<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>Louisa Temme &#8211; openlabnotebooks.org</title>
	<atom:link href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/louisa-temme/feed/" rel="self" type="application/rss+xml" />
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<description>A growing team of groundbreaking scientists around the world are now sharing their lab notebooks online</description>
	<lastBuildDate>Mon, 13 May 2019 13:19:13 +0000</lastBuildDate>
	<language>en-GB</language>
	<sy:updatePeriod>hourly</sy:updatePeriod>
	<sy:updateFrequency>1</sy:updateFrequency>
	<generator>https://wordpress.org/?v=4.9.10</generator>

<image>
	<url>https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/cropped-lab-jar02-32x32.png</url>
	<title>Louisa Temme &#8211; openlabnotebooks.org</title>
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<width>32</width>
	<height>32</height>
</image> 
	<item>
		<title>Growing co-crystallized fragments to ligands of the transcription factor brachyury (T)</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/growing-co-crystallized-fragments-to-ligands-of-the-transcription-factor-brachyury-t/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/growing-co-crystallized-fragments-to-ligands-of-the-transcription-factor-brachyury-t/#respond</comments>
		<pubDate>Wed, 24 Apr 2019 21:15:14 +0000</pubDate>
		<dc:creator><![CDATA[Louisa Temme]]></dc:creator>
				<category><![CDATA[Chordoma]]></category>
		<category><![CDATA[Louisa Temme]]></category>
		<category><![CDATA[Understudied Kinases]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2758</guid>
		<description><![CDATA[In order to develop ligands addressing different possible binding sites of the transcription factor brachyury, fragments co-crystallized with this protein should be developed into ligands by using the program SparkTM (Cresset®). Brachyury (T, TBXT) is an essential regulator of the notochord development. Brachyury is primarily expressed in the embryo but not in the majority of normal <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/growing-co-crystallized-fragments-to-ligands-of-the-transcription-factor-brachyury-t/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>In order to develop ligands addressing different possible binding sites of the transcription factor brachyury, fragments co-crystallized with this protein should be developed into ligands by using the program Spark<sup>TM </sup>(Cresset<sup><span style="font-size: 11.25px;">®</span></sup>).</p>
<p>Brachyury (T, TBXT) is an essential regulator of the notochord development. Brachyury is primarily expressed in the embryo but not in the majority of normal adult tissue. It is known that brachyury is overexpressed in chordoma, a primary bone cancer, and plays a role in the disease pathogenesis. Chordoma affects the skull-base, the mobile spine and the sacrum. It is a slow-growing, malign cancer that is locally invasive and has a tendency to recur although a surgical and/or radiation therapy was conducted.<sup>1</sup></p>
<p>It is hypothesized that chordoma is originated from embryonic notochordal remnants. Brachyury is proposed to act as an aberrantly activated transcription factor and thereby be oncogenic. No approved targeted therapies, chemotherapies or immunotherapies are available for chordoma and no small-molecule inhibitors for brachyury have been developed.<sup>1 </sup>Therefore, a fragment-based approach should be explored to develop ligands for brachyury to study this protein.</p>
<p>Our colleagues Joseph Newman and Opher Gileadi at SGC Oxford co-crystallized wild-type human brachyury protein with various fragments which bind to the sites A-E (Figure 1, left). Thankfully, they provided the fragment and protein structures to us so that we can explore the usage of the Spark<sup>TM </sup>growing approach to develop ligands for these binding sites.</p>
<p><img class="wp-image-2759 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Picture1-300x121.png" alt="" width="592" height="239" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Picture1-300x121.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Picture1-768x310.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Picture1.png 974w" sizes="(max-width: 592px) 100vw, 592px" />Figure 1. Left: Aligned X-ray structures of the human wild-type brachyury protein in complex with fragments at different binding sites (A-E). These first crystallization structures contain Cd<sup>2+</sup>ions (green balls). Right: Binding site A close to loop with D177 and in proximity to residue Y88.<sup>2</sup></p>
<p>First, we focused on binding site A since most fragments bind to this binding site and site A is close to a loop that is mutated in chordoma-linked variant G177D<sup>3 </sup>and to the amino acid residue Y88 that, if mutated, affects binding to effector proteins<sup>4 </sup>(Figure 1, left).</p>
<p><img class="wp-image-2761 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Picture2-300x86.png" alt="" width="607" height="174" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Picture2-300x86.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Picture2-768x220.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Picture2.png 974w" sizes="(max-width: 607px) 100vw, 607px" /></p>
<p>Figure 2. Fragment <strong>1 </strong>and <strong>2 </strong>co-crystallized in the binding site A of wild-type TBXT.</p>
<p>21 fragments were co-crystallized to binding site A. In Figure 2, the binding modes of two co-crystallized fragments are displayed. Fragments <strong>1 </strong>and <strong>2 </strong>show interactions or close proximity to two of the three amino acids Ser129, Ser89 and Arg180. Therefore, the growing approach of the program Spark<sup>TM </sup>should be used to identify new ligands that are able to form all three interactions. One growing approach started from the amino group of fragment <strong>1 </strong>and was guided by fragment <strong>2 </strong>towards the region of its carboxylate group (Figure 3).</p>
<p><img class="wp-image-2763 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Picture3-300x171.png" alt="" width="484" height="276" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Picture3-300x171.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Picture3-768x438.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Picture3-780x445.png 780w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Picture3.png 781w" sizes="(max-width: 484px) 100vw, 484px" /></p>
<p>Figure 3: Spark<sup>TM </sup>growing approach starting from fragment <strong>1 </strong>towards Arg180 guided by fragment <strong>2</strong>. The functional groups of fragments <strong>1 </strong>and <strong>2 </strong>revealing H-bond interactions with amino acid residues are marked with the following colors: Ser129 (red), Ser89 (blue), Arg180 (green).</p>
<p>To achieve new ligands with similar properties in certain areas so-called field points were used. They are ‘condensed representations of electrostatic, hydrophobic and shape properties’ (Cresset<sup><span style="font-size: 11.25px;">®</span></sup>) of those areas where they are placed by the program Spark<sup>TM</sup>. Regarding the growing approach of fragment <strong>1 </strong>and <strong>2 </strong>the hydrophobic field point of the phenyl ring and the negative field point of the carboxylate group of fragment <strong>2 </strong>were used. In total three of the 21 fragments bound to site A were used as starting points and each of them were grown in the area of two different fragments. The results were clustered by Spark<sup>TM </sup>depending on their structural similarity. Those compounds with the highest score of the first 20 clusters ranked according their scores were chosen to analyze their interactions by docking into binding site A using the docking program Flare<sup>TM </sup>(Cresset<sup><span style="font-size: 11.25px;">®</span></sup>). 4,000 docking results were inspected visually and compounds with a high docking score, high electrostatic complementarity and high number of interactions to the binding site were picked.</p>
<p>Figure 4 shows the docking pose of the example compound <strong>3 </strong>that was obtained by docking <strong>3 </strong>in binding site A of brachyury using Flare<sup>TM</sup>. Besides other criteria like docking score and electrostatic complementarity following interactions of <strong>3 </strong>are promising for the co-crystallization of this compound with brachyury: two H-bond interactions of the positively charged amino moiety to Ser129 and His126, the proximity of the amide carbonyl-group to Ser89 and the H-bond interactions of the oxazole O-atom and N-atom towards Arg180.</p>
<p><img class="wp-image-2765 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Picture4-300x232.png" alt="" width="454" height="351" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Picture4-300x232.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Picture4.png 633w" sizes="(max-width: 454px) 100vw, 454px" />Figure 4. Binding mode of new compound <strong>3 </strong>obtained by docking into binding site A of wild-type brachyury with Flare<sup>TM</sup>. H-bond interactions are marked with green dotted lines.</p>
<p>Additionally, new compounds were obtained by searching in the Enamine REAL database for similar compounds to those compounds that were found by the Spark<sup>TM </sup>approach and that showed promising docking results in Flare<sup>TM</sup>.</p>
<p>A selection of compounds that are identical or slightly different than those found in the Spark<sup>TM </sup>approach as well as compounds that represent similar ones to the Spark<sup>TM </sup>hits were purchased and will be tested regarding their ability to co-crystallize with brachyury. If the compounds bind to this transcription factor, they will represent promising starting points for future ligands of brachyury.</p>
<p>&nbsp;</p>
<p>(1) Sharifnia, T., <em> Med. (N. Y., NY, U. S.)</em>, <strong>2019</strong>, <em>25</em>, 292-300.</p>
<p>(2) Joseph Newman and Opher Gileadi, SGC Oxford, University of Oxford, UK.</p>
<p>(3) Pillay, N., <em>Nat. Genet.</em>, <strong>2012</strong>, <em>44</em>, 1185-1187.</p>
<p>(4) Beisaw, A., <em>EMBO reports</em>, <strong>2018</strong>, <em>19</em>, 118-134.</p>
<p>&nbsp;</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/growing-co-crystallized-fragments-to-ligands-of-the-transcription-factor-brachyury-t/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Selective CDKL2 inhibitors &#8211; molecular modeling</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/selective-cdkl2-inhibitors-molecular-modeling/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/selective-cdkl2-inhibitors-molecular-modeling/#respond</comments>
		<pubDate>Fri, 21 Dec 2018 02:02:27 +0000</pubDate>
		<dc:creator><![CDATA[Louisa Temme]]></dc:creator>
				<category><![CDATA[Louisa Temme]]></category>
		<category><![CDATA[Understudied Kinases]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2325</guid>
		<description><![CDATA[At SGC we are interested in developing protein kinase inhibitors with a high selectivity towards other protein kinases. Based on lead compounds from the literature and our group new and selective CDKL2 inhibitors should be designed, synthesized and tested. Carla Alamillo, a former member of our group, worked on this topic and mentioned in her <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/selective-cdkl2-inhibitors-molecular-modeling/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p style="text-align: justify;">At SGC we are interested in developing protein kinase inhibitors with a high selectivity towards other protein kinases. Based on lead compounds from the literature and our group new and selective CDKL2 inhibitors should be designed, synthesized and tested. Carla Alamillo, a former member of our group, worked on this topic and mentioned in her last entry in the open notebook about this topic  (<a href="https://thesgc.github.io/static-openlabnotebooks/sns-032-analogues-update/">https://thesgc.github.io/static-openlabnotebooks/sns-032-analogues-update/</a>) the pro-mising role of SNS-032 as a starting point for new protein kinase inhibitors (Figure 1).<br />
SNS-032 shows a low K<sub>d</sub>value for CDKL2 (48 nM) but also for many other kinases. This means the selectivity of this compound towards other protein kinases is low. In contrast, CAF-181 synthesized by Carla (compound 11 in Carla’s entry) shows low K<sub>d</sub>values for CDKL2 and CDK7 and a high selectivity towards the other kinases tested. Additionally, CAF-181 reveals a moderate CDKL2 IC<sub>50</sub>value of 417 nM. Therefore, CAF-181 represents a good lead compound for the development of new CDKL2 inhibitors with improved selectivity towards CDK7.</p>
<p style="text-align: justify;"><img class="wp-image-2327 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/122018_Picture1-1-300x174.png" alt="" width="694" height="402" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/122018_Picture1-1-300x174.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/122018_Picture1-1.png 668w" sizes="(max-width: 694px) 100vw, 694px" /></p>
<p style="text-align: justify;"><strong>Figure 1. </strong>Lead compounds SNS-032 and CAF-181 and their respective K<sub>d </sub>values.</p>
<p style="text-align: justify;">To design new compounds the molecular modeling software Flare<sup>TM</sup>(Cresset<sup>®</sup>) was used. The first step for the <em>in silico </em>docking is the selection of the crystal structure of the respective protein. Two crystal structures of CDKL2 are available on the PDB website (<a href="https://www.rcsb.org/">https://www.rcsb.org</a>) but not published in a journal yet. One crystal structure is co-crystallized with the compound TCS 2312 (PDB-ID 4BBM) and the other with the compound DKI (PDB-ID 4AAA) (Figure 2). The 2D interaction maps (<a href="https://www.rcsb.org/structure/4AAA">https://www.rcsb.org/structure/4AAA</a> and <a href="https://www.rcsb.org/structure/4BBM">https://www.rcsb.org/structure/4BBM</a>) have to be flipped vertically to match the same amino acid residues. Interestingly, Val83 (Figure 2, blue box) seems to be an impor-tant interaction partner for the ligands since both compounds reveal two H-bond interactions with Val83.</p>
<p style="text-align: justify;"><img class="wp-image-2328 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/122018_Picture2-300x181.png" alt="" width="629" height="380" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/122018_Picture2-300x181.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/122018_Picture2.png 600w" sizes="(max-width: 629px) 100vw, 629px" /></p>
<p style="text-align: justify;"><strong>Figure 2. </strong>Crystal structures of CDKL2: co-crystallized with TCS 2312 (A, PDB-ID 4BBM, DOI: 10.2210/pdb4BBM/pdb) and co-crystallized with DKI (B, PDB-ID 4AAA, DOI: 10.2210/pdb4AAA/pdb). Black dashed lines indicate hydrogen bonds, salt bridges, and metal interactions. Green solid lines show hydrophobic interactions and green dashed lines show π-π and π-cation interactions (<a href="https://www.rcsb.org/structure/4BBM">https://www.rcsb.org/structure/4BBM</a> and <a href="https://www.rcsb.org/structure/4AAA">https://www.rcsb.org/structure/4AAA</a>).</p>
<p style="text-align: justify;">During the <em>in silico </em>docking studies the crystal structure 4BBM was used because the structure 4AAA reveals an unresolved amino acid sequence ending with Ala148 and NME148A (Figure 3B, black box). Unfortunately, this sequence part also contains the amino acid Phe147 that forms a part of the active site and likely interacts with inhibitors. However, the protein structure used for the docking should contain a fully resolved amino acid sequence in this area. The protein structure 4BBM provides a fully resolved amino acid sequence which makes it suitable for <em>in silico </em>docking.</p>
<p style="text-align: justify;"><img class="wp-image-2329 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/122018_Picture3-300x81.png" alt="" width="870" height="235" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/122018_Picture3-300x81.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/122018_Picture3-768x208.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/122018_Picture3.png 974w" sizes="(max-width: 870px) 100vw, 870px" /></p>
<p style="text-align: justify;"><strong>Figure 3. </strong>Structure of CDKL2 co-crystallized with either TCS 2312 (orange, A, PDB-ID 4BBM) or DKI (dark red, B, PDB-ID 4AAA). The dashed green and orange lines indicate H-bond interactions and the purple lines CH-π interactions. Docked ligands are shown as sticks. Oxygen and nitrogen atoms are colored in red and blue, respectively.</p>
<p style="text-align: justify;">However, Phe147 in structure 4BBM is not clarified. Instead of the phenyl ring a methyl group is visible. Capping by the docking program Flare<sup>TM</sup> did not result in phenylalanine but just in a partially aromatic phenyl ring of this amino acid (Figure 3A, black box). This problem was solved by mutation of this position into a phenylalanine.</p>
<p style="text-align: justify;">To investigate the selectivity of newly designed compounds they should also be investigated during the <em>in silico </em>docking into the crystal structure of CDK7 with the PDB-ID 1UA2.</p>
<p style="text-align: justify;">The next step of <em>in silico</em> docking into the CDKL2 structure 4BBM included the validation of the crystal structure and the docking protocol. After preparation and minimization of the crystal structure the co-crystallized ligand TCS 2312 should be redocked into the active site. However, the co-crystallized ligand TCS 2312 shows different bond orders/tautomeric states as well as a different protonation state on the basic nitrogen than the literature compound TCS 2312. To calculate the RMSD value between the binding mode of the structurally corrected and redocked TCS 2312 and the co-crystallized ligand the co-crystallized binding mode had to be corrected because Flare<sup>TM</sup> can only calculate a RMSD value between structurally identical compounds. Therefore, the co-crystallized ligand was corrected and only the resorcinol moiety was minimized to achieve a planar aromatic system. In this way the binding mode of the co-crystallized ligand was changed as little as possible. Then, the corrected, co-crystallized ligand TCS  2312 was minimized and the structure at physiological pH was generated. Afterwards, this compound was docked into the crystal structure 4BBM. Figure 4A shows the similar orientation of corrected, co-crystallized TCS 2312 (yellow) and the redocked ligand (purple). The similar orientation of both binding modes and the hereby resulting low RMSD value<br />
(1.65 Å) suggest that the corrected CDKL2 protein crystal structure 4BBM and the docking protocol are suitable for the <em>in silico </em>docking of new compounds.</p>
<p style="text-align: justify;"><img class="wp-image-2330 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/122018_Picture4-300x181.png" alt="" width="762" height="460" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/122018_Picture4-300x181.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/122018_Picture4-768x463.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/122018_Picture4.png 943w" sizes="(max-width: 762px) 100vw, 762px" /></p>
<p style="text-align: justify;"><strong>Figure 4. </strong>Predicted binding modes of TCS 2312 (A, purple), CAF-181 (B) and SNS-032 (C and D) obtained by docking into the CDKL2 crystal structure with the PDB-ID 4BBM.Binding mode of the corrected, co-crystallized ligand TCS 2312 (A, yellow). The dashed green and orange lines indicate H-bond interactions and the purple lines CH-π interactions. Docked ligands are shown as sticks. Oxygen and nitrogen atoms are colored in red and blue, respectively.</p>
<p>&nbsp;</p>
<p style="text-align: justify;">In the Figures 4B-D the docking results of CAF-181 and SNS-032 are shown. The <em>tert</em>-butyl group of both compounds occupies the inner part of the active site whereas the piperazine ring of CAF-181 and the piperidine of SNS 032 point towards the solvent accessible area. The N-atom of the SNS-032 piperidine ring is part of an H-bond interaction with one of the carboxylic acid moieties of either Glu130 (C) or Asp89 (D). Therefore, it might be promising to introduce amide substituents that can interact with those acidic amino acids.</p>
<p style="text-align: justify;">CAF compounds, synthesized and tested previously in our group, were docked into the CDKL2 crystal structure 4BBM and in the CDK7 crystal structure 1UA2. The respective docking scores and electrostatic complementarity<sup>TM </sup>scores of the resulting docking poses reveal a good correlation to the K<sub>d </sub>values of those compounds regarding CDKL2. However, no clear correlation was found regarding docking of CAF compounds in CDK7. This observation could be explained with the larger active site of CDK7 and its many solvent accessible areas that could facilitate binding of structurally diverse compounds.</p>
<p style="text-align: justify;">Interestingly, <em>in silico</em> docking of SNS-032 and CAF-181 in the active site of CDK7 reveal  curved orientations of both compounds (Figure 5). The green color of the electrostatic complementarity<sup>TM </sup>surface of the SNS-032 binding mode in Figure 5C indicates a high electrostatic complementarity<sup>TM </sup>of the ligand to this active site which could be the reason for the high inhibitory activity of SNS-032 regarding CDK7.</p>
<p style="text-align: justify;"><em><img class="wp-image-2331 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/122018_Picture5-300x180.png" alt="" width="751" height="451" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/122018_Picture5-300x180.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/122018_Picture5-768x461.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/122018_Picture5.png 962w" sizes="(max-width: 751px) 100vw, 751px" /></em></p>
<p style="text-align: justify;"><strong>Figure 5. </strong>Predicted binding modes obtained by docking into the CDK7 crystal structure with the PDB-ID 1UA2:<br />
(A) SNS-032. (B) SNS-032 with the electrostatic potential surface of CDK7. (C) SNS-032 with its electrostatic complementarity<sup>TM </sup>surface. (D) CAF-181. The dashed green and orange lines indicate H-bond interactions and the purple lines CH-π interactions. Docked ligands are shown as sticks. Oxygen and nitrogen atoms are colored in red and blue, respectively.</p>
<p style="text-align: justify;">Based on this observation new CDKL2 inhibitors should reveal restrained flexibility to avoid the curved orientation in CDK7 and thereby increase the selectivity towards CDK7. Among others this could be achieved by replacement of the flexible thio-methylene linker by a more rigid linker. In my next entry I will show the docking results of more rigid compounds obtained by both <em>in silico </em>docking into the crystal structure of CDKL2 and CDK7.</p>
<p style="text-align: justify;">Please feel free to suggest other ideas for CDKL2 inhibitors with increased selectivity towards CDK7. If you have any questions or comments, please let me know.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/selective-cdkl2-inhibitors-molecular-modeling/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
	</channel>
</rss>
