<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>Scientist &#8211; openlabnotebooks.org</title>
	<atom:link href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/feed/" rel="self" type="application/rss+xml" />
	<link>http://thesgc.github.io/static-openlabnotebooks/</link>
	<description>A growing team of groundbreaking scientists around the world are now sharing their lab notebooks online</description>
	<lastBuildDate>Thu, 31 Jan 2019 23:10:29 +0000</lastBuildDate>
	<language>en-GB</language>
	<sy:updatePeriod>hourly</sy:updatePeriod>
	<sy:updateFrequency>1</sy:updateFrequency>
	<generator>https://wordpress.org/?v=4.9.9</generator>

<image>
	<url>https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/cropped-lab-jar02-32x32.png</url>
	<title>Scientist &#8211; openlabnotebooks.org</title>
	<link>http://thesgc.github.io/static-openlabnotebooks/</link>
	<width>32</width>
	<height>32</height>
</image> 
	<item>
		<title>Welcome to the SGC-Karolinska side of ALS-RAP: the scientists, the screening, and the antibody selections</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/welcome-to-the-sgc-karolinska-side-of-als-rap-the-scientists-the-screening-and-the-antibody-selections/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/welcome-to-the-sgc-karolinska-side-of-als-rap-the-scientists-the-screening-and-the-antibody-selections/#respond</comments>
		<pubDate>Thu, 31 Jan 2019 23:10:29 +0000</pubDate>
		<dc:creator><![CDATA[Carolyn Marks]]></dc:creator>
				<category><![CDATA[ALS-Reproducible Antibody Platform (ALS-RAP)]]></category>
		<category><![CDATA[Amyotrophic Lateral Sclerosis (ALS)]]></category>
		<category><![CDATA[Carolyn Marks]]></category>

		<guid isPermaLink="false">https://openlabnotebooks.org/?p=2419</guid>
		<description><![CDATA[Hello fellow scientists and open lab notebook readers! I’m Carolyn Marks and I’m working in the ALS-RAP partnership with Susanne Gräslund, who leads the Recombinant Antibodies Group at SGC-Karolinska in Stockholm, Sweden. SGC-Karolinska has had a long-standing collaboration with Helena Persson Lotsholm, Director of the Human Antibody Therapeutics Facility within the Drug Discovery and Development <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/welcome-to-the-sgc-karolinska-side-of-als-rap-the-scientists-the-screening-and-the-antibody-selections/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p style="text-align: justify;">Hello fellow scientists and open lab notebook readers!</p>
<p style="text-align: justify;">I’m Carolyn Marks and I’m working in the ALS-RAP partnership with <a href="https://thesgc.org/profile/karolinska/sgraslund">Susanne Gräslund</a>, who leads the Recombinant Antibodies Group at <a href="https://www.thesgc.org/karolinska">SGC-Karolinska</a> in Stockholm, Sweden. SGC-Karolinska has had a long-standing collaboration with <a href="https://www.scilifelab.se/facilities/human-antibody-therapeutics/">Helena Persson Lotsholm</a>, Director of the Human Antibody Therapeutics Facility within the <a href="https://www.scilifelab.se/platforms/ddd">Drug Discovery and Development Platform at Science for Life Laboratory in Stockholm</a>. I am fortunate to work with and learn from several experts to generate the highest quality antibodies for the ALS-Reproducible Antibody Platform. I joined the <a href="https://www.als-rap.org/team">ALS-RAP team</a> in September of last year and from here on out I’ll be sharing live updates of our work with this community.</p>
<p style="text-align: justify;">Perhaps I should tell you a little bit about me, what we do at SGC-Karolinska, and how I got here. I’m a wet lab kind of gal and a neuroscientist at heart with a lot of years of experience under my belt. The methods that I use here at SGC-Karolinska for the ALS-RAP are broad and range from phage display technology to cloning to high-throughput screening to patient-derived cell-based assays. Actually, many of the methods that I&#8217;m using now to generate and validate recombinant antibodies for the ALS-RAP, I used for many years to study neurodegenerative diseases, including ALS.</p>
<p style="text-align: justify;">I´m originally from Texas where my love of neurophysiology began, and where I was initially trained as an electrophysiologist. From there, I deferred medical school and took an internship at NINDS at the NIH to have a break before med school began. My 1-year internship turned into two, and I quickly understood why everyone at NIH kept telling me “Carolyn, you reek of a PhD!” I soon found the NIH-Karolinska Graduate Partnerships Program and hopped on a plane. That&#8217;s how the girl from Texas ended up in Sweden. I received my PhD in 2012 from the NIH-Karolinska Collaborative Doctoral Program in Neuroscience; and I recently completed my Post-Doc, also at Karolinska, characterizing the ubiquitin proteasome system as therapeutic targets in neurodegenerative diseases.</p>
<p style="text-align: justify;">How did I find my way to the SGC, you might wonder? In my research over the years, I have experienced first-hand the need for reliable high-quality antibodies – and the lack thereof, as well. I hate that we as scientists are limited by the tools that companies provide. These limitations are crippling our research. Sadly, the ones who suffer the most are the patients. This was my motivation for joining SGC-Karolinska and particularly this project. I wanted to make better tools for the research community. Consequently, I believe that better research tools will yield better science and hopefully better therapeutics for the patients who need it. That&#8217;s why I&#8217;m here.</p>
<p style="text-align: justify;">How does the <a href="https://www.als-rap.org/overview">ALS-RAP</a> work? Our team at Karolinska is only 1 of many. We work together primarily with scientists at <a href="https://openlabnotebooks.org/introducing-the-team-the-gileadi-group-in-oxford/">Oxford</a> and in <a href="https://www.mcgill.ca/neuro/research/researchers/thomas-m-durcan-phd">Montreal</a>. The ALS-RAP was created by three major ALS charities from Canada, the UK, and the US. The long-term goal of this partnership is to provide the ALS research community with reproducible, thoroughly validated antibodies for selected ALS-related proteins. This includes testing existing commercial antibodies as well as generating new ones. The work is done in three different labs, as recently outlined by my fellow ALS-RAP partner and open-lab-notebooker, <a href="https://openlabnotebooks.org/validated-antibodies-for-als-research-the-als-rap-project/">Opher Gileadi</a>.</p>
<p style="text-align: justify;">In the Stockholm team, we use phage display technology to create recombinant antibodies. Don´t know what phage display is? The inventors of this technique were actually awarded the <a href="https://www.nobelprize.org/prizes/chemistry/2018/summary/">Nobel Prize in Chemistry in 2018</a>. While this technology has many uses, we use phage display to generate recombinant antibodies. How does it work you might wonder? When we receive antigens of our ALS proteins from our friends in <a href="https://www.ndm.ox.ac.uk/principal-investigators/researcher/opher-gileadi">Oxford</a>, we then use the phage display technique to select out antibodies that binds these antigens. We use extensive libraries of human antibody candidates where high-quality binders can be obtained with the phage display method. We then perform several rounds of screening and selection to ensure that we get the best possible antibodies. We then evaluate their specificity and affinity and finally we validate the antibodies using immunoprecipitation followed by mass spectrometry. It sounds simple, but trust me it’s a thorough process! Last but not least, we will release the sequence of the best antibody for each target on our website and make it available to anyone who wants to use it. Our collaborators in Montreal are creating CRISPR-knockouts of these targets in both standard cell lines and iPSCs for further validation and application specificity.</p>
<p style="text-align: justify;">We have a really exciting <a href="https://als-rap.org/target-list">target list</a>, built by the help of our ALS-RAP advisory board. Check it out and see if your favorite protein or that much needed antibody is on our list. If not, click <a href="https://als-rap.org/your-input">here</a> and request it. So far, we have generated and selected our first set of antibodies for 7 of these targets. Now onto primary and secondary validation.</p>
<p style="text-align: justify;">Stay tuned to find out who the lucky 7 are…</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/welcome-to-the-sgc-karolinska-side-of-als-rap-the-scientists-the-screening-and-the-antibody-selections/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Testing Hit Analogues against USP5 Zf-UBD with SPR Assay #1</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/testing-hit-analogues-against-usp5-zf-ubd-with-spr-assay-1/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/testing-hit-analogues-against-usp5-zf-ubd-with-spr-assay-1/#respond</comments>
		<pubDate>Thu, 24 Jan 2019 15:34:40 +0000</pubDate>
		<dc:creator><![CDATA[Mandeep Mann]]></dc:creator>
				<category><![CDATA[Mandeep Mann]]></category>
		<category><![CDATA[USP5]]></category>

		<guid isPermaLink="false">https://openlabnotebooks.org/?p=2393</guid>
		<description><![CDATA[In a previous post, I described using hit expansion tools such as commercial substructure searches, docking, and free energy perturbation (FEP) simulations to explore the structure activity relationship (SAR) of the chemical series focused on the Zf-UBD of USP5. The first batch of commercially ordered compounds arrived just before the Christmas holidays. I decided to <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/testing-hit-analogues-against-usp5-zf-ubd-with-spr-assay-1/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>In a <a href="https://openlabnotebooks.org/hit-expansion-using-substructure-search-free-energy-perturbation/">previous post</a>, I described using hit expansion tools such as commercial substructure searches, docking, and free energy perturbation (FEP) simulations to explore the structure activity relationship (SAR) of the chemical series focused on the Zf-UBD of USP5. The first batch of commercially ordered compounds arrived just before the Christmas holidays. I decided to tackle testing the chemical analogues in the New Year. I used a surface plasmon resonance (SPR) assay to determine binding affinities of commercially ordered analogues, designed from compounds hits listed in Table 1. You can see details on <a href="https://zenodo.org/record/2548710#.XEnZclxKjIU">Zenodo</a><strong>. </strong></p>
<p style="text-align: left;"><strong>Table 1</strong>. Compound Hits from which chemical analogues were designed</p>
<p><img class="wp-image-2394 alignleft" src="https://openlabnotebooks.org/wp-content/uploads/2019/01/20190124a-300x153.png" alt="" width="451" height="231" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/20190124a-300x153.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/20190124a.png 678w" sizes="(max-width: 451px) 100vw, 451px" /></p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p>Of the 19 analogues tested, compound AE-641/11456811, an analogue of DAT194 (K<sub>D</sub>= 215 µM; n=9) had a binding affinity of approximately 65 µM (n=6) and an excellent ligand efficiency of 0.36 (Figure 1). The addition of a methyl group increased potency 3-fold!</p>
<p><img class=" wp-image-2395 aligncenter" src="https://openlabnotebooks.org/wp-content/uploads/2019/01/20190125b-300x83.png" alt="" width="473" height="131" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/20190125b-300x83.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/20190125b-768x212.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/20190125b-780x218.png 780w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/20190125b.png 788w" sizes="(max-width: 473px) 100vw, 473px" /></p>
<p style="text-align: center;">Figure 1. A) Compound DAT194 B) Compound AE-641/11456811</p>
<p>Compound DAT194 (PDB: 6NFT) and the docked pose of AE-641/11456811 are shown in Figure 2a. The addition of the methyl group leads to increased hydrophobic interactions in the binding pocket with W209. In Figure 2b, the docked pose of AE-641/11456811 shows the methyl group fits into a small groove in the pocket. Based on the docked pose, it may be possible to further extend the non-polar group into the pocket to increase the potency further. For this reason, compounds in Figure 3 were ordered as follow up compounds of AE-641/11456811.</p>
<p><img class="wp-image-2396 aligncenter" src="https://openlabnotebooks.org/wp-content/uploads/2019/01/20190124c-300x101.png" alt="" width="529" height="178" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/20190124c-300x101.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/20190124c-768x258.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/20190124c.png 900w" sizes="(max-width: 529px) 100vw, 529px" /></p>
<p style="text-align: center;"><strong>Figure 2.</strong> A) Compound DAT194 (PDB: 6NFT) in magenta and docked pose of AE-641/11456811 (yellow) B) Binding pocket of USP5 Zf-UBD and docked pose AE-641/11456811 (yellow)</p>
<p><img class=" wp-image-2397 aligncenter" src="https://openlabnotebooks.org/wp-content/uploads/2019/01/20190124d-300x181.png" alt="" width="290" height="175" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/20190124d-300x181.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/20190124d.png 708w" sizes="(max-width: 290px) 100vw, 290px" /></p>
<p style="text-align: center;"><strong>Figure 3</strong>. Follow up compounds of AE-641/11456811</p>
<p>Analogues of DAT180 and DAT201 that were tested did not show a significant improvement in potency. So far, I’ve tested 19 of the 39 analogue compounds of DAT180, DAT194 and DAT201 that were selected in the hit expansion. Some compounds required custom synthesis and were not ordered, and others are still in transit. I will be testing the rest of the analogues once the next batch of compounds arrive, as well as testing follow up compounds of AE-641/11456811. I’ll also be setting up co-crystal trays of USP5 Zf-UBD and AE-641/11456811 for structure determination. Stay tuned!</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/testing-hit-analogues-against-usp5-zf-ubd-with-spr-assay-1/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Optimising seeding of normal and FOP cells for a later assay &#8211; prettier than expected</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/optimising-seeding-of-normal-and-fop-cells-for-a-later-assay-prettier-than-expected/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/optimising-seeding-of-normal-and-fop-cells-for-a-later-assay-prettier-than-expected/#respond</comments>
		<pubDate>Tue, 22 Jan 2019 19:22:08 +0000</pubDate>
		<dc:creator><![CDATA[Elizabeth Brown]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Brain Tumours]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>
		<category><![CDATA[Liz Brown]]></category>
		<category><![CDATA[Understudied Kinases]]></category>

		<guid isPermaLink="false">https://openlabnotebooks.org/?p=2388</guid>
		<description><![CDATA[Further down the line I want to be able to test the activation of the BMP signalling pathway downstream of the ALK2 receptor. For this I plan to use the alkaline phosphatase gene as a readout &#8211; it&#8217;s a well known gene activated downstream of BMP signalling so a lot of easy to use assays <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/optimising-seeding-of-normal-and-fop-cells-for-a-later-assay-prettier-than-expected/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Further down the line I want to be able to test the activation of the BMP signalling pathway downstream of the ALK2 receptor. For this I plan to use the alkaline phosphatase gene as a readout &#8211; it&#8217;s a well known gene activated downstream of BMP signalling so a lot of easy to use assays already exist for it.</p>
<p>Unfortunately <a href="https://openlabnotebooks.org/optimising-an-assay-to-detect-alkaline-phosphatase-activity-in-c2c12-cells/">when I tested this assay with the standard C2C12 cell line it didn&#8217;t work particularly well</a> so I&#8217;ve decided to try again with a new set of cells. This time I&#8217;m choosing connective tissue cells (fibroblasts) from FOP patients as a reporter cell line because they respond more strongly to the signals that activate the BMP pathway (called BMP ligands).</p>
<p>I&#8217;ve never worked with these cells before so, again, I needed to work out how many I&#8217;ll need to get a good result from the assay. These cells however grow as a very thin sheet which is so see-through that I initially couldn&#8217;t tell if the wells were full. To solve this I used stains called Calcein AM and Hoechst. Calcein AM can stain any living cell because they metabolise it from a see-through molecule into a fluorescent green product, whereas Hoechst can stick to DNA and turn all the cell nuclei blue, and so suddenly, all of my cells glow:</p>
<div id="attachment_2389" style="width: 1382px" class="wp-caption aligncenter"><a href="https://openlabnotebooks.org/wp-content/uploads/2019/01/Calcein-AM-example.png"><img class="wp-image-2389 size-full" src="https://openlabnotebooks.org/wp-content/uploads/2019/01/Calcein-AM-example.png" alt="Fibroblast cells stained with Calcein AM (green) and Hoechst (blue)" width="1372" height="784" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/Calcein-AM-example.png 1372w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/Calcein-AM-example-300x171.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/Calcein-AM-example-768x439.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/Calcein-AM-example-1024x585.png 1024w" sizes="(max-width: 1372px) 100vw, 1372px" /></a><p class="wp-caption-text">Fibroblast cells stained with Calcein AM (green) and Hoechst (blue)</p></div>
<p>Aren&#8217;t they pretty?</p>
<p>Once they&#8217;re this easy to see, it&#8217;s even easier to see how many cells are needed to fill a well, but not get overcrowded. You can read more about how I did that in my <a href="https://zenodo.org/record/2546911">Zenodo post</a>.</p>
<p>The only question I have left is why some cells didn&#8217;t stain with Calcein AM, even though their nucleus looks fine? Anyone know the answer?</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/optimising-seeding-of-normal-and-fop-cells-for-a-later-assay-prettier-than-expected/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>More crystals through XChem</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/more-crystals-through-xchem/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/more-crystals-through-xchem/#respond</comments>
		<pubDate>Thu, 10 Jan 2019 11:54:08 +0000</pubDate>
		<dc:creator><![CDATA[Jose Brandao-Neto]]></dc:creator>
				<category><![CDATA[Jose Brandao-Neto]]></category>
		<category><![CDATA[XChem]]></category>

		<guid isPermaLink="false">https://openlabnotebooks.org/?p=2372</guid>
		<description><![CDATA[Following an early commissioning stage, XChem crystallised its core xray collection protocol as 180 degrees rotation in 60 seconds, which provides PanDDA with good datasets. But what has never been fully characterised is the lowest dose per crystal required for PanDDA to work. XChem now has an established hit rate profile where, based on our <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/more-crystals-through-xchem/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Following an early commissioning stage, XChem crystallised its core xray collection protocol as 180 degrees rotation in 60 seconds, which provides PanDDA with good datasets. But what has never been fully characterised is the lowest dose per crystal required for PanDDA to work. XChem now has an established hit rate profile where, based on our core xray protocol, we expect between 8 and 40 fragment hits out of the standrad supply screen of 400 fragments from the DSIP library (2% to 10% hit rate using the Diamond, SGC and Enamine fragment library poised for follow up [cox et al 2017]).</p>
<p>In a short series of posts I will tell my journey to arrive at the first round of xray dose optimisation, notably to reduce the core expriment to 30s per 180 degree rotation, currently (December 2018) at its first pilot runs as a protocol after 3 rounds of characterisation experiments.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/more-crystals-through-xchem/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>A new ALK2 structure, and proof that mistakes in the lab aren’t always a bad thing</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/a-new-alk2-structure-and-proof-that-mistakes-in-the-lab-arent-always-a-bad-thing/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/a-new-alk2-structure-and-proof-that-mistakes-in-the-lab-arent-always-a-bad-thing/#respond</comments>
		<pubDate>Fri, 04 Jan 2019 15:28:47 +0000</pubDate>
		<dc:creator><![CDATA[Roslin Adamson]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Brain Tumours]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>
		<category><![CDATA[Roslin Adamson]]></category>

		<guid isPermaLink="false">https://openlabnotebooks.org/?p=2364</guid>
		<description><![CDATA[Hello everyone and happy New Year! I really, really wanted to end the year on a high note in December before I went off on holiday, but as it turned out, I didn’t have enough time to set up 18 plates and write a blog post on the last day. So I’ll just have to <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/a-new-alk2-structure-and-proof-that-mistakes-in-the-lab-arent-always-a-bad-thing/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Hello everyone and happy New Year! I really, really wanted to end the year on a high note in December before I went off on holiday, but as it turned out, I didn’t have enough time to set up 18 plates and write a blog post on the last day. So I’ll just have to start the year on a high note.</p>
<p>I’m very happy to say I got another ACVR1/ALK2 structure in December, at 2 Å resolution, co-crystallised with M4K2158 (see images below, and the .mtz autoprocessed file is available <a href="https://zenodo.org/record/2531714#.XC94jlz7SUk">here</a> on Zenodo). The structure is not fully refined yet, but what’s interesting is the solvent side of the molecule is being stabilised by either a water contact (Chain A) or Asp293 (Chain B). This gives it really good density.</p>
<p>&nbsp;</p>
<div id="attachment_2367" style="width: 865px" class="wp-caption aligncenter"><img class="wp-image-2367 " src="https://openlabnotebooks.org/wp-content/uploads/2019/01/M4K2158_structure-1024x576.png" alt="" width="855" height="481" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/M4K2158_structure-1024x576.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/M4K2158_structure-300x169.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/M4K2158_structure-768x432.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/M4K2158_structure.png 1280w" sizes="(max-width: 855px) 100vw, 855px" /><p class="wp-caption-text">ACVR1 bound to M4K2158 at ~2 Å, showing interactions with His286 and a water at the back of the pocket</p></div>
<p>&nbsp;</p>
<p>I also seem to have some new <span style="font-size: 1.5rem;">pretty </span><span style="font-size: 1.5rem;">hits with M4K2118, M4K2149 and M4K3003 (crystal pics below). What’s interesting about the M4K3003 hits is that I’ve really struggled to get crystals with this compound (as well as M4K3007 and 3010), and it is a pretty desirable structure. These hits came about as a result of a mistake, where I somehow managed to mix up my tubes and put ACVR1 with M4K3003 into my mega composite screen without seeds, and TNIK into a fine screen I had intended for ACVR1 with seeds. I only discovered this because I mass specced the TNIK just before setting up the plates, then rushed off to do them as it was the SGC Christmas party that night and I needed to finish (best not to rush experiments). When I checked the mass spec result later, I got the exact mass of ACVR1! So I went back to the crystallography room and recovered what protein I could from the source plate for the TNIK plates, and mass specced it again. Same result – ACVR1. But when I looked at the drop images for those plates (now containing ACVR1 + M4K3003), I found a bunch of crystals! Happy serendipity!! These will go on the trip to Diamond on the 21</span><sup>st</sup><span style="font-size: 1.5rem;"> of January and then we’ll know whether the mistake really ended well or not. Fingers crossed!</span></p>
<p>&nbsp;</p>
<div id="attachment_2366" style="width: 893px" class="wp-caption aligncenter"><img class="wp-image-2366" src="https://openlabnotebooks.org/wp-content/uploads/2019/01/Crystal-hits-Dec_Jan-2018_9-1024x576.png" alt="" width="883" height="497" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/Crystal-hits-Dec_Jan-2018_9-1024x576.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/Crystal-hits-Dec_Jan-2018_9-300x169.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/Crystal-hits-Dec_Jan-2018_9-768x432.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/Crystal-hits-Dec_Jan-2018_9.png 1280w" sizes="(max-width: 883px) 100vw, 883px" /><p class="wp-caption-text">ACVR1 co-crystallised with M4K2149, 2118 and 3003 &#8211; hits to send to the synchrotron</p></div>
<p>&nbsp;</p>
<p>You may well ask, why was I setting up plates with TNIK? As it happens, some of our collaborators’ assays show M4K2009, our main compound of interest, hits both TNIK and DDR1 with pretty high affinity. We’ve been struggling to get crystals of ACVR1 with M4K2009 – for whatever reason, it doesn’t want to make crystals. So it seemed like a good idea to see if TNIK or DDR1 will co-crystallise with it. TNIK (TRAF2 and NCK Interacting Kinase) is a serine/threonine kinase that functions as an activator of the Wnt signalling pathway. DDR1 (Discoidin Domain Receptor Tyrosine Kinase 1) is a tyrosine kinase that functions as cell surface receptor for fibrillar collagen and regulates cell attachment to the extracellular matrix, remodelling of the extracellular matrix, cell migration, differentiation, survival and cell proliferation (according to its GeneCard).</p>
<p>While I was about it, I thought it would be useful to set up plates with BMPR1A (ALK3), for the same reason I set up TGFBR1 (ALK5) before – to see what off-target structures look like with some of these compounds. The purification protocols for TNIK and BMPR1A were as standard, but they’re available <a href="https://zenodo.org/record/2531714#.XC94jlz7SUk">here</a> on Zenodo.</p>
<p>Some other good news is that I’ve figured out how to reproduce my XChem crystals fairly reliably. On the last day before the holidays I set up 6 plates of ACVR1 with M4K2117, which gives really beautiful, well-diffracting crystals. I used 10, 12 and 15 mg/ml protein, and made plates with and without seeds. Well, the ones without the seeds grew fabulously, within 2 weeks, and the ones with seeds didn’t! The next step with these then is to test what happens if I soak them in different formulations of lower salt with and without some PEG3350. That&#8217;ll be for next week. Have a great weekend!</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/a-new-alk2-structure-and-proof-that-mistakes-in-the-lab-arent-always-a-bad-thing/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Selective CDKL2 inhibitors &#8211; molecular modeling</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/selective-cdkl2-inhibitors-molecular-modeling/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/selective-cdkl2-inhibitors-molecular-modeling/#respond</comments>
		<pubDate>Fri, 21 Dec 2018 02:02:27 +0000</pubDate>
		<dc:creator><![CDATA[Louisa Temme]]></dc:creator>
				<category><![CDATA[Louisa Temme]]></category>
		<category><![CDATA[Understudied Kinases]]></category>

		<guid isPermaLink="false">https://openlabnotebooks.org/?p=2325</guid>
		<description><![CDATA[At SGC we are interested in developing protein kinase inhibitors with a high selectivity towards other protein kinases. Based on lead compounds from the literature and our group new and selective CDKL2 inhibitors should be designed, synthesized and tested. Carla Alamillo, a former member of our group, worked on this topic and mentioned in her <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/selective-cdkl2-inhibitors-molecular-modeling/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p style="text-align: justify;">At SGC we are interested in developing protein kinase inhibitors with a high selectivity towards other protein kinases. Based on lead compounds from the literature and our group new and selective CDKL2 inhibitors should be designed, synthesized and tested. Carla Alamillo, a former member of our group, worked on this topic and mentioned in her last entry in the open notebook about this topic  (<a href="https://openlabnotebooks.org/sns-032-analogues-update/">https://openlabnotebooks.org/sns-032-analogues-update/</a>) the pro-mising role of SNS-032 as a starting point for new protein kinase inhibitors (Figure 1).<br />
SNS-032 shows a low K<sub>d</sub>value for CDKL2 (48 nM) but also for many other kinases. This means the selectivity of this compound towards other protein kinases is low. In contrast, CAF-181 synthesized by Carla (compound 11 in Carla’s entry) shows low K<sub>d</sub>values for CDKL2 and CDK7 and a high selectivity towards the other kinases tested. Additionally, CAF-181 reveals a moderate CDKL2 IC<sub>50</sub>value of 417 nM. Therefore, CAF-181 represents a good lead compound for the development of new CDKL2 inhibitors with improved selectivity towards CDK7.</p>
<p style="text-align: justify;"><img class="wp-image-2327 aligncenter" src="https://openlabnotebooks.org/wp-content/uploads/2018/12/122018_Picture1-1-300x174.png" alt="" width="694" height="402" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/122018_Picture1-1-300x174.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/122018_Picture1-1.png 668w" sizes="(max-width: 694px) 100vw, 694px" /></p>
<p style="text-align: justify;"><strong>Figure 1. </strong>Lead compounds SNS-032 and CAF-181 and their respective K<sub>d </sub>values.</p>
<p style="text-align: justify;">To design new compounds the molecular modeling software Flare<sup>TM</sup>(Cresset<sup>®</sup>) was used. The first step for the <em>in silico </em>docking is the selection of the crystal structure of the respective protein. Two crystal structures of CDKL2 are available on the PDB website (<a href="https://www.rcsb.org/">https://www.rcsb.org</a>) but not published in a journal yet. One crystal structure is co-crystallized with the compound TCS 2312 (PDB-ID 4BBM) and the other with the compound DKI (PDB-ID 4AAA) (Figure 2). The 2D interaction maps (<a href="https://www.rcsb.org/structure/4AAA">https://www.rcsb.org/structure/4AAA</a> and <a href="https://www.rcsb.org/structure/4BBM">https://www.rcsb.org/structure/4BBM</a>) have to be flipped vertically to match the same amino acid residues. Interestingly, Val83 (Figure 2, blue box) seems to be an impor-tant interaction partner for the ligands since both compounds reveal two H-bond interactions with Val83.</p>
<p style="text-align: justify;"><img class="wp-image-2328 aligncenter" src="https://openlabnotebooks.org/wp-content/uploads/2018/12/122018_Picture2-300x181.png" alt="" width="629" height="380" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/122018_Picture2-300x181.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/122018_Picture2.png 600w" sizes="(max-width: 629px) 100vw, 629px" /></p>
<p style="text-align: justify;"><strong>Figure 2. </strong>Crystal structures of CDKL2: co-crystallized with TCS 2312 (A, PDB-ID 4BBM, DOI: 10.2210/pdb4BBM/pdb) and co-crystallized with DKI (B, PDB-ID 4AAA, DOI: 10.2210/pdb4AAA/pdb). Black dashed lines indicate hydrogen bonds, salt bridges, and metal interactions. Green solid lines show hydrophobic interactions and green dashed lines show π-π and π-cation interactions (<a href="https://www.rcsb.org/structure/4BBM">https://www.rcsb.org/structure/4BBM</a> and <a href="https://www.rcsb.org/structure/4AAA">https://www.rcsb.org/structure/4AAA</a>).</p>
<p style="text-align: justify;">During the <em>in silico </em>docking studies the crystal structure 4BBM was used because the structure 4AAA reveals an unresolved amino acid sequence ending with Ala148 and NME148A (Figure 3B, black box). Unfortunately, this sequence part also contains the amino acid Phe147 that forms a part of the active site and likely interacts with inhibitors. However, the protein structure used for the docking should contain a fully resolved amino acid sequence in this area. The protein structure 4BBM provides a fully resolved amino acid sequence which makes it suitable for <em>in silico </em>docking.</p>
<p style="text-align: justify;"><img class="wp-image-2329 aligncenter" src="https://openlabnotebooks.org/wp-content/uploads/2018/12/122018_Picture3-300x81.png" alt="" width="870" height="235" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/122018_Picture3-300x81.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/122018_Picture3-768x208.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/122018_Picture3.png 974w" sizes="(max-width: 870px) 100vw, 870px" /></p>
<p style="text-align: justify;"><strong>Figure 3. </strong>Structure of CDKL2 co-crystallized with either TCS 2312 (orange, A, PDB-ID 4BBM) or DKI (dark red, B, PDB-ID 4AAA). The dashed green and orange lines indicate H-bond interactions and the purple lines CH-π interactions. Docked ligands are shown as sticks. Oxygen and nitrogen atoms are colored in red and blue, respectively.</p>
<p style="text-align: justify;">However, Phe147 in structure 4BBM is not clarified. Instead of the phenyl ring a methyl group is visible. Capping by the docking program Flare<sup>TM</sup> did not result in phenylalanine but just in a partially aromatic phenyl ring of this amino acid (Figure 3A, black box). This problem was solved by mutation of this position into a phenylalanine.</p>
<p style="text-align: justify;">To investigate the selectivity of newly designed compounds they should also be investigated during the <em>in silico </em>docking into the crystal structure of CDK7 with the PDB-ID 1UA2.</p>
<p style="text-align: justify;">The next step of <em>in silico</em> docking into the CDKL2 structure 4BBM included the validation of the crystal structure and the docking protocol. After preparation and minimization of the crystal structure the co-crystallized ligand TCS 2312 should be redocked into the active site. However, the co-crystallized ligand TCS 2312 shows different bond orders/tautomeric states as well as a different protonation state on the basic nitrogen than the literature compound TCS 2312. To calculate the RMSD value between the binding mode of the structurally corrected and redocked TCS 2312 and the co-crystallized ligand the co-crystallized binding mode had to be corrected because Flare<sup>TM</sup> can only calculate a RMSD value between structurally identical compounds. Therefore, the co-crystallized ligand was corrected and only the resorcinol moiety was minimized to achieve a planar aromatic system. In this way the binding mode of the co-crystallized ligand was changed as little as possible. Then, the corrected, co-crystallized ligand TCS  2312 was minimized and the structure at physiological pH was generated. Afterwards, this compound was docked into the crystal structure 4BBM. Figure 4A shows the similar orientation of corrected, co-crystallized TCS 2312 (yellow) and the redocked ligand (purple). The similar orientation of both binding modes and the hereby resulting low RMSD value<br />
(1.65 Å) suggest that the corrected CDKL2 protein crystal structure 4BBM and the docking protocol are suitable for the <em>in silico </em>docking of new compounds.</p>
<p style="text-align: justify;"><img class="wp-image-2330 aligncenter" src="https://openlabnotebooks.org/wp-content/uploads/2018/12/122018_Picture4-300x181.png" alt="" width="762" height="460" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/122018_Picture4-300x181.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/122018_Picture4-768x463.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/122018_Picture4.png 943w" sizes="(max-width: 762px) 100vw, 762px" /></p>
<p style="text-align: justify;"><strong>Figure 4. </strong>Predicted binding modes of TCS 2312 (A, purple), CAF-181 (B) and SNS-032 (C and D) obtained by docking into the CDKL2 crystal structure with the PDB-ID 4BBM.Binding mode of the corrected, co-crystallized ligand TCS 2312 (A, yellow). The dashed green and orange lines indicate H-bond interactions and the purple lines CH-π interactions. Docked ligands are shown as sticks. Oxygen and nitrogen atoms are colored in red and blue, respectively.</p>
<p>&nbsp;</p>
<p style="text-align: justify;">In the Figures 4B-D the docking results of CAF-181 and SNS-032 are shown. The <em>tert</em>-butyl group of both compounds occupies the inner part of the active site whereas the piperazine ring of CAF-181 and the piperidine of SNS 032 point towards the solvent accessible area. The N-atom of the SNS-032 piperidine ring is part of an H-bond interaction with one of the carboxylic acid moieties of either Glu130 (C) or Asp89 (D). Therefore, it might be promising to introduce amide substituents that can interact with those acidic amino acids.</p>
<p style="text-align: justify;">CAF compounds, synthesized and tested previously in our group, were docked into the CDKL2 crystal structure 4BBM and in the CDK7 crystal structure 1UA2. The respective docking scores and electrostatic complementarity<sup>TM </sup>scores of the resulting docking poses reveal a good correlation to the K<sub>d </sub>values of those compounds regarding CDKL2. However, no clear correlation was found regarding docking of CAF compounds in CDK7. This observation could be explained with the larger active site of CDK7 and its many solvent accessible areas that could facilitate binding of structurally diverse compounds.</p>
<p style="text-align: justify;">Interestingly, <em>in silico</em> docking of SNS-032 and CAF-181 in the active site of CDK7 reveal  curved orientations of both compounds (Figure 5). The green color of the electrostatic complementarity<sup>TM </sup>surface of the SNS-032 binding mode in Figure 5C indicates a high electrostatic complementarity<sup>TM </sup>of the ligand to this active site which could be the reason for the high inhibitory activity of SNS-032 regarding CDK7.</p>
<p style="text-align: justify;"><em><img class="wp-image-2331 aligncenter" src="https://openlabnotebooks.org/wp-content/uploads/2018/12/122018_Picture5-300x180.png" alt="" width="751" height="451" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/122018_Picture5-300x180.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/122018_Picture5-768x461.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/122018_Picture5.png 962w" sizes="(max-width: 751px) 100vw, 751px" /></em></p>
<p style="text-align: justify;"><strong>Figure 5. </strong>Predicted binding modes obtained by docking into the CDK7 crystal structure with the PDB-ID 1UA2:<br />
(A) SNS-032. (B) SNS-032 with the electrostatic potential surface of CDK7. (C) SNS-032 with its electrostatic complementarity<sup>TM </sup>surface. (D) CAF-181. The dashed green and orange lines indicate H-bond interactions and the purple lines CH-π interactions. Docked ligands are shown as sticks. Oxygen and nitrogen atoms are colored in red and blue, respectively.</p>
<p style="text-align: justify;">Based on this observation new CDKL2 inhibitors should reveal restrained flexibility to avoid the curved orientation in CDK7 and thereby increase the selectivity towards CDK7. Among others this could be achieved by replacement of the flexible thio-methylene linker by a more rigid linker. In my next entry I will show the docking results of more rigid compounds obtained by both <em>in silico </em>docking into the crystal structure of CDKL2 and CDK7.</p>
<p style="text-align: justify;">Please feel free to suggest other ideas for CDKL2 inhibitors with increased selectivity towards CDK7. If you have any questions or comments, please let me know.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/selective-cdkl2-inhibitors-molecular-modeling/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Determining the nanoBRET ALK2 IC50 values of 24 new ACVR1/ALK2 inhibitors</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/determining-the-nanobret-al2k-ic50-values-of-24-new-acvr1-alk2-inhibitors/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/determining-the-nanobret-al2k-ic50-values-of-24-new-acvr1-alk2-inhibitors/#respond</comments>
		<pubDate>Wed, 19 Dec 2018 06:39:05 +0000</pubDate>
		<dc:creator><![CDATA[Jong Fu]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Brain Tumours]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Jong Fu Wong]]></category>

		<guid isPermaLink="false">https://openlabnotebooks.org/?p=2314</guid>
		<description><![CDATA[In an effort to develop clinical compounds for Diffused Intrinsic Pontine Glioma (DIPG) treatment, new analogues of ACVR1/ALK2 inhibitors are continuously synthesised by Ontario Institute for Cancer Research (OICR) and Charles River Laboratories (CRL). I will provide prompt feedback of the cellular assay results to guide their design of new compounds. I determined the potency <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/determining-the-nanobret-al2k-ic50-values-of-24-new-acvr1-alk2-inhibitors/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>In an effort to develop clinical compounds for Diffused Intrinsic Pontine Glioma (DIPG) treatment, new analogues of ACVR1/ALK2 inhibitors are continuously synthesised by <a href="https://oicr.on.ca/">Ontario Institute for Cancer Research (OICR)</a> and <a href="https://www.criver.com/">Charles River Laboratories (CRL)</a>. I will provide prompt feedback of the cellular assay results to guide their design of new compounds.</p>
<div id="attachment_2315" style="width: 397px" class="wp-caption alignnone"><img class=" wp-image-2315" src="https://openlabnotebooks.org/wp-content/uploads/2018/12/Transfection-control.png" alt="" width="387" height="387" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/Transfection-control.png 575w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/Transfection-control-150x150.png 150w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/Transfection-control-300x300.png 300w" sizes="(max-width: 387px) 100vw, 387px" /><p class="wp-caption-text">HEK293 cells were nicely transfected, as indicated by GFP signal. I can proceed with the experiment.</p></div>
<div id="attachment_2316" style="width: 656px" class="wp-caption alignnone"><img class="size-full wp-image-2316" src="https://openlabnotebooks.org/wp-content/uploads/2018/12/Figure-2.png" alt="" width="646" height="720" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/Figure-2.png 646w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/Figure-2-269x300.png 269w" sizes="(max-width: 646px) 100vw, 646px" /><p class="wp-caption-text">ALK2 NanoBRET IC50 curves and chemical structures of the first dozen of new ACVR1/ALK2 inhibitors. IC50 values estimated by GraphPad Prism are shown in green.</p></div>
<div id="attachment_2317" style="width: 679px" class="wp-caption alignnone"><img class="size-full wp-image-2317" src="https://openlabnotebooks.org/wp-content/uploads/2018/12/Figure-3.png" alt="" width="669" height="720" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/Figure-3.png 669w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/Figure-3-279x300.png 279w" sizes="(max-width: 669px) 100vw, 669px" /><p class="wp-caption-text">ALK2 NanoBRET IC50 curves and chemical structures of the second dozen of new ACVR1/ALK2 inhibitors. IC50 values estimated by GraphPad Prism are shown in green.</p></div>
<p>I determined the potency of 24 new ACVR1/ALK2 inhibitors synthesised by OICR. M4K2009 in particular is quite potent (<a href="https://en.wikipedia.org/wiki/IC50">larger values indicates lower potency</a>). This compound differs from legacy compound M4K1055 by the position of its methyl group (CH3) in the middle of the molecule. Attempts to replace the tri-methoxy group (the three CH3-O groups bonded to the phenyl ring) were not successful. Since a robust nanoBRET tracer is not available for TGFBR1/ALK5, I will use dual luciferase promoter assay (orthologous assay) to determine the off-target activity of these compounds towards TGFBR1/ALK5.</p>
<div id="attachment_2322" style="width: 440px" class="wp-caption alignnone"><img class=" wp-image-2322" src="https://openlabnotebooks.org/wp-content/uploads/2018/12/Comparing-M4K2009-to-legacy-M4K1055.png" alt="" width="430" height="376" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/Comparing-M4K2009-to-legacy-M4K1055.png 326w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/Comparing-M4K2009-to-legacy-M4K1055-300x262.png 300w" sizes="(max-width: 430px) 100vw, 430px" /><p class="wp-caption-text">Comparing the chemical structures of M4K2009 to legacy compound M4K1055. ALK2 nanoBRET IC50 values of the compounds are also shown (lower values reflects increased potency).</p></div>
<p>The M4K monthly meeting where this result was presented has been recorded and posted on <a href="https://www.youtube.com/watch?v=YAsVIt_iBes&amp;t=1464s">youtube</a>.</p>
<p>For detailed experimental protocols, please refer to my <a href="https://zenodo.org/record/2410328#.XBnjM1X7Spo">Zenodo page</a>.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/determining-the-nanobret-al2k-ic50-values-of-24-new-acvr1-alk2-inhibitors/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Update on batch deposition of XChem structures</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/update-on-batch-deposition-of-xchem-structures/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/update-on-batch-deposition-of-xchem-structures/#respond</comments>
		<pubDate>Tue, 18 Dec 2018 12:21:14 +0000</pubDate>
		<dc:creator><![CDATA[Tobias Krojer]]></dc:creator>
				<category><![CDATA[Tobias Krojer]]></category>
		<category><![CDATA[XChem]]></category>

		<guid isPermaLink="false">https://openlabnotebooks.org/?p=2292</guid>
		<description><![CDATA[The year is coming to an end and I want to start my blog by giving you an overview of a project that kept me busy for the better part of it. Our existing Protein Data Bank (PDB) batch deposition mechanism underwent a major overhaul recently and we are finally in a position where we <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/update-on-batch-deposition-of-xchem-structures/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>The year is coming to an end and I want to start my blog by giving you an overview of a project that kept me busy for the better part of it. Our existing <a href="https://www.wwpdb.org/">Protein Data Bank (PDB)</a> batch deposition mechanism underwent a major overhaul recently and we are finally in a position where we can relatively quickly deposit all the structures from an <a href="https://www.diamond.ac.uk/Instruments/Mx/Fragment-Screening.html">XChem fragment screening</a> campaign.</p>
<p>&nbsp;</p>
<p>I assume not everyone will be familiar with what this is about, so here is some background of the problems we are trying to solve: once the XChem facility started churning out more and more protein-fragment complex structures, it became obvious that we needed new tools to support PDB deposition. Doing it one-by-one through the wwPDB website would be too time-consuming. Additionally, the multi-dataset nature of the experiment and the way we analyse the data made it necessary to think about new ways to help the end-user understand the basis of the modelling decisions taken by the depositor.</p>
<p>&nbsp;</p>
<h4>Summary</h4>
<p>The post is rather long, because it contains quite a few details which might be of interest if you are confronted with having to deposit many similar structures or if you want to publish findings based on a <a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5413968/">PanDDA</a> analysis. Here&#8217;s a quick summary if you are in a rush:</p>
<ul>
<li>Each deposited structure contains direct evidence for the modelled ligand, i.e. the PanDDA event map</li>
<li>All apo structures that were collected during an XChem experiment are deposited as a single PDB entry so that the data can be used to reproduce the PanDDA analysis</li>
<li>File preparation for batch deposition of ligand bound and apo structures is facilitated by XChemExplorer</li>
</ul>
<p>&nbsp;</p>
<h3>PDB group depositions</h3>
<p>The first problem, i.e. deposition of many structures, was solved quickly, because, fortunately for us, the PDB has already developed a new, but not very well known, portal to support parallel structure depositions, so-called <a href="https://deposit-group-1.rcsb.rutgers.edu/groupdeposit/">group depositions</a>. It’s pretty nifty, you can bundle all your structure and structure factor mmcif files in a single tar archive, then you upload the file and press submit. That&#8217;s it! Well, almost. One still needs to provide the usual experimental data like protein sequence, crystallization condition, authors etc. in the header of each mmcif file, but I’ve implemented a solution in the<a href="https://github.com/xchem/XChemExplorer"> XChemExplorer (XCE)</a> software so that one only needs to do it once and the information then gets propagated to all the structures in a group (Figure 1). Once the user has provided this information in XCE, they need to flag which structures they want to deposit and the program will generate the respective files, ready for upload. That was the – conceptually – easy bit. Next, we had to think about ways to provide evidence for the modelled ligands.</p>
<div id="attachment_2294" style="width: 523px" class="wp-caption aligncenter"><img class=" wp-image-2294" src="https://openlabnotebooks.org/wp-content/uploads/2018/12/figure_1-300x213.png" alt="" width="513" height="364" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/figure_1-300x213.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/figure_1-768x544.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/figure_1.png 907w" sizes="(max-width: 513px) 100vw, 513px" /><p class="wp-caption-text">Figure 1. XChemExplorer (XCE) enables preparation of files for mass structure deposition through the PDB group deposition portal. Users only need to provide a minimal amount of information which is then propagated to all files in a group. The remaining information is captured from the XCE database and the project directory. XCE uses PDB_EXTRACT and SF_CONVERT to generate structure and structure factor mmcif files, respectively. XCE can generate and bundle mmcif files of ligand-bound models, as well as of the ground-state model and the corresponding apo structure factor mmcif files.</p></div>
<h3></h3>
<h3>Evidence for modelled ligands</h3>
<p>We routinely use the <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5413968/">Pan-Dataset Density Analysis (PanDDA)</a> method to identify bound fragments and because the algorithm is so sensitive, we are often able to detect weakly bound compounds which would go unnoticed in 2mFo-DFc maps. However, the concept of PanDDA event maps is still just emerging in the community and so the question is how can we provide the rest of the world with evidence that these ligands really are bound? The last few years have seen a few efforts to improve the quality of protein-ligand structures in the PDB and there have been several publications which highlight dubiously modelled ligands. So, what to do in order to avoid that XChem will soon find itself in the pillory? Together with Stephen Burley’s team at the RCSB, we enabled inclusion of PanDDA event maps in the deposited structure factor mmcif file (Figure 2). This allows everyone – with a bit of crystallographic knowledge – to download the mmcif file and recreate the maps that were used to model the bound ligand and eventual changes in the protein structure. Ideally, the event maps would be displayed on the different PDB websites, but in the meantime, we have created our own <a href="https://www.thesgc.org/fragment-screening">interactive HTML summaries</a> which users can use to share their results with the world. On a personal note, I would like to add that there certainly are ‘fantasy’ models in the PDB, but this is probably unavoidable given the sheer number of structures in the archive and that overall, protein crystallographers are most likely not too dissimilar to the rest of humanity when it comes to the distribution of certain traits. Still, I am sure that most scientists are aware of the limitations of some of their models, but they have no possibility to communicate their concerns. However, this is another topic which I will elaborate on in one of my next blog posts.</p>
<p>&nbsp;</p>
<h3>Reproduction of results</h3>
<p>Finally, we also thought about ways to enable anyone to reproduce the event maps. Ultimately, PanDDA event maps cannot be reproduced unless you have all the apo datasets available to calculate the respective statistical maps. Making all the data available, also ensures that PanDDA users can be confident when publishing their results. Technically, this should be trivial in the year 2018 given that the file sizes involved are negligible by today’s standards. The more important (and contentious) question is ‘<em>where should the data live?</em>’. The PDB recommended to deposit them in their archive so that they will be properly curated and remain available in perpetuity. Hence, our initial strategy in 2017 was to bundle all the apo structures in another tar archive and deposit them in a separate group deposition session. But that meant that every apo structure was treated as a unique structure and therefore got its own PDB ID. Needless to say that this created confusion in some parts of the community who were wondering what we are up to. This did clearly not work as expected, not least because even if someone would like the run PanDDA again on the data, they would need a lot of patience if they want to download all the datasets from the PDB. We contemplated to use the <a href="https://zenodo.org/">ZENODO</a> data repository as an alternative (and we still recommend to use this platform to upload the complete PanDDA analysis anyway), but finally we came up with a, which at least I believe, much more elegant solution. Instead of depositing every apo PDB file (which are almost identical after initial refinement), we deposit only the ground-state model (and the corresponding mean map) and then include the structure factors of every dataset that was collected during the XChem experiment in the final structure factor mmcif file (Figure 2). The file will often be several Gb large, but users will now be able to download all the data that they need to reproduce a PanDDA analysis in one go.</p>
<div id="attachment_2295" style="width: 523px" class="wp-caption aligncenter"><img class="wp-image-2295" src="https://openlabnotebooks.org/wp-content/uploads/2018/12/figure_2-300x214.png" alt="" width="513" height="365" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/figure_2-300x214.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/figure_2-768x547.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/figure_2.png 850w" sizes="(max-width: 513px) 100vw, 513px" /><p class="wp-caption-text">Figure 2. File structure and contents of the deposited mmcif files. Each ligand-bound structure (changed_state.pdb) does consist of an mmcif file containing its coordinates and experimental data and a structure factor mmcif file which contains the measured data as well as an additional data block for each PanDDA event map. The ground-state model will be included in a separate mmcif file. The corresponding structure factor mmcif file does contain one block for the mean map and then one block each for every datasets which was part of the XChem experiment. In case someone wants to reproduce the PanDDA analysis, they can download the ground-state deposition, use the model to run initial refinement on every dataset and then subject it to PanDDA analysis. Additionally, there is a possibility to upload the complete PanDDA analysis into the ZENODO data repository and point to it in the PDB deposition.</p></div>
<p>&nbsp;</p>
<p>We have recently made our first apo deposition (G_1002058) and are still sorting out a few technicalities. Once this is done, I’m planning to make a few changes to the XCE gui, so that users get a simple step-by-step description what they need to do. Also, I am currently working on a publication which we will hopefully submit early next year and which will provide more details.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/update-on-batch-deposition-of-xchem-structures/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Synthesis and testing of follow up compounds based on HAO1 fragment 6, bound at the tetramer interface.</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/synthesis-and-testing-of-follow-up-compounds-based-on-hao1-fragment-6-bound-at-the-tetramer-interface/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/synthesis-and-testing-of-follow-up-compounds-based-on-hao1-fragment-6-bound-at-the-tetramer-interface/#comments</comments>
		<pubDate>Mon, 17 Dec 2018 03:03:00 +0000</pubDate>
		<dc:creator><![CDATA[Sabrina Mackinnon]]></dc:creator>
				<category><![CDATA[HAO1]]></category>
		<category><![CDATA[Primary Hyperoxaluria]]></category>
		<category><![CDATA[Sabrina Mackinnon]]></category>

		<guid isPermaLink="false">https://openlabnotebooks.org/?p=2279</guid>
		<description><![CDATA[Hello again! A few posts ago, when I shared my fragment screening results (here), I mentioned that we had one fragment that bound at the interface between two subunits of the HAO1 tetramer (figure 1 will refresh your memory). The simplest way to turn this into an inhibitor would be to make it bigger so <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/synthesis-and-testing-of-follow-up-compounds-based-on-hao1-fragment-6-bound-at-the-tetramer-interface/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Hello again! A few posts ago, when I shared my fragment screening results (<a href="https://openlabnotebooks.org/analysis-of-fragments-that-bind-to-hao1/">here</a>), I mentioned that we had one fragment that bound at the interface between two subunits of the HAO1 tetramer (figure 1 will refresh your memory). The simplest way to turn this into an inhibitor would be to make it bigger so that it disrupts or otherwise changes the HAO1 tetramer. This could inhibit HAO1 activity either directly, where compound-induced changes in the amount, shape or flexibility of the tetramer reduce HAO1 efficiency (<strong>less HAO1 activity</strong><span style="font-size: 1.5rem;">), or indirectly, where disrupting the tetramer reduces the stability of HAO1 protein (<strong>less HAO1 amount</strong>).</span></p>
<p>&nbsp;</p>
<p><img class="aligncenter size-large wp-image-2282" src="https://openlabnotebooks.org/wp-content/uploads/2018/12/post7-fig1-1024x572.png" alt="" width="780" height="436" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/post7-fig1-1024x572.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/post7-fig1-300x168.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/post7-fig1-768x429.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/post7-fig1.png 1380w" sizes="(max-width: 780px) 100vw, 780px" /></p>
<p style="text-align: center;"><em><strong>Figure 1: Fragment 6 binds at the interface between two subunits of the HAO1 tetramer. </strong>A. Location of fragment binding in relation to the HAO1 tetramer. B. Surface model showing fragment binds in pocket between two subunits. C. Ribbon and stick model showing fragment 6 interacts with both subunits, displacing R341 (subunit 1) and pi stacking with Y163 (subunit 2).</em></p>
<p>So, with help from Tamas Szommer in Paul Brennan’s group (check out the group page <a href="https://www.brennanresearchgroup.com/">here</a>), I made a small set of fifteen bigger compounds to see if we can develop inhibitors targeting the subunit interface. I wanted to preserve the aromatic ring (called a pyrimidine because of its two nitrogen groups), which engages in pi stacking with one subunit of HAO1, and the chlorine substituent, which displaces an arginine on the other subunit, to try to ensure that the compounds still bind to the same place, while replacing the five-membered, oxygen-containing ring (called a tetrahydrofuran) with larger substituents.</p>
<p>To do this I used a common chemical reaction, called <a href="https://chem.libretexts.org/Textbook_Maps/Organic_Chemistry/Map%3A_Organic_Chemistry_(McMurry)/Chapter_16%3A_Chemistry_of_Benzene_-_Electrophilic_Aromatic_Substitution/16.07_Nucleophilic_Aromatic_Substitution">nucleophilic aromatic substitution</a>, to replace one chlorine in a dichloro-pyrimidine with the amine (-NH) group of various building blocks. In this reaction (shown in figure 2A), a nucleophilic group (in this case the amine of the building block, R1-NH-R2) is added to an aromatic ring, delocalising the electrons and forming an intermediate known as the Meisenheimer complex. Then, the halogen atom (in this case the second chlorine) is eliminated from the ring, allowing it to return to its preferred, lower energy, aromatic conformation. I chose 15 compounds that were diverse in structure and greater than 150 Daltons in size (a tetrahydrofuran is 72 Daltons) to be my building blocks (figure 2B).</p>
<p>&nbsp;</p>
<p><img class="aligncenter size-full wp-image-2281" src="https://openlabnotebooks.org/wp-content/uploads/2018/12/post7-fig2.png" alt="" width="941" height="492" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/post7-fig2.png 941w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/post7-fig2-300x157.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/post7-fig2-768x402.png 768w" sizes="(max-width: 941px) 100vw, 941px" /></p>
<p style="text-align: center;"><em><strong>Figure 2: SNAr reaction mechanism and amines used to synthesise follow up compounds. </strong>A. SNAr reaction scheme. Inset: Meisenheimer complex intermediate. B. 15 building blocks selected based on size, diversity and inclusion of a free amine (NH or NH2).</em></p>
<p>As with the loop 4 and active site follow up compounds (discussed in my <a href="https://openlabnotebooks.org/screening-follow-up-compounds-from-active-site-and-gating-loop-hao1-fragment-hits/">last post</a>), I tested my new compounds for inhibition of HAO1 using a fluorescence-based activity assay. Of the 15 compounds, only three inhibited more than 50% of HAO1 activity at 1 mM, allowing me to determine an IC50 for them. The best compound, SM001-10, has an IC50 of 86 μM. Some compounds showed no inhibition at all at 1 mM and this is also useful in determining how (and how not) to expand the good compounds. More details about the synthesis and testing of these compounds can be found on my Zenodo post <a href="https://doi.org/10.5281/zenodo.2341173">here</a>.</p>
<p>&nbsp;</p>
<p><img class="aligncenter size-full wp-image-2280" src="https://openlabnotebooks.org/wp-content/uploads/2018/12/post7-fig3.png" alt="" width="836" height="672" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/post7-fig3.png 836w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/post7-fig3-300x241.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/post7-fig3-768x617.png 768w" sizes="(max-width: 836px) 100vw, 836px" /></p>
<p style="text-align: center;"><em><strong>Figure 3: Effect of fragment 6 and related compounds on HAO1 activity. </strong>A. Original hit, fragment 6, structure and activity. B. Extensions that did not result in inhibition of HAO1. C. Structure and IC50 results for the three compounds showing significant inhibition of HAO1 at less than 1 mM concentration.</em></p>
<p>We need two more pieces of information to guide the development of these compound into good inhibitors for HAO1: x-ray crystal structures to tell us how exactly the compounds bind and where we could grow the compounds to improve them; and an assay that tells us more about <em>how</em> these compounds are inhibiting HAO1 activity i.e. do they disrupt the HAO1 tetramer? Or change the shape of the tetramer so that it is less effective at turning over the substrate?</p>
<p>Well, with any luck, we will find out together in the New Year! Until then – Happy holidays everyone!</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/synthesis-and-testing-of-follow-up-compounds-based-on-hao1-fragment-6-bound-at-the-tetramer-interface/feed/</wfw:commentRss>
		<slash:comments>1</slash:comments>
		</item>
		<item>
		<title>USP Zf-UBD Crystallography Pipeline Pt 2.</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/usp-zf-ubd-crystallography-pipeline-pt-2/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/usp-zf-ubd-crystallography-pipeline-pt-2/#respond</comments>
		<pubDate>Fri, 14 Dec 2018 22:28:00 +0000</pubDate>
		<dc:creator><![CDATA[Mandeep Mann]]></dc:creator>
				<category><![CDATA[Mandeep Mann]]></category>
		<category><![CDATA[USP5]]></category>

		<guid isPermaLink="false">https://openlabnotebooks.org/?p=2266</guid>
		<description><![CDATA[In a previous post, I described the importance of solving the structure of other USPs that have a Zf-UBD for the rational design of selective compounds. The last post described the design, cloning, E.coli expression, growth and purification of some of the USP Zf-UBD constructs. As a continuation in my quest to try and crystallize <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/usp-zf-ubd-crystallography-pipeline-pt-2/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>In a previous <a href="https://openlabnotebooks.org/usp-zf-ubd-crystallography-pipeline/">post</a>, I described the importance of solving the structure of other USPs that have a Zf-UBD for the rational design of selective compounds. The last post described the design, cloning, <em>E.coli </em>expression, growth and purification of some of the USP Zf-UBD constructs. As a continuation in my quest to try and crystallize the other USP Zf-UBDs I’ve finished purifying and setting up crystallization screens of the ‘TOC013’ constructs in an attempt to crystallize these protein samples for subsequent structure determination by X-ray crystallography. You can see details on <a href="https://zenodo.org/record/2284636#.XBQt_lxKiUk"><strong>Zenodo</strong></a>.  I previously described getting crystals of USP33 that did not diffract and USP20 microcrystals that were too small to mount. I tried ‘seeding’ these initial crystal conditions in the hopes of getting larger crystals that diffract. Micro-seeding is the process of adding a dilute solution of m<span style="font-size: 1.5rem;">icro-crystals to a crystallizing solution to nucleate more crystals. This helps in controlling size and reproducibility.</span></p>
<p>So far, I have not had any luck with seeding but it may just be a matter of time- some proteins take longer to crystallize than others. I’ll have to keep an eye on my plates. Hopefully I’ll have some luck in the New Year and come back to crystals after the holidays! The USP Zf-UBD crystallization pipeline to date is summarized in Table 1.</p>
<p style="text-align: center;"><strong>Table 1</strong>. Summary of USP Zf-UBD Crystallization</p>
<table style="height: 2843px;" width="694">
<tbody>
<tr>
<td style="width: 73.9205px;"><strong>Clone </strong></td>
<td style="width: 41.1932px;"><strong>Target</strong></td>
<td style="width: 26.6477px;"><strong>AA </strong></td>
<td style="width: 43.0114px;"><strong>Vector</strong></td>
<td style="width: 37.5568px;"><strong>Media</strong></td>
<td style="width: 49.375px;"><strong>Amount Grown (L)</strong></td>
<td style="width: 72.1023px;"><strong>Purification Yield (mg) </strong></td>
<td style="width: 44.8295px;"><strong>Crystal Plate Setup (Y/N) </strong></td>
<td style="width: 84.8295px;"><strong>Crystals? (Y/N) </strong></td>
<td style="width: 65.7386px;"><strong>Diffraction (Y/N)</strong></td>
</tr>
<tr>
<td style="width: 73.9205px;">TOC013E09</td>
<td style="width: 41.1932px;">USP44</td>
<td style="width: 26.6477px;">27-110</td>
<td style="width: 43.0114px;">pET28-MHL</td>
<td style="width: 37.5568px;">M9</td>
<td style="width: 49.375px;">12</td>
<td style="width: 72.1023px;">No/low expression</td>
<td style="width: 44.8295px;">Y</td>
<td style="width: 84.8295px;">N</td>
<td style="width: 65.7386px;"></td>
</tr>
<tr>
<td style="width: 73.9205px;">TOC013F02</td>
<td style="width: 41.1932px;">USP49</td>
<td style="width: 26.6477px;">15-105</td>
<td style="width: 43.0114px;">pET28-MHL</td>
<td style="width: 37.5568px;">M9</td>
<td style="width: 49.375px;">12</td>
<td style="width: 72.1023px;">0.25 mg</td>
<td style="width: 44.8295px;">Y</td>
<td style="width: 84.8295px;">N</td>
<td style="width: 65.7386px;"></td>
</tr>
<tr>
<td style="width: 73.9205px;">TOC013D10</td>
<td style="width: 41.1932px;">USP33</td>
<td style="width: 26.6477px;">37-124</td>
<td style="width: 43.0114px;">pET28-MHL</td>
<td style="width: 37.5568px;">M9</td>
<td style="width: 49.375px;">6</td>
<td style="width: 72.1023px;">15.7 mg</td>
<td style="width: 44.8295px;">Y</td>
<td style="width: 84.8295px;">Y</p>
<p>-seeding:100 nL of 1:1000 and 1:10000 SM of 1 µL drops of RWD01, RWE01, SGCF03 small microcrystals (SM concentration too high?)</td>
<td style="width: 65.7386px;">N</td>
</tr>
<tr>
<td style="width: 73.9205px;">TOC013D11</td>
<td style="width: 41.1932px;">USP33</td>
<td style="width: 26.6477px;">38-124</td>
<td style="width: 43.0114px;">pET28-MHL</td>
<td style="width: 37.5568px;">M9</td>
<td style="width: 49.375px;">6</td>
<td style="width: 72.1023px;">0.35 mg</td>
<td style="width: 44.8295px;">Y</td>
<td style="width: 84.8295px;">N</td>
<td style="width: 65.7386px;"></td>
</tr>
<tr>
<td style="width: 73.9205px;">TOC013E02</td>
<td style="width: 41.1932px;">USP39</td>
<td style="width: 26.6477px;">97-197</td>
<td style="width: 43.0114px;">pET28-MHL</td>
<td style="width: 37.5568px;">M9</td>
<td style="width: 49.375px;">6</td>
<td style="width: 72.1023px;">1.6 mg</td>
<td style="width: 44.8295px;">Y</td>
<td style="width: 84.8295px;">N</td>
<td style="width: 65.7386px;"></td>
</tr>
<tr>
<td style="width: 73.9205px;">TOC013E05</td>
<td style="width: 41.1932px;">USP39</td>
<td style="width: 26.6477px;">97-205</td>
<td style="width: 43.0114px;">pET28-MHL</td>
<td style="width: 37.5568px;">M9</td>
<td style="width: 49.375px;">6</td>
<td style="width: 72.1023px;">1.1 mg</td>
<td style="width: 44.8295px;">Y</td>
<td style="width: 84.8295px;">N</td>
<td style="width: 65.7386px;"></td>
</tr>
<tr>
<td style="width: 73.9205px;">TOC013C02</td>
<td style="width: 41.1932px;">USP13</td>
<td style="width: 26.6477px;">188-292</td>
<td style="width: 43.0114px;">pET28-MHL</td>
<td style="width: 37.5568px;">M9</td>
<td style="width: 49.375px;">6</td>
<td style="width: 72.1023px;">Insoluble</td>
<td style="width: 44.8295px;"></td>
<td style="width: 84.8295px;"></td>
<td style="width: 65.7386px;"></td>
</tr>
<tr>
<td style="width: 73.9205px;">TOC013C03</td>
<td style="width: 41.1932px;">USP13</td>
<td style="width: 26.6477px;">208-292</td>
<td style="width: 43.0114px;">pET28-MHL</td>
<td style="width: 37.5568px;">M9</td>
<td style="width: 49.375px;">6</td>
<td style="width: 72.1023px;">Insoluble</td>
<td style="width: 44.8295px;"></td>
<td style="width: 84.8295px;"></td>
<td style="width: 65.7386px;"></td>
</tr>
<tr>
<td style="width: 73.9205px;">TOC013C04</td>
<td style="width: 41.1932px;">USP13</td>
<td style="width: 26.6477px;">208-308</td>
<td style="width: 43.0114px;">pET28-MHL</td>
<td style="width: 37.5568px;">TB</td>
<td style="width: 49.375px;">6</td>
<td style="width: 72.1023px;">2.4 mg</td>
<td style="width: 44.8295px;">Y</td>
<td style="width: 84.8295px;">N</td>
<td style="width: 65.7386px;"></td>
</tr>
<tr>
<td style="width: 73.9205px;">TOC013B11</td>
<td style="width: 41.1932px;">USP03</td>
<td style="width: 26.6477px;">10-152</td>
<td style="width: 43.0114px;">pET28-MHL</td>
<td style="width: 37.5568px;">TB</td>
<td style="width: 49.375px;">6</td>
<td style="width: 72.1023px;">2 mg</td>
<td style="width: 44.8295px;">Y</td>
<td style="width: 84.8295px;">N</td>
<td style="width: 65.7386px;"></td>
</tr>
<tr>
<td style="width: 73.9205px;">TOC013C08</td>
<td style="width: 41.1932px;">USP16</td>
<td style="width: 26.6477px;">26-146</td>
<td style="width: 43.0114px;">pET28-MHL</td>
<td style="width: 37.5568px;">M9</td>
<td style="width: 49.375px;">6</td>
<td style="width: 72.1023px;">0.7 mg</td>
<td style="width: 44.8295px;">Y</td>
<td style="width: 84.8295px;">N</td>
<td style="width: 65.7386px;"></td>
</tr>
<tr>
<td style="width: 73.9205px;">TOC013C09</td>
<td style="width: 41.1932px;">USP16</td>
<td style="width: 26.6477px;">32-146</td>
<td style="width: 43.0114px;">pET28-MHL</td>
<td style="width: 37.5568px;">TB</td>
<td style="width: 49.375px;">6</td>
<td style="width: 72.1023px;">0.8 mg</td>
<td style="width: 44.8295px;">Y</td>
<td style="width: 84.8295px;">N</td>
<td style="width: 65.7386px;"></td>
</tr>
<tr>
<td style="width: 73.9205px;">TOC013C11</td>
<td style="width: 41.1932px;">USP20</td>
<td style="width: 26.6477px;">1-100</td>
<td style="width: 43.0114px;">pET28-MHL</td>
<td style="width: 37.5568px;">M9</td>
<td style="width: 49.375px;">6</td>
<td style="width: 72.1023px;">4.8 mg</td>
<td style="width: 44.8295px;">Y</td>
<td style="width: 84.8295px;">N</td>
<td style="width: 65.7386px;"></td>
</tr>
<tr>
<td style="width: 73.9205px;">TOC013D03</td>
<td style="width: 41.1932px;">USP20</td>
<td style="width: 26.6477px;">1-110</td>
<td style="width: 43.0114px;">pET28-MHL</td>
<td style="width: 37.5568px;">M9</td>
<td style="width: 49.375px;">6</td>
<td style="width: 72.1023px;">13.2 mg</td>
<td style="width: 44.8295px;">Y</td>
<td style="width: 84.8295px;">Y: small microcrystals SGCC08</p>
<p>&nbsp;</p>
<p>optimizing:</p>
<p>1) different protein concentrations (1, 2, 3, 4 mg/mL); no crystals</p>
<p>2) seeding with SGCC08; no crystals</p>
<p>&nbsp;</td>
<td style="width: 65.7386px;"></td>
</tr>
<tr>
<td style="width: 73.9205px;">TOC013D04</td>
<td style="width: 41.1932px;">USP20</td>
<td style="width: 26.6477px;">8-110</td>
<td style="width: 43.0114px;">pET28-MHL</td>
<td style="width: 37.5568px;">M9</td>
<td style="width: 49.375px;">6</td>
<td style="width: 72.1023px;">7.2 mg</td>
<td style="width: 44.8295px;">Y</td>
<td style="width: 84.8295px;">N</td>
<td style="width: 65.7386px;"></td>
</tr>
<tr>
<td style="width: 73.9205px;">TOC013-D07</td>
<td style="width: 41.1932px;">USP22</td>
<td style="width: 26.6477px;">43-142</td>
<td style="width: 43.0114px;">pET28-MHL</td>
<td style="width: 37.5568px;">M9</td>
<td style="width: 49.375px;">6</td>
<td style="width: 72.1023px;">No/low expression</td>
<td style="width: 44.8295px;">N</td>
<td style="width: 84.8295px;">N</td>
<td style="width: 65.7386px;"></td>
</tr>
<tr>
<td style="width: 73.9205px;">TOC013D08</td>
<td style="width: 41.1932px;">USP22</td>
<td style="width: 26.6477px;">49-142</td>
<td style="width: 43.0114px;">pET28-MHL</td>
<td style="width: 37.5568px;">M9</td>
<td style="width: 49.375px;">6</td>
<td style="width: 72.1023px;">No/low expression</td>
<td style="width: 44.8295px;">Y</td>
<td style="width: 84.8295px;">N</td>
<td style="width: 65.7386px;"></td>
</tr>
<tr>
<td style="width: 73.9205px;"></td>
<td style="width: 41.1932px;"></td>
<td style="width: 26.6477px;"></td>
<td style="width: 43.0114px;"></td>
<td style="width: 37.5568px;"></td>
<td style="width: 49.375px;"></td>
<td style="width: 72.1023px;"></td>
<td style="width: 44.8295px;"></td>
<td style="width: 84.8295px;"></td>
<td style="width: 65.7386px;"></td>
</tr>
<tr>
<td style="width: 708.466px;" colspan="10">Constructs previously available in SGC database that have high expression:</td>
</tr>
<tr>
<td style="width: 73.9205px;">SDC093D04</td>
<td style="width: 41.1932px;">USP13</td>
<td style="width: 26.6477px;">183-307</td>
<td style="width: 43.0114px;">pET28-MHL</td>
<td style="width: 37.5568px;">TB</td>
<td style="width: 49.375px;">6</td>
<td style="width: 72.1023px;">26.3 mg</td>
<td style="width: 44.8295px;"></td>
<td style="width: 84.8295px;"></td>
<td style="width: 65.7386px;"></td>
</tr>
<tr>
<td style="width: 73.9205px;">SDC093D05</td>
<td style="width: 41.1932px;">USP13</td>
<td style="width: 26.6477px;">187-301</td>
<td style="width: 43.0114px;">pET28-MHL</td>
<td style="width: 37.5568px;">TB</td>
<td style="width: 49.375px;">6</td>
<td style="width: 72.1023px;"></td>
<td style="width: 44.8295px;"></td>
<td style="width: 84.8295px;"></td>
<td style="width: 65.7386px;"></td>
</tr>
<tr>
<td style="width: 73.9205px;">SDC024F07</td>
<td style="width: 41.1932px;">USP16</td>
<td style="width: 26.6477px;">34-185</td>
<td style="width: 43.0114px;">pET28-LIC</td>
<td style="width: 37.5568px;">TB</td>
<td style="width: 49.375px;">6</td>
<td style="width: 72.1023px;"></td>
<td style="width: 44.8295px;"></td>
<td style="width: 84.8295px;"></td>
<td style="width: 65.7386px;"></td>
</tr>
<tr>
<td style="width: 73.9205px;">SDC024F09</td>
<td style="width: 41.1932px;">USP16</td>
<td style="width: 26.6477px;">16-185</td>
<td style="width: 43.0114px;">pET28-LIC</td>
<td style="width: 37.5568px;">TB</td>
<td style="width: 49.375px;">6</td>
<td style="width: 72.1023px;"></td>
<td style="width: 44.8295px;"></td>
<td style="width: 84.8295px;"></td>
<td style="width: 65.7386px;"></td>
</tr>
<tr>
<td style="width: 73.9205px;">SDC024F10</td>
<td style="width: 41.1932px;">USP16</td>
<td style="width: 26.6477px;">8-185</td>
<td style="width: 43.0114px;">pET28-LIC</td>
<td style="width: 37.5568px;">TB</td>
<td style="width: 49.375px;">6</td>
<td style="width: 72.1023px;"></td>
<td style="width: 44.8295px;"></td>
<td style="width: 84.8295px;"></td>
<td style="width: 65.7386px;"></td>
</tr>
<tr>
<td style="width: 73.9205px;">SDC036B06</td>
<td style="width: 41.1932px;">USP20</td>
<td style="width: 26.6477px;">1-141</td>
<td style="width: 43.0114px;">pET28-LIC</td>
<td style="width: 37.5568px;">TB</td>
<td style="width: 49.375px;">6</td>
<td style="width: 72.1023px;"></td>
<td style="width: 44.8295px;"></td>
<td style="width: 84.8295px;"></td>
<td style="width: 65.7386px;"></td>
</tr>
<tr>
<td style="width: 73.9205px;">SDC042G05</td>
<td style="width: 41.1932px;">USP33</td>
<td style="width: 26.6477px;">7-139</td>
<td style="width: 43.0114px;">pET28-LIC</td>
<td style="width: 37.5568px;">TB</td>
<td style="width: 49.375px;">6</td>
<td style="width: 72.1023px;"></td>
<td style="width: 44.8295px;"></td>
<td style="width: 84.8295px;"></td>
<td style="width: 65.7386px;"></td>
</tr>
<tr>
<td style="width: 73.9205px;">SDC042G06</td>
<td style="width: 41.1932px;">USP33</td>
<td style="width: 26.6477px;">7-167</td>
<td style="width: 43.0114px;">pET28-LIC</td>
<td style="width: 37.5568px;">TB</td>
<td style="width: 49.375px;">6</td>
<td style="width: 72.1023px;"></td>
<td style="width: 44.8295px;"></td>
<td style="width: 84.8295px;"></td>
<td style="width: 65.7386px;"></td>
</tr>
<tr>
<td style="width: 73.9205px;">SDC042G07</td>
<td style="width: 41.1932px;">USP33</td>
<td style="width: 26.6477px;">7-183</td>
<td style="width: 43.0114px;">pET28-LIC</td>
<td style="width: 37.5568px;">TB</td>
<td style="width: 49.375px;">6</td>
<td style="width: 72.1023px;"></td>
<td style="width: 44.8295px;"></td>
<td style="width: 84.8295px;"></td>
<td style="width: 65.7386px;"></td>
</tr>
<tr>
<td style="width: 73.9205px;">SDC168E10</td>
<td style="width: 41.1932px;">USP44</td>
<td style="width: 26.6477px;">1-167</td>
<td style="width: 43.0114px;">pET28-MHL</td>
<td style="width: 37.5568px;">TB</td>
<td style="width: 49.375px;">6</td>
<td style="width: 72.1023px;"></td>
<td style="width: 44.8295px;"></td>
<td style="width: 84.8295px;"></td>
<td style="width: 65.7386px;"></td>
</tr>
<tr>
<td style="width: 73.9205px;">SDC231F11</td>
<td style="width: 41.1932px;">USP51</td>
<td style="width: 26.6477px;">160-350</td>
<td style="width: 43.0114px;">pET28-LIC</td>
<td style="width: 37.5568px;">TB</td>
<td style="width: 49.375px;">6</td>
<td style="width: 72.1023px;"></td>
<td style="width: 44.8295px;"></td>
<td style="width: 84.8295px;"></td>
<td style="width: 65.7386px;"></td>
</tr>
<tr>
<td style="width: 73.9205px;">SDC231-G03</td>
<td style="width: 41.1932px;">USP51</td>
<td style="width: 26.6477px;">176-350</td>
<td style="width: 43.0114px;">pET28-LIC</td>
<td style="width: 37.5568px;">TB</td>
<td style="width: 49.375px;">6</td>
<td style="width: 72.1023px;"></td>
<td style="width: 44.8295px;"></td>
<td style="width: 84.8295px;"></td>
<td style="width: 65.7386px;"></td>
</tr>
</tbody>
</table>
<p>&nbsp;</p>
<p>&nbsp;</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/usp-zf-ubd-crystallography-pipeline-pt-2/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
	</channel>
</rss>
