<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>Scientist &#8211; openlabnotebooks.org</title>
	<atom:link href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/feed/" rel="self" type="application/rss+xml" />
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<description>A growing team of groundbreaking scientists around the world are now sharing their lab notebooks online</description>
	<lastBuildDate>Fri, 27 Nov 2020 20:40:26 +0000</lastBuildDate>
	<language>en-GB</language>
	<sy:updatePeriod>
	hourly	</sy:updatePeriod>
	<sy:updateFrequency>
	1	</sy:updateFrequency>
	<generator>https://wordpress.org/?v=5.5.3</generator>

<image>
	<url>https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/cropped-lab-jar02-32x32.png</url>
	<title>Scientist &#8211; openlabnotebooks.org</title>
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<width>32</width>
	<height>32</height>
</image> 
	<item>
		<title>Distinguishing between catalytic and non-catalytic pockets in the ligandable human genome</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/distinguishing-between-catalytic-and-non-catalytic-pockets-in-the-ligandable-human-genome/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/distinguishing-between-catalytic-and-non-catalytic-pockets-in-the-ligandable-human-genome/#respond</comments>
		
		<dc:creator><![CDATA[Evianne Rovers]]></dc:creator>
		<pubDate>Fri, 27 Nov 2020 20:30:07 +0000</pubDate>
				<category><![CDATA[Evianne Rovers]]></category>
		<category><![CDATA[Ligandable Genome]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=5422</guid>

					<description><![CDATA[My goal is to find druggable pockets in human enzymes that are non-catalytic. These non-catalytic druggable pockets may then be exploited for proximity pharmacology (ProxPharm), a novel paradigm in drug discovery where chimeric compounds bring two proteins in close proximity to elicit an effect of one protein on the other1. For example, PROTACs simultaneously bind <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/distinguishing-between-catalytic-and-non-catalytic-pockets-in-the-ligandable-human-genome/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p>My goal is to find druggable pockets in human enzymes that are non-catalytic. These non-catalytic druggable pockets may then be exploited for proximity pharmacology (ProxPharm), a novel paradigm in drug discovery where chimeric compounds bring two proteins in close proximity to elicit an effect of one protein on the other<sup>1</sup>. For example, PROTACs simultaneously bind an E3 ligase and a substrate protein, leading to ubiquitination and subsequent degradation of the substrate<sup>2</sup>. ProxPharm compounds should not inhibit the recruited enzyme (Figure 1). Therefore, a binding location which is not located in the catalytic domain, or is in the catalytic domain but distant from the catalytic site, is favored.</p>
<div id="attachment_5433" style="width: 730px" class="wp-caption aligncenter"><img aria-describedby="caption-attachment-5433" loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Figure-3-Labbookpost-1.png" alt="" width="720" height="405" class="wp-image-5433 size-full" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Figure-3-Labbookpost-1.png 720w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Figure-3-Labbookpost-1-300x169.png 300w" sizes="(max-width: 720px) 100vw, 720px" /><p id="caption-attachment-5433" class="wp-caption-text"><strong>Figure 1.</strong> ProxPharm compound binds enzyme (B) and substrate (A) without inhibiting the enzyme.</p></div>
<p>In our lab, Setayesh Yazdani initiated the Ligandable Genome project based on the project of Jiayan Wang et al.<sup>3</sup> Jiayan’s project analysed exclusively human protein structures <u>with bound ligands</u> and assessed the druggability of the corresponding protein domains that were occupied by at least one drug-like ligand in the PDB. Setayesh extended the analysis by mapping cavities in all human proteins in the PDB, whether or not they were bound to small-molecule ligands. She identified the pockets and calculated their properties by using the icmPocketFinder method in ICM (Molsoft, San Diego).</p>
<div id="attachment_5431" style="width: 1450px" class="wp-caption aligncenter"><img aria-describedby="caption-attachment-5431" loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Figure-1.-Labbook-post-1.png" alt="" width="1440" height="1440" class="wp-image-5431" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Figure-1.-Labbook-post-1.png 720w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Figure-1.-Labbook-post-1-300x300.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Figure-1.-Labbook-post-1-150x150.png 150w" sizes="(max-width: 1440px) 100vw, 1440px" /><p id="caption-attachment-5431" class="wp-caption-text"><strong>Figure </strong><strong>2</strong><strong>.</strong>  The workflow to distinguish between catalytic and non-catalytic pockets.</p></div>
<p>The next important step in this project is to distinguish between catalytic and non-catalytic pockets found by icmPocketfinder in human enzymes. To do this I measure the distance between catalytic/active site residues available from the Mechanism and Catalytic Site Atlas (M-CSA) database<sup>4 </sup>or available from the UniprotKB database<sup>5 </sup>and pockets found with IcmPocketFinder (Figure 2). One of the benefits from this approach is that the proximity of the pocket to the catalytic domain is quantified. Figure 3. shows that the minimum distance between the catalytic domain of 5’-nucleotidase (CD73) (PDB code 6TVE<sup>6</sup>) and pocket 13 is 13 Å.</p>
<div id="attachment_5446" style="width: 1376px" class="wp-caption alignnone"><img aria-describedby="caption-attachment-5446" loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Screen-Shot-2020-11-26-at-7.13.16-PM.png" alt="" width="1366" height="652" class="wp-image-5446 size-full" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Screen-Shot-2020-11-26-at-7.13.16-PM.png 1366w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Screen-Shot-2020-11-26-at-7.13.16-PM-300x143.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Screen-Shot-2020-11-26-at-7.13.16-PM-1024x489.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Screen-Shot-2020-11-26-at-7.13.16-PM-768x367.png 768w" sizes="(max-width: 1366px) 100vw, 1366px" /><p id="caption-attachment-5446" class="wp-caption-text"><strong>Figure 3.</strong> Discriminating between catalytic and non-catalytic pockets for 5’-nucleotidase (CD73) (PDB code 6TVE)<sup>6</sup>. Chimeric compounds targeting the purple pocket to recruit the enzyme to a new substrate are far from catalytic residues, and therefore unlikely to inhibit the enzyme. A) Pocket 13 (purple) distance to catalytic residues is 13.36 Å. B) Pocket 11 (green) distance to catalytic residues is 0.70 Å and distinguished as catalytic site.</p></div>
<p>Using the workflow and the methods listed above, catalytic residues could be assigned to 1487 human enzymes with structures in the PDB, corresponding to 19,510 PDB structures where I identified 140,276 pockets. In my next post, I will analyze which of these pockets are druggable, which are remote from the catalytic site, and will compile a list of enzymes that may be recruited by hetero-bifunctional small molecules to new substrates. If you would like to read the scripts and detailed description of the methods used in this project, please refer to my Zenodo report <a href="https://zenodo.org/record/4294090#.X8FkDy-71QI">here</a>.</p>
<p><strong>References:</strong></p>
<ol>
<li>Gerry, C. J. &amp; Schreiber, S. L. Unifying principles of bifunctional, proximity-inducing small molecules. <em>Nat. Chem. Biol.</em> <strong>16</strong>, 369–378 (2020).</li>
<li>Nalawansha, D. A. &amp; Crews, C. M. PROTACs: An Emerging Therapeutic Modality in Precision Medicine. <em>Cell Chem. Biol.</em> <strong>27</strong>, 998–1014 (2020).</li>
<li>Wang, J., Yazdani, S., Han, A. &amp; Schapira, M. Structure-based view of the druggable genome. <em>Drug Discov. Today</em> <strong>25</strong>, 561–567 (2020).</li>
<li>Ribeiro, A. J. M. <em>et al.</em> Mechanism and Catalytic Site Atlas (M-CSA): a database of enzyme reaction mechanisms and active sites. <em>Nucleic Acids Res.</em> <strong>46</strong>, D618–D623 (2018).</li>
<li>Consortium, T. U. UniProt: a worldwide hub of protein knowledge. <em>Nucleic Acids Res.</em> <strong>47</strong>, D506–D515 (2019).</li>
<li>Bhattarai, S. <em>et al.</em> 2-Substituted α,β-Methylene-ADP Derivatives: Potent Competitive Ecto-5′-nucleotidase (CD73) Inhibitors with Variable Binding Modes. <em>J. Med. Chem.</em> <strong>63</strong>, 2941–2957 (2020).</li>
</ol>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/distinguishing-between-catalytic-and-non-catalytic-pockets-in-the-ligandable-human-genome/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>Druggability and Genetic Variability of the ATPase Site and Central Channel of SARS-CoV-2 nsp13 Helicase Across Coronaviruses and SARS-CoV-2 Samples &#8211; Post 23</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/druggability-and-genetic-variability-of-the-atpase-site-and-central-channel-of-sars-cov-2-nsp13-helicase-across-coronaviruses-and-sars-cov-2-samples-post-23/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/druggability-and-genetic-variability-of-the-atpase-site-and-central-channel-of-sars-cov-2-nsp13-helicase-across-coronaviruses-and-sars-cov-2-samples-post-23/#comments</comments>
		
		<dc:creator><![CDATA[Setayesh Yazdani]]></dc:creator>
		<pubDate>Thu, 12 Nov 2020 07:56:23 +0000</pubDate>
				<category><![CDATA[COVID-19]]></category>
		<category><![CDATA[SARS-CoV-2 computational genomics]]></category>
		<category><![CDATA[Setayesh Yazdani]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=5388</guid>

					<description><![CDATA[One of the important enzymes in the replication cycle of the SARS-CoV-2 virus is the helicase, which is also known as non-structural protein 13 (nsp13). During the viral life cycle, the holo-RNA-dependent RNA polymerase, also known as nsp12, is thought to coordinate with several additional factors, including the nsp13 helicase (Snijder et al., 2016; Sola <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/druggability-and-genetic-variability-of-the-atpase-site-and-central-channel-of-sars-cov-2-nsp13-helicase-across-coronaviruses-and-sars-cov-2-samples-post-23/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p>One of the important enzymes in the replication cycle of the SARS-CoV-2 virus is the helicase, which is also known as non-structural protein 13 (nsp13). During the viral life cycle, the holo-RNA-dependent RNA polymerase, also known as nsp12, is thought to coordinate with several additional factors, including the nsp13 helicase (Snijder et al., 2016; Sola et al., 2015) as shown in figure 1.</p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/possible_targets_SARS-CoV-2.png" alt="" width="938" height="507" class="alignnone wp-image-5389 size-full" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/possible_targets_SARS-CoV-2.png 938w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/possible_targets_SARS-CoV-2-300x162.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/possible_targets_SARS-CoV-2-768x415.png 768w" sizes="(max-width: 938px) 100vw, 938px" /></p>
<p><strong>Figure 1. The possible targets for drug development against SARS-CoV-2.</strong> Adapted from Habtemariam et al., 2020.</p>
<p>The ATPase domain of nsp13 catalyzes an NTP-dependent unwinding of duplex RNAs into single strands in the 5’ to 3’ direction. (Habtemariam et al., 2020, Snijder et al., 2016) Previous reports on potent inhibitors targeting the ATPase domain of herpes simplex virus helicase suggest that helicase enzyme is chemically tractable. (Kleymann et al., 2002)</p>
<p>This post provides an overview of the druggability and amino acid variability of both the helicase ATP binding site and its central channel where the RNA molecule binds (figure 2) using the SARS-CoV-2 nsp13 helicase structures available from the protein data bank (PDB).</p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/atpase_site_central_channel_6zsl.png" alt="" width="975" height="645" class="alignnone wp-image-5390 size-full" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/atpase_site_central_channel_6zsl.png 975w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/atpase_site_central_channel_6zsl-300x198.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/atpase_site_central_channel_6zsl-768x508.png 768w" sizes="(max-width: 975px) 100vw, 975px" /></p>
<p><strong>Figure 2. Nsp13 helicase ATPase site and central channel (PDB: 6zsl).</strong> The RNA molecule is shown in ribbon diagram from the superimposed Upf1 complex (PDB code: 2xzl) and the ADP molecule shown in purple stick representation superimposed from SARS-CoV-2 nsp13 helicase structure bound to ADP (PDB: 6xez).</p>
<p>Using SiteMap (Schrodinger, NY) we assessed the druggability of the binding sites using the druggability score (Dscore). The ATP binding site of nsp13 and central channel have Dscores of 1.054 and 1.091 respectively. Both Dscores are indicative of druggable sites.</p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/druggability_nsp13.png" alt="" width="983" height="930" class="alignnone wp-image-5393 size-full" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/druggability_nsp13.png 983w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/druggability_nsp13-300x284.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/druggability_nsp13-768x727.png 768w" sizes="(max-width: 983px) 100vw, 983px" /></p>
<p><strong>Figure 3. Druggability analysis of SARS-CoV-2 nsp13 helicase ATPase site (top panel, PDB: 6xez chain F) and central channel (bottom pane, PDB: 6zsl). </strong></p>
<p>We then determined the residues that line the nsp13 helicase ATPase site and central channel using ICM PocketFinder <span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9">(</span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SpellingErrorV2 SCXW7658465 BCX9">Molsoft</span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9">, S</span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9">an Diego</span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9">) </span></span>and the nsp13 helicase structures. We identified 38 amino acid sidechains lining the nsp13 ATPase site (figure 5, <a href="https://doi.org/10.5281/zenodo.4269234">Zenodo report</a>) and 25 amino acid sidechains lining the central channel (figure 6, <a href="https://doi.org/10.5281/zenodo.4269234">Zenodo report</a>).</p>
<p>We conducted a broad survey of viral proteins from Alpha- and Betacoronaviruses to identify the most conserved druggable sites. By integrating variability and druggability data, we can then infer the most promising strategies for developing broad-spectrum viral inhibitors. This analysis can be valuable in the context of the emergence of future coronaviruses from circulating viral strains in bat reservoir species. We looked at twenty-seven reviewed nsp13 sequences from Uniprot, where six belong to Alphacoronavirus genus entries, and twenty-one belong to the Betacoronavirus genus. We then assessed the variability of amino acid lining the nsp13 helicase ATP binding site and its central channel by performing multiple sequence alignment.</p>
<p>We made diversity dendrograms focusing on the 38 sidechains of the ATPase site (figure 7, <a href="https://doi.org/10.5281/zenodo.4269234">zenodo report</a>) and the 25 sidechains of the central channel (figure 8, <a href="https://doi.org/10.5281/zenodo.4269234">zenodo report</a>) from step2. At the ATPase site, 26 out of 38 residues were found to be identical across these 27 entries which translates to 68% identity, indicating a relatively conserved site. At the nsp13 central channel, only 3 out of 25 residues were non-identical; 88% of residues were identical. This shows that the nsp13 helicase central channel where the RNA molecule binds is highly conserved.</p>
<p>We assessed the genetic variability of the ATPase site and the central channel of nsp13 among SARS-CoV-2 samples. Nicola De Maio, our collaborator from Nick Goldman’s lab at the European Bioinformatics Institute, looked at more than 39000 SARS-CoV-2 genome samples from COVID-19 patients and identified all mutations at the two sites of interest of nsp13 helicase. He identified 12 non-synonymous variants at the ATPase site (table 1, <a href="https://doi.org/10.5281/zenodo.4269234">zenodo report</a>) and no non-synonymous variant at the central channel of nsp13 helicase.</p>
<p>We conclude that the ATPase site of SARS-CoV-2 nsp13 helicase is less conserved than its RNA-binding central channel across Alpha- and Betacoronaviruses. We observe a similar pattern by looking at non-synonymous variants across SARS-CoV-2 samples from COVID-19 patients.  While the ATPase site where ADP binds is druggable and relatively conserved (%68), the central channel of nsp13 helicase where RNA binds is druggable and highly conserved (88%). Our finding supports the design of pan-coronavirus inhibitors targeting the central channel of nsp13 helicase.</p>
<p>You can view the full report on <a href="https://doi.org/10.5281/zenodo.4269234">Zenodo</a>.</p>
<p>Please contact me via the “Leave a comment” link at the top of this post. Stay Tuned for more updates on this project.</p>
<p><strong>References</strong></p>
<p>Habtemariam, S., Nabavi, S. F., Banach, M., Berindan-Neagoe, I., Sarkar, K., Sil, P. C., &amp; Nabavi, S. M. (2020). Should we try SARS-Cov-2 Helicase inhibitors for COVID-19 therapy? <em>Archives of Medical Research</em>, <em>51</em>(7), 733-735. <a href="https://doi.org/10.1016/j.arcmed.2020.05.024">https://doi.org/10.1016/j.arcmed.2020.05.024</a></p>
<p>Kleymann, G., Fischer, R., Betz, U. A., Hendrix, M., Bender, W., Schneider, U., Handke, G., Eckenberg, P., Hewlett, G., Pevzner, V., Baumeister, J., Weber, O., Henninger, K., Keldenich, J., Jensen, A., Kolb, J., Bach, U., Popp, A., Mäben, J., … Rübsamen-Waigmann, H. (2002). New helicase-primase inhibitors as drug candidates for the treatment of herpes simplex disease. <em>Nature Medicine</em>, <em>8</em>(4), 392-398. <a href="https://doi.org/10.1038/nm0402-392">https://doi.org/10.1038/nm0402-392</a></p>
<p>Snijder, E., Decroly, E., &amp; Ziebuhr, J. (2016). The Nonstructural proteins directing coronavirus RNA synthesis and processing. <em>Coronaviruses</em>, 59-126. <a href="https://doi.org/10.1016/bs.aivir.2016.08.008">https://doi.org/10.1016/bs.aivir.2016.08.008</a></p>
<p>Sola, I., Almazán, F., Zúñiga, S., &amp; Enjuanes, L. (2015). Continuous and discontinuous RNA synthesis in coronaviruses. <em>Annual Review of Virology</em>, <em>2</em>(1), 265-288. <a href="https://doi.org/10.1146/annurev-virology-100114-055218">https://doi.org/10.1146/annurev-virology-100114-055218</a></p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/druggability-and-genetic-variability-of-the-atpase-site-and-central-channel-of-sars-cov-2-nsp13-helicase-across-coronaviruses-and-sars-cov-2-samples-post-23/feed/</wfw:commentRss>
			<slash:comments>2</slash:comments>
		
		
			</item>
		<item>
		<title>Druggability and Genetic Variability of the ADP-bound Pocket of SARS-CoV-2 RNA-dependent RNA polymerase NiRAN domain Across Coronaviruses and SARS-CoV-2 Samples &#8211; Post 22</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/druggability-and-genetic-variability-of-the-adp-bound-pocket-of-sars-cov-2-rna-dependent-rna-polymerase-niran-domain-across-coronaviruses-and-sars-cov-2-samples-post-22/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/druggability-and-genetic-variability-of-the-adp-bound-pocket-of-sars-cov-2-rna-dependent-rna-polymerase-niran-domain-across-coronaviruses-and-sars-cov-2-samples-post-22/#respond</comments>
		
		<dc:creator><![CDATA[Setayesh Yazdani]]></dc:creator>
		<pubDate>Thu, 12 Nov 2020 03:39:52 +0000</pubDate>
				<category><![CDATA[COVID-19]]></category>
		<category><![CDATA[Ligandable Genome]]></category>
		<category><![CDATA[SARS-CoV-2 computational genomics]]></category>
		<category><![CDATA[Setayesh Yazdani]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=5384</guid>

					<description><![CDATA[The RNA-dependent RNA-polymerase (RdRp) also known as non-structural protein 12 (nsp12) is the target of antiviral agent remdesivir. Nsp12 has an important role in viral genome replication and transcription. (Chen et al., 2020)  Chen et al. identified a new pocket on the N-terminal extension of nsp12 occupied by ADP-Mg2+ after solving the structure of the helicase-polymerase complex. This ADP-bound pocket is on the N-terminal nidovirus RdRp-associated nucleotidyltransferase (NiRAN) domain which <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/druggability-and-genetic-variability-of-the-adp-bound-pocket-of-sars-cov-2-rna-dependent-rna-polymerase-niran-domain-across-coronaviruses-and-sars-cov-2-samples-post-22/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p><span data-contrast="auto">The RNA-dependent RNA-polymerase (RdRp) also known as non-structural protein 12 (nsp12) is the target of antiviral agent remdesivir. Nsp12 has an important role in viral genome replication and transcription. </span><span data-contrast="auto">(Chen et al., 2020)</span><span data-ccp-props="{&quot;201341983&quot;:0,&quot;335559739&quot;:160,&quot;335559740&quot;:259}"> </span></p>
<p><span data-contrast="auto">Chen et al. identified a new pocket on </span><span data-contrast="auto">the N-terminal extension of </span><span data-contrast="auto">nsp12 </span><span data-contrast="auto">occupied</span><span data-contrast="auto"> </span><span data-contrast="auto">by ADP-Mg2+ after solving the </span><span data-contrast="auto">structure of </span><span data-contrast="auto">the </span><span data-contrast="auto">helicase-polymerase complex. This </span><span data-contrast="auto">AD</span><span data-contrast="auto">P</span><span data-contrast="auto">-bound </span><span data-contrast="auto">pocket is on the N-terminal </span><span data-contrast="auto">nidovirus</span><span data-contrast="auto"> </span><span data-contrast="auto">RdRp</span><span data-contrast="auto">-associated </span><span data-contrast="auto">nucleotidyltransferase</span><span data-contrast="auto"> </span><span data-contrast="auto">(</span><span data-contrast="auto">NiRAN</span><span data-contrast="auto">) domain</span><span data-contrast="auto"> which is also well-resolved</span><span data-contrast="auto">.</span><span data-contrast="auto"> </span><span data-contrast="auto">(Chen et al., 2020)</span><span data-contrast="auto">.</span><span data-ccp-props="{&quot;201341983&quot;:0,&quot;335559739&quot;:160,&quot;335559740&quot;:259}"> </span></p>
<p><span data-contrast="auto">Based on </span><span data-contrast="auto">conservation profile</span><span data-contrast="auto"> across </span><span data-contrast="auto">nido</span><span data-contrast="auto">virales</span><span data-contrast="auto"> (an order of </span><span data-contrast="auto">positive-stranded RNA viruses</span><span data-contrast="auto"> including SARS-CoV-2)</span><span data-contrast="auto">, </span><span data-contrast="auto">the </span><span data-contrast="auto">NiRAN</span><span data-contrast="auto"> domain is beli</span><span data-contrast="auto">eved to have</span><span data-contrast="auto"> </span><span data-contrast="auto">nucleotidylation</span><span data-contrast="auto"> activity</span><span data-contrast="auto">, though the protein </span><span data-contrast="auto">substrates to which nucleotides are added </span><span data-contrast="auto">by the </span><span data-contrast="auto">NiRAN</span><span data-contrast="auto"> domain </span><span data-contrast="auto">are unknown</span><span data-contrast="auto">. </span><span data-contrast="auto"> </span><span data-contrast="auto">Reverse genetics was used to</span><span data-contrast="auto"> establish the importance of </span><span data-contrast="auto">NiRAN</span><span data-contrast="auto"> domain for </span><span data-contrast="auto">nidoviral</span><span data-contrast="auto"> </span><span data-contrast="auto">replication:</span><span data-contrast="auto"> m</span><span data-contrast="auto">utati</span><span data-contrast="auto">ng</span><span data-contrast="auto"> residues conserved in </span><span data-contrast="auto">NirAN</span><span data-contrast="auto"> domains i</span><span data-contrast="auto">nduces</span> <span data-contrast="auto">significant reduction of </span><span data-contrast="auto">SARS-CoV</span><span data-contrast="auto">-1</span><span data-contrast="auto"> and Equine viral arteritis (EAV) </span><span data-contrast="auto">viral titers in cell culture</span><span data-contrast="auto"> </span><span data-contrast="auto">(Lehmann et al., 2015)</span><span data-contrast="auto">.</span><span data-contrast="auto"> </span><span data-contrast="auto">Previous structural homology search of the SARS-CoV</span><span data-contrast="auto">-1</span><span data-contrast="auto"> nsp12 </span><span data-contrast="auto">NiRAN</span><span data-contrast="auto"> domain showed significant homology with protein kinases, including some key catalytic residues such as Asn209 and </span><span data-contrast="auto">NiRAN</span><span data-contrast="auto"> </span><span data-contrast="auto">conserved residues As218 and Phe219. (</span><span data-contrast="auto">Kirchdoerfer</span><span data-contrast="auto"> et al., 2019)</span><span data-ccp-props="{&quot;201341983&quot;:0,&quot;335559739&quot;:160,&quot;335559740&quot;:259}"> </span></p>
<p><span data-contrast="auto">For our project, w</span><span data-contrast="auto">e were interested in assessing the </span><span data-contrast="auto">druggability</span><span data-contrast="auto"> of this pocket and finding out how conserved is this </span><span data-contrast="auto">site both </span><span data-contrast="auto">across </span><span data-contrast="auto">past and </span><span data-contrast="auto">novel </span><span data-contrast="auto">coronaviruses and across SARS-CoV-2 samples</span><span data-contrast="auto"> </span><span data-contrast="auto">from COVID-19 patients. </span><span data-ccp-props="{&quot;201341983&quot;:0,&quot;335559739&quot;:160,&quot;335559740&quot;:259}"> </span></p>
<p><span class="TextRun SCXW204407608 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW204407608 BCX9">To achieve our first aim, we used a tool called sitemap (Schrodinger, NY) to assess the </span></span><span class="TextRun SCXW204407608 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SpellingErrorV2 SCXW204407608 BCX9">druggability</span></span><span class="TextRun SCXW204407608 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW204407608 BCX9"> of </span></span><span class="TextRun SCXW204407608 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW204407608 BCX9">the </span></span><span class="TextRun SCXW204407608 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW204407608 BCX9">ADP-bound pocket of </span></span><span class="TextRun SCXW204407608 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW204407608 BCX9">NiRAN</span></span><span class="TextRun SCXW204407608 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW204407608 BCX9"> domain of </span></span><span class="TextRun SCXW204407608 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SpellingErrorV2 SCXW204407608 BCX9">RdRp</span></span><span class="TextRun SCXW204407608 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW204407608 BCX9"> using the available nsp12 structure in </span></span><span class="TextRun SCXW204407608 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW204407608 BCX9">the </span></span><span class="TextRun SCXW204407608 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW204407608 BCX9">protein data bank (PDB code: </span></span><span class="TextRun SCXW204407608 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW204407608 BCX9">6xez</span></span><span class="TextRun SCXW204407608 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW204407608 BCX9">). </span></span>T<span class="TextRun SCXW2849510 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun CommentStart SCXW2849510 BCX0">he result indicate</span></span><span class="TextRun SCXW2849510 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW2849510 BCX0">s</span></span><span class="TextRun SCXW2849510 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW2849510 BCX0"> that this </span></span><span class="TextRun SCXW2849510 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW2849510 BCX0">highly polar </span></span><span class="TextRun SCXW2849510 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW2849510 BCX0">pocket</span></span><span class="TextRun SCXW2849510 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW2849510 BCX0"> is </span></span><span class="TextRun SCXW2849510 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW2849510 BCX0">a difficult </span></span><span class="TextRun SCXW2849510 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW2849510 BCX0">druggable </span></span><span class="TextRun SCXW2849510 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW2849510 BCX0">site </span></span><span class="TextRun SCXW2849510 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW2849510 BCX0">(</span></span><span class="TextRun SCXW2849510 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SpellingErrorV2 SCXW2849510 BCX0">Dscore</span></span><span class="TextRun SCXW2849510 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW2849510 BCX0"> =</span></span><span class="TextRun SCXW2849510 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW2849510 BCX0"> </span></span><span class="TextRun SCXW2849510 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW2849510 BCX0">0.85</span></span><span class="TextRun SCXW2849510 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW2849510 BCX0">)</span></span><span class="TextRun SCXW2849510 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW2849510 BCX0"> as shown in figure 1. (Halgren, 2009) </span></span><span class="EOP SCXW2849510 BCX0" data-ccp-props="{&quot;201341983&quot;:0,&quot;335559739&quot;:160,&quot;335559740&quot;:259}"> </span></p>
<p>&nbsp;</p>
<div id="attachment_5405" style="width: 985px" class="wp-caption alignnone"><img aria-describedby="caption-attachment-5405" loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/druggability_assessment_ADP-bound_pocket_NiRAN_domain_RdRp-1.png" alt="" width="975" height="471" class="wp-image-5405 size-full" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/druggability_assessment_ADP-bound_pocket_NiRAN_domain_RdRp-1.png 975w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/druggability_assessment_ADP-bound_pocket_NiRAN_domain_RdRp-1-300x145.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/druggability_assessment_ADP-bound_pocket_NiRAN_domain_RdRp-1-768x371.png 768w" sizes="(max-width: 975px) 100vw, 975px" /><p id="caption-attachment-5405" class="wp-caption-text">Figure 1. Druggability assessment of the ADP-bound pocket of NiRAN domain of RdRp.</p></div>
<p><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9">We picked 30 amino acid </span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9">residues </span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9">that li</span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9">n</span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9">e this</span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9"> site </span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9">of</span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9"> interest as identified by their proximity (~2.8</span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9">Å</span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9">) </span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9">to both</span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9"> the</span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9"> bound</span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9"> ADP</span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9"> molecule</span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9"> </span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9">and </span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9">the </span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9">V</span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9">a</span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9">n der Waals boundaries of the corresponding pocket</span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9"> </span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9">mapped by</span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9"> ICM</span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9">.</span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9"> (</span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SpellingErrorV2 SCXW7658465 BCX9">Molsoft</span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9">, S</span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9">an Diego</span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9">)</span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9">.</span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9"> (Figure 2, <a href="https://doi.org/10.5281/zenodo.4270496">Zenodo report</a>)</span></span></p>
<p><span class="TextRun SCXW159757282 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW159757282 BCX9">As part of our second aim, w</span></span><span class="TextRun SCXW159757282 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW159757282 BCX9">e conduct</span></span><span class="TextRun SCXW159757282 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW159757282 BCX9">ed</span></span><span class="TextRun SCXW159757282 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW159757282 BCX9"> a broad survey of viral proteins from Alpha- and </span></span><span class="TextRun SCXW159757282 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SpellingErrorV2 SCXW159757282 BCX9">Betacoronaviruses</span></span><span class="TextRun SCXW159757282 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW159757282 BCX9"> to identify the most conserved druggable sites. </span></span><span class="TextRun SCXW159757282 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW159757282 BCX9">By</span></span><span class="TextRun SCXW159757282 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW159757282 BCX9"> integrating variability and </span></span><span class="TextRun SCXW159757282 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SpellingErrorV2 SCXW159757282 BCX9">druggability</span></span><span class="TextRun SCXW159757282 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW159757282 BCX9"> data </span></span><span class="TextRun SCXW159757282 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW159757282 BCX9">we can</span></span><span class="TextRun SCXW159757282 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW159757282 BCX9"> infer the most promising strategies for developing broad-spectrum viral inhibitors</span></span><span class="TextRun SCXW159757282 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW159757282 BCX9"> against coronaviruses</span></span><span class="TextRun SCXW159757282 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW159757282 BCX9">. This analysis can be valuable in the context of the emergence of future coronaviruses circulating in bat reservoir species. </span></span><span class="EOP SCXW159757282 BCX9" data-ccp-props="{&quot;201341983&quot;:0,&quot;335559739&quot;:160,&quot;335559740&quot;:259}"> </span></p>
<p><span class="TextRun SCXW113687124 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW113687124 BCX9">To assess this site&#8217;s genetic variability, we made a diversity dendrogram by performing a multiple sequence alignment using reviewed sequences from </span></span><span class="TextRun SCXW113687124 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SpellingErrorV2 SCXW113687124 BCX9">Uniprot</span></span><span class="TextRun SCXW113687124 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW113687124 BCX9">, of which 6 are from the Alphacoronavirus genus and 21 from the </span><span class="NormalTextRun SpellingErrorV2 SCXW113687124 BCX9">Betacoronavirus</span><span class="NormalTextRun SCXW113687124 BCX9"> genus. (Figure </span></span><span class="TextRun SCXW113687124 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW113687124 BCX9">3</span></span><span class="TextRun SCXW113687124 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW113687124 BCX9">,<a href="https://doi.org/10.5281/zenodo.4270496"> Zenodo report</a>) <span class="TextRun SCXW93444954 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW93444954 BCX9">We observe that 21 out of </span></span><span class="TextRun SCXW93444954 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW93444954 BCX9">3</span></span><span class="TextRun SCXW93444954 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW93444954 BCX9">0</span></span><span class="TextRun SCXW93444954 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW93444954 BCX9"> residues are identical at this site of RdRp which translates into </span></span><span class="TextRun SCXW93444954 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW93444954 BCX9">70</span></span><span class="TextRun SCXW93444954 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW93444954 BCX9">% identity</span></span><span class="TextRun SCXW93444954 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW93444954 BCX9">. </span></span></span></span></p>
<p><span class="TextRun SCXW31895145 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW31895145 BCX9">As a follow-up to assessing the genetic variability among Alphacoronavirus and </span></span><span class="TextRun SCXW31895145 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SpellingErrorV2 SCXW31895145 BCX9">Betacoronavirus</span></span><span class="TextRun SCXW31895145 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW31895145 BCX9"> entries, </span></span><span class="TextRun SCXW31895145 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW31895145 BCX9">we also look</span></span><span class="TextRun SCXW31895145 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW31895145 BCX9">ed</span></span><span class="TextRun SCXW31895145 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW31895145 BCX9"> at the conservation of this site across SARS-CoV-2 samples. O</span></span><span class="TextRun SCXW31895145 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW31895145 BCX9">ur collaborator, Nicola De Maio, from Nick Goldman’s lab at the European Bioinformatics Institute has identified the non-synonymous mutations at </span></span><span class="TextRun SCXW31895145 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW31895145 BCX9">ADP-bound pocket of </span></span><span class="TextRun SCXW31895145 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW31895145 BCX9">NiRAN</span></span><span class="TextRun SCXW31895145 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW31895145 BCX9"> domain of </span></span><span class="TextRun SCXW31895145 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SpellingErrorV2 SCXW31895145 BCX9">RdRp</span></span><span class="TextRun SCXW31895145 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW31895145 BCX9"> </span></span><span class="TextRun SCXW31895145 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW31895145 BCX9">across more than </span></span><span class="TextRun SCXW31895145 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW31895145 BCX9">39000</span></span><span class="TextRun SCXW31895145 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW31895145 BCX9"> SARS-CoV-2 samples. Altogether he identified non-synonymous mutations at </span></span><span class="TextRun SCXW31895145 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW31895145 BCX9">6</span></span><span class="TextRun SCXW31895145 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW31895145 BCX9"> unique amino acid residues consisting of </span></span><span class="TextRun SCXW31895145 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW31895145 BCX9">6</span></span><span class="TextRun SCXW31895145 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW31895145 BCX9"> unique non-synonymous variants. In table 1 of the <a href="https://doi.org/10.5281/zenodo.4270496">Zenodo report</a>, there are details about the mutation events underlying these variants. </span></span><span class="EOP SCXW31895145 BCX9" data-ccp-props="{&quot;201341983&quot;:0,&quot;335559739&quot;:160,&quot;335559740&quot;:259}"> </span></p>
<p><b><span data-contrast="auto">Comparison with previously analyzed </span></b><b><span data-contrast="auto">RdRp </span></b><b><span data-contrast="auto">pockets, <a href="https://doi.org/10.5281/zenodo.4041302">RdRp active site,</a> and <a href="https://doi.org/10.5281/zenodo.4247424">allosteric site (M666 pocket</a>):</span></b><span data-ccp-props="{&quot;201341983&quot;:0,&quot;335559739&quot;:160,&quot;335559740&quot;:259}"> </span></p>
<p>We previously analyzed the druggability and conservation of the catalytic site and an ectopic site (that we called M666 pocket) of RdRp. We find that the and ectopic pocket (Dscore = 1.03) is more druggable than the catalytic pocket (Dscore 0.86) occupied by RNA-conjugated Rmedesivir and NiRAN pocket (Dscore=0.85).</p>
<p><span data-contrast="auto">On the other end, the catalytic site o</span><span data-contrast="auto">f</span><span data-contrast="auto"> </span><span data-contrast="auto">RdRp</span><span data-contrast="auto"> is among the most conserved </span><span data-contrast="auto">pockets </span><span data-contrast="auto">that </span><span data-contrast="auto">we have seen across SARS-CoV-2 protein structures</span><span data-contrast="auto"> (83% conserved </span><span data-contrast="auto">in coronaviruses)</span><span data-contrast="auto"> follow</span><span data-contrast="auto">ed by </span><span data-contrast="auto">the </span><span data-contrast="auto">NiRAN</span><span data-contrast="auto"> pocket (</span><span data-contrast="auto">68% conserved) and the M666 pocket (</span><span data-contrast="auto">33% conserved).</span><span data-ccp-props="{&quot;201341983&quot;:0,&quot;335559739&quot;:160,&quot;335559740&quot;:259}"> </span></p>
<p><span data-contrast="auto">We observe a related</span><span data-contrast="auto"> trend when looking at mutation rates a</span><span data-contrast="auto">cross COVID-19 samples:</span><span data-contrast="auto"> </span><span data-contrast="auto">t</span><span data-contrast="auto">he </span><span data-contrast="auto">catalytic site and </span><span data-contrast="auto">NiRAN</span><span data-contrast="auto"> pocket were genetically more stable, with</span><span data-contrast="auto"> 6 unique </span><span data-contrast="auto">mutations</span><span data-contrast="auto"> </span><span data-contrast="auto">found in</span><span data-contrast="auto"> 6</span><span data-contrast="auto"> </span><span data-contrast="auto">samples</span><span data-contrast="auto"> each</span><span data-contrast="auto">, while the M666 pocket w</span><span data-contrast="auto">as more </span><span data-contrast="auto">prone to mutations (10 mutations found in </span><span data-contrast="auto">13 </span><span data-contrast="auto">variants</span><span data-contrast="auto">)</span><span data-contrast="auto">.</span><span data-ccp-props="{&quot;201341983&quot;:0,&quot;335559739&quot;:160,&quot;335559740&quot;:259}"> </span></p>
<p><span class="TextRun SCXW154553115 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun CommentStart SCXW154553115 BCX0">A</span></span><span class="TextRun SCXW154553115 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW154553115 BCX0"> strategy may be to develop compounds </span></span><span class="TextRun SCXW154553115 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW154553115 BCX0">that include a chelating moiety to engage the Mg2+ </span></span><span class="TextRun SCXW154553115 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW154553115 BCX0">observed</span></span><span class="TextRun SCXW154553115 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW154553115 BCX0"> in</span></span><span class="TextRun SCXW154553115 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW154553115 BCX0"> the ADP-bound</span></span><span class="TextRun SCXW154553115 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW154553115 BCX0"> </span></span><span class="TextRun SCXW154553115 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW154553115 BCX0">NiRAN pocket</span></span><span class="TextRun SCXW154553115 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW154553115 BCX0">. </span></span><span class="TextRun SCXW154553115 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW154553115 BCX0">This site</span></span><span class="TextRun SCXW154553115 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW154553115 BCX0"> </span></span><span class="TextRun SCXW154553115 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW154553115 BCX0">has</span></span><span class="TextRun SCXW154553115 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW154553115 BCX0"> </span></span><span class="TextRun SCXW154553115 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW154553115 BCX0">reasonable genetic stability and is </span></span><span class="TextRun SCXW154553115 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW154553115 BCX0">potentially </span></span><span class="TextRun SCXW154553115 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW154553115 BCX0">an interesting </span></span><span class="TextRun SCXW154553115 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW154553115 BCX0">target for the development of </span></span><span class="TextRun SCXW154553115 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW154553115 BCX0">broad-spectrum</span></span><span class="TextRun SCXW154553115 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW154553115 BCX0"> inhibitors</span></span><span class="TextRun SCXW154553115 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW154553115 BCX0">. </span></span><span class="EOP SCXW154553115 BCX0" data-ccp-props="{&quot;201341983&quot;:0,&quot;335559739&quot;:160,&quot;335559740&quot;:259}"> </span></p>
<p>You can view this report on <a href="https://doi.org/10.5281/zenodo.4270496">Zenodo</a> as well.</p>
<p>Please contact me via the “Leave a comment” link at the top of this post. Stay Tuned for more updates on this project.</p>
<p><b><span data-contrast="auto">References</span></b><span data-ccp-props="{&quot;201341983&quot;:0,&quot;335559739&quot;:160,&quot;335559740&quot;:259}"> </span></p>
<p><span data-contrast="auto">Chen, J., Malone, B., Llewellyn, E., Grasso, M., Shelton, P. M., </span><span data-contrast="auto">Olinares</span><span data-contrast="auto">, P. D., Maruthi, K., </span><span data-contrast="auto">Eng</span><span data-contrast="auto">, E., </span><span data-contrast="auto">Vatandaslar</span><span data-contrast="auto">, H., Chait, B. T., Kapoor, T., Darst, S. A., &amp; Campbell, E. A. (2020). Structural basis for helicase-polymerase coupling in the SARS-Cov-2 replication-transcription complex. </span><a href="https://doi.org/10.1101/2020.07.08.194084"><span data-contrast="none">https://doi.org/10.1101/2020.07.08.194084</span></a><span data-ccp-props="{&quot;201341983&quot;:0,&quot;335559685&quot;:720,&quot;335559739&quot;:160,&quot;335559740&quot;:259,&quot;335559991&quot;:720}"> </span></p>
<p><span data-contrast="auto">Halgren, T. A. (2009). Identifying and characterizing binding sites and assessing </span><span data-contrast="auto">Druggability</span><span data-contrast="auto">. </span><i><span data-contrast="auto">Journal of Chemical Information and Modeling</span></i><span data-contrast="auto">, </span><i><span data-contrast="auto">49</span></i><span data-contrast="auto">(2), 377-389. </span><a href="https://doi.org/10.1021/ci800324m"><span data-contrast="none">https://doi.org/10.1021/ci800324m</span></a><span data-ccp-props="{&quot;201341983&quot;:0,&quot;335559685&quot;:720,&quot;335559739&quot;:160,&quot;335559740&quot;:259,&quot;335559991&quot;:720}"> </span></p>
<p><span data-contrast="auto">Kirchdoerfer</span><span data-contrast="auto">, R. N., &amp; Ward, A. B. (2019). Structure of the SARS-</span><span data-contrast="auto">Cov</span><span data-contrast="auto"> NSP12 polymerase bound to NSP7 and NSP8 Co-factors. </span><a href="https://doi.org/10.1101/551986"><span data-contrast="none">https://doi.org/10.1101/551986</span></a><span data-ccp-props="{&quot;201341983&quot;:0,&quot;335559685&quot;:720,&quot;335559739&quot;:160,&quot;335559740&quot;:259,&quot;335559991&quot;:720}"> </span></p>
<p><span data-contrast="auto">Lehmann, K. C., Gulyaeva, A., </span><span data-contrast="auto">Zevenhoven</span><span data-contrast="auto">-Dobbe, J. C., Janssen, G., Ruben, M., </span><span data-contrast="auto">Overkleeft</span><span data-contrast="auto">, H. S., Van </span><span data-contrast="auto">Veelen</span><span data-contrast="auto">, P. A., </span><span data-contrast="auto">Samborskiy</span><span data-contrast="auto">, D., Kravchenko, A. A., </span><span data-contrast="auto">Leontovich</span><span data-contrast="auto">, A. M., Sidorov, I. A., Snijder, E. J., Posthuma, C. C., &amp; </span><span data-contrast="auto">Gorbalenya</span><span data-contrast="auto">, A. E. (2015). undefined. </span><i><span data-contrast="auto">Nucleic Acids Research</span></i><span data-contrast="auto">, </span><i><span data-contrast="auto">43</span></i><span data-contrast="auto">(17), 8416-8434. </span><a href="https://doi.org/10.1093/nar/gkv838"><span data-contrast="none">https://doi.org/10.1093/nar/gkv838</span></a><span data-contrast="none"> </span><span data-ccp-props="{&quot;201341983&quot;:0,&quot;335559685&quot;:720,&quot;335559739&quot;:160,&quot;335559740&quot;:259,&quot;335559991&quot;:720}"> </span></p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/druggability-and-genetic-variability-of-the-adp-bound-pocket-of-sars-cov-2-rna-dependent-rna-polymerase-niran-domain-across-coronaviruses-and-sars-cov-2-samples-post-22/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>Interrogating PR/SET domains for the ability to bind SAH</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/interrogating-pr-set-domains-for-the-ability-to-bind-sah/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/interrogating-pr-set-domains-for-the-ability-to-bind-sah/#respond</comments>
		
		<dc:creator><![CDATA[Danton Ivanochko]]></dc:creator>
		<pubDate>Mon, 09 Nov 2020 14:37:06 +0000</pubDate>
				<category><![CDATA[Cheryl Arrowsmith (Univ. of Toronto)]]></category>
		<category><![CDATA[Danton Ivanochko]]></category>
		<category><![CDATA[PRDM methyltransferases]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=5380</guid>

					<description><![CDATA[This is an excerpt from my PhD thesis where I investigated 13 human PR/SET domains from the PRDM protein family and assessed their ability to bind to a fluorinated SAH analog. The research helps to identify which of the PRDMs could be methyltransferases. This work was carried out in the laboratory of Dr. Cheryl Arrowsmith <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/interrogating-pr-set-domains-for-the-ability-to-bind-sah/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<blockquote><p>This is an excerpt from my PhD thesis where I investigated 13 human PR/SET domains from the PRDM protein family and assessed their ability to bind to a fluorinated SAH analog. The research helps to identify which of the PRDMs could be methyltransferases. This work was carried out in the laboratory of Dr. Cheryl Arrowsmith at the University of Toronto and the University Health Network.</p></blockquote>
<h1><strong>Interrogating PR/SET domains for the ability to bind SAH</strong></h1>
<p><strong><br />
</strong><br />
<strong>Authorship attribution:<br />
</strong>This chapter contains unpublished work and I am responsible for the overall generation and interpretation of the data included herein with the following mentions: The F-SAH compound was synthesized and kindly provided by Carlos Zepeda, Ontario Institute for Cancer Research (<em>OICR</em>). Shili Duan provided guidance and technical assistance in protein production. Additional proteins were kindly provided by Taraneh Hajian, Elisa Gibson and Dr. Masoud Vedadi from the SGC Toronto. Dr. Scott Houliston advised and supported all NMR experiments, and Drs. Houliston and Cheryl Arrowsmith aided in data analysis and provided guidance throughout the study. This project was prematurely halted due to a catastrophic freezer malfunction during the COVID-19 shutdown.</p>
<p>&nbsp;</p>
<p><strong><u>Introduction </u></strong><strong><br />
</strong><br />
Lysine methyltransferases are a group epigenetic regulators currently being investigated in clinical and pharmaceutical studies for the treatment of certain cancers (Rugo, et al., 2020) (Ilango, et al., 2020) (Hoy, 2020) (Dilworth &amp; Barsyte-Lovejoy, 2019). The majority of all KMT proteins possess a domain with a SET-fold (Schapira, 2011). The PRDM protein family is characterized by the presence of a PRDI-BF1 and RIZ1 (PR/SET) domain, which is a subgroup of the SET domain (<strong>FIGURE A1</strong>). Currently KMT activity has been reported for some PRDMs (<strong>TABLE A1</strong>), while some PRDMs are thought to be inactive pseudoenzymes. Since PRDMs are less represented in the academic literature compared to canonical SET proteins (<strong>FIGURE A4</strong>), it is unclear if these PRDMs are truly pseudoenzyme or if their KMT activity is only yet to be characterized. Distinguishing which PRDMs possess KMT activity would further our understanding of lysine epigenetics and pseudoenzyme-driven biology (Arrowsmith, et al., 2012) (Ribeiro, et al., 2019).</p>
<p>All domains with a SET-fold interact with SAM through a conserved hydrogen bonding network split across two distinct domain regions (Campagna-Slater, et al., 2011). These SAM binding residues in the canonical SET-domains are well conserved, however the corresponding residues in PR/SET domains are less well conserved (<strong>FIGURE A2</strong>) (H-binding clusters at Regions 1 and 2)]. The only holo-enzyme structure of a PR/SET domains is of mouse PRDM9 bound to a histone H3 peptide and SAH (Wu, et al., 2013). The human and mouse PRDM9 PR/SET domains possess many of the highly conserved SAM-binding residues found in the canonical SET-domains. Lacking any structural evidence, it is unclear if and how many of the PR/SET domains may bind to SAM. Therefore, detecting cofactor binding in PR/SET domains would aid in distinguishing whether certain PRDMs are enzymes or pseudoenzymes.</p>
<p>Here I investigate cofactor binding to understand which PR/SET domains may have the ability to catalyze the KMT reaction. Using <sup>19</sup>F-nuclear magnetic resonance (NMR) spectroscopy, we measured the binding response for the majority of the human PR/SET domains with a fluorinated analog of the methyltransferase by-product S-adenosylhomocysteine (SAH), 2-fluoro-SAH (F-SAH). <sup>19</sup>F-NMR is a sensitive and versatile method to study protein-ligand interactions (Dalvit &amp; Vulpetti, 2019). Using the F-SAH binding response with the PRDM9 PR/SET domain as a positive control for the screening assays, we identified several PR/SET domains that bind F-SAH and therefore likely bind to SAM. Additionally, we detected several PR/SET domains that do not show any evidence of binding to F-SAH, which suggests a lesser affinity for SAM or potentially no affinity. This dataset can be used as a starting point for future discovery of KMT activity in PRDM proteins.</p>
<p><strong><u>RESULTS</u></strong></p>
<p><strong>Establishment of the F-SAH binding assay</strong></p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/FIGURE_K1-Copy-295x300.png" alt="" class="alignnone wp-image-5381" width="955" height="971" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/FIGURE_K1-Copy-295x300.png 295w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/FIGURE_K1-Copy-1008x1024.png 1008w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/FIGURE_K1-Copy-768x780.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/FIGURE_K1-Copy-1512x1536.png 1512w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/FIGURE_K1-Copy.png 1887w" sizes="(max-width: 955px) 100vw, 955px" /><br />
<strong>Figure K1. Characterization of F-SAH binding to PRDM9. </strong>(<strong>a</strong>) Chemical structure of 2-fluoro-SAH (F-SAH) and (<strong>b</strong>) model of F-SAH bound to PRDM9 based on a SAM-bound structure (PDBID: 6nm4). Colour coding corresponds to (<strong>a</strong>). The effects of F-SAH binding to PRDM9 were measured by <sup>19</sup>F-NMR. (<strong>c-d</strong>) Spectra of 16 µM F-SAH alone and with 1X and 5X molar equivalents of PRDM9 display a concentration dependent decrease in the unbound F-SAH peak and the emergence of a bound F-SAH peak.</p>
<p>&nbsp;</p>
<p>We utilized <sup>19</sup>F nuclear magnetic resonance spectroscopy (NMR) spectroscopy to assess whether PR/SET domains may have the ability to bind to the methyltransferase enzyme cofactor. Fluorinated SAH was chosen, rather than fluorinated SAM because SAH and its chemical precursors are more stable across a wide pH range and were therefore more suited to chemical synthesis. The chemical structure of 2-fluoro-SAH (F-SAH) shows that SAH is fluorinated at the 2-position of the adenine ring<strong> (FIGURE K1a)</strong>. Using our structure of the human PRDM9 PR/SET domain bound by SAM, we generated a theoretical model of F-SAH bound to PRDM9 <strong>(FIGURE K1b)</strong>. We observed that the fluorine atom projects toward the solvent and therefore is unlikely to alter the interaction with the protein. Nevertheless, because fluorine’s chemical shift is highly sensitive to any change in its microenvironment, we expected to see perturbation and/or broadening of the <sup>19</sup>F resonance when F-SAH binds to a PR/SET domain.</p>
<p>First, we investigated F-SAH alone by measuring a 1D <sup>19</sup>F spectrum of F-SAH, which gave rise to a single peak at ~-52.4 ppm. Next, we monitored for F-SAH binding to PRDM9, which is the most extensively characterized methyltransferase among all the PR/SET domain containing proteins (Eram, et al., 2014) (Blazer, et al., 2016) (Allali-Hassani, et al., 2019) (Powers, et al., 2016). We assessed the F-SAH NMR signal alone and in the presence of equal and 5-fold molar equivalents of PRDM9 PR/SET domain, we detected a concentration-dependent decrease in the intensity of the unbound F-SAH <sup>19</sup>F resonance <strong>(FIGURE K1c-d)</strong>.  Moreover, we observed the emergence of a second <sup>19</sup>F signal (~ 1045 Hz up-field at -54.3 ppm) that is from protein-bound F-SAH. As these results were reproducible using two separate PRDM9 protein preparations, we selected 5-fold molar equivalents of PR/SET domain to F-SAH for further experiments.</p>
<p><strong>Screening human PR/SET domains with the F-SAH binding assay</strong></p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/FIGURE_K2-Copy-204x300.png" alt="" class="alignnone wp-image-5382" width="869" height="1278" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/FIGURE_K2-Copy-204x300.png 204w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/FIGURE_K2-Copy-1391x2048.png 1391w" sizes="(max-width: 869px) 100vw, 869px" /><br />
<strong>Figure K2. Screening for F-SAH binders among PR domains.</strong> (<strong>a</strong>) <sup>19</sup>F-NMR spectra showing 16 µM F-SAH in the presence of 80 µM of the indicated PR/SET domains. (<strong>b</strong>) Plots of <sup>19</sup>F peak integrals for F-SAH in the presence of individual PR/SET domains. All <sup>19</sup>F peaks were normalized to the peak integral of F-SAH alone in buffer and the normalized integrals of the Bound and 1-Unbound peaks relative to F-SAH alone were plotted (i.e. F-SAH alone would be located at position [0,0]).</p>
<p>&nbsp;</p>
<p>We performed F-SAH binding assays with PR/SET domains from 13 of the 19 human PRDM family members. The PR/SET domains of PRDM 1, 2, 4, 5, 6, 11, 12, 14 and 17 were provided by the SGC Toronto and I purified PRDM 3, 7, 9 and 16. I was able to purify PR/SET protein constructs for PRDM 8, 10, 13 and 15, however these constructs suffered stability issues in our <sup>19</sup>F-NMR buffer during dialysis (see Methods) and were therefore not suitable for further study. We collected <sup>19</sup>F-NMR spectra for F-SAH with 5-fold molar equivalents of each of the 13 human PR/SET domains (<strong>Figure K2a</strong>). The relative intensities of the unbound and bound F-SAH peaks varied considerably among the different PR/SET proteins indicating variable binding affinities across the panel of PRDM proteins. To quantify the binding for each PRDM PR/SET domain, we normalized the peak integrals of the unbound and bound peaks for each protein to the unbound peak of F-SAH alone. Specifically, we divided the integral of each peak by the integral of the peak for F-SAH alone. We plotted the normalized integrals of the 1 minus the unbound peak against the bound peak for each PR/SET domain to differentiate each protein’s affinity for F-SAH (<strong>Figure K2b</strong>). PR/SET datapoints located in the upper, right region of the plot displayed strong binding to F-SAH, while datapoints at the origin indicate no evidence of binding. We used the F-SAH signal in the presence of PRDM9 as an internal control to distinguish a strong binding response. PRDM 4 and 5 were plotted near to PRDM9, indicating a strong evidence response. Additionally, PRDM 14, 12, 7, 11 and 17 exhibited a very strong binding response. We were unable to detect any binding response for F-SAH in the presence of PRDM 3, 1 or 16, using our assay conditions, though we did detected a weak binding response for PRDM 2 and 6. Taken together, this data provides evidence to help prioritize further assays to identify methyltransferase activity among the PRDM family members.<em><br />
</em></p>
<p><strong><u>Discussion</u></strong><strong><br />
</strong><br />
<strong>Using F-SAH to probe KMT domains</strong></p>
<p>Here we used <sup>19</sup>F-NMR to investigate the potential for 13 of the 19 human PR/SET domains to bind a fluorinated analog of the methylation reaction by-product SAH. Using our F-SAH binding assays with PR/SET domains, we identified 7 domains (PRDM 4, 5, 7, 11, 12, 14 and 17) that showed a binding response similar to, or stronger than PRDM9. Of these ‘binding-competent’ domains only PRDM 9 and 7 have reported KMT activity, with accompanying enzyme kinetics data (Eram, et al., 2014) (Blazer, et al., 2016). Interestingly, PRDM7 showed stronger binding to F-SAH compared to PRDM9 (<strong>Figure K2</strong>). PRDM7 possesses much weaker KMT activity for H3K4me2 substrate peptide compared to PRDM9, with a 100-fold lower rate of catalysis (<em>k</em><sub>cat</sub>=1.9&#215;10<sup>2</sup>/<em>h </em>vs 1.9&#215;10<sup>4</sup>/h, for PRDM 7 and 9, respectively) (Blazer, et al., 2016) (Eram, et al., 2014). These catalytic rate differences are mostly attributed to the distinct capacities to facilitate the methyl cation transferase by the catalytic residues (Y357 and S357, in PRDM 9 and 7, respectively). Based on our data, another contributing factor could be that PRDM7 has a stronger affinity for the reaction by-product SAH, which inhibits the catalytic rate. In our F-SAH binding assays, PRDM 2 and 6 showed weak, but detectable, binding compared to PRDM9 (<strong>Figure K2</strong>). KMT activities towards H3K9 and H4K20 have been reported for PRDM 2 and 6, respectively (Kim, et al., 2003) (Congdon, et al., 2014) (Wu, et al., 2008). This indicates that a weak binding response for F-SAH is associated with known active KMTs. Finally, in all cases where we observed F-SAH binding, we measured a consistent up-field shift (~1040 Hz) in the F-SAH <sup>19</sup>F resonance (<strong>Figure K2a</strong>). This indicates there is a common binding modality with a highly similar microenvironment formed in the bound state across the family.</p>
<p>We also demonstrated the utility of F-SAH as a synthetic chemical probe for methyltransferase proteins. Our approach is comparable to a previous study that reported the binding of the MLL1 SET-domain to a SAM-analog conjugated with a fluorescent tag (Luan, et al., 2016). Comparatively, F-SAH benefits from the smaller size of fluorine, which is positioned on the adenine ring in an orientation that is unlikely to interfere with binding (<strong>Figure K2a-b</strong>). Additionally, <sup>19</sup>F-NMR is capable of detecting low affinity protein-ligand interactions, such as is required for fragment-based screening (Dalvit &amp; Vulpetti, 2019). We surveyed two publicly available databases that quantify protein-ligand interaction data (<strong>TABLE K1</strong>). We found that SET-fold containing proteins have a range of affinities for SAH with dissociation constants (K<sub>D</sub>) from low µM to high nM, as well as a similar range for reported half-maximal inhibition (IC<sub>50</sub>) by SAH. Only RBCMT had affinity data for both SAH and SAM, showing that SAM binds RBCMT with an ~70-fold higher affinity when compared with SAH  (Couture, et al., 2006). The authors reported conserved binding modes in crystal structures of RBCMT-SAM and RBCMT-SAH implying that the variations in affinity could not be attributed to the binding conformations, but may rather be attributed to a higher entropic penalty for bound SAH, which possesses additional bond rotational freedom compared to SAM (Couture, et al., 2006). Taken together, we can predict that the upper-limit for the affinity of PRDM9 for F-SAH is less than the known affinity of SAM, which is 18.6 µM (Eram, et al., 2014).</p>
<p><strong>Table K1. Known dissociation constant (K<sub>D</sub>) and half maximal inhibition values (IC<sub>50</sub>) of SAM and SAH for SET domain proteins. </strong>Binding affinity data available from the BindingDB (www.bindingdb.org) and BindingMOAD (www.bindingmoad.org/) was accessed from the RCSB PDB (www.rcsb.org). Note that BindingDB reports a large range of IC<sub>50</sub> values for EZH2.</p>
<table width="95%">
<tbody>
<tr>
<td style="width: 95.5333px;"><strong>Protein</strong></td>
<td style="width: 207.883px;"><strong>Source</strong></td>
<td style="width: 115.967px;"><strong>Ligand</strong></td>
<td style="width: 115.967px;"><strong>K<sub>D</sub> (µM)</strong></td>
<td style="width: 115.967px;"><strong>IC<sub>50</sub> (µM)</strong></td>
<td style="width: 228.283px;"><strong>Source</strong></td>
</tr>
<tr>
<td style="width: 95.5333px;">EHMT1</td>
<td style="width: 207.883px;"><em>Homo sapiens</em></td>
<td style="width: 115.967px;">SAH</td>
<td style="width: 115.967px;"></td>
<td style="width: 115.967px;">2.3</td>
<td style="width: 228.283px;">BindingDB</td>
</tr>
<tr>
<td style="width: 95.5333px;">EHMT2</td>
<td style="width: 207.883px;"><em>Homo sapiens</em></td>
<td style="width: 115.967px;">SAH</td>
<td style="width: 115.967px;">0.57</td>
<td style="width: 115.967px;">2.0</td>
<td style="width: 228.283px;">BindingDB</td>
</tr>
<tr>
<td style="width: 95.5333px;">EZH2</td>
<td style="width: 207.883px;"><em>Homo sapiens</em></td>
<td style="width: 115.967px;">SAH</td>
<td style="width: 115.967px;"></td>
<td style="width: 115.967px;">0.263-16.6</td>
<td style="width: 228.283px;">BindingDB</td>
</tr>
<tr>
<td style="width: 95.5333px;">KMT2A</td>
<td style="width: 207.883px;"><em>Homo sapiens</em></td>
<td style="width: 115.967px;">SAH</td>
<td style="width: 115.967px;"></td>
<td style="width: 115.967px;">2.3</td>
<td style="width: 228.283px;">BindingDB</td>
</tr>
<tr>
<td style="width: 95.5333px;">Kmt5b</td>
<td style="width: 207.883px;"><em>Mus musculus</em></td>
<td style="width: 115.967px;">SAM</td>
<td style="width: 115.967px;">11.2</td>
<td style="width: 115.967px;"></td>
<td style="width: 228.283px;">BindingMOAD</td>
</tr>
<tr>
<td style="width: 95.5333px;">Kmt5c</td>
<td style="width: 207.883px;"><em>Mus musculus</em></td>
<td style="width: 115.967px;">SAH</td>
<td style="width: 115.967px;">17.2</td>
<td style="width: 115.967px;">10</td>
<td style="width: 228.283px;">BindingDB</td>
</tr>
<tr>
<td style="width: 95.5333px;">PRDM9</td>
<td style="width: 207.883px;"><em>Homo sapiens</em></td>
<td style="width: 115.967px;">SAM</td>
<td style="width: 115.967px;">18.6</td>
<td style="width: 115.967px;"></td>
<td style="width: 228.283px;">(Eram, et al., 2014)</td>
</tr>
<tr>
<td style="width: 95.5333px;">RBCMT</td>
<td style="width: 207.883px;"><em>Pisum sativum</em></td>
<td style="width: 115.967px;">SAM</td>
<td style="width: 115.967px;">0.29</td>
<td style="width: 115.967px;"></td>
<td style="width: 228.283px;">BindingMOAD</td>
</tr>
<tr>
<td style="width: 95.5333px;">RBCMT</td>
<td style="width: 207.883px;"><em>Pisum sativum</em></td>
<td style="width: 115.967px;">SAH</td>
<td style="width: 115.967px;">21.2</td>
<td style="width: 115.967px;"></td>
<td style="width: 228.283px;">BindingMOAD</td>
</tr>
<tr>
<td style="width: 95.5333px;">SETD7</td>
<td style="width: 207.883px;"><em>Homo sapiens</em></td>
<td style="width: 115.967px;">SAH</td>
<td style="width: 115.967px;"></td>
<td style="width: 115.967px;">30</td>
<td style="width: 228.283px;">BindingDB</td>
</tr>
<tr>
<td style="width: 95.5333px;">mSMYD2</td>
<td style="width: 207.883px;"><em>Mus musculus</em></td>
<td style="width: 115.967px;">SAH</td>
<td style="width: 115.967px;"></td>
<td style="width: 115.967px;">0.18</td>
<td style="width: 228.283px;">BindingDB</td>
</tr>
<tr>
<td style="width: 95.5333px;">SMYD2</td>
<td style="width: 207.883px;"><em>Homo sapiens</em></td>
<td style="width: 115.967px;">SAH</td>
<td style="width: 115.967px;"></td>
<td style="width: 115.967px;">0.18</td>
<td style="width: 228.283px;">BindingDB</td>
</tr>
</tbody>
</table>
<p><strong>Interpreting the absence of binding evidence</strong></p>
<p>We identified that PRDM 1, 3, and 16 do not bind to F-SAH in our assay conditions, which is suggestive of the inability to bind to SAM. PRDM1 (initially identified as Blimp-1) is widely understood to lack KMT activity and instead functions as a gene repressor by recruiting specific co-repressors proteins to PRDM1-binding sites throughout the genome (Minnich, et al., 2016). PRDM1 does affect histone methylation, but it does so through interactions with the KMT protein G9a/EHMT2 (Gyory, et al., 2004), the arginine methyltransferase PRMT5 (Ancelin, et al., 2006) and the lysine demethylase LSD1 (Su, et al., 2009). Whether the PR/SET domain of PRDM1 functions as a “reader domain” for specific histone tail modifications is undetermined, however our data suggests that it is unlikely that SAH or SAM would play a role in this ability.</p>
<p>We found that the PR/SET domains of PRDM 3 and 16 do not bind to F-SAH under our assay conditions. However, this does not exclude the possibility that larger or differently generated constructs may possess the ability to bind SAM or SAH. For instance, longer or full-length PRDM 3 and 16 protein constructs were reported to possess KMT activity when purified from either HeLa cells and mouse fibroblasts or SF9 insect cells (Pinheiro, et al., 2012) (Zhou, et al., 2016). Our PRDM 3 and 16 constructs were purified from E. coli and each contained the PR/SET domain along with the first proximal C-terminal zinc-finger motif (ZnF1), which is similar to previous structural and enzyme characterization studies of the PR/SET and ZnF1 of PRDM9 (Eram, et al., 2014) (Wu, et al., 2013). It is unclear if the additional regions outside the PRDM 3 and 16 PR/SET domains and ZnF1 could function to stabilize the cofactor binding site in the PR/SET domain. Interestingly, a previously solved NMR solution structure of PRDM16 demonstrated the absence of internal β-strands that are present in crystal and solution structures of all other PR/SET domains, which could possibly destabilize the cofactor and substrate binding sites (<strong>FIGURE A3d</strong>). Furthermore, it is unclear if additional factors present in eukaryotic expression systems enable cofactor binding or enzymatic activity.</p>
<p><strong>Future analysis of PRDMs<br />
</strong><br />
Future investigation to complete the F-SAH binding screen with the untested members of the PRDM family (PRDM 8, 10, 13, 15, FOG1, and FOG2) would provide valuable data.  Additionally, assaying F-SAH binding in competition with (unlabeled) SAM could qualitatively delineate the affinities of the methyltransferase reaction cofactor and by-product, providing further evidence of potential enzymatic function. We detected F-SAH binding in PRDM 4, 5, 11, 12, 14, and 17, and we could not find any studies reporting KMT activity with these proteins. Intriguingly, PRDM 4, 5, 11, 12, and 14 all possess the highly conserved “catalytic tyrosine” (<strong>FIGURE A2</strong>), which is further evidence supporting KMT activity for these proteins. Importantly, several studies have attributed oncogenic roles for PRDM 4, 12 and 14 in specific cancers (Casamassimi, et al., 2020). Future discovery of enzymatic function in any of these PR/SET domains could provide a novel, actionable target for oncological drug discovery.</p>
<p>&nbsp;</p>
<p><strong><u>Methods</u></strong><strong><br />
</strong><strong>Protein production </strong><br />
PRDM9 (195-385) and PRDM7 (195-392) with an N-terminal 6xHis-tag and TEV protease site were expressed in <em>Escherichia coli</em> BL21 (DE3) codon plus cells from a pET28-MHL vector. PRDM3 (69-235) and PRDM16 (73-256) with an N-terminal 6xHis-tag and TEV protease site were expressed in <em>Escherichia coli</em> BL21 (DE3) codon plus cells from a pET15b/MHL vector. Cell were grown at 37°C in M9 minimal medium in the presence of 50 µg/ml of kanamycin (PRDM 7 and 9) or 100 µg/ml of ampicillin (PRDM 3 and 16) to an OD600 of 0.8 and induced by isopropyl-1-thio-D-galactopyranoside (IPTG), final concentration 0.5 mM and incubated overnight at 15°C. Cells were harvested by centrifugation at 7,000 rpm and cell pellets were stored at -80°C. Cell were lyzed on ice in lysis buffer (20 mM Tris pH 7.5, 300 mM NaCl, 10 mM imidazole, 50 µM ZnSO<sub>4</sub>, 5 mM β-mercaptoethanol, 0.5 mM TCEP, 0.1% Triton-X 100, 2.5% glycerol, 1mM PMSF, 2 mM benzamidine and Roche complete EDTA-free protease inhibitor cocktail tablet) using a probe sonicator. The crude extract was cleared by ultracentrifugation for 50 min at 50000xg and the supernatant was incubated with TALON Metal Affinity Resin (Takara) at 4<sup>o</sup>C for 120 minutes with agitation. Resin was washed in wash buffer (lysis buffer with 0.01% Triton-X 100 and lacking protease inhibitors) and bound proteins were eluted using elution buffer (wash buffer with 400 mM imidazole), monitored by Bradford analysis. Protein was dialyzed overnight at 4°C in TEV dialysis buffer (20 mM Tris pH 7.5, 300 mM NaCl, 20µM ZnCL<sub>2</sub>, 2.5% glycerol, 5 mM β-mercaptoethanol and 0.5 mM TCEP) and incubated with His-tagged TEV protease produced in house at a 1/20 dilution by mass. 0.5 mM CHAPS was added to the TEV dialysis buffer for PRDM 9 and 7, but not PRDM 3 and 16. TEV and uncleaved proteins were removed using TALON resin and soluble  protein was loaded onto a Superdex 75 Increase 10/300 GL colum (GE Healthcare), equilibrated with 20 mM Tris-HCl buffer, pH 7.5, and 150 mM NaCl, 2.5% glycerol, 5 mM β-mercaptoethanol and 0.5 mM TCEP at flow rate 0.7 ml/min. Fractions containing PRDM9 protein were pooled and dialyzed into low salt buffer (20 mM Tris-HCl pH 8.5, 100 mM NaCl, 5% glycerol, 5 mM β-mercaptoethanol and 0.5 mM TCEP). PRDM9 protein was further purified by anion-exchange chromatography on 2 tandemly joined 5 ml HiTrap<img src="https://s.w.org/images/core/emoji/13.0.0/72x72/2122.png" alt="™" class="wp-smiley" style="height: 1em; max-height: 1em;" /> DEAE FF (GE Healthcare) columns along a linear gradient up to 1.0 M NaCl. Purified PRDM9 protein was dialyzed into a final buffer containing 20 mM Tris pH 7.5, 150 mM NaCl and 2 mM TCEP. The proteins for PRDM1 (38-223), PRDM2 (2-160), PRDM4 (390-540), PRDM5 (2-226), PRDM6 (194-405), PRDM11 (79-314), PRDM12 (60-229), PRDM14 (212-422), and PRDM17 (1-205) were purified from E. coli expression and kindly provided by Taraneh Hajian, Elisa Gibson and Dr. Masoud Vedadi from the SGC Toronto.</p>
<p><strong>F-SAH binding by <sup>19</sup>F NMR<br />
</strong>F-SAH was synthesized and kindly provided by Carlos Zepeda, Ontario Institute for Cancer Research (<em>OICR</em>). All PR/SET domain containing proteins were dialyzed overnight at 4<sup>o</sup>C into <sup>19</sup>F-NMR buffer (20 mM Tris pH 7.3, 150 mM NaCl, 2 mM TCEP and 20 µM ZnCl<sub>2</sub>) also containing 16 µM sodium trifluoroacetate. The concentration of each protein was measured using an absorbance reading at 280 nm and evaluated using their respective theoretical extinction coefficients calculated using ProtParam (Gasteiger, et al., 2005). 100 mM F-SAH in d6-DMSO was diluted to 16 µM F-SAH by adding the dialyzed protein solution to a final protein concentration of 80 µM and then topping off with <sup>19</sup>F-NMR buffer to a final volume of 500 µl. For the F-SAH alone sample, only <sup>19</sup>F-NMR buffer was added. The solutions were mixed in 1.5 ml polypropylene tubes and then transferred to 5 mm glass NMR tubes.</p>
<p>1D <sup>19</sup>F spectra were acquired at 298K on a Bruker Avance III spectrometer operating at 600 MHz and equipped with a QCI cryoprobe with an independent <sup>19</sup>F detection coil. Each spectrum was acquired with 3072 scans, an acquisition time of 577 ms, and processed by applying an exponential window function (LB = 20). There was no difference observed in the F-SAH <sup>19</sup>F line-shape when signal acquisition was acquired with or without proton decoupling; therefore, all spectra were acquired without proton decoupling.  The spectral window was measured from -37.89 ppm to -88.09 ppm and <sup>19</sup>F peaks were identified at -75.53 ppm, -52.40 ppm, and -54.21 ppm, for trifluoroacetate, F-SAH unbound, and F-SAH bound, respectively. Processing and analysis were carried out with Topspin 3.5 (Bruker BioSpin). To control for pipetting errors that could result in differences in F-SAH signal intensities, the integral of the trifluoroacetate <sup>19</sup>F peak of each PR/SET containing sample was set to equal the integral of the trifluoroacetate <sup>19</sup>F peak of the sample with F-SAH alone. To normalize F-SAH peaks, the integral for each F-SAH peak was divided by the integral of the F-SAH peak from the sample with F-SAH alone (i.e. integral of F-SAH alone = 1).</p>
<h1>References</h1>
<p>Allali-Hassani, A. et al., 2019. Discovery of a chemical probe for PRDM9.. <em>Nat Commun., </em>10(1), p. 5759.</p>
<p>Ancelin, K. et al., 2006. Blimp1 associates with Prmt5 and directs histone arginine methylation in mouse germ cells.. <em>Nat Cell Biol, </em>8(6), pp. 623-30.</p>
<p>Arrowsmith, C. et al., 2012. Epigenetic protein families: a new frontier for drug discovery.. <em>Nat Rev Drug Discov, </em>11(5), pp. 384-400.</p>
<p>Blazer, L. et al., 2016. PR Domain-containing Protein 7 (PRDM7) Is a Histone 3 Lysine 4 Trimethyltransferase.. <em>J Biol Chem, </em>291(26), pp. 13509-19.</p>
<p>Campagna-Slater, V. et al., 2011. Structural chemistry of the histone methyltransferases cofactor binding site.. <em>J Chem Inf Model, </em>51(3), pp. 612-23.</p>
<p>Casamassimi, A. et al., 2020. Multifaceted Role of PRDM Proteins in Human Cancer.. <em>Int J Mol Sci., </em>21(7), p. E2648.</p>
<p>Congdon, L., Sims, J., Tuzon, C. &amp; Rice, J., 2014. The PR-Set7 binding domain of Riz1 is required for the H4K20me1-H3K9me1 trans-tail &#8216;histone code&#8217; and Riz1 tumor suppressor function.. <em>Nucleic Acids Res, </em>42(6), pp. 3580-9.</p>
<p>Couture, J. et al., 2006. Catalytic roles for carbon-oxygen hydrogen bonding in SET domain lysine methyltransferases.. <em>J Biol Chem, </em>281(28), pp. 19280-7.</p>
<p>Dalvit, C. &amp; Vulpetti, A., 2019. Ligand-Based Fluorine NMR Screening: Principles and Applications in Drug Discovery Projects.. <em>J Med Chem, </em>62(5), pp. 2218-44.</p>
<p>Dilworth, D. &amp; Barsyte-Lovejoy, D., 2019. Targeting protein methylation: from chemical tools to precision medicines.. <em>Cell Mol Life Sci, </em>76(15), pp. 2967-85.</p>
<p>Eram, M. et al., 2014. Trimethylation of histone H3 lysine 36 by human methyltransferase PRDM9 protein.. <em>J Biol Chem, </em>289(17), pp. 12177-88.</p>
<p>Gasteiger, E. et al., 2005. Protein Identification and Analysis Tools on the ExPASy Server. In: J. M. Walker, ed. <em>The Proteomics Protocols Handbook. </em>s.l.:Humana Press, pp. 571-607.</p>
<p>Gyory, I. et al., 2004. PRDI-BF1 recruits the histone H3 methyltransferase G9a in transcriptional silencing.. <em>Nat Immunol, </em>5(3), pp. 299-08.</p>
<p>Hoy, S., 2020. Tazemetostat: First Approval.. <em>Drugs, </em>80(5), pp. 513-21.</p>
<p>Ilango, S. et al., 2020. Epigenetic alterations in cancer.. <em>Front Biosci (Landmark Ed), </em>25(1), pp. 1058-109.</p>
<p>Kim, K., Geng, L. &amp; Huang, S., 2003. Inactivation of a histone methyltransferase by mutations in human cancers.. <em>Cancer Res, </em>63(22), pp. 7619-23.</p>
<p>Luan, Y. et al., 2016. Design of a fluorescent ligand targeting the S-adenosylmethionine binding site of the histone methyltransferase MLL1. <em>Org Biomol Chem, </em>14(2), pp. 631-8.</p>
<p>Minnich, M. et al., 2016. Multifunctional role of the transcription factor Blimp-1 in coordinating plasma cell differentiation.. <em>Nat Immunol, </em>17(3), pp. 331-43.</p>
<p>Pinheiro, I. et al., 2012. Prdm3 and Prdm16 are H3K9me1 methyltransferases required for mammalian heterochromatin integrity.. <em>Cell, </em>150(5), pp. 948-60.</p>
<p>Powers, N. et al., 2016. The Meiotic Recombination Activator PRDM9 Trimethylates Both H3K36 and H3K4 at Recombination Hotspots In Vivo.. <em>PLoS Genet, </em>12(6), p. e1006146.</p>
<p>Ribeiro, A. et al., 2019. Emerging concepts in pseudoenzyme classification, evolution, and signaling.. <em>Sci Signal, </em>12(594), p. eaat9797.</p>
<p>Rugo, H. et al., 2020. The Promise for Histone Methyltransferase Inhibitors for Epigenetic Therapy in Clinical Oncology: A Narrative Review.. <em>Adv Ther. , </em>37(7), pp. 3059-82.</p>
<p>Schapira, M., 2011. Structural Chemistry of Human SET Domain Protein Methyltransferases.. <em>Curr Chem Genomics, </em>5(S1), pp. 85-94.</p>
<p>Su, S. et al., 2009. Involvement of histone demethylase LSD1 in Blimp-1-mediated gene repression during plasma cell differentiation.. <em>Mol Cell Biol, </em>29(6), pp. 1421-31.</p>
<p>Wu, H. et al., 2013. Molecular basis for the regulation of the H3K4 methyltransferase activity of PRDM9.. <em>Cell Rep, </em>5(1), pp. 13-20.</p>
<p>Wu, Y. et al., 2008. PRDM6 is enriched in vascular precursors during development and inhibits endothelial cell proliferation, survival, and differentiation.. <em>J Mol Cell Cardiol, </em>44(1), pp. 47-58.</p>
<p>Zhou, B. et al., 2016. PRDM16 Suppresses MLL1r Leukemia via Intrinsic Histone Methyltransferase Activity.. <em>Mol Cell, </em>62(2), pp. 222-36.</p>
<p>&nbsp;</p>
<p><a href="https://zenodo.org/record/4253145#.X6lMCVB7lEY">Download this as a PDF from Zenodo.</a></p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/interrogating-pr-set-domains-for-the-ability-to-bind-sah/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>Production and purification of Ectodomain of SARS-CoV-2 Spike (S) protein</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/production-and-purification-of-ectodomain-of-sars-cov-2-spike-s-protein/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/production-and-purification-of-ectodomain-of-sars-cov-2-spike-s-protein/#respond</comments>
		
		<dc:creator><![CDATA[Alma Seitova]]></dc:creator>
		<pubDate>Sat, 07 Nov 2020 23:35:17 +0000</pubDate>
				<category><![CDATA[Alma Seitova]]></category>
		<category><![CDATA[COVID-19]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=5353</guid>

					<description><![CDATA[Production and purification of Ectodomain of SARS-CoV-2 Spike (S) protein Two different expression plasmids for prefusion S ectodomain residues 1−1208 of 2019-nCoV S (GenBank: MN908947) with proline substitutions (2P) at residues 986 and 987(1) and with six proline substitutions (HexaPro (at residues 817, 892, 899, 942, 986 and 987 (2), a “GSAS” substitution at the <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/production-and-purification-of-ectodomain-of-sars-cov-2-spike-s-protein/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p></a><strong>Production and purification of Ectodomain of SARS-CoV-2 Spike (S) protein</strong><br />
      Two different expression plasmids for prefusion S ectodomain residues 1−1208 of 2019-nCoV S (GenBank: MN908947) with proline substitutions (2P) at residues 986 and 987(1) and with six proline substitutions (HexaPro (at residues 817, 892, 899, 942, 986 and 987 (2), a “GSAS” substitution at the furin cleavage site (residues 682–685), a C-terminal T4 fibritin trimerization motif, an HRV3C protease cleavage site, a TwinStrepTag and an 8XHisTag in the mammalian expression vector pαH were generously gifted by Dr.McLellan’s lab (1,2). </p>
<p>I. Production and purification of the 2P version of S-Ectodomain.<br />
During the first purifications from a small-scale production we have faced some challenges with the 2P version of this protein. (Figure 1.) </p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/1as.png" alt="" width="1032" height="575" class="alignnone size-full wp-image-5354" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/1as.png 1032w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/1as-300x167.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/1as-1024x571.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/1as-768x428.png 768w" sizes="(max-width: 1032px) 100vw, 1032px" /></p>
<p>Purification from a big scale-up production using another buffer (1x PBS) led to the same problems, and we evaluated protein concentrations by Pierce BCA Protein Assay (Please see Figure 2&#038;3) and comparative gel analysis. </p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Figure-2.png" alt="" width="1088" height="612" class="alignnone size-full wp-image-5362" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Figure-2.png 1088w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Figure-2-300x169.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Figure-2-1024x576.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Figure-2-768x432.png 768w" sizes="(max-width: 1088px) 100vw, 1088px" /></p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Figure-3.-as.png" alt="" width="1105" height="622" class="alignnone size-full wp-image-5368" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Figure-3.-as.png 1105w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Figure-3.-as-300x169.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Figure-3.-as-1024x576.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Figure-3.-as-768x432.png 768w" sizes="(max-width: 1105px) 100vw, 1105px" /></p>
<p>We have obtained total 12mg of Histag purified and dialyzed protein and run 9mg of protein  on the size-exclusion column. SEC profiles showed that a big portion of the protein were in aggregates (Figure 4).</p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Figure-4.-AS.png" alt="" width="1116" height="628" class="alignnone size-full wp-image-5364" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Figure-4.-AS.png 1116w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Figure-4.-AS-300x169.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Figure-4.-AS-1024x576.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Figure-4.-AS-768x432.png 768w" sizes="(max-width: 1116px) 100vw, 1116px" /></p>
<p>On the size-exclusion profile we observed 2 peaks of the aggregates and collected only a 3rd peak corresponding to the trimer of S- Ectodomain running close to 670kD.<br />
Based on the results of the purifications by affinity, size-exclusion protocols and exchanging notes with a Biotechnology group of SGC Oxford, we’ve come to conclusions that S- Ectodomain is prone to the aggregation almost at the every stage of purification, one nucleation of the aggregates “ curls up” a most part of the protein.<br />
We have optimised a few steps in the production and purification of this target to prevent or minimize aggregation:<br />
• Cultured cells were harvested in a different hours of the post-transfection times<br />
• Harvested culture supernatant clarified by two steps of centrifugation at RT was passed through a<br />
  0.22 μm filter before IMAC batch absorption<br />
• After IMAC purification, combined peak fractions were filtered through a 0.22 μm filter, before<br />
  concentrating and loading to the SEC<br />
• Concentrated combined fractions of the eluted protein at 16 degree celsius and  run on SEC<br />
Size-exclusion profiles of S-Ectodomain from these 4 purifications shown no major aggregates and a main fraction of the protein was corresponding to the expected MV as 670kD (Figure 5.), but a final yield of the purified by two-steps protein were very low as 0.2-0.3mg/L.  </p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Figure-5.-AS.png" alt="" width="1035" height="583" class="alignnone size-full wp-image-5365" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Figure-5.-AS.png 1035w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Figure-5.-AS-300x169.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Figure-5.-AS-1024x577.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Figure-5.-AS-768x433.png 768w" sizes="(max-width: 1035px) 100vw, 1035px" /></p>
<p>We have run protein standards on the SEC column used for this protein to confirm accuracy of the MV of the eluted protein (Figure 6.) </p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Figure-6.-as.png" alt="" width="1116" height="628" class="alignnone size-full wp-image-5366" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Figure-6.-as.png 1116w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Figure-6.-as-300x169.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Figure-6.-as-1024x576.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Figure-6.-as-768x432.png 768w" sizes="(max-width: 1116px) 100vw, 1116px" /></p>
<p>II. Production and purification of the HexaPro version of S-Ectodomain<br />
We have used the same optimized protocol for the HexaPro version of the S-Ectodomain (Figure 7).</p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Figure-7.-.png" alt="" width="1101" height="620" class="alignnone size-full wp-image-5370" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Figure-7.-.png 1101w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Figure-7.--300x169.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Figure-7.--1024x577.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Figure-7.--768x432.png 768w" sizes="(max-width: 1101px) 100vw, 1101px" /></p>
<p>Final yield for this constructs was higher than for 2P version of S-Ectodomain, we’ve obtained 1mg/L from the first 800ml of production by transient transfection of EXPI293 cells. Further optimization might improve a final yield to the higher expression for this construct, but so far we&#8217;ve done only one production and purification trial with HexaPro. </p>
<p>References:<br />
1.Cryo-EM structure of the 2019-nCoV spike in the prefusion conformation<br />
Daniel Wrapp, Nianshuang Wang, Kizzmekia S. Corbett, Jory A. Goldsmith, Ching-Lin Hsieh, Olubukola Abiona, Barney S. Graham, Jason S. McLellan<br />
Science 367, 1260–1263 (2020) 13 March 2020</p>
<p>2. Structure-based design of prefusion-stabilized SARS-CoV-2 spikes<br />
Ching-Lin Hsieh, Jory A. Goldsmith, Jeffrey M. Schaub, Andrea M. DiVenere, Hung-Che Kuo,<br />
Kamyab Javanmardi, Kevin C. Le, Daniel Wrapp, Alison G. Lee, Yutong Liu, Chia-Wei Chou,<br />
Patrick O. Byrne, Christy K. Hjorth, Nicole V. Johnson, John Ludes-Meyers, Annalee W. Nguyen,<br />
Juyeon Park1, Nianshuang Wang1, Dzifa Amengor1, Jason J. Lavinder1,2, Gregory C. Ippolito,<br />
Jennifer A. Maynard, Ilya J. Finkelstein, Jason S. McLellan<br />
Science 369, 1501–1505 (2020) 18 September 2020              </p>
<p><a href="https://zenodo.org/record/4256998#.X6cqUS0ZPBI">Please see Zenodo page for the production and purification protocols</a></p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/production-and-purification-of-ectodomain-of-sars-cov-2-spike-s-protein/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>Development and Optimization of an IQF USP5 Activity Assay</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/development-and-optimization-of-an-iqf-usp5-activity-assay/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/development-and-optimization-of-an-iqf-usp5-activity-assay/#respond</comments>
		
		<dc:creator><![CDATA[Mandeep Mann]]></dc:creator>
		<pubDate>Thu, 05 Nov 2020 16:58:16 +0000</pubDate>
				<category><![CDATA[Mandeep Mann]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<category><![CDATA[USP5]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=5335</guid>

					<description><![CDATA[In a previous post, I described a ubiquitin-rhodamine110 (UbRho110) assay, and showed that a ZnF-UBD ligand antagonizes USP5 catalytic activity. UbRho110 is not a native substrate, and validation with native polyubiquitin species in an orthogonal assay is required. To address this, I developed and optimized an internally quenched fluorophore pair (IQF) assay. You can see <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/development-and-optimization-of-an-iqf-usp5-activity-assay/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p>In a <a href="https://thesgc.github.io/static-openlabnotebooks/usp5-ubiquitin-rhodamine110-catalytic-activity-assay/">previous post</a>, I described a ubiquitin-rhodamine110 (UbRho110) assay, and showed that a ZnF-UBD ligand antagonizes USP5 catalytic activity. UbRho110 is not a native substrate, and validation with native polyubiquitin species in an orthogonal assay is required. To address this, I developed and optimized an internally quenched fluorophore pair (IQF) assay. You can see details on <strong><a href="https://zenodo.org/record/4247576#.X6QridBKg2w">Zenodo. </a></strong></p>
<p>The IQF assay uses di-ubiquitin substrates, one of which carries a fluorophore that is silenced by a nearby quencher. After the di-ubiquitin substrate is cleaved by a deubiquitinase (DUB) such as USP5, the free fluorophore can be detected through a fluorescence signal (Figure 1).</p>
<p style="text-align: center;"><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/20201105_fig1-300x146.jpg" alt="" width="335" height="163" class=" wp-image-5336 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/20201105_fig1-300x146.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/20201105_fig1.jpg 624w" sizes="(max-width: 335px) 100vw, 335px" /><strong>Figure 1</strong>. Summary of IQF assay. Image taken from <a href="https://lifesensors.com/iqf/">LifeSensors</a>.</p>
<p>I tested two di-ubiquitin IQF substrates: Ub2K63 and Ub2K48. A ZnF-UBD ligand, <a href="https://thesgc.github.io/static-openlabnotebooks/expansion-of-ubtr012574-chemical-series/">UBTR020994a</a> (K<sub>D</sub>= 8 ± 2 µM [n=3]) equipotently antagonizes USP5 DUB activity for both diubiquitin substrates in a dose-dependent manner (Figure 2).</p>
<p style="text-align: center;"><strong>a)</strong><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/20201105_fig2a-300x96.png" alt="" width="178" height="57" class="wp-image-5337 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/20201105_fig2a-300x96.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/20201105_fig2a.png 638w" sizes="(max-width: 178px) 100vw, 178px" /></p>
<p style="text-align: center;"><strong>b) </strong></p>
<p style="text-align: center;"><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/20201105_fig2b-300x159.png" alt="" width="230" height="122" class="alignnone wp-image-5338" style="font-size: 1.5rem;" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/20201105_fig2b-300x159.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/20201105_fig2b.png 649w" sizes="(max-width: 230px) 100vw, 230px" /></p>
<p style="text-align: center;"><strong>c)</strong><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/20201105_fig2c-300x214.png" alt="" width="233" height="166" class=" wp-image-5339 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/20201105_fig2c-300x214.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/20201105_fig2c.png 577w" sizes="(max-width: 233px) 100vw, 233px" /></p>
<p style="text-align: center;"><strong>Figure 2</strong>. a) Chemical structure of UBTR020994a (K<sub>D</sub>= 8 ± 2 µM [n=3]) b) Dose-response of UBTR020994a and Ub2K63 (n=3)  IC50  ̴ 63 µM c) Dose-response of UBTR020994a and Ub2K48 (n=3) IC50=45 ± 17 µM</p>
<p>In addition to the wild-type (WT) USP5, I tested a R221A and C335S mutant. The R221A is a ZnF-UBD mutant, where R221 co-ordinates ubiquitin binding at the ZnF-UBD. C335S is a mutant of the catalytic cysteine and prevents the cleavage of the isopeptide bond in ubiquitin chains. The R221A mutant has little to no activity with Ub2K48 substrate-comparable to inhibition of the WT-USP5 with a ZnF-UBD antagonist, UBTR020994a. IQF Ub2K63 has approximately 50% activity for R221A mutant, suggesting R221 may be more important for recruitment and placement of Ub2K48 substrate than Ub2K63 substrate.  C335S is inactive in the assay.</p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/20201105_fig3d.png" alt="" width="99" height="30" class=" wp-image-5340 aligncenter" /><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/20201105_fig2a-300x96.png" alt="" width="113" height="36" class="wp-image-5337 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/20201105_fig2a-300x96.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/20201105_fig2a.png 638w" sizes="(max-width: 113px) 100vw, 113px" /></p>
<p><strong>a)</strong>                                                        <strong>  b)</strong></p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/20201105_fig3a-300x290.png" alt="" width="226" height="218" class="alignnone wp-image-5341" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/20201105_fig3a-300x290.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/20201105_fig3a.png 433w" sizes="(max-width: 226px) 100vw, 226px" /><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/20201105_fig3b-300x269.png" alt="" width="242" height="217" class="alignnone wp-image-5342" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/20201105_fig3b-300x269.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/20201105_fig3b.png 477w" sizes="(max-width: 242px) 100vw, 242px" /></p>
<p style="text-align: center;"><strong>Figure 3</strong>. USP5 DUB Activity for wild-type (WT) USP5, a ZnF-UBD mutant (R221A), a catalytic mutant (C335S) ± 2 mM UBTR020994a: a) Ub2K63 (N=2) b) Ub2K48 (N=3)</p>
<p>In the future, I will be repeating experiments with the USP5 mutants and dose-response curves for UBTR020994a for both IQF substrates to confirm the results. I also hope to test other lysine-linked di-ubiquitin substrates, such as K11, using this IQF assay.</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/development-and-optimization-of-an-iqf-usp5-activity-assay/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>Genetic Variability at An Allosteric site of SARS-CoV-2 RNA-dependent RNA polymerase Across Coronaviruses and SARS-CoV-2 Samples &#8211; Post 21</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/genetic-variability-at-an-allosteric-site-of-sars-cov-2-rna-dependent-rna-polymerase-across-coronaviruses-and-sars-cov-2-samples/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/genetic-variability-at-an-allosteric-site-of-sars-cov-2-rna-dependent-rna-polymerase-across-coronaviruses-and-sars-cov-2-samples/#respond</comments>
		
		<dc:creator><![CDATA[Setayesh Yazdani]]></dc:creator>
		<pubDate>Thu, 05 Nov 2020 00:08:24 +0000</pubDate>
				<category><![CDATA[COVID-19]]></category>
		<category><![CDATA[SARS-CoV-2 computational genomics]]></category>
		<category><![CDATA[Setayesh Yazdani]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=5319</guid>

					<description><![CDATA[SARS-CoV-2 is a positive-strand RNA virus, depending on its multi-subunit machinery to replicate its RNA. This machinery is known as RNA-dependent RNA polymerase (RdRp). The catalytic subunit of RdRp, which is the core component of this machinery, is called nsp12. Nsp12 has little catalytic activity on its own and relies on accessory subunits to have <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/genetic-variability-at-an-allosteric-site-of-sars-cov-2-rna-dependent-rna-polymerase-across-coronaviruses-and-sars-cov-2-samples/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p>SARS-CoV-2 is a positive-strand RNA virus, depending on its multi-subunit machinery to replicate its RNA. This machinery is known as RNA-dependent RNA polymerase (RdRp). The catalytic subunit of RdRp, which is the core component of this machinery, is called nsp12. Nsp12 has little catalytic activity on its own and relies on accessory subunits to have full activity. These accessory factors, nsp7 and nsp8, increase RdRp template binding and processivity. (Yin et al., 2020)</p>
<p>RdRp is a well-established target for a class of antiviral drugs known as nucleotide analogs. Remdesivir is an example of such a group of drugs that inhibits RdRp. Remdesivir has been authorized to be used for special COVID-19 cases with severe symptoms in Canada. (https://healthycanadians.gc.ca/recall-alert-rappel-avis/hc-sc/2020/73621a-eng.php)</p>
<p>We previously analyzed the remdesivir-binding site of RdRp. We then shifted our focus on another druggable pocket of the RdRp complex, an allosteric site on the RdRp complex, which is this post&#8217;s topic.</p>
<p>In figure 1, we show the RdRp complex and highlight the site of interest, which is lined by amino acid residue M666 and which we refer to here as the M666 pocket. This site has a druggability score of 1.008 based on our results from Sitemap (Schrodinger, NY), indicating a druggable site (figure 2, <a href="https://doi.org/10.5281/zenodo.4247424">Zenodo report</a>).</p>
<p>M666 site is at the nsp8 binding site of nsp12. A compound binding this site with high potency may antagonize the formation of an active RdRp complex. It may inhibit the RdRp function, which is necessary for viral replication.</p>
<p>&nbsp;</p>
<div id="attachment_5321" style="width: 636px" class="wp-caption alignnone"><img aria-describedby="caption-attachment-5321" loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/nsp12-nsp82-nsp7.png" alt="" width="626" height="775" class="wp-image-5321 size-full" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/nsp12-nsp82-nsp7.png 626w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/nsp12-nsp82-nsp7-242x300.png 242w" sizes="(max-width: 626px) 100vw, 626px" /><p id="caption-attachment-5321" class="wp-caption-text">Figure 1. The apo nsp12-nsp7-nsp8 RdRp complex and the positioning of the M666 pocket at the interface of nsp12-nsp8.</p></div>
<p>&nbsp;</p>
<p>We then selected the residues surrounding this pocket in proximity to bound nsp8. The RdRp M666 pocket is lined by 24 amino acid sidechains (figure 3, <a href="https://doi.org/10.5281/zenodo.4247424">Zenodo report</a>).</p>
<p>We then conduct a broad survey of viral proteins from Alpha- and Betacoronaviruses to identify the most conserved druggable sites. We are integrating variability and druggability data to infer the most promising strategies for developing broad-spectrum viral inhibitors. This analysis can be valuable in the context of the emergence of future coronaviruses circulating in bat reservoir species.</p>
<p>To assess this site&#8217;s genetic variability, we first looked at 27 reviewed sequences from Uniprot, of which 6 are from the Alphacoronavirus genus and 21 from the Betacoronavirus genus. Altogether, eight out of twenty-four residues at M666 were identical across the 27 entries, equivalent to 33% identity at this site (figure 4, <a href="https://doi.org/10.5281/zenodo.4247424">Zenodo report</a>).</p>
<p>As a follow-up to assessing the genetic variability among Alphacoronavirus and Betacoronavirus entries, our collaborator, Nicola De Maio, from Nick Goldman’s lab at the European Bioinformatics Institute has identified the non-synonymous mutations at M666 sites across more than 15000 SARS-CoV-2 samples. Altogether he identified non-synonymous mutations at 10 unique amino acid residues consisting of 13 unique non-synonymous variants. In table 1 on our <a href="https://doi.org/10.5281/zenodo.4247424">Zenodo report</a>, there are details about the mutation events underlying these variants.</p>
<p>In terms of druggability, the M666 pocket (Dscore=1.028) is superior to the RdRp active site (where remdesivir binds) (Dscore=0.859). While the active site is highly conserved (83% identity), the M666 pocket is poorly conserved (33% identity). Similarly, across SARS-CoV-2 samples at the M666 site, there is high genetic variability compared to the remdesivir site, which is more genetically stable; only 6 out of 48 residues were mutated.</p>
<p>Remdesivir use for COVID-19 is only recommended for hospitalized patients who need supplemental oxygen therapy (Clinical Trials.gov number, NCT04280705), and only in those severe cases, remdesivir was superior to placebo in shortening the time to recovery. (Goldman et al., 2020) This phase3 study showed that despite remdesivir, there was still high mortality because of COVID-19.  Therefore, clinically we still need a better drug to treat COVID-19 through different approaches.  (https://www.nejm.org/doi/full/10.1056/NEJMoa2007764)</p>
<p>Targeting the M666 provides an alternate strategy to target RdRp function by disturbing its interaction with a key accessory factor, nsp8. It is important to note the high genetic variability of this site among coronaviruses and SARS-CoV-2 samples, which deprioritizes it as a target for pan-coronavirus anti-SARS-CoV-2 drugs.</p>
<p>You can view this report on <a href="https://doi.org/10.5281/zenodo.4247424">Zenodo</a> as well.</p>
<p>Please contact me via the “Leave a comment” link at the top of this post. Stay Tuned for more updates on this project.</p>
<p><strong>References:</strong></p>
<p>Yin, W., Mao, C., Luan, X., Shen, D., Shen, Q., Su, H., Wang, X., Zhou, F., Zhao, W., Gao, M., Chang, S., Xie, Y., Tian, G., Jiang, H., Tao, S., Shen, J., Jiang, Y., Jiang, H., Xu, Y., &amp; Xu, H. (2020). Structural basis for inhibition of the RNA-dependent RNA polymerase from SARS-CoV-2 by remdesivir. <em>Science</em>, <em>368</em>(6498), 1499-1504. <a href="https://doi.org/10.1126/science.abc1560">https://doi.org/10.1126/science.abc1560</a></p>
<p>Goldman, J. D., Lye, D. C.B., Hui, D. S., Marks, K. M., Bruno, R., Montejano, R., Spinner, C. D., Galli, M., Ahn, M., Nahass, R. G., Chen, Y., SenGupta, D., Hyland, R. H., Osinusi, A. O., Cao, H., Blair, C., Wei, X., Gaggar, A., Brainard, D. M., &amp; Subramanian, A. (2020). Remdesivir for 5 or 10 Days in Patients with Severe Covid-19. <em>The New England Journal of Medicine</em>. <a href="https://doi.org/10.1056/NEJMoa2015301">https://doi.org/10.1056/NEJMoa2015301</a></p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/genetic-variability-at-an-allosteric-site-of-sars-cov-2-rna-dependent-rna-polymerase-across-coronaviruses-and-sars-cov-2-samples/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>FBXW11 interaction with β-catenin &#8211; cellular assay</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/fbxw11-interaction-with-%ce%b2-catenin-cellular-assay/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/fbxw11-interaction-with-%ce%b2-catenin-cellular-assay/#respond</comments>
		
		<dc:creator><![CDATA[Magdalena Szewczyk]]></dc:creator>
		<pubDate>Mon, 26 Oct 2020 17:05:14 +0000</pubDate>
				<category><![CDATA[Cell assay reports]]></category>
		<category><![CDATA[E3 ligases]]></category>
		<category><![CDATA[Magdalena Szewczyk]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=5309</guid>

					<description><![CDATA[Background The SCF (SKP1-CUL1-F-box protein) E3 ubiquitin ligase is a complex of proteins that helps facilitate the ubiquitination of substrates for targeted proteasomal degradation. FBXW11 is F-box containing protein that has 11 WD-repeats and is a substrate recognition component of the SCF complex. FBXW11 recognizes and binds its substrates, including beta-catenin, at phosphorylated sites or <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/fbxw11-interaction-with-%ce%b2-catenin-cellular-assay/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p><strong>Background</strong></p>
<p>The SCF (SKP1-CUL1-F-box protein) E3 ubiquitin ligase is a complex of proteins that helps facilitate the ubiquitination of substrates for targeted proteasomal degradation. FBXW11 is F-box containing protein that has 11 WD-repeats and is a substrate recognition component of the SCF complex. FBXW11 recognizes and binds its substrates, including beta-catenin, at phosphorylated sites or phosphodegrons, and recruits it to the SCF complex for subsequent ubiquitination and degradation (PMID: 10321728)</p>
<p><strong>Assay validation</strong></p>
<p>Assay measuring the interaction of FBXW11 with B-catenin was developed using NanoBRET technology &#8211; a proximity-based assay that can detect protein interactions by measuring energy transfer from a bioluminescent protein donor (NanoLuc luciferase, NL) to a fluorescent protein acceptor (fluorescent ligand bound to HaloTag, HT). In the beginning, we determined the optimal tag orientation of donor constructs: N or C-terminally NL-tagged β-catenin and acceptor constructs: N or C-terminally HT-tagged FBXW11 in HEK293T cells. NL and HT proteins, expressed alone, were used as controls to determine background signal. The best results were obtained with C-terminally NL-tagged B-catenin and C-terminally HT-tagged FBXW11 (Fig.1). The assay was further validated with binding deficient FBXW11 mutant (R447E) designed by Matthieu Schapira. R447E mutation resulted in a significant decrease in the interaction of FBXW11 with B-catenin (Fig.2).</p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/f11b-300x134.png" alt="" width="582" height="260" class=" wp-image-5311 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/f11b-300x134.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/f11b-768x343.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/f11b.png 993w" sizes="(max-width: 582px) 100vw, 582px" /></p>
<p><strong>Fig.1. FBXW11 interaction with B-catenin &#8211; NanoBRET assay. </strong>HEK293T cells were co-transfected with C- or N-terminally NanoLuc (NL)-tagged B-catenin or NL alone and C- or N-terminally Halo-tagged (HT) FBXW11 or HT alone for 24 h. The interaction was measured using NanoBRET assay. The results are MEAN of 2 technical replicates.</p>
<p><strong> <img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/fbxw11-300x168.png" alt="" width="491" height="275" class="wp-image-5310 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/fbxw11-300x168.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/fbxw11-1024x572.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/fbxw11-768x429.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/fbxw11.png 1063w" sizes="(max-width: 491px) 100vw, 491px" /></strong></p>
<p><strong>Fig.2. </strong><strong>R447E mutation decreases the interaction of FBXW11 with B-catenin.</strong> HEK293T cells were co-transfected with C-terminally Halo-tagged (HT) FBXW11 or HT and C-terminally NanoLuc (NL)-tagged B-catenin for 24 h. The interaction was measured using NanoBRET assay. The results are MEAN of 2 technical replicates.</p>
<p>Please go to <a href="https://zenodo.org/record/4134923#.X5ccxohKg2w">Zenodo</a> for experimental details.</p>
<p><strong>Acknowledgments:</strong></p>
<p>Matthieu Schapira &#8211; mutant design, Aidan Levinsky &#8211; help with mutagenesis</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/fbxw11-interaction-with-%ce%b2-catenin-cellular-assay/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>FBXW7 interaction with Cyclin E1 &#8211; cellular assay</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/fbxw7-interaction-with-cyclin-e1-cellular-assay/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/fbxw7-interaction-with-cyclin-e1-cellular-assay/#respond</comments>
		
		<dc:creator><![CDATA[Magdalena Szewczyk]]></dc:creator>
		<pubDate>Mon, 26 Oct 2020 16:42:54 +0000</pubDate>
				<category><![CDATA[Cell assay reports]]></category>
		<category><![CDATA[E3 ligases]]></category>
		<category><![CDATA[Magdalena Szewczyk]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=5302</guid>

					<description><![CDATA[Background The SCF (SKP1-CUL1-F-box protein) E3 ubiquitin ligase is a complex of proteins that helps facilitate the ubiquitination of substrates for targeted proteasomal degradation. FBXW7 (F-box/WD repeat-containing protein 7) is an F-box protein component of the SCF complex and is involved in substrate recognition. The F-box domain tethers FBXW7 to the SCF complex by interacting <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/fbxw7-interaction-with-cyclin-e1-cellular-assay/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p><strong>Background</strong></p>
<p>The SCF (SKP1-CUL1-F-box protein) E3 ubiquitin ligase is a complex of proteins that helps facilitate the ubiquitination of substrates for targeted proteasomal degradation. FBXW7 (F-box/WD repeat-containing protein 7) is an F-box protein component of the SCF complex and is involved in substrate recognition. The F-box domain tethers FBXW7 to the SCF complex by interacting with the SKP1 component while the WD40 repeats are involved in substrate binding. FBXW7 recognizes and binds its substrates at phosphorylated sites or phosphodegrons and recruits them to the SCF complex for subsequent ubiquitination and degradation (PMID:17434132). A variety of substrates exist for FBXW7, many of which are potentially oncogenic, including cyclin E1, MYC, JUN, and NOTCH (PMID: 11565034, 15103331, 14739463, 29149593). The targeted degradation of growth-promoting factors by FBXW7 suggests it likely plays an important role as a tumor suppressor. FBXW7 Arg→Cys mutation at position 465 hinders its ability to interact with SKP1 and thus the SCF complex; this mutation is associated with acute lymphoblastic leukemia (PMID: 11565033, 28007894).</p>
<p><strong>Assay validation</strong></p>
<p>Assay measuring the interaction of FBXW7 with cyclin E1 was developed using NanoBRET technology &#8211; a proximity-based assay that can detect protein interactions by measuring energy transfer from a bioluminescent protein donor (NanoLuc, NL) to a fluorescent protein acceptor (fluorescent ligand bound to HaloTag, HT). In the beginning, we determined the optimal tag orientation of donor constructs: N or C-terminally NL-tagged cyclin E1 and acceptor constructs: N or C-terminally HT- tagged FBXW7 in HEK293T cells. NanoLuc and halo tag proteins, expressed alone, were used as controls to determine background signal. The best results were obtained with C-terminally NL-tagged cyclin E1 and C-terminally HT-tagged FBXW7 (Fig.1). The assay was further validated with a binding deficient FBXW7 mutant (R465E, R505E) designed by Matthieu Schapira. R465E, R505E double mutation resulted in a significant decrease in the interaction of FBXW7 with cyclin E1 (Fig.2).</p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/fbx711-300x132.jpg" alt="" width="598" height="263" class=" wp-image-5307 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/fbx711-300x132.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/fbx711-1024x451.jpg 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/fbx711-768x338.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/fbx711.jpg 1280w" sizes="(max-width: 598px) 100vw, 598px" /></p>
<p><strong>Fig.1. FBXW7 interaction with cyclin E1 &#8211; NanoBRET assay. </strong>HEK293T cells were co-transfected with C- or N-terminally NanoLuc (NL)-tagged cyclin E1 or NL alone and C- or N-terminally Halo-tagged (HT) FBXW7 or HT alone for 24 h. The interaction was measured using NanoBRET assay. The results are MEAN of 2 technical replicates.</p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/fbx7a-300x119.jpg" alt="" width="635" height="252" class=" wp-image-5306 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/fbx7a-300x119.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/fbx7a-1024x405.jpg 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/fbx7a-768x304.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/fbx7a.jpg 1145w" sizes="(max-width: 635px) 100vw, 635px" /></p>
<p><strong> </strong><strong>Fig.2. R465E, R505E double</strong><strong> mutation decreases the interaction of FBXW7 with cyclin E1.</strong> HEK293T cells were co-transfected with C-terminally Halo-tagged (HT) FBXW7 (WT or R465E/R505E or HT and C-terminally NanoLuc (NL)-tagged cyclin E1 for 24 h. The interaction was measured using NanoBRET assay. The results are MEAN of 2 technical replicates.</p>
<p>Please go to<a href="https://zenodo.org/record/4134887#.X5b8RohKg2w"> Zenodo</a> for experimental details.</p>
<p><strong>Acknowledgments:</strong></p>
<p>Matthieu Schapira &#8211; mutant design, Aidan Levinsky &#8211; help with mutagenesis</p>
<p>&nbsp;</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/fbxw7-interaction-with-cyclin-e1-cellular-assay/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>FBXW7_349-707_NHis_Purification Protocol</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/fbxw7_349-707_nhis_purification-protocol/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/fbxw7_349-707_nhis_purification-protocol/#respond</comments>
		
		<dc:creator><![CDATA[Taraneh Hajian]]></dc:creator>
		<pubDate>Thu, 22 Oct 2020 16:41:30 +0000</pubDate>
				<category><![CDATA[Protein purification protocols]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<category><![CDATA[Taraneh Hajian]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=5295</guid>

					<description><![CDATA[Please see FBXW7-349-707_Nhis Expression and Purification protocol in the link below: https://zenodo.org/record/4118423]]></description>
										<content:encoded><![CDATA[<p>Please see FBXW7-349-707_Nhis Expression and Purification protocol in the link below:</p>
<p><a href="https://zenodo.org/record/4118423">https://zenodo.org/record/4118423</a></p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/fbxw7_349-707_nhis_purification-protocol/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
	</channel>
</rss>
