<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>Scientist &#8211; openlabnotebooks.org</title>
	<atom:link href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/feed/" rel="self" type="application/rss+xml" />
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<description>A growing team of groundbreaking scientists around the world are now sharing their lab notebooks online</description>
	<lastBuildDate>Mon, 21 Sep 2020 13:09:33 +0000</lastBuildDate>
	<language>en-GB</language>
	<sy:updatePeriod>
	hourly	</sy:updatePeriod>
	<sy:updateFrequency>
	1	</sy:updateFrequency>
	<generator>https://wordpress.org/?v=5.4.2</generator>

<image>
	<url>https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/cropped-lab-jar02-32x32.png</url>
	<title>Scientist &#8211; openlabnotebooks.org</title>
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<width>32</width>
	<height>32</height>
</image> 
	<item>
		<title>Genetic Variability at the Remdesivir-binding Pocket of SARS-CoV-2 RNA-dependent RNA polymerase Across Coronaviruses and SARS-CoV-2 Samples &#8211; Post 20</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/genetic-variability-at-the-remdesivir-binding-pocket-of-sars-cov-2-rna-dependent-rna-polymerase-across-coronaviruses-and-sars-cov-2-samples-post-20/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/genetic-variability-at-the-remdesivir-binding-pocket-of-sars-cov-2-rna-dependent-rna-polymerase-across-coronaviruses-and-sars-cov-2-samples-post-20/#respond</comments>
		
		<dc:creator><![CDATA[Setayesh Yazdani]]></dc:creator>
		<pubDate>Mon, 21 Sep 2020 13:09:33 +0000</pubDate>
				<category><![CDATA[COVID-19]]></category>
		<category><![CDATA[SARS-CoV-2 computational genomics]]></category>
		<category><![CDATA[Setayesh Yazdani]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=5176</guid>

					<description><![CDATA[In my previous post 19, I showed how we assessed the druggability of the remdesivir-binding site of SARS-CoV-2 RNA-dependent RNA polymerase (RdRp) and determined the residues lining that site. In this post, we analyze the genetic diversity of RdRp catalytic site across Alpha- and Betacoronavirus entries from UniProt and among SARS-CoV-2 samples. In the context <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/genetic-variability-at-the-remdesivir-binding-pocket-of-sars-cov-2-rna-dependent-rna-polymerase-across-coronaviruses-and-sars-cov-2-samples-post-20/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p>In my previous post <a href="https://thesgc.github.io/static-openlabnotebooks/the-next-target-sars-cov-2-rna-dependent-rna-polymerase-and-its-druggability-post-19/">19</a>, I showed how we assessed the druggability of the remdesivir-binding site of SARS-CoV-2 RNA-dependent RNA polymerase (RdRp) and determined the residues lining that site. In this post, we analyze the genetic diversity of RdRp catalytic site across Alpha- and Betacoronavirus entries from UniProt and among SARS-CoV-2 samples.</p>
<p>In the context of the emergence of future MERS-like or SARS-like coronaviruses from the bat strains circulating in bat reservoir species, it is imperative to do a broad survey of viral proteins to identify the best strategies for the development of broad-spectrum viral inhibitors. (Sheahan et al., 2020) For this exact reason, we are looking at twenty-seven reviewed sequences from Uniprot, where six belong to entries of the Alphacoronavirus genus and twenty-one belong to the Betacoronavirus genus. We then assess the variability of amino acid lining the binding pockets found at the RdRP active site (PDB: 7bv2).</p>
<p>I made a diversity dendrogram focusing on the <a href="https://thesgc.github.io/static-openlabnotebooks/the-next-target-sars-cov-2-rna-dependent-rna-polymerase-and-its-druggability-post-19/">48 amino acid sidechains lining the pockets</a> juxtaposed to Remdesivir. Shown below in figure 1, the consensus profile at each of the 48 sidechains is shown. Strikingly, 83% of residues, 40 out of 48, are conserved across these 27 entries. This is one of the most conserved SARS-CoV-2 binding sites that we have seen so far (I will post a systematic comparison once I have reviewed all sites on all SARS-CoV-2 proteins in the PDB). Here are the 8 non-conserved residues: F442, A443, Y453, Y455, M543, N552, A558, S681. Across the 21 entries of the Betacoronavirus genus (including SARS-CoV-2), 46 out of 48 residues are conserved at the RdRp active site. Here are the non-conserved residues: F442 (almost perfectly conserved), and A443.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/09/RdRp-alpha-beta-dendrogram.png" alt="" width="1616" height="760" class="alignnone wp-image-5184 size-full" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/09/RdRp-alpha-beta-dendrogram.png 1616w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/09/RdRp-alpha-beta-dendrogram-300x141.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/09/RdRp-alpha-beta-dendrogram-1024x482.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/09/RdRp-alpha-beta-dendrogram-768x361.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/09/RdRp-alpha-beta-dendrogram-1536x722.png 1536w" sizes="(max-width: 1616px) 100vw, 1616px" /></p>
<p><strong>Figure 1. </strong><strong>The diversity dendrogram of the active site of RdRp for the entries of Alpha- and Betacoronavirus genera</strong><strong>.</strong></p>
<p>We mapped the variations shown among the entries of Alpha- and Betacoronavirus genera onto the SARS-CoV-2 RdRp crystal structure (PDB: 7bv2) such that sidechains that are variable among the entries are highlighted in orange and the conserved sidechains are shown in green.</p>
<p>&nbsp;</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/09/color-coded-RdRp-alpha-beta.png" alt="" width="975" height="684" class="alignnone wp-image-5191 size-full" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/09/color-coded-RdRp-alpha-beta.png 975w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/09/color-coded-RdRp-alpha-beta-300x210.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/09/color-coded-RdRp-alpha-beta-768x539.png 768w" sizes="(max-width: 975px) 100vw, 975px" /></p>
<p><strong>Figure 2. Mapping the genetic variation of UniProt entries belonging to Alpha- and Betacoronavirus genera onto SARS-CoV-2 RdRp Crystal structure (PDB: 7bv2). </strong>The eight non-conserved residues among the entries of both Alpha- and Betacoronavirus include F442, A443, Y453, Y455, M543, N552, A558, S681 while the two non-conserved residues among the Betacoronavirus entries include F442, A443.</p>
<p>Our next step was to assess the genetic variability of the catalytic site of RdRp among SARS-CoV-2 samples. Nicola De Maio, our collaborator from Nick Goldman’s lab at European Bioinformatics Institute, looked at more than 15000 SARS-CoV-2 samples and identified all mutations at the RdRp catalytic site pockets. He identified 6 non-synonymous variants as shown in table 1. For example, at residue number 443, the wild-type sidechain in SARS-CoV-2 RdRp is alanine. In the sample batch Nicola has looked at, he saw that 15870 samples had alanine at that position while there were 2 samples with valine at this exact position.</p>
<table width="629" style="width: 75.6566%;" height="365">
<tbody>
<tr>
<td width="47" style="width: 8.60861%;">Index</td>
<td width="123" style="width: 13.013%;">Non-synonymous Variants</td>
<td width="129" style="width: 12.1121%;">variants</td>
<td width="117" style="width: 11.9119%;">codon one</td>
<td width="108" style="width: 12.3123%;">codon two</td>
<td width="105" style="width: 14.3143%;">codon three</td>
</tr>
<tr>
<td width="47" style="width: 8.60861%;">1</td>
<td width="123" style="width: 13.013%;">A443</td>
<td width="129" style="width: 12.1121%;">A(15870), V(2)</td>
<td width="117" style="width: 11.9119%;">g(15872)</td>
<td width="108" style="width: 12.3123%;">c(15870), t(2)</td>
<td width="105" style="width: 14.3143%;">t(15872)</td>
</tr>
<tr>
<td width="47" style="width: 8.60861%;">2</td>
<td width="123" style="width: 13.013%;">Q444</td>
<td width="129" style="width: 12.1121%;">Q(15870), H(2)</td>
<td width="117" style="width: 11.9119%;">c(15872)</td>
<td width="108" style="width: 12.3123%;">a(15872)</td>
<td width="105" style="width: 14.3143%;">t(2), g(15870)</td>
</tr>
<tr>
<td width="47" style="width: 8.60861%;">3</td>
<td width="123" style="width: 13.013%;">M542</td>
<td width="129" style="width: 12.1121%;">M(15873), L(1)</td>
<td width="117" style="width: 11.9119%;">a(15874), c(1)</td>
<td width="108" style="width: 12.3123%;">t(15875)</td>
<td width="105" style="width: 14.3143%;">g(15874)</td>
</tr>
<tr>
<td width="47" style="width: 8.60861%;">4</td>
<td width="123" style="width: 13.013%;">A547</td>
<td width="129" style="width: 12.1121%;">A(15874), V(1)</td>
<td width="117" style="width: 11.9119%;">g(15875)</td>
<td width="108" style="width: 12.3123%;">c(15874), t(1)</td>
<td width="105" style="width: 14.3143%;">c(15875)</td>
</tr>
<tr>
<td width="47" style="width: 8.60861%;">5</td>
<td width="123" style="width: 13.013%;">A554</td>
<td width="129" style="width: 12.1121%;">A(15874), V(1)</td>
<td width="117" style="width: 11.9119%;">g(15875)</td>
<td width="108" style="width: 12.3123%;">c(15874), t(1)</td>
<td width="105" style="width: 14.3143%;">c(1), t(15874)</td>
</tr>
<tr>
<td width="47" style="width: 8.60861%;">6</td>
<td width="123" style="width: 13.013%;">V667</td>
<td width="129" style="width: 12.1121%;">V(15870), I(5)</td>
<td width="117" style="width: 11.9119%;">a(5), g(15870)</td>
<td width="108" style="width: 12.3123%;">t(15875)</td>
<td width="105" style="width: 14.3143%;">c(15875)</td>
</tr>
</tbody>
</table>
<p><strong>Table 1. </strong><strong>The non-synonymous variants at </strong><strong>RdRp active site </strong><strong>across more than 15000 SARS-CoV-2 samples.</strong></p>
<p>We could not predict the effect of these mutations on remdesivir binding due to the lack of RdRp structure where remdesivir has not been incorporated into the RNA primer sequence while it is sitting in a pocket in RdRp. Therefore, we can only comment on how the nature of the variant sidechains differ from the wild-type sidechains according to their physiochemical properties: the size and polarity of mutated residues are largely conserved in all cases (A-&gt;V, Q-&gt;H, V-&gt;I), suggesting high chances of a light effect on ligand binding.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/09/color-coded-RdRp-sars-cov-2.png" alt="" width="975" height="334" class="alignnone wp-image-5182 size-full" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/09/color-coded-RdRp-sars-cov-2.png 975w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/09/color-coded-RdRp-sars-cov-2-300x103.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/09/color-coded-RdRp-sars-cov-2-768x263.png 768w" sizes="(max-width: 975px) 100vw, 975px" /></p>
<p><strong>Figure 2. Left Panel: The surface representation of SARS-CoV-2 RdRp crystal structure with the non-conserved amino acid positions highlighted in orange. Right Panel: The sticks representation of the non-conserved residues among SARS-CoV-2 samples.</strong></p>
<p>In conclusion, from a structural standpoint, the catalytic site of RdRp is a good target for broad-spectrum inhibitors against coronaviruses considering that only 17% of the residues lining catalytic site binding pockets were not conserved across 27 Alpha- and Betacoronavirus entries. It is worth mentioning that clinically, the inhibition of RdRp which is achieved by remdesivir has not been fully confirmed to be an effective therapy for COVID-19.(Goldman et al, 2020)</p>
<p>According to the SARS-CoV-2 variants reported in table 1, the properties of most of the non-synonymous variants at the catalytic site were conserved, suggesting high chances of a light effect on ligand binding.</p>
<p>It would be crucial to investigate the structure of RdRp combined with remdesivir in the state right before it gets incorporated into the RNA primer. This would reveal the main interactions that remdesivir makes with RdRp sidechains and allow to better predict the effect of mutations both across coronaviruses broadly and more specifically across samples from COVID-19 patients on remdesivir binding.</p>
<p>To view the full report on SARS-CoV-2 RdRp, please visit this <a href="https://doi.org/10.5281/zenodo.4041302">Zenodo report. </a></p>
<p>As always, I would be happy to hear your comments. Please contact me via the <strong>“Leave a comment”</strong> link at the top of this post. Stay Tuned for my next post!</p>
<p><strong>References:</strong></p>
<p>Sheahan, T. P., Sims, A. C., Zhou, S., Graham, R. L., Hill, C. S., Leist, S. R., Schäfer, A., Dinnon, K. H., Montgomery, S. A., Agostini, M. L., Pruijssers, A. J., Chapell, J. D., Brown, A. J., Bluemling, G. R., Natchus, M. G., Saindane, M., Kolykhalov, A. A., Painter, G., Harcourt, J., … Baric, R. S. (2020). An orally bioavailable broad-spectrum antiviral inhibits SARS-Cov-2 and multiple endemic, epidemic and bat coronavirus. <a href="https://doi.org/10.1101/2020.03.19.997890">https://doi.org/10.1101/2020.03.19.997890</a></p>
<p>Goldman, J. D., Lye, D. C.B., Hui, D. S., Marks, K. M., Bruno, R., Montejano, R., Spinner, C. D., Galli, M., Ahn, M., Nahass, R. G., Chen, Y., SenGupta, D., Hyland, R. H., Osinusi, A. O., Cao, H., Blair, C., Wei, X., Gaggar, A., Brainard, D. M., &amp; Subramanian, A. (2020). Remdesivir for 5 or 10 Days in Patients with Severe Covid-19. <em>The New England Journal of Medicine</em>. <a href="https://doi.org/10.1056/NEJMoa2015301">https://doi.org/10.1056/NEJMoa2015301</a></p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/genetic-variability-at-the-remdesivir-binding-pocket-of-sars-cov-2-rna-dependent-rna-polymerase-across-coronaviruses-and-sars-cov-2-samples-post-20/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>The next target: SARS-CoV-2 RNA-dependent RNA polymerase and Its Druggability &#8211; Post 19</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/the-next-target-sars-cov-2-rna-dependent-rna-polymerase-and-its-druggability-post-19/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/the-next-target-sars-cov-2-rna-dependent-rna-polymerase-and-its-druggability-post-19/#respond</comments>
		
		<dc:creator><![CDATA[Setayesh Yazdani]]></dc:creator>
		<pubDate>Tue, 15 Sep 2020 16:35:45 +0000</pubDate>
				<category><![CDATA[COVID-19]]></category>
		<category><![CDATA[Setayesh Yazdani]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=5147</guid>

					<description><![CDATA[Previously we reported our analysis on the structural diversity of binding pockets found on the SARS-CoV-2 main protease, methyltransferase, and papain-like protease, macrodomain. We then shifted our focus on the SARS-CoV-2 RNA-dependent RNA polymerase (RdRp). SARS-CoV-2 is a positive-strand RNA virus, depending on its multi-subunit machinery to replicate its RNA. The catalytic subunit of RdRp <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/the-next-target-sars-cov-2-rna-dependent-rna-polymerase-and-its-druggability-post-19/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p>Previously we reported our analysis on the structural diversity of binding pockets found on the SARS-CoV-2 main protease, methyltransferase, and papain-like protease, macrodomain. We then shifted our focus on the SARS-CoV-2 RNA-dependent RNA polymerase (RdRp).</p>
<p>SARS-CoV-2 is a positive-strand RNA virus, depending on its multi-subunit machinery to replicate its RNA. The catalytic subunit of RdRp is the core component of this machinery, which is also known as nsp12. Nsp12 relies on accessory subunits to have full activity. The accessory factors, nsp7 and nsp8, increase RdRp template binding and processivity. (Yin et al., 2020)</p>
<p>RdRp is a well-established target for a class of antiviral drugs known as nucleotide analogs. One of the known and only FDA and Health Canada approved medications for COVID-19, remdesivir, targets RdRp.</p>
<p>Remdesivir is converted to its active form, remdesivir triphosphate (RTP), that gets incorporated into RNA and causes chain termination.</p>
<p>It is important to note that remdesivir has been tested in phase III clinical trial. The results were shared in The New England Journal of Medicine on May 27, 2020, by Goldman and his colleagues. (Goldman et al, 2020) The use of remdesivir for 5 to 10 days in patients with severe COVID-19 showed that there was no significant difference between the 5-day treatment and 10-day treatment groups. The overall effect on the survival rate was rather low. It is important to note this study did not have a placebo control arm and that means the magnitude of the benefit cannot be determined.</p>
<p>Yin et al solved the crystal structure of RdRp in complex with a remdesivir-incorporated RNA strand (PDB code 7bv2). The authors published two structures of RdRp; one in the apo form containing one nsp12, nsp7, and two nsp8 proteins (PDB: 7bv1) shown in figure 1A and another bound to the remdesivir-incorporated RNA, containing nsp12, one nsp7, and one nsp8 (PDB: 7bv2) shown in figure 1B.</p>
<p>Upon incorporation of the remdesivir into the primer strand, there is immediate termination of chain elongation. This inhibitory mechanism is called nonobligate RNA chain termination by remdesivir where the prodrug needs conversion to the active triphosphate form, RTP. (Yin et al., 2020)The interactions between remdesivir monophosphate (RMP) and RdRp are shown in figure 2.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/09/RdRp-1AB.png" alt="" width="506" height="339" class="alignnone wp-image-5150 " srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/09/RdRp-1AB.png 903w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/09/RdRp-1AB-300x201.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/09/RdRp-1AB-768x515.png 768w" sizes="(max-width: 506px) 100vw, 506px" /></p>
<p><strong>Figure 1. (A) Views of Cryo-electron microscopy structure of nsp12-nsp7-nsp8 RdRp complex</strong> <strong>(B) and nsp12-nsp7-nsp8 in complex with the primer RNA template and remdesivir</strong> (Yin et al., 2020)</p>
<p>Once RdRp is in the template-RTP form, it goes into a closed conformation where the 11 base-pairs template primer is held by the finger-palm-thumb subdomains. There are also extensive RNA-protein interactions that involve the RNA phosphate-ribose backbone. In the absence of specific interactions between any specific base-pairs and RdRp, the binding of RNA by RdRp becomes independent of the RNA sequence which explains why no specific sequence is needed for RdRp’s enzymatic activity. (Yin et al., 2020)</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/09/RdRp-2.png.jpg" alt="" width="303" height="264" class="alignnone wp-image-5162 " srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/09/RdRp-2.png.jpg 476w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/09/RdRp-2.png-300x262.jpg 300w" sizes="(max-width: 303px) 100vw, 303px" /></p>
<p><strong>Figure 2. The chemical interactions between the RdRp catalytic site residues and RMP (purple). </strong>(Yin et al., 2020)</p>
<p>We aim to analyze the genetic diversity of RdRp catalytic site across coronaviruses and SARS-CoV-2 samples. To achieve this goal, we first assessed the druggability of the remdesivir-binding site of RdRp and determined the residues lining that site which I explain in this post.</p>
<p>Using SiteMap (Schrodinger, NY) we assessed the druggability of the binding sites using the druggability score (Dscore). We found that the active site of RdRp in the vicinity of bound remdesivir has a Dscore of 0.859 indicative of a difficult druggable site. We show a schematic of this site in figure 3.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/09/RdRp-druggability-rem.png" alt="" width="1004" height="570" class="alignnone wp-image-5153 size-full" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/09/RdRp-druggability-rem.png 1004w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/09/RdRp-druggability-rem-300x170.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/09/RdRp-druggability-rem-768x436.png 768w" sizes="(max-width: 1004px) 100vw, 1004px" /></p>
<p><strong>Figure 3. </strong><strong>Druggability analysis of SARS-CoV-2 RdRp active site. </strong>Cavities at the catalytic site (orange and magenta blobs) are found next to the position occupied by remdesivir monophosphate incorporated into the RNA template<strong> (purple stick). </strong></p>
<p>Using ICM PocketFinder, we find that RMP is not occupying a clearly defined binding pocket but is juxtaposed to two cavities. This probably reflects the fact that we are looking at a structure where remdesivir was already incorporated into RNA. The site is structurally conserved in the apo form (PDB: 7bv1). We hypothesize that compounds occupying these cavities would block the catalytic activity of RdRp. We identified 48 amino acid sidechains lining the pockets juxtaposed to Remdesivir shown in figures 3 and 4 (within 2.8 Å of the pocket): V410, F440, F441, F442, A443, Q444, D452, Y453, Y455, Y456, Y458, T540, M542, N543, L544, K545, Y546, A547, I548, S549, K551, N552, R553, A554, R555, T556, V557, A558, G559, Y619, P620, K621, C622, D623, R624, V667, K676, G679, T680, S681, S682, G683, D684, T687, A688, N691, S759, D760.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/09/RdRp-4-neigh_res_rem.png" alt="" width="671" height="467" class="alignnone wp-image-5154 " srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/09/RdRp-4-neigh_res_rem.png 1266w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/09/RdRp-4-neigh_res_rem-300x209.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/09/RdRp-4-neigh_res_rem-1024x713.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/09/RdRp-4-neigh_res_rem-768x534.png 768w" sizes="(max-width: 671px) 100vw, 671px" /></p>
<p><strong>Figure 4. </strong><strong>The neighboring residues lining the active site of RdRp.</strong></p>
<p>In my next post, I will show the sequence diversity across UniProt entries from the Alpha- and Betacoronavirus genera and map that to the RdRp remdesivir-binding site using its crystal structure.</p>
<p>As always, I would be happy to hear your comments. Please contact me via the “Leave a comment” link at the top of this post. Stay Tuned for my next post!</p>
<p><strong>References:</strong></p>
<p>Yin, W., Mao, C., Luan, X., Shen, D., Shen, Q., Su, H., Wang, X., Zhou, F., Zhao, W., Gao, M., Chang, S., Xie, Y., Tian, G., Jiang, H., Tao, S., Shen, J., Jiang, Y., Jiang, H., Xu, Y., &amp; Xu, H. (2020). Structural basis for inhibition of the RNA-dependent RNA polymerase from SARS-CoV-2 by remdesivir. <em>Science</em>, <em>368</em>(6498), 1499-1504. <a href="https://doi.org/10.1126/science.abc1560">https://doi.org/10.1126/science.abc1560</a></p>
<p>Goldman, J. D., Lye, D. C.B., Hui, D. S., Marks, K. M., Bruno, R., Montejano, R., Spinner, C. D., Galli, M., Ahn, M., Nahass, R. G., Chen, Y., SenGupta, D., Hyland, R. H., Osinusi, A. O., Cao, H., Blair, C., Wei, X., Gaggar, A., Brainard, D. M., &amp; Subramanian, A. (2020). Remdesivir for 5 or 10 Days in Patients with Severe Covid-19. <em>The New England Journal of Medicine</em>. <a href="https://doi.org/10.1056/NEJMoa2015301">https://doi.org/10.1056/NEJMoa2015301</a></p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/the-next-target-sars-cov-2-rna-dependent-rna-polymerase-and-its-druggability-post-19/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>Calculating the change in Gibbs free energy (ddGbind) of an improved alpha-ketoamide inhibitor binding associated with genetic variations of SARS-CoV-2 main protease &#8211; post18</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/calculating-the-change-in-gibbs-free-energy-ddgbind-of-an-improved-alpha-ketoamide-inhibitor-binding-associated-with-genetic-variations-of-sars-cov-2-main-protease-post18/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/calculating-the-change-in-gibbs-free-energy-ddgbind-of-an-improved-alpha-ketoamide-inhibitor-binding-associated-with-genetic-variations-of-sars-cov-2-main-protease-post18/#respond</comments>
		
		<dc:creator><![CDATA[Setayesh Yazdani]]></dc:creator>
		<pubDate>Mon, 24 Aug 2020 21:20:22 +0000</pubDate>
				<category><![CDATA[COVID-19]]></category>
		<category><![CDATA[SARS-CoV-2 computational genomics]]></category>
		<category><![CDATA[Setayesh Yazdani]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=5099</guid>

					<description><![CDATA[Since we posted our analysis of genetic variation of SARS-CoV-2 main protease (MPro) with inhibitor N3 (PDB: 7bqy), there have been other inhibitors co-crystallized with the MPro (PDB: 6lze, PDB: 6m0k, and PDB: 6y2f). In my last post, I talked about the two inhibitors, 11a and 11b (PDB: 6lze, PDB: 6m0k) and in today&#8217;s post, <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/calculating-the-change-in-gibbs-free-energy-ddgbind-of-an-improved-alpha-ketoamide-inhibitor-binding-associated-with-genetic-variations-of-sars-cov-2-main-protease-post18/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p>Since we posted <a href="https://thesgc.github.io/static-openlabnotebooks/calculating-the-change-in-gibbs-free-energy-ddgbind-of-inhibitor-binding-associated-with-genetic-variations-of-sars-cov-2-main-protease/">our analysis of genetic variation of SARS-CoV-2 main protease (MPro) with inhibitor N3</a> (PDB: 7bqy), there have been other inhibitors co-crystallized with the MPro (PDB: 6lze, PDB: 6m0k, and PDB: 6y2f).</p>
<p>In my <a href="https://thesgc.github.io/static-openlabnotebooks/back-to-sars-cov-2-main-protease-calculating-the-change-in-gibbs-free-energy-ddgbind-of-two-peptidomimetic-aldehyde-inhibitors-binding-associated-with-genetic-variations-of-sars-cov-2-main-protease/">last post</a>, I talked about the two inhibitors, 11a and 11b (PDB: 6lze, PDB: 6m0k) and in today&#8217;s post, we focus on another MPro structure co-crystallized with an inhibitor called 13b (PDB: 6y2f).</p>
<p>The premise for the MPro inhibitor discussed in this post is based on the authors&#8217; previous work where they designed a peptidomimetic α-ketoamide, 11r (Figure 1), as a broad-spectrum inhibitor of coronavirus and enterovirus replication. (1) In a subsequent study (2), the authors attempt to improve the half-life of the previous inhibitor in plasma by incorporating the p3-p2 amide bond in 11r into a pyridine ring in 13a. (Figure 1) They speculated that this change might hide the amide bond from cellular proteases and hence prevent its cleavage. Furthermore, the authors justified that to enhance the antiviral activity against betacoronaviruses of clade b (SARS-CoV-2 and SARS-CoV) they replaced the cyclohexyl moiety of 13a by cyclopropyl in 13b which is a smaller moiety. By changing 13a to 13b, they sacrificed the goal of broad-spectrum inhibitor activity of their compound, as the cyclohexyl in 13a was intended to fill in the enterovirus MPro which is replaced by a smaller cyclopropyl moiety in 13b.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/11r-13b.png" alt="" width="550" height="337" class="alignnone wp-image-5116 " srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/11r-13b.png 1294w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/11r-13b-300x184.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/11r-13b-1024x628.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/11r-13b-768x471.png 768w" sizes="(max-width: 550px) 100vw, 550px" /></p>
<p><strong>Figure 1. The design rationale for an improved alpha-ketoamide inhibitor, the path from 11r to 13b.</strong></p>
<p>The IC50 value of 13b was at 0.67 ± 0.18 μM for SARS-CoV-2 MPro and 0.90 ± 0.29 μM and 0.58 ± 0.22 μM for SARS-CoV Mpro and the MERS-CoV Mpro respectively. Additionally, the researchers in this study showed that 13b has lung tropism which means that 13b was detectable in mice’s lung tissue 4 hours following its subcutaneous administration. Lung tropism is desirable considering the tissue that COVID-19 mostly affects.</p>
<p>The catalytic residue of the MPro catalytic site, Cys145, performs a nucleophilic attack onto the α-keto group of 13b to form a thiohemiketal bond in a reversible reaction and thus MPro is inhibited.</p>
<p>Their study provided a framework for the development of pyridone-containing inhibitors to be used as antiviral drugs.</p>
<p>In this post, I will explain how the non-synonymous variants found at SARS-CoV-2 MPro catalytic site across thousands of SARS-CoV-2 samples from COVID-19 patients are predicted to affect 13b binding. (3)</p>
<p>To predict the effect of those identified mutations on inhibitor (13b) binding, we use ICM software (Molsoft, SD). In ICM, by mutating the residues, we can see the change in Gibbs free energy of inhibitor binding (ddGbind) which would indicate if the mutation penalizes the binding of the inhibitor significantly or not.</p>
<p>To see the detailed methods on how the change in Gibbs free energy of ligand binding is calculated, please view this Zenodo post <a href="https://doi.org/10.5281/zenodo.3928729"><u>here</u></a>. Below in table 1, we show the four non-synonymous mutations and the corresponding ddGbind values.</p>
<table width="463" style="width: 76.0487%; height: 214px;" height="313">
<tbody>
<tr style="height: 58px;">
<td width="75" style="height: 58px;">Residue number</td>
<td width="66" style="height: 58px;">Wild Type</td>
<td width="59" style="height: 58px;">Mutant</td>
<td width="73" style="height: 58px;"><strong>ddGbind</strong></td>
<td width="90" style="height: 58px;">dGbind_wild</td>
<td width="99" style="height: 58px;">dGbind_mut</td>
</tr>
<tr style="height: 39px;">
<td width="75" style="height: 39px;">49</td>
<td width="66" style="height: 39px;">Met</td>
<td width="59" style="height: 39px;">Ile</td>
<td width="73" style="height: 39px;">-0.6</td>
<td width="90" style="height: 39px;">-27.1</td>
<td width="99" style="height: 39px;">-27.7</td>
</tr>
<tr style="height: 39px;">
<td width="75" style="height: 39px;">52</td>
<td width="66" style="height: 39px;">Pro</td>
<td width="59" style="height: 39px;">Ser</td>
<td width="73" style="height: 39px;">-0.1</td>
<td width="90" style="height: 39px;">-27.1</td>
<td width="99" style="height: 39px;">-27.2</td>
</tr>
<tr style="height: 39px;">
<td width="75" style="height: 39px;">142</td>
<td width="66" style="height: 39px;">Asn</td>
<td width="59" style="height: 39px;">Ser</td>
<td width="73" style="height: 39px;">1.7</td>
<td width="90" style="height: 39px;">-27.1</td>
<td width="99" style="height: 39px;">-25.4</td>
</tr>
<tr style="height: 39px;">
<td width="75" style="height: 39px;">168</td>
<td width="66" style="height: 39px;">Pro</td>
<td width="59" style="height: 39px;">Ser</td>
<td width="73" style="height: 39px;">0.4</td>
<td width="90" style="height: 39px;">-27.1</td>
<td width="99" style="height: 39px;">-26.7</td>
</tr>
</tbody>
</table>
<p><strong>Table 1.</strong> <strong>The predicted change in Gibbs free energy associated with the non-synonymous variants found at MPro catalytic site across more than 15000 SARS-CoV-2 samples.</strong> ddGbind values are in Kcal/mol.</p>
<p>Based on the precision of our method, none of the non-synonymous mutations are predicted to affect 13b binding significantly, since all ddGbind values are less than 2.0 Kcal/mol. (4) Prediction of no significant changes in ddGbind values for the four non-synonymous mutations (M49I, P52S, N142S, and P168S) is possibly due to the conservation of key ligand-protein interactions after these residues are mutated. In figure 2, we show the 2D ligand-protein interaction diagram which outlines the chemical interactions between 13b and MPro catalytic site residues.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/13b-2D-ligand-protein-interactions.png" alt="" width="407" height="349" class="alignnone wp-image-5117 " srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/13b-2D-ligand-protein-interactions.png 683w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/13b-2D-ligand-protein-interactions-300x257.png 300w" sizes="(max-width: 407px) 100vw, 407px" /></p>
<p><strong>Figure 2. The 2D ligand-protein interaction diagram between ligand 13b and the residues of the MPro active site. (PDB: 6y2f).</strong> This diagram was created using Pose.View in Proteins.Plus.</p>
<p>We created an energy matrix table where we consider the effect of every possible mutation on the binding of the ligand (inhibitor 13b) with SARS-CoV-2-MPro. Table 2 includes a list of the 20 residues at MPro catalytic site expect Cys145. Cys145 is not included as it makes a covalent bond with 13b: any mutation will severely affect 13b binding.</p>
<p>It is likely that the residues of the rows with green color are not making optimal interactions with 13b. This is reflected in the very slight and non-significant lowering in ddGbind values of these residues (mutations are stabilizing) and hence the green color. On the other hand, the sidechains whose mutations are shown in orange are likely the positions where MPro would better interact with 13b which include H41, S144, M165, E166, P168, D187. As a result, the mutations of those sidechains are predicted to penalize inhibitor-binding significantly and hence the orange color. According to Zhang and her colleagues (3), His41 provides a hydrogen bond to stabilize the oxyanion group of thiohemiketal link between 13b which is also reflected in the orange row in table 3. The amide oxygen of 13b partly accepts a hydrogen bond from the main-chain amide of Ser144, which forms the canonical “oxyanion hole” of the cysteine protease, making S144 an important residue for 13b inhibitory mechanism.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/13b-energy-matrix.png" alt="" width="1908" height="728" class="alignnone wp-image-5118 size-full" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/13b-energy-matrix.png 1908w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/13b-energy-matrix-300x114.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/13b-energy-matrix-1024x391.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/13b-energy-matrix-768x293.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/13b-energy-matrix-1536x586.png 1536w" sizes="(max-width: 1908px) 100vw, 1908px" /></p>
<p><strong>Table 2. </strong><strong>The energy matrix showing the effect of all possible mutations at the SARS-COV-2 MPro catalytic site and the predicted effect on inhibitor 13b binding.</strong> No mutations at Cys145 were considered since it makes the covalent thiohemiketal linkage with 13b which is essential for the inhibitory mechanism in this scenario. The energy unit is kcal/mol.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/energy-matrix-color-coding.png" alt="" width="133" height="118" class="alignnone wp-image-5041" /></p>
<p>Altogether, for the improved α-ketoamide inhibitor, 13b, we can conclude that the rare variants found in SARS-CoV-2 samples are only expected to minimally affect 13b binding to the MPro.</p>
<p><strong>References:</strong></p>
<ol>
<li>Zhang, Linlin., α-Ketoamides as broad-spectrum inhibitors of coronavirus and enterovirus replication: Structure-based design, synthesis, and activity assessment. <em>Journal of Medicinal Chemistry</em>. 63 (9), 4562–4578 (2020). <a href="https://doi.org/10.1021/acs.jmedchem.9b01828">https://doi.org/10.1021/acs.jmedchem.9b01828</a></li>
<li>Zhang, Linlin., Crystal structure of SARS-CoV-2 main protease provides a basis for design of improved α-ketoamide inhibitors. <em>Science</em> 368(6489), 409-412 (2020). <a href="https://doi.org/10.1126/science.abb3405">https://doi.org/10.1126/science.abb3405</a></li>
<li>Nicola De Maio. <em>Genetic diversity at the catalytic site of the main SARS-Cov-2 protease.</em> (2020) openlabnotebooks.org –https://thesgc.github.io/static-openlabnotebooks/genetic-diversity-at-the-catalytic-site-of-the-main-sars-cov-2-protease/</li>
<li>Schapira, M., Totrov, M., Abagyan, R. Prediction of the Binding Energy for Small Molecules, Peptides and Proteins. <em>Journal of Molecular Recognition </em>12(3), 177-90 (1999). <a href="https://doi.org/10.1002/(SICI)1099-1352(199905/06)12:3%3c177::AID-JMR451%3e3.0.CO;2-Z">https://doi.org/10.1002/(SICI)1099-1352(199905/06)12:3&lt;177::AID-JMR451&gt;3.0.CO;2-Z</a></li>
</ol>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/calculating-the-change-in-gibbs-free-energy-ddgbind-of-an-improved-alpha-ketoamide-inhibitor-binding-associated-with-genetic-variations-of-sars-cov-2-main-protease-post18/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>Brachyury (TBXT) ligands for chordoma: 1. Fragment optimization strategy 2. fragment F9000505, B-site, PDB 5QRU</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/brachyury-tbxt-ligands-for-chordoma-1-fragment-optimization-strategy-2-fragment-f9000505-b-site-pdb-5qru/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/brachyury-tbxt-ligands-for-chordoma-1-fragment-optimization-strategy-2-fragment-f9000505-b-site-pdb-5qru/#respond</comments>
		
		<dc:creator><![CDATA[David Drewry]]></dc:creator>
		<pubDate>Mon, 24 Aug 2020 18:33:44 +0000</pubDate>
				<category><![CDATA[Chordoma]]></category>
		<category><![CDATA[David Drewry]]></category>
		<category><![CDATA[SGC UNC Chapel Hill]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=5102</guid>

					<description><![CDATA[Introduction In this blog post I will share details on the new molecules we have designed and synthesized or purchased based on one of our original hits from the x-ray crystallography fragment screen. The compound is called F9000505 and it is bound to pocket B. The crystal structure was deposited in the Protein Data Bank <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/brachyury-tbxt-ligands-for-chordoma-1-fragment-optimization-strategy-2-fragment-f9000505-b-site-pdb-5qru/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p><strong><u>Introduction</u></strong></p>
<p>In this blog post I will share details on the new molecules we have designed and synthesized or purchased based on one of our original hits from the x-ray crystallography fragment screen. The compound is called F9000505 and it is bound to pocket B. The crystal structure was deposited in the Protein Data Bank (pdb) with PDB ID 5QRU. This post will give you a feel for some of the ways we are approaching fragment optimization. Posts in the near future will describe strategies at other fragment sites and also provide data on the binding of these compounds to brachyury as it emerges.</p>
<p><strong><u>Background / Project Goal</u></strong></p>
<p>The expression of brachyury (TBXT) is a hallmark of chordoma. A growing body of data suggests that inhibition of TBXT protein activity, expression, or interactions with protein partners are predicted to be effective and specific means for treating chordoma. A key aim of this project is to discover compounds that bind directly to brachyury with sufficient affinity so that they ca be optimized into clinical candidates for the treatment of chordoma.</p>
<p><strong><u>Initial Fragment Optimization Strategy </u></strong></p>
<p>Fragment-based screening by x-ray crystallography is very powerful in that the location of the fragment hits on the target protein (brachyury) are known after the experiment, and the structure provides details of the interactions that the fragments make with brachyury. Crystallography can identify fragments that bind very weakly, so there is often a long way to go to grow the hit molecules into compound that bind potently enough to elicit the desired biological effects. We are taking an aggressive approach to exploring each of our hits so that we can soon focus efforts on the most promising molecules and binding sites.</p>
<p>The two sources of new molecules are purchase of commercially available compounds, and design and synthesis of new compounds. Purchase of compounds is cost effective and rapid in the early stages of a project but becomes limited as molecules become more complicated to capture the intricacies of the binding sites.</p>
<p>We have purchased compounds primarily through a company called ChemSpace. We performed substructure and similarity searches in the ChemSpace available compound database for each x-ray crystallography hit (“query” molecules for the searching). A substructure search means specific details of the hit need to be present in the molecules identified by the search. A similarity search relaxes the criteria and allows for selection of molecules that the computer tells us have certain similarities to our query molecules. These similarities could be in categories like size, shape, and electronic properties. Similarity searches can identify compounds that are more different from the hits which may be important if they identify new interactions, scaffolds that are easier to make new analogs of, or compound that explore different vectors (directions) around the hits.</p>
<p>We chose initial analogues for purchase to provide important information around each fragment in several domains:</p>
<ol>
<li>Compounds with functional groups that will serve as efficient handles for further elaboration.</li>
<li>Compounds that explore growth of the hit fragment in logical directions based on the crystal structure.</li>
<li>Compounds that add additional functionality that can make new interactions in order to enhance potency.</li>
<li>Compounds that teach us where linkers may be added in order to develop brachyury degraders.</li>
</ol>
<p>By making the fragments larger we can understand where growth is beneficial to potency. This will provide focus to future rounds of purchase or chemistry. Incorporation of chemical handles that can be rapidly elaborated with a variety of building blocks will allow us to make additional analogs quickly. Adding functionality (for example amides, ureas, phenols, heterocycles, substituted phenyls) that can make energetically favorable interactions (hydrogen bonds, pi-pi stacking, lipophilic interactions, charge-charge interactions) improves our chances of increasing potency. Utilizing commercially available analogs of the initial fragments found bound to the TBXT crystal structure provides the means to quickly generate structure activity relationships (SAR – this means how changing the structure changes how well the compound binds to brachyury) around each fragment. Initial SAR is critical as it provides confidence around each hit (can it be optimized, do several similar compounds also bind) and aids in site selection to narrow our focus.</p>
<p><strong><u>Fragment F9000505, binding site B, PDB ID: 5QRU </u></strong></p>
<p><em> </em><em>Compounds for purchase</em></p>
<p><strong>Figure 1</strong> depicts all the compounds that we could buy identified from substructure and similarity searching around fragment hit F9000505. This compound binds to site B, and more details on the way it binds will be found below in the new compound design section. Each dot represents a compound and they are organized in the plot according to two easy to calculate and important properties: lipophilicity (clogP) and molecular weight (MW). The green circles are the compounds that we have purchased and are screening. If any of these are active, we can purchase other compounds (blue dots) that can further supplement the SAR. This iterative cycle of design, synthesis (purchase if possible), and test is the life blood of medicinal chemistry compound optimization.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig1-1024x725.png" alt="" width="780" height="552" class="aligncenter wp-image-5104 size-large" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig1-1024x725.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig1-300x213.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig1-768x544.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig1.png 1200w" sizes="(max-width: 780px) 100vw, 780px" /></p>
<p><strong>Figure 2</strong> depicts examples of molecules we purchased from this set that demonstrate the availability of useful molecules that can answer some of the specific medicinal chemistry questions we need to ask. The blue boxes highlight the areas with structural modifications to the hit compound.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-2.png" alt="" width="643" height="604" class="aligncenter wp-image-5105 " srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-2.png 978w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-2-300x282.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-2-768x721.png 768w" sizes="(max-width: 643px) 100vw, 643px" /></p>
<p><em>Design and Synthesis of New Compounds</em></p>
<p>We have chemists on our team at UNC and also a talented team of chemists at a company called Piramal. We have been able to make excellent progress on new compound synthesis even with the initial lab shutdowns and then the reduced capacity necessitated by Covid-19. The team at Piramal has focused on preparation of analogs of fragments for which there are fewer commercial analogs, and on more elaborate chemistry to explore promising fragment sites in greater detail using the same principles outlined above.</p>
<p><strong>Figure 3</strong> depicts the crystal structure of fragment F9000505 bound to pocket B of brachyury. This simple compound results from the acylation of 2-methoxy-5-methylaniline with the amino acid glycine. The phenyl ring makes lipophilic contacts with the protein and the 5-methyl group tucks into a lipophilic pocket. The oxygen of the 2-methoxy group makes an intramolecular hydrogen bond with the adjacent NH, helping to orient the trajectory of the carbonyl of the amide and also the terminal amine of the glycine. In this locked in orientation the carbonyl of the fragment forms a hydrogen bond with Ala119 of brachyury and the primary amine forms a hydrogen bond to the carbonyl of Pro117.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog2-fig3-286x300.png" alt="" width="315" height="330" class="aligncenter wp-image-5106 " srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog2-fig3-286x300.png 286w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog2-fig3-768x806.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog2-fig3.png 806w" sizes="(max-width: 315px) 100vw, 315px" /></p>
<p>Based on this structure there are a number of avenues that need to be explored for optimization of binding. The methyl substituent in the lipophilic pocket may not be ideal. Groups with different size and electronic characteristics need to be explored (for example: F, Cl, ethyl, cyclopropyl). There is plenty of room to extend off of the methoxy group. Doing this may allow us to pick up additional interactions with the surface of the protein. In addition, this may be an attractive position for a linker that will be suitable for synthesis of a bifunctional degrader. Another strategy is to extend the primary amine in order to reach Glu116 which could increase affinity. The alpha carbon of the glycine moiety presents another opportunity. Simple substituents like methyl will add conformational constraint which can enhance potency. Because this building block comes from amino acids, there is opportunity to quickly explore stereochemistry at this position and a range of functional groups that have potential to form new interactions. Finally, we can form a 5- or 6-membered ring between the NH and the O of the fragment. This will serve to further lockdown the orientation of the carbonyl and NH2 that make key hydrogen bonds and offer different vectors for exploring interactions with other brachyury residues. Our initial compound design targeted 68 molecules. These molecules have all been synthesized and submitted from screening in the surface plasmon resonance (SPR) binding assay. A sampling of the 68 compounds are depicted in <strong>Figure 4</strong>. These show example modifications of the methyl group (<strong>4A</strong>), primary amine (<strong>4B</strong>), the alpha carbon of the glycine moiety (<strong>4C</strong>), and the methyl ether (<strong>4D</strong>).</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-4.png" alt="" width="861" height="573" class="wp-image-5107 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-4.png 1538w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-4-300x200.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-4-1024x682.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-4-768x511.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-4-1536x1023.png 1536w" sizes="(max-width: 861px) 100vw, 861px" /></p>
<p><em>Synthetic chemistry schemes</em></p>
<p><strong>Figure 5</strong> show the chemistry scheme used to make many of the 68 new analogues of F9000505 synthesized for this project.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-5.png" alt="" width="698" height="172" class="alignnone wp-image-5108 " srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-5.png 1434w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-5-300x74.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-5-1024x253.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-5-768x190.png 768w" sizes="(max-width: 698px) 100vw, 698px" /></p>
<p><strong>Figure 6</strong> demonstrates utilization of an easily synthesized key intermediate to explore the addition of new functionality at three different distances from the core, linked with an amide. If any offer enhanced potency, follow up will focus on modifications of the pendant aryl ring.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-6.png" alt="" width="584" height="275" class="wp-image-5109 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-6.png 1332w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-6-300x141.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-6-1024x483.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-6-768x362.png 768w" sizes="(max-width: 584px) 100vw, 584px" /></p>
<p><strong>Figure 7</strong> depicts three different ways to functionalize the same key intermediate with robust chemistry that provides different sets of products. A key element of the initial fragment analogue strategy is to generate and use key intermediates that can be diversified in the last synthetic step.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-7.png" alt="" width="396" height="220" class="wp-image-5110 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-7.png 910w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-7-300x167.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-7-768x427.png 768w" sizes="(max-width: 396px) 100vw, 396px" /></p>
<p>The crystal structure suggests that new interactions may be possible if we grow the molecule in the ether position. In addition to making several different ethers, we have opted to incorporate both an alkyne linker and a triazole linker in this position. These choices offer different spacing, different angles, different functionalities, and can be expanded with robust chemistry. An overview of this linker modification chemistry is shown in <strong>figure 8</strong>.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-8.png" alt="" width="797" height="458" class="aligncenter wp-image-5111 " srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-8.png 1494w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-8-300x172.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-8-1024x588.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-8-768x441.png 768w" sizes="(max-width: 797px) 100vw, 797px" /></p>
<p>Finally, a key element of the strategy is to identify positions on the molecules where linkers can be attached to generate bifunctional degradation reagents. Crystallography will of course provide critical information to guide linker location. To improve our chances of quickly identifying potential linker locations, we are routinely incorporating some building blocks with functionality that lends itself to linker friendly chemistry. Currently these are mostly simple amides and Boc-protected amines, but azides and alkynes would also be valuable additions. Some newly synthesized compounds with “linker-friendly” functionality are shown in <strong>figure 9</strong>.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog2-fig-9.png" alt="" width="605" height="294" class="wp-image-5112 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog2-fig-9.png 1290w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog2-fig-9-300x146.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog2-fig-9-1024x499.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog2-fig-9-768x374.png 768w" sizes="(max-width: 605px) 100vw, 605px" /></p>
<p>&nbsp;</p>
<p>Stay tuned for details on compounds we have made and purchased for other sites and, importantly, the SPR screening data that should be available soon that lets us know how we are doing in terms of binding affinity!</p>
<p>Don&#8217;t hesitate to reach out with questions and comments.</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/brachyury-tbxt-ligands-for-chordoma-1-fragment-optimization-strategy-2-fragment-f9000505-b-site-pdb-5qru/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>Electroporation: a second look</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/electroporation-a-second-look/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/electroporation-a-second-look/#respond</comments>
		
		<dc:creator><![CDATA[Elizabeth Brown]]></dc:creator>
		<pubDate>Wed, 19 Aug 2020 17:07:37 +0000</pubDate>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Brain Tumours]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>
		<category><![CDATA[Liz Brown]]></category>
		<category><![CDATA[SGC Univ. of Oxford]]></category>
		<category><![CDATA[Understudied Kinases]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=5092</guid>

					<description><![CDATA[It’s never too late to take another look at your data. The incredibly knowledgeable Sally Cowley suggested that I shouldn’t just think about how many cells contain any AlexFluor488 antibody after electroporation, but how bright each cell is (i.e. the amplitude of fluorescence). Looking at this instead will give me an idea of how much <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/electroporation-a-second-look/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p>It’s never too late to take another look at your data.</p>
<p>The incredibly knowledgeable <a href="https://www.stemcells.ox.ac.uk/sally-cowley">Sally Cowley</a> suggested that I shouldn’t just think about how many cells contain <em>any </em>AlexFluor488 antibody after electroporation, but <em>how bright each cell is </em>(i.e. the amplitude of fluorescence). Looking at this instead will give me an idea of <em>how much</em> protein enters each cell. So. I went back to the old data, saw that the electroporation pre-sets <em>do</em> have an effect on fluorescence amplitude, and added a new gate to quantify the difference.</p>
<p>With this I could see that my previous best pre-set produced only 27% “hyper-positive” cells, compared with 93% hyper-positive cells in the new Best Electroporation Pre-set!</p>
<p>&nbsp;</p>
<p><a href="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/Image-1.png"><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/Image-1-1024x478.png" alt="Two side by side graphs entitled 'old &quot;best&quot; settings' and 'new best settings!' that display the number of cells versus the amplitude of Alexa-Fluor 488 flourescence. The right hand graph show that these settings produce more highly fluorescent cells." class="aligncenter wp-image-5093 size-large" width="780" height="364" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/Image-1-1024x478.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/Image-1-300x140.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/Image-1-768x359.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/Image-1-1536x717.png 1536w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/Image-1-2048x956.png 2048w" sizes="(max-width: 780px) 100vw, 780px" /></a></p>
<p>At the same time I can see that the three best pre-sets also produce the highest levels of cell death, but I think this is actually a good sign. If the electroporation has more of an impact on the cell membrane it makes sense that more protein can enter the cell, but also that more of them will die. And the level of cell death is still low enough that in the end there are more hyper-positive cells.</p>
<p>Hopefully this rethink will make my future editing attempts even more effective!</p>
<p>You can read the full details of what I did in <a href="https://zenodo.org/record/3990047#.Xz1cM0HTVaQ">my Zenodo report</a>.</p>
<p>&nbsp;</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/electroporation-a-second-look/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>Structure and X-ray Fragment screening of SARS-Cov-2 helicase (Nsp13)</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/structure-and-x-ray-fragment-screening-of-sars-cov-2-helicase-nsp13/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/structure-and-x-ray-fragment-screening-of-sars-cov-2-helicase-nsp13/#respond</comments>
		
		<dc:creator><![CDATA[Joseph Newman]]></dc:creator>
		<pubDate>Mon, 17 Aug 2020 18:53:46 +0000</pubDate>
				<category><![CDATA[COVID-19]]></category>
		<category><![CDATA[Joseph Newman]]></category>
		<category><![CDATA[SGC Univ. of Oxford]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=5084</guid>

					<description><![CDATA[My name is Joseph Newman and I have been working in Lab of Opher Gileadi at SGC Oxford since 2013 mainly on targeting DNA repair factors as synthetic lethal targets for early stage development of new cancer therapeutics. Like many scientists my normal laboratory based research activities were disrupted by the lockdown, and when Opher <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/structure-and-x-ray-fragment-screening-of-sars-cov-2-helicase-nsp13/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p>My name is Joseph Newman and I have been working in Lab of Opher Gileadi at SGC Oxford since 2013 mainly on targeting DNA repair factors as synthetic lethal targets for early stage development of new cancer therapeutics. Like many scientists my normal laboratory based research activities were disrupted by the lockdown, and when Opher suggested to work on targeting the SARS-Cov-2 helicase (a family that I have significant experience and interest in) I was keen to give it a try. The SARA-Cov-2 helicase is well conserved across and appears to be indispensable across all coronavirus species, and for this reason has been highlighted as one of the attractive targets for the development of anti-viral drugs(1, 2). The helicase is a Upf1 like SF1b helicase that unwinds both DNA and RNA in a 5’ to 3’ direction with a relaxed nucleotide specificity <em>in vitro</em>. Whilst its precise role in viral replication in not currently well-defined recent studies show it acts in concert with the replication-transcription complex (Nsp7/Nsp8<sub>2</sub>/Nsp12), possibly being involved in either disrupting downstream RNA secondary structures or template switching(3).</p>
<p>Previously crystal structures of Nsp13 have been solved for MERS-CoV(4) and SARS-Cov-1(5) to 3.0 Å and 2.8 Å respectively which indicated that the protein may be crystallizable, although the resolution of these structures is probably too low to reliably detect fragment binders in a X-ray Fragment screen. We began lab work on this project in mid-April and made quite rapid progress obtaining a new crystal form that is reproducible and diffracts routinely to around 2.0 Å. We have deposited the Nnsp13 structure in the pdb (6ZSL) and recently ran a full fragment screen with the help of XChem team at Diamond Light Source. The screen went about as well as could be expected producing around 50-60 hits from a screen of around 600 compounds with several compounds binding in both the nucleotide and RNA binding sites. I am still working on finalising the hits and the results should be available soon on the Diamond fragalysis user interface, but I thought I would show you an overview of what we found in the image below<a href="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/nsp13-DNA-atp.tif"><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/nsp13-DNA-atp.tif" alt="" class="alignnone size-medium wp-image-5085" width="1" height="1" /></a></p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/nsp13-DNA-atp.tif" alt="" class="alignnone size-medium wp-image-5085" width="1" height="1" /><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/nsp13-DNA-atp-300x120.jpg" alt="" class="alignnone wp-image-5088" width="1225" height="490" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/nsp13-DNA-atp-300x120.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/nsp13-DNA-atp-1024x411.jpg 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/nsp13-DNA-atp-768x308.jpg 768w" sizes="(max-width: 1225px) 100vw, 1225px" /></p>
<p><em>The structure of Nsp13 is shown on the left panel along with the expected positions of the Nucleotide and RNA which have been positioned into the enzyme based on structures of homologues. The right hand panel shows a subset of the fragment hits which can be seen to cluster around areas of potential interest for inhibition.</em></p>
<p>This project was a team effort and I want to thank Yuliana Yosaatmadja for help with the protein work and the whole XChem team at Diamond in particular Alice Douangamath for help in setting crystallization experiments and mounting all the crystals. If you are reading this and are interested in collaborating towards the development of Nsp13 inhibitors, then please get in touch.</p>
<p>For a detailed protocol of Nsp13 purification and crystallization please refer to my<a href="https://zenodo.org/record/3988536"> Zenodo post</a>.</p>
<p><strong>References</strong></p>
<ol>
<li>Habtemariam S, Nabavi SF, Banach M, Berindan-Neagoe I, Sarkar K, Sil PC, et al. Should We Try SARS-CoV-2 Helicase Inhibitors for COVID-19 Therapy? Arch Med Res. 2020.</li>
<li>Keum YS, Jeong YJ. Development of chemical inhibitors of the SARS coronavirus: viral helicase as a potential target. Biochem Pharmacol. 2012;84(10):1351-8.</li>
<li>Chen J, Malone B, Llewellyn E, Grasso M, Shelton PMM, Olinares PDB, et al. Structural basis for helicase-polymerase coupling in the SARS-CoV-2 replication-transcription complex. bioRxiv. 2020.</li>
<li>Hao W, Wojdyla JA, Zhao R, Han R, Das R, Zlatev I, et al. Crystal structure of Middle East respiratory syndrome coronavirus helicase. PLoS Pathog. 2017;13(6):e1006474.</li>
<li>Jia Z, Yan L, Ren Z, Wu L, Wang J, Guo J, et al. Delicate structural coordination of the Severe Acute Respiratory Syndrome coronavirus Nsp13 upon ATP hydrolysis. Nucleic Acids Res. 2019;47(12):6538-50.</li>
</ol>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/structure-and-x-ray-fragment-screening-of-sars-cov-2-helicase-nsp13/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>Back to SARS-CoV-2 Main Protease: Calculating the change in Gibbs free energy (ddGbind) of two peptidomimetic aldehyde Inhibitors binding associated with genetic variations of SARS-CoV-2 main protease &#8211; post17</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/back-to-sars-cov-2-main-protease-calculating-the-change-in-gibbs-free-energy-ddgbind-of-two-peptidomimetic-aldehyde-inhibitors-binding-associated-with-genetic-variations-of-sars-cov-2-main-protease/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/back-to-sars-cov-2-main-protease-calculating-the-change-in-gibbs-free-energy-ddgbind-of-two-peptidomimetic-aldehyde-inhibitors-binding-associated-with-genetic-variations-of-sars-cov-2-main-protease/#respond</comments>
		
		<dc:creator><![CDATA[Setayesh Yazdani]]></dc:creator>
		<pubDate>Tue, 04 Aug 2020 15:54:13 +0000</pubDate>
				<category><![CDATA[COVID-19]]></category>
		<category><![CDATA[Setayesh Yazdani]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=5058</guid>

					<description><![CDATA[Since we posted our analysis of genetic variation of SARS-CoV-2 main protease (MPro) with inhibitor N3 (PDB: 7bqy), there have been other inhibitors co-crystallized with the MPro (PDB: 6lze and PDB: 6m0k). MPro acts like a scissor by cutting SARS-CoV-2 polyproteins into functional pieces, making it a critical target for antiviral therapies against SARS-CoV-2. In <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/back-to-sars-cov-2-main-protease-calculating-the-change-in-gibbs-free-energy-ddgbind-of-two-peptidomimetic-aldehyde-inhibitors-binding-associated-with-genetic-variations-of-sars-cov-2-main-protease/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p>Since we posted our analysis of genetic variation of SARS-CoV-2 main protease (MPro) with inhibitor N3 (PDB: 7bqy), there have been other inhibitors co-crystallized with the MPro (PDB: 6lze and PDB: 6m0k).</p>
<p>MPro acts like a scissor by cutting SARS-CoV-2 polyproteins into functional pieces, making it a critical target for antiviral therapies against SARS-CoV-2.</p>
<p>In June 2020, Dai and colleagues published a paper where they described their structure-based approach to designing two antiviral candidates targeting the MPro of SARS-CoV-2.</p>
<p>They reported two lead compounds, 11a and 11b, and both showed inhibitory activity against the MPro with IC50 values of 53 ± 5 nM and 40 ± 2 nM respectively. Both compounds inhibit MPro through a similar mechanism of action by making covalent linkages via an aldehyde group with cysteine 145 (C145). They also included in vivo pharmacokinetic data for both compounds and concluded that both could be very promising candidates for further development as anti-SARS-CoV-2 candidates.</p>
<p>11a and 11b are pretty much the same molecules with a difference in their p2 region as shown in figure 1 where 11a has a cyclohexyl and 11b has a 3-fluorophenyl at that position.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/design_strategy_11a_11b-1024x411.png" alt="" width="780" height="313" class="alignnone wp-image-5059 size-large" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/design_strategy_11a_11b-1024x411.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/design_strategy_11a_11b-300x121.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/design_strategy_11a_11b-768x309.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/design_strategy_11a_11b.png 1359w" sizes="(max-width: 780px) 100vw, 780px" /></p>
<p><strong>Figure 1. The design strategy for developing inhibitors against SARS-CoV-2 MPro. (Dai et al., 2020)</strong></p>
<p>In this post, I will describe how the non-synonymous mutations identified in more than 15000 SARS-CoV-2 samples at the main protease catalytic site are predicted to affect the binding of these two inhibitors to the MPro.</p>
<p>If you would like to read the details of the previous MPro analysis, please visit this <a href="https://doi.org/10.5281/zenodo.3928729"><u>Zenodo</u></a> report.</p>
<p>According to <a href="https://thesgc.github.io/static-openlabnotebooks/genetic-diversity-at-the-catalytic-site-of-the-main-sars-cov-2-protease/"><u>Nicola De Maio</u></a>, our collaborator on this project from European Bioinformatics Institute, there were 4 non-synonymous variants at the MPro catalytic site across the SARS-CoV-2 samples from COVID-19 patients.  (Table 1)</p>
<p>Next, to predict the effect of those identified mutations on inhibitor (11a and 11b) binding, we use ICM software (Molsoft, SD). In ICM, by mutating the residue, we can see the change in Gibbs free energy of inhibitor binding (ddGbind) which would indicate if the mutation penalizes the binding of the inhibitor significantly or not. Table 1 shows two columns of ddGbind values for each of the inhibitors.</p>
<table width="349" style="width: 68.6869%;">
<tbody>
<tr>
<td width="64" style="width: 13.5227%;"><strong>Residue Number</strong></td>
<td width="64" style="width: 13.5228%;"><strong>Wild Type Residue</strong></td>
<td width="67" style="width: 14.7727%;"><strong>Mutated Residue</strong></td>
<td width="79" style="width: 13.9773%;"><strong>ddGbind (11a)</strong></td>
<td width="76" style="width: 11.8182%;"><strong>ddGbind (11b)</strong></td>
</tr>
<tr>
<td width="64" style="width: 13.5227%;">49</td>
<td width="64" style="width: 13.5228%;">Met</td>
<td width="67" style="width: 14.7727%;">Ile</td>
<td width="79" style="width: 13.9773%;">4.8</td>
<td width="76" style="width: 11.8182%;">0.4</td>
</tr>
<tr>
<td width="64" style="width: 13.5227%;">52</td>
<td width="64" style="width: 13.5228%;">Pro</td>
<td width="67" style="width: 14.7727%;">Ser</td>
<td width="79" style="width: 13.9773%;">-0.1</td>
<td width="76" style="width: 11.8182%;">-0.1</td>
</tr>
<tr>
<td width="64" style="width: 13.5227%;">142</td>
<td width="64" style="width: 13.5228%;">Asn</td>
<td width="67" style="width: 14.7727%;">Ser</td>
<td width="79" style="width: 13.9773%;">1.3</td>
<td width="76" style="width: 11.8182%;">0.7</td>
</tr>
<tr>
<td width="64" style="width: 13.5227%;">168</td>
<td width="64" style="width: 13.5228%;">Pro</td>
<td width="67" style="width: 14.7727%;">Ser</td>
<td width="79" style="width: 13.9773%;">0.7</td>
<td width="76" style="width: 11.8182%;">0.7</td>
</tr>
</tbody>
</table>
<p><strong>Table 1.</strong> <strong>The predicted change in Gibbs free energy associated with the non-synonymous variants found at MPro catalytic site across more than 15000 SARS-CoV-2 samples.</strong> ddGbind values are in Kcal/mol.</p>
<p>While M49I is predicted to penalize 11a binding to MPro significantly (ddGbind &gt; 2.0 Kcal/mol), that is not the case with 11b. M49 is juxtaposed to the functional group that is diverging between the two inhibitors (cyclohexyl in 11a, 3-fluorophenyl in 11b), which could be a reason for the observed difference (Fig. 2). However, we also note a small but non-negligible variation backbone conformation of the loop where M49 is located (Fig. 2). Importantly, sidechains are flexible, but the protein backbone is not when calculating ddGbind with ICM. It is, therefore, possible that an Ile is acceptable at position 49 in the context of the loop conformation observed in one structure and not the other. Molecular dynamics simulations may help better analyze the effect of the M149I mutation on the binding 11a. At the moment we cannot reliably predict this effect.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/M49_11a_11b.png" alt="" width="659" height="439" class="alignnone wp-image-5060 size-full" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/M49_11a_11b.png 659w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/M49_11a_11b-300x200.png 300w" sizes="(max-width: 659px) 100vw, 659px" /></p>
<p><strong>Figure 2. M49 in proximity to 11a (PDB: 6lze &#8211; yellow) and 11b (PDB: 6m0k &#8211; cyan) in the SARS-CoV-2 MPro crystal structure. </strong></p>
<p>Lastly, we used two-dimensional diagrams as shown in figure 3 to illustrate the chemical interactions between 11a and 11b and how they are different from one another.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/2D_11a_11b-1024x559.png" alt="" width="780" height="426" class="alignnone wp-image-5061 size-large" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/2D_11a_11b-1024x559.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/2D_11a_11b-300x164.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/2D_11a_11b-768x419.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/2D_11a_11b.png 1315w" sizes="(max-width: 780px) 100vw, 780px" /></p>
<p><strong>Figure 3. Two-dimensional diagrams representing the chemical interactions between the main protease and the two inhibitors, 11a and 11b. </strong>These diagrams were created using Pose.View in Proteins.Plus.</p>
<p>From the ddGbind values in table 1, we can conclude that the variants found in SARS-CoV-2 samples are both rare and only expected to minimally affect the binding of 11b to MPro.</p>
<p>We created an energy matrix where we consider the effect of every possible mutation at the MPro catalytic site on the binding of 11a and 11b with MPro. Table 2 includes a list of 20 residues lining the MPro catalytic pocket. No mutations at Cys145 were considered since it makes the covalent linkage with the inhibitors 11a and 11b which is essential for their inhibitory mechanisms against MPro.</p>
<p>It is likely that the residues of the rows with green color are not making optimal interactions with the inhibitors. This is reflected in the very slight and non-significant lowering in ddGbind values of these residues (mutations are stabilizing) and hence the green color. On the other hand, the sidechains whose mutations are shown in orange are likely the positions where the MPro would better interact with its inhibitors. As a result, the mutations of those sidechains are predicted to penalize inhibitor binding significantly and hence the orange color as shown in Table 2 and Table 3.<img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/11a_energy_matrix.png" alt="" width="1890" height="712" class="alignnone wp-image-5065 size-full" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/11a_energy_matrix.png 1890w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/11a_energy_matrix-300x113.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/11a_energy_matrix-1024x386.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/11a_energy_matrix-768x289.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/11a_energy_matrix-1536x579.png 1536w" sizes="(max-width: 1890px) 100vw, 1890px" /><strong>Table 2.</strong> <strong>The energy matrix showing the effect of all possible mutations at the SARS-COV-2 MPro catalytic site and the predicted effect on 11a binding (PDB: 6lze).</strong> The energy unit is kcal/mol. WT = wild type residue.</p>
<p><img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAJ8AAACICAYAAAD5yDnDAAARnElEQVR4Ae2dy6slVxWH79+gAyMaQ2wEZz4GikHNQE2LOBIJokZRVGwh+EJRMvGREFtEwSeioojSiEYyCIiIgrZk4kBBUAg0iDh34sCBcOS7yXf6d1d21Tm3+95bdar2hrp77/Xea/3OqqrTh+Ro00fPwEQZOJrIb3fbM7Dp4OsgmCwDHXyTpb473ht8ly5d2hwdHR1fV69eHc3ctWvXNleuXBmVOU9m+ifu69ev35a7tHdbhvZQJsdrGXud9PLly5sEHPsbN24M5ugii9UK4qz9n7W9VszSOvjMxGZz3DXoHkPDbshsh6nFyq4Jj6EMeoC5NVp6dFR9ZnfFXtLlYdu4sJf6+kzdVjwZa+Xrkzk/oNIzd3me/PAqa2zGlXPqEg+Dc6EjbyiPaWdO652dz8S3guawmQiSyEgdkmNRSDgyzMgo37KNDro50MkEk3Ro2lUWGXVZCz78GUvGZUzoY7OOGisy2kxZz4MP/cjHnzrE6zmYsc+oflJXe56VGXv41O5QXNqZ23xb4DPZHorDkxSSaPGlKSMYUgYe8tjjgqecesrAc1hkkm8x4aXttJPxpkzSWXOGHCkLXb+sM27tCIqMlTzAzwv9mh9tpP8q45nquYklfaaNOa53gq8eMA9RE2WSsljS1DNxKSMvZ+WSVpMrCGqMaTvtZLwpQ4yCwg6TflMWevpF15H2oRGv/JoHdSq92kCuynimeu6aH33Mdd4JPg+fReGQdAeSQmEYJII9I4tFotRFx+SmzLFS+YMPrhzYwZ4Df/hNu/Cgq2uhoOubtf7RTZvazhnZ1NVvFp91yqiPrD7MgzxmfGOfAb9lAxl186zpH33OrK1jgzP/sxf4OANJ8aqJkE5iGBbWs8tnJmEtGWVzTj2TSiGkGwc6Fg5e+kden/AcKQNAtMmsfMqmfYENP+PBDgO+9lJWGjN6DMEEDR/aOGbGn9Q1PmbtILpY8EUeFrWkgAmQBOWiDjrDw9xsBTMM7qJCqp3vovyu3U8H39oRMOH5O/gmTP7aXXfwrR0BE56/g2/C5M/RdX2DPs8YTw0+vg7gqqMVdH49kF8JQG8NZPxKpcVv0YZstWRbtLN8u/WrDmzmVybkxj1n9CupjIec5lt38lrrWgfy0LLb0pVmvO6ZW7Tk77tu4aHqtlFQpWJfDy0rnfndFTQHegLrdgGjTebbtXWW4BNgxJUfJOjm4qz8ZR2wb24zN7vWLaDlGXbpj/ETD0Nye4GPg1JkLgJmz8BB0u1uKdNyrB11/cRmMkhC+lUGe+ohzzoHMWUCLbYfCHUtlnxspH/9aFtf6KsrjzntsDcO6OZF+YxPWurjK8/e8gefS1ntMOPPcyKDvntmc1nPm4BhDZ9Y0dGPdtKfMvCMNW2lbK5PVi45z6xNoiwccSBGPYhJZkZvaGSQJhDZTAZ+2DNSBtse0KRWP+gOJRhZeMgwsKGf9A+POJXxzOzVPWY+8we+cUnHXuZIesYnrcahj5p/5fGHbXMuHTuVJo+5+smY8/z4xb51ZJ15Mh/QXPsBZ0ZvLA5i2Qm+DBYFHHFlAaGnM5watAUgeA/K2pF28vBZoLSddGykLW0aY+UTF/JcFjfPl/5TN8+gvufTZ9WFrr8qm/lRf584lGX2jMRDDh2tOOAZNzMyjCoLz5E5h5Z5z1iTjpxnq/razfmmt6TGOh1B9tAJGujpjEMhlyMPmodMOymTh0rbScd+2tKf8hk7axLDSJ8pk/7TdqXrJ+cqkzHoV3kL5J55nzhS3jqgR04cNQ7o0KzHkB/iRc5h/O4z72kj6ch6tqqvnZx3gg8jeTiK7UFY+6nDqUmGBg9dRyYFHgdgYMtDp0weKg+Cj9TFVmugbyLgZ8JYe6akZyzQtQ3ds7V8QSN288Ie+54/Y5Zn3rSXcWQe4BuHssz40h/yXNJrrGkvZSudGByZc2hZj4wVX8Zh3ZGv+trNuV25lHgmsSSAC8c6Y530PLSByGc24RyEg8vTXSYjD5sHSbvEgVxrwMN+DmShMauXiUTWmIxPfc4mr9pFJu3gG30H8euP+F3Lr/qZB3gtf/jgcmATPwzWxopM5gyesaWf6iNzrk3rl2eFpy9mY6j6x4GVPyerU5h9e7oMtEBVLdTCVf4Ue4AiIC/SfwffGWY7O8mQWTqoHWRI5qLp+8R9HjF18J1HVrvNvTJwlPfrvr75a+2eiwvIxX8ef3jTr56DKTBwNIXT7rODHQx08PXOP9mdr4Ovg6+Db4pb8Qfe/KoNl76/+N7LJ/bSx2ZeTNIGsux9YcFmSx+6MlW/Jb9E2uo7HwD463c/cXyxrkUGGC0AoYP8jz55/wnw/ebRD27uvuM5WzvaT7vqSkMePfdrmVcPPrvdEMgAgl0KoFVgVPBVO+yrnj61xZ7L/Vrm1YOPQtN5slsNFR8g1U7WAl+CrQWsSqs2hvwvjd7Bxyv/Mz+a2FVcwZe3yAqc2ukq0PBRadXGrjiWwl89+AQCoGHdKix0n+8qvwKn2qlgRB970LVlDO7XMq8afPXBv95SAUEFUwVGBR/7vIW3bNI5oWurv3Cs8PuuN77iJSdeBmpHEhytWQABIi/AjCx2pQFGaAJdGUCtDH5bPpZOW3XnW3px536+Dr4Vdvy5gLL/pCpum94G+3zzUeJcc/HfPz226VfPwRQYOJrCaffZwQ4GOvh655/sztfB18HXwXcet+IXv/CODVfL9t8f//b2ezYeqlsyLdqPH/n4CZu/+97DJ/YtnUr70Nuf/heTpGPXh/s3veblzXjwpczQudLm3NeL7XwUkyIPFYkCI0OBHnnwgWNZ1shT5LHCpW5LfgyQ6lbAs+cDgV9lagzpi7MRd5U5pP1iwUehKCZzLYhdT3qVo/gJBuWcBRfgRVZ6zvoY4if4qh0/OGlPn9LYD9lWZu7zIsFHV6CAFVQWo0VPMCgHcFvghe+tE1vKt2ZAgu3apdJfBVsLWJXWOkPL/5xpiwNfFmmoQC16gsGCAbwWHb7dEX/Kt2b42ACsyU+7HXwLeeujw1DYemXhAV8Wv4JRYEFPPdcCHNAM3fr0McRP/9VOBSN+8Zld2BiM6RDnxXW+LEIFVfIoJEWGdtoXDnQpPrqASzvar0CRnnOCT6AK9pZNdNHRb3/hmHm3rOCjqD57UUSKyZUdJQHSWidQ4e8DtLSDL/0yGw+z9LxF54cEkCsz1FHT19zXi+58c0/+2uPr4Jt5914yQI//E2lLPmA/23x/xNB/z9d4M/a5qs/P/tbgTHPyv3/9bdOvnoMpMHA0hdPus4MdDHTw9c4/2Z2vg6+Dr4PvPG7FH37PO7Zfyj760CebSf7pt76ylbnv3tc2ZVqxYZtLHvZzL33f2VifevLXW5upe+muO7dx/v6XP2nKpPwhrBfb+SiiYGDNW1qrsEkHfICR4lLsXQVUV/tVHv9DoE9Z9PGNz1aMCex9Y0v7c10vFnw14a3CArTsduwrYJNfbQqKMZAhA0ixXfXd4wPQtWJEBnp2uyE57R3KvArwDXWLBBsFQ66CDRrgGepgAIFrV8EBqJ0yZTOGIVBVOjEmGNPeIa1XAT6K1ypWFp6ijYHPjliLC6C4Kr3uBV+NI3UryLRR6R18B/KWSKGGbnnQs9MlGH2OS75gcKYbeusd6ozQAVgrBoAoeHOuAK0fngpG4zm0edGdD+AMgYJCCTAf8gUqxafAY8VUVxnAox1pY8+CyuQ8BCrs2Hn3iS1tznm9WPDRabKbsAaIgkag2JngW+B9CiZQlcXOafTVyznBh73sunkWY0/dQ1wvFnyHWIy1xdzBdyDPrksEZv9JVf9J1bMeT/IWf67rvf7HqF2oZ+AcMtD/Z8/nkNRucr8MdPDtl6cudQ4Z6OA7h6R2k/tlYBXgu3Tp0vah+vr1683MXLlyZStz9erVpkyLePny5U3KYyf3LZ0Wjbh8uCfe1rhx48ZWBtlDH4d/gh0VuHbt2oaLwdwqLIVPOoWl0MgDrrEhIJCpdtTDhjFIq3P6qYBWNu0AcIB+yGPx4MviAJQEmbzardgLFjvSWKHVT3Bo2xkZQI29XQNg1e6ZIEd/6Cy7bM+JvyrwDXWLBBvFaRUfMAIeQVmLCC+7V+W7RwbZscEHpIK0BbZddsZ8zIE3noU5RHjKGAAOReHK7lE7R5o9DfjSpja03eqqyjgLPnRaA/vEU0cHX83IAe0B41DBKXaCKsG4722Xjni7t11ttNIqwOW1wCjvUObFdb5W4gFevY2lHEXPriVQx8CgPnbVrQBRZgyUytQYpOeMH+QYQx0y5ee+Xjz46GKAKS8AY0ezQN4OkbPA8sZmAJHArl10TDd5GR9rAU1cdmVjTn7aOLT14sF3aAVZU7wdfGuq9szO2n9SVW7J9fbX9ycfWc40H//891ObfvUcTIGBoymcdp8d7GCgg693/snufB18HXwdfOd1K/7jn397/B3fA+9/ZzPJ3/zB17bfAd77htc1ZVqxYS9tfvZznzqxb+nsQ8OOD/VpP3Uf+9XT/86M3F13v2jvmNPGHNaL73yCZKiQFBCAUgzABxgp7j5FVVeA14LiEzBV+tg+PwDYJ5YqT2zSb8VHtTfVftHgo0AUB0C1wAc9i51yAir5tUh2uzEA2MmwXfV37fEtyJRlnx8M9mMxqjfHedHgo0iAKEGVRaj0ViGh0YGGOhg+EgxpP9cA1E6Z9KG14K/8GiNy+/ivduawXwz4ABLF5bITCZgKMhNf6bWwyAm+VueEr09tDs2CD3vKCDBsVADRzYhPWecaYwffDN8YKZ7AcK63J4qbtASjwEi+AHAG3FyCXXrO8PHfAlLK5XrMHuBLoFYwpp25rxfT+cYSnaBKOQHGDB2gIVsLnDqu1XUPwLQjbQxEytQZHa5Kzz2+iBHarfhIW1OuVwc+QSNQ7EwUdFfRs1ACVZod0P2tzACKOPIyJrqd3ZNZmbHOfCsxXKTOKsB3kQntvvb/p8MOvhk+r64FwMe/51vLYfs59+9KF5Gr/nu+8ozls1SfTz57nks+nvzHE5t+9RxMgYGjKZx2nx3sYKCDr3f+ye58HXwdfB1853UrfvXrX7l5wV3PP5Hgj3zmfdsvad/27rec4BnHF77x6a0MNqTvmrGXNvGV+136u/icpZ5nl85c+YvufBQJEGWxfvGH7x+DyoLA+87Pv/wscPF2hyxygA87VVcbdVZ3SP5WAUkMADnPU30f0n7R4KMQACCLVQvPniuLRpGz21l0ZQBX2pTurA+AUm0rg03sDPGVyxmf9TzJP7T1KsGXBa/AooCVRmdMMCJjVxu6pQKUMYAKFGIBhK3uqwwzfoirg2+GD+l2EgqZgKjFothnCb4KSgFDHFzuh2bBR/wpoz5zgr+eJ3UObb3KzpfgrGCkgAAhQcU+dQDEWFfTJjoJ9ASHH5Yhfsoik2B0nTKHuF4d+OgiFM9iAaJ6y/OWyowcQAQs0tVtzVUGX9pRHjAlmKXvM2NrDPj72JiLzKLBZ4dwBkAknsJLy84DTSBmtzkNUASqBb4doGkj5wo+/OUZUnbu60WDb+7JX3t8HXwzfHlaCyj7T6r6T6q2jyA+ilzY/LO/fHXTr56DKTBwNIXT7rODHQx08PXOP9mdr4Ovg6+Dby634vvuv+f4ATzj+eiXHtg+lL/snpc2i/X5Hz64lXnenc9tyqRN1+/62Fs3aRNfuVduiXPvfNH5KLpAy2Lz9vf1Jx46BpQyyWcN4AAgawAMqFinbtVxn7r7yKt36HMHX4DPYgIA17UTsQdc8pkBXXY79tm94KXN1GWtDwBbbVfZJe07+PYAXwKiAgswVBpdMsEoYABgglI6M/QxgKbsUtarBR+AodgtQCQIaqerQAMIlTYGvhYosWF39Pa+FICNnWO14BtNygS3XZ8Ts8uOxbgEXgffjtsunYhOaEcaeuFAhg4IKAQS69QdAkzK0AG1MyS/FHoHX4DPWx9g4PKNlVladibkuS0DBmZlhp7rWqBJoGrnNPotm4dC6+AL8B1K0ZYSZ/9VS/9Vy7Zj27kvav4/BPXH34vgg6oAAAAASUVORK5CYII=" /><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/11b_energy_matrix.png" alt="" width="1902" height="729" class="alignnone wp-image-5066 size-full" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/11b_energy_matrix.png 1902w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/11b_energy_matrix-300x115.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/11b_energy_matrix-1024x392.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/11b_energy_matrix-768x294.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/11b_energy_matrix-1536x589.png 1536w" sizes="(max-width: 1902px) 100vw, 1902px" /><strong>Table 3.</strong> <strong>The energy matrix showing the effect of all possible mutations at the SARS-COV-2 MPro catalytic site and the predicted effect on 11b binding (PDB: 6m0k).</strong> The energy unit is kcal/mol. WT = wild type residue.</p>
<p><img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAJ8AAACICAYAAAD5yDnDAAARnElEQVR4Ae2dy6slVxWH79+gAyMaQ2wEZz4GikHNQE2LOBIJokZRVGwh+EJRMvGREFtEwSeioojSiEYyCIiIgrZk4kBBUAg0iDh34sCBcOS7yXf6d1d21Tm3+95bdar2hrp77/Xea/3OqqrTh+Ro00fPwEQZOJrIb3fbM7Dp4OsgmCwDHXyTpb473ht8ly5d2hwdHR1fV69eHc3ctWvXNleuXBmVOU9m+ifu69ev35a7tHdbhvZQJsdrGXud9PLly5sEHPsbN24M5ugii9UK4qz9n7W9VszSOvjMxGZz3DXoHkPDbshsh6nFyq4Jj6EMeoC5NVp6dFR9ZnfFXtLlYdu4sJf6+kzdVjwZa+Xrkzk/oNIzd3me/PAqa2zGlXPqEg+Dc6EjbyiPaWdO652dz8S3guawmQiSyEgdkmNRSDgyzMgo37KNDro50MkEk3Ro2lUWGXVZCz78GUvGZUzoY7OOGisy2kxZz4MP/cjHnzrE6zmYsc+oflJXe56VGXv41O5QXNqZ23xb4DPZHorDkxSSaPGlKSMYUgYe8tjjgqecesrAc1hkkm8x4aXttJPxpkzSWXOGHCkLXb+sM27tCIqMlTzAzwv9mh9tpP8q45nquYklfaaNOa53gq8eMA9RE2WSsljS1DNxKSMvZ+WSVpMrCGqMaTvtZLwpQ4yCwg6TflMWevpF15H2oRGv/JoHdSq92kCuynimeu6aH33Mdd4JPg+fReGQdAeSQmEYJII9I4tFotRFx+SmzLFS+YMPrhzYwZ4Df/hNu/Cgq2uhoOubtf7RTZvazhnZ1NVvFp91yqiPrD7MgzxmfGOfAb9lAxl186zpH33OrK1jgzP/sxf4OANJ8aqJkE5iGBbWs8tnJmEtGWVzTj2TSiGkGwc6Fg5e+kden/AcKQNAtMmsfMqmfYENP+PBDgO+9lJWGjN6DMEEDR/aOGbGn9Q1PmbtILpY8EUeFrWkgAmQBOWiDjrDw9xsBTMM7qJCqp3vovyu3U8H39oRMOH5O/gmTP7aXXfwrR0BE56/g2/C5M/RdX2DPs8YTw0+vg7gqqMVdH49kF8JQG8NZPxKpcVv0YZstWRbtLN8u/WrDmzmVybkxj1n9CupjIec5lt38lrrWgfy0LLb0pVmvO6ZW7Tk77tu4aHqtlFQpWJfDy0rnfndFTQHegLrdgGjTebbtXWW4BNgxJUfJOjm4qz8ZR2wb24zN7vWLaDlGXbpj/ETD0Nye4GPg1JkLgJmz8BB0u1uKdNyrB11/cRmMkhC+lUGe+ohzzoHMWUCLbYfCHUtlnxspH/9aFtf6KsrjzntsDcO6OZF+YxPWurjK8/e8gefS1ntMOPPcyKDvntmc1nPm4BhDZ9Y0dGPdtKfMvCMNW2lbK5PVi45z6xNoiwccSBGPYhJZkZvaGSQJhDZTAZ+2DNSBtse0KRWP+gOJRhZeMgwsKGf9A+POJXxzOzVPWY+8we+cUnHXuZIesYnrcahj5p/5fGHbXMuHTuVJo+5+smY8/z4xb51ZJ15Mh/QXPsBZ0ZvLA5i2Qm+DBYFHHFlAaGnM5watAUgeA/K2pF28vBZoLSddGykLW0aY+UTF/JcFjfPl/5TN8+gvufTZ9WFrr8qm/lRf584lGX2jMRDDh2tOOAZNzMyjCoLz5E5h5Z5z1iTjpxnq/razfmmt6TGOh1B9tAJGujpjEMhlyMPmodMOymTh0rbScd+2tKf8hk7axLDSJ8pk/7TdqXrJ+cqkzHoV3kL5J55nzhS3jqgR04cNQ7o0KzHkB/iRc5h/O4z72kj6ch6tqqvnZx3gg8jeTiK7UFY+6nDqUmGBg9dRyYFHgdgYMtDp0weKg+Cj9TFVmugbyLgZ8JYe6akZyzQtQ3ds7V8QSN288Ie+54/Y5Zn3rSXcWQe4BuHssz40h/yXNJrrGkvZSudGByZc2hZj4wVX8Zh3ZGv+trNuV25lHgmsSSAC8c6Y530PLSByGc24RyEg8vTXSYjD5sHSbvEgVxrwMN+DmShMauXiUTWmIxPfc4mr9pFJu3gG30H8euP+F3Lr/qZB3gtf/jgcmATPwzWxopM5gyesaWf6iNzrk3rl2eFpy9mY6j6x4GVPyerU5h9e7oMtEBVLdTCVf4Ue4AiIC/SfwffGWY7O8mQWTqoHWRI5qLp+8R9HjF18J1HVrvNvTJwlPfrvr75a+2eiwvIxX8ef3jTr56DKTBwNIXT7rODHQx08PXOP9mdr4Ovg6+Db4pb8Qfe/KoNl76/+N7LJ/bSx2ZeTNIGsux9YcFmSx+6MlW/Jb9E2uo7HwD463c/cXyxrkUGGC0AoYP8jz55/wnw/ebRD27uvuM5WzvaT7vqSkMePfdrmVcPPrvdEMgAgl0KoFVgVPBVO+yrnj61xZ7L/Vrm1YOPQtN5slsNFR8g1U7WAl+CrQWsSqs2hvwvjd7Bxyv/Mz+a2FVcwZe3yAqc2ukq0PBRadXGrjiWwl89+AQCoGHdKix0n+8qvwKn2qlgRB970LVlDO7XMq8afPXBv95SAUEFUwVGBR/7vIW3bNI5oWurv3Cs8PuuN77iJSdeBmpHEhytWQABIi/AjCx2pQFGaAJdGUCtDH5bPpZOW3XnW3px536+Dr4Vdvy5gLL/pCpum94G+3zzUeJcc/HfPz226VfPwRQYOJrCaffZwQ4GOvh655/sztfB18HXwXcet+IXv/CODVfL9t8f//b2ezYeqlsyLdqPH/n4CZu/+97DJ/YtnUr70Nuf/heTpGPXh/s3veblzXjwpczQudLm3NeL7XwUkyIPFYkCI0OBHnnwgWNZ1shT5LHCpW5LfgyQ6lbAs+cDgV9lagzpi7MRd5U5pP1iwUehKCZzLYhdT3qVo/gJBuWcBRfgRVZ6zvoY4if4qh0/OGlPn9LYD9lWZu7zIsFHV6CAFVQWo0VPMCgHcFvghe+tE1vKt2ZAgu3apdJfBVsLWJXWOkPL/5xpiwNfFmmoQC16gsGCAbwWHb7dEX/Kt2b42ACsyU+7HXwLeeujw1DYemXhAV8Wv4JRYEFPPdcCHNAM3fr0McRP/9VOBSN+8Zld2BiM6RDnxXW+LEIFVfIoJEWGdtoXDnQpPrqASzvar0CRnnOCT6AK9pZNdNHRb3/hmHm3rOCjqD57UUSKyZUdJQHSWidQ4e8DtLSDL/0yGw+z9LxF54cEkCsz1FHT19zXi+58c0/+2uPr4Jt5914yQI//E2lLPmA/23x/xNB/z9d4M/a5qs/P/tbgTHPyv3/9bdOvnoMpMHA0hdPus4MdDHTw9c4/2Z2vg6+Dr4PvPG7FH37PO7Zfyj760CebSf7pt76ylbnv3tc2ZVqxYZtLHvZzL33f2VifevLXW5upe+muO7dx/v6XP2nKpPwhrBfb+SiiYGDNW1qrsEkHfICR4lLsXQVUV/tVHv9DoE9Z9PGNz1aMCex9Y0v7c10vFnw14a3CArTsduwrYJNfbQqKMZAhA0ixXfXd4wPQtWJEBnp2uyE57R3KvArwDXWLBBsFQ66CDRrgGepgAIFrV8EBqJ0yZTOGIVBVOjEmGNPeIa1XAT6K1ypWFp6ijYHPjliLC6C4Kr3uBV+NI3UryLRR6R18B/KWSKGGbnnQs9MlGH2OS75gcKYbeusd6ozQAVgrBoAoeHOuAK0fngpG4zm0edGdD+AMgYJCCTAf8gUqxafAY8VUVxnAox1pY8+CyuQ8BCrs2Hn3iS1tznm9WPDRabKbsAaIgkag2JngW+B9CiZQlcXOafTVyznBh73sunkWY0/dQ1wvFnyHWIy1xdzBdyDPrksEZv9JVf9J1bMeT/IWf67rvf7HqF2oZ+AcMtD/Z8/nkNRucr8MdPDtl6cudQ4Z6OA7h6R2k/tlYBXgu3Tp0vah+vr1683MXLlyZStz9erVpkyLePny5U3KYyf3LZ0Wjbh8uCfe1rhx48ZWBtlDH4d/gh0VuHbt2oaLwdwqLIVPOoWl0MgDrrEhIJCpdtTDhjFIq3P6qYBWNu0AcIB+yGPx4MviAJQEmbzardgLFjvSWKHVT3Bo2xkZQI29XQNg1e6ZIEd/6Cy7bM+JvyrwDXWLBBvFaRUfMAIeQVmLCC+7V+W7RwbZscEHpIK0BbZddsZ8zIE3noU5RHjKGAAOReHK7lE7R5o9DfjSpja03eqqyjgLPnRaA/vEU0cHX83IAe0B41DBKXaCKsG4722Xjni7t11ttNIqwOW1wCjvUObFdb5W4gFevY2lHEXPriVQx8CgPnbVrQBRZgyUytQYpOeMH+QYQx0y5ee+Xjz46GKAKS8AY0ezQN4OkbPA8sZmAJHArl10TDd5GR9rAU1cdmVjTn7aOLT14sF3aAVZU7wdfGuq9szO2n9SVW7J9fbX9ycfWc40H//891ObfvUcTIGBoymcdp8d7GCgg693/snufB18HXwdfOd1K/7jn397/B3fA+9/ZzPJ3/zB17bfAd77htc1ZVqxYS9tfvZznzqxb+nsQ8OOD/VpP3Uf+9XT/86M3F13v2jvmNPGHNaL73yCZKiQFBCAUgzABxgp7j5FVVeA14LiEzBV+tg+PwDYJ5YqT2zSb8VHtTfVftHgo0AUB0C1wAc9i51yAir5tUh2uzEA2MmwXfV37fEtyJRlnx8M9mMxqjfHedHgo0iAKEGVRaj0ViGh0YGGOhg+EgxpP9cA1E6Z9KG14K/8GiNy+/ivduawXwz4ABLF5bITCZgKMhNf6bWwyAm+VueEr09tDs2CD3vKCDBsVADRzYhPWecaYwffDN8YKZ7AcK63J4qbtASjwEi+AHAG3FyCXXrO8PHfAlLK5XrMHuBLoFYwpp25rxfT+cYSnaBKOQHGDB2gIVsLnDqu1XUPwLQjbQxEytQZHa5Kzz2+iBHarfhIW1OuVwc+QSNQ7EwUdFfRs1ACVZod0P2tzACKOPIyJrqd3ZNZmbHOfCsxXKTOKsB3kQntvvb/p8MOvhk+r64FwMe/51vLYfs59+9KF5Gr/nu+8ozls1SfTz57nks+nvzHE5t+9RxMgYGjKZx2nx3sYKCDr3f+ye58HXwdfB1853UrfvXrX7l5wV3PP5Hgj3zmfdsvad/27rec4BnHF77x6a0MNqTvmrGXNvGV+136u/icpZ5nl85c+YvufBQJEGWxfvGH7x+DyoLA+87Pv/wscPF2hyxygA87VVcbdVZ3SP5WAUkMADnPU30f0n7R4KMQACCLVQvPniuLRpGz21l0ZQBX2pTurA+AUm0rg03sDPGVyxmf9TzJP7T1KsGXBa/AooCVRmdMMCJjVxu6pQKUMYAKFGIBhK3uqwwzfoirg2+GD+l2EgqZgKjFothnCb4KSgFDHFzuh2bBR/wpoz5zgr+eJ3UObb3KzpfgrGCkgAAhQcU+dQDEWFfTJjoJ9ASHH5Yhfsoik2B0nTKHuF4d+OgiFM9iAaJ6y/OWyowcQAQs0tVtzVUGX9pRHjAlmKXvM2NrDPj72JiLzKLBZ4dwBkAknsJLy84DTSBmtzkNUASqBb4doGkj5wo+/OUZUnbu60WDb+7JX3t8HXwzfHlaCyj7T6r6T6q2jyA+ilzY/LO/fHXTr56DKTBwNIXT7rODHQx08PXOP9mdr4Ovg6+Dby634vvuv+f4ATzj+eiXHtg+lL/snpc2i/X5Hz64lXnenc9tyqRN1+/62Fs3aRNfuVduiXPvfNH5KLpAy2Lz9vf1Jx46BpQyyWcN4AAgawAMqFinbtVxn7r7yKt36HMHX4DPYgIA17UTsQdc8pkBXXY79tm94KXN1GWtDwBbbVfZJe07+PYAXwKiAgswVBpdMsEoYABgglI6M/QxgKbsUtarBR+AodgtQCQIaqerQAMIlTYGvhYosWF39Pa+FICNnWO14BtNygS3XZ8Ts8uOxbgEXgffjtsunYhOaEcaeuFAhg4IKAQS69QdAkzK0AG1MyS/FHoHX4DPWx9g4PKNlVladibkuS0DBmZlhp7rWqBJoGrnNPotm4dC6+AL8B1K0ZYSZ/9VS/9Vy7Zj27kvav4/BPXH34vgg6oAAAAASUVORK5CYII=" /></p>
<p>Please contact me via the “Leave a comment” link at the top of this post. Stay Tuned for more updates on this project.</p>
<p><strong>References:</strong></p>
<p>Dai, W., Zhang, B., Jiang, X., Su, H., Li, J., Zhao, Y., Xie, X., Jin, Z., Peng, J., Liu, F., Li, C., Li, Y., Bai, F., Wang, H., Chen, X., Cen, X., Hu, S., Yang, X., Wang, J., … Liu, H. (2020). Structure-based design of antiviral drug candidates targeting the SARS-CoV-2 main protease. <em>Science</em>, <em>368</em>(6497), 1331-1335. <a href="https://doi.org/10.1126/science.abb4489">https://doi.org/10.1126/science.abb4489</a></p>
<p>&nbsp;</p>
<p>&nbsp;</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/back-to-sars-cov-2-main-protease-calculating-the-change-in-gibbs-free-energy-ddgbind-of-two-peptidomimetic-aldehyde-inhibitors-binding-associated-with-genetic-variations-of-sars-cov-2-main-protease/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>Generation of stable mammalian cell lines for the expression of proteins related to COVID-19 through random integration</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/generation-of-stable-mammalian-cell-lines-for-the-expression-of-proteins-related-to-covid-19-through-random-integration/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/generation-of-stable-mammalian-cell-lines-for-the-expression-of-proteins-related-to-covid-19-through-random-integration/#respond</comments>
		
		<dc:creator><![CDATA[Jong Fu]]></dc:creator>
		<pubDate>Fri, 24 Jul 2020 09:27:17 +0000</pubDate>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[COVID-19]]></category>
		<category><![CDATA[Jong Fu Wong]]></category>
		<category><![CDATA[SGC Univ. of Oxford]]></category>
		<category><![CDATA[Spike]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=5048</guid>

					<description><![CDATA[Background: Scientists centred around University of Oxford came together in the current COVID-19 pandemic to contribute to relevant studies collaboratively. Researchers with no prior experience in virology studies, myself included, had volunteered our time to contribute whichever way we can based on our field of expertise. The Biotech team of SGC Oxford, headed by Nicola <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/generation-of-stable-mammalian-cell-lines-for-the-expression-of-proteins-related-to-covid-19-through-random-integration/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p><strong>Background:</strong></p>
<p>Scientists centred around University of Oxford came together in the current COVID-19 pandemic to contribute to relevant studies collaboratively. Researchers with no prior experience in virology studies, myself included, had volunteered our time to contribute whichever way we can based on our field of expertise. The Biotech team of SGC Oxford, headed by Nicola Burgess-Brown had taken up the role of producing proteins used in serological assays, basic research to understand COVID-19 pathophysiology and hunt for treatment compounds. The initial production of these proteins was based on transient transfection of expression construct DNA into mammalian cells. This approach was laborious and consumed large amounts of plasmid DNA. Therefore, I volunteered to generate stable expression cell lines that will negate the constant need for transfection.</p>
<p><strong>Experimental design:</strong></p>
<p>I aimed to stably integrate the expression constructs into the genome of Expi293F human cells commonly used in large scale protein production. The expression constructs carried neomycin marker for selection in mammalian cells. Toxicity to the cells is not a concern in this case because the expressed proteins are secreted into the medium and do not build up in the cells. The expression constructs are randomly integrated into the genome mainly via non-homologous end-joining (NHEJ). An advantage of this approach is that molecular sub-cloning is not necessary.</p>
<p>In order to maximise the rate of correct integration of the expression constructs, restriction digestion was performed to cut once between the expression cassette and antibiotic resistance cassette (refer to cartoon in Figure 1). Without this step, the construct DNA will be broken at a random position during the integration step, potentially disrupting the expression cassette. I have also tested the effect of actively creating double strand breaks (DSBs) in the genome of the cells on the integration efficiency. More DSBs in the genome might increase the incidence of NHEJ and integration of our constructs. This was achieved by co-transfection of AAVS1 CRISPR/eSpCas9(1.1) construct (refer to cartoon in Figure 1). The eSpCas9(1.1)_No_FLAG_AAVS1_T2 construct was a gift from Yannick Doyon (Addgene plasmid # 79888 ; http://n2t.net/addgene:79888 ; RRID:Addgene_79888)</p>
<p><strong>Results:</strong></p>
<div id="attachment_5049" style="width: 1209px" class="wp-caption alignnone"><img aria-describedby="caption-attachment-5049" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/Figure-1.jpg" alt="" width="1199" height="661" class="size-full wp-image-5049" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/Figure-1.jpg 1199w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/Figure-1-300x165.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/Figure-1-1024x565.jpg 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/Figure-1-768x423.jpg 768w" sizes="(max-width: 1199px) 100vw, 1199px" /><p id="caption-attachment-5049" class="wp-caption-text">Cartoons of approaches to enhance desired stable integration of protein expression constructs.</p></div>
<div id="attachment_5050" style="width: 1108px" class="wp-caption alignnone"><img aria-describedby="caption-attachment-5050" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/Figure-2.jpg" alt="" width="1098" height="623" class="size-full wp-image-5050" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/Figure-2.jpg 1098w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/Figure-2-300x170.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/Figure-2-1024x581.jpg 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/Figure-2-768x436.jpg 768w" sizes="(max-width: 1098px) 100vw, 1098px" /><p id="caption-attachment-5050" class="wp-caption-text">Schematic of the workflow to generate stable cell lines</p></div>
<div id="attachment_5051" style="width: 557px" class="wp-caption alignnone"><img aria-describedby="caption-attachment-5051" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/Figure-3.jpg" alt="" width="547" height="579" class="size-full wp-image-5051" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/Figure-3.jpg 547w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/Figure-3-283x300.jpg 283w" sizes="(max-width: 547px) 100vw, 547px" /><p id="caption-attachment-5051" class="wp-caption-text">Comparison of the efficiency of correct stable integration under different conditions. Numbers of surviving cells were counted after G418 antibiotic selection and the expression of desired proteins was detected using Western Blot.</p></div>
<p><strong>Findings:</strong></p>
<ol>
<li>Linearisation of expression construct increased stable integration.</li>
<li>Induction of AAVS1 double strand break enhanced stable integration further.</li>
<li>The efficiency of stable integration is highly dependent on construct size. E.g. 12.3 kb construct is about 30 times less efficiently integrated than 8.2 kb construct.</li>
<li>More than 5-fold increase in purified proteins compared to same volume of transient transfected cells.</li>
</ol>
<p><strong> </strong><strong>In a nut shell:</strong></p>
<p>Stable cells that continuously produce and secrete high level of COVID-19-related proteins have been successfully generated. These cells are suitable for large scale culture and protein purification. These stable cells will help to streamline protein production for serological assays and other COVID-19-related studies.</p>
<p>For detailed protocol and information, please refer to my <a href="https://zenodo.org/record/3958921#.XxqpYFVKipo">Zenodo post</a>.</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/generation-of-stable-mammalian-cell-lines-for-the-expression-of-proteins-related-to-covid-19-through-random-integration/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>Predicting the effect of all possible mutations at nsp3-Mac1 ADPr-binding site on ADPr binding &#8211; Post 16</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/predicting-the-effect-of-all-possible-mutations-at-nsp3-mac1-adpr-binding-site-on-adpr-binding-post-16/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/predicting-the-effect-of-all-possible-mutations-at-nsp3-mac1-adpr-binding-site-on-adpr-binding-post-16/#respond</comments>
		
		<dc:creator><![CDATA[Setayesh Yazdani]]></dc:creator>
		<pubDate>Wed, 22 Jul 2020 16:05:33 +0000</pubDate>
				<category><![CDATA[COVID-19]]></category>
		<category><![CDATA[Setayesh Yazdani]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=5038</guid>

					<description><![CDATA[In my last post, I showed you the genetic variants at the ADP-ribose (ADPr) binding site of SARS-CoV-2 nsp3-Mac1 and the predicted effects of those mutations on ADPr binding with Mac1. Next, we wanted to assess the effect of all possible mutations at the sidechains lining the ADPr binding site of nsp3-Mac1 and how that <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/predicting-the-effect-of-all-possible-mutations-at-nsp3-mac1-adpr-binding-site-on-adpr-binding-post-16/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p>In my last <a href="https://thesgc.github.io/static-openlabnotebooks/genetic-variability-at-the-adp-ribose-binding-site-of-the-sars-cov-2-nsp3-mac1-and-predicted-effects-of-mutations-on-adp-ribose-binding-post-15/">post</a>, I showed you the genetic variants at the ADP-ribose (ADPr) binding site of SARS-CoV-2 nsp3-Mac1 and the predicted effects of those mutations on ADPr binding with Mac1.</p>
<p>Next, we wanted to assess the effect of all possible mutations at the sidechains lining the ADPr binding site of nsp3-Mac1 and how that would affect ADPr binding. ADPr is the endogenous ligand for nsp3-Mac1 that Michalska and their colleagues have crystalized with SARS-CoV-2 Mac1 (PDB: 6w02).</p>
<p>We created an energy matrix where we consider the effect of every possible mutation on the binding of ADPr with nsp3-Mac1. Table 1 includes a list of 29 residues lining the pockets.</p>
<p>It is likely that the residues of the rows with green color are not making optimal interactions with ADPr. This is reflected in the very slight and non-significant lowering in ddGbind values of these residues (mutations are stabilizing) and hence the green color. On the other hand, the sidechains whose mutations are shown in orange are likely the positions where macrodomain would better interact with ADPr. As a result, the mutations of those sidechains are predicted to penalize its binding significantly and hence the orange color.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/energy-matrix-mac1.png" alt="" width="1802" height="870" class="alignnone wp-image-5040 size-full" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/energy-matrix-mac1.png 1802w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/energy-matrix-mac1-300x145.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/energy-matrix-mac1-1024x494.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/energy-matrix-mac1-768x371.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/energy-matrix-mac1-1536x742.png 1536w" sizes="(max-width: 1802px) 100vw, 1802px" /></p>
<p><strong>Table 1.</strong> <strong>The energy matrix showing the effect of all possible mutations at SARS-COV-2 nsp3-Mac1 ADPr binding site and the predicted effect on ADPr binding.</strong> The energy unit is kcal/mol. WT = wild type residue.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/energy-matrix-color-coding.png" alt="" width="134" height="119" class="alignnone wp-image-5041" /></p>
<p>Most of the hydrogen bonds in ADPr binding pocket make use of main chain atoms and only a few hydrogen bonds involve protein side chains. Since main sidechain interactions can be achieved using several combinations of sidechains, there is less pressure on amino acid preservation. (Michalska et al., 2020)</p>
<p>In the absence of a catalytic residue at ADPr-binding site, Jankevicius et al. proposed substrate-assisted catalysis. Through this enzymatic mechanism, a water molecule is responsible for nucleophilic attack on the ribose anomeric carbon atom which is activated by the Pa group. (Jankevicius et al., 2013) Based on the current macrodomain/ADPr structure (PDB: 6w02), the candidate water (Wat) molecule binds to the amide group of A50, also the carbonyl of A38, and the oxygen atom of Pa and OH1’ group of the proximal ribose ring of ADPr which is shown in figure 1. (Michalska et al., 2020)</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/substrate-assiated-catalysis-mac1-300x249.png" alt="" width="300" height="249" class="alignnone size-medium wp-image-5042" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/substrate-assiated-catalysis-mac1-300x249.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/substrate-assiated-catalysis-mac1.png 389w" sizes="(max-width: 300px) 100vw, 300px" /></p>
<p><strong>Figure 1. ADPr binding site in nsp3-Mac1 (PDB:6w02). (</strong>Michalska et al., 2020)</p>
<p>Based on the diversity dendrograms in post <a href="https://thesgc.github.io/static-openlabnotebooks/mapping-the-genetic-variations-of-alpha-and-betacoronavirus-uniprot-entries-onto-sars-cov-2-nsp3-mac1-crystal-structure-post-14/">14</a>, it is evident that A50 is preserved among the entries of Alpha- and Betacoronavirus entries while A38 is not. On the other hand, both A38 and A50 are conserved in the SARS-CoV-2 samples from the samples of COVID-19 patients Nicola De Maio has looked at.</p>
<p>From table 1, we see that many cells of the rows belonging to A38 and A50 are highlighted in orange which further points at the importance of these two amino acids in substrate-assisted catalysis proposed by Jankevicius et al. (Jankevicius et al., 2013).</p>
<p><strong>Conclusion</strong></p>
<p>We see a lot of structural diversity at the ADP-ribose binding site of the first macrodomain of SARS-CoV-2 nsp3, both across diverse coronaviruses and across diverse SARS-CoV-2 samples. However, most of these variations are predicted to have little impact on ADPr binding. While this structural diversity is not favorable for the design of broad-spectrum ligands, competitive inhibitors mimicking interactions made by ADPr may have a broad spectrum of action.</p>
<p>This post marks the end of nsp3-Mac1 analysis. To see the full report on nsp3-Mac1, please refer to my <a href="https://doi.org/10.5281/zenodo.3956555">Zenodo</a> report.</p>
<p>Please contact me via the “Leave a comment” link at the top of this post. Stay Tuned for more updates on this project!</p>
<p><strong>References:</strong></p>
<p>Michalska, K., Kim, Y., Jedrzejczak, R., Maltseva, N. I., Stols, L., Endres, M., &amp; Joachimiak, A. (2020). Crystal structures of SARS-Cov-2 ADP-ribose phosphatase (ADRP): From the apo form to ligand complexes. <a href="https://doi.org/10.1101/2020.05.14.096081">https://doi.org/10.1101/2020.05.14.096081</a></p>
<p>Jankevicius, G., Hassler, M., Golia, B., Rybin, V., Zacharias, M., Timinszky, G., &amp; Ladurner, A. G. (2013). A family of macrodomain proteins reverses cellular mono-ADP-ribosylation. Nature Structural &amp; Molecular Biology, 20(4), 508-514. <a href="https://doi.org/10.1038/nsmb.2523">https://doi.org/10.1038/nsmb.2523</a></p>
<p>&nbsp;</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/predicting-the-effect-of-all-possible-mutations-at-nsp3-mac1-adpr-binding-site-on-adpr-binding-post-16/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>Genetic variability at the ADP-ribose Binding Site of the SARS-CoV-2 nsp3-Mac1 and predicted effects of mutations on ADP-ribose binding &#8211; Post 15</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/genetic-variability-at-the-adp-ribose-binding-site-of-the-sars-cov-2-nsp3-mac1-and-predicted-effects-of-mutations-on-adp-ribose-binding-post-15/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/genetic-variability-at-the-adp-ribose-binding-site-of-the-sars-cov-2-nsp3-mac1-and-predicted-effects-of-mutations-on-adp-ribose-binding-post-15/#respond</comments>
		
		<dc:creator><![CDATA[Setayesh Yazdani]]></dc:creator>
		<pubDate>Wed, 22 Jul 2020 16:02:54 +0000</pubDate>
				<category><![CDATA[COVID-19]]></category>
		<category><![CDATA[Setayesh Yazdani]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=5033</guid>

					<description><![CDATA[In two of my previous posts 13 and 14, I showed how we found the residues lining the ADP ribose (ADPr) binding site of SARS-CoV-2 nsp3-Mac1 and we looked at the variability of this site across Alpha- and Betacoronavirus entries from UniProt. In addition to the UniPro sequences, we were interested in looking at variants <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/genetic-variability-at-the-adp-ribose-binding-site-of-the-sars-cov-2-nsp3-mac1-and-predicted-effects-of-mutations-on-adp-ribose-binding-post-15/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p>In two of my previous posts <a href="https://thesgc.github.io/static-openlabnotebooks/the-next-target-sars-cov-2-macrodomain-and-its-druggability-post-13/">13</a> and <a href="https://thesgc.github.io/static-openlabnotebooks/mapping-the-genetic-variations-of-alpha-and-betacoronavirus-uniprot-entries-onto-sars-cov-2-nsp3-mac1-crystal-structure-post-14/">14</a>, I showed how we found the residues lining the ADP ribose (ADPr) binding site of SARS-CoV-2 nsp3-Mac1 and we looked at the variability of this site across Alpha- and Betacoronavirus entries from UniProt.</p>
<p>In addition to the UniPro sequences, we were interested in looking at variants from COVID-19 patients. Nicola De Maio, our collaborator from Nick Goldman’s lab at EBI, looked at more than 15000 SARS-CoV-2 samples and identified all mutations at the ADPr binding site.</p>
<p>He identified 15 non-synonymous variants as shown in table 1. For example, at residue number 23, the wild-type sidechain in SARS-CoV-2 Mac1 is isoleucine. In the sample batch Nicola has looked at, he saw that 15872 samples had isoleucine at that position while there were 2 samples with other variants at this position; one was a valine and another was a serine.</p>
<table style="width: 43.8542%; height: 564px;" height="918">
<tbody>
<tr style="height: 18px;">
<td style="width: 168.935px; height: 18px;"><strong>Number</strong></td>
<td style="width: 183.009px; height: 18px;"><strong>Wild Type</strong></td>
<td style="width: 470.417px; height: 18px;"><strong>Non-synonymous Variants</strong></td>
</tr>
<tr style="height: 10px;">
<td style="width: 168.935px; height: 10px;">1</td>
<td style="width: 183.009px; height: 10px;">Ile23</td>
<td style="width: 470.417px; height: 10px;">I (15872), V (1)</td>
</tr>
<tr style="height: 38px;">
<td style="width: 168.935px; height: 38px;">2</td>
<td style="width: 183.009px; height: 38px;">Ile23</td>
<td style="width: 470.417px; height: 38px;">I (15872), S (1)</td>
</tr>
<tr style="height: 38px;">
<td style="width: 168.935px; height: 38px;">3</td>
<td style="width: 183.009px; height: 38px;">Ala39</td>
<td style="width: 470.417px; height: 38px;">A (15872), V (3)</td>
</tr>
<tr style="height: 38px;">
<td style="width: 168.935px; height: 38px;">4</td>
<td style="width: 183.009px; height: 38px;">Asn40</td>
<td style="width: 470.417px; height: 38px;">S (1), N (15874)</td>
</tr>
<tr style="height: 38px;">
<td style="width: 168.935px; height: 38px;">5</td>
<td style="width: 183.009px; height: 38px;">His45</td>
<td style="width: 470.417px; height: 38px;">Y (1), H (15874)</td>
</tr>
<tr style="height: 38px;">
<td style="width: 168.935px; height: 38px;">6</td>
<td style="width: 183.009px; height: 38px;">Gly46</td>
<td style="width: 470.417px; height: 38px;">*(1), G (15874)</td>
</tr>
<tr style="height: 38px;">
<td style="width: 168.935px; height: 38px;">7</td>
<td style="width: 183.009px; height: 38px;">Gly48</td>
<td style="width: 470.417px; height: 38px;">C (1), G (15873)</td>
</tr>
<tr style="height: 38px;">
<td style="width: 168.935px; height: 38px;">8</td>
<td style="width: 183.009px; height: 38px;">Val49</td>
<td style="width: 470.417px; height: 38px;">I (2), V (15871)</td>
</tr>
<tr style="height: 38px;">
<td style="width: 168.935px; height: 38px;">9</td>
<td style="width: 183.009px; height: 38px;">Ala52</td>
<td style="width: 470.417px; height: 38px;">A (15873), V (1)</td>
</tr>
<tr style="height: 38px;">
<td style="width: 168.935px; height: 38px;">10</td>
<td style="width: 183.009px; height: 38px;">Ala129</td>
<td style="width: 470.417px; height: 38px;">A (15781), V (1)</td>
</tr>
<tr style="height: 38px;">
<td style="width: 168.935px; height: 38px;">11</td>
<td style="width: 183.009px; height: 38px;">Ile131</td>
<td style="width: 470.417px; height: 38px;">I (15781), F (1)</td>
</tr>
<tr style="height: 39px;">
<td style="width: 168.935px; height: 39px;">12</td>
<td style="width: 183.009px; height: 39px;">Phe132</td>
<td style="width: 470.417px; height: 39px;">V (1), F (15781)</td>
</tr>
<tr style="height: 39px;">
<td style="width: 168.935px; height: 39px;">13</td>
<td style="width: 183.009px; height: 39px;">Pro136</td>
<td style="width: 470.417px; height: 39px;">P (15760), L (6)</td>
</tr>
<tr style="height: 39px;">
<td style="width: 168.935px; height: 39px;">14</td>
<td style="width: 183.009px; height: 39px;">Pro136</td>
<td style="width: 470.417px; height: 39px;">P (15760), S (6)</td>
</tr>
<tr style="height: 39px;">
<td style="width: 168.935px; height: 39px;">15</td>
<td style="width: 183.009px; height: 39px;">Leu160</td>
<td style="width: 470.417px; height: 39px;">L (15870), F (1)</td>
</tr>
</tbody>
</table>
<p><strong>Table 1. The non-synonymous variants at Mac1 ADPr binding site across more than 15000 SARS-CoV-2 samples.</strong> * denotes that the mutation has caused a premature stop codon at position Gly46, and the patient’s status with this mutation in their viral sample was marked as “released”.</p>
<p>The next step for us was to perform energy calculations for the SARS-CoV-2 variants at nsp3-MAC1 ADPr binding site in order to predict their effects on ADPr binding.</p>
<p>To predict the effect of these mutations on the binding of a known Mac1 ligand, ADPr (PDB: 6w02), we estimated the difference in the binding energy of the ligand between the wild type and the mutated forms of the protein with ICM (Molsoft, San Diego). ICM calculates the change in Gibbs binding energy (dG<sub>bind</sub> in kcal/mol) of the ligand as the energy of the protein-ligand complex minus the energy of the isolated protein and ligand. The difference in binding energy (ddG<sub>bind</sub>), which is what we are interested in, is dG<sub>bind</sub> [mutant] minus dG<sub>bind</sub> [wild-type]. Since the precision of the method is about 2 kcal/mol, ddG<sub>bind</sub> values &gt; 2 kcal/mol indicate mutations that significantly penalize ligand binding. (Schapira et al., 1999) The color-coded structure in figure 1 highlights the variants among the SARS-CoV-2 samples at ADPr binding site of nsp3-Mac1 based on their effects on ligand binding.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/non-syn-variants-mac-color-coded.png" alt="" width="1514" height="750" class="alignnone wp-image-5034 size-full" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/non-syn-variants-mac-color-coded.png 1514w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/non-syn-variants-mac-color-coded-300x149.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/non-syn-variants-mac-color-coded-1024x507.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/non-syn-variants-mac-color-coded-768x380.png 768w" sizes="(max-width: 1514px) 100vw, 1514px" /></p>
<p><strong>Figure 1.</strong> <strong>Energy calculations </strong><strong>of </strong><strong>non-synonymous mutations at SARS-CoV-2 nsp3-Mac1 ADPr binding site to predict their effect on ADPr binding.</strong> The energy unit for ddGbind values is Kcal/mol. The green color represents residues with mutations with ddGbind values below 2.0 kcal/mol and orange color shows residues with mutations that have ddGbind values greater than 2.0 kcal/mol.</p>
<p>Looking at figure 1, we see that the ddGbind values for G48C, I131F are above 2.0 Kcal/mol which is considered a significant change in binding according to a previous study. (Schapira et al., 1999) The remainder of the variants are predicted to have minor effects on ligand binding (-2kcal/mol &lt; ddGbind &lt; 2kcal/mol).</p>
<p>Based on previous studies, coronaviruses with mutations in their macrodomains are highly attenuated and in vivo cause minimal disease. This is due to the role of macrodomain in reversing ADP-ribosylation which disrupts the host’s immune response. (Grunewald et al., 2019)</p>
<p>Below, I use surface representation to model the two SARS-CoV-2 nsp3-Mac1 variants, G48C (figure 2) and I131F (figure 3), as visualization aids. In my <a href="https://doi.org/10.5281/zenodo.3956555">Zenodo</a> report, you can also view the modeled mutations using sticks representation.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/G48C-Mac1.png" alt="" width="1078" height="367" class="alignnone wp-image-5035 size-full" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/G48C-Mac1.png 1078w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/G48C-Mac1-300x102.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/G48C-Mac1-1024x349.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/G48C-Mac1-768x261.png 768w" sizes="(max-width: 1078px) 100vw, 1078px" /></p>
<p><strong>Figure 2.</strong> <strong>The surface representation of SARS-CoV-2 macrodomain variant: G48C.</strong> The corresponding ddGbind value for this mutation is 5.0 Kcal/mol.</p>
<p>C48 is a bulkier residue than glycine as shown in figure 2 which could be the reason for unfavorable and destabilizing predicted effect on ADPr binding with the nsp3-Mac1 at ddGbind value of 5.0 Kcal/mol. The diphosphate moiety binds to the glycine-rich segment (Gly46-Gly47-Gly48) loop on the protein which provides stabilizing effect for the proximal ribose ring in the pocket through hydrogen bonds with G46 (OH2’), G48 (OH1’). (Michalska et al., 2020)</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/I131F-Mac1.png" alt="" width="775" height="355" class="alignnone wp-image-5036 " srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/I131F-Mac1.png 947w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/I131F-Mac1-300x137.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/I131F-Mac1-768x352.png 768w" sizes="(max-width: 775px) 100vw, 775px" /></p>
<p><strong>Figure 3.</strong> <strong>The surface representation of SARS-CoV-2 macrodomain variant: I131F. </strong>The corresponding ddGbind value for this mutation is 4.3 Kcal/mol.</p>
<p>Although phenylalanine is a bulkier residue than isoleucine, its bulkiness is protruded outside the pocket rather than inside the pocket where the phosphate linker is located. According to Karolina Michalska and their colleagues, the proximal ribose ring is stabilized in the pocket by hydrophobic interactions with I131. The mutation to phenylalanine may disturb this hydrophobic interaction.  (Michalska et al., 2020)</p>
<p>To view the 2D ligand-protein diagram showing the chemical interactions between ADPr and nsp3-Mac1, please refer to figure 11 on my <a href="https://doi.org/10.5281/zenodo.3956555">Zenodo</a> report.</p>
<p>We see a lot of structural diversity at the ADP-ribose binding site of the first macrodomain of SARS-CoV-2 nsp3, both across diverse coronaviruses, and across diverse SARS-CoV-2 samples. However, most of these variations are predicted to have little impact on ADPr binding. While this structural diversity is not favorable for the design of broad-spectrum ligands, competitive inhibitors mimicking interactions made by ADPr may have a broad spectrum of action.</p>
<p><strong>References:</strong></p>
<p>Schapira, M., Totrov, M., Abagyan, R. (1999). Prediction of the Binding Energy for Small Molecules, Peptides and Proteins. <em>Journal of Molecular Recognition </em>12(3), 177-90 <a href="https://doi.org/10.1002/(SICI)1099-1352(199905/06)12:3%3c177::AID-JMR451%3e3.0.CO;2-Z">https://doi.org/10.1002/(SICI)1099-1352(199905/06)12:3&lt;177::AID-JMR451&gt;3.0.CO;2-Z</a></p>
<p>Grunewald, M. E., Chen, Y., Kuny, C., Maejima, T., Lease, R., Ferraris, D., Aikawa, M., Sullivan, C. S., Perlman, S., &amp; Fehr, A. R. (2019). The coronavirus macrodomain is required to prevent PARP-mediated inhibition of virus replication and enhancement of IFN expression. <em>PLOS Pathogens</em>, <em>15</em>(5), e1007756. <a href="https://doi.org/10.1371/journal.ppat.1007756">https://doi.org/10.1371/journal.ppat.1007756</a></p>
<p>Michalska, K., Kim, Y., Jedrzejczak, R., Maltseva, N. I., Stols, L., Endres, M., &amp; Joachimiak, A. (2020). Crystal structures of SARS-Cov-2 ADP-ribose phosphatase (ADRP): From the apo form to ligand complexes. <a href="https://doi.org/10.1101/2020.05.14.096081">https://doi.org/10.1101/2020.05.14.096081</a></p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/genetic-variability-at-the-adp-ribose-binding-site-of-the-sars-cov-2-nsp3-mac1-and-predicted-effects-of-mutations-on-adp-ribose-binding-post-15/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
	</channel>
</rss>
