<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>Understudied Kinases &#8211; openlabnotebooks.org</title>
	<atom:link href="https://thesgc.github.io/static-openlabnotebooks/category/project/understudied-kinases/feed/" rel="self" type="application/rss+xml" />
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<description>A growing team of groundbreaking scientists around the world are now sharing their lab notebooks online</description>
	<lastBuildDate>Sun, 02 Jun 2019 11:37:14 +0000</lastBuildDate>
	<language>en-GB</language>
	<sy:updatePeriod>hourly</sy:updatePeriod>
	<sy:updateFrequency>1</sy:updateFrequency>
	<generator>https://wordpress.org/?v=4.9.10</generator>

<image>
	<url>https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/cropped-lab-jar02-32x32.png</url>
	<title>Understudied Kinases &#8211; openlabnotebooks.org</title>
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<width>32</width>
	<height>32</height>
</image> 
	<item>
		<title>How many cells in DIPG behave like stem cells?</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/how-many-cells-in-dipg-behave-like-stem-cells/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/how-many-cells-in-dipg-behave-like-stem-cells/#respond</comments>
		<pubDate>Mon, 06 May 2019 15:40:30 +0000</pubDate>
		<dc:creator><![CDATA[Elizabeth Brown]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Brain Tumours]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Liz Brown]]></category>
		<category><![CDATA[Understudied Kinases]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2804</guid>
		<description><![CDATA[Stem cells are a set of cells that, unlike most cells in the body, can divide as many times as they want, and can turn into any cell in the body. The mutations in cancer often make the cells behave more like stem cells, because in order to grow into a large tumour they may <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/how-many-cells-in-dipg-behave-like-stem-cells/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p><a href="https://stemcells.nih.gov/info/basics/1.htm">Stem cells</a> are a set of cells that, unlike most cells in the body, can divide as many times as they want, and can turn into any cell in the body. The mutations in cancer often make the cells behave more like stem cells, because in order to grow into a large tumour they may need to divide hundreds or thousands of cells, and it can allow a single cell to escape to another part of the body and grow into a second tumour (this is called metastasis). Knowing whether all of the cells in a tumour, or even a small proportion of them behave like this, and whether they&#8217;re killed by new therapies can be important information &#8211; if just one of these cells survives theoretically they can regrow into an entire new tumour.</p>
<p>And this ability to regenerate an tumour is just how we test how many of these cells are present: I developed an assay where I dilute the cells until I can seed just a handful into each well, and then I count how many new colonies (like tiny tumours) have grown.</p>
<p>I then used a statistical rule (called <a href="http://www.biology-pages.info/L/LimitingDilution.html">the Poisson distribution</a>) and a programming package (called R) to work out at what dilution there is just one stem-like cell present i.e. if only one stem cell is present when I seeded 5 cells then 20% are stem-like.</p>
<div id="attachment_2805" style="width: 310px" class="wp-caption aligncenter"><a href="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/05/poisson-graph.png"><img class="size-medium wp-image-2805" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/05/poisson-graph-300x300.png" alt="A graph showing how the program mathematically transforms the data and then finds a straight line that fits it. Where this line crosses -1 indicates where 1 stem-like cell is present." width="300" height="300" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/05/poisson-graph-300x300.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/05/poisson-graph-150x150.png 150w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/05/poisson-graph-768x768.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/05/poisson-graph.png 960w" sizes="(max-width: 300px) 100vw, 300px" /></a><p class="wp-caption-text">The program mathematically transforms the data and then finds a straight line that fits it. Where this line crosses -1 indicates where 1 stem-like cell is present.</p></div>
<p>Although I need to repeat it to make sure the result is reliable, it seems that almost all of the cells in this line are stem-like! <a href="https://zenodo.org/record/2669443">If you&#8217;d like to read about the technical details of this experiment you can do so here.</a></p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/how-many-cells-in-dipg-behave-like-stem-cells/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Growing co-crystallized fragments to ligands of the transcription factor brachyury (T)</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/growing-co-crystallized-fragments-to-ligands-of-the-transcription-factor-brachyury-t/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/growing-co-crystallized-fragments-to-ligands-of-the-transcription-factor-brachyury-t/#respond</comments>
		<pubDate>Wed, 24 Apr 2019 21:15:14 +0000</pubDate>
		<dc:creator><![CDATA[Louisa Temme]]></dc:creator>
				<category><![CDATA[Chordoma]]></category>
		<category><![CDATA[Louisa Temme]]></category>
		<category><![CDATA[Understudied Kinases]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2758</guid>
		<description><![CDATA[In order to develop ligands addressing different possible binding sites of the transcription factor brachyury, fragments co-crystallized with this protein should be developed into ligands by using the program SparkTM (Cresset®). Brachyury (T, TBXT) is an essential regulator of the notochord development. Brachyury is primarily expressed in the embryo but not in the majority of normal <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/growing-co-crystallized-fragments-to-ligands-of-the-transcription-factor-brachyury-t/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>In order to develop ligands addressing different possible binding sites of the transcription factor brachyury, fragments co-crystallized with this protein should be developed into ligands by using the program Spark<sup>TM </sup>(Cresset<sup><span style="font-size: 11.25px;">®</span></sup>).</p>
<p>Brachyury (T, TBXT) is an essential regulator of the notochord development. Brachyury is primarily expressed in the embryo but not in the majority of normal adult tissue. It is known that brachyury is overexpressed in chordoma, a primary bone cancer, and plays a role in the disease pathogenesis. Chordoma affects the skull-base, the mobile spine and the sacrum. It is a slow-growing, malign cancer that is locally invasive and has a tendency to recur although a surgical and/or radiation therapy was conducted.<sup>1</sup></p>
<p>It is hypothesized that chordoma is originated from embryonic notochordal remnants. Brachyury is proposed to act as an aberrantly activated transcription factor and thereby be oncogenic. No approved targeted therapies, chemotherapies or immunotherapies are available for chordoma and no small-molecule inhibitors for brachyury have been developed.<sup>1 </sup>Therefore, a fragment-based approach should be explored to develop ligands for brachyury to study this protein.</p>
<p>Our colleagues Joseph Newman and Opher Gileadi at SGC Oxford co-crystallized wild-type human brachyury protein with various fragments which bind to the sites A-E (Figure 1, left). Thankfully, they provided the fragment and protein structures to us so that we can explore the usage of the Spark<sup>TM </sup>growing approach to develop ligands for these binding sites.</p>
<p><img class="wp-image-2759 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Picture1-300x121.png" alt="" width="592" height="239" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Picture1-300x121.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Picture1-768x310.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Picture1.png 974w" sizes="(max-width: 592px) 100vw, 592px" />Figure 1. Left: Aligned X-ray structures of the human wild-type brachyury protein in complex with fragments at different binding sites (A-E). These first crystallization structures contain Cd<sup>2+</sup>ions (green balls). Right: Binding site A close to loop with D177 and in proximity to residue Y88.<sup>2</sup></p>
<p>First, we focused on binding site A since most fragments bind to this binding site and site A is close to a loop that is mutated in chordoma-linked variant G177D<sup>3 </sup>and to the amino acid residue Y88 that, if mutated, affects binding to effector proteins<sup>4 </sup>(Figure 1, left).</p>
<p><img class="wp-image-2761 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Picture2-300x86.png" alt="" width="607" height="174" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Picture2-300x86.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Picture2-768x220.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Picture2.png 974w" sizes="(max-width: 607px) 100vw, 607px" /></p>
<p>Figure 2. Fragment <strong>1 </strong>and <strong>2 </strong>co-crystallized in the binding site A of wild-type TBXT.</p>
<p>21 fragments were co-crystallized to binding site A. In Figure 2, the binding modes of two co-crystallized fragments are displayed. Fragments <strong>1 </strong>and <strong>2 </strong>show interactions or close proximity to two of the three amino acids Ser129, Ser89 and Arg180. Therefore, the growing approach of the program Spark<sup>TM </sup>should be used to identify new ligands that are able to form all three interactions. One growing approach started from the amino group of fragment <strong>1 </strong>and was guided by fragment <strong>2 </strong>towards the region of its carboxylate group (Figure 3).</p>
<p><img class="wp-image-2763 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Picture3-300x171.png" alt="" width="484" height="276" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Picture3-300x171.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Picture3-768x438.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Picture3-780x445.png 780w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Picture3.png 781w" sizes="(max-width: 484px) 100vw, 484px" /></p>
<p>Figure 3: Spark<sup>TM </sup>growing approach starting from fragment <strong>1 </strong>towards Arg180 guided by fragment <strong>2</strong>. The functional groups of fragments <strong>1 </strong>and <strong>2 </strong>revealing H-bond interactions with amino acid residues are marked with the following colors: Ser129 (red), Ser89 (blue), Arg180 (green).</p>
<p>To achieve new ligands with similar properties in certain areas so-called field points were used. They are ‘condensed representations of electrostatic, hydrophobic and shape properties’ (Cresset<sup><span style="font-size: 11.25px;">®</span></sup>) of those areas where they are placed by the program Spark<sup>TM</sup>. Regarding the growing approach of fragment <strong>1 </strong>and <strong>2 </strong>the hydrophobic field point of the phenyl ring and the negative field point of the carboxylate group of fragment <strong>2 </strong>were used. In total three of the 21 fragments bound to site A were used as starting points and each of them were grown in the area of two different fragments. The results were clustered by Spark<sup>TM </sup>depending on their structural similarity. Those compounds with the highest score of the first 20 clusters ranked according their scores were chosen to analyze their interactions by docking into binding site A using the docking program Flare<sup>TM </sup>(Cresset<sup><span style="font-size: 11.25px;">®</span></sup>). 4,000 docking results were inspected visually and compounds with a high docking score, high electrostatic complementarity and high number of interactions to the binding site were picked.</p>
<p>Figure 4 shows the docking pose of the example compound <strong>3 </strong>that was obtained by docking <strong>3 </strong>in binding site A of brachyury using Flare<sup>TM</sup>. Besides other criteria like docking score and electrostatic complementarity following interactions of <strong>3 </strong>are promising for the co-crystallization of this compound with brachyury: two H-bond interactions of the positively charged amino moiety to Ser129 and His126, the proximity of the amide carbonyl-group to Ser89 and the H-bond interactions of the oxazole O-atom and N-atom towards Arg180.</p>
<p><img class="wp-image-2765 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Picture4-300x232.png" alt="" width="454" height="351" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Picture4-300x232.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Picture4.png 633w" sizes="(max-width: 454px) 100vw, 454px" />Figure 4. Binding mode of new compound <strong>3 </strong>obtained by docking into binding site A of wild-type brachyury with Flare<sup>TM</sup>. H-bond interactions are marked with green dotted lines.</p>
<p>Additionally, new compounds were obtained by searching in the Enamine REAL database for similar compounds to those compounds that were found by the Spark<sup>TM </sup>approach and that showed promising docking results in Flare<sup>TM</sup>.</p>
<p>A selection of compounds that are identical or slightly different than those found in the Spark<sup>TM </sup>approach as well as compounds that represent similar ones to the Spark<sup>TM </sup>hits were purchased and will be tested regarding their ability to co-crystallize with brachyury. If the compounds bind to this transcription factor, they will represent promising starting points for future ligands of brachyury.</p>
<p>&nbsp;</p>
<p>(1) Sharifnia, T., <em> Med. (N. Y., NY, U. S.)</em>, <strong>2019</strong>, <em>25</em>, 292-300.</p>
<p>(2) Joseph Newman and Opher Gileadi, SGC Oxford, University of Oxford, UK.</p>
<p>(3) Pillay, N., <em>Nat. Genet.</em>, <strong>2012</strong>, <em>44</em>, 1185-1187.</p>
<p>(4) Beisaw, A., <em>EMBO reports</em>, <strong>2018</strong>, <em>19</em>, 118-134.</p>
<p>&nbsp;</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/growing-co-crystallized-fragments-to-ligands-of-the-transcription-factor-brachyury-t/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Novel Hinge Binders: N-Methyl-azaindoles</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/novel-hinge-binders-n-methyl-azaindoles/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/novel-hinge-binders-n-methyl-azaindoles/#respond</comments>
		<pubDate>Tue, 02 Apr 2019 18:30:55 +0000</pubDate>
		<dc:creator><![CDATA[Sean O'Byrne]]></dc:creator>
				<category><![CDATA[Sean O'Byrne]]></category>
		<category><![CDATA[Understudied Kinases]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2644</guid>
		<description><![CDATA[One goal of the CaMKK2 project is to explore the chemistry and biology of novel hinge binders. Various hinge binders showing CaMKK2 inhibition have been identified in the literature and through testing of compounds synthesized in the SGC. Of note in this search was a recent publication by Price et al. (Bioorg. Med. Chem. Lett., <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/novel-hinge-binders-n-methyl-azaindoles/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>One goal of the CaMKK2 project is to explore the chemistry and biology of novel hinge binders. Various hinge binders showing CaMKK2 inhibition have been identified in the literature and through testing of compounds synthesized in the SGC. Of note in this search was a recent publication by Price <em>et al</em>. (<em>Bioorg. Med. Chem. Lett</em>., <strong>2018</strong>, <em>28</em>, 1958–1963), in which molecules based on 7-azaindole (<strong>1,2</strong>) and 2-amino-pyrimidine (<strong>3</strong>) (<strong>Figure 1</strong>) cores showed excellent activity in an enzymatic CaMKK2 assay.</p>
<div id="attachment_2645" style="width: 790px" class="wp-caption alignnone"><img class="wp-image-2645 size-large" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Figure-1-1024x283.png" alt="" width="780" height="216" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Figure-1-1024x283.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Figure-1-300x83.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Figure-1-768x213.png 768w" sizes="(max-width: 780px) 100vw, 780px" /><p class="wp-caption-text"><strong>Figure 1</strong>: Azaindole based CaMKK2 inhibitors</p></div>
<p>Although the compounds have good potency, they are promiscuous, showing poor selectivity in a small screen against 48-86 different kinases. Indeed between 15-30% of the kinases in the panel were inhibited with an IC<sub>50</sub> &lt;100 nM.</p>
<p>Small modifications to the hinge binding region of pharmacophore can have drastic effects on the binding mode, and therefore the selectivity and potency, of inhibitors. It was decided to probe if <em>N</em>-methylation of the azaindole core could be tolerated in CaMKK2. <em>N</em>-methyl azaindole <strong>4 </strong>(<strong>Figure 1</strong>) was synthesized and sent for testing in a CaMKK2 enzyme inhibition assay. The compounds showed moderate activity giving an IC<sub>50 </sub>of 183 nM. The compound also gave had good solubility (125 µM) and was membrane permeable. It was decided to synthesis a small library of compounds based on <strong>4</strong> to see if more potent compounds based on this scaffold.</p>
<p>The compounds were relatively easy to access synthetically. <em>N</em>-methylation of 2-iodo-4-chloro-7-azaindole followed by two Suzuki couplings and an ester hydrolysis allowed quick access to the desired compounds. Conveniently, both Suzuki coupling could be carried using microwave conditions with reactions going to completion in 30 minutes (<strong>Scheme 1</strong>).</p>
<div id="attachment_2647" style="width: 790px" class="wp-caption alignnone"><img class="size-large wp-image-2647" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Scheme-1-1-1024x245.png" alt="" width="780" height="187" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Scheme-1-1-1024x245.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Scheme-1-1-300x72.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Scheme-1-1-768x184.png 768w" sizes="(max-width: 780px) 100vw, 780px" /><p class="wp-caption-text"><strong>Scheme 1</strong>: Sequential Suzuki reactions utilizing a microwave reactor to greatly reduce reaction times.</p></div>
<p>In choosing what analogues to make in the library I wanted to probe the effects of electron withdrawing and donating groups, lipophilicity and to determine the effect of extending the molecules with morpholino- groups pointing out of the binding pocket towards the solvent. The library of compounds synthesized is shown in <strong>Figure 2</strong>.</p>
<div id="attachment_2648" style="width: 790px" class="wp-caption alignnone"><img class="size-large wp-image-2648" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Figure-2-1024x515.png" alt="" width="780" height="392" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Figure-2-1024x515.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Figure-2-300x151.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Figure-2-768x386.png 768w" sizes="(max-width: 780px) 100vw, 780px" /><p class="wp-caption-text">Figure 2: Library of N-methyl-azaindoles synthesized and sent for a CaMKK2 enzyme inhibition assay.</p></div>
<p>The compounds have been shipped and we are awaiting the results from the enzyme inhibition assay. If any of the compounds have good activity, the best will be sent for a selectivity profiling using KINOMEscan screening platform provided by DiscoverX. From this we will learn if <em>N</em>-methylation can indeed modulate selectivity across the kinome.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/novel-hinge-binders-n-methyl-azaindoles/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Synthesis of promiscuous ALK2 inhibitors</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/synthesis-of-promiscuous-alk2-inhibitors/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/synthesis-of-promiscuous-alk2-inhibitors/#respond</comments>
		<pubDate>Tue, 19 Mar 2019 20:34:09 +0000</pubDate>
		<dc:creator><![CDATA[Alfredo Picado]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Alfredo Picado]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>
		<category><![CDATA[Understudied Kinases]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2598</guid>
		<description><![CDATA[In a collaboration between SGC-University of North Carolina-Chapel Hill and M4K Pharma, we plan to synthesize 3 different imidazopyridin pyrazines as promiscuous kinase ALK2 inhibitors, compounds 1-3, Scheme 1. The first step will be the synthesis of key intermediate 10, Scheme 2.  The proposed synthesis begins with a nucleophilic aromatic substitution on compound 4 to <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/synthesis-of-promiscuous-alk2-inhibitors/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>In a collaboration between SGC-University of North Carolina-Chapel Hill and M4K Pharma, we plan to synthesize 3 different imidazopyridin pyrazines as promiscuous kinase ALK2 inhibitors, compounds <strong>1-3</strong>, Scheme 1.</p>
<div id="attachment_2596" style="width: 994px" class="wp-caption alignnone"><img class="size-full wp-image-2596" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Screen-Shot-2019-03-19-at-4.22.27-PM.png" alt="" width="984" height="346" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Screen-Shot-2019-03-19-at-4.22.27-PM.png 984w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Screen-Shot-2019-03-19-at-4.22.27-PM-300x105.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Screen-Shot-2019-03-19-at-4.22.27-PM-768x270.png 768w" sizes="(max-width: 984px) 100vw, 984px" /><p class="wp-caption-text">Scheme 1. ALK2 inhibitors</p></div>
<p>The first step will be the synthesis of key intermediate <strong>10</strong>, Scheme 2.  The proposed synthesis begins with a nucleophilic aromatic substitution on compound <strong>4</strong> to get nitropyridine <strong>5</strong>, then reduction with hydrogen on palladium/carbon to obtain the pyridine-3,4-diamine <strong>6</strong>.</p>
<p>On the other hand, aminopyrazine <strong>7</strong> will be saponified to the aminopyrazine sodium salt <strong>8</strong>, which will be coupled with  compound <strong>6</strong> using peptide coupling reagent HATU to form bromopyrazine carboxamide <strong>9</strong>, followed by heating in presence of acetic acid to yield the cyclized intermediate 5-bromo-3-(1-ethyl-1<em>H</em>-imidazo[4,5-<em>c</em>]pyridin-2-yl)pyrazin-2-amine <strong>10</strong>.</p>
<div id="attachment_2597" style="width: 928px" class="wp-caption alignnone"><img class="size-full wp-image-2597" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Screen-Shot-2019-03-19-at-4.22.42-PM.png" alt="" width="918" height="650" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Screen-Shot-2019-03-19-at-4.22.42-PM.png 918w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Screen-Shot-2019-03-19-at-4.22.42-PM-300x212.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Screen-Shot-2019-03-19-at-4.22.42-PM-768x544.png 768w" sizes="(max-width: 918px) 100vw, 918px" /><p class="wp-caption-text">Scheme 2. Synthesis route 1</p></div>
<p>If we succeed with the proposed synthesis of imidazopyridine pyrazine <strong>10</strong>, then soon we will present next steps to get to target compounds <strong>1-3</strong></p>
<p>This collaboration is the under supervision of Dr. David Drewry.</p>
<p>Synthesis routes were adapted from international patents WO 2006/063167 A1, WO 2008/071937 and <em>ACS Med. Chem. Lett</em>. 2016, 7, 5, 514-519.</p>
<p>&nbsp;</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/synthesis-of-promiscuous-alk2-inhibitors/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Checking I haven&#8217;t got my cells mixed up&#8230;</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/checking-i-havent-got-my-cells-mixed-up/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/checking-i-havent-got-my-cells-mixed-up/#respond</comments>
		<pubDate>Thu, 28 Feb 2019 18:56:05 +0000</pubDate>
		<dc:creator><![CDATA[Elizabeth Brown]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Brain Tumours]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>
		<category><![CDATA[Liz Brown]]></category>
		<category><![CDATA[Understudied Kinases]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2542</guid>
		<description><![CDATA[Did you know as many as 30% of the cells in labs could be misidentified? And what a nightmare it would be to find out years of your work are just completely wrong because you got your skin cells mixed up with your brain cells&#8230; So to avoid that catastrophe, I took some time to <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/checking-i-havent-got-my-cells-mixed-up/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Did you know as many as 30% of the cells in labs could be misidentified? And what a nightmare it would be to find out years of your work are just completely wrong because you got your skin cells mixed up with your brain cells&#8230;</p>
<p>So to avoid that catastrophe, I took some time to check the DNA sequence of the ACVR1 gene, because for this gene we know what should be mutated (or not) for each of my cell lines. This way I can make sure that I haven&#8217;t mixed them up in the past year (<em>and</em> that they haven&#8217;t picked up new mutations in this important area all by themselves).</p>
<p>All I had to do was collect the genomes of the cells, amplify specifically the ACVR1 gene, send it for overnight sequencing, and then I get a trace like this back from the facility:</p>
<div id="attachment_2544" style="width: 790px" class="wp-caption aligncenter"><a href="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/R206H-sequence-1.png"><img class="wp-image-2544 size-large" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/R206H-sequence-1-1024x396.png" alt="Sequencing trace covering the R206H mutation in ACVR1 found in FOP and DIPG" width="780" height="302" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/R206H-sequence-1-1024x396.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/R206H-sequence-1-300x116.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/R206H-sequence-1-768x297.png 768w" sizes="(max-width: 780px) 100vw, 780px" /></a><p class="wp-caption-text">Sequencing trace covering the R206H mutation in ACVR1 found in FOP and DIPG</p></div>
<p>Each DNA base has one peak, except one that has both a guanine (G) and an adenine (A) DNA base. The adenine, present on one chromosome, causes the change from an arginine (R) to a histidine (H) at the 206th position of the ACVR1 protein i.e. the R206H mutation.</p>
<p>This trace is from the HSJD-DIPG-007 cell line and is the mutation I expected. According to all my sequencing, none of my cell lines have gotten mixed up, yay!</p>
<p>You can read the full details of how I did this, <a href="https://zenodo.org/record/2579809#.XHgumMD7SUk">here on Zenodo</a>.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/checking-i-havent-got-my-cells-mixed-up/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Quinazoline Series: Synthesis</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/quinazoline-series-synthesis/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/quinazoline-series-synthesis/#respond</comments>
		<pubDate>Sun, 24 Feb 2019 22:58:15 +0000</pubDate>
		<dc:creator><![CDATA[Benjamin Eduful]]></dc:creator>
				<category><![CDATA[Ben Eduful]]></category>
		<category><![CDATA[Understudied Kinases]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2434</guid>
		<description><![CDATA[We have received some very interesting biological data for our CaMKK2 inhibitor series (furopyridine, Azaindole and quinoline) and I will be sharing the data in the coming weeks, but before that, I wanted to share my synthetic efforts towards our quinoline series. Synthesis of Quinoline series: Starting with 6-Bromo-4-chloroquinoline, a two (2) step synthetic scheme <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/quinazoline-series-synthesis/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p style="text-align: justify;">We have received some very interesting biological data for our CaMKK2 inhibitor series (furopyridine, Azaindole and quinoline) and I will be sharing the data in the coming weeks, but before that, I wanted to share my synthetic efforts towards our quinoline series.</p>
<p><strong>Synthesis of Quinoline series</strong>:</p>
<p style="text-align: justify;">Starting with 6-Bromo-4-chloroquinoline, a two (2) step synthetic scheme was devised where the bromine (Br) synthetic handle was substituted first (with an appropriate boronic acid/ester), followed by the chlorine (Cl) , <strong>Figure 1</strong>, after testing many different reaction conditions for selectivity.</p>
<p style="text-align: center;"><strong>Figure 1: Synthesis of quinoline series</strong></p>
<p><img class=" wp-image-2526 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/Quinoline-1-300x67.png" alt="" width="416" height="93" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/Quinoline-1-300x67.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/Quinoline-1.png 569w" sizes="(max-width: 416px) 100vw, 416px" /></p>
<p style="text-align: justify;">A number of analogs were thus synthesized using the above synthetic strategy, for example, synthesis of the quinoline analog: 2-cyclopentyl-4-(6-(4-(methylsulfonyl)phenyl)quinolin-4-yl)benzoic acid was achieved using this method, <strong>Scheme 1</strong>.</p>
<p style="text-align: center;"><strong>Scheme 1: Synthesis of quinoline analog</strong></p>
<p><img class="wp-image-2527 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/Quinoline-2-300x88.png" alt="" width="701" height="206" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/Quinoline-2-300x88.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/Quinoline-2-768x226.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/Quinoline-2.png 936w" sizes="(max-width: 701px) 100vw, 701px" /></p>
<p style="text-align: justify;">Although the synthetic strategy above led to the synthesis of some analogs, it was not without challenges. For instance, selectivity for the Br was not always achieved as disubstituted products were common in some cases due to the activated 4-position of the Cl.</p>
<p style="text-align: justify;">In order to overcome these challenges and to allow for rapid synthesis of different analogs over a short period of time, a strategy to first install the cyclopentyl derivative was sort after. In the synthetic scheme below, <strong>Scheme 2</strong>, 6-chloroquinolin-4-ol, <strong>101</strong>, was treated with triflic anhydride in the presence of DIPEA in DCM to obtain 6-chloroquinolin-4-yl trifluoromethanesulfonate, <strong>102</strong>. This compound was able to undergo the Suzuki cross coupling reaction to obtain compound <strong>103</strong>. With this compound in hand, a number of different analogs are being generated in our search for active CaMKK2 chemical probes, for example compound <strong>105</strong>, <strong>Scheme 2</strong>.</p>
<p style="text-align: center;"><strong>Scheme 2: Synthesis of quinoline analog via new route</strong></p>
<p><img class=" wp-image-2528 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/Quinoline-3-300x71.png" alt="" width="758" height="179" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/Quinoline-3-300x71.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/Quinoline-3-768x182.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/Quinoline-3-1024x242.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/Quinoline-3.png 1282w" sizes="(max-width: 758px) 100vw, 758px" /></p>
<p>We are also looking to exploring the 7-position of the quinoline with similar substitutions to the 6-position (<strong>Figure 2</strong>), quinoline derivative 1. Other anticipated analogs will include O-alkylations (quinoline derivatives 2 and 3), N-alkylations or peptide couplings or Hartwig-Buchwald aminations (quinolines derivatives 4 and 5) etc. Details of such reactions will be communicated in due course.</p>
<p style="text-align: center;"><strong>Figure 2: Exploration of other quinoline analogs</strong></p>
<p><img class=" wp-image-2529 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/Quinoline-4-300x81.png" alt="" width="504" height="136" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/Quinoline-4-300x81.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/Quinoline-4-768x207.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/Quinoline-4.png 858w" sizes="(max-width: 504px) 100vw, 504px" /></p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/quinazoline-series-synthesis/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Optimising seeding of normal and FOP cells for a later assay &#8211; prettier than expected</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/optimising-seeding-of-normal-and-fop-cells-for-a-later-assay-prettier-than-expected/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/optimising-seeding-of-normal-and-fop-cells-for-a-later-assay-prettier-than-expected/#comments</comments>
		<pubDate>Tue, 22 Jan 2019 19:22:08 +0000</pubDate>
		<dc:creator><![CDATA[Elizabeth Brown]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Brain Tumours]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>
		<category><![CDATA[Liz Brown]]></category>
		<category><![CDATA[Understudied Kinases]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2388</guid>
		<description><![CDATA[Further down the line I want to be able to test the activation of the BMP signalling pathway downstream of the ALK2 receptor. For this I plan to use the alkaline phosphatase gene as a readout &#8211; it&#8217;s a well known gene activated downstream of BMP signalling so a lot of easy to use assays <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/optimising-seeding-of-normal-and-fop-cells-for-a-later-assay-prettier-than-expected/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Further down the line I want to be able to test the activation of the BMP signalling pathway downstream of the ALK2 receptor. For this I plan to use the alkaline phosphatase gene as a readout &#8211; it&#8217;s a well known gene activated downstream of BMP signalling so a lot of easy to use assays already exist for it.</p>
<p>Unfortunately <a href="https://thesgc.github.io/static-openlabnotebooks/optimising-an-assay-to-detect-alkaline-phosphatase-activity-in-c2c12-cells/">when I tested this assay with the standard C2C12 cell line it didn&#8217;t work particularly well</a> so I&#8217;ve decided to try again with a new set of cells. This time I&#8217;m choosing connective tissue cells (fibroblasts) from FOP patients as a reporter cell line because they respond more strongly to the signals that activate the BMP pathway (called BMP ligands).</p>
<p>I&#8217;ve never worked with these cells before so, again, I needed to work out how many I&#8217;ll need to get a good result from the assay. These cells however grow as a very thin sheet which is so see-through that I initially couldn&#8217;t tell if the wells were full. To solve this I used stains called Calcein AM and Hoechst. Calcein AM can stain any living cell because they metabolise it from a see-through molecule into a fluorescent green product, whereas Hoechst can stick to DNA and turn all the cell nuclei blue, and so suddenly, all of my cells glow:</p>
<div id="attachment_2389" style="width: 1382px" class="wp-caption aligncenter"><a href="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/Calcein-AM-example.png"><img class="wp-image-2389 size-full" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/Calcein-AM-example.png" alt="Fibroblast cells stained with Calcein AM (green) and Hoechst (blue)" width="1372" height="784" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/Calcein-AM-example.png 1372w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/Calcein-AM-example-300x171.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/Calcein-AM-example-768x439.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/Calcein-AM-example-1024x585.png 1024w" sizes="(max-width: 1372px) 100vw, 1372px" /></a><p class="wp-caption-text">Fibroblast cells stained with Calcein AM (green) and Hoechst (blue)</p></div>
<p>Aren&#8217;t they pretty?</p>
<p>Once they&#8217;re this easy to see, it&#8217;s even easier to see how many cells are needed to fill a well, but not get overcrowded. You can read more about how I did that in my <a href="https://zenodo.org/record/2546911">Zenodo post</a>.</p>
<p>The only question I have left is why some cells didn&#8217;t stain with Calcein AM, even though their nucleus looks fine? Anyone know the answer?</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/optimising-seeding-of-normal-and-fop-cells-for-a-later-assay-prettier-than-expected/feed/</wfw:commentRss>
		<slash:comments>1</slash:comments>
		</item>
		<item>
		<title>Selective CDKL2 inhibitors &#8211; molecular modeling</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/selective-cdkl2-inhibitors-molecular-modeling/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/selective-cdkl2-inhibitors-molecular-modeling/#respond</comments>
		<pubDate>Fri, 21 Dec 2018 02:02:27 +0000</pubDate>
		<dc:creator><![CDATA[Louisa Temme]]></dc:creator>
				<category><![CDATA[Louisa Temme]]></category>
		<category><![CDATA[Understudied Kinases]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2325</guid>
		<description><![CDATA[At SGC we are interested in developing protein kinase inhibitors with a high selectivity towards other protein kinases. Based on lead compounds from the literature and our group new and selective CDKL2 inhibitors should be designed, synthesized and tested. Carla Alamillo, a former member of our group, worked on this topic and mentioned in her <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/selective-cdkl2-inhibitors-molecular-modeling/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p style="text-align: justify;">At SGC we are interested in developing protein kinase inhibitors with a high selectivity towards other protein kinases. Based on lead compounds from the literature and our group new and selective CDKL2 inhibitors should be designed, synthesized and tested. Carla Alamillo, a former member of our group, worked on this topic and mentioned in her last entry in the open notebook about this topic  (<a href="https://thesgc.github.io/static-openlabnotebooks/sns-032-analogues-update/">https://thesgc.github.io/static-openlabnotebooks/sns-032-analogues-update/</a>) the pro-mising role of SNS-032 as a starting point for new protein kinase inhibitors (Figure 1).<br />
SNS-032 shows a low K<sub>d</sub>value for CDKL2 (48 nM) but also for many other kinases. This means the selectivity of this compound towards other protein kinases is low. In contrast, CAF-181 synthesized by Carla (compound 11 in Carla’s entry) shows low K<sub>d</sub>values for CDKL2 and CDK7 and a high selectivity towards the other kinases tested. Additionally, CAF-181 reveals a moderate CDKL2 IC<sub>50</sub>value of 417 nM. Therefore, CAF-181 represents a good lead compound for the development of new CDKL2 inhibitors with improved selectivity towards CDK7.</p>
<p style="text-align: justify;"><img class="wp-image-2327 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/122018_Picture1-1-300x174.png" alt="" width="694" height="402" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/122018_Picture1-1-300x174.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/122018_Picture1-1.png 668w" sizes="(max-width: 694px) 100vw, 694px" /></p>
<p style="text-align: justify;"><strong>Figure 1. </strong>Lead compounds SNS-032 and CAF-181 and their respective K<sub>d </sub>values.</p>
<p style="text-align: justify;">To design new compounds the molecular modeling software Flare<sup>TM</sup>(Cresset<sup>®</sup>) was used. The first step for the <em>in silico </em>docking is the selection of the crystal structure of the respective protein. Two crystal structures of CDKL2 are available on the PDB website (<a href="https://www.rcsb.org/">https://www.rcsb.org</a>) but not published in a journal yet. One crystal structure is co-crystallized with the compound TCS 2312 (PDB-ID 4BBM) and the other with the compound DKI (PDB-ID 4AAA) (Figure 2). The 2D interaction maps (<a href="https://www.rcsb.org/structure/4AAA">https://www.rcsb.org/structure/4AAA</a> and <a href="https://www.rcsb.org/structure/4BBM">https://www.rcsb.org/structure/4BBM</a>) have to be flipped vertically to match the same amino acid residues. Interestingly, Val83 (Figure 2, blue box) seems to be an impor-tant interaction partner for the ligands since both compounds reveal two H-bond interactions with Val83.</p>
<p style="text-align: justify;"><img class="wp-image-2328 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/122018_Picture2-300x181.png" alt="" width="629" height="380" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/122018_Picture2-300x181.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/122018_Picture2.png 600w" sizes="(max-width: 629px) 100vw, 629px" /></p>
<p style="text-align: justify;"><strong>Figure 2. </strong>Crystal structures of CDKL2: co-crystallized with TCS 2312 (A, PDB-ID 4BBM, DOI: 10.2210/pdb4BBM/pdb) and co-crystallized with DKI (B, PDB-ID 4AAA, DOI: 10.2210/pdb4AAA/pdb). Black dashed lines indicate hydrogen bonds, salt bridges, and metal interactions. Green solid lines show hydrophobic interactions and green dashed lines show π-π and π-cation interactions (<a href="https://www.rcsb.org/structure/4BBM">https://www.rcsb.org/structure/4BBM</a> and <a href="https://www.rcsb.org/structure/4AAA">https://www.rcsb.org/structure/4AAA</a>).</p>
<p style="text-align: justify;">During the <em>in silico </em>docking studies the crystal structure 4BBM was used because the structure 4AAA reveals an unresolved amino acid sequence ending with Ala148 and NME148A (Figure 3B, black box). Unfortunately, this sequence part also contains the amino acid Phe147 that forms a part of the active site and likely interacts with inhibitors. However, the protein structure used for the docking should contain a fully resolved amino acid sequence in this area. The protein structure 4BBM provides a fully resolved amino acid sequence which makes it suitable for <em>in silico </em>docking.</p>
<p style="text-align: justify;"><img class="wp-image-2329 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/122018_Picture3-300x81.png" alt="" width="870" height="235" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/122018_Picture3-300x81.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/122018_Picture3-768x208.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/122018_Picture3.png 974w" sizes="(max-width: 870px) 100vw, 870px" /></p>
<p style="text-align: justify;"><strong>Figure 3. </strong>Structure of CDKL2 co-crystallized with either TCS 2312 (orange, A, PDB-ID 4BBM) or DKI (dark red, B, PDB-ID 4AAA). The dashed green and orange lines indicate H-bond interactions and the purple lines CH-π interactions. Docked ligands are shown as sticks. Oxygen and nitrogen atoms are colored in red and blue, respectively.</p>
<p style="text-align: justify;">However, Phe147 in structure 4BBM is not clarified. Instead of the phenyl ring a methyl group is visible. Capping by the docking program Flare<sup>TM</sup> did not result in phenylalanine but just in a partially aromatic phenyl ring of this amino acid (Figure 3A, black box). This problem was solved by mutation of this position into a phenylalanine.</p>
<p style="text-align: justify;">To investigate the selectivity of newly designed compounds they should also be investigated during the <em>in silico </em>docking into the crystal structure of CDK7 with the PDB-ID 1UA2.</p>
<p style="text-align: justify;">The next step of <em>in silico</em> docking into the CDKL2 structure 4BBM included the validation of the crystal structure and the docking protocol. After preparation and minimization of the crystal structure the co-crystallized ligand TCS 2312 should be redocked into the active site. However, the co-crystallized ligand TCS 2312 shows different bond orders/tautomeric states as well as a different protonation state on the basic nitrogen than the literature compound TCS 2312. To calculate the RMSD value between the binding mode of the structurally corrected and redocked TCS 2312 and the co-crystallized ligand the co-crystallized binding mode had to be corrected because Flare<sup>TM</sup> can only calculate a RMSD value between structurally identical compounds. Therefore, the co-crystallized ligand was corrected and only the resorcinol moiety was minimized to achieve a planar aromatic system. In this way the binding mode of the co-crystallized ligand was changed as little as possible. Then, the corrected, co-crystallized ligand TCS  2312 was minimized and the structure at physiological pH was generated. Afterwards, this compound was docked into the crystal structure 4BBM. Figure 4A shows the similar orientation of corrected, co-crystallized TCS 2312 (yellow) and the redocked ligand (purple). The similar orientation of both binding modes and the hereby resulting low RMSD value<br />
(1.65 Å) suggest that the corrected CDKL2 protein crystal structure 4BBM and the docking protocol are suitable for the <em>in silico </em>docking of new compounds.</p>
<p style="text-align: justify;"><img class="wp-image-2330 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/122018_Picture4-300x181.png" alt="" width="762" height="460" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/122018_Picture4-300x181.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/122018_Picture4-768x463.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/122018_Picture4.png 943w" sizes="(max-width: 762px) 100vw, 762px" /></p>
<p style="text-align: justify;"><strong>Figure 4. </strong>Predicted binding modes of TCS 2312 (A, purple), CAF-181 (B) and SNS-032 (C and D) obtained by docking into the CDKL2 crystal structure with the PDB-ID 4BBM.Binding mode of the corrected, co-crystallized ligand TCS 2312 (A, yellow). The dashed green and orange lines indicate H-bond interactions and the purple lines CH-π interactions. Docked ligands are shown as sticks. Oxygen and nitrogen atoms are colored in red and blue, respectively.</p>
<p>&nbsp;</p>
<p style="text-align: justify;">In the Figures 4B-D the docking results of CAF-181 and SNS-032 are shown. The <em>tert</em>-butyl group of both compounds occupies the inner part of the active site whereas the piperazine ring of CAF-181 and the piperidine of SNS 032 point towards the solvent accessible area. The N-atom of the SNS-032 piperidine ring is part of an H-bond interaction with one of the carboxylic acid moieties of either Glu130 (C) or Asp89 (D). Therefore, it might be promising to introduce amide substituents that can interact with those acidic amino acids.</p>
<p style="text-align: justify;">CAF compounds, synthesized and tested previously in our group, were docked into the CDKL2 crystal structure 4BBM and in the CDK7 crystal structure 1UA2. The respective docking scores and electrostatic complementarity<sup>TM </sup>scores of the resulting docking poses reveal a good correlation to the K<sub>d </sub>values of those compounds regarding CDKL2. However, no clear correlation was found regarding docking of CAF compounds in CDK7. This observation could be explained with the larger active site of CDK7 and its many solvent accessible areas that could facilitate binding of structurally diverse compounds.</p>
<p style="text-align: justify;">Interestingly, <em>in silico</em> docking of SNS-032 and CAF-181 in the active site of CDK7 reveal  curved orientations of both compounds (Figure 5). The green color of the electrostatic complementarity<sup>TM </sup>surface of the SNS-032 binding mode in Figure 5C indicates a high electrostatic complementarity<sup>TM </sup>of the ligand to this active site which could be the reason for the high inhibitory activity of SNS-032 regarding CDK7.</p>
<p style="text-align: justify;"><em><img class="wp-image-2331 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/122018_Picture5-300x180.png" alt="" width="751" height="451" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/122018_Picture5-300x180.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/122018_Picture5-768x461.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/122018_Picture5.png 962w" sizes="(max-width: 751px) 100vw, 751px" /></em></p>
<p style="text-align: justify;"><strong>Figure 5. </strong>Predicted binding modes obtained by docking into the CDK7 crystal structure with the PDB-ID 1UA2:<br />
(A) SNS-032. (B) SNS-032 with the electrostatic potential surface of CDK7. (C) SNS-032 with its electrostatic complementarity<sup>TM </sup>surface. (D) CAF-181. The dashed green and orange lines indicate H-bond interactions and the purple lines CH-π interactions. Docked ligands are shown as sticks. Oxygen and nitrogen atoms are colored in red and blue, respectively.</p>
<p style="text-align: justify;">Based on this observation new CDKL2 inhibitors should reveal restrained flexibility to avoid the curved orientation in CDK7 and thereby increase the selectivity towards CDK7. Among others this could be achieved by replacement of the flexible thio-methylene linker by a more rigid linker. In my next entry I will show the docking results of more rigid compounds obtained by both <em>in silico </em>docking into the crystal structure of CDKL2 and CDK7.</p>
<p style="text-align: justify;">Please feel free to suggest other ideas for CDKL2 inhibitors with increased selectivity towards CDK7. If you have any questions or comments, please let me know.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/selective-cdkl2-inhibitors-molecular-modeling/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Shorter procedure to access Thieno[2,3-d]pyrimidines</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/shorter-procedure-to-access-thieno23-dpyrimidines/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/shorter-procedure-to-access-thieno23-dpyrimidines/#respond</comments>
		<pubDate>Thu, 13 Dec 2018 02:23:38 +0000</pubDate>
		<dc:creator><![CDATA[Alfredo Picado]]></dc:creator>
				<category><![CDATA[Alfredo Picado]]></category>
		<category><![CDATA[Understudied Kinases]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2244</guid>
		<description><![CDATA[Continuing with the line of delivering a chemical probe for DRAK2/STK17B kinase, we decided to revise the procedure to synthesize analogs with a thieno[2,3-d]pyrimidine core, which are intended to be used as negative control in our DRAK2 inhibition study. We shortened the synthesis of these pyrimidines to 2 steps from the starting material 6-bromo-4-chlorothieno[2,3-d]pyrimidine 1 <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/shorter-procedure-to-access-thieno23-dpyrimidines/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Continuing with the line of delivering a chemical probe for DRAK2/STK17B kinase, we decided to revise the procedure to synthesize analogs with a thieno[2,3-d]pyrimidine core, which are intended to be used as negative control in our DRAK2 inhibition study.</p>
<p>We shortened the synthesis of these pyrimidines to 2 steps from the starting material 6-bromo-4-chlorothieno[2,3-d]pyrimidine <strong>1</strong> and mercaptoacetic acid <strong>2</strong>, Figure 1.  Here, we eliminated a previous step involving the saponification of an ester moiety to get to the carboxylic acid <strong>3</strong>, which is obtained in quantitative yield and easily purified by rinsing with water.</p>
<p>Using this shorter route, we prepared a few analogs <strong>AP-90, AP-94, AP-262-264</strong>, which were recently screened as DRAK2 inhibitors, showing all of them cell potency &gt;10,000 nM; in other words, they are inactive against DRAK2 and because of their structural similarity with the active compounds in this series, they can be used as negative control for future experiments and probe development.</p>
<div id="attachment_2246" style="width: 974px" class="wp-caption alignnone"><img class="size-full wp-image-2246" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/Screen-Shot-2018-12-12-at-9.07.31-PM.png" alt="" width="964" height="964" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/Screen-Shot-2018-12-12-at-9.07.31-PM.png 964w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/Screen-Shot-2018-12-12-at-9.07.31-PM-150x150.png 150w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/Screen-Shot-2018-12-12-at-9.07.31-PM-300x300.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/Screen-Shot-2018-12-12-at-9.07.31-PM-768x768.png 768w" sizes="(max-width: 964px) 100vw, 964px" /><p class="wp-caption-text">Figure 1. Synthesis of thieno[2,3-d]pyrimidine derivatives.</p></div>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/shorter-procedure-to-access-thieno23-dpyrimidines/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Screening ACVR1 inhibitors on mutant and non-mutant ACVR1 DIPG cells &#8211; effectiveness may vary</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/screening-acvr1-inhibitors-on-mutant-and-wild-type-acvr1-dipg-cells-effectiveness-may-vary/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/screening-acvr1-inhibitors-on-mutant-and-wild-type-acvr1-dipg-cells-effectiveness-may-vary/#respond</comments>
		<pubDate>Fri, 07 Dec 2018 17:08:35 +0000</pubDate>
		<dc:creator><![CDATA[Elizabeth Brown]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Brain Tumours]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Liz Brown]]></category>
		<category><![CDATA[Understudied Kinases]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2226</guid>
		<description><![CDATA[Hi there! The last month of my life was taken over by making sure my PhD first year report was beautifully polished, but I have returned with results of a small compound screen: These are all compounds that Jong Fu has already tested with his assays so we know they effectively inhibit ACVR1, but I <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/screening-acvr1-inhibitors-on-mutant-and-wild-type-acvr1-dipg-cells-effectiveness-may-vary/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Hi there! The last month of my life was taken over by making sure my PhD first year report was beautifully polished, but I have returned with results of a small compound screen:</p>
<p>These are all compounds that <a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/jong-fu-wong/">Jong Fu</a> has already tested with his assays so we know they effectively inhibit ACVR1, but I thought it would be a good idea to try them out on my DIPG cells and see how well they can kill them. I included a few different variables in this screen to test what could make a difference to the outcome:</p>
<ul>
<li>I tested cells grown in 2D and 3D because this has been noted to change the outcome of compound screens in other systems</li>
<li>I tested the compounds alone, or in addition to radiation in order to test whether the compounds were effective in addition to current DIPG treatments.</li>
</ul>
<p>I tested how effective each compound is by growing cells for 7 days in the presence of the compound. At the end I measured how alive or healthy the cells were (i.e. their viability) by using a pre-developed kit that measures the amount of ATP inside the cell by using an enzyme that can produce light when ATP is present. ATP is measured because this is sometimes called the &#8216;energy currency of the cell&#8217; so living cells have to have ATP inside them, whereas dead cells won&#8217;t and won&#8217;t be counted.</p>
<p>This screen produced a lot of data at once which is best visualised as heatmaps like these:</p>
<div id="attachment_2227" style="width: 790px" class="wp-caption aligncenter"><a href="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/ALK2i-figure-heatmap.png"><img class="size-large wp-image-2227" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/ALK2i-figure-heatmap-860x1024.png" alt="" width="780" height="929" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/ALK2i-figure-heatmap-860x1024.png 860w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/ALK2i-figure-heatmap-252x300.png 252w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/ALK2i-figure-heatmap-768x914.png 768w" sizes="(max-width: 780px) 100vw, 780px" /></a><p class="wp-caption-text"><strong>ACVR1 inhibitor M4K2009 is best at reducing mutant ACVR1 DIPG cell viability, but is not effective in ACVR1 wild-type cells</strong><br />Heatmaps showing the viability (relative to an untreated control) of a 2D culture of mutant ACVR1 DIPG cells (A) or wild-type ACVR1 DIPG cells (B) following 7 days in the presence of 1μM of the indicated ALK2 inhibitors. The colour of each box indicates how viable the cells were at the end (where blue indicates an improvement in viability and red indicates a drop in viability). Each column indicates an individual (technical repeats, n1-3) grouped into those done at the same time (biological repeats 1-3).</p></div>
<p>From that heatmap you can see that M4K2009 is the best at killing ACVR1 mutant cells (the top heatmap), but it isn&#8217;t as effective in cells where the ACVR1 gene is not mutated (&#8220;wild-type&#8221; cells). In those cells instead M4K2096 is the most effective compound (but still not to the same extent as M4K2009).</p>
<p>But this is only the data for cells grown in 2 dimensions without a radiation treatment. If you want to see the full dataset, you can read my <a href="https://zenodo.org/record/2000673#.XAqo_FX7SUk">Zenodo post</a>!</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/screening-acvr1-inhibitors-on-mutant-and-wild-type-acvr1-dipg-cells-effectiveness-may-vary/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
	</channel>
</rss>
