<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>Project &#8211; openlabnotebooks.org</title>
	<atom:link href="https://thesgc.github.io/static-openlabnotebooks/category/project/feed/" rel="self" type="application/rss+xml" />
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<description>A growing team of groundbreaking scientists around the world are now sharing their lab notebooks online</description>
	<lastBuildDate>Thu, 14 May 2020 17:24:00 +0000</lastBuildDate>
	<language>en-GB</language>
	<sy:updatePeriod>
	hourly	</sy:updatePeriod>
	<sy:updateFrequency>
	1	</sy:updateFrequency>
	<generator>https://wordpress.org/?v=5.4.1</generator>

<image>
	<url>https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/cropped-lab-jar02-32x32.png</url>
	<title>Project &#8211; openlabnotebooks.org</title>
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<width>32</width>
	<height>32</height>
</image> 
	<item>
		<title>PRDM9 cellular assay</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/prdm9-cellular-assay/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/prdm9-cellular-assay/#respond</comments>
		
		<dc:creator><![CDATA[Magdalena Szewczyk]]></dc:creator>
		<pubDate>Thu, 14 May 2020 17:24:00 +0000</pubDate>
				<category><![CDATA[Cell assay reports]]></category>
		<category><![CDATA[Magdalena Szewczyk]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=4697</guid>

					<description><![CDATA[Background PRDM9 (PR domain-containing protein 9) contains a subclass of the SET (PR/SET) domain that trimethylates histone 3 on lysine 4 and 36 (PIMID:24095733). It is expressed mainly in germ cells playing an important role in meiotic recombination (PIMID:27362481). Its aberrant expression has also been associated with genomic instability in cancer (PIMID:30341163). Assay Validation Consistent <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/prdm9-cellular-assay/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p><strong>Background</strong></p>
<p>PRDM9 (PR domain-containing protein 9) contains a subclass of the SET (PR/SET) domain that trimethylates histone 3 on lysine 4 and 36 (PIMID:24095733). It is expressed mainly in germ cells playing an important role in meiotic recombination (PIMID:27362481). Its aberrant expression has also been associated with genomic instability in cancer (PIMID:30341163).</p>
<p><strong>Assay Validation</strong></p>
<p>Consistent with previous findings, the overexpression of FLAG-tagged wild type PRDM9 but not its catalytic mutant (Y357A) led to an increase in histone 3 lysine 4 trimethylation levels (H3K4me3) of exogenous (GFP-tagged) as well as endogenous histone H3 in cells (Fig.1). Due to high basal H3K4me3 levels, only a small increase in methylation was observed upon PRDM9 overexpression in endogenous H3, making the system with exogenous histone H3 preferential assay for compounds screening. The assay was additionally validated with PRDM9 selective inhibitor MRK-740 (PIMID:31848333). MRK-740 decreased PRDM9 dependent H3K4 trimethylation of exogenous histone H3 in a dose-dependent manner (PIMID:31848333, Fig.2). The assay z factor equals 0.83.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/PRDM9-2-268x300.png" alt="" width="338" height="378" class="wp-image-4699 alignleft" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/PRDM9-2-268x300.png 268w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/PRDM9-2.png 552w" sizes="(max-width: 338px) 100vw, 338px" /></p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p><strong>Fig.1.</strong><strong> The wild type but not catalytic mutant of PRDM9 methylated exogenous and endogenous histone H3K4.</strong>  HEK293T cells were co-transfected with GFP-tagged histone H3 and empty vector(control), FLAG-tagged wild type, or Y357A catalytic mutant PRDM9 24 h<strong>.</strong> The methylation levels were analyzed in Western Blot. (Shawna Organ)</p>
<p>&nbsp;</p>
<p><strong> </strong></p>
<p><strong> </strong><strong> </strong></p>
<p><strong> </strong><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/PRDM9-1-300x94.png" alt="" width="714" height="224" class="alignnone wp-image-4698" style="font-size: 1.5rem;" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/PRDM9-1-300x94.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/PRDM9-1-768x241.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/PRDM9-1.png 959w" sizes="(max-width: 714px) 100vw, 714px" /></p>
<p><strong>Fig.2.</strong><strong> MRK-740 decreases PRDM9 dependent H3K4 trimethylation in cells</strong>. HEK293T cells were co-transfected with FLAG-tagged PRMT9 (wild type or Y357A mutant) and GFP-tagged histone H3 and treated with inhibitor for 20 h. H3K4me3 levels were determined by Western blot. The graph represents the nonlinear fit of H3K4me3 signal intensities normalized to GFP. The results are mean +/- SEM of 3 replicates. (Magdalena Szewczyk)</p>
<p>Experimental details are posted on <a href="https://zenodo.org/record/3827543#.Xr1-LGhKg2w">Zenodo</a>.</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/prdm9-cellular-assay/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>SMYD2 cellular assay</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/smyd2-cellular-assay/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/smyd2-cellular-assay/#respond</comments>
		
		<dc:creator><![CDATA[Magdalena Szewczyk]]></dc:creator>
		<pubDate>Wed, 13 May 2020 17:42:48 +0000</pubDate>
				<category><![CDATA[Cell assay reports]]></category>
		<category><![CDATA[Magdalena Szewczyk]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=4674</guid>

					<description><![CDATA[Background SMYD2 is protein-lysine N-methyltransferase that methylates both histones and non-histone proteins, including p53. It monomethylates K370 of p53 leading to decreased DNA-binding and consequently the lower transcriptional regulation activity of p53 (PIMID:17108971). Assay validation Based on the findings (PIMID:17108971), we developed SMYD2 cellular assay in which exogenous FLAG-tagged p53 is methylated by exogenous V5-tagged <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/smyd2-cellular-assay/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p><strong>Background</strong></p>
<p>SMYD2 is protein-lysine N-methyltransferase that methylates both histones and non-histone proteins, including p53. It monomethylates K370 of p53 leading to decreased DNA-binding and consequently the lower transcriptional regulation activity of p53 (PIMID:17108971).</p>
<p><strong>Assay validation </strong></p>
<p>Based on the findings (PIMID:17108971), we developed SMYD2 cellular assay in which exogenous FLAG-tagged p53 is methylated by exogenous V5-tagged SMYD2. The assay was validated with two SMYD2 selective chemical probes: BAY-598 and PFI-5 (PIMID: 27075367, 31415173). Both compounds decreased monomethylation of exogenous p53-K370 in cells overexpressed with V5-SMYD2 (Fig.1,2). The z factor for the assay equals 0.87 (n=3).</p>
<p>&nbsp;</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/smyd2-1-300x187.png" alt="" width="529" height="330" class="wp-image-4678 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/smyd2-1-300x187.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/smyd2-1.png 654w" sizes="(max-width: 529px) 100vw, 529px" /></p>
<p><strong>Fig.1. BAY-598 decreases SMYD2 dependent p53K370 monomethylation in cells.</strong> HEK293T cells were co-transfected with V5-tagged SMYD2 and FLAG-tagged p53 and treated with 1 µM BAY-598 for 20 h. The monomethylation levels of p53K370 were analyzed in Western blot.</p>
<p><strong> <img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/smyd2-2-300x74.png" alt="" width="693" height="171" class="alignnone wp-image-4677" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/smyd2-2-300x74.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/smyd2-2.png 635w" sizes="(max-width: 693px) 100vw, 693px" /></strong></p>
<p><strong>Fig.2. PFI-5 decreases SMYD2 dependent p53K370 monomethylation in cells in a dose-dependent manner.</strong> MCF7 cells were co-transfected with V5-tagged SMYD2 and FLAG-tagged p53 and treated with PFI-5 for 20 h. The monomethylation levels of p53K370 were analyzed in Western Blot. The graph represents the nonlinear fit of p53K370me signal intensities normalized to p53(FLAG). The results are mean +/- SEM of 3 replicates.</p>
<p>Go to <a href="https://zenodo.org/record/3825216#.XrwxOmhKg2w">Zenodo</a> for experimental details.</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/smyd2-cellular-assay/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>GID4 (human) cellular assay</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/gid4-human-cellular-assay/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/gid4-human-cellular-assay/#respond</comments>
		
		<dc:creator><![CDATA[Magdalena Szewczyk]]></dc:creator>
		<pubDate>Wed, 06 May 2020 17:21:48 +0000</pubDate>
				<category><![CDATA[Cell assay reports]]></category>
		<category><![CDATA[E3 ligases]]></category>
		<category><![CDATA[Magdalena Szewczyk]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=4651</guid>

					<description><![CDATA[Background GID4 is the substrate-recognition subunit of the CTLH E3 ubiquitin-protein ligase complex which has been shown to possess E3-ligase enzymatic activity (PIMID:31885576). GID4 was shown to bind proteins and peptides with a Pro/N-degron consisting of an unmodified N-terminal Pro and has the highest affinity for the peptide Pro-Gly-Leu-Trp (PIMID:29632410). In yeast Gid4 specifically targets <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/gid4-human-cellular-assay/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p><strong>Background</strong></p>
<p>GID4 is the substrate-recognition subunit of the CTLH E3 ubiquitin-protein ligase complex which has been shown to possess E3-ligase enzymatic activity (PIMID:<a href="https://www.ncbi.nlm.nih.gov/pubmed/31885576">31885576</a>). GID4 was shown to bind proteins and peptides with a Pro/N-degron consisting of an unmodified N-terminal Pro and has the highest affinity for the peptide Pro-Gly-Leu-Trp (PIMID:29632410). In yeast Gid4 specifically targets the gluconeogenic enzymes (Fbp1, Icl1, and Mdh2) for proteasome-mediated degradation (PIMID: 31863529), however, substrate specificity for human GID4 has not been fully understood.</p>
<p><strong>Assay validation</strong></p>
<p>Cell-based thermal shift assays (CETSA) provide a native environment for measuring compounds’ target engagement and cell permeability. Standard western blot-based CETSA protocols are laborious, require high-quality antibodies to the target, and are generally not amenable to high-throughput compound screening. Therefore, we have adopted the HiBiT technology-based CETSA, which measures the thermal stability of a NanoLuc Luciferase fusion protein, to measure GID4 cellular target engagement in a 384-well plate format (Fig.1). The assay was validated with two compounds: 20964 and 12966, which showed significant dose-dependent stabilization of GID4 when tested <em>in vitro</em> by DSF. Using both compounds we observed reproducible stabilization of GID4 (Fig.2A) in cells. Since compound 20964 gave better T<sub>m</sub>-shift (∆T<sub>m</sub>), we tested if the stabilization is dose-dependent in a temperature range which gave the biggest difference between treated and untreated samples.  Compound 20964 showed the dose-dependent stabilization of GID4 (Fig.2B). The effect was significant at 1 µM concentration (Fig.2C). These results validate the utility of the HiBiT based CETSA assay to measure cellular target engagement of GID4.</p>
<p><strong> <img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/gid4a-300x144.png" alt="" width="583" height="280" class="wp-image-4652 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/gid4a-300x144.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/gid4a-768x369.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/gid4a.png 788w" sizes="(max-width: 583px) 100vw, 583px" /></strong></p>
<p><strong>Fig.1. HiBIT protein tagging system</strong>. HiBiT is an 11 amino acid peptide tag that can be attached to the N or C terminus of the protein. The tag is detected with a reagent containing an inactive luciferase subunit, Large Bit (LgBiT), which rapidly binds to HiBiT to produce a functional NanoLuc luciferase enzyme that relies on the substrate furimazine to produce high intensity, glow-type luminescence.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/gid4new-300x169.png" alt="" width="646" height="364" class="wp-image-4653 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/gid4new-300x169.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/gid4new-1024x576.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/gid4new-768x432.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/gid4new.png 1280w" sizes="(max-width: 646px) 100vw, 646px" /></p>
<p><strong>Fig.2. GID4 HiBiT CETSA target engagement assay validation. </strong>HeLa cells were transfected with N-terminally HiBiT-tagged GID4. Cells were treated with compounds and DMSO control for an hour before heating. After heating, cells were lysed and LgBiT and Luciferase substrate were added and the signal was measured. <strong>(A) </strong>Effect of 20964 (10 µM) and 12966 (30 µM) on the thermal stability of HiBiT-tagged GID4 relative to DMSO control. <strong>(B)</strong> 20964 showed the dose-dependent stabilization of GID4 in an indicated temperature range. <strong>(C)</strong> 20964 significantly stabilized GID4 at 1 µM in an indicated temperature range. The results are mean ± SEM of 4 technical replicates.</p>
<p>Experimental details can be found on <a href="https://zenodo.org/record/3803314#.XrLxcahKg2w">Zenodo</a>.</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/gid4-human-cellular-assay/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>PRMT3 cellular assay</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/prmt3-cellular-assay/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/prmt3-cellular-assay/#respond</comments>
		
		<dc:creator><![CDATA[Magdalena Szewczyk]]></dc:creator>
		<pubDate>Tue, 05 May 2020 18:31:31 +0000</pubDate>
				<category><![CDATA[Cell assay reports]]></category>
		<category><![CDATA[Magdalena Szewczyk]]></category>
		<category><![CDATA[PRMTs]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=4644</guid>

					<description><![CDATA[Background PRMT3 is type I protein arginine methyltransferase which mono- and asymmetrically dimethylates proteins, mainly in glycine and arginine-rich motifs (Fig.1, PIMID: 15175657). Fig.1. Mammalian PRMTs. There are 9 members and 3 types of the PRMT family. Type I, II and III PRMTs catalyze the formation of monomethyl arginines (Rme1). Type I PRMTs catalyze asymmetric arginine <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/prmt3-cellular-assay/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p><strong>Background</strong></p>
<p>PRMT3 is type I protein arginine methyltransferase which mono- and asymmetrically dimethylates proteins, mainly in glycine and arginine-rich motifs (Fig.1, PIMID: 15175657).</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1-300x119.png" alt="" width="630" height="250" class="wp-image-4626 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1-300x119.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1-1024x405.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1-768x304.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1.png 1214w" sizes="(max-width: 630px) 100vw, 630px" /></p>
<p><strong>Fig.1. </strong><strong>Mammalian PRMTs. </strong>There are 9 members and 3 types of the PRMT family. Type I, II and III PRMTs catalyze the formation of monomethyl arginines (Rme1). Type I PRMTs catalyze asymmetric arginine methylation (Rme2a), while type II PRMTs form symmetric arginine methylation (Rme2s). Type III PRMT can only monomethylate arginine residues. Modified from (PMID:29378138).</p>
<p><strong>Assay validation</strong></p>
<p>PRMT3 was shown to asymmetrically dimethylate histone H4 arginine 3 (H4R3) <em>in vitro </em>(PMID:22795084). Consistent with <em>in vitro </em>findings overexpression of wild type PRMT3 but not its catalytic mutant (E338Q) led to an increase in H4R3me2a (Fig.2). The assay was further validated with PRMT3 selective inhibitor SGC707, which decreased H4R3me2a levels in cells overexpressed with PRMT3 in a dose-dependant manner (Fig.3). The Z factor for the assay equals 0.59.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p3a-300x160.png" alt="" width="437" height="233" class="wp-image-4646 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p3a-300x160.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p3a-768x409.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p3a.png 846w" sizes="(max-width: 437px) 100vw, 437px" /></p>
<p><strong>Fig.2. The wild type but not catalytic mutant (E338Q) of PRMT3 methylated endogenous H4R3.</strong> HEK293T cells were transfected with empty vector (control), FLAG-tagged PRMT3 wild type (PRMT3wt) or catalytic mutant (PRMT3mut) for 24 h. The methylation levels of H4R3 were analyzed in Western blot.</p>
<p>&nbsp;</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p3-300x86.png" alt="" width="691" height="198" class="alignnone wp-image-4645" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p3-300x86.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p3-1024x293.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p3-768x220.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p3.png 1280w" sizes="(max-width: 691px) 100vw, 691px" /></p>
<p><strong>Fig.3.</strong> <strong>SGC707 decreases PRMT3 dependent H4R3 asymmetric dimethylation in cells</strong>. HEK293T cells were transfected with FLAG-tagged PRMT3 wild type (PRMT3wt) or catalytic (E338Q) mutant (PRMT3mut) and treated with PRMT3 selective inhibitor, SGC7070, for 20 h. H4R3me2a levels were determined by Western blot. The graph represents the nonlinear fit of H4R3me2a signal intensities normalized to total histone H4. The results are mean +/- SEM of 3 replicates.</p>
<p>Please visit <a href="https://zenodo.org/record/3787392#.XrGwgahKg2w">Zenodo</a>, for experimental details.</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/prmt3-cellular-assay/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>PRMT6 cellular assay</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/prmt6-cellular-assay/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/prmt6-cellular-assay/#respond</comments>
		
		<dc:creator><![CDATA[Magdalena Szewczyk]]></dc:creator>
		<pubDate>Tue, 05 May 2020 16:58:34 +0000</pubDate>
				<category><![CDATA[Cell assay reports]]></category>
		<category><![CDATA[Magdalena Szewczyk]]></category>
		<category><![CDATA[PRMTs]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=4638</guid>

					<description><![CDATA[Background PRMT6 is type I protein arginine methyltransferase which mono- and asymmetrically dimethylates histone and nonhistone proteins, mainly in glycine and arginine rich motifs (Fig.1). PRMT6 shows predominant nuclear localization and is implicated in the regulation of nuclear processes such as DNA repair and gene expression (PIMID: 11724789, 18079182).  Fig.1. Mammalian PRMTs. There are 9 <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/prmt6-cellular-assay/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p><strong>Background</strong></p>
<p>PRMT6 is type I protein arginine methyltransferase which mono- and asymmetrically dimethylates histone and nonhistone proteins, mainly in glycine and arginine rich motifs (Fig.1). PRMT6 shows predominant nuclear localization and is implicated in the regulation of nuclear processes such as DNA repair and gene expression (PIMID: 11724789, 18079182).</p>
<p><strong> <img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1-300x119.png" alt="" width="550" height="218" class="wp-image-4626 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1-300x119.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1-1024x405.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1-768x304.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1.png 1214w" sizes="(max-width: 550px) 100vw, 550px" /></strong></p>
<p><strong> </strong><strong>Fig.1. </strong><strong>Mammalian PRMTs. </strong>There are 9 members and 3 types of the PRMT family. Type I, II and III PRMTs catalyze the formation of monomethyl arginines (Rme1). Type I PRMTs catalyze asymmetric arginine methylation (Rme2a), while type II PRMTs form symmetric arginine methylation (Rme2s). Type III PRMT can only monomethylate arginine residues. Modified from (PMID:29378138).</p>
<p><strong>Assay validation</strong></p>
<p>It has been reported that PRMT6 is the main contributor to histone H3 arginine 2 asymmetric dimethylation (H3R2me2a) in cells (PIMID: 17898714). We found that knocking down of PRMT6 for 3 days is not sufficient to observe a decent decrease in H3R2me2a levels, however, overexpression of wild type PRMT6 but not its catalytic mutant (V86K/D88A) led to increase in H3R2me2a levels (Fig.2). Overexpression of PRMT6 also increased H3 arginine 8 asymmetric dimethylation (H3R8me2a) and H4 arginine 3 asymmetric dimethylation (H4R3me2a) levels (Fig.2). The assay was further validated with PRMT Type I chemical probe (MS023). MS023 decreased PRMT6 dependent H3R2 asymmetric dimethylation in cells overexpressed with PRMT6 in a dose-dependent manner (Fig.3). The Z factor for the assay equals 0.63.</p>
<p><strong> </strong><strong> <img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/prmt6-209x300.png" alt="" width="298" height="428" class="wp-image-4639 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/prmt6-209x300.png 209w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/prmt6.png 411w" sizes="(max-width: 298px) 100vw, 298px" /></strong></p>
<p><strong>Fig.2. The wild type but not catalytic mutant of PRMT6 asymmetrically dimethylated endogenous H3R2, H3R8, and H4R3.</strong>  HEK293T cells were transfected with FLAG-tagged PRMT6 wild type or V86K/D88A catalytic mutant (PRMT6mut) or empty vector (control) for 24 h. The methylation levels were analyzed in Western Blot.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/prmt6a-300x75.png" alt="" width="724" height="181" class="alignnone wp-image-4640" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/prmt6a-300x75.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/prmt6a-1024x257.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/prmt6a-768x193.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/prmt6a.png 1260w" sizes="(max-width: 724px) 100vw, 724px" /></p>
<p><strong>Fig.3.</strong> <strong>MS023 decreases PRMT6 dependent H3R2 asymmetric dimethylation in cells</strong>. HEK293T cells were transfected with FLAG-tagged PRMT6 wild type or V86K/D88A catalytic mutant (PRMT6mut) and treated with inhibitor for 20 h. H3R2me2a levels were determined by Western blot. The graph represents the nonlinear fit of H3R2me2a signal intensities normalized to total histone H3. The results are mean +/- SEM of 3 replicates.</p>
<p>Experimental details are posted on <a href="https://zenodo.org/record/3787176#.XrGbAqhKg2w">Zenodo</a>.</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/prmt6-cellular-assay/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>PRMT4 (CARM1) cellular assay</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/prmt4-carm1-cellular-assay/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/prmt4-carm1-cellular-assay/#respond</comments>
		
		<dc:creator><![CDATA[Magdalena Szewczyk]]></dc:creator>
		<pubDate>Mon, 04 May 2020 16:08:06 +0000</pubDate>
				<category><![CDATA[Cell assay reports]]></category>
		<category><![CDATA[E3 ligases]]></category>
		<category><![CDATA[Magdalena Szewczyk]]></category>
		<category><![CDATA[PRMTs]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=4630</guid>

					<description><![CDATA[Background PRMT4 (CARM1) is type I protein arginine methyltransferase which mono- or asymmetrically dimethylates proteins (Fig.1) within proline-methionine rich motives, that are involved in transcription, splicing and RNA processing such as H3R17, EP300, PABP1, CA150, BAF155, MED12 (PIMID:19150423). Fig.1. Mammalian PRMTs. There are 9 members and 3 types of the PRMT family. Type I, II <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/prmt4-carm1-cellular-assay/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p><strong>Background</strong></p>
<p>PRMT4 (CARM1) is type I protein arginine methyltransferase which mono- or asymmetrically dimethylates proteins (Fig.1) within proline-methionine rich motives, that are involved in transcription, splicing and RNA processing such as H3R17, EP300, PABP1, CA150, BAF155, MED12 (PIMID:19150423).</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1-300x119.png" alt="" width="527" height="209" class="wp-image-4626 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1-300x119.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1-1024x405.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1-768x304.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1.png 1214w" sizes="(max-width: 527px) 100vw, 527px" /></p>
<p><strong>Fig.1. </strong><strong>Mammalian PRMTs. </strong>There are 9 members and 3 types of the PRMT family. Type I, II and III PRMTs catalyze the formation of monomethyl arginines (Rme1). Type I PRMTs catalyze asymmetric arginine methylation (Rme2a), while type II PRMTs form symmetric arginine methylation (Rme2s). Type III PRMT can only monomethylate arginine residues. Modified from (PMID:29378138).</p>
<p><strong>Assay validation</strong></p>
<p>It has been reported that the RNA polymerase II mediator complex subunit 12 (MED12) and BAF155, are PRMT4 substrates (PIMID: 26601288, 25927994). Consistently with previous findings, PRMT4 knock-down decreased asymmetric dimethylation of MED12 and BAF155 (Fig.2). The assay was validated with PRMT4 selective chemical probe TP-064 (PIMID: 29719619). TP-064 decreased PRMT4 asymmetric dimethylation of MED12 (Z factor = 0.6) and BAF155 (Z factor = 0.76) in a dose-dependent manner (Fig.3).</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p42-300x245.png" alt="" width="417" height="340" class="alignnone wp-image-4631" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p42-300x245.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p42-768x627.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p42.png 846w" sizes="(max-width: 417px) 100vw, 417px" /></p>
<p><strong>Fig.2. PRMT4 knock-down resulted in a decrease of MED12 and BAF155 asymmetric dimethylation (Rme2a) levels. </strong>HEK293T cells were treated with siRNA for 3 days and 6 days. The levels of methylation were analyzed in Western Blot.<strong> </strong></p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p4-3-300x120.png" alt="" width="623" height="249" class="alignnone wp-image-4632" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p4-3-300x120.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p4-3-1024x410.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p4-3-768x308.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p4-3.png 1261w" sizes="(max-width: 623px) 100vw, 623px" /></p>
<p><strong>Fig. 3.</strong> <strong>TP-064 </strong><strong>decreases PRMT4 dependent MED12 and BAF155 asymmetric dimethylation (Rme2a) levels.</strong> HEK293T cells were treated with PRMT4 selective inhibitor (TP-064) at indicated concentrations for 48 h and BAF155-Rme2a (A) and MED12-Rme2a (B) levels were determined by Western blot. The graphs represent nonlinear fits of methyl signal intensities normalized to total protein levels. The results are mean +/- SEM of 3 replicates.</p>
<p>Please go to <a href="https://zenodo.org/record/3785034#.XrA9AKhKg2w">Zenodo</a> for experimental details.</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/prmt4-carm1-cellular-assay/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>PRMT5 cellular assay</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/prmt5-cellular-assay/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/prmt5-cellular-assay/#respond</comments>
		
		<dc:creator><![CDATA[Magdalena Szewczyk]]></dc:creator>
		<pubDate>Mon, 27 Apr 2020 15:15:00 +0000</pubDate>
				<category><![CDATA[Cell assay reports]]></category>
		<category><![CDATA[Magdalena Szewczyk]]></category>
		<category><![CDATA[PRMTs]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=4602</guid>

					<description><![CDATA[Background PRMT5 in complex with the WD-repeat protein MEP50 is the major type II protein arginine methyltransferase responsible for mono- and symmetric dimethylation of arginine in a great variety of proteins (Fig.1). It plays a critical role in diverse cellular processes such as genome organization, transcription, translation, differentiation, cell cycle, DNA repair,  RNA processing, and <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/prmt5-cellular-assay/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p><strong>Background</strong></p>
<p>PRMT5 in complex with the WD-repeat protein MEP50 is the major type II protein arginine methyltransferase responsible for mono- and symmetric dimethylation of arginine in a great variety of proteins (Fig.1). It plays a critical role in diverse cellular processes such as genome organization, transcription, translation, differentiation, cell cycle, DNA repair,  RNA processing, and metabolism (PMID:25662273). PRMT5 expression levels and methyltransferase activity were found to be dysregulated in a variety of cancers (PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25662273">25662273</a>).</p>
<p><strong> <img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1-300x119.png" alt="" width="608" height="241" class="wp-image-4626 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1-300x119.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1-1024x405.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1-768x304.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1.png 1214w" sizes="(max-width: 608px) 100vw, 608px" /><br />
</strong></p>
<p><strong>Fig.1. </strong><strong>Mammalian PRMTs. </strong>There are 9 members and 3 types of the PRMT family. Type I, II and III PRMTs catalyze the formation of monomethyl arginines (Rme1). Type I PRMTs catalyze asymmetric arginine methylation (Rme2a), while type II PRMTs form symmetric arginine methylation (Rme2s). Type III PRMT can only monomethylate arginine residues. Modified from (PMID:29378138).</p>
<p><strong> </strong></p>
<p><strong>Assay validation</strong></p>
<p>It has been reported previously, that the various SMN complex proteins, including SmBB’ are PRMT5 substrates (PMID:19520849). We found that the knocking down of PRMT5, but not PRMT1, 3, 4, 6 and  7 resulted in decreased SmBB’ symmetric dimethylation levels (Fig.2). PRMT9, another type II PRMT, is also not affecting SmBB’ Rme2s levels (PIMID: 25737013). The assay was further validated by treatment with selective PRMT5 chemical probe, GSK591. GSK591 decreased SmBB’-Rme2s levels in a dose-dependent manner (Fig.3). The assay was also used to validate the cellular activity of another PRMT5 selective chemical probe LLY-283 (PMID:30034588). Similar results were obtained with different cell lines (data not shown).</p>
<p>&nbsp;</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/p5-2-300x225.png" alt="" width="712" height="534" class="wp-image-4604 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/p5-2-300x225.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/p5-2-768x576.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/p5-2-350x263.png 350w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/p5-2-780x586.png 780w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/p5-2-250x188.png 250w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/p5-2.png 945w" sizes="(max-width: 712px) 100vw, 712px" /></p>
<p><strong>Fig.2. PRMT5 knock-down results in decrease in SmBB’-Rme2s levels. </strong>MCF7 cells were treated with siRNA targeted for different PRMTs and control siRNA for 6 (<strong>A</strong>) and 3 (<strong>B</strong>) days. <strong>A.</strong> Western blot analysis with an antibody recognizing symmetric arginine methylation (Rme2s) of various proteins. PRMTs expression after knock-down (KD) and β-actin controls are shown. Note that the loss of PTMT1 leads to decrease of Rme2a modification and the shared substrates become targets for PRMT5 leading to increase in Rme2s levels.<strong>B.</strong> Western blot analysis of the SmBB’-Rme2s levels after PRMT5 KD. <strong>C.</strong> Quantification of signal intensities of SmBB’-Rme2s normalized to total SmBB’ (n=1).</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/p5-3-300x83.png" alt="" width="647" height="179" class="wp-image-4603 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/p5-3-300x83.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/p5-3-1024x282.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/p5-3-768x212.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/p5-3.png 1280w" sizes="(max-width: 647px) 100vw, 647px" /></p>
<p><strong>Fig.3.</strong> <strong>PRMT5 chemical probe, GSK591, decreases SmBB’ symmetric dimethylation in a dose-dependent manner.</strong> MCF7 cells were treated with inhibitor at indicated concentrations for 48 h and SmBB’-Rme2s and total SmBB’ levels were determined by Western blot. The graph represents nonlinear fits of SmBB’-Rme2s signal intensities normalized to total SmBB’. The results are mean +/- SEM of 3 replicates. The Z factor for the assay equals 0.67 (n=3).</p>
<p>Please visit <a href="https://zenodo.org/record/3774168#.Xqhm52hKg2w">Zenodo</a> for experimental details.</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/prmt5-cellular-assay/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>MYST1 (KAT8, MOF) and histone H4 cellular assay</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/myst1-kat8-mof-and-histone-h4-cellular-assay/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/myst1-kat8-mof-and-histone-h4-cellular-assay/#respond</comments>
		
		<dc:creator><![CDATA[Magdalena Szewczyk]]></dc:creator>
		<pubDate>Thu, 23 Apr 2020 17:28:13 +0000</pubDate>
				<category><![CDATA[Cell assay reports]]></category>
		<category><![CDATA[Chromatin factors]]></category>
		<category><![CDATA[Magdalena Szewczyk]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=4584</guid>

					<description><![CDATA[Background The mammalian MYST1 is a member of the MYST family of histone acetyltransferases and is the primary enzyme that catalyzes the acetylation of histone H4 on lysine 16 (H4K16). Acetylation of K16 is a prevalent mark associated with chromatin decondensation. MYST1 has been shown to play an essential role in embryonic formation and development, <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/myst1-kat8-mof-and-histone-h4-cellular-assay/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p><strong>Background</strong></p>
<p>The mammalian MYST1 is a member of the MYST family of histone acetyltransferases and is the primary enzyme that catalyzes the acetylation of histone H4 on lysine 16 (H4K16). Acetylation of K16 is a prevalent mark associated with chromatin decondensation. MYST1 has been shown to play an essential role in embryonic formation and development, chromatin assembly, transcription activation, cellular apoptosis, double‐strand break repair, cellular stress response, and tumorigenesis (PMID:24452550, 26387537, 17967868, 24452485).</p>
<p><strong>Assay validation</strong></p>
<p>It was shown that MYST1 participates in at least two distinct multiprotein complexes (MSL and NSL) in human cells. Both complexes can acetylate H4K16; however, the NSL complex possesses broader substrate specificity and can also acetylate histone H4 at K5 and K8 (PMID:26784169).  Consistent with previous findings, the overexpression of MYST1 led to an increase in total lysine acetylation levels and acetylation of H4K16 on exogenous (GFP-tagged) as well as endogenous histone H4 (Fig.1 and 2). Due to high basal H4 acetylation levels, only a small increase in acetylation was observed upon MYST1 overexpression in endogenous H4 (Fig.2), making the system with exogenous histone H4 preferential assay for compounds screening. The assay was additionally validated with MYST1 catalytic mutant (K274R), which overexpression in contrast to WT MYST1 did not alter basal exogenous H4 acetylation levels (Fig1).</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/MYST1-1-300x131.png" alt="" width="671" height="293" class="wp-image-4588 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/MYST1-1-300x131.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/MYST1-1-768x336.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/MYST1-1.png 959w" sizes="(max-width: 671px) 100vw, 671px" /></p>
<p><strong>Fig.1. The wild type but not catalytic mutant of MYST1 acetylated exogenous histone H4.</strong>  HEK293T cells were co-transfected with GFP-tagged histone H4 and FLAG-tagged MYST1 WT/MutK276R or empty vector for 24 h. <strong>A.</strong> Western blot analysis of the levels of acetyl-lysine histone H4 (PanAc) and H4K16. <strong>B.</strong> Quantification of signal intensities of acetyl-lysine histone H4 and H4K16 normalized to GFP. The results are mean +/- SEM of 3 replicates. Antibody detecting acetylated lysine residues showed higher background to noise ratio (Z factor = 0.94) than those detecting H4K16Ac levels (Z factor = 0.91).</p>
<p>&nbsp;</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/MYST1-2-300x136.png" alt="" width="657" height="298" class="wp-image-4589 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/MYST1-2-300x136.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/MYST1-2-768x348.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/MYST1-2.png 862w" sizes="(max-width: 657px) 100vw, 657px" /></p>
<p><strong>Fig.2. MYST1 overexpression slightly increased endogenous histone H4 acetylation.</strong>  HEK293T cells were transfected with FLAG-tagged MYST1 or empty vector for 24 h. <strong>A.</strong> Western blot analysis of the levels of acetyl-lysine histone H4 (PanAc) and H4K16. <strong>B.</strong> Quantification of signal intensities of acetyl-lysine histone H4 and H4K16 normalized to histone H4. The results are mean +/- SEM of 3 replicates.</p>
<p>For assay details, please go to <a href="https://zenodo.org/record/3774172#.XqhpKWhKg2w">Zenodo</a>.</p>
<p><strong> </strong></p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/myst1-kat8-mof-and-histone-h4-cellular-assay/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>PRMT1 cellular assay</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/prmt1-cell-assay-report/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/prmt1-cell-assay-report/#respond</comments>
		
		<dc:creator><![CDATA[Magdalena Szewczyk]]></dc:creator>
		<pubDate>Thu, 23 Apr 2020 15:51:40 +0000</pubDate>
				<category><![CDATA[Cell assay reports]]></category>
		<category><![CDATA[Magdalena Szewczyk]]></category>
		<category><![CDATA[PRMTs]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=4569</guid>

					<description><![CDATA[Background PRMT1 is the type I protein arginine methyltransferase (Fig.1) which mono- and asymmetrically dimethylates histone and nonhistone proteins, mainly in glycine and arginine-rich motifs and is involved in various cellular processes such as transcription, RNA processing, DNA repair, cell cycle or signal transduction (PMID:26612103). It is responsible for depositing over 90% of the total <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/prmt1-cell-assay-report/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p><strong>Background</strong></p>
<p>PRMT1 is the type I protein arginine methyltransferase (Fig.1) which mono- and asymmetrically dimethylates histone and nonhistone proteins, mainly in glycine and arginine-rich motifs and is involved in various cellular processes such as transcription, RNA processing, DNA repair, cell cycle or signal transduction (PMID:26612103). It is responsible for depositing over 90% of the total asymmetric arginine dimethyl mark (PMID:23419748). It has been identified, that at least seven distinct PRMT1 isoforms are generated by complex alternative splicing in the 5’ region of its pre-mRNA, however, their exact role has not been fully understood (PMID:11448779, 11387442). Deregulated PRMT1 expression has been observed in a number of tumor types including brain, lung, breast, colon, bladder, and leukemia, which make it a potential target for anti-cancer therapy (PMID:23235912, 28061334).</p>
<p><strong> <img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1-300x119.png" alt="" width="545" height="216" class="wp-image-4626 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1-300x119.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1-1024x405.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1-768x304.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1.png 1214w" sizes="(max-width: 545px) 100vw, 545px" /> </strong></p>
<p><strong>Fig.1. </strong><strong>Mammalian PRMTs. </strong>There are 9 members and 3 types of the PRMT family. Type I, II and III PRMTs catalyze the formation of monomethyl arginines (Rme1). Type I PRMTs catalyze asymmetric arginine methylation (Rme2a), while type II PRMTs form symmetric arginine methylation (Rme2s). Type III PRMT can only monomethylate arginine residues. Modified from (PMID:29378138).</p>
<p><strong>PRMT1 assay validation</strong></p>
<p>It has been reported that PRMT1 is the main contributor to H4R3 (histone 4 arginine 3) asymmetric dimethylation (H4R3me2a) in cells (PMID:17848568). Moreover, with the loss of PRMT1, a large number of substrates become targets for Type II and III PRMTs, because these substrates are presumably no longer blocked by Rme2a modification (PMID: 23419748). We found that knocking down of PRMT1, but not PRMT3, 4, 5, 6, or 7 decreased basal H4R3me2a levels and increased Rme1 and Rme2s levels in MCF7 cells (Fig.2). MCF7 cell line was chosen for the PRMT1 cellular assay due to the high basal levels of H4R3me2a (Fig.3). The assay was further validated with PRMT Type I chemical probe MS023 (PMID:26598975). MS023 decreased the levels of H4R3me2a in a dose-dependent manner (Fig. 4). The Z factor for the assay equals 0.83 (n=3).</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/p1fig2-1-140x300.png" alt="" width="280" height="600" class="wp-image-4576 alignleft" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/p1fig2-1-140x300.png 140w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/p1fig2-1-477x1024.png 477w" sizes="(max-width: 280px) 100vw, 280px" /></p>
<p><strong>Fig.2. PRMT1 knock-down results in decrease in histone 4 arginine 3 asymmetric dimethylation levels and increase in global arginine mono- and dimethylation.</strong> MCF7 cells were treated with siRNA targeted for different PRMTs and control siRNA for 6 days and analysed by western blot for PRMTs expression, histone 4 arginine 3 asymmetric dimethylation (H4R3me2a), symmetric arginine methylation (Rme2s) and monomethylation (Rme1) levels.</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/p1fig3a-300x157.png" alt="" width="655" height="343" class="wp-image-4575 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/p1fig3a-300x157.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/p1fig3a-1024x535.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/p1fig3a-768x402.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/p1fig3a.png 1440w" sizes="(max-width: 655px) 100vw, 655px" /></p>
<p><strong>Fig.3. MCF7 cells have high basal levels of H4R3Rme2a mark. </strong>Western blot analysis of basal H4R3me2a levels in various cell lines. The bars represent H4R3me2a fluorescent intensities normalized to total H4 (n=1).</p>
<p><strong> <img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/p1f4-300x81.png" alt="" width="737" height="199" class="wp-image-4578 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/p1f4-300x81.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/p1f4-1024x278.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/p1f4.png 1440w" sizes="(max-width: 737px) 100vw, 737px" /></strong></p>
<p><strong>Fig.4.</strong> <strong>MS023 decreases PRMT1 dependent H4R3 asymmetric dimethylation in cells</strong>. MCF7 cells were treated with inhibitor at indicated concentrations for 48 h and H4R3me2a levels were determined by western blot. The graph represents nonlinear fits of H4R3me2a signal intensities normalized to total histone H4. The results are mean +/- SEM of 3 replicates.</p>
<p>For assay details, please go to <a href="https://zenodo.org/record/3774163#.Xqhl9mhKg2w">Zenodo</a>.</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/prmt1-cell-assay-report/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>A blind date with an old friend: crystal structures of Cys-S-sulfonic acid GAPDH</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/a-blind-date-with-an-old-friend-crystal-structures-of-cys-s-sulfonic-acid-gapdh/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/a-blind-date-with-an-old-friend-crystal-structures-of-cys-s-sulfonic-acid-gapdh/#respond</comments>
		
		<dc:creator><![CDATA[Pietro Roversi]]></dc:creator>
		<pubDate>Tue, 21 Apr 2020 15:56:58 +0000</pubDate>
				<category><![CDATA[Open Secretion Rescue]]></category>
		<category><![CDATA[Pietro Roversi]]></category>
		<category><![CDATA[Pietro Roversi (Univ. of Leicester)]]></category>
		<category><![CDATA[Rare disease]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=4525</guid>

					<description><![CDATA[Biologists interested in the function or the 3D shape of a certain protein face the task of isolating billions of copies of that protein from cells (a process called &#8220;protein purification&#8221; which does not involve washing or holy water). The aim of protein purification is to start with cells that contain the protein of interest <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/a-blind-date-with-an-old-friend-crystal-structures-of-cys-s-sulfonic-acid-gapdh/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<div id="attachment_4526" style="width: 310px" class="wp-caption alignleft"><img aria-describedby="caption-attachment-4526" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/Wheel-300x180.jpg" alt="The familiar, improved" class="size-medium wp-image-4526" width="300" height="180" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/Wheel-300x180.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/Wheel.jpg 700w" sizes="(max-width: 300px) 100vw, 300px" /><p id="caption-attachment-4526" class="wp-caption-text">The familiar, rediscovered.</p></div>
<p>Biologists interested in the function or the 3D shape of a certain protein face the task of isolating billions of copies of that protein from cells (a process called &#8220;protein purification&#8221; which does not involve washing or holy water). The aim of protein purification is to start with cells that contain the protein of interest in large amounts and arrive at a sample made of billions of copies of that protein &#8211; and not much else. Typically &#8211; the pure protein ends up in a plastic tube, in water and perhaps some salt or some pH buffer to make sure that the proteins are happy and stable in the water solution. Protein purification has <a href="https://www.biologicscorp.com/blog/overview-of-protein-purification-and-characterization/">an illustrious history,</a> and early pioneers such as <a class="" title="James B. Sumner" href="https://www.nobelprize.org/prizes/chemistry/1946/sumner/facts/">James B. Sumner, </a><a class="" title="John H. Northrop" href="https://www.nobelprize.org/prizes/chemistry/1946/northrop/facts/">John H. Northrop </a>and<a class="" title="Wendell M. Stanley" href="https://www.nobelprize.org/prizes/chemistry/1946/stanley/facts/"> Wendell M. Stanley</a> won the <a href="https://www.nobelprize.org/prizes/chemistry/1946/speedread/">Nobel Prize for preparing pure proteins</a>. Once a sample of protein is pure, depending on the questions surrounding it, it can either be used for biochemical assays, or looked at with electromagnetic radiation of some kind, and the results add to the knowledge about the protein, answering questions such as what does the protein do? how does it do it? what 3D shape does it have? how does it move in time? and so on.</p>
<p>Of course &#8211; however simple or complex a human endeavour may be &#8211; what can go wrong will go wrong &#8211; and protein purification is no exception. Proteins are not visible to the naked eye, nor with an optical microscope, and the starting cellular sample contains tens of thousands of different proteins &#8211; so every so often biologists set out to isolate protein X &#8211; and end up with protein Y (&#8220;a <a href="https://en.wikipedia.org/wiki/Contamination">contaminant</a>&#8220;) in their tube (see for example <a href="https://doi.org/10.1016/j.jmb.2006.11.023">here</a>). This is an unfortunate outcome: protein purification (like most complex human endeavours) is not for free. At what point is the mistake apparent? Wise biologists set out to check their samples early during the process, and in fact the most important skill for successful protein purification is to be able to track the protein one wants along the potentially long and laborious steps with which it is progressively separated from the initial mix. This way, as soon as it transpires that the protein of interest is lost (this indeed can happen &#8211; or perhaps it was not there in the first place?) one goes back to square one without further costly efforts. What is not there (anymore) cannot be isolated.</p>
<div id="attachment_4545" style="width: 235px" class="wp-caption alignright"><img aria-describedby="caption-attachment-4545" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/Serendipity-225x300.gif" alt="Serendipity" class="size-medium wp-image-4545" width="225" height="300" /><p id="caption-attachment-4545" class="wp-caption-text">Serendipity</p></div>
<p>However, there&#8217;s always <a href="https://www.jbc.org/content/284/16/10285.full">serendipity</a>. Perhaps protein Y is a novel, interesting one &#8211; and its accidental purification will constitute a good result &#8211; it may answer a different set of questions. Or perhaps one isolates protein X with another protein Y &#8211; and the fact that they stick together reveals that they interact. The proteins that nobody wants to accidentally purify are the ones that we know very well already: ending up with one of those in the pure protein tube is tantamount to discovering that one&#8217;s blind date is an old friend. And of course the more abundant and common a protein is &#8211; the higher the likelihood that it will be the protein Y that the biologist purifies without wanting to. One such very abundant protein is one of the very first enzymes studied, Glyceraldehyde 3-phosphate dehydrogenase aka <a href="https://en.wikipedia.org/wiki/Glyceraldehyde_3-phosphate_dehydrogenase">GAPDH</a>. It is so necessary to cells across all animal kingdoms that it is found in large amounts in bacteria, fungi and animals; its stability makes it a very successful contaminant, and one that has been serendipitously isolated in many a study (see for example <a href="https://doi.org/10.1107/S0907444903041441">[1]</a>, <a href="doi: 10.1074/jbc.M801401200">[2]</a>, <a href="doi: 10.1074/jbc.M113.466870">[3]</a>). The repeated unexpected isolation of GAPDH from protein samples in which biologists did not expect it to be present in the first place has actually revealed that the protein moonlights in multiple roles and very diverse cellular and extracellular environments. Or looking at it from a different angle, a promiscuous protein with a track record for surprise, and one that may well end up turning up on one&#8217;s blind date.</p>
<div id="attachment_4547" style="width: 310px" class="wp-caption alignleft"><img aria-describedby="caption-attachment-4547" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/CSUs-300x220.png" alt="GAPDH Cys-S-sulfonated" class="size-medium wp-image-4547" width="300" height="220" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/CSUs-300x220.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/CSUs.png 661w" sizes="(max-width: 300px) 100vw, 300px" /><p id="caption-attachment-4547" class="wp-caption-text">Structures of human GAPDH carrying S-sulfonic acid on the catalytic Cys152. Eight different conformations of the Cys152 S-sulfonic acid (from as many monomers in the asymmetric unit of the crystal) are superposed and represented in sticks (cyan: C atoms; yellow: S atoms; red: O atoms).</p></div>
<p>We have now joined the group of biologists stumbling on the disappointing discovery of GAPDH in their pure protein tube. In the course of a research effort aimed at the purification of an unrelated enzyme from a fungus, <a href="https://doi.org/10.5281/zenodo.3760776">we purified, crystallised and determined crystal structures of human GAPDH</a>. The protein made it into our sample from the human cells we were using to purify the protein wanted. At the time of this writing, the <a href="https://www.rcsb.org/">Protein Data Bank</a> (PDB) contains more than 142 GAPDH entries, eleven of which are of human GAPDH isoforms &#8211; so these structures are not exactly a big deal. On the other hand, our crystals turned out to contain GAPDH protein that is chemically modified and bears a Cys-S-sulfonic acid on its catalytic Cystein residue. Although GAPDH Cys-S-sulfonation is known to occur and has already been detected and described, one of these novel human GAPDH crystal structures is the highest resolution human GAPDH structure to date &#8211; and the first Cys-S-sulfonated human GAPDH  structure to be determined. Sometimes the old friend one ends up in a blind date with is not as familiar as one thought &#8211; and may still make for an interesting night out after all.</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/a-blind-date-with-an-old-friend-crystal-structures-of-cys-s-sulfonic-acid-gapdh/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
	</channel>
</rss>
