<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>Project &#8211; openlabnotebooks.org</title>
	<atom:link href="https://thesgc.github.io/static-openlabnotebooks/category/project/feed/" rel="self" type="application/rss+xml" />
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<description>A growing team of groundbreaking scientists around the world are now sharing their lab notebooks online</description>
	<lastBuildDate>Fri, 05 Apr 2019 14:33:26 +0000</lastBuildDate>
	<language>en-GB</language>
	<sy:updatePeriod>hourly</sy:updatePeriod>
	<sy:updateFrequency>1</sy:updateFrequency>
	<generator>https://wordpress.org/?v=4.9.10</generator>

<image>
	<url>https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/cropped-lab-jar02-32x32.png</url>
	<title>Project &#8211; openlabnotebooks.org</title>
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<width>32</width>
	<height>32</height>
</image> 
	<item>
		<title>Structure-Based PROTAC Design: Project Overview</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/2714-2/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/2714-2/#respond</comments>
		<pubDate>Fri, 05 Apr 2019 13:56:33 +0000</pubDate>
		<dc:creator><![CDATA[Haresh Ajani]]></dc:creator>
				<category><![CDATA[Haresh Ajani]]></category>
		<category><![CDATA[In silico PROTACs design]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2714</guid>
		<description><![CDATA[Proteolysis Targeting Chimeras (PROTACs) are small molecules that induce the degradation of their target and have shown considerable promise as a novel therapeutic modality.1-3 PROTACs are heterobifunctional compounds containing one chemical moiety that binds to the target protein of interest and a chemical handle that binds to an E3 ubiquitin ligase (Figure 1). Recruitment of <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/2714-2/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p style="text-align: justify;">Proteolysis Targeting Chimeras (PROTACs) are small molecules that induce the degradation of their target and have shown considerable promise as a novel therapeutic modality.<sup>1-3</sup> PROTACs are heterobifunctional compounds containing one chemical moiety that binds to the target protein of interest and a chemical handle that binds to an E3 ubiquitin ligase (Figure 1). Recruitment of the E3 ligase leads to ubiquitination and subsequent degradation of the target by the proteasome. The design of PROTACs remains an empirical process, likely because of an insufficient understanding of the molecular basis for target recruitment and formation of a ternary complex.<sup>4,5 </sup>Not all ligases are compatible with all targets, and linker chemistry and attachment points play a crucial role but are poorly understood.</p>
<p>&nbsp;</p>
<p style="text-align: left;"><img class="aligncenter wp-image-2688 size-full" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/PROTAC_TERNARY_COMPLEX.png" alt="" width="931" height="566" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/PROTAC_TERNARY_COMPLEX.png 931w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/PROTAC_TERNARY_COMPLEX-300x182.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/PROTAC_TERNARY_COMPLEX-768x467.png 768w" sizes="(max-width: 931px) 100vw, 931px" /><strong> .                                </strong></p>
<h4 style="text-align: left;"><strong>                          Figure 1: a PROTAC induces the formation of a ternary complex.</strong></h4>
<p>&nbsp;</p>
<p style="text-align: justify;">The goal of this research project is to test whether computational techniques can be used to rationally guide the design of PROTACs. I will be asking four questions:</p>
<p><img class="alignnone wp-image-2694 size-full" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/PROTAC_NEW-1.png" alt="" width="1581" height="2250" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/PROTAC_NEW-1.png 1581w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/PROTAC_NEW-1-211x300.png 211w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/PROTAC_NEW-1-768x1093.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/PROTAC_NEW-1-720x1024.png 720w" sizes="(max-width: 1581px) 100vw, 1581px" /></p>
<p style="text-align: justify;">Virtual screening and structure-based drug design are quite unreliable techniques for drug-like molecules but are still used because they are generally performing better than random. Considering the structural complexity of the ternary complex central to PROTAC activity, and the chemical complexity of PROTAC molecules, in silico design of PROTACs seems even more challenging. Can rational approaches accelerate PROTAC discovery? I will try to systematically address this important question.</p>
<p>&nbsp;</p>
<p style="text-align: justify;"><strong>Reference:</strong></p>
<ol>
<li>Crews, C. M. et al. Induced protein degradation: an emerging drug discovery paradigm. Nat. Rev. Drug. Discov., <strong>16,</strong> 101-114 (2017).</li>
<li>Scudellari, M. et al. Protein-slaying drugs could be the next blockbuster therapies. Nature., <strong>567,</strong> 298-300 (2019).</li>
<li>Churcher, I. Protac-Induced Protein Degradation in Drug Discovery: Breaking the Rules or Just Making New Ones? J. Med. Chem., <strong>61,</strong> 444-452 (2018).</li>
<li>Fischer, E. S. et al. Plasticity in binding confers selectivity in ligand-induced protein degradation. Nat. Chem. Biol., <strong>14,</strong> 706–714 (2018).</li>
<li>Ciulli, A. et al. Structural basis of PROTAC cooperative recognition for selective protein degradation. Nat. Chem. Biol., <strong>13,</strong> 514–521 (2017).</li>
<li>Williams, C. I. et al. <em>In Silico </em>modeling of PROTAC-medicated ternary complexes: Validation and application. J. Chem. Inf. Model. <strong>Doi:</strong>1021/acs.jcim.8b00872</li>
</ol>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/2714-2/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Determining whether bespoke tracers work with ALK5 in nanoBRET target engagement assay</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/determining-whether-bespoke-tracers-work-with-alk5-in-nanobret-target-engagement-assay/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/determining-whether-bespoke-tracers-work-with-alk5-in-nanobret-target-engagement-assay/#comments</comments>
		<pubDate>Thu, 04 Apr 2019 15:23:54 +0000</pubDate>
		<dc:creator><![CDATA[Jong Fu]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Brain Tumours]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Jong Fu Wong]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2687</guid>
		<description><![CDATA[Special thanks to: David Drewry – Helped with designing the nanoBRET tracers M4K pharma OICR chemist team – Synthesised the M4K1046-linker derivatives Carrow Wells (UNC) – Conjugated the M4K1046 to nanoBRET fluorophore Background: I have always wanted to establish nanoBRET target engagement assay for ALK5. Relative to dual luciferase promoter assay and immunofluorescent staining, nanoBRET <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/determining-whether-bespoke-tracers-work-with-alk5-in-nanobret-target-engagement-assay/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p><strong>Special thanks to:</strong></p>
<p>David Drewry – Helped with designing the nanoBRET tracers</p>
<p>M4K pharma OICR chemist team – Synthesised the M4K1046-linker derivatives</p>
<p>Carrow Wells (UNC) – Conjugated the M4K1046 to nanoBRET fluorophore</p>
<p><strong>Background:</strong></p>
<p>I have always wanted to establish nanoBRET target engagement assay for ALK5. Relative to dual luciferase promoter assay and immunofluorescent staining, nanoBRET is many times faster. With a robust ALK5 nanoBRET assay, I will be able to rapidly screen for cellular off-target activity.</p>
<p>Since none of the commercial nanoBRET tracers worked with ALK5, we attempted to generate our own tracers. We chose to create these bespoke tracers based on M4K1046 because it has cellular IC50 of ~50nM for ALK5. David Drewry has helped to design linkers that will attach fluorophores (nanoBRET energy acceptor) to the solvent-facing end of M4K1046. Based on known structures, additional bulk in this region should not hinder the binding of the compounds to ALK5. Two versions of M4K1046 (with different linker length) were synthesised by the M4K pharma chemist team in OICR. They were subsequently sent to Carrow Wells for conjugation to nanoBRET fluorophore.</p>
<p><strong>Experimental design:</strong></p>
<p>I wanted to determine the binding of these tracers to ALK5-nanoluciferase fusion. Therefore, I incubated increasing concentrations of tracer with HEK293 cells expressing ALK5-nanoluciferase fusion protein. If the tracers can bind nicely to the ATP pocket of ALK5, incubation with increasing concentrations of tracer will result in increasing BRET signal (wavelength = 610 nm). The resulting magnitude of BRET and EC50 estimated from the curve are good indicators to whether these tracers can be used for nanoBRET assay with ALK5.</p>
<p>To identify any background signal, I have included a replicate of the above experiment with 10,000nM of parent M4K1046. These unlabelled compound will saturate ALK5 ATP pockets and prevent the binding of nanoBRET tracers. All signal from this set of experiment is contributed by non-specific background.</p>
<p>To be eliminate doubts of any technical or reagent issues with the nanoBRET experiment itself, I have replicated both experiments above, substituting ALK5 with ALK2.</p>
<p><strong>Results:</strong></p>
<div id="attachment_2691" style="width: 610px" class="wp-caption alignnone"><img class="size-full wp-image-2691" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Fig-1.png" alt="" width="600" height="362" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Fig-1.png 600w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Fig-1-300x181.png 300w" sizes="(max-width: 600px) 100vw, 600px" /><p class="wp-caption-text">EGFP signal in transfected cells. HEK293 were transfected efficiently. Cells can be harvested for nanoBRET assay.</p></div>
<div id="attachment_2692" style="width: 579px" class="wp-caption alignnone"><img class="size-full wp-image-2692" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Fig-2.png" alt="" width="569" height="293" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Fig-2.png 569w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Fig-2-300x154.png 300w" sizes="(max-width: 569px) 100vw, 569px" /><p class="wp-caption-text">EC50 curves of the first bespoke ALK5 tracer (UNC-CIW-001-011) with ALK2-nanoluc and ALK5-nanoluc. The chemical structures of parent M4K1046 and the bespoke tracer are shown on the side of the graphs. This tracer cannot bind to ALK5.</p></div>
<div id="attachment_2693" style="width: 580px" class="wp-caption alignnone"><img class=" wp-image-2693" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Fig-3.png" alt="" width="570" height="301" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Fig-3.png 555w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Fig-3-300x158.png 300w" sizes="(max-width: 570px) 100vw, 570px" /><p class="wp-caption-text">EC50 curves of the second bespoke ALK5 tracer (UNC-CIW-001-012) with ALK2-nanoluc and ALK5-nanoluc. The chemical structures of parent M4K1046 and the bespoke tracer are shown on the side of the graphs. This tracer cannot bind to ALK5.</p></div>
<p><strong>Conclusion:</strong></p>
<p>Both bespoke tracers cannot be used for ALK5 nanoBRET. Both of them did not bind to ALK5. The nanoBRET experiment itself was a success since both tracers worked well with ALK2 (over 20 folds assay windows). The background noise was low for both tracers. ALK5 seems to intrinsically not tolerate bulky additions on the solvent end of M4K1046. This might be additional avenue for improving ALK2 vs ALK5 selectivity. At the very least, we have gained this additional information from these efforts.</p>
<p>For additional experimental details, please refer to my <a href="https://zenodo.org/record/2628675#.XKYfNlVKiUk">Zenodo post</a>.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/determining-whether-bespoke-tracers-work-with-alk5-in-nanobret-target-engagement-assay/feed/</wfw:commentRss>
		<slash:comments>2</slash:comments>
		</item>
		<item>
		<title>Novel Hinge Binders: N-Methyl-azaindoles</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/novel-hinge-binders-n-methyl-azaindoles/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/novel-hinge-binders-n-methyl-azaindoles/#respond</comments>
		<pubDate>Tue, 02 Apr 2019 18:30:55 +0000</pubDate>
		<dc:creator><![CDATA[Sean O'Byrne]]></dc:creator>
				<category><![CDATA[Sean O'Byrne]]></category>
		<category><![CDATA[Understudied Kinases]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2644</guid>
		<description><![CDATA[One goal of the CaMKK2 project is to explore the chemistry and biology of novel hinge binders. Various hinge binders showing CaMKK2 inhibition have been identified in the literature and through testing of compounds synthesized in the SGC. Of note in this search was a recent publication by Price et al. (Bioorg. Med. Chem. Lett., <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/novel-hinge-binders-n-methyl-azaindoles/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>One goal of the CaMKK2 project is to explore the chemistry and biology of novel hinge binders. Various hinge binders showing CaMKK2 inhibition have been identified in the literature and through testing of compounds synthesized in the SGC. Of note in this search was a recent publication by Price <em>et al</em>. (<em>Bioorg. Med. Chem. Lett</em>., <strong>2018</strong>, <em>28</em>, 1958–1963), in which molecules based on 7-azaindole (<strong>1,2</strong>) and 2-amino-pyrimidine (<strong>3</strong>) (<strong>Figure 1</strong>) cores showed excellent activity in an enzymatic CaMKK2 assay.</p>
<div id="attachment_2645" style="width: 790px" class="wp-caption alignnone"><img class="wp-image-2645 size-large" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Figure-1-1024x283.png" alt="" width="780" height="216" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Figure-1-1024x283.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Figure-1-300x83.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Figure-1-768x213.png 768w" sizes="(max-width: 780px) 100vw, 780px" /><p class="wp-caption-text"><strong>Figure 1</strong>: Azaindole based CaMKK2 inhibitors</p></div>
<p>Although the compounds have good potency, they are promiscuous, showing poor selectivity in a small screen against 48-86 different kinases. Indeed between 15-30% of the kinases in the panel were inhibited with an IC<sub>50</sub> &lt;100 nM.</p>
<p>Small modifications to the hinge binding region of pharmacophore can have drastic effects on the binding mode, and therefore the selectivity and potency, of inhibitors. It was decided to probe if <em>N</em>-methylation of the azaindole core could be tolerated in CaMKK2. <em>N</em>-methyl azaindole <strong>4 </strong>(<strong>Figure 1</strong>) was synthesized and sent for testing in a CaMKK2 enzyme inhibition assay. The compounds showed moderate activity giving an IC<sub>50 </sub>of 183 nM. The compound also gave had good solubility (125 µM) and was membrane permeable. It was decided to synthesis a small library of compounds based on <strong>4</strong> to see if more potent compounds based on this scaffold.</p>
<p>The compounds were relatively easy to access synthetically. <em>N</em>-methylation of 2-iodo-4-chloro-7-azaindole followed by two Suzuki couplings and an ester hydrolysis allowed quick access to the desired compounds. Conveniently, both Suzuki coupling could be carried using microwave conditions with reactions going to completion in 30 minutes (<strong>Scheme 1</strong>).</p>
<div id="attachment_2647" style="width: 790px" class="wp-caption alignnone"><img class="size-large wp-image-2647" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Scheme-1-1-1024x245.png" alt="" width="780" height="187" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Scheme-1-1-1024x245.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Scheme-1-1-300x72.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Scheme-1-1-768x184.png 768w" sizes="(max-width: 780px) 100vw, 780px" /><p class="wp-caption-text"><strong>Scheme 1</strong>: Sequential Suzuki reactions utilizing a microwave reactor to greatly reduce reaction times.</p></div>
<p>In choosing what analogues to make in the library I wanted to probe the effects of electron withdrawing and donating groups, lipophilicity and to determine the effect of extending the molecules with morpholino- groups pointing out of the binding pocket towards the solvent. The library of compounds synthesized is shown in <strong>Figure 2</strong>.</p>
<div id="attachment_2648" style="width: 790px" class="wp-caption alignnone"><img class="size-large wp-image-2648" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Figure-2-1024x515.png" alt="" width="780" height="392" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Figure-2-1024x515.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Figure-2-300x151.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Figure-2-768x386.png 768w" sizes="(max-width: 780px) 100vw, 780px" /><p class="wp-caption-text">Figure 2: Library of N-methyl-azaindoles synthesized and sent for a CaMKK2 enzyme inhibition assay.</p></div>
<p>The compounds have been shipped and we are awaiting the results from the enzyme inhibition assay. If any of the compounds have good activity, the best will be sent for a selectivity profiling using KINOMEscan screening platform provided by DiscoverX. From this we will learn if <em>N</em>-methylation can indeed modulate selectivity across the kinome.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/novel-hinge-binders-n-methyl-azaindoles/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Molecular Forecaster Inc (MFI): USP5 Zf-UBD</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/molecular-forecaster-inc-mfi-usp5-zf-ubd/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/molecular-forecaster-inc-mfi-usp5-zf-ubd/#respond</comments>
		<pubDate>Mon, 01 Apr 2019 12:30:17 +0000</pubDate>
		<dc:creator><![CDATA[Mandeep Mann]]></dc:creator>
				<category><![CDATA[Mandeep Mann]]></category>
		<category><![CDATA[USP5]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2625</guid>
		<description><![CDATA[In my last post, I mentioned I would be doing an internship at a biotech company in Montreal, Molecular Forecaster Inc as a part of a NSERC ChemNet CREATE grant. I’ve been in Montreal for about three weeks now and have had a chance to play around with the Forecaster software. I did preliminary docking <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/molecular-forecaster-inc-mfi-usp5-zf-ubd/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>In my last post, I mentioned I would be doing an internship at a biotech company in Montreal, Molecular Forecaster Inc as a part of a NSERC ChemNet CREATE grant. I’ve been in Montreal for about three weeks now and have had a chance to play around with the Forecaster software. I did preliminary docking validation studies for USP5 Zf-UBD using the Forecaster FITTED program. You can find experimental details on <a href="https://zenodo.org/record/2620208#.XKIBqihKiUk">Zenodo</a>.</p>
<p>First, I did self-docking studies with USP5 Zf-UBD co-crystal structures to assess the success of ligand docking. Self-docking is a method where a ligand is docked into the protein structure it was extracted from. To evaluate the correctness of the predicted pose, root mean square deviation values (RMSD) are evaluated; the lower the RMSD, the better the docked pose. The predicted binding poses of self-docked ligands was successful (RMSD &lt;2 Å), as shown in Figure 1.</p>
<p><img class=" wp-image-2629 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/20190401b-300x187.png" alt="" width="412" height="257" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/20190401b-300x187.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/20190401b.png 468w" sizes="(max-width: 412px) 100vw, 412px" /></p>
<p style="text-align: center;"><strong>Figure 1.</strong> USP5 Zf-UBD self-docking RMSD</p>
<p>An interesting aspect of the Forecaster platform is MATCHUP, where you can dock ligands to a protein and simulate the flexibility of the protein by providing several conformations of structures. The predicted binding poses of ligands docked using MATCHUP was also successful (RMSD &lt;2 Å), as shown in Figure 2, however the docked poses of compound DAT201 and compound DAT180 had higher RMSDs (RMSD &gt;1.5 Å) with flexible docking than rigid self-docking (Figure 3).</p>
<p><img class=" wp-image-2630 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/20190401c-300x168.png" alt="" width="447" height="250" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/20190401c-300x168.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/20190401c.png 536w" sizes="(max-width: 447px) 100vw, 447px" /></p>
<p style="text-align: center;"><strong>Figure 2</strong>. Self-docking (rigid) vs. flexible docking (MATCHUP)</p>
<p>&nbsp;</p>
<p><img class=" wp-image-2628 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/20190401a-300x148.png" alt="" width="527" height="260" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/20190401a-300x148.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/20190401a-768x380.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/20190401a-1024x506.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/20190401a.png 1456w" sizes="(max-width: 527px) 100vw, 527px" /></p>
<p style="text-align: center;"><strong>Figure 3</strong>. Flexible docked pose (green) versus co-crystallized ligand (grey) a) DAT201 b) DAT180 c) DAT194</p>
<p>I also performed rigid cross-docking of the available USP5 Zf-UBD structures. Cross-docking is when different ligands are docked to a protein structure from a complex; this is useful when deciding which structure to use for virtual screening. Table 1 summarizes the results of the cross-docking study, where structure 6NFT had the best overall average RMSD.</p>
<p style="text-align: center;"><strong>Table 1</strong>. Cross-docking: RMSDs of ligand pose with best energy complex from 3 poses</p>
<p><img class="alignnone wp-image-2631 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/20190401d-300x91.png" alt="" width="333" height="101" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/20190401d-300x91.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/20190401d.png 444w" sizes="(max-width: 333px) 100vw, 333px" /></p>
<p>Lastly, I did a virtual screen of a library of USP5 Zf-UBD compounds that have already been experimentally tested to determine the best ranking docking method (Figure 3). Flexible docking with MATCHUP results in the best overall ranking of USP5 Zf-UBD compounds, when comparing ligand activity and docking scores, as shown by the area under the curve (AUC) values. The AUC of the docking methods are marginally better than random (Figure 3-red (AUC=0.5)) which may be because all compounds in the library tested, include a carboxylic chain that drives binding in the pocket, making it challenging to distinguish between active and inactive ligands. Next, I’ll be using Forecaster’s flexible protein docking with MATCHUP to screen larger commercial libraries in the search for different ligand scaffolds against USP5 Zf-UBD. Stay tuned!</p>
<p><img class=" wp-image-2632 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/20190401e-300x40.png" alt="" width="353" height="47" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/20190401e-300x40.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/20190401e.png 476w" sizes="(max-width: 353px) 100vw, 353px" /></p>
<p><img class=" wp-image-2633 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/20190401f-300x174.png" alt="" width="445" height="258" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/20190401f-300x174.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/20190401f.png 624w" sizes="(max-width: 445px) 100vw, 445px" /></p>
<p style="text-align: center;"><strong>Figure 3. </strong>ROC of virtual screen of USP5 Zf-UBD compounds</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p><u>References </u></p>
<p>Corbeil C.R., Englebienne P., Moitessier N. Docking ligands into flexible and solvated macromolecules. 1. Development and validation of FITTED 1.0<strong> </strong><em>J. Chem. Inf. Model.</em> (2007), 47, 2, 435-449</p>
<p>Corbeil C.R., Moitessier N. Docking ligands into flexible and solvated macromolecules. 3. Impact of input ligand conformation, protein flexibility, and water molecules on the accuracy of docking programs<strong> </strong><em>J. Chem. Inf. Model.</em> (2009), 49, 4, 997-1009</p>
<p>Moitessier N., Pottel J., Therrien E., Englebienne P., Liu Z., Tomberg A., Corbeil C.R. Medicinal Chemistry Projects Requiring Imaginative Structure-Based Drug Design Methods <em>Acc. Chem. Res.</em> (2016), 49 (9), 1646-1657</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/molecular-forecaster-inc-mfi-usp5-zf-ubd/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Cell types we work with</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/cell-types-we-work-with/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/cell-types-we-work-with/#respond</comments>
		<pubDate>Sat, 23 Mar 2019 18:06:56 +0000</pubDate>
		<dc:creator><![CDATA[Thomas Durcan]]></dc:creator>
				<category><![CDATA[ALS-Reproducible Antibody Platform (ALS-RAP)]]></category>
		<category><![CDATA[Amyotrophic Lateral Sclerosis (ALS)]]></category>
		<category><![CDATA[iPSC Disease modelling]]></category>
		<category><![CDATA[iPSCs]]></category>
		<category><![CDATA[Parkinson's disease and neurodevelopmental disorders]]></category>
		<category><![CDATA[Thomas M. Durcan]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2607</guid>
		<description><![CDATA[Hello again, Three weeks have now passed since my first blog, and the response has been overwhelming, who knew this kid with a love of dinosaurs could become an internet blogger. Your encouragement to make my t-Rex has motivated me to do just that, so let’s see how it all goes. Also fun fact, the <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/cell-types-we-work-with/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Hello again,</p>
<p>Three weeks have now passed since my first blog, and the response has been overwhelming, who knew this kid with a love of dinosaurs could become an internet blogger. Your encouragement to make my t-Rex has motivated me to do just that, so let’s see how it all goes. Also fun fact, the biggest and oldest T-Rex was found in <a href="https://www.ctvnews.ca/sci-tech/paleontologists-discover-world-s-biggest-and-oldest-tyrannosaurus-rex-in-canada-1.4348206">Canada</a>, so looks like I am in right place for dinosaurs.</p>
<p>But, in the meantime, I do have a day, night and weekend job running a lab and academic drug discovery group, and so its time to discuss science. More specifically, what cells we work with in our work. Our bread and butter are the iPSC, short for induced pluripotent stem cells. Made from the blood of patients and healthy volunteers, when you look at them, they look like regular old cells, tightly packed into colonies, as they love being close to their fellow cells, problems arise when they want to move away and get some distance from their neighbors. Below are some images of what they look like, courtesy of Carol Chen and Narges Abdian, who help manage and oversee our catalog of stem cell lines in my group, along with Zhipeng You. As you can see, the cells are tightly packed, with each cell having the potential to grow into any cell type you want, all it needs is the right ingredients to direct it into the cell type of your choosing.</p>
<p><a href="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/iPSC-cell-image.tif"><img class="alignnone size-medium wp-image-2610" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/iPSC-cell-image.tif" alt="" width="1" height="1" /><img class="size-medium wp-image-2611 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Image-of-IPSCs_forblog-300x249.jpg" alt="" width="300" height="249" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Image-of-IPSCs_forblog-300x249.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Image-of-IPSCs_forblog-768x638.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Image-of-IPSCs_forblog-1024x851.jpg 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Image-of-IPSCs_forblog.jpg 1175w" sizes="(max-width: 300px) 100vw, 300px" /></a></p>
<p>And what might these cells be, well we are interested in diseases of the brain, and in particular the cells lost during the progression of these degenerative disorders. This means our focus is on getting these stem cells to form neurons, of which everyone of us has over 100 billion. We can make many different types and over the coming months, I will be putting online our different SOPs from our group outlining how we make many of these different neurons. One of the main interests for our group is Parkinson’s disease, and the cells lost in this disease are dopaminergic neurons, making it our mission to understand why these neurons are being lost in the disease. Other cell types we make are motor neurons, sensory neurons and cortical neurons, all different types of neurons that are involved in different diseases, playing different roles. For those who wonder what they look like, here are some images below, staining for proteins that are specific for these neurons (Dopaminergic-Top; Cortical- Bottom). Images are courtesy of Carol Chen and Cecilia DeRocha Desouza.</p>
<p><img class="wp-image-2612 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/DA-neurons-300x232.jpg" alt="" width="299" height="231" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/DA-neurons-300x232.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/DA-neurons-768x594.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/DA-neurons-1024x792.jpg 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/DA-neurons.jpg 1323w" sizes="(max-width: 299px) 100vw, 299px" /><img class="wp-image-2613 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/cortical-neurons.jpg" alt="" width="296" height="234" /></p>
<p>In addition to neurons, we also make the support cells for these neurons, and one of these cells is astrocytes. This cell can be the good guy-bad guy and has been demonstrated to support neuronal survival, and on the flip side, to promote the loss of neurons. Courtesy of Vincent Soubannier, here are some images of what these cells look like in the dish.<img class="size-medium wp-image-2615 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/astrocytes-1-300x241.jpg" alt="" width="300" height="241" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/astrocytes-1-300x241.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/astrocytes-1-768x617.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/astrocytes-1.jpg 908w" sizes="(max-width: 300px) 100vw, 300px" /></p>
<p>Now, I know, you are saying Tom, these neurons and astrocytes are beautiful, but they are flat, having been grown in 2D dishes. This isn’t like a brain; the brain is three dimensional and complex. And you know what, you would be right. Growing cells in 2D is artificial, but it lets us grow and image neurons and astrocytes for doing research on human cells that otherwise wouldn’t be accessible. But it wasn’t until the pioneering work by Dr <a href="https://www2.mrc-lmb.cam.ac.uk/groups/lancaster/">Madeline Lancaster</a> and others that the idea for making complex brain structures emerged. From pioneering work in her lab, she showed that neuronal organoids or “minibrains” could be formed in a dish. More complex then anything grown in 2D, these minibrains could be expanded, grown for several months, if not years, and show many characteristics of different regions within the human brain. Over 18 months ago, we turned our attention to minibrains and now after all this time, we can make three different types in our group: midbrain, cerebral and forebrain neuronal organoids or “minibrains”. You can learn more about this work through the enclosed link, with <a href="https://www.mcgill.ca/neuro/category/article-categories/neuroxxceptional-stories/nguyen-vi-mohamed">Dr Nguyen-Vi Mohamed</a> showing how these are made in the lab.</p>
<p>These can grow up to 4mm in size depending on the minibrain grown, and are comprised of neurons, astrocytes and many other cell types. They can be maintained for months, and our oldest minibrains were over 14 months old. Just as a preview of what they look like, below is a section through one of these minibrains at 30 days, courtesy of Dr Nguyen-Vi Mohamed.</p>
<p><img class=" wp-image-2616 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/minibrain-section-300x244.jpg" alt="" width="304" height="247" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/minibrain-section-300x244.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/minibrain-section-768x624.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/minibrain-section-1024x832.jpg 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/minibrain-section.jpg 1281w" sizes="(max-width: 304px) 100vw, 304px" /></p>
<p>With incubators packed with these little guys, we now have over 5000 growing with the capacity to maintain close to 10,000 at any one time. These are probably the coolest things I have ever seen and am amazed at the things that can be done with them. Its such a new model, we are barely scratching the surface at what can be done, but I am looking forward to trying new things and seeing what others do. To end, I wanted to bring peoples attention to some remarkable work from from the Lancaster group published this <a href="https://www.theguardian.com/science/2019/mar/18/scientists-grow-mini-brain-on-the-move-that-can-contract-muscle">week</a> (I am like a fanboy of her work). They could connect a minibrain and muscle, with the minibrain taking control over the muscle and causing it to contract. This is just the tip of the iceberg and now I can’t wait to see what other structures we can connect to a minibrain. It won’t be long before we have little mini bodies on a dish, with minibrains connected to miniature versions of all the major organs in the body. But for now, we wait and see how the field progresses. For those wondering how we make minibrains, our <a href="https://mniopenresearch.org/articles/3-1/v1">methods paper</a> is coming out in next week or so, and once its out, I will use this blog to cover the main points on how we make these organoid structures amongst other hot topics in the stem cell field. Next blog, I will be discuss how to thaw out a vial of iPSCs for all those new users wondering what I should do when I get a tube of these cells. Well don’t panic, I am here to help you. Feel free to connect with me on <a href="https://www.linkedin.com/in/thomas-durcan-18593b23/">LinkedIn</a>, if I don’t know you please at least introduce yourself and who you are. I also now on the twitter (@thomas.durcan), so you can follow me there too, I am a relative novice at this whole social media malarkey, so we shall see how that goes.</p>
<p>&nbsp;</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/cell-types-we-work-with/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Update on meetings, manuscripts and mighty cool experiments…</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/update-on-meetings-manuscripts-and-mighty-cool-experiments/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/update-on-meetings-manuscripts-and-mighty-cool-experiments/#respond</comments>
		<pubDate>Thu, 21 Mar 2019 16:00:43 +0000</pubDate>
		<dc:creator><![CDATA[Rachel Harding]]></dc:creator>
				<category><![CDATA[Huntingtin]]></category>
		<category><![CDATA[Huntington’s disease]]></category>
		<category><![CDATA[Rachel Harding]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2674</guid>
		<description><![CDATA[Update on meetings, manuscripts and mighty cool experiments&#8230;]]></description>
				<content:encoded><![CDATA[<blockquote class="wp-embedded-content" data-secret="jM2a5gTbw1"><p><a href="http://labscribbles.com/2019/03/21/update-on-meetings-manuscripts-and-mighty-cool-experiments/">Update on meetings, manuscripts and mighty cool experiments&#8230;</a></p></blockquote>
<p><iframe class="wp-embedded-content" sandbox="allow-scripts" security="restricted" src="http://labscribbles.com/2019/03/21/update-on-meetings-manuscripts-and-mighty-cool-experiments/embed/#?secret=jM2a5gTbw1" data-secret="jM2a5gTbw1" width="600" height="338" title="&#8220;Update on meetings, manuscripts and mighty cool experiments&#8230;&#8221; &#8212; LabScribbles" frameborder="0" marginwidth="0" marginheight="0" scrolling="no"></iframe></p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/update-on-meetings-manuscripts-and-mighty-cool-experiments/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Synthesis of promiscuous ALK2 inhibitors</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/synthesis-of-promiscuous-alk2-inhibitors/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/synthesis-of-promiscuous-alk2-inhibitors/#respond</comments>
		<pubDate>Tue, 19 Mar 2019 20:34:09 +0000</pubDate>
		<dc:creator><![CDATA[Alfredo Picado]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Alfredo Picado]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>
		<category><![CDATA[Understudied Kinases]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2598</guid>
		<description><![CDATA[In a collaboration between SGC-University of North Carolina-Chapel Hill and M4K Pharma, we plan to synthesize 3 different imidazopyridin pyrazines as promiscuous kinase ALK2 inhibitors, compounds 1-3, Scheme 1. The first step will be the synthesis of key intermediate 10, Scheme 2.  The proposed synthesis begins with a nucleophilic aromatic substitution on compound 4 to <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/synthesis-of-promiscuous-alk2-inhibitors/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>In a collaboration between SGC-University of North Carolina-Chapel Hill and M4K Pharma, we plan to synthesize 3 different imidazopyridin pyrazines as promiscuous kinase ALK2 inhibitors, compounds <strong>1-3</strong>, Scheme 1.</p>
<div id="attachment_2596" style="width: 994px" class="wp-caption alignnone"><img class="size-full wp-image-2596" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Screen-Shot-2019-03-19-at-4.22.27-PM.png" alt="" width="984" height="346" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Screen-Shot-2019-03-19-at-4.22.27-PM.png 984w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Screen-Shot-2019-03-19-at-4.22.27-PM-300x105.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Screen-Shot-2019-03-19-at-4.22.27-PM-768x270.png 768w" sizes="(max-width: 984px) 100vw, 984px" /><p class="wp-caption-text">Scheme 1. ALK2 inhibitors</p></div>
<p>The first step will be the synthesis of key intermediate <strong>10</strong>, Scheme 2.  The proposed synthesis begins with a nucleophilic aromatic substitution on compound <strong>4</strong> to get nitropyridine <strong>5</strong>, then reduction with hydrogen on palladium/carbon to obtain the pyridine-3,4-diamine <strong>6</strong>.</p>
<p>On the other hand, aminopyrazine <strong>7</strong> will be saponified to the aminopyrazine sodium salt <strong>8</strong>, which will be coupled with  compound <strong>6</strong> using peptide coupling reagent HATU to form bromopyrazine carboxamide <strong>9</strong>, followed by heating in presence of acetic acid to yield the cyclized intermediate 5-bromo-3-(1-ethyl-1<em>H</em>-imidazo[4,5-<em>c</em>]pyridin-2-yl)pyrazin-2-amine <strong>10</strong>.</p>
<div id="attachment_2597" style="width: 928px" class="wp-caption alignnone"><img class="size-full wp-image-2597" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Screen-Shot-2019-03-19-at-4.22.42-PM.png" alt="" width="918" height="650" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Screen-Shot-2019-03-19-at-4.22.42-PM.png 918w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Screen-Shot-2019-03-19-at-4.22.42-PM-300x212.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Screen-Shot-2019-03-19-at-4.22.42-PM-768x544.png 768w" sizes="(max-width: 918px) 100vw, 918px" /><p class="wp-caption-text">Scheme 2. Synthesis route 1</p></div>
<p>If we succeed with the proposed synthesis of imidazopyridine pyrazine <strong>10</strong>, then soon we will present next steps to get to target compounds <strong>1-3</strong></p>
<p>This collaboration is the under supervision of Dr. David Drewry.</p>
<p>Synthesis routes were adapted from international patents WO 2006/063167 A1, WO 2008/071937 and <em>ACS Med. Chem. Lett</em>. 2016, 7, 5, 514-519.</p>
<p>&nbsp;</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/synthesis-of-promiscuous-alk2-inhibitors/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Phosphorylation over time and phosphomapping experiments to investigate the influence of ALK2 mutations on activity.</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/phosphorylation-over-time-and-phosphomapping-experiments-to-investigate-the-influence-of-alk2-mutations-on-activity/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/phosphorylation-over-time-and-phosphomapping-experiments-to-investigate-the-influence-of-alk2-mutations-on-activity/#respond</comments>
		<pubDate>Tue, 12 Mar 2019 11:56:29 +0000</pubDate>
		<dc:creator><![CDATA[Ellie Williams]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Ellie Williams]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2579</guid>
		<description><![CDATA[In this post I’d like to return to looking at my work on phosphorylation. Phosphorylation is a key signalling method for activating the BMP pathway and inducing activity. Specifically activation of ALK2 by phosphorylation leads to phosphorylation of SMAD1 which leads to gene transcription that ultimately is responsible for bone formation. In this post I’m <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/phosphorylation-over-time-and-phosphomapping-experiments-to-investigate-the-influence-of-alk2-mutations-on-activity/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>In this post I’d like to return to looking at my work on phosphorylation. Phosphorylation is a key signalling method for activating the BMP pathway and inducing activity. Specifically activation of ALK2 by phosphorylation leads to phosphorylation of SMAD1 which leads to gene transcription that ultimately is responsible for bone formation.</p>
<p>In this post I’m going to talk about two experiments – the first is looking at the rate of phosphorylation (or how active) of two ALK2 mutants, L196P and R258S. The second is to look at mapping the exact sites of phosphorylation on two of the more common ALK2 mutations in FOP, R206H and Q027E.</p>
<p>&nbsp;</p>
<p><strong>Phosphorylation over time in ALK2 L196P and R258S.</strong></p>
<p>I’ve talked <a href="https://thesgc.github.io/static-openlabnotebooks/phosphorylation-of-smad1-rates-as-a-function-of-alk2-mutations-and-type-ii-association/">previously</a> about how phosphorylation is effected by the Q207E and R206H mutations in ALK2. This time I wanted to look at some additional mutants that are found in a small proportion of cases of FOP, specifically L196P, R258S, G328E and R375P. I purified the long forms of these ALK2 mutations in a standard fashion. Regrettably only L196P and R258S gave a good yield of expressed protein while the other two constructs gave me no expression of protein. I set up my standard activation assay with these ALK2 mutants (I mixed them with SMAD1 and with either ACVR2 or BMPR2, before adding ATP, DTT and Mg2+/Mn2+ in order to start the reaction), taking samples over time periodically up to 30 minutes. The samples were then analysed by mass spec to look for intact mass shifts corresponding to phosphorylation.</p>
<p>I plotted the peaks corresponding to phosphorylated states of both ALK2 and SMAD1 at the different time points and used this 3D graph to compare different mutant’s behaviour with respect to the activity of ALK2. I also compared this data to previously obtained data from the previous blog post.</p>
<p><img class="alignnone size-large wp-image-2580" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/MS_timecourse_blogpost-806x1024.jpg" alt="" width="780" height="991" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/MS_timecourse_blogpost-806x1024.jpg 806w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/MS_timecourse_blogpost-236x300.jpg 236w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/MS_timecourse_blogpost-768x975.jpg 768w" sizes="(max-width: 780px) 100vw, 780px" /></p>
<p><em>Time course plots for phosphorylation of SMAD1 (top) and ALK2 L196P or R258S (bottom) in the presence of ACVR2 (left) or BMPR2 (right). Legend shows time point in seconds.</em></p>
<p>&nbsp;</p>
<p>Overall we can see that R258S appears to be more activating than L196P with both more SMAD activation over the first five minutes (including the appearance of a potentially interesting third SMAD binding site). This is the case when using either ACVR2 or BMPR2. Also in general ACVR2 appears to lead to more SMAD activation than BMPR2 regardless of mutation.</p>
<p>When looking at ALK2 phosphorylation, we can see that L196P is quickly phosphorylated up to three times compared to R258S but that over the course of five minutes, R258S accumulates up to 6 phosphorylation’s much more rapidly than L196P. This effect is most pronounced when paired with ACVR2.</p>
<p>What this means in very broad terms is that the different ALK2 FOP mutants are differently activating and that their response to different type II receptors change depending on which mutation is present. This is similar to the results seen with other ALK2 mutants studied previously. This builds towards understanding why the ALK2 FOP mutations leads to increased signalling and thus increased bone formation and exactly what is going on.</p>
<p>&nbsp;</p>
<p><strong>Phosphomapping of ALK2 R206H and Q207E:</strong></p>
<p>Another aspect of looking at the phosphorylation of ALK2 and SMAD1 is not just how much it is phosphorylated but also where it is phosphorylated. For this I worked with Rod Chalk in our mass spec department to map the phosphorylation sites present in samples of protein incubated as previously mentioned. In this case I was looking at Q207E and R206H mutants of ALK2 paired with either ACVR2 or BMPR2, all in the presence of SMAD. This suggested that many of the phosphorylations could be traced to the following positions in the GS region of the kinase regardless of ALK2 mutation:</p>
<p><strong><u>ST</u></strong>LADLLDH<strong><u>S</u></strong>C<strong><u>TS</u></strong>G<strong><u>S</u></strong>G<strong><u>S</u></strong>GLPFLVQRTV</p>
<p>However not all these sites are seen at the same time and it is hard to determine which are essential for transmission of signal and ALK2 activation.</p>
<p>Similarly phosphorylation on SMAD1 could be traced to the following common locations at the C-terminal tail of the protein:</p>
<p>TQMGSPHNPI<strong><u>SS</u></strong>V<strong><u>S</u></strong></p>
<p>In this case it is the last three serines in the tail that are able to be phosphorylated and again, are seen with both mutations. Combinations of the three possible sites and all three at once were seen as possibilities within the phosphomapping data.</p>
<p>What this means however is a bit less clear. When considering the time course data from the previous post linked as well as the new mutant time course data and the phosphomapping it seems to suggest that you require multiple (up to 6) ALK2 phosphorylations to get maximal (3 site) SMAD1 phosphorylation and that there are at least this many sites in the GS loop and the region just prior to it.</p>
<p>In order to properly probe the necessity of each of these sites, it is going to be necessary to repeat these mass spec experiments on knock out mutants that remove the phosphorylation sites in various combinations, to really see which are essential for ALK2 activation. These are currently being cloned so hopefully I’ll be able to test them soon.</p>
<p>&nbsp;</p>
<p>You can find the technical details for all these experiments over on <a href="https://doi.org/10.5281/zenodo.2591062">Zenodo.</a></p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/phosphorylation-over-time-and-phosphomapping-experiments-to-investigate-the-influence-of-alk2-mutations-on-activity/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>USP5 Zf-UBD Co-Crystal Structure with Compound XSR00035795a &#038; Testing Selectivity</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/usp5-zf-ubd-co-crystal-structure-with-compound-xsr00035795a-testing-selectivity/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/usp5-zf-ubd-co-crystal-structure-with-compound-xsr00035795a-testing-selectivity/#respond</comments>
		<pubDate>Fri, 01 Mar 2019 19:24:16 +0000</pubDate>
		<dc:creator><![CDATA[Mandeep Mann]]></dc:creator>
				<category><![CDATA[Mandeep Mann]]></category>
		<category><![CDATA[USP5]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2548</guid>
		<description><![CDATA[In a previous post, I tested hit analogues against USP5 Zf-UBD using a SPR assay. Compound AE-641/11456811, now aptly re-named XSR00035795a (an SGC global ID- since our database infrastructure is finally re-vamped), had a binding affinity of approximately 60 µM. The addition of a methyl group on the carboxylic chain of the compound increased potency <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/usp5-zf-ubd-co-crystal-structure-with-compound-xsr00035795a-testing-selectivity/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>In a <a href="https://thesgc.github.io/static-openlabnotebooks/testing-hit-analogues-against-usp5-zf-ubd-with-spr-assay-1/">previous post</a>, I tested hit analogues against USP5 Zf-UBD using a SPR assay. Compound AE-641/11456811, now aptly re-named XSR00035795a (an SGC global ID- since our database infrastructure is finally re-vamped), had a binding affinity of approximately 60 µM. The addition of a methyl group on the carboxylic chain of the compound increased potency 3-fold!</p>
<p>You can find details of this experiment on <a href="https://zenodo.org/record/2581522#.XHmFHIhKjIU">Zenodo</a>. I crystallized the complex of USP5 Zf-UBD with XSR00035795a, and the structure was solved by Dr. Wolfram Tempel. The addition of the methyl group leads to increased hydrophobic interactions in the binding pocket with W209; the side chain of R221 also rotates out of the pocket (Figure 1).</p>
<p><img class="wp-image-2549 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/20190301e-300x122.png" alt="" width="585" height="237" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/20190301e-300x122.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/20190301e-768x313.png 768w" sizes="(max-width: 585px) 100vw, 585px" /></p>
<p style="text-align: center;"><strong>Figure 1</strong>. a) Superimposed USP5 structures of compound DAT194 (PDB: 6NFT; cyan) and compound XSR00035795a (magenta) b) Binding pocket of co-crystal structure of USP5 Zf-UBD and XSR00035795a</p>
<p>Next, I tested this compound against HDAC6 Zf-UBD to see if it confers selectivity against USP5 Zf-UBD using a SPR assay (Figure 2). Compound XSR00035795a (LE=0.36) was not selective for USP5 Zf-UBD. XSR00035795a and HDAC6 Zf-UBD has a K<sub>D</sub> of 15 ± 0.4 µM (n=2) whereas USP5 Zf-UBD has an average K<sub>D</sub> of 60 ± 8 µM (n=3).</p>
<p style="text-align: left;">a)                                                                                                        b)<img class="wp-image-2550 alignnone" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/20190301f-300x75.png" alt="" width="648" height="162" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/20190301f-300x75.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/20190301f-768x193.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/20190301f-1024x257.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/20190301f.png 1372w" sizes="(max-width: 648px) 100vw, 648px" /></p>
<p style="text-align: center;"><strong>Figure 2</strong>. Representative SPR binding curves and sensograms of compound XSR00035795a with a) HDAC6 Zf-UBD (K<sub>D</sub>= 15 ± 0.4 µM; n=2) and b) USP5 Zf-UBD (K<sub>D</sub>= 60 ± 8 µM; n=3)</p>
<p>When co-crystal structures of HDAC6 Zf-UBD and USP5 Zf-UBD are superimposed, binding of XSR00035795a to HDAC6 Zf-UBD makes sense. Figure 3 shows the binding pocket of HDAC6 Zf-UBD is able to accommodate the methyl group of compound XSR00035795a. Replacing a valine residue in the primary pocket of HDAC6 with an alanine in USP5 generates a small cavity that may be exploited by extending the aliphatic group on the carboxylic chain to confer selectivity towards USP5.</p>
<p><img class=" wp-image-2551 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/20190301g-300x226.png" alt="" width="448" height="336" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/20190301g-300x226.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/20190301g-350x263.png 350w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/20190301g-250x188.png 250w" sizes="(max-width: 448px) 100vw, 448px" /></p>
<p style="text-align: center;"><strong>Figure 3</strong>. Superimposed CPK representation of HDAC6 Zf-UBD (purple PDB: 6CE8) and USP5 Zf-UBD and compound XSR00035795a co-crystal structure (cyan)</p>
<p>Commercial compounds with extended aliphatic groups on the scaffold have been ordered but unfortunately due to vendor delays, I haven’t received them! Here’s hoping the compounds will arrive soon…</p>
<p>On another note, I will be doing a three month internship at the Montreal biotech, Molecular Forecaster Inc., as part of a NSERC-CREATE ChemNET grant. I will further explore computational chemistry approaches and structure-based drug design to design inhibitors against USP5 Zf-UBD and use the docking platform to dock compounds against available subfamily protein domain structures, as a predictive tool for selectivity. I’ll be heading to Montreal next week! Stay tuned for some neat computational simulations against USP5 Zf-UBD and the results of the extended aliphatic groups on the XSR00035795a scaffold.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/usp5-zf-ubd-co-crystal-structure-with-compound-xsr00035795a-testing-selectivity/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Checking I haven&#8217;t got my cells mixed up&#8230;</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/checking-i-havent-got-my-cells-mixed-up/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/checking-i-havent-got-my-cells-mixed-up/#respond</comments>
		<pubDate>Thu, 28 Feb 2019 18:56:05 +0000</pubDate>
		<dc:creator><![CDATA[Elizabeth Brown]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Brain Tumours]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>
		<category><![CDATA[Liz Brown]]></category>
		<category><![CDATA[Understudied Kinases]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2542</guid>
		<description><![CDATA[Did you know as many as 30% of the cells in labs could be misidentified? And what a nightmare it would be to find out years of your work are just completely wrong because you got your skin cells mixed up with your brain cells&#8230; So to avoid that catastrophe, I took some time to <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/checking-i-havent-got-my-cells-mixed-up/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Did you know as many as 30% of the cells in labs could be misidentified? And what a nightmare it would be to find out years of your work are just completely wrong because you got your skin cells mixed up with your brain cells&#8230;</p>
<p>So to avoid that catastrophe, I took some time to check the DNA sequence of the ACVR1 gene, because for this gene we know what should be mutated (or not) for each of my cell lines. This way I can make sure that I haven&#8217;t mixed them up in the past year (<em>and</em> that they haven&#8217;t picked up new mutations in this important area all by themselves).</p>
<p>All I had to do was collect the genomes of the cells, amplify specifically the ACVR1 gene, send it for overnight sequencing, and then I get a trace like this back from the facility:</p>
<div id="attachment_2544" style="width: 790px" class="wp-caption aligncenter"><a href="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/R206H-sequence-1.png"><img class="wp-image-2544 size-large" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/R206H-sequence-1-1024x396.png" alt="Sequencing trace covering the R206H mutation in ACVR1 found in FOP and DIPG" width="780" height="302" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/R206H-sequence-1-1024x396.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/R206H-sequence-1-300x116.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/R206H-sequence-1-768x297.png 768w" sizes="(max-width: 780px) 100vw, 780px" /></a><p class="wp-caption-text">Sequencing trace covering the R206H mutation in ACVR1 found in FOP and DIPG</p></div>
<p>Each DNA base has one peak, except one that has both a guanine (G) and an adenine (A) DNA base. The adenine, present on one chromosome, causes the change from an arginine (R) to a histidine (H) at the 206th position of the ACVR1 protein i.e. the R206H mutation.</p>
<p>This trace is from the HSJD-DIPG-007 cell line and is the mutation I expected. According to all my sequencing, none of my cell lines have gotten mixed up, yay!</p>
<p>You can read the full details of how I did this, <a href="https://zenodo.org/record/2579809#.XHgumMD7SUk">here on Zenodo</a>.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/checking-i-havent-got-my-cells-mixed-up/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
	</channel>
</rss>
