<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>Project &#8211; openlabnotebooks.org</title>
	<atom:link href="https://thesgc.github.io/static-openlabnotebooks/category/project/feed/" rel="self" type="application/rss+xml" />
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<description>A growing team of groundbreaking scientists around the world are now sharing their lab notebooks online</description>
	<lastBuildDate>Fri, 01 Mar 2019 09:51:22 +0000</lastBuildDate>
	<language>en-GB</language>
	<sy:updatePeriod>hourly</sy:updatePeriod>
	<sy:updateFrequency>1</sy:updateFrequency>
	<generator>https://wordpress.org/?v=4.9.9</generator>

<image>
	<url>https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/cropped-lab-jar02-32x32.png</url>
	<title>Project &#8211; openlabnotebooks.org</title>
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<width>32</width>
	<height>32</height>
</image> 
	<item>
		<title>Checking I haven&#8217;t got my cells mixed up&#8230;</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/checking-i-havent-got-my-cells-mixed-up/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/checking-i-havent-got-my-cells-mixed-up/#respond</comments>
		<pubDate>Thu, 28 Feb 2019 18:56:05 +0000</pubDate>
		<dc:creator><![CDATA[Elizabeth Brown]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Brain Tumours]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>
		<category><![CDATA[Liz Brown]]></category>
		<category><![CDATA[Understudied Kinases]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2542</guid>
		<description><![CDATA[Did you know as many as 30% of the cells in labs could be misidentified? And what a nightmare it would be to find out years of your work are just completely wrong because you got your skin cells mixed up with your brain cells&#8230; So to avoid that catastrophe, I took some time to <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/checking-i-havent-got-my-cells-mixed-up/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Did you know as many as 30% of the cells in labs could be misidentified? And what a nightmare it would be to find out years of your work are just completely wrong because you got your skin cells mixed up with your brain cells&#8230;</p>
<p>So to avoid that catastrophe, I took some time to check the DNA sequence of the ACVR1 gene, because for this gene we know what should be mutated (or not) for each of my cell lines. This way I can make sure that I haven&#8217;t mixed them up in the past year (<em>and</em> that they haven&#8217;t picked up new mutations in this important area all by themselves).</p>
<p>All I had to do was collect the genomes of the cells, amplify specifically the ACVR1 gene, send it for overnight sequencing, and then I get a trace like this back from the facility:</p>
<div id="attachment_2544" style="width: 790px" class="wp-caption aligncenter"><a href="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/R206H-sequence-1.png"><img class="wp-image-2544 size-large" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/R206H-sequence-1-1024x396.png" alt="Sequencing trace covering the R206H mutation in ACVR1 found in FOP and DIPG" width="780" height="302" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/R206H-sequence-1-1024x396.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/R206H-sequence-1-300x116.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/R206H-sequence-1-768x297.png 768w" sizes="(max-width: 780px) 100vw, 780px" /></a><p class="wp-caption-text">Sequencing trace covering the R206H mutation in ACVR1 found in FOP and DIPG</p></div>
<p>Each DNA base has one peak, except one that has both a guanine (G) and an adenine (A) DNA base. The adenine, present on one chromosome, causes the change from an arginine (R) to a histidine (H) at the 206th position of the ACVR1 protein i.e. the R206H mutation.</p>
<p>This trace is from the HSJD-DIPG-007 cell line and is the mutation I expected. According to all my sequencing, none of my cell lines have gotten mixed up, yay!</p>
<p>You can read the full details of how I did this, <a href="https://zenodo.org/record/2579809#.XHgumMD7SUk">here on Zenodo</a>.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/checking-i-havent-got-my-cells-mixed-up/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Modelling diseases of the brain through stem cells</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/modelling-diseases-of-the-brain-through-stem-cells/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/modelling-diseases-of-the-brain-through-stem-cells/#respond</comments>
		<pubDate>Thu, 28 Feb 2019 00:41:34 +0000</pubDate>
		<dc:creator><![CDATA[Thomas Durcan]]></dc:creator>
				<category><![CDATA[ALS-Reproducible Antibody Platform (ALS-RAP)]]></category>
		<category><![CDATA[Amyotrophic Lateral Sclerosis (ALS)]]></category>
		<category><![CDATA[iPSC Disease modelling]]></category>
		<category><![CDATA[iPSCs]]></category>
		<category><![CDATA[Parkinson's disease and neurodevelopmental disorders]]></category>
		<category><![CDATA[Thomas M. Durcan]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2536</guid>
		<description><![CDATA[Hello SGC members and to the millions of scientific readers across the globe, My name is Thomas Durcan, and I am an assistant professor at the Montreal Neurological Institute (MNI) and McGill University, where I also lead the SGC tissue platform in Montreal. You can call me the stem cell guy, as everything I and my <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/modelling-diseases-of-the-brain-through-stem-cells/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Hello SGC members and to the millions of scientific readers across the globe,</p>
<p>My name is Thomas Durcan, and I am an assistant professor at the <a href="https://www.mcgill.ca/neuro/">Montreal Neurological Institute (MNI) </a>and <a href="https://www.mcgill.ca/">McGill University</a>, where I also lead the <a href="https://ultra-dd.org/tissue-platforms">SGC tissue platform</a> in Montreal. You can call me the stem cell guy, as everything I and my team does is centered on human stem cells, developing disease models for understanding and treating diseases of the brain. So who am I, well let me introduce myself. I hail from Dublin, Ireland where I was born and raised. I wanted to clone dinosaurs from the moment I saw Jurassic park, so I did everything in my power to become a scientist. During my time at <a href="https://www.ucd.ie/">University College Dublin</a>, I looked at the DNA of plants, although no breakthroughs emerged in my quest for dinosaurs. With a degree in hand, I headed across to North America, first to the USA and the <a href="https://www.nd.edu/">University of Notre Dame</a>, and then onto colder climes in Canada, settling in Montreal, Quebec. It is here in Montreal, where I have learned to survive winter, to sample the finest bagels, to develop a passion for wine and where I got the opportunity to understand the cellular and molecular causes of Parkinson&#8217;s disease working in the lab of <a href="https://www.mcgill.ca/neuro/research/researchers/fon">Dr. Edward Fon</a>.</p>
<p>However, all my studies were with regular old Hela or HEK293T cells, cells that are clearly not neuronal. When studying a disease of the brain, it would make sense to actually study a human neuron, but these aren&#8217;t readily accessible. I can&#8217;t go up to someone and ask them for their neurons as what you are born with is kind of what you need. So for years, our field was stuck, we had some advances, but it was always in a Hela-type cell, or animal model, which often lacked many of the features of the human disease itself. Then the breakthrough came, in 2006 with the Yamanaka factors. A set of four transcription factors that when introduced into skin cells from a mouse or human, brought them back to the beginning. It reprogrammed the cell, and made it pluripotent, coining the term induced pluripotent stem cell or iPSC. By reprogramming the skin cell, it removed all epigenetics and created a stem cell, that was capable of generating any tissue or cell type within the human body. All it needed, was the correct set of factors to do so.</p>
<p>SInce this seminal publication, the field has grown exponentially and I started working on stem cells in 2014, leading to the creation of the <a href="https://www.mcgill.ca/ipsquebec-mni/">iPSC platform</a> at the MNI, under the direction of Dr. Edward Fon, and myself as group leader, and in partnership with the iPSC platform at the Universite of Laval, under the leadership of <a href="http://www.jpuymiratlab.crchul.ulaval.ca/">Jack Puymerat</a>.</p>
<p>When we began, we were three people, and fast forward to 2019, our group is now over 30 full time staff and students, and growing every month, with the iPSC platform starting its transition this year into the Early Drug Discovery Unit (Neuro-EDDU- further details to follow). With iPSCs at the core of our mission we are now engaged in many early drug discovery projects with researchers, biotech and large pharma, generating neurons for studying Parkinson&#8217;s disease, Amyotrophic Lateral Sclerosis (ALS) and neurodevelopmental biology. Not only are we generating neurons on 2D dishes, but we have been growing minibrains by the thousands, with these 3D neuronal organoids mimicking the human brain more closely then anything seen with 2D cultures. At the same time, we have been fortunate that CRISPR genome editing has emerged, providing us the tools to edit a gene at the base pair level. This technology is at the core of our involvement with the <a href="https://als-rap.org/">ALS-RAP</a> project, in which we are making a panel of over 30 KO cells against ALS-associated genes, using these cells to validate antibodies. With all these tools now set up within the group, new big data approaches are being explored by members of my team, both for image based and MultiOmics based analysis of neurons and other disease-relevant cells generated from patient iPSCs.</p>
<p>This is just a snap-shot into the type of things we are doing, and we are now a step closer to making dinosaur stem cells, fulfilling a life long dream of mine, while also advancing new therapies into the clinic that we hope will one day treat these devastating disorders. Yet, we can&#8217;t do this alone, which is why for my blog I will be starting to make available all our SOPs as part of the open science mission of the MNI and TOSI. All our tips and tricks will be posted on a regular basis, from how to grow an iPSC all the way to to how to make a minibrain will be added to the blog over the coming months, to show everyone how to work with iPSCs. In parallel, as one of the three leads on the <a href="https://als-rap.org/">ALS-RAP</a> project, I will be posting updates on our antibody validation efforts and progress on this project, keeping people up to date on antibodies we are testing, and pointing to ones you should consider working with.</p>
<p>I look forward to opening the doors to our group, and hopefully over the coming months you will enjoy hearing about the work being done in Montreal, and at the MNI on stem cells and antibody validation. Next time, I will start by introducing our SOPs for working with iPSCs, starting with how to thaw a frozen vial of iPSCs. To talk or to lean more  feel free to reach me through <a href="https://www.linkedin.com/in/thomas-durcan-18593b23/">linkedin</a>.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/modelling-diseases-of-the-brain-through-stem-cells/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Quinazoline Series: Synthesis</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/quinazoline-series-synthesis/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/quinazoline-series-synthesis/#respond</comments>
		<pubDate>Sun, 24 Feb 2019 22:58:15 +0000</pubDate>
		<dc:creator><![CDATA[Benjamin Eduful]]></dc:creator>
				<category><![CDATA[Ben Eduful]]></category>
		<category><![CDATA[Understudied Kinases]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2434</guid>
		<description><![CDATA[We have received some very interesting biological data for our CaMKK2 inhibitor series (furopyridine, Azaindole and quinoline) and I will be sharing the data in the coming weeks, but before that, I wanted to share my synthetic efforts towards our quinoline series. Synthesis of Quinoline series: Starting with 6-Bromo-4-chloroquinoline, a two (2) step synthetic scheme <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/quinazoline-series-synthesis/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p style="text-align: justify;">We have received some very interesting biological data for our CaMKK2 inhibitor series (furopyridine, Azaindole and quinoline) and I will be sharing the data in the coming weeks, but before that, I wanted to share my synthetic efforts towards our quinoline series.</p>
<p><strong>Synthesis of Quinoline series</strong>:</p>
<p style="text-align: justify;">Starting with 6-Bromo-4-chloroquinoline, a two (2) step synthetic scheme was devised where the bromine (Br) synthetic handle was substituted first (with an appropriate boronic acid/ester), followed by the chlorine (Cl) , <strong>Figure 1</strong>, after testing many different reaction conditions for selectivity.</p>
<p style="text-align: center;"><strong>Figure 1: Synthesis of quinoline series</strong></p>
<p><img class=" wp-image-2526 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/Quinoline-1-300x67.png" alt="" width="416" height="93" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/Quinoline-1-300x67.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/Quinoline-1.png 569w" sizes="(max-width: 416px) 100vw, 416px" /></p>
<p style="text-align: justify;">A number of analogs were thus synthesized using the above synthetic strategy, for example, synthesis of the quinoline analog: 2-cyclopentyl-4-(6-(4-(methylsulfonyl)phenyl)quinolin-4-yl)benzoic acid was achieved using this method, <strong>Scheme 1</strong>.</p>
<p style="text-align: center;"><strong>Scheme 1: Synthesis of quinoline analog</strong></p>
<p><img class="wp-image-2527 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/Quinoline-2-300x88.png" alt="" width="701" height="206" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/Quinoline-2-300x88.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/Quinoline-2-768x226.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/Quinoline-2.png 936w" sizes="(max-width: 701px) 100vw, 701px" /></p>
<p style="text-align: justify;">Although the synthetic strategy above led to the synthesis of some analogs, it was not without challenges. For instance, selectivity for the Br was not always achieved as disubstituted products were common in some cases due to the activated 4-position of the Cl.</p>
<p style="text-align: justify;">In order to overcome these challenges and to allow for rapid synthesis of different analogs over a short period of time, a strategy to first install the cyclopentyl derivative was sort after. In the synthetic scheme below, <strong>Scheme 2</strong>, 6-chloroquinolin-4-ol, <strong>101</strong>, was treated with triflic anhydride in the presence of DIPEA in DCM to obtain 6-chloroquinolin-4-yl trifluoromethanesulfonate, <strong>102</strong>. This compound was able to undergo the Suzuki cross coupling reaction to obtain compound <strong>103</strong>. With this compound in hand, a number of different analogs are being generated in our search for active CaMKK2 chemical probes, for example compound <strong>105</strong>, <strong>Scheme 2</strong>.</p>
<p style="text-align: center;"><strong>Scheme 2: Synthesis of quinoline analog via new route</strong></p>
<p><img class=" wp-image-2528 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/Quinoline-3-300x71.png" alt="" width="758" height="179" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/Quinoline-3-300x71.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/Quinoline-3-768x182.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/Quinoline-3-1024x242.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/Quinoline-3.png 1282w" sizes="(max-width: 758px) 100vw, 758px" /></p>
<p>We are also looking to exploring the 7-position of the quinoline with similar substitutions to the 6-position (<strong>Figure 2</strong>), quinoline derivative 1. Other anticipated analogs will include O-alkylations (quinoline derivatives 2 and 3), N-alkylations or peptide couplings or Hartwig-Buchwald aminations (quinolines derivatives 4 and 5) etc. Details of such reactions will be communicated in due course.</p>
<p style="text-align: center;"><strong>Figure 2: Exploration of other quinoline analogs</strong></p>
<p><img class=" wp-image-2529 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/Quinoline-4-300x81.png" alt="" width="504" height="136" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/Quinoline-4-300x81.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/Quinoline-4-768x207.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/Quinoline-4.png 858w" sizes="(max-width: 504px) 100vw, 504px" /></p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/quinazoline-series-synthesis/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>ZFYVE9C purification and association test with TGFBR1 and TGFBR2</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/zfyve9c-purification-and-association-test-with-tgfbr1-and-tgfbr2/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/zfyve9c-purification-and-association-test-with-tgfbr1-and-tgfbr2/#comments</comments>
		<pubDate>Fri, 22 Feb 2019 14:05:47 +0000</pubDate>
		<dc:creator><![CDATA[Ellie Williams]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Ellie Williams]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2515</guid>
		<description><![CDATA[ZFYVE9C (Otherwise known as SARA) is a scaffold protein involved in the association of various protein components as part of the TGFB signalling pathway. The equivalent in the BMP signalling pathway is a protein known as Endofin so studying SARA (which is slightly easier) might give us useful insights for both proteins. For more details <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/zfyve9c-purification-and-association-test-with-tgfbr1-and-tgfbr2/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>ZFYVE9C (Otherwise known as SARA) is a scaffold protein involved in the association of various protein components as part of the TGFB signalling pathway. The equivalent in the BMP signalling pathway is a protein known as Endofin so studying SARA (which is slightly easier) might give us useful insights for both proteins. For more details on this project please see <a href="https://thesgc.github.io/static-openlabnotebooks/a-summary-of-my-alk2-project-goals/">this blog post.</a></p>
<p>One of the unanswered questions about SARA is what does one of the major domains that it has actually do? (The domain is known as a DUF domain, or a Domain of Unknown Function!) We know the structure but that hasn’t given us any clues as to its function. One thought was that it might be involved in recruiting type I or type II receptors close to their binding partners such as one of the SMADs to allow signal propagation.</p>
<p>I purified the DUF domain of SARA and mixed it with either excess TGFBR1 or excess TGFBR2 (the two relevant receptors in the TGFB pathway). I tested the receptors in both the unphosphorylated (inactive) and phosphorylated (active) states. After incubating the samples for 30 minutes while concentrating the volume back down to &lt;5ml, I ran them down a gel filtration column to see if the two proteins bound together sufficiently to shift the peaks coming off the column which would indicate they were acting as a complex of larger mass.</p>
<p><img class="alignnone size-large wp-image-2516" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/gf_trace_zfyve9c-1024x491.png" alt="" width="780" height="374" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/gf_trace_zfyve9c-1024x491.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/gf_trace_zfyve9c-300x144.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/gf_trace_zfyve9c-768x368.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/gf_trace_zfyve9c.png 1578w" sizes="(max-width: 780px) 100vw, 780px" /></p>
<p><em>Overlay of the UV traces from all the size exclusion gel filtration runs to show the relative elution volumes of the peaks confirming no shift in the ZFYVE9C peak consistent with no complex being formed.</em></p>
<p>&nbsp;</p>
<p>Unfortunately I didn’t see any shift in the peak for SARA suggesting that either the two proteins don’t bind or that the affinity is weak enough that they fall apart under the conditions of the gel filtration column.</p>
<p>The next thing to try will be to purify the proteins again but not to cleave the tag on one of the binding partners (either the receptors or the SARA protein) and try to do a pull down experiment to test whether this shows any association or not.</p>
<p>&nbsp;</p>
<p>You can find more details over at<a href="https://doi.org/10.5281/zenodo.2575581"> Zenodo.</a></p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/zfyve9c-purification-and-association-test-with-tgfbr1-and-tgfbr2/feed/</wfw:commentRss>
		<slash:comments>2</slash:comments>
		</item>
		<item>
		<title>Three new ALK2 structures, one of them not the one I thought!</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/three-new-alk2-structures-one-of-them-not-the-one-i-thought/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/three-new-alk2-structures-one-of-them-not-the-one-i-thought/#respond</comments>
		<pubDate>Tue, 19 Feb 2019 17:46:47 +0000</pubDate>
		<dc:creator><![CDATA[Roslin Adamson]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Brain Tumours]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>
		<category><![CDATA[Roslin Adamson]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2507</guid>
		<description><![CDATA[I’ve not posted in a while, mainly because things have been really busy, but good busy, as I now have 3 new datasets/structures, of ALK2 with M4K2118, M4K2149 and, amazingly, M4K2009. As you know, this is our lead candidate, which until now has not behaved very well in crystal trials, and it hasn’t been possible <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/three-new-alk2-structures-one-of-them-not-the-one-i-thought/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>I’ve not posted in a while, mainly because things have been really busy, but good busy, as I now have 3 new datasets/structures, of ALK2 with M4K2118, M4K2149 and, amazingly, M4K2009. As you know, this is our lead candidate, which until now has not behaved very well in crystal trials, and it hasn’t been possible to get decent crystals. In my last post I told you how I had somehow managed to mix up the TNIK and ACVR1/ALK2 tubes when I was setting up the plates, and accidentally put ACVR1 into a coarse screen and TNIK into a fine screen for ACVR1. Well, after I got lovely big crystals for what I thought was ACVR1 with M4K3003, I sent them to the synchrotron, got beautiful data to 1.4 Å, phased it, and lo and behold! It was M4K2009 snugly fitted into the binding pocket (see the ligand density after molecular replacement below). Of course, I didn’t notice this was a trimethoxy containing compound (how??) before I excitedly emailed everyone to tell them I had M4K3003, and then had to retract … However, the story ends well as M4K2009 is really important.</p>
<p><img class="aligncenter  wp-image-2511" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/M4k2009-1024x576.jpg" alt="" width="904" height="509" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/M4k2009-1024x576.jpg 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/M4k2009-300x169.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/M4k2009-768x432.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/M4k2009.jpg 1280w" sizes="(max-width: 904px) 100vw, 904px" /></p>
<p>Additionally, I got data for M4K2149 (~2.6 Å), one of the amide head group containing compounds we’re interested to see, and M4K2118 (~1.4 Å), which is doing well in some of the other assays too. I’ve put the .mtz files <a href="https://zenodo.org/record/2573142#.XGw9g-j7SUk">here</a> on Zenodo if you want to play with high resolution data, and some images below showing the ligand density and the ligands binding in the pocket. The amide head group of M4K2149 appears to preserve the water network in the pocket that we’ve observed in other structures, but as the resolution is too low for waters, we can’t tell til we get higher resolution data. Which hopefully will be this week, as I’ve grown some really pretty crystals for M4K2149 that I hope will diffract better. These appeared quite by accident again, as I’d designed a fine screen around the D5 well condition of my mega composite screen, but neglected to notice when I validated the screen that I’d put 10 x less ammonium sulphate in one row than intended – should have been 0.9 M, but ended up being 0.09 M. I should probably start calling myself ‘the accidental crystallographer’ at this rate.</p>
<p><img class="aligncenter  wp-image-2509" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/M4K2118-2149-1024x576.png" alt="" width="883" height="497" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/M4K2118-2149-1024x576.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/M4K2118-2149-300x169.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/M4K2118-2149-768x432.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/M4K2118-2149.png 1280w" sizes="(max-width: 883px) 100vw, 883px" /></p>
<p>In the meantime, I’ve got some small crystals for M4K3007, another structure we really want, and some pretty ones for M4K2184, another amide head group compound. I’ve followed up the M4K3007, just waiting to see if anything appears, and will send the tiny ones to the synchrotron while waiting.</p>
<p><img class="aligncenter  wp-image-2508" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/crystal-pics-1024x576.png" alt="" width="849" height="478" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/crystal-pics-1024x576.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/crystal-pics-300x169.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/crystal-pics-768x432.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/crystal-pics.png 1280w" sizes="(max-width: 849px) 100vw, 849px" /></p>
<p>I’m also very happy to say that our new M4K2117 crystal system for XChem fragment screening has worked very nicely so far, in the sense that the crystals are large and relatively robust and can handle soaking in low salt conditions as well as compounds with DMSO for a few hours. They then still diffract to around 1.2-1.8 Å, consistently in the same space group, C 1 2 1. This is a vast improvement over the previous system with LDN-193189. We did a solvent characterisation and pre-screen a couple of weeks ago with 100 crystals, and of the 20 or so hits we got, none was in the allosteric pocket we are looking for. This could be because of the 1.4 M ammonium sulphate and 0.2 M Na/K tartrate I have in there, which might prevent access of fragments to the pocket. Thus, using the remarkable shifter that Nathan Wright at SGC designed, we’ll be transferring loads of crystals from grow plates to low salt soak plates before doing the compound soaks this week, with the same 100 compounds, in order to check if the hit rate/location changes.</p>
<p>I will mention though, for anyone who may never have noticed this, the DMSO concentration in the protein solution for crystallisation is very important! I had previously been using Batch 1 of M4K2117 for my co-crystallisation, but it ran out so I used Batch 2, which was a 50 mM stock, as opposed to the previous 25 mM stock. The plates I set up with this, the same in every other way, gave showers of rather ugly, crunchy, small crystals. So I tried a few things – diluted the stock to 25 mM in DMSO and set up plates with that, with both lower and higher concentrations of protein (8 and 14 mg/ml along with the usual 10-11 mg/ml), as well as using the stock as is, with lower and higher protein concentrations. The lower stock concentrations brought back the big, perfect, beautiful crystals from before, the higher stock, no matter the protein concentration, gave ugly chunks and showers. The final concentration of compound was the same in all, so clearly the DMSO percentage is a very important factor.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/three-new-alk2-structures-one-of-them-not-the-one-i-thought/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Collecting HAO1 crystal structures with follow-up compounds based on fragment hits</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/collecting-hao1-crystal-structures-with-follow-up-compounds-based-on-fragment-hits/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/collecting-hao1-crystal-structures-with-follow-up-compounds-based-on-fragment-hits/#respond</comments>
		<pubDate>Tue, 12 Feb 2019 16:32:27 +0000</pubDate>
		<dc:creator><![CDATA[Sabrina Mackinnon]]></dc:creator>
				<category><![CDATA[HAO1]]></category>
		<category><![CDATA[Primary Hyperoxaluria]]></category>
		<category><![CDATA[Sabrina Mackinnon]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2488</guid>
		<description><![CDATA[A belated Happy New Year everyone!  It has been a good while since my last post so I wanted to update you on what I have been spending 2019 on so far. After identifying some really promising follow-up compounds in my activity assay, the next step was to find out how they bind to HAO1, <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/collecting-hao1-crystal-structures-with-follow-up-compounds-based-on-fragment-hits/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>A belated Happy New Year everyone!  It has been a good while since my last post so I wanted to update you on what I have been spending 2019 on so far.</p>
<p>After identifying some really promising follow-up compounds in my activity assay, the next step was to find out how they bind to HAO1, as illustrated in figure 1. Specifically I want to know how different chemical constituents of the new compounds interact with and/or cause conformational changes to amino acids within the HAO1 structure. By understanding which groups improve binding (e.g. that stabilise the bound state by interacting with nearby amino acids), which group increase inhibition (e.g. by disrupting active site residues like Trp110) and which groups are not required for the compound to work (i.e. no apparent interaction with the protein), we can rationally design compounds that combine the most effective chemical groups to make more potent inhibitors.</p>
<p>In practical terms, this means that I have made &gt;250 HAO1 crystals and soaked each of my follow up compounds twice (127 compounds times 2 crystals each = 254 crystals soaked). This duplication helps ensure complete coverage of all compounds (e.g. if one replicate is not mounted/collected correctly) and also improves confidence in my results i.e. if I see the same compound making the same interactions in two independent structures, I feel confident that the interactions I see are significant changes dependent on compound-binding.</p>
<p><img class="aligncenter size-large wp-image-2489" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/post8_figure1-1024x935.png" alt="" width="780" height="712" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/post8_figure1-1024x935.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/post8_figure1-300x274.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/post8_figure1-768x702.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/post8_figure1.png 1144w" sizes="(max-width: 780px) 100vw, 780px" /></p>
<p><strong><em>Figure 1: Experimental workflow for characterising follow-up compounds by x-ray crystallography. </em></strong><em>Follow-up compounds from fragments 1, 2 and 5 are described <a href="https://thesgc.github.io/static-openlabnotebooks/screening-follow-up-compounds-from-active-site-and-gating-loop-hao1-fragment-hits/">here</a>, whereas those from fragment 6 are described <a href="https://thesgc.github.io/static-openlabnotebooks/synthesis-and-testing-of-follow-up-compounds-based-on-hao1-fragment-6-bound-at-the-tetramer-interface/">here</a>. Completed experiments are in light blue boxes and ongoing experiment is in dark blue.</em></p>
<p>I have collected a total of 227 good datasets (resolution range of 1.2 to 2.2 Å, 90% better than 1.7 Å) and now I am hard at work processing these through the <a href="http://journals.iucr.org/d/issues/2017/03/00/ba5267/">XChemExplorer pipeline</a>, ready to tell you all about the compound-bound structures in the next few weeks.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/collecting-hao1-crystal-structures-with-follow-up-compounds-based-on-fragment-hits/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Last post on Split-SCN1A</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/last-post-on-split-scn1a/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/last-post-on-split-scn1a/#respond</comments>
		<pubDate>Sun, 10 Feb 2019 22:47:17 +0000</pubDate>
		<dc:creator><![CDATA[Wen Yih Aw]]></dc:creator>
				<category><![CDATA[Dravet syndrome]]></category>
		<category><![CDATA[Gene therapy for Dravet Syndrome]]></category>
		<category><![CDATA[Wen Yih Aw]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2452</guid>
		<description><![CDATA[The idea for split-intein based SCN1A reconstitution started when I first learned of the split-Cas9 system. Long story short, packaging Cas9 with a guideRNA sequence into an AAV vector is pushing the packaging capacity of AAV vector. To circumvent this issue, the authors design a split-Cas9 system where the coding sequence for Cas9 is split <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/last-post-on-split-scn1a/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>The idea for split-intein based SCN1A reconstitution started when I first learned of the split-Cas9 system. Long story short, packaging Cas9 with a guideRNA sequence into an AAV vector is pushing the packaging capacity of AAV vector. To circumvent this issue, the authors design a split-Cas9 system where the coding sequence for Cas9 is split into two fragments, each fused to a split-inteins sequence. When co-expressed in cells, the split-inteins can function as glue joining two protein fragments together.</p>
<p>In an attempt to “shrink” SCN1A to be packageable in AAV vectors, I ran into several design and cloning issues, ultimately leading to the decision to terminate this project. Cas9 is a well-studied cytoplasmic protein with structural information while SCN1A is a large multi-pass transmembrane protein that weaves in and out of the plasma membrane. While success packaging and application of the split-Cas9 system for gene editing is encouraging, my attempt to translate similar design to SCN1A, without structural information, proves to be an oversight. Design aside, the coding sequence for human SCN1A is difficult to maintain. The SCN1A cDNA sequence tends to accumulate deletions and rearranges when transformed and propagated in E. coli. One way to improve plasmid stability and reduce unwanted rearrangement is by propagating plasmid in E. coli strains with recA1 genotype at room temperature. recA1 confers a loss of function in RecA1, a protein that is essential for homologous recombinational DNA repair in E. coli. While this strategy helps to alleviate the problem, it is time-consuming and eventually, I decided to make use of a codon-optimized human SCN1A cDNA as the cloning template for split-SCN1A constructs. While this solves the plasmid stability issue, the codon-optimized cDNA has not been well characterized and while the proteins expressed reconstituted, they failed to traffic to the correct cellular compartment. These raise an issue that needs to be addressed – how to propagate mutation-free gene therapy vectors with SCN1A sequence in large scale, have any of us managed to move the project to treatment pipeline.</p>
<p>Thank you all for reading. While this project didn&#8217;t work out as planned, I truly believe that this is an exciting time for Dravet Syndrome. With recent positive Fenfluramine clinical trial results, exciting new drugs in the pipeline, and cross-continental efforts in pushing various gene therapy approaches for treating Dravet, I am hopeful that we can see these exciting results moving into medical treatment in the foreseeable future.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/last-post-on-split-scn1a/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>A summary of my ALK2 project goals.</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/a-summary-of-my-alk2-project-goals/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/a-summary-of-my-alk2-project-goals/#respond</comments>
		<pubDate>Tue, 05 Feb 2019 17:01:21 +0000</pubDate>
		<dc:creator><![CDATA[Ellie Williams]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Ellie Williams]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2439</guid>
		<description><![CDATA[I thought I’d spend a blog post just talking about the various paths of my research and try and tie it all together. A lot of my experiments do dot around these various arms of investigation and so one experimental post doesn’t necessarily tie in directly with the previous one or the next one and <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/a-summary-of-my-alk2-project-goals/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>I thought I’d spend a blog post just talking about the various paths of my research and try and tie it all together. A lot of my experiments do dot around these various arms of investigation and so one experimental post doesn’t necessarily tie in directly with the previous one or the next one and I thought it might be a good idea to explain why and summarise where I’m up to with each strand.</p>
<p>My main focus of research is to investigate the cause of the disease fibrodysplasia ossificans progressiva (FOP). We know that it’s caused by a mutation in ALK2 which leads to excessive bone formation. This also nicely dovetails with research done into DIPG where 25% of cases share a mutation with FOP in ALK2.</p>
<p>My research can be broken down in to three main strands:</p>
<ul>
<li>The first is looking at characterising the binding of ligands to ALK2 in order to optimise binding and work towards finding a potent and specific inhibitor. This is the focus of a lot of the M4K work that Jong Fu and Ros are doing and I’ve been doing a little bit of that as well but it’s not the main focus of my work. This endeavour is going fairly well – despite setbacks relating to protein expression (which are not entirely solved – we now have regained expression of the kinase domain alone but we still don’t have expression of the kinase domain and GS loop construct) and Ros has a number of structures that she’s working on at the moment that have been recently collected. Alas I’ve not had as much luck with new structures recently. We frequently get new deliveries of compounds from M4K to test – Jong Fu looks at them in cells while Ros and I try and crystallise the most promising candidates with varying degrees of success. Ros and I have also attempted XChem fragment screening which is where we use high throughput crystallography to identify small weakly bound fragments that might identify new leads for drug design or in our case, help us identify drugs that might bind to a second allosteric site within ALK2 away from the main ligand binding pocket. An XChem update will be coming soon from either Ros or myself as I’ve recently started helping her with this aspect of the work and so have been shadowing her on our last couple of visits to Diamond.</li>
</ul>
<p>&nbsp;</p>
<ul>
<li>The second is investigating the interactions of the type I receptor (ALK2) with its partnering type II in order to activate SMAD 1/5/8. This covers looking at receptor cross reactivity, SMAD cross reactivity, and the influence of mutations on this. This also involves looking at crystal structures of the mutants where possible, attempting to make complexes of the type I and type II receptors in order to try and crystallise them and probing the activities of the proteins and the various disease mutants. Covered in this is everything I’ve done using western blots and mass spec to identify the dependencies of this complex as required for SMAD phosphorylation. So far I know that the mutants are more active than the wild type (with R206H being the most active ALK2 mutation), that the type II must be present but does not need to be active to allow for SMAD activation and that increased activity is not solely dependent upon the removal of inhibition by FKBP12. Covered in this is also the work looking at mapping the phosphorylation sites in ALK2 to try and identify if some of these sites are more critical than others with relation to SMAD activation. The cloning I recently talked about is part of this investigation, making mutations to remove certain phosphorylation sites and comparing the activity of these with the wild type. As soon as these constructs have been cloned into suitable expression vectors I’ll be starting that experiment too. Another aspect of this is looking at the association of other proteins with the receptors, such as the possible link between them and XIAP as I talked about in one earlier blog post.</li>
</ul>
<p>&nbsp;</p>
<ul>
<li>The third aspect is looking other proteins related to the ALK2 pathway such as Endofin. Endofin is a membrane associated protein with multiple domains including a SMAD binding domain and large areas of unknown function. It is involved with the co-localisation of various parts of the pathway and appears to act as something of a scaffold protein to hold things in close proximity for signal propagation. The equivalent protein in the TGFβ pathway is called SARA and appears to have a similar function in that pathway. Some time ago I solved the structure for a domain of unknown function within SARA (4BKW) which contained a novel fold and showed structural (but not sequence) similarity to a protein called SUFU, which is part of the Hedgehog signalling pathway. We still don’t know what this protein and domain does or what it binds to or how it functions and so this is another area of investigation. If we can uncover the function of SARA we can also try and uncover the function of Endofin and this might provide a better understanding of the way FOP mutations work and help us see if there are any other ways we can target the pathway that are not exclusively based on inhibiting ALK2 activity. However given that the DUF domain has very little information about it, our investigations into what it does have been limited thus far. My goals with this are to try and probe the role of the DUF domain looking for whether it binds to either of the type I or type II receptors, and if so which parts of the receptors, or if it associates with a different protein all together. Given we know very little about what it does and the structure offers few clues (the lack of sequence similarity to SUFU means that although we can predict a possible binding mode, the nature of what binds there is very hard to determine as there are no conserved residues) this is quite a difficult problem to approach.</li>
</ul>
<p><img class="alignnone size-large wp-image-2440" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/summary-1024x606.png" alt="" width="780" height="462" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/summary-1024x606.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/summary-300x177.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/summary-768x454.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/summary.png 1324w" sizes="(max-width: 780px) 100vw, 780px" /></p>
<p>&nbsp;</p>
<p>My work moves between these three strands depending on how much and which proteins I’ve got available to me &#8211; do I have any in the freezer already purified or do I need to make fresh? And that’s if I have the pellets in the freezer of the expressed protein in the first place. It also varies depending on whether we have new compounds we’re interested in testing immediately from either M4K or other collaborators, or whether I’ve got data I need to analyse from previous successful crystallisation attempts. Sometimes I’ll be looking at my own structural data and sometimes I’ll be looking at someone else’s structure which we call ‘proofreading’ to double check it for errors and make sure nothing has been missed. Sometimes we’ll have time at Diamond and so everything else has to be put on hold in order to make use of that visit (such as XChem trips) and sometimes there’s a lot of data analysis to do on other experiments &#8211; writing it up and understanding what the data is telling us is just as important for progress as the experiments themselves.</p>
<p>Having the three strands means I can move from one to another as each strand has its own peaks and lulls in activity at different times. I’m afraid it does make for a slightly more disjointed blog journey but I hope you’ll bear with me through it all as I try and make sense of the problems I’m investigating.</p>
<p>I’ll try and post again sooner rather than later on what I’ve been doing in January as it has been really rather bitty and frustrating in terms of things actually working but hopefully February will bring with it some more productive experiments.</p>
<p>&nbsp;</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/a-summary-of-my-alk2-project-goals/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Welcome to the SGC-Karolinska side of ALS-RAP: the scientists, the screening, and the antibody selections</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/welcome-to-the-sgc-karolinska-side-of-als-rap-the-scientists-the-screening-and-the-antibody-selections/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/welcome-to-the-sgc-karolinska-side-of-als-rap-the-scientists-the-screening-and-the-antibody-selections/#respond</comments>
		<pubDate>Thu, 31 Jan 2019 23:10:29 +0000</pubDate>
		<dc:creator><![CDATA[Carolyn Marks]]></dc:creator>
				<category><![CDATA[ALS-Reproducible Antibody Platform (ALS-RAP)]]></category>
		<category><![CDATA[Amyotrophic Lateral Sclerosis (ALS)]]></category>
		<category><![CDATA[Carolyn Marks]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2419</guid>
		<description><![CDATA[Hello fellow scientists and open lab notebook readers! I’m Carolyn Marks and I’m working in the ALS-RAP partnership with Susanne Gräslund, who leads the Recombinant Antibodies Group at SGC-Karolinska in Stockholm, Sweden. SGC-Karolinska has had a long-standing collaboration with Helena Persson Lotsholm, Director of the Human Antibody Therapeutics Facility within the Drug Discovery and Development <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/welcome-to-the-sgc-karolinska-side-of-als-rap-the-scientists-the-screening-and-the-antibody-selections/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p style="text-align: justify;">Hello fellow scientists and open lab notebook readers!</p>
<p style="text-align: justify;">I’m Carolyn Marks and I’m working in the ALS-RAP partnership with <a href="https://thesgc.org/profile/karolinska/sgraslund">Susanne Gräslund</a>, who leads the Recombinant Antibodies Group at <a href="https://www.thesgc.org/karolinska">SGC-Karolinska</a> in Stockholm, Sweden. SGC-Karolinska has had a long-standing collaboration with <a href="https://www.scilifelab.se/facilities/human-antibody-therapeutics/">Helena Persson Lotsholm</a>, Director of the Human Antibody Therapeutics Facility within the <a href="https://www.scilifelab.se/platforms/ddd">Drug Discovery and Development Platform at Science for Life Laboratory in Stockholm</a>. I am fortunate to work with and learn from several experts to generate the highest quality antibodies for the ALS-Reproducible Antibody Platform. I joined the <a href="https://www.als-rap.org/team">ALS-RAP team</a> in September of last year and from here on out I’ll be sharing live updates of our work with this community.</p>
<p style="text-align: justify;">Perhaps I should tell you a little bit about me, what we do at SGC-Karolinska, and how I got here. I’m a wet lab kind of gal and a neuroscientist at heart with a lot of years of experience under my belt. The methods that I use here at SGC-Karolinska for the ALS-RAP are broad and range from phage display technology to cloning to high-throughput screening to patient-derived cell-based assays. Actually, many of the methods that I&#8217;m using now to generate and validate recombinant antibodies for the ALS-RAP, I used for many years to study neurodegenerative diseases, including ALS.</p>
<p style="text-align: justify;">I´m originally from Texas where my love of neurophysiology began, and where I was initially trained as an electrophysiologist. From there, I deferred medical school and took an internship at NINDS at the NIH to have a break before med school began. My 1-year internship turned into two, and I quickly understood why everyone at NIH kept telling me “Carolyn, you reek of a PhD!” I soon found the NIH-Karolinska Graduate Partnerships Program and hopped on a plane. That&#8217;s how the girl from Texas ended up in Sweden. I received my PhD in 2012 from the NIH-Karolinska Collaborative Doctoral Program in Neuroscience; and I recently completed my Post-Doc, also at Karolinska, characterizing the ubiquitin proteasome system as therapeutic targets in neurodegenerative diseases.</p>
<p style="text-align: justify;">How did I find my way to the SGC, you might wonder? In my research over the years, I have experienced first-hand the need for reliable high-quality antibodies – and the lack thereof, as well. I hate that we as scientists are limited by the tools that companies provide. These limitations are crippling our research. Sadly, the ones who suffer the most are the patients. This was my motivation for joining SGC-Karolinska and particularly this project. I wanted to make better tools for the research community. Consequently, I believe that better research tools will yield better science and hopefully better therapeutics for the patients who need it. That&#8217;s why I&#8217;m here.</p>
<p style="text-align: justify;">How does the <a href="https://www.als-rap.org/overview">ALS-RAP</a> work? Our team at Karolinska is only 1 of many. We work together primarily with scientists at <a href="https://thesgc.github.io/static-openlabnotebooks/introducing-the-team-the-gileadi-group-in-oxford/">Oxford</a> and in <a href="https://www.mcgill.ca/neuro/research/researchers/thomas-m-durcan-phd">Montreal</a>. The ALS-RAP was created by three major ALS charities from Canada, the UK, and the US. The long-term goal of this partnership is to provide the ALS research community with reproducible, thoroughly validated antibodies for selected ALS-related proteins. This includes testing existing commercial antibodies as well as generating new ones. The work is done in three different labs, as recently outlined by my fellow ALS-RAP partner and open-lab-notebooker, <a href="https://thesgc.github.io/static-openlabnotebooks/validated-antibodies-for-als-research-the-als-rap-project/">Opher Gileadi</a>.</p>
<p style="text-align: justify;">In the Stockholm team, we use phage display technology to create recombinant antibodies. Don´t know what phage display is? The inventors of this technique were actually awarded the <a href="https://www.nobelprize.org/prizes/chemistry/2018/summary/">Nobel Prize in Chemistry in 2018</a>. While this technology has many uses, we use phage display to generate recombinant antibodies. How does it work you might wonder? When we receive antigens of our ALS proteins from our friends in <a href="https://www.ndm.ox.ac.uk/principal-investigators/researcher/opher-gileadi">Oxford</a>, we then use the phage display technique to select out antibodies that binds these antigens. We use extensive libraries of human antibody candidates where high-quality binders can be obtained with the phage display method. We then perform several rounds of screening and selection to ensure that we get the best possible antibodies. We then evaluate their specificity and affinity and finally we validate the antibodies using immunoprecipitation followed by mass spectrometry. It sounds simple, but trust me it’s a thorough process! Last but not least, we will release the sequence of the best antibody for each target on our website and make it available to anyone who wants to use it. Our collaborators in Montreal are creating CRISPR-knockouts of these targets in both standard cell lines and iPSCs for further validation and application specificity.</p>
<p style="text-align: justify;">We have a really exciting <a href="https://als-rap.org/target-list">target list</a>, built by the help of our ALS-RAP advisory board. Check it out and see if your favorite protein or that much needed antibody is on our list. If not, click <a href="https://als-rap.org/your-input">here</a> and request it. So far, we have generated and selected our first set of antibodies for 7 of these targets. Now onto primary and secondary validation.</p>
<p style="text-align: justify;">Stay tuned to find out who the lucky 7 are…</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/welcome-to-the-sgc-karolinska-side-of-als-rap-the-scientists-the-screening-and-the-antibody-selections/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Testing Hit Analogues against USP5 Zf-UBD with SPR Assay #1</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/testing-hit-analogues-against-usp5-zf-ubd-with-spr-assay-1/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/testing-hit-analogues-against-usp5-zf-ubd-with-spr-assay-1/#respond</comments>
		<pubDate>Thu, 24 Jan 2019 15:34:40 +0000</pubDate>
		<dc:creator><![CDATA[Mandeep Mann]]></dc:creator>
				<category><![CDATA[Mandeep Mann]]></category>
		<category><![CDATA[USP5]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2393</guid>
		<description><![CDATA[In a previous post, I described using hit expansion tools such as commercial substructure searches, docking, and free energy perturbation (FEP) simulations to explore the structure activity relationship (SAR) of the chemical series focused on the Zf-UBD of USP5. The first batch of commercially ordered compounds arrived just before the Christmas holidays. I decided to <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/testing-hit-analogues-against-usp5-zf-ubd-with-spr-assay-1/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>In a <a href="https://thesgc.github.io/static-openlabnotebooks/hit-expansion-using-substructure-search-free-energy-perturbation/">previous post</a>, I described using hit expansion tools such as commercial substructure searches, docking, and free energy perturbation (FEP) simulations to explore the structure activity relationship (SAR) of the chemical series focused on the Zf-UBD of USP5. The first batch of commercially ordered compounds arrived just before the Christmas holidays. I decided to tackle testing the chemical analogues in the New Year. I used a surface plasmon resonance (SPR) assay to determine binding affinities of commercially ordered analogues, designed from compounds hits listed in Table 1. You can see details on <a href="https://zenodo.org/record/2548710#.XEnZclxKjIU">Zenodo</a><strong>. </strong></p>
<p style="text-align: left;"><strong>Table 1</strong>. Compound Hits from which chemical analogues were designed</p>
<p><img class="wp-image-2394 alignleft" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/20190124a-300x153.png" alt="" width="451" height="231" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/20190124a-300x153.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/20190124a.png 678w" sizes="(max-width: 451px) 100vw, 451px" /></p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p>Of the 19 analogues tested, compound AE-641/11456811, an analogue of DAT194 (K<sub>D</sub>= 215 µM; n=9) had a binding affinity of approximately 65 µM (n=6) and an excellent ligand efficiency of 0.36 (Figure 1). The addition of a methyl group increased potency 3-fold!</p>
<p><img class=" wp-image-2395 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/20190125b-300x83.png" alt="" width="473" height="131" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/20190125b-300x83.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/20190125b-768x212.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/20190125b-780x218.png 780w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/20190125b.png 788w" sizes="(max-width: 473px) 100vw, 473px" /></p>
<p style="text-align: center;">Figure 1. A) Compound DAT194 B) Compound AE-641/11456811</p>
<p>Compound DAT194 (PDB: 6NFT) and the docked pose of AE-641/11456811 are shown in Figure 2a. The addition of the methyl group leads to increased hydrophobic interactions in the binding pocket with W209. In Figure 2b, the docked pose of AE-641/11456811 shows the methyl group fits into a small groove in the pocket. Based on the docked pose, it may be possible to further extend the non-polar group into the pocket to increase the potency further. For this reason, compounds in Figure 3 were ordered as follow up compounds of AE-641/11456811.</p>
<p><img class="wp-image-2396 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/20190124c-300x101.png" alt="" width="529" height="178" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/20190124c-300x101.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/20190124c-768x258.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/20190124c.png 900w" sizes="(max-width: 529px) 100vw, 529px" /></p>
<p style="text-align: center;"><strong>Figure 2.</strong> A) Compound DAT194 (PDB: 6NFT) in magenta and docked pose of AE-641/11456811 (yellow) B) Binding pocket of USP5 Zf-UBD and docked pose AE-641/11456811 (yellow)</p>
<p><img class=" wp-image-2397 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/20190124d-300x181.png" alt="" width="290" height="175" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/20190124d-300x181.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/20190124d.png 708w" sizes="(max-width: 290px) 100vw, 290px" /></p>
<p style="text-align: center;"><strong>Figure 3</strong>. Follow up compounds of AE-641/11456811</p>
<p>Analogues of DAT180 and DAT201 that were tested did not show a significant improvement in potency. So far, I’ve tested 19 of the 39 analogue compounds of DAT180, DAT194 and DAT201 that were selected in the hit expansion. Some compounds required custom synthesis and were not ordered, and others are still in transit. I will be testing the rest of the analogues once the next batch of compounds arrive, as well as testing follow up compounds of AE-641/11456811. I’ll also be setting up co-crystal trays of USP5 Zf-UBD and AE-641/11456811 for structure determination. Stay tuned!</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/testing-hit-analogues-against-usp5-zf-ubd-with-spr-assay-1/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
	</channel>
</rss>
