<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>Ellie Williams &#8211; openlabnotebooks.org</title>
	<atom:link href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/ellie-williams/feed/" rel="self" type="application/rss+xml" />
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<description>A growing team of groundbreaking scientists around the world are now sharing their lab notebooks online</description>
	<lastBuildDate>Mon, 13 May 2019 13:19:13 +0000</lastBuildDate>
	<language>en-GB</language>
	<sy:updatePeriod>hourly</sy:updatePeriod>
	<sy:updateFrequency>1</sy:updateFrequency>
	<generator>https://wordpress.org/?v=4.9.10</generator>

<image>
	<url>https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/cropped-lab-jar02-32x32.png</url>
	<title>Ellie Williams &#8211; openlabnotebooks.org</title>
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<width>32</width>
	<height>32</height>
</image> 
	<item>
		<title>ACVR1 Crystals taken to the Diamond Light Source (Beamline I04.) &#8211; no data sets collected.</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/acvr1-crystals-taken-to-the-diamond-light-source-beamline-i04-no-data-sets-collected/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/acvr1-crystals-taken-to-the-diamond-light-source-beamline-i04-no-data-sets-collected/#respond</comments>
		<pubDate>Mon, 13 May 2019 13:19:13 +0000</pubDate>
		<dc:creator><![CDATA[Ellie Williams]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Ellie Williams]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2861</guid>
		<description><![CDATA[Following up on the crystals from my last post I sent them along on the last Diamond trip we had to beamline I04 but regrettably didn’t get any usable datasets. Perhaps fortunately there is a possible explanation for this that isn’t just ‘the crystals were a bit too rubbish’. We send our crystals to the Diamond <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/acvr1-crystals-taken-to-the-diamond-light-source-beamline-i04-no-data-sets-collected/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Following up on the crystals from my last post I sent them along on the last Diamond trip we had to <a href="https://www.diamond.ac.uk/Instruments/Mx/I04">beamline I04</a> but regrettably didn’t get any usable datasets. Perhaps fortunately there is a possible explanation for this that isn’t just ‘the crystals were a bit too rubbish’. We send our crystals to the Diamond Light Source and because of the high number of crystals sent on each trip by the department (124 on the last trip) not everyone goes along on every trip and we quite often use something called ‘auto-centering’ that uses a computer algorithm to find the edge of the loop and to align the loop and your crystal in the beam. Centering is necessary but also quite tricky, because to collect a data set you need ‘slices’ of data taken with the crystal at different angles so the loop the crystals are mounted in, is rotated by a robotic arm so that the crystal also rotates around inside the x-ray beam. Thus if your crystal isn’t aligned properly, as the loop rotates it moves you crystal out of the beam and thus you get no data at that point.</p>
<p>Usually the automatic alignment is really good, however this time around it didn’t work so well and quite often either my crystal wasn’t in the beam at all (it mistook the loop stem for the crystal) or it started in the beam and then rotated out of it (which is what I think happened in the case shown below where you can see two ‘periods’ of data with dips in between. This is, of course, suboptimal and leads to a data set that you can’t really use.</p>
<p>I’m resending the crystals to the beamline this Friday and hopefully going with them myself. The silver lining to this is that I know that some of them definitely diffract which is encouraging. Hopefully they’ll behave well when fully in the beam.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/05/xtal_centering.png" alt="" width="940" height="430" class="alignnone size-full wp-image-2862" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/05/xtal_centering.png 940w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/05/xtal_centering-300x137.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/05/xtal_centering-768x351.png 768w" sizes="(max-width: 940px) 100vw, 940px" /></p>
<p><em>Fig. 1 – Beam aligned at very edge of loop showing cyclic diffraction suggesting the crystal was moving in and out of the beam rather than degrading in the beam.</em></p>
<p>&nbsp;</p>
<p>You can read more about this <a href="https://doi.org/10.5281/zenodo.2788112">over on Zenodo.</a></p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/acvr1-crystals-taken-to-the-diamond-light-source-beamline-i04-no-data-sets-collected/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Re-Purification of ALK2 for crystallisation with different compounds from M4K.</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/re-purification-of-alk2-for-crystallisation-with-different-compounds-from-m4k/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/re-purification-of-alk2-for-crystallisation-with-different-compounds-from-m4k/#respond</comments>
		<pubDate>Wed, 24 Apr 2019 14:50:59 +0000</pubDate>
		<dc:creator><![CDATA[Ellie Williams]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Ellie Williams]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2753</guid>
		<description><![CDATA[I’ve been lucky enough to have a student (Mira) in the lab working with me for the past few weeks and so I’d like to write about the work that we’ve been doing together. We spent some time cleaning up an old sample of ALK2 that Ros had left over in the freezer to set <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/re-purification-of-alk2-for-crystallisation-with-different-compounds-from-m4k/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>I’ve been lucky enough to have a student (Mira) in the lab working with me for the past few weeks and so I’d like to write about the work that we’ve been doing together.</p>
<p>We spent some time cleaning up an old sample of ALK2 that Ros had left over in the freezer to set up in crystal plates with some compounds from M4K Pharma that they were relatively keen to see the structures of.</p>
<p>We used a specialised coarse screen that Ros had designed containing only conditions where we had previously seen crystals in the wider coarse screens. This means that we can narrow down the field where we’re expecting to find crystals and test fewer conditions to find the one we need – this means setting up fewer plates and thus needing less protein per compound tested, so is overall more efficient.</p>
<p>We set up the plates (at 4C and 20C) at around 9mg/ml with compounds M4K2163, M4K2143, M4K2192, M4K2207 and M4K2194 at 1mM concentration from 25mM stocks.</p>
<p>We found some interesting crystals, one of which was mountable as is (from M4K2192) and several across all the compounds, which showed small crystals that would be worth following up in more specific conditions.</p>
<p><img class="alignnone size-full wp-image-2754" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/M4K2192.png" alt="" width="403" height="294" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/M4K2192.png 403w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/M4K2192-300x219.png 300w" sizes="(max-width: 403px) 100vw, 403px" /></p>
<p><em>Crystals of ALK2/ACVR1A with compound M4K2192</em></p>
<p>&nbsp;</p>
<p>So Mira designed some follow up screens based on the conditions that showed small crystals and we set up some more plates with those conditions in the hope of turning those small crystals into larger, better crystals. We had enough protein to set up followup plates for compounds M4K2163 and M4K2143 at 9.5mg/ml. From these plates we mounted 5 crystals across the board and combined with those above ended up with crystals of unknown quality across three compounds. These have been confirmed as containing protein by imaging the drops by UV light (UV light interacts with protein to make crystals only visible under UV light if they contain protein rather than just being salt) and are now waiting to be sent to the synchrotron to see if they actually diffract or not.  You can read more about the technical details <a href="https://zenodo.org/record/2650203#.XMB29jBKiUk">here on Zenodo.</a></p>
<p>&nbsp;</p>
<p><span style="font-size: 1.5rem;">Yesterday (23</span><span style="font-size: 11.25px; height: 0px; line-height: 0; position: relative; vertical-align: baseline; bottom: 1ex;">rd</span><span style="font-size: 1.5rem;">) also marked FOP awareness day and so, shamefully a day late, I’d like to add my picture to the many others out there celebrating</span><span style="font-size: 1.5rem;"> </span><strong style="font-size: 1.5rem;">#FunFeet4FOP</strong><span style="font-size: 1.5rem;">! Here are my toes.</span></p>
<p><img class="alignnone size-medium wp-image-2755" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Feet4FOP_crop-300x223.jpg" alt="" width="300" height="223" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Feet4FOP_crop-300x223.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Feet4FOP_crop-768x570.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Feet4FOP_crop-1024x761.jpg 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Feet4FOP_crop.jpg 1143w" sizes="(max-width: 300px) 100vw, 300px" /></p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/re-purification-of-alk2-for-crystallisation-with-different-compounds-from-m4k/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>The Benefits of Conference attendance:  BMP Signalling in Cancer II.</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/the-benefits-of-conference-attendance-bmp-signalling-in-cancer-ii/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/the-benefits-of-conference-attendance-bmp-signalling-in-cancer-ii/#respond</comments>
		<pubDate>Mon, 15 Apr 2019 10:44:07 +0000</pubDate>
		<dc:creator><![CDATA[Ellie Williams]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Ellie Williams]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2728</guid>
		<description><![CDATA[I’ve got a few things to catch up with on the blog – one of them is to talk about a conference I attended in Oxford recently. I attended the Biochemical Society BMP Signalling in Cancer II meeting which was held in St. Annes College, Oxford. At this meeting I presented a poster on the <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/the-benefits-of-conference-attendance-bmp-signalling-in-cancer-ii/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>I’ve got a few things to catch up with on the blog – one of them is to talk about a conference I attended in Oxford recently. I attended the <a href="https://www.biochemistry.org/Events/tabid/379/MeetingNo/SA217/view/Conference/Default.aspx">Biochemical Society BMP Signalling in Cancer II</a> meeting which was held in St. Annes College, Oxford. At this meeting I presented a poster on the work I’ve been doing (and which has been written about here as it’s happened) on looking at how the mutations in ALK2 effect the underlying activity of the kinase as well as effecting how well FKBP12 can inhibit ALK2.</p>
<p>The conference brought together 60 researchers in the field from around the world talking about a variety of different cancers but also other BMP related conditions such as FOP. Over three days there were some fantastic talks by the leaders in the field including from Dr. Caroline Hill from the Crick Institute in London who spoke about receptor cross talk in BMP signalling and how that might affect signalling through ALK2 and Prof. Eileen Shore from the university of Pennsylvania who spoke about mechanosensing and tissue stiffness in FOP.</p>
<p>I had a chance to get into some really interesting conversations with some of the attendees about the way their work and my work intersected and how we might be able to collaborate in the future – particularly interesting was talking to some Postdocs from Caroline Hill’s group who are looking at similar problems to me but from an in-cell perspective rather than using purified proteins. From that some interesting questions have been thrown up that I now want to test in my in-vitro systems and that will be detailed here as and when I’ve had a chance to try them. Hopefully this can lead to validation of both my work and theirs by complementing in-vivo and in-vitro approaches.</p>
<p>I thought it would be good to share the poster I created here – so you can find it <a href="https://zenodo.org/record/2640488#.XLRgljBKiUk">over on Zenodo</a>. It is designed for a scientific audience but not necessarily an FOP specialist audience so hopefully it should be fairly easy to follow. It leads the reader through the story of explaining that Gain of Function (GOF) mutations in ALK2 are involved in diseases like FOP and DIPG. I then go on to present data that shows that these mutations disrupt the binding of the inhibitor protein FKBP12 but that this isn’t the whole story (as regular readers of this blog will know). Then I talk about what I found out about what is needed for SMAD activation by ALK2 before looking at the increased phosphorylation of SMAD1 by the mutant ALK2 proteins compared to the wild type. This covers both the western blot data that shows more SMAD1 activation but also the mass spec data I’ve shared previously that looks at more SMAD1 and more ALK2 phosphorylation. Finally I showed the phosphomapping data on ALK2 that is starting to pick apart the exact phosphorylation locations on ALK2 that are necessary for SMAD1 activation (Also data I’ve talked about on this blog before). In this case it was nice to see that all the effort Rod and I had put into phosphomapping had paid off! Other groups had said they’d tried and been unable to map ALK2 using trypsin alone but our method of using 5 different enzymes both enriched and unenriched seems to have done the trick (despite it being 10x more work!).</p>
<p>There is great value from attending a conference like this and in this case I feel that I really did make some very important connections and had some very informative discussions that were directly relevant to my research and that will hopefully bear fruit in time and guide the direction of my experiments.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/the-benefits-of-conference-attendance-bmp-signalling-in-cancer-ii/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Phosphorylation over time and phosphomapping experiments to investigate the influence of ALK2 mutations on activity.</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/phosphorylation-over-time-and-phosphomapping-experiments-to-investigate-the-influence-of-alk2-mutations-on-activity/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/phosphorylation-over-time-and-phosphomapping-experiments-to-investigate-the-influence-of-alk2-mutations-on-activity/#respond</comments>
		<pubDate>Tue, 12 Mar 2019 11:56:29 +0000</pubDate>
		<dc:creator><![CDATA[Ellie Williams]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Ellie Williams]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2579</guid>
		<description><![CDATA[In this post I’d like to return to looking at my work on phosphorylation. Phosphorylation is a key signalling method for activating the BMP pathway and inducing activity. Specifically activation of ALK2 by phosphorylation leads to phosphorylation of SMAD1 which leads to gene transcription that ultimately is responsible for bone formation. In this post I’m <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/phosphorylation-over-time-and-phosphomapping-experiments-to-investigate-the-influence-of-alk2-mutations-on-activity/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>In this post I’d like to return to looking at my work on phosphorylation. Phosphorylation is a key signalling method for activating the BMP pathway and inducing activity. Specifically activation of ALK2 by phosphorylation leads to phosphorylation of SMAD1 which leads to gene transcription that ultimately is responsible for bone formation.</p>
<p>In this post I’m going to talk about two experiments – the first is looking at the rate of phosphorylation (or how active) of two ALK2 mutants, L196P and R258S. The second is to look at mapping the exact sites of phosphorylation on two of the more common ALK2 mutations in FOP, R206H and Q027E.</p>
<p>&nbsp;</p>
<p><strong>Phosphorylation over time in ALK2 L196P and R258S.</strong></p>
<p>I’ve talked <a href="https://thesgc.github.io/static-openlabnotebooks/phosphorylation-of-smad1-rates-as-a-function-of-alk2-mutations-and-type-ii-association/">previously</a> about how phosphorylation is effected by the Q207E and R206H mutations in ALK2. This time I wanted to look at some additional mutants that are found in a small proportion of cases of FOP, specifically L196P, R258S, G328E and R375P. I purified the long forms of these ALK2 mutations in a standard fashion. Regrettably only L196P and R258S gave a good yield of expressed protein while the other two constructs gave me no expression of protein. I set up my standard activation assay with these ALK2 mutants (I mixed them with SMAD1 and with either ACVR2 or BMPR2, before adding ATP, DTT and Mg2+/Mn2+ in order to start the reaction), taking samples over time periodically up to 30 minutes. The samples were then analysed by mass spec to look for intact mass shifts corresponding to phosphorylation.</p>
<p>I plotted the peaks corresponding to phosphorylated states of both ALK2 and SMAD1 at the different time points and used this 3D graph to compare different mutant’s behaviour with respect to the activity of ALK2. I also compared this data to previously obtained data from the previous blog post.</p>
<p><img class="alignnone size-large wp-image-2580" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/MS_timecourse_blogpost-806x1024.jpg" alt="" width="780" height="991" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/MS_timecourse_blogpost-806x1024.jpg 806w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/MS_timecourse_blogpost-236x300.jpg 236w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/MS_timecourse_blogpost-768x975.jpg 768w" sizes="(max-width: 780px) 100vw, 780px" /></p>
<p><em>Time course plots for phosphorylation of SMAD1 (top) and ALK2 L196P or R258S (bottom) in the presence of ACVR2 (left) or BMPR2 (right). Legend shows time point in seconds.</em></p>
<p>&nbsp;</p>
<p>Overall we can see that R258S appears to be more activating than L196P with both more SMAD activation over the first five minutes (including the appearance of a potentially interesting third SMAD binding site). This is the case when using either ACVR2 or BMPR2. Also in general ACVR2 appears to lead to more SMAD activation than BMPR2 regardless of mutation.</p>
<p>When looking at ALK2 phosphorylation, we can see that L196P is quickly phosphorylated up to three times compared to R258S but that over the course of five minutes, R258S accumulates up to 6 phosphorylation’s much more rapidly than L196P. This effect is most pronounced when paired with ACVR2.</p>
<p>What this means in very broad terms is that the different ALK2 FOP mutants are differently activating and that their response to different type II receptors change depending on which mutation is present. This is similar to the results seen with other ALK2 mutants studied previously. This builds towards understanding why the ALK2 FOP mutations leads to increased signalling and thus increased bone formation and exactly what is going on.</p>
<p>&nbsp;</p>
<p><strong>Phosphomapping of ALK2 R206H and Q207E:</strong></p>
<p>Another aspect of looking at the phosphorylation of ALK2 and SMAD1 is not just how much it is phosphorylated but also where it is phosphorylated. For this I worked with Rod Chalk in our mass spec department to map the phosphorylation sites present in samples of protein incubated as previously mentioned. In this case I was looking at Q207E and R206H mutants of ALK2 paired with either ACVR2 or BMPR2, all in the presence of SMAD. This suggested that many of the phosphorylations could be traced to the following positions in the GS region of the kinase regardless of ALK2 mutation:</p>
<p><strong><u>ST</u></strong>LADLLDH<strong><u>S</u></strong>C<strong><u>TS</u></strong>G<strong><u>S</u></strong>G<strong><u>S</u></strong>GLPFLVQRTV</p>
<p>However not all these sites are seen at the same time and it is hard to determine which are essential for transmission of signal and ALK2 activation.</p>
<p>Similarly phosphorylation on SMAD1 could be traced to the following common locations at the C-terminal tail of the protein:</p>
<p>TQMGSPHNPI<strong><u>SS</u></strong>V<strong><u>S</u></strong></p>
<p>In this case it is the last three serines in the tail that are able to be phosphorylated and again, are seen with both mutations. Combinations of the three possible sites and all three at once were seen as possibilities within the phosphomapping data.</p>
<p>What this means however is a bit less clear. When considering the time course data from the previous post linked as well as the new mutant time course data and the phosphomapping it seems to suggest that you require multiple (up to 6) ALK2 phosphorylations to get maximal (3 site) SMAD1 phosphorylation and that there are at least this many sites in the GS loop and the region just prior to it.</p>
<p>In order to properly probe the necessity of each of these sites, it is going to be necessary to repeat these mass spec experiments on knock out mutants that remove the phosphorylation sites in various combinations, to really see which are essential for ALK2 activation. These are currently being cloned so hopefully I’ll be able to test them soon.</p>
<p>&nbsp;</p>
<p>You can find the technical details for all these experiments over on <a href="https://doi.org/10.5281/zenodo.2591062">Zenodo.</a></p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/phosphorylation-over-time-and-phosphomapping-experiments-to-investigate-the-influence-of-alk2-mutations-on-activity/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>ZFYVE9C purification and association test with TGFBR1 and TGFBR2</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/zfyve9c-purification-and-association-test-with-tgfbr1-and-tgfbr2/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/zfyve9c-purification-and-association-test-with-tgfbr1-and-tgfbr2/#comments</comments>
		<pubDate>Fri, 22 Feb 2019 14:05:47 +0000</pubDate>
		<dc:creator><![CDATA[Ellie Williams]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Ellie Williams]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2515</guid>
		<description><![CDATA[ZFYVE9C (Otherwise known as SARA) is a scaffold protein involved in the association of various protein components as part of the TGFB signalling pathway. The equivalent in the BMP signalling pathway is a protein known as Endofin so studying SARA (which is slightly easier) might give us useful insights for both proteins. For more details <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/zfyve9c-purification-and-association-test-with-tgfbr1-and-tgfbr2/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>ZFYVE9C (Otherwise known as SARA) is a scaffold protein involved in the association of various protein components as part of the TGFB signalling pathway. The equivalent in the BMP signalling pathway is a protein known as Endofin so studying SARA (which is slightly easier) might give us useful insights for both proteins. For more details on this project please see <a href="https://thesgc.github.io/static-openlabnotebooks/a-summary-of-my-alk2-project-goals/">this blog post.</a></p>
<p>One of the unanswered questions about SARA is what does one of the major domains that it has actually do? (The domain is known as a DUF domain, or a Domain of Unknown Function!) We know the structure but that hasn’t given us any clues as to its function. One thought was that it might be involved in recruiting type I or type II receptors close to their binding partners such as one of the SMADs to allow signal propagation.</p>
<p>I purified the DUF domain of SARA and mixed it with either excess TGFBR1 or excess TGFBR2 (the two relevant receptors in the TGFB pathway). I tested the receptors in both the unphosphorylated (inactive) and phosphorylated (active) states. After incubating the samples for 30 minutes while concentrating the volume back down to &lt;5ml, I ran them down a gel filtration column to see if the two proteins bound together sufficiently to shift the peaks coming off the column which would indicate they were acting as a complex of larger mass.</p>
<p><img class="alignnone size-large wp-image-2516" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/gf_trace_zfyve9c-1024x491.png" alt="" width="780" height="374" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/gf_trace_zfyve9c-1024x491.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/gf_trace_zfyve9c-300x144.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/gf_trace_zfyve9c-768x368.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/gf_trace_zfyve9c.png 1578w" sizes="(max-width: 780px) 100vw, 780px" /></p>
<p><em>Overlay of the UV traces from all the size exclusion gel filtration runs to show the relative elution volumes of the peaks confirming no shift in the ZFYVE9C peak consistent with no complex being formed.</em></p>
<p>&nbsp;</p>
<p>Unfortunately I didn’t see any shift in the peak for SARA suggesting that either the two proteins don’t bind or that the affinity is weak enough that they fall apart under the conditions of the gel filtration column.</p>
<p>The next thing to try will be to purify the proteins again but not to cleave the tag on one of the binding partners (either the receptors or the SARA protein) and try to do a pull down experiment to test whether this shows any association or not.</p>
<p>&nbsp;</p>
<p>You can find more details over at<a href="https://doi.org/10.5281/zenodo.2575581"> Zenodo.</a></p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/zfyve9c-purification-and-association-test-with-tgfbr1-and-tgfbr2/feed/</wfw:commentRss>
		<slash:comments>2</slash:comments>
		</item>
		<item>
		<title>A summary of my ALK2 project goals.</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/a-summary-of-my-alk2-project-goals/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/a-summary-of-my-alk2-project-goals/#respond</comments>
		<pubDate>Tue, 05 Feb 2019 17:01:21 +0000</pubDate>
		<dc:creator><![CDATA[Ellie Williams]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Ellie Williams]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2439</guid>
		<description><![CDATA[I thought I’d spend a blog post just talking about the various paths of my research and try and tie it all together. A lot of my experiments do dot around these various arms of investigation and so one experimental post doesn’t necessarily tie in directly with the previous one or the next one and <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/a-summary-of-my-alk2-project-goals/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>I thought I’d spend a blog post just talking about the various paths of my research and try and tie it all together. A lot of my experiments do dot around these various arms of investigation and so one experimental post doesn’t necessarily tie in directly with the previous one or the next one and I thought it might be a good idea to explain why and summarise where I’m up to with each strand.</p>
<p>My main focus of research is to investigate the cause of the disease fibrodysplasia ossificans progressiva (FOP). We know that it’s caused by a mutation in ALK2 which leads to excessive bone formation. This also nicely dovetails with research done into DIPG where 25% of cases share a mutation with FOP in ALK2.</p>
<p>My research can be broken down in to three main strands:</p>
<ul>
<li>The first is looking at characterising the binding of ligands to ALK2 in order to optimise binding and work towards finding a potent and specific inhibitor. This is the focus of a lot of the M4K work that Jong Fu and Ros are doing and I’ve been doing a little bit of that as well but it’s not the main focus of my work. This endeavour is going fairly well – despite setbacks relating to protein expression (which are not entirely solved – we now have regained expression of the kinase domain alone but we still don’t have expression of the kinase domain and GS loop construct) and Ros has a number of structures that she’s working on at the moment that have been recently collected. Alas I’ve not had as much luck with new structures recently. We frequently get new deliveries of compounds from M4K to test – Jong Fu looks at them in cells while Ros and I try and crystallise the most promising candidates with varying degrees of success. Ros and I have also attempted XChem fragment screening which is where we use high throughput crystallography to identify small weakly bound fragments that might identify new leads for drug design or in our case, help us identify drugs that might bind to a second allosteric site within ALK2 away from the main ligand binding pocket. An XChem update will be coming soon from either Ros or myself as I’ve recently started helping her with this aspect of the work and so have been shadowing her on our last couple of visits to Diamond.</li>
</ul>
<p>&nbsp;</p>
<ul>
<li>The second is investigating the interactions of the type I receptor (ALK2) with its partnering type II in order to activate SMAD 1/5/8. This covers looking at receptor cross reactivity, SMAD cross reactivity, and the influence of mutations on this. This also involves looking at crystal structures of the mutants where possible, attempting to make complexes of the type I and type II receptors in order to try and crystallise them and probing the activities of the proteins and the various disease mutants. Covered in this is everything I’ve done using western blots and mass spec to identify the dependencies of this complex as required for SMAD phosphorylation. So far I know that the mutants are more active than the wild type (with R206H being the most active ALK2 mutation), that the type II must be present but does not need to be active to allow for SMAD activation and that increased activity is not solely dependent upon the removal of inhibition by FKBP12. Covered in this is also the work looking at mapping the phosphorylation sites in ALK2 to try and identify if some of these sites are more critical than others with relation to SMAD activation. The cloning I recently talked about is part of this investigation, making mutations to remove certain phosphorylation sites and comparing the activity of these with the wild type. As soon as these constructs have been cloned into suitable expression vectors I’ll be starting that experiment too. Another aspect of this is looking at the association of other proteins with the receptors, such as the possible link between them and XIAP as I talked about in one earlier blog post.</li>
</ul>
<p>&nbsp;</p>
<ul>
<li>The third aspect is looking other proteins related to the ALK2 pathway such as Endofin. Endofin is a membrane associated protein with multiple domains including a SMAD binding domain and large areas of unknown function. It is involved with the co-localisation of various parts of the pathway and appears to act as something of a scaffold protein to hold things in close proximity for signal propagation. The equivalent protein in the TGFβ pathway is called SARA and appears to have a similar function in that pathway. Some time ago I solved the structure for a domain of unknown function within SARA (4BKW) which contained a novel fold and showed structural (but not sequence) similarity to a protein called SUFU, which is part of the Hedgehog signalling pathway. We still don’t know what this protein and domain does or what it binds to or how it functions and so this is another area of investigation. If we can uncover the function of SARA we can also try and uncover the function of Endofin and this might provide a better understanding of the way FOP mutations work and help us see if there are any other ways we can target the pathway that are not exclusively based on inhibiting ALK2 activity. However given that the DUF domain has very little information about it, our investigations into what it does have been limited thus far. My goals with this are to try and probe the role of the DUF domain looking for whether it binds to either of the type I or type II receptors, and if so which parts of the receptors, or if it associates with a different protein all together. Given we know very little about what it does and the structure offers few clues (the lack of sequence similarity to SUFU means that although we can predict a possible binding mode, the nature of what binds there is very hard to determine as there are no conserved residues) this is quite a difficult problem to approach.</li>
</ul>
<p><img class="alignnone size-large wp-image-2440" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/summary-1024x606.png" alt="" width="780" height="462" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/summary-1024x606.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/summary-300x177.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/summary-768x454.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/summary.png 1324w" sizes="(max-width: 780px) 100vw, 780px" /></p>
<p>&nbsp;</p>
<p>My work moves between these three strands depending on how much and which proteins I’ve got available to me &#8211; do I have any in the freezer already purified or do I need to make fresh? And that’s if I have the pellets in the freezer of the expressed protein in the first place. It also varies depending on whether we have new compounds we’re interested in testing immediately from either M4K or other collaborators, or whether I’ve got data I need to analyse from previous successful crystallisation attempts. Sometimes I’ll be looking at my own structural data and sometimes I’ll be looking at someone else’s structure which we call ‘proofreading’ to double check it for errors and make sure nothing has been missed. Sometimes we’ll have time at Diamond and so everything else has to be put on hold in order to make use of that visit (such as XChem trips) and sometimes there’s a lot of data analysis to do on other experiments &#8211; writing it up and understanding what the data is telling us is just as important for progress as the experiments themselves.</p>
<p>Having the three strands means I can move from one to another as each strand has its own peaks and lulls in activity at different times. I’m afraid it does make for a slightly more disjointed blog journey but I hope you’ll bear with me through it all as I try and make sense of the problems I’m investigating.</p>
<p>I’ll try and post again sooner rather than later on what I’ve been doing in January as it has been really rather bitty and frustrating in terms of things actually working but hopefully February will bring with it some more productive experiments.</p>
<p>&nbsp;</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/a-summary-of-my-alk2-project-goals/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Purification and crystallisation of an ALK2/ACVR2/FKPB12 complex.</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/purification-and-crystallisation-of-an-alk2-acvr2-fkpb12-complex/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/purification-and-crystallisation-of-an-alk2-acvr2-fkpb12-complex/#respond</comments>
		<pubDate>Fri, 14 Dec 2018 15:30:55 +0000</pubDate>
		<dc:creator><![CDATA[Ellie Williams]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Ellie Williams]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2261</guid>
		<description><![CDATA[One problem with working with the isolated kinase domains of ALK2 and it’s type II binding partners, is that the affinity between the kinase domains alone is insufficient to allow complex formation. We know that the two proteins associate with each other in vivo and are co-localised by the domains on the outside of the <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/purification-and-crystallisation-of-an-alk2-acvr2-fkpb12-complex/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>One problem with working with the isolated kinase domains of ALK2 and it’s type II binding partners, is that the affinity between the kinase domains alone is insufficient to allow complex formation. We know that the two proteins associate with each other in vivo and are co-localised by the domains on the outside of the membrane. However when we just mix the two kinase domains we don’t see a strong enough association to allow a complex to be purified. We know from in-vitro tests such as the phosphorylation tests I’ve mentioned previously that there must at least be transient contact to allow phosphorylation but that isn’t sufficient to set up crystal plates of the complex.</p>
<p>One way to try and get around this is to artificially tether the two kinase domains together with a flexible loop. I have a construct which does exactly this, tethering one end of ACVR2 to ALK2. It’s known as QQ02ACVR2A. I purified this protein and then formed a complex with the binding partner FKBP12 which will bind to the GS domain of ALK2 present in the construct.</p>
<p>The first test during purification was to see if a complex could be assembled between the fusion construct and FKBP12. As they are very differences sizes (72 kDa vs. 12 kDa) purification by gel filtration gives a very clear answer. We can clearly see co-elution of the fusion and FKBP12 which is only possible if they were present as a complex. This in its self is quite exciting as it shows the fusion is still able to bind to its physiological binding partner.</p>
<p>The second stage after purification is to attempt to crystallise the complex. I concentrated the purified protein down and added compound K02288. K02288 definitely binds ALK2 and has some affinity for ACVR2 which is necessary for stabilising both kinase domains sufficiently to crystallise. I’ve tried crystallising ALK2 alone without compound before and had zero success so it was important to add a compound to this crystallisation attempt.</p>
<p>I set up four coarse screen plates and got the following in the HCS screen:</p>
<p><img class="alignnone size-medium wp-image-2262" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/initial_crystals-300x286.png" alt="" width="300" height="286" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/initial_crystals-300x286.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/initial_crystals.png 598w" sizes="(max-width: 300px) 100vw, 300px" /></p>
<p>B6: 20% PEG8000, 0.2M Mg Acetate, 0.1M Cacodylate pH6.5</p>
<p>A7: 1.4M Na Acetate, 0.1M Cacodylate pH6.5</p>
<p>A8: 30% 2-propanol, 0.2M Na Citrate tribasic, 0.1M Cacodylate pH6.5</p>
<p>These are crystalline but not crystals so I designed a follow up screen based around these conditions. I then used what was left of my complex to set up two plates in this followup screen at 4C and 20C.</p>
<p>After 7 days I found the following in my plates:</p>
<p><img class="alignnone size-large wp-image-2263" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/FU_xtals-1024x633.png" alt="" width="780" height="482" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/FU_xtals-1024x633.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/FU_xtals-300x185.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/FU_xtals-768x475.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/FU_xtals.png 1160w" sizes="(max-width: 780px) 100vw, 780px" /></p>
<p>I mounted five examples and sent them to the synchrotron. Alas they all came back as either salt or no diffraction. While the salt must be ignored, it’s possible that the no diffraction crystals are protein but just do not diffract. The next step will be to make seed stocks from these wells, purify more protein and set up some more plates to see if I can get any better crystals.</p>
<p>You can find more details <a href="https://doi.org/10.5281/zenodo.2276755">over at Zenodo.</a></p>
<p>I&#8217;m off for Christmas the middle of next week but in the new year I&#8217;ll be trying again at crystallising a variety of different proteins and compounds as well as working on characterising the phospho-mutants of ALK2 I posted about last time. Have a great holiday!</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/purification-and-crystallisation-of-an-alk2-acvr2-fkpb12-complex/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Generating mutations in the GS domain of ALK2</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/generating-mutations-in-the-gs-domain-of-alk2/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/generating-mutations-in-the-gs-domain-of-alk2/#respond</comments>
		<pubDate>Fri, 23 Nov 2018 15:53:04 +0000</pubDate>
		<dc:creator><![CDATA[Ellie Williams]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Ellie Williams]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2176</guid>
		<description><![CDATA[This week one of the things I’ve been working on is the final set of mutations in the GS domain of Alk2 to help us unpick the influence of phosphorylation on function. That, however, is a bit of a mouthful so let’s unpack it a bit. I said in my last blog post that we <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/generating-mutations-in-the-gs-domain-of-alk2/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>This week one of the things I’ve been working on is the final set of mutations in the GS domain of Alk2 to help us unpick the influence of phosphorylation on function.</p>
<p>That, however, is a bit of a mouthful so let’s unpack it a bit.</p>
<p>I said in my last blog post that we know that for Alk2 to phosphorylate SMAD1 it must contain the GS domain. This is a short glycine and serine rich loop of the protein that comes before the main kinase domain. The serine residues in this region can be phosphorylated and this seems to be key to activating Alk2. The pattern of phosphorylation of these residues may well be important and split in specific ways between auto phosphorylation and trans phosphorylation by a type II receptor.</p>
<p>In order to work out which phosphorylation’s are key for SMAD activation, we designed mutants that would change the serine residues (where phosphorylation occurs) to alanine residues, thus preventing phosphorylation on that residue. In different combinations this allows us to test which phosphorylations are key to activation.</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p><img class="alignnone size-large wp-image-2177" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/Phos_mutation-1024x375.png" alt="" width="780" height="286" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/Phos_mutation-1024x375.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/Phos_mutation-300x110.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/Phos_mutation-768x281.png 768w" sizes="(max-width: 780px) 100vw, 780px" /></p>
<p><em>Mutations in the GS domain (blue) of Alk2, immediately prior to the kinase domain (green). The table shows the 11 different mutants that probe the different combinations of phosphorylation events necessary for SMAD1 activation. Row 11 in yellow is the mutant recently cloned.</em></p>
<p>My colleague George Kerr designed and cloned most of the constructs but moved onto a new role before the final one was made. I’ve now tried to make the final clone (highlighted in yellow) based on her primer designs.</p>
<p>I got colonies from my cloning and picked four to send for sequencing. Sequencing revealed that all four colonies contained the correct mutation and no additional mutations.</p>
<p>These mutant proteins can now be used to do in-cell assays, used as templates to make constructs suitable for expression and from there used in many of the in-vitro experiments I’ve already described to help work out the influence each of these phosphorylation events has.</p>
<p>For more details on the experimental methods please look at <a href="https://doi.org/10.5281/zenodo.1495120">Zenodo here.</a></p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/generating-mutations-in-the-gs-domain-of-alk2/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Phosphorylation of SMAD1 &#8211; rates as a function of Alk2 mutations and type II association.</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/phosphorylation-of-smad1-rates-as-a-function-of-alk2-mutations-and-type-ii-association/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/phosphorylation-of-smad1-rates-as-a-function-of-alk2-mutations-and-type-ii-association/#respond</comments>
		<pubDate>Wed, 07 Nov 2018 17:48:18 +0000</pubDate>
		<dc:creator><![CDATA[Ellie Williams]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Ellie Williams]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2114</guid>
		<description><![CDATA[One of the questions to be asked when looking at the mutation in ALK2 that causes FOP, is how exactly does that mutation cause the disease. ALK2 we know is part of the BMP signalling pathway responsible for bone formation and the mutation results in excess bone formation – thus we can summarise that somehow <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/phosphorylation-of-smad1-rates-as-a-function-of-alk2-mutations-and-type-ii-association/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>One of the questions to be asked when looking at the mutation in ALK2 that causes FOP, is how exactly does that mutation cause the disease. ALK2 we know is part of the BMP signalling pathway responsible for bone formation and the mutation results in excess bone formation – thus we can summarise that somehow the mutation leads to increased signal. The question is how?</p>
<p>Alk2 is known to phosphorylate SMADs 1, 5 and 8 (summarised as SMAD1/5/8) in order to signal downstream in the BMP signalling pathway. The instigation of this signal comes from the binding of an external BMP ligand to the domains of Alk2 and a type II receptor on the outside of the cell. This co-locates the internal domains (which incidentally is also where the Alk2 mutation is located) and seems to drive signalling.</p>
<p>I have been looking at the activation of SMAD1 by ALK2 in the presence or absence of a Type II receptor and in the presence or absence of the FOP ALK2 mutation. The activation of SMAD1 results in its phosphorylation which can be monitored by mass spectrometry.</p>
<ul>
<li>We know from previous experiments that Alk2 alone cannot activate SMAD1 even if the mutation is present, thus the Type II receptor is needed.</li>
<li>We know that Type II activity isn’t necessary for SMAD1 activation as using a kinase dead version of BMPR2 still works.</li>
<li>We know from previous experiments that you need both the kinase and GS domains of Alk2 in order to get SMAD1 activation.</li>
<li>We know from previous experiments that BMPR2 and ACVR2 both work to allow SMAD1 activation.</li>
<li>We know that Alk2 autophosphorylates its self to some degree, focused on the GS loop.</li>
<li>We know that the type II receptor transphosphorylates Alk2 adding additional phosphorylation sites.</li>
</ul>
<p><img class="alignnone size-medium wp-image-2116" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/Receptor_interactions-190x300.png" alt="" width="190" height="300" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/Receptor_interactions-190x300.png 190w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/Receptor_interactions.png 543w" sizes="(max-width: 190px) 100vw, 190px" /></p>
<p>Interaction of the type I and type II receptors in the membrane.</p>
<p>So what don’t we know?</p>
<ul>
<li>We don’t know how the mutation in Alk2 effects the rate of SMAD1 phosphorylation</li>
<li>We don’t know how transphosphorylation effects the rate of SMAD1 phosphorylation.</li>
<li>We don’t’ know how the mutation effects the rate of autophosphorylation of Alk2.</li>
</ul>
<p>This experiment can be performed using isolated purified proteins. I’ve been busy purifying Alk2 (wild type and two mutant versions – Q207E and R206H), ACVR2 and BMPR2. I’m now ready to combine these with samples of the Kinase Dead version of BMPR2 and SMAD1 that I have in the freezer into a mass spec based assay.</p>
<p>I mixed all the components together at appropriate concentrations, in this case all proteins were at 40uM, ATP was at 0.5mM, Magnesium ions were at 5mM, Manganese ions were at 2mM and DTT (a reducing agent to stop the proteins precipitating) was at 5mM. These concentrations are needed because if you drop lower, you lose the association between the type I and II receptors. In the cell they’d be brought together in the membrane by a ligand binding outside the cell which would co-locate the inner domains. Since we’re only looking at the inner domains in vitro, we need to artificially increase the concentrations to allow them to interact. 40uM was previously determined as being the lowest concentration at which this interaction can reliably be measured.</p>
<p>Once the ATP had been added, this started the phosphorylation reaction. I took a small sample at various time points (30s, 1min, 90s, 2min, 150s, 3min, 4min, 5min, 6min, 7min, 8min, 9min, 10min, 15min, 20min 25min) from my main reaction mix, and acid quenched the reaction in this sample to stop it going any further. This was done by diluting the sample in mass spectrometry running buffer containing 0.1% formic acid.</p>
<p>The samples were run overnight on the mass spectrometry machine to measure the mass of each of the proteins and the masses of any phosphorylated proteins. The peak heights for each of the phosphorylated states (I looked at up to 2 phosphorylation states for SMAD1 and 8 for Alk2) was extracted from the data and used to monitor the rate of phosphorylation of both Alk2 and SMAD1 over time.</p>
<p><img class="alignnone size-large wp-image-2118" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/total_phos-1024x683.png" alt="" width="780" height="520" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/total_phos-1024x683.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/total_phos-300x200.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/total_phos-768x512.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/total_phos.png 1560w" sizes="(max-width: 780px) 100vw, 780px" /></p>
<p><em>Total phosphorylation of Q207E after 25 minutes incubation with ACVR2.</em></p>
<p>I used the following combination of proteins in my reaction mixes:</p>
<table style="width: 510px;">
<tbody>
<tr>
<td style="width: 141px;">&nbsp;</td>
<td style="width: 88px;">ACVR2 active</td>
<td style="width: 93px;">BMPR2 active</td>
<td style="width: 109px;">BMPR2 kinase dead</td>
</tr>
<tr>
<td style="width: 141px;">Alk2 Wild type</td>
<td style="width: 88px;">X</td>
<td style="width: 93px;">X</td>
<td style="width: 109px;">X</td>
</tr>
<tr>
<td style="width: 141px;">Alk2 R206H mutant</td>
<td style="width: 88px;">X</td>
<td style="width: 93px;">X</td>
<td style="width: 109px;">X</td>
</tr>
<tr>
<td style="width: 141px;">Alk2 Q207E mutant</td>
<td style="width: 88px;">X</td>
<td style="width: 93px;">X</td>
<td style="width: 109px;">X</td>
</tr>
</tbody>
</table>
<p>&nbsp;</p>
<p>This gave me the following results: In each case the important bit to look at is the initial slope of the reaction, not the absolute magnitude as that is more subject to Mass Spec variance.</p>
<p><strong>Phosphorylation on SMAD:</strong></p>
<p><img class="alignnone size-full wp-image-2117" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/SMAD1_P.png" alt="" width="1475" height="598" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/SMAD1_P.png 1475w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/SMAD1_P-300x122.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/SMAD1_P-768x311.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/SMAD1_P-1024x415.png 1024w" sizes="(max-width: 1475px) 100vw, 1475px" /></p>
<p><em>Graphs of SMAD1 phosphorylation comparing the effect of different Alk2 forms (WT, Q207E and R206H mutations) and different type II receptors (ACVR2, BMPR2 active and BMPR2 kinase dead)</em></p>
<p>&nbsp;</p>
<p><strong>Phosphorylation on Alk2</strong></p>
<p><img class="alignnone size-full wp-image-2115" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/Alk2_P.png" alt="" width="1479" height="618" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/Alk2_P.png 1479w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/Alk2_P-300x125.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/Alk2_P-768x321.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/Alk2_P-1024x428.png 1024w" sizes="(max-width: 1479px) 100vw, 1479px" /></p>
<p><em>Graphs of Alk2 phosphorylation comparing the effect of different Alk2 forms (WT, Q207E and R206H mutations) and different type II receptors (ACVR2, BMPR2 active and BMPR2 kinase dead)</em></p>
<p>&nbsp;</p>
<p>Interpreting this data is quite tricky because you can look at phosphorylation on either protein while at the same time comparing the effects of the Alk2 mutations or you can compare the effects of the different type II receptors. I haven’t got enough data to be able to get any quantitative data on initial rates of reaction but the trends shown in this experiment match the trends seen in the previous experiment I ran of this type some time ago (before I started writing this blog).</p>
<p>So, what can we conclude from this experiment?</p>
<ul>
<li>SMAD1 phosphorylation is slower with wild type Alk2 than with either of the mutations (levels out at 20 minutes compared to 5 minutes aprox.)</li>
<li>When looking at SMAD1 phosphorylation, BMPR2 is a less good companion than ACVR2 when wild type Alk2 is used. Q207E somewhat counteracts this effect and R206H has the most pronounced counteraction bringing the rate of phosphorylation with BMPR2 to nearly equivalent to ACVR2.</li>
<li>In general the Q207E mutation is a greater facilitator of SMAD1 phosphorylation than wild type. The R206H mutation is better than both of them.</li>
<li>The rate of ALK2 phosphorylation is not effected by the type II protein used.</li>
<li>The rate of Alk2 phosphorylation does change depending on which construct is used. WT has the lowest rate, Q207E has a faster rate while R206H has the fastest rate.</li>
<li>The kinase dead version of BMPR2 leads to minimal difference in Alk2 phosphorylation but does reduce SMAD1 phosphorylation when compared to the BMPR2.</li>
</ul>
<p>&nbsp;</p>
<p>Next step: Try and repeat it again, however for that I need more wild type protein and we still don’t have the expression system back up and running for that.</p>
<p>For more details please look at the <a href="http://doi.org/10.5281/zenodo.1479804">Zenodo entry</a> for this pos</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/phosphorylation-of-smad1-rates-as-a-function-of-alk2-mutations-and-type-ii-association/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Structure Solution for BMPR1B/FKBP12 complex bound to M4K2009.</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/structure-solution-for-bmpr1b-fkbp12-complex-bound-to-m4k2009/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/structure-solution-for-bmpr1b-fkbp12-complex-bound-to-m4k2009/#respond</comments>
		<pubDate>Mon, 01 Oct 2018 14:52:06 +0000</pubDate>
		<dc:creator><![CDATA[Ellie Williams]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Brain Tumours]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Ellie Williams]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=1946</guid>
		<description><![CDATA[Structure Solution for BMPR1B/FKBP12 complex bound to M4K2009. Good news everybody! From the crystals I sent to the Diamond Light Source (a synchrotron, or particle accelerator, that gives very high quality x-rays) last week, I got a data set for BMPR1B/FKBP12 with M4K2009 bound to it! This is one of the compounds that the M4K <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/structure-solution-for-bmpr1b-fkbp12-complex-bound-to-m4k2009/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Structure Solution for BMPR1B/FKBP12 complex bound to M4K2009.</p>
<p>Good news everybody!</p>
<p>From the crystals I sent to the Diamond Light Source (a synchrotron, or particle accelerator, that gives very high quality x-rays) last week, I got a data set for BMPR1B/FKBP12 with M4K2009 bound to it! This is one of the compounds that the M4K Pharma company is very keen to see in situ to understand it&#8217;s action. While it would ideally be bound to ALK2 (something Ros is working on), seeing it bound to BMPR1B is also useful for understanding it&#8217;s selectivity (or lack of it) across various BMP signalling proteins.</p>
<p>The data looks like it refines down to around 2.34 Å. I’ve then spent much of last week analysing the data and trying to get a decent model of the proteins.</p>
<p>The data is in the P1 21 1 crystal form (this is just a way of describing the symmetry within the crystal) and has two copies of the complex in the asymmetric unit (which is another way of defining how the protein is packed inside the crystal). It’s refining nicely and I’ve got a very nice clear look at the compound bound to the binding site however I also noticed that several (but not all) exposed cysteines have been covalently modified by something in the plate well condition to give very distinct blobs of electron density that need to be identified.</p>
<p><img class="alignnone size-large wp-image-1948" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/10/compound_density-1024x739.jpg" alt="" width="780" height="563" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/10/compound_density-1024x739.jpg 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/10/compound_density-300x217.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/10/compound_density-768x555.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/10/compound_density.jpg 1238w" sizes="(max-width: 780px) 100vw, 780px" /></p>
<p><em>Compound M4K2009 bound to BMPR1B.</em></p>
<p>Looking back, the crystal was grown in 20% PEG8000, 0.2M Magnesium Acetate and 0.1M Cacodylate pH6.5.  Under these conditions it’s most likely that the cacodylate has reacted with the cys residue, most likely catalysed by neighbouring side chains deprotonating the SH group and allowing it to react with the arsenic at the centre of Cacodylate. However, there are two possible reactions that could occur, one of which leaves you with a sulphur bound to an arsenic with two methyl groups and an oxygen attached to it, and the other which leaves you with just two methyl groups bound to the arsenic. Now – consulting with my chemist colleagues they believe that the oxygen containing result is most likely and I’m inclined to agree as it is the simplest and most easily catalysed reaction. However, looking at the density and modelling in both options, it looks like the second might fit better. Either there is a more complex reaction occurring or it’s possible the oxygen has been knocked off by radiation damage during data collection.</p>
<p>Currently I’ve modeled them all as the second option since as a general rule what the data is telling you should win out over the prediction, but I can relatively easily change that if necessary. I need to get a second opinion on it at some point and any thoughts are greatly received.</p>
<p><img class="alignnone size-full wp-image-1949" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/10/cacodylate_modification.png" alt="" width="311" height="125" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/10/cacodylate_modification.png 311w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/10/cacodylate_modification-300x121.png 300w" sizes="(max-width: 311px) 100vw, 311px" /></p>
<p><em>Two possible options of cacodylate modification to cys residues.</em></p>
<p>Fortunately, none of the cys residues are near the binding site however so this isn’t an issue that is going to effect that result and I can be fairly confident that compound binding is not influenced by it.</p>
<p>You can read more details on the <a href="http://doi.org/10.5281/zenodo.1441877">processing on Zenodo</a>.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/structure-solution-for-bmpr1b-fkbp12-complex-bound-to-m4k2009/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
	</channel>
</rss>
