<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>USPs &#8211; openlabnotebooks.org</title>
	<atom:link href="https://thesgc.github.io/static-openlabnotebooks/category/project/usps/feed/" rel="self" type="application/rss+xml" />
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<description>A growing team of groundbreaking scientists around the world are now sharing their lab notebooks online</description>
	<lastBuildDate>Tue, 21 Jul 2020 12:14:51 +0000</lastBuildDate>
	<language>en-GB</language>
	<sy:updatePeriod>
	hourly	</sy:updatePeriod>
	<sy:updateFrequency>
	1	</sy:updateFrequency>
	<generator>https://wordpress.org/?v=5.5.3</generator>

<image>
	<url>https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/cropped-lab-jar02-32x32.png</url>
	<title>USPs &#8211; openlabnotebooks.org</title>
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<width>32</width>
	<height>32</height>
</image> 
	<item>
		<title>WDR5 and histone H3 interaction cell assay report</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/wdr5-and-histone-h3-interaction-cell-assay-report/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/wdr5-and-histone-h3-interaction-cell-assay-report/#respond</comments>
		
		<dc:creator><![CDATA[Magdalena Szewczyk]]></dc:creator>
		<pubDate>Tue, 21 Jul 2020 12:14:51 +0000</pubDate>
				<category><![CDATA[Cell assay reports]]></category>
		<category><![CDATA[Chromatin factors]]></category>
		<category><![CDATA[Magdalena Szewczyk]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<category><![CDATA[USPs]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=5016</guid>

					<description><![CDATA[  Background WDR5 belongs to the WD-repeat protein family and is an important component of various complexes, including SET1/MLL, NuRD, and NSL complex. It interacts with several binding partners via the &#8220;WDR5-interacting&#8221; (Win) site (e.g. symmetrically dimethylated histone H3R2, MLL1-4, SET1-2) or &#8220;WDR5-binding motif&#8221; (WBM) site (e.g. c,l,n-MYC, RBBP5). WDR5 emerged as a prime target <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/wdr5-and-histone-h3-interaction-cell-assay-report/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p><strong> </strong></p>
<p><strong>Background</strong></p>
<p>WDR5 belongs to the WD-repeat protein family and is an important component of various complexes, including SET1/MLL, NuRD, and NSL complex. It interacts with several binding partners via the &#8220;WDR5-interacting&#8221; (Win) site (e.g. symmetrically dimethylated histone H3R2, MLL1-4, SET1-2) or &#8220;WDR5-binding motif&#8221; (WBM) site (e.g. c,l,n-MYC, RBBP5). WDR5 emerged as a prime target for anti-cancer therapy due to its important roles in gene transcription, proliferation, and self-renewal (PIMID 29385767).</p>
<p><strong> </strong><strong>Assay validation</strong></p>
<p>Assay measuring the interaction of WDR5 with histone H3 was developed using NanoBRET technology &#8211; a proximity-based assay that can detect protein interactions by measuring energy transfer from a bioluminescent protein donor to a fluorescent protein acceptor. We tested different donor (N- or C-terminally NanoLuc® tagged WDR5) to acceptor (C-terminally HaloTag®Fusion-tagged histone H3.3) ratios in two different cell lines (U2Os and HEK293T). The assay was validated with WDR5 Win site antagonist &#8211; OICR9429 (PIMID:2695870). The best NanoBRET ratio and the greatest compound effect was obtained in U2Os cells transfected with 0.01 ug of C-terminally HaloTag®Fusion-tagged histone H3.3 and 0.001 µg of C- terminally NanoLuc® -tagged-WDR5 per 96-well (Fig.1).</p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/wdr5-228x300.jpg" alt="" width="368" height="484" class="wp-image-5018 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/wdr5-228x300.jpg 228w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/wdr5.jpg 490w" sizes="(max-width: 368px) 100vw, 368px" /></p>
<p><strong>Fig.1. OICR9429 decreases WDR5 interaction with histone H3.3 &#8211; NanoBRET assay.  </strong>U2Os or HEK293T cells were co-transfected with 0.001 µg of N- or C-terminally NL-tagged WDR5 and different amounts of C-terminally HT-tagged histone H3.3, as indicated (per 96/well) and treated with OICR9429 for 4 h. * &#8211; best assay condition,  NL-NanoLuc® tag, HT- HaloTag®Fusion tag</p>
<p>Experimental details can be found on <a href="https://zenodo.org/record/3953928#.Xxbb955Kg2w">Zenodo</a>.</p>
<p>&nbsp;</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/wdr5-and-histone-h3-interaction-cell-assay-report/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>p300/CBP cellular assay</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/p300-cbp-cellular-assay/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/p300-cbp-cellular-assay/#respond</comments>
		
		<dc:creator><![CDATA[Magdalena Szewczyk]]></dc:creator>
		<pubDate>Thu, 28 May 2020 16:23:26 +0000</pubDate>
				<category><![CDATA[Cell assay reports]]></category>
		<category><![CDATA[Magdalena Szewczyk]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<category><![CDATA[USPs]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=4793</guid>

					<description><![CDATA[Background Transcriptional coactivators p300 and CBP catalyze the acetylation of lysine residues of a large variety of histone and non-histone proteins (PMID:28426192). p300 and CBP are considered functional homologous proteins that share around 86% amino acid sequence identity in the HAT domain (PIMID:7870179).  The upregulation of p300/CBP has been associated with various cancers, indicating an important role in <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/p300-cbp-cellular-assay/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p><strong>Background</strong></p>
<p>Transcriptional coactivators p300 and CBP catalyze the acetylation of lysine residues of a large variety of histone and non-histone proteins (PMID:28426192). p300 and CBP are considered functional homologous proteins that share around 86% amino acid sequence identity in the HAT domain (PIMID:7870179).  The upregulation of p300/CBP has been associated with various cancers, indicating an important role in tumorigenesis (PIMID:27881443).</p>
<p><strong>Assay validation</strong></p>
<p>P300/CBP has been reported previously to acetylate p53 at lysine 320 and histone H3 at lysine 27 (PIMID: 25545885, 29804834). Based on the findings, we developed p300 cellular assays measuring p53 acetylation. As shown in Fig.1 overexpression of p300 but not its catalytic mutant (Y1467F) increased p53 acetylation levels. We also developed p300/CBP cellular assay measuring H3K27 acetylation.  Knocking down of p300/CBP resulted in a global decrease of H3K27 acetylation levels (Fig 2A). The H3K27 assay was further validated with p300/CBP selective inhibitor A-485 (PIMID: 29769713). A-485 decreased H3K27 acetylation levels in a dose-dependent manner in HFF cells (Fig.2B). Similar results were obtained in other cell lines (data not shown). The z factor for the assay equals 0.7.</p>
<p><strong> </strong></p>
<p><strong> <img loading="lazy" class="wp-image-4795 alignleft" height="363" width="290" alt="" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p300-1.png" /></strong></p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p><strong>Fig.1.</strong><strong> The wild type but not catalytic mutant of p300 acetylated exogenous p53.</strong>  HEK293T cells were co-transfected with FLAG-tagged p53 and empty vector(control),  p300 wild type, or p300 Y1467F catalytic mutant for 24 h<strong>.</strong> The acetylation levels were analyzed in Western blot. (Taylor Mitchell)</p>
<p><strong> </strong></p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p><strong> <img loading="lazy" class="alignnone wp-image-4794" height="248" width="845" alt="" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p300-2-300x88.png" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p300-2-300x88.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p300-2-1024x301.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p300-2-768x226.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p300-2.png 1280w" sizes="(max-width: 845px) 100vw, 845px" /></strong></p>
<p><strong> </strong><strong>Fig.2. The loss of p300/CBP activity results in a decrease of H3K27 acetylation levels. A. </strong>p300/CBP knock-down. HEK293T and MCF7 cells were treated with siRNA targeted for p300/CBP  and control siRNA for 3 days. <strong>B.</strong> p300/CBP inhibition. HFF cells were treated with p300/CBP selective inhibitor A-481 at indicated concentrations for 3 days. H3K27 acetylation levels were determined by Western blot. (Dalia Barsyte, Magdalena Szewczyk)</p>
<p>Experimental details are posted on <a href="https://zenodo.org/record/3862780#.Xs_k6DpKg2w">Zenodo</a>.</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/p300-cbp-cellular-assay/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
	</channel>
</rss>
