<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>PRMTs &#8211; openlabnotebooks.org</title>
	<atom:link href="https://thesgc.github.io/static-openlabnotebooks/category/project/prmts/feed/" rel="self" type="application/rss+xml" />
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<description>A growing team of groundbreaking scientists around the world are now sharing their lab notebooks online</description>
	<lastBuildDate>Tue, 05 May 2020 18:31:31 +0000</lastBuildDate>
	<language>en-GB</language>
	<sy:updatePeriod>
	hourly	</sy:updatePeriod>
	<sy:updateFrequency>
	1	</sy:updateFrequency>
	<generator>https://wordpress.org/?v=5.4.1</generator>

<image>
	<url>https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/cropped-lab-jar02-32x32.png</url>
	<title>PRMTs &#8211; openlabnotebooks.org</title>
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<width>32</width>
	<height>32</height>
</image> 
	<item>
		<title>PRMT3 cellular assay</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/prmt3-cellular-assay/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/prmt3-cellular-assay/#respond</comments>
		
		<dc:creator><![CDATA[Magdalena Szewczyk]]></dc:creator>
		<pubDate>Tue, 05 May 2020 18:31:31 +0000</pubDate>
				<category><![CDATA[Cell assay reports]]></category>
		<category><![CDATA[Magdalena Szewczyk]]></category>
		<category><![CDATA[PRMTs]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=4644</guid>

					<description><![CDATA[Background PRMT3 is type I protein arginine methyltransferase which mono- and asymmetrically dimethylates proteins, mainly in glycine and arginine-rich motifs (Fig.1, PIMID: 15175657). Fig.1. Mammalian PRMTs. There are 9 members and 3 types of the PRMT family. Type I, II and III PRMTs catalyze the formation of monomethyl arginines (Rme1). Type I PRMTs catalyze asymmetric arginine <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/prmt3-cellular-assay/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p><strong>Background</strong></p>
<p>PRMT3 is type I protein arginine methyltransferase which mono- and asymmetrically dimethylates proteins, mainly in glycine and arginine-rich motifs (Fig.1, PIMID: 15175657).</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1-300x119.png" alt="" width="630" height="250" class="wp-image-4626 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1-300x119.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1-1024x405.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1-768x304.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1.png 1214w" sizes="(max-width: 630px) 100vw, 630px" /></p>
<p><strong>Fig.1. </strong><strong>Mammalian PRMTs. </strong>There are 9 members and 3 types of the PRMT family. Type I, II and III PRMTs catalyze the formation of monomethyl arginines (Rme1). Type I PRMTs catalyze asymmetric arginine methylation (Rme2a), while type II PRMTs form symmetric arginine methylation (Rme2s). Type III PRMT can only monomethylate arginine residues. Modified from (PMID:29378138).</p>
<p><strong>Assay validation</strong></p>
<p>PRMT3 was shown to asymmetrically dimethylate histone H4 arginine 3 (H4R3) <em>in vitro </em>(PMID:22795084). Consistent with <em>in vitro </em>findings overexpression of wild type PRMT3 but not its catalytic mutant (E338Q) led to an increase in H4R3me2a (Fig.2). The assay was further validated with PRMT3 selective inhibitor SGC707, which decreased H4R3me2a levels in cells overexpressed with PRMT3 in a dose-dependant manner (Fig.3). The Z factor for the assay equals 0.59.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p3a-300x160.png" alt="" width="437" height="233" class="wp-image-4646 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p3a-300x160.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p3a-768x409.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p3a.png 846w" sizes="(max-width: 437px) 100vw, 437px" /></p>
<p><strong>Fig.2. The wild type but not catalytic mutant (E338Q) of PRMT3 methylated endogenous H4R3.</strong> HEK293T cells were transfected with empty vector (control), FLAG-tagged PRMT3 wild type (PRMT3wt) or catalytic mutant (PRMT3mut) for 24 h. The methylation levels of H4R3 were analyzed in Western blot.</p>
<p>&nbsp;</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p3-300x86.png" alt="" width="691" height="198" class="alignnone wp-image-4645" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p3-300x86.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p3-1024x293.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p3-768x220.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p3.png 1280w" sizes="(max-width: 691px) 100vw, 691px" /></p>
<p><strong>Fig.3.</strong> <strong>SGC707 decreases PRMT3 dependent H4R3 asymmetric dimethylation in cells</strong>. HEK293T cells were transfected with FLAG-tagged PRMT3 wild type (PRMT3wt) or catalytic (E338Q) mutant (PRMT3mut) and treated with PRMT3 selective inhibitor, SGC7070, for 20 h. H4R3me2a levels were determined by Western blot. The graph represents the nonlinear fit of H4R3me2a signal intensities normalized to total histone H4. The results are mean +/- SEM of 3 replicates.</p>
<p>Please visit <a href="https://zenodo.org/record/3787392#.XrGwgahKg2w">Zenodo</a>, for experimental details.</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/prmt3-cellular-assay/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>PRMT6 cellular assay</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/prmt6-cellular-assay/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/prmt6-cellular-assay/#respond</comments>
		
		<dc:creator><![CDATA[Magdalena Szewczyk]]></dc:creator>
		<pubDate>Tue, 05 May 2020 16:58:34 +0000</pubDate>
				<category><![CDATA[Cell assay reports]]></category>
		<category><![CDATA[Magdalena Szewczyk]]></category>
		<category><![CDATA[PRMTs]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=4638</guid>

					<description><![CDATA[Background PRMT6 is type I protein arginine methyltransferase which mono- and asymmetrically dimethylates histone and nonhistone proteins, mainly in glycine and arginine rich motifs (Fig.1). PRMT6 shows predominant nuclear localization and is implicated in the regulation of nuclear processes such as DNA repair and gene expression (PIMID: 11724789, 18079182).  Fig.1. Mammalian PRMTs. There are 9 <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/prmt6-cellular-assay/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p><strong>Background</strong></p>
<p>PRMT6 is type I protein arginine methyltransferase which mono- and asymmetrically dimethylates histone and nonhistone proteins, mainly in glycine and arginine rich motifs (Fig.1). PRMT6 shows predominant nuclear localization and is implicated in the regulation of nuclear processes such as DNA repair and gene expression (PIMID: 11724789, 18079182).</p>
<p><strong> <img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1-300x119.png" alt="" width="550" height="218" class="wp-image-4626 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1-300x119.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1-1024x405.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1-768x304.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1.png 1214w" sizes="(max-width: 550px) 100vw, 550px" /></strong></p>
<p><strong> </strong><strong>Fig.1. </strong><strong>Mammalian PRMTs. </strong>There are 9 members and 3 types of the PRMT family. Type I, II and III PRMTs catalyze the formation of monomethyl arginines (Rme1). Type I PRMTs catalyze asymmetric arginine methylation (Rme2a), while type II PRMTs form symmetric arginine methylation (Rme2s). Type III PRMT can only monomethylate arginine residues. Modified from (PMID:29378138).</p>
<p><strong>Assay validation</strong></p>
<p>It has been reported that PRMT6 is the main contributor to histone H3 arginine 2 asymmetric dimethylation (H3R2me2a) in cells (PIMID: 17898714). We found that knocking down of PRMT6 for 3 days is not sufficient to observe a decent decrease in H3R2me2a levels, however, overexpression of wild type PRMT6 but not its catalytic mutant (V86K/D88A) led to increase in H3R2me2a levels (Fig.2). Overexpression of PRMT6 also increased H3 arginine 8 asymmetric dimethylation (H3R8me2a) and H4 arginine 3 asymmetric dimethylation (H4R3me2a) levels (Fig.2). The assay was further validated with PRMT Type I chemical probe (MS023). MS023 decreased PRMT6 dependent H3R2 asymmetric dimethylation in cells overexpressed with PRMT6 in a dose-dependent manner (Fig.3). The Z factor for the assay equals 0.63.</p>
<p><strong> </strong><strong> <img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/prmt6-209x300.png" alt="" width="298" height="428" class="wp-image-4639 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/prmt6-209x300.png 209w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/prmt6.png 411w" sizes="(max-width: 298px) 100vw, 298px" /></strong></p>
<p><strong>Fig.2. The wild type but not catalytic mutant of PRMT6 asymmetrically dimethylated endogenous H3R2, H3R8, and H4R3.</strong>  HEK293T cells were transfected with FLAG-tagged PRMT6 wild type or V86K/D88A catalytic mutant (PRMT6mut) or empty vector (control) for 24 h. The methylation levels were analyzed in Western Blot.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/prmt6a-300x75.png" alt="" width="724" height="181" class="alignnone wp-image-4640" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/prmt6a-300x75.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/prmt6a-1024x257.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/prmt6a-768x193.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/prmt6a.png 1260w" sizes="(max-width: 724px) 100vw, 724px" /></p>
<p><strong>Fig.3.</strong> <strong>MS023 decreases PRMT6 dependent H3R2 asymmetric dimethylation in cells</strong>. HEK293T cells were transfected with FLAG-tagged PRMT6 wild type or V86K/D88A catalytic mutant (PRMT6mut) and treated with inhibitor for 20 h. H3R2me2a levels were determined by Western blot. The graph represents the nonlinear fit of H3R2me2a signal intensities normalized to total histone H3. The results are mean +/- SEM of 3 replicates.</p>
<p>Experimental details are posted on <a href="https://zenodo.org/record/3787176#.XrGbAqhKg2w">Zenodo</a>.</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/prmt6-cellular-assay/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>PRMT4 (CARM1) cellular assay</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/prmt4-carm1-cellular-assay/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/prmt4-carm1-cellular-assay/#respond</comments>
		
		<dc:creator><![CDATA[Magdalena Szewczyk]]></dc:creator>
		<pubDate>Mon, 04 May 2020 16:08:06 +0000</pubDate>
				<category><![CDATA[Cell assay reports]]></category>
		<category><![CDATA[E3 ligases]]></category>
		<category><![CDATA[Magdalena Szewczyk]]></category>
		<category><![CDATA[PRMTs]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=4630</guid>

					<description><![CDATA[Background PRMT4 (CARM1) is type I protein arginine methyltransferase which mono- or asymmetrically dimethylates proteins (Fig.1) within proline-methionine rich motives, that are involved in transcription, splicing and RNA processing such as H3R17, EP300, PABP1, CA150, BAF155, MED12 (PIMID:19150423). Fig.1. Mammalian PRMTs. There are 9 members and 3 types of the PRMT family. Type I, II <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/prmt4-carm1-cellular-assay/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p><strong>Background</strong></p>
<p>PRMT4 (CARM1) is type I protein arginine methyltransferase which mono- or asymmetrically dimethylates proteins (Fig.1) within proline-methionine rich motives, that are involved in transcription, splicing and RNA processing such as H3R17, EP300, PABP1, CA150, BAF155, MED12 (PIMID:19150423).</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1-300x119.png" alt="" width="527" height="209" class="wp-image-4626 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1-300x119.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1-1024x405.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1-768x304.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1.png 1214w" sizes="(max-width: 527px) 100vw, 527px" /></p>
<p><strong>Fig.1. </strong><strong>Mammalian PRMTs. </strong>There are 9 members and 3 types of the PRMT family. Type I, II and III PRMTs catalyze the formation of monomethyl arginines (Rme1). Type I PRMTs catalyze asymmetric arginine methylation (Rme2a), while type II PRMTs form symmetric arginine methylation (Rme2s). Type III PRMT can only monomethylate arginine residues. Modified from (PMID:29378138).</p>
<p><strong>Assay validation</strong></p>
<p>It has been reported that the RNA polymerase II mediator complex subunit 12 (MED12) and BAF155, are PRMT4 substrates (PIMID: 26601288, 25927994). Consistently with previous findings, PRMT4 knock-down decreased asymmetric dimethylation of MED12 and BAF155 (Fig.2). The assay was validated with PRMT4 selective chemical probe TP-064 (PIMID: 29719619). TP-064 decreased PRMT4 asymmetric dimethylation of MED12 (Z factor = 0.6) and BAF155 (Z factor = 0.76) in a dose-dependent manner (Fig.3).</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p42-300x245.png" alt="" width="417" height="340" class="alignnone wp-image-4631" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p42-300x245.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p42-768x627.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p42.png 846w" sizes="(max-width: 417px) 100vw, 417px" /></p>
<p><strong>Fig.2. PRMT4 knock-down resulted in a decrease of MED12 and BAF155 asymmetric dimethylation (Rme2a) levels. </strong>HEK293T cells were treated with siRNA for 3 days and 6 days. The levels of methylation were analyzed in Western Blot.<strong> </strong></p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p4-3-300x120.png" alt="" width="623" height="249" class="alignnone wp-image-4632" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p4-3-300x120.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p4-3-1024x410.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p4-3-768x308.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p4-3.png 1261w" sizes="(max-width: 623px) 100vw, 623px" /></p>
<p><strong>Fig. 3.</strong> <strong>TP-064 </strong><strong>decreases PRMT4 dependent MED12 and BAF155 asymmetric dimethylation (Rme2a) levels.</strong> HEK293T cells were treated with PRMT4 selective inhibitor (TP-064) at indicated concentrations for 48 h and BAF155-Rme2a (A) and MED12-Rme2a (B) levels were determined by Western blot. The graphs represent nonlinear fits of methyl signal intensities normalized to total protein levels. The results are mean +/- SEM of 3 replicates.</p>
<p>Please go to <a href="https://zenodo.org/record/3785034#.XrA9AKhKg2w">Zenodo</a> for experimental details.</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/prmt4-carm1-cellular-assay/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>PRMT5 cellular assay</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/prmt5-cellular-assay/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/prmt5-cellular-assay/#respond</comments>
		
		<dc:creator><![CDATA[Magdalena Szewczyk]]></dc:creator>
		<pubDate>Mon, 27 Apr 2020 15:15:00 +0000</pubDate>
				<category><![CDATA[Cell assay reports]]></category>
		<category><![CDATA[Magdalena Szewczyk]]></category>
		<category><![CDATA[PRMTs]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=4602</guid>

					<description><![CDATA[Background PRMT5 in complex with the WD-repeat protein MEP50 is the major type II protein arginine methyltransferase responsible for mono- and symmetric dimethylation of arginine in a great variety of proteins (Fig.1). It plays a critical role in diverse cellular processes such as genome organization, transcription, translation, differentiation, cell cycle, DNA repair,  RNA processing, and <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/prmt5-cellular-assay/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p><strong>Background</strong></p>
<p>PRMT5 in complex with the WD-repeat protein MEP50 is the major type II protein arginine methyltransferase responsible for mono- and symmetric dimethylation of arginine in a great variety of proteins (Fig.1). It plays a critical role in diverse cellular processes such as genome organization, transcription, translation, differentiation, cell cycle, DNA repair,  RNA processing, and metabolism (PMID:25662273). PRMT5 expression levels and methyltransferase activity were found to be dysregulated in a variety of cancers (PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25662273">25662273</a>).</p>
<p><strong> <img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1-300x119.png" alt="" width="608" height="241" class="wp-image-4626 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1-300x119.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1-1024x405.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1-768x304.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1.png 1214w" sizes="(max-width: 608px) 100vw, 608px" /><br />
</strong></p>
<p><strong>Fig.1. </strong><strong>Mammalian PRMTs. </strong>There are 9 members and 3 types of the PRMT family. Type I, II and III PRMTs catalyze the formation of monomethyl arginines (Rme1). Type I PRMTs catalyze asymmetric arginine methylation (Rme2a), while type II PRMTs form symmetric arginine methylation (Rme2s). Type III PRMT can only monomethylate arginine residues. Modified from (PMID:29378138).</p>
<p><strong> </strong></p>
<p><strong>Assay validation</strong></p>
<p>It has been reported previously, that the various SMN complex proteins, including SmBB’ are PRMT5 substrates (PMID:19520849). We found that the knocking down of PRMT5, but not PRMT1, 3, 4, 6 and  7 resulted in decreased SmBB’ symmetric dimethylation levels (Fig.2). PRMT9, another type II PRMT, is also not affecting SmBB’ Rme2s levels (PIMID: 25737013). The assay was further validated by treatment with selective PRMT5 chemical probe, GSK591. GSK591 decreased SmBB’-Rme2s levels in a dose-dependent manner (Fig.3). The assay was also used to validate the cellular activity of another PRMT5 selective chemical probe LLY-283 (PMID:30034588). Similar results were obtained with different cell lines (data not shown).</p>
<p>&nbsp;</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/p5-2-300x225.png" alt="" width="712" height="534" class="wp-image-4604 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/p5-2-300x225.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/p5-2-768x576.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/p5-2-350x263.png 350w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/p5-2-780x586.png 780w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/p5-2-250x188.png 250w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/p5-2.png 945w" sizes="(max-width: 712px) 100vw, 712px" /></p>
<p><strong>Fig.2. PRMT5 knock-down results in decrease in SmBB’-Rme2s levels. </strong>MCF7 cells were treated with siRNA targeted for different PRMTs and control siRNA for 6 (<strong>A</strong>) and 3 (<strong>B</strong>) days. <strong>A.</strong> Western blot analysis with an antibody recognizing symmetric arginine methylation (Rme2s) of various proteins. PRMTs expression after knock-down (KD) and β-actin controls are shown. Note that the loss of PTMT1 leads to decrease of Rme2a modification and the shared substrates become targets for PRMT5 leading to increase in Rme2s levels.<strong>B.</strong> Western blot analysis of the SmBB’-Rme2s levels after PRMT5 KD. <strong>C.</strong> Quantification of signal intensities of SmBB’-Rme2s normalized to total SmBB’ (n=1).</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/p5-3-300x83.png" alt="" width="647" height="179" class="wp-image-4603 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/p5-3-300x83.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/p5-3-1024x282.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/p5-3-768x212.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/p5-3.png 1280w" sizes="(max-width: 647px) 100vw, 647px" /></p>
<p><strong>Fig.3.</strong> <strong>PRMT5 chemical probe, GSK591, decreases SmBB’ symmetric dimethylation in a dose-dependent manner.</strong> MCF7 cells were treated with inhibitor at indicated concentrations for 48 h and SmBB’-Rme2s and total SmBB’ levels were determined by Western blot. The graph represents nonlinear fits of SmBB’-Rme2s signal intensities normalized to total SmBB’. The results are mean +/- SEM of 3 replicates. The Z factor for the assay equals 0.67 (n=3).</p>
<p>Please visit <a href="https://zenodo.org/record/3774168#.Xqhm52hKg2w">Zenodo</a> for experimental details.</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/prmt5-cellular-assay/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>PRMT1 cellular assay</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/prmt1-cell-assay-report/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/prmt1-cell-assay-report/#respond</comments>
		
		<dc:creator><![CDATA[Magdalena Szewczyk]]></dc:creator>
		<pubDate>Thu, 23 Apr 2020 15:51:40 +0000</pubDate>
				<category><![CDATA[Cell assay reports]]></category>
		<category><![CDATA[Magdalena Szewczyk]]></category>
		<category><![CDATA[PRMTs]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=4569</guid>

					<description><![CDATA[Background PRMT1 is the type I protein arginine methyltransferase (Fig.1) which mono- and asymmetrically dimethylates histone and nonhistone proteins, mainly in glycine and arginine-rich motifs and is involved in various cellular processes such as transcription, RNA processing, DNA repair, cell cycle or signal transduction (PMID:26612103). It is responsible for depositing over 90% of the total <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/prmt1-cell-assay-report/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p><strong>Background</strong></p>
<p>PRMT1 is the type I protein arginine methyltransferase (Fig.1) which mono- and asymmetrically dimethylates histone and nonhistone proteins, mainly in glycine and arginine-rich motifs and is involved in various cellular processes such as transcription, RNA processing, DNA repair, cell cycle or signal transduction (PMID:26612103). It is responsible for depositing over 90% of the total asymmetric arginine dimethyl mark (PMID:23419748). It has been identified, that at least seven distinct PRMT1 isoforms are generated by complex alternative splicing in the 5’ region of its pre-mRNA, however, their exact role has not been fully understood (PMID:11448779, 11387442). Deregulated PRMT1 expression has been observed in a number of tumor types including brain, lung, breast, colon, bladder, and leukemia, which make it a potential target for anti-cancer therapy (PMID:23235912, 28061334).</p>
<p><strong> <img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1-300x119.png" alt="" width="545" height="216" class="wp-image-4626 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1-300x119.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1-1024x405.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1-768x304.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1.png 1214w" sizes="(max-width: 545px) 100vw, 545px" /> </strong></p>
<p><strong>Fig.1. </strong><strong>Mammalian PRMTs. </strong>There are 9 members and 3 types of the PRMT family. Type I, II and III PRMTs catalyze the formation of monomethyl arginines (Rme1). Type I PRMTs catalyze asymmetric arginine methylation (Rme2a), while type II PRMTs form symmetric arginine methylation (Rme2s). Type III PRMT can only monomethylate arginine residues. Modified from (PMID:29378138).</p>
<p><strong>PRMT1 assay validation</strong></p>
<p>It has been reported that PRMT1 is the main contributor to H4R3 (histone 4 arginine 3) asymmetric dimethylation (H4R3me2a) in cells (PMID:17848568). Moreover, with the loss of PRMT1, a large number of substrates become targets for Type II and III PRMTs, because these substrates are presumably no longer blocked by Rme2a modification (PMID: 23419748). We found that knocking down of PRMT1, but not PRMT3, 4, 5, 6, or 7 decreased basal H4R3me2a levels and increased Rme1 and Rme2s levels in MCF7 cells (Fig.2). MCF7 cell line was chosen for the PRMT1 cellular assay due to the high basal levels of H4R3me2a (Fig.3). The assay was further validated with PRMT Type I chemical probe MS023 (PMID:26598975). MS023 decreased the levels of H4R3me2a in a dose-dependent manner (Fig. 4). The Z factor for the assay equals 0.83 (n=3).</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/p1fig2-1-140x300.png" alt="" width="280" height="600" class="wp-image-4576 alignleft" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/p1fig2-1-140x300.png 140w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/p1fig2-1-477x1024.png 477w" sizes="(max-width: 280px) 100vw, 280px" /></p>
<p><strong>Fig.2. PRMT1 knock-down results in decrease in histone 4 arginine 3 asymmetric dimethylation levels and increase in global arginine mono- and dimethylation.</strong> MCF7 cells were treated with siRNA targeted for different PRMTs and control siRNA for 6 days and analysed by western blot for PRMTs expression, histone 4 arginine 3 asymmetric dimethylation (H4R3me2a), symmetric arginine methylation (Rme2s) and monomethylation (Rme1) levels.</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/p1fig3a-300x157.png" alt="" width="655" height="343" class="wp-image-4575 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/p1fig3a-300x157.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/p1fig3a-1024x535.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/p1fig3a-768x402.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/p1fig3a.png 1440w" sizes="(max-width: 655px) 100vw, 655px" /></p>
<p><strong>Fig.3. MCF7 cells have high basal levels of H4R3Rme2a mark. </strong>Western blot analysis of basal H4R3me2a levels in various cell lines. The bars represent H4R3me2a fluorescent intensities normalized to total H4 (n=1).</p>
<p><strong> <img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/p1f4-300x81.png" alt="" width="737" height="199" class="wp-image-4578 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/p1f4-300x81.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/p1f4-1024x278.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/p1f4.png 1440w" sizes="(max-width: 737px) 100vw, 737px" /></strong></p>
<p><strong>Fig.4.</strong> <strong>MS023 decreases PRMT1 dependent H4R3 asymmetric dimethylation in cells</strong>. MCF7 cells were treated with inhibitor at indicated concentrations for 48 h and H4R3me2a levels were determined by western blot. The graph represents nonlinear fits of H4R3me2a signal intensities normalized to total histone H4. The results are mean +/- SEM of 3 replicates.</p>
<p>For assay details, please go to <a href="https://zenodo.org/record/3774163#.Xqhl9mhKg2w">Zenodo</a>.</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/prmt1-cell-assay-report/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>Identifying mono methyl arginines with  PTMScan® Mono-Methyl Arginine Motif [mme-RG] Kit from CST</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/identifying-mono-methyl-arginines-with-ptmscan-mono-methyl-arginine-motif-mme-rg-kit-from-cst/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/identifying-mono-methyl-arginines-with-ptmscan-mono-methyl-arginine-motif-mme-rg-kit-from-cst/#comments</comments>
		
		<dc:creator><![CDATA[Magdalena Szewczyk]]></dc:creator>
		<pubDate>Mon, 09 Dec 2019 12:59:19 +0000</pubDate>
				<category><![CDATA[Cell assay reports]]></category>
		<category><![CDATA[Magdalena Szewczyk]]></category>
		<category><![CDATA[PRMTs]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=3855</guid>

					<description><![CDATA[We have used the PTMScan® Mono-Methyl Arginine Motif [mme-RG] Kit #12235 from CST in order to determine novel PRMT7 substrates. In preliminary experiments done few years ago by our pharma partner several novel potential PRMT7 substrates were discovered with the kit, including HSP70, which was further validated by different methods (https://www.biorxiv.org/content/10.1101/503136v2). In order to obtain biological replicates, <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/identifying-mono-methyl-arginines-with-ptmscan-mono-methyl-arginine-motif-mme-rg-kit-from-cst/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p>We have used the PTMScan<sup>®</sup> Mono-Methyl Arginine Motif [mme-RG] Kit #12235 from CST in order to determine novel PRMT7 substrates. In preliminary experiments done few years ago by our pharma partner several novel potential PRMT7 substrates were discovered with the kit, including HSP70, which was further validated by different methods (<a href="https://www.biorxiv.org/content/10.1101/503136v2">https://www.biorxiv.org/content/10.1101/503136v2</a>). In order to obtain biological replicates, recently, we used the same kit. In the initial experiment we followed exactly CST’s protocol ( <a href="https://media.cellsignal.com/pdf/12235.pdf">https://media.cellsignal.com/pdf/12235.pdf</a>) and observed low peptide counts and high background in the IP-ed samples and no presence of our positive control HSP70 peptide FELTGIPPAPR. Since there was some ambiguity about the desalting step in section 5, we had several discussions with CST support. The fruitful discussions compelled us to share a more detailed protocol with the goal of improving the success rates of these enrichments.</p>
<p>In our second attempt, the input or ‘pre-IP’ samples had high counts of HSP70 peptide FELTGIPPAPR, which R is methylated but not heavy labelled in PRMT7 WT and unmethylated in heavy labelled peptide (PRMT7 KO), which is with agreement with our previous findings Fig.1. The HSP70 methylation is visibly detected by anti-Rme1 CST antibody (the same one used in the kit) in western blot, so it should be immunoprecipitated, however the peptide could only be detected in one of the 3 immunoprecipitated samples and at very low counts. We have no idea why it is not working in our hands, but if anyone consider using this kit, I recommend looking at my comments on <a href="https://zenodo.org/record/3567916#.Xe5D-ehKg2w">zenodo</a> . Good luck!</p>
<p>&nbsp;</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/12/Overview-SILAC-300x169.png" alt="" class="alignnone wp-image-3874" width="1093" height="616" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/12/Overview-SILAC-300x169.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/12/Overview-SILAC-768x432.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/12/Overview-SILAC-1024x576.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/12/Overview-SILAC.png 1280w" sizes="(max-width: 1093px) 100vw, 1093px" /></p>
<p>Fig.1. The MS data generated by Suzanne Ackloo and Amber Couzens.</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/identifying-mono-methyl-arginines-with-ptmscan-mono-methyl-arginine-motif-mme-rg-kit-from-cst/feed/</wfw:commentRss>
			<slash:comments>1</slash:comments>
		
		
			</item>
	</channel>
</rss>
