<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>Jolene Ho &#8211; openlabnotebooks.org</title>
	<atom:link href="https://thesgc.github.io/static-openlabnotebooks/category/past-scientist/jolene-ho/feed/" rel="self" type="application/rss+xml" />
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<description>A growing team of groundbreaking scientists around the world are now sharing their lab notebooks online</description>
	<lastBuildDate>Thu, 09 May 2019 14:48:52 +0000</lastBuildDate>
	<language>en-GB</language>
	<sy:updatePeriod>hourly</sy:updatePeriod>
	<sy:updateFrequency>1</sy:updateFrequency>
	<generator>https://wordpress.org/?v=4.9.10</generator>

<image>
	<url>https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/cropped-lab-jar02-32x32.png</url>
	<title>Jolene Ho &#8211; openlabnotebooks.org</title>
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<width>32</width>
	<height>32</height>
</image> 
	<item>
		<title>Further Investigation of the Reduction of HTT Protein through Chemical Inhibition</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/further-investigation-of-the-reduction-of-htt-protein-through-chemical-inhibition/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/further-investigation-of-the-reduction-of-htt-protein-through-chemical-inhibition/#respond</comments>
		<pubDate>Tue, 15 Jan 2019 18:56:28 +0000</pubDate>
		<dc:creator><![CDATA[Jolene Ho]]></dc:creator>
				<category><![CDATA[Huntingtin]]></category>
		<category><![CDATA[Huntington’s disease]]></category>
		<category><![CDATA[Jolene Ho]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2378</guid>
		<description><![CDATA[The experimental set-up and results covered in this post can be found here: 10.5281/zenodo.2541053. A lot has happened in the past few months and it is with mixed feelings that I am finishing up my postdoc here at SGC! I feel very privileged to have been part of the great science going on at SGC <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/further-investigation-of-the-reduction-of-htt-protein-through-chemical-inhibition/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p style="text-align: justify;">The experimental set-up and results covered in this post can be found here: <a href="http://doi.org/10.5281/zenodo.2541053">10.5281/zenodo.2541053</a>.</p>
<p style="text-align: justify;">A lot has happened in the past few months and it is with mixed feelings that I am finishing up my postdoc here at SGC! I feel very privileged to have been part of the great science going on at SGC and a pioneer of the Open Notebook initiative here, contributing to open science. Nonetheless, the science has to go on! Hopefully, my posts in this Open Notebook have provided new perspectives to people working on similar projects and trying to learn more about Huntington’s Disease and huntingtin (HTT), inspired people to strive on for science despite the negative and/or inconclusive data, or at least given some insight to how life on the bench is like in general.</p>
<p style="text-align: justify;">Previously, I showed that the use of PRMT5 inhibitors from the SGC probe library, namely GSK591 and LLY283, were able to decrease levels of HTT in HeLa cells. SGC2096, the inactive control to GSK591, was not able to decrease HTT protein levels. This, together with the different mechanisms of action of GSK591 and LLY283 in inhibiting PRMT5, demonstrates that it is indeed through the specific inhibition of PRMT5 that HTT protein levels were decreased. So far, I have not come across any drug that is able to regulate the levels of HTT protein. As such, the inhibition of PRMT5 to reduce the expression of HTT at the protein level presents a potential and exciting therapeutic avenue for the treatment of Huntington’s disease.</p>
<p style="text-align: justify;">The findings that PRMT5 inhibition itself can affect the expression of HTT at the protein level suggests that there might be some interaction and possible feedback loop in play between PRMT5 and HTT. We thus further investigated how PRMT5 inhibition may contribute to the decrease in HTT protein levels to elucidate the mechanisms involved in this process. As PRMT5 is the main enzyme known to mediate symmetric demethylation of arginine (SDMA), we thus checked how the levels of SDMA are affected by PRMT5 inhibition and HTT knockdown.</p>
<p style="text-align: justify;"><img class="alignnone wp-image-2379" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/HeLa-G3BP1-PRMT5i-in-HTT-KD-300x225.jpg" alt="" width="771" height="578" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/HeLa-G3BP1-PRMT5i-in-HTT-KD-300x225.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/HeLa-G3BP1-PRMT5i-in-HTT-KD-768x576.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/HeLa-G3BP1-PRMT5i-in-HTT-KD-350x263.jpg 350w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/HeLa-G3BP1-PRMT5i-in-HTT-KD-780x586.jpg 780w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/HeLa-G3BP1-PRMT5i-in-HTT-KD-250x188.jpg 250w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/HeLa-G3BP1-PRMT5i-in-HTT-KD.jpg 960w" sizes="(max-width: 771px) 100vw, 771px" /></p>
<p style="text-align: justify;">Unsurprisingly, PRMT5 inhibition with GSK591 or LLY283 led to the decrease in SDMA. However, we also noticed a slight decrease in SDMA across all 3 HTT knockdown samples! Although this decrease is marginal, its occurrence across all the HTT knockdown samples we tested gives confidence to this observation and corroborates the hypothesis that PRMT5 and HTT are part of a feedback loop rather than that HTT is simply downstream of PRMT5, since the inhibition of PRMT5 leads to a decrease in HTT, and a decrease in HTT by way of HTT knockdown leads to a decrease in SDMA, which is mainly mediated by PRMT5.</p>
<p style="text-align: justify;">The two-way crosstalk demonstrated by PRMT5 activity levels affecting the expression of HTT and the presence (or absence, since we used a knockdown model) of HTT likewise decreasing SDMA levels suggests that the relationship between PRMT5 and HTT is complicated and reciprocal, with one regulating the other possibly through post-translational modifications such as SDMA and other epigenetic mechanisms, and warrants further investigation as to how the activity and levels of PRMT5 and HTT are related. Some important future experiments would be to test if PRMT inhibition is also able to decrease the protein levels of mutant HTT similarly to how it affects wild-type HTT, and if so, this will be an exciting opportunity to follow up on in the search of a potential therapy for Huntington’s disease.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/further-investigation-of-the-reduction-of-htt-protein-through-chemical-inhibition/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Reduction of HTT Protein through Chemical Inhibition</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/reduction-of-htt-protein-through-chemical-inhibition/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/reduction-of-htt-protein-through-chemical-inhibition/#comments</comments>
		<pubDate>Thu, 22 Nov 2018 20:34:38 +0000</pubDate>
		<dc:creator><![CDATA[Jolene Ho]]></dc:creator>
				<category><![CDATA[Huntingtin]]></category>
		<category><![CDATA[Huntington’s disease]]></category>
		<category><![CDATA[Jolene Ho]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2169</guid>
		<description><![CDATA[The experimental set-up and results covered in this post can be found here: 10.5281/zenodo.1495129. Apologies for the long delay between this post and my last post, it really has been quite some time! As postdocs, we generally have several projects running simultaneously and we have to split our time between these projects. I am no <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/reduction-of-htt-protein-through-chemical-inhibition/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p style="text-align: justify;">The experimental set-up and results covered in this post can be found here: <a href="http://doi.org/10.5281/zenodo.1495129">10.5281/zenodo.1495129</a>.</p>
<p style="text-align: justify;">Apologies for the long delay between this post and my last post, it really has been quite some time! As postdocs, we generally have several projects running simultaneously and we have to split our time between these projects. I am no exception and lately I have been caught up in trying to wrap up several projects. Besides my work in HTT, I am involved in screening various cell lines against our SGC probe library and identifying probes that have a differential effect in these cell lines. At the SGC, chemical compounds only qualify as probes if they have an in vitro potency of &lt;100 nM, &gt;30-fold selectivity vs other subfamilies, and have significant on-target cell activity at 1 µM. These probes are essential for the early stages of drug development as positive hits in our screens allow us to identify potential drug targets, protein families, and pathways for which we can further investigate in terms of disease mechanism and therapeutic strategies. These probes identified as positive hits in our screens may then be further developed and refined as drugs which may be used in the clinics eventually. You can read more about the SGC’s probes here: <a href="https://www.thesgc.org/chemical-probes">https://www.thesgc.org/chemical-probes</a>.</p>
<p style="text-align: justify;">In one of the other projects I’m working on, we identified PRMT5 as a potential therapeutic target in glioblastoma through the action of GSK591 and LLY283, 2 probes that inhibit PRMT5 through different and independent mechanisms. The inhibition of PRMT5 as a therapeutic strategy is not novel in itself, with several other groups having already published on this.</p>
<p>&nbsp;</p>
<p style="text-align: justify;">PRMT5 as a druggable target for glioblastoma therapy.</p>
<p style="text-align: justify;">Banasavadi-Siddegowda YK, Welker AM, An M, Yang X, Zhou W, Shi G, Imitola J, Li C, Hsu S, Wang J, Phelps M, Zhang J, Beattie CE, Baiocchi R, Kaur B.</p>
<p style="text-align: justify;">Neuro Oncol. 2018 May 18;20(6):753-763. doi: <a href="http://doi.org/10.1093/neuonc/nox206">10.1093/neuonc/nox206</a>.</p>
<p>&nbsp;</p>
<p style="text-align: justify;">Coordinated Splicing of Regulatory Detained Introns within Oncogenic Transcripts Creates an Exploitable Vulnerability in Malignant Glioma.</p>
<p style="text-align: justify;">Braun CJ, Stanciu M, Boutz PL, Patterson JC, Calligaris D, Higuchi F, Neupane R, Fenoglio S, Cahill DP, Wakimoto H, Agar NYR, Yaffe MB, Sharp PA, Hemann MT, Lees JA.</p>
<p style="text-align: justify;">Cancer Cell. 2017 Oct 9;32(4):411-426.e11. doi: <a href="http://doi.org/10.1016/j.ccell.2017.08.018">10.1016/j.ccell.2017.08.018</a>. Epub 2017 Sep 28.</p>
<p>&nbsp;</p>
<p style="text-align: justify;">However, we had some interesting findings through the RNA-sequencing done on cells treated with these PRMT5 inhibitors. Serendipitously, HTT emerged as one of the genes whose expression was consistently downregulated with PRMT5 inhibition! I tested these PRMT5 inhibitors in the HeLa cells I used to generate the 3 stable HTT knockdown lines previously (read more about that here: https://thesgc.github.io/static-openlabnotebooks/generating-htt-knockdown-models/) and looked at how the HTT protein levels were affected in these lines.</p>
<p style="text-align: justify;"><img class="alignnone wp-image-2170" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/20181116-HeLa-G3BP1-PRMT5i-and-HTT-KD-300x225.jpg" alt="" width="841" height="631" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/20181116-HeLa-G3BP1-PRMT5i-and-HTT-KD-300x225.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/20181116-HeLa-G3BP1-PRMT5i-and-HTT-KD-768x576.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/20181116-HeLa-G3BP1-PRMT5i-and-HTT-KD-350x263.jpg 350w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/20181116-HeLa-G3BP1-PRMT5i-and-HTT-KD-780x586.jpg 780w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/20181116-HeLa-G3BP1-PRMT5i-and-HTT-KD-250x188.jpg 250w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/20181116-HeLa-G3BP1-PRMT5i-and-HTT-KD.jpg 960w" sizes="(max-width: 841px) 100vw, 841px" /></p>
<p style="text-align: justify;">HTT protein levels were clearly decreased with PRMT5 inhibition! I ran these samples alongside the stable HTT knockdown lines and although the HTT levels did not decrease as drastically as with the knockdowns, the decrease with the PRMT5 inhibitors is still substantial. I also included SGC2096, the inactive control to GSK591, and HTT protein levels were not affected by it, showing that it is indeed through the activity of PRMT5 that HTT levels were decreased.</p>
<p style="text-align: justify;">The mechanism by which HTT is downregulated by PRMT5 inhibitors is still unclear and it will be very interesting to further investigate the link between HTT and PRMT5. As of now, the only publication I can find linking HTT to PRMT5 is the following:</p>
<p>&nbsp;</p>
<p style="text-align: justify;">PRMT5- mediated symmetric arginine dimethylation is attenuated by mutant huntingtin and is impaired in Huntington&#8217;s disease (HD).</p>
<p style="text-align: justify;">Ratovitski T, Arbez N, Stewart JC, Chighladze E, Ross CA.</p>
<p style="text-align: justify;">Cell Cycle. 2015;14(11):1716-29. doi: <a href="http://doi.org/10.1080/15384101.2015.1033595">10.1080/15384101.2015.1033595</a>.</p>
<p>&nbsp;</p>
<p style="text-align: justify;">This paper showed the colocalisation and interaction of HTT and PRMT5 and shows that this interaction results in the stimulation of PRMT5 activity by HTT in vitro. Mutant HTT has a greater affinity for PRMT5; however, it has a reduced ability to stimulate PRMT5 activity, as demonstrated by the reduction of symmetric dimethylation of arginine (SDMA), which is mainly mediated by PRMT5, of histones H2 and H4. Decreased levels of SDMA on known PRMT5 targets were detected in HD brain samples, suggesting that the reduction in PRMT5 activity may contribute to the pathogenesis of HD. The knockdown of JMJD6, which removes arginine demethylation, was able to rescue the phenotype of increased cell death seen with mutant HTT, further suggesting that the restoration of SDMA to normal levels may be a potential therapeutic strategy.</p>
<p style="text-align: justify;">However, our findings that PRMT5 inhibition itself can affect the expression of HTT at the protein level suggests that there might be a feedback loop in play between PRMT5 and HTT. It will be interesting to further investigate how the levels of SDMA and the other proteins involved are affected with PRMT5 inhibition and elucidate the mechanisms involved in this process. It should also be noted that our results only reflect the expression of wild-type HTT and not mutant (expanded) HTT, and it will be highly interesting and of great clinical importance to find out if PRMT5 inhibition is able to decrease the protein levels of mutant HTT similarly to how it affects wild-type HTT.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/reduction-of-htt-protein-through-chemical-inhibition/feed/</wfw:commentRss>
		<slash:comments>4</slash:comments>
		</item>
		<item>
		<title>Generating HTT Knockdown Models</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/generating-htt-knockdown-models/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/generating-htt-knockdown-models/#comments</comments>
		<pubDate>Mon, 17 Sep 2018 20:45:58 +0000</pubDate>
		<dc:creator><![CDATA[Jolene Ho]]></dc:creator>
				<category><![CDATA[Huntingtin]]></category>
		<category><![CDATA[Huntington’s disease]]></category>
		<category><![CDATA[Jolene Ho]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=1860</guid>
		<description><![CDATA[The experimental set-up and results covered in this post can be found here: doi.org/10.5281/zenodo.1420537 I was previously using an HTT overexpression model to look at any changes in proteins related to HTT or its function. Unfortunately, none of the related proteins I looked at showed any significant change in protein levels with HTT overexpression. There <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/generating-htt-knockdown-models/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>The experimental set-up and results covered in this post can be found here: <a href="http://doi.org/10.5281/zenodo.1420537">doi.org/10.5281/zenodo.1420537</a></p>
<p>I was previously using an HTT overexpression model to look at any changes in proteins related to HTT or its function. Unfortunately, none of the related proteins I looked at showed any significant change in protein levels with HTT overexpression. There are several reasons for this: the HTT overexpression might not be strong enough (especially for the larger constructs of D4 and D6, which are harder to express) or the level of HTT protein in the cells is already saturated, hence an overexpression does not further affect these proteins of interest. I therefore decided to use a knockdown model instead, which might provide more insight into the functions and effects of HTT.</p>
<p>A slightly different system was used to generate the HTT knockdown models, compared to the HTT overexpression model. For the overexpression model, a baculovirus system was used. In the knockdown model, a lentivirus system was used.</p>
<p>Lentiviruses can be easily produced through the transfection of HEK293T cells with several plasmids which code for different components of the lentivirus. The HEK293T cells, which are very easy to transfect, can then produce the various components of the lentivirus and assemble the lentiviruses, releasing them into the cell media. As HEK293T cells are grown adherently, these lentiviruses can be harvested by simply collecting the supernatant. When the plasmids encoding the different components of the lentivirus are transfected together with a plasmid encoding an shRNA for the gene of interest (in this case, HTT), the lentiviruses produced have the ability to disrupt the gene of interest through RNA interference (RNAi) when they transduce the target cells.</p>
<p><img class="alignnone  wp-image-1862" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/09/Lentivirus-Production-300x225.jpg" alt="" width="771" height="578" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/09/Lentivirus-Production-300x225.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/09/Lentivirus-Production-768x576.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/09/Lentivirus-Production-350x263.jpg 350w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/09/Lentivirus-Production-780x586.jpg 780w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/09/Lentivirus-Production-250x188.jpg 250w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/09/Lentivirus-Production.jpg 960w" sizes="(max-width: 771px) 100vw, 771px" /></p>
<p>Figure adapted from Figure 4 of Nucleic Acid Delivery: Lentiviral and Retroviral Vectors by Erin P O’Keefe; Princeton University, United States (MATER METHODS 2013;3:174; <a href="http://dx.doi.org/10.13070/mm.en.3.174">dx.doi.org/10.13070/mm.en.3.174</a>).</p>
<p>&nbsp;</p>
<p>Upon transduction of the target cells, the lentivirus delivers the plasmid encoding the shRNA to the nucleus. The plasmid produces the shRNA, which is processed by the target cell, leading to the production of a sequence complementary to that of the mRNA of the gene of interest, HTT. The binding of this complementary sequence to the mRNA of the gene of interest causes the formation of the RNA-induced silencing complex (RISC). The RISC, consisting of both the complementary sequence from the shRNA and the mRNA of the gene of interest, is then degraded, preventing the expression of the target protein, which is HTT in this case, leading to the knockdown of HTT. As the plasmid in the target cell continuously produces the shRNA, the mRNA is continuously degraded and the cells are stably knocked down for HTT.</p>
<p>&nbsp;</p>
<p><img class="alignnone  wp-image-1863" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/09/DdRNAi_diagram-211x300.jpg" alt="" width="325" height="462" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/09/DdRNAi_diagram-211x300.jpg 211w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/09/DdRNAi_diagram.jpg 571w" sizes="(max-width: 325px) 100vw, 325px" /></p>
<p>Figure from Figure 1 of DNA-directed RNA interference (<a href="http://www.wikiwand.com/en/DNA-directed_RNA_interference">http://www.wikiwand.com/en/DNA-directed_RNA_interference</a>).</p>
<p>&nbsp;</p>
<p>Using this method, I managed to establish 3 stable HTT knockdowns using 3 independent shRNA constructs in HeLa cells.</p>
<p><img class="alignnone  wp-image-1861" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/09/20180620-HeLa-G3BP1-HTT-KD245-Stable-300x225.jpg" alt="" width="773" height="580" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/09/20180620-HeLa-G3BP1-HTT-KD245-Stable-300x225.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/09/20180620-HeLa-G3BP1-HTT-KD245-Stable-768x576.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/09/20180620-HeLa-G3BP1-HTT-KD245-Stable-350x263.jpg 350w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/09/20180620-HeLa-G3BP1-HTT-KD245-Stable-780x586.jpg 780w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/09/20180620-HeLa-G3BP1-HTT-KD245-Stable-250x188.jpg 250w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/09/20180620-HeLa-G3BP1-HTT-KD245-Stable.jpg 960w" sizes="(max-width: 773px) 100vw, 773px" /></p>
<p>As you can see, the levels of HTT are much lower in the HTT knockdown cells compared to the no treatment and H4 (luciferase) controls. I will next use these cell lines to identify any changes in expression of HTT-related proteins.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/generating-htt-knockdown-models/feed/</wfw:commentRss>
		<slash:comments>1</slash:comments>
		</item>
		<item>
		<title>Checking Disruption of Proteins of Interest with Overexpression of HTT</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/checking-disruption-of-proteins-of-interest-with-overexpression-of-htt/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/checking-disruption-of-proteins-of-interest-with-overexpression-of-htt/#comments</comments>
		<pubDate>Mon, 30 Jul 2018 22:45:47 +0000</pubDate>
		<dc:creator><![CDATA[Jolene Ho]]></dc:creator>
				<category><![CDATA[Huntingtin]]></category>
		<category><![CDATA[Huntington’s disease]]></category>
		<category><![CDATA[Jolene Ho]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=1582</guid>
		<description><![CDATA[The experimental set-up and results covered in this post can be found here: 10.5281/zenodo.1323902 As I’ve been having problems overexpression HTT properly in my cancer cell lines, I decided to continue looking at other proteins of interest using our best overexpression model so far, which is in HEK293T cells. Although it may not be a cancer <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/checking-disruption-of-proteins-of-interest-with-overexpression-of-htt/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p style="text-align: justify;">The experimental set-up and results covered in this post can be found here: <a href="http://doi.org/10.5281/zenodo.1323902">10.5281/zenodo.1323902</a></p>
<p style="text-align: justify;">
<p style="text-align: justify;">As I’ve been having problems overexpression HTT properly in my cancer cell lines, I decided to continue looking at other proteins of interest using our best overexpression model so far, which is in HEK293T cells. Although it may not be a cancer cell line, it is very widely used due to the ease of genetic manipulation, as evident in my success with overexpressing HTT in this line, and would be useful for the elucidation of possible mechanisms of action, which can then be validated in other more relevant but difficult cells.</p>
<p style="text-align: justify;">
<p style="text-align: justify;">On my hunt for possible target proteins to look at in my HTT overexpression model, I found several interesting papers on posttranslational modifications of proteins. Posttranslational modifications occur on proteins after they have been translated from RNA to protein and are important in the normal function of proteins, including ensuring that proteins are folded properly and do not aggregate. As the pathological feature of Huntington’s disease is the formation of protein aggregates due to the expanded polyQ region, it is highly relevant to look at changes in posttranslational modifications in the context of huntingtin overexpression.</p>
<p style="text-align: justify;">
<p>&nbsp;</p>
<p style="text-align: justify;"><strong>PRMT5- mediated symmetric arginine dimethylation is attenuated by mutant huntingtin and is impaired in Huntington&#8217;s disease (HD)</strong></p>
<p style="text-align: justify;">Ratovitski T, Arbez N, Stewart JC, Chighladze E, Ross CA.</p>
<p style="text-align: justify;">Cell Cycle. 2015;14(11):1716-29.</p>
<p style="text-align: justify;"><a href="https://doi.org/10.1080/15384101.2015.1033595">https://doi.org/10.1080/15384101.2015.1033595</a></p>
<p style="text-align: justify;">
<p>&nbsp;</p>
<p style="text-align: justify;">This paper showed that arginine dimethylation was decreased by mutant huntingtin in Huntington’s disease. Interestingly, normal HTT, but not mutant HTT, stimulated arginine methylation mediated by PRMT5 activity. The main epigenetic mark mediated by PRMT5 is symmetric dimethylation of arginine (RMe2s). I thus decided to look at the levels of RMe2s in cells with HTT OE to see if there were any changes in PRMT5 activity. As a control, I also looked at asymmetric demethylation of arginine (RMe2a), which should not be affected by changes in PRMT5 activity.</p>
<p style="text-align: justify;">
<p style="text-align: justify;">I also found several other pathways that have been reported to be disrupted by HTT. In particular, HTT is known to be important for nuclear integrity and nuclear export, and several papers looked at how the nuclear lamina is regulated by Huntingtin. I decided to examine the nuclear integrity of HTT OE cells by looking at Lamin B1, which is a protein involved in the nuclear lamina and which was also used in some of these papers to study nuclear integrity.</p>
<p style="text-align: justify;">
<p>&nbsp;</p>
<p style="text-align: justify;">
<p style="text-align: justify;"><strong>Polyglutamine-Expanded Huntingtin Exacerbates Age-Related Disruption of Nuclear Integrity and Nucleocytoplasmic Transport</strong></p>
<p style="text-align: justify;">Gasset-Rosa F, Chillon-Marinas C, Goginashvili A, Atwal RS, Artates JW, Tabet R, Wheeler VC, Bang AG, Cleveland DW, Lagier-Tourenne C.</p>
<p style="text-align: justify;">Neuron. 2017 Apr 5;94(1):48-57.e4.</p>
<p style="text-align: justify;"><a href="https://doi.org/10.1016/j.neuron.2017.03.027">https://doi.org/10.1016/j.neuron.2017.03.027</a></p>
<p style="text-align: justify;">
<p>&nbsp;</p>
<p style="text-align: justify;"><strong>Focal distortion of the nuclear envelope by huntingtin aggregates revealed by lamin immunostaining</strong></p>
<p style="text-align: justify;">Chapple JP, Bros-Facer V, Butler R, Gallo JM.</p>
<p style="text-align: justify;">Neurosci Lett. 2008 Dec 12;447(2-3):172-4. Epub 2008 Oct 1.</p>
<p style="text-align: justify;"><a href="https://doi.org/10.1016/j.neulet.2008.09.075">https://doi.org/10.1016/j.neulet.2008.09.075</a></p>
<p style="text-align: justify;">
<p>&nbsp;</p>
<p style="text-align: justify;"><strong>B10 Nuclear Lamina Is Differentially Altered In Huntington’s Disease Brain Regions</strong></p>
<p style="text-align: justify;">J Neurol Neurosurg Psychiatry 2014;85:A12.</p>
<p style="text-align: justify;">Alcalá R, Creus-Muncunill J, Azkona G, Alberch J, Pérez-Navarro E.</p>
<p style="text-align: justify;"><a href="http://doi.org/10.1136/jnnp-2014-309032.38">http://doi.org/10.1136/jnnp-2014-309032.38</a></p>
<p style="text-align: justify;">
<p>&nbsp;</p>
<p style="text-align: justify;">Huntingtin was also previously reported to be involved in perinuclear localisation and centrosomal organisation. PCNT is a protein that localises to the centrosome and is involved in the formation of the pericentriolar matrix (PCM), which is key for proper centrosome formation and cell cycle progression. Its role in both perinuclear localisation and centrosomal organisation makes it an important candidate to look at in the context of Huntington’s disease. Moreover, it is also involved in ciliogenesis, which has been reported to be disrupted in patients with Huntington’s disease.</p>
<p style="text-align: justify;">
<p>&nbsp;</p>
<p style="text-align: justify;"><strong>Perinuclear localization of huntingtin as a consequence of its binding to microtubules through an interaction with beta-tubulin: relevance to Huntington&#8217;s disease</strong></p>
<p style="text-align: justify;">Hoffner G, Kahlem P, Djian P.</p>
<p style="text-align: justify;">J Cell Sci. 2002 Mar 1;115(Pt 5):941-8.</p>
<p style="text-align: justify;"><a href="http://jcs.biologists.org/content/115/5/941.long">http://jcs.biologists.org/content/115/5/941.long</a></p>
<p style="text-align: justify;">
<p>&nbsp;</p>
<p style="text-align: justify;"><strong>Centrosome disorganization in fibroblast cultures derived from R6/2 Huntington&#8217;s disease (HD) transgenic mice and HD patients</strong></p>
<p style="text-align: justify;">Sathasivam K, Woodman B, Mahal A, Bertaux F, Wanker EE, Shima DT, Bates GP.</p>
<p style="text-align: justify;">Hum Mol Genet. 2001 Oct 1;10(21):2425-35.</p>
<p style="text-align: justify;"><a href="http://doi.org/10.1093/hmg/10.21.2425">http://doi.org/10.1093/hmg/10.21.2425</a></p>
<p style="text-align: justify;">
<p>&nbsp;</p>
<p style="text-align: justify;"><strong>Ciliogenesis is regulated by a huntingtin-HAP1-PCM1 pathway and is altered in Huntington disease</strong></p>
<p style="text-align: justify;">Keryer G, Pineda JR, Liot G, Kim J, Dietrich P, Benstaali C, Smith K, Cordelières FP, Spassky N, Ferrante RJ, Dragatsis I, Saudou F.</p>
<p style="text-align: justify;">J Clin Invest. 2011 Nov;121(11):4372-82. Epub 2011 Oct 10.</p>
<p style="text-align: justify;"><a href="http://doi.org/10.1172/JCI57552">http://doi.org/10.1172/JCI57552</a></p>
<p style="text-align: justify;">
<p>&nbsp;</p>
<p style="text-align: justify;">Finally, I also looked at coilin, a major component of cajal bodies. There have been reports that cajal bodies may play a role in the pathogenesis of CAG repeat diseases such as Huntington’s disease. Furthermore, it is a known substrate of PRMT5 and may be affected indirectly by Huntingtin through methylation by PRMT5.</p>
<p style="text-align: justify;">
<p>&nbsp;</p>
<p style="text-align: justify;"><strong>Interaction between neuronal intranuclear inclusions and promyelocytic leukemia protein nuclear and coiled bodies in CAG repeat diseases</strong></p>
<p style="text-align: justify;">Yamada M, Sato T, Shimohata T, Hayashi S, Igarashi S, Tsuji S, Takahashi H.</p>
<p style="text-align: justify;">Am J Pathol. 2001 Nov;159(5):1785-95.</p>
<p style="text-align: justify;"><a href="http://doi.org/10.1016/S0002-9440(10)63025-8">http://doi.org/10.1016/S0002-9440(10)63025-8</a></p>
<p style="text-align: justify;">
<p>&nbsp;</p>
<p style="text-align: justify;">My final list of proteins to look thus consisted of: RMe2s, RMe2a, Lamin B1, PCNT and coilin, with GAPDH as the loading control.</p>
<p style="text-align: justify;"><img class="alignnone  wp-image-1583" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/07/20180524-HeLa-G3BP1-HEK293T-HTT-OE-300x225.jpg" alt="" width="827" height="620" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/07/20180524-HeLa-G3BP1-HEK293T-HTT-OE-300x225.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/07/20180524-HeLa-G3BP1-HEK293T-HTT-OE-768x576.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/07/20180524-HeLa-G3BP1-HEK293T-HTT-OE-350x263.jpg 350w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/07/20180524-HeLa-G3BP1-HEK293T-HTT-OE-780x586.jpg 780w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/07/20180524-HeLa-G3BP1-HEK293T-HTT-OE-250x188.jpg 250w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/07/20180524-HeLa-G3BP1-HEK293T-HTT-OE.jpg 960w" sizes="(max-width: 827px) 100vw, 827px" /></p>
<p style="text-align: justify;">Unfortunately, none of these proteins showed any significant change in protein levels! It is possible that the HTT overexpression was not good enough, especially for the larger constructs of D4 and D6. However, even comparing between the HTT WT overexpression (D1: Q23) and the PBS control, there is no obvious difference in protein levels. Another possibility is that HTT levels is not limiting in the pathways looked at and hence an overexpression does not affect these proteins of interest. It might thus be more prudent to look at knockdowns of HTT instead.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/checking-disruption-of-proteins-of-interest-with-overexpression-of-htt/feed/</wfw:commentRss>
		<slash:comments>2</slash:comments>
		</item>
		<item>
		<title>Overexpression of Expanded HTT in Brain Cancer Cell Lines</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/overexpression-of-expanded-htt-in-brain-cancer-cell-lines/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/overexpression-of-expanded-htt-in-brain-cancer-cell-lines/#respond</comments>
		<pubDate>Tue, 19 Jun 2018 17:05:54 +0000</pubDate>
		<dc:creator><![CDATA[Jolene Ho]]></dc:creator>
				<category><![CDATA[Huntingtin]]></category>
		<category><![CDATA[Huntington’s disease]]></category>
		<category><![CDATA[Jolene Ho]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=1366</guid>
		<description><![CDATA[The experimental set-up and results covered in this post can be found here: 10.5281/zenodo.1292884. A few posts back, I tried expressing HTT with different lengths of Q repeats in mammalian cells using baculovirus (read more here). However, expression was not very good, especially for HTT with longer Q repeats (have a look at these results here). <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/overexpression-of-expanded-htt-in-brain-cancer-cell-lines/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p style="text-align: justify;">The experimental set-up and results covered in this post can be found here: <a href="https://doi.org/10.5281/zenodo.1292884">10.5281/zenodo.1292884</a>.</p>
<p style="text-align: justify;">A few posts back, I tried expressing HTT with different lengths of Q repeats in mammalian cells using baculovirus (read more <a href="https://thesgc.github.io/static-openlabnotebooks/overexpressing-htt-in-mammalian-cells/">here</a>). However, expression was not very good, especially for HTT with longer Q repeats (have a look at these results <a href="https://thesgc.github.io/static-openlabnotebooks/overexpression-of-expanded-htt/">here</a>). Anyway, forging ahead, I decided to try the baculovirus overexpression in brain cells, which would be more relevant to HD.</p>
<p style="text-align: justify;">As our lab generally works on cancer, we have the brain cancer lines LN18 and T98G. These lines were both established from patients with glioblastoma. Although, being brain cancer lines, these may not be ideal for the investigation of HD, especially with previous reports indicating that people with HD might have a lower incidence of cancer. Here is a study showing that the lower incidence of cancer in HD patients might be due to the mutant HTT protein causing more apoptosis in pre-cancer cells:</p>
<p>&nbsp;</p>
<p style="text-align: justify;">Sørensen SA, Fenger K, Olsen JH.</p>
<p style="text-align: justify;">Significantly lower incidence of cancer among patients with Huntington disease: An apoptotic effect of an expanded polyglutamine tract?</p>
<p style="text-align: justify;">Cancer. 1999 Oct 1;86(7):1342-6.</p>
<p style="text-align: justify;"><a href="https://doi.org/10.1002/(SICI)1097-0142(19991001)86:7%3c1342::AID-CNCR33%3e3.0.CO;2-3">https://doi.org/10.1002/(SICI)1097-0142(19991001)86:7&lt;1342::AID-CNCR33&gt;3.0.CO;2-3</a></p>
<p>&nbsp;</p>
<p style="text-align: justify;">More recently, another paper has shown that the siRNAs based on the Q repeats in HTT could kill cancer cells!</p>
<p>&nbsp;</p>
<p style="text-align: justify;">Murmann AE, Gao QQ, Putzbach WE, Patel M, Bartom ET, Law CY, Bridgeman B, Chen S, McMahon KM, Thaxton CS, Peter ME.</p>
<p style="text-align: justify;">Small interfering RNAs based on huntingtin trinucleotide repeats are highly toxic to cancer cells.</p>
<p style="text-align: justify;">EMBO Rep. 2018 Mar;19(3). pii: e45336. doi: 10.15252/embr.201745336. Epub 2018 Feb 12.</p>
<p style="text-align: justify;"><a href="http://doi.org/10.15252/embr.201745336">http://doi.org/10.15252/embr.201745336</a></p>
<p>&nbsp;</p>
<p style="text-align: justify;">There is clearly a link between cancer and the etiology of HD, mutant HTT or even wild-type HTT that would be interesting to look at. So, I tried overexpressing the HTT constructs in the brain cancer cell lines LN18 and T98G. Here are my Western blot results showing levels of HTT and FLAG.</p>
<p style="text-align: justify;">  <img class="alignnone wp-image-1376" style="font-size: 1.5rem;" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/06/LN18-HTT-300x225.jpg" alt="" width="767" height="575" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/06/LN18-HTT-300x225.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/06/LN18-HTT-768x576.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/06/LN18-HTT-350x263.jpg 350w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/06/LN18-HTT-780x586.jpg 780w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/06/LN18-HTT-250x188.jpg 250w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/06/LN18-HTT.jpg 960w" sizes="(max-width: 767px) 100vw, 767px" /></p>
<p style="text-align: justify;"><img class="alignnone wp-image-1375" style="font-size: 1.5rem;" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/06/LN18-FLAG-300x225.jpg" alt="" width="765" height="573" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/06/LN18-FLAG-300x225.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/06/LN18-FLAG-768x576.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/06/LN18-FLAG-350x263.jpg 350w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/06/LN18-FLAG-250x188.jpg 250w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/06/LN18-FLAG.jpg 960w" sizes="(max-width: 765px) 100vw, 765px" /></p>
<p style="text-align: justify;"><img class="alignnone wp-image-1374" style="font-size: 1.5rem;" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/06/T98G-HTT-300x225.jpg" alt="" width="752" height="564" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/06/T98G-HTT-300x225.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/06/T98G-HTT-768x576.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/06/T98G-HTT-350x263.jpg 350w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/06/T98G-HTT-780x586.jpg 780w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/06/T98G-HTT-250x188.jpg 250w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/06/T98G-HTT.jpg 960w" sizes="(max-width: 752px) 100vw, 752px" /></p>
<p style="text-align: justify;"><img class="alignnone wp-image-1373" style="font-size: 1.5rem;" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/06/T98G-FLAG-300x225.jpg" alt="" width="740" height="555" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/06/T98G-FLAG-300x225.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/06/T98G-FLAG-768x576.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/06/T98G-FLAG-350x263.jpg 350w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/06/T98G-FLAG-780x586.jpg 780w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/06/T98G-FLAG-250x188.jpg 250w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/06/T98G-FLAG.jpg 960w" sizes="(max-width: 740px) 100vw, 740px" /></p>
<p style="text-align: justify;">Alas, the overexpression was even worse than what I got previously! This is not surprising though, as the cells used previously, HEK293T, are known for being easy to grow and manipulate. Although there is a bit of increase in the levels of HTT, FLAG levels were barely detectable, and there was no increase in HTT levels for the D4 and D6 constructs. We might need to go back to troubleshooting ways to improve the expression of these constructs with baculovirus constructs, especially in more relevant cell lines.</p>
<p style="text-align: justify;">
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/overexpression-of-expanded-htt-in-brain-cancer-cell-lines/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Visualisation of Purified HTT Proteins (2)</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/visualisation-of-purified-htt-proteins-2/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/visualisation-of-purified-htt-proteins-2/#respond</comments>
		<pubDate>Fri, 25 May 2018 20:36:00 +0000</pubDate>
		<dc:creator><![CDATA[Jolene Ho]]></dc:creator>
				<category><![CDATA[Huntingtin]]></category>
		<category><![CDATA[Huntington’s disease]]></category>
		<category><![CDATA[Jolene Ho]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=1268</guid>
		<description><![CDATA[The experimental set-up and results covered in this post can be found here: 10.5281/zenodo.1253365. Recently, I helped run some Western blots for Rachel Harding, a fellow postdoc and open notebooker at SGC (read more about her work here: http://labscribbles.com/), for a publication on a toolkit to enable people to more easily purify and study HTT (you <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/visualisation-of-purified-htt-proteins-2/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>The experimental set-up and results covered in this post can be found here: <a href="http://doi.org/10.5281/zenodo.1253365">10.5281/zenodo.1253365</a>.</p>
<p>Recently, I helped run some Western blots for Rachel Harding, a fellow postdoc and open notebooker at SGC (read more about her work here: <a href="http://labscribbles.com/">http://labscribbles.com/</a>), for a publication on a toolkit to enable people to more easily purify and study HTT (you can read a draft of the manuscript here: <a href="https://zenodo.org/record/1194724#.Ws9pli7wapp">https://zenodo.org/record/1194724#.Ws9pli7wapp</a>). The samples I ran were purified protein of different HTT constructs, D1 to D6, and have different numbers of Q repeats as follows:</p>
<p>D1: 23 Q repeats</p>
<p>D2: 54 Q repeats</p>
<p>D4: 79 Q repeats</p>
<p>D6: 145 Q repeats</p>
<p>As these are purified HTT proteins, we cannot use the BCA assay or a housekeeping protein to normalise the samples, so I manually adjust the volumes of protein loaded for the Western blot. Unfortunately, after normalisation, although clear bands can be seen in all samples, the D4 and D6 transduction samples were still fainter compared to the other samples. Rachel thus provided me with fresh samples with higher concentrations of protein to run the gels with.</p>
<p>For the first batch of samples, I started off running equal volumes of each construct just to see how the relative concentrations were like across the samples, but I’ve since realised that generally, the larger constructs express much more poorly. I thus loaded a smaller volume of protein for the smaller constructs AND used the concentrated eluates (EC) of the larger constructs from the get-go.</p>
<p><img class="alignnone  wp-image-1270" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/05/20180413-Rachels-HTT-samples-2-300x225.jpg" alt="" width="793" height="594" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/05/20180413-Rachels-HTT-samples-2-300x225.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/05/20180413-Rachels-HTT-samples-2-768x576.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/05/20180413-Rachels-HTT-samples-2-350x263.jpg 350w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/05/20180413-Rachels-HTT-samples-2-250x188.jpg 250w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/05/20180413-Rachels-HTT-samples-2.jpg 960w" sizes="(max-width: 793px) 100vw, 793px" /></p>
<p>Not as bad as my first run with the first batch of samples, but still lots of normalisation to do. After several rounds of quantification of the bands, dilutions and normalisation of the loading volumes, this was the final run.</p>
<p><img class="alignnone  wp-image-1269" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/05/20180426-Rachels-HTT-samples-2-300x225.jpg" alt="" width="701" height="526" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/05/20180426-Rachels-HTT-samples-2-300x225.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/05/20180426-Rachels-HTT-samples-2-768x576.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/05/20180426-Rachels-HTT-samples-2-350x263.jpg 350w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/05/20180426-Rachels-HTT-samples-2-780x586.jpg 780w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/05/20180426-Rachels-HTT-samples-2-250x188.jpg 250w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/05/20180426-Rachels-HTT-samples-2.jpg 960w" sizes="(max-width: 701px) 100vw, 701px" /></p>
<p>The signal for this second batch of samples was much better than that of the first batch. The D1, F1 and F6 samples still had to be diluted down by quite a bit, even when using the unconcentrated eluate, but clear even bands can be seen in all samples.</p>
<p>&nbsp;</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/visualisation-of-purified-htt-proteins-2/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Visualisation of Purified HTT Proteins</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/visualisation-of-purified-htt-proteins/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/visualisation-of-purified-htt-proteins/#respond</comments>
		<pubDate>Mon, 30 Apr 2018 14:44:37 +0000</pubDate>
		<dc:creator><![CDATA[Jolene Ho]]></dc:creator>
				<category><![CDATA[Huntingtin]]></category>
		<category><![CDATA[Huntington’s disease]]></category>
		<category><![CDATA[Jolene Ho]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=1123</guid>
		<description><![CDATA[The experimental set-up and results covered in this post can be found here: 10.5281/zenodo.1233602. &#160; Lately, I’ve been helping Rachel Harding, a fellow postdoc at SGC and one of the pioneers of the Extreme Open Science Initiative at SGC, with some HTT experiments. She is also working on HTT, and has in fact been working on <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/visualisation-of-purified-htt-proteins/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>The experimental set-up and results covered in this post can be found here: <a href="http://doi.org/10.5281/zenodo.1233602">10.5281/zenodo.1233602</a>.</p>
<p>&nbsp;</p>
<p>Lately, I’ve been helping Rachel Harding, a fellow postdoc at SGC and one of the pioneers of the Extreme Open Science Initiative at SGC, with some HTT experiments. She is also working on HTT, and has in fact been working on HTT way longer than I have. Read more about her work here: <a href="http://labscribbles.com/">http://labscribbles.com/</a>.</p>
<p>&nbsp;</p>
<p>Rachel has been working on a publication on a toolkit to enable people to more easily purify and study HTT and she has very kindly included me to help show the expression of the HTT constructs by Western blot. You can read a draft of the manuscript here: <a href="https://zenodo.org/record/1194724#.Ws9pli7wapp">https://zenodo.org/record/1194724#.Ws9pli7wapp</a>.</p>
<p>&nbsp;</p>
<p>The work here is slightly different from what I have been doing previously. I usually express these constructs in mammalian cells and harvest whole cell lysates, after which I quantify the total amount of protein using a bicinchoninic acid assay (BCA assay). I then normalise the protein concentrations of all my samples and carry out my Western blot, probing for my protein of interest (eg, HTT) and a housekeeping protein such as GAPDH, which should be equally abundant across all samples after normalisation and ensures that any differences in abundance of my protein of interest are due to the treatment and not due to improper loading.</p>
<p>&nbsp;</p>
<p>For this experiment, Rachel provided me with purified HTT protein. As such, we cannot use the BCA assay or a housekeeping protein to normalise the samples, so we’ll need to manually adjust the volumes of protein loaded after each Western blot. For the first run, as I did not know what to expect, I ran equal volumes of each construct just to see how the relative concentrations were like across the samples.</p>
<p><img class="alignnone  wp-image-1153" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/04/20180308-Rachels-HTT-samples-E-HTT-300x225.jpg" alt="" width="802" height="601" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/04/20180308-Rachels-HTT-samples-E-HTT-300x225.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/04/20180308-Rachels-HTT-samples-E-HTT-768x576.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/04/20180308-Rachels-HTT-samples-E-HTT-350x263.jpg 350w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/04/20180308-Rachels-HTT-samples-E-HTT-780x586.jpg 780w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/04/20180308-Rachels-HTT-samples-E-HTT-250x188.jpg 250w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/04/20180308-Rachels-HTT-samples-E-HTT.jpg 960w" sizes="(max-width: 802px) 100vw, 802px" /></p>
<p>As you can see, the protein concentrations were very variable across the samples. As with my previous experiments, the constructs D1 to D6 are as follows:</p>
<p>D1: 23 Q repeats</p>
<p>D2: 54 Q repeats</p>
<p>D4: 79 Q repeats</p>
<p>D6: 145 Q repeats</p>
<p>In agreement with my previous experiment in mammalian cells, the D1 construct could be detected for both the transduction (TD) and transfection (TF) samples, but the other samples were much more diluted compared the D1 samples. Here’s an overexposed image, we can see some protein in the other lanes but the background is too high to quantify the band intensities accurately.</p>
<p><img class="alignnone  wp-image-1150" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/04/20180308-Rachels-HTT-samples-E-HTT-bright-300x225.jpg" alt="" width="757" height="568" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/04/20180308-Rachels-HTT-samples-E-HTT-bright-300x225.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/04/20180308-Rachels-HTT-samples-E-HTT-bright-768x576.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/04/20180308-Rachels-HTT-samples-E-HTT-bright-350x263.jpg 350w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/04/20180308-Rachels-HTT-samples-E-HTT-bright-780x586.jpg 780w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/04/20180308-Rachels-HTT-samples-E-HTT-bright-250x188.jpg 250w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/04/20180308-Rachels-HTT-samples-E-HTT-bright.jpg 960w" sizes="(max-width: 757px) 100vw, 757px" /></p>
<p>Luckily, Rachel provided me with some concentrated samples as well, so I used these for the D2, D4 and D6 constructs AND diluted down the unconcentrated D1 constructs. Here’s my second go at it.</p>
<p><img class="alignnone  wp-image-1154" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/04/20180313-Rachels-HTT-samples-EC-300x225.jpg" alt="" width="805" height="605" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/04/20180313-Rachels-HTT-samples-EC-300x225.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/04/20180313-Rachels-HTT-samples-EC-768x576.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/04/20180313-Rachels-HTT-samples-EC-350x263.jpg 350w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/04/20180313-Rachels-HTT-samples-EC-780x586.jpg 780w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/04/20180313-Rachels-HTT-samples-EC-250x188.jpg 250w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/04/20180313-Rachels-HTT-samples-EC.jpg 960w" sizes="(max-width: 805px) 100vw, 805px" /></p>
<p>Better but not quite there yet, so I did some quantification of the bands and normalisation of the loading volumes. After several more runs of quantification and normalisation, this was my final run!</p>
<p><img class="alignnone  wp-image-1155" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/04/20180329-Rachels-HTT-samples-EC-300x225.jpg" alt="" width="797" height="598" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/04/20180329-Rachels-HTT-samples-EC-300x225.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/04/20180329-Rachels-HTT-samples-EC-768x576.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/04/20180329-Rachels-HTT-samples-EC-350x263.jpg 350w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/04/20180329-Rachels-HTT-samples-EC-780x586.jpg 780w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/04/20180329-Rachels-HTT-samples-EC-250x188.jpg 250w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/04/20180329-Rachels-HTT-samples-EC.jpg 960w" sizes="(max-width: 797px) 100vw, 797px" /></p>
<p>Unfortunately, this was the best I could get. The D4 and D6 transduction samples were still fainter compared to the other samples, but clear bands can be seen in all samples. I could reduce the amount of protein I load for the other samples to levels similar to that of the D4 and D6 samples but all the bands would be fainter. We’ll see what Rachel makes of this and decide how to proceed.</p>
<p>&nbsp;</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/visualisation-of-purified-htt-proteins/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Overexpression of Expanded HTT</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/overexpression-of-expanded-htt/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/overexpression-of-expanded-htt/#comments</comments>
		<pubDate>Sat, 14 Apr 2018 19:14:14 +0000</pubDate>
		<dc:creator><![CDATA[Jolene Ho]]></dc:creator>
				<category><![CDATA[Huntingtin]]></category>
		<category><![CDATA[Huntington’s disease]]></category>
		<category><![CDATA[Jolene Ho]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=1037</guid>
		<description><![CDATA[The experimental set-up and results covered in this post can be found here: 10.5281/zenodo.1218375 The etiology of HD is the expansion of Q repeats or CAG repeats in the HTT gene, which produces the huntingtin protein. This causes improper folding of the protein and leads to protein aggregation in neuronal cells, which ultimately causes the <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/overexpression-of-expanded-htt/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>The experimental set-up and results covered in this post can be found here: <a href="http://doi.org/10.5281/zenodo.1218375">10.5281/zenodo.1218375</a></p>
<p>The etiology of HD is the expansion of Q repeats or CAG repeats in the HTT gene, which produces the huntingtin protein. This causes improper folding of the protein and leads to protein aggregation in neuronal cells, which ultimately causes the movement, cognitive and psychiatric impairment we see in people with HD. You can read more about HTT and HD in my earlier blog post here (<a href="https://thesgc.github.io/static-openlabnotebooks/background-on-htt-and-hd/">https://thesgc.github.io/static-openlabnotebooks/background-on-htt-and-hd/</a>).</p>
<p>Although we know that HD is caused by mutant HTT, no one has yet developed a cure for it. As I am investigating the effects the number of Q repeats in HTT has on the biological function in cells, I will attempt to recapitulate the disease in a cell model by introducing exogenous HTT with different numbers of Q repeats into mammalian cells and studying the phenotype of the cells.</p>
<p>In my previous post, I managed to use baculovirus to overexpress wild-type HTT tagged with FLAG in mammalian cells (read more about my optimisation of baculovirus transduction here: <a href="https://thesgc.github.io/static-openlabnotebooks/overexpressing-htt-in-mammalian-cells/">https://thesgc.github.io/static-openlabnotebooks/overexpressing-htt-in-mammalian-cells/</a>). Using the same pipeline and protocol, I tried expressing HTT with different lengths of Q repeats in mammalian cells.</p>
<p>D1: 23 Q repeats</p>
<p>D2: 54 Q repeats</p>
<p>D4: 79 Q repeats</p>
<p>D6: 145 Q repeats</p>
<p>Generally, a normal person would have up to about 35 Q repeats in HTT while people with HD have more 45 Q repeats, with the severity of the disease increasing with the number of Q repeats. Here is a set of graphs I used in my earlier blog post on the background of HTT and HD which you can use to correlate severity of the disease (measured by the age of onset and age at death) with the number of Q repeats in the constructs I am using for my experiment.</p>
<p><img class="alignnone  wp-image-534" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/Fig-1-Am-J-Hum-Genet-2016-300x99.jpg" alt="" width="808" height="267" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/Fig-1-Am-J-Hum-Genet-2016-300x99.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/Fig-1-Am-J-Hum-Genet-2016-768x253.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/Fig-1-Am-J-Hum-Genet-2016-1024x337.jpg 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/Fig-1-Am-J-Hum-Genet-2016.jpg 1308w" sizes="(max-width: 808px) 100vw, 808px" /></p>
<p><strong>Figure 1, Keum <em>et al.</em>, Am J Hum Genet 2016</strong></p>
<p>As with all the other baculoviruses used in my experiments, these HTT baculoviruses were developed here at SGC Toronto by Rachel Harding, Peter Loppnau and Alma Seitova, who designed, cloned and produced the HTT baculoviruses.</p>
<p><img class="alignnone  wp-image-1038" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/04/FLAG-300x225.jpg" alt="" width="776" height="582" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/04/FLAG-300x225.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/04/FLAG-768x576.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/04/FLAG-350x263.jpg 350w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/04/FLAG-780x586.jpg 780w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/04/FLAG-250x188.jpg 250w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/04/FLAG.jpg 960w" sizes="(max-width: 776px) 100vw, 776px" /></p>
<p>As you can see, I included the HTT C-terminal FLAG-tag and N-terminal FLAG-tag baculoviruses used previously as controls and I got the same results as I did previously, only the N-terminal FLAG-tag was detected. I also included the GFP control.</p>
<p>The D1 construct was expressed quite well, but D2 was not as good, and there was almost no expression of D4 and D6. This is not so surprising as the constructs are much bigger with increasing number of Q repeats, making it harder for both the delivery of the construct into the cells and the expression of the construct once it gets into the cell. Here’s another image showing the levels of HTT in the cells, with GAPDH included as the loading control.</p>
<p><img class="alignnone  wp-image-1039" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/04/HTT-and-GAPDH-300x225.jpg" alt="" width="832" height="624" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/04/HTT-and-GAPDH-300x225.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/04/HTT-and-GAPDH-768x576.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/04/HTT-and-GAPDH-350x263.jpg 350w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/04/HTT-and-GAPDH-780x586.jpg 780w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/04/HTT-and-GAPDH-250x188.jpg 250w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/04/HTT-and-GAPDH.jpg 960w" sizes="(max-width: 832px) 100vw, 832px" /></p>
<p>As with the previous image showing FLAG expression, HTT expression in D4 and D6 are comparable to that of the GFP and PBS controls. More optimisation will be needed to further improve the expression of these larger constructs.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/overexpression-of-expanded-htt/feed/</wfw:commentRss>
		<slash:comments>5</slash:comments>
		</item>
		<item>
		<title>N-terminal vs C-terminal FLAG-tagged HTT</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/n-terminal-vs-c-terminal-flag-tagged-htt/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/n-terminal-vs-c-terminal-flag-tagged-htt/#respond</comments>
		<pubDate>Sun, 18 Mar 2018 06:57:16 +0000</pubDate>
		<dc:creator><![CDATA[Jolene Ho]]></dc:creator>
				<category><![CDATA[Huntingtin]]></category>
		<category><![CDATA[Huntington’s disease]]></category>
		<category><![CDATA[Jolene Ho]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=928</guid>
		<description><![CDATA[The experimental set-up and results covered in this post can be found here: 10.5281/zenodo.1202292 In my previous post, I used baculovirus to overexpress HTT in mammalian cells. Although I managed to overexpress HTT, this HTT is actually FLAG-tagged, and very strangely, I could not detect FLAG! Let me first explain what a FLAG-tag is. FLAG is <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/n-terminal-vs-c-terminal-flag-tagged-htt/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p style="text-align: justify;">The experimental set-up and results covered in this post can be found here: <a href="http://doi.org/10.5281/zenodo.1202292">10.5281/zenodo.1202292</a></p>
<p style="text-align: justify;">In my previous post, I used baculovirus to overexpress HTT in mammalian cells. Although I managed to overexpress HTT, this HTT is actually FLAG-tagged, and very strangely, I could not detect FLAG!</p>
<p style="text-align: justify;">Let me first explain what a FLAG-tag is. FLAG is a short protein that is often attached to the end of an artificially expressed protein to ‘tag’ it and differentiate it from endogenous protein. It only consists of 8 amino acids, with sequence DYKDDDDK, where D is aspartic acid, Y is tyrosine and K is lysine. As it is so short, it is unlikely that it will affect the normal function of the protein that it is tagged to. It is used for a variety of purposes such as for cellular localisation, protein purification, immunoprecipitation etc. It is especially useful when you do not have a good antibody against the protein-of-interest. Attaching a FLAG-tag allows you to use a FLAG antibody instead, which generally works quite well.</p>
<p style="text-align: justify;">The HTT protein that I used has a FLAG-tag on the C-terminal end. Although FLAG usually does not interfere with the normal functions of the protein, it can sometimes be subject to post-translational modifications such as cleavage of the tag from the protein, or it might not be detectable due to the way the protein it is tagged to is folded.</p>
<p style="text-align: justify;">I thus decided to repeat the experiment of overexpressing HTT in mammalian cells using two different baculoviruses, one expressing the C-terminal FLAG-tagged HTT and the other expressing the FLAG-tag at the N-terminal. Both HTT baculoviruses were developed here at SGC Toronto by Rachel Harding, Peter Loppnau and Alma Seitova, who designed, cloned and produced the HTT baculoviruses.</p>
<p style="text-align: justify;"><img class="alignnone wp-image-929" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/03/G523-300x225.jpg" alt="" width="813" height="610" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/03/G523-300x225.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/03/G523-768x576.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/03/G523-350x263.jpg 350w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/03/G523-780x586.jpg 780w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/03/G523-250x188.jpg 250w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/03/G523.jpg 960w" sizes="(max-width: 813px) 100vw, 813px" /></p>
<p style="text-align: justify;"><img class="alignnone wp-image-930" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/03/HEK283T-300x225.jpg" alt="" width="791" height="593" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/03/HEK283T-300x225.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/03/HEK283T-768x576.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/03/HEK283T-350x263.jpg 350w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/03/HEK283T-780x586.jpg 780w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/03/HEK283T-250x188.jpg 250w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/03/HEK283T.jpg 960w" sizes="(max-width: 791px) 100vw, 791px" /></p>
<p style="text-align: justify;">I was only able to detect the N-terminal FLAG-tag and not the C-terminal FLAG-tag in the same cell line as used previously. To be sure, I also repeated this experiment in a second cell line and obtained the same results. A FLAG-tagged green fluorescence protein (GFP) control was also used in my second experiment. Having a GFP control is useful as cells which are successfully able to express GFP can be detected under a fluorescence microscope, allowing me to determine if the protein has been successfully expressed in the cells without needing to do the Western. It would be great if we could express a GFP-tagged HTT using baculovirus, but this is not a trivial feat as HTT is such a large protein.</p>
<p style="text-align: justify;">It seems like an N-terminal FLAG is better in the case of HTT for protein overexpression. However, this is not always the case for every protein. In fact, an N-terminal FLAG has a higher chance of interfering with the normal localisation of a protein as this is usually where targeting signals that direct the protein to the correct compartments of the cell are found. We will need to keep this in mind as we move forward with further experiments using the N-terminal FLAG-tagged HTT.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/n-terminal-vs-c-terminal-flag-tagged-htt/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Overexpressing HTT in mammalian cells</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/overexpressing-htt-in-mammalian-cells/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/overexpressing-htt-in-mammalian-cells/#comments</comments>
		<pubDate>Tue, 06 Feb 2018 22:21:53 +0000</pubDate>
		<dc:creator><![CDATA[Jolene Ho]]></dc:creator>
				<category><![CDATA[Huntingtin]]></category>
		<category><![CDATA[Huntington’s disease]]></category>
		<category><![CDATA[Jolene Ho]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=666</guid>
		<description><![CDATA[The experimental set-up and results covered in this post can be found here: 10.5281/zenodo.1172644 &#160; One of the main aims of my project is to understand the role of normal and mutant HTT in mammalian cells. To do so, I will be overexpressing HTT in mammalian cells using baculovirus. The use of baculovirus for protein expression <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/overexpressing-htt-in-mammalian-cells/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>The experimental set-up and results covered in this post can be found here: <a href="http://doi.org/10.5281/zenodo.1172644">10.5281/zenodo.1172644</a></p>
<p>&nbsp;</p>
<p style="text-align: justify;">One of the main aims of my project is to understand the role of normal and mutant HTT in mammalian cells. To do so, I will be overexpressing HTT in mammalian cells using baculovirus.</p>
<p style="text-align: justify;">The use of baculovirus for protein expression in mammalian cells is not new, but it has become more widely used over the years. This technique is termed BacMam. You can read up more on this technology in this review which gives a good overview of the advances and improvements in the use of baculoviruses:</p>
<p style="text-align: justify;"><strong> </strong></p>
<p style="text-align: justify;"><strong>Baculovirus as versatile vectors for protein expression in insect and mammalian cells.</strong></p>
<p style="text-align: justify;">Kost TA, Condreay JP, Jarvis DL.</p>
<p style="text-align: justify;">Nat Biotechnol. 2005 May;23(5):567-75. DOI: <a href="http://doi.org/10.1038/nbt1095">10.1038/nbt1095</a></p>
<p>&nbsp;</p>
<p style="text-align: justify;">Briefly, the DNA of the baculovirus is modified to include the gene of interest. In our case, this will be <em>HTT</em>. Baculovirus containing the modified DNA is typically produced using Sf9 insect cells, which packages the modified DNA into fully formed baculoviruses and releases them into the cell media. These baculoviruses can be concentrated by centrifugation and used directly to infect mammalian cells, allowing the gene of interest to be produced in the mammalian cells.</p>
<p style="text-align: justify;"><img class="alignnone wp-image-667" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/02/BacMam-1-300x120.jpg" alt="" width="808" height="323" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/02/BacMam-1-300x120.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/02/BacMam-1-768x308.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/02/BacMam-1.jpg 837w" sizes="(max-width: 808px) 100vw, 808px" /></p>
<p style="text-align: justify;">Image source: <a href="https://ars.els-cdn.com/content/image/1-s2.0-S1359644607001006-gr1.jpg">https://ars.els-cdn.com/content/image/1-s2.0-S1359644607001006-gr1.jpg</a> from Fig 1 of the following article.</p>
<p style="text-align: justify;"><strong>Implementation of BacMam virus gene delivery technology in a drug discovery setting.</strong></p>
<p style="text-align: justify;">Kost TA, Condreay JP, Ames RS, Rees S, Romanos MA.</p>
<p style="text-align: justify;">Drug Discov Today. 2007 May;12(9-10):396-403. Epub 2007 Mar 23. DOI: <a href="http://doi.org/10.1016/j.drudis.2007.02.017">10.1016/j.drudis.2007.02.017</a></p>
<p style="text-align: justify;"><img class="alignnone wp-image-677" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/02/BacMam-2-300x287.jpg" alt="" width="429" height="410" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/02/BacMam-2-300x287.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/02/BacMam-2.jpg 668w" sizes="(max-width: 429px) 100vw, 429px" /></p>
<p style="text-align: justify;">Image source: <a href="http://www.thermofisher.com/content/dam/LifeTech/migration/images/drug-discovery-development/data.par.0356.image.420.382.1.dat-s006918-bacmam-gif.gif">http://www.thermofisher.com/content/dam/LifeTech/migration/images/drug-discovery-development/data.par.0356.image.420.382.1.dat-s006918-bacmam-gif.gif</a> from Fig 1 of <strong>ThermoFisher Scientific’s BacMam Technology Overview</strong>, which gives a more detailed workflow of BacMam production here: <a href="http://www.thermofisher.com/ca/en/home/industrial/pharma-biopharma/drug-discovery-development/target-and-lead-identification-and-validation/pathway-biology/cellular-pathway-analysis/bacmam-system/bacmam-technology-overview.html">http://www.thermofisher.com/ca/en/home/industrial/pharma-biopharma/drug-discovery-development/target-and-lead-identification-and-validation/pathway-biology/cellular-pathway-analysis/bacmam-system/bacmam-technology-overview.html</a></p>
<p style="text-align: justify;">The HTT baculovirus that I will be using was developed here at SGC Toronto by Rachel Harding, Peter Loppnau and Alma Seitova, who designed, cloned and produced the HTT baculovirus. As I have not used baculoviruses before, I started out by scouring the literature on how baculovirus transduction in mammalian cells is done. One very comprehensive paper I found was this:</p>
<p style="text-align: justify;"><strong> </strong></p>
<p style="text-align: justify;"><strong>Investigation of optimal transduction conditions for baculovirus-mediated gene delivery into mammalian cells.</strong></p>
<p style="text-align: justify;">Hsu CS, Ho YC, Wang KC, Hu YC.</p>
<p style="text-align: justify;">Biotechnol Bioeng. 2004 Oct 5;88(1):42-51. DOI: <a href="http://doi.org/10.1002/bit.20213">10.1002/bit.20213</a></p>
<p>&nbsp;</p>
<p style="text-align: justify;">According to this paper, these are the main factors that affect transduction efficiency into mammalian cells (HeLa cells were used in this paper).</p>
<ol style="text-align: justify;">
<li>Virus dosage – increasing amounts of virus will improve transduction efficiency</li>
<li>Incubation time – increasing incubation time with virus will improve transduction efficiency</li>
<li>Incubation temperature – transduction occurs better at room temperature (25˚C) rather than at 37˚C due to a decrease in virus titre at higher temperatures.</li>
<li>Surrounding solution (cell media) – D-PBS resulted in better transduction efficiency compared to the other media tested, which included DMEM</li>
<li>Cations – The presence of cations such as Ca<sup>2+</sup> and Mg<sup>2+</sup> did not significantly affect transduction efficiency</li>
</ol>
<p style="text-align: justify;">I also found this paper:</p>
<p>&nbsp;</p>
<p style="text-align: justify;"><strong>Improving promiscuous mammalian cell entry by the baculovirus Autographa californica multiple nuclear polyhedrosis virus.</strong></p>
<p style="text-align: justify;">O&#8217;Flynn NM, Patel A, Kadlec J, Jones IM.</p>
<p style="text-align: justify;">Biosci Rep. 2012 Nov 30;33(1):23-36. DOI: <a href="http://doi.org/10.1042/BSR20120093">10.1042/BSR20120093</a></p>
<p>&nbsp;</p>
<p style="text-align: justify;">Following up on point 5 on cations from the previous paper, a common cation added to aid virus entry into cells is polybrene. However, this paper shows that the addition of polybrene for baculovirus entry actually impairs transduction efficiency.</p>
<p style="text-align: justify;">With these factors in mind, I set out to optimise the conditions for baculovirus transduction in my cells.</p>
<p style="text-align: justify;">In short, I tested the following six conditions in two 24-well plates, one of which was incubated at 37˚C throughout while the other was kept at room temperature for 1h immediately following addition of baculovirus, then incubated at 37˚C:</p>
<ol style="text-align: justify;">
<li>375 µl of PBS + 125 µl of virus/PBS, replaced with 500 µl NS-A media after 1h</li>
<li>375 µl of NS-A media + 125 µl of virus/PBS, replaced with 500 µl NS-A media after 1h</li>
<li>375 µl of NS-A media + 125 µl of virus/PBS, media not changed</li>
<li>125 µl of virus, topped up with 375 µl NS-A media after 1h</li>
<li>125 µl of virus, replaced with 500 µl NS-A media after 1h</li>
<li>500 µl of NS-A media + 25 µl of virus, replaced with 500 µl NS-A media after 1 day</li>
</ol>
<p style="text-align: justify;">For each condition, a control where the same volume of PBS was added in place of baculovirus was included.</p>
<p style="text-align: justify;">Cells were harvested for protein 3 days post-transduction. Western blot was used to assess the levels of HTT present in the cells under the different conditions. You can find my full experimental set-up, the Western blot images and quantification (normalised to GAPDH and the respective PBS controls) here: <a href="http://doi.org/10.5281/zenodo.1167896">http://doi.org/10.5281/zenodo.1167896</a></p>
<p style="text-align: justify;">The clear winners are conditions 3 and 4!</p>
<ol style="text-align: justify;" start="3">
<li>375 µl of NS-A media + 125 µl of virus/PBS, media not changed</li>
<li>125 µl of virus, topped up with 375 µl NS-A media after 1h</li>
</ol>
<p style="text-align: justify;"><img class="alignnone wp-image-681" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/02/Western-300x225.jpg" alt="" width="715" height="537" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/02/Western-300x225.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/02/Western-768x576.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/02/Western-350x263.jpg 350w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/02/Western-780x586.jpg 780w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/02/Western-250x188.jpg 250w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/02/Western.jpg 960w" sizes="(max-width: 715px) 100vw, 715px" /></p>
<p style="text-align: justify;">It appears that the greatest factor contributing to increased HTT expression is the duration at which the cells are exposed to the virus. Conditions in which the cell medium was replaced with fresh medium not containing any baculovirus had the lowest levels of HTT (conditions 1, 2, 5 and 6). Between the plate incubated at 37˚C throughout and the plate kept at room temperature for 1h, there was almost no difference in HTT levels across the different conditions. In fact, by looking at the Western blot image, transduction using condition 4 at 37˚C seems to be better than at room temperature. This might be due to the cells themselves being happier at 37˚C although the viruses may fare better at lower temperatures.</p>
<p style="text-align: justify;">Subsequent experiments will be carried out using conditions 3 and/or 4.</p>
<p style="text-align: justify;">
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/overexpressing-htt-in-mammalian-cells/feed/</wfw:commentRss>
		<slash:comments>1</slash:comments>
		</item>
	</channel>
</rss>
