<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>Opher Gileadi &#8211; openlabnotebooks.org</title>
	<atom:link href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/opher-gileadi/feed/" rel="self" type="application/rss+xml" />
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<description>A growing team of groundbreaking scientists around the world are now sharing their lab notebooks online</description>
	<lastBuildDate>Tue, 16 Jul 2019 04:01:32 +0000</lastBuildDate>
	<language>en-GB</language>
	<sy:updatePeriod>hourly</sy:updatePeriod>
	<sy:updateFrequency>1</sy:updateFrequency>
	<generator>https://wordpress.org/?v=4.9.10</generator>

<image>
	<url>https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/cropped-lab-jar02-32x32.png</url>
	<title>Opher Gileadi &#8211; openlabnotebooks.org</title>
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<width>32</width>
	<height>32</height>
</image> 
	<item>
		<title>Validated antibodies for ALS research: the ALS-RAP project</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/validated-antibodies-for-als-research-the-als-rap-project/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/validated-antibodies-for-als-research-the-als-rap-project/#respond</comments>
		<pubDate>Mon, 03 Dec 2018 10:06:28 +0000</pubDate>
		<dc:creator><![CDATA[Opher Gileadi]]></dc:creator>
				<category><![CDATA[ALS-Reproducible Antibody Platform (ALS-RAP)]]></category>
		<category><![CDATA[Amyotrophic Lateral Sclerosis (ALS)]]></category>
		<category><![CDATA[Opher Gileadi]]></category>
		<category><![CDATA[SGC Univ. of Oxford]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2212</guid>
		<description><![CDATA[We are part of ALS-RAP, the ALS (Amyotrophic Lateral Sclerosis) Reproducible Antibody Platform. We’re addressing one of the big conundrums to biomedical research: the lack, or the poor quality, of many antibodies used for research ranging from cell biology to histopathology. ALS, also called MND (motor neurone disease), is a group of diseases that cause <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/validated-antibodies-for-als-research-the-als-rap-project/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>We are part of <a href="https://www.als-rap.org/">ALS-RAP</a>, the ALS (Amyotrophic Lateral Sclerosis) Reproducible Antibody Platform. We’re addressing one of the big conundrums to biomedical research: the lack, or the poor quality, of many antibodies used for research ranging from cell biology to histopathology.</p>
<p>ALS, also called MND (motor neurone disease), is a group of diseases that cause degeneration of motor neurones. The symptoms vary, but the disease is debilitating, life-threatening and currently incurable. Around 5,000 people in the UK, and 12,000 people in the US, are currently affected. There are charities in several countries devoted to helping patients and supporting research, including the <a href="http://www.alsa.org/">ALS Association</a> (US), <a href="https://www.mndassociation.org/">MNDA association</a> (UK) and the <a href="https://www.als.ca/">ALS Society of Canada</a>; they all have very informative web sites. These three charities have joined forces to fund the ALS-RAP project</p>
<p>Up to 10% of cases are hereditary, and the genetic changes have been mapped for many of these cases. Mutations of C9ORF72 and SOD1, TDP-43 and FUS are most common, but there are dozens of other genes that are mutated less frequently. Studying the ALS-related genes and their roles in healthy nerve function and in disease are keys to understanding the biology and pathology and, eventually, finding cures. To do that, we need high-quality, validated reagents; in many cases, these do not exist (or we have no way of knowing, because we haven’t tested them properly!)</p>
<p>So here’s our plan. We have set up a collaboration with Susanne Gräslund at the Karolinska Institute in Stockholm and with Tom Durcan and Peter McPherson at the Montreal Neurological Institute (McGill University).</p>
<p>First, we consulted our advisory board to build an initial <a href="https://www.als-rap.org/target-list">target list</a>; this is mostly based on genetic data, nicely summarized <a href="https://doi.org/10.2147/DNND.S84956">here</a>.</p>
<p>The Canadian groups are establishing a robust validation process: basically, using cells that express “native” levels of the target protein, and matched CRISPR knockout cells, to probe the sensitivity and specificity of antibodies. They have started to probe commercial antibodies</p>
<p>The Stockholm team has libraries of single-chain antibody variable domains (scFv) in bacteriophage; they will screen the library to isolate and characterize antibodies to each target protein. We also expect contributions of new antibodies from commercial collaborators, who will (of course) put these on their catalogues.</p>
<p>The Oxford team (that’s Tracy Keates and me) are expressing and purifying the target proteins as biotinylated, well-folded proteins or protein domains, to be used as antigens for antibody screening.</p>
<p><strong>You can see the details in our web page: <a href="http://www.als-rap.org">www.als-rap.org</a> , including the target list, target status, plasmids and protocols, validation data (soon) etc. </strong></p>
<p>In the next blog, I’ll tell more about our cloning campaign to generate the antigens.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/validated-antibodies-for-als-research-the-als-rap-project/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Introducing the team: The Gileadi Group in Oxford.</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/introducing-the-team-the-gileadi-group-in-oxford/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/introducing-the-team-the-gileadi-group-in-oxford/#respond</comments>
		<pubDate>Fri, 30 Nov 2018 16:31:10 +0000</pubDate>
		<dc:creator><![CDATA[Opher Gileadi]]></dc:creator>
				<category><![CDATA[Alzheimer’s disease]]></category>
		<category><![CDATA[Amyotrophic Lateral Sclerosis (ALS)]]></category>
		<category><![CDATA[Chordoma]]></category>
		<category><![CDATA[Opher Gileadi]]></category>
		<category><![CDATA[SGC Univ. of Oxford]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2198</guid>
		<description><![CDATA[Hi! I’m Opher Gileadi, I head a group at the Structural Genomics Consortium (SGC) in Oxford. For many years, we’ve been looking at proteins involved in human. We’ve looked at a wide variety of proteins: from DNA repair enzymes and transcription factors to enzymes involved in signalling. In most cases, we focus on early work: <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/introducing-the-team-the-gileadi-group-in-oxford/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Hi! I’m Opher Gileadi, I head a group at the Structural Genomics Consortium (SGC) in Oxford. For many years, we’ve been looking at proteins involved in human. We’ve looked at a wide variety of proteins: from DNA repair enzymes and transcription factors to enzymes involved in signalling. In most cases, we focus on early work: structure determination, activity assays, antibody generation, and finding initial small molecule “hits”. Having the right reagents are the key to enabling new drug targets; we expect that our data will allow – or encourage – further work by medicinal chemists and biologists to seriously test therapeutic hypotheses (we call our data and reagents sets <a href="https://www.thesgc.org/tep">TEPs</a> – Target Enabling Packages).</p>
<p>My team and I will be blogging and sharing our experiments on three areas of interest:</p>
<p><strong>Alzheimer’s disease</strong>: We are funded by the NIH through the <a href="https://www.nia.nih.gov/research/amp-ad">AMP-AD</a> project (Accelerating Medicines Partnership &#8211; Alzheimer&#8217;s Disease). The project has integrated data from brain autopsies and model systems to identify new possible protein targets for treatment or prevention of Alzheimer’s (annotated in the <a href="https://agora.ampadportal.org/genes/(genes-router:genes-list)">Agora</a> web site, run by <a href="http://sagebionetworks.org/research-projects/agora/">SAGE Bionetworks</a>. Our group is generating TEPs for some of these targets – we’ll share our progress in future posts.</p>
<p><strong>ALS (Amyotrophic Lateral Sclerosis)</strong>: We are part of <a href="https://www.als-rap.org/">ALS-RAP</a>, the ALS reproducible antibody platform. We’re addressing one of the big conundrums to biomedical research: the lack, or the poor quality, of many antibodies used for research ranging from cell biology to histopathology.</p>
<p>Together with colleagues in Stockholm and Montreal we aim to validate a set of antibodies to 30 proteins involved in the disease. We will test commercial antibodies as well as generate new ones through phage display technologies.</p>
<p><strong>Chordoma</strong>: <a href="https://www.chordomafoundation.org/understanding-chordoma/">Chordoma</a> is a rare type of cancer that occurs in the bones of the skull base and spine. It is resistant to all forms of chemotherapy; patients are treated by surgery and radiation, but recurrence is common and, eventually, untreatable. A unifying molecular characteristic of chordoma is overexpression of a variant of the transcription factor T-Brachyury (TBXT), which is essential for survival of the cancer cells. Joseph Newman in my group has solved structures of the human Brachyury protein (PDB codes <a href="http://www.rcsb.org/structure/6F58">6F58</a>, <a href="http://www.rcsb.org/structure/6F59">6F59</a>); we are working together with groups in the University of North Carolina and in Baylor College (Texas) to explore TBXT as a drug target.</p>
<p>That’s all for today – we’ll follow-up with stories in these three areas.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/introducing-the-team-the-gileadi-group-in-oxford/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
	</channel>
</rss>
