<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>openlabnotebooks.org</title>
	<atom:link href="https://thesgc.github.io/static-openlabnotebooks/feed/" rel="self" type="application/rss+xml" />
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<description>A growing team of groundbreaking scientists around the world are now sharing their lab notebooks online</description>
	<lastBuildDate>Mon, 26 Oct 2020 19:00:59 +0000</lastBuildDate>
	<language>en-GB</language>
	<sy:updatePeriod>
	hourly	</sy:updatePeriod>
	<sy:updateFrequency>
	1	</sy:updateFrequency>
	<generator>https://wordpress.org/?v=5.5.3</generator>

<image>
	<url>https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/cropped-lab-jar02-32x32.png</url>
	<title>openlabnotebooks.org</title>
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<width>32</width>
	<height>32</height>
</image> 
	<item>
		<title>FBXW11 interaction with β-catenin &#8211; cellular assay</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/fbxw11-interaction-with-%ce%b2-catenin-cellular-assay/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/fbxw11-interaction-with-%ce%b2-catenin-cellular-assay/#respond</comments>
		
		<dc:creator><![CDATA[Magdalena Szewczyk]]></dc:creator>
		<pubDate>Mon, 26 Oct 2020 17:05:14 +0000</pubDate>
				<category><![CDATA[Cell assay reports]]></category>
		<category><![CDATA[E3 ligases]]></category>
		<category><![CDATA[Magdalena Szewczyk]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=5309</guid>

					<description><![CDATA[Background The SCF (SKP1-CUL1-F-box protein) E3 ubiquitin ligase is a complex of proteins that helps facilitate the ubiquitination of substrates for targeted proteasomal degradation. FBXW11 is F-box containing protein that has 11 WD-repeats and is a substrate recognition component of the SCF complex. FBXW11 recognizes and binds its substrates, including beta-catenin, at phosphorylated sites or <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/fbxw11-interaction-with-%ce%b2-catenin-cellular-assay/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p><strong>Background</strong></p>
<p>The SCF (SKP1-CUL1-F-box protein) E3 ubiquitin ligase is a complex of proteins that helps facilitate the ubiquitination of substrates for targeted proteasomal degradation. FBXW11 is F-box containing protein that has 11 WD-repeats and is a substrate recognition component of the SCF complex. FBXW11 recognizes and binds its substrates, including beta-catenin, at phosphorylated sites or phosphodegrons, and recruits it to the SCF complex for subsequent ubiquitination and degradation (PMID: 10321728)</p>
<p><strong>Assay validation</strong></p>
<p>Assay measuring the interaction of FBXW11 with B-catenin was developed using NanoBRET technology &#8211; a proximity-based assay that can detect protein interactions by measuring energy transfer from a bioluminescent protein donor (NanoLuc luciferase, NL) to a fluorescent protein acceptor (fluorescent ligand bound to HaloTag, HT). In the beginning, we determined the optimal tag orientation of donor constructs: N or C-terminally NL-tagged β-catenin and acceptor constructs: N or C-terminally HT-tagged FBXW11 in HEK293T cells. NL and HT proteins, expressed alone, were used as controls to determine background signal. The best results were obtained with C-terminally NL-tagged B-catenin and C-terminally HT-tagged FBXW11 (Fig.1). The assay was further validated with binding deficient FBXW11 mutant (R447E) designed by Matthieu Schapira. R447E mutation resulted in a significant decrease in the interaction of FBXW11 with B-catenin (Fig.2).</p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/f11b-300x134.png" alt="" width="582" height="260" class=" wp-image-5311 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/f11b-300x134.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/f11b-768x343.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/f11b.png 993w" sizes="(max-width: 582px) 100vw, 582px" /></p>
<p><strong>Fig.1. FBXW11 interaction with B-catenin &#8211; NanoBRET assay. </strong>HEK293T cells were co-transfected with C- or N-terminally NanoLuc (NL)-tagged B-catenin or NL alone and C- or N-terminally Halo-tagged (HT) FBXW11 or HT alone for 24 h. The interaction was measured using NanoBRET assay. The results are MEAN of 2 technical replicates.</p>
<p><strong> <img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/fbxw11-300x168.png" alt="" width="491" height="275" class="wp-image-5310 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/fbxw11-300x168.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/fbxw11-1024x572.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/fbxw11-768x429.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/fbxw11.png 1063w" sizes="(max-width: 491px) 100vw, 491px" /></strong></p>
<p><strong>Fig.2. </strong><strong>R447E mutation decreases the interaction of FBXW11 with B-catenin.</strong> HEK293T cells were co-transfected with C-terminally Halo-tagged (HT) FBXW11 or HT and C-terminally NanoLuc (NL)-tagged B-catenin for 24 h. The interaction was measured using NanoBRET assay. The results are MEAN of 2 technical replicates.</p>
<p>Please go to <a href="https://zenodo.org/record/4134923#.X5ccxohKg2w">Zenodo</a> for experimental details.</p>
<p><strong>Acknowledgments:</strong></p>
<p>Matthieu Schapira &#8211; mutant design, Aidan Levinsky &#8211; help with mutagenesis</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/fbxw11-interaction-with-%ce%b2-catenin-cellular-assay/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>FBXW7 interaction with Cyclin E1 &#8211; cellular assay</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/fbxw7-interaction-with-cyclin-e1-cellular-assay/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/fbxw7-interaction-with-cyclin-e1-cellular-assay/#respond</comments>
		
		<dc:creator><![CDATA[Magdalena Szewczyk]]></dc:creator>
		<pubDate>Mon, 26 Oct 2020 16:42:54 +0000</pubDate>
				<category><![CDATA[Cell assay reports]]></category>
		<category><![CDATA[E3 ligases]]></category>
		<category><![CDATA[Magdalena Szewczyk]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=5302</guid>

					<description><![CDATA[Background The SCF (SKP1-CUL1-F-box protein) E3 ubiquitin ligase is a complex of proteins that helps facilitate the ubiquitination of substrates for targeted proteasomal degradation. FBXW7 (F-box/WD repeat-containing protein 7) is an F-box protein component of the SCF complex and is involved in substrate recognition. The F-box domain tethers FBXW7 to the SCF complex by interacting <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/fbxw7-interaction-with-cyclin-e1-cellular-assay/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p><strong>Background</strong></p>
<p>The SCF (SKP1-CUL1-F-box protein) E3 ubiquitin ligase is a complex of proteins that helps facilitate the ubiquitination of substrates for targeted proteasomal degradation. FBXW7 (F-box/WD repeat-containing protein 7) is an F-box protein component of the SCF complex and is involved in substrate recognition. The F-box domain tethers FBXW7 to the SCF complex by interacting with the SKP1 component while the WD40 repeats are involved in substrate binding. FBXW7 recognizes and binds its substrates at phosphorylated sites or phosphodegrons and recruits them to the SCF complex for subsequent ubiquitination and degradation (PMID:17434132). A variety of substrates exist for FBXW7, many of which are potentially oncogenic, including cyclin E1, MYC, JUN, and NOTCH (PMID: 11565034, 15103331, 14739463, 29149593). The targeted degradation of growth-promoting factors by FBXW7 suggests it likely plays an important role as a tumor suppressor. FBXW7 Arg→Cys mutation at position 465 hinders its ability to interact with SKP1 and thus the SCF complex; this mutation is associated with acute lymphoblastic leukemia (PMID: 11565033, 28007894).</p>
<p><strong>Assay validation</strong></p>
<p>Assay measuring the interaction of FBXW7 with cyclin E1 was developed using NanoBRET technology &#8211; a proximity-based assay that can detect protein interactions by measuring energy transfer from a bioluminescent protein donor (NanoLuc, NL) to a fluorescent protein acceptor (fluorescent ligand bound to HaloTag, HT). In the beginning, we determined the optimal tag orientation of donor constructs: N or C-terminally NL-tagged cyclin E1 and acceptor constructs: N or C-terminally HT- tagged FBXW7 in HEK293T cells. NanoLuc and halo tag proteins, expressed alone, were used as controls to determine background signal. The best results were obtained with C-terminally NL-tagged cyclin E1 and C-terminally HT-tagged FBXW7 (Fig.1). The assay was further validated with a binding deficient FBXW7 mutant (R465E, R505E) designed by Matthieu Schapira. R465E, R505E double mutation resulted in a significant decrease in the interaction of FBXW7 with cyclin E1 (Fig.2).</p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/fbx711-300x132.jpg" alt="" width="598" height="263" class=" wp-image-5307 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/fbx711-300x132.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/fbx711-1024x451.jpg 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/fbx711-768x338.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/fbx711.jpg 1280w" sizes="(max-width: 598px) 100vw, 598px" /></p>
<p><strong>Fig.1. FBXW7 interaction with cyclin E1 &#8211; NanoBRET assay. </strong>HEK293T cells were co-transfected with C- or N-terminally NanoLuc (NL)-tagged cyclin E1 or NL alone and C- or N-terminally Halo-tagged (HT) FBXW7 or HT alone for 24 h. The interaction was measured using NanoBRET assay. The results are MEAN of 2 technical replicates.</p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/fbx7a-300x119.jpg" alt="" width="635" height="252" class=" wp-image-5306 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/fbx7a-300x119.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/fbx7a-1024x405.jpg 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/fbx7a-768x304.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/fbx7a.jpg 1145w" sizes="(max-width: 635px) 100vw, 635px" /></p>
<p><strong> </strong><strong>Fig.2. R465E, R505E double</strong><strong> mutation decreases the interaction of FBXW7 with cyclin E1.</strong> HEK293T cells were co-transfected with C-terminally Halo-tagged (HT) FBXW7 (WT or R465E/R505E or HT and C-terminally NanoLuc (NL)-tagged cyclin E1 for 24 h. The interaction was measured using NanoBRET assay. The results are MEAN of 2 technical replicates.</p>
<p>Please go to<a href="https://zenodo.org/record/4134887#.X5b8RohKg2w"> Zenodo</a> for experimental details.</p>
<p><strong>Acknowledgments:</strong></p>
<p>Matthieu Schapira &#8211; mutant design, Aidan Levinsky &#8211; help with mutagenesis</p>
<p>&nbsp;</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/fbxw7-interaction-with-cyclin-e1-cellular-assay/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>FBXW7_349-707_NHis_Purification Protocol</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/fbxw7_349-707_nhis_purification-protocol/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/fbxw7_349-707_nhis_purification-protocol/#respond</comments>
		
		<dc:creator><![CDATA[Taraneh Hajian]]></dc:creator>
		<pubDate>Thu, 22 Oct 2020 16:41:30 +0000</pubDate>
				<category><![CDATA[Protein purification protocols]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<category><![CDATA[Taraneh Hajian]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=5295</guid>

					<description><![CDATA[Please see FBXW7-349-707_Nhis Expression and Purification protocol in the link below: https://zenodo.org/record/4118423]]></description>
										<content:encoded><![CDATA[<p>Please see FBXW7-349-707_Nhis Expression and Purification protocol in the link below:</p>
<p><a href="https://zenodo.org/record/4118423">https://zenodo.org/record/4118423</a></p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/fbxw7_349-707_nhis_purification-protocol/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>TEAD4_212-434_NHis_Expression and Purification Protocol</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/tead4_212-434_nhis_expression-and-purification-protocol/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/tead4_212-434_nhis_expression-and-purification-protocol/#respond</comments>
		
		<dc:creator><![CDATA[Taraneh Hajian]]></dc:creator>
		<pubDate>Wed, 21 Oct 2020 18:54:16 +0000</pubDate>
				<category><![CDATA[Protein purification protocols]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<category><![CDATA[Taraneh Hajian]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=5293</guid>

					<description><![CDATA[Below is the link for expression and purification TEAD4:212-434 protocol on Zonodo.https://doi.org/10.5281/zenodo.4116056 &#160; &#160;]]></description>
										<content:encoded><![CDATA[<p>Below is the link for expression and purification TEAD4:212-434 protocol on Zonodo.<a href="https://doi.org/10.5281/zenodo.4116056">https://doi.org/10.5281/zenodo.4116056</a></p>
<p>&nbsp;</p>
<p>&nbsp;</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/tead4_212-434_nhis_expression-and-purification-protocol/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>NSD3_247-402_NHis_Purification Protocol</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/nsd3_247-402_nhis_purification-protocol/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/nsd3_247-402_nhis_purification-protocol/#respond</comments>
		
		<dc:creator><![CDATA[Taraneh Hajian]]></dc:creator>
		<pubDate>Wed, 21 Oct 2020 16:06:29 +0000</pubDate>
				<category><![CDATA[Protein purification protocols]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<category><![CDATA[Taraneh Hajian]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=5291</guid>

					<description><![CDATA[The NSD protein family has been implicated in cancers, Therefore they are considered as potential targets for cancer drug discovery. Here is the Expression and purification protocol of recombinant NSD3 PWWP1 for further research. You can read more details on Zonodo in link below: https://doi.org/10.5281/zenodo.4115939]]></description>
										<content:encoded><![CDATA[<h6>The NSD protein family has been implicated in cancers, Therefore they are considered as potential targets for cancer drug discovery. Here is the Expression and purification protocol of recombinant NSD3 PWWP1 for further research.</h6>
<p>You can read more details on Zonodo in link below:</p>
<p><a href="https://doi.org/10.5281/zenodo.4115939">https://doi.org/10.5281/zenodo.4115939</a></p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/nsd3_247-402_nhis_purification-protocol/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>NSD2_208-368_NHis _Purification Protocol</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/nsd2_208-368_nhis-_purification-protocol/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/nsd2_208-368_nhis-_purification-protocol/#respond</comments>
		
		<dc:creator><![CDATA[Taraneh Hajian]]></dc:creator>
		<pubDate>Wed, 21 Oct 2020 15:29:15 +0000</pubDate>
				<category><![CDATA[Protein purification protocols]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<category><![CDATA[Taraneh Hajian]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=5288</guid>

					<description><![CDATA[The NSD protein family has been implicated in cancers, Therefore they are considered as potential targets for cancer drug discovery. Here is the Expression and purification protocol of recombinant NSD2 PWWP1 for further research. You can read the purification protocol on zenodo. &#160;]]></description>
										<content:encoded><![CDATA[<h6>The NSD protein family has been implicated in cancers, Therefore they are considered as potential targets for cancer drug discovery. Here is the Expression and purification protocol of recombinant NSD2 PWWP1 for further research. You can read the purification protocol on <a href="https://doi.org/10.5281/zenodo.4115640">zenodo</a>.</h6>
<p>&nbsp;</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/nsd2_208-368_nhis-_purification-protocol/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>Production and purification of recombinant SARS COV-2 RBDSD1-Fc protein</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/production-and-purification-of-recombinant-sars-cov-2-rbdsd1-fc-protein/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/production-and-purification-of-recombinant-sars-cov-2-rbdsd1-fc-protein/#respond</comments>
		
		<dc:creator><![CDATA[Alma Seitova]]></dc:creator>
		<pubDate>Sun, 18 Oct 2020 12:32:26 +0000</pubDate>
				<category><![CDATA[Alma Seitova]]></category>
		<category><![CDATA[COVID-19]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=5250</guid>

					<description><![CDATA[Production and purification of recombinant SARS COV-2 RBDSD1-Fc protein from a transiently transfected EXPI293 mammalian cells. Expression plasmid for CoV-9-RBDSD1, residues of RBD R319−S591 with a C-terminal HRV3C protease cleavage site, a monomeric Fc tag and an 8XHisTag was generously gifted by Dr. McLellan’s lab (1). Secreted proteins were purified from a suspension culture cell <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/production-and-purification-of-recombinant-sars-cov-2-rbdsd1-fc-protein/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p><strong>Production and purification of recombinant SARS COV-2 RBDSD1-Fc protein from a transiently transfected EXPI293 mammalian cells.<br />
Expression plasmid for CoV-9-RBDSD1, residues of RBD R319−S591 with a C-terminal HRV3C protease cleavage site, a monomeric Fc tag and an 8XHisTag was generously gifted by Dr. McLellan’s lab (1). Secreted proteins were purified from a suspension culture cell supernatant with an expression level 15-20mg/L. For different application RBD-SD1 was purified by affinity and gel-filtration protocols, with and without purification tag (Figure 1.) </p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/RBDSD1.png" alt="" width="900" height="507" class="alignnone size-full wp-image-5242" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/RBDSD1.png 900w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/RBDSD1-300x169.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/RBDSD1-768x433.png 768w" sizes="(max-width: 900px) 100vw, 900px" /></p>
<p>Reference: 1.Cryo-EM structure of the 2019-nCoV spike in the prefusion conformation. Daniel Wrapp, Nianshuang Wang, Kizzmekia S. Corbett,<a href="https://zenodo.org/record/4085730#.X4crSdBKhaQ"></a>A. Goldsmith, Ching-Lin Hsieh,Olubukola Abiona, Barney S. Graham, Jason S. McLellan .Science 367, 1260–1263 (2020) 13 March 2020</p>
<p><a href="https://zenodo.org/record/4091746#.X4zVwC0ZNAY">Please visit corresponding Zenodo page for &#8220;Materials and Methods&#8221;</a></p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/production-and-purification-of-recombinant-sars-cov-2-rbdsd1-fc-protein/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>Production and purifications of the SARS-CoV-2 related proteins from the transiently transfected mammalian cells</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/production-and-purifications-of-the-sars-cov-2-related-proteins-from-the-transiently-transfected-mammalian-cells/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/production-and-purifications-of-the-sars-cov-2-related-proteins-from-the-transiently-transfected-mammalian-cells/#respond</comments>
		
		<dc:creator><![CDATA[Alma Seitova]]></dc:creator>
		<pubDate>Wed, 14 Oct 2020 16:48:32 +0000</pubDate>
				<category><![CDATA[Alma Seitova]]></category>
		<category><![CDATA[COVID-19]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=5215</guid>

					<description><![CDATA[Over the last few months, we have all become familiar with the COVID-19 global pandemic. The coronavirus is “decorated” with the Spike (S) protein, one of the 25 proteins of SARS-CoV-2 that are encoded in the viral genome. The infection starts with the attachment of the virus to the cell membrane to enter host cells. <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/production-and-purifications-of-the-sars-cov-2-related-proteins-from-the-transiently-transfected-mammalian-cells/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p>Over the last few months, we have all become familiar with the COVID-19 global pandemic. The coronavirus is “decorated” with the Spike (S) protein, one of the 25 proteins of SARS-CoV-2 that are encoded in the viral genome.<br />
   The infection starts with the attachment of the virus to the cell membrane to enter host cells.  S- protein mediates infection by interacting with the human angiotensin-converting enzyme 2 (ACE2) receptor (Figure 1).  This interaction occurs when the receptor-binding domain (RBD) of the S- protein which recognizes and binds ACE2. This interaction mediates membrane fusion and viral entry into human cells.</p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/Covid-image.png" alt="" width="576" height="324" class="alignnone size-full wp-image-5216" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/Covid-image.png 576w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/Covid-image-300x169.png 300w" sizes="(max-width: 576px) 100vw, 576px" /></p>
<p>So, it is no wonder why the S-protein, RBD of S-protein and human ACE2 became key targets in Covid-19 related scientific research and in the worldwide collaborative efforts to develop diagnostics such as therapeutic antibodies or a vaccine.</p>
<p>To support the fight against COVID-19 and fulfill the high demand for these proteins, I volunteered, together with my colleague, Ashley Hutchinson to produce and purify high-quality S-protein, RBD and human ACE2 in mammalian cells. It is not a trivial contribution. ACE2 protein, for example, would usually cost hundreds of dollars per ~10 µg. We started this challenging journey back in April, during the lockdown of the city and of the University of Toronto.  Since then, 64 aliquots of our products have been distributed among different labs in Toronto, across Canada and abroad. We are very happy to have feedback from researches indicating that our products are performing well in the projects and in some cases even better than proteins obtained from commercial sources. </p>
<p>Please visit corresponding <a href="https://zenodo.org/record/4085730#.X4crSdBKhaQ">Zenodo</a> page for the “Materials and Methods” and details of how we produced and purified these targets to share with the global research community. </p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/production-and-purifications-of-the-sars-cov-2-related-proteins-from-the-transiently-transfected-mammalian-cells/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>RBBP7__1-425_NHis_Expression Purification Protocol</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/rbbp7__1-425_nhis_expression-purification-protocol/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/rbbp7__1-425_nhis_expression-purification-protocol/#respond</comments>
		
		<dc:creator><![CDATA[Taraneh Hajian]]></dc:creator>
		<pubDate>Wed, 14 Oct 2020 16:17:16 +0000</pubDate>
				<category><![CDATA[Protein purification protocols]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<category><![CDATA[Taraneh Hajian]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=5235</guid>

					<description><![CDATA[RBBP4 and RBBP7 (RBBP4/7) are highly homologous nuclear WD40 motif-containing proteins widely implicated in various cancers and are valuable drug targets. Below is the link to the expression and purification of the protocol for RBBP7. RBBP7 protein is used to develop drug screening assays towards the discovery of novel cancer therapeutics. https://zenodo.org/record/4086006#.X4ckLWhKi70]]></description>
										<content:encoded><![CDATA[<p>RBBP4 and RBBP7 (RBBP4/7) are highly homologous nuclear WD40 motif-containing proteins widely implicated in various cancers and are valuable drug targets. Below is the link to the expression and purification of the protocol for RBBP7. RBBP7 protein is used to develop drug screening assays towards the discovery of novel cancer therapeutics.</p>
<p><a href="https://zenodo.org/record/4086006#.X4ckLWhKi70">https://zenodo.org/record/4086006#.X4ckLWhKi70</a></p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/rbbp7__1-425_nhis_expression-purification-protocol/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>RBBP4__1-425_NHis_Expression and Purification Protocol</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/rbbp4__1-425_nhis_expression-and-purification-protocol/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/rbbp4__1-425_nhis_expression-and-purification-protocol/#respond</comments>
		
		<dc:creator><![CDATA[Taraneh Hajian]]></dc:creator>
		<pubDate>Wed, 14 Oct 2020 16:14:47 +0000</pubDate>
				<category><![CDATA[Protein purification protocols]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<category><![CDATA[Taraneh Hajian]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=5230</guid>

					<description><![CDATA[RBBP4 and RBBP7 (RBBP4/7) are highly homologous nuclear WD40 motif-containing proteins widely implicated in various cancers and are valuable drug targets. Below is the link to the expression and purification of the protocol for RBBP4. RBBP4 protein is used to develop drug screening assays towards the discovery of novel cancer therapeutics. &#160; https://zenodo.org/record/4085998#.X4cfCmhKi70]]></description>
										<content:encoded><![CDATA[<p>RBBP4 and RBBP7 (RBBP4/7) are highly homologous nuclear WD40 motif-containing proteins widely implicated in various cancers and are valuable drug targets. Below is the link to the expression and purification of the protocol for RBBP4. RBBP4 protein is used to develop drug screening assays towards the discovery of novel cancer therapeutics.</p>
<p>&nbsp;</p>
<p><a href="https://zenodo.org/record/4085998#.X4cfCmhKi70">https://zenodo.org/record/4085998#.X4cfCmhKi70</a></p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/rbbp4__1-425_nhis_expression-and-purification-protocol/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
	</channel>
</rss>
