<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>openlabnotebooks.org</title>
	<atom:link href="https://thesgc.github.io/static-openlabnotebooks/feed/" rel="self" type="application/rss+xml" />
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<description>A growing team of groundbreaking scientists around the world are now sharing their lab notebooks online</description>
	<lastBuildDate>Tue, 23 Apr 2019 19:45:27 +0000</lastBuildDate>
	<language>en-GB</language>
	<sy:updatePeriod>hourly</sy:updatePeriod>
	<sy:updateFrequency>1</sy:updateFrequency>
	<generator>https://wordpress.org/?v=4.9.10</generator>

<image>
	<url>https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/cropped-lab-jar02-32x32.png</url>
	<title>openlabnotebooks.org</title>
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<width>32</width>
	<height>32</height>
</image> 
	<item>
		<title>Why the heck does huntingtin bind PAR?</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/why-the-heck-does-huntingtin-bind-par/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/why-the-heck-does-huntingtin-bind-par/#respond</comments>
		<pubDate>Tue, 23 Apr 2019 19:45:27 +0000</pubDate>
		<dc:creator><![CDATA[Tamara Maiuri]]></dc:creator>
				<category><![CDATA[Huntingtin]]></category>
		<category><![CDATA[Huntington’s disease]]></category>
		<category><![CDATA[Tamara Maiuri]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2742</guid>
		<description><![CDATA[Updates on the huntingtin PAR binding story can be found on the Truant lab website.]]></description>
				<content:encoded><![CDATA[<p>Updates on the huntingtin PAR binding story can be found <a href="https://raytruantlab.ca/2019/04/01/why-the-heck-does-huntingtin-bind-par/">on the Truant lab website</a>.</p>
<div id="attachment_2746" style="width: 310px" class="wp-caption alignnone"><img class="size-medium wp-image-2746" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/2019-04-01-FRAP-crop-300x151.jpg" alt="" width="300" height="151" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/2019-04-01-FRAP-crop-300x151.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/2019-04-01-FRAP-crop-768x388.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/2019-04-01-FRAP-crop-1024x517.jpg 1024w" sizes="(max-width: 300px) 100vw, 300px" /><p class="wp-caption-text">FRAP experiment testing nuclear huntingtin dynamics in response to oxidative stress.</p></div>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/why-the-heck-does-huntingtin-bind-par/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>PhD position available in malaria parasite genomics and transcriptomics</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/phd-position-available-in-malaria-parasite-genomics-and-transcriptomics/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/phd-position-available-in-malaria-parasite-genomics-and-transcriptomics/#respond</comments>
		<pubDate>Tue, 23 Apr 2019 13:49:53 +0000</pubDate>
		<dc:creator><![CDATA[Antoine Claessens]]></dc:creator>
				<category><![CDATA[Antoine Claessens]]></category>
		<category><![CDATA[Malaria]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2737</guid>
		<description><![CDATA[Summary: –       3-year PhD funded by CNRS for genomics and transcriptomics data analysis of the Plasmodium falciparum malaria parasite. –       In a young dynamic team based in Montpellier, South of France, with many worldwide collaborators –       Keywords: population genetics &#38; genomics, single-cell RNA-seq, molecular epidemiology, antigenic variation.   Our research Most malaria deaths are caused <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/phd-position-available-in-malaria-parasite-genomics-and-transcriptomics/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p><strong>Summary:</strong></p>
<p><strong>–       3-year PhD funded by CNRS for genomics and transcriptomics data analysis of the <em>Plasmodium falciparum</em> malaria parasite.</strong></p>
<p><strong>–       In a young dynamic team based in Montpellier, South of France, with many worldwide collaborators</strong></p>
<p><strong>–       Keywords: population genetics &amp; genomics, single-cell RNA-seq, molecular epidemiology, antigenic variation.</strong></p>
<p><strong> </strong></p>
<p><strong>Our research</strong></p>
<p>Most malaria deaths are caused by <em>Plasmodium falciparum</em>, a unicellular eukaryote parasite transmitted by Anopheles mosquitoes. Despite the heavy burden of the disease, most human infections are asymptomatic (afebrile) and can last for months/years. In regions where malaria is seasonal, such as in The Gambia, West Africa, these asymptomatic infections probably act as a reservoir during the dry season when there is no or little transmission. <strong>Long-lasting infections could be turned to our advantage if we study them as a new host-pathogen interaction model</strong>, by regular sampling of an asymptomatic volunteer, investigating the parasite biology in its natural environment. More details on our research can be found on this <a href="https://thesgc.github.io/static-openlabnotebooks/plasmodium-falciparum-asymptomatic-infections-in-search-of-the-elephant-in-the-room/">on this post</a>.</p>
<p>From 2014 to 2017, we collected fingerprick blood samples every 3 months in a population of ~1000 inhabitants in a Gambian village. At the start of the 2017 dry season, we also recruited a cohort of 50 volunteers who were all asymptomatic carriers for <em>P. falciparum</em>. Monthly blood samples were taken for 6 months over the entire dry season. A total of 435 blood samples were collected, including this time series of monthly samples from the same volunteers over a 6-month period. <strong>This dataset represents a unique opportunity to address novel questions about the parasite biology</strong>. More specifically, we are interested in discovering how the parasite establishes a chronic infection, through changes at the transcriptomic levels.</p>
<p><strong>Your projects</strong></p>
<p>Projects here under only a suggestion, the exact project will be defined based on the student’s abilities and expertise. Your work will be entirely computer-based, but you will be part of a  team mixing ‘dry’ and ‘wet’ lab research.</p>
<p><strong>Project 1: <em>P. falciparum</em> population genetics and genomics</strong></p>
<p>As outlined above, we collected blood samples all year round from a village over two and a half year. <em>P. falciparum</em> DNA was extracted and genotyped and/or whole genome sequenced in collaboration with MalariaGEN at the Sanger Institute. A dataset of about 400 parasite DNA barcodes and 100 parasite genomes is now available. Here we want to describe the first parasite genomes sequenced from asymptomatic infections, and investigate the impact of the dry season selective pressure on the parasite population. This longitudinal approach also allows us to tackle the question of the duration of a <em>P. falciparum</em> chronic infection. Preliminary analysis revealed, for example, that a child was infected asymptomatically with the same <em>P. falciparum </em>strain for over 2.5 years. This overturns the dogma on duration of a <em>P. falciparum</em> infection.</p>
<p>Your task is to fully characterise the parasite population genetic diversity, using genotyping, genomic and epidemiologic data. The analysis will include PCA, Fst, chromosome painting, EHH, iHS, etc.</p>
<p><strong>Project 2: Can <em><u>P. falciparum</u></em> sense its environment and adapt to it via transcriptional regulation?</strong></p>
<p>In areas where rains are seasonal, two distinct patterns of malaria fluctuate, from high prevalence with a wide range of clinical outcome in the wet season, to low prevalence mainly asymptomatic in the dry season. The mechanisms allowing the parasite to survive in chronic infections during the dry season, and restart transmission as vector population increases with the ensuing wet season, remain to be investigated. We hypothesize that <em>P. falciparum</em> is able to sense its environment and dampens its virulence with seasonality.</p>
<p>In collaboration with colleagues at the London School of Hygiene &amp; Trop Med, we are currently optimizing a single-cell RNA-seq approach to <em>P. falciparum</em>.</p>
<p>Your task, using ‘bulk’ and single-cell parasite transcriptomes, is to identify differentially regulated genes in the dry vs wet season, in symptomatic vs asymptomatic infections, and identify putative candidates involved in cell growth. The analysis will include normalization, PCA for clustering and tSNE for representation, correlating bulk and single cell data with Pearson correlation, drawing gene co-expression networks within each identified cluster (subpopulation), etc.</p>
<p>The transcriptome dataset, which will be linked with <em>in vitro </em>growth assays, will be crucial in understanding how the parasite successfully establishes a chronic infection over the dry season, yet triggers a new epidemic over the following transmission season. It also has the potential of identifying dormant parasites, a discovery that would guide malaria elimination strategies.</p>
<p><strong>Other projects:</strong></p>
<p>– Population genomics with over 100 <em>P. falciparum</em> genomes collected from severe malaria or uncomplicated cases in Benin. In collaboration with Gwladys Bertin (Paris), Joe Zhu and Jacob Garcia (Oxford, UK).</p>
<p>– Identification of the micro-indels and structural variant mutation rates in <em>P. falciparum</em> in vitro using data from “clone trees” (Hamilton 2016 NAR and unpublished data).</p>
<p>– And many, many more…</p>
<p>&nbsp;</p>
<p><strong>Your skills (essentials)</strong></p>
<ul>
<li>A Master in biostatistics / bioinformatics / population genetics, or in molecular biology with a strong background in biostatistics and programming</li>
<li>Demonstrated experience in “Omics” data analysis</li>
<li>Able to work independently</li>
<li>Great communication skills in English</li>
<li>Stay focused on answering a specific biological question, from the initial analysis to publication</li>
<li>Passionate about scientific discoveries</li>
</ul>
<p><strong>Your skills (desirable)</strong></p>
<ul>
<li>Knowledge about malaria biology and population genetics</li>
<li>Wet-lab experience is a plus, but not essential</li>
<li>Willing to supervise a Master student in bioinformatics / epidemiology</li>
<li>Participate actively in the lab and institute life</li>
</ul>
<p><strong>Who we are</strong></p>
<p>Antoine Claessens (PI) is a malariologist who trained at Edinburgh University, the Sanger Institute, LSHTM and the MRC-Gambia. He recently joined Montpellier University as a “Chargé de Recherche INSERM”. He was awarded an ATIP and an ANR-JC.</p>
<p>Currently the team consists of one post-doc and one Master student. Two PhD students will be recruited around Sep 2019, one for lab-based research and one for data analysis (this advert).  Each year, 2 Master students in bioinformatics / epidemiology will join the team for up to 6 months. We are likely to recruit another post-doc towards late 2019.</p>
<p><strong>What you can expect / what we can offer</strong></p>
<p>The 3-year PhD will be funded by CNRS (substantially higher salary than other sources of funding). You will be part of <a href="http://www.dimnp.univ-montp2.fr/">DIMNP</a> and you will closely interact with experienced bioinformaticians from the bioinformatics platform and with population geneticists from <a href="https://www.mivegec.ird.fr/fr/">MIVEGEC</a>. Both ‘UMR’ are very international, with highly competent and friendly scientists. More generally, Montpellier is a large hub for research in Life Sciences, particularly in the field of evolutionary biology.</p>
<p>I will try my best to transmit my passion about <em>P. falciparum</em> biology; you might find yourself dreaming about <em>var</em> genes too. As we will be a small team, you can expect more of my time dedicated to you, but I do not want to micromanage. I expect you to work hard, yet you will have freedom on working hours etc. All our current research projects are in collaboration with international laboratories, there is an opportunity to work in the USA/UK for a short period of time. If you are planning to stay in academic research, I will help you secure a post-doc position.</p>
<p>If you are from abroad: Being able to speak French is not a requirement; your colleagues will speak English fluently. We will help you with the administrative task of moving to France. Many aspects of the cost of life, such as childcare, are relatively cheap. Most importantly, the quality of life in South of France is hard to beat!</p>
<p><strong>Key collaborators and recent publications:<br />
</strong>Umberto D’Alessandro (MRC-Gambia)<br />
Teun Bousema (Radboud UMC, Netherlands)<br />
David Conway (LSHTM, UK)<br />
Dominic Kwiatkowski, MalariaGEN, Lia Chappell (Sanger Institute)<br />
Franck Prugnolle and Virginie Rougeron (MIVEGEC, Montpellier)<br />
Arthur Talman (Sanger / MIVEGEC)<br />
Silvia Portugal (Heidelberg, Germany)<br />
Ian Cheeseman (Texas Bio Med, USA)</p>
<table width="510">
<tbody>
<tr>
<td width="326"><strong>Selected publications:</strong></td>
<td width="184"><strong>Key discovery</strong></td>
</tr>
<tr>
<td width="326"><u>Claessens A</u>, Adams Y, …, Bozdech Z and Rowe JA. Group A-like PfEMP1s mediate cytoadherence to human brain endothelial cells<em>.</em> <strong>Proc Natl Acad Sci USA</strong> 2012.</td>
<td width="184">How <em>P. falciparum</em> bind to blood microvessels in vitro (microarray transcriptomic approach)</td>
</tr>
<tr>
<td width="326"><u>Claessens A</u>*, Hamilton W*, …, Rayner J, Kwiatkowski D. Generation of antigenic diversity in Plasmodium falciparum by structured rearrangement of var genes during mitosis. <strong>PLOS Genetics</strong> 2014. *joint first author.</td>
<td width="184">New chimeric <em>var</em> gene sequences are generated during parasite growth <em>in vitro</em>.</td>
</tr>
<tr>
<td width="326">Hamilton W*, <u>Claessens A</u>*, Otto T, …, Rayner J, Kwiatkowski D. Extreme mutation bias and high AT content in Plasmodium falciparum<em>.</em> <strong>Nucleic Acid Research</strong> 2016<em>. </em></td>
<td width="184">An excess of G:C to A:T transitions explains the AT-richness of <em>P.f.</em> genome</td>
</tr>
<tr>
<td width="326"><u>Claessens A</u>, Affara M, Assefa S, Kwiatkowski D &amp; Conway D. Culture adaptation of malaria parasites selects for convergent loss-of-function mutants<em>. </em><strong>Scientific Reports</strong> 2017<em>. </em></td>
<td width="184"><em>In vitro</em> culture selects for loss-of-function mutations (genomic approach)</td>
</tr>
<tr>
<td width="326"><u>Claessens A*</u>, Harris LM*, Stanojcic S*, Merrick C. RecQ helicases in the malaria parasite Plasmodium falciparum affect genome stability, gene expression patterns and DNA replication dynamics. <strong>PLOS Genetics</strong> 2018.</td>
<td width="184">Helicase PfWRN maintains genome integrity</td>
</tr>
</tbody>
</table>
<p>&nbsp;</p>
<p><strong>Interested in applying? Please send an email to <a href="mailto:antoineclaessens@gmail.com">antoineclaessens@gmail.com</a> with subject “PhD #0002 – keep me updated”. Starting date: September 2019 (to be discussed).</strong></p>
<p>&nbsp;</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/phd-position-available-in-malaria-parasite-genomics-and-transcriptomics/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Results from soaking follow-up compounds into preformed HAO1 crystals</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/results-from-soaking-follow-up-compounds-into-preformed-hao1-crystals/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/results-from-soaking-follow-up-compounds-into-preformed-hao1-crystals/#respond</comments>
		<pubDate>Tue, 23 Apr 2019 13:39:23 +0000</pubDate>
		<dc:creator><![CDATA[Sabrina Mackinnon]]></dc:creator>
				<category><![CDATA[HAO1]]></category>
		<category><![CDATA[Primary Hyperoxaluria]]></category>
		<category><![CDATA[Sabrina Mackinnon]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2732</guid>
		<description><![CDATA[In my last post, I told you I had soaked my follow-up compounds into HAO1 crystals to start developing structural explanations of how these compounds work. So far I have analysed four interesting protein-compound structures: 2 at the active site, 1 at the gating loop and 1 at the oligomeric interface. The first structure had <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/results-from-soaking-follow-up-compounds-into-preformed-hao1-crystals/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>In my <a href="https://thesgc.github.io/static-openlabnotebooks/collecting-hao1-crystal-structures-with-follow-up-compounds-based-on-fragment-hits/">last post</a>, I told you I had soaked my follow-up compounds into HAO1 crystals to start developing structural explanations of how these compounds work. So far I have analysed four interesting protein-compound structures: 2 at the active site, 1 at the gating loop and 1 at the oligomeric interface.</p>
<p>The first structure had a compound derived from fragment 1 bound at the active site. This compound (5-bromo-N-methyl-1H-indazole-3-carboxamide; FUP1A for short) expands fragment 1 in two directions: adding a bulky, hydrophobic bromine to the aromatic ring and adding a methyl (CH<sub>3</sub>) to the carboxamide (CO-NH) at the other end of the molecule (figure 1C). FUP1A shows more typical displacement of the substrate-coordinating tryptophan (Trp110), rotating this residues by 180° so that it points away from the substrate binding pocket (figure 1A). FUP1A binds further into the substrate binding pocket than the fragment, likely due to the addition of the bulky bromine, so that it is now in the same orientation as the published HAO1 inhibitors (CDST and CCPST, PDB codes <a href="https://www.rcsb.org/structure/2RDT">2RDT</a> and<a href="https://www.rcsb.org/structure/6GMC"> 6GMC</a> respectively) as illustrated in figure 1D.</p>
<p>When taken together with the potent activity assay result – an IC50 of 62 µM, 7X improvement over fragment 1 – for a similar compound (6-bromo-N-ethyl-1H-indazole-3-carboxamide; FUP1B; structure shown in figure 1D) with the bromine one position away and a longer extension (ethyl, CH<sub>2</sub>CH<sub>3</sub>, rather than methyl) on the other side, this structure opens up some interesting questions. Does the orientation of fragment 1 reflect a more optimal position for the core scaffold? If so, does the ethyl group and/or the different position of the ethyl group of FUP1B allow it to maintain this position? Or do one or both of these groups disrupt the active site residues in an entirely different way? To answer these questions, I am currently working to obtain a structure with FUP1B bound.</p>
<p>The second structure was with a compound derived from fragment 2, also at the active site. This compound (6-amino-1-benzyl-5-(methylamino) pyrimidine-2, 4(1H, 3H)-dione; FUP2A) was chosen to explore alternate groups for the second half of fragment 2, replacing the original ring (cyclohexane) with an aromatic one (benzene), since the former did not seem to contribute to fragment binding. FUP2A did not show much improvement in the activity assay over fragment 2 and this is reflected by the bound structure, which shows that FUP2A and fragment 2 bind in very similar poses with the same displacement of Trp110 (figure 1B). As well as validating this scaffold as a consistent HAO1-binder, this follow up structure suggests that the best way to continue development would be to expand from other positions on the substituted ring, to better disrupt active site residues.</p>
<p><img class="aligncenter size-large wp-image-2733" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/post9-fig1-1024x553.png" alt="" width="780" height="421" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/post9-fig1-1024x553.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/post9-fig1-300x162.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/post9-fig1-768x414.png 768w" sizes="(max-width: 780px) 100vw, 780px" /></p>
<p style="text-align: center;"><em><b>Figure 1: New structures of active site inhibitors of HAO1. </b>A. Fragment 1 (red) and the related compound, FUP1A (purple). B. Fragment 2 (orange) and the related compound, FUP2A (yellow). C. Chemical structures of interesting compounds. D. Comparison of FUP1A (purple) and FUP2A (yellow) with the published inhibitors, CDST (green) and CCPST (orange), shown as sticks.</em></p>
<p>The third structure is of a compound bound at the gating loop and derived from fragment 5 (figure 2B). This compound (2-((4H-1, 2, 4-triazol-3-yl) thio)-1-(4-(3-chlorophenyl) piperazin-1-yl) ethan-1-one; FUP5A) replaces the fluoride on the original phenyl ring with a chloride at a different position (4-fluoro to 3-chloro) and extends the opposite end of the compound with a sulphur and a nitrogen-rich aromatic ring (triazole) (Figure 3B). FUP5A superimposes fairly well with its parent fragment at the piperazine (six-membered, non-aromatic ring with a nitrogen at either end) with the piperazine displacing the sidechain of Tyr134 and the substituted phenyl ring continuing to disrupt the gating loop (Figure 2A). the extension at the opposite end of the molecule does not cause the same conformational change (shift of helix 4) but rather points mostly out towards the solvent, not forming further interactions with the protein (Figure 3A). This structure suggests that larger extensions from the phenyl end may be more effective in disrupting the gating loop but care is needed in extending the other end of the molecule to avoid losing efficiency by including non-interacting chemical groups.</p>
<p><img class="aligncenter size-large wp-image-2735" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/post9-fig2-1024x542.png" alt="" width="780" height="413" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/post9-fig2-1024x542.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/post9-fig2-300x159.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/post9-fig2-768x407.png 768w" sizes="(max-width: 780px) 100vw, 780px" /></p>
<p style="text-align: center;"><em><b>Figure </b><b>2</b><b>: New structure of gating loop site inhibitor of HAO1. </b>A. Fragment 5 (blue) and the related compound, FUP5A (green). B. Chemical structures of interesting compounds.</em></p>
<p>The final structure has a compound derived from fragment 6 bound at the subunit interface. This compound (1-(5-chloro-pyrimidin-4-yl)-4-(pyridin-2-yl)-piperazine; FUP6A) extends the fragment scaffold, essentially in a straight line, by the addition of two further rings and the structure shows that it binds in the same manner –  through hydrophobic interactions with both subunits and pi-stacking with the ring of Tyr163 on one subunit (figure 3A). The fact that FUP6A can bind with the HAO1 tetramer intact explains why it did not show much inhibition in the activity assay and this structure also helps to further map the binding pocket at the interface. Taken together with the more potent follow up compounds (e.g. FUP6B, FUP6C, shown in figure 3B), this structure validates the need for bulky additions perpendicular to the core scaffold (like the two fluorides in FUP6B) to more effectively disrupt the HAO1 tetramer.</p>
<p><img class="aligncenter size-large wp-image-2734" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/post9-fig3-1024x782.png" alt="" width="780" height="596" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/post9-fig3-1024x782.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/post9-fig3-300x229.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/post9-fig3-768x587.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/post9-fig3.png 1046w" sizes="(max-width: 780px) 100vw, 780px" /></p>
<p style="text-align: center;"><em><b>Figure 3: New structure of interface site inhibitor of HAO1. </b>A. Fragment 6 (purple) and the related compound, FUP6A (green). B. Chemical structures of interesting compounds.</em></p>
<p>Since some compounds may require greater movements of the gating loop or tetramer subunits than can be freely accommodated within preformed crystals, I am also trying to co-crystallise with the most promising hits. This involves adding one compound per aliquot of freshly purified HAO1 protein before setting up trials to screen a wide range of crystallisation conditions. Co-crystallisation experiments requiring more time and resources than soaking but do have a few benefits: a greater number of conditions can be tested to look for those that only yield crystals with a given ligand; a lower concentration of compound can be used to achieve an equivalent saturation of the protein; and binding of the compound occurs <em>in solution,</em> accommodating a greater degree of flexibility. To this end, I have set up crystal plates with the 13 top compounds (4 active site, 6 gating loop and 3 interface), which have yielded a number of likely-looking crystals that I will be screening over the coming weeks.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/results-from-soaking-follow-up-compounds-into-preformed-hao1-crystals/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>The Benefits of Conference attendance:  BMP Signalling in Cancer II.</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/the-benefits-of-conference-attendance-bmp-signalling-in-cancer-ii/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/the-benefits-of-conference-attendance-bmp-signalling-in-cancer-ii/#respond</comments>
		<pubDate>Mon, 15 Apr 2019 10:44:07 +0000</pubDate>
		<dc:creator><![CDATA[Ellie Williams]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Ellie Williams]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2728</guid>
		<description><![CDATA[I’ve got a few things to catch up with on the blog – one of them is to talk about a conference I attended in Oxford recently. I attended the Biochemical Society BMP Signalling in Cancer II meeting which was held in St. Annes College, Oxford. At this meeting I presented a poster on the <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/the-benefits-of-conference-attendance-bmp-signalling-in-cancer-ii/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>I’ve got a few things to catch up with on the blog – one of them is to talk about a conference I attended in Oxford recently. I attended the <a href="https://www.biochemistry.org/Events/tabid/379/MeetingNo/SA217/view/Conference/Default.aspx">Biochemical Society BMP Signalling in Cancer II</a> meeting which was held in St. Annes College, Oxford. At this meeting I presented a poster on the work I’ve been doing (and which has been written about here as it’s happened) on looking at how the mutations in ALK2 effect the underlying activity of the kinase as well as effecting how well FKBP12 can inhibit ALK2.</p>
<p>The conference brought together 60 researchers in the field from around the world talking about a variety of different cancers but also other BMP related conditions such as FOP. Over three days there were some fantastic talks by the leaders in the field including from Dr. Caroline Hill from the Crick Institute in London who spoke about receptor cross talk in BMP signalling and how that might affect signalling through ALK2 and Prof. Eileen Shore from the university of Pennsylvania who spoke about mechanosensing and tissue stiffness in FOP.</p>
<p>I had a chance to get into some really interesting conversations with some of the attendees about the way their work and my work intersected and how we might be able to collaborate in the future – particularly interesting was talking to some Postdocs from Caroline Hill’s group who are looking at similar problems to me but from an in-cell perspective rather than using purified proteins. From that some interesting questions have been thrown up that I now want to test in my in-vitro systems and that will be detailed here as and when I’ve had a chance to try them. Hopefully this can lead to validation of both my work and theirs by complementing in-vivo and in-vitro approaches.</p>
<p>I thought it would be good to share the poster I created here – so you can find it <a href="https://zenodo.org/record/2640488#.XLRgljBKiUk">over on Zenodo</a>. It is designed for a scientific audience but not necessarily an FOP specialist audience so hopefully it should be fairly easy to follow. It leads the reader through the story of explaining that Gain of Function (GOF) mutations in ALK2 are involved in diseases like FOP and DIPG. I then go on to present data that shows that these mutations disrupt the binding of the inhibitor protein FKBP12 but that this isn’t the whole story (as regular readers of this blog will know). Then I talk about what I found out about what is needed for SMAD activation by ALK2 before looking at the increased phosphorylation of SMAD1 by the mutant ALK2 proteins compared to the wild type. This covers both the western blot data that shows more SMAD1 activation but also the mass spec data I’ve shared previously that looks at more SMAD1 and more ALK2 phosphorylation. Finally I showed the phosphomapping data on ALK2 that is starting to pick apart the exact phosphorylation locations on ALK2 that are necessary for SMAD1 activation (Also data I’ve talked about on this blog before). In this case it was nice to see that all the effort Rod and I had put into phosphomapping had paid off! Other groups had said they’d tried and been unable to map ALK2 using trypsin alone but our method of using 5 different enzymes both enriched and unenriched seems to have done the trick (despite it being 10x more work!).</p>
<p>There is great value from attending a conference like this and in this case I feel that I really did make some very important connections and had some very informative discussions that were directly relevant to my research and that will hopefully bear fruit in time and guide the direction of my experiments.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/the-benefits-of-conference-attendance-bmp-signalling-in-cancer-ii/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Structure-Based PROTAC Design: Project Overview</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/2714-2/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/2714-2/#respond</comments>
		<pubDate>Fri, 05 Apr 2019 13:56:33 +0000</pubDate>
		<dc:creator><![CDATA[Haresh Ajani]]></dc:creator>
				<category><![CDATA[Haresh Ajani]]></category>
		<category><![CDATA[In silico PROTACs design]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2714</guid>
		<description><![CDATA[Proteolysis Targeting Chimeras (PROTACs) are small molecules that induce the degradation of their target and have shown considerable promise as a novel therapeutic modality.1-3 PROTACs are heterobifunctional compounds containing one chemical moiety that binds to the target protein of interest and a chemical handle that binds to an E3 ubiquitin ligase (Figure 1). Recruitment of <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/2714-2/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p style="text-align: justify;">Proteolysis Targeting Chimeras (PROTACs) are small molecules that induce the degradation of their target and have shown considerable promise as a novel therapeutic modality.<sup>1-3</sup> PROTACs are heterobifunctional compounds containing one chemical moiety that binds to the target protein of interest and a chemical handle that binds to an E3 ubiquitin ligase (Figure 1). Recruitment of the E3 ligase leads to ubiquitination and subsequent degradation of the target by the proteasome. The design of PROTACs remains an empirical process, likely because of an insufficient understanding of the molecular basis for target recruitment and formation of a ternary complex.<sup>4,5 </sup>Not all ligases are compatible with all targets, and linker chemistry and attachment points play a crucial role but are poorly understood.</p>
<p>&nbsp;</p>
<p style="text-align: left;"><img class="aligncenter wp-image-2688 size-full" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/PROTAC_TERNARY_COMPLEX.png" alt="" width="931" height="566" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/PROTAC_TERNARY_COMPLEX.png 931w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/PROTAC_TERNARY_COMPLEX-300x182.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/PROTAC_TERNARY_COMPLEX-768x467.png 768w" sizes="(max-width: 931px) 100vw, 931px" /><strong> .                                </strong></p>
<h4 style="text-align: left;"><strong>                          Figure 1: a PROTAC induces the formation of a ternary complex.</strong></h4>
<p>&nbsp;</p>
<p style="text-align: justify;">The goal of this research project is to test whether computational techniques can be used to rationally guide the design of PROTACs. I will be asking four questions:</p>
<p><img class="alignnone wp-image-2694 size-full" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/PROTAC_NEW-1.png" alt="" width="1581" height="2250" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/PROTAC_NEW-1.png 1581w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/PROTAC_NEW-1-211x300.png 211w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/PROTAC_NEW-1-768x1093.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/PROTAC_NEW-1-720x1024.png 720w" sizes="(max-width: 1581px) 100vw, 1581px" /></p>
<p style="text-align: justify;">Virtual screening and structure-based drug design are quite unreliable techniques for drug-like molecules but are still used because they are generally performing better than random. Considering the structural complexity of the ternary complex central to PROTAC activity, and the chemical complexity of PROTAC molecules, in silico design of PROTACs seems even more challenging. Can rational approaches accelerate PROTAC discovery? I will try to systematically address this important question.</p>
<p>&nbsp;</p>
<p style="text-align: justify;"><strong>Reference:</strong></p>
<ol>
<li>Crews, C. M. et al. Induced protein degradation: an emerging drug discovery paradigm. Nat. Rev. Drug. Discov., <strong>16,</strong> 101-114 (2017).</li>
<li>Scudellari, M. et al. Protein-slaying drugs could be the next blockbuster therapies. Nature., <strong>567,</strong> 298-300 (2019).</li>
<li>Churcher, I. Protac-Induced Protein Degradation in Drug Discovery: Breaking the Rules or Just Making New Ones? J. Med. Chem., <strong>61,</strong> 444-452 (2018).</li>
<li>Fischer, E. S. et al. Plasticity in binding confers selectivity in ligand-induced protein degradation. Nat. Chem. Biol., <strong>14,</strong> 706–714 (2018).</li>
<li>Ciulli, A. et al. Structural basis of PROTAC cooperative recognition for selective protein degradation. Nat. Chem. Biol., <strong>13,</strong> 514–521 (2017).</li>
<li>Williams, C. I. et al. <em>In Silico </em>modeling of PROTAC-medicated ternary complexes: Validation and application. J. Chem. Inf. Model. <strong>Doi:</strong>1021/acs.jcim.8b00872</li>
</ol>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/2714-2/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Successful generation of fragments of the HTT protein and improving the purification procedure for the HTT-HAP40 complex</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/successful-generation-of-fragments-of-the-htt-protein-and-improving-the-purification-procedure-for-the-htt-hap40-complex/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/successful-generation-of-fragments-of-the-htt-protein-and-improving-the-purification-procedure-for-the-htt-hap40-complex/#respond</comments>
		<pubDate>Thu, 04 Apr 2019 15:36:43 +0000</pubDate>
		<dc:creator><![CDATA[Rachel Harding]]></dc:creator>
				<category><![CDATA[Huntington’s disease]]></category>
		<category><![CDATA[Rachel Harding]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2702</guid>
		<description><![CDATA[It has now been almost 2 years since I set out to try and make fragments of the huntingtin protein which might be amenable to structural analysis with X-ray crystallography. X-ray crystallography is a fantastic method and allows us to see the molecules in very fine atomic detail which is important if we are to <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/successful-generation-of-fragments-of-the-htt-protein-and-improving-the-purification-procedure-for-the-htt-hap40-complex/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>It has now been almost 2 years since I set out to try and make fragments of the huntingtin protein which might be amenable to structural analysis with X-ray crystallography. X-ray crystallography is a fantastic method and allows us to see the molecules in very fine atomic detail which is important if we are to understand the intricacies of the elusive huntingtin protein molecule. It has been a long hard road with almost none of our extensive cloning efforts producing any expression constructs which made sufficient yields of protein. X-ray crystallography is a protein-expensive method so we need milligram (lots of protein). But finally, I have purified milligrams of different huntingtin fragment protein samples!!!</p>
<p>It should be made very clear that this has been a big team effort with cloner extraordinaire, Peter Loppnau, the eukaryotic production, Ashley Hutchinson and Alma Seitova as well as Linda Lin doing a lot of the heavy lifting on our cloning and eukaryotic production pipeline so I am very grateful for all of their hard work. Turns out that my construct design was fine but the expression vector we used made a huge difference (this is the piece of DNA which we insert different parts of the huntingtin gene). Switching from pFBOH-MHL to pBMDEL gave us great yields! Here is the C-terminal HEAT domain protein I purified &#8211; so much pure protein! All of the other data can be found on <a href="https://zenodo.org/record/2628060">Zenodo</a>.</p>
<div id="attachment_2707" style="width: 999px" class="wp-caption alignnone"><img class="wp-image-2707 size-full" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/C-term_large-scale_20190404.png" alt="" width="989" height="908" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/C-term_large-scale_20190404.png 989w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/C-term_large-scale_20190404-300x275.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/C-term_large-scale_20190404-768x705.png 768w" sizes="(max-width: 989px) 100vw, 989px" /><p class="wp-caption-text">The gel filtration fractions were run on the gel before I had measured protein concentration – there is too much sample on the gel, but it looks great!</p></div>
<p>I have also been working to make the cleanest possible sample of the HTT-HAP40 protein complex &#8211; all data are on <a href="https://zenodo.org/record/2628064">Zenodo</a>. Having very clean samples should reduce prep-to-prep variation and reduce contamination of pesky proteins which are not either HTT or HAP40 in the sample. However, there is a trade off; more purification steps are needed to make a very pure sample but for every additional step the yield will decrease so I end up with less final material. However, I am very pleased with this sample (tempted to say cleanest prep ever?!).</p>
<div id="attachment_2706" style="width: 1047px" class="wp-caption alignnone"><img class="wp-image-2706 size-full" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/HTT-HAP40_20190404.png" alt="" width="1037" height="439" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/HTT-HAP40_20190404.png 1037w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/HTT-HAP40_20190404-300x127.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/HTT-HAP40_20190404-768x325.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/HTT-HAP40_20190404-1024x433.png 1024w" sizes="(max-width: 1037px) 100vw, 1037px" /><p class="wp-caption-text">Here you can see fractions taken throughout the HTT-HAP40 prep from starting material on the left to the final sample (FS) on the right.</p></div>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/successful-generation-of-fragments-of-the-htt-protein-and-improving-the-purification-procedure-for-the-htt-hap40-complex/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Determining whether bespoke tracers work with ALK5 in nanoBRET target engagement assay</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/determining-whether-bespoke-tracers-work-with-alk5-in-nanobret-target-engagement-assay/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/determining-whether-bespoke-tracers-work-with-alk5-in-nanobret-target-engagement-assay/#comments</comments>
		<pubDate>Thu, 04 Apr 2019 15:23:54 +0000</pubDate>
		<dc:creator><![CDATA[Jong Fu]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Brain Tumours]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Jong Fu Wong]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2687</guid>
		<description><![CDATA[Special thanks to: David Drewry – Helped with designing the nanoBRET tracers M4K pharma OICR chemist team – Synthesised the M4K1046-linker derivatives Carrow Wells (UNC) – Conjugated the M4K1046 to nanoBRET fluorophore Background: I have always wanted to establish nanoBRET target engagement assay for ALK5. Relative to dual luciferase promoter assay and immunofluorescent staining, nanoBRET <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/determining-whether-bespoke-tracers-work-with-alk5-in-nanobret-target-engagement-assay/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p><strong>Special thanks to:</strong></p>
<p>David Drewry – Helped with designing the nanoBRET tracers</p>
<p>M4K pharma OICR chemist team – Synthesised the M4K1046-linker derivatives</p>
<p>Carrow Wells (UNC) – Conjugated the M4K1046 to nanoBRET fluorophore</p>
<p><strong>Background:</strong></p>
<p>I have always wanted to establish nanoBRET target engagement assay for ALK5. Relative to dual luciferase promoter assay and immunofluorescent staining, nanoBRET is many times faster. With a robust ALK5 nanoBRET assay, I will be able to rapidly screen for cellular off-target activity.</p>
<p>Since none of the commercial nanoBRET tracers worked with ALK5, we attempted to generate our own tracers. We chose to create these bespoke tracers based on M4K1046 because it has cellular IC50 of ~50nM for ALK5. David Drewry has helped to design linkers that will attach fluorophores (nanoBRET energy acceptor) to the solvent-facing end of M4K1046. Based on known structures, additional bulk in this region should not hinder the binding of the compounds to ALK5. Two versions of M4K1046 (with different linker length) were synthesised by the M4K pharma chemist team in OICR. They were subsequently sent to Carrow Wells for conjugation to nanoBRET fluorophore.</p>
<p><strong>Experimental design:</strong></p>
<p>I wanted to determine the binding of these tracers to ALK5-nanoluciferase fusion. Therefore, I incubated increasing concentrations of tracer with HEK293 cells expressing ALK5-nanoluciferase fusion protein. If the tracers can bind nicely to the ATP pocket of ALK5, incubation with increasing concentrations of tracer will result in increasing BRET signal (wavelength = 610 nm). The resulting magnitude of BRET and EC50 estimated from the curve are good indicators to whether these tracers can be used for nanoBRET assay with ALK5.</p>
<p>To identify any background signal, I have included a replicate of the above experiment with 10,000nM of parent M4K1046. These unlabelled compound will saturate ALK5 ATP pockets and prevent the binding of nanoBRET tracers. All signal from this set of experiment is contributed by non-specific background.</p>
<p>To be eliminate doubts of any technical or reagent issues with the nanoBRET experiment itself, I have replicated both experiments above, substituting ALK5 with ALK2.</p>
<p><strong>Results:</strong></p>
<div id="attachment_2691" style="width: 610px" class="wp-caption alignnone"><img class="size-full wp-image-2691" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Fig-1.png" alt="" width="600" height="362" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Fig-1.png 600w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Fig-1-300x181.png 300w" sizes="(max-width: 600px) 100vw, 600px" /><p class="wp-caption-text">EGFP signal in transfected cells. HEK293 were transfected efficiently. Cells can be harvested for nanoBRET assay.</p></div>
<div id="attachment_2692" style="width: 579px" class="wp-caption alignnone"><img class="size-full wp-image-2692" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Fig-2.png" alt="" width="569" height="293" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Fig-2.png 569w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Fig-2-300x154.png 300w" sizes="(max-width: 569px) 100vw, 569px" /><p class="wp-caption-text">EC50 curves of the first bespoke ALK5 tracer (UNC-CIW-001-011) with ALK2-nanoluc and ALK5-nanoluc. The chemical structures of parent M4K1046 and the bespoke tracer are shown on the side of the graphs. This tracer cannot bind to ALK5.</p></div>
<div id="attachment_2693" style="width: 580px" class="wp-caption alignnone"><img class=" wp-image-2693" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Fig-3.png" alt="" width="570" height="301" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Fig-3.png 555w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Fig-3-300x158.png 300w" sizes="(max-width: 570px) 100vw, 570px" /><p class="wp-caption-text">EC50 curves of the second bespoke ALK5 tracer (UNC-CIW-001-012) with ALK2-nanoluc and ALK5-nanoluc. The chemical structures of parent M4K1046 and the bespoke tracer are shown on the side of the graphs. This tracer cannot bind to ALK5.</p></div>
<p><strong>Conclusion:</strong></p>
<p>Both bespoke tracers cannot be used for ALK5 nanoBRET. Both of them did not bind to ALK5. The nanoBRET experiment itself was a success since both tracers worked well with ALK2 (over 20 folds assay windows). The background noise was low for both tracers. ALK5 seems to intrinsically not tolerate bulky additions on the solvent end of M4K1046. This might be additional avenue for improving ALK2 vs ALK5 selectivity. At the very least, we have gained this additional information from these efforts.</p>
<p>For additional experimental details, please refer to my <a href="https://zenodo.org/record/2628675#.XKYfNlVKiUk">Zenodo post</a>.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/determining-whether-bespoke-tracers-work-with-alk5-in-nanobret-target-engagement-assay/feed/</wfw:commentRss>
		<slash:comments>2</slash:comments>
		</item>
		<item>
		<title>Novel Hinge Binders: N-Methyl-azaindoles</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/novel-hinge-binders-n-methyl-azaindoles/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/novel-hinge-binders-n-methyl-azaindoles/#respond</comments>
		<pubDate>Tue, 02 Apr 2019 18:30:55 +0000</pubDate>
		<dc:creator><![CDATA[Sean O'Byrne]]></dc:creator>
				<category><![CDATA[Sean O'Byrne]]></category>
		<category><![CDATA[Understudied Kinases]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2644</guid>
		<description><![CDATA[One goal of the CaMKK2 project is to explore the chemistry and biology of novel hinge binders. Various hinge binders showing CaMKK2 inhibition have been identified in the literature and through testing of compounds synthesized in the SGC. Of note in this search was a recent publication by Price et al. (Bioorg. Med. Chem. Lett., <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/novel-hinge-binders-n-methyl-azaindoles/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>One goal of the CaMKK2 project is to explore the chemistry and biology of novel hinge binders. Various hinge binders showing CaMKK2 inhibition have been identified in the literature and through testing of compounds synthesized in the SGC. Of note in this search was a recent publication by Price <em>et al</em>. (<em>Bioorg. Med. Chem. Lett</em>., <strong>2018</strong>, <em>28</em>, 1958–1963), in which molecules based on 7-azaindole (<strong>1,2</strong>) and 2-amino-pyrimidine (<strong>3</strong>) (<strong>Figure 1</strong>) cores showed excellent activity in an enzymatic CaMKK2 assay.</p>
<div id="attachment_2645" style="width: 790px" class="wp-caption alignnone"><img class="wp-image-2645 size-large" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Figure-1-1024x283.png" alt="" width="780" height="216" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Figure-1-1024x283.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Figure-1-300x83.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Figure-1-768x213.png 768w" sizes="(max-width: 780px) 100vw, 780px" /><p class="wp-caption-text"><strong>Figure 1</strong>: Azaindole based CaMKK2 inhibitors</p></div>
<p>Although the compounds have good potency, they are promiscuous, showing poor selectivity in a small screen against 48-86 different kinases. Indeed between 15-30% of the kinases in the panel were inhibited with an IC<sub>50</sub> &lt;100 nM.</p>
<p>Small modifications to the hinge binding region of pharmacophore can have drastic effects on the binding mode, and therefore the selectivity and potency, of inhibitors. It was decided to probe if <em>N</em>-methylation of the azaindole core could be tolerated in CaMKK2. <em>N</em>-methyl azaindole <strong>4 </strong>(<strong>Figure 1</strong>) was synthesized and sent for testing in a CaMKK2 enzyme inhibition assay. The compounds showed moderate activity giving an IC<sub>50 </sub>of 183 nM. The compound also gave had good solubility (125 µM) and was membrane permeable. It was decided to synthesis a small library of compounds based on <strong>4</strong> to see if more potent compounds based on this scaffold.</p>
<p>The compounds were relatively easy to access synthetically. <em>N</em>-methylation of 2-iodo-4-chloro-7-azaindole followed by two Suzuki couplings and an ester hydrolysis allowed quick access to the desired compounds. Conveniently, both Suzuki coupling could be carried using microwave conditions with reactions going to completion in 30 minutes (<strong>Scheme 1</strong>).</p>
<div id="attachment_2647" style="width: 790px" class="wp-caption alignnone"><img class="size-large wp-image-2647" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Scheme-1-1-1024x245.png" alt="" width="780" height="187" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Scheme-1-1-1024x245.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Scheme-1-1-300x72.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Scheme-1-1-768x184.png 768w" sizes="(max-width: 780px) 100vw, 780px" /><p class="wp-caption-text"><strong>Scheme 1</strong>: Sequential Suzuki reactions utilizing a microwave reactor to greatly reduce reaction times.</p></div>
<p>In choosing what analogues to make in the library I wanted to probe the effects of electron withdrawing and donating groups, lipophilicity and to determine the effect of extending the molecules with morpholino- groups pointing out of the binding pocket towards the solvent. The library of compounds synthesized is shown in <strong>Figure 2</strong>.</p>
<div id="attachment_2648" style="width: 790px" class="wp-caption alignnone"><img class="size-large wp-image-2648" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Figure-2-1024x515.png" alt="" width="780" height="392" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Figure-2-1024x515.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Figure-2-300x151.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Figure-2-768x386.png 768w" sizes="(max-width: 780px) 100vw, 780px" /><p class="wp-caption-text">Figure 2: Library of N-methyl-azaindoles synthesized and sent for a CaMKK2 enzyme inhibition assay.</p></div>
<p>The compounds have been shipped and we are awaiting the results from the enzyme inhibition assay. If any of the compounds have good activity, the best will be sent for a selectivity profiling using KINOMEscan screening platform provided by DiscoverX. From this we will learn if <em>N</em>-methylation can indeed modulate selectivity across the kinome.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/novel-hinge-binders-n-methyl-azaindoles/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Molecular Forecaster Inc (MFI): USP5 Zf-UBD</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/molecular-forecaster-inc-mfi-usp5-zf-ubd/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/molecular-forecaster-inc-mfi-usp5-zf-ubd/#respond</comments>
		<pubDate>Mon, 01 Apr 2019 12:30:17 +0000</pubDate>
		<dc:creator><![CDATA[Mandeep Mann]]></dc:creator>
				<category><![CDATA[Mandeep Mann]]></category>
		<category><![CDATA[USP5]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2625</guid>
		<description><![CDATA[In my last post, I mentioned I would be doing an internship at a biotech company in Montreal, Molecular Forecaster Inc as a part of a NSERC ChemNet CREATE grant. I’ve been in Montreal for about three weeks now and have had a chance to play around with the Forecaster software. I did preliminary docking <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/molecular-forecaster-inc-mfi-usp5-zf-ubd/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>In my last post, I mentioned I would be doing an internship at a biotech company in Montreal, Molecular Forecaster Inc as a part of a NSERC ChemNet CREATE grant. I’ve been in Montreal for about three weeks now and have had a chance to play around with the Forecaster software. I did preliminary docking validation studies for USP5 Zf-UBD using the Forecaster FITTED program. You can find experimental details on <a href="https://zenodo.org/record/2620208#.XKIBqihKiUk">Zenodo</a>.</p>
<p>First, I did self-docking studies with USP5 Zf-UBD co-crystal structures to assess the success of ligand docking. Self-docking is a method where a ligand is docked into the protein structure it was extracted from. To evaluate the correctness of the predicted pose, root mean square deviation values (RMSD) are evaluated; the lower the RMSD, the better the docked pose. The predicted binding poses of self-docked ligands was successful (RMSD &lt;2 Å), as shown in Figure 1.</p>
<p><img class=" wp-image-2629 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/20190401b-300x187.png" alt="" width="412" height="257" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/20190401b-300x187.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/20190401b.png 468w" sizes="(max-width: 412px) 100vw, 412px" /></p>
<p style="text-align: center;"><strong>Figure 1.</strong> USP5 Zf-UBD self-docking RMSD</p>
<p>An interesting aspect of the Forecaster platform is MATCHUP, where you can dock ligands to a protein and simulate the flexibility of the protein by providing several conformations of structures. The predicted binding poses of ligands docked using MATCHUP was also successful (RMSD &lt;2 Å), as shown in Figure 2, however the docked poses of compound DAT201 and compound DAT180 had higher RMSDs (RMSD &gt;1.5 Å) with flexible docking than rigid self-docking (Figure 3).</p>
<p><img class=" wp-image-2630 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/20190401c-300x168.png" alt="" width="447" height="250" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/20190401c-300x168.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/20190401c.png 536w" sizes="(max-width: 447px) 100vw, 447px" /></p>
<p style="text-align: center;"><strong>Figure 2</strong>. Self-docking (rigid) vs. flexible docking (MATCHUP)</p>
<p>&nbsp;</p>
<p><img class=" wp-image-2628 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/20190401a-300x148.png" alt="" width="527" height="260" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/20190401a-300x148.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/20190401a-768x380.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/20190401a-1024x506.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/20190401a.png 1456w" sizes="(max-width: 527px) 100vw, 527px" /></p>
<p style="text-align: center;"><strong>Figure 3</strong>. Flexible docked pose (green) versus co-crystallized ligand (grey) a) DAT201 b) DAT180 c) DAT194</p>
<p>I also performed rigid cross-docking of the available USP5 Zf-UBD structures. Cross-docking is when different ligands are docked to a protein structure from a complex; this is useful when deciding which structure to use for virtual screening. Table 1 summarizes the results of the cross-docking study, where structure 6NFT had the best overall average RMSD.</p>
<p style="text-align: center;"><strong>Table 1</strong>. Cross-docking: RMSDs of ligand pose with best energy complex from 3 poses</p>
<p><img class="alignnone wp-image-2631 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/20190401d-300x91.png" alt="" width="333" height="101" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/20190401d-300x91.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/20190401d.png 444w" sizes="(max-width: 333px) 100vw, 333px" /></p>
<p>Lastly, I did a virtual screen of a library of USP5 Zf-UBD compounds that have already been experimentally tested to determine the best ranking docking method (Figure 3). Flexible docking with MATCHUP results in the best overall ranking of USP5 Zf-UBD compounds, when comparing ligand activity and docking scores, as shown by the area under the curve (AUC) values. The AUC of the docking methods are marginally better than random (Figure 3-red (AUC=0.5)) which may be because all compounds in the library tested, include a carboxylic chain that drives binding in the pocket, making it challenging to distinguish between active and inactive ligands. Next, I’ll be using Forecaster’s flexible protein docking with MATCHUP to screen larger commercial libraries in the search for different ligand scaffolds against USP5 Zf-UBD. Stay tuned!</p>
<p><img class=" wp-image-2632 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/20190401e-300x40.png" alt="" width="353" height="47" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/20190401e-300x40.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/20190401e.png 476w" sizes="(max-width: 353px) 100vw, 353px" /></p>
<p><img class=" wp-image-2633 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/20190401f-300x174.png" alt="" width="445" height="258" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/20190401f-300x174.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/20190401f.png 624w" sizes="(max-width: 445px) 100vw, 445px" /></p>
<p style="text-align: center;"><strong>Figure 3. </strong>ROC of virtual screen of USP5 Zf-UBD compounds</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p><u>References </u></p>
<p>Corbeil C.R., Englebienne P., Moitessier N. Docking ligands into flexible and solvated macromolecules. 1. Development and validation of FITTED 1.0<strong> </strong><em>J. Chem. Inf. Model.</em> (2007), 47, 2, 435-449</p>
<p>Corbeil C.R., Moitessier N. Docking ligands into flexible and solvated macromolecules. 3. Impact of input ligand conformation, protein flexibility, and water molecules on the accuracy of docking programs<strong> </strong><em>J. Chem. Inf. Model.</em> (2009), 49, 4, 997-1009</p>
<p>Moitessier N., Pottel J., Therrien E., Englebienne P., Liu Z., Tomberg A., Corbeil C.R. Medicinal Chemistry Projects Requiring Imaginative Structure-Based Drug Design Methods <em>Acc. Chem. Res.</em> (2016), 49 (9), 1646-1657</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/molecular-forecaster-inc-mfi-usp5-zf-ubd/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Cell types we work with</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/cell-types-we-work-with/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/cell-types-we-work-with/#respond</comments>
		<pubDate>Sat, 23 Mar 2019 18:06:56 +0000</pubDate>
		<dc:creator><![CDATA[Thomas Durcan]]></dc:creator>
				<category><![CDATA[ALS-Reproducible Antibody Platform (ALS-RAP)]]></category>
		<category><![CDATA[Amyotrophic Lateral Sclerosis (ALS)]]></category>
		<category><![CDATA[iPSC Disease modelling]]></category>
		<category><![CDATA[iPSCs]]></category>
		<category><![CDATA[Parkinson's disease and neurodevelopmental disorders]]></category>
		<category><![CDATA[Thomas M. Durcan]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2607</guid>
		<description><![CDATA[Hello again, Three weeks have now passed since my first blog, and the response has been overwhelming, who knew this kid with a love of dinosaurs could become an internet blogger. Your encouragement to make my t-Rex has motivated me to do just that, so let’s see how it all goes. Also fun fact, the <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/cell-types-we-work-with/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Hello again,</p>
<p>Three weeks have now passed since my first blog, and the response has been overwhelming, who knew this kid with a love of dinosaurs could become an internet blogger. Your encouragement to make my t-Rex has motivated me to do just that, so let’s see how it all goes. Also fun fact, the biggest and oldest T-Rex was found in <a href="https://www.ctvnews.ca/sci-tech/paleontologists-discover-world-s-biggest-and-oldest-tyrannosaurus-rex-in-canada-1.4348206">Canada</a>, so looks like I am in right place for dinosaurs.</p>
<p>But, in the meantime, I do have a day, night and weekend job running a lab and academic drug discovery group, and so its time to discuss science. More specifically, what cells we work with in our work. Our bread and butter are the iPSC, short for induced pluripotent stem cells. Made from the blood of patients and healthy volunteers, when you look at them, they look like regular old cells, tightly packed into colonies, as they love being close to their fellow cells, problems arise when they want to move away and get some distance from their neighbors. Below are some images of what they look like, courtesy of Carol Chen and Narges Abdian, who help manage and oversee our catalog of stem cell lines in my group, along with Zhipeng You. As you can see, the cells are tightly packed, with each cell having the potential to grow into any cell type you want, all it needs is the right ingredients to direct it into the cell type of your choosing.</p>
<p><a href="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/iPSC-cell-image.tif"><img class="alignnone size-medium wp-image-2610" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/iPSC-cell-image.tif" alt="" width="1" height="1" /><img class="size-medium wp-image-2611 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Image-of-IPSCs_forblog-300x249.jpg" alt="" width="300" height="249" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Image-of-IPSCs_forblog-300x249.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Image-of-IPSCs_forblog-768x638.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Image-of-IPSCs_forblog-1024x851.jpg 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Image-of-IPSCs_forblog.jpg 1175w" sizes="(max-width: 300px) 100vw, 300px" /></a></p>
<p>And what might these cells be, well we are interested in diseases of the brain, and in particular the cells lost during the progression of these degenerative disorders. This means our focus is on getting these stem cells to form neurons, of which everyone of us has over 100 billion. We can make many different types and over the coming months, I will be putting online our different SOPs from our group outlining how we make many of these different neurons. One of the main interests for our group is Parkinson’s disease, and the cells lost in this disease are dopaminergic neurons, making it our mission to understand why these neurons are being lost in the disease. Other cell types we make are motor neurons, sensory neurons and cortical neurons, all different types of neurons that are involved in different diseases, playing different roles. For those who wonder what they look like, here are some images below, staining for proteins that are specific for these neurons (Dopaminergic-Top; Cortical- Bottom). Images are courtesy of Carol Chen and Cecilia DeRocha Desouza.</p>
<p><img class="wp-image-2612 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/DA-neurons-300x232.jpg" alt="" width="299" height="231" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/DA-neurons-300x232.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/DA-neurons-768x594.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/DA-neurons-1024x792.jpg 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/DA-neurons.jpg 1323w" sizes="(max-width: 299px) 100vw, 299px" /><img class="wp-image-2613 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/cortical-neurons.jpg" alt="" width="296" height="234" /></p>
<p>In addition to neurons, we also make the support cells for these neurons, and one of these cells is astrocytes. This cell can be the good guy-bad guy and has been demonstrated to support neuronal survival, and on the flip side, to promote the loss of neurons. Courtesy of Vincent Soubannier, here are some images of what these cells look like in the dish.<img class="size-medium wp-image-2615 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/astrocytes-1-300x241.jpg" alt="" width="300" height="241" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/astrocytes-1-300x241.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/astrocytes-1-768x617.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/astrocytes-1.jpg 908w" sizes="(max-width: 300px) 100vw, 300px" /></p>
<p>Now, I know, you are saying Tom, these neurons and astrocytes are beautiful, but they are flat, having been grown in 2D dishes. This isn’t like a brain; the brain is three dimensional and complex. And you know what, you would be right. Growing cells in 2D is artificial, but it lets us grow and image neurons and astrocytes for doing research on human cells that otherwise wouldn’t be accessible. But it wasn’t until the pioneering work by Dr <a href="https://www2.mrc-lmb.cam.ac.uk/groups/lancaster/">Madeline Lancaster</a> and others that the idea for making complex brain structures emerged. From pioneering work in her lab, she showed that neuronal organoids or “minibrains” could be formed in a dish. More complex then anything grown in 2D, these minibrains could be expanded, grown for several months, if not years, and show many characteristics of different regions within the human brain. Over 18 months ago, we turned our attention to minibrains and now after all this time, we can make three different types in our group: midbrain, cerebral and forebrain neuronal organoids or “minibrains”. You can learn more about this work through the enclosed link, with <a href="https://www.mcgill.ca/neuro/category/article-categories/neuroxxceptional-stories/nguyen-vi-mohamed">Dr Nguyen-Vi Mohamed</a> showing how these are made in the lab.</p>
<p>These can grow up to 4mm in size depending on the minibrain grown, and are comprised of neurons, astrocytes and many other cell types. They can be maintained for months, and our oldest minibrains were over 14 months old. Just as a preview of what they look like, below is a section through one of these minibrains at 30 days, courtesy of Dr Nguyen-Vi Mohamed.</p>
<p><img class=" wp-image-2616 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/minibrain-section-300x244.jpg" alt="" width="304" height="247" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/minibrain-section-300x244.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/minibrain-section-768x624.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/minibrain-section-1024x832.jpg 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/minibrain-section.jpg 1281w" sizes="(max-width: 304px) 100vw, 304px" /></p>
<p>With incubators packed with these little guys, we now have over 5000 growing with the capacity to maintain close to 10,000 at any one time. These are probably the coolest things I have ever seen and am amazed at the things that can be done with them. Its such a new model, we are barely scratching the surface at what can be done, but I am looking forward to trying new things and seeing what others do. To end, I wanted to bring peoples attention to some remarkable work from from the Lancaster group published this <a href="https://www.theguardian.com/science/2019/mar/18/scientists-grow-mini-brain-on-the-move-that-can-contract-muscle">week</a> (I am like a fanboy of her work). They could connect a minibrain and muscle, with the minibrain taking control over the muscle and causing it to contract. This is just the tip of the iceberg and now I can’t wait to see what other structures we can connect to a minibrain. It won’t be long before we have little mini bodies on a dish, with minibrains connected to miniature versions of all the major organs in the body. But for now, we wait and see how the field progresses. For those wondering how we make minibrains, our <a href="https://mniopenresearch.org/articles/3-1/v1">methods paper</a> is coming out in next week or so, and once its out, I will use this blog to cover the main points on how we make these organoid structures amongst other hot topics in the stem cell field. Next blog, I will be discuss how to thaw out a vial of iPSCs for all those new users wondering what I should do when I get a tube of these cells. Well don’t panic, I am here to help you. Feel free to connect with me on <a href="https://www.linkedin.com/in/thomas-durcan-18593b23/">LinkedIn</a>, if I don’t know you please at least introduce yourself and who you are. I also now on the twitter (@thomas.durcan), so you can follow me there too, I am a relative novice at this whole social media malarkey, so we shall see how that goes.</p>
<p>&nbsp;</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/cell-types-we-work-with/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
	</channel>
</rss>
