<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>openlabnotebooks.org</title>
	<atom:link href="https://thesgc.github.io/static-openlabnotebooks/feed/" rel="self" type="application/rss+xml" />
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<description>A growing team of groundbreaking scientists around the world are now sharing their lab notebooks online</description>
	<lastBuildDate>Sat, 23 Mar 2019 18:06:56 +0000</lastBuildDate>
	<language>en-GB</language>
	<sy:updatePeriod>hourly</sy:updatePeriod>
	<sy:updateFrequency>1</sy:updateFrequency>
	<generator>https://wordpress.org/?v=4.9.10</generator>

<image>
	<url>https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/cropped-lab-jar02-32x32.png</url>
	<title>openlabnotebooks.org</title>
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<width>32</width>
	<height>32</height>
</image> 
	<item>
		<title>Cell types we work with</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/cell-types-we-work-with/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/cell-types-we-work-with/#respond</comments>
		<pubDate>Sat, 23 Mar 2019 18:06:56 +0000</pubDate>
		<dc:creator><![CDATA[Thomas Durcan]]></dc:creator>
				<category><![CDATA[ALS-Reproducible Antibody Platform (ALS-RAP)]]></category>
		<category><![CDATA[Amyotrophic Lateral Sclerosis (ALS)]]></category>
		<category><![CDATA[iPSC Disease modelling]]></category>
		<category><![CDATA[iPSCs]]></category>
		<category><![CDATA[Parkinson's disease and neurodevelopmental disorders]]></category>
		<category><![CDATA[Thomas M. Durcan]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2607</guid>
		<description><![CDATA[Hello again, Three weeks have now passed since my first blog, and the response has been overwhelming, who knew this kid with a love of dinosaurs could become an internet blogger. Your encouragement to make my t-Rex has motivated me to do just that, so let’s see how it all goes. Also fun fact, the <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/cell-types-we-work-with/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Hello again,</p>
<p>Three weeks have now passed since my first blog, and the response has been overwhelming, who knew this kid with a love of dinosaurs could become an internet blogger. Your encouragement to make my t-Rex has motivated me to do just that, so let’s see how it all goes. Also fun fact, the biggest and oldest T-Rex was found in <a href="https://www.ctvnews.ca/sci-tech/paleontologists-discover-world-s-biggest-and-oldest-tyrannosaurus-rex-in-canada-1.4348206">Canada</a>, so looks like I am in right place for dinosaurs.</p>
<p>But, in the meantime, I do have a day, night and weekend job running a lab and academic drug discovery group, and so its time to discuss science. More specifically, what cells we work with in our work. Our bread and butter are the iPSC, short for induced pluripotent stem cells. Made from the blood of patients and healthy volunteers, when you look at them, they look like regular old cells, tightly packed into colonies, as they love being close to their fellow cells, problems arise when they want to move away and get some distance from their neighbors. Below are some images of what they look like, courtesy of Carol Chen and Narges Abdian, who help manage and oversee our catalog of stem cell lines in my group, along with Zhipeng You. As you can see, the cells are tightly packed, with each cell having the potential to grow into any cell type you want, all it needs is the right ingredients to direct it into the cell type of your choosing.</p>
<p><a href="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/iPSC-cell-image.tif"><img class="alignnone size-medium wp-image-2610" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/iPSC-cell-image.tif" alt="" width="1" height="1" /><img class="size-medium wp-image-2611 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Image-of-IPSCs_forblog-300x249.jpg" alt="" width="300" height="249" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Image-of-IPSCs_forblog-300x249.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Image-of-IPSCs_forblog-768x638.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Image-of-IPSCs_forblog-1024x851.jpg 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Image-of-IPSCs_forblog.jpg 1175w" sizes="(max-width: 300px) 100vw, 300px" /></a></p>
<p>And what might these cells be, well we are interested in diseases of the brain, and in particular the cells lost during the progression of these degenerative disorders. This means our focus is on getting these stem cells to form neurons, of which everyone of us has over 100 billion. We can make many different types and over the coming months, I will be putting online our different SOPs from our group outlining how we make many of these different neurons. One of the main interests for our group is Parkinson’s disease, and the cells lost in this disease are dopaminergic neurons, making it our mission to understand why these neurons are being lost in the disease. Other cell types we make are motor neurons, sensory neurons and cortical neurons, all different types of neurons that are involved in different diseases, playing different roles. For those who wonder what they look like, here are some images below, staining for proteins that are specific for these neurons (Dopaminergic-Top; Cortical- Bottom). Images are courtesy of Carol Chen and Cecilia DeRocha Desouza.</p>
<p><img class="wp-image-2612 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/DA-neurons-300x232.jpg" alt="" width="299" height="231" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/DA-neurons-300x232.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/DA-neurons-768x594.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/DA-neurons-1024x792.jpg 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/DA-neurons.jpg 1323w" sizes="(max-width: 299px) 100vw, 299px" /><img class="wp-image-2613 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/cortical-neurons.jpg" alt="" width="296" height="234" /></p>
<p>In addition to neurons, we also make the support cells for these neurons, and one of these cells is astrocytes. This cell can be the good guy-bad guy and has been demonstrated to support neuronal survival, and on the flip side, to promote the loss of neurons. Courtesy of Vincent Soubannier, here are some images of what these cells look like in the dish.<img class="size-medium wp-image-2615 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/astrocytes-1-300x241.jpg" alt="" width="300" height="241" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/astrocytes-1-300x241.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/astrocytes-1-768x617.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/astrocytes-1.jpg 908w" sizes="(max-width: 300px) 100vw, 300px" /></p>
<p>Now, I know, you are saying Tom, these neurons and astrocytes are beautiful, but they are flat, having been grown in 2D dishes. This isn’t like a brain; the brain is three dimensional and complex. And you know what, you would be right. Growing cells in 2D is artificial, but it lets us grow and image neurons and astrocytes for doing research on human cells that otherwise wouldn’t be accessible. But it wasn’t until the pioneering work by Dr <a href="https://www2.mrc-lmb.cam.ac.uk/groups/lancaster/">Madeline Lancaster</a> and others that the idea for making complex brain structures emerged. From pioneering work in her lab, she showed that neuronal organoids or “minibrains” could be formed in a dish. More complex then anything grown in 2D, these minibrains could be expanded, grown for several months, if not years, and show many characteristics of different regions within the human brain. Over 18 months ago, we turned our attention to minibrains and now after all this time, we can make three different types in our group: midbrain, cerebral and forebrain neuronal organoids or “minibrains”. You can learn more about this work through the enclosed link, with <a href="https://www.mcgill.ca/neuro/category/article-categories/neuroxxceptional-stories/nguyen-vi-mohamed">Dr Nguyen-Vi Mohamed</a> showing how these are made in the lab.</p>
<p>These can grow up to 4mm in size depending on the minibrain grown, and are comprised of neurons, astrocytes and many other cell types. They can be maintained for months, and our oldest minibrains were over 14 months old. Just as a preview of what they look like, below is a section through one of these minibrains at 30 days, courtesy of Dr Nguyen-Vi Mohamed.</p>
<p><img class=" wp-image-2616 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/minibrain-section-300x244.jpg" alt="" width="304" height="247" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/minibrain-section-300x244.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/minibrain-section-768x624.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/minibrain-section-1024x832.jpg 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/minibrain-section.jpg 1281w" sizes="(max-width: 304px) 100vw, 304px" /></p>
<p>With incubators packed with these little guys, we now have over 5000 growing with the capacity to maintain close to 10,000 at any one time. These are probably the coolest things I have ever seen and am amazed at the things that can be done with them. Its such a new model, we are barely scratching the surface at what can be done, but I am looking forward to trying new things and seeing what others do. To end, I wanted to bring peoples attention to some remarkable work from from the Lancaster group published this <a href="https://www.theguardian.com/science/2019/mar/18/scientists-grow-mini-brain-on-the-move-that-can-contract-muscle">week</a> (I am like a fanboy of her work). They could connect a minibrain and muscle, with the minibrain taking control over the muscle and causing it to contract. This is just the tip of the iceberg and now I can’t wait to see what other structures we can connect to a minibrain. It won’t be long before we have little mini bodies on a dish, with minibrains connected to miniature versions of all the major organs in the body. But for now, we wait and see how the field progresses. For those wondering how we make minibrains, our methods paper is coming out in next week or so, and once its out, I will use this blog to cover the main points on how we make these organoid structures amongst other hot topics in the stem cell field. Next blog, I will be discuss how to thaw out a vial of iPSCs for all those new users wondering what I should do when I get a tube of these cells. Well don’t panic, I am here to help you. Feel free to connect with me on <a href="https://www.linkedin.com/in/thomas-durcan-18593b23/">LinkedIn</a>, if I don’t know you please at least introduce yourself and who you are. I also now on the twitter (@thomas.durcan), so you can follow me there too, I am a relative novice at this whole social media malarkey, so we shall see how that goes.</p>
<p>&nbsp;</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/cell-types-we-work-with/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Synthesis of promiscuous ALK2 inhibitors</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/synthesis-of-promiscuous-alk2-inhibitors/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/synthesis-of-promiscuous-alk2-inhibitors/#respond</comments>
		<pubDate>Tue, 19 Mar 2019 20:34:09 +0000</pubDate>
		<dc:creator><![CDATA[Alfredo Picado]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Alfredo Picado]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>
		<category><![CDATA[Understudied Kinases]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2598</guid>
		<description><![CDATA[In a collaboration between SGC-University of North Carolina-Chapel Hill and M4K Pharma, we plan to synthesize 3 different imidazopyridin pyrazines as promiscuous kinase ALK2 inhibitors, compounds 1-3, Scheme 1. The first step will be the synthesis of key intermediate 10, Scheme 2.  The proposed synthesis begins with a nucleophilic aromatic substitution on compound 4 to <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/synthesis-of-promiscuous-alk2-inhibitors/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>In a collaboration between SGC-University of North Carolina-Chapel Hill and M4K Pharma, we plan to synthesize 3 different imidazopyridin pyrazines as promiscuous kinase ALK2 inhibitors, compounds <strong>1-3</strong>, Scheme 1.</p>
<div id="attachment_2596" style="width: 994px" class="wp-caption alignnone"><img class="size-full wp-image-2596" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Screen-Shot-2019-03-19-at-4.22.27-PM.png" alt="" width="984" height="346" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Screen-Shot-2019-03-19-at-4.22.27-PM.png 984w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Screen-Shot-2019-03-19-at-4.22.27-PM-300x105.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Screen-Shot-2019-03-19-at-4.22.27-PM-768x270.png 768w" sizes="(max-width: 984px) 100vw, 984px" /><p class="wp-caption-text">Scheme 1. ALK2 inhibitors</p></div>
<p>The first step will be the synthesis of key intermediate <strong>10</strong>, Scheme 2.  The proposed synthesis begins with a nucleophilic aromatic substitution on compound <strong>4</strong> to get nitropyridine <strong>5</strong>, then reduction with hydrogen on palladium/carbon to obtain the pyridine-3,4-diamine <strong>6</strong>.</p>
<p>On the other hand, aminopyrazine <strong>7</strong> will be saponified to the aminopyrazine sodium salt <strong>8</strong>, which will be coupled with  compound <strong>6</strong> using peptide coupling reagent HATU to form bromopyrazine carboxamide <strong>9</strong>, followed by heating in presence of acetic acid to yield the cyclized intermediate 5-bromo-3-(1-ethyl-1<em>H</em>-imidazo[4,5-<em>c</em>]pyridin-2-yl)pyrazin-2-amine <strong>10</strong>.</p>
<div id="attachment_2597" style="width: 928px" class="wp-caption alignnone"><img class="size-full wp-image-2597" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Screen-Shot-2019-03-19-at-4.22.42-PM.png" alt="" width="918" height="650" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Screen-Shot-2019-03-19-at-4.22.42-PM.png 918w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Screen-Shot-2019-03-19-at-4.22.42-PM-300x212.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Screen-Shot-2019-03-19-at-4.22.42-PM-768x544.png 768w" sizes="(max-width: 918px) 100vw, 918px" /><p class="wp-caption-text">Scheme 2. Synthesis route 1</p></div>
<p>If we succeed with the proposed synthesis of imidazopyridine pyrazine <strong>10</strong>, then soon we will present next steps to get to target compounds <strong>1-3</strong></p>
<p>This collaboration is the under supervision of Dr. David Drewry.</p>
<p>Synthesis routes were adapted from international patents WO 2006/063167 A1, WO 2008/071937 and <em>ACS Med. Chem. Lett</em>. 2016, 7, 5, 514-519.</p>
<p>&nbsp;</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/synthesis-of-promiscuous-alk2-inhibitors/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Phosphorylation over time and phosphomapping experiments to investigate the influence of ALK2 mutations on activity.</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/phosphorylation-over-time-and-phosphomapping-experiments-to-investigate-the-influence-of-alk2-mutations-on-activity/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/phosphorylation-over-time-and-phosphomapping-experiments-to-investigate-the-influence-of-alk2-mutations-on-activity/#respond</comments>
		<pubDate>Tue, 12 Mar 2019 11:56:29 +0000</pubDate>
		<dc:creator><![CDATA[Ellie Williams]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Ellie Williams]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2579</guid>
		<description><![CDATA[In this post I’d like to return to looking at my work on phosphorylation. Phosphorylation is a key signalling method for activating the BMP pathway and inducing activity. Specifically activation of ALK2 by phosphorylation leads to phosphorylation of SMAD1 which leads to gene transcription that ultimately is responsible for bone formation. In this post I’m <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/phosphorylation-over-time-and-phosphomapping-experiments-to-investigate-the-influence-of-alk2-mutations-on-activity/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>In this post I’d like to return to looking at my work on phosphorylation. Phosphorylation is a key signalling method for activating the BMP pathway and inducing activity. Specifically activation of ALK2 by phosphorylation leads to phosphorylation of SMAD1 which leads to gene transcription that ultimately is responsible for bone formation.</p>
<p>In this post I’m going to talk about two experiments – the first is looking at the rate of phosphorylation (or how active) of two ALK2 mutants, L196P and R258S. The second is to look at mapping the exact sites of phosphorylation on two of the more common ALK2 mutations in FOP, R206H and Q027E.</p>
<p>&nbsp;</p>
<p><strong>Phosphorylation over time in ALK2 L196P and R258S.</strong></p>
<p>I’ve talked <a href="https://thesgc.github.io/static-openlabnotebooks/phosphorylation-of-smad1-rates-as-a-function-of-alk2-mutations-and-type-ii-association/">previously</a> about how phosphorylation is effected by the Q207E and R206H mutations in ALK2. This time I wanted to look at some additional mutants that are found in a small proportion of cases of FOP, specifically L196P, R258S, G328E and R375P. I purified the long forms of these ALK2 mutations in a standard fashion. Regrettably only L196P and R258S gave a good yield of expressed protein while the other two constructs gave me no expression of protein. I set up my standard activation assay with these ALK2 mutants (I mixed them with SMAD1 and with either ACVR2 or BMPR2, before adding ATP, DTT and Mg2+/Mn2+ in order to start the reaction), taking samples over time periodically up to 30 minutes. The samples were then analysed by mass spec to look for intact mass shifts corresponding to phosphorylation.</p>
<p>I plotted the peaks corresponding to phosphorylated states of both ALK2 and SMAD1 at the different time points and used this 3D graph to compare different mutant’s behaviour with respect to the activity of ALK2. I also compared this data to previously obtained data from the previous blog post.</p>
<p><img class="alignnone size-large wp-image-2580" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/MS_timecourse_blogpost-806x1024.jpg" alt="" width="780" height="991" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/MS_timecourse_blogpost-806x1024.jpg 806w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/MS_timecourse_blogpost-236x300.jpg 236w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/MS_timecourse_blogpost-768x975.jpg 768w" sizes="(max-width: 780px) 100vw, 780px" /></p>
<p><em>Time course plots for phosphorylation of SMAD1 (top) and ALK2 L196P or R258S (bottom) in the presence of ACVR2 (left) or BMPR2 (right). Legend shows time point in seconds.</em></p>
<p>&nbsp;</p>
<p>Overall we can see that R258S appears to be more activating than L196P with both more SMAD activation over the first five minutes (including the appearance of a potentially interesting third SMAD binding site). This is the case when using either ACVR2 or BMPR2. Also in general ACVR2 appears to lead to more SMAD activation than BMPR2 regardless of mutation.</p>
<p>When looking at ALK2 phosphorylation, we can see that L196P is quickly phosphorylated up to three times compared to R258S but that over the course of five minutes, R258S accumulates up to 6 phosphorylation’s much more rapidly than L196P. This effect is most pronounced when paired with ACVR2.</p>
<p>What this means in very broad terms is that the different ALK2 FOP mutants are differently activating and that their response to different type II receptors change depending on which mutation is present. This is similar to the results seen with other ALK2 mutants studied previously. This builds towards understanding why the ALK2 FOP mutations leads to increased signalling and thus increased bone formation and exactly what is going on.</p>
<p>&nbsp;</p>
<p><strong>Phosphomapping of ALK2 R206H and Q207E:</strong></p>
<p>Another aspect of looking at the phosphorylation of ALK2 and SMAD1 is not just how much it is phosphorylated but also where it is phosphorylated. For this I worked with Rod Chalk in our mass spec department to map the phosphorylation sites present in samples of protein incubated as previously mentioned. In this case I was looking at Q207E and R206H mutants of ALK2 paired with either ACVR2 or BMPR2, all in the presence of SMAD. This suggested that many of the phosphorylations could be traced to the following positions in the GS region of the kinase regardless of ALK2 mutation:</p>
<p><strong><u>ST</u></strong>LADLLDH<strong><u>S</u></strong>C<strong><u>TS</u></strong>G<strong><u>S</u></strong>G<strong><u>S</u></strong>GLPFLVQRTV</p>
<p>However not all these sites are seen at the same time and it is hard to determine which are essential for transmission of signal and ALK2 activation.</p>
<p>Similarly phosphorylation on SMAD1 could be traced to the following common locations at the C-terminal tail of the protein:</p>
<p>TQMGSPHNPI<strong><u>SS</u></strong>V<strong><u>S</u></strong></p>
<p>In this case it is the last three serines in the tail that are able to be phosphorylated and again, are seen with both mutations. Combinations of the three possible sites and all three at once were seen as possibilities within the phosphomapping data.</p>
<p>What this means however is a bit less clear. When considering the time course data from the previous post linked as well as the new mutant time course data and the phosphomapping it seems to suggest that you require multiple (up to 6) ALK2 phosphorylations to get maximal (3 site) SMAD1 phosphorylation and that there are at least this many sites in the GS loop and the region just prior to it.</p>
<p>In order to properly probe the necessity of each of these sites, it is going to be necessary to repeat these mass spec experiments on knock out mutants that remove the phosphorylation sites in various combinations, to really see which are essential for ALK2 activation. These are currently being cloned so hopefully I’ll be able to test them soon.</p>
<p>&nbsp;</p>
<p>You can find the technical details for all these experiments over on <a href="https://doi.org/10.5281/zenodo.2591062">Zenodo.</a></p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/phosphorylation-over-time-and-phosphomapping-experiments-to-investigate-the-influence-of-alk2-mutations-on-activity/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>USP5 Zf-UBD Co-Crystal Structure with Compound XSR00035795a &#038; Testing Selectivity</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/usp5-zf-ubd-co-crystal-structure-with-compound-xsr00035795a-testing-selectivity/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/usp5-zf-ubd-co-crystal-structure-with-compound-xsr00035795a-testing-selectivity/#respond</comments>
		<pubDate>Fri, 01 Mar 2019 19:24:16 +0000</pubDate>
		<dc:creator><![CDATA[Mandeep Mann]]></dc:creator>
				<category><![CDATA[Mandeep Mann]]></category>
		<category><![CDATA[USP5]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2548</guid>
		<description><![CDATA[In a previous post, I tested hit analogues against USP5 Zf-UBD using a SPR assay. Compound AE-641/11456811, now aptly re-named XSR00035795a (an SGC global ID- since our database infrastructure is finally re-vamped), had a binding affinity of approximately 60 µM. The addition of a methyl group on the carboxylic chain of the compound increased potency <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/usp5-zf-ubd-co-crystal-structure-with-compound-xsr00035795a-testing-selectivity/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>In a <a href="https://thesgc.github.io/static-openlabnotebooks/testing-hit-analogues-against-usp5-zf-ubd-with-spr-assay-1/">previous post</a>, I tested hit analogues against USP5 Zf-UBD using a SPR assay. Compound AE-641/11456811, now aptly re-named XSR00035795a (an SGC global ID- since our database infrastructure is finally re-vamped), had a binding affinity of approximately 60 µM. The addition of a methyl group on the carboxylic chain of the compound increased potency 3-fold!</p>
<p>You can find details of this experiment on <a href="https://zenodo.org/record/2581522#.XHmFHIhKjIU">Zenodo</a>. I crystallized the complex of USP5 Zf-UBD with XSR00035795a, and the structure was solved by Dr. Wolfram Tempel. The addition of the methyl group leads to increased hydrophobic interactions in the binding pocket with W209; the side chain of R221 also rotates out of the pocket (Figure 1).</p>
<p><img class="wp-image-2549 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/20190301e-300x122.png" alt="" width="585" height="237" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/20190301e-300x122.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/20190301e-768x313.png 768w" sizes="(max-width: 585px) 100vw, 585px" /></p>
<p style="text-align: center;"><strong>Figure 1</strong>. a) Superimposed USP5 structures of compound DAT194 (PDB: 6NFT; cyan) and compound XSR00035795a (magenta) b) Binding pocket of co-crystal structure of USP5 Zf-UBD and XSR00035795a</p>
<p>Next, I tested this compound against HDAC6 Zf-UBD to see if it confers selectivity against USP5 Zf-UBD using a SPR assay (Figure 2). Compound XSR00035795a (LE=0.36) was not selective for USP5 Zf-UBD. XSR00035795a and HDAC6 Zf-UBD has a K<sub>D</sub> of 15 ± 0.4 µM (n=2) whereas USP5 Zf-UBD has an average K<sub>D</sub> of 60 ± 8 µM (n=3).</p>
<p style="text-align: left;">a)                                                                                                        b)<img class="wp-image-2550 alignnone" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/20190301f-300x75.png" alt="" width="648" height="162" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/20190301f-300x75.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/20190301f-768x193.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/20190301f-1024x257.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/20190301f.png 1372w" sizes="(max-width: 648px) 100vw, 648px" /></p>
<p style="text-align: center;"><strong>Figure 2</strong>. Representative SPR binding curves and sensograms of compound XSR00035795a with a) HDAC6 Zf-UBD (K<sub>D</sub>= 15 ± 0.4 µM; n=2) and b) USP5 Zf-UBD (K<sub>D</sub>= 60 ± 8 µM; n=3)</p>
<p>When co-crystal structures of HDAC6 Zf-UBD and USP5 Zf-UBD are superimposed, binding of XSR00035795a to HDAC6 Zf-UBD makes sense. Figure 3 shows the binding pocket of HDAC6 Zf-UBD is able to accommodate the methyl group of compound XSR00035795a. Replacing a valine residue in the primary pocket of HDAC6 with an alanine in USP5 generates a small cavity that may be exploited by extending the aliphatic group on the carboxylic chain to confer selectivity towards USP5.</p>
<p><img class=" wp-image-2551 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/20190301g-300x226.png" alt="" width="448" height="336" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/20190301g-300x226.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/20190301g-350x263.png 350w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/20190301g-250x188.png 250w" sizes="(max-width: 448px) 100vw, 448px" /></p>
<p style="text-align: center;"><strong>Figure 3</strong>. Superimposed CPK representation of HDAC6 Zf-UBD (purple PDB: 6CE8) and USP5 Zf-UBD and compound XSR00035795a co-crystal structure (cyan)</p>
<p>Commercial compounds with extended aliphatic groups on the scaffold have been ordered but unfortunately due to vendor delays, I haven’t received them! Here’s hoping the compounds will arrive soon…</p>
<p>On another note, I will be doing a three month internship at the Montreal biotech, Molecular Forecaster Inc., as part of a NSERC-CREATE ChemNET grant. I will further explore computational chemistry approaches and structure-based drug design to design inhibitors against USP5 Zf-UBD and use the docking platform to dock compounds against available subfamily protein domain structures, as a predictive tool for selectivity. I’ll be heading to Montreal next week! Stay tuned for some neat computational simulations against USP5 Zf-UBD and the results of the extended aliphatic groups on the XSR00035795a scaffold.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/usp5-zf-ubd-co-crystal-structure-with-compound-xsr00035795a-testing-selectivity/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Checking I haven&#8217;t got my cells mixed up&#8230;</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/checking-i-havent-got-my-cells-mixed-up/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/checking-i-havent-got-my-cells-mixed-up/#respond</comments>
		<pubDate>Thu, 28 Feb 2019 18:56:05 +0000</pubDate>
		<dc:creator><![CDATA[Elizabeth Brown]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Brain Tumours]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>
		<category><![CDATA[Liz Brown]]></category>
		<category><![CDATA[Understudied Kinases]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2542</guid>
		<description><![CDATA[Did you know as many as 30% of the cells in labs could be misidentified? And what a nightmare it would be to find out years of your work are just completely wrong because you got your skin cells mixed up with your brain cells&#8230; So to avoid that catastrophe, I took some time to <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/checking-i-havent-got-my-cells-mixed-up/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Did you know as many as 30% of the cells in labs could be misidentified? And what a nightmare it would be to find out years of your work are just completely wrong because you got your skin cells mixed up with your brain cells&#8230;</p>
<p>So to avoid that catastrophe, I took some time to check the DNA sequence of the ACVR1 gene, because for this gene we know what should be mutated (or not) for each of my cell lines. This way I can make sure that I haven&#8217;t mixed them up in the past year (<em>and</em> that they haven&#8217;t picked up new mutations in this important area all by themselves).</p>
<p>All I had to do was collect the genomes of the cells, amplify specifically the ACVR1 gene, send it for overnight sequencing, and then I get a trace like this back from the facility:</p>
<div id="attachment_2544" style="width: 790px" class="wp-caption aligncenter"><a href="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/R206H-sequence-1.png"><img class="wp-image-2544 size-large" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/R206H-sequence-1-1024x396.png" alt="Sequencing trace covering the R206H mutation in ACVR1 found in FOP and DIPG" width="780" height="302" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/R206H-sequence-1-1024x396.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/R206H-sequence-1-300x116.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/R206H-sequence-1-768x297.png 768w" sizes="(max-width: 780px) 100vw, 780px" /></a><p class="wp-caption-text">Sequencing trace covering the R206H mutation in ACVR1 found in FOP and DIPG</p></div>
<p>Each DNA base has one peak, except one that has both a guanine (G) and an adenine (A) DNA base. The adenine, present on one chromosome, causes the change from an arginine (R) to a histidine (H) at the 206th position of the ACVR1 protein i.e. the R206H mutation.</p>
<p>This trace is from the HSJD-DIPG-007 cell line and is the mutation I expected. According to all my sequencing, none of my cell lines have gotten mixed up, yay!</p>
<p>You can read the full details of how I did this, <a href="https://zenodo.org/record/2579809#.XHgumMD7SUk">here on Zenodo</a>.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/checking-i-havent-got-my-cells-mixed-up/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Modelling diseases of the brain through stem cells</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/modelling-diseases-of-the-brain-through-stem-cells/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/modelling-diseases-of-the-brain-through-stem-cells/#respond</comments>
		<pubDate>Thu, 28 Feb 2019 00:41:34 +0000</pubDate>
		<dc:creator><![CDATA[Thomas Durcan]]></dc:creator>
				<category><![CDATA[ALS-Reproducible Antibody Platform (ALS-RAP)]]></category>
		<category><![CDATA[Amyotrophic Lateral Sclerosis (ALS)]]></category>
		<category><![CDATA[iPSC Disease modelling]]></category>
		<category><![CDATA[iPSCs]]></category>
		<category><![CDATA[Parkinson's disease and neurodevelopmental disorders]]></category>
		<category><![CDATA[Thomas M. Durcan]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2536</guid>
		<description><![CDATA[Hello SGC members and to the millions of scientific readers across the globe, My name is Thomas Durcan, and I am an assistant professor at the Montreal Neurological Institute (MNI) and McGill University, where I also lead the SGC tissue platform in Montreal. You can call me the stem cell guy, as everything I and my <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/modelling-diseases-of-the-brain-through-stem-cells/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Hello SGC members and to the millions of scientific readers across the globe,</p>
<p>My name is Thomas Durcan, and I am an assistant professor at the <a href="https://www.mcgill.ca/neuro/">Montreal Neurological Institute (MNI) </a>and <a href="https://www.mcgill.ca/">McGill University</a>, where I also lead the <a href="https://ultra-dd.org/tissue-platforms">SGC tissue platform</a> in Montreal. You can call me the stem cell guy, as everything I and my team does is centered on human stem cells, developing disease models for understanding and treating diseases of the brain. So who am I, well let me introduce myself. I hail from Dublin, Ireland where I was born and raised. I wanted to clone dinosaurs from the moment I saw Jurassic park, so I did everything in my power to become a scientist. During my time at <a href="https://www.ucd.ie/">University College Dublin</a>, I looked at the DNA of plants, although no breakthroughs emerged in my quest for dinosaurs. With a degree in hand, I headed across to North America, first to the USA and the <a href="https://www.nd.edu/">University of Notre Dame</a>, and then onto colder climes in Canada, settling in Montreal, Quebec. It is here in Montreal, where I have learned to survive winter, to sample the finest bagels, to develop a passion for wine and where I got the opportunity to understand the cellular and molecular causes of Parkinson&#8217;s disease working in the lab of <a href="https://www.mcgill.ca/neuro/research/researchers/fon">Dr. Edward Fon</a>.</p>
<p>However, all my studies were with regular old Hela or HEK293T cells, cells that are clearly not neuronal. When studying a disease of the brain, it would make sense to actually study a human neuron, but these aren&#8217;t readily accessible. I can&#8217;t go up to someone and ask them for their neurons as what you are born with is kind of what you need. So for years, our field was stuck, we had some advances, but it was always in a Hela-type cell, or animal model, which often lacked many of the features of the human disease itself. Then the breakthrough came, in 2006 with the Yamanaka factors. A set of four transcription factors that when introduced into skin cells from a mouse or human, brought them back to the beginning. It reprogrammed the cell, and made it pluripotent, coining the term induced pluripotent stem cell or iPSC. By reprogramming the skin cell, it removed all epigenetics and created a stem cell, that was capable of generating any tissue or cell type within the human body. All it needed, was the correct set of factors to do so.</p>
<p>SInce this seminal publication, the field has grown exponentially and I started working on stem cells in 2014, leading to the creation of the <a href="https://www.mcgill.ca/ipsquebec-mni/">iPSC platform</a> at the MNI, under the direction of Dr. Edward Fon, and myself as group leader, and in partnership with the iPSC platform at the Universite of Laval, under the leadership of <a href="http://www.jpuymiratlab.crchul.ulaval.ca/">Jack Puymerat</a>.</p>
<p>When we began, we were three people, and fast forward to 2019, our group is now over 30 full time staff and students, and growing every month, with the iPSC platform starting its transition this year into the Early Drug Discovery Unit (Neuro-EDDU- further details to follow). With iPSCs at the core of our mission we are now engaged in many early drug discovery projects with researchers, biotech and large pharma, generating neurons for studying Parkinson&#8217;s disease, Amyotrophic Lateral Sclerosis (ALS) and neurodevelopmental biology. Not only are we generating neurons on 2D dishes, but we have been growing minibrains by the thousands, with these 3D neuronal organoids mimicking the human brain more closely then anything seen with 2D cultures. At the same time, we have been fortunate that CRISPR genome editing has emerged, providing us the tools to edit a gene at the base pair level. This technology is at the core of our involvement with the <a href="https://als-rap.org/">ALS-RAP</a> project, in which we are making a panel of over 30 KO cells against ALS-associated genes, using these cells to validate antibodies. With all these tools now set up within the group, new big data approaches are being explored by members of my team, both for image based and MultiOmics based analysis of neurons and other disease-relevant cells generated from patient iPSCs.</p>
<p>This is just a snap-shot into the type of things we are doing, and we are now a step closer to making dinosaur stem cells, fulfilling a life long dream of mine, while also advancing new therapies into the clinic that we hope will one day treat these devastating disorders. Yet, we can&#8217;t do this alone, which is why for my blog I will be starting to make available all our SOPs as part of the open science mission of the MNI and TOSI. All our tips and tricks will be posted on a regular basis, from how to grow an iPSC all the way to to how to make a minibrain will be added to the blog over the coming months, to show everyone how to work with iPSCs. In parallel, as one of the three leads on the <a href="https://als-rap.org/">ALS-RAP</a> project, I will be posting updates on our antibody validation efforts and progress on this project, keeping people up to date on antibodies we are testing, and pointing to ones you should consider working with.</p>
<p>I look forward to opening the doors to our group, and hopefully over the coming months you will enjoy hearing about the work being done in Montreal, and at the MNI on stem cells and antibody validation. Next time, I will start by introducing our SOPs for working with iPSCs, starting with how to thaw a frozen vial of iPSCs. To talk or to lean more  feel free to reach me through <a href="https://www.linkedin.com/in/thomas-durcan-18593b23/">linkedin</a>.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/modelling-diseases-of-the-brain-through-stem-cells/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Quinazoline Series: Synthesis</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/quinazoline-series-synthesis/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/quinazoline-series-synthesis/#respond</comments>
		<pubDate>Sun, 24 Feb 2019 22:58:15 +0000</pubDate>
		<dc:creator><![CDATA[Benjamin Eduful]]></dc:creator>
				<category><![CDATA[Ben Eduful]]></category>
		<category><![CDATA[Understudied Kinases]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2434</guid>
		<description><![CDATA[We have received some very interesting biological data for our CaMKK2 inhibitor series (furopyridine, Azaindole and quinoline) and I will be sharing the data in the coming weeks, but before that, I wanted to share my synthetic efforts towards our quinoline series. Synthesis of Quinoline series: Starting with 6-Bromo-4-chloroquinoline, a two (2) step synthetic scheme <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/quinazoline-series-synthesis/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p style="text-align: justify;">We have received some very interesting biological data for our CaMKK2 inhibitor series (furopyridine, Azaindole and quinoline) and I will be sharing the data in the coming weeks, but before that, I wanted to share my synthetic efforts towards our quinoline series.</p>
<p><strong>Synthesis of Quinoline series</strong>:</p>
<p style="text-align: justify;">Starting with 6-Bromo-4-chloroquinoline, a two (2) step synthetic scheme was devised where the bromine (Br) synthetic handle was substituted first (with an appropriate boronic acid/ester), followed by the chlorine (Cl) , <strong>Figure 1</strong>, after testing many different reaction conditions for selectivity.</p>
<p style="text-align: center;"><strong>Figure 1: Synthesis of quinoline series</strong></p>
<p><img class=" wp-image-2526 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/Quinoline-1-300x67.png" alt="" width="416" height="93" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/Quinoline-1-300x67.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/Quinoline-1.png 569w" sizes="(max-width: 416px) 100vw, 416px" /></p>
<p style="text-align: justify;">A number of analogs were thus synthesized using the above synthetic strategy, for example, synthesis of the quinoline analog: 2-cyclopentyl-4-(6-(4-(methylsulfonyl)phenyl)quinolin-4-yl)benzoic acid was achieved using this method, <strong>Scheme 1</strong>.</p>
<p style="text-align: center;"><strong>Scheme 1: Synthesis of quinoline analog</strong></p>
<p><img class="wp-image-2527 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/Quinoline-2-300x88.png" alt="" width="701" height="206" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/Quinoline-2-300x88.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/Quinoline-2-768x226.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/Quinoline-2.png 936w" sizes="(max-width: 701px) 100vw, 701px" /></p>
<p style="text-align: justify;">Although the synthetic strategy above led to the synthesis of some analogs, it was not without challenges. For instance, selectivity for the Br was not always achieved as disubstituted products were common in some cases due to the activated 4-position of the Cl.</p>
<p style="text-align: justify;">In order to overcome these challenges and to allow for rapid synthesis of different analogs over a short period of time, a strategy to first install the cyclopentyl derivative was sort after. In the synthetic scheme below, <strong>Scheme 2</strong>, 6-chloroquinolin-4-ol, <strong>101</strong>, was treated with triflic anhydride in the presence of DIPEA in DCM to obtain 6-chloroquinolin-4-yl trifluoromethanesulfonate, <strong>102</strong>. This compound was able to undergo the Suzuki cross coupling reaction to obtain compound <strong>103</strong>. With this compound in hand, a number of different analogs are being generated in our search for active CaMKK2 chemical probes, for example compound <strong>105</strong>, <strong>Scheme 2</strong>.</p>
<p style="text-align: center;"><strong>Scheme 2: Synthesis of quinoline analog via new route</strong></p>
<p><img class=" wp-image-2528 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/Quinoline-3-300x71.png" alt="" width="758" height="179" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/Quinoline-3-300x71.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/Quinoline-3-768x182.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/Quinoline-3-1024x242.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/Quinoline-3.png 1282w" sizes="(max-width: 758px) 100vw, 758px" /></p>
<p>We are also looking to exploring the 7-position of the quinoline with similar substitutions to the 6-position (<strong>Figure 2</strong>), quinoline derivative 1. Other anticipated analogs will include O-alkylations (quinoline derivatives 2 and 3), N-alkylations or peptide couplings or Hartwig-Buchwald aminations (quinolines derivatives 4 and 5) etc. Details of such reactions will be communicated in due course.</p>
<p style="text-align: center;"><strong>Figure 2: Exploration of other quinoline analogs</strong></p>
<p><img class=" wp-image-2529 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/Quinoline-4-300x81.png" alt="" width="504" height="136" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/Quinoline-4-300x81.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/Quinoline-4-768x207.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/Quinoline-4.png 858w" sizes="(max-width: 504px) 100vw, 504px" /></p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/quinazoline-series-synthesis/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>ZFYVE9C purification and association test with TGFBR1 and TGFBR2</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/zfyve9c-purification-and-association-test-with-tgfbr1-and-tgfbr2/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/zfyve9c-purification-and-association-test-with-tgfbr1-and-tgfbr2/#comments</comments>
		<pubDate>Fri, 22 Feb 2019 14:05:47 +0000</pubDate>
		<dc:creator><![CDATA[Ellie Williams]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Ellie Williams]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2515</guid>
		<description><![CDATA[ZFYVE9C (Otherwise known as SARA) is a scaffold protein involved in the association of various protein components as part of the TGFB signalling pathway. The equivalent in the BMP signalling pathway is a protein known as Endofin so studying SARA (which is slightly easier) might give us useful insights for both proteins. For more details <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/zfyve9c-purification-and-association-test-with-tgfbr1-and-tgfbr2/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>ZFYVE9C (Otherwise known as SARA) is a scaffold protein involved in the association of various protein components as part of the TGFB signalling pathway. The equivalent in the BMP signalling pathway is a protein known as Endofin so studying SARA (which is slightly easier) might give us useful insights for both proteins. For more details on this project please see <a href="https://thesgc.github.io/static-openlabnotebooks/a-summary-of-my-alk2-project-goals/">this blog post.</a></p>
<p>One of the unanswered questions about SARA is what does one of the major domains that it has actually do? (The domain is known as a DUF domain, or a Domain of Unknown Function!) We know the structure but that hasn’t given us any clues as to its function. One thought was that it might be involved in recruiting type I or type II receptors close to their binding partners such as one of the SMADs to allow signal propagation.</p>
<p>I purified the DUF domain of SARA and mixed it with either excess TGFBR1 or excess TGFBR2 (the two relevant receptors in the TGFB pathway). I tested the receptors in both the unphosphorylated (inactive) and phosphorylated (active) states. After incubating the samples for 30 minutes while concentrating the volume back down to &lt;5ml, I ran them down a gel filtration column to see if the two proteins bound together sufficiently to shift the peaks coming off the column which would indicate they were acting as a complex of larger mass.</p>
<p><img class="alignnone size-large wp-image-2516" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/gf_trace_zfyve9c-1024x491.png" alt="" width="780" height="374" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/gf_trace_zfyve9c-1024x491.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/gf_trace_zfyve9c-300x144.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/gf_trace_zfyve9c-768x368.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/gf_trace_zfyve9c.png 1578w" sizes="(max-width: 780px) 100vw, 780px" /></p>
<p><em>Overlay of the UV traces from all the size exclusion gel filtration runs to show the relative elution volumes of the peaks confirming no shift in the ZFYVE9C peak consistent with no complex being formed.</em></p>
<p>&nbsp;</p>
<p>Unfortunately I didn’t see any shift in the peak for SARA suggesting that either the two proteins don’t bind or that the affinity is weak enough that they fall apart under the conditions of the gel filtration column.</p>
<p>The next thing to try will be to purify the proteins again but not to cleave the tag on one of the binding partners (either the receptors or the SARA protein) and try to do a pull down experiment to test whether this shows any association or not.</p>
<p>&nbsp;</p>
<p>You can find more details over at<a href="https://doi.org/10.5281/zenodo.2575581"> Zenodo.</a></p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/zfyve9c-purification-and-association-test-with-tgfbr1-and-tgfbr2/feed/</wfw:commentRss>
		<slash:comments>2</slash:comments>
		</item>
		<item>
		<title>Three new ALK2 structures, one of them not the one I thought!</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/three-new-alk2-structures-one-of-them-not-the-one-i-thought/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/three-new-alk2-structures-one-of-them-not-the-one-i-thought/#respond</comments>
		<pubDate>Tue, 19 Feb 2019 17:46:47 +0000</pubDate>
		<dc:creator><![CDATA[Roslin Adamson]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Brain Tumours]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>
		<category><![CDATA[Roslin Adamson]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2507</guid>
		<description><![CDATA[I’ve not posted in a while, mainly because things have been really busy, but good busy, as I now have 3 new datasets/structures, of ALK2 with M4K2118, M4K2149 and, amazingly, M4K2009. As you know, this is our lead candidate, which until now has not behaved very well in crystal trials, and it hasn’t been possible <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/three-new-alk2-structures-one-of-them-not-the-one-i-thought/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>I’ve not posted in a while, mainly because things have been really busy, but good busy, as I now have 3 new datasets/structures, of ALK2 with M4K2118, M4K2149 and, amazingly, M4K2009. As you know, this is our lead candidate, which until now has not behaved very well in crystal trials, and it hasn’t been possible to get decent crystals. In my last post I told you how I had somehow managed to mix up the TNIK and ACVR1/ALK2 tubes when I was setting up the plates, and accidentally put ACVR1 into a coarse screen and TNIK into a fine screen for ACVR1. Well, after I got lovely big crystals for what I thought was ACVR1 with M4K3003, I sent them to the synchrotron, got beautiful data to 1.4 Å, phased it, and lo and behold! It was M4K2009 snugly fitted into the binding pocket (see the ligand density after molecular replacement below). Of course, I didn’t notice this was a trimethoxy containing compound (how??) before I excitedly emailed everyone to tell them I had M4K3003, and then had to retract … However, the story ends well as M4K2009 is really important.</p>
<p><img class="aligncenter  wp-image-2511" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/M4k2009-1024x576.jpg" alt="" width="904" height="509" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/M4k2009-1024x576.jpg 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/M4k2009-300x169.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/M4k2009-768x432.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/M4k2009.jpg 1280w" sizes="(max-width: 904px) 100vw, 904px" /></p>
<p>Additionally, I got data for M4K2149 (~2.6 Å), one of the amide head group containing compounds we’re interested to see, and M4K2118 (~1.4 Å), which is doing well in some of the other assays too. I’ve put the .mtz files <a href="https://zenodo.org/record/2573142#.XGw9g-j7SUk">here</a> on Zenodo if you want to play with high resolution data, and some images below showing the ligand density and the ligands binding in the pocket. The amide head group of M4K2149 appears to preserve the water network in the pocket that we’ve observed in other structures, but as the resolution is too low for waters, we can’t tell til we get higher resolution data. Which hopefully will be this week, as I’ve grown some really pretty crystals for M4K2149 that I hope will diffract better. These appeared quite by accident again, as I’d designed a fine screen around the D5 well condition of my mega composite screen, but neglected to notice when I validated the screen that I’d put 10 x less ammonium sulphate in one row than intended – should have been 0.9 M, but ended up being 0.09 M. I should probably start calling myself ‘the accidental crystallographer’ at this rate.</p>
<p><img class="aligncenter  wp-image-2509" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/M4K2118-2149-1024x576.png" alt="" width="883" height="497" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/M4K2118-2149-1024x576.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/M4K2118-2149-300x169.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/M4K2118-2149-768x432.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/M4K2118-2149.png 1280w" sizes="(max-width: 883px) 100vw, 883px" /></p>
<p>In the meantime, I’ve got some small crystals for M4K3007, another structure we really want, and some pretty ones for M4K2184, another amide head group compound. I’ve followed up the M4K3007, just waiting to see if anything appears, and will send the tiny ones to the synchrotron while waiting.</p>
<p><img class="aligncenter  wp-image-2508" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/crystal-pics-1024x576.png" alt="" width="849" height="478" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/crystal-pics-1024x576.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/crystal-pics-300x169.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/crystal-pics-768x432.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/crystal-pics.png 1280w" sizes="(max-width: 849px) 100vw, 849px" /></p>
<p>I’m also very happy to say that our new M4K2117 crystal system for XChem fragment screening has worked very nicely so far, in the sense that the crystals are large and relatively robust and can handle soaking in low salt conditions as well as compounds with DMSO for a few hours. They then still diffract to around 1.2-1.8 Å, consistently in the same space group, C 1 2 1. This is a vast improvement over the previous system with LDN-193189. We did a solvent characterisation and pre-screen a couple of weeks ago with 100 crystals, and of the 20 or so hits we got, none was in the allosteric pocket we are looking for. This could be because of the 1.4 M ammonium sulphate and 0.2 M Na/K tartrate I have in there, which might prevent access of fragments to the pocket. Thus, using the remarkable shifter that Nathan Wright at SGC designed, we’ll be transferring loads of crystals from grow plates to low salt soak plates before doing the compound soaks this week, with the same 100 compounds, in order to check if the hit rate/location changes.</p>
<p>I will mention though, for anyone who may never have noticed this, the DMSO concentration in the protein solution for crystallisation is very important! I had previously been using Batch 1 of M4K2117 for my co-crystallisation, but it ran out so I used Batch 2, which was a 50 mM stock, as opposed to the previous 25 mM stock. The plates I set up with this, the same in every other way, gave showers of rather ugly, crunchy, small crystals. So I tried a few things – diluted the stock to 25 mM in DMSO and set up plates with that, with both lower and higher concentrations of protein (8 and 14 mg/ml along with the usual 10-11 mg/ml), as well as using the stock as is, with lower and higher protein concentrations. The lower stock concentrations brought back the big, perfect, beautiful crystals from before, the higher stock, no matter the protein concentration, gave ugly chunks and showers. The final concentration of compound was the same in all, so clearly the DMSO percentage is a very important factor.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/three-new-alk2-structures-one-of-them-not-the-one-i-thought/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Collecting HAO1 crystal structures with follow-up compounds based on fragment hits</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/collecting-hao1-crystal-structures-with-follow-up-compounds-based-on-fragment-hits/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/collecting-hao1-crystal-structures-with-follow-up-compounds-based-on-fragment-hits/#respond</comments>
		<pubDate>Tue, 12 Feb 2019 16:32:27 +0000</pubDate>
		<dc:creator><![CDATA[Sabrina Mackinnon]]></dc:creator>
				<category><![CDATA[HAO1]]></category>
		<category><![CDATA[Primary Hyperoxaluria]]></category>
		<category><![CDATA[Sabrina Mackinnon]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2488</guid>
		<description><![CDATA[A belated Happy New Year everyone!  It has been a good while since my last post so I wanted to update you on what I have been spending 2019 on so far. After identifying some really promising follow-up compounds in my activity assay, the next step was to find out how they bind to HAO1, <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/collecting-hao1-crystal-structures-with-follow-up-compounds-based-on-fragment-hits/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>A belated Happy New Year everyone!  It has been a good while since my last post so I wanted to update you on what I have been spending 2019 on so far.</p>
<p>After identifying some really promising follow-up compounds in my activity assay, the next step was to find out how they bind to HAO1, as illustrated in figure 1. Specifically I want to know how different chemical constituents of the new compounds interact with and/or cause conformational changes to amino acids within the HAO1 structure. By understanding which groups improve binding (e.g. that stabilise the bound state by interacting with nearby amino acids), which group increase inhibition (e.g. by disrupting active site residues like Trp110) and which groups are not required for the compound to work (i.e. no apparent interaction with the protein), we can rationally design compounds that combine the most effective chemical groups to make more potent inhibitors.</p>
<p>In practical terms, this means that I have made &gt;250 HAO1 crystals and soaked each of my follow up compounds twice (127 compounds times 2 crystals each = 254 crystals soaked). This duplication helps ensure complete coverage of all compounds (e.g. if one replicate is not mounted/collected correctly) and also improves confidence in my results i.e. if I see the same compound making the same interactions in two independent structures, I feel confident that the interactions I see are significant changes dependent on compound-binding.</p>
<p><img class="aligncenter size-large wp-image-2489" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/post8_figure1-1024x935.png" alt="" width="780" height="712" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/post8_figure1-1024x935.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/post8_figure1-300x274.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/post8_figure1-768x702.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/post8_figure1.png 1144w" sizes="(max-width: 780px) 100vw, 780px" /></p>
<p><strong><em>Figure 1: Experimental workflow for characterising follow-up compounds by x-ray crystallography. </em></strong><em>Follow-up compounds from fragments 1, 2 and 5 are described <a href="https://thesgc.github.io/static-openlabnotebooks/screening-follow-up-compounds-from-active-site-and-gating-loop-hao1-fragment-hits/">here</a>, whereas those from fragment 6 are described <a href="https://thesgc.github.io/static-openlabnotebooks/synthesis-and-testing-of-follow-up-compounds-based-on-hao1-fragment-6-bound-at-the-tetramer-interface/">here</a>. Completed experiments are in light blue boxes and ongoing experiment is in dark blue.</em></p>
<p>I have collected a total of 227 good datasets (resolution range of 1.2 to 2.2 Å, 90% better than 1.7 Å) and now I am hard at work processing these through the <a href="http://journals.iucr.org/d/issues/2017/03/00/ba5267/">XChemExplorer pipeline</a>, ready to tell you all about the compound-bound structures in the next few weeks.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/collecting-hao1-crystal-structures-with-follow-up-compounds-based-on-fragment-hits/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
	</channel>
</rss>
