<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>E3 ligases &#8211; openlabnotebooks.org</title>
	<atom:link href="https://thesgc.github.io/static-openlabnotebooks/category/project/e3-ligases/feed/" rel="self" type="application/rss+xml" />
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<description>A growing team of groundbreaking scientists around the world are now sharing their lab notebooks online</description>
	<lastBuildDate>Mon, 26 Oct 2020 19:00:59 +0000</lastBuildDate>
	<language>en-GB</language>
	<sy:updatePeriod>
	hourly	</sy:updatePeriod>
	<sy:updateFrequency>
	1	</sy:updateFrequency>
	<generator>https://wordpress.org/?v=5.5.3</generator>

<image>
	<url>https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/cropped-lab-jar02-32x32.png</url>
	<title>E3 ligases &#8211; openlabnotebooks.org</title>
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<width>32</width>
	<height>32</height>
</image> 
	<item>
		<title>FBXW11 interaction with β-catenin &#8211; cellular assay</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/fbxw11-interaction-with-%ce%b2-catenin-cellular-assay/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/fbxw11-interaction-with-%ce%b2-catenin-cellular-assay/#respond</comments>
		
		<dc:creator><![CDATA[Magdalena Szewczyk]]></dc:creator>
		<pubDate>Mon, 26 Oct 2020 17:05:14 +0000</pubDate>
				<category><![CDATA[Cell assay reports]]></category>
		<category><![CDATA[E3 ligases]]></category>
		<category><![CDATA[Magdalena Szewczyk]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=5309</guid>

					<description><![CDATA[Background The SCF (SKP1-CUL1-F-box protein) E3 ubiquitin ligase is a complex of proteins that helps facilitate the ubiquitination of substrates for targeted proteasomal degradation. FBXW11 is F-box containing protein that has 11 WD-repeats and is a substrate recognition component of the SCF complex. FBXW11 recognizes and binds its substrates, including beta-catenin, at phosphorylated sites or <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/fbxw11-interaction-with-%ce%b2-catenin-cellular-assay/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p><strong>Background</strong></p>
<p>The SCF (SKP1-CUL1-F-box protein) E3 ubiquitin ligase is a complex of proteins that helps facilitate the ubiquitination of substrates for targeted proteasomal degradation. FBXW11 is F-box containing protein that has 11 WD-repeats and is a substrate recognition component of the SCF complex. FBXW11 recognizes and binds its substrates, including beta-catenin, at phosphorylated sites or phosphodegrons, and recruits it to the SCF complex for subsequent ubiquitination and degradation (PMID: 10321728)</p>
<p><strong>Assay validation</strong></p>
<p>Assay measuring the interaction of FBXW11 with B-catenin was developed using NanoBRET technology &#8211; a proximity-based assay that can detect protein interactions by measuring energy transfer from a bioluminescent protein donor (NanoLuc luciferase, NL) to a fluorescent protein acceptor (fluorescent ligand bound to HaloTag, HT). In the beginning, we determined the optimal tag orientation of donor constructs: N or C-terminally NL-tagged β-catenin and acceptor constructs: N or C-terminally HT-tagged FBXW11 in HEK293T cells. NL and HT proteins, expressed alone, were used as controls to determine background signal. The best results were obtained with C-terminally NL-tagged B-catenin and C-terminally HT-tagged FBXW11 (Fig.1). The assay was further validated with binding deficient FBXW11 mutant (R447E) designed by Matthieu Schapira. R447E mutation resulted in a significant decrease in the interaction of FBXW11 with B-catenin (Fig.2).</p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/f11b-300x134.png" alt="" width="582" height="260" class=" wp-image-5311 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/f11b-300x134.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/f11b-768x343.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/f11b.png 993w" sizes="(max-width: 582px) 100vw, 582px" /></p>
<p><strong>Fig.1. FBXW11 interaction with B-catenin &#8211; NanoBRET assay. </strong>HEK293T cells were co-transfected with C- or N-terminally NanoLuc (NL)-tagged B-catenin or NL alone and C- or N-terminally Halo-tagged (HT) FBXW11 or HT alone for 24 h. The interaction was measured using NanoBRET assay. The results are MEAN of 2 technical replicates.</p>
<p><strong> <img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/fbxw11-300x168.png" alt="" width="491" height="275" class="wp-image-5310 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/fbxw11-300x168.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/fbxw11-1024x572.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/fbxw11-768x429.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/fbxw11.png 1063w" sizes="(max-width: 491px) 100vw, 491px" /></strong></p>
<p><strong>Fig.2. </strong><strong>R447E mutation decreases the interaction of FBXW11 with B-catenin.</strong> HEK293T cells were co-transfected with C-terminally Halo-tagged (HT) FBXW11 or HT and C-terminally NanoLuc (NL)-tagged B-catenin for 24 h. The interaction was measured using NanoBRET assay. The results are MEAN of 2 technical replicates.</p>
<p>Please go to <a href="https://zenodo.org/record/4134923#.X5ccxohKg2w">Zenodo</a> for experimental details.</p>
<p><strong>Acknowledgments:</strong></p>
<p>Matthieu Schapira &#8211; mutant design, Aidan Levinsky &#8211; help with mutagenesis</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/fbxw11-interaction-with-%ce%b2-catenin-cellular-assay/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>FBXW7 interaction with Cyclin E1 &#8211; cellular assay</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/fbxw7-interaction-with-cyclin-e1-cellular-assay/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/fbxw7-interaction-with-cyclin-e1-cellular-assay/#respond</comments>
		
		<dc:creator><![CDATA[Magdalena Szewczyk]]></dc:creator>
		<pubDate>Mon, 26 Oct 2020 16:42:54 +0000</pubDate>
				<category><![CDATA[Cell assay reports]]></category>
		<category><![CDATA[E3 ligases]]></category>
		<category><![CDATA[Magdalena Szewczyk]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=5302</guid>

					<description><![CDATA[Background The SCF (SKP1-CUL1-F-box protein) E3 ubiquitin ligase is a complex of proteins that helps facilitate the ubiquitination of substrates for targeted proteasomal degradation. FBXW7 (F-box/WD repeat-containing protein 7) is an F-box protein component of the SCF complex and is involved in substrate recognition. The F-box domain tethers FBXW7 to the SCF complex by interacting <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/fbxw7-interaction-with-cyclin-e1-cellular-assay/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p><strong>Background</strong></p>
<p>The SCF (SKP1-CUL1-F-box protein) E3 ubiquitin ligase is a complex of proteins that helps facilitate the ubiquitination of substrates for targeted proteasomal degradation. FBXW7 (F-box/WD repeat-containing protein 7) is an F-box protein component of the SCF complex and is involved in substrate recognition. The F-box domain tethers FBXW7 to the SCF complex by interacting with the SKP1 component while the WD40 repeats are involved in substrate binding. FBXW7 recognizes and binds its substrates at phosphorylated sites or phosphodegrons and recruits them to the SCF complex for subsequent ubiquitination and degradation (PMID:17434132). A variety of substrates exist for FBXW7, many of which are potentially oncogenic, including cyclin E1, MYC, JUN, and NOTCH (PMID: 11565034, 15103331, 14739463, 29149593). The targeted degradation of growth-promoting factors by FBXW7 suggests it likely plays an important role as a tumor suppressor. FBXW7 Arg→Cys mutation at position 465 hinders its ability to interact with SKP1 and thus the SCF complex; this mutation is associated with acute lymphoblastic leukemia (PMID: 11565033, 28007894).</p>
<p><strong>Assay validation</strong></p>
<p>Assay measuring the interaction of FBXW7 with cyclin E1 was developed using NanoBRET technology &#8211; a proximity-based assay that can detect protein interactions by measuring energy transfer from a bioluminescent protein donor (NanoLuc, NL) to a fluorescent protein acceptor (fluorescent ligand bound to HaloTag, HT). In the beginning, we determined the optimal tag orientation of donor constructs: N or C-terminally NL-tagged cyclin E1 and acceptor constructs: N or C-terminally HT- tagged FBXW7 in HEK293T cells. NanoLuc and halo tag proteins, expressed alone, were used as controls to determine background signal. The best results were obtained with C-terminally NL-tagged cyclin E1 and C-terminally HT-tagged FBXW7 (Fig.1). The assay was further validated with a binding deficient FBXW7 mutant (R465E, R505E) designed by Matthieu Schapira. R465E, R505E double mutation resulted in a significant decrease in the interaction of FBXW7 with cyclin E1 (Fig.2).</p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/fbx711-300x132.jpg" alt="" width="598" height="263" class=" wp-image-5307 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/fbx711-300x132.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/fbx711-1024x451.jpg 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/fbx711-768x338.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/fbx711.jpg 1280w" sizes="(max-width: 598px) 100vw, 598px" /></p>
<p><strong>Fig.1. FBXW7 interaction with cyclin E1 &#8211; NanoBRET assay. </strong>HEK293T cells were co-transfected with C- or N-terminally NanoLuc (NL)-tagged cyclin E1 or NL alone and C- or N-terminally Halo-tagged (HT) FBXW7 or HT alone for 24 h. The interaction was measured using NanoBRET assay. The results are MEAN of 2 technical replicates.</p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/fbx7a-300x119.jpg" alt="" width="635" height="252" class=" wp-image-5306 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/fbx7a-300x119.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/fbx7a-1024x405.jpg 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/fbx7a-768x304.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/fbx7a.jpg 1145w" sizes="(max-width: 635px) 100vw, 635px" /></p>
<p><strong> </strong><strong>Fig.2. R465E, R505E double</strong><strong> mutation decreases the interaction of FBXW7 with cyclin E1.</strong> HEK293T cells were co-transfected with C-terminally Halo-tagged (HT) FBXW7 (WT or R465E/R505E or HT and C-terminally NanoLuc (NL)-tagged cyclin E1 for 24 h. The interaction was measured using NanoBRET assay. The results are MEAN of 2 technical replicates.</p>
<p>Please go to<a href="https://zenodo.org/record/4134887#.X5b8RohKg2w"> Zenodo</a> for experimental details.</p>
<p><strong>Acknowledgments:</strong></p>
<p>Matthieu Schapira &#8211; mutant design, Aidan Levinsky &#8211; help with mutagenesis</p>
<p>&nbsp;</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/fbxw7-interaction-with-cyclin-e1-cellular-assay/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>STIP1(HOP) and STUB1(CHIP) interaction with HSPA8 &#8211; cellular assay</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/stip1hop-and-stub1chip-interaction-with-hspa8-cellular-assay/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/stip1hop-and-stub1chip-interaction-with-hspa8-cellular-assay/#respond</comments>
		
		<dc:creator><![CDATA[Magdalena Szewczyk]]></dc:creator>
		<pubDate>Fri, 05 Jun 2020 19:02:55 +0000</pubDate>
				<category><![CDATA[Cell assay reports]]></category>
		<category><![CDATA[E3 ligases]]></category>
		<category><![CDATA[Magdalena Szewczyk]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=4866</guid>

					<description><![CDATA[Background HSP70 heat-shock proteins family, including constitutively expressed HSPA8 variant, facilitate correct folding of client proteins and degradation of unfolded clients. To achieve folding or degradation of client proteins, HSP70 cooperates with co-chaperones, which mostly bind to the highly conserved sequence EEVD in the C-terminal domain of HSP70 via their tetratricopeptide repeat (TPR) domain (PMID: <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/stip1hop-and-stub1chip-interaction-with-hspa8-cellular-assay/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p><strong>Background</strong></p>
<p>HSP70 heat-shock proteins family, including constitutively expressed HSPA8 variant, facilitate correct folding of client proteins and degradation of unfolded clients. To achieve folding or degradation of client proteins, HSP70 cooperates with co-chaperones, which mostly bind to the highly conserved sequence EEVD in the C-terminal domain of HSP70 via their tetratricopeptide repeat (TPR) domain (PMID: 26565746). STIP1(HOP) and ubiquitin ligase STUB1(CHIP) and are TPR-containing co-chaperones that assist with protein folding or degradation, respectively (PIMID:22824801). It was shown previously that mutations within TPR domains of STIP1 (K8A) and STUB1 (K30A) abolished their interaction with HSPA8 (PIMID: 23865999, 26565746).</p>
<p><strong>Assay validation</strong></p>
<p>Assay measuring the interaction of HSPA8 to STIP1 and STUB1 co-chaperons was developed using NanoBRET technology &#8211; a proximity-based assay that can detect protein interactions by measuring energy transfer from a bioluminescent protein donor to a fluorescent protein acceptor. At the beginning we determined the optimal tag orientation and donor to acceptor ratio of donor constructs: N or C-terminally NanoLuc® tagged STIP1/STIP1 K8A mutant or STUB1/STUB1 K30A and acceptor construct: C-terminally HaloTag®Fusion tagged HSPA8. The best results were obtained with C-terminally tagged donor constructs and 1:9 donor to acceptor ratio. Consistent with previous findings the interaction (NanoBRET ratio) of HSPA8 and mutated co-chaperons was significantly lower compared to wild type proteins (Fig.1).</p>
<p><strong> <img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/stipstub-300x170.jpg" alt="" width="671" height="380" class="wp-image-4867 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/stipstub-300x170.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/stipstub-768x434.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/stipstub.jpg 945w" sizes="(max-width: 671px) 100vw, 671px" /></strong></p>
<p><strong>Fig.1. HSPA8 interaction with STIP1 (HOP) and STUB1 (CHIP) co-chaperones. </strong>HEK293T cells were co-transfected with Nanoluc (NL)-tagged STIP1 (WT or K8A mutant) or NL-tagged STUB1 (WT or K30A mutant) and Halo-tagged (HT) HSPA8 for 24 h. The interaction was measured using NanoBRET assay. The results are MEAN+/-SD of 3 technical replicates.</p>
<p>Experimental details are posted on <a href="https://zenodo.org/record/3879514#.XtqWgzpKg2w">Zenodo</a>.</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/stip1hop-and-stub1chip-interaction-with-hspa8-cellular-assay/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>UBE3A Wnt signaling Cell Assay</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/ube3a-wnt-signaling-cell-assay/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/ube3a-wnt-signaling-cell-assay/#respond</comments>
		
		<dc:creator><![CDATA[Victoria Vu]]></dc:creator>
		<pubDate>Mon, 01 Jun 2020 21:14:49 +0000</pubDate>
				<category><![CDATA[E3 ligases]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<category><![CDATA[Victoria Vu]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=4826</guid>

					<description><![CDATA[Background Apart from my HTT protein project, I also work on UBE3A, a HECT domain E3 ligase. E3 ligases work as a part of the ubiquitination system interacting with E2 ubiquitin-conjugating enzymes to transfer ubiquitin onto target substrate proteins. Ubiquitination of substrate proteins regulate a diversity of cell processes and is also important for ubiquitin-mediated <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/ube3a-wnt-signaling-cell-assay/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p><strong><em><u>Background</u></em></strong></p>
<p>Apart from my HTT protein project, I also work on UBE3A, a HECT domain E3 ligase. E3 ligases work as a part of the ubiquitination system interacting with E2 ubiquitin-conjugating enzymes to transfer ubiquitin onto target substrate proteins. Ubiquitination of substrate proteins regulate a diversity of cell processes and is also important for ubiquitin-mediated proteosomal degradation of proteins.</p>
<p>UBE3A reported substrates include the tumor suppressor p53 (<a href="https://pubmed.ncbi.nlm.nih.gov/8221889/">PMID8221889</a>), the PDZ domain-containing protein scribble (<a href="https://pubmed.ncbi.nlm.nih.gov/11027293/">PMID11027293</a>), and NFX1-91, a transcriptional repressor of the gene encoding hTERT (<a href="https://pubmed.ncbi.nlm.nih.gov/15371341/">PMID15371341</a>). UBE3A was initially identified as the cellular factor that stimulates ubiquitin-dependent degradation of the tumor suppressor p53 driven by the E6 protein of cervical cancer-associated human papillomavirus (HPV) types 16 and 18 (<a href="https://pubmed.ncbi.nlm.nih.gov/8380895/">PMID8380895</a>). UBE3A also functions as a transcriptional coactivator for steroid hormone receptors (<a href="https://pubmed.ncbi.nlm.nih.gov/18432313/">PMID18432313</a>).</p>
<p>I am interested in understanding the role of UBE3A as an E3 ligase in the context of neurological diseases. UBE3A shows brain-specific imprinting and the loss of function of maternally inherited UBE3A has been linked to Angelman syndrome (<a href="https://pubmed.ncbi.nlm.nih.gov/8988171/">PMID8988171</a>). In addition, the autism-linked UBE3A-T485A mutation disrupts phosphorylation control of UBE3A and enhances UBE3A ubiquitin ligase activity (<a href="https://pubmed.ncbi.nlm.nih.gov/28559284/">PMID28559284</a>). This suggest that dysregulation of UBE3A, both downregulation and upregulation, can affect neurological function.</p>
<p>To better understand the cellular function of UBE3A, we can use chemical probes to inhibit it. To identify compounds that affect UBE3A, we can develop reporter assays that detection activation of pathways involving our protein of interest. UBE3A can stimulate Wnt pathway activation and protect β-catenin from proteasomal degradation with these activities requiring the ubiquitin ligase activity of UBE3A (<a href="https://pubmed.ncbi.nlm.nih.gov/27902311/">PMID27902311</a>) (Fig 1). Here, I describe a UBE3A Wnt-signalling reporter cell assay developed in collaboration with Dalia Barsyte-Lovejoy. The methods described in this post can be found on <a href="http://doi.org/10.5281/zenodo.3872219">Zenodo</a>.</p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/zbc0311770290007-936x1024.jpg" alt="" width="266" height="291" class="wp-image-4831 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/zbc0311770290007-936x1024.jpg 936w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/zbc0311770290007-274x300.jpg 274w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/zbc0311770290007-768x840.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/zbc0311770290007.jpg 989w" sizes="(max-width: 266px) 100vw, 266px" /></p>
<p><strong>Figure 1. UBE3A regulation of Wnt pathway (from <a href="https://pubmed.ncbi.nlm.nih.gov/28559284/">PMID28559284</a>).</strong></p>
<p><strong><em><u>Results</u></em></strong></p>
<p>To assess the transcriptional output of Wnt pathway we utilized the TCF response element 8xTOP-Flash-FLuc (Firefly luciferase) reporter that is co-transfected into HEK293 cells together with increasing amounts of UBE3A and Renilla luciferase (Rluc) expressing plasmid that is used for transfection efficiency normalization purposes (Fig 2).</p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/Fig2-1024x183.png" alt="" width="780" height="139" class="aligncenter size-large wp-image-4827" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/Fig2-1024x183.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/Fig2-300x54.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/Fig2-768x138.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/Fig2.png 1189w" sizes="(max-width: 780px) 100vw, 780px" /></p>
<p><strong>Figure 2. Wnt activity reporter assay system schematic. Reporter constructs were previously described in <a href="https://pubmed.ncbi.nlm.nih.gov/12699626/">PMID12699626</a>.</strong></p>
<p>To determine if UBE3A specifically affects the Wnt reporter and is not affecting the control Rluc or the mutant TCF response element reporter Fluc, we assessed the Rluc and Fluc separately and as a normalized ratio (Fig 3). UBE3A drastically upregulated the 8xTOP reporter signal, slightly upregulated Rluc driven by TK promoter and had no effect on mutant FOP reporter. We also use a GSK3 inhibitor to ensure that the 8XTOP-Flash-FLuc reporter responds to Wnt activation. Normally, GSK3 phosphorylates β-catenin triggering its degradation.  Inhibition of GSK3 will stabilize β-catenin, translating to increased Wnt signalling. The 8xTOP Flash reporter is activated by GSK3 inhibition and resulting Wnt activation, but the mutant FOP Flash reporter is not.</p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/Fig3-1024x373.png" alt="" width="780" height="284" class="aligncenter size-large wp-image-4828" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/Fig3-1024x373.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/Fig3-300x109.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/Fig3-768x280.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/Fig3.png 1445w" sizes="(max-width: 780px) 100vw, 780px" /></p>
<p><strong>Figure 3. UBE3A reporter assay specificity testing. Left panel represents data on baseline reporter activity, while right graph is data obtained upon the activation of wnt pathway by inhibiting GSK3. </strong>(Dalia Barsyte-Lovejoy)</p>
<p>Apart from testing the specificity of the assay, we also want to show that the system is it is responsive to varying levels of Wnt activation. We find that increasing amounts of three different isoforms of UBE3A resulted in Wnt reporter activation in a dose-response manner (Fig 4). UBE3A isoform III gave the highest relative activity Fluc/Rluc (i.e. lowest signal:background ratio) which makes it the most suitable for chemical probe screening. We will use this isoform moving forward.</p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/Fig4-1024x436.png" alt="" width="780" height="332" class="aligncenter size-large wp-image-4829" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/Fig4-1024x436.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/Fig4-300x128.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/Fig4-768x327.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/Fig4.png 1396w" sizes="(max-width: 780px) 100vw, 780px" /></p>
<p><strong>Figure 4. Wnt reporter activity in response to increasing UBE3A. Left panel represents data on baseline reporter activity, while the right graph is data obtained upon the activation of Wnt pathway by inhibiting GSK3. </strong>(Dalia Barsyte-Lovejoy)</p>
<p>To further validate the assay, we test whether the Wnt activation is dependent on the catalytic activity of UBE3A. Using site-directed mutagenesis, I made the UBE3A-C833A catalytically inactive mutant (<a href="https://pubmed.ncbi.nlm.nih.gov/9497376/">PMID9497376</a>) where the alanine substitution of the active site cysteine obliterates UBE3A’s ability of forming a thiolester bond with ubiquitin. The wildtype UBE3A protein showed Wnt reporter activation, while the catalytic cysteine C833 mutant UBE3A did not (Fig 5).</p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/Fig5.png" alt="" width="362" height="332" class="aligncenter size-full wp-image-4830" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/Fig5.png 362w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/Fig5-300x275.png 300w" sizes="(max-width: 362px) 100vw, 362px" /></p>
<p><strong>Figure 5. UBE3A driven Wnt activation is dependent on its catalytic activity. </strong>(Victoria Vu)</p>
<p>Now that I have a Wnt activation cell assay that is specific, dose-responsive and depends on the catalytic activity of UBE3A, I can screen for compounds that may inhibit UBE3A catalytic activity and use it to better understand the function of UBE3A in neurological disorders.</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/ube3a-wnt-signaling-cell-assay/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>RPA2 and RFWD3 interaction cellular assay</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/rpa2-and-rfwd3-interaction-cellular-assay/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/rpa2-and-rfwd3-interaction-cellular-assay/#respond</comments>
		
		<dc:creator><![CDATA[Magdalena Szewczyk]]></dc:creator>
		<pubDate>Thu, 21 May 2020 16:00:40 +0000</pubDate>
				<category><![CDATA[Cell assay reports]]></category>
		<category><![CDATA[E3 ligases]]></category>
		<category><![CDATA[Magdalena Szewczyk]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=4771</guid>

					<description><![CDATA[Background RFWD3 (E3 ligase RING finger and WD repeat domain-containing protein 3) and RPA2 (Replication protein A2) were shown to functionally interact and participate in replication checkpoint control. At stalled replication forks, the RPA complex binds single-stranded DNA to prevent nucleolytic digestions and also to initiate replication checkpoint signaling by recruitment checkpoint proteins. RFWD3 is <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/rpa2-and-rfwd3-interaction-cellular-assay/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p><strong>Background</strong></p>
<p>RFWD3 (E3 ligase RING finger and WD repeat domain-containing protein 3) and RPA2 (Replication protein A2) were shown to functionally interact and participate in replication checkpoint control. At stalled replication forks, the RPA complex binds single-stranded DNA to prevent nucleolytic digestions and also to initiate replication checkpoint signaling by recruitment checkpoint proteins. RFWD3 is recruited to stalled replication forks and interacts directly with RPA2 in response to replication stress. The deletion of the RPA2 binding region on RFWD3 impairs its localization to stalled replication forks (PIMID: 21504906).</p>
<p><strong>Summary</strong></p>
<p>Assay measuring the interaction of RPA2 to RFWD3 was developed using NanoBRET technology &#8211; a proximity-based assay that can detect protein interactions by measuring energy transfer from a bioluminescent protein donor to a fluorescent protein acceptor. We determined the optimal donor to acceptor ratio and levels of expression of 2 donor constructs: C- or N-terminally NanoLuc®(NL)- tagged RFWD3 and acceptor construct: N-terminally HaloTag®Fusion (HT)-tagged RPA2. We compared the results to background NanoBRET signal from HT-RPA2 and NL protein alone. The best results were obtained with N-terminally NL-tagged RFWD3 (0.001 µg/96-well) and N-terminally HT- tagged RPA2 (0.03 µg/96-well) (Fig.1)</p>
<p><strong> </strong></p>
<p><strong> <img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/rpa2-300x152.png" alt="" width="513" height="260" class="wp-image-4772 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/rpa2-300x152.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/rpa2-768x390.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/rpa2.png 962w" sizes="(max-width: 513px) 100vw, 513px" /></strong></p>
<p><strong>Fig 1. RPA2 – RFWD3</strong><strong> nanoBRET assay. </strong>HEK293T cells were plated in 96-well plates and transfected with indicated amounts of C- or N- terminally NL-tagged RFWD3 and N-terminally HT-tagged RPA2. Next day cells were treated +/- fluorescent ligand and 4 h later the luciferase substrate was added, and the signal was read. The results are mean +/- SD of 3 replicates.</p>
<p>Go to <a href="https://zenodo.org/record/3837867#.Xsak7WhKg2w">Zenodo</a> for experimental details.</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/rpa2-and-rfwd3-interaction-cellular-assay/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>GID4 (human) cellular assay</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/gid4-human-cellular-assay/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/gid4-human-cellular-assay/#respond</comments>
		
		<dc:creator><![CDATA[Magdalena Szewczyk]]></dc:creator>
		<pubDate>Wed, 06 May 2020 17:21:48 +0000</pubDate>
				<category><![CDATA[Cell assay reports]]></category>
		<category><![CDATA[E3 ligases]]></category>
		<category><![CDATA[Magdalena Szewczyk]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=4651</guid>

					<description><![CDATA[Background GID4 is the substrate-recognition subunit of the CTLH E3 ubiquitin-protein ligase complex which has been shown to possess E3-ligase enzymatic activity (PIMID:31885576). GID4 was shown to bind proteins and peptides with a Pro/N-degron consisting of an unmodified N-terminal Pro and has the highest affinity for the peptide Pro-Gly-Leu-Trp (PIMID:29632410). In yeast Gid4 specifically targets <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/gid4-human-cellular-assay/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p><strong>Background</strong></p>
<p>GID4 is the substrate-recognition subunit of the CTLH E3 ubiquitin-protein ligase complex which has been shown to possess E3-ligase enzymatic activity (PIMID:<a href="https://www.ncbi.nlm.nih.gov/pubmed/31885576">31885576</a>). GID4 was shown to bind proteins and peptides with a Pro/N-degron consisting of an unmodified N-terminal Pro and has the highest affinity for the peptide Pro-Gly-Leu-Trp (PIMID:29632410). In yeast Gid4 specifically targets the gluconeogenic enzymes (Fbp1, Icl1, and Mdh2) for proteasome-mediated degradation (PIMID: 31863529), however, substrate specificity for human GID4 has not been fully understood.</p>
<p><strong>Assay validation</strong></p>
<p>Cell-based thermal shift assays (CETSA) provide a native environment for measuring compounds’ target engagement and cell permeability. Standard western blot-based CETSA protocols are laborious, require high-quality antibodies to the target, and are generally not amenable to high-throughput compound screening. Therefore, we have adopted the HiBiT technology-based CETSA, which measures the thermal stability of a NanoLuc Luciferase fusion protein, to measure GID4 cellular target engagement in a 384-well plate format (Fig.1). The assay was validated with two compounds: 20964 and 12966, which showed significant dose-dependent stabilization of GID4 when tested <em>in vitro</em> by DSF. Using both compounds we observed reproducible stabilization of GID4 (Fig.2A) in cells. Since compound 20964 gave better T<sub>m</sub>-shift (∆T<sub>m</sub>), we tested if the stabilization is dose-dependent in a temperature range which gave the biggest difference between treated and untreated samples.  Compound 20964 showed the dose-dependent stabilization of GID4 (Fig.2B). The effect was significant at 1 µM concentration (Fig.2C). These results validate the utility of the HiBiT based CETSA assay to measure cellular target engagement of GID4.</p>
<p><strong> <img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/gid4a-300x144.png" alt="" width="583" height="280" class="wp-image-4652 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/gid4a-300x144.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/gid4a-768x369.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/gid4a.png 788w" sizes="(max-width: 583px) 100vw, 583px" /></strong></p>
<p><strong>Fig.1. HiBIT protein tagging system</strong>. HiBiT is an 11 amino acid peptide tag that can be attached to the N or C terminus of the protein. The tag is detected with a reagent containing an inactive luciferase subunit, Large Bit (LgBiT), which rapidly binds to HiBiT to produce a functional NanoLuc luciferase enzyme that relies on the substrate furimazine to produce high intensity, glow-type luminescence.</p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/gid4new-300x169.png" alt="" width="646" height="364" class="wp-image-4653 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/gid4new-300x169.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/gid4new-1024x576.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/gid4new-768x432.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/gid4new.png 1280w" sizes="(max-width: 646px) 100vw, 646px" /></p>
<p><strong>Fig.2. GID4 HiBiT CETSA target engagement assay validation. </strong>HeLa cells were transfected with N-terminally HiBiT-tagged GID4. Cells were treated with compounds and DMSO control for an hour before heating. After heating, cells were lysed and LgBiT and Luciferase substrate were added and the signal was measured. <strong>(A) </strong>Effect of 20964 (10 µM) and 12966 (30 µM) on the thermal stability of HiBiT-tagged GID4 relative to DMSO control. <strong>(B)</strong> 20964 showed the dose-dependent stabilization of GID4 in an indicated temperature range. <strong>(C)</strong> 20964 significantly stabilized GID4 at 1 µM in an indicated temperature range. The results are mean ± SEM of 4 technical replicates.</p>
<p>Experimental details can be found on <a href="https://zenodo.org/record/3803314#.XrLxcahKg2w">Zenodo</a>.</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/gid4-human-cellular-assay/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>PRMT4 (CARM1) cellular assay</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/prmt4-carm1-cellular-assay/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/prmt4-carm1-cellular-assay/#respond</comments>
		
		<dc:creator><![CDATA[Magdalena Szewczyk]]></dc:creator>
		<pubDate>Mon, 04 May 2020 16:08:06 +0000</pubDate>
				<category><![CDATA[Cell assay reports]]></category>
		<category><![CDATA[E3 ligases]]></category>
		<category><![CDATA[Magdalena Szewczyk]]></category>
		<category><![CDATA[PRMTs]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=4630</guid>

					<description><![CDATA[Background PRMT4 (CARM1) is type I protein arginine methyltransferase which mono- or asymmetrically dimethylates proteins (Fig.1) within proline-methionine rich motives, that are involved in transcription, splicing and RNA processing such as H3R17, EP300, PABP1, CA150, BAF155, MED12 (PIMID:19150423). Fig.1. Mammalian PRMTs. There are 9 members and 3 types of the PRMT family. Type I, II <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/prmt4-carm1-cellular-assay/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p><strong>Background</strong></p>
<p>PRMT4 (CARM1) is type I protein arginine methyltransferase which mono- or asymmetrically dimethylates proteins (Fig.1) within proline-methionine rich motives, that are involved in transcription, splicing and RNA processing such as H3R17, EP300, PABP1, CA150, BAF155, MED12 (PIMID:19150423).</p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1-300x119.png" alt="" width="527" height="209" class="wp-image-4626 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1-300x119.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1-1024x405.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1-768x304.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1.png 1214w" sizes="(max-width: 527px) 100vw, 527px" /></p>
<p><strong>Fig.1. </strong><strong>Mammalian PRMTs. </strong>There are 9 members and 3 types of the PRMT family. Type I, II and III PRMTs catalyze the formation of monomethyl arginines (Rme1). Type I PRMTs catalyze asymmetric arginine methylation (Rme2a), while type II PRMTs form symmetric arginine methylation (Rme2s). Type III PRMT can only monomethylate arginine residues. Modified from (PMID:29378138).</p>
<p><strong>Assay validation</strong></p>
<p>It has been reported that the RNA polymerase II mediator complex subunit 12 (MED12) and BAF155, are PRMT4 substrates (PIMID: 26601288, 25927994). Consistently with previous findings, PRMT4 knock-down decreased asymmetric dimethylation of MED12 and BAF155 (Fig.2). The assay was validated with PRMT4 selective chemical probe TP-064 (PIMID: 29719619). TP-064 decreased PRMT4 asymmetric dimethylation of MED12 (Z factor = 0.6) and BAF155 (Z factor = 0.76) in a dose-dependent manner (Fig.3).</p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p42-300x245.png" alt="" width="417" height="340" class="alignnone wp-image-4631" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p42-300x245.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p42-768x627.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p42.png 846w" sizes="(max-width: 417px) 100vw, 417px" /></p>
<p><strong>Fig.2. PRMT4 knock-down resulted in a decrease of MED12 and BAF155 asymmetric dimethylation (Rme2a) levels. </strong>HEK293T cells were treated with siRNA for 3 days and 6 days. The levels of methylation were analyzed in Western Blot.<strong> </strong></p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p4-3-300x120.png" alt="" width="623" height="249" class="alignnone wp-image-4632" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p4-3-300x120.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p4-3-1024x410.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p4-3-768x308.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p4-3.png 1261w" sizes="(max-width: 623px) 100vw, 623px" /></p>
<p><strong>Fig. 3.</strong> <strong>TP-064 </strong><strong>decreases PRMT4 dependent MED12 and BAF155 asymmetric dimethylation (Rme2a) levels.</strong> HEK293T cells were treated with PRMT4 selective inhibitor (TP-064) at indicated concentrations for 48 h and BAF155-Rme2a (A) and MED12-Rme2a (B) levels were determined by Western blot. The graphs represent nonlinear fits of methyl signal intensities normalized to total protein levels. The results are mean +/- SEM of 3 replicates.</p>
<p>Please go to <a href="https://zenodo.org/record/3785034#.XrA9AKhKg2w">Zenodo</a> for experimental details.</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/prmt4-carm1-cellular-assay/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
	</channel>
</rss>
