<!DOCTYPE html>
<html lang="en-GB"><head><meta charset="UTF-8"><meta name="viewport" content="width=device-width, initial-scale=1, minimum-scale=1"><title>Interrogating PR/SET domains for the ability to bind SAH – openlabnotebooks.org</title><link rel="dns-prefetch" href="//www.google.com"><link rel="dns-prefetch" href="//ajax.googleapis.com"><link rel="dns-prefetch" href="//s.w.org"><link rel="stylesheet" id="pt-cv-public-style-css" href="https://thesgc.github.io/static-openlabnotebooks/wp-content/plugins/content-views-query-and-display-post-page/public/assets/css/cv.css" type="text/css" media="all"><link rel="stylesheet" id="wp-block-library-css" href="https://thesgc.github.io/static-openlabnotebooks/wp-includes/css/dist/block-library/style.min.css" type="text/css" media="all"><link rel="stylesheet" id="contact-form-7-css" href="https://thesgc.github.io/static-openlabnotebooks/wp-content/plugins/contact-form-7/includes/css/styles.css" type="text/css" media="all"><link rel="stylesheet" id="email-subscribers-css" href="https://thesgc.github.io/static-openlabnotebooks/wp-content/plugins/email-subscribers/lite/public/css/email-subscribers-public.css" type="text/css" media="all"><link rel="stylesheet" id="custom-css-css" href="https://thesgc.github.io/static-openlabnotebooks/wp-content/plugins/sgc-chem-structures/css/sgc-chem-style.css" type="text/css" media="all"><link rel="stylesheet" id="less-style-css" href="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/wp-less/clean-journal-pro-child/less/custom-410e72baa3.css" type="text/css" media="all"><link rel="stylesheet" id="cookie-consent-style-css" href="https://thesgc.github.io/static-openlabnotebooks/wp-content/plugins/uk-cookie-consent/assets/css/style.css" type="text/css" media="all"><link rel="stylesheet" id="clean-journal-pro-style-css" href="https://thesgc.github.io/static-openlabnotebooks/wp-content/themes/clean-journal-pro/style.css" type="text/css" media="all"><link rel="stylesheet" id="child-style-css" href="https://thesgc.github.io/static-openlabnotebooks/wp-content/themes/clean-journal-pro-child/style.css" type="text/css" media="all"><link rel="stylesheet" id="jquery-ui-css-css" href="https://ajax.googleapis.com/ajax/libs/jqueryui/1.12.1/themes/smoothness/jquery-ui.css?ver=1.12.1" type="text/css" media=""><link rel="stylesheet" id="clean-journal-web-font-css" href="//fonts.googleapis.com/css?family=Source+Sans+Pro:300,300italic,regular,italic,600,600italic%7cMontserrat:300,300italic,regular,italic,600,600italic%7cMerriweather:300,300italic,regular,italic,600,600italic&subset=latin" type="text/css" media="all"><link rel="stylesheet" id="clean-journal-style-css" href="https://thesgc.github.io/static-openlabnotebooks/wp-content/themes/clean-journal-pro-child/style.css" type="text/css" media="all"><link rel="stylesheet" id="genericons-css" href="https://thesgc.github.io/static-openlabnotebooks/wp-content/themes/clean-journal-pro/css/genericons/genericons.css" type="text/css" media="all"><script type="text/javascript" src="https://thesgc.github.io/static-openlabnotebooks/wp-includes/js/jquery/jquery.js" id="jquery-core-js"></script><script type="text/javascript" id="email-subscribers-js-extra">
/* <![CDATA[ */
var es_data = {"messages":{"es_empty_email_notice":"Please enter email address","es_rate_limit_notice":"You need to wait for sometime before subscribing again","es_single_optin_success_message":"Successfully Subscribed.","es_email_exists_notice":"Email Address already exists!","es_unexpected_error_notice":"Oops.. Unexpected error occurred.","es_invalid_email_notice":"Invalid email address","es_try_later_notice":"Please try after some time"},"es_ajax_url":"https:\/\/thesgc.github.io\/static-openlabnotebooks\/wp-admin\/admin-ajax.php"};
/* ]]> */
</script><script type="text/javascript" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/plugins/email-subscribers/lite/public/js/email-subscribers-public.js" id="email-subscribers-js"></script><script type="text/javascript" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/plugins/sgc-chem-structures/js/sgc-chem-scripts.js" id="custom-js-js"></script><script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jqueryui/1.12.1/jquery-ui.min.js?ver=1.12.1" id="jquery-ui-js"></script><script type="text/javascript" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/themes/clean-journal-pro-child/js/scripts.js" id="custom-script-js"></script><script type="text/javascript" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/themes/clean-journal-pro/js/jquery.sidr.min.js" id="sidr-js"></script><script type="text/javascript" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/themes/clean-journal-pro/js/clean-journal-custom-scripts.min.js" id="clean-journal-custom-scripts-js"></script><script type="text/javascript">
			var _gaq = _gaq || [];
			_gaq.push(['_setAccount', 'UA-110085046-1']);
			_gaq.push(['_trackPageview']);
			(function() {
				var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
				ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
				var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
			})();
		</script><style id="ctcc-css" type="text/css" media="screen">
				#catapult-cookie-bar {
					box-sizing: border-box;
					max-height: 0;
					opacity: 0;
					z-index: 99999;
					overflow: hidden;
					color: #ddd;
					position: fixed;
					left: 0;
					bottom: 0;
					width: 100%;
					background-color: #464646;
				}
				#catapult-cookie-bar a {
					color: #fff;
				}
				#catapult-cookie-bar .x_close span {
					background-color: ;
				}
				button#catapultCookie {
					background:;
					color: ;
					border: 0; padding: 6px 9px; border-radius: 3px;
				}
				#catapult-cookie-bar h3 {
					color: #ddd;
				}
				.has-cookie-bar #catapult-cookie-bar {
					opacity: 1;
					max-height: 999px;
					min-height: 30px;
				}</style><style type="text/css">.recentcomments a{display:inline !important;padding:0 !important;margin:0 !important;}</style><link rel="icon" href="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/cropped-lab-jar02-32x32.png" sizes="32x32"><link rel="icon" href="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/cropped-lab-jar02-192x192.png" sizes="192x192"><link rel="apple-touch-icon" href="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/cropped-lab-jar02-180x180.png"><meta name="msapplication-TileImage" content="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/cropped-lab-jar02-270x270.png"><style type="text/css" media="screen">
#site-generator { background-color: #f5f5f5; color: #404040; }
#site-generator a { color: #1b8be0; }
body, button, input, select, textarea { font-family: "Source Sans Pro", sans-serif; }
</style></head><body class="post-template-default single single-post postid-5380 single-format-link wp-custom-logo category-155 group-blog two-columns content-left excerpt-image-left mobile-menu-one">
	<div id="myModal" class="modal">

		<div class="modal-content">
			<span class="title"><h2>Structure search</h2></span>
			<span class="close">×</span>
			<iframe src="" id="ketcher-frame"></iframe>
			
			<div class="search-options">
				<a id="search-ok">SEARCH</a>
				<a id="search-error">SEARCH</a>
				
				<div class="search-option">
					<label for="threshold">Similarity threshold:</label>
					<input type="number" id="threshold" step="0.1" min="0.1" max="1" disabled></div>
				<div class="search-option">
					<label for="select-type">Search type:</label>
						<select id="select-type"><option value="substructure">Substructure</option><option value="similarity">Similarity</option></select></div>
			
			</div>
			<div id="search-results">
				<span id="loading"></span>
			</div>
		</div>

	</div>
			<div id="page" class="hfeed site">
		    <div id="header-top" class="header-top-bar">
        <div class="wrapper">
                            <div class="header-top-right">
                    <section class="widget widget_clean_journal_social_icons" id="header-right-social-icons"><div class="widget-wrap">
                            <a class="genericon_parent genericon genericon-twitter" target="_blank" title="Twitter" href="https://twitter.com/SGClabNotebooks"><span class="screen-reader-text">Twitter</span> </a><a id="custom-icon-1" class="custom-icon" target="_blank" title="Subscribe" href="/subscription-form">
					<img alt="Subscribe" class="icon-static" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/02/mail-icon.png"></a><a id="custom-icon-2" class="custom-icon" target="_blank" title="Download" href="https://thesgc.github.io/static-openlabnotebooks/custom-code/">
					<img alt="Download" class="icon-static" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/08/downwards-arrow.png"></a>                        </div>
                    </section></div>
                    </div>
    </div>

		<header id="masthead" role="banner"><div class="wrapper">
		
    <div id="mobile-header-left-menu" class="mobile-menu-anchor primary-menu">
        <a href="#mobile-header-left-nav" id="header-left-menu" class="genericon genericon-menu">
            <span class="mobile-menu-text">Menu</span>
        </a>
    </div>

    <div id="site-branding" class="logo-left">
				<div id="site-logo"><a href="https://thesgc.github.io/static-openlabnotebooks/" class="custom-logo-link" rel="home"><img width="99" height="120" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/04/cropped-SGC_logo.png" class="custom-logo" alt="openlabnotebooks.org"></a></div><div id="site-header"><p class="site-title"><a href="https://thesgc.github.io/static-openlabnotebooks/">openlabnotebooks.org</a></p><p class="site-description">A growing team of groundbreaking scientists around the world are now sharing their lab notebooks online</p>
		</div></div>

<aside class="sidebar sidebar-header-right widget-area"><section class="widget widget_search" id="header-right-search"><div class="widget-wrap">
				
<form role="search" method="get" class="search-form" action="https://thesgc.github.io/static-openlabnotebooks/">
	<label>
		<span class="screen-reader-text">Search for:</span>
		<input type="search" class="search-field" placeholder="Search..." value="" name="s" title="Search for:"></label>
	<input type="submit" class="search-submit" value="Search"></form>
			</div>
		</section></aside><a class="sub-search">STRUCTURE SEARCH</a>	
				</div>
		</header><nav class="nav-primary search-enabled" role="navigation"><div class="wrapper">
                <h3 class="assistive-text">Primary Menu</h3>
                <div class="screen-reader-text skip-link"><a href="#content" title="Skip to content">Skip to content</a></div>
                <ul id="menu-main-navigation" class="menu clean-journal-nav-menu"><li id="menu-item-27" class="menu-item menu-item-type-custom menu-item-object-custom menu-item-home menu-item-27"><a href="https://thesgc.github.io/static-openlabnotebooks">Home</a></li>
<li id="menu-item-3061" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-3061"><a href="https://thesgc.github.io/static-openlabnotebooks/laboratories/">Laboratories</a>
<ul class="sub-menu"><li id="menu-item-3060" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-3060"><a href="https://thesgc.github.io/static-openlabnotebooks/active-laboratories/">Active</a>
	<ul class="sub-menu"><li id="menu-item-3373" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-3373"><a href="https://thesgc.github.io/static-openlabnotebooks/category/laboratory/amy-ramsey-univ-of-toronto/">Amy Ramsey (Univ. of Toronto)</a></li>
		<li id="menu-item-4874" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-4874"><a href="https://thesgc.github.io/static-openlabnotebooks/category/laboratory/bccrc-univ-of-british-columbia/">BCCRC, Univ. of British Columbia</a></li>
		<li id="menu-item-5125" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-5125"><a href="https://thesgc.github.io/static-openlabnotebooks/category/laboratory/biozone-univ-of-toronto/">BioZone (Univ. of Toronto)</a></li>
		<li id="menu-item-3077" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-3077"><a href="https://thesgc.github.io/static-openlabnotebooks/category/laboratory/unc-catalyst/">Catalyst UNC Chapel Hill</a></li>
		<li id="menu-item-5284" class="menu-item menu-item-type-taxonomy menu-item-object-category current-post-ancestor current-menu-parent current-post-parent menu-item-5284"><a href="https://thesgc.github.io/static-openlabnotebooks/category/laboratory/cheryl-arrowsmith-univ-of-toronto/">Cheryl Arrowsmith (Univ. of Toronto)</a></li>
		<li id="menu-item-3067" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-3067"><a href="https://thesgc.github.io/static-openlabnotebooks/category/laboratory/diamond-light-source/">Diamond Light Source</a></li>
		<li id="menu-item-3068" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-3068"><a href="https://thesgc.github.io/static-openlabnotebooks/category/laboratory/michael-taylor-sickkids/">Michael Taylor (SickKids Hospital Toronto)</a></li>
		<li id="menu-item-3069" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-3069"><a href="https://thesgc.github.io/static-openlabnotebooks/category/laboratory/montreal-neurological-institute/">Montreal Neurological Institute</a></li>
		<li id="menu-item-4785" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-4785"><a href="https://thesgc.github.io/static-openlabnotebooks/category/laboratory/nick-goldman-ebi/">Nick Goldman (EBI)</a></li>
		<li id="menu-item-3071" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-3071"><a href="https://thesgc.github.io/static-openlabnotebooks/category/laboratory/pietro-roversi-leic/">Pietro Roversi (Univ. of Leicester)</a></li>
		<li id="menu-item-3072" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-3072"><a href="https://thesgc.github.io/static-openlabnotebooks/category/laboratory/ray-truant-mcmaster/">Ray Truant (McMaster Univ.)</a></li>
		<li id="menu-item-3073" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-3073"><a href="https://thesgc.github.io/static-openlabnotebooks/category/laboratory/sgc-unc/">SGC UNC Chapel Hill</a></li>
		<li id="menu-item-3074" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-3074"><a href="https://thesgc.github.io/static-openlabnotebooks/category/laboratory/sgc-oxford/">SGC Univ. of Oxford</a></li>
		<li id="menu-item-3075" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-3075"><a href="https://thesgc.github.io/static-openlabnotebooks/category/laboratory/sgc-toronto/">SGC Univ. of Toronto</a></li>
	</ul></li>
	<li id="menu-item-3059" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-3059"><a href="https://thesgc.github.io/static-openlabnotebooks/past-laboratories/">Past</a>
	<ul class="sub-menu"><li id="menu-item-3076" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-3076"><a href="https://thesgc.github.io/static-openlabnotebooks/category/laboratory/antoine-claessens-laboratories/">Antoine Claessens</a></li>
		<li id="menu-item-3078" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-3078"><a href="https://thesgc.github.io/static-openlabnotebooks/category/laboratory/sgc-karolinska/">SGC Karolinska</a></li>
	</ul></li>
</ul></li>
<li id="menu-item-115" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-115"><a href="https://thesgc.github.io/static-openlabnotebooks/people/">People</a>
<ul class="sub-menu"><li id="menu-item-2962" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-2962"><a href="https://thesgc.github.io/static-openlabnotebooks/active-people/">Active</a>
	<ul class="sub-menu"><li id="menu-item-123" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-123"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/alfredo-picado/">Alfredo Picado</a></li>
		<li id="menu-item-5129" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-5129"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/alma-seitova/">Alma Seitova</a></li>
		<li id="menu-item-3372" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-3372"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/amy-ramsey/">Amy Ramsey</a></li>
		<li id="menu-item-5137" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-5137"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/anwar-hossain/">Anwar Hossain</a></li>
		<li id="menu-item-3651" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-3651"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/ashley-hutchinson/">Ashley Hutchinson</a></li>
		<li id="menu-item-1118" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-1118"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/ben-eduful/">Ben Eduful</a></li>
		<li id="menu-item-3102" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-3102"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/claudia-alvarez/">Claudia Alvarez</a></li>
		<li id="menu-item-5286" class="menu-item menu-item-type-taxonomy menu-item-object-category current-post-ancestor current-menu-parent current-post-parent menu-item-5286"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/danton-ivanochko/">Danton Ivanochko</a></li>
		<li id="menu-item-257" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-257"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/david-dilworth/">David Dilworth</a></li>
		<li id="menu-item-4506" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-4506"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/david-drewry/">David Drewry</a></li>
		<li id="menu-item-4556" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-4556"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/ed-anderson/">Edward Anderson</a></li>
		<li id="menu-item-1050" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-1050"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/ellie-williams/">Ellie Williams</a></li>
		<li id="menu-item-5223" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-5223"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/evianne-rovers/">Evianne Rovers</a></li>
		<li id="menu-item-5413" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-5413"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/fang-chi-chang/">Fang-Chi Chang</a></li>
		<li id="menu-item-4598" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-4598"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/hadley-sheppard/">Hadley Sheppard</a></li>
		<li id="menu-item-176" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-176"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/jong-fu-wong/">Jong Fu Wong</a></li>
		<li id="menu-item-1993" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-1993"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/jose-brandao-neto/">Jose Brandao-Neto</a></li>
		<li id="menu-item-5082" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-5082"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/joseph-newman/">Joseph Newman</a></li>
		<li id="menu-item-3414" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-3414"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/katheron-intson/">Katheron Intson</a></li>
		<li id="menu-item-177" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-177"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/liz-brown/">Liz Brown</a></li>
		<li id="menu-item-2124" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-2124"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/louisa-temme/">Louisa Temme</a></li>
		<li id="menu-item-2790" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-2790"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/magdalena-szewczyk/">Magdalena Szewczyk</a></li>
		<li id="menu-item-135" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-135"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/mandeep-mann/">Mandeep Mann</a></li>
		<li id="menu-item-4787" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-4787"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/nicola-de-maio/">Nicola De Maio</a></li>
		<li id="menu-item-2203" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-2203"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/opher-gileadi/">Opher Gileadi</a></li>
		<li id="menu-item-5126" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-5126"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/peter-stogios/">Peter Stogios</a></li>
		<li id="menu-item-1700" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-1700"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/pietro-roversi/">Pietro Roversi</a></li>
		<li id="menu-item-281" class="menu-item menu-item-type-custom menu-item-object-custom menu-item-281"><a target="_blank" rel="noopener noreferrer" href="http://labscribbles.com/">Rachel Harding</a></li>
		<li id="menu-item-1772" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-1772"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/sabrina-mackinnon/">Sabrina Mackinnon</a></li>
		<li id="menu-item-2867" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-2867"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/serap-beldar/">Serap Beldar</a></li>
		<li id="menu-item-3761" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-3761"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/setayesh-yazdani/">Setayesh Yazdani</a></li>
		<li id="menu-item-2956" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-2956"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/stephen-armstrong/">Stephen R. Armstrong</a></li>
		<li id="menu-item-2461" class="menu-item menu-item-type-custom menu-item-object-custom menu-item-2461"><a target="_blank" rel="noopener noreferrer" href="https://raytruantlab.ca/real-time-open-reports/">Tamara Maiuri</a></li>
		<li id="menu-item-4513" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-4513"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/taraneh-hajian/">Taraneh Hajian</a></li>
		<li id="menu-item-2447" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-2447"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/thomas-durcan/">Thomas M. Durcan</a></li>
		<li id="menu-item-2311" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-2311"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/tobias-krojer/">Tobias Krojer</a></li>
		<li id="menu-item-2882" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-2882"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/victoria-vu/">Victoria Vu</a></li>
		<li id="menu-item-3415" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-3415"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/yuanye-yan/">Yuanye Yan</a></li>
	</ul></li>
	<li id="menu-item-2961" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-2961"><a href="https://thesgc.github.io/static-openlabnotebooks/past-people/">Past</a>
	<ul class="sub-menu"><li id="menu-item-1162" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-1162"><a href="https://thesgc.github.io/static-openlabnotebooks/category/past-scientist/antoine-claessens/">Antoine Claessens</a></li>
		<li id="menu-item-2963" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-2963"><a href="https://thesgc.github.io/static-openlabnotebooks/category/past-scientist/carla-alamillo/">Carla Alamillo</a></li>
		<li id="menu-item-3346" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-3346"><a href="https://thesgc.github.io/static-openlabnotebooks/category/past-scientist/carmen-jimenez-antunez/">Carmen Jimenez Antunez</a></li>
		<li id="menu-item-2029" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-2029"><a href="https://thesgc.github.io/static-openlabnotebooks/category/past-scientist/carolyn-marks/">Carolyn Marks</a></li>
		<li id="menu-item-308" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-308"><a href="https://thesgc.github.io/static-openlabnotebooks/category/past-scientist/genna-luciani/">Genna Luciani</a></li>
		<li id="menu-item-2681" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-2681"><a href="https://thesgc.github.io/static-openlabnotebooks/category/past-scientist/haresh-ajani/">Haresh Ajani</a></li>
		<li id="menu-item-2902" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-2902"><a href="https://thesgc.github.io/static-openlabnotebooks/category/past-scientist/jacob-mcauley/">Jacob McAuley</a></li>
		<li id="menu-item-2968" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-2968"><a href="https://thesgc.github.io/static-openlabnotebooks/category/past-scientist/jolene-ho/">Jolene Ho</a></li>
		<li id="menu-item-2964" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-2964"><a href="https://thesgc.github.io/static-openlabnotebooks/category/past-scientist/kaleb-naegeli/">Kaleb Naegeli</a></li>
		<li id="menu-item-2965" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-2965"><a href="https://thesgc.github.io/static-openlabnotebooks/category/past-scientist/megha-abbey/">Megha Abbey</a></li>
		<li id="menu-item-2966" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-2966"><a href="https://thesgc.github.io/static-openlabnotebooks/category/past-scientist/nirav-kapadia/">Nirav Kapadia</a></li>
		<li id="menu-item-376" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-376"><a href="https://thesgc.github.io/static-openlabnotebooks/category/past-scientist/roslin-adamson/">Roslin Adamson</a></li>
		<li id="menu-item-1972" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-1972"><a href="https://thesgc.github.io/static-openlabnotebooks/category/past-scientist/sean-obyrne/">Sean O’Byrne</a></li>
		<li id="menu-item-2967" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-2967"><a href="https://thesgc.github.io/static-openlabnotebooks/category/past-scientist/wen-yih-aw/">Wen Yih Aw</a></li>
	</ul></li>
</ul></li>
<li id="menu-item-118" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-118"><a href="https://thesgc.github.io/static-openlabnotebooks/disease-foundations/">Diseases</a>
<ul class="sub-menu"><li id="menu-item-1761" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-1761"><a href="https://thesgc.github.io/static-openlabnotebooks/active-diseases/">Active</a>
	<ul class="sub-menu"><li id="menu-item-1855" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-1855"><a href="https://thesgc.github.io/static-openlabnotebooks/category/disease-foundation/alzheimers-disease/">Alzheimer’s disease</a></li>
		<li id="menu-item-2026" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-2026"><a href="https://thesgc.github.io/static-openlabnotebooks/category/disease-foundation/amyotrophic-lateral-sclerosis-als/">Amyotrophic Lateral Sclerosis (ALS)</a></li>
		<li id="menu-item-4555" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-4555"><a href="https://thesgc.github.io/static-openlabnotebooks/category/disease-foundation/batten-disease/">Batten disease</a></li>
		<li id="menu-item-1140" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-1140"><a href="https://thesgc.github.io/static-openlabnotebooks/category/disease-foundation/chordoma/">Chordoma</a></li>
		<li id="menu-item-4780" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-4780"><a href="https://thesgc.github.io/static-openlabnotebooks/category/disease-foundation/covid-19/">COVID-19</a></li>
		<li id="menu-item-144" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-144"><a href="https://thesgc.github.io/static-openlabnotebooks/category/disease-foundation/dipg/">Diffuse intrinsic pontine glioma</a></li>
		<li id="menu-item-3759" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-3759"><a href="https://thesgc.github.io/static-openlabnotebooks/category/disease-foundation/ependymoma/">Ependymoma</a></li>
		<li id="menu-item-16" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-16"><a href="https://thesgc.github.io/static-openlabnotebooks/category/disease-foundation/fop/">Fibrodysplasia ossificans progressiva</a></li>
		<li id="menu-item-3374" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-3374"><a href="https://thesgc.github.io/static-openlabnotebooks/category/disease-foundation/grin-disorder/">GRIN disorder</a></li>
		<li id="menu-item-600" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-600"><a href="https://thesgc.github.io/static-openlabnotebooks/category/disease-foundation/chdi/">Huntington’s disease</a></li>
		<li id="menu-item-2504" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-2504"><a href="https://thesgc.github.io/static-openlabnotebooks/category/disease-foundation/ipsc-disease-modelling/">iPSC Disease modelling</a></li>
		<li id="menu-item-255" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-255"><a href="https://thesgc.github.io/static-openlabnotebooks/category/disease-foundation/leukemia/">Leukemia</a></li>
		<li id="menu-item-1144" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-1144"><a href="https://thesgc.github.io/static-openlabnotebooks/category/disease-foundation/malaria/">Malaria</a></li>
		<li id="menu-item-2449" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-2449"><a href="https://thesgc.github.io/static-openlabnotebooks/category/disease-foundation/parkinsons-disease-and-neurodevelopmental-disorders/">Parkinson’s disease and neurodevelopmental disorders</a></li>
		<li id="menu-item-1774" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-1774"><a href="https://thesgc.github.io/static-openlabnotebooks/category/disease-foundation/primary-hyperoxaluria/">Primary Hyperoxaluria</a></li>
		<li id="menu-item-1702" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-1702"><a href="https://thesgc.github.io/static-openlabnotebooks/category/disease-foundation/rare-disease/">Rare disease</a></li>
	</ul></li>
	<li id="menu-item-1760" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-1760"><a href="https://thesgc.github.io/static-openlabnotebooks/past-diseases/">Past</a>
	<ul class="sub-menu"><li id="menu-item-1873" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-1873"><a href="https://thesgc.github.io/static-openlabnotebooks/category/disease-foundation/dravet-syndrome/">Dravet syndrome</a></li>
	</ul></li>
</ul></li>
<li id="menu-item-155" class="menu-item menu-item-type-post_type menu-item-object-page current-post-parent menu-item-has-children menu-item-155"><a href="https://thesgc.github.io/static-openlabnotebooks/projects/">Projects</a>
<ul class="sub-menu"><li id="menu-item-1759" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-1759"><a href="https://thesgc.github.io/static-openlabnotebooks/active-projects/">Active</a>
	<ul class="sub-menu"><li id="menu-item-356" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-356"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/acvr1-alk2/">ACVR1/ALK2</a></li>
		<li id="menu-item-2027" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-2027"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/als-reproducible-antibody-platform-als-rap/">ALS-Reproducible Antibody Platform (ALS-RAP)</a></li>
		<li id="menu-item-4496" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-4496"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/cell-assay-reports/">Cell assay reports</a></li>
		<li id="menu-item-4387" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-4387"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/chromatin-factors/">Chromatin factors</a></li>
		<li id="menu-item-2883" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-2883"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/e3-ligases/">E3 ligases</a></li>
		<li id="menu-item-2771" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-2771"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/epigenetic-targets-in-medulloblastoma/">Epigenetic targets in Medulloblastoma</a></li>
		<li id="menu-item-3760" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-3760"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/ezhip/">EZHIP</a></li>
		<li id="menu-item-1856" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-1856"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/gtpase-gef-complex/">GTPase-GEF complex</a></li>
		<li id="menu-item-1773" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-1773"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/hao1/">HAO1</a></li>
		<li id="menu-item-451" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-451"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/hdacs/">HDACs</a></li>
		<li id="menu-item-3295" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-3295"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/hero-glove/">HERO Glove</a></li>
		<li id="menu-item-601" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-601"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/huntingtin/">Huntingtin</a></li>
		<li id="menu-item-3184" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-3184"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/investigating-outdoor-mobility-of-older-adults/">Investigating Outdoor Mobility of Older Adults</a></li>
		<li id="menu-item-2503" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-2503"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/ipscs/">iPSCs</a></li>
		<li id="menu-item-4286" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-4286"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/ligandable-genome/">Ligandable Genome</a></li>
		<li id="menu-item-3375" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-3375"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/nmda/">NMDA</a></li>
		<li id="menu-item-357" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-357"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/nsd3/">NSD3</a></li>
		<li id="menu-item-1703" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-1703"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/open-secretion-rescue/">Open Secretion Rescue</a></li>
		<li id="menu-item-504" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-504"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/prc2/">PRC2</a></li>
		<li id="menu-item-5285" class="menu-item menu-item-type-taxonomy menu-item-object-category current-post-ancestor current-menu-parent current-post-parent menu-item-5285"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/prdm-methyltransferases/">PRDM methyltransferases</a></li>
		<li id="menu-item-3847" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-3847"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/prmts/">PRMTs</a></li>
		<li id="menu-item-4509" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-4509"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/protein-purification-protocols/">Protein purification protocols</a></li>
		<li id="menu-item-4786" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-4786"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/sars-cov-2-computational-genomics/">SARS-CoV-2 computational genomics</a></li>
		<li id="menu-item-2868" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-2868"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/setdb1/">SETDB1</a></li>
		<li id="menu-item-5024" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-5024"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/spike/">Spike</a></li>
		<li id="menu-item-5138" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-5138"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/tbxt-brachyury-inhibitors/">TBXT (brachyury) inhibitors</a></li>
		<li id="menu-item-254" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-254"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/understudied-kinases/">Understudied Kinases</a></li>
		<li id="menu-item-157" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-157"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/usp5/">USP5</a></li>
		<li id="menu-item-4388" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-4388"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/usps/">USPs</a></li>
		<li id="menu-item-5414" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-5414"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/wdr-proteins/">WDR proteins</a></li>
		<li id="menu-item-3652" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-3652"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/wdr41/">WDR41</a></li>
		<li id="menu-item-1960" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-1960"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/xchem/">XChem</a></li>
	</ul></li>
	<li id="menu-item-1758" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-1758"><a href="https://thesgc.github.io/static-openlabnotebooks/past-projects/">Past</a>
	<ul class="sub-menu"><li id="menu-item-1141" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-1141"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/egfr-and-gak/">EGFR and GAK</a></li>
		<li id="menu-item-1874" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-1874"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/gene-therapy-for-dravet-syndrome/">Gene therapy for Dravet Syndrome</a></li>
		<li id="menu-item-2682" class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-2682"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/in-silico-protacs-design/">In silico PROTACs design</a></li>
	</ul></li>
</ul></li>
<li id="menu-item-923" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-923"><a href="https://thesgc.github.io/static-openlabnotebooks/the-team/">The team</a></li>
<li id="menu-item-919" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-919"><a href="https://thesgc.github.io/static-openlabnotebooks/about/">About</a></li>
<li id="menu-item-4663" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-4663"><a href="https://thesgc.github.io/static-openlabnotebooks/my-research-in-2-minutes/">My research in 2 minutes</a>
<ul class="sub-menu"><li id="menu-item-4729" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-4729"><a href="https://thesgc.github.io/static-openlabnotebooks/artem-babaian-ultra-deep-search-for-novel-coronaviruses/">Artem Babaian: ultra-deep search for novel coronaviruses</a></li>
	<li id="menu-item-4694" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-4694"><a href="https://thesgc.github.io/static-openlabnotebooks/danton-ivanochko-a-chemical-probe-for-prdm9/">Danton Ivanochko: a chemical probe for PRDM9</a></li>
	<li id="menu-item-4695" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-4695"><a href="https://thesgc.github.io/static-openlabnotebooks/mandeep-mann-a-chemical-probe-for-usp5-2/">Mandeep Mann: a chemical probe for USP5</a></li>
	<li id="menu-item-5207" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-5207"><a href="https://thesgc.github.io/static-openlabnotebooks/natasha-anita-depressive-biomarkers-in-diabetes/">Natasha Anita: depressive biomarkers in diabetes</a></li>
	<li id="menu-item-4693" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-4693"><a href="https://thesgc.github.io/static-openlabnotebooks/rachel-harding-3d-structure-of-huntingtin/">Rachel Harding: 3D structure of Huntingtin</a></li>
	<li id="menu-item-4706" class="menu-item menu-item-type-post_type menu-item-object-page menu-item-4706"><a href="https://thesgc.github.io/static-openlabnotebooks/setayesh-yazdani-the-druggable-human-genome/">Setayesh Yazdani: the druggable human genome</a></li>
</ul></li>
</ul><div id="search-toggle" class="genericon">
                            <a class="screen-reader-text" href="#search-container">Search</a>
                        </div>

                        <div id="search-container" class="displaynone">
                            
<form role="search" method="get" class="search-form" action="https://thesgc.github.io/static-openlabnotebooks/">
	<label>
		<span class="screen-reader-text">Search for:</span>
		<input type="search" class="search-field" placeholder="Search..." value="" name="s" title="Search for:"></label>
	<input type="submit" class="search-submit" value="Search"></form>
                        </div>
                                	</div>
        </nav><div id="content" class="site-content">
			<div class="wrapper">
	
	<main id="main" class="site-main" role="main"><article id="post-5380" class="post-5380 post type-post status-publish format-link hentry category-cheryl-arrowsmith-univ-of-toronto category-danton-ivanochko category-prdm-methyltransferases post_format-post-format-link"><div class="entry-container">
		<header class="entry-header"><h1 class="entry-title">Interrogating PR/SET domains for the ability to bind SAH</h1>

			<p class="entry-meta"><span class="posted-on"><span class="screen-reader-text">Posted on</span><a href="https://thesgc.github.io/static-openlabnotebooks/interrogating-pr-set-domains-for-the-ability-to-bind-sah/" rel="bookmark"><time class="entry-date published updated" datetime="2020-11-09T14:37:06+00:00">9th November 2020</time></a></span><span class="byline"><span class="author vcard"><span class="screen-reader-text">Author</span><a class="url fn n" href="">Danton Ivanochko</a></span></span><span class="comments-link"><a href="https://thesgc.github.io/static-openlabnotebooks/interrogating-pr-set-domains-for-the-ability-to-bind-sah/#respond">Leave a comment</a></span></p>		</header><div class="entry-content">
			<blockquote><p>This is an excerpt from my PhD thesis where I investigated 13 human PR/SET domains from the PRDM protein family and assessed their ability to bind to a fluorinated SAH analog. The research helps to identify which of the PRDMs could be methyltransferases. This work was carried out in the laboratory of Dr. Cheryl Arrowsmith at the University of Toronto and the University Health Network.</p></blockquote>
<h1><strong>Interrogating PR/SET domains for the ability to bind SAH</strong></h1>
<p><strong><br></strong><br><strong>Authorship attribution:<br></strong>This chapter contains unpublished work and I am responsible for the overall generation and interpretation of the data included herein with the following mentions: The F-SAH compound was synthesized and kindly provided by Carlos Zepeda, Ontario Institute for Cancer Research (<em>OICR</em>). Shili Duan provided guidance and technical assistance in protein production. Additional proteins were kindly provided by Taraneh Hajian, Elisa Gibson and Dr. Masoud Vedadi from the SGC Toronto. Dr. Scott Houliston advised and supported all NMR experiments, and Drs. Houliston and Cheryl Arrowsmith aided in data analysis and provided guidance throughout the study. This project was prematurely halted due to a catastrophic freezer malfunction during the COVID-19 shutdown.</p>
<p> </p>
<p><strong><u>Introduction </u></strong><strong><br></strong><br>
Lysine methyltransferases are a group epigenetic regulators currently being investigated in clinical and pharmaceutical studies for the treatment of certain cancers (Rugo, et al., 2020) (Ilango, et al., 2020) (Hoy, 2020) (Dilworth & Barsyte-Lovejoy, 2019). The majority of all KMT proteins possess a domain with a SET-fold (Schapira, 2011). The PRDM protein family is characterized by the presence of a PRDI-BF1 and RIZ1 (PR/SET) domain, which is a subgroup of the SET domain (<strong>FIGURE A1</strong>). Currently KMT activity has been reported for some PRDMs (<strong>TABLE A1</strong>), while some PRDMs are thought to be inactive pseudoenzymes. Since PRDMs are less represented in the academic literature compared to canonical SET proteins (<strong>FIGURE A4</strong>), it is unclear if these PRDMs are truly pseudoenzyme or if their KMT activity is only yet to be characterized. Distinguishing which PRDMs possess KMT activity would further our understanding of lysine epigenetics and pseudoenzyme-driven biology (Arrowsmith, et al., 2012) (Ribeiro, et al., 2019).</p>
<p>All domains with a SET-fold interact with SAM through a conserved hydrogen bonding network split across two distinct domain regions (Campagna-Slater, et al., 2011). These SAM binding residues in the canonical SET-domains are well conserved, however the corresponding residues in PR/SET domains are less well conserved (<strong>FIGURE A2</strong>) (H-binding clusters at Regions 1 and 2)]. The only holo-enzyme structure of a PR/SET domains is of mouse PRDM9 bound to a histone H3 peptide and SAH (Wu, et al., 2013). The human and mouse PRDM9 PR/SET domains possess many of the highly conserved SAM-binding residues found in the canonical SET-domains. Lacking any structural evidence, it is unclear if and how many of the PR/SET domains may bind to SAM. Therefore, detecting cofactor binding in PR/SET domains would aid in distinguishing whether certain PRDMs are enzymes or pseudoenzymes.</p>
<p>Here I investigate cofactor binding to understand which PR/SET domains may have the ability to catalyze the KMT reaction. Using <sup>19</sup>F-nuclear magnetic resonance (NMR) spectroscopy, we measured the binding response for the majority of the human PR/SET domains with a fluorinated analog of the methyltransferase by-product S-adenosylhomocysteine (SAH), 2-fluoro-SAH (F-SAH). <sup>19</sup>F-NMR is a sensitive and versatile method to study protein-ligand interactions (Dalvit & Vulpetti, 2019). Using the F-SAH binding response with the PRDM9 PR/SET domain as a positive control for the screening assays, we identified several PR/SET domains that bind F-SAH and therefore likely bind to SAM. Additionally, we detected several PR/SET domains that do not show any evidence of binding to F-SAH, which suggests a lesser affinity for SAM or potentially no affinity. This dataset can be used as a starting point for future discovery of KMT activity in PRDM proteins.</p>
<p><strong><u>RESULTS</u></strong></p>
<p><strong>Establishment of the F-SAH binding assay</strong></p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/FIGURE_K1-Copy-295x300.png" alt="" class="alignnone wp-image-5381" width="955" height="971" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/FIGURE_K1-Copy-295x300.png 295w,https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/FIGURE_K1-Copy-1008x1024.png 1008w,https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/FIGURE_K1-Copy-768x780.png 768w,https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/FIGURE_K1-Copy-1512x1536.png 1512w,https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/FIGURE_K1-Copy.png 1887w" sizes="(max-width: 955px) 100vw, 955px"><br><strong>Figure K1. Characterization of F-SAH binding to PRDM9. </strong>(<strong>a</strong>) Chemical structure of 2-fluoro-SAH (F-SAH) and (<strong>b</strong>) model of F-SAH bound to PRDM9 based on a SAM-bound structure (PDBID: 6nm4). Colour coding corresponds to (<strong>a</strong>). The effects of F-SAH binding to PRDM9 were measured by <sup>19</sup>F-NMR. (<strong>c-d</strong>) Spectra of 16 µM F-SAH alone and with 1X and 5X molar equivalents of PRDM9 display a concentration dependent decrease in the unbound F-SAH peak and the emergence of a bound F-SAH peak.</p>
<p> </p>
<p>We utilized <sup>19</sup>F nuclear magnetic resonance spectroscopy (NMR) spectroscopy to assess whether PR/SET domains may have the ability to bind to the methyltransferase enzyme cofactor. Fluorinated SAH was chosen, rather than fluorinated SAM because SAH and its chemical precursors are more stable across a wide pH range and were therefore more suited to chemical synthesis. The chemical structure of 2-fluoro-SAH (F-SAH) shows that SAH is fluorinated at the 2-position of the adenine ring<strong> (FIGURE K1a)</strong>. Using our structure of the human PRDM9 PR/SET domain bound by SAM, we generated a theoretical model of F-SAH bound to PRDM9 <strong>(FIGURE K1b)</strong>. We observed that the fluorine atom projects toward the solvent and therefore is unlikely to alter the interaction with the protein. Nevertheless, because fluorine’s chemical shift is highly sensitive to any change in its microenvironment, we expected to see perturbation and/or broadening of the <sup>19</sup>F resonance when F-SAH binds to a PR/SET domain.</p>
<p>First, we investigated F-SAH alone by measuring a 1D <sup>19</sup>F spectrum of F-SAH, which gave rise to a single peak at ~-52.4 ppm. Next, we monitored for F-SAH binding to PRDM9, which is the most extensively characterized methyltransferase among all the PR/SET domain containing proteins (Eram, et al., 2014) (Blazer, et al., 2016) (Allali-Hassani, et al., 2019) (Powers, et al., 2016). We assessed the F-SAH NMR signal alone and in the presence of equal and 5-fold molar equivalents of PRDM9 PR/SET domain, we detected a concentration-dependent decrease in the intensity of the unbound F-SAH <sup>19</sup>F resonance <strong>(FIGURE K1c-d)</strong>.  Moreover, we observed the emergence of a second <sup>19</sup>F signal (~ 1045 Hz up-field at -54.3 ppm) that is from protein-bound F-SAH. As these results were reproducible using two separate PRDM9 protein preparations, we selected 5-fold molar equivalents of PR/SET domain to F-SAH for further experiments.</p>
<p><strong>Screening human PR/SET domains with the F-SAH binding assay</strong></p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/FIGURE_K2-Copy-204x300.png" alt="" class="alignnone wp-image-5382" width="869" height="1278" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/FIGURE_K2-Copy-204x300.png 204w,https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/FIGURE_K2-Copy-1391x2048.png 1391w" sizes="(max-width: 869px) 100vw, 869px"><br><strong>Figure K2. Screening for F-SAH binders among PR domains.</strong> (<strong>a</strong>) <sup>19</sup>F-NMR spectra showing 16 µM F-SAH in the presence of 80 µM of the indicated PR/SET domains. (<strong>b</strong>) Plots of <sup>19</sup>F peak integrals for F-SAH in the presence of individual PR/SET domains. All <sup>19</sup>F peaks were normalized to the peak integral of F-SAH alone in buffer and the normalized integrals of the Bound and 1-Unbound peaks relative to F-SAH alone were plotted (i.e. F-SAH alone would be located at position [0,0]).</p>
<p> </p>
<p>We performed F-SAH binding assays with PR/SET domains from 13 of the 19 human PRDM family members. The PR/SET domains of PRDM 1, 2, 4, 5, 6, 11, 12, 14 and 17 were provided by the SGC Toronto and I purified PRDM 3, 7, 9 and 16. I was able to purify PR/SET protein constructs for PRDM 8, 10, 13 and 15, however these constructs suffered stability issues in our <sup>19</sup>F-NMR buffer during dialysis (see Methods) and were therefore not suitable for further study. We collected <sup>19</sup>F-NMR spectra for F-SAH with 5-fold molar equivalents of each of the 13 human PR/SET domains (<strong>Figure K2a</strong>). The relative intensities of the unbound and bound F-SAH peaks varied considerably among the different PR/SET proteins indicating variable binding affinities across the panel of PRDM proteins. To quantify the binding for each PRDM PR/SET domain, we normalized the peak integrals of the unbound and bound peaks for each protein to the unbound peak of F-SAH alone. Specifically, we divided the integral of each peak by the integral of the peak for F-SAH alone. We plotted the normalized integrals of the 1 minus the unbound peak against the bound peak for each PR/SET domain to differentiate each protein’s affinity for F-SAH (<strong>Figure K2b</strong>). PR/SET datapoints located in the upper, right region of the plot displayed strong binding to F-SAH, while datapoints at the origin indicate no evidence of binding. We used the F-SAH signal in the presence of PRDM9 as an internal control to distinguish a strong binding response. PRDM 4 and 5 were plotted near to PRDM9, indicating a strong evidence response. Additionally, PRDM 14, 12, 7, 11 and 17 exhibited a very strong binding response. We were unable to detect any binding response for F-SAH in the presence of PRDM 3, 1 or 16, using our assay conditions, though we did detected a weak binding response for PRDM 2 and 6. Taken together, this data provides evidence to help prioritize further assays to identify methyltransferase activity among the PRDM family members.<em><br></em></p>
<p><strong><u>Discussion</u></strong><strong><br></strong><br><strong>Using F-SAH to probe KMT domains</strong></p>
<p>Here we used <sup>19</sup>F-NMR to investigate the potential for 13 of the 19 human PR/SET domains to bind a fluorinated analog of the methylation reaction by-product SAH. Using our F-SAH binding assays with PR/SET domains, we identified 7 domains (PRDM 4, 5, 7, 11, 12, 14 and 17) that showed a binding response similar to, or stronger than PRDM9. Of these ‘binding-competent’ domains only PRDM 9 and 7 have reported KMT activity, with accompanying enzyme kinetics data (Eram, et al., 2014) (Blazer, et al., 2016). Interestingly, PRDM7 showed stronger binding to F-SAH compared to PRDM9 (<strong>Figure K2</strong>). PRDM7 possesses much weaker KMT activity for H3K4me2 substrate peptide compared to PRDM9, with a 100-fold lower rate of catalysis (<em>k</em><sub>cat</sub>=1.9×10<sup>2</sup>/<em>h </em>vs 1.9×10<sup>4</sup>/h, for PRDM 7 and 9, respectively) (Blazer, et al., 2016) (Eram, et al., 2014). These catalytic rate differences are mostly attributed to the distinct capacities to facilitate the methyl cation transferase by the catalytic residues (Y357 and S357, in PRDM 9 and 7, respectively). Based on our data, another contributing factor could be that PRDM7 has a stronger affinity for the reaction by-product SAH, which inhibits the catalytic rate. In our F-SAH binding assays, PRDM 2 and 6 showed weak, but detectable, binding compared to PRDM9 (<strong>Figure K2</strong>). KMT activities towards H3K9 and H4K20 have been reported for PRDM 2 and 6, respectively (Kim, et al., 2003) (Congdon, et al., 2014) (Wu, et al., 2008). This indicates that a weak binding response for F-SAH is associated with known active KMTs. Finally, in all cases where we observed F-SAH binding, we measured a consistent up-field shift (~1040 Hz) in the F-SAH <sup>19</sup>F resonance (<strong>Figure K2a</strong>). This indicates there is a common binding modality with a highly similar microenvironment formed in the bound state across the family.</p>
<p>We also demonstrated the utility of F-SAH as a synthetic chemical probe for methyltransferase proteins. Our approach is comparable to a previous study that reported the binding of the MLL1 SET-domain to a SAM-analog conjugated with a fluorescent tag (Luan, et al., 2016). Comparatively, F-SAH benefits from the smaller size of fluorine, which is positioned on the adenine ring in an orientation that is unlikely to interfere with binding (<strong>Figure K2a-b</strong>). Additionally, <sup>19</sup>F-NMR is capable of detecting low affinity protein-ligand interactions, such as is required for fragment-based screening (Dalvit & Vulpetti, 2019). We surveyed two publicly available databases that quantify protein-ligand interaction data (<strong>TABLE K1</strong>). We found that SET-fold containing proteins have a range of affinities for SAH with dissociation constants (K<sub>D</sub>) from low µM to high nM, as well as a similar range for reported half-maximal inhibition (IC<sub>50</sub>) by SAH. Only RBCMT had affinity data for both SAH and SAM, showing that SAM binds RBCMT with an ~70-fold higher affinity when compared with SAH  (Couture, et al., 2006). The authors reported conserved binding modes in crystal structures of RBCMT-SAM and RBCMT-SAH implying that the variations in affinity could not be attributed to the binding conformations, but may rather be attributed to a higher entropic penalty for bound SAH, which possesses additional bond rotational freedom compared to SAM (Couture, et al., 2006). Taken together, we can predict that the upper-limit for the affinity of PRDM9 for F-SAH is less than the known affinity of SAM, which is 18.6 µM (Eram, et al., 2014).</p>
<p><strong>Table K1. Known dissociation constant (K<sub>D</sub>) and half maximal inhibition values (IC<sub>50</sub>) of SAM and SAH for SET domain proteins. </strong>Binding affinity data available from the BindingDB (www.bindingdb.org) and BindingMOAD (www.bindingmoad.org/) was accessed from the RCSB PDB (www.rcsb.org). Note that BindingDB reports a large range of IC<sub>50</sub> values for EZH2.</p>
<table width="95%"><tbody><tr><td style="width: 95.5333px;"><strong>Protein</strong></td>
<td style="width: 207.883px;"><strong>Source</strong></td>
<td style="width: 115.967px;"><strong>Ligand</strong></td>
<td style="width: 115.967px;"><strong>K<sub>D</sub> (µM)</strong></td>
<td style="width: 115.967px;"><strong>IC<sub>50</sub> (µM)</strong></td>
<td style="width: 228.283px;"><strong>Source</strong></td>
</tr><tr><td style="width: 95.5333px;">EHMT1</td>
<td style="width: 207.883px;"><em>Homo sapiens</em></td>
<td style="width: 115.967px;">SAH</td>
<td style="width: 115.967px;"></td>
<td style="width: 115.967px;">2.3</td>
<td style="width: 228.283px;">BindingDB</td>
</tr><tr><td style="width: 95.5333px;">EHMT2</td>
<td style="width: 207.883px;"><em>Homo sapiens</em></td>
<td style="width: 115.967px;">SAH</td>
<td style="width: 115.967px;">0.57</td>
<td style="width: 115.967px;">2.0</td>
<td style="width: 228.283px;">BindingDB</td>
</tr><tr><td style="width: 95.5333px;">EZH2</td>
<td style="width: 207.883px;"><em>Homo sapiens</em></td>
<td style="width: 115.967px;">SAH</td>
<td style="width: 115.967px;"></td>
<td style="width: 115.967px;">0.263-16.6</td>
<td style="width: 228.283px;">BindingDB</td>
</tr><tr><td style="width: 95.5333px;">KMT2A</td>
<td style="width: 207.883px;"><em>Homo sapiens</em></td>
<td style="width: 115.967px;">SAH</td>
<td style="width: 115.967px;"></td>
<td style="width: 115.967px;">2.3</td>
<td style="width: 228.283px;">BindingDB</td>
</tr><tr><td style="width: 95.5333px;">Kmt5b</td>
<td style="width: 207.883px;"><em>Mus musculus</em></td>
<td style="width: 115.967px;">SAM</td>
<td style="width: 115.967px;">11.2</td>
<td style="width: 115.967px;"></td>
<td style="width: 228.283px;">BindingMOAD</td>
</tr><tr><td style="width: 95.5333px;">Kmt5c</td>
<td style="width: 207.883px;"><em>Mus musculus</em></td>
<td style="width: 115.967px;">SAH</td>
<td style="width: 115.967px;">17.2</td>
<td style="width: 115.967px;">10</td>
<td style="width: 228.283px;">BindingDB</td>
</tr><tr><td style="width: 95.5333px;">PRDM9</td>
<td style="width: 207.883px;"><em>Homo sapiens</em></td>
<td style="width: 115.967px;">SAM</td>
<td style="width: 115.967px;">18.6</td>
<td style="width: 115.967px;"></td>
<td style="width: 228.283px;">(Eram, et al., 2014)</td>
</tr><tr><td style="width: 95.5333px;">RBCMT</td>
<td style="width: 207.883px;"><em>Pisum sativum</em></td>
<td style="width: 115.967px;">SAM</td>
<td style="width: 115.967px;">0.29</td>
<td style="width: 115.967px;"></td>
<td style="width: 228.283px;">BindingMOAD</td>
</tr><tr><td style="width: 95.5333px;">RBCMT</td>
<td style="width: 207.883px;"><em>Pisum sativum</em></td>
<td style="width: 115.967px;">SAH</td>
<td style="width: 115.967px;">21.2</td>
<td style="width: 115.967px;"></td>
<td style="width: 228.283px;">BindingMOAD</td>
</tr><tr><td style="width: 95.5333px;">SETD7</td>
<td style="width: 207.883px;"><em>Homo sapiens</em></td>
<td style="width: 115.967px;">SAH</td>
<td style="width: 115.967px;"></td>
<td style="width: 115.967px;">30</td>
<td style="width: 228.283px;">BindingDB</td>
</tr><tr><td style="width: 95.5333px;">mSMYD2</td>
<td style="width: 207.883px;"><em>Mus musculus</em></td>
<td style="width: 115.967px;">SAH</td>
<td style="width: 115.967px;"></td>
<td style="width: 115.967px;">0.18</td>
<td style="width: 228.283px;">BindingDB</td>
</tr><tr><td style="width: 95.5333px;">SMYD2</td>
<td style="width: 207.883px;"><em>Homo sapiens</em></td>
<td style="width: 115.967px;">SAH</td>
<td style="width: 115.967px;"></td>
<td style="width: 115.967px;">0.18</td>
<td style="width: 228.283px;">BindingDB</td>
</tr></tbody></table><p><strong>Interpreting the absence of binding evidence</strong></p>
<p>We identified that PRDM 1, 3, and 16 do not bind to F-SAH in our assay conditions, which is suggestive of the inability to bind to SAM. PRDM1 (initially identified as Blimp-1) is widely understood to lack KMT activity and instead functions as a gene repressor by recruiting specific co-repressors proteins to PRDM1-binding sites throughout the genome (Minnich, et al., 2016). PRDM1 does affect histone methylation, but it does so through interactions with the KMT protein G9a/EHMT2 (Gyory, et al., 2004), the arginine methyltransferase PRMT5 (Ancelin, et al., 2006) and the lysine demethylase LSD1 (Su, et al., 2009). Whether the PR/SET domain of PRDM1 functions as a “reader domain” for specific histone tail modifications is undetermined, however our data suggests that it is unlikely that SAH or SAM would play a role in this ability.</p>
<p>We found that the PR/SET domains of PRDM 3 and 16 do not bind to F-SAH under our assay conditions. However, this does not exclude the possibility that larger or differently generated constructs may possess the ability to bind SAM or SAH. For instance, longer or full-length PRDM 3 and 16 protein constructs were reported to possess KMT activity when purified from either HeLa cells and mouse fibroblasts or SF9 insect cells (Pinheiro, et al., 2012) (Zhou, et al., 2016). Our PRDM 3 and 16 constructs were purified from E. coli and each contained the PR/SET domain along with the first proximal C-terminal zinc-finger motif (ZnF1), which is similar to previous structural and enzyme characterization studies of the PR/SET and ZnF1 of PRDM9 (Eram, et al., 2014) (Wu, et al., 2013). It is unclear if the additional regions outside the PRDM 3 and 16 PR/SET domains and ZnF1 could function to stabilize the cofactor binding site in the PR/SET domain. Interestingly, a previously solved NMR solution structure of PRDM16 demonstrated the absence of internal β-strands that are present in crystal and solution structures of all other PR/SET domains, which could possibly destabilize the cofactor and substrate binding sites (<strong>FIGURE A3d</strong>). Furthermore, it is unclear if additional factors present in eukaryotic expression systems enable cofactor binding or enzymatic activity.</p>
<p><strong>Future analysis of PRDMs<br></strong><br>
Future investigation to complete the F-SAH binding screen with the untested members of the PRDM family (PRDM 8, 10, 13, 15, FOG1, and FOG2) would provide valuable data.  Additionally, assaying F-SAH binding in competition with (unlabeled) SAM could qualitatively delineate the affinities of the methyltransferase reaction cofactor and by-product, providing further evidence of potential enzymatic function. We detected F-SAH binding in PRDM 4, 5, 11, 12, 14, and 17, and we could not find any studies reporting KMT activity with these proteins. Intriguingly, PRDM 4, 5, 11, 12, and 14 all possess the highly conserved “catalytic tyrosine” (<strong>FIGURE A2</strong>), which is further evidence supporting KMT activity for these proteins. Importantly, several studies have attributed oncogenic roles for PRDM 4, 12 and 14 in specific cancers (Casamassimi, et al., 2020). Future discovery of enzymatic function in any of these PR/SET domains could provide a novel, actionable target for oncological drug discovery.</p>
<p> </p>
<p><strong><u>Methods</u></strong><strong><br></strong><strong>Protein production </strong><br>
PRDM9 (195-385) and PRDM7 (195-392) with an N-terminal 6xHis-tag and TEV protease site were expressed in <em>Escherichia coli</em> BL21 (DE3) codon plus cells from a pET28-MHL vector. PRDM3 (69-235) and PRDM16 (73-256) with an N-terminal 6xHis-tag and TEV protease site were expressed in <em>Escherichia coli</em> BL21 (DE3) codon plus cells from a pET15b/MHL vector. Cell were grown at 37°C in M9 minimal medium in the presence of 50 µg/ml of kanamycin (PRDM 7 and 9) or 100 µg/ml of ampicillin (PRDM 3 and 16) to an OD600 of 0.8 and induced by isopropyl-1-thio-D-galactopyranoside (IPTG), final concentration 0.5 mM and incubated overnight at 15°C. Cells were harvested by centrifugation at 7,000 rpm and cell pellets were stored at -80°C. Cell were lyzed on ice in lysis buffer (20 mM Tris pH 7.5, 300 mM NaCl, 10 mM imidazole, 50 µM ZnSO<sub>4</sub>, 5 mM β-mercaptoethanol, 0.5 mM TCEP, 0.1% Triton-X 100, 2.5% glycerol, 1mM PMSF, 2 mM benzamidine and Roche complete EDTA-free protease inhibitor cocktail tablet) using a probe sonicator. The crude extract was cleared by ultracentrifugation for 50 min at 50000xg and the supernatant was incubated with TALON Metal Affinity Resin (Takara) at 4<sup>o</sup>C for 120 minutes with agitation. Resin was washed in wash buffer (lysis buffer with 0.01% Triton-X 100 and lacking protease inhibitors) and bound proteins were eluted using elution buffer (wash buffer with 400 mM imidazole), monitored by Bradford analysis. Protein was dialyzed overnight at 4°C in TEV dialysis buffer (20 mM Tris pH 7.5, 300 mM NaCl, 20µM ZnCL<sub>2</sub>, 2.5% glycerol, 5 mM β-mercaptoethanol and 0.5 mM TCEP) and incubated with His-tagged TEV protease produced in house at a 1/20 dilution by mass. 0.5 mM CHAPS was added to the TEV dialysis buffer for PRDM 9 and 7, but not PRDM 3 and 16. TEV and uncleaved proteins were removed using TALON resin and soluble  protein was loaded onto a Superdex 75 Increase 10/300 GL colum (GE Healthcare), equilibrated with 20 mM Tris-HCl buffer, pH 7.5, and 150 mM NaCl, 2.5% glycerol, 5 mM β-mercaptoethanol and 0.5 mM TCEP at flow rate 0.7 ml/min. Fractions containing PRDM9 protein were pooled and dialyzed into low salt buffer (20 mM Tris-HCl pH 8.5, 100 mM NaCl, 5% glycerol, 5 mM β-mercaptoethanol and 0.5 mM TCEP). PRDM9 protein was further purified by anion-exchange chromatography on 2 tandemly joined 5 ml HiTrap™ DEAE FF (GE Healthcare) columns along a linear gradient up to 1.0 M NaCl. Purified PRDM9 protein was dialyzed into a final buffer containing 20 mM Tris pH 7.5, 150 mM NaCl and 2 mM TCEP. The proteins for PRDM1 (38-223), PRDM2 (2-160), PRDM4 (390-540), PRDM5 (2-226), PRDM6 (194-405), PRDM11 (79-314), PRDM12 (60-229), PRDM14 (212-422), and PRDM17 (1-205) were purified from E. coli expression and kindly provided by Taraneh Hajian, Elisa Gibson and Dr. Masoud Vedadi from the SGC Toronto.</p>
<p><strong>F-SAH binding by <sup>19</sup>F NMR<br></strong>F-SAH was synthesized and kindly provided by Carlos Zepeda, Ontario Institute for Cancer Research (<em>OICR</em>). All PR/SET domain containing proteins were dialyzed overnight at 4<sup>o</sup>C into <sup>19</sup>F-NMR buffer (20 mM Tris pH 7.3, 150 mM NaCl, 2 mM TCEP and 20 µM ZnCl<sub>2</sub>) also containing 16 µM sodium trifluoroacetate. The concentration of each protein was measured using an absorbance reading at 280 nm and evaluated using their respective theoretical extinction coefficients calculated using ProtParam (Gasteiger, et al., 2005). 100 mM F-SAH in d6-DMSO was diluted to 16 µM F-SAH by adding the dialyzed protein solution to a final protein concentration of 80 µM and then topping off with <sup>19</sup>F-NMR buffer to a final volume of 500 µl. For the F-SAH alone sample, only <sup>19</sup>F-NMR buffer was added. The solutions were mixed in 1.5 ml polypropylene tubes and then transferred to 5 mm glass NMR tubes.</p>
<p>1D <sup>19</sup>F spectra were acquired at 298K on a Bruker Avance III spectrometer operating at 600 MHz and equipped with a QCI cryoprobe with an independent <sup>19</sup>F detection coil. Each spectrum was acquired with 3072 scans, an acquisition time of 577 ms, and processed by applying an exponential window function (LB = 20). There was no difference observed in the F-SAH <sup>19</sup>F line-shape when signal acquisition was acquired with or without proton decoupling; therefore, all spectra were acquired without proton decoupling.  The spectral window was measured from -37.89 ppm to -88.09 ppm and <sup>19</sup>F peaks were identified at -75.53 ppm, -52.40 ppm, and -54.21 ppm, for trifluoroacetate, F-SAH unbound, and F-SAH bound, respectively. Processing and analysis were carried out with Topspin 3.5 (Bruker BioSpin). To control for pipetting errors that could result in differences in F-SAH signal intensities, the integral of the trifluoroacetate <sup>19</sup>F peak of each PR/SET containing sample was set to equal the integral of the trifluoroacetate <sup>19</sup>F peak of the sample with F-SAH alone. To normalize F-SAH peaks, the integral for each F-SAH peak was divided by the integral of the F-SAH peak from the sample with F-SAH alone (i.e. integral of F-SAH alone = 1).</p>
<h1>References</h1>
<p>Allali-Hassani, A. et al., 2019. Discovery of a chemical probe for PRDM9.. <em>Nat Commun., </em>10(1), p. 5759.</p>
<p>Ancelin, K. et al., 2006. Blimp1 associates with Prmt5 and directs histone arginine methylation in mouse germ cells.. <em>Nat Cell Biol, </em>8(6), pp. 623-30.</p>
<p>Arrowsmith, C. et al., 2012. Epigenetic protein families: a new frontier for drug discovery.. <em>Nat Rev Drug Discov, </em>11(5), pp. 384-400.</p>
<p>Blazer, L. et al., 2016. PR Domain-containing Protein 7 (PRDM7) Is a Histone 3 Lysine 4 Trimethyltransferase.. <em>J Biol Chem, </em>291(26), pp. 13509-19.</p>
<p>Campagna-Slater, V. et al., 2011. Structural chemistry of the histone methyltransferases cofactor binding site.. <em>J Chem Inf Model, </em>51(3), pp. 612-23.</p>
<p>Casamassimi, A. et al., 2020. Multifaceted Role of PRDM Proteins in Human Cancer.. <em>Int J Mol Sci., </em>21(7), p. E2648.</p>
<p>Congdon, L., Sims, J., Tuzon, C. & Rice, J., 2014. The PR-Set7 binding domain of Riz1 is required for the H4K20me1-H3K9me1 trans-tail ‘histone code’ and Riz1 tumor suppressor function.. <em>Nucleic Acids Res, </em>42(6), pp. 3580-9.</p>
<p>Couture, J. et al., 2006. Catalytic roles for carbon-oxygen hydrogen bonding in SET domain lysine methyltransferases.. <em>J Biol Chem, </em>281(28), pp. 19280-7.</p>
<p>Dalvit, C. & Vulpetti, A., 2019. Ligand-Based Fluorine NMR Screening: Principles and Applications in Drug Discovery Projects.. <em>J Med Chem, </em>62(5), pp. 2218-44.</p>
<p>Dilworth, D. & Barsyte-Lovejoy, D., 2019. Targeting protein methylation: from chemical tools to precision medicines.. <em>Cell Mol Life Sci, </em>76(15), pp. 2967-85.</p>
<p>Eram, M. et al., 2014. Trimethylation of histone H3 lysine 36 by human methyltransferase PRDM9 protein.. <em>J Biol Chem, </em>289(17), pp. 12177-88.</p>
<p>Gasteiger, E. et al., 2005. Protein Identification and Analysis Tools on the ExPASy Server. In: J. M. Walker, ed. <em>The Proteomics Protocols Handbook. </em>s.l.:Humana Press, pp. 571-607.</p>
<p>Gyory, I. et al., 2004. PRDI-BF1 recruits the histone H3 methyltransferase G9a in transcriptional silencing.. <em>Nat Immunol, </em>5(3), pp. 299-08.</p>
<p>Hoy, S., 2020. Tazemetostat: First Approval.. <em>Drugs, </em>80(5), pp. 513-21.</p>
<p>Ilango, S. et al., 2020. Epigenetic alterations in cancer.. <em>Front Biosci (Landmark Ed), </em>25(1), pp. 1058-109.</p>
<p>Kim, K., Geng, L. & Huang, S., 2003. Inactivation of a histone methyltransferase by mutations in human cancers.. <em>Cancer Res, </em>63(22), pp. 7619-23.</p>
<p>Luan, Y. et al., 2016. Design of a fluorescent ligand targeting the S-adenosylmethionine binding site of the histone methyltransferase MLL1. <em>Org Biomol Chem, </em>14(2), pp. 631-8.</p>
<p>Minnich, M. et al., 2016. Multifunctional role of the transcription factor Blimp-1 in coordinating plasma cell differentiation.. <em>Nat Immunol, </em>17(3), pp. 331-43.</p>
<p>Pinheiro, I. et al., 2012. Prdm3 and Prdm16 are H3K9me1 methyltransferases required for mammalian heterochromatin integrity.. <em>Cell, </em>150(5), pp. 948-60.</p>
<p>Powers, N. et al., 2016. The Meiotic Recombination Activator PRDM9 Trimethylates Both H3K36 and H3K4 at Recombination Hotspots In Vivo.. <em>PLoS Genet, </em>12(6), p. e1006146.</p>
<p>Ribeiro, A. et al., 2019. Emerging concepts in pseudoenzyme classification, evolution, and signaling.. <em>Sci Signal, </em>12(594), p. eaat9797.</p>
<p>Rugo, H. et al., 2020. The Promise for Histone Methyltransferase Inhibitors for Epigenetic Therapy in Clinical Oncology: A Narrative Review.. <em>Adv Ther. , </em>37(7), pp. 3059-82.</p>
<p>Schapira, M., 2011. Structural Chemistry of Human SET Domain Protein Methyltransferases.. <em>Curr Chem Genomics, </em>5(S1), pp. 85-94.</p>
<p>Su, S. et al., 2009. Involvement of histone demethylase LSD1 in Blimp-1-mediated gene repression during plasma cell differentiation.. <em>Mol Cell Biol, </em>29(6), pp. 1421-31.</p>
<p>Wu, H. et al., 2013. Molecular basis for the regulation of the H3K4 methyltransferase activity of PRDM9.. <em>Cell Rep, </em>5(1), pp. 13-20.</p>
<p>Wu, Y. et al., 2008. PRDM6 is enriched in vascular precursors during development and inhibits endothelial cell proliferation, survival, and differentiation.. <em>J Mol Cell Cardiol, </em>44(1), pp. 47-58.</p>
<p>Zhou, B. et al., 2016. PRDM16 Suppresses MLL1r Leukemia via Intrinsic Histone Methyltransferase Activity.. <em>Mol Cell, </em>62(2), pp. 222-36.</p>
<p> </p>
<p><a href="https://zenodo.org/record/4253145#.X6lMCVB7lEY">Download this as a PDF from Zenodo.</a></p>
			
			
						
			
					</div>

		<footer class="entry-footer"><p class="entry-meta"><span class="cat-links"><span class="screen-reader-text">Categories</span><a href="https://thesgc.github.io/static-openlabnotebooks/category/laboratory/cheryl-arrowsmith-univ-of-toronto/" rel="category tag">Cheryl Arrowsmith (Univ. of Toronto)</a>, <a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/danton-ivanochko/" rel="category tag">Danton Ivanochko</a>, <a href="https://thesgc.github.io/static-openlabnotebooks/category/project/prdm-methyltransferases/" rel="category tag">PRDM methyltransferases</a></span></p>		</footer></div>
</article><nav class="navigation post-navigation" role="navigation" aria-label="Posts"><h2 class="screen-reader-text">Post navigation</h2>
		<div class="nav-links"><div class="nav-previous"><a href="https://thesgc.github.io/static-openlabnotebooks/production-and-purification-of-ectodomain-of-sars-cov-2-spike-s-protein/" rel="prev"><span class="meta-nav" aria-hidden="true">← Previous</span> <span class="screen-reader-text">Previous post:</span> <span class="post-title">Production and purification of Ectodomain of SARS-CoV-2 Spike (S) protein</span></a></div><div class="nav-next"><a href="https://thesgc.github.io/static-openlabnotebooks/druggability-and-genetic-variability-of-the-adp-bound-pocket-of-sars-cov-2-rna-dependent-rna-polymerase-niran-domain-across-coronaviruses-and-sars-cov-2-samples-post-22/" rel="next"><span class="meta-nav" aria-hidden="true">Next →</span> <span class="screen-reader-text">Next post:</span> <span class="post-title">Druggability and Genetic Variability of the ADP-bound Pocket of SARS-CoV-2 RNA-dependent RNA polymerase NiRAN domain Across Coronaviruses and SARS-CoV-2 Samples – Post 22</span></a></div></div>
	</nav><div id="comments" class="comments-area">

	
	
	
		<div id="respond" class="comment-respond">
		<h3 id="reply-title" class="comment-reply-title">Leave a Reply <small><a rel="nofollow" id="cancel-comment-reply-link" href="/interrogating-pr-set-domains-for-the-ability-to-bind-sah/#respond" style="display:none;">Cancel reply</a></small></h3><form action="https://thesgc.github.io/static-openlabnotebooks/wp-comments-post.php" method="post" id="commentform" class="comment-form"><p class="comment-notes"><span id="email-notes">Your email address will not be published.</span> Required fields are marked <span class="required">*</span></p><p class="comment-form-comment"><label for="comment">Comment</label> <textarea id="comment" name="comment" cols="45" rows="8" maxlength="65525" required="required"></textarea></p><p class="comment-form-author"><label for="author">Name <span class="required">*</span></label> <input id="author" name="author" type="text" value="" size="30" maxlength="245" required="required"></p>
<p class="comment-form-email"><label for="email">Email <span class="required">*</span></label> <input id="email" name="email" type="text" value="" size="30" maxlength="100" aria-describedby="email-notes" required="required"></p>
<p class="comment-form-url"><label for="url">Website</label> <input id="url" name="url" type="text" value="" size="30" maxlength="200"></p>
<p class="form-submit"><input name="submit" type="submit" id="submit" class="submit" value="Post Comment"><input type="hidden" name="comment_post_ID" value="5380" id="comment_post_ID"><input type="hidden" name="comment_parent" id="comment_parent" value="0"></p><p style="display: none;"><input type="hidden" id="akismet_comment_nonce" name="akismet_comment_nonce" value="078028c33a"></p><p style="display: none;"><input type="hidden" id="ak_js" name="ak_js" value="50"></p></form>	</div>
	
</div>		
		<div class="contact-form-link" id="id-70484"> 
		<a>Contact Danton Ivanochko</a>	</div>
	
	
	<div id="popup-id-70484" title=" Contact Danton Ivanochko" class="read-more-popup inactive">
		<div class="read-more-text"></div>
			<div class="read-more-link"><div role="form" class="wpcf7" id="wpcf7-f545-o1" lang="en-GB" dir="ltr">
<div class="screen-reader-response"><p role="status" aria-live="polite" aria-atomic="true"></p> <ul></ul></div>
<form action="/interrogating-pr-set-domains-for-the-ability-to-bind-sah/#wpcf7-f545-o1" method="post" class="wpcf7-form init" novalidate="novalidate" data-status="init">
<div style="display: none;">
<input type="hidden" name="_wpcf7" value="545"><input type="hidden" name="_wpcf7_version" value="5.3.1"><input type="hidden" name="_wpcf7_locale" value="en_GB"><input type="hidden" name="_wpcf7_unit_tag" value="wpcf7-f545-o1"><input type="hidden" name="_wpcf7_container_post" value="0"><input type="hidden" name="_wpcf7_posted_data_hash" value=""><input type="hidden" name="_wpcf7_recaptcha_response" value=""></div>
<p><label> Your Name (required)<br><span class="wpcf7-form-control-wrap your-name"><input type="text" name="your-name" value="" size="40" class="wpcf7-form-control wpcf7-text wpcf7-validates-as-required" aria-required="true" aria-invalid="false"></span> </label></p>
<p><label> Your Email (required)<br><span class="wpcf7-form-control-wrap your-email"><input type="email" name="your-email" value="" size="40" class="wpcf7-form-control wpcf7-text wpcf7-email wpcf7-validates-as-required wpcf7-validates-as-email" aria-required="true" aria-invalid="false"></span> </label></p>
<p><label> Subject<br><span class="wpcf7-form-control-wrap your-subject"><input type="text" name="your-subject" value="" size="40" class="wpcf7-form-control wpcf7-text" aria-invalid="false"></span> </label></p>
<p><label> Your Message<br><span class="wpcf7-form-control-wrap your-message"><textarea name="your-message" cols="40" rows="4" class="wpcf7-form-control wpcf7-textarea" aria-invalid="false"></textarea></span> </label></p>

<p><input type="submit" value="Send" class="wpcf7-form-control wpcf7-submit"></p>
<p><span class="wpcf7-form-control-wrap post-author-shortcode"><input type="hidden" name="post-author-shortcode" value="dantonivanochko@gmail.com" size="40" class="wpcf7-form-control wpcf7dtx-dynamictext wpcf7-dynamichidden" aria-invalid="false"></span></p>
<div class="wpcf7-response-output" aria-hidden="true"></div></form></div></div>
	</div>
	
	</main><aside class="sidebar sidebar-primary widget-area" role="complementary"><section id="search-2" class="widget widget_search"><div class="widget-wrap">
<form role="search" method="get" class="search-form" action="https://thesgc.github.io/static-openlabnotebooks/">
	<label>
		<span class="screen-reader-text">Search for:</span>
		<input type="search" class="search-field" placeholder="Search..." value="" name="s" title="Search for:"></label>
	<input type="submit" class="search-submit" value="Search"></form>
</div></section><section id="recent-posts-2" class="widget widget_recent_entries"><div class="widget-wrap">
		<h4 class="widget-title">Recent Posts</h4>
		<ul><li>
					<a href="https://thesgc.github.io/static-openlabnotebooks/distinguishing-between-catalytic-and-non-catalytic-pockets-in-the-ligandable-human-genome/">Distinguishing between catalytic and non-catalytic pockets in the ligandable human genome</a>
									</li>
											<li>
					<a href="https://thesgc.github.io/static-openlabnotebooks/druggability-and-genetic-variability-of-the-atpase-site-and-central-channel-of-sars-cov-2-nsp13-helicase-across-coronaviruses-and-sars-cov-2-samples-post-23/">Druggability and Genetic Variability of the ATPase Site and Central Channel of SARS-CoV-2 nsp13 Helicase Across Coronaviruses and SARS-CoV-2 Samples – Post 23</a>
									</li>
											<li>
					<a href="https://thesgc.github.io/static-openlabnotebooks/druggability-and-genetic-variability-of-the-adp-bound-pocket-of-sars-cov-2-rna-dependent-rna-polymerase-niran-domain-across-coronaviruses-and-sars-cov-2-samples-post-22/">Druggability and Genetic Variability of the ADP-bound Pocket of SARS-CoV-2 RNA-dependent RNA polymerase NiRAN domain Across Coronaviruses and SARS-CoV-2 Samples – Post 22</a>
									</li>
											<li>
					<a href="https://thesgc.github.io/static-openlabnotebooks/interrogating-pr-set-domains-for-the-ability-to-bind-sah/" aria-current="page">Interrogating PR/SET domains for the ability to bind SAH</a>
									</li>
											<li>
					<a href="https://thesgc.github.io/static-openlabnotebooks/production-and-purification-of-ectodomain-of-sars-cov-2-spike-s-protein/">Production and purification of Ectodomain of SARS-CoV-2 Spike (S) protein</a>
									</li>
					</ul></div></section><section id="recent-comments-2" class="widget widget_recent_comments"><div class="widget-wrap"><h4 class="widget-title">Recent Comments</h4><ul id="recentcomments"><li class="recentcomments"><span class="comment-author-link">Setayesh Yazdani</span> on <a href="https://thesgc.github.io/static-openlabnotebooks/druggability-and-genetic-variability-of-the-atpase-site-and-central-channel-of-sars-cov-2-nsp13-helicase-across-coronaviruses-and-sars-cov-2-samples-post-23/#comment-1854">Druggability and Genetic Variability of the ATPase Site and Central Channel of SARS-CoV-2 nsp13 Helicase Across Coronaviruses and SARS-CoV-2 Samples – Post 23</a></li><li class="recentcomments"><span class="comment-author-link"><a href="http://DNDi.org" rel="external nofollow ugc" class="url">Charles Mowbray</a></span> on <a href="https://thesgc.github.io/static-openlabnotebooks/druggability-and-genetic-variability-of-the-atpase-site-and-central-channel-of-sars-cov-2-nsp13-helicase-across-coronaviruses-and-sars-cov-2-samples-post-23/#comment-1853">Druggability and Genetic Variability of the ATPase Site and Central Channel of SARS-CoV-2 nsp13 Helicase Across Coronaviruses and SARS-CoV-2 Samples – Post 23</a></li><li class="recentcomments"><span class="comment-author-link">Elizabeth Brown</span> on <a href="https://thesgc.github.io/static-openlabnotebooks/shockingly-effective-electroporation-of-ribonucleoprotein-complexes-into-dipg-cells/#comment-1560">Shockingly effective – electroporation of ribonucleoprotein complexes into DIPG cells</a></li><li class="recentcomments"><span class="comment-author-link">Elizabeth Brown</span> on <a href="https://thesgc.github.io/static-openlabnotebooks/shockingly-effective-electroporation-of-ribonucleoprotein-complexes-into-dipg-cells/#comment-1531">Shockingly effective – electroporation of ribonucleoprotein complexes into DIPG cells</a></li><li class="recentcomments"><span class="comment-author-link">Jong Fu</span> on <a href="https://thesgc.github.io/static-openlabnotebooks/hdac11-target-engagement-assay-development-in-cells-cetsa/#comment-1478">HDAC11 target engagement assay development in cells – CETSA</a></li></ul></div></section><section id="archives-2" class="widget widget_archive"><div class="widget-wrap"><h4 class="widget-title">Archives</h4>
			<ul><li><a href="https://thesgc.github.io/static-openlabnotebooks/2020/11/">November 2020</a></li>
	<li><a href="https://thesgc.github.io/static-openlabnotebooks/2020/10/">October 2020</a></li>
	<li><a href="https://thesgc.github.io/static-openlabnotebooks/2020/09/">September 2020</a></li>
	<li><a href="https://thesgc.github.io/static-openlabnotebooks/2020/08/">August 2020</a></li>
	<li><a href="https://thesgc.github.io/static-openlabnotebooks/2020/07/">July 2020</a></li>
	<li><a href="https://thesgc.github.io/static-openlabnotebooks/2020/06/">June 2020</a></li>
	<li><a href="https://thesgc.github.io/static-openlabnotebooks/2020/05/">May 2020</a></li>
	<li><a href="https://thesgc.github.io/static-openlabnotebooks/2020/04/">April 2020</a></li>
	<li><a href="https://thesgc.github.io/static-openlabnotebooks/2020/03/">March 2020</a></li>
	<li><a href="https://thesgc.github.io/static-openlabnotebooks/2020/02/">February 2020</a></li>
	<li><a href="https://thesgc.github.io/static-openlabnotebooks/2020/01/">January 2020</a></li>
	<li><a href="https://thesgc.github.io/static-openlabnotebooks/2019/12/">December 2019</a></li>
	<li><a href="https://thesgc.github.io/static-openlabnotebooks/2019/11/">November 2019</a></li>
	<li><a href="https://thesgc.github.io/static-openlabnotebooks/2019/10/">October 2019</a></li>
	<li><a href="https://thesgc.github.io/static-openlabnotebooks/2019/09/">September 2019</a></li>
	<li><a href="https://thesgc.github.io/static-openlabnotebooks/2019/08/">August 2019</a></li>
	<li><a href="https://thesgc.github.io/static-openlabnotebooks/2019/07/">July 2019</a></li>
	<li><a href="https://thesgc.github.io/static-openlabnotebooks/2019/06/">June 2019</a></li>
	<li><a href="https://thesgc.github.io/static-openlabnotebooks/2019/05/">May 2019</a></li>
	<li><a href="https://thesgc.github.io/static-openlabnotebooks/2019/04/">April 2019</a></li>
	<li><a href="https://thesgc.github.io/static-openlabnotebooks/2019/03/">March 2019</a></li>
	<li><a href="https://thesgc.github.io/static-openlabnotebooks/2019/02/">February 2019</a></li>
	<li><a href="https://thesgc.github.io/static-openlabnotebooks/2019/01/">January 2019</a></li>
	<li><a href="https://thesgc.github.io/static-openlabnotebooks/2018/12/">December 2018</a></li>
	<li><a href="https://thesgc.github.io/static-openlabnotebooks/2018/11/">November 2018</a></li>
	<li><a href="https://thesgc.github.io/static-openlabnotebooks/2018/10/">October 2018</a></li>
	<li><a href="https://thesgc.github.io/static-openlabnotebooks/2018/09/">September 2018</a></li>
	<li><a href="https://thesgc.github.io/static-openlabnotebooks/2018/08/">August 2018</a></li>
	<li><a href="https://thesgc.github.io/static-openlabnotebooks/2018/07/">July 2018</a></li>
	<li><a href="https://thesgc.github.io/static-openlabnotebooks/2018/06/">June 2018</a></li>
	<li><a href="https://thesgc.github.io/static-openlabnotebooks/2018/05/">May 2018</a></li>
	<li><a href="https://thesgc.github.io/static-openlabnotebooks/2018/04/">April 2018</a></li>
	<li><a href="https://thesgc.github.io/static-openlabnotebooks/2018/03/">March 2018</a></li>
	<li><a href="https://thesgc.github.io/static-openlabnotebooks/2018/02/">February 2018</a></li>
	<li><a href="https://thesgc.github.io/static-openlabnotebooks/2018/01/">January 2018</a></li>
	<li><a href="https://thesgc.github.io/static-openlabnotebooks/2017/12/">December 2017</a></li>
	<li><a href="https://thesgc.github.io/static-openlabnotebooks/2017/11/">November 2017</a></li>
	<li><a href="https://thesgc.github.io/static-openlabnotebooks/2017/10/">October 2017</a></li>
	<li><a href="https://thesgc.github.io/static-openlabnotebooks/2017/09/">September 2017</a></li>
	<li><a href="https://thesgc.github.io/static-openlabnotebooks/2017/08/">August 2017</a></li>
	<li><a href="https://thesgc.github.io/static-openlabnotebooks/2017/07/">July 2017</a></li>
			</ul></div></section><section id="categories-2" class="widget widget_categories"><div class="widget-wrap"><h4 class="widget-title">Categories</h4>
			<ul><li class="cat-item cat-item-28"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/acvr1-alk2/">ACVR1/ALK2</a>
</li>
	<li class="cat-item cat-item-11"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/alfredo-picado/" title="My name is Alfredo Picado and I am a native of Costa Rica. I studied chemistry at University of Costa Rica before moving to the USA for graduate school. I obtained my Ph.D. in Organic Chemistry from Clemson University in South Carolina under the supervision of Dr. Karl Dieter.  My research was focused on the development of new organocopper reagents for synthesis of highly functionalized synthons and heterocycles.

During my graduate studies I spent 1 year at GlaxoSmithKline in Philadelphia, working on the synthesis of small molecule inhibitors of ion channels involved in heart failure. During this time I developed a keen interest in medicinal chemistry and drug discovery.

After finishing my doctoral degree, I taught organic chemistry and biochemistry at Pfeiffer University, a private college in North Carolina. 

Currently, I am a postdoctoral researcher in the SGC at the University of North Carolina Chapel Hill working under the supervision of Dr. David Drewry. I am currently developing chemical probes for the understudied protein kinases MST1-4 and MAP3K2. My research is jointly funded by the NIH and Luceome Biotechnologies.

Alfredo Picado, Ph.D.">Alfredo Picado</a>
</li>
	<li class="cat-item cat-item-149"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/alma-seitova/" title="Alma received her MSc degree from Kazakh State University and her PhD from Institute of General Genetics at the Academy of Science in Moscow, Russia. She joined SGC, Toronto in 2005. Currently, Alma is leading the Eukaryotic Expression Platform (EEP) dedicated to the high-throughput (HTP) test expression and production of challenging proteins/protein complexes, which require eukaryotic expression machinery including Baculovirus mediated and BacMam/Mammalian expression systems. The EEP team optimizes a pipeline that includes designing expression constructs, choosing expression systems, cloning, test expression, production, and purification of milligram quantities of various recombinant proteins. The EEP platform is currently producing high-quality SARS-CoV-2 ( COVID19) related proteins, specifically S-Ectodomain, RBD of the Spike and human ACE2, as a part of The Toronto Open Access COVID-19 Protein Manufacturing Center and makes proteins available to any Canadian scientist conducting COVID-19 research (www.youtube.com/watch?v=bYkoyX5T9fE).">Alma Seitova</a>
</li>
	<li class="cat-item cat-item-65"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/als-reproducible-antibody-platform-als-rap/">ALS-Reproducible Antibody Platform (ALS-RAP)</a>
</li>
	<li class="cat-item cat-item-55"><a href="https://thesgc.github.io/static-openlabnotebooks/category/disease-foundation/alzheimers-disease/">Alzheimer’s disease</a>
</li>
	<li class="cat-item cat-item-115"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/amy-ramsey/" title="Dr. Ramsey is an Associate Professor in the Department of Pharmacology and Toxicology at the University of Toronto. She earned a B.S. in Microbiology from the University of Kentucky, and a Ph.D. in Genetics and Molecular Biology from the University of North Carolina at Chapel Hill. Her postdoctoral training was conducted with Dr. Marc Caron at Duke University Medical Center. As a graduate student at UNC, she developed a mouse model with a genetic deficiency of NMDA receptors caused by a mutation in Grin1. Since the creation of the mice in 1999, Dr. Ramsey has studied the physiological and behavioural consequences of Grin1 loss-of-function. She began working with GRIN families in 2018 to discover treatments for patients using mice that carry disease-causing mutations in Grin1, and serves a member of the scientific advisory board for CureGRIN.">Amy Ramsey</a>
</li>
	<li class="cat-item cat-item-116"><a href="https://thesgc.github.io/static-openlabnotebooks/category/laboratory/amy-ramsey-univ-of-toronto/">Amy Ramsey (Univ. of Toronto)</a>
</li>
	<li class="cat-item cat-item-64"><a href="https://thesgc.github.io/static-openlabnotebooks/category/disease-foundation/amyotrophic-lateral-sclerosis-als/">Amyotrophic Lateral Sclerosis (ALS)</a>
</li>
	<li class="cat-item cat-item-46"><a href="https://thesgc.github.io/static-openlabnotebooks/category/past-scientist/antoine-claessens/" title="Antoine Claessens is an INSERM researcher working on malaria genomics and transcriptomics. Biochemist by training, his interest in the Plasmodium parasite started with his PhD at Edinburgh university during which he developed an in vitro model for cerebral malaria. After a post-doc at the Sanger Institute on parasite mutation rate using whole genome sequencing, he was awarded a fellowship based at the London School of Hygiene & Tropical Medicine and at the MRC-Gambia. In Gambia, he recruited a unique cohort of P. falciparum infected asymptomatic carriers to study host-pathogen interactions over a long period of time. From the bench to bioinformatics via the bush, he is now an all-rounder malariologist starting his own group at MIVEGEC, Montpellier, France.  He recently joined the ‘Open Science movement’ as he is convinced that better sharing of data and methods across scientists can only speed up the elimination of malaria.">Antoine Claessens</a>
</li>
	<li class="cat-item cat-item-101"><a href="https://thesgc.github.io/static-openlabnotebooks/category/laboratory/antoine-claessens-laboratories/">Antoine Claessens</a>
</li>
	<li class="cat-item cat-item-121"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/ashley-hutchinson/" title="Ashley earned her advanced diploma form Sir Sandford Fleming College, where she was awarded the Dean’s Award, for top marks and for her commitment to the field of community and science. Ashley joined SGC under the tutelage of Dr. Alma Seitova to work on Eukaryotic Expression platform, including Baculovirus mediated expression system for protein production and purification. Currently Ashley is involved in structural characterization of WDR41-SMCR8-C9ORF72 trimeric complex under the supervision of Dr. Levon Halabelian, which brings us to the open lab notebook.">Ashley Hutchinson</a>
</li>
	<li class="cat-item cat-item-42"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/ben-eduful/" title="Benjamin Eduful studied chemistry at the University of Cape Coast, Ghana, and with a strong interest in drug discovery and development, and the study of the interactions between xenobiotics and living systems, he pursued a MS in Clinical Pharmacology at the University of Aberdeen, Scotland, UK.

He completed his PhD in Medicinal Chemistry at the University of South Florida under the direct supervision of Prof. James W. Leahy. Benjamin’s dissertation was focused on the design, synthesis and biological evaluation of small molecules as treatment for infectious and neurodegenerative diseases.

He joined SGC at the University of North Carolina as a postdoctoral researcher in April 2018, working under Professor David Drewry. Benjamin’s research work is focused on the design and synthesis of small-molecule inhibitors for historically understudied protein kinases (specifically the Calcium/calmodulin-dependent protein kinase kinase 2, CaMKK2).">Ben Eduful</a>
</li>
	<li class="cat-item cat-item-5"><a href="https://thesgc.github.io/static-openlabnotebooks/category/disease-foundation/tbtc/">Brain Tumours</a>
</li>
	<li class="cat-item cat-item-114"><a href="https://thesgc.github.io/static-openlabnotebooks/category/past-scientist/carmen-jimenez-antunez/" title="Carmen is a postdoctoral researcher at the SGC Oxford. She completed her PhD in Medicinal Chemistry at Cardiff University. Her doctoral work focused on the design, synthesis and evaluation of new prodrugs that would enhance the anticancer activity of nucleoside analogues.
In 2018 she joined the SGC working in the lab of Professor Paul Brennan and Dr Opher Gileadi. Carmen works as part of a multidisciplinary project for the development of target enable packages helping the discovery of drug development for Alzheimer Research.">Carmen Jimenez Antunez</a>
</li>
	<li class="cat-item cat-item-66"><a href="https://thesgc.github.io/static-openlabnotebooks/category/past-scientist/carolyn-marks/" title="Carolyn Marks is Biotechnology Research Engineer at SGC Karolinska in Stockholm, Sweden. She obtained her PhD from the NIH-Karolinska Collaborative Doctoral Program in Neuroscience. Carolyn completed her Post-Doc at Karolinska & Science for Life Laboratory characterizing the ubiquitin proteasome system as therapeutic targets in neurodegenerative diseases. Carolyn has recently joined Susanne Gräslund and the SGC Karolinska Recombinant Antibody Production Team. At SGC Karolinska, Carolyn will primarily produce and validate gold standard antibodies for the ALS Reproducible Antibody Platform. Carolyn will merge the expertise of the SGC with the Human Antibody Therapeutics facility at the Drug Development and Discovery Platform of Science for Life Laboratory in Stockholm to generate, identify, and characterize the highest quality, renewable, recombinant antibodies for ALS-relevant antigens and make them freely available to the ALS research community.">Carolyn Marks</a>
</li>
	<li class="cat-item cat-item-102"><a href="https://thesgc.github.io/static-openlabnotebooks/category/laboratory/unc-catalyst/">Catalyst UNC Chapel Hill</a>
</li>
	<li class="cat-item cat-item-131"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/cell-assay-reports/">Cell assay reports</a>
</li>
	<li class="cat-item cat-item-155"><a href="https://thesgc.github.io/static-openlabnotebooks/category/laboratory/cheryl-arrowsmith-univ-of-toronto/">Cheryl Arrowsmith (Univ. of Toronto)</a>
</li>
	<li class="cat-item cat-item-44"><a href="https://thesgc.github.io/static-openlabnotebooks/category/disease-foundation/chordoma/">Chordoma</a>
</li>
	<li class="cat-item cat-item-129"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/chromatin-factors/">Chromatin factors</a>
</li>
	<li class="cat-item cat-item-104"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/claudia-alvarez/" title="Claudia completed her PhD in Chemistry in 2018 under the supervision of Dr. Voula Kanelis at the University of Toronto. She studied the structural and functional consequences of diabetes- and hyperinsulinism-causing mutations in an ABC transporter protein called the sulfonylurea receptor using NMR, fluorescence spectroscopy and other biophysical techniques.
She is working as a postdoctoral fellow at the SGC under the supervision of Dr. Cheryl Arrowsmith. Claudia's research focuses on determining the crystal structure and function of individual domains of the huntingtin protein, in order to elucidate its role in Huntington disease.">Claudia Alvarez</a>
</li>
	<li class="cat-item cat-item-139"><a href="https://thesgc.github.io/static-openlabnotebooks/category/disease-foundation/covid-19/">COVID-19</a>
</li>
	<li class="cat-item cat-item-153"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/danton-ivanochko/" title="Danton is a PhD candidate in the department of Medical Biophysics at the University of Toronto. Working under the supervision of Dr. Cheryl Arrowsmith, Danton’s PhD research has employed structural biology, biophysics, and bioinformatics, to uncover important biology related to methyltransferase enzymes. Additionally, Danton received a Mitacs Accelerate Fellowship to work on a PROTAC-development project at Boehringer Ingelheim in Vienna, Austria. Prior to coming to Toronto, Danton received his BSc in Biochemistry and an MSc in Human Molecular Genetics from the University of Ottawa under the supervision of Dr. David Picketts, where he studied how epigenetic mechanisms regulate neurodevelopment.">Danton Ivanochko</a>
</li>
	<li class="cat-item cat-item-24"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/david-dilworth/" title="David is a postdoctoral fellow in the Cell Biology Group at the Structural Genomics Consortium Toronto. His research focuses on the epigenetic determinants of cancer, with an interest in how the transcriptional networks that drive the disease are established and maintained. David received his PhD in Biochemistry from the University of Victoria, Canada. His doctoral work focused on a noncovalent chromatin modification called proline isomerization. This research identified several novel functions for the nuclear proline isomerase FKBP25 in the maintenance of genomic stability. At the SGC, David will be using chemical probes (ie. precision inhibitors) that target epigenetic pathways relevant to human disease to study chromatin-centric phenomenon.

A complete list of David’s publications can be found here.">David Dilworth</a>
</li>
	<li class="cat-item cat-item-132"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/david-drewry/" title="David Drewry, Ph.D., is a leader in the medicinal chemistry of protein kinases and is one of the principal architects of the research strategy at the SGC-UNC to build an open and collaborative research network to promote target discovery. He previously enjoyed more than 24 years as a medicinal chemist with GlaxoSmithKline and legacy companies, where he led teams working across the preclinical spectrum of drug discovery. His research interests include the art and science of medicinal chemistry, kinase inhibitor design, utilization of annotated sets of kinase inhibitors to build understanding of signaling networks and precompetitive chemical biology to facilitate target identification. After earning a Bachelor’s of Science degree, cum laude, in chemistry from Yale University, Drewry earned his doctorate at the University of California, Berkeley in the laboratory of Paul Bartlett, working on the design, synthesis and mechanistic studies of zinc protease inhibitors. David spent one year as the head of chemistry at Meryx Pharmaceuticals, a biotech startup focused on small-molecule inhibitors of Mer kinase that was a spinoff from the UNC Eshelman School of Pharmacy. Now at the SGC-UNC in the UNC Eshelman School of Pharmacy, he focuses on building the KCGS (Kinase Chemogenomics Set, more information here  https://www.sgc-unc.org/kcgs), design of chemical probes for understudied kinases, and finding treatments for the rare cancer chordoma. For chordoma, his lab is working on discovery of ligands that bind to the transcription factor TBXT, a key vulnerability of chordoma. This collaborative lead discovery project incorporates x-ray crystallography and structure based drug design, organic synthesis and iterative medicinal chemistry, and biophysical assays to measure affinity.">David Drewry</a>
</li>
	<li class="cat-item cat-item-90"><a href="https://thesgc.github.io/static-openlabnotebooks/category/laboratory/diamond-light-source/">Diamond Light Source</a>
</li>
	<li class="cat-item cat-item-15"><a href="https://thesgc.github.io/static-openlabnotebooks/category/disease-foundation/dipg/">Diffuse intrinsic pontine glioma</a>
</li>
	<li class="cat-item cat-item-58"><a href="https://thesgc.github.io/static-openlabnotebooks/category/disease-foundation/dravet-syndrome/">Dravet syndrome</a>
</li>
	<li class="cat-item cat-item-86"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/e3-ligases/">E3 ligases</a>
</li>
	<li class="cat-item cat-item-43"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/egfr-and-gak/">EGFR and GAK</a>
</li>
	<li class="cat-item cat-item-39"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/ellie-williams/" title="Ellie Williams studied the structure and dynamics of HSP90 at University College London. After graduation she came to work at the SGC and has been working in the lab of Alex Bullock since 2010. Her work has been focusing on understanding the mutations in ALK2 which cause the rare connective tissue disorder Fibrodysplasia Ossificans Progressiva (FOP) – She has spent her time looking at the structure of ALK2 by x-ray crystallography, characterising the binding of drug molecules and compounds to the protein, investigating the specificity of binding, looking at the interaction of ALK2 with its interacting partners and characterising the effect of mutations on binding and activity.  Her work is funded by the UK charity, FOP Friends. 
Since 2015 she has also been responsible for the public engagement activity within the SGC in Oxford, organising schools visits, festival participation and art exhibitions. When not in the lab Ellie spends her time costume making, reading and looking after her daughter.
">Ellie Williams</a>
</li>
	<li class="cat-item cat-item-125"><a href="https://thesgc.github.io/static-openlabnotebooks/category/disease-foundation/ependymoma/">Ependymoma</a>
</li>
	<li class="cat-item cat-item-78"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/epigenetic-targets-in-medulloblastoma/">Epigenetic targets in Medulloblastoma</a>
</li>
	<li class="cat-item cat-item-152"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/evianne-rovers/" title="Evianne graduated from Amsterdam University College with a B.Sc Liberal Arts and Sciences with Distinction in the tracks Biomedical Sciences and Chemistry in July 2020 and is now a M.Sc student with Direct-entry in the PhD program at the University of Toronto, Department of Pharmacology and Toxicology. At the Structural Genomics Consortium, she is working under the supervision of Matthieu Schapira on a bioinformatics project called the Ligandable Human Genome. In this project, potential binding pockets are identified and analyzed in the known PDB structures of the human genome. The next step is to determine if a pocket is either catalytic or non-catalytic by using the MCSA and Interpro database. The goal for this project is to identify non-catalytic pockets that could potentially be exploited for the design of ProxPharm chemicals (i.e. PROTACs). Outside the lab, Evianne enjoys playing for the Varsity Women’s Field Hockey team of UofT, going on hiking trips, being first year representative for thesis-based pharmacology program and spending time with friends and family.">Evianne Rovers</a>
</li>
	<li class="cat-item cat-item-124"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/ezhip/">EZHIP</a>
</li>
	<li class="cat-item cat-item-6"><a href="https://thesgc.github.io/static-openlabnotebooks/category/disease-foundation/fop/">Fibrodysplasia ossificans progressiva</a>
</li>
	<li class="cat-item cat-item-57"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/gene-therapy-for-dravet-syndrome/">Gene therapy for Dravet Syndrome</a>
</li>
	<li class="cat-item cat-item-26"><a href="https://thesgc.github.io/static-openlabnotebooks/category/past-scientist/genna-luciani/" title="Genna received her PhD from the University of Toronto in the department of Molecular Genetics. Her doctoral research focused on small molecule identification and characterization using the nematode C. elegans as a model system.  She joined the SGC in 2015 in the cell biology group at SGC Toronto. In collaboration with Dr. Mark Minden’s lab, she is investigating the effect of SGC’s epigenetic chemical probes in poor prognosis acute myeloid leukemia patient cells.Her open lab notebook will focus on characterizing chemical probes targeting the polycomb repressive complex 2 (PRC2) in AML. This project is funded by the Leukemia and Lymphoma Society of Canada (LLSC). Outside of the lab, Genna enjoys all things food related, reading, crafting and obsessing over her many favourite animals.">Genna Luciani</a>
</li>
	<li class="cat-item cat-item-117"><a href="https://thesgc.github.io/static-openlabnotebooks/category/disease-foundation/grin-disorder/">GRIN disorder</a>
</li>
	<li class="cat-item cat-item-54"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/gtpase-gef-complex/">GTPase-GEF complex</a>
</li>
	<li class="cat-item cat-item-138"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/hadley-sheppard/" title=" 
Hadley began studying chordoma during her PhD at Baylor College of Medicine. Since completing her program, Hadley is an incoming Post-Doctoral Fellow in the lab of Paul Workman at the Institute of Cancer Research (ICR). Together in collaboration with the labs of Opher Gileadi and David Drewry, Hadley and the team at ICR will work to directly drug the transcription factor brachyury - a unique transcriptional addiction in chordoma.">Hadley Sheppard</a>
</li>
	<li class="cat-item cat-item-51"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/hao1/">HAO1</a>
</li>
	<li class="cat-item cat-item-76"><a href="https://thesgc.github.io/static-openlabnotebooks/category/past-scientist/haresh-ajani/" title="Haresh received his Ph.D. in computational chemistry from the Palacky University, and Institute of Organic Chemistry and Biochemistry AS CR, Prague, Czech Republic in 2018. He completed his Ph.D. in structure based drug design, focusing on receptor-ligand scoring functions based on semi-empirical quantum mechanics (SQM).  He joined the Research Informatics group at SGC-Toronto in 2019 as a post-doctoral fellow, where he is contributing to the discovery of ligands for proteins involved in the ubiquitin proteasome system, and assesses in silico methods for the structure-based design of proteolysis targeting chimeras (PROTACs).">Haresh Ajani</a>
</li>
	<li class="cat-item cat-item-32"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/hdacs/">HDACs</a>
</li>
	<li class="cat-item cat-item-37"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/huntingtin/">Huntingtin</a>
</li>
	<li class="cat-item cat-item-3"><a href="https://thesgc.github.io/static-openlabnotebooks/category/disease-foundation/chdi/">Huntington’s disease</a>
</li>
	<li class="cat-item cat-item-77"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/in-silico-protacs-design/">In silico PROTACs design</a>
</li>
	<li class="cat-item cat-item-73"><a href="https://thesgc.github.io/static-openlabnotebooks/category/disease-foundation/ipsc-disease-modelling/">iPSC Disease modelling</a>
</li>
	<li class="cat-item cat-item-70"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/ipscs/">iPSCs</a>
</li>
	<li class="cat-item cat-item-87"><a href="https://thesgc.github.io/static-openlabnotebooks/category/past-scientist/jacob-mcauley/" title="Jacob McAuley is going into his third year at the University of Toronto completing his double major in Molecular Genetics and Immunology. Over the Summer, he will be working in the SGC under the guidance of Dr. Rachel Harding in the lab of Dr. Cheryl Arrowsmith. During his time at the SGC, he will focus on the huntingtin project, working to purify possible interaction partners of Huntingtin protein to aid in the understanding of its functions within cells. Mutations in this protein result in the development of Huntington’s Disease, a rare neurological disorder that currently has no cure. In the future, Jacob McAuley plans to pursue a carrier in the sciences with the eventual goal of obtaining a Ph.D. centered around genetic diseases such as Huntington’s.">Jacob McAuley</a>
</li>
	<li class="cat-item cat-item-30"><a href="https://thesgc.github.io/static-openlabnotebooks/category/past-scientist/jolene-ho/" title="Jolene Ho obtained her Ph.D. in Cancer Biology under Prof Lorenz Poellinger’s lab at the Cancer Science Institute (CSI) of Singapore, National University of Singapore (NUS). After a postdoc in the same lab, she worked in Veredus Laboratories, a diagnostics company in Singapore. Currently, she is a postdoctoral fellow at the Structural Genomics Consortium (SGC) in Toronto, in the lab of Takis Prinos.

Her projects are mostly focused on the role of regulatory proteins in cancer and other neurological disorders, and she has been studying the effects of molecular inhibitors on various regulators through methods such as qPCR, Western blotting, tissue culture and other functional in vitro assays. Currently, she is investigating mutations in huntingtin (HTT), which are heavily implicated in the onset of Huntington’s disease.">Jolene Ho</a>
</li>
	<li class="cat-item cat-item-21"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/jong-fu-wong/" title="Jong Fu Wong studied the novel cytosolic roles of epigenetic modifier EZH2 for his PhD at Nanyang Technological University (NTU), Singapore. After graduating, he decided to delve into more application-oriented studies. He now works for the Structural Genomics Consortium (SGC) at Oxford University as a Cell Biologist under the supervision of Associate Professor Alex Bullock.He is funded by The Brain Tumour Charity to help expedite the development of treatment for Diffused Intrinsic Pontine Gliomas (DIPG), a rare form of brain cancer that occurs in young children. Specifically, using patient-derived DIPG cell lines, he is screening through iterations of compounds rationally-designed by Professor Paul Brennan’s team based in Target Discovery Institute (TDI) to identify ones that can potently yet selectively inhibit ACVR1/ALK2, a kinase frequently mutated in DIPG. Similar ACVR1/ALK2 mutations are also responsible for the development of Fibrodysplasia Ossificans Progressiva (FOP), another rare genetic disease where connective tissues progressively ossified, freezing joints permanently and restrict movement. Therefore, compounds identified from the screens will also contribute to the development of FOP treatment.Besides that, he is interested to unravel some of the underlying mechanistic causes of DIPG and to explore the prospect of combination treatment. His many alter egos include an aspiring aquarist, a food junky and a dummy Judoka.">Jong Fu Wong</a>
</li>
	<li class="cat-item cat-item-62"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/jose-brandao-neto/" title="The main focus of my current research is twofold:

i) the nucleotide metabolism of Schistosoma mansoni using structural molecular biology, an effort that has led to the determination of 8 novel structures and a total of 19 unique proteins solubilised between 2014 and 2018.
ii) improving the quality of crystallographic models by introducing excited states, which will allow for direct interpretation of behaviour of metal centres in XFEL and synchrotron alike, a step beyond what is currently done through indirect assessment with multiple spectroscopic techniques.

At the beamline level, I am interested in methods for improving the throughput while keeping the same data quality level, optimising the usage of the hardware, and developing standards for quantitative assessment of diffraction from protein crystals.">Jose Brandao-Neto</a>
</li>
	<li class="cat-item cat-item-146"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/joseph-newman/" title="I studied my undergraduate degree in Biochemistry at the University of Edinburgh and did a PhD in crystallography in the University of Sheffield working in the lab of Prof David Rice. Following my PhD I did a postdoc at the University of Newcastle with Prof Rick Lewis. I am currently a senior Post Doctoral Scientist based in the laboratory of Prof Opher Gileadi at the Structural Genomics Consortium, University of Oxford, UK. My research is focused on determining the structures and mechanisms of human proteins of therapeutic interest with a particular focus on proteins involved in the maintenance of genome integrity and DNA repair pathways. I have a broad background in biochemistry and specific expertise in structural biology primarily through the field of X-ray crystallography, which has been my primary research tool throughout my PhD and postdoctoral studies.">Joseph Newman</a>
</li>
	<li class="cat-item cat-item-119"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/katheron-intson/" title="Kath is a PhD candidate at the University of Toronto. While earning a BScH from Queen’s University, she began working under Dr. Amy Ramsey to study potential drug treatments for abnormal behaviours in Grin1 loss-of-function mice. Since beginning her doctorate with Amy, she has focused on contributions of different populations of NMDA-receptor-containing cells to brain function and the physiological consequences of social isolation.">Katheron Intson</a>
</li>
	<li class="cat-item cat-item-23"><a href="https://thesgc.github.io/static-openlabnotebooks/category/disease-foundation/leukemia/">Leukemia</a>
</li>
	<li class="cat-item cat-item-128"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/ligandable-genome/">Ligandable Genome</a>
</li>
	<li class="cat-item cat-item-20"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/liz-brown/" title="I grew up in Northern Ireland and moved to Cambridge, England to do undergraduate degrees in Pathology and Biochemistry. After three years of study, and two months of research experience at the Karolinska Institute in Stockholm, I realised I wanted to focus my research on cancer. From there I completed undergraduate projects in mammary cell death, mammary tumour 3D culture, and a Master's project focused on the effect of heterogeneity on mammary tumours as well as training an in silico model of signalling within the mammary epithelium.
I am now completing my PhD studies in the Structural Genomics Consortium in Oxford under the supervision of Alex Bullock. The focus of my study is the effect of ACVR1 mutations in diffuse intrinsic pontine glioma (DIPG), a paediatric brainstem tumour, and fibrodysplasia ossificans progressiva (FOP), a genetic disorder that causes extraskeletal bone formation. I aim to use patient derived cell lines to characterise how the mutation changes intracellular signalling and the behaviour of DIPG and FOP cells in a way that leads to disease.
When I'm not making science, I can be found powerlifting in the gym, writing silly stories, or cosied up watching one of my favourite films.
">Liz Brown</a>
</li>
	<li class="cat-item cat-item-67"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/louisa-temme/" title="Louisa is a Postdoctoral Research Associate in the SGC at the University of North Carolina at Chapel Hill. She is developing chemical probes for understudied protein kinases under the supervision of Prof. David Drewry. Louisa is a pharmacist and obtained her doctoral degree in medicinal chemistry from Westfälische-Wilhelms-Universität Münster, Germany. In the working group of Prof. Bernhard Wünsch she gained experience in organic synthesis, medicinal chemistry and functional assay establishment during the development of novel neuroprotective molecules for the treatment of Morbus Alzheimer’s and Parkinson’s disease. To identify and analyze drug-target interactions of these negative allosteric modulators for the ifenprodil binding site of the GluN1a/GluN2B NMDA receptor she additionally learned and performed in silico docking and virtual screening of chemical libraries during a research stay at the lab of Prof. Wolfgang Sippl (Martin-Luther-Universität Halle-Wittenberg, Germany).">Louisa Temme</a>
</li>
	<li class="cat-item cat-item-80"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/magdalena-szewczyk/" title="Magda received her M.Sc. degree from University of Lodz (Poland) and PhD degree from McMaster University (Canada). Currently, she is a research associate in the Cell Biology Group at the Structural Genomics Consortium Toronto. At the SGC, Magda is developing cellular assays for the validation of chemical probes of epigenetic modifiers and readers as well as researching their role in cancer cell biology. Since joining SGC-Toronto team in 2014, Magda has developed several assays for testing the enzymatic activity of PRMT1,3,5,6,8, 9 as well as SMYD2. She also utilized NanoLuc-Halo tag system for protein-protein interaction assay development of L3MBTL1/histoneH3, NSD3/histoneH3, RBBP4,7/MTA2, YAP1/TEAD1 and HDAC6/ISG15. Other than research Magda loves hiking, camping, traveling and organizing fundraising events. Her open lab book will focus on target validation and assay development for variety of epigenetic targets.">Magdalena Szewczyk</a>
</li>
	<li class="cat-item cat-item-45"><a href="https://thesgc.github.io/static-openlabnotebooks/category/disease-foundation/malaria/">Malaria</a>
</li>
	<li class="cat-item cat-item-13"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/mandeep-mann/" title="Mandeep graduated from the University of Waterloo with a B.Sc in Honor’s Biochemistry in 2016 and is now a PhD candidate at the University of Toronto, Department of Pharmacology and Toxicology. She is working at the Structural Genomics Consortium, supervised by Dr. Matthieu Schapira, developing drug-like inhibitors against oncology targets. When Mandeep is not in the lab, she enjoys playing volleyball with friends, reading and travelling.

USP5 Project
Ubiquitin specific proteases (USPs) are the largest family of deubiquitinases and are responsible for removing ubiquitin from proteins. USPs have been on the radar of the drug discovery community for over a decade for the treatment of diseases such as cancer and neurodegeneration, but, with a few, rare exceptions, efforts to develop catalytic inhibitors have all failed thus far. In addition to the catalytic domain, some USPs, such as USP5, contain a zinc-finger ubiquitin binding domain (Zf-UBD). The goal of this research project is to develop ZnF-UBD chemical probes to see if pharmacologically targeting the non-catalytic domain of USP5 is a valid strategy. To the best of our knowledge, this is the first ever chemical biology open notebook.">Mandeep Mann</a>
</li>
	<li class="cat-item cat-item-91"><a href="https://thesgc.github.io/static-openlabnotebooks/category/laboratory/michael-taylor-sickkids/">Michael Taylor (SickKids Hospital Toronto)</a>
</li>
	<li class="cat-item cat-item-92"><a href="https://thesgc.github.io/static-openlabnotebooks/category/laboratory/montreal-neurological-institute/">Montreal Neurological Institute</a>
</li>
	<li class="cat-item cat-item-141"><a href="https://thesgc.github.io/static-openlabnotebooks/category/laboratory/nick-goldman-ebi/">Nick Goldman (EBI)</a>
</li>
	<li class="cat-item cat-item-140"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/nicola-de-maio/" title="Nicola De Maio studied Mathematics in Italy, and completed his PhD in phyloenetics and population genetics at the VetMedUni Vienna. Between 2013 and 2017 Nicola was a postdoct in the group of Prof Daniel Wilson in Oxford, sutdying within and between-host evolution of viruses and bacteria, and developing methods to infer transmission between individuals and across countries using sequencing and epidemiological data. Nicola is now a Scientist in molecular evolution at the European Bioinformatics Institute (EMBL-EBI) in Hinxton, UK. Nicola develops mathematical models and software to study evolution from DNA sequence data, in particular working in phylogenetic, alignment, and phylogeographic inference.">Nicola De Maio</a>
</li>
	<li class="cat-item cat-item-118"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/nmda/">NMDA</a>
</li>
	<li class="cat-item cat-item-29"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/nsd3/">NSD3</a>
</li>
	<li class="cat-item cat-item-49"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/open-secretion-rescue/">Open Secretion Rescue</a>
</li>
	<li class="cat-item cat-item-68"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/opher-gileadi/" title="I grew up in Israel and studied for my BSc and PhD at the Hebrew University in Jerusalem. I then did a post-doc in Roger Kornberg's lab at Stanford, and started my own group at the Weizmann Institute in Rehovot, Israel. Since 2004 I've been at the Structural Genomics Consortium at the University of Oxford. My team looks at the structure and function of proteins involved in human diseases; we aim to build the tools and knowledge to test the utility of these proteins as drug targets. While focussing for many years on DNA repair and cancer, we are now looking at new targets for Alzheimer's disease and other neurodegenerative diseases.">Opher Gileadi</a>
</li>
	<li class="cat-item cat-item-71"><a href="https://thesgc.github.io/static-openlabnotebooks/category/disease-foundation/parkinsons-disease-and-neurodevelopmental-disorders/">Parkinson's disease and neurodevelopmental disorders</a>
</li>
	<li class="cat-item cat-item-47"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/pietro-roversi/" title="Pietro grew up in Verona, Italy and studied Chemistry at the University of Milano (1987-1993) where he graduated with honours and a passion for structural science. He stayed on in Milano to study for a PhD in structural chemistry under the supervision of Riccardo Destro, working on the multipole and Maximum Entropy modelling of charge densities in crystalline solids (1993-1997). With Gérard Bricogne at the MRC Laboratory of Molecular Biology and at Global Phasing Ltd. (1997-2003) Pietro learnt macromolecular crystallographic theory and computing, and contributed to the writing of the macromolecular crystallography refinement package Buster-TNT (now autoBUSTER).

With Susan Lea at the Biochemistry and Pathology Departments of Oxford University Pietro learnt protein chemistry and determined macromolecular structures of human complement proteins and bacterial proteins involved in infection mechanisms (2003-2012). After a 1-year Ikerbasque Fellowship in Bilbao (Spain) Pietro returned to Oxford in 2013 to join the group of Nicole Zitzmann at the Biochemistry Department, where he determined the first structure of the ER glycoprotein quality control (ERQC) checkpoint enzyme, UGGT.

At Leicester, a 2-year LISCB-WT ISSF Fellowship enables Pietro to start the development of inhibitors of ERQC and ER associated degradation (ERAD) machineries. These inhibitors have the potential to become broad-spectrum rescuers of secretion of responsive mutant glycoproteins in rare disease. As Dr. Strangelove would put it, one day Pietro hopes to train cells to stop checking and love secretion.">Pietro Roversi</a>
</li>
	<li class="cat-item cat-item-93"><a href="https://thesgc.github.io/static-openlabnotebooks/category/laboratory/pietro-roversi-leic/">Pietro Roversi (Univ. of Leicester)</a>
</li>
	<li class="cat-item cat-item-34"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/prc2/">PRC2</a>
</li>
	<li class="cat-item cat-item-154"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/prdm-methyltransferases/">PRDM methyltransferases</a>
</li>
	<li class="cat-item cat-item-52"><a href="https://thesgc.github.io/static-openlabnotebooks/category/disease-foundation/primary-hyperoxaluria/">Primary Hyperoxaluria</a>
</li>
	<li class="cat-item cat-item-127"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/prmts/">PRMTs</a>
</li>
	<li class="cat-item cat-item-134"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/protein-purification-protocols/">Protein purification protocols</a>
</li>
	<li class="cat-item cat-item-10"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/rachel-harding/" title="Rachel Harding completed her PhD at the University of Oxford and now works for the SGC at the University of Toronto as a Postdoctoral Fellow supervised by Professor Cheryl Arrowsmith. Rachel is funded by the CHDI foundation and is studying Huntington’s disease, in particular, researching the protein encoded by the gene mutated in HD patients called huntingtin. Rachel is interested in the 3D structure of the huntingtin protein as well as how it functions in cells and is studying huntingtin protein by both cryo electron microscopy and X-ray crystallography. Rachel is interested in Open Science and Open Data and actively advocates for scientists to share their research outputs freely and without restriction. Since February 2016, Rachel has put all of her research findings on her Huntington’s disease project into an open notebook, LabScribbles, and curates the LabScribbles data collection on the repository Zenodo. Outside of the lab, Rachel is a keen cyclist and enjoys riding and exploring the Canadian countryside on her Brompton folding-bicycle. ">Rachel Harding</a>
</li>
	<li class="cat-item cat-item-48"><a href="https://thesgc.github.io/static-openlabnotebooks/category/disease-foundation/rare-disease/">Rare disease</a>
</li>
	<li class="cat-item cat-item-94"><a href="https://thesgc.github.io/static-openlabnotebooks/category/laboratory/ray-truant-mcmaster/">Ray Truant (McMaster Univ.)</a>
</li>
	<li class="cat-item cat-item-31"><a href="https://thesgc.github.io/static-openlabnotebooks/category/past-scientist/roslin-adamson/" title="Roslin completed her DPhil in the Department of Biochemistry at Oxford University working on the interactions between G protein-coupled receptors and G proteins. She now works at the SGC in Oxford as a post-doctoral research associate supervised by Alex Bullock.She was funded by CHDI initially to work on the structural biology of KEAP1 bound to an E3 ubiquitin ligase, and the involvement of this complex in Huntington’s disease. She is now funded by FOP Friends and M4K Pharma and is working on ACVR1/ALK2, a protein involved in Fibrodysplasia Ossificans Progressiva (FOP), a rare connective tissue disease. This work includes solving the 3D x-ray crystallography structure of the protein bound to various different small compounds, with the aim to find potential inhibitor compounds for development into drugs. It also includes developing activity assays for the protein so that we can screen more compounds. In her spare time Roslin is a keen sculptor of tree-like objects.">Roslin Adamson</a>
</li>
	<li class="cat-item cat-item-50"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/sabrina-mackinnon/" title="Sabrina completed her undergraduate studies in Bioscience and Biomedical Sciences at Robert Gordon University, Aberdeen with a growing interest in studying the group of rare metabolic disorders known as inborn errors of metabolism. Sabrina is currently undertaking her DPhil studies at SGC University of Oxford, under the supervision of Wyatt Yue and Paul Brennan. The focus of her project is to develop small molecule therapies for inborn errors of metabolism by substrate reduction and pharmacological chaperoning. When not in the lab, Sabrina enjoys trips to the seaside and reading the most ridiculous science fiction she can find.

Primary hyperoxaluria project

Primary hyperoxalurias are rare genetic disorders caused by inherited mutations that lead to a build-up of glyoxylate, a normal metabolite in amino acid breakdown and production. When glyoxylate is not converted to more useful substances, it accumulates and is converted instead to oxalate, which is very damaging to the kidneys. One potential approach to treating these disorders, is to reduce the production of glyoxylate by inhibiting the enzyme that produces it: hydroxyacid oxidase 1 (HAO1, also called glycolate oxidase). The aim of Sabrina’s project is to identify, characterise and validate small molecule inhibitors of HAO1, combining cutting-edge methods of fragment screening, x-ray crystallography, and chemical biology.">Sabrina Mackinnon</a>
</li>
	<li class="cat-item cat-item-142"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/sars-cov-2-computational-genomics/">SARS-CoV-2 computational genomics</a>
</li>
	<li class="cat-item cat-item-61"><a href="https://thesgc.github.io/static-openlabnotebooks/category/past-scientist/sean-obyrne/" title="Sean is a medicinal chemistry post-doctoral researcher in the Structural Genomics Consortium in UNC.  He has a keen interest in the synthesis and applications of protein inhibitors and activity-based probes to elucidate drug targets and identify therapeutic opportunities. He is also interested in drug repurposing, especially using human kinase inhibitors to target the kinome of neglected tropical diseases.

He has extensive experience in route design, multistep synthesis, purification and characterization of complex small molecules, as well as expertise in the design and synthesis of BODIPY fluorescent dyes. During his PhD research in the University of Glasgow, he worked on two distinct projects. He developed an effective anti-parasitic drug delivery system which selectively targets infected host cells. His second project involved the synthesis of self-reporting activity-based probes for kinases, allowing for detection of covalent bond formation between proteins and ligands.">Sean O'Byrne</a>
</li>
	<li class="cat-item cat-item-83"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/serap-beldar/" title="Serap Beldar received her B.Sc. degree in Molecular Biology and Genetics from Izmir Institute of Technology, and M.Sc. degree in Computational Sciences and Engineering from Koç University. She recently earned her Ph.D. (in Biological Sciences) from Nanyang Technological University as a Singapore International Graduate Award (SINGA) scholar. During her doctoral studies, she has specialized in the production and biophysical characterization of proteins. Her current research involves structural characterization of histone-modifying enzymes and deciphering molecular interactions with their interacting partners.">Serap Beldar</a>
</li>
	<li class="cat-item cat-item-123"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/setayesh-yazdani/" title="Setayesh received her H.B.Sc. from the University of Toronto in the Biomedical Toxicology Specialist program in June 2018. During her fourth thesis project at Sunnybrook Research Institute, she examined the relationship between type 2 diabetes mellitus and Alzheimer’s disease by specifically looking at the cognitive scores, anatomical structures, and blood biomarker changes in patients with Alzheimer’s disease. She joined Structural Genomic Consortium (SGC) in 2019 as a M.Sc. student in the department of Pharmacology and Toxicology, University of Toronto. At SGC, Setayesh is currently working on a project related to Enhancer of Zeste Homologs inhibitory protein (EZHIP), an important protein in Posterior fossa type A (PFA) ependymomas oncogenicity pathway.  Her goal is to find out if there is a structural domain within EZHIP which can be targeted by a ligand. If that is the case, using PROTAC design, she hopes to target EZHIP for degradation.

Setayesh concurrently works on another bioinformatics project called the Ligandable Human Genome. For that she uses the software tool ICM where she can integrate and analyze different sources of data such as from Protein Data Bank, DrugBank, and Cancer Dependency Map Databases from Sanger and Broad Institutes.

Outside the lab, She is an enthusiastic fitness instructor and works as a coach at Fight Fitness Markham on her weekends. She enjoys leadership activities on campus; she is one of the first-year representatives of her graduate pharmacology cohort. At Iranian Student Association at UofT, Setayesh focuses on membership outreach by increasing community engagement through their mentorship program and other social events.">Setayesh Yazdani</a>
</li>
	<li class="cat-item cat-item-84"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/setdb1/">SETDB1</a>
</li>
	<li class="cat-item cat-item-103"><a href="https://thesgc.github.io/static-openlabnotebooks/category/laboratory/sgc-karolinska/">SGC Karolinska</a>
</li>
	<li class="cat-item cat-item-95"><a href="https://thesgc.github.io/static-openlabnotebooks/category/laboratory/sgc-unc/">SGC UNC Chapel Hill</a>
</li>
	<li class="cat-item cat-item-96"><a href="https://thesgc.github.io/static-openlabnotebooks/category/laboratory/sgc-oxford/">SGC Univ. of Oxford</a>
</li>
	<li class="cat-item cat-item-97"><a href="https://thesgc.github.io/static-openlabnotebooks/category/laboratory/sgc-toronto/">SGC Univ. of Toronto</a>
</li>
	<li class="cat-item cat-item-145"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/spike/">Spike</a>
</li>
	<li class="cat-item cat-item-88"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/stephen-armstrong/" title="I first began studying cancer under Dr. Carita Lanner at the Northern Ontario School of Medicine, investigating the transcriptomes of ovarian cancer cell lines that had acquired drug resistance to broad range chemotherapeutics. Acquired resistance to more than one drug (ex. dual-resistant cell lines) often proved to be sufficient for development of multi-drug resistance, a process where cancer cells become resistant to multiple broad-range chemotherapeutics including those it has not been previously exposed to. This phenomenon highlighted a common problem with nonspecific drugs used to treat common cancers.The primary focus of modern day cancer treatment involves targeted therapy, in which drugs are designed to exploit specific processes that cancer cells rely on, avoiding the induction broad-range resistance and toxicity associated with classical chemotherapy.  These often seek to hobble key oncogenic processes that drives tumor proliferation in specific cancer types (ex. Herceptin targeting the HER-2 receptor in HER-2+ breast cancer), or enable the function of tumor suppressor proteins (ex. MDM2 interference to induce activation of the p53 tumor suppressor protein).  The major problem with targeted therapy is that most cancers lack targeted therapy options.  In 2011, I travelled to Edmonton, Alberta to take up a doctoral position under Dr. Roger Leng studying the p53 tumor suppressor family, a family that is dysregulated in roughly 80% of all cancers, and is the most striking commonality shared between cancers in general.  Developing targeted therapy options for these family members would be ground-breaking in the treatment of cancer, however, transcriptional factors have proven difficult to exploit.  During my studies there, I investigated the regulation of the metastatic TAp63 protein by the ubiquitin-proteasome system and discovered a cross-talk regulatory pathway between proteasome-mediated degradation and the heat shock response pathway that proved to be a major regulator of this transcription factor, and therefore, a possible anti-metastatic target.  After graduating from the University of Alberta in 2016, I sought a postdoctoral position at the Hospital for Sick Children (Toronto, Ontario), and was successful in gaining one under the lab of Dr. Michael Taylor studying childhood brain malignancies, the most predominant of which being medulloblastoma.  Children suffering from medulloblastoma have a specific need for targeted therapy options because radiation treatment common used to treat these cancers is associated with severe developmental defects and reduction in quality of life.  The major problem is the most metastatic form of medulloblastoma lacks clear mutational drivers.  Despite this problem, epigenetic aberrations have been observed in these cells, and our hypothesis is that these epigenetic defects are driving tumorigenesis.  To this end, I am collaborating with the Structural Genomics Consortium (University of Toronto) under Dr. Panagiotis Prinos, to screen medulloblastoma cells with an ever-expanding library of probes designed to interfere with specific epigenetic proteins.  From this research, I hope to drive progress forward in discovering and developing targeted therapies for children suffering from this disease.

2008 – B.Sc. Biomedical Biology (Laurentian University – Sudbury, Ontario)
2010 – M.Sc. Biology (Laurentian University – Sudbury, Ontario)
2016 – Ph.D. Medical Sciences (University of Alberta – Edmonton, Alberta)">Stephen R. Armstrong</a>
</li>
	<li class="cat-item cat-item-72"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/tamara-maiuri/" title="Tamara Maiuri completed her PhD at the University of Toronto and currently works as a Research Associate in Dr. Ray Truant's group at McMaster University, Hamilton, Canada. The Truant lab studies the normal biological functions of the huntingtin protein in hopes of understanding how they may be disrupted upon inheritance of the expanded huntingtin gene that causes Huntington’s disease. Tamara is investigating the role of the huntingtin protein in the process of DNA repair, in a project funded by the HDSA Berman/Topper Fellowship. She has openly shared her results in blog form (raytruantlab.ca/real-time-open-reports/) and through the Truant lab Zenodo community since 2017. Tamara is also a member of the band Eli & the Straw Man, who has partnered with the Huntington Society of Canada since 2016 to raise funds and awareness for Huntington’s disease through their music.">Tamara Maiuri</a>
</li>
	<li class="cat-item cat-item-135"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/taraneh-hajian/" title="With a Masters degree in molecular virology, Taraneh joined SGC Toronto at 2008. Working as a technician in the Molecular Biophysics group, she gained more than 10 years of experience in different aspects of molecular biology and biochemistry including planning, developing and implementing large scale expressions and purification of many proteins including epigenetic targets, transcription factors, enzymes and multimeric complex proteins. The produced proteins have since been used in a myriad of research projects and collaborations, leading to discovery of numerous molecular probes.">Taraneh Hajian</a>
</li>
	<li class="cat-item cat-item-69"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/thomas-durcan/" title="As an assistant professor at the Montreal Neurological Institute (MNI) and McGill University https://www.youtube.com/watch?v=kuO2IKE4JME), my research focus is on applying patient-derived stem cells towards the development of phenotypic discovery assays and 3D mini-brain models (https://www.youtube.com/watch?v=zjOS_trumik) for both neurodegenerative and neurodevelopmental disorders. As group leader of the iPSC platform at the Montreal Neurological Institute (MNI), I oversee a team of 30 research staff and students and growing, committed to applying novel stem cell technology, combined with CRISPR genome editing, mini-brain models and new microfluidic technologies towards elucidating the underlying causes of these complex disorders. Leading the SGC tissue platform, NeuroSGC; I oversee a team committed to assay development, in parallel with leading the establishment of NeuroCDRD, towards the automation of our open assays for small molecule screens. In addition, I am also part of antibody validation efforts (https://www.als-rap.org) between the MNI, Oxford and the Karolinska to generate CRISPR KO cell-lines for validating commercial antibodies against high-value ALS targets. Combined with new approaches in the group towards building multiomic profiles on the patient-derived IPSC cells within the group, the long term strategy over the coming years is to identify new personalized precision therapies that can be applied towards building clinical trials on a dish. Further information on the iPSC platform is found here (http://www.mcgill.ca/ipsquebec-mni/)">Thomas M. Durcan</a>
</li>
	<li class="cat-item cat-item-63"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/tobias-krojer/" title="Tobias works as a Team Leader in the PX group at SGC Oxford. He joined Frank’s group in 2009 and the main focus of his research is fragment-based lead discovery and crystallographic software development.

Tobias studied Biotechnology at FH Weihenstephan (Germany) in the late nineties. He trained as a structural biologist at the Max Planck Institute of Biochemistry in Martinsried (Germany) and received a PhD from Cardiff University in 2005. Afterwards, he worked as a postdoc at the Institute of Molecular Pathology in Vienna (Austria).">Tobias Krojer</a>
</li>
	<li class="cat-item cat-item-22"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/understudied-kinases/">Understudied Kinases</a>
</li>
	<li class="cat-item cat-item-18"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/usp5/">USP5</a>
</li>
	<li class="cat-item cat-item-130"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/usps/">USPs</a>
</li>
	<li class="cat-item cat-item-85"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/victoria-vu/" title="Victoria received her PhD from the University of Toronto, Canada. Her grad studies focused on the effects of natural variation in genetic backgrounds on perturbed phenotypes using C. elegans as a model organism. Victoria works in Dalia Barsyte-Lovejoy’s group at SGC Toronto focusing on the development of cellular assays for E3 ligases.">Victoria Vu</a>
</li>
	<li class="cat-item cat-item-122"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/wdr41/">WDR41</a>
</li>
	<li class="cat-item cat-item-56"><a href="https://thesgc.github.io/static-openlabnotebooks/category/past-scientist/wen-yih-aw/" title="Wen completed her Ph.D. in Molecular Biology at Princeton University studying how cells establish a sense of direction within a 2D plane. She is now a Postdoctoral Research Associate at the UNC Catalyst for Rare Diseases, developing a dual-vector gene therapy strategy for Dravet Syndrome.

Dravet Syndrome is a rare and severe form of epilepsy that begins in infancy. It is commonly caused by haploinsufficiency in scn1a gene, a gene that codes for voltage-gated sodium channel. Specifically, her project will focus on the development of a protein trans-splicing system that allows us to bypass the packaging size limit of AAV vectors to enable gene therapy for Dravet Syndrome.">Wen Yih Aw</a>
</li>
	<li class="cat-item cat-item-60"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/xchem/">XChem</a>
</li>
			</ul></div></section><section id="meta-2" class="widget widget_meta"><div class="widget-wrap"><h4 class="widget-title">Meta</h4>
		<ul><li><a href="https://thesgc.github.io/static-openlabnotebooks/wp-login.php">Log in</a></li>
			<li><a href="https://thesgc.github.io/static-openlabnotebooks/feed/">Entries feed</a></li>
			<li><a href="https://thesgc.github.io/static-openlabnotebooks/comments/feed/">Comments feed</a></li>

			<li><a href="https://en-gb.wordpress.org/">WordPress.org</a></li>
		</ul></div></section></aside></div>
	    </div>
		            
	<footer id="colophon" class="site-footer" role="contentinfo"><div id="site-generator">
	    		<div class="wrapper">
	    			<div id="footer-content" class="copyright"><div class="left-side"><a href="https://thesgc.github.io/static-openlabnotebooks/">openlabnotebooks.org</a> is licensed under a <a href="https://creativecommons.org/licenses/by/4.0" target="_blank">CC-BY</a> licence. 2020 <a target="_blank" href="http://www.thesgc.org/science/research-informatics">SGC RI</a></div><div class="right-side"><a href="/accessibility">Accessibility</a></div></div>
	    		</div>
			</div>	</footer></div>
		
<a href="#masthead" id="scrollup" class="genericon"><span class="screen-reader-text">Scroll Up</span></a><nav id="mobile-header-left-nav" class="mobile-menu" role="navigation"><ul id="header-left-nav" class="menu"><li class="menu-item menu-item-type-custom menu-item-object-custom menu-item-home menu-item-27"><a href="https://thesgc.github.io/static-openlabnotebooks">Home</a></li>
<li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-3061"><a href="https://thesgc.github.io/static-openlabnotebooks/laboratories/">Laboratories</a>
<ul class="sub-menu"><li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-3060"><a href="https://thesgc.github.io/static-openlabnotebooks/active-laboratories/">Active</a>
	<ul class="sub-menu"><li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-3373"><a href="https://thesgc.github.io/static-openlabnotebooks/category/laboratory/amy-ramsey-univ-of-toronto/">Amy Ramsey (Univ. of Toronto)</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-4874"><a href="https://thesgc.github.io/static-openlabnotebooks/category/laboratory/bccrc-univ-of-british-columbia/">BCCRC, Univ. of British Columbia</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-5125"><a href="https://thesgc.github.io/static-openlabnotebooks/category/laboratory/biozone-univ-of-toronto/">BioZone (Univ. of Toronto)</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-3077"><a href="https://thesgc.github.io/static-openlabnotebooks/category/laboratory/unc-catalyst/">Catalyst UNC Chapel Hill</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category current-post-ancestor current-menu-parent current-post-parent menu-item-5284"><a href="https://thesgc.github.io/static-openlabnotebooks/category/laboratory/cheryl-arrowsmith-univ-of-toronto/">Cheryl Arrowsmith (Univ. of Toronto)</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-3067"><a href="https://thesgc.github.io/static-openlabnotebooks/category/laboratory/diamond-light-source/">Diamond Light Source</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-3068"><a href="https://thesgc.github.io/static-openlabnotebooks/category/laboratory/michael-taylor-sickkids/">Michael Taylor (SickKids Hospital Toronto)</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-3069"><a href="https://thesgc.github.io/static-openlabnotebooks/category/laboratory/montreal-neurological-institute/">Montreal Neurological Institute</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-4785"><a href="https://thesgc.github.io/static-openlabnotebooks/category/laboratory/nick-goldman-ebi/">Nick Goldman (EBI)</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-3071"><a href="https://thesgc.github.io/static-openlabnotebooks/category/laboratory/pietro-roversi-leic/">Pietro Roversi (Univ. of Leicester)</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-3072"><a href="https://thesgc.github.io/static-openlabnotebooks/category/laboratory/ray-truant-mcmaster/">Ray Truant (McMaster Univ.)</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-3073"><a href="https://thesgc.github.io/static-openlabnotebooks/category/laboratory/sgc-unc/">SGC UNC Chapel Hill</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-3074"><a href="https://thesgc.github.io/static-openlabnotebooks/category/laboratory/sgc-oxford/">SGC Univ. of Oxford</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-3075"><a href="https://thesgc.github.io/static-openlabnotebooks/category/laboratory/sgc-toronto/">SGC Univ. of Toronto</a></li>
	</ul></li>
	<li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-3059"><a href="https://thesgc.github.io/static-openlabnotebooks/past-laboratories/">Past</a>
	<ul class="sub-menu"><li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-3076"><a href="https://thesgc.github.io/static-openlabnotebooks/category/laboratory/antoine-claessens-laboratories/">Antoine Claessens</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-3078"><a href="https://thesgc.github.io/static-openlabnotebooks/category/laboratory/sgc-karolinska/">SGC Karolinska</a></li>
	</ul></li>
</ul></li>
<li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-115"><a href="https://thesgc.github.io/static-openlabnotebooks/people/">People</a>
<ul class="sub-menu"><li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-2962"><a href="https://thesgc.github.io/static-openlabnotebooks/active-people/">Active</a>
	<ul class="sub-menu"><li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-123"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/alfredo-picado/">Alfredo Picado</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-5129"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/alma-seitova/">Alma Seitova</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-3372"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/amy-ramsey/">Amy Ramsey</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-5137"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/anwar-hossain/">Anwar Hossain</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-3651"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/ashley-hutchinson/">Ashley Hutchinson</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-1118"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/ben-eduful/">Ben Eduful</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-3102"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/claudia-alvarez/">Claudia Alvarez</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category current-post-ancestor current-menu-parent current-post-parent menu-item-5286"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/danton-ivanochko/">Danton Ivanochko</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-257"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/david-dilworth/">David Dilworth</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-4506"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/david-drewry/">David Drewry</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-4556"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/ed-anderson/">Edward Anderson</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-1050"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/ellie-williams/">Ellie Williams</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-5223"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/evianne-rovers/">Evianne Rovers</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-5413"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/fang-chi-chang/">Fang-Chi Chang</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-4598"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/hadley-sheppard/">Hadley Sheppard</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-176"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/jong-fu-wong/">Jong Fu Wong</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-1993"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/jose-brandao-neto/">Jose Brandao-Neto</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-5082"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/joseph-newman/">Joseph Newman</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-3414"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/katheron-intson/">Katheron Intson</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-177"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/liz-brown/">Liz Brown</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-2124"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/louisa-temme/">Louisa Temme</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-2790"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/magdalena-szewczyk/">Magdalena Szewczyk</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-135"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/mandeep-mann/">Mandeep Mann</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-4787"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/nicola-de-maio/">Nicola De Maio</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-2203"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/opher-gileadi/">Opher Gileadi</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-5126"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/peter-stogios/">Peter Stogios</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-1700"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/pietro-roversi/">Pietro Roversi</a></li>
		<li class="menu-item menu-item-type-custom menu-item-object-custom menu-item-281"><a target="_blank" rel="noopener noreferrer" href="http://labscribbles.com/">Rachel Harding</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-1772"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/sabrina-mackinnon/">Sabrina Mackinnon</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-2867"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/serap-beldar/">Serap Beldar</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-3761"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/setayesh-yazdani/">Setayesh Yazdani</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-2956"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/stephen-armstrong/">Stephen R. Armstrong</a></li>
		<li class="menu-item menu-item-type-custom menu-item-object-custom menu-item-2461"><a target="_blank" rel="noopener noreferrer" href="https://raytruantlab.ca/real-time-open-reports/">Tamara Maiuri</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-4513"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/taraneh-hajian/">Taraneh Hajian</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-2447"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/thomas-durcan/">Thomas M. Durcan</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-2311"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/tobias-krojer/">Tobias Krojer</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-2882"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/victoria-vu/">Victoria Vu</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-3415"><a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/yuanye-yan/">Yuanye Yan</a></li>
	</ul></li>
	<li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-2961"><a href="https://thesgc.github.io/static-openlabnotebooks/past-people/">Past</a>
	<ul class="sub-menu"><li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-1162"><a href="https://thesgc.github.io/static-openlabnotebooks/category/past-scientist/antoine-claessens/">Antoine Claessens</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-2963"><a href="https://thesgc.github.io/static-openlabnotebooks/category/past-scientist/carla-alamillo/">Carla Alamillo</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-3346"><a href="https://thesgc.github.io/static-openlabnotebooks/category/past-scientist/carmen-jimenez-antunez/">Carmen Jimenez Antunez</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-2029"><a href="https://thesgc.github.io/static-openlabnotebooks/category/past-scientist/carolyn-marks/">Carolyn Marks</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-308"><a href="https://thesgc.github.io/static-openlabnotebooks/category/past-scientist/genna-luciani/">Genna Luciani</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-2681"><a href="https://thesgc.github.io/static-openlabnotebooks/category/past-scientist/haresh-ajani/">Haresh Ajani</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-2902"><a href="https://thesgc.github.io/static-openlabnotebooks/category/past-scientist/jacob-mcauley/">Jacob McAuley</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-2968"><a href="https://thesgc.github.io/static-openlabnotebooks/category/past-scientist/jolene-ho/">Jolene Ho</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-2964"><a href="https://thesgc.github.io/static-openlabnotebooks/category/past-scientist/kaleb-naegeli/">Kaleb Naegeli</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-2965"><a href="https://thesgc.github.io/static-openlabnotebooks/category/past-scientist/megha-abbey/">Megha Abbey</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-2966"><a href="https://thesgc.github.io/static-openlabnotebooks/category/past-scientist/nirav-kapadia/">Nirav Kapadia</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-376"><a href="https://thesgc.github.io/static-openlabnotebooks/category/past-scientist/roslin-adamson/">Roslin Adamson</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-1972"><a href="https://thesgc.github.io/static-openlabnotebooks/category/past-scientist/sean-obyrne/">Sean O’Byrne</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-2967"><a href="https://thesgc.github.io/static-openlabnotebooks/category/past-scientist/wen-yih-aw/">Wen Yih Aw</a></li>
	</ul></li>
</ul></li>
<li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-118"><a href="https://thesgc.github.io/static-openlabnotebooks/disease-foundations/">Diseases</a>
<ul class="sub-menu"><li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-1761"><a href="https://thesgc.github.io/static-openlabnotebooks/active-diseases/">Active</a>
	<ul class="sub-menu"><li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-1855"><a href="https://thesgc.github.io/static-openlabnotebooks/category/disease-foundation/alzheimers-disease/">Alzheimer’s disease</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-2026"><a href="https://thesgc.github.io/static-openlabnotebooks/category/disease-foundation/amyotrophic-lateral-sclerosis-als/">Amyotrophic Lateral Sclerosis (ALS)</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-4555"><a href="https://thesgc.github.io/static-openlabnotebooks/category/disease-foundation/batten-disease/">Batten disease</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-1140"><a href="https://thesgc.github.io/static-openlabnotebooks/category/disease-foundation/chordoma/">Chordoma</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-4780"><a href="https://thesgc.github.io/static-openlabnotebooks/category/disease-foundation/covid-19/">COVID-19</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-144"><a href="https://thesgc.github.io/static-openlabnotebooks/category/disease-foundation/dipg/">Diffuse intrinsic pontine glioma</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-3759"><a href="https://thesgc.github.io/static-openlabnotebooks/category/disease-foundation/ependymoma/">Ependymoma</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-16"><a href="https://thesgc.github.io/static-openlabnotebooks/category/disease-foundation/fop/">Fibrodysplasia ossificans progressiva</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-3374"><a href="https://thesgc.github.io/static-openlabnotebooks/category/disease-foundation/grin-disorder/">GRIN disorder</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-600"><a href="https://thesgc.github.io/static-openlabnotebooks/category/disease-foundation/chdi/">Huntington’s disease</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-2504"><a href="https://thesgc.github.io/static-openlabnotebooks/category/disease-foundation/ipsc-disease-modelling/">iPSC Disease modelling</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-255"><a href="https://thesgc.github.io/static-openlabnotebooks/category/disease-foundation/leukemia/">Leukemia</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-1144"><a href="https://thesgc.github.io/static-openlabnotebooks/category/disease-foundation/malaria/">Malaria</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-2449"><a href="https://thesgc.github.io/static-openlabnotebooks/category/disease-foundation/parkinsons-disease-and-neurodevelopmental-disorders/">Parkinson’s disease and neurodevelopmental disorders</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-1774"><a href="https://thesgc.github.io/static-openlabnotebooks/category/disease-foundation/primary-hyperoxaluria/">Primary Hyperoxaluria</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-1702"><a href="https://thesgc.github.io/static-openlabnotebooks/category/disease-foundation/rare-disease/">Rare disease</a></li>
	</ul></li>
	<li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-1760"><a href="https://thesgc.github.io/static-openlabnotebooks/past-diseases/">Past</a>
	<ul class="sub-menu"><li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-1873"><a href="https://thesgc.github.io/static-openlabnotebooks/category/disease-foundation/dravet-syndrome/">Dravet syndrome</a></li>
	</ul></li>
</ul></li>
<li class="menu-item menu-item-type-post_type menu-item-object-page current-post-parent menu-item-has-children menu-item-155"><a href="https://thesgc.github.io/static-openlabnotebooks/projects/">Projects</a>
<ul class="sub-menu"><li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-1759"><a href="https://thesgc.github.io/static-openlabnotebooks/active-projects/">Active</a>
	<ul class="sub-menu"><li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-356"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/acvr1-alk2/">ACVR1/ALK2</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-2027"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/als-reproducible-antibody-platform-als-rap/">ALS-Reproducible Antibody Platform (ALS-RAP)</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-4496"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/cell-assay-reports/">Cell assay reports</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-4387"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/chromatin-factors/">Chromatin factors</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-2883"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/e3-ligases/">E3 ligases</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-2771"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/epigenetic-targets-in-medulloblastoma/">Epigenetic targets in Medulloblastoma</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-3760"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/ezhip/">EZHIP</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-1856"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/gtpase-gef-complex/">GTPase-GEF complex</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-1773"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/hao1/">HAO1</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-451"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/hdacs/">HDACs</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-3295"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/hero-glove/">HERO Glove</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-601"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/huntingtin/">Huntingtin</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-3184"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/investigating-outdoor-mobility-of-older-adults/">Investigating Outdoor Mobility of Older Adults</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-2503"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/ipscs/">iPSCs</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-4286"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/ligandable-genome/">Ligandable Genome</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-3375"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/nmda/">NMDA</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-357"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/nsd3/">NSD3</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-1703"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/open-secretion-rescue/">Open Secretion Rescue</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-504"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/prc2/">PRC2</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category current-post-ancestor current-menu-parent current-post-parent menu-item-5285"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/prdm-methyltransferases/">PRDM methyltransferases</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-3847"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/prmts/">PRMTs</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-4509"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/protein-purification-protocols/">Protein purification protocols</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-4786"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/sars-cov-2-computational-genomics/">SARS-CoV-2 computational genomics</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-2868"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/setdb1/">SETDB1</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-5024"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/spike/">Spike</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-5138"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/tbxt-brachyury-inhibitors/">TBXT (brachyury) inhibitors</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-254"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/understudied-kinases/">Understudied Kinases</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-157"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/usp5/">USP5</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-4388"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/usps/">USPs</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-5414"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/wdr-proteins/">WDR proteins</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-3652"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/wdr41/">WDR41</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-1960"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/xchem/">XChem</a></li>
	</ul></li>
	<li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-1758"><a href="https://thesgc.github.io/static-openlabnotebooks/past-projects/">Past</a>
	<ul class="sub-menu"><li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-1141"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/egfr-and-gak/">EGFR and GAK</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-1874"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/gene-therapy-for-dravet-syndrome/">Gene therapy for Dravet Syndrome</a></li>
		<li class="menu-item menu-item-type-taxonomy menu-item-object-category menu-item-2682"><a href="https://thesgc.github.io/static-openlabnotebooks/category/project/in-silico-protacs-design/">In silico PROTACs design</a></li>
	</ul></li>
</ul></li>
<li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-923"><a href="https://thesgc.github.io/static-openlabnotebooks/the-team/">The team</a></li>
<li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-919"><a href="https://thesgc.github.io/static-openlabnotebooks/about/">About</a></li>
<li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-has-children menu-item-4663"><a href="https://thesgc.github.io/static-openlabnotebooks/my-research-in-2-minutes/">My research in 2 minutes</a>
<ul class="sub-menu"><li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-4729"><a href="https://thesgc.github.io/static-openlabnotebooks/artem-babaian-ultra-deep-search-for-novel-coronaviruses/">Artem Babaian: ultra-deep search for novel coronaviruses</a></li>
	<li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-4694"><a href="https://thesgc.github.io/static-openlabnotebooks/danton-ivanochko-a-chemical-probe-for-prdm9/">Danton Ivanochko: a chemical probe for PRDM9</a></li>
	<li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-4695"><a href="https://thesgc.github.io/static-openlabnotebooks/mandeep-mann-a-chemical-probe-for-usp5-2/">Mandeep Mann: a chemical probe for USP5</a></li>
	<li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-5207"><a href="https://thesgc.github.io/static-openlabnotebooks/natasha-anita-depressive-biomarkers-in-diabetes/">Natasha Anita: depressive biomarkers in diabetes</a></li>
	<li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-4693"><a href="https://thesgc.github.io/static-openlabnotebooks/rachel-harding-3d-structure-of-huntingtin/">Rachel Harding: 3D structure of Huntingtin</a></li>
	<li class="menu-item menu-item-type-post_type menu-item-object-page menu-item-4706"><a href="https://thesgc.github.io/static-openlabnotebooks/setayesh-yazdani-the-druggable-human-genome/">Setayesh Yazdani: the druggable human genome</a></li>
</ul></li>
</ul></nav><script type="text/javascript" id="contact-form-7-js-extra">
/* <![CDATA[ */
var wpcf7 = {"apiSettings":{"root":"https:\/\/thesgc.github.io\/static-openlabnotebooks\/wp-json\/contact-form-7\/v1","namespace":"contact-form-7\/v1"}};
/* ]]> */
</script><script type="text/javascript" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/plugins/contact-form-7/includes/js/scripts.js" id="contact-form-7-js"></script><script type="text/javascript" id="pt-cv-content-views-script-js-extra">
/* <![CDATA[ */
var PT_CV_PUBLIC = {"_prefix":"pt-cv-","page_to_show":"5","_nonce":"8816a71cba","is_admin":"","is_mobile":"","ajaxurl":"https:\/\/thesgc.github.io\/static-openlabnotebooks\/wp-admin\/admin-ajax.php","lang":"","loading_image_src":"data:image\/gif;base64,R0lGODlhDwAPALMPAMrKygwMDJOTkz09PZWVla+vr3p6euTk5M7OzuXl5TMzMwAAAJmZmWZmZszMzP\/\/\/yH\/C05FVFNDQVBFMi4wAwEAAAAh+QQFCgAPACwAAAAADwAPAAAEQvDJaZaZOIcV8iQK8VRX4iTYoAwZ4iCYoAjZ4RxejhVNoT+mRGP4cyF4Pp0N98sBGIBMEMOotl6YZ3S61Bmbkm4mAgAh+QQFCgAPACwAAAAADQANAAAENPDJSRSZeA418itN8QiK8BiLITVsFiyBBIoYqnoewAD4xPw9iY4XLGYSjkQR4UAUD45DLwIAIfkEBQoADwAsAAAAAA8ACQAABC\/wyVlamTi3nSdgwFNdhEJgTJoNyoB9ISYoQmdjiZPcj7EYCAeCF1gEDo4Dz2eIAAAh+QQFCgAPACwCAAAADQANAAAEM\/DJBxiYeLKdX3IJZT1FU0iIg2RNKx3OkZVnZ98ToRD4MyiDnkAh6BkNC0MvsAj0kMpHBAAh+QQFCgAPACwGAAAACQAPAAAEMDC59KpFDll73HkAA2wVY5KgiK5b0RRoI6MuzG6EQqCDMlSGheEhUAgqgUUAFRySIgAh+QQFCgAPACwCAAIADQANAAAEM\/DJKZNLND\/kkKaHc3xk+QAMYDKsiaqmZCxGVjSFFCxB1vwy2oOgIDxuucxAMTAJFAJNBAAh+QQFCgAPACwAAAYADwAJAAAEMNAs86q1yaWwwv2Ig0jUZx3OYa4XoRAfwADXoAwfo1+CIjyFRuEho60aSNYlOPxEAAAh+QQFCgAPACwAAAIADQANAAAENPA9s4y8+IUVcqaWJ4qEQozSoAzoIyhCK2NFU2SJk0hNnyEOhKR2AzAAj4Pj4GE4W0bkJQIAOw=="};
var PT_CV_PAGINATION = {"first":"\u00ab","prev":"\u2039","next":"\u203a","last":"\u00bb","goto_first":"Go to first page","goto_prev":"Go to previous page","goto_next":"Go to next page","goto_last":"Go to last page","current_page":"Current page is","goto_page":"Go to page"};
/* ]]> */
</script><script type="text/javascript" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/plugins/content-views-query-and-display-post-page/public/assets/js/cv.js" id="pt-cv-content-views-script-js"></script><script type="text/javascript" id="cookie-consent-js-extra">
/* <![CDATA[ */
var ctcc_vars = {"expiry":"30","method":"1","version":"1"};
/* ]]> */
</script><script type="text/javascript" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/plugins/uk-cookie-consent/assets/js/uk-cookie-consent-js.js" id="cookie-consent-js"></script><script type="text/javascript" src="https://www.google.com/recaptcha/api.js?render=6Lf14Y0UAAAAAODV6ncwHqbFUCAo4VV5YoU3_2Q4&ver=3.0" id="google-recaptcha-js"></script><script type="text/javascript" id="wpcf7-recaptcha-js-extra">
/* <![CDATA[ */
var wpcf7_recaptcha = {"sitekey":"6Lf14Y0UAAAAAODV6ncwHqbFUCAo4VV5YoU3_2Q4","actions":{"homepage":"homepage","contactform":"contactform"}};
/* ]]> */
</script><script type="text/javascript" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/plugins/contact-form-7/modules/recaptcha/script.js" id="wpcf7-recaptcha-js"></script><script type="text/javascript" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/themes/clean-journal-pro/js/navigation.min.js" id="clean-journal-navigation-js"></script><script type="text/javascript" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/themes/clean-journal-pro/js/skip-link-focus-fix.min.js" id="clean-journal-skip-link-focus-fix-js"></script><script type="text/javascript" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/themes/clean-journal-pro/js/fitvids.min.js" id="jquery-fitvids-js"></script><script type="text/javascript" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/themes/clean-journal-pro/js/clean-journal-scrollup.min.js" id="clean-journal-scrollup-js"></script><script async="async" type="text/javascript" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/plugins/akismet/_inc/form.js" id="akismet-form-js"></script><script type="text/javascript">
					jQuery(document).ready(function($){
												if(!catapultReadCookie("catAccCookies")){ // If the cookie has not been set then show the bar
							$("html").addClass("has-cookie-bar");
							$("html").addClass("cookie-bar-bottom-bar");
							$("html").addClass("cookie-bar-bar");
													}
																			ctccFirstPage();
											});
				</script><div id="catapult-cookie-bar" class=""><div class="ctcc-inner "><span class="ctcc-left-side">This site uses cookies:  <a class="ctcc-more-info-link" tabindex="0" target="_blank" href="https://thesgc.github.io/static-openlabnotebooks/cookie-policy/">Find out more.</a></span><span class="ctcc-right-side"><button id="catapultCookie" tabindex="0" onclick="catapultAcceptCookies();">Okay, thank you</button></span></div></div>
</body></html>
