<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>Pietro Roversi &#8211; openlabnotebooks.org</title>
	<atom:link href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/pietro-roversi/feed/" rel="self" type="application/rss+xml" />
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<description>A growing team of groundbreaking scientists around the world are now sharing their lab notebooks online</description>
	<lastBuildDate>Sun, 19 Jan 2020 16:47:35 +0000</lastBuildDate>
	<language>en-GB</language>
	<sy:updatePeriod>
	hourly	</sy:updatePeriod>
	<sy:updateFrequency>
	1	</sy:updateFrequency>
	<generator>https://wordpress.org/?v=5.3.2</generator>

<image>
	<url>https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/cropped-lab-jar02-32x32.png</url>
	<title>Pietro Roversi &#8211; openlabnotebooks.org</title>
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<width>32</width>
	<height>32</height>
</image> 
	<item>
		<title>UGGT:  a fly-paper glycoprotein catching troublesome flies?</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/uggt-is-the-misfolding-glycoprotein-checkpoint-like-misfolded-glycoprotein-fly-paper/</link>
				<comments>https://thesgc.github.io/static-openlabnotebooks/uggt-is-the-misfolding-glycoprotein-checkpoint-like-misfolded-glycoprotein-fly-paper/#respond</comments>
				<pubDate>Sun, 19 Jan 2020 15:04:40 +0000</pubDate>
		<dc:creator><![CDATA[Pietro Roversi]]></dc:creator>
				<category><![CDATA[Open Secretion Rescue]]></category>
		<category><![CDATA[Pietro Roversi]]></category>
		<category><![CDATA[Pietro Roversi (Univ. of Leicester)]]></category>
		<category><![CDATA[Rare disease]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=4013</guid>
				<description><![CDATA[&#160; About one quarter of the Eukaryotic genome encodes glycoproteins that are made in the Endoplasmic Reticulum (ER), and travel to special destinations such as the extracellular environment, the cell membrane or other organelles like the lysosomes. We can think of the ER as a nursery where baby glycoproteins quickly grow to attain their adult <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/uggt-is-the-misfolding-glycoprotein-checkpoint-like-misfolded-glycoprotein-fly-paper/">Read More ...</a>]]></description>
								<content:encoded><![CDATA[<p>&nbsp;</p>
<div id="attachment_4017" style="width: 255px" class="wp-caption alignleft"><img aria-describedby="caption-attachment-4017" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/01/FlyPaper-245x300.png" alt="" class="size-medium wp-image-4017" width="245" height="300" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/01/FlyPaper-245x300.png 245w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/01/FlyPaper-768x940.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/01/FlyPaper-837x1024.png 837w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/01/FlyPaper.png 1500w" sizes="(max-width: 245px) 100vw, 245px" /><p id="caption-attachment-4017" class="wp-caption-text">Mis-folded glycoproteins are caught by UGGT and cannot leave the ER.</p></div>
<p>About one quarter of the Eukaryotic genome encodes <a href="https://en.wikipedia.org/wiki/Glycoprotein">glycoprotein</a>s that are made in the <a href="https://en.wikipedia.org/wiki/Endoplasmic_reticulum">Endoplasmic Reticulum</a> (ER), and travel to special destinations such as the extracellular environment, the cell membrane or other organelles like the lysosomes. We can think of the ER as a nursery where baby glycoproteins quickly grow to attain their adult shape and size, in conditions that closely mimic the ones they evolved to find themselves in.</p>
<p>But how is it decided that a glycoprotein is fully and perfectly formed and can leave the ER? As it turns out, the Eukaryotic cell has evolved a formidable glycoprotein folding quality control (ERQC) system, which ensures that glycoproteins that are mis-folded &#8211; i.e. are still forming &#8211; cannot prematurely leave the ER. The checkpoint of this quality control system is the enzyme that detects a defects in a folding glycoprotein and flags it for ER retention: UDP-glucose Glycoprotein Glucosyl-Transferase (<a href="https://en.wikipedia.org/wiki/UGGT">UGGT</a>).</p>
<p>Thinking of the variety of sizes and shapes of the thousands of <a href="https://en.wikipedia.org/wiki/Secretory_protein">secretory proteins</a> that travel from the ER down the secretory pathway, a big question comes to mind: how can UGGT &#8211; a single enzyme &#8211; survey folding of so many and such different glycoprotein clients? It has been hypothesized that UGGT recognizes exposed &#8220;sticky&#8221; <a href="https://en.wikipedia.org/wiki/Amino_acid">amino acid</a> residues on the surface of a defective glycoprotein, but nobody has yet discovered how the recognition of that &#8220;stickiness&#8221; happens.</p>
<p>In the process of carrying out a control experiment, we have now obtained serendipitous data suggesting that UGGT itself may possess a &#8220;sticky&#8221; mis-folded patch, and that it may therefore catch glycoproteins that carry a folding defect via a &#8220;mis-fold:mis-fold&#8221; molecular recognition event. We prepared a special version of recombinant UGGT, treating mammalian cells with the mannosidase inhibitor <a href="https://en.wikipedia.org/wiki/Kifunensine">kifunensine</a>: this was meant to produce a UGGT molecule (which we call UGGT<sub>Kif</sub> ) bearing glycans identical to the ones carried by all glycoproteins in the ER (including UGGT&#8217;s substrates). Surprisingly, <a href="https://zenodo.org/record/3608191#.XiR76RfgrNA">the UGGT<sub>Kif </sub>crystal structure</a> revealed that one of the domains of the protein is mis-folded!</p>
<p>This observation &#8211; together with <a href="http://10.1016/j.bbrc.2015.04.105">published evidence</a> that UGGT bears de-mannosylated glycans (a hallmark of defective glycoproteins flagged for <a href="https://en.wikipedia.org/wiki/Endoplasmic-reticulum-associated_protein_degradation">ER associated degradation</a>) &#8211; now begs the question whether UGGT has evolved to maintain a mis-folded domain in order to catch mis-folded glycoproteins. Clearly, a physiologically mis-folded UGGT would be steadily degraded in the ER, so if nature has come up with this solution to the problem of mis-fold recognition &#8211; Eukaryotic cells would need to afford quick catabolism of UGGT and the costs associated with constant replenishment of ER UGGT levels.</p>
<p>In summary, when it comes to glycoproteins, <a href="https://doi.org/10.1101/2019.12.25.888438">does it take one to know one</a>? We are now pursuing experiments to test the hypothesis that UGGT be like mis-folded glycoprotein fly-paper, catching mis-folded glycoprotein flies. Watch this space!</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
]]></content:encoded>
							<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/uggt-is-the-misfolding-glycoprotein-checkpoint-like-misfolded-glycoprotein-fly-paper/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
							</item>
		<item>
		<title>Strippers from outer space: recombinant PNGase F.</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/strippers-from-outer-space-recombinant-pngase-f/</link>
				<comments>https://thesgc.github.io/static-openlabnotebooks/strippers-from-outer-space-recombinant-pngase-f/#comments</comments>
				<pubDate>Sun, 30 Jun 2019 18:41:29 +0000</pubDate>
		<dc:creator><![CDATA[Pietro Roversi]]></dc:creator>
				<category><![CDATA[Open Secretion Rescue]]></category>
		<category><![CDATA[Pietro Roversi]]></category>
		<category><![CDATA[Pietro Roversi (Univ. of Leicester)]]></category>
		<category><![CDATA[Rare disease]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=3041</guid>
				<description><![CDATA[Nature has evolved molecules called glycoproteins that are proteins carrying sugar chains (&#8220;glycans&#8221;). Glycans can make proteins more soluble, they can shield them from the environment, they can be used as calling cards to enable protein-protein recognition; in Eukaryotes, a glucose-on/glucose-off quality control system uses glycans to monitor glycoproteins folding. In general, glycans contribute to <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/strippers-from-outer-space-recombinant-pngase-f/">Read More ...</a>]]></description>
								<content:encoded><![CDATA[<div id="attachment_3042" style="width: 239px" class="wp-caption alignnone"><img aria-describedby="caption-attachment-3042" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/06/mir.jpg" alt="" class="size-full wp-image-3042" width="229" height="220" /><p id="caption-attachment-3042" class="wp-caption-text">The enzyme PNGase F comes from a bacterium first isolated on the MIR space station.</p></div>
<p>Nature has evolved molecules called glycoproteins that are proteins carrying sugar chains (&#8220;glycans&#8221;). Glycans can make proteins more soluble, they can shield them from the environment, they can be used as calling cards to enable protein-protein recognition; in Eukaryotes, a glucose-on/glucose-off quality control system uses glycans to monitor glycoproteins folding.</p>
<p>In general, glycans contribute to expanding the repertoire of protein shapes and sizes that genomes code for, and sugars on proteins are a great asset to nature. Protein chemists on the other hand are not always happy with the glycans on the proteins they produce. This is especially true of molecular biologists forcing cells to synthesise custom glycoproteins: these &#8216;recombinant glycoproteins&#8217; may end up with glycans that differ from naturally occurring ones.</p>
<p>As it turns out, the ability of stripping sugars from glycoproteins has helped a number of species to survive, and enzymes that can cleave glycans from glycoproteins (&#8220;deglycosylases&#8221;) are encoded in many a genome. By offering such enzymes, nature comes therefore to the succour of molecular biologists and protein chemists who are unhappy with the glycans on the proteins they handle: by treating a glycoprotein with a deglycosylase, it is possible to get rid of the sugars and obtain a sugar-free protein.</p>
<p>In our case, crystals of a glycoprotein we are studying grew a flimsy lattice formed by glycan-glycan contacts, making the resulting glycoprotein structure all fuzzy. We are hoping that one way of improving on these crystals and obtaining a better structure would be to strip the glycoprotein of the glycans altogether, perhaps enabling the growth of different crystals, kept together by more stable, less mobile protein-protein contacts.</p>
<p>For this reason, we decided to treat our glycoprotein with a special deglycosylase called PNGase F. The enzyme belongs to a rare bacterium called <a href="https://en.wikipedia.org/wiki/Elizabethkingia_miricola"><em>Elizabethkingia miricola</em>, </a>first isolated from condensation water in the <a href="https://en.wikipedia.org/wiki/Mir">Space Station Mir</a>. But a small vial of the enzyme can cost up to £1000 when purchased from commercial sources! We therefore endeavoured to produce our own PNGase F, using <em>E.coli</em> bacteria transformed with the <em>Elizabethkingia miricola</em> PNGase F gene.</p>
<p>After ten days of work, <a href="https://zenodo.org/record/3263902#.XRj3jXt7l2Y">we obtained our stripper from outer space</a>, and were able to confirm its activity. Watch this less-than-outer space for the next experiments in which we shall try and grow our crystals again, this time with the protein without glycans!</p>
]]></content:encoded>
							<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/strippers-from-outer-space-recombinant-pngase-f/feed/</wfw:commentRss>
		<slash:comments>1</slash:comments>
							</item>
		<item>
		<title>Crystal structures of the CtUGGT double mutant S180C:T742C</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/crystal-structures-of-the-ctuggt-double-mutant-s180ct742c/</link>
				<comments>https://thesgc.github.io/static-openlabnotebooks/crystal-structures-of-the-ctuggt-double-mutant-s180ct742c/#comments</comments>
				<pubDate>Wed, 15 Aug 2018 14:59:01 +0000</pubDate>
		<dc:creator><![CDATA[Pietro Roversi]]></dc:creator>
				<category><![CDATA[Open Secretion Rescue]]></category>
		<category><![CDATA[Pietro Roversi]]></category>
		<category><![CDATA[Pietro Roversi (Univ. of Leicester)]]></category>
		<category><![CDATA[Rare disease]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=1705</guid>
				<description><![CDATA[The inter-domain conformational mobility of UGGT, the eukaryotic secretion glycoprotein folding checkpoint, is likely key to its ability of recognising and re-glucosylating a vast number of different misfolded glycoproteins, bearing N-linked glycan(s) at various different distances from the site(s) of misfold. Ongoing Molecular Dynamics simulations by Juan Blanco, Carlos Modenutti, and Marcelo Martí in Buenos <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/crystal-structures-of-the-ctuggt-double-mutant-s180ct742c/">Read More ...</a>]]></description>
								<content:encoded><![CDATA[<p>The inter-domain conformational mobility of UGGT, the eukaryotic secretion glycoprotein folding checkpoint, is likely key to its ability of recognising and re-glucosylating a vast number of different misfolded glycoproteins, bearing N-linked glycan(s) at various different distances from the site(s) of <a href="https://www.ncbi.nlm.nih.gov/pubmed/28739903">misfold.<br />
</a></p>
<p>Ongoing Molecular Dynamics simulations by <a href="http://www.iquibicen.fcen.uba.ar/investigador/marcelo-marti/">Juan Blanco, Carlos Modenutti, and Marcelo Martí</a> in Buenos Aires have highlighted a motion called clamping, which brings the TRXL1 and the TRXL3 domains of UGGT close together. To test the importance of this clamping motion we have engineered a double Cys mutant of <em>Chaetomium thermophilum</em> UGGT (aka <em>Ct</em>UGGT) which locks the clamping motions shut: S180C:T742C. This mutant is called <em>Ct</em>UGGT-N for short.</p>
<p>The cloning, protein expression and purification were successful. Mass spectrometry confirmed the existence of the desired disulphide bond. We have managed to crystallise the <em>Ct</em>UGGT-N protein in two distinct crystal forms and we have determined <a href="https://zenodo.org/record/1345671#.W3Q61X4nYfw">the structures</a>.</p>
<p>In the trigonal form, the orientation of TRXL2 with respect to the rest of the protein is the most opened ever observed so far (a rotation of 60 degrees away from the TRXL2 in <a href="https://www.rcsb.org/structure/5NV4">PDB ID 5NV4</a>). The orthorhombic form proves that the clamped shut structure (<em>Ct</em>UGGT-N P2<sub>1</sub>2<sub>1</sub>2<sub>1</sub>) is compatible with the bent shut structure (<em>Ct</em>UGGT<img class="size-medium wp-image-1728 alignleft" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/08/Slide4-300x225.jpg" alt="" width="300" height="225" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/08/Slide4-300x225.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/08/Slide4-768x576.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/08/Slide4-350x263.jpg 350w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/08/Slide4-780x586.jpg 780w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/08/Slide4-250x188.jpg 250w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/08/Slide4.jpg 841w" sizes="(max-width: 300px) 100vw, 300px" />-H, P2<sub>1</sub>2<sub>1</sub>2<sub>1</sub>, PDB ID 5NV4) <em>i.e.</em> both clamping and bending movements can close at the same time. A quadruple <em>Ct</em>UGGT mutant, containing both D611C:G1050C and S180C:T742C, should be feasible and its human UGGT1 equivalent could provide the best rigidification of the human structure conceivable so far &#8211; in order to aid crystallisation and obtain the so far elusive human protein crystal structure.</p>
]]></content:encoded>
							<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/crystal-structures-of-the-ctuggt-double-mutant-s180ct742c/feed/</wfw:commentRss>
		<slash:comments>1</slash:comments>
							</item>
	</channel>
</rss>
