<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>Diffuse intrinsic pontine glioma &#8211; openlabnotebooks.org</title>
	<atom:link href="https://thesgc.github.io/static-openlabnotebooks/category/disease-foundation/dipg/feed/" rel="self" type="application/rss+xml" />
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<description>A growing team of groundbreaking scientists around the world are now sharing their lab notebooks online</description>
	<lastBuildDate>Fri, 01 Mar 2019 09:51:22 +0000</lastBuildDate>
	<language>en-GB</language>
	<sy:updatePeriod>hourly</sy:updatePeriod>
	<sy:updateFrequency>1</sy:updateFrequency>
	<generator>https://wordpress.org/?v=4.9.9</generator>

<image>
	<url>https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/cropped-lab-jar02-32x32.png</url>
	<title>Diffuse intrinsic pontine glioma &#8211; openlabnotebooks.org</title>
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<width>32</width>
	<height>32</height>
</image> 
	<item>
		<title>Checking I haven&#8217;t got my cells mixed up&#8230;</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/checking-i-havent-got-my-cells-mixed-up/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/checking-i-havent-got-my-cells-mixed-up/#respond</comments>
		<pubDate>Thu, 28 Feb 2019 18:56:05 +0000</pubDate>
		<dc:creator><![CDATA[Elizabeth Brown]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Brain Tumours]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>
		<category><![CDATA[Liz Brown]]></category>
		<category><![CDATA[Understudied Kinases]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2542</guid>
		<description><![CDATA[Did you know as many as 30% of the cells in labs could be misidentified? And what a nightmare it would be to find out years of your work are just completely wrong because you got your skin cells mixed up with your brain cells&#8230; So to avoid that catastrophe, I took some time to <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/checking-i-havent-got-my-cells-mixed-up/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Did you know as many as 30% of the cells in labs could be misidentified? And what a nightmare it would be to find out years of your work are just completely wrong because you got your skin cells mixed up with your brain cells&#8230;</p>
<p>So to avoid that catastrophe, I took some time to check the DNA sequence of the ACVR1 gene, because for this gene we know what should be mutated (or not) for each of my cell lines. This way I can make sure that I haven&#8217;t mixed them up in the past year (<em>and</em> that they haven&#8217;t picked up new mutations in this important area all by themselves).</p>
<p>All I had to do was collect the genomes of the cells, amplify specifically the ACVR1 gene, send it for overnight sequencing, and then I get a trace like this back from the facility:</p>
<div id="attachment_2544" style="width: 790px" class="wp-caption aligncenter"><a href="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/R206H-sequence-1.png"><img class="wp-image-2544 size-large" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/R206H-sequence-1-1024x396.png" alt="Sequencing trace covering the R206H mutation in ACVR1 found in FOP and DIPG" width="780" height="302" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/R206H-sequence-1-1024x396.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/R206H-sequence-1-300x116.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/R206H-sequence-1-768x297.png 768w" sizes="(max-width: 780px) 100vw, 780px" /></a><p class="wp-caption-text">Sequencing trace covering the R206H mutation in ACVR1 found in FOP and DIPG</p></div>
<p>Each DNA base has one peak, except one that has both a guanine (G) and an adenine (A) DNA base. The adenine, present on one chromosome, causes the change from an arginine (R) to a histidine (H) at the 206th position of the ACVR1 protein i.e. the R206H mutation.</p>
<p>This trace is from the HSJD-DIPG-007 cell line and is the mutation I expected. According to all my sequencing, none of my cell lines have gotten mixed up, yay!</p>
<p>You can read the full details of how I did this, <a href="https://zenodo.org/record/2579809#.XHgumMD7SUk">here on Zenodo</a>.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/checking-i-havent-got-my-cells-mixed-up/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>ZFYVE9C purification and association test with TGFBR1 and TGFBR2</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/zfyve9c-purification-and-association-test-with-tgfbr1-and-tgfbr2/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/zfyve9c-purification-and-association-test-with-tgfbr1-and-tgfbr2/#comments</comments>
		<pubDate>Fri, 22 Feb 2019 14:05:47 +0000</pubDate>
		<dc:creator><![CDATA[Ellie Williams]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Ellie Williams]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2515</guid>
		<description><![CDATA[ZFYVE9C (Otherwise known as SARA) is a scaffold protein involved in the association of various protein components as part of the TGFB signalling pathway. The equivalent in the BMP signalling pathway is a protein known as Endofin so studying SARA (which is slightly easier) might give us useful insights for both proteins. For more details <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/zfyve9c-purification-and-association-test-with-tgfbr1-and-tgfbr2/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>ZFYVE9C (Otherwise known as SARA) is a scaffold protein involved in the association of various protein components as part of the TGFB signalling pathway. The equivalent in the BMP signalling pathway is a protein known as Endofin so studying SARA (which is slightly easier) might give us useful insights for both proteins. For more details on this project please see <a href="https://thesgc.github.io/static-openlabnotebooks/a-summary-of-my-alk2-project-goals/">this blog post.</a></p>
<p>One of the unanswered questions about SARA is what does one of the major domains that it has actually do? (The domain is known as a DUF domain, or a Domain of Unknown Function!) We know the structure but that hasn’t given us any clues as to its function. One thought was that it might be involved in recruiting type I or type II receptors close to their binding partners such as one of the SMADs to allow signal propagation.</p>
<p>I purified the DUF domain of SARA and mixed it with either excess TGFBR1 or excess TGFBR2 (the two relevant receptors in the TGFB pathway). I tested the receptors in both the unphosphorylated (inactive) and phosphorylated (active) states. After incubating the samples for 30 minutes while concentrating the volume back down to &lt;5ml, I ran them down a gel filtration column to see if the two proteins bound together sufficiently to shift the peaks coming off the column which would indicate they were acting as a complex of larger mass.</p>
<p><img class="alignnone size-large wp-image-2516" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/gf_trace_zfyve9c-1024x491.png" alt="" width="780" height="374" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/gf_trace_zfyve9c-1024x491.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/gf_trace_zfyve9c-300x144.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/gf_trace_zfyve9c-768x368.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/gf_trace_zfyve9c.png 1578w" sizes="(max-width: 780px) 100vw, 780px" /></p>
<p><em>Overlay of the UV traces from all the size exclusion gel filtration runs to show the relative elution volumes of the peaks confirming no shift in the ZFYVE9C peak consistent with no complex being formed.</em></p>
<p>&nbsp;</p>
<p>Unfortunately I didn’t see any shift in the peak for SARA suggesting that either the two proteins don’t bind or that the affinity is weak enough that they fall apart under the conditions of the gel filtration column.</p>
<p>The next thing to try will be to purify the proteins again but not to cleave the tag on one of the binding partners (either the receptors or the SARA protein) and try to do a pull down experiment to test whether this shows any association or not.</p>
<p>&nbsp;</p>
<p>You can find more details over at<a href="https://doi.org/10.5281/zenodo.2575581"> Zenodo.</a></p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/zfyve9c-purification-and-association-test-with-tgfbr1-and-tgfbr2/feed/</wfw:commentRss>
		<slash:comments>2</slash:comments>
		</item>
		<item>
		<title>Three new ALK2 structures, one of them not the one I thought!</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/three-new-alk2-structures-one-of-them-not-the-one-i-thought/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/three-new-alk2-structures-one-of-them-not-the-one-i-thought/#respond</comments>
		<pubDate>Tue, 19 Feb 2019 17:46:47 +0000</pubDate>
		<dc:creator><![CDATA[Roslin Adamson]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Brain Tumours]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>
		<category><![CDATA[Roslin Adamson]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2507</guid>
		<description><![CDATA[I’ve not posted in a while, mainly because things have been really busy, but good busy, as I now have 3 new datasets/structures, of ALK2 with M4K2118, M4K2149 and, amazingly, M4K2009. As you know, this is our lead candidate, which until now has not behaved very well in crystal trials, and it hasn’t been possible <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/three-new-alk2-structures-one-of-them-not-the-one-i-thought/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>I’ve not posted in a while, mainly because things have been really busy, but good busy, as I now have 3 new datasets/structures, of ALK2 with M4K2118, M4K2149 and, amazingly, M4K2009. As you know, this is our lead candidate, which until now has not behaved very well in crystal trials, and it hasn’t been possible to get decent crystals. In my last post I told you how I had somehow managed to mix up the TNIK and ACVR1/ALK2 tubes when I was setting up the plates, and accidentally put ACVR1 into a coarse screen and TNIK into a fine screen for ACVR1. Well, after I got lovely big crystals for what I thought was ACVR1 with M4K3003, I sent them to the synchrotron, got beautiful data to 1.4 Å, phased it, and lo and behold! It was M4K2009 snugly fitted into the binding pocket (see the ligand density after molecular replacement below). Of course, I didn’t notice this was a trimethoxy containing compound (how??) before I excitedly emailed everyone to tell them I had M4K3003, and then had to retract … However, the story ends well as M4K2009 is really important.</p>
<p><img class="aligncenter  wp-image-2511" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/M4k2009-1024x576.jpg" alt="" width="904" height="509" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/M4k2009-1024x576.jpg 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/M4k2009-300x169.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/M4k2009-768x432.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/M4k2009.jpg 1280w" sizes="(max-width: 904px) 100vw, 904px" /></p>
<p>Additionally, I got data for M4K2149 (~2.6 Å), one of the amide head group containing compounds we’re interested to see, and M4K2118 (~1.4 Å), which is doing well in some of the other assays too. I’ve put the .mtz files <a href="https://zenodo.org/record/2573142#.XGw9g-j7SUk">here</a> on Zenodo if you want to play with high resolution data, and some images below showing the ligand density and the ligands binding in the pocket. The amide head group of M4K2149 appears to preserve the water network in the pocket that we’ve observed in other structures, but as the resolution is too low for waters, we can’t tell til we get higher resolution data. Which hopefully will be this week, as I’ve grown some really pretty crystals for M4K2149 that I hope will diffract better. These appeared quite by accident again, as I’d designed a fine screen around the D5 well condition of my mega composite screen, but neglected to notice when I validated the screen that I’d put 10 x less ammonium sulphate in one row than intended – should have been 0.9 M, but ended up being 0.09 M. I should probably start calling myself ‘the accidental crystallographer’ at this rate.</p>
<p><img class="aligncenter  wp-image-2509" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/M4K2118-2149-1024x576.png" alt="" width="883" height="497" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/M4K2118-2149-1024x576.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/M4K2118-2149-300x169.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/M4K2118-2149-768x432.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/M4K2118-2149.png 1280w" sizes="(max-width: 883px) 100vw, 883px" /></p>
<p>In the meantime, I’ve got some small crystals for M4K3007, another structure we really want, and some pretty ones for M4K2184, another amide head group compound. I’ve followed up the M4K3007, just waiting to see if anything appears, and will send the tiny ones to the synchrotron while waiting.</p>
<p><img class="aligncenter  wp-image-2508" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/crystal-pics-1024x576.png" alt="" width="849" height="478" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/crystal-pics-1024x576.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/crystal-pics-300x169.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/crystal-pics-768x432.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/crystal-pics.png 1280w" sizes="(max-width: 849px) 100vw, 849px" /></p>
<p>I’m also very happy to say that our new M4K2117 crystal system for XChem fragment screening has worked very nicely so far, in the sense that the crystals are large and relatively robust and can handle soaking in low salt conditions as well as compounds with DMSO for a few hours. They then still diffract to around 1.2-1.8 Å, consistently in the same space group, C 1 2 1. This is a vast improvement over the previous system with LDN-193189. We did a solvent characterisation and pre-screen a couple of weeks ago with 100 crystals, and of the 20 or so hits we got, none was in the allosteric pocket we are looking for. This could be because of the 1.4 M ammonium sulphate and 0.2 M Na/K tartrate I have in there, which might prevent access of fragments to the pocket. Thus, using the remarkable shifter that Nathan Wright at SGC designed, we’ll be transferring loads of crystals from grow plates to low salt soak plates before doing the compound soaks this week, with the same 100 compounds, in order to check if the hit rate/location changes.</p>
<p>I will mention though, for anyone who may never have noticed this, the DMSO concentration in the protein solution for crystallisation is very important! I had previously been using Batch 1 of M4K2117 for my co-crystallisation, but it ran out so I used Batch 2, which was a 50 mM stock, as opposed to the previous 25 mM stock. The plates I set up with this, the same in every other way, gave showers of rather ugly, crunchy, small crystals. So I tried a few things – diluted the stock to 25 mM in DMSO and set up plates with that, with both lower and higher concentrations of protein (8 and 14 mg/ml along with the usual 10-11 mg/ml), as well as using the stock as is, with lower and higher protein concentrations. The lower stock concentrations brought back the big, perfect, beautiful crystals from before, the higher stock, no matter the protein concentration, gave ugly chunks and showers. The final concentration of compound was the same in all, so clearly the DMSO percentage is a very important factor.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/three-new-alk2-structures-one-of-them-not-the-one-i-thought/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>A summary of my ALK2 project goals.</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/a-summary-of-my-alk2-project-goals/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/a-summary-of-my-alk2-project-goals/#respond</comments>
		<pubDate>Tue, 05 Feb 2019 17:01:21 +0000</pubDate>
		<dc:creator><![CDATA[Ellie Williams]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Ellie Williams]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2439</guid>
		<description><![CDATA[I thought I’d spend a blog post just talking about the various paths of my research and try and tie it all together. A lot of my experiments do dot around these various arms of investigation and so one experimental post doesn’t necessarily tie in directly with the previous one or the next one and <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/a-summary-of-my-alk2-project-goals/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>I thought I’d spend a blog post just talking about the various paths of my research and try and tie it all together. A lot of my experiments do dot around these various arms of investigation and so one experimental post doesn’t necessarily tie in directly with the previous one or the next one and I thought it might be a good idea to explain why and summarise where I’m up to with each strand.</p>
<p>My main focus of research is to investigate the cause of the disease fibrodysplasia ossificans progressiva (FOP). We know that it’s caused by a mutation in ALK2 which leads to excessive bone formation. This also nicely dovetails with research done into DIPG where 25% of cases share a mutation with FOP in ALK2.</p>
<p>My research can be broken down in to three main strands:</p>
<ul>
<li>The first is looking at characterising the binding of ligands to ALK2 in order to optimise binding and work towards finding a potent and specific inhibitor. This is the focus of a lot of the M4K work that Jong Fu and Ros are doing and I’ve been doing a little bit of that as well but it’s not the main focus of my work. This endeavour is going fairly well – despite setbacks relating to protein expression (which are not entirely solved – we now have regained expression of the kinase domain alone but we still don’t have expression of the kinase domain and GS loop construct) and Ros has a number of structures that she’s working on at the moment that have been recently collected. Alas I’ve not had as much luck with new structures recently. We frequently get new deliveries of compounds from M4K to test – Jong Fu looks at them in cells while Ros and I try and crystallise the most promising candidates with varying degrees of success. Ros and I have also attempted XChem fragment screening which is where we use high throughput crystallography to identify small weakly bound fragments that might identify new leads for drug design or in our case, help us identify drugs that might bind to a second allosteric site within ALK2 away from the main ligand binding pocket. An XChem update will be coming soon from either Ros or myself as I’ve recently started helping her with this aspect of the work and so have been shadowing her on our last couple of visits to Diamond.</li>
</ul>
<p>&nbsp;</p>
<ul>
<li>The second is investigating the interactions of the type I receptor (ALK2) with its partnering type II in order to activate SMAD 1/5/8. This covers looking at receptor cross reactivity, SMAD cross reactivity, and the influence of mutations on this. This also involves looking at crystal structures of the mutants where possible, attempting to make complexes of the type I and type II receptors in order to try and crystallise them and probing the activities of the proteins and the various disease mutants. Covered in this is everything I’ve done using western blots and mass spec to identify the dependencies of this complex as required for SMAD phosphorylation. So far I know that the mutants are more active than the wild type (with R206H being the most active ALK2 mutation), that the type II must be present but does not need to be active to allow for SMAD activation and that increased activity is not solely dependent upon the removal of inhibition by FKBP12. Covered in this is also the work looking at mapping the phosphorylation sites in ALK2 to try and identify if some of these sites are more critical than others with relation to SMAD activation. The cloning I recently talked about is part of this investigation, making mutations to remove certain phosphorylation sites and comparing the activity of these with the wild type. As soon as these constructs have been cloned into suitable expression vectors I’ll be starting that experiment too. Another aspect of this is looking at the association of other proteins with the receptors, such as the possible link between them and XIAP as I talked about in one earlier blog post.</li>
</ul>
<p>&nbsp;</p>
<ul>
<li>The third aspect is looking other proteins related to the ALK2 pathway such as Endofin. Endofin is a membrane associated protein with multiple domains including a SMAD binding domain and large areas of unknown function. It is involved with the co-localisation of various parts of the pathway and appears to act as something of a scaffold protein to hold things in close proximity for signal propagation. The equivalent protein in the TGFβ pathway is called SARA and appears to have a similar function in that pathway. Some time ago I solved the structure for a domain of unknown function within SARA (4BKW) which contained a novel fold and showed structural (but not sequence) similarity to a protein called SUFU, which is part of the Hedgehog signalling pathway. We still don’t know what this protein and domain does or what it binds to or how it functions and so this is another area of investigation. If we can uncover the function of SARA we can also try and uncover the function of Endofin and this might provide a better understanding of the way FOP mutations work and help us see if there are any other ways we can target the pathway that are not exclusively based on inhibiting ALK2 activity. However given that the DUF domain has very little information about it, our investigations into what it does have been limited thus far. My goals with this are to try and probe the role of the DUF domain looking for whether it binds to either of the type I or type II receptors, and if so which parts of the receptors, or if it associates with a different protein all together. Given we know very little about what it does and the structure offers few clues (the lack of sequence similarity to SUFU means that although we can predict a possible binding mode, the nature of what binds there is very hard to determine as there are no conserved residues) this is quite a difficult problem to approach.</li>
</ul>
<p><img class="alignnone size-large wp-image-2440" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/summary-1024x606.png" alt="" width="780" height="462" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/summary-1024x606.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/summary-300x177.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/summary-768x454.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/summary.png 1324w" sizes="(max-width: 780px) 100vw, 780px" /></p>
<p>&nbsp;</p>
<p>My work moves between these three strands depending on how much and which proteins I’ve got available to me &#8211; do I have any in the freezer already purified or do I need to make fresh? And that’s if I have the pellets in the freezer of the expressed protein in the first place. It also varies depending on whether we have new compounds we’re interested in testing immediately from either M4K or other collaborators, or whether I’ve got data I need to analyse from previous successful crystallisation attempts. Sometimes I’ll be looking at my own structural data and sometimes I’ll be looking at someone else’s structure which we call ‘proofreading’ to double check it for errors and make sure nothing has been missed. Sometimes we’ll have time at Diamond and so everything else has to be put on hold in order to make use of that visit (such as XChem trips) and sometimes there’s a lot of data analysis to do on other experiments &#8211; writing it up and understanding what the data is telling us is just as important for progress as the experiments themselves.</p>
<p>Having the three strands means I can move from one to another as each strand has its own peaks and lulls in activity at different times. I’m afraid it does make for a slightly more disjointed blog journey but I hope you’ll bear with me through it all as I try and make sense of the problems I’m investigating.</p>
<p>I’ll try and post again sooner rather than later on what I’ve been doing in January as it has been really rather bitty and frustrating in terms of things actually working but hopefully February will bring with it some more productive experiments.</p>
<p>&nbsp;</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/a-summary-of-my-alk2-project-goals/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Optimising seeding of normal and FOP cells for a later assay &#8211; prettier than expected</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/optimising-seeding-of-normal-and-fop-cells-for-a-later-assay-prettier-than-expected/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/optimising-seeding-of-normal-and-fop-cells-for-a-later-assay-prettier-than-expected/#respond</comments>
		<pubDate>Tue, 22 Jan 2019 19:22:08 +0000</pubDate>
		<dc:creator><![CDATA[Elizabeth Brown]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Brain Tumours]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>
		<category><![CDATA[Liz Brown]]></category>
		<category><![CDATA[Understudied Kinases]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2388</guid>
		<description><![CDATA[Further down the line I want to be able to test the activation of the BMP signalling pathway downstream of the ALK2 receptor. For this I plan to use the alkaline phosphatase gene as a readout &#8211; it&#8217;s a well known gene activated downstream of BMP signalling so a lot of easy to use assays <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/optimising-seeding-of-normal-and-fop-cells-for-a-later-assay-prettier-than-expected/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Further down the line I want to be able to test the activation of the BMP signalling pathway downstream of the ALK2 receptor. For this I plan to use the alkaline phosphatase gene as a readout &#8211; it&#8217;s a well known gene activated downstream of BMP signalling so a lot of easy to use assays already exist for it.</p>
<p>Unfortunately <a href="https://thesgc.github.io/static-openlabnotebooks/optimising-an-assay-to-detect-alkaline-phosphatase-activity-in-c2c12-cells/">when I tested this assay with the standard C2C12 cell line it didn&#8217;t work particularly well</a> so I&#8217;ve decided to try again with a new set of cells. This time I&#8217;m choosing connective tissue cells (fibroblasts) from FOP patients as a reporter cell line because they respond more strongly to the signals that activate the BMP pathway (called BMP ligands).</p>
<p>I&#8217;ve never worked with these cells before so, again, I needed to work out how many I&#8217;ll need to get a good result from the assay. These cells however grow as a very thin sheet which is so see-through that I initially couldn&#8217;t tell if the wells were full. To solve this I used stains called Calcein AM and Hoechst. Calcein AM can stain any living cell because they metabolise it from a see-through molecule into a fluorescent green product, whereas Hoechst can stick to DNA and turn all the cell nuclei blue, and so suddenly, all of my cells glow:</p>
<div id="attachment_2389" style="width: 1382px" class="wp-caption aligncenter"><a href="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/Calcein-AM-example.png"><img class="wp-image-2389 size-full" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/Calcein-AM-example.png" alt="Fibroblast cells stained with Calcein AM (green) and Hoechst (blue)" width="1372" height="784" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/Calcein-AM-example.png 1372w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/Calcein-AM-example-300x171.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/Calcein-AM-example-768x439.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/Calcein-AM-example-1024x585.png 1024w" sizes="(max-width: 1372px) 100vw, 1372px" /></a><p class="wp-caption-text">Fibroblast cells stained with Calcein AM (green) and Hoechst (blue)</p></div>
<p>Aren&#8217;t they pretty?</p>
<p>Once they&#8217;re this easy to see, it&#8217;s even easier to see how many cells are needed to fill a well, but not get overcrowded. You can read more about how I did that in my <a href="https://zenodo.org/record/2546911">Zenodo post</a>.</p>
<p>The only question I have left is why some cells didn&#8217;t stain with Calcein AM, even though their nucleus looks fine? Anyone know the answer?</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/optimising-seeding-of-normal-and-fop-cells-for-a-later-assay-prettier-than-expected/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>A new ALK2 structure, and proof that mistakes in the lab aren’t always a bad thing</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/a-new-alk2-structure-and-proof-that-mistakes-in-the-lab-arent-always-a-bad-thing/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/a-new-alk2-structure-and-proof-that-mistakes-in-the-lab-arent-always-a-bad-thing/#respond</comments>
		<pubDate>Fri, 04 Jan 2019 15:28:47 +0000</pubDate>
		<dc:creator><![CDATA[Roslin Adamson]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Brain Tumours]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>
		<category><![CDATA[Roslin Adamson]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2364</guid>
		<description><![CDATA[Hello everyone and happy New Year! I really, really wanted to end the year on a high note in December before I went off on holiday, but as it turned out, I didn’t have enough time to set up 18 plates and write a blog post on the last day. So I’ll just have to <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/a-new-alk2-structure-and-proof-that-mistakes-in-the-lab-arent-always-a-bad-thing/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Hello everyone and happy New Year! I really, really wanted to end the year on a high note in December before I went off on holiday, but as it turned out, I didn’t have enough time to set up 18 plates and write a blog post on the last day. So I’ll just have to start the year on a high note.</p>
<p>I’m very happy to say I got another ACVR1/ALK2 structure in December, at 2 Å resolution, co-crystallised with M4K2158 (see images below, and the .mtz autoprocessed file is available <a href="https://zenodo.org/record/2531714#.XC94jlz7SUk">here</a> on Zenodo). The structure is not fully refined yet, but what’s interesting is the solvent side of the molecule is being stabilised by either a water contact (Chain A) or Asp293 (Chain B). This gives it really good density.</p>
<p>&nbsp;</p>
<div id="attachment_2367" style="width: 865px" class="wp-caption aligncenter"><img class="wp-image-2367 " src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/M4K2158_structure-1024x576.png" alt="" width="855" height="481" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/M4K2158_structure-1024x576.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/M4K2158_structure-300x169.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/M4K2158_structure-768x432.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/M4K2158_structure.png 1280w" sizes="(max-width: 855px) 100vw, 855px" /><p class="wp-caption-text">ACVR1 bound to M4K2158 at ~2 Å, showing interactions with His286 and a water at the back of the pocket</p></div>
<p>&nbsp;</p>
<p>I also seem to have some new <span style="font-size: 1.5rem;">pretty </span><span style="font-size: 1.5rem;">hits with M4K2118, M4K2149 and M4K3003 (crystal pics below). What’s interesting about the M4K3003 hits is that I’ve really struggled to get crystals with this compound (as well as M4K3007 and 3010), and it is a pretty desirable structure. These hits came about as a result of a mistake, where I somehow managed to mix up my tubes and put ACVR1 with M4K3003 into my mega composite screen without seeds, and TNIK into a fine screen I had intended for ACVR1 with seeds. I only discovered this because I mass specced the TNIK just before setting up the plates, then rushed off to do them as it was the SGC Christmas party that night and I needed to finish (best not to rush experiments). When I checked the mass spec result later, I got the exact mass of ACVR1! So I went back to the crystallography room and recovered what protein I could from the source plate for the TNIK plates, and mass specced it again. Same result – ACVR1. But when I looked at the drop images for those plates (now containing ACVR1 + M4K3003), I found a bunch of crystals! Happy serendipity!! These will go on the trip to Diamond on the 21</span><sup>st</sup><span style="font-size: 1.5rem;"> of January and then we’ll know whether the mistake really ended well or not. Fingers crossed!</span></p>
<p>&nbsp;</p>
<div id="attachment_2366" style="width: 893px" class="wp-caption aligncenter"><img class="wp-image-2366" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/Crystal-hits-Dec_Jan-2018_9-1024x576.png" alt="" width="883" height="497" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/Crystal-hits-Dec_Jan-2018_9-1024x576.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/Crystal-hits-Dec_Jan-2018_9-300x169.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/Crystal-hits-Dec_Jan-2018_9-768x432.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/Crystal-hits-Dec_Jan-2018_9.png 1280w" sizes="(max-width: 883px) 100vw, 883px" /><p class="wp-caption-text">ACVR1 co-crystallised with M4K2149, 2118 and 3003 &#8211; hits to send to the synchrotron</p></div>
<p>&nbsp;</p>
<p>You may well ask, why was I setting up plates with TNIK? As it happens, some of our collaborators’ assays show M4K2009, our main compound of interest, hits both TNIK and DDR1 with pretty high affinity. We’ve been struggling to get crystals of ACVR1 with M4K2009 – for whatever reason, it doesn’t want to make crystals. So it seemed like a good idea to see if TNIK or DDR1 will co-crystallise with it. TNIK (TRAF2 and NCK Interacting Kinase) is a serine/threonine kinase that functions as an activator of the Wnt signalling pathway. DDR1 (Discoidin Domain Receptor Tyrosine Kinase 1) is a tyrosine kinase that functions as cell surface receptor for fibrillar collagen and regulates cell attachment to the extracellular matrix, remodelling of the extracellular matrix, cell migration, differentiation, survival and cell proliferation (according to its GeneCard).</p>
<p>While I was about it, I thought it would be useful to set up plates with BMPR1A (ALK3), for the same reason I set up TGFBR1 (ALK5) before – to see what off-target structures look like with some of these compounds. The purification protocols for TNIK and BMPR1A were as standard, but they’re available <a href="https://zenodo.org/record/2531714#.XC94jlz7SUk">here</a> on Zenodo.</p>
<p>Some other good news is that I’ve figured out how to reproduce my XChem crystals fairly reliably. On the last day before the holidays I set up 6 plates of ACVR1 with M4K2117, which gives really beautiful, well-diffracting crystals. I used 10, 12 and 15 mg/ml protein, and made plates with and without seeds. Well, the ones without the seeds grew fabulously, within 2 weeks, and the ones with seeds didn’t! The next step with these then is to test what happens if I soak them in different formulations of lower salt with and without some PEG3350. That&#8217;ll be for next week. Have a great weekend!</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/a-new-alk2-structure-and-proof-that-mistakes-in-the-lab-arent-always-a-bad-thing/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Determining the nanoBRET ALK2 IC50 values of 24 new ACVR1/ALK2 inhibitors</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/determining-the-nanobret-al2k-ic50-values-of-24-new-acvr1-alk2-inhibitors/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/determining-the-nanobret-al2k-ic50-values-of-24-new-acvr1-alk2-inhibitors/#respond</comments>
		<pubDate>Wed, 19 Dec 2018 06:39:05 +0000</pubDate>
		<dc:creator><![CDATA[Jong Fu]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Brain Tumours]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Jong Fu Wong]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2314</guid>
		<description><![CDATA[In an effort to develop clinical compounds for Diffused Intrinsic Pontine Glioma (DIPG) treatment, new analogues of ACVR1/ALK2 inhibitors are continuously synthesised by Ontario Institute for Cancer Research (OICR) and Charles River Laboratories (CRL). I will provide prompt feedback of the cellular assay results to guide their design of new compounds. I determined the potency <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/determining-the-nanobret-al2k-ic50-values-of-24-new-acvr1-alk2-inhibitors/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>In an effort to develop clinical compounds for Diffused Intrinsic Pontine Glioma (DIPG) treatment, new analogues of ACVR1/ALK2 inhibitors are continuously synthesised by <a href="https://oicr.on.ca/">Ontario Institute for Cancer Research (OICR)</a> and <a href="https://www.criver.com/">Charles River Laboratories (CRL)</a>. I will provide prompt feedback of the cellular assay results to guide their design of new compounds.</p>
<div id="attachment_2315" style="width: 397px" class="wp-caption alignnone"><img class=" wp-image-2315" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/Transfection-control.png" alt="" width="387" height="387" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/Transfection-control.png 575w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/Transfection-control-150x150.png 150w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/Transfection-control-300x300.png 300w" sizes="(max-width: 387px) 100vw, 387px" /><p class="wp-caption-text">HEK293 cells were nicely transfected, as indicated by GFP signal. I can proceed with the experiment.</p></div>
<div id="attachment_2316" style="width: 656px" class="wp-caption alignnone"><img class="size-full wp-image-2316" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/Figure-2.png" alt="" width="646" height="720" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/Figure-2.png 646w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/Figure-2-269x300.png 269w" sizes="(max-width: 646px) 100vw, 646px" /><p class="wp-caption-text">ALK2 NanoBRET IC50 curves and chemical structures of the first dozen of new ACVR1/ALK2 inhibitors. IC50 values estimated by GraphPad Prism are shown in green.</p></div>
<div id="attachment_2317" style="width: 679px" class="wp-caption alignnone"><img class="size-full wp-image-2317" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/Figure-3.png" alt="" width="669" height="720" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/Figure-3.png 669w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/Figure-3-279x300.png 279w" sizes="(max-width: 669px) 100vw, 669px" /><p class="wp-caption-text">ALK2 NanoBRET IC50 curves and chemical structures of the second dozen of new ACVR1/ALK2 inhibitors. IC50 values estimated by GraphPad Prism are shown in green.</p></div>
<p>I determined the potency of 24 new ACVR1/ALK2 inhibitors synthesised by OICR. M4K2009 in particular is quite potent (<a href="https://en.wikipedia.org/wiki/IC50">larger values indicates lower potency</a>). This compound differs from legacy compound M4K1055 by the position of its methyl group (CH3) in the middle of the molecule. Attempts to replace the tri-methoxy group (the three CH3-O groups bonded to the phenyl ring) were not successful. Since a robust nanoBRET tracer is not available for TGFBR1/ALK5, I will use dual luciferase promoter assay (orthologous assay) to determine the off-target activity of these compounds towards TGFBR1/ALK5.</p>
<div id="attachment_2322" style="width: 440px" class="wp-caption alignnone"><img class=" wp-image-2322" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/Comparing-M4K2009-to-legacy-M4K1055.png" alt="" width="430" height="376" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/Comparing-M4K2009-to-legacy-M4K1055.png 326w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/Comparing-M4K2009-to-legacy-M4K1055-300x262.png 300w" sizes="(max-width: 430px) 100vw, 430px" /><p class="wp-caption-text">Comparing the chemical structures of M4K2009 to legacy compound M4K1055. ALK2 nanoBRET IC50 values of the compounds are also shown (lower values reflects increased potency).</p></div>
<p>The M4K monthly meeting where this result was presented has been recorded and posted on <a href="https://www.youtube.com/watch?v=YAsVIt_iBes&amp;t=1464s">youtube</a>.</p>
<p>For detailed experimental protocols, please refer to my <a href="https://zenodo.org/record/2410328#.XBnjM1X7Spo">Zenodo page</a>.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/determining-the-nanobret-al2k-ic50-values-of-24-new-acvr1-alk2-inhibitors/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Purification and crystallisation of an ALK2/ACVR2/FKPB12 complex.</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/purification-and-crystallisation-of-an-alk2-acvr2-fkpb12-complex/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/purification-and-crystallisation-of-an-alk2-acvr2-fkpb12-complex/#respond</comments>
		<pubDate>Fri, 14 Dec 2018 15:30:55 +0000</pubDate>
		<dc:creator><![CDATA[Ellie Williams]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Ellie Williams]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2261</guid>
		<description><![CDATA[One problem with working with the isolated kinase domains of ALK2 and it’s type II binding partners, is that the affinity between the kinase domains alone is insufficient to allow complex formation. We know that the two proteins associate with each other in vivo and are co-localised by the domains on the outside of the <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/purification-and-crystallisation-of-an-alk2-acvr2-fkpb12-complex/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>One problem with working with the isolated kinase domains of ALK2 and it’s type II binding partners, is that the affinity between the kinase domains alone is insufficient to allow complex formation. We know that the two proteins associate with each other in vivo and are co-localised by the domains on the outside of the membrane. However when we just mix the two kinase domains we don’t see a strong enough association to allow a complex to be purified. We know from in-vitro tests such as the phosphorylation tests I’ve mentioned previously that there must at least be transient contact to allow phosphorylation but that isn’t sufficient to set up crystal plates of the complex.</p>
<p>One way to try and get around this is to artificially tether the two kinase domains together with a flexible loop. I have a construct which does exactly this, tethering one end of ACVR2 to ALK2. It’s known as QQ02ACVR2A. I purified this protein and then formed a complex with the binding partner FKBP12 which will bind to the GS domain of ALK2 present in the construct.</p>
<p>The first test during purification was to see if a complex could be assembled between the fusion construct and FKBP12. As they are very differences sizes (72 kDa vs. 12 kDa) purification by gel filtration gives a very clear answer. We can clearly see co-elution of the fusion and FKBP12 which is only possible if they were present as a complex. This in its self is quite exciting as it shows the fusion is still able to bind to its physiological binding partner.</p>
<p>The second stage after purification is to attempt to crystallise the complex. I concentrated the purified protein down and added compound K02288. K02288 definitely binds ALK2 and has some affinity for ACVR2 which is necessary for stabilising both kinase domains sufficiently to crystallise. I’ve tried crystallising ALK2 alone without compound before and had zero success so it was important to add a compound to this crystallisation attempt.</p>
<p>I set up four coarse screen plates and got the following in the HCS screen:</p>
<p><img class="alignnone size-medium wp-image-2262" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/initial_crystals-300x286.png" alt="" width="300" height="286" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/initial_crystals-300x286.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/initial_crystals.png 598w" sizes="(max-width: 300px) 100vw, 300px" /></p>
<p>B6: 20% PEG8000, 0.2M Mg Acetate, 0.1M Cacodylate pH6.5</p>
<p>A7: 1.4M Na Acetate, 0.1M Cacodylate pH6.5</p>
<p>A8: 30% 2-propanol, 0.2M Na Citrate tribasic, 0.1M Cacodylate pH6.5</p>
<p>These are crystalline but not crystals so I designed a follow up screen based around these conditions. I then used what was left of my complex to set up two plates in this followup screen at 4C and 20C.</p>
<p>After 7 days I found the following in my plates:</p>
<p><img class="alignnone size-large wp-image-2263" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/FU_xtals-1024x633.png" alt="" width="780" height="482" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/FU_xtals-1024x633.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/FU_xtals-300x185.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/FU_xtals-768x475.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/FU_xtals.png 1160w" sizes="(max-width: 780px) 100vw, 780px" /></p>
<p>I mounted five examples and sent them to the synchrotron. Alas they all came back as either salt or no diffraction. While the salt must be ignored, it’s possible that the no diffraction crystals are protein but just do not diffract. The next step will be to make seed stocks from these wells, purify more protein and set up some more plates to see if I can get any better crystals.</p>
<p>You can find more details <a href="https://doi.org/10.5281/zenodo.2276755">over at Zenodo.</a></p>
<p>I&#8217;m off for Christmas the middle of next week but in the new year I&#8217;ll be trying again at crystallising a variety of different proteins and compounds as well as working on characterising the phospho-mutants of ALK2 I posted about last time. Have a great holiday!</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/purification-and-crystallisation-of-an-alk2-acvr2-fkpb12-complex/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Screening ACVR1 inhibitors on mutant and non-mutant ACVR1 DIPG cells &#8211; effectiveness may vary</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/screening-acvr1-inhibitors-on-mutant-and-wild-type-acvr1-dipg-cells-effectiveness-may-vary/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/screening-acvr1-inhibitors-on-mutant-and-wild-type-acvr1-dipg-cells-effectiveness-may-vary/#respond</comments>
		<pubDate>Fri, 07 Dec 2018 17:08:35 +0000</pubDate>
		<dc:creator><![CDATA[Elizabeth Brown]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Brain Tumours]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Liz Brown]]></category>
		<category><![CDATA[Understudied Kinases]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2226</guid>
		<description><![CDATA[Hi there! The last month of my life was taken over by making sure my PhD first year report was beautifully polished, but I have returned with results of a small compound screen: These are all compounds that Jong Fu has already tested with his assays so we know they effectively inhibit ACVR1, but I <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/screening-acvr1-inhibitors-on-mutant-and-wild-type-acvr1-dipg-cells-effectiveness-may-vary/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Hi there! The last month of my life was taken over by making sure my PhD first year report was beautifully polished, but I have returned with results of a small compound screen:</p>
<p>These are all compounds that <a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/jong-fu-wong/">Jong Fu</a> has already tested with his assays so we know they effectively inhibit ACVR1, but I thought it would be a good idea to try them out on my DIPG cells and see how well they can kill them. I included a few different variables in this screen to test what could make a difference to the outcome:</p>
<ul>
<li>I tested cells grown in 2D and 3D because this has been noted to change the outcome of compound screens in other systems</li>
<li>I tested the compounds alone, or in addition to radiation in order to test whether the compounds were effective in addition to current DIPG treatments.</li>
</ul>
<p>I tested how effective each compound is by growing cells for 7 days in the presence of the compound. At the end I measured how alive or healthy the cells were (i.e. their viability) by using a pre-developed kit that measures the amount of ATP inside the cell by using an enzyme that can produce light when ATP is present. ATP is measured because this is sometimes called the &#8216;energy currency of the cell&#8217; so living cells have to have ATP inside them, whereas dead cells won&#8217;t and won&#8217;t be counted.</p>
<p>This screen produced a lot of data at once which is best visualised as heatmaps like these:</p>
<div id="attachment_2227" style="width: 790px" class="wp-caption aligncenter"><a href="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/ALK2i-figure-heatmap.png"><img class="size-large wp-image-2227" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/ALK2i-figure-heatmap-860x1024.png" alt="" width="780" height="929" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/ALK2i-figure-heatmap-860x1024.png 860w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/ALK2i-figure-heatmap-252x300.png 252w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/ALK2i-figure-heatmap-768x914.png 768w" sizes="(max-width: 780px) 100vw, 780px" /></a><p class="wp-caption-text"><strong>ACVR1 inhibitor M4K2009 is best at reducing mutant ACVR1 DIPG cell viability, but is not effective in ACVR1 wild-type cells</strong><br />Heatmaps showing the viability (relative to an untreated control) of a 2D culture of mutant ACVR1 DIPG cells (A) or wild-type ACVR1 DIPG cells (B) following 7 days in the presence of 1μM of the indicated ALK2 inhibitors. The colour of each box indicates how viable the cells were at the end (where blue indicates an improvement in viability and red indicates a drop in viability). Each column indicates an individual (technical repeats, n1-3) grouped into those done at the same time (biological repeats 1-3).</p></div>
<p>From that heatmap you can see that M4K2009 is the best at killing ACVR1 mutant cells (the top heatmap), but it isn&#8217;t as effective in cells where the ACVR1 gene is not mutated (&#8220;wild-type&#8221; cells). In those cells instead M4K2096 is the most effective compound (but still not to the same extent as M4K2009).</p>
<p>But this is only the data for cells grown in 2 dimensions without a radiation treatment. If you want to see the full dataset, you can read my <a href="https://zenodo.org/record/2000673#.XAqo_FX7SUk">Zenodo post</a>!</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/screening-acvr1-inhibitors-on-mutant-and-wild-type-acvr1-dipg-cells-effectiveness-may-vary/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Update on crystal hits, XChem and more purification</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/update-on-crystal-hits-xchem-and-more-purification/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/update-on-crystal-hits-xchem-and-more-purification/#comments</comments>
		<pubDate>Wed, 28 Nov 2018 16:22:44 +0000</pubDate>
		<dc:creator><![CDATA[Roslin Adamson]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Brain Tumours]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>
		<category><![CDATA[Roslin Adamson]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2191</guid>
		<description><![CDATA[Things have been really busy in the lab the last few weeks. I’ve done a couple of purifications of ACVR1 (ALK2), which I’ll post in Zenodo (getting loads of protein now), but probably won’t write up the details of every purification every single time anymore, as it’s the same process every time for this construct. <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/update-on-crystal-hits-xchem-and-more-purification/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Things have been really busy in the lab the last few weeks. I’ve done a couple of purifications of ACVR1 (ALK2), which I’ll post in <a href="https://zenodo.org/record/1639900#.W_6-6Wj7SUk">Zenodo</a> (getting loads of protein now), but probably won’t write up the details of every purification every single time anymore, as it’s the same process every time for this construct. Please feel free to access earlier Zenodo posts for the protocol, or contact me and I’ll send you the links. I’ve also set up at least 50 crystal plates over the past month, most using the composite screen I made a while back, as well as follow up screens for hits in those conditions.</p>
<p>&nbsp;</p>
<p>I haven’t been lucky with other compounds again like I was with the big, beautiful M4K2117 crystals, but, after initial apparent abject failure to reproduce these, I now appear to have grown a plate of simply gorgeous beauties (about 40 wells with lovely crystals, which in theory should diffract around 1.5 Å or better) (see pics below, verified as protein with UV imaging). I’ve done this for the fragment screening side of the project, where we’re using x-ray crystallography to screen compound fragments in search of an allosteric binding site on ACVR1, as described in earlier posts. The crystal system I’ve been using is with LDN-193189 co-crystallised with ACVR1. The crystals are big and beautiful, but suffer from some issues: they give variable space groups, tricky for the pandda software to deal with, they have thick skins, which makes soaking somewhat variable and necessarily increases the soak time, as well as making mounting tricky. In many cases while mounting, I noticed the skin was formed almost as an envelope around the top and bottom of the drop, thus when you try to pull back the skin, it pulls the crystal with it and then wraps it up tightly in a big lump of sticky skin. I’m not that great at the micro-surgery needed to separate the mess out. Additionally, many of these crystals melt or become friable when soaked too long (more than ~2 hours).</p>
<p>&nbsp;</p>
<p>The M4K2117 ACVR1 co-crystals however are in nice clear drops, although the conditions are very salty (0.1 M citrate pH 5.0 – 6.0, 1.4M ammonium sulphate, 0.2M sodium/potassium tartrate). This could be an issue for fragment binding – acidic conditions with lots of charged species around. Thus, I intend to test what happens if I soak them in different formulations of lower salt with and without some PEG3350. Also, I need to figure out how to grow them reproducibly, as only those 40 wells from one plate out of 6 set up actually grew. It’s unclear why 2 other plates the same conditions and 3 other plates with essentially the same condition didn’t grow. Or perhaps I just need to wait a little longer (they take about a month to grow).  Many of the wells without crystals have a lot of phase separation, as shown in the pic below as well.</p>
<p>&nbsp;</p>
<div id="attachment_2194" style="width: 521px" class="wp-caption aligncenter"><img class="wp-image-2194 size-full" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/Allosteric-pocket.png" alt="" width="511" height="687" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/Allosteric-pocket.png 511w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/Allosteric-pocket-223x300.png 223w" sizes="(max-width: 511px) 100vw, 511px" /><p class="wp-caption-text">The allosteric site in question, and the current LDN-193189 co-crystals we are using</p></div>
<div id="attachment_2195" style="width: 902px" class="wp-caption aligncenter"><img class="wp-image-2195 size-full" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/M4K2117-1.png" alt="" width="892" height="657" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/M4K2117-1.png 892w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/M4K2117-1-300x221.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/M4K2117-1-768x566.png 768w" sizes="(max-width: 892px) 100vw, 892px" /><p class="wp-caption-text">ACVR1-M4K2117 co-crystals, with corresponding UV images showing these are indeed protein. The lower right pics show the phase separation that is in the majority of wells.</p></div>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/update-on-crystal-hits-xchem-and-more-purification/feed/</wfw:commentRss>
		<slash:comments>2</slash:comments>
		</item>
	</channel>
</rss>
