<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>ACVR1/ALK2 &#8211; openlabnotebooks.org</title>
	<atom:link href="https://thesgc.github.io/static-openlabnotebooks/category/project/acvr1-alk2/feed/" rel="self" type="application/rss+xml" />
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<description>A growing team of groundbreaking scientists around the world are now sharing their lab notebooks online</description>
	<lastBuildDate>Sun, 02 Jun 2019 11:37:14 +0000</lastBuildDate>
	<language>en-GB</language>
	<sy:updatePeriod>hourly</sy:updatePeriod>
	<sy:updateFrequency>1</sy:updateFrequency>
	<generator>https://wordpress.org/?v=4.9.10</generator>

<image>
	<url>https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/cropped-lab-jar02-32x32.png</url>
	<title>ACVR1/ALK2 &#8211; openlabnotebooks.org</title>
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<width>32</width>
	<height>32</height>
</image> 
	<item>
		<title>ACVR1 Crystals taken to the Diamond Light Source (Beamline I04.) &#8211; no data sets collected.</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/acvr1-crystals-taken-to-the-diamond-light-source-beamline-i04-no-data-sets-collected/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/acvr1-crystals-taken-to-the-diamond-light-source-beamline-i04-no-data-sets-collected/#respond</comments>
		<pubDate>Mon, 13 May 2019 13:19:13 +0000</pubDate>
		<dc:creator><![CDATA[Ellie Williams]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Ellie Williams]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2861</guid>
		<description><![CDATA[Following up on the crystals from my last post I sent them along on the last Diamond trip we had to beamline I04 but regrettably didn’t get any usable datasets. Perhaps fortunately there is a possible explanation for this that isn’t just ‘the crystals were a bit too rubbish’. We send our crystals to the Diamond <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/acvr1-crystals-taken-to-the-diamond-light-source-beamline-i04-no-data-sets-collected/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Following up on the crystals from my last post I sent them along on the last Diamond trip we had to <a href="https://www.diamond.ac.uk/Instruments/Mx/I04">beamline I04</a> but regrettably didn’t get any usable datasets. Perhaps fortunately there is a possible explanation for this that isn’t just ‘the crystals were a bit too rubbish’. We send our crystals to the Diamond Light Source and because of the high number of crystals sent on each trip by the department (124 on the last trip) not everyone goes along on every trip and we quite often use something called ‘auto-centering’ that uses a computer algorithm to find the edge of the loop and to align the loop and your crystal in the beam. Centering is necessary but also quite tricky, because to collect a data set you need ‘slices’ of data taken with the crystal at different angles so the loop the crystals are mounted in, is rotated by a robotic arm so that the crystal also rotates around inside the x-ray beam. Thus if your crystal isn’t aligned properly, as the loop rotates it moves you crystal out of the beam and thus you get no data at that point.</p>
<p>Usually the automatic alignment is really good, however this time around it didn’t work so well and quite often either my crystal wasn’t in the beam at all (it mistook the loop stem for the crystal) or it started in the beam and then rotated out of it (which is what I think happened in the case shown below where you can see two ‘periods’ of data with dips in between. This is, of course, suboptimal and leads to a data set that you can’t really use.</p>
<p>I’m resending the crystals to the beamline this Friday and hopefully going with them myself. The silver lining to this is that I know that some of them definitely diffract which is encouraging. Hopefully they’ll behave well when fully in the beam.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/05/xtal_centering.png" alt="" width="940" height="430" class="alignnone size-full wp-image-2862" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/05/xtal_centering.png 940w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/05/xtal_centering-300x137.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/05/xtal_centering-768x351.png 768w" sizes="(max-width: 940px) 100vw, 940px" /></p>
<p><em>Fig. 1 – Beam aligned at very edge of loop showing cyclic diffraction suggesting the crystal was moving in and out of the beam rather than degrading in the beam.</em></p>
<p>&nbsp;</p>
<p>You can read more about this <a href="https://doi.org/10.5281/zenodo.2788112">over on Zenodo.</a></p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/acvr1-crystals-taken-to-the-diamond-light-source-beamline-i04-no-data-sets-collected/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>How many cells in DIPG behave like stem cells?</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/how-many-cells-in-dipg-behave-like-stem-cells/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/how-many-cells-in-dipg-behave-like-stem-cells/#respond</comments>
		<pubDate>Mon, 06 May 2019 15:40:30 +0000</pubDate>
		<dc:creator><![CDATA[Elizabeth Brown]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Brain Tumours]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Liz Brown]]></category>
		<category><![CDATA[Understudied Kinases]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2804</guid>
		<description><![CDATA[Stem cells are a set of cells that, unlike most cells in the body, can divide as many times as they want, and can turn into any cell in the body. The mutations in cancer often make the cells behave more like stem cells, because in order to grow into a large tumour they may <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/how-many-cells-in-dipg-behave-like-stem-cells/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p><a href="https://stemcells.nih.gov/info/basics/1.htm">Stem cells</a> are a set of cells that, unlike most cells in the body, can divide as many times as they want, and can turn into any cell in the body. The mutations in cancer often make the cells behave more like stem cells, because in order to grow into a large tumour they may need to divide hundreds or thousands of cells, and it can allow a single cell to escape to another part of the body and grow into a second tumour (this is called metastasis). Knowing whether all of the cells in a tumour, or even a small proportion of them behave like this, and whether they&#8217;re killed by new therapies can be important information &#8211; if just one of these cells survives theoretically they can regrow into an entire new tumour.</p>
<p>And this ability to regenerate an tumour is just how we test how many of these cells are present: I developed an assay where I dilute the cells until I can seed just a handful into each well, and then I count how many new colonies (like tiny tumours) have grown.</p>
<p>I then used a statistical rule (called <a href="http://www.biology-pages.info/L/LimitingDilution.html">the Poisson distribution</a>) and a programming package (called R) to work out at what dilution there is just one stem-like cell present i.e. if only one stem cell is present when I seeded 5 cells then 20% are stem-like.</p>
<div id="attachment_2805" style="width: 310px" class="wp-caption aligncenter"><a href="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/05/poisson-graph.png"><img class="size-medium wp-image-2805" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/05/poisson-graph-300x300.png" alt="A graph showing how the program mathematically transforms the data and then finds a straight line that fits it. Where this line crosses -1 indicates where 1 stem-like cell is present." width="300" height="300" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/05/poisson-graph-300x300.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/05/poisson-graph-150x150.png 150w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/05/poisson-graph-768x768.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/05/poisson-graph.png 960w" sizes="(max-width: 300px) 100vw, 300px" /></a><p class="wp-caption-text">The program mathematically transforms the data and then finds a straight line that fits it. Where this line crosses -1 indicates where 1 stem-like cell is present.</p></div>
<p>Although I need to repeat it to make sure the result is reliable, it seems that almost all of the cells in this line are stem-like! <a href="https://zenodo.org/record/2669443">If you&#8217;d like to read about the technical details of this experiment you can do so here.</a></p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/how-many-cells-in-dipg-behave-like-stem-cells/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Re-Purification of ALK2 for crystallisation with different compounds from M4K.</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/re-purification-of-alk2-for-crystallisation-with-different-compounds-from-m4k/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/re-purification-of-alk2-for-crystallisation-with-different-compounds-from-m4k/#respond</comments>
		<pubDate>Wed, 24 Apr 2019 14:50:59 +0000</pubDate>
		<dc:creator><![CDATA[Ellie Williams]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Ellie Williams]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2753</guid>
		<description><![CDATA[I’ve been lucky enough to have a student (Mira) in the lab working with me for the past few weeks and so I’d like to write about the work that we’ve been doing together. We spent some time cleaning up an old sample of ALK2 that Ros had left over in the freezer to set <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/re-purification-of-alk2-for-crystallisation-with-different-compounds-from-m4k/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>I’ve been lucky enough to have a student (Mira) in the lab working with me for the past few weeks and so I’d like to write about the work that we’ve been doing together.</p>
<p>We spent some time cleaning up an old sample of ALK2 that Ros had left over in the freezer to set up in crystal plates with some compounds from M4K Pharma that they were relatively keen to see the structures of.</p>
<p>We used a specialised coarse screen that Ros had designed containing only conditions where we had previously seen crystals in the wider coarse screens. This means that we can narrow down the field where we’re expecting to find crystals and test fewer conditions to find the one we need – this means setting up fewer plates and thus needing less protein per compound tested, so is overall more efficient.</p>
<p>We set up the plates (at 4C and 20C) at around 9mg/ml with compounds M4K2163, M4K2143, M4K2192, M4K2207 and M4K2194 at 1mM concentration from 25mM stocks.</p>
<p>We found some interesting crystals, one of which was mountable as is (from M4K2192) and several across all the compounds, which showed small crystals that would be worth following up in more specific conditions.</p>
<p><img class="alignnone size-full wp-image-2754" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/M4K2192.png" alt="" width="403" height="294" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/M4K2192.png 403w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/M4K2192-300x219.png 300w" sizes="(max-width: 403px) 100vw, 403px" /></p>
<p><em>Crystals of ALK2/ACVR1A with compound M4K2192</em></p>
<p>&nbsp;</p>
<p>So Mira designed some follow up screens based on the conditions that showed small crystals and we set up some more plates with those conditions in the hope of turning those small crystals into larger, better crystals. We had enough protein to set up followup plates for compounds M4K2163 and M4K2143 at 9.5mg/ml. From these plates we mounted 5 crystals across the board and combined with those above ended up with crystals of unknown quality across three compounds. These have been confirmed as containing protein by imaging the drops by UV light (UV light interacts with protein to make crystals only visible under UV light if they contain protein rather than just being salt) and are now waiting to be sent to the synchrotron to see if they actually diffract or not.  You can read more about the technical details <a href="https://zenodo.org/record/2650203#.XMB29jBKiUk">here on Zenodo.</a></p>
<p>&nbsp;</p>
<p><span style="font-size: 1.5rem;">Yesterday (23</span><span style="font-size: 11.25px; height: 0px; line-height: 0; position: relative; vertical-align: baseline; bottom: 1ex;">rd</span><span style="font-size: 1.5rem;">) also marked FOP awareness day and so, shamefully a day late, I’d like to add my picture to the many others out there celebrating</span><span style="font-size: 1.5rem;"> </span><strong style="font-size: 1.5rem;">#FunFeet4FOP</strong><span style="font-size: 1.5rem;">! Here are my toes.</span></p>
<p><img class="alignnone size-medium wp-image-2755" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Feet4FOP_crop-300x223.jpg" alt="" width="300" height="223" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Feet4FOP_crop-300x223.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Feet4FOP_crop-768x570.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Feet4FOP_crop-1024x761.jpg 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Feet4FOP_crop.jpg 1143w" sizes="(max-width: 300px) 100vw, 300px" /></p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/re-purification-of-alk2-for-crystallisation-with-different-compounds-from-m4k/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>The Benefits of Conference attendance:  BMP Signalling in Cancer II.</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/the-benefits-of-conference-attendance-bmp-signalling-in-cancer-ii/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/the-benefits-of-conference-attendance-bmp-signalling-in-cancer-ii/#respond</comments>
		<pubDate>Mon, 15 Apr 2019 10:44:07 +0000</pubDate>
		<dc:creator><![CDATA[Ellie Williams]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Ellie Williams]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2728</guid>
		<description><![CDATA[I’ve got a few things to catch up with on the blog – one of them is to talk about a conference I attended in Oxford recently. I attended the Biochemical Society BMP Signalling in Cancer II meeting which was held in St. Annes College, Oxford. At this meeting I presented a poster on the <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/the-benefits-of-conference-attendance-bmp-signalling-in-cancer-ii/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>I’ve got a few things to catch up with on the blog – one of them is to talk about a conference I attended in Oxford recently. I attended the <a href="https://www.biochemistry.org/Events/tabid/379/MeetingNo/SA217/view/Conference/Default.aspx">Biochemical Society BMP Signalling in Cancer II</a> meeting which was held in St. Annes College, Oxford. At this meeting I presented a poster on the work I’ve been doing (and which has been written about here as it’s happened) on looking at how the mutations in ALK2 effect the underlying activity of the kinase as well as effecting how well FKBP12 can inhibit ALK2.</p>
<p>The conference brought together 60 researchers in the field from around the world talking about a variety of different cancers but also other BMP related conditions such as FOP. Over three days there were some fantastic talks by the leaders in the field including from Dr. Caroline Hill from the Crick Institute in London who spoke about receptor cross talk in BMP signalling and how that might affect signalling through ALK2 and Prof. Eileen Shore from the university of Pennsylvania who spoke about mechanosensing and tissue stiffness in FOP.</p>
<p>I had a chance to get into some really interesting conversations with some of the attendees about the way their work and my work intersected and how we might be able to collaborate in the future – particularly interesting was talking to some Postdocs from Caroline Hill’s group who are looking at similar problems to me but from an in-cell perspective rather than using purified proteins. From that some interesting questions have been thrown up that I now want to test in my in-vitro systems and that will be detailed here as and when I’ve had a chance to try them. Hopefully this can lead to validation of both my work and theirs by complementing in-vivo and in-vitro approaches.</p>
<p>I thought it would be good to share the poster I created here – so you can find it <a href="https://zenodo.org/record/2640488#.XLRgljBKiUk">over on Zenodo</a>. It is designed for a scientific audience but not necessarily an FOP specialist audience so hopefully it should be fairly easy to follow. It leads the reader through the story of explaining that Gain of Function (GOF) mutations in ALK2 are involved in diseases like FOP and DIPG. I then go on to present data that shows that these mutations disrupt the binding of the inhibitor protein FKBP12 but that this isn’t the whole story (as regular readers of this blog will know). Then I talk about what I found out about what is needed for SMAD activation by ALK2 before looking at the increased phosphorylation of SMAD1 by the mutant ALK2 proteins compared to the wild type. This covers both the western blot data that shows more SMAD1 activation but also the mass spec data I’ve shared previously that looks at more SMAD1 and more ALK2 phosphorylation. Finally I showed the phosphomapping data on ALK2 that is starting to pick apart the exact phosphorylation locations on ALK2 that are necessary for SMAD1 activation (Also data I’ve talked about on this blog before). In this case it was nice to see that all the effort Rod and I had put into phosphomapping had paid off! Other groups had said they’d tried and been unable to map ALK2 using trypsin alone but our method of using 5 different enzymes both enriched and unenriched seems to have done the trick (despite it being 10x more work!).</p>
<p>There is great value from attending a conference like this and in this case I feel that I really did make some very important connections and had some very informative discussions that were directly relevant to my research and that will hopefully bear fruit in time and guide the direction of my experiments.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/the-benefits-of-conference-attendance-bmp-signalling-in-cancer-ii/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Determining whether bespoke tracers work with ALK5 in nanoBRET target engagement assay</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/determining-whether-bespoke-tracers-work-with-alk5-in-nanobret-target-engagement-assay/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/determining-whether-bespoke-tracers-work-with-alk5-in-nanobret-target-engagement-assay/#comments</comments>
		<pubDate>Thu, 04 Apr 2019 15:23:54 +0000</pubDate>
		<dc:creator><![CDATA[Jong Fu]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Brain Tumours]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Jong Fu Wong]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2687</guid>
		<description><![CDATA[Special thanks to: David Drewry – Helped with designing the nanoBRET tracers M4K pharma OICR chemist team – Synthesised the M4K1046-linker derivatives Carrow Wells (UNC) – Conjugated the M4K1046 to nanoBRET fluorophore Background: I have always wanted to establish nanoBRET target engagement assay for ALK5. Relative to dual luciferase promoter assay and immunofluorescent staining, nanoBRET <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/determining-whether-bespoke-tracers-work-with-alk5-in-nanobret-target-engagement-assay/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p><strong>Special thanks to:</strong></p>
<p>David Drewry – Helped with designing the nanoBRET tracers</p>
<p>M4K pharma OICR chemist team – Synthesised the M4K1046-linker derivatives</p>
<p>Carrow Wells (UNC) – Conjugated the M4K1046 to nanoBRET fluorophore</p>
<p><strong>Background:</strong></p>
<p>I have always wanted to establish nanoBRET target engagement assay for ALK5. Relative to dual luciferase promoter assay and immunofluorescent staining, nanoBRET is many times faster. With a robust ALK5 nanoBRET assay, I will be able to rapidly screen for cellular off-target activity.</p>
<p>Since none of the commercial nanoBRET tracers worked with ALK5, we attempted to generate our own tracers. We chose to create these bespoke tracers based on M4K1046 because it has cellular IC50 of ~50nM for ALK5. David Drewry has helped to design linkers that will attach fluorophores (nanoBRET energy acceptor) to the solvent-facing end of M4K1046. Based on known structures, additional bulk in this region should not hinder the binding of the compounds to ALK5. Two versions of M4K1046 (with different linker length) were synthesised by the M4K pharma chemist team in OICR. They were subsequently sent to Carrow Wells for conjugation to nanoBRET fluorophore.</p>
<p><strong>Experimental design:</strong></p>
<p>I wanted to determine the binding of these tracers to ALK5-nanoluciferase fusion. Therefore, I incubated increasing concentrations of tracer with HEK293 cells expressing ALK5-nanoluciferase fusion protein. If the tracers can bind nicely to the ATP pocket of ALK5, incubation with increasing concentrations of tracer will result in increasing BRET signal (wavelength = 610 nm). The resulting magnitude of BRET and EC50 estimated from the curve are good indicators to whether these tracers can be used for nanoBRET assay with ALK5.</p>
<p>To identify any background signal, I have included a replicate of the above experiment with 10,000nM of parent M4K1046. These unlabelled compound will saturate ALK5 ATP pockets and prevent the binding of nanoBRET tracers. All signal from this set of experiment is contributed by non-specific background.</p>
<p>To be eliminate doubts of any technical or reagent issues with the nanoBRET experiment itself, I have replicated both experiments above, substituting ALK5 with ALK2.</p>
<p><strong>Results:</strong></p>
<div id="attachment_2691" style="width: 610px" class="wp-caption alignnone"><img class="size-full wp-image-2691" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Fig-1.png" alt="" width="600" height="362" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Fig-1.png 600w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Fig-1-300x181.png 300w" sizes="(max-width: 600px) 100vw, 600px" /><p class="wp-caption-text">EGFP signal in transfected cells. HEK293 were transfected efficiently. Cells can be harvested for nanoBRET assay.</p></div>
<div id="attachment_2692" style="width: 579px" class="wp-caption alignnone"><img class="size-full wp-image-2692" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Fig-2.png" alt="" width="569" height="293" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Fig-2.png 569w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Fig-2-300x154.png 300w" sizes="(max-width: 569px) 100vw, 569px" /><p class="wp-caption-text">EC50 curves of the first bespoke ALK5 tracer (UNC-CIW-001-011) with ALK2-nanoluc and ALK5-nanoluc. The chemical structures of parent M4K1046 and the bespoke tracer are shown on the side of the graphs. This tracer cannot bind to ALK5.</p></div>
<div id="attachment_2693" style="width: 580px" class="wp-caption alignnone"><img class=" wp-image-2693" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Fig-3.png" alt="" width="570" height="301" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Fig-3.png 555w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Fig-3-300x158.png 300w" sizes="(max-width: 570px) 100vw, 570px" /><p class="wp-caption-text">EC50 curves of the second bespoke ALK5 tracer (UNC-CIW-001-012) with ALK2-nanoluc and ALK5-nanoluc. The chemical structures of parent M4K1046 and the bespoke tracer are shown on the side of the graphs. This tracer cannot bind to ALK5.</p></div>
<p><strong>Conclusion:</strong></p>
<p>Both bespoke tracers cannot be used for ALK5 nanoBRET. Both of them did not bind to ALK5. The nanoBRET experiment itself was a success since both tracers worked well with ALK2 (over 20 folds assay windows). The background noise was low for both tracers. ALK5 seems to intrinsically not tolerate bulky additions on the solvent end of M4K1046. This might be additional avenue for improving ALK2 vs ALK5 selectivity. At the very least, we have gained this additional information from these efforts.</p>
<p>For additional experimental details, please refer to my <a href="https://zenodo.org/record/2628675#.XKYfNlVKiUk">Zenodo post</a>.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/determining-whether-bespoke-tracers-work-with-alk5-in-nanobret-target-engagement-assay/feed/</wfw:commentRss>
		<slash:comments>2</slash:comments>
		</item>
		<item>
		<title>Synthesis of promiscuous ALK2 inhibitors</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/synthesis-of-promiscuous-alk2-inhibitors/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/synthesis-of-promiscuous-alk2-inhibitors/#respond</comments>
		<pubDate>Tue, 19 Mar 2019 20:34:09 +0000</pubDate>
		<dc:creator><![CDATA[Alfredo Picado]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Alfredo Picado]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>
		<category><![CDATA[Understudied Kinases]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2598</guid>
		<description><![CDATA[In a collaboration between SGC-University of North Carolina-Chapel Hill and M4K Pharma, we plan to synthesize 3 different imidazopyridin pyrazines as promiscuous kinase ALK2 inhibitors, compounds 1-3, Scheme 1. The first step will be the synthesis of key intermediate 10, Scheme 2.  The proposed synthesis begins with a nucleophilic aromatic substitution on compound 4 to <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/synthesis-of-promiscuous-alk2-inhibitors/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>In a collaboration between SGC-University of North Carolina-Chapel Hill and M4K Pharma, we plan to synthesize 3 different imidazopyridin pyrazines as promiscuous kinase ALK2 inhibitors, compounds <strong>1-3</strong>, Scheme 1.</p>
<div id="attachment_2596" style="width: 994px" class="wp-caption alignnone"><img class="size-full wp-image-2596" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Screen-Shot-2019-03-19-at-4.22.27-PM.png" alt="" width="984" height="346" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Screen-Shot-2019-03-19-at-4.22.27-PM.png 984w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Screen-Shot-2019-03-19-at-4.22.27-PM-300x105.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Screen-Shot-2019-03-19-at-4.22.27-PM-768x270.png 768w" sizes="(max-width: 984px) 100vw, 984px" /><p class="wp-caption-text">Scheme 1. ALK2 inhibitors</p></div>
<p>The first step will be the synthesis of key intermediate <strong>10</strong>, Scheme 2.  The proposed synthesis begins with a nucleophilic aromatic substitution on compound <strong>4</strong> to get nitropyridine <strong>5</strong>, then reduction with hydrogen on palladium/carbon to obtain the pyridine-3,4-diamine <strong>6</strong>.</p>
<p>On the other hand, aminopyrazine <strong>7</strong> will be saponified to the aminopyrazine sodium salt <strong>8</strong>, which will be coupled with  compound <strong>6</strong> using peptide coupling reagent HATU to form bromopyrazine carboxamide <strong>9</strong>, followed by heating in presence of acetic acid to yield the cyclized intermediate 5-bromo-3-(1-ethyl-1<em>H</em>-imidazo[4,5-<em>c</em>]pyridin-2-yl)pyrazin-2-amine <strong>10</strong>.</p>
<div id="attachment_2597" style="width: 928px" class="wp-caption alignnone"><img class="size-full wp-image-2597" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Screen-Shot-2019-03-19-at-4.22.42-PM.png" alt="" width="918" height="650" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Screen-Shot-2019-03-19-at-4.22.42-PM.png 918w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Screen-Shot-2019-03-19-at-4.22.42-PM-300x212.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Screen-Shot-2019-03-19-at-4.22.42-PM-768x544.png 768w" sizes="(max-width: 918px) 100vw, 918px" /><p class="wp-caption-text">Scheme 2. Synthesis route 1</p></div>
<p>If we succeed with the proposed synthesis of imidazopyridine pyrazine <strong>10</strong>, then soon we will present next steps to get to target compounds <strong>1-3</strong></p>
<p>This collaboration is the under supervision of Dr. David Drewry.</p>
<p>Synthesis routes were adapted from international patents WO 2006/063167 A1, WO 2008/071937 and <em>ACS Med. Chem. Lett</em>. 2016, 7, 5, 514-519.</p>
<p>&nbsp;</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/synthesis-of-promiscuous-alk2-inhibitors/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Phosphorylation over time and phosphomapping experiments to investigate the influence of ALK2 mutations on activity.</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/phosphorylation-over-time-and-phosphomapping-experiments-to-investigate-the-influence-of-alk2-mutations-on-activity/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/phosphorylation-over-time-and-phosphomapping-experiments-to-investigate-the-influence-of-alk2-mutations-on-activity/#respond</comments>
		<pubDate>Tue, 12 Mar 2019 11:56:29 +0000</pubDate>
		<dc:creator><![CDATA[Ellie Williams]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Ellie Williams]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2579</guid>
		<description><![CDATA[In this post I’d like to return to looking at my work on phosphorylation. Phosphorylation is a key signalling method for activating the BMP pathway and inducing activity. Specifically activation of ALK2 by phosphorylation leads to phosphorylation of SMAD1 which leads to gene transcription that ultimately is responsible for bone formation. In this post I’m <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/phosphorylation-over-time-and-phosphomapping-experiments-to-investigate-the-influence-of-alk2-mutations-on-activity/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>In this post I’d like to return to looking at my work on phosphorylation. Phosphorylation is a key signalling method for activating the BMP pathway and inducing activity. Specifically activation of ALK2 by phosphorylation leads to phosphorylation of SMAD1 which leads to gene transcription that ultimately is responsible for bone formation.</p>
<p>In this post I’m going to talk about two experiments – the first is looking at the rate of phosphorylation (or how active) of two ALK2 mutants, L196P and R258S. The second is to look at mapping the exact sites of phosphorylation on two of the more common ALK2 mutations in FOP, R206H and Q027E.</p>
<p>&nbsp;</p>
<p><strong>Phosphorylation over time in ALK2 L196P and R258S.</strong></p>
<p>I’ve talked <a href="https://thesgc.github.io/static-openlabnotebooks/phosphorylation-of-smad1-rates-as-a-function-of-alk2-mutations-and-type-ii-association/">previously</a> about how phosphorylation is effected by the Q207E and R206H mutations in ALK2. This time I wanted to look at some additional mutants that are found in a small proportion of cases of FOP, specifically L196P, R258S, G328E and R375P. I purified the long forms of these ALK2 mutations in a standard fashion. Regrettably only L196P and R258S gave a good yield of expressed protein while the other two constructs gave me no expression of protein. I set up my standard activation assay with these ALK2 mutants (I mixed them with SMAD1 and with either ACVR2 or BMPR2, before adding ATP, DTT and Mg2+/Mn2+ in order to start the reaction), taking samples over time periodically up to 30 minutes. The samples were then analysed by mass spec to look for intact mass shifts corresponding to phosphorylation.</p>
<p>I plotted the peaks corresponding to phosphorylated states of both ALK2 and SMAD1 at the different time points and used this 3D graph to compare different mutant’s behaviour with respect to the activity of ALK2. I also compared this data to previously obtained data from the previous blog post.</p>
<p><img class="alignnone size-large wp-image-2580" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/MS_timecourse_blogpost-806x1024.jpg" alt="" width="780" height="991" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/MS_timecourse_blogpost-806x1024.jpg 806w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/MS_timecourse_blogpost-236x300.jpg 236w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/MS_timecourse_blogpost-768x975.jpg 768w" sizes="(max-width: 780px) 100vw, 780px" /></p>
<p><em>Time course plots for phosphorylation of SMAD1 (top) and ALK2 L196P or R258S (bottom) in the presence of ACVR2 (left) or BMPR2 (right). Legend shows time point in seconds.</em></p>
<p>&nbsp;</p>
<p>Overall we can see that R258S appears to be more activating than L196P with both more SMAD activation over the first five minutes (including the appearance of a potentially interesting third SMAD binding site). This is the case when using either ACVR2 or BMPR2. Also in general ACVR2 appears to lead to more SMAD activation than BMPR2 regardless of mutation.</p>
<p>When looking at ALK2 phosphorylation, we can see that L196P is quickly phosphorylated up to three times compared to R258S but that over the course of five minutes, R258S accumulates up to 6 phosphorylation’s much more rapidly than L196P. This effect is most pronounced when paired with ACVR2.</p>
<p>What this means in very broad terms is that the different ALK2 FOP mutants are differently activating and that their response to different type II receptors change depending on which mutation is present. This is similar to the results seen with other ALK2 mutants studied previously. This builds towards understanding why the ALK2 FOP mutations leads to increased signalling and thus increased bone formation and exactly what is going on.</p>
<p>&nbsp;</p>
<p><strong>Phosphomapping of ALK2 R206H and Q207E:</strong></p>
<p>Another aspect of looking at the phosphorylation of ALK2 and SMAD1 is not just how much it is phosphorylated but also where it is phosphorylated. For this I worked with Rod Chalk in our mass spec department to map the phosphorylation sites present in samples of protein incubated as previously mentioned. In this case I was looking at Q207E and R206H mutants of ALK2 paired with either ACVR2 or BMPR2, all in the presence of SMAD. This suggested that many of the phosphorylations could be traced to the following positions in the GS region of the kinase regardless of ALK2 mutation:</p>
<p><strong><u>ST</u></strong>LADLLDH<strong><u>S</u></strong>C<strong><u>TS</u></strong>G<strong><u>S</u></strong>G<strong><u>S</u></strong>GLPFLVQRTV</p>
<p>However not all these sites are seen at the same time and it is hard to determine which are essential for transmission of signal and ALK2 activation.</p>
<p>Similarly phosphorylation on SMAD1 could be traced to the following common locations at the C-terminal tail of the protein:</p>
<p>TQMGSPHNPI<strong><u>SS</u></strong>V<strong><u>S</u></strong></p>
<p>In this case it is the last three serines in the tail that are able to be phosphorylated and again, are seen with both mutations. Combinations of the three possible sites and all three at once were seen as possibilities within the phosphomapping data.</p>
<p>What this means however is a bit less clear. When considering the time course data from the previous post linked as well as the new mutant time course data and the phosphomapping it seems to suggest that you require multiple (up to 6) ALK2 phosphorylations to get maximal (3 site) SMAD1 phosphorylation and that there are at least this many sites in the GS loop and the region just prior to it.</p>
<p>In order to properly probe the necessity of each of these sites, it is going to be necessary to repeat these mass spec experiments on knock out mutants that remove the phosphorylation sites in various combinations, to really see which are essential for ALK2 activation. These are currently being cloned so hopefully I’ll be able to test them soon.</p>
<p>&nbsp;</p>
<p>You can find the technical details for all these experiments over on <a href="https://doi.org/10.5281/zenodo.2591062">Zenodo.</a></p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/phosphorylation-over-time-and-phosphomapping-experiments-to-investigate-the-influence-of-alk2-mutations-on-activity/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Checking I haven&#8217;t got my cells mixed up&#8230;</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/checking-i-havent-got-my-cells-mixed-up/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/checking-i-havent-got-my-cells-mixed-up/#respond</comments>
		<pubDate>Thu, 28 Feb 2019 18:56:05 +0000</pubDate>
		<dc:creator><![CDATA[Elizabeth Brown]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Brain Tumours]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>
		<category><![CDATA[Liz Brown]]></category>
		<category><![CDATA[Understudied Kinases]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2542</guid>
		<description><![CDATA[Did you know as many as 30% of the cells in labs could be misidentified? And what a nightmare it would be to find out years of your work are just completely wrong because you got your skin cells mixed up with your brain cells&#8230; So to avoid that catastrophe, I took some time to <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/checking-i-havent-got-my-cells-mixed-up/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Did you know as many as 30% of the cells in labs could be misidentified? And what a nightmare it would be to find out years of your work are just completely wrong because you got your skin cells mixed up with your brain cells&#8230;</p>
<p>So to avoid that catastrophe, I took some time to check the DNA sequence of the ACVR1 gene, because for this gene we know what should be mutated (or not) for each of my cell lines. This way I can make sure that I haven&#8217;t mixed them up in the past year (<em>and</em> that they haven&#8217;t picked up new mutations in this important area all by themselves).</p>
<p>All I had to do was collect the genomes of the cells, amplify specifically the ACVR1 gene, send it for overnight sequencing, and then I get a trace like this back from the facility:</p>
<div id="attachment_2544" style="width: 790px" class="wp-caption aligncenter"><a href="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/R206H-sequence-1.png"><img class="wp-image-2544 size-large" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/R206H-sequence-1-1024x396.png" alt="Sequencing trace covering the R206H mutation in ACVR1 found in FOP and DIPG" width="780" height="302" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/R206H-sequence-1-1024x396.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/R206H-sequence-1-300x116.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/R206H-sequence-1-768x297.png 768w" sizes="(max-width: 780px) 100vw, 780px" /></a><p class="wp-caption-text">Sequencing trace covering the R206H mutation in ACVR1 found in FOP and DIPG</p></div>
<p>Each DNA base has one peak, except one that has both a guanine (G) and an adenine (A) DNA base. The adenine, present on one chromosome, causes the change from an arginine (R) to a histidine (H) at the 206th position of the ACVR1 protein i.e. the R206H mutation.</p>
<p>This trace is from the HSJD-DIPG-007 cell line and is the mutation I expected. According to all my sequencing, none of my cell lines have gotten mixed up, yay!</p>
<p>You can read the full details of how I did this, <a href="https://zenodo.org/record/2579809#.XHgumMD7SUk">here on Zenodo</a>.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/checking-i-havent-got-my-cells-mixed-up/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>ZFYVE9C purification and association test with TGFBR1 and TGFBR2</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/zfyve9c-purification-and-association-test-with-tgfbr1-and-tgfbr2/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/zfyve9c-purification-and-association-test-with-tgfbr1-and-tgfbr2/#comments</comments>
		<pubDate>Fri, 22 Feb 2019 14:05:47 +0000</pubDate>
		<dc:creator><![CDATA[Ellie Williams]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Ellie Williams]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2515</guid>
		<description><![CDATA[ZFYVE9C (Otherwise known as SARA) is a scaffold protein involved in the association of various protein components as part of the TGFB signalling pathway. The equivalent in the BMP signalling pathway is a protein known as Endofin so studying SARA (which is slightly easier) might give us useful insights for both proteins. For more details <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/zfyve9c-purification-and-association-test-with-tgfbr1-and-tgfbr2/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>ZFYVE9C (Otherwise known as SARA) is a scaffold protein involved in the association of various protein components as part of the TGFB signalling pathway. The equivalent in the BMP signalling pathway is a protein known as Endofin so studying SARA (which is slightly easier) might give us useful insights for both proteins. For more details on this project please see <a href="https://thesgc.github.io/static-openlabnotebooks/a-summary-of-my-alk2-project-goals/">this blog post.</a></p>
<p>One of the unanswered questions about SARA is what does one of the major domains that it has actually do? (The domain is known as a DUF domain, or a Domain of Unknown Function!) We know the structure but that hasn’t given us any clues as to its function. One thought was that it might be involved in recruiting type I or type II receptors close to their binding partners such as one of the SMADs to allow signal propagation.</p>
<p>I purified the DUF domain of SARA and mixed it with either excess TGFBR1 or excess TGFBR2 (the two relevant receptors in the TGFB pathway). I tested the receptors in both the unphosphorylated (inactive) and phosphorylated (active) states. After incubating the samples for 30 minutes while concentrating the volume back down to &lt;5ml, I ran them down a gel filtration column to see if the two proteins bound together sufficiently to shift the peaks coming off the column which would indicate they were acting as a complex of larger mass.</p>
<p><img class="alignnone size-large wp-image-2516" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/gf_trace_zfyve9c-1024x491.png" alt="" width="780" height="374" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/gf_trace_zfyve9c-1024x491.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/gf_trace_zfyve9c-300x144.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/gf_trace_zfyve9c-768x368.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/gf_trace_zfyve9c.png 1578w" sizes="(max-width: 780px) 100vw, 780px" /></p>
<p><em>Overlay of the UV traces from all the size exclusion gel filtration runs to show the relative elution volumes of the peaks confirming no shift in the ZFYVE9C peak consistent with no complex being formed.</em></p>
<p>&nbsp;</p>
<p>Unfortunately I didn’t see any shift in the peak for SARA suggesting that either the two proteins don’t bind or that the affinity is weak enough that they fall apart under the conditions of the gel filtration column.</p>
<p>The next thing to try will be to purify the proteins again but not to cleave the tag on one of the binding partners (either the receptors or the SARA protein) and try to do a pull down experiment to test whether this shows any association or not.</p>
<p>&nbsp;</p>
<p>You can find more details over at<a href="https://doi.org/10.5281/zenodo.2575581"> Zenodo.</a></p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/zfyve9c-purification-and-association-test-with-tgfbr1-and-tgfbr2/feed/</wfw:commentRss>
		<slash:comments>2</slash:comments>
		</item>
		<item>
		<title>Three new ALK2 structures, one of them not the one I thought!</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/three-new-alk2-structures-one-of-them-not-the-one-i-thought/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/three-new-alk2-structures-one-of-them-not-the-one-i-thought/#respond</comments>
		<pubDate>Tue, 19 Feb 2019 17:46:47 +0000</pubDate>
		<dc:creator><![CDATA[Roslin Adamson]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Brain Tumours]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>
		<category><![CDATA[Roslin Adamson]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2507</guid>
		<description><![CDATA[I’ve not posted in a while, mainly because things have been really busy, but good busy, as I now have 3 new datasets/structures, of ALK2 with M4K2118, M4K2149 and, amazingly, M4K2009. As you know, this is our lead candidate, which until now has not behaved very well in crystal trials, and it hasn’t been possible <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/three-new-alk2-structures-one-of-them-not-the-one-i-thought/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>I’ve not posted in a while, mainly because things have been really busy, but good busy, as I now have 3 new datasets/structures, of ALK2 with M4K2118, M4K2149 and, amazingly, M4K2009. As you know, this is our lead candidate, which until now has not behaved very well in crystal trials, and it hasn’t been possible to get decent crystals. In my last post I told you how I had somehow managed to mix up the TNIK and ACVR1/ALK2 tubes when I was setting up the plates, and accidentally put ACVR1 into a coarse screen and TNIK into a fine screen for ACVR1. Well, after I got lovely big crystals for what I thought was ACVR1 with M4K3003, I sent them to the synchrotron, got beautiful data to 1.4 Å, phased it, and lo and behold! It was M4K2009 snugly fitted into the binding pocket (see the ligand density after molecular replacement below). Of course, I didn’t notice this was a trimethoxy containing compound (how??) before I excitedly emailed everyone to tell them I had M4K3003, and then had to retract … However, the story ends well as M4K2009 is really important.</p>
<p><img class="aligncenter  wp-image-2511" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/M4k2009-1024x576.jpg" alt="" width="904" height="509" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/M4k2009-1024x576.jpg 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/M4k2009-300x169.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/M4k2009-768x432.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/M4k2009.jpg 1280w" sizes="(max-width: 904px) 100vw, 904px" /></p>
<p>Additionally, I got data for M4K2149 (~2.6 Å), one of the amide head group containing compounds we’re interested to see, and M4K2118 (~1.4 Å), which is doing well in some of the other assays too. I’ve put the .mtz files <a href="https://zenodo.org/record/2573142#.XGw9g-j7SUk">here</a> on Zenodo if you want to play with high resolution data, and some images below showing the ligand density and the ligands binding in the pocket. The amide head group of M4K2149 appears to preserve the water network in the pocket that we’ve observed in other structures, but as the resolution is too low for waters, we can’t tell til we get higher resolution data. Which hopefully will be this week, as I’ve grown some really pretty crystals for M4K2149 that I hope will diffract better. These appeared quite by accident again, as I’d designed a fine screen around the D5 well condition of my mega composite screen, but neglected to notice when I validated the screen that I’d put 10 x less ammonium sulphate in one row than intended – should have been 0.9 M, but ended up being 0.09 M. I should probably start calling myself ‘the accidental crystallographer’ at this rate.</p>
<p><img class="aligncenter  wp-image-2509" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/M4K2118-2149-1024x576.png" alt="" width="883" height="497" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/M4K2118-2149-1024x576.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/M4K2118-2149-300x169.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/M4K2118-2149-768x432.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/M4K2118-2149.png 1280w" sizes="(max-width: 883px) 100vw, 883px" /></p>
<p>In the meantime, I’ve got some small crystals for M4K3007, another structure we really want, and some pretty ones for M4K2184, another amide head group compound. I’ve followed up the M4K3007, just waiting to see if anything appears, and will send the tiny ones to the synchrotron while waiting.</p>
<p><img class="aligncenter  wp-image-2508" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/crystal-pics-1024x576.png" alt="" width="849" height="478" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/crystal-pics-1024x576.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/crystal-pics-300x169.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/crystal-pics-768x432.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/crystal-pics.png 1280w" sizes="(max-width: 849px) 100vw, 849px" /></p>
<p>I’m also very happy to say that our new M4K2117 crystal system for XChem fragment screening has worked very nicely so far, in the sense that the crystals are large and relatively robust and can handle soaking in low salt conditions as well as compounds with DMSO for a few hours. They then still diffract to around 1.2-1.8 Å, consistently in the same space group, C 1 2 1. This is a vast improvement over the previous system with LDN-193189. We did a solvent characterisation and pre-screen a couple of weeks ago with 100 crystals, and of the 20 or so hits we got, none was in the allosteric pocket we are looking for. This could be because of the 1.4 M ammonium sulphate and 0.2 M Na/K tartrate I have in there, which might prevent access of fragments to the pocket. Thus, using the remarkable shifter that Nathan Wright at SGC designed, we’ll be transferring loads of crystals from grow plates to low salt soak plates before doing the compound soaks this week, with the same 100 compounds, in order to check if the hit rate/location changes.</p>
<p>I will mention though, for anyone who may never have noticed this, the DMSO concentration in the protein solution for crystallisation is very important! I had previously been using Batch 1 of M4K2117 for my co-crystallisation, but it ran out so I used Batch 2, which was a 50 mM stock, as opposed to the previous 25 mM stock. The plates I set up with this, the same in every other way, gave showers of rather ugly, crunchy, small crystals. So I tried a few things – diluted the stock to 25 mM in DMSO and set up plates with that, with both lower and higher concentrations of protein (8 and 14 mg/ml along with the usual 10-11 mg/ml), as well as using the stock as is, with lower and higher protein concentrations. The lower stock concentrations brought back the big, perfect, beautiful crystals from before, the higher stock, no matter the protein concentration, gave ugly chunks and showers. The final concentration of compound was the same in all, so clearly the DMSO percentage is a very important factor.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/three-new-alk2-structures-one-of-them-not-the-one-i-thought/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
	</channel>
</rss>
