<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>ACVR1/ALK2 &#8211; openlabnotebooks.org</title>
	<atom:link href="https://thesgc.github.io/static-openlabnotebooks/category/project/acvr1-alk2/feed/" rel="self" type="application/rss+xml" />
	<link>http://thesgc.github.io/static-openlabnotebooks/</link>
	<description>A growing team of groundbreaking scientists around the world are now sharing their lab notebooks online</description>
	<lastBuildDate>Thu, 31 Jan 2019 23:10:29 +0000</lastBuildDate>
	<language>en-GB</language>
	<sy:updatePeriod>hourly</sy:updatePeriod>
	<sy:updateFrequency>1</sy:updateFrequency>
	<generator>https://wordpress.org/?v=4.9.9</generator>

<image>
	<url>https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/cropped-lab-jar02-32x32.png</url>
	<title>ACVR1/ALK2 &#8211; openlabnotebooks.org</title>
	<link>http://thesgc.github.io/static-openlabnotebooks/</link>
	<width>32</width>
	<height>32</height>
</image> 
	<item>
		<title>Optimising seeding of normal and FOP cells for a later assay &#8211; prettier than expected</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/optimising-seeding-of-normal-and-fop-cells-for-a-later-assay-prettier-than-expected/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/optimising-seeding-of-normal-and-fop-cells-for-a-later-assay-prettier-than-expected/#respond</comments>
		<pubDate>Tue, 22 Jan 2019 19:22:08 +0000</pubDate>
		<dc:creator><![CDATA[Elizabeth Brown]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Brain Tumours]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>
		<category><![CDATA[Liz Brown]]></category>
		<category><![CDATA[Understudied Kinases]]></category>

		<guid isPermaLink="false">https://openlabnotebooks.org/?p=2388</guid>
		<description><![CDATA[Further down the line I want to be able to test the activation of the BMP signalling pathway downstream of the ALK2 receptor. For this I plan to use the alkaline phosphatase gene as a readout &#8211; it&#8217;s a well known gene activated downstream of BMP signalling so a lot of easy to use assays <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/optimising-seeding-of-normal-and-fop-cells-for-a-later-assay-prettier-than-expected/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Further down the line I want to be able to test the activation of the BMP signalling pathway downstream of the ALK2 receptor. For this I plan to use the alkaline phosphatase gene as a readout &#8211; it&#8217;s a well known gene activated downstream of BMP signalling so a lot of easy to use assays already exist for it.</p>
<p>Unfortunately <a href="https://openlabnotebooks.org/optimising-an-assay-to-detect-alkaline-phosphatase-activity-in-c2c12-cells/">when I tested this assay with the standard C2C12 cell line it didn&#8217;t work particularly well</a> so I&#8217;ve decided to try again with a new set of cells. This time I&#8217;m choosing connective tissue cells (fibroblasts) from FOP patients as a reporter cell line because they respond more strongly to the signals that activate the BMP pathway (called BMP ligands).</p>
<p>I&#8217;ve never worked with these cells before so, again, I needed to work out how many I&#8217;ll need to get a good result from the assay. These cells however grow as a very thin sheet which is so see-through that I initially couldn&#8217;t tell if the wells were full. To solve this I used stains called Calcein AM and Hoechst. Calcein AM can stain any living cell because they metabolise it from a see-through molecule into a fluorescent green product, whereas Hoechst can stick to DNA and turn all the cell nuclei blue, and so suddenly, all of my cells glow:</p>
<div id="attachment_2389" style="width: 1382px" class="wp-caption aligncenter"><a href="https://openlabnotebooks.org/wp-content/uploads/2019/01/Calcein-AM-example.png"><img class="wp-image-2389 size-full" src="https://openlabnotebooks.org/wp-content/uploads/2019/01/Calcein-AM-example.png" alt="Fibroblast cells stained with Calcein AM (green) and Hoechst (blue)" width="1372" height="784" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/Calcein-AM-example.png 1372w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/Calcein-AM-example-300x171.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/Calcein-AM-example-768x439.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/Calcein-AM-example-1024x585.png 1024w" sizes="(max-width: 1372px) 100vw, 1372px" /></a><p class="wp-caption-text">Fibroblast cells stained with Calcein AM (green) and Hoechst (blue)</p></div>
<p>Aren&#8217;t they pretty?</p>
<p>Once they&#8217;re this easy to see, it&#8217;s even easier to see how many cells are needed to fill a well, but not get overcrowded. You can read more about how I did that in my <a href="https://zenodo.org/record/2546911">Zenodo post</a>.</p>
<p>The only question I have left is why some cells didn&#8217;t stain with Calcein AM, even though their nucleus looks fine? Anyone know the answer?</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/optimising-seeding-of-normal-and-fop-cells-for-a-later-assay-prettier-than-expected/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>A new ALK2 structure, and proof that mistakes in the lab aren’t always a bad thing</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/a-new-alk2-structure-and-proof-that-mistakes-in-the-lab-arent-always-a-bad-thing/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/a-new-alk2-structure-and-proof-that-mistakes-in-the-lab-arent-always-a-bad-thing/#respond</comments>
		<pubDate>Fri, 04 Jan 2019 15:28:47 +0000</pubDate>
		<dc:creator><![CDATA[Roslin Adamson]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Brain Tumours]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>
		<category><![CDATA[Roslin Adamson]]></category>

		<guid isPermaLink="false">https://openlabnotebooks.org/?p=2364</guid>
		<description><![CDATA[Hello everyone and happy New Year! I really, really wanted to end the year on a high note in December before I went off on holiday, but as it turned out, I didn’t have enough time to set up 18 plates and write a blog post on the last day. So I’ll just have to <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/a-new-alk2-structure-and-proof-that-mistakes-in-the-lab-arent-always-a-bad-thing/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Hello everyone and happy New Year! I really, really wanted to end the year on a high note in December before I went off on holiday, but as it turned out, I didn’t have enough time to set up 18 plates and write a blog post on the last day. So I’ll just have to start the year on a high note.</p>
<p>I’m very happy to say I got another ACVR1/ALK2 structure in December, at 2 Å resolution, co-crystallised with M4K2158 (see images below, and the .mtz autoprocessed file is available <a href="https://zenodo.org/record/2531714#.XC94jlz7SUk">here</a> on Zenodo). The structure is not fully refined yet, but what’s interesting is the solvent side of the molecule is being stabilised by either a water contact (Chain A) or Asp293 (Chain B). This gives it really good density.</p>
<p>&nbsp;</p>
<div id="attachment_2367" style="width: 865px" class="wp-caption aligncenter"><img class="wp-image-2367 " src="https://openlabnotebooks.org/wp-content/uploads/2019/01/M4K2158_structure-1024x576.png" alt="" width="855" height="481" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/M4K2158_structure-1024x576.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/M4K2158_structure-300x169.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/M4K2158_structure-768x432.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/M4K2158_structure.png 1280w" sizes="(max-width: 855px) 100vw, 855px" /><p class="wp-caption-text">ACVR1 bound to M4K2158 at ~2 Å, showing interactions with His286 and a water at the back of the pocket</p></div>
<p>&nbsp;</p>
<p>I also seem to have some new <span style="font-size: 1.5rem;">pretty </span><span style="font-size: 1.5rem;">hits with M4K2118, M4K2149 and M4K3003 (crystal pics below). What’s interesting about the M4K3003 hits is that I’ve really struggled to get crystals with this compound (as well as M4K3007 and 3010), and it is a pretty desirable structure. These hits came about as a result of a mistake, where I somehow managed to mix up my tubes and put ACVR1 with M4K3003 into my mega composite screen without seeds, and TNIK into a fine screen I had intended for ACVR1 with seeds. I only discovered this because I mass specced the TNIK just before setting up the plates, then rushed off to do them as it was the SGC Christmas party that night and I needed to finish (best not to rush experiments). When I checked the mass spec result later, I got the exact mass of ACVR1! So I went back to the crystallography room and recovered what protein I could from the source plate for the TNIK plates, and mass specced it again. Same result – ACVR1. But when I looked at the drop images for those plates (now containing ACVR1 + M4K3003), I found a bunch of crystals! Happy serendipity!! These will go on the trip to Diamond on the 21</span><sup>st</sup><span style="font-size: 1.5rem;"> of January and then we’ll know whether the mistake really ended well or not. Fingers crossed!</span></p>
<p>&nbsp;</p>
<div id="attachment_2366" style="width: 893px" class="wp-caption aligncenter"><img class="wp-image-2366" src="https://openlabnotebooks.org/wp-content/uploads/2019/01/Crystal-hits-Dec_Jan-2018_9-1024x576.png" alt="" width="883" height="497" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/Crystal-hits-Dec_Jan-2018_9-1024x576.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/Crystal-hits-Dec_Jan-2018_9-300x169.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/Crystal-hits-Dec_Jan-2018_9-768x432.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/Crystal-hits-Dec_Jan-2018_9.png 1280w" sizes="(max-width: 883px) 100vw, 883px" /><p class="wp-caption-text">ACVR1 co-crystallised with M4K2149, 2118 and 3003 &#8211; hits to send to the synchrotron</p></div>
<p>&nbsp;</p>
<p>You may well ask, why was I setting up plates with TNIK? As it happens, some of our collaborators’ assays show M4K2009, our main compound of interest, hits both TNIK and DDR1 with pretty high affinity. We’ve been struggling to get crystals of ACVR1 with M4K2009 – for whatever reason, it doesn’t want to make crystals. So it seemed like a good idea to see if TNIK or DDR1 will co-crystallise with it. TNIK (TRAF2 and NCK Interacting Kinase) is a serine/threonine kinase that functions as an activator of the Wnt signalling pathway. DDR1 (Discoidin Domain Receptor Tyrosine Kinase 1) is a tyrosine kinase that functions as cell surface receptor for fibrillar collagen and regulates cell attachment to the extracellular matrix, remodelling of the extracellular matrix, cell migration, differentiation, survival and cell proliferation (according to its GeneCard).</p>
<p>While I was about it, I thought it would be useful to set up plates with BMPR1A (ALK3), for the same reason I set up TGFBR1 (ALK5) before – to see what off-target structures look like with some of these compounds. The purification protocols for TNIK and BMPR1A were as standard, but they’re available <a href="https://zenodo.org/record/2531714#.XC94jlz7SUk">here</a> on Zenodo.</p>
<p>Some other good news is that I’ve figured out how to reproduce my XChem crystals fairly reliably. On the last day before the holidays I set up 6 plates of ACVR1 with M4K2117, which gives really beautiful, well-diffracting crystals. I used 10, 12 and 15 mg/ml protein, and made plates with and without seeds. Well, the ones without the seeds grew fabulously, within 2 weeks, and the ones with seeds didn’t! The next step with these then is to test what happens if I soak them in different formulations of lower salt with and without some PEG3350. That&#8217;ll be for next week. Have a great weekend!</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/a-new-alk2-structure-and-proof-that-mistakes-in-the-lab-arent-always-a-bad-thing/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Determining the nanoBRET ALK2 IC50 values of 24 new ACVR1/ALK2 inhibitors</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/determining-the-nanobret-al2k-ic50-values-of-24-new-acvr1-alk2-inhibitors/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/determining-the-nanobret-al2k-ic50-values-of-24-new-acvr1-alk2-inhibitors/#respond</comments>
		<pubDate>Wed, 19 Dec 2018 06:39:05 +0000</pubDate>
		<dc:creator><![CDATA[Jong Fu]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Brain Tumours]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Jong Fu Wong]]></category>

		<guid isPermaLink="false">https://openlabnotebooks.org/?p=2314</guid>
		<description><![CDATA[In an effort to develop clinical compounds for Diffused Intrinsic Pontine Glioma (DIPG) treatment, new analogues of ACVR1/ALK2 inhibitors are continuously synthesised by Ontario Institute for Cancer Research (OICR) and Charles River Laboratories (CRL). I will provide prompt feedback of the cellular assay results to guide their design of new compounds. I determined the potency <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/determining-the-nanobret-al2k-ic50-values-of-24-new-acvr1-alk2-inhibitors/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>In an effort to develop clinical compounds for Diffused Intrinsic Pontine Glioma (DIPG) treatment, new analogues of ACVR1/ALK2 inhibitors are continuously synthesised by <a href="https://oicr.on.ca/">Ontario Institute for Cancer Research (OICR)</a> and <a href="https://www.criver.com/">Charles River Laboratories (CRL)</a>. I will provide prompt feedback of the cellular assay results to guide their design of new compounds.</p>
<div id="attachment_2315" style="width: 397px" class="wp-caption alignnone"><img class=" wp-image-2315" src="https://openlabnotebooks.org/wp-content/uploads/2018/12/Transfection-control.png" alt="" width="387" height="387" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/Transfection-control.png 575w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/Transfection-control-150x150.png 150w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/Transfection-control-300x300.png 300w" sizes="(max-width: 387px) 100vw, 387px" /><p class="wp-caption-text">HEK293 cells were nicely transfected, as indicated by GFP signal. I can proceed with the experiment.</p></div>
<div id="attachment_2316" style="width: 656px" class="wp-caption alignnone"><img class="size-full wp-image-2316" src="https://openlabnotebooks.org/wp-content/uploads/2018/12/Figure-2.png" alt="" width="646" height="720" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/Figure-2.png 646w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/Figure-2-269x300.png 269w" sizes="(max-width: 646px) 100vw, 646px" /><p class="wp-caption-text">ALK2 NanoBRET IC50 curves and chemical structures of the first dozen of new ACVR1/ALK2 inhibitors. IC50 values estimated by GraphPad Prism are shown in green.</p></div>
<div id="attachment_2317" style="width: 679px" class="wp-caption alignnone"><img class="size-full wp-image-2317" src="https://openlabnotebooks.org/wp-content/uploads/2018/12/Figure-3.png" alt="" width="669" height="720" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/Figure-3.png 669w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/Figure-3-279x300.png 279w" sizes="(max-width: 669px) 100vw, 669px" /><p class="wp-caption-text">ALK2 NanoBRET IC50 curves and chemical structures of the second dozen of new ACVR1/ALK2 inhibitors. IC50 values estimated by GraphPad Prism are shown in green.</p></div>
<p>I determined the potency of 24 new ACVR1/ALK2 inhibitors synthesised by OICR. M4K2009 in particular is quite potent (<a href="https://en.wikipedia.org/wiki/IC50">larger values indicates lower potency</a>). This compound differs from legacy compound M4K1055 by the position of its methyl group (CH3) in the middle of the molecule. Attempts to replace the tri-methoxy group (the three CH3-O groups bonded to the phenyl ring) were not successful. Since a robust nanoBRET tracer is not available for TGFBR1/ALK5, I will use dual luciferase promoter assay (orthologous assay) to determine the off-target activity of these compounds towards TGFBR1/ALK5.</p>
<div id="attachment_2322" style="width: 440px" class="wp-caption alignnone"><img class=" wp-image-2322" src="https://openlabnotebooks.org/wp-content/uploads/2018/12/Comparing-M4K2009-to-legacy-M4K1055.png" alt="" width="430" height="376" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/Comparing-M4K2009-to-legacy-M4K1055.png 326w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/Comparing-M4K2009-to-legacy-M4K1055-300x262.png 300w" sizes="(max-width: 430px) 100vw, 430px" /><p class="wp-caption-text">Comparing the chemical structures of M4K2009 to legacy compound M4K1055. ALK2 nanoBRET IC50 values of the compounds are also shown (lower values reflects increased potency).</p></div>
<p>The M4K monthly meeting where this result was presented has been recorded and posted on <a href="https://www.youtube.com/watch?v=YAsVIt_iBes&amp;t=1464s">youtube</a>.</p>
<p>For detailed experimental protocols, please refer to my <a href="https://zenodo.org/record/2410328#.XBnjM1X7Spo">Zenodo page</a>.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/determining-the-nanobret-al2k-ic50-values-of-24-new-acvr1-alk2-inhibitors/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Purification and crystallisation of an ALK2/ACVR2/FKPB12 complex.</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/purification-and-crystallisation-of-an-alk2-acvr2-fkpb12-complex/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/purification-and-crystallisation-of-an-alk2-acvr2-fkpb12-complex/#respond</comments>
		<pubDate>Fri, 14 Dec 2018 15:30:55 +0000</pubDate>
		<dc:creator><![CDATA[Ellie Williams]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Ellie Williams]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>

		<guid isPermaLink="false">https://openlabnotebooks.org/?p=2261</guid>
		<description><![CDATA[One problem with working with the isolated kinase domains of ALK2 and it’s type II binding partners, is that the affinity between the kinase domains alone is insufficient to allow complex formation. We know that the two proteins associate with each other in vivo and are co-localised by the domains on the outside of the <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/purification-and-crystallisation-of-an-alk2-acvr2-fkpb12-complex/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>One problem with working with the isolated kinase domains of ALK2 and it’s type II binding partners, is that the affinity between the kinase domains alone is insufficient to allow complex formation. We know that the two proteins associate with each other in vivo and are co-localised by the domains on the outside of the membrane. However when we just mix the two kinase domains we don’t see a strong enough association to allow a complex to be purified. We know from in-vitro tests such as the phosphorylation tests I’ve mentioned previously that there must at least be transient contact to allow phosphorylation but that isn’t sufficient to set up crystal plates of the complex.</p>
<p>One way to try and get around this is to artificially tether the two kinase domains together with a flexible loop. I have a construct which does exactly this, tethering one end of ACVR2 to ALK2. It’s known as QQ02ACVR2A. I purified this protein and then formed a complex with the binding partner FKBP12 which will bind to the GS domain of ALK2 present in the construct.</p>
<p>The first test during purification was to see if a complex could be assembled between the fusion construct and FKBP12. As they are very differences sizes (72 kDa vs. 12 kDa) purification by gel filtration gives a very clear answer. We can clearly see co-elution of the fusion and FKBP12 which is only possible if they were present as a complex. This in its self is quite exciting as it shows the fusion is still able to bind to its physiological binding partner.</p>
<p>The second stage after purification is to attempt to crystallise the complex. I concentrated the purified protein down and added compound K02288. K02288 definitely binds ALK2 and has some affinity for ACVR2 which is necessary for stabilising both kinase domains sufficiently to crystallise. I’ve tried crystallising ALK2 alone without compound before and had zero success so it was important to add a compound to this crystallisation attempt.</p>
<p>I set up four coarse screen plates and got the following in the HCS screen:</p>
<p><img class="alignnone size-medium wp-image-2262" src="https://openlabnotebooks.org/wp-content/uploads/2018/12/initial_crystals-300x286.png" alt="" width="300" height="286" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/initial_crystals-300x286.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/initial_crystals.png 598w" sizes="(max-width: 300px) 100vw, 300px" /></p>
<p>B6: 20% PEG8000, 0.2M Mg Acetate, 0.1M Cacodylate pH6.5</p>
<p>A7: 1.4M Na Acetate, 0.1M Cacodylate pH6.5</p>
<p>A8: 30% 2-propanol, 0.2M Na Citrate tribasic, 0.1M Cacodylate pH6.5</p>
<p>These are crystalline but not crystals so I designed a follow up screen based around these conditions. I then used what was left of my complex to set up two plates in this followup screen at 4C and 20C.</p>
<p>After 7 days I found the following in my plates:</p>
<p><img class="alignnone size-large wp-image-2263" src="https://openlabnotebooks.org/wp-content/uploads/2018/12/FU_xtals-1024x633.png" alt="" width="780" height="482" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/FU_xtals-1024x633.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/FU_xtals-300x185.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/FU_xtals-768x475.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/FU_xtals.png 1160w" sizes="(max-width: 780px) 100vw, 780px" /></p>
<p>I mounted five examples and sent them to the synchrotron. Alas they all came back as either salt or no diffraction. While the salt must be ignored, it’s possible that the no diffraction crystals are protein but just do not diffract. The next step will be to make seed stocks from these wells, purify more protein and set up some more plates to see if I can get any better crystals.</p>
<p>You can find more details <a href="https://doi.org/10.5281/zenodo.2276755">over at Zenodo.</a></p>
<p>I&#8217;m off for Christmas the middle of next week but in the new year I&#8217;ll be trying again at crystallising a variety of different proteins and compounds as well as working on characterising the phospho-mutants of ALK2 I posted about last time. Have a great holiday!</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/purification-and-crystallisation-of-an-alk2-acvr2-fkpb12-complex/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Screening ACVR1 inhibitors on mutant and non-mutant ACVR1 DIPG cells &#8211; effectiveness may vary</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/screening-acvr1-inhibitors-on-mutant-and-wild-type-acvr1-dipg-cells-effectiveness-may-vary/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/screening-acvr1-inhibitors-on-mutant-and-wild-type-acvr1-dipg-cells-effectiveness-may-vary/#respond</comments>
		<pubDate>Fri, 07 Dec 2018 17:08:35 +0000</pubDate>
		<dc:creator><![CDATA[Elizabeth Brown]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Brain Tumours]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Liz Brown]]></category>
		<category><![CDATA[Understudied Kinases]]></category>

		<guid isPermaLink="false">https://openlabnotebooks.org/?p=2226</guid>
		<description><![CDATA[Hi there! The last month of my life was taken over by making sure my PhD first year report was beautifully polished, but I have returned with results of a small compound screen: These are all compounds that Jong Fu has already tested with his assays so we know they effectively inhibit ACVR1, but I <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/screening-acvr1-inhibitors-on-mutant-and-wild-type-acvr1-dipg-cells-effectiveness-may-vary/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Hi there! The last month of my life was taken over by making sure my PhD first year report was beautifully polished, but I have returned with results of a small compound screen:</p>
<p>These are all compounds that <a href="https://openlabnotebooks.org/category/scientist/jong-fu-wong/">Jong Fu</a> has already tested with his assays so we know they effectively inhibit ACVR1, but I thought it would be a good idea to try them out on my DIPG cells and see how well they can kill them. I included a few different variables in this screen to test what could make a difference to the outcome:</p>
<ul>
<li>I tested cells grown in 2D and 3D because this has been noted to change the outcome of compound screens in other systems</li>
<li>I tested the compounds alone, or in addition to radiation in order to test whether the compounds were effective in addition to current DIPG treatments.</li>
</ul>
<p>I tested how effective each compound is by growing cells for 7 days in the presence of the compound. At the end I measured how alive or healthy the cells were (i.e. their viability) by using a pre-developed kit that measures the amount of ATP inside the cell by using an enzyme that can produce light when ATP is present. ATP is measured because this is sometimes called the &#8216;energy currency of the cell&#8217; so living cells have to have ATP inside them, whereas dead cells won&#8217;t and won&#8217;t be counted.</p>
<p>This screen produced a lot of data at once which is best visualised as heatmaps like these:</p>
<div id="attachment_2227" style="width: 790px" class="wp-caption aligncenter"><a href="https://openlabnotebooks.org/wp-content/uploads/2018/12/ALK2i-figure-heatmap.png"><img class="size-large wp-image-2227" src="https://openlabnotebooks.org/wp-content/uploads/2018/12/ALK2i-figure-heatmap-860x1024.png" alt="" width="780" height="929" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/ALK2i-figure-heatmap-860x1024.png 860w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/ALK2i-figure-heatmap-252x300.png 252w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/ALK2i-figure-heatmap-768x914.png 768w" sizes="(max-width: 780px) 100vw, 780px" /></a><p class="wp-caption-text"><strong>ACVR1 inhibitor M4K2009 is best at reducing mutant ACVR1 DIPG cell viability, but is not effective in ACVR1 wild-type cells</strong><br />Heatmaps showing the viability (relative to an untreated control) of a 2D culture of mutant ACVR1 DIPG cells (A) or wild-type ACVR1 DIPG cells (B) following 7 days in the presence of 1μM of the indicated ALK2 inhibitors. The colour of each box indicates how viable the cells were at the end (where blue indicates an improvement in viability and red indicates a drop in viability). Each column indicates an individual (technical repeats, n1-3) grouped into those done at the same time (biological repeats 1-3).</p></div>
<p>From that heatmap you can see that M4K2009 is the best at killing ACVR1 mutant cells (the top heatmap), but it isn&#8217;t as effective in cells where the ACVR1 gene is not mutated (&#8220;wild-type&#8221; cells). In those cells instead M4K2096 is the most effective compound (but still not to the same extent as M4K2009).</p>
<p>But this is only the data for cells grown in 2 dimensions without a radiation treatment. If you want to see the full dataset, you can read my <a href="https://zenodo.org/record/2000673#.XAqo_FX7SUk">Zenodo post</a>!</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/screening-acvr1-inhibitors-on-mutant-and-wild-type-acvr1-dipg-cells-effectiveness-may-vary/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Update on crystal hits, XChem and more purification</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/update-on-crystal-hits-xchem-and-more-purification/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/update-on-crystal-hits-xchem-and-more-purification/#comments</comments>
		<pubDate>Wed, 28 Nov 2018 16:22:44 +0000</pubDate>
		<dc:creator><![CDATA[Roslin Adamson]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Brain Tumours]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>
		<category><![CDATA[Roslin Adamson]]></category>

		<guid isPermaLink="false">https://openlabnotebooks.org/?p=2191</guid>
		<description><![CDATA[Things have been really busy in the lab the last few weeks. I’ve done a couple of purifications of ACVR1 (ALK2), which I’ll post in Zenodo (getting loads of protein now), but probably won’t write up the details of every purification every single time anymore, as it’s the same process every time for this construct. <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/update-on-crystal-hits-xchem-and-more-purification/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Things have been really busy in the lab the last few weeks. I’ve done a couple of purifications of ACVR1 (ALK2), which I’ll post in <a href="https://zenodo.org/record/1639900#.W_6-6Wj7SUk">Zenodo</a> (getting loads of protein now), but probably won’t write up the details of every purification every single time anymore, as it’s the same process every time for this construct. Please feel free to access earlier Zenodo posts for the protocol, or contact me and I’ll send you the links. I’ve also set up at least 50 crystal plates over the past month, most using the composite screen I made a while back, as well as follow up screens for hits in those conditions.</p>
<p>&nbsp;</p>
<p>I haven’t been lucky with other compounds again like I was with the big, beautiful M4K2117 crystals, but, after initial apparent abject failure to reproduce these, I now appear to have grown a plate of simply gorgeous beauties (about 40 wells with lovely crystals, which in theory should diffract around 1.5 Å or better) (see pics below, verified as protein with UV imaging). I’ve done this for the fragment screening side of the project, where we’re using x-ray crystallography to screen compound fragments in search of an allosteric binding site on ACVR1, as described in earlier posts. The crystal system I’ve been using is with LDN-193189 co-crystallised with ACVR1. The crystals are big and beautiful, but suffer from some issues: they give variable space groups, tricky for the pandda software to deal with, they have thick skins, which makes soaking somewhat variable and necessarily increases the soak time, as well as making mounting tricky. In many cases while mounting, I noticed the skin was formed almost as an envelope around the top and bottom of the drop, thus when you try to pull back the skin, it pulls the crystal with it and then wraps it up tightly in a big lump of sticky skin. I’m not that great at the micro-surgery needed to separate the mess out. Additionally, many of these crystals melt or become friable when soaked too long (more than ~2 hours).</p>
<p>&nbsp;</p>
<p>The M4K2117 ACVR1 co-crystals however are in nice clear drops, although the conditions are very salty (0.1 M citrate pH 5.0 – 6.0, 1.4M ammonium sulphate, 0.2M sodium/potassium tartrate). This could be an issue for fragment binding – acidic conditions with lots of charged species around. Thus, I intend to test what happens if I soak them in different formulations of lower salt with and without some PEG3350. Also, I need to figure out how to grow them reproducibly, as only those 40 wells from one plate out of 6 set up actually grew. It’s unclear why 2 other plates the same conditions and 3 other plates with essentially the same condition didn’t grow. Or perhaps I just need to wait a little longer (they take about a month to grow).  Many of the wells without crystals have a lot of phase separation, as shown in the pic below as well.</p>
<p>&nbsp;</p>
<div id="attachment_2194" style="width: 521px" class="wp-caption aligncenter"><img class="wp-image-2194 size-full" src="https://openlabnotebooks.org/wp-content/uploads/2018/11/Allosteric-pocket.png" alt="" width="511" height="687" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/Allosteric-pocket.png 511w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/Allosteric-pocket-223x300.png 223w" sizes="(max-width: 511px) 100vw, 511px" /><p class="wp-caption-text">The allosteric site in question, and the current LDN-193189 co-crystals we are using</p></div>
<div id="attachment_2195" style="width: 902px" class="wp-caption aligncenter"><img class="wp-image-2195 size-full" src="https://openlabnotebooks.org/wp-content/uploads/2018/11/M4K2117-1.png" alt="" width="892" height="657" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/M4K2117-1.png 892w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/M4K2117-1-300x221.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/M4K2117-1-768x566.png 768w" sizes="(max-width: 892px) 100vw, 892px" /><p class="wp-caption-text">ACVR1-M4K2117 co-crystals, with corresponding UV images showing these are indeed protein. The lower right pics show the phase separation that is in the majority of wells.</p></div>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/update-on-crystal-hits-xchem-and-more-purification/feed/</wfw:commentRss>
		<slash:comments>2</slash:comments>
		</item>
		<item>
		<title>Generating mutations in the GS domain of ALK2</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/generating-mutations-in-the-gs-domain-of-alk2/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/generating-mutations-in-the-gs-domain-of-alk2/#respond</comments>
		<pubDate>Fri, 23 Nov 2018 15:53:04 +0000</pubDate>
		<dc:creator><![CDATA[Ellie Williams]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Ellie Williams]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>

		<guid isPermaLink="false">https://openlabnotebooks.org/?p=2176</guid>
		<description><![CDATA[This week one of the things I’ve been working on is the final set of mutations in the GS domain of Alk2 to help us unpick the influence of phosphorylation on function. That, however, is a bit of a mouthful so let’s unpack it a bit. I said in my last blog post that we <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/generating-mutations-in-the-gs-domain-of-alk2/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>This week one of the things I’ve been working on is the final set of mutations in the GS domain of Alk2 to help us unpick the influence of phosphorylation on function.</p>
<p>That, however, is a bit of a mouthful so let’s unpack it a bit.</p>
<p>I said in my last blog post that we know that for Alk2 to phosphorylate SMAD1 it must contain the GS domain. This is a short glycine and serine rich loop of the protein that comes before the main kinase domain. The serine residues in this region can be phosphorylated and this seems to be key to activating Alk2. The pattern of phosphorylation of these residues may well be important and split in specific ways between auto phosphorylation and trans phosphorylation by a type II receptor.</p>
<p>In order to work out which phosphorylation’s are key for SMAD activation, we designed mutants that would change the serine residues (where phosphorylation occurs) to alanine residues, thus preventing phosphorylation on that residue. In different combinations this allows us to test which phosphorylations are key to activation.</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p><img class="alignnone size-large wp-image-2177" src="https://openlabnotebooks.org/wp-content/uploads/2018/11/Phos_mutation-1024x375.png" alt="" width="780" height="286" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/Phos_mutation-1024x375.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/Phos_mutation-300x110.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/Phos_mutation-768x281.png 768w" sizes="(max-width: 780px) 100vw, 780px" /></p>
<p><em>Mutations in the GS domain (blue) of Alk2, immediately prior to the kinase domain (green). The table shows the 11 different mutants that probe the different combinations of phosphorylation events necessary for SMAD1 activation. Row 11 in yellow is the mutant recently cloned.</em></p>
<p>My colleague George Kerr designed and cloned most of the constructs but moved onto a new role before the final one was made. I’ve now tried to make the final clone (highlighted in yellow) based on her primer designs.</p>
<p>I got colonies from my cloning and picked four to send for sequencing. Sequencing revealed that all four colonies contained the correct mutation and no additional mutations.</p>
<p>These mutant proteins can now be used to do in-cell assays, used as templates to make constructs suitable for expression and from there used in many of the in-vitro experiments I’ve already described to help work out the influence each of these phosphorylation events has.</p>
<p>For more details on the experimental methods please look at <a href="https://doi.org/10.5281/zenodo.1495120">Zenodo here.</a></p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/generating-mutations-in-the-gs-domain-of-alk2/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Some structure pictures for your viewing pleasure</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/some-structure-pictures-for-your-viewing-pleasure/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/some-structure-pictures-for-your-viewing-pleasure/#respond</comments>
		<pubDate>Thu, 08 Nov 2018 16:04:46 +0000</pubDate>
		<dc:creator><![CDATA[Roslin Adamson]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Brain Tumours]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>
		<category><![CDATA[Roslin Adamson]]></category>

		<guid isPermaLink="false">https://openlabnotebooks.org/?p=2127</guid>
		<description><![CDATA[So I reported here a couple of weeks ago that I&#8217;d obtained datasets for ACVR1 bound to M4K2117 and M4K2121, at 1.25 and 2 Å resolution, respectively, and I put up a link to the data. I&#8217;m now refining the data and got some pretty good structures so far. These are what the current stats look <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/some-structure-pictures-for-your-viewing-pleasure/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>So I reported here a couple of weeks ago that I&#8217;d obtained datasets for ACVR1 bound to M4K2117 and M4K2121, at 1.25 and 2 Å resolution, respectively, and I put up a link to the data. I&#8217;m now refining the data and got some pretty good structures so far. These are what the current stats look like, below. There&#8217;s still some way to go with the refinement, but they look pretty good so far!</p>
<p><img class="aligncenter  wp-image-2129" src="https://openlabnotebooks.org/wp-content/uploads/2018/11/Structure-stats-1024x576.png" alt="" width="873" height="491" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/Structure-stats-1024x576.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/Structure-stats-300x169.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/Structure-stats-768x432.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/Structure-stats.png 1280w" sizes="(max-width: 873px) 100vw, 873px" /></p>
<p>I&#8217;ve also made some pretty pictures of the bound ligands, some or which were used for the M4K progress meeting yesterday. As can be seen below, the water molecule co-ordinating two of the Os on the trimethoxyphenyl is there, as in previous structures, and this is coordinated by another water and Lys235. The interaction of the hinge region of both ligands with His286 is also there. The solvent side is uncoordinated in the M4K2117 structure, but there&#8217;s a water there in M4K2121, which is less mobile in that region anyway, so that is not too surprising. It&#8217;s truly fabulous how we can easily see in these structures the chemical space that can still be explored to fill up the pocket, but I&#8217;m acutely aware of how many other important criteria a drug candidate has to fulfil in order to actually do its job properly. Hopefully, with the excellent expertise we have working on this project in all the different areas needed to make an effective drug, the structural information will be of use. Now, I have some more compounds to screen &#8230;</p>
<p><img class="aligncenter  wp-image-2130" src="https://openlabnotebooks.org/wp-content/uploads/2018/11/M4K2117-1024x576.png" alt="" width="842" height="474" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/M4K2117-1024x576.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/M4K2117-300x169.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/M4K2117-768x432.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/M4K2117.png 1280w" sizes="(max-width: 842px) 100vw, 842px" /> <img class="aligncenter  wp-image-2131" src="https://openlabnotebooks.org/wp-content/uploads/2018/11/M4K2121-1024x576.png" alt="" width="830" height="466" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/M4K2121-1024x576.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/M4K2121-300x169.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/M4K2121-768x432.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/M4K2121.png 1152w" sizes="(max-width: 830px) 100vw, 830px" /></p>
<p>&nbsp;</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/some-structure-pictures-for-your-viewing-pleasure/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Phosphorylation of SMAD1 &#8211; rates as a function of Alk2 mutations and type II association.</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/phosphorylation-of-smad1-rates-as-a-function-of-alk2-mutations-and-type-ii-association/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/phosphorylation-of-smad1-rates-as-a-function-of-alk2-mutations-and-type-ii-association/#respond</comments>
		<pubDate>Wed, 07 Nov 2018 17:48:18 +0000</pubDate>
		<dc:creator><![CDATA[Ellie Williams]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Ellie Williams]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>

		<guid isPermaLink="false">https://openlabnotebooks.org/?p=2114</guid>
		<description><![CDATA[One of the questions to be asked when looking at the mutation in ALK2 that causes FOP, is how exactly does that mutation cause the disease. ALK2 we know is part of the BMP signalling pathway responsible for bone formation and the mutation results in excess bone formation – thus we can summarise that somehow <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/phosphorylation-of-smad1-rates-as-a-function-of-alk2-mutations-and-type-ii-association/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>One of the questions to be asked when looking at the mutation in ALK2 that causes FOP, is how exactly does that mutation cause the disease. ALK2 we know is part of the BMP signalling pathway responsible for bone formation and the mutation results in excess bone formation – thus we can summarise that somehow the mutation leads to increased signal. The question is how?</p>
<p>Alk2 is known to phosphorylate SMADs 1, 5 and 8 (summarised as SMAD1/5/8) in order to signal downstream in the BMP signalling pathway. The instigation of this signal comes from the binding of an external BMP ligand to the domains of Alk2 and a type II receptor on the outside of the cell. This co-locates the internal domains (which incidentally is also where the Alk2 mutation is located) and seems to drive signalling.</p>
<p>I have been looking at the activation of SMAD1 by ALK2 in the presence or absence of a Type II receptor and in the presence or absence of the FOP ALK2 mutation. The activation of SMAD1 results in its phosphorylation which can be monitored by mass spectrometry.</p>
<ul>
<li>We know from previous experiments that Alk2 alone cannot activate SMAD1 even if the mutation is present, thus the Type II receptor is needed.</li>
<li>We know that Type II activity isn’t necessary for SMAD1 activation as using a kinase dead version of BMPR2 still works.</li>
<li>We know from previous experiments that you need both the kinase and GS domains of Alk2 in order to get SMAD1 activation.</li>
<li>We know from previous experiments that BMPR2 and ACVR2 both work to allow SMAD1 activation.</li>
<li>We know that Alk2 autophosphorylates its self to some degree, focused on the GS loop.</li>
<li>We know that the type II receptor transphosphorylates Alk2 adding additional phosphorylation sites.</li>
</ul>
<p><img class="alignnone size-medium wp-image-2116" src="https://openlabnotebooks.org/wp-content/uploads/2018/11/Receptor_interactions-190x300.png" alt="" width="190" height="300" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/Receptor_interactions-190x300.png 190w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/Receptor_interactions.png 543w" sizes="(max-width: 190px) 100vw, 190px" /></p>
<p>Interaction of the type I and type II receptors in the membrane.</p>
<p>So what don’t we know?</p>
<ul>
<li>We don’t know how the mutation in Alk2 effects the rate of SMAD1 phosphorylation</li>
<li>We don’t know how transphosphorylation effects the rate of SMAD1 phosphorylation.</li>
<li>We don’t’ know how the mutation effects the rate of autophosphorylation of Alk2.</li>
</ul>
<p>This experiment can be performed using isolated purified proteins. I’ve been busy purifying Alk2 (wild type and two mutant versions – Q207E and R206H), ACVR2 and BMPR2. I’m now ready to combine these with samples of the Kinase Dead version of BMPR2 and SMAD1 that I have in the freezer into a mass spec based assay.</p>
<p>I mixed all the components together at appropriate concentrations, in this case all proteins were at 40uM, ATP was at 0.5mM, Magnesium ions were at 5mM, Manganese ions were at 2mM and DTT (a reducing agent to stop the proteins precipitating) was at 5mM. These concentrations are needed because if you drop lower, you lose the association between the type I and II receptors. In the cell they’d be brought together in the membrane by a ligand binding outside the cell which would co-locate the inner domains. Since we’re only looking at the inner domains in vitro, we need to artificially increase the concentrations to allow them to interact. 40uM was previously determined as being the lowest concentration at which this interaction can reliably be measured.</p>
<p>Once the ATP had been added, this started the phosphorylation reaction. I took a small sample at various time points (30s, 1min, 90s, 2min, 150s, 3min, 4min, 5min, 6min, 7min, 8min, 9min, 10min, 15min, 20min 25min) from my main reaction mix, and acid quenched the reaction in this sample to stop it going any further. This was done by diluting the sample in mass spectrometry running buffer containing 0.1% formic acid.</p>
<p>The samples were run overnight on the mass spectrometry machine to measure the mass of each of the proteins and the masses of any phosphorylated proteins. The peak heights for each of the phosphorylated states (I looked at up to 2 phosphorylation states for SMAD1 and 8 for Alk2) was extracted from the data and used to monitor the rate of phosphorylation of both Alk2 and SMAD1 over time.</p>
<p><img class="alignnone size-large wp-image-2118" src="https://openlabnotebooks.org/wp-content/uploads/2018/11/total_phos-1024x683.png" alt="" width="780" height="520" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/total_phos-1024x683.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/total_phos-300x200.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/total_phos-768x512.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/total_phos.png 1560w" sizes="(max-width: 780px) 100vw, 780px" /></p>
<p><em>Total phosphorylation of Q207E after 25 minutes incubation with ACVR2.</em></p>
<p>I used the following combination of proteins in my reaction mixes:</p>
<table style="width: 510px;">
<tbody>
<tr>
<td style="width: 141px;">&nbsp;</td>
<td style="width: 88px;">ACVR2 active</td>
<td style="width: 93px;">BMPR2 active</td>
<td style="width: 109px;">BMPR2 kinase dead</td>
</tr>
<tr>
<td style="width: 141px;">Alk2 Wild type</td>
<td style="width: 88px;">X</td>
<td style="width: 93px;">X</td>
<td style="width: 109px;">X</td>
</tr>
<tr>
<td style="width: 141px;">Alk2 R206H mutant</td>
<td style="width: 88px;">X</td>
<td style="width: 93px;">X</td>
<td style="width: 109px;">X</td>
</tr>
<tr>
<td style="width: 141px;">Alk2 Q207E mutant</td>
<td style="width: 88px;">X</td>
<td style="width: 93px;">X</td>
<td style="width: 109px;">X</td>
</tr>
</tbody>
</table>
<p>&nbsp;</p>
<p>This gave me the following results: In each case the important bit to look at is the initial slope of the reaction, not the absolute magnitude as that is more subject to Mass Spec variance.</p>
<p><strong>Phosphorylation on SMAD:</strong></p>
<p><img class="alignnone size-full wp-image-2117" src="https://openlabnotebooks.org/wp-content/uploads/2018/11/SMAD1_P.png" alt="" width="1475" height="598" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/SMAD1_P.png 1475w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/SMAD1_P-300x122.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/SMAD1_P-768x311.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/SMAD1_P-1024x415.png 1024w" sizes="(max-width: 1475px) 100vw, 1475px" /></p>
<p><em>Graphs of SMAD1 phosphorylation comparing the effect of different Alk2 forms (WT, Q207E and R206H mutations) and different type II receptors (ACVR2, BMPR2 active and BMPR2 kinase dead)</em></p>
<p>&nbsp;</p>
<p><strong>Phosphorylation on Alk2</strong></p>
<p><img class="alignnone size-full wp-image-2115" src="https://openlabnotebooks.org/wp-content/uploads/2018/11/Alk2_P.png" alt="" width="1479" height="618" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/Alk2_P.png 1479w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/Alk2_P-300x125.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/Alk2_P-768x321.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/Alk2_P-1024x428.png 1024w" sizes="(max-width: 1479px) 100vw, 1479px" /></p>
<p><em>Graphs of Alk2 phosphorylation comparing the effect of different Alk2 forms (WT, Q207E and R206H mutations) and different type II receptors (ACVR2, BMPR2 active and BMPR2 kinase dead)</em></p>
<p>&nbsp;</p>
<p>Interpreting this data is quite tricky because you can look at phosphorylation on either protein while at the same time comparing the effects of the Alk2 mutations or you can compare the effects of the different type II receptors. I haven’t got enough data to be able to get any quantitative data on initial rates of reaction but the trends shown in this experiment match the trends seen in the previous experiment I ran of this type some time ago (before I started writing this blog).</p>
<p>So, what can we conclude from this experiment?</p>
<ul>
<li>SMAD1 phosphorylation is slower with wild type Alk2 than with either of the mutations (levels out at 20 minutes compared to 5 minutes aprox.)</li>
<li>When looking at SMAD1 phosphorylation, BMPR2 is a less good companion than ACVR2 when wild type Alk2 is used. Q207E somewhat counteracts this effect and R206H has the most pronounced counteraction bringing the rate of phosphorylation with BMPR2 to nearly equivalent to ACVR2.</li>
<li>In general the Q207E mutation is a greater facilitator of SMAD1 phosphorylation than wild type. The R206H mutation is better than both of them.</li>
<li>The rate of ALK2 phosphorylation is not effected by the type II protein used.</li>
<li>The rate of Alk2 phosphorylation does change depending on which construct is used. WT has the lowest rate, Q207E has a faster rate while R206H has the fastest rate.</li>
<li>The kinase dead version of BMPR2 leads to minimal difference in Alk2 phosphorylation but does reduce SMAD1 phosphorylation when compared to the BMPR2.</li>
</ul>
<p>&nbsp;</p>
<p>Next step: Try and repeat it again, however for that I need more wild type protein and we still don’t have the expression system back up and running for that.</p>
<p>For more details please look at the <a href="http://doi.org/10.5281/zenodo.1479804">Zenodo entry</a> for this pos</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/phosphorylation-of-smad1-rates-as-a-function-of-alk2-mutations-and-type-ii-association/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Some good news! I have data! Not just any data &#8211; 1.25 Å data!</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/some-good-news-i-have-data-not-just-any-data-1-25-a-data/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/some-good-news-i-have-data-not-just-any-data-1-25-a-data/#comments</comments>
		<pubDate>Fri, 19 Oct 2018 15:58:24 +0000</pubDate>
		<dc:creator><![CDATA[Roslin Adamson]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Brain Tumours]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>
		<category><![CDATA[Roslin Adamson]]></category>

		<guid isPermaLink="false">https://openlabnotebooks.org/?p=2019</guid>
		<description><![CDATA[So, the week has been super busy, but all for good reasons this time!! I&#8217;m absolutely thrilled to say that amongst all the salt crystals I found in my crystal plates, I also found the crystals below, one of which diffracted to 1.25 Å. This is definitely the highest resolution I&#8217;ve ever worked with. It&#8217;s amazing! <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/some-good-news-i-have-data-not-just-any-data-1-25-a-data/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>So, the week has been super busy, but all for good reasons this time!! I&#8217;m absolutely thrilled to say that amongst all the salt crystals I found in my crystal plates, I also found the crystals below, one of which diffracted to 1.25 Å. This is definitely the highest resolution I&#8217;ve ever worked with. It&#8217;s amazing! You can see everything &#8211; as shown in the pic below, phenix found the ligand quite easily. This was with M4K2117. The other two datasets I got were at about 2 Å, also really good, and they&#8217;re for M4K2121. Clearly the use of the composite screen I set up has increased the hit rate, and allowed me to screen a lot more compounds more quickly and at both room temperature and 4 degrees.</p>
<p>The link to the autoprocessed .mtz files is <a href="https://zenodo.org/record/1475852#.W9mR4JP7SUk">here.</a></p>
<p>I&#8217;ve already set up 20 more plates with 10 more compounds &#8211; fingers crossed we get more crystals and more data! A brilliant end to a tough few weeks!</p>
<p>&nbsp;</p>
<p><img class="aligncenter size-large wp-image-2020" src="https://openlabnotebooks.org/wp-content/uploads/2018/10/ACVR1A-c096-datasets-1024x576.png" alt="" width="780" height="439" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/10/ACVR1A-c096-datasets-1024x576.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/10/ACVR1A-c096-datasets-300x169.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/10/ACVR1A-c096-datasets-768x432.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/10/ACVR1A-c096-datasets.png 1280w" sizes="(max-width: 780px) 100vw, 780px" /></p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/some-good-news-i-have-data-not-just-any-data-1-25-a-data/feed/</wfw:commentRss>
		<slash:comments>5</slash:comments>
		</item>
	</channel>
</rss>
