<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>Disease &#8211; openlabnotebooks.org</title>
	<atom:link href="https://thesgc.github.io/static-openlabnotebooks/category/disease-foundation/feed/" rel="self" type="application/rss+xml" />
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<description>A growing team of groundbreaking scientists around the world are now sharing their lab notebooks online</description>
	<lastBuildDate>Tue, 06 Aug 2019 13:00:51 +0000</lastBuildDate>
	<language>en-GB</language>
	<sy:updatePeriod>hourly</sy:updatePeriod>
	<sy:updateFrequency>1</sy:updateFrequency>
	<generator>https://wordpress.org/?v=4.9.10</generator>

<image>
	<url>https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/cropped-lab-jar02-32x32.png</url>
	<title>Disease &#8211; openlabnotebooks.org</title>
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<width>32</width>
	<height>32</height>
</image> 
	<item>
		<title>Years and Years… of Dopaminergic neurons</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/years-and-years-of-dopaminergic-neurons/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/years-and-years-of-dopaminergic-neurons/#respond</comments>
		<pubDate>Tue, 06 Aug 2019 12:34:52 +0000</pubDate>
		<dc:creator><![CDATA[Thomas Durcan]]></dc:creator>
				<category><![CDATA[ALS-Reproducible Antibody Platform (ALS-RAP)]]></category>
		<category><![CDATA[Amyotrophic Lateral Sclerosis (ALS)]]></category>
		<category><![CDATA[iPSC Disease modelling]]></category>
		<category><![CDATA[iPSCs]]></category>
		<category><![CDATA[Montreal Neurological Institute]]></category>
		<category><![CDATA[Parkinson's disease and neurodevelopmental disorders]]></category>
		<category><![CDATA[Thomas M. Durcan]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=3273</guid>
		<description><![CDATA[Well hello there, did you miss me. No, I didn’t get stuck on a roundabout for days, preventing me from writing my next blog. I was simply enjoying the Montreal sunshine and those umbrella-containing beverages. Whoever came up with the idea of an umbrella in a drink is a genius. The umbrella shades the drink, keeping <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/years-and-years-of-dopaminergic-neurons/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Well hello there, did you miss me. No, I didn’t get stuck on a <a href="https://thesgc.github.io/static-openlabnotebooks/the-never-ending-roundabout-of-ipsc-quality-control/">roundabout</a> for days, preventing me from writing my next blog. I was simply enjoying the Montreal sunshine and those umbrella-containing beverages. Whoever came up with the idea of an umbrella in a drink is a genius. The umbrella shades the drink, keeping the ice cold, so you can enjoy the cool refreshing taste of your drink for so longer. It&#8217;s simply &#8220;<em><strong>brilliant</strong></em>&#8220;. All drinks should have umbrellas in my opinion, with little dinosaur pictures or little plastic dinosaurs hanging onto the side. Maybe I will make it my next venture, keep an eye out for jurassic umbrellas.</p>
<p>So we are now at the halfway point since I went and wrote an <a href="https://theconversation.com/to-cure-brain-diseases-neuroscientists-must-collaborate-thats-why-im-giving-my-data-away-118672">article</a> saying I will give away my SOPs and do <a href="https://thesgc.github.io/static-openlabnotebooks/the-summer-notebook-of-a-principal-investigator/">10 blogs</a> over the summer months. I am now at  Blog #5 of 10, so I kept my promise and did what I said I would do. But I am not finished here, I am only getting started, with more SOPs, data and other exciting goodies coming your way in August and beyond.</p>
<p>Now besides umbrellas another reason for my delay was I got hooked on a new UK show. Can you guess what it is from the title? No, its not “<strong><em>Of dopaminergic neurons</em></strong>”, though that is a new documentary I am developing with the hope of bringing it to TIFF in 2021. I mean &#8220;<em><strong>Years and Years</strong></em>&#8220;. For those not familiar, it follows the Lyons family from Manchester and the series progresses over a 15-year span from 2019 to 2034, tracking how society and our own lives might change with each passing year. It was truly fascinating, and I binged through it in under a week, which is a rare feat for me.</p>
<p>So why do I mention this? Well by looking ahead into the future about what might come, it got me nostalgic and started me thinking about how far things have come with my own stem cell group. Leading me to reminisce for days of yore. When people see our group now, they think we have our stuff together. Which is kind of true, though we do have our own mishaps and disasters from time to time. But, it wasn’t always like that. When we started in 2015, we were an utter gongshow in so many ways, figuring things out on the fly and going day by day. There was me, Carol Chen and Andrea Schreij, with Ted Fon providing us guidance and leadership to keep going. We hadn’t a notion about what we were getting into, we knew stem cells were cool, but the first 6 months were exhausting. Between the three of us we were working 7-day weeks to keep our cells alive and happy. This didn’t even take into account all the effort that went into making neurons from these stem cells. But gradually, with time and perseverance we did it, we made our first dopaminergic neurons, and below is the fluorescence images of these first neurons. This picture is stuck on the door of our cell culture room to remind us of how hard it was (but how much fun it was) during the first year and to remind us how far we have come. Many people doubted us, but we worked hard, got better and now have a healthy respect for the stem cells and all their uses.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/DA-neurons-300x232.jpg" alt="" width="338" height="261" class=" wp-image-2612 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/DA-neurons-300x232.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/DA-neurons-768x594.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/DA-neurons-1024x792.jpg 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/DA-neurons.jpg 1323w" sizes="(max-width: 338px) 100vw, 338px" /></p>
<p><a href="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/iPSCs-to-DA-neurons_the-process.tif"><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/iPSCs-to-DA-neurons_the-process.tif" alt="" width="1" height="1" class="alignnone size-medium wp-image-3276" /></a>Which is why over my next two blogs, I will be putting out three protocols on how we generate dopaminergic neurons from iPSCs going from the iPSC to the neuron. Yes its tough to make neurons, but its doable and we are here to help  as best we can, starting with the <a href="https://zenodo.org/record/3361000#.XUjw-uhKg2w">protocols</a> we use, going from an iPSC to <span style="font-size: 1.5rem;">cortical neurons, or the </span><span style="font-size: 1.5rem;">Dopaminergic neurons you see below (Adapted from our open access <a href="https://www.frontiersin.org/articles/10.3389/fnins.2018.00047/full">Frontiers review</a>).</span></p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/iPSCs-to-DA-neurons_the-process-300x200.jpg" alt="" width="548" height="365" class=" wp-image-3278 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/iPSCs-to-DA-neurons_the-process-300x200.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/iPSCs-to-DA-neurons_the-process-768x511.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/iPSCs-to-DA-neurons_the-process-1024x681.jpg 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/iPSCs-to-DA-neurons_the-process.jpg 1117w" sizes="(max-width: 548px) 100vw, 548px" /></p>
<p>For today, I will be starting in the future, by looking at how we work with dopaminergic neuronal progenitors , directing them into dopaminergic neurons. This protocol also applies for making cortical neurons, as it follows the same workflow, albeit with different factors used in making the progenitors. How we make the progenitors is what I will explain in my next installment.</p>
<p>When we make dopaminergic neurons, we always make a large batch of progenitors, so we can use the same batch in a given study as much as possible. These are cells primed to become neurons, but they still divide and proliferate until we give them the media and factors to drive them into neurons. A lot of the details are in the SOP, so I won’t go too much into it. But once you have progenitors, you can easily generate the dopaminergic or cortical neurons needed for your experiments, depending on the type of progenitor you made at the start.</p>
<p>Suffice to say, be careful with your <span style="text-decoration: underline;"><strong>cell density</strong></span>, this is crucial in getting good neuronal cultures, too many cells or too few cells and you won’t get good neurons. So figuring this out for each cell-line is crucial. Neurons grow differently from line to line, which also needs to be accounted for.</p>
<p>Also look out for <span style="text-decoration: underline;"><strong>proliferating cells</strong></span>. Often, when you start neuronal differentiation, you have cells that won’t differentiate, and they remain proliferative. If left unchecked, they can grow out of hand and contaminate your cultures. This is why we use mitomycin C to reduce these cells, leading to more pure cultures, although you will always have other cell types, including astrocytes. Having too pure neurons is also an issue, as having astrocytes and other glial cells can be good for maturing and maintaining your neuronal cultures, so its something to consider.</p>
<p>These neurons can be grown for many weeks in 2D and are a great source of human neurons for modelling diseases of the brain in a dish. But our story doesn&#8217;t end here, as 3D neuronal organoids take things one step further. This is something I will explain before the month is over. So, enjoy our two new SOPs, one is our <a href="https://zenodo.org/record/3361000#.XUjw-uhKg2w">regular pdf</a> for generating neurons from progenitors, and the other is our <a href="https://zenodo.org/record/3361006#.XUjyqehKg2w">interactive one</a> to help with training. I hope you enjoy the trip down memory lane with me as you learn a little about our humble beginnings, remember everyone has to start somewhere. On Friday, I will describe how we generate these neuronal progenitors from iPSCs. Until next time, think of me the next time you enjoy a cool umbrella containing drink.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/years-and-years-of-dopaminergic-neurons/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Project Overview: Overexpressing HTT in neuronal cells</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/project-overview-overexpressing-htt-in-neuronal-cells/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/project-overview-overexpressing-htt-in-neuronal-cells/#respond</comments>
		<pubDate>Tue, 06 Aug 2019 04:19:23 +0000</pubDate>
		<dc:creator><![CDATA[Victoria Vu]]></dc:creator>
				<category><![CDATA[Huntingtin]]></category>
		<category><![CDATA[Huntington’s disease]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<category><![CDATA[Victoria Vu]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=3260</guid>
		<description><![CDATA[My main project resolves around the E3 ligase UBE3A (I will be writing a project overview for this later), but I am also continuing some work in the same vein as Jolene Ho&#8217;s work looking at role of HTT in neurological disorders. HTT encodes a protein called huntingtin whose function is not known but appears <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/project-overview-overexpressing-htt-in-neuronal-cells/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>My main project resolves around the E3 ligase UBE3A (I will be writing a project overview for this later), but I am also continuing some work in the same vein as <a href="https://thesgc.github.io/static-openlabnotebooks/category/past-scientist/jolene-ho/">Jolene Ho&#8217;s work</a> looking at role of HTT in neurological disorders.</p>
<p>HTT encodes a protein called huntingtin whose function is not known but appears to be crucial to neuron function.  Mutant HTT is frequently associated with Huntington&#8217;s Disease (HD), where the 5&#8242; end of the <em>HTT</em> gene is observed to have an expanded CAG repeat region &#8211; the trinucleotide is repeated more times in individuals with HD than in normal individuals. In HD, mutant HTT is observed to affect neuronal function and structure leading to progressive neurodegeneration, however the exact disease mechanism is not well understood.</p>
<p>To study the function of HTT and the effects of mutant HTT protein <em>in vivo</em>, I will try to introduce trackable copies of <em>HTT </em>into mammalian cells, specifically neuronal cells. By overexpressing the HTT protein in varying amounts and with varying number of CAG repeats, we can try to elucidate the normal function of HTT in a non-disease context and examine how mutant HTT protein affects cell function and leads to neurodegeneration.</p>
<p>As seen in <a href="https://thesgc.github.io/static-openlabnotebooks/category/past-scientist/jolene-ho/">Jolene&#8217;s previous posts</a>, it is can be difficult to express such a large protein in cells. Jolene attempted to express HTT in <a href="https://thesgc.github.io/static-openlabnotebooks/overexpression-of-expanded-htt-in-brain-cancer-cell-lines/">two brain cancer cell lines</a>, but found limited success. I am using the same <a href="https://thesgc.github.io/static-openlabnotebooks/overexpressing-htt-in-mammalian-cells/">BacMAM virus system</a> previously established at the SGC and attempting to apply it in the widely used neuroblast cell line SH-SY5Y. The protocols for the experiments described below and additional data can be found on <a href="https://zenodo.org/record/3361015#.XUj8POhKiMo">Zenodo</a>.</p>
<p>With limited examples of the use of BacMAM in SH-SY5Y in the literature, I decided to test a range of virus amounts and included a BacMAM GFP control which I could visualize under the microscope to see how effective it was in SH-SY5Y. In line with <a href="https://thesgc.github.io/static-openlabnotebooks/overexpressing-htt-in-mammalian-cells/">Jolene&#8217;s optimization experiments</a> and a paper I found using BacMAM to overexpress wildtype and mutant LRRK2 implicated in Parkinson&#8217;s Disease  (PMID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3429506/">22952710</a>), I used up to 25% v/v of p3 (passage 3) BacMAM virus to transduce SH-SY5Y and included HEK293 for comparison.</p>
<p>The LRRK2 article (PMID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3429506/">22952710</a>) assayed cells for protein expression 48 hours post-transduction, so I thought I would get decent HTT expression within the same time-frame. I tracked GFP expression for 48 hours to get an overall idea of how effective the virus is in SH-SHY5. To detect HTT expression, I followed-up with FLAG-tag immunofluorescence (IF) in the SH-SY5Y (note: HEK293 don&#8217;t adhere very well and wash off easily so IF can&#8217;t be done with them). I find that GFP is very well expressed in both the HEK293 and SH-SY5Y cell lines after 48hrs, however, I did not detect any FLAG-tagged HTT in my IF (see RFP signal in figure below). This suggest that expression levels of the FLAG-tagged HTT from my BacMAM virus is likely too low to be detected using IF.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/BacMAM_HTT_48hrs-01-300x137.png" alt="" width="654" height="299" class="aligncenter wp-image-3285" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/BacMAM_HTT_48hrs-01-300x137.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/BacMAM_HTT_48hrs-01-768x350.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/BacMAM_HTT_48hrs-01-1024x467.png 1024w" sizes="(max-width: 654px) 100vw, 654px" /></p>
<p>I decided to repeat the transduction in a larger 24-well format and harvest cells for a western to see if I could detect any HTT expression. I also changed the harvest point to 3 days post-transduction as Jolene successfully in detected overexpressed HTT in HEK293 with a longer 3-day post-transduction expression period. I observed some SH-SHY5 cells began dying 2 days post-transduction, with more cells dying and lifting off in higher v/v of BacMAM virus. After 3 days, I was only able to harvest the cells with 5% BacMAM virus for my western as most of the cells at higher v/v BacMAM virus died.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/HTT_combined-01-300x148.png" alt="" width="675" height="333" class="aligncenter wp-image-3284" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/HTT_combined-01-300x148.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/HTT_combined-01-768x379.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/HTT_combined-01-1024x506.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/HTT_combined-01.png 1338w" sizes="(max-width: 675px) 100vw, 675px" /></p>
<p>In my western, I detected FLAG-tagged HTT in the positive controls (D1 &#8211; 23 CAG repeats, D2 &#8211; 54 CAG repeats), but I did not detect any in both my SH-SY5Y grown for 3 days and 6 days post-transduction. It is possible that 5% v/v BacMAM virus is too low, but before I can move forward with higher v/v, I need to optimize conditions for the transduction in SH-SY5Y so that fewer cells die. The SH-SY5Y cells may be dying from either too much virus, the side effects of overexpressing HTT, or incompatibility with the Sf9 media the BacMAM virus is in. Moving forward, I will have to test these possibilities and optimize transduction conditions to maximize overexpression HTT while reducing cell death.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/project-overview-overexpressing-htt-in-neuronal-cells/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Purification of the HTT C-HEAT domain</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/purification-of-the-htt-c-heat-domain/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/purification-of-the-htt-c-heat-domain/#respond</comments>
		<pubDate>Fri, 02 Aug 2019 13:40:02 +0000</pubDate>
		<dc:creator><![CDATA[Claudia Alvarez]]></dc:creator>
				<category><![CDATA[Claudia Alvarez]]></category>
		<category><![CDATA[Huntingtin]]></category>
		<category><![CDATA[Huntington’s disease]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=3263</guid>
		<description><![CDATA[I am very excited to write my first post in open notebook. My post on Zenodo summarizes the results of my first ever purification of a huntingtin (HTT) domain. I was able to purify successfully the HTT C-HEAT domain encompassing residues 2088-3144. Figure 1. Anti-FLAG purification of the HTT C-HEAT domain. Eluted protein is seen <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/purification-of-the-htt-c-heat-domain/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>I am very excited to write my first post in open notebook. My post on <a href="https://zenodo.org/record/3358864#.XUQ673spA2w">Zenodo</a> summarizes the results of my first ever purification of a huntingtin (HTT) domain. I was able to purify successfully the HTT C-HEAT domain encompassing residues 2088-3144.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/C-TERM-FLAG-for-post-e1564753579324-281x300.png" alt="" class="alignnone wp-image-3269 size-medium" width="281" height="300" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/C-TERM-FLAG-for-post-e1564753579324-281x300.png 281w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/C-TERM-FLAG-for-post-e1564753579324-768x821.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/C-TERM-FLAG-for-post-e1564753579324.png 849w" sizes="(max-width: 281px) 100vw, 281px" /></p>
<p>Figure 1. Anti-FLAG purification of the HTT C-HEAT domain. Eluted protein is seen in the red box!</p>
<p>This purification was first done by <a href="http://labscribbles.com/">Dr. Rachel Harding</a> (<a href="https://zenodo.org/record/2628060#.XUQ4snspA2x">Zenodo</a>). Following her purification protocol, I obtained good yields of protein (0.45 mg of protein per L of Sp9 cells) which I immediately used to set crystal trays and to initiate biophysical characterization.</p>
<p>In future purifications, my aim is to improve purity as I noticed the sample was not very clean (low molecular weight proteins and nucleic acid). Further, I want to do a more stringent analysis to determine the most ideal buffer conditions for the sample using DSLS and DSF. I also would like to use SDS-PAGE and native PAGE to determine the progression of the limited proteolysis over time.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/purification-of-the-htt-c-heat-domain/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Project Overview</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/project-overview/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/project-overview/#respond</comments>
		<pubDate>Wed, 24 Jul 2019 15:33:27 +0000</pubDate>
		<dc:creator><![CDATA[Claudia Alvarez]]></dc:creator>
				<category><![CDATA[Claudia Alvarez]]></category>
		<category><![CDATA[Huntingtin]]></category>
		<category><![CDATA[Huntington’s disease]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=3212</guid>
		<description><![CDATA[I joined the group of Dr. Cheryl Arrowsmith 2 months ago. Currently, I am working on the Huntington’s disease project along with Dr. Rachel Harding and Jacob McAuley. During these past two months, I have been learning how to purify huntingtin (HTT) protein and overall getting up to date with the project. A brief introduction <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/project-overview/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>I joined the group of Dr. Cheryl Arrowsmith 2 months ago. Currently, I am working on the Huntington’s disease project along with <a href="https://labscribbles.com/">Dr. Rachel Harding</a> and <a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/jacob-mcauley/">Jacob McAuley</a>. During these past two months, I have been learning how to purify huntingtin (HTT) protein and overall getting up to date with the project.</p>
<p>A brief introduction to the project: HTT is a 348 KDa protein that is vital for development <sup>1</sup>. HTT has a polyglutamine repeat at its N-terminus which is normally no longer than 35 repeats (Figure 1). When this number of repeats exceeds 38 repeats, it causes Huntington’s disease (HD), a dominantly inherited neurological disorder that occurs in adulthood and sometimes in children <sup>2</sup>. Thus, like many other neurological disorders such as Parkinson’s and Alzheimer’s disease, HD is also caused by a protein with an abnormal function in the cell. There are very limited treatments available for HD patients in part because there is little information about the function and structure of HTT and mutant HTT.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/Project-overview-post_3-284x300.jpg" alt="" class="alignnone size-medium wp-image-3227" width="284" height="300" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/Project-overview-post_3-284x300.jpg 284w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/Project-overview-post_3.jpg 718w" sizes="(max-width: 284px) 100vw, 284px" /></p>
<p>Figure 1. The huntingtin protein in normal population and in Huntington&#8217;s disease patients. Figure courtesy of <a href="https://labscribbles.com/">Dr. Rachel Harding</a>.</p>
<p>In the cell, HTT is thought to be involved in several important protein-protein interactions that are critical for cellular vesicular transport, endocytosis, autophagy and the regulation of transcription <sup>3,7</sup>. However, the complete picture of the function of HTT in the cell is yet not fully determined.</p>
<p>HTT is made up of three different domains. Two HEAT <sup>8,9</sup> (<u>H</u>untingtin, <u>E</u>longation factor 3, protein phosphatase 2<u>A</u>, <u>T</u>arget of rapamycin 1) domains at the N-terminus and C-terminus which are connected by a bridge. However, HTT has several disordered regions within these domains, which makes structural studies of HTT challenging. Recently, the cryo-EM structure of the protein was obtained making a huge step forward to understand more about HTT <sup>10</sup>. However, almost a quarter of the structure of HTT is still unknown in part due to the dynamic nature of this protein.</p>
<p>The main focus of my project is to obtain the 3D structure of the individual domains of the HTT protein, namely the N-terminal HEAT domain, the bridge and C-terminal HEAT domain. With this approach, we hope to better define structural elements in the protein. Further, studies with the individual domains will also be useful to understand what the role of each of the domains is within HTT. By learning about the structure and function of the individual domains, we can learn about HTT’s function inside the cell. I will be posting updates on the research progress so stay tuned!</p>
<p style="margin: 0cm; margin-bottom: .0001pt;">1. F. Saudou and S. Humbert. <em>Neuron, </em>vol. 89, no. 5, pp. 910-926, doi: <a href="https://doi.org/10.1016/j.neuron.2016.02.003">https://doi.org/10.1016/j.neuron.2016.02.003</a>.</p>
<p style="margin: 0cm; margin-bottom: .0001pt;">2. M. E. MacDonald<em> et al.</em>, &#8221; <em>Cell, </em>vol. 72, no. 6, pp. 971-983, 1993/03/26/ 1993, doi: <a href="https://doi.org/10.1016/0092-8674(93)90585-E">https://doi.org/10.1016/0092-8674(93)90585-E</a>.</p>
<p style="margin: 0cm; margin-bottom: .0001pt;">3. F. Mochel and R. G. Haller. (2011)<em>The Journal of Clinical Investigation, </em>vol. 121, no. 2, pp. 493-499, doi: <a href="https://doi.org/10.1172/JCI45691">https://doi.org/10.1172/JCI45691</a>.</p>
<p style="margin: 0cm; margin-bottom: .0001pt;">4. P. Harjes and E. E. Wanker. <em>Trends in Biochemical Sciences, </em>vol. 28, no. 8, pp. 425-433, 2003/08/01/ 2003, doi: <a href="https://doi.org/10.1016/S0968-0004(03)00168-3">https://doi.org/10.1016/S0968-0004(03)00168-3</a>.</p>
<p style="margin: 0cm; margin-bottom: .0001pt;">5. S.-H. Li and X.-J. Li, <em>The Neuroscientist, </em>vol. 10, no. 5, pp. 467-475, 2004/10/01 2004, doi: <a href="https://doi.org/10.1177/1073858404266777">https://doi.org/10.1177/1073858404266777</a>.</p>
<p style="margin: 0cm; margin-bottom: .0001pt;">6. J. P. Caviston and E. L. F. Holzbaur. <em>Trends in Cell Biology, </em>vol. 19, no. 4, pp. 147-155, 2009/04/01/ 2009, doi: <a href="https://doi.org/10.1016/j.tcb.2009.01.005">https://doi.org/10.1016/j.tcb.2009.01.005</a>.</p>
<p style="margin: 0cm; margin-bottom: .0001pt;">7. <a href="https://oxfordmedicine.com/view/10.1093/med/9780199929146.001.0001/med-9780199929146-chapter-11">Z. Chiara and C. Elena. Oxford, UK: Oxford University Press</a>.</p>
<p style="margin: 0cm; margin-bottom: .0001pt;">8. M. A. Andrade and P. Bork, <em>Nature Genetics, </em>vol. 11, no. 2, pp. 115-116, 1995/10/01 1995, <a href="https://doi.org/10.1038/ng1095-115">doi: 10.1038/ng1095-115</a>.</p>
<p style="margin: 0cm; margin-bottom: .0001pt;">9. J. Perry and N. Kleckner,  <em>Cell, </em>vol. 112, no. 2, pp. 151-155, 2003, doi: <a href="https://doi.org/10.1016/S0092-8674(03)00033-3">10.1016/S0092-8674(03)00033-3</a>.</p>
<p style="margin: 0cm; margin-bottom: .0001pt;">10. Q. Guo<em> et al.</em>, <em>Nature, </em>vol. 555, no. 7694, pp. 117-+, Mar 2018, <a href="https://doi.org/10.1038/nature25502">doi: 10.1038/nature25502</a>.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/project-overview/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>The never-ending roundabout of iPSC quality control</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/the-never-ending-roundabout-of-ipsc-quality-control/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/the-never-ending-roundabout-of-ipsc-quality-control/#respond</comments>
		<pubDate>Wed, 24 Jul 2019 11:32:18 +0000</pubDate>
		<dc:creator><![CDATA[Thomas Durcan]]></dc:creator>
				<category><![CDATA[ALS-Reproducible Antibody Platform (ALS-RAP)]]></category>
		<category><![CDATA[Amyotrophic Lateral Sclerosis (ALS)]]></category>
		<category><![CDATA[iPSC Disease modelling]]></category>
		<category><![CDATA[iPSCs]]></category>
		<category><![CDATA[Montreal Neurological Institute]]></category>
		<category><![CDATA[Parkinson's disease and neurodevelopmental disorders]]></category>
		<category><![CDATA[Thomas M. Durcan]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=3207</guid>
		<description><![CDATA[As an Irishman, I am well versed in roundabouts, we love them. Entering the roundabout, going around it and then hoping to get off it when its your exit. If you don’t exit, you just keep going round and around and around, well you get the drift. It reminds me of that great scene in <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/the-never-ending-roundabout-of-ipsc-quality-control/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>As an Irishman, I am well versed in roundabouts, we love them. Entering the roundabout, going around it and then hoping to get off it when its your exit. If you don’t exit, you just keep going round and around and around, well you get the drift. It reminds me of that great scene in National Lampoon&#8217;s European vacation where the Griswalds enter an English roundabout and just keep going around in circles for hours. If you don’t know what I am talking about, then check it out, I can <a href="https://www.youtube.com/watch?v=iAgX6qlJEMc">wait</a>.</p>
<p>Ok, welcome back, you enjoy the video, of course you did, how could you not. Now, I know you must be wondering, Tom what are you on about, just get to the point. Did you hit your head on a Montreal construction cone, not once have you mentioned dinosaurs. Well I am taking a break from dinosaurs today. And the reason for my earlier meander about roundabouts, is because I am going to talk a little about the unending circle of iPSC quality control (QC), which in my mind is just like any major roundabout in Dublin or London. Always moving in a circular fashion as we check and recheck our cells to ensure we are working with top-notch iPSCs. And the reason for this, your stem cell culture is the core of all your experiments, have a bad cell or a bad batch, and you will be having one bad week, month or even year. Trust me, I know all too well this feeling which is why our group, and in particular Carol (I talked about her in my last <a href="https://thesgc.github.io/static-openlabnotebooks/phew-its-hot-today-lets-chill-out-and-freeze-down-some-ipscs/">blog</a>) and her fellow iPSC specialist Narges Abdian have gone to great lengths to implement our roundabout of iPSC quality control, which I encourage you all to adopt or to share with me ideas on what you do in your group or ideas we could take on board.</p>
<p>I will only loosely explain it here, but I do encourage you to check out our <a href="https://zenodo.org/record/3348428#.XTfc_-hKg2w">poster</a> on this. Carol presented this at the recent International Society for Stem Cell Research (ISSCR) meeting in LA at the end of June. For those who missed her poster, we are making it available for all to see, to read and to start a dialog over the importance of quality control.</p>
<p>For us, we take several steps to ensure that when we make a new line or receive a line from elsewhere that it checks all the boxes of a “<em>bona fide”</em> good line to be used in experiments. These steps include:</p>
<ol>
<li><strong><u>Check your iPSC is pluripotent:</u></strong> We image cells for common pluripotency markers and also check for these markers by qPCR. You can also look at the morphology of your cells, and if they don&#8217;t look like an iPSC, then you have big problems. But if you pass this first step, now it is on to <strong>Step 2</strong>.</li>
<li><strong><u>Does your cell grow well</u></strong>: For this we do a combination of live cell imaging in two different stem cell medias (mTESR and Essential 8) and check for cell growth and cell viability through a combination of live cell imaging and crystal violet staining. Some lines grow well on both medias, but there are times when a line might prefer one line versus another, so that’s important to keep in mind.</li>
<li><strong><u>Is the genome of the cell stable</u></strong>: For this you can look at total chromosome number, along with genetic duplications that often arise. These can often confer selective growth advantages for your cell-line so its important to check this when working with any cell-line. You can also do other methods like whole-genome sequencing to check the entire genetic sequence of your cell-line, and this is currently in progress for many of our control iPSCs and the lines they were derived from, so this data we will soon have for a number of our lines being made available.</li>
<li><strong><u>Can my stem cell form other cells:</u></strong> Obviously when you are working with a stem cell, you might want a specific cell type. So, if your cell is good, you can always go ahead and try your SOP for generating the cell-line you want. But another way to test the differentiation potential is through a <a href="https://www.stemcell.com/stemdiff-trilineage-differentiation-kit.html">trilineage test</a> in which you promote the formation of endoderm, mesoderm and ectoderm.</li>
</ol>
<p>There you have it, our roundabout of iPSC QC; our little circle of life. We will be adding more to this process over the next month, as we are in the preparation phase of a manuscript on iPSC QC. More data is being added so we will return to this roundabout in due time.</p>
<p>You will also notice from the poster we have made and profiled 10 healthy control lines relative to cells from other sources. Yes, you can now access these iPSCs through the <a href="https://www.mcgill.ca/c-bigneuro/">MNI C-BIG</a> and <a href="cbig.mni@mcgill.ca">requests</a> for the lines can be made. Other lines are entering their freezers in coming weeks and months from our Parkinson&#8217;s disease and ALS cohorts, and I will keep you all updated as they become available for access under the <a href="https://www.mcgill.ca/neuro/open-science-0">open science</a> principles of the MNI.</p>
<p><span style="font-size: 1.5rem;">Next week, I start to outline how we make and work with neurons from patient-derived iPSCs. Got to run, </span><span style="font-size: 1.5rem;">this is my exit, or I will be on this roundabout forever. </span></p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/the-never-ending-roundabout-of-ipsc-quality-control/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Phew, its hot today. Let’s chill out and freeze down some iPSCs</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/phew-its-hot-today-lets-chill-out-and-freeze-down-some-ipscs/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/phew-its-hot-today-lets-chill-out-and-freeze-down-some-ipscs/#respond</comments>
		<pubDate>Thu, 18 Jul 2019 18:18:10 +0000</pubDate>
		<dc:creator><![CDATA[Thomas Durcan]]></dc:creator>
				<category><![CDATA[ALS-Reproducible Antibody Platform (ALS-RAP)]]></category>
		<category><![CDATA[Amyotrophic Lateral Sclerosis (ALS)]]></category>
		<category><![CDATA[iPSC Disease modelling]]></category>
		<category><![CDATA[iPSCs]]></category>
		<category><![CDATA[Montreal Neurological Institute]]></category>
		<category><![CDATA[Parkinson's disease and neurodevelopmental disorders]]></category>
		<category><![CDATA[Thomas M. Durcan]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=3187</guid>
		<description><![CDATA[In the immortal words of Ron Burgundy “Its so hot… milk was a bad choice”. Man, it is a scorcher today, with a humidex of really humid, now I know how cells feel like living in an incubator. But when its hot, what better way to cool down then with an ice cold beverage (I <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/phew-its-hot-today-lets-chill-out-and-freeze-down-some-ipscs/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>In the immortal words of Ron Burgundy “<em><strong>Its so hot… milk was a bad choice</strong></em>”. Man, it is a scorcher today, with a humidex of really humid, now I know how cells feel like living in an incubator. But when its hot, what better way to cool down then with an ice cold beverage (I mean freshly frozen tube of stem cells). As promised, the continuation of <span style="font-size: 1.5rem;">Tuesday&#8217;s </span><span style="font-size: 1.5rem;"><a href="https://thesgc.github.io/static-openlabnotebooks/thawing-your-tube-of-ipscs/">blog</a> in which we learn to maintain iPSCs, to split iPSCs, and yes, as it says it in the title, to freeze down iPSCs. Before I get started, I have been inundated with requests from people asking how the dinosaurs are doing, I don’t know if they are mocking me, or if they are serious. But thank you for asking, they are doing well and no they don&#8217;t have their own instagram account, here is my first and last picture of my collection as they like their privacy. As you </span><span style="font-size: 1.5rem;">can </span><span style="font-size: 1.5rem;">see, blue has a special place in my heart looking after my stress ball brain, he sits in on all my meetings giving constructive advice as we all work to take a bite out of the modern scientific problems of our day (haha, sorry that was an awful pun, please forgive me).</span></p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/dino-image-1_compressed-300x225.jpg" alt="" width="330" height="247" class="wp-image-3188 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/dino-image-1_compressed-300x225.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/dino-image-1_compressed-768x576.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/dino-image-1_compressed-1024x768.jpg 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/dino-image-1_compressed-350x263.jpg 350w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/dino-image-1_compressed-780x586.jpg 780w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/dino-image-1_compressed-250x188.jpg 250w" sizes="(max-width: 330px) 100vw, 330px" /></p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/dino-image-2_compressed-300x225.jpg" alt="" width="330" height="247" class=" wp-image-3189 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/dino-image-2_compressed-300x225.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/dino-image-2_compressed-768x576.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/dino-image-2_compressed-1024x768.jpg 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/dino-image-2_compressed-350x263.jpg 350w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/dino-image-2_compressed-780x586.jpg 780w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/dino-image-2_compressed-250x188.jpg 250w" sizes="(max-width: 330px) 100vw, 330px" /></p>
<p>Ok, enough of the fun and games, lets get down to serious business. Your cells are attached and growing, so every day you must now keep an eye on them to make sure they are happy. Also, every day you need to change the media. How all this works is in the <a href="https://zenodo.org/record/3337402#.XS-1vehKg2x">SOP</a> we published on Tuesday. There are some new approaches in which you can skips days but the preference for our group is daily media changes. As your cells grow, they will form tightly packed colonies that grow and expand in size over about 6-7 days until they reach confluency. But the growth rate can vary from line to line, and depends on the media you use, so be mindful of this when doing your cultures. Also look out for differentiation, or when the cells become bad. This is like when they lose their superpowers and are no longer a superhero but just a regular person, or in stem cell terms, they lose their pluripotency. Images of what this looks like are below, monitor daily for the bad cells, when you see them try to remove them, it helps in keeping the cells all nice and pluripotent.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/NCRM1_diff_mTeSR1_10x-300x143.jpg" alt="" width="611" height="291" class="aligncenter wp-image-3190" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/NCRM1_diff_mTeSR1_10x-300x143.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/NCRM1_diff_mTeSR1_10x-768x366.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/NCRM1_diff_mTeSR1_10x-1024x488.jpg 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/NCRM1_diff_mTeSR1_10x.jpg 1260w" sizes="(max-width: 611px) 100vw, 611px" /></p>
<p>Once the cells have reached critical mass and in our <a href="https://zenodo.org/record/3337402#.XS-1vehKg2x">SOP</a>, we have recommendations on when this is, then you are ready to passage. This means removing the cells from the dish they are on, to put a small amount of these cells onto a new dish to grow again. To do so, we use gentle cell dissociation media to lift the cells from the plates. These cells are pelleted, resuspended in media, and then carefully, pipetted to break the aggregates down to add onto a new plate. This plate is coated like we describe in Tuesday’s <a href="https://thesgc.github.io/static-openlabnotebooks/thawing-your-tube-of-ipscs/">blog</a>. If you don’t have a coated dish, then gulp. You better pray someone else does, or your best bet is to simply freeze the cells down.</p>
<p>Now for freezing cells, its pretty much the same as passaging, with one small exception, this is the end of the cells current journey as they are heading to the liquid nitrogen tank in the big storage room down the hall. Once you have your cell pellet, resuspend in freeze down media, and calculate the cell viability and total cell number. Based on your numbers, pipette your cell suspension into multiple cryogenic vials, and then voila, ready for the freezer. This is a stepwise freezing process, so start at -20 degrees, then -80 after a few hours, and then the next day to their final destination, the liquid nitrogen tank where it stays until you need it again, or until someone else needs it.</p>
<p>So there we have it, the highest-level overview of doing iPSC culture, but all the details are <a href="https://zenodo.org/record/3337402#.XS-1vehKg2x">here</a>.  Everything we do is outlined in the <a href="https://thesgc.github.io/static-openlabnotebooks/thawing-your-tube-of-ipscs/">blog</a> and <a href="https://zenodo.org/record/3337402#.XS_oFuhKg2x">here</a>, I can’t take the credit, actually I get zero credit, this is all my team, I just write about it. The real credit in particular goes to the person who trained me on iPSCs, and who is my real boss, <a href="xiuqing.chen@mcgill.ca">Dr. Carol Chen</a> pictured below. Fun fact about Carol, she started a year before me in the lab, and I have driven her crazy every day for 12 years, but she is a fountain of knowledge and puts my lab skills to shame, while keeping a steady flow of chocolate coming my way. She trains everyone in our group for iPSCs, and when people need to be trained from other groups, she is the go to person for helping people to learn and is one of many great team members I get to come and work with every day. You will get to meet many of them in the coming weeks and months.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/Carol-action-shot_compressed-300x200.jpg" alt="" width="300" height="200" class="size-medium wp-image-3192 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/Carol-action-shot_compressed-300x200.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/Carol-action-shot_compressed-768x513.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/Carol-action-shot_compressed-1024x683.jpg 1024w" sizes="(max-width: 300px) 100vw, 300px" /></p>
<p>And its not just our regular <a href="https://zenodo.org/record/3337402#.XS_o3ehKg2x">SOP</a> that we use for training. We are trying something a little different and more immersive, so this is brand new and we hope you like it. If you have comments or feedback we would love to hear from you and this goes for all SOPs, if you think, oh gosh Tom, your such an idiot, then please tell me, we want to do the best work we can and if people have advice then please lets talk, if you have editing errors you spot, then please go ahead and provide me a <a href="thomas.durcan@mcgill.ca">list</a>. As you can tell, I wasn&#8217;t very good with English grammar at school.</p>
<p>Now for our SOP pdf, we took it, tore it apart, put it back together again and created a hands-on training module, that is now on tablets in our culture room. This means when people learn iPSCs, no need for paper. A real-time, <a href="https://zenodo.org/record/3340189#.XS_u3uhKg2w">clickable SOP</a> on a tablet for people to work off, paired with the interactive <a href="https://zenodo.org/record/3270914#.XS-5cOhKg2x">reagent list</a>. All credit for this goes to <a href="xiuqing.chen@mcgill.ca">Carol Chen</a>, <a href="cecilia.rocha@mcgill.ca">Cecilia Rocha</a> and <a href="trisha.rao@mcgill.ca">Trisha Rao</a> who took this on board and led this, and I hope you agree with me, its great and we are hoping to test it over coming months within our group and adapt it for a number of our other protocols. And hopefully it will be a great <a href="https://zenodo.org/record/3340189#.XS_u3uhKg2w">learning tool</a> for anyone new who wants to work with stem cells no matter where you are.</p>
<p>Ok, so this is our first part done, three blogs down, seven to go. You sick of me yet, course you are. My dedicated fans still stick with me (hi mom)  Next week, we will discuss our pipeline for monitoring iPSC quality. For this, I will be making available Carol’s poster from the ISSCR meeting in LA a few weeks ago, which I know you will all love. So enjoy the weekend and after all this talk of cold drinks, I better go get myself a cold beverage and cool off. Over the next week will try something new which is hats. As I approach 40, I am trying new things, so hats seemed like a good thing to try, I might have a picture next week of me in a hat depending on how I think i look, and we can decide if its good or bad for Tom to wear a hat with my sizeable noggin. Until next week, you stay classy San Diego.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/phew-its-hot-today-lets-chill-out-and-freeze-down-some-ipscs/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Investigation of HAO1 co-factor loading effects on crystal form and protein activity</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/investigation-of-hao1-co-factor-loading-effects-on-crystal-form-and-protein-activity/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/investigation-of-hao1-co-factor-loading-effects-on-crystal-form-and-protein-activity/#respond</comments>
		<pubDate>Wed, 17 Jul 2019 12:01:13 +0000</pubDate>
		<dc:creator><![CDATA[Sabrina Mackinnon]]></dc:creator>
				<category><![CDATA[HAO1]]></category>
		<category><![CDATA[Primary Hyperoxaluria]]></category>
		<category><![CDATA[Sabrina Mackinnon]]></category>
		<category><![CDATA[SGC Univ. of Oxford]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=3168</guid>
		<description><![CDATA[In my last post, I told you about my initial results soaking follow-up compounds into preformed HAO1 crystals. In order to generate more crystals (and so more structures), I purified some new protein (I’ll call it ‘batch #2’ for simplicity). However, this second batch, when crystallised, did not take up any of the soaked in <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/investigation-of-hao1-co-factor-loading-effects-on-crystal-form-and-protein-activity/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>In my last post, I told you about my initial results soaking follow-up compounds into preformed HAO1 crystals. In order to generate more crystals (and so more structures), I purified some new protein (I’ll call it ‘batch #2’ for simplicity). However, this second batch, when crystallised, did not take up any of the soaked in compounds and so I set out to find out why. The first indication was in the structure itself which showed that there was much less density for FMN in the second batch (Figure 1D) than in the first (Figure 1C) with the map showing negative (red) density when I placed FMN into its accustomed place.</p>
<p>I confirmed that the second batch was less loaded with FMN by measuring the FMN concentration, relative to protein, in solution by absorbance. As you can see in  Figure 1A, the protein peak (at 280 nm) is very similar for both batches but the dual peaks (376 nm and 450 nm) expected for FMN are much smaller in batch #2 relative to batch #1, indicating much less FMN per unit protein. Further investigation also revealed that: a) batch #2 was much less active than batch #1 (Figure 1B, blue/green); b) adding extra FMN did not improve protein activity (Figure 1B, red/purple); and c) soaking of batch #2 crystals with FMN did not improve FMN occupancy in the structures, further supporting that co-factor binding could not be restored in purified protein.</p>
<p style="text-align: center;"><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/post10-fig1-1024x510.png" alt="" width="780" height="388" class="aligncenter size-large wp-image-3169" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/post10-fig1-1024x510.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/post10-fig1-300x150.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/post10-fig1-768x383.png 768w" sizes="(max-width: 780px) 100vw, 780px" /></p>
<p style="text-align: center;"><b><i>Figure 1: </i></b><b><i>Comparison of FMN content of HAO1 batches. </i></b><i>A. Measurement of FMN by absorbance. Protein peak at 280 nm, FMN peaks at 376 nm and 450 nm. B. Measurement of HAO1 assay with and without equimolar (1:1) added FMN. C. Full occupancy of FMN in batch #1 protein crystal. D. Poor occupancy of FMN in batch #2 protein crystal</i></p>
<p>To explain how a lack of FMN could explain a lack of binding of soaked compounds, I theorised that there must be some changes in flexibility of the gating loop so that in the absence of FMN the loop does not move to allow ligand binding. This theory was initially based on the fact that, unlike previous apo (ligand free) structures of HAO1, the entire gating loop of the batch #2 crystal structures had sufficient density to be modelled (Figure 2B). In previous structures, a fully ordered gating loop was only observed when the loop was in effect ‘locked-down’ by the binding of substrate to batch #1 protein (e.g. PDB code 6GMB; Figure 2C), whereas batch #1 protein crystals without soaked substrate had flexible loops with some residues not able to be built (Figure 2A).</p>
<p style="text-align: center;"><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/post10-fig2-1024x484.png" alt="" width="780" height="369" class="aligncenter size-large wp-image-3170" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/post10-fig2-1024x484.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/post10-fig2-300x142.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/post10-fig2-768x363.png 768w" sizes="(max-width: 780px) 100vw, 780px" /></p>
<p style="text-align: center;"><b><i>Figure </i></b><b><i>2: Comparison of loop flexibility of HAO1 structures. </i></b><i>A. Batch #</i><i>1, </i><i>used for fragment screening. In the absence of substrate, the loop shows high mobility. B. Batch #</i><i>2, unsuccessful in soaking compounds. </i><i>Lower mobility loop. C. Batch #1 crystallised with substrate to ‘lock down’ loop. Low mobility loop.</i></p>
<p>This theory is supported by comparing the <em>B-factors</em> of the loop regions in the different structures. Each atom or group of atoms within a crystal structure is assigned a B-factor which represents the uncertainty in their position dependent on three factors: <em>static</em> disorder, caused by different protein molecules within the same crystal having different conformations; <em>dynamic</em> disorder, representing the movement of the atoms themselves; and <em>coordinate error</em>, attributable to atom(s) that are placed wrongly in the model. A combination of the former two (alongside a minimisation of the latter) gives a good indication of the relative mobility of different protein regions. In this case, we can see from figure 2 that the protein crystals I have used to successfully soak my compounds had highly mobile/disordered loops (Figure 2A, batch #1 without substrate) whereas later structures, from batch #2 (Figure 2B), have much lower b-factors in the loop region. As expected, the least mobile gating loop (lowest b-factors) is found in the structure of HAO1 with substrate (batch #1 with substrate, Figure 2C).</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/investigation-of-hao1-co-factor-loading-effects-on-crystal-form-and-protein-activity/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Thawing your tube of iPSCs</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/thawing-your-tube-of-ipscs/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/thawing-your-tube-of-ipscs/#comments</comments>
		<pubDate>Tue, 16 Jul 2019 04:01:32 +0000</pubDate>
		<dc:creator><![CDATA[Thomas Durcan]]></dc:creator>
				<category><![CDATA[ALS-Reproducible Antibody Platform (ALS-RAP)]]></category>
		<category><![CDATA[Amyotrophic Lateral Sclerosis (ALS)]]></category>
		<category><![CDATA[iPSC Disease modelling]]></category>
		<category><![CDATA[iPSCs]]></category>
		<category><![CDATA[Montreal Neurological Institute]]></category>
		<category><![CDATA[Parkinson's disease and neurodevelopmental disorders]]></category>
		<category><![CDATA[Thomas M. Durcan]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=3150</guid>
		<description><![CDATA[Good morning and how is everyone doing, enjoying that first cup of tea or coffee of the morning I hope. Myself, I am a tea person, always have been, my Irish nature, we drink it by the liter. Can’t beat a good of tea in the morning to revive oneself. And if it’s the afternoon <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/thawing-your-tube-of-ipscs/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Good morning and how is everyone doing, enjoying that first cup of tea or coffee of the morning I hope. Myself, I am a tea person, always have been, my Irish nature, we drink it by the liter. Can’t beat a good of tea in the morning to revive oneself. And if it’s the afternoon or evening, well I hope you are still enjoying a caffeinated, or decaffeinated beverage, your choice. Man, what a week. From the blogs, a kind individual/individuals anonymously sent me a set of toy dinosaurs to play with, so have been having immense fun with my new set. I will add a photo in my next blog to show you my collection, it really is impressive, I just need the right lighting. So, I survived another Monday, always a slog. It always feels like I run at 20% Monday and about 100% Friday, go figure. But survive I did, and now we are ready for the next installment of my fun with Tom blog, looking at how to thaw iPSCs.</p>
<p>For those who didn’t read the last <a href="https://thesgc.github.io/static-openlabnotebooks/i-want-to-work-with-ipscs-what-do-i-need-to-get-started/">installment</a>, our list of <a href="https://zenodo.org/record/3270914#.XS1FC-hKg2x">reagents</a> is online, free to download for anyone interested. Today, we have now added our iPSC passage <a href="https://zenodo.org/record/3337402#.XTCduOhKiUl">SOP</a>, and for explaining what we do, I will break it into two parts. The first part today I will discuss how to thaw your iPSCs, and then on Thursday (Yes, Thursday, you are getting two installments of me this week, contain yourself, when down in the cottage this will be a must read) I will go into iPSC passage, freeze down and have a special surprise for you all at the end, and no its not a pony.</p>
<p>So, fun and games are now over, lets get down to business. Here are your five easy steps to thawing and growing your iPSCs, these points will be short, and all the details can be found in our online <a href="https://zenodo.org/record/3337402#.XS1Gm-hKg2w">SOP</a>.</p>
<p><strong>Coating of dishes: </strong>For growing our iPSCs, we use a matrix to grow the cells, and the preferred choice for our group is matrigel. There are other alternatives out there, and some groups even prefer to use feeder layers, in which your seed a layer of fibroblasts, followed by seeding your iPSCs. All have their plus and minus points. Matrigel is far from perfect, but the critical thing to remember, is keep it on ice at all times. Thaw on ice. Keep the media you suspend it in on ice, even the tips you use, keep in the freezer. You want to prevent it from gelating until you have added it to your dishes. Once added onto your dish, which for thawing cells is a 60mm dish, swirl around to coat the plate evenly and place in your cell culture incubator for at least 1h. People vary times for coating, etc; but our standard is always a minimum of 1h in the incubator.</p>
<p><strong>Warming of media: </strong>While your plate is coating, you need to warm your media. But iPSC media is special, unlike regular old cell culture media where you warm it in a 37 degree celsius waterbath, the media that is used in iPSC cell culture, which for us is either mTESr or Essential 8, is warmed at room temperature. More details on working with mTESR and Essential 8 in our <a href="https://zenodo.org/record/3337402#.XS1Gm-hKg2w">SOP</a>.</p>
<p><strong>Warming of cells: </strong>The time has now arrived. Thawing the cells. The nerves are high, so breathe slowly, you got this. Once you remove your tube of cells from liquid Nitrogen, place it in a small liquid Nitrogen container, or on dry ice to bring the cells into the culture room. The next part is done quickly, remove the frozen tube and gently shake continuously until only a small, frozen cell pellet remains. Transfer the cells from the tube into a bigger tube, containing some media and spin down to pellet your cells.</p>
<p><strong>Addition of cells to plate: </strong>Don’t fret, your almost there. Once the cells are pelleted, remove the old media, add some fresh media, and voila, after some careful pipetting, add in a dropwise fashion onto your coated dish. Before you do, make sure to remove any matrigel containing media from the plates. Now put the cells back in the incubator, toss your lab coat onto its hanger (wait, no stop), clean up after yourself. And now you are breezy, free to do whatever you wish, all while waiting and hoping the next day brings new hope, or in this case, cells growing and attached onto your dish.</p>
<p><b>Check Up</b><strong> on cells the next day: </strong>It has been a rough night, you barely slept a wink at night dreaming about your cells, don’t believe me, I have spent roughly 1562 nights dreaming about my cells during the course of my scientific career, its like counting sheep, why do you think I look about 70 years old. You open the incubator, put the cells on the microscope, and success. The cells have attached, they look great as shown below.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/iPSC-cell-image_Day-1-300x175.jpg" alt="" width="532" height="310" class=" wp-image-3153 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/iPSC-cell-image_Day-1-300x175.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/iPSC-cell-image_Day-1-768x448.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/iPSC-cell-image_Day-1-1024x597.jpg 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/iPSC-cell-image_Day-1.jpg 1431w" sizes="(max-width: 532px) 100vw, 532px" /></p>
<p><a href="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/iPSC-cell-image_Day-1.tif"><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/iPSC-cell-image_Day-1.tif" alt="" width="1" height="1" class="alignnone size-medium wp-image-3152" /></a>Now they are ready for the next step in their journey. Which is</p>
<p>TO BE CONTINUED. Cue exit music.</p>
<p>I kid, I kid, Tom has not left the building. The rest comes Thursday outlining how to maintain cells, pass cells and freeze cells, all must have skills if you want to be an iPSC master.</p>
<p>So, thanks for reading. Hope you join me again next time. I am off to practice this thing called walking, am doing a <a href="http://donate.parkinson.ca/site/TR/SuperWalk2019/PSC_SuperWalk?pg=team&amp;fr_id=2323&amp;team_id=8983">Parkinson’s super walk</a> in September as my group along with Dr Fon are doing a battle of labs to raise awareness and fundraise for Parkinson’s research. So, if you are Parkinson’s lab in Canada, sign up and just bring it. As it will be brought from our side, and it all goes to a good cause.</p>
<p>Ok, enjoy your day and see you all Thursday again as we continue our journey with stem cells and dinosaurs into the EDDU.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/thawing-your-tube-of-ipscs/feed/</wfw:commentRss>
		<slash:comments>1</slash:comments>
		</item>
		<item>
		<title>Huntingtin chromatin retention in PARP knock out cells</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/huntingtin-chromatin-retention-in-parp-knock-out-cells/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/huntingtin-chromatin-retention-in-parp-knock-out-cells/#respond</comments>
		<pubDate>Mon, 15 Jul 2019 18:38:46 +0000</pubDate>
		<dc:creator><![CDATA[Tamara Maiuri]]></dc:creator>
				<category><![CDATA[Huntingtin]]></category>
		<category><![CDATA[Huntington’s disease]]></category>
		<category><![CDATA[Ray Truant (McMaster Univ.)]]></category>
		<category><![CDATA[Tamara Maiuri]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=3145</guid>
		<description><![CDATA[More forgotten experiments revived by my year-end report. Huntingtin chromatin retention is reduced in PARP KO cells, but in frustration over some confusing results, I overlooked this important data. Lesson: pause and look at the data before rushing on! All the details and links to Zenodo deposits can be found on the Truant lab website.]]></description>
				<content:encoded><![CDATA[<p>More forgotten experiments revived by my <a href="https://raytruantlab.ca/2019/06/27/the-year-in-review/">year-end report</a>. Huntingtin chromatin retention is reduced in PARP KO cells, but in frustration over some confusing results, I overlooked this important data. Lesson: pause and look at the data before rushing on!</p>
<p>All the details and links to Zenodo deposits can be found on the <a href="https://raytruantlab.ca/2019/07/15/huntingtin-chromatin-retention-in-parp-knockout-cells/">Truant lab website</a>.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/Blog-post-15-fig1-copy-300x204.jpg" alt="" width="360" height="245" class="alignnone  wp-image-3146" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/Blog-post-15-fig1-copy-300x204.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/Blog-post-15-fig1-copy-768x521.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/Blog-post-15-fig1-copy-1024x695.jpg 1024w" sizes="(max-width: 360px) 100vw, 360px" /></p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/huntingtin-chromatin-retention-in-parp-knock-out-cells/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Different methods to evaluate the effects of M4K compounds on DIPG patient-derived cell lines</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/different-methods-to-evaluate-the-effects-of-m4k-compounds-on-dipg-patient-derived-cell-lines/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/different-methods-to-evaluate-the-effects-of-m4k-compounds-on-dipg-patient-derived-cell-lines/#respond</comments>
		<pubDate>Fri, 12 Jul 2019 13:18:26 +0000</pubDate>
		<dc:creator><![CDATA[Jong Fu]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Brain Tumours]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Jong Fu Wong]]></category>
		<category><![CDATA[SGC Univ. of Oxford]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=3132</guid>
		<description><![CDATA[Background: Aside from determining the optimal numbers of DIPG cells to be seeded for viability assay, it is also important to validate that measurements using CellTiter Glo are in agreement with other methods of determining cell viability. Experimental design: I chose to compare the EC50 values of M4K2009 on HSJD-DIPG-007 when measured using different methods. <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/different-methods-to-evaluate-the-effects-of-m4k-compounds-on-dipg-patient-derived-cell-lines/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p><strong>Background:</strong></p>
<p>Aside from determining the optimal numbers of DIPG cells to be seeded for viability assay, it is also important to validate that measurements using CellTiter Glo are in agreement with other methods of determining cell viability.</p>
<p><strong>Experimental design:</strong></p>
<p>I chose to compare the EC50 values of M4K2009 on HSJD-DIPG-007 when measured using different methods. A simplified comparison of the different measurement methods is summarised in the following table.</p>
<div id="attachment_3133" style="width: 632px" class="wp-caption alignnone"><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/Figures-for-opennotebook.png" alt="" width="622" height="668" class="size-full wp-image-3133" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/Figures-for-opennotebook.png 622w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/Figures-for-opennotebook-279x300.png 279w" sizes="(max-width: 622px) 100vw, 622px" /><p class="wp-caption-text">Comparison of the different methods</p></div>
<div id="attachment_3134" style="width: 620px" class="wp-caption alignnone"><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/Slide1.png" alt="" width="610" height="314" class="size-full wp-image-3134" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/Slide1.png 610w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/Slide1-300x154.png 300w" sizes="(max-width: 610px) 100vw, 610px" /><p class="wp-caption-text">The timeline of this experiment</p></div>
<div id="attachment_3135" style="width: 622px" class="wp-caption alignnone"><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/Slide2.png" alt="" width="612" height="614" class="size-full wp-image-3135" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/Slide2.png 612w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/Slide2-150x150.png 150w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/Slide2-300x300.png 300w" sizes="(max-width: 612px) 100vw, 612px" /><p class="wp-caption-text">Demonstration of how different stains serve as the indicator of various cell aspects. Dead cells (marked as 1, 2 and 3) that are stained by Propidium Iodide (Red) are not stained by Calcein AM (Green). They are still stained by Hoechst 33342 (Blue), albeit at slight lower intensity due to DNA degradation. Depending on the form (<a href="https://en.wikipedia.org/wiki/Apoptosis">apoptosis</a> vs <a href="https://en.wikipedia.org/wiki/Necrosis">necrosis</a>) and extend of cell death, they might no longer be visible in brightfield (based on <a href="https://en.wikipedia.org/wiki/Differential_interference_contrast_microscopy">Differential Interference Contrast</a>).</p></div>
<div id="attachment_3138" style="width: 817px" class="wp-caption alignnone"><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/Slide5.png" alt="" width="807" height="352" class="size-full wp-image-3138" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/Slide5.png 807w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/Slide5-300x131.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/Slide5-768x335.png 768w" sizes="(max-width: 807px) 100vw, 807px" /><p class="wp-caption-text">EC50 of M4K2009 determined based on the amount of ATP in each well (left) and total area occupied by cells (right). Brightfield images were acquired every 24 hours.</p></div>
<div id="attachment_3139" style="width: 1273px" class="wp-caption alignnone"><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/Slide7.png" alt="" width="1263" height="349" class="size-full wp-image-3139" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/Slide7.png 1263w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/Slide7-300x83.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/Slide7-768x212.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/Slide7-1024x283.png 1024w" sizes="(max-width: 1263px) 100vw, 1263px" /><p class="wp-caption-text">EC50 of M4K2009 determined based on the intensity of dead cell stain (left) and area occupied by the stain (right). Propidium iodide signal was imaged every 24 hours.</p></div>
<div id="attachment_3136" style="width: 773px" class="wp-caption alignnone"><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/Slide3.png" alt="" width="763" height="332" class="size-full wp-image-3136" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/Slide3.png 763w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/Slide3-300x131.png 300w" sizes="(max-width: 763px) 100vw, 763px" /><p class="wp-caption-text">EC50 of M4K2009 determined based on the metabolic activity of the cells (reduction of Calcein AM stain). Both signal intensity (left) and area covered by the signal (right) were examined.</p></div>
<div id="attachment_3137" style="width: 768px" class="wp-caption alignnone"><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/Slide4.png" alt="" width="758" height="338" class="size-full wp-image-3137" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/Slide4.png 758w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/07/Slide4-300x134.png 300w" sizes="(max-width: 758px) 100vw, 758px" /><p class="wp-caption-text">EC50 of M4K2009 determined based on the total number of cells (staining of all cell nuclei). Both signal intensity (left) and area covered by the signal (right) were examined.</p></div>
<p><strong>Findings:</strong></p>
<p>1) 7 days of compound treatment gave the best assay window for EC50 estimated based on the confluency of the cells.</p>
<p>2) CellTiter Glo is superior over other methods of quantification in term of robustness and number of steps involved.</p>
<p>3) Propidium iodide staining of dead cells is not suitable for EC50 estimation because most dead cells likely became too degraded to be stained after 7 days of compound treatment.</p>
<p>4) EC50 estimation based on Calcein AM staining of metabolically active cells was robust but not as sensitive as CellTiter Glo or cell confluency.</p>
<p>5) EC50 estimation based on the amount of nuclei stained by Hoechst 33342 is relatively variable.</p>
<p><strong>In a nut shell:</strong></p>
<p>In future experiments, the EC50 of M4K compounds on the viability of DIPG patient-derived cell lines will be estimated based on CellTiter Glo signal and cell confluency.</p>
<p>For experimental details, please refer to my <a href="https://zenodo.org/record/3333613#.XSiGDutKiUk">Zenodo</a> page.</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/different-methods-to-evaluate-the-effects-of-m4k-compounds-on-dipg-patient-derived-cell-lines/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
	</channel>
</rss>
