<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>Disease &#8211; openlabnotebooks.org</title>
	<atom:link href="https://thesgc.github.io/static-openlabnotebooks/category/disease-foundation/feed/" rel="self" type="application/rss+xml" />
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<description>A growing team of groundbreaking scientists around the world are now sharing their lab notebooks online</description>
	<lastBuildDate>Thu, 20 Jun 2019 11:26:10 +0000</lastBuildDate>
	<language>en-GB</language>
	<sy:updatePeriod>hourly</sy:updatePeriod>
	<sy:updateFrequency>1</sy:updateFrequency>
	<generator>https://wordpress.org/?v=4.9.10</generator>

<image>
	<url>https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/cropped-lab-jar02-32x32.png</url>
	<title>Disease &#8211; openlabnotebooks.org</title>
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<width>32</width>
	<height>32</height>
</image> 
	<item>
		<title>Optimisation of viability assay for DIPG patient-derived cell lines</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/optimisation-of-viability-assay-for-dipg-patient-derived-cell-lines/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/optimisation-of-viability-assay-for-dipg-patient-derived-cell-lines/#respond</comments>
		<pubDate>Thu, 13 Jun 2019 22:41:42 +0000</pubDate>
		<dc:creator><![CDATA[Jong Fu]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Brain Tumours]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Jong Fu Wong]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2935</guid>
		<description><![CDATA[Background: Evaluation of the efficacy of M4K compounds in DIPG patient-derived cell lines is essential before any promising compounds can be further tested in mouse xenograft models. This approach can aid in narrowing down clinical compound candidates and reduce the time, resources and animal sacrifice needed downstream. A robust and efficient readout for the changes <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/optimisation-of-viability-assay-for-dipg-patient-derived-cell-lines/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p><strong>Background:</strong></p>
<p>Evaluation of the efficacy of M4K compounds in DIPG patient-derived cell lines is essential before any promising compounds can be further tested in mouse xenograft models. This approach can aid in narrowing down clinical compound candidates and reduce the time, resources and animal sacrifice needed downstream.</p>
<p>A robust and efficient readout for the changes in the viability of the DIPG cells needs to be established before it can be used to evaluate the M4K compounds. In addition, the amount of cells to be seeded at the beginning of the experiment has to be optimised to avoid overcrowding and starvation of the cells after extended culture times. Overcrowding and starvation will lead to increased cell death and prevent accurate estimation of the potency of M4K compounds (<a href="https://en.wikipedia.org/wiki/EC50">EC50</a>).</p>
<p><strong>Experimental design:</strong></p>
<p>The wells of 96-well plate with transparent bottom will be coated with laminin to synchronise the mode of growth (adherent or non-adherent) among the different patient-derived lines. Additionally, this will promote even distribution of cells and allow most cells to be in focus when images are taken.</p>
<p>An increasing cell number (125 to 20000) will be seeded into each well. The amount of cells after 7 days of growth in 100µl of medium will be measured based on the area occupied and the total amount of ATP present in each well. In addition, propidium iodide which stains dead cells will be added at the beginning the experiment. At seeding number that are not saturating, the amount of cells after 7 days should remain proportional to the initial seeding number.</p>
<p><strong>Results:</strong></p>
<p><a href="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/06/Figure-1.-Experimental-design.tif"><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/06/Figure-1.-Experimental-design.tif" alt="" class="alignnone size-full wp-image-2936" /></a></p>
<div id="attachment_2938" style="width: 436px" class="wp-caption alignnone"><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/06/Figure-1.-Experimental-design.gif" alt="" width="426" height="159" class=" wp-image-2938" /><p class="wp-caption-text">Timeline of experiment</p></div>
<p>&nbsp;</p>
<div id="attachment_2939" style="width: 973px" class="wp-caption alignnone"><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/06/Figure-2.-Quantification-of-confluency-by-Celigo-algorithms.gif" alt="" width="963" height="792" class="wp-image-2939" /><p class="wp-caption-text">Demonstration of how Celigo algorithm quantifies the area occupied by total cells and area occupied by propidium iodide signal. Images of HSJD-DIPG-007 cells after 7 days in culture were used.</p></div>
<div id="attachment_2940" style="width: 966px" class="wp-caption alignnone"><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/06/Figure-3.-Percentage-confluency.gif" alt="" width="956" height="471" class="wp-image-2940" /><p class="wp-caption-text">Percentage of area occupied by cells after 7 days in culture.</p></div>
<div id="attachment_2941" style="width: 971px" class="wp-caption alignnone"><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/06/Figure-4.-CellTiter-Glo.gif" alt="" width="961" height="453" class="wp-image-2941" /><p class="wp-caption-text">Luminescence signal from CellTiter-Glo at endpoint. ATP content is proportional to the amount of luminescence signal and is an indicator of cell viability.</p></div>
<div id="attachment_2942" style="width: 973px" class="wp-caption alignnone"><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/06/Figure-5.-Near-perfect-linearity-up-to-5000-cells-per-well.gif" alt="" width="963" height="465" class="wp-image-2942" /><p class="wp-caption-text">Near perfect linearity. The viability of the cells (CellTiter Glo signal) correlates almost perfectly with the area occupied by the cells (confluency) (below 5000 cells seeded per well). R squared values of the two measurements are indicated next to the fitted curves. A perfect correlation will have an R squared value of 1.</p></div>
<div id="attachment_2943" style="width: 841px" class="wp-caption alignnone"><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/06/Figure-6.-DIPG-cell-seeding-number-chosen.gif" alt="" width="831" height="458" class="wp-image-2943 size-full" /><p class="wp-caption-text">DIPG cell seeding numbers selected for future experiments</p></div>
<p><strong>Findings:</strong></p>
<p>1) Celigo and its algorithm is able to quantify the area occupied by the cells accurately.</p>
<p>2) SU-DIPG-IV and HSJD-DIPG-007 proliferate quickly. Seeding too many of these cells at the beginning of the experiment will saturate both confluency and CellTiter Glo readouts.</p>
<p>3) CellTiter Glo signal and confluency are good indicators of the actual quantity of cells in the well. Below 5000 cells seeded per well, both sets of measurements correlated almost perfectly with each other.</p>
<p>4) Cell death (as indicated by propidium iodide signal) in relation to total cell at endpoint increases when too many cells were seeded at the beginning of the experiment.</p>
<p>5) Initial cell seeding numbers for each patient-derived cell lines that do not lead to overcrowding at the end of the experiment can be chosen based on the linearity of CellTiter Glo signal and the intensity of propidium iodide.</p>
<p><strong>In a nut shell:</strong></p>
<p>We now have the parameters to go ahead and evaluate how well M4K compounds can stop the growth or kill DIPG cells.</p>
<p>For detailed experimental descriptions, please visit my <strong><a href="https://zenodo.org/record/3245503#.XQLMWRZKiUk">Zenodo</a> </strong>page.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/optimisation-of-viability-assay-for-dipg-patient-derived-cell-lines/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Synthesis of promiscuous ALK2 inhibitors-continuation-2</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/synthesis-of-promiscuous-alk2-inhibitors-continuation-2/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/synthesis-of-promiscuous-alk2-inhibitors-continuation-2/#respond</comments>
		<pubDate>Thu, 13 Jun 2019 20:34:07 +0000</pubDate>
		<dc:creator><![CDATA[Alfredo Picado]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Alfredo Picado]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>
		<category><![CDATA[Understudied Kinases]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2924</guid>
		<description><![CDATA[Continuing with our goal to synthesize three compounds as ALK2 inhibitors, let’s refer back to Scheme 2 in the previous post on this topic, compounds 6, 8 and 9 were synthesized in good yields (65-90%), however compound 10 could not be obtained. &#160; Different conditions were tested to induce the cyclization of the bromo-pyrazine-2-carboxamide 9 <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/synthesis-of-promiscuous-alk2-inhibitors-continuation-2/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p style="text-align: justify;">Continuing with our goal to synthesize three compounds as ALK2 inhibitors, let’s refer back to Scheme 2 in the previous post on this topic, compounds <strong>6</strong>, <strong>8</strong> and <strong>9</strong> were synthesized in good yields (65-90%), however compound <strong>10</strong> could not be obtained.</p>
<div id="attachment_2597" style="width: 748px" class="wp-caption alignnone"><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Screen-Shot-2019-03-19-at-4.22.42-PM.png" alt="" width="738" height="522" class=" wp-image-2597" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Screen-Shot-2019-03-19-at-4.22.42-PM.png 918w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Screen-Shot-2019-03-19-at-4.22.42-PM-300x212.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Screen-Shot-2019-03-19-at-4.22.42-PM-768x544.png 768w" sizes="(max-width: 738px) 100vw, 738px" /><p class="wp-caption-text">Scheme 2. Synthesis route 1</p></div>
<p>&nbsp;</p>
<p style="text-align: justify;">Different conditions were tested to induce the cyclization of the bromo-pyrazine-2-carboxamide <strong>9</strong> to the target bromo-pyrazin-2-amine <strong>10</strong>. This reaction mostly yielded starting material <strong>9 </strong>and by-product <strong>11</strong> in low yields (&lt;15%). After several attempts, this approach (acid/heat) was abandoned.</p>
<p>&nbsp;</p>
<div id="attachment_2926" style="width: 746px" class="wp-caption alignnone"><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/06/Screen-Shot-2019-06-13-at-4.17.14-PM.png" alt="" width="736" height="592" class=" wp-image-2926" /><p class="wp-caption-text">Scheme 3. Synthesis route 2</p></div>
<p style="text-align: justify;">An alternative synthetic route is shown in Scheme 3.  Here, aminopirazine <strong>12</strong> will be reduced with diisobutylaluminium hydride (DIBAL-H) to yield carbaldehyde <strong>13</strong>, which then will be reacted with diamine <strong>6</strong> (previously synthesized) at 200 <sup>o</sup>C using dimethylacetamide (DMA) as a solvent. This last step can be carried out in a microwave reactor and should yield the cyclized compound pyridinpyrazin-2-amine <strong>14</strong>, which can be brominated with <em>N</em>-Bromosuccinimide (NBS) to yield compound <strong>10</strong>.</p>
<p style="text-align: justify;">As it was mentioned before, <strong>10 </strong>could not be synthesized by the previously proposed route. The new route is adapted from international patent WO 2008/071937 and we hope in this way we can access to this key intermediate in 150-200 mg scale to move forward with more reactions.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/synthesis-of-promiscuous-alk2-inhibitors-continuation-2/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Project Overview: Huntingtin and its interaction partners</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/project-overview-huntingtin-and-its-interaction-partners/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/project-overview-huntingtin-and-its-interaction-partners/#respond</comments>
		<pubDate>Mon, 10 Jun 2019 14:15:20 +0000</pubDate>
		<dc:creator><![CDATA[Jacob McAuley]]></dc:creator>
				<category><![CDATA[Huntingtin]]></category>
		<category><![CDATA[Huntington’s disease]]></category>
		<category><![CDATA[Jacob McAuley]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2905</guid>
		<description><![CDATA[&#160; Huntington’s disease (HD) is a progressive neurological disorder caused by a mutation in the HTT gene which encodes for the huntingtin protein. The current understanding of the function of HTT is limited, and there are only a few sources that indicate that it likely plays a critical role in several intracellular processes such as <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/project-overview-huntingtin-and-its-interaction-partners/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>&nbsp;</p>
<p>Huntington’s disease (HD) is a progressive neurological disorder caused by a mutation in the HTT gene which encodes for the huntingtin protein. The current understanding of the function of HTT is limited, and there are only a few sources that indicate that it likely plays a critical role in several intracellular processes such as proper neural development. This gap in the scientific literature continues onto the mutant version of the protein. In patients with HD, we know that the mutant protein misfolds and forms aggregates. However, it is unclear if this is the direct cause for the symptoms of the disease or if it is a downstream symptom of some other problem. To obtain a more conclusive understanding of huntingtins role in the cell, the SGC is trying to obtain a structure for huntingtin protein.<br />
So now the question is, how do we image such a molecule? At the SGC we use a technique called X-ray Crystallography. This is a process by which a very pure protein sample is crystallized and expose to X-rays. Using the diffraction of light through the crystal, we can piece together an atom by atom computer model of the sample protein. One complication with the huntingtin protein is that it contains several disordered regions which would likely prevent the formation of a consistent structure in the crystal. To get a full image of this protein, we need to find a way to force a consistent structure onto the disordered regions. Binding a protein to the disordered region causes consistent reordering of this region. The use of a <a href="https://zenodo.org/record/3234159#.XP5doIhKjcs">BioID experiment</a> has identified several proteins that bind huntingtin. In effect, this experiment consists of arming the huntingtin protein with a spray can. When another protein comes within its vicinity, it is tagged by the huntingtin molecule. By looking at tagged proteins, several interaction partners of huntingtin were identified.<br />
During my time at the SGC, I plan to go through this list of tentative interaction partners, producing and purifying them for use in future experiments. Furthermore, this could allow us to obtain crystal structures of huntingtin in complex with an interaction partner. Obtaining a list of all the interaction partners of any given protein can give us a good idea of the function of the protein. This is because we can then attribute the interaction partners function to that of the protein of interest. In the coming weeks, I will continue to post updates on the progress of the purification of the hits from the BioID experiment.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/project-overview-huntingtin-and-its-interaction-partners/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Identifying new members of the huntingtin interactome with BioID and more updates</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/identifying-new-members-of-the-huntingtin-interactome-with-bioid-and-more-updates/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/identifying-new-members-of-the-huntingtin-interactome-with-bioid-and-more-updates/#respond</comments>
		<pubDate>Thu, 30 May 2019 20:38:45 +0000</pubDate>
		<dc:creator><![CDATA[Rachel Harding]]></dc:creator>
				<category><![CDATA[Huntington’s disease]]></category>
		<category><![CDATA[Rachel Harding]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2890</guid>
		<description><![CDATA[I have been busy in the lab working at the bench and training our new recruits to the Huntington’s disease research team! Welcome Jacob and Claudia! Using a technique called BioID, we have identified proteins which are proximal or nearby to the HTT protein in cells. To date, no published literature details the use of <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/identifying-new-members-of-the-huntingtin-interactome-with-bioid-and-more-updates/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>I have been busy in the lab working at the bench and training our new recruits to the Huntington’s disease research team! Welcome Jacob and Claudia!</p>
<p>Using a technique called <a href="https://zenodo.org/record/3234159">BioID</a>, we have identified proteins which are proximal or nearby to the HTT protein in cells. To date, no published literature details the use of this methodology to identify proteins which might interact with HTT so this represents a novel approach. Working in collaboration with Prof. Anne-Claude Gingras (Lunenfeld Tanenbaum Research Institute, University of Toronto) and her postdoctoral fellow Dr. Geoff Hesketh, we have used this technique to finalise a short-list of supposed HTT interacting proteins, many of which are newly described in their connection to Huntington’s disease. This is an exciting step forward to this project and I am keen to follow up all of these new leads.</p>
<p>Working in collaboration with Prof. Hilal Lashuel and Driss Boudeffa as well as our cryoEM collaborators Prof. Susan Lea and Dr. Justin Deme, we have analysed how altering the <a href="https://zenodo.org/record/3234163">phosphorylation state of HTT might change the global structure</a>. We analysed our dephosphorylated samples by cryoEM and we can see the global structure of the protein from <a href="https://zenodo.org/record/3234167">analysis of the images</a> collected by Susan and Justin. I am looking forward to their next update on the remaining samples.</p>
<p>The stringent protein purification protocol for HTT is now <a href="https://zenodo.org/record/3234174">optimised</a> and we used this to purify both <a href="https://zenodo.org/record/3234182">HTT and HTT-HAP40 Q23</a> . These samples are ready for analysis and use in different interaction studies.</p>
<p>I am away from the lab for the next few weeks as I will be travelling to Europe to attend the <a href="https://www.grc.org/cag-triplet-repeat-disorders-conference/2019/">Gordon Research Conference on Triplet Repeat Disorders in Italy</a>. I am excited to give my talk and present my poster as well as catch up with many of my HD colleagues and collaborators.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/identifying-new-members-of-the-huntingtin-interactome-with-bioid-and-more-updates/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>ACVR1 Crystals taken to the Diamond Light Source (Beamline I04.) &#8211; no data sets collected.</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/acvr1-crystals-taken-to-the-diamond-light-source-beamline-i04-no-data-sets-collected/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/acvr1-crystals-taken-to-the-diamond-light-source-beamline-i04-no-data-sets-collected/#respond</comments>
		<pubDate>Mon, 13 May 2019 13:19:13 +0000</pubDate>
		<dc:creator><![CDATA[Ellie Williams]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Ellie Williams]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2861</guid>
		<description><![CDATA[Following up on the crystals from my last post I sent them along on the last Diamond trip we had to beamline I04 but regrettably didn’t get any usable datasets. Perhaps fortunately there is a possible explanation for this that isn’t just ‘the crystals were a bit too rubbish’. We send our crystals to the Diamond <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/acvr1-crystals-taken-to-the-diamond-light-source-beamline-i04-no-data-sets-collected/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Following up on the crystals from my last post I sent them along on the last Diamond trip we had to <a href="https://www.diamond.ac.uk/Instruments/Mx/I04">beamline I04</a> but regrettably didn’t get any usable datasets. Perhaps fortunately there is a possible explanation for this that isn’t just ‘the crystals were a bit too rubbish’. We send our crystals to the Diamond Light Source and because of the high number of crystals sent on each trip by the department (124 on the last trip) not everyone goes along on every trip and we quite often use something called ‘auto-centering’ that uses a computer algorithm to find the edge of the loop and to align the loop and your crystal in the beam. Centering is necessary but also quite tricky, because to collect a data set you need ‘slices’ of data taken with the crystal at different angles so the loop the crystals are mounted in, is rotated by a robotic arm so that the crystal also rotates around inside the x-ray beam. Thus if your crystal isn’t aligned properly, as the loop rotates it moves you crystal out of the beam and thus you get no data at that point.</p>
<p>Usually the automatic alignment is really good, however this time around it didn’t work so well and quite often either my crystal wasn’t in the beam at all (it mistook the loop stem for the crystal) or it started in the beam and then rotated out of it (which is what I think happened in the case shown below where you can see two ‘periods’ of data with dips in between. This is, of course, suboptimal and leads to a data set that you can’t really use.</p>
<p>I’m resending the crystals to the beamline this Friday and hopefully going with them myself. The silver lining to this is that I know that some of them definitely diffract which is encouraging. Hopefully they’ll behave well when fully in the beam.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/05/xtal_centering.png" alt="" width="940" height="430" class="alignnone size-full wp-image-2862" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/05/xtal_centering.png 940w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/05/xtal_centering-300x137.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/05/xtal_centering-768x351.png 768w" sizes="(max-width: 940px) 100vw, 940px" /></p>
<p><em>Fig. 1 – Beam aligned at very edge of loop showing cyclic diffraction suggesting the crystal was moving in and out of the beam rather than degrading in the beam.</em></p>
<p>&nbsp;</p>
<p>You can read more about this <a href="https://doi.org/10.5281/zenodo.2788112">over on Zenodo.</a></p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/acvr1-crystals-taken-to-the-diamond-light-source-beamline-i04-no-data-sets-collected/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>CRISPR knockout of EZH1 in AML cell line</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/crispr-knockout-of-ezh1-in-aml-cell-line/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/crispr-knockout-of-ezh1-in-aml-cell-line/#respond</comments>
		<pubDate>Fri, 10 May 2019 21:37:11 +0000</pubDate>
		<dc:creator><![CDATA[Genna Luciani]]></dc:creator>
				<category><![CDATA[Genna Luciani]]></category>
		<category><![CDATA[Leukemia]]></category>
		<category><![CDATA[PRC2]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2855</guid>
		<description><![CDATA[Hello again. My apologies for the delay in posting &#8211; I&#8217;ve been busy with other projects lately. As a follow up to my previous post, we wanted to further confirm our EZH1 antibody and test it in a knockout line. To do this, we generated a CRISPR knockout of EZH1 in an AML cell line <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/crispr-knockout-of-ezh1-in-aml-cell-line/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Hello again. My apologies for the delay in posting &#8211; I&#8217;ve been busy with other projects lately. As a follow up to my <a href="https://thesgc.github.io/static-openlabnotebooks/finally-finding-an-antibody-to-detect-ezh1-protein-expression/">previous</a> post, we wanted to further confirm our EZH1 antibody and test it in a knockout line. To do this, we generated a CRISPR knockout of EZH1 in an AML cell line that will be a great tool for future experiments. To generate the knockout line, we were helped by Fraser Soares, a postdoctoral fellow in Hansen He&#8217;s lab. He provided us with an AML cell line (OCI-AML-3) that expresses Cas9, along with guide RNAs that would allow us to knockout EZH1. Once the knockouts were generated, we tested the expression of EZH1 and EZH2. We determined that the guides knocked out EZH1, but had no effect on related EZH2 expression levels (Figure 1). We can now conclude we have a specific EZH1 antibody and EZH1 knockout lines that we can use in the future. For full experimental details, please see <a href="https://zenodo.org/record/2718713#.XNXV6TBKjIU">Zenodo</a> post.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/05/EZH1-crispr-2-300x100.png" alt="" width="342" height="114" class="wp-image-2857 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/05/EZH1-crispr-2-300x100.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/05/EZH1-crispr-2-768x255.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/05/EZH1-crispr-2-1024x340.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/05/EZH1-crispr-2.png 1462w" sizes="(max-width: 342px) 100vw, 342px" /></p>
<p>Figure 1: Westerns of Crispr knockouts in OCI-AML-3 Cas9 cells with either control guide RNA (sgLacZ) or guide RNA to EZH1. Levels of EZH1 were undetectable with knockout (A) and did not affect EZH2 levels (B).</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/crispr-knockout-of-ezh1-in-aml-cell-line/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>How many cells in DIPG behave like stem cells?</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/how-many-cells-in-dipg-behave-like-stem-cells/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/how-many-cells-in-dipg-behave-like-stem-cells/#respond</comments>
		<pubDate>Mon, 06 May 2019 15:40:30 +0000</pubDate>
		<dc:creator><![CDATA[Elizabeth Brown]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Brain Tumours]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Liz Brown]]></category>
		<category><![CDATA[Understudied Kinases]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2804</guid>
		<description><![CDATA[Stem cells are a set of cells that, unlike most cells in the body, can divide as many times as they want, and can turn into any cell in the body. The mutations in cancer often make the cells behave more like stem cells, because in order to grow into a large tumour they may <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/how-many-cells-in-dipg-behave-like-stem-cells/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p><a href="https://stemcells.nih.gov/info/basics/1.htm">Stem cells</a> are a set of cells that, unlike most cells in the body, can divide as many times as they want, and can turn into any cell in the body. The mutations in cancer often make the cells behave more like stem cells, because in order to grow into a large tumour they may need to divide hundreds or thousands of cells, and it can allow a single cell to escape to another part of the body and grow into a second tumour (this is called metastasis). Knowing whether all of the cells in a tumour, or even a small proportion of them behave like this, and whether they&#8217;re killed by new therapies can be important information &#8211; if just one of these cells survives theoretically they can regrow into an entire new tumour.</p>
<p>And this ability to regenerate an tumour is just how we test how many of these cells are present: I developed an assay where I dilute the cells until I can seed just a handful into each well, and then I count how many new colonies (like tiny tumours) have grown.</p>
<p>I then used a statistical rule (called <a href="http://www.biology-pages.info/L/LimitingDilution.html">the Poisson distribution</a>) and a programming package (called R) to work out at what dilution there is just one stem-like cell present i.e. if only one stem cell is present when I seeded 5 cells then 20% are stem-like.</p>
<div id="attachment_2805" style="width: 310px" class="wp-caption aligncenter"><a href="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/05/poisson-graph.png"><img class="size-medium wp-image-2805" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/05/poisson-graph-300x300.png" alt="A graph showing how the program mathematically transforms the data and then finds a straight line that fits it. Where this line crosses -1 indicates where 1 stem-like cell is present." width="300" height="300" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/05/poisson-graph-300x300.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/05/poisson-graph-150x150.png 150w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/05/poisson-graph-768x768.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/05/poisson-graph.png 960w" sizes="(max-width: 300px) 100vw, 300px" /></a><p class="wp-caption-text">The program mathematically transforms the data and then finds a straight line that fits it. Where this line crosses -1 indicates where 1 stem-like cell is present.</p></div>
<p>Although I need to repeat it to make sure the result is reliable, it seems that almost all of the cells in this line are stem-like! <a href="https://zenodo.org/record/2669443">If you&#8217;d like to read about the technical details of this experiment you can do so here.</a></p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/how-many-cells-in-dipg-behave-like-stem-cells/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Growing co-crystallized fragments to ligands of the transcription factor brachyury (T)</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/growing-co-crystallized-fragments-to-ligands-of-the-transcription-factor-brachyury-t/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/growing-co-crystallized-fragments-to-ligands-of-the-transcription-factor-brachyury-t/#respond</comments>
		<pubDate>Wed, 24 Apr 2019 21:15:14 +0000</pubDate>
		<dc:creator><![CDATA[Louisa Temme]]></dc:creator>
				<category><![CDATA[Chordoma]]></category>
		<category><![CDATA[Louisa Temme]]></category>
		<category><![CDATA[Understudied Kinases]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2758</guid>
		<description><![CDATA[In order to develop ligands addressing different possible binding sites of the transcription factor brachyury, fragments co-crystallized with this protein should be developed into ligands by using the program SparkTM (Cresset®). Brachyury (T, TBXT) is an essential regulator of the notochord development. Brachyury is primarily expressed in the embryo but not in the majority of normal <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/growing-co-crystallized-fragments-to-ligands-of-the-transcription-factor-brachyury-t/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>In order to develop ligands addressing different possible binding sites of the transcription factor brachyury, fragments co-crystallized with this protein should be developed into ligands by using the program Spark<sup>TM </sup>(Cresset<sup><span style="font-size: 11.25px;">®</span></sup>).</p>
<p>Brachyury (T, TBXT) is an essential regulator of the notochord development. Brachyury is primarily expressed in the embryo but not in the majority of normal adult tissue. It is known that brachyury is overexpressed in chordoma, a primary bone cancer, and plays a role in the disease pathogenesis. Chordoma affects the skull-base, the mobile spine and the sacrum. It is a slow-growing, malign cancer that is locally invasive and has a tendency to recur although a surgical and/or radiation therapy was conducted.<sup>1</sup></p>
<p>It is hypothesized that chordoma is originated from embryonic notochordal remnants. Brachyury is proposed to act as an aberrantly activated transcription factor and thereby be oncogenic. No approved targeted therapies, chemotherapies or immunotherapies are available for chordoma and no small-molecule inhibitors for brachyury have been developed.<sup>1 </sup>Therefore, a fragment-based approach should be explored to develop ligands for brachyury to study this protein.</p>
<p>Our colleagues Joseph Newman and Opher Gileadi at SGC Oxford co-crystallized wild-type human brachyury protein with various fragments which bind to the sites A-E (Figure 1, left). Thankfully, they provided the fragment and protein structures to us so that we can explore the usage of the Spark<sup>TM </sup>growing approach to develop ligands for these binding sites.</p>
<p><img class="wp-image-2759 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Picture1-300x121.png" alt="" width="592" height="239" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Picture1-300x121.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Picture1-768x310.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Picture1.png 974w" sizes="(max-width: 592px) 100vw, 592px" />Figure 1. Left: Aligned X-ray structures of the human wild-type brachyury protein in complex with fragments at different binding sites (A-E). These first crystallization structures contain Cd<sup>2+</sup>ions (green balls). Right: Binding site A close to loop with D177 and in proximity to residue Y88.<sup>2</sup></p>
<p>First, we focused on binding site A since most fragments bind to this binding site and site A is close to a loop that is mutated in chordoma-linked variant G177D<sup>3 </sup>and to the amino acid residue Y88 that, if mutated, affects binding to effector proteins<sup>4 </sup>(Figure 1, left).</p>
<p><img class="wp-image-2761 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Picture2-300x86.png" alt="" width="607" height="174" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Picture2-300x86.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Picture2-768x220.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Picture2.png 974w" sizes="(max-width: 607px) 100vw, 607px" /></p>
<p>Figure 2. Fragment <strong>1 </strong>and <strong>2 </strong>co-crystallized in the binding site A of wild-type TBXT.</p>
<p>21 fragments were co-crystallized to binding site A. In Figure 2, the binding modes of two co-crystallized fragments are displayed. Fragments <strong>1 </strong>and <strong>2 </strong>show interactions or close proximity to two of the three amino acids Ser129, Ser89 and Arg180. Therefore, the growing approach of the program Spark<sup>TM </sup>should be used to identify new ligands that are able to form all three interactions. One growing approach started from the amino group of fragment <strong>1 </strong>and was guided by fragment <strong>2 </strong>towards the region of its carboxylate group (Figure 3).</p>
<p><img class="wp-image-2763 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Picture3-300x171.png" alt="" width="484" height="276" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Picture3-300x171.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Picture3-768x438.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Picture3-780x445.png 780w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Picture3.png 781w" sizes="(max-width: 484px) 100vw, 484px" /></p>
<p>Figure 3: Spark<sup>TM </sup>growing approach starting from fragment <strong>1 </strong>towards Arg180 guided by fragment <strong>2</strong>. The functional groups of fragments <strong>1 </strong>and <strong>2 </strong>revealing H-bond interactions with amino acid residues are marked with the following colors: Ser129 (red), Ser89 (blue), Arg180 (green).</p>
<p>To achieve new ligands with similar properties in certain areas so-called field points were used. They are ‘condensed representations of electrostatic, hydrophobic and shape properties’ (Cresset<sup><span style="font-size: 11.25px;">®</span></sup>) of those areas where they are placed by the program Spark<sup>TM</sup>. Regarding the growing approach of fragment <strong>1 </strong>and <strong>2 </strong>the hydrophobic field point of the phenyl ring and the negative field point of the carboxylate group of fragment <strong>2 </strong>were used. In total three of the 21 fragments bound to site A were used as starting points and each of them were grown in the area of two different fragments. The results were clustered by Spark<sup>TM </sup>depending on their structural similarity. Those compounds with the highest score of the first 20 clusters ranked according their scores were chosen to analyze their interactions by docking into binding site A using the docking program Flare<sup>TM </sup>(Cresset<sup><span style="font-size: 11.25px;">®</span></sup>). 4,000 docking results were inspected visually and compounds with a high docking score, high electrostatic complementarity and high number of interactions to the binding site were picked.</p>
<p>Figure 4 shows the docking pose of the example compound <strong>3 </strong>that was obtained by docking <strong>3 </strong>in binding site A of brachyury using Flare<sup>TM</sup>. Besides other criteria like docking score and electrostatic complementarity following interactions of <strong>3 </strong>are promising for the co-crystallization of this compound with brachyury: two H-bond interactions of the positively charged amino moiety to Ser129 and His126, the proximity of the amide carbonyl-group to Ser89 and the H-bond interactions of the oxazole O-atom and N-atom towards Arg180.</p>
<p><img class="wp-image-2765 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Picture4-300x232.png" alt="" width="454" height="351" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Picture4-300x232.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Picture4.png 633w" sizes="(max-width: 454px) 100vw, 454px" />Figure 4. Binding mode of new compound <strong>3 </strong>obtained by docking into binding site A of wild-type brachyury with Flare<sup>TM</sup>. H-bond interactions are marked with green dotted lines.</p>
<p>Additionally, new compounds were obtained by searching in the Enamine REAL database for similar compounds to those compounds that were found by the Spark<sup>TM </sup>approach and that showed promising docking results in Flare<sup>TM</sup>.</p>
<p>A selection of compounds that are identical or slightly different than those found in the Spark<sup>TM </sup>approach as well as compounds that represent similar ones to the Spark<sup>TM </sup>hits were purchased and will be tested regarding their ability to co-crystallize with brachyury. If the compounds bind to this transcription factor, they will represent promising starting points for future ligands of brachyury.</p>
<p>&nbsp;</p>
<p>(1) Sharifnia, T., <em> Med. (N. Y., NY, U. S.)</em>, <strong>2019</strong>, <em>25</em>, 292-300.</p>
<p>(2) Joseph Newman and Opher Gileadi, SGC Oxford, University of Oxford, UK.</p>
<p>(3) Pillay, N., <em>Nat. Genet.</em>, <strong>2012</strong>, <em>44</em>, 1185-1187.</p>
<p>(4) Beisaw, A., <em>EMBO reports</em>, <strong>2018</strong>, <em>19</em>, 118-134.</p>
<p>&nbsp;</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/growing-co-crystallized-fragments-to-ligands-of-the-transcription-factor-brachyury-t/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Re-Purification of ALK2 for crystallisation with different compounds from M4K.</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/re-purification-of-alk2-for-crystallisation-with-different-compounds-from-m4k/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/re-purification-of-alk2-for-crystallisation-with-different-compounds-from-m4k/#respond</comments>
		<pubDate>Wed, 24 Apr 2019 14:50:59 +0000</pubDate>
		<dc:creator><![CDATA[Ellie Williams]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Ellie Williams]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2753</guid>
		<description><![CDATA[I’ve been lucky enough to have a student (Mira) in the lab working with me for the past few weeks and so I’d like to write about the work that we’ve been doing together. We spent some time cleaning up an old sample of ALK2 that Ros had left over in the freezer to set <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/re-purification-of-alk2-for-crystallisation-with-different-compounds-from-m4k/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>I’ve been lucky enough to have a student (Mira) in the lab working with me for the past few weeks and so I’d like to write about the work that we’ve been doing together.</p>
<p>We spent some time cleaning up an old sample of ALK2 that Ros had left over in the freezer to set up in crystal plates with some compounds from M4K Pharma that they were relatively keen to see the structures of.</p>
<p>We used a specialised coarse screen that Ros had designed containing only conditions where we had previously seen crystals in the wider coarse screens. This means that we can narrow down the field where we’re expecting to find crystals and test fewer conditions to find the one we need – this means setting up fewer plates and thus needing less protein per compound tested, so is overall more efficient.</p>
<p>We set up the plates (at 4C and 20C) at around 9mg/ml with compounds M4K2163, M4K2143, M4K2192, M4K2207 and M4K2194 at 1mM concentration from 25mM stocks.</p>
<p>We found some interesting crystals, one of which was mountable as is (from M4K2192) and several across all the compounds, which showed small crystals that would be worth following up in more specific conditions.</p>
<p><img class="alignnone size-full wp-image-2754" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/M4K2192.png" alt="" width="403" height="294" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/M4K2192.png 403w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/M4K2192-300x219.png 300w" sizes="(max-width: 403px) 100vw, 403px" /></p>
<p><em>Crystals of ALK2/ACVR1A with compound M4K2192</em></p>
<p>&nbsp;</p>
<p>So Mira designed some follow up screens based on the conditions that showed small crystals and we set up some more plates with those conditions in the hope of turning those small crystals into larger, better crystals. We had enough protein to set up followup plates for compounds M4K2163 and M4K2143 at 9.5mg/ml. From these plates we mounted 5 crystals across the board and combined with those above ended up with crystals of unknown quality across three compounds. These have been confirmed as containing protein by imaging the drops by UV light (UV light interacts with protein to make crystals only visible under UV light if they contain protein rather than just being salt) and are now waiting to be sent to the synchrotron to see if they actually diffract or not.  You can read more about the technical details <a href="https://zenodo.org/record/2650203#.XMB29jBKiUk">here on Zenodo.</a></p>
<p>&nbsp;</p>
<p><span style="font-size: 1.5rem;">Yesterday (23</span><span style="font-size: 11.25px; height: 0px; line-height: 0; position: relative; vertical-align: baseline; bottom: 1ex;">rd</span><span style="font-size: 1.5rem;">) also marked FOP awareness day and so, shamefully a day late, I’d like to add my picture to the many others out there celebrating</span><span style="font-size: 1.5rem;"> </span><strong style="font-size: 1.5rem;">#FunFeet4FOP</strong><span style="font-size: 1.5rem;">! Here are my toes.</span></p>
<p><img class="alignnone size-medium wp-image-2755" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Feet4FOP_crop-300x223.jpg" alt="" width="300" height="223" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Feet4FOP_crop-300x223.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Feet4FOP_crop-768x570.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Feet4FOP_crop-1024x761.jpg 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Feet4FOP_crop.jpg 1143w" sizes="(max-width: 300px) 100vw, 300px" /></p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/re-purification-of-alk2-for-crystallisation-with-different-compounds-from-m4k/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Why the heck does huntingtin bind PAR?</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/why-the-heck-does-huntingtin-bind-par/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/why-the-heck-does-huntingtin-bind-par/#respond</comments>
		<pubDate>Tue, 23 Apr 2019 19:45:27 +0000</pubDate>
		<dc:creator><![CDATA[Tamara Maiuri]]></dc:creator>
				<category><![CDATA[Huntingtin]]></category>
		<category><![CDATA[Huntington’s disease]]></category>
		<category><![CDATA[Tamara Maiuri]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2742</guid>
		<description><![CDATA[Updates on the huntingtin PAR binding story can be found on the Truant lab website.]]></description>
				<content:encoded><![CDATA[<p>Updates on the huntingtin PAR binding story can be found <a href="https://raytruantlab.ca/2019/04/01/why-the-heck-does-huntingtin-bind-par/">on the Truant lab website</a>.</p>
<div id="attachment_2746" style="width: 310px" class="wp-caption alignnone"><img class="size-medium wp-image-2746" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/2019-04-01-FRAP-crop-300x151.jpg" alt="" width="300" height="151" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/2019-04-01-FRAP-crop-300x151.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/2019-04-01-FRAP-crop-768x388.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/2019-04-01-FRAP-crop-1024x517.jpg 1024w" sizes="(max-width: 300px) 100vw, 300px" /><p class="wp-caption-text">FRAP experiment testing nuclear huntingtin dynamics in response to oxidative stress.</p></div>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/why-the-heck-does-huntingtin-bind-par/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
	</channel>
</rss>
