<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>Disease &#8211; openlabnotebooks.org</title>
	<atom:link href="https://thesgc.github.io/static-openlabnotebooks/category/disease-foundation/feed/" rel="self" type="application/rss+xml" />
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<description>A growing team of groundbreaking scientists around the world are now sharing their lab notebooks online</description>
	<lastBuildDate>Fri, 27 Sep 2019 13:24:29 +0000</lastBuildDate>
	<language>en-GB</language>
	<sy:updatePeriod>hourly</sy:updatePeriod>
	<sy:updateFrequency>1</sy:updateFrequency>
	<generator>https://wordpress.org/?v=4.9.11</generator>

<image>
	<url>https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/cropped-lab-jar02-32x32.png</url>
	<title>Disease &#8211; openlabnotebooks.org</title>
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<width>32</width>
	<height>32</height>
</image> 
	<item>
		<title>Purification of HTT N-HEAT_81-1643</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/purification-of-htt-n-heat_81-1643/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/purification-of-htt-n-heat_81-1643/#respond</comments>
		<pubDate>Fri, 27 Sep 2019 13:24:29 +0000</pubDate>
		<dc:creator><![CDATA[Claudia Alvarez]]></dc:creator>
				<category><![CDATA[Claudia Alvarez]]></category>
		<category><![CDATA[Huntingtin]]></category>
		<category><![CDATA[Huntington’s disease]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=3587</guid>
		<description><![CDATA[It’s being a while since I wrote my first post. I have been busy in the lab trying to isolate monomeric and stable samples of the N-HEAT domain just as we demonstrated with the C-HEAT domain (and by Dr. Rachel Harding). In this post, I summarize the results of the first round of purification of <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/purification-of-htt-n-heat_81-1643/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>It’s being a while since I wrote my first post. I have been busy in the lab trying to isolate monomeric and stable samples of the N-HEAT domain just as we demonstrated with the <a href="https://thesgc.github.io/static-openlabnotebooks/purification-of-the-htt-c-heat-domain/">C-HEAT</a> domain (and by <a href="https://zenodo.org/record/2628060#.XYzlUmYpDb0">Dr. Rachel Harding)</a>.</p>
<p>In this post, I summarize the results of the first round of purification of the huntingtin N-HEAT domain with boundaries 81-1643 (<a href="https://zenodo.org/record/3462496#.XYzkY2YpDb0">HTT N-HEAT_81-1643</a>).</p>
<p>The construct HTT N-HEAT_81-1643 encompasses the entire N &#8211; terminal HEAT domain (all 21 HEAT repeats) (Figure 1). This construct and a few others N-HEAT constructs (which I will be posting about later) were selected among several N-HEAT constructs initially made due to high levels of expression in Sf9 cells (<a href="https://zenodo.org/record/2600051#.XKU89aeZPOQ">Dr. Rachel Harding</a>). The first large scale purification of HTT N-HEAT_81-1643 showed this construct elutes early from the Superdex 200 10/300 column (<a href="https://zenodo.org/record/2628060#.XULMtnspDb0">Dr. Rachel Harding</a>). This implied the construct was either in a high oligomeric state or an extended conformation.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/09/HTT_N-HEAT_B08_illustrator_2-e1569590474153-300x244.png" alt="" class="wp-image-3607 aligncenter" width="330" height="268" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/09/HTT_N-HEAT_B08_illustrator_2-e1569590474153-300x244.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/09/HTT_N-HEAT_B08_illustrator_2-e1569590474153-768x624.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/09/HTT_N-HEAT_B08_illustrator_2-e1569590474153-1024x833.png 1024w" sizes="(max-width: 330px) 100vw, 330px" /></p>
<p>Figure 1. Hunting structure (PDB 6ez8)<sup>1</sup>. N-HEAT (91-1641) colored in cyan</p>
<p>In <a href="https://zenodo.org/record/3462496#.XYzkY2YpDb0">this purification</a>, I used a different gel filtration column (Superpose 6 10/300) which has a broader fractionation range of MW weights than the Superdex 200 column used initially. By using this column, I could better idea of the particle size of HTT N-HEAT construct. The results from this purification showed the construct also eluted early from the Superose 6 column (MW ~670 KDa- according to protein standards). Note that, if the construct was a monomer and had globular shape, it should a elute at a volume of around 16 &#8211; 17 mL (the monomer = 172 KDa). The gel filtration results imply that the construct is in a high oligomeric state likely a tetramer. However, it is still also possible that the construct is in an extended conformation and this would also result in an early elution volume from the gel filtration column (large hydrodynamic ratio). Yet another possible explanation is that HTT N-HEAT_81-1643 is in a complex with nucleic acid material as final A<sub>260/280</sub> readings of the purified samples of the HTT N-HEAT_81-1643 show high content of nucleic acid material (A<sub>260/280</sub> = 0.7). Future experiments with this HTT N-HEAT_81-1643 will aim at testing these hypotheses.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/09/20190603_N-term-HTT-B08-superose_2-e1569590429899-300x174.png" alt="" class="size-medium wp-image-3609 aligncenter" width="300" height="174" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/09/20190603_N-term-HTT-B08-superose_2-e1569590429899-300x174.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/09/20190603_N-term-HTT-B08-superose_2-e1569590429899-768x445.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/09/20190603_N-term-HTT-B08-superose_2-e1569590429899-1024x594.png 1024w" sizes="(max-width: 300px) 100vw, 300px" /></p>
<p>Figure 2. Gel filtration profile N-HEAT_81-1643 using a Superose 6 10/300 column. Elution volume of construct N-HEAT_81-1643 is 12.6 mL.</p>
<p>The sample that was obtained from this purification was used to set crystal trays and to test sample stability. To date, no crystals have been observed in any of the trays set for this construct. Regarding the sample stability, the samples are easily degraded at 4 °C in comparison with the <a href="https://thesgc.github.io/static-openlabnotebooks/purification-of-the-htt-c-heat-domain/">C-HEAT</a> domain.</p>
<ol>
<li>Guo, Q.; Huang, B.;  Cheng, J. D.;  Seefelder, M.;  Engler, T.;  Pfeifer, G.;  Oeckl, P.;  Otto, M.;  Moser, F.;  Maurer, M.;  Pautsch, A.;  Baumeister, W.;  Fernandez-Busnadiego, R.; Kochanek, S., The cryo-electron microscopy structure of huntingtin. <em>Nature </em><strong>2018,</strong> <em>555</em> (7694), 117-+.</li>
</ol>
<p>&nbsp;</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/purification-of-htt-n-heat_81-1643/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Update on the huntingtin PAR binding story</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/update-on-the-huntingtin-par-binding-story/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/update-on-the-huntingtin-par-binding-story/#respond</comments>
		<pubDate>Thu, 26 Sep 2019 13:33:56 +0000</pubDate>
		<dc:creator><![CDATA[Tamara Maiuri]]></dc:creator>
				<category><![CDATA[Huntingtin]]></category>
		<category><![CDATA[Huntington’s disease]]></category>
		<category><![CDATA[Ray Truant (McMaster Univ.)]]></category>
		<category><![CDATA[Tamara Maiuri]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=3581</guid>
		<description><![CDATA[One of the major findings of the HDSA-funded project to find huntingtin interactors relevant DNA repair was that many proteins that interact with huntingtin are modified by poly ADP ribose (PAR). Previous posts have described our hypothesis that huntingtin binds poly ADP ribose, our preliminary evidence that it does so, frustrating attempts to understand why it <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/update-on-the-huntingtin-par-binding-story/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/09/Blog-post-18-image-300x146.png" alt="" width="679" height="331" class="alignnone wp-image-3582" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/09/Blog-post-18-image-300x146.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/09/Blog-post-18-image-768x374.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/09/Blog-post-18-image-1024x499.png 1024w" sizes="(max-width: 679px) 100vw, 679px" /></p>
<p>One of the major findings of the HDSA-funded project to find huntingtin interactors relevant DNA repair was that many proteins that interact with huntingtin are modified by poly ADP ribose (PAR). Previous posts have described our <a href="https://raytruantlab.ca/2018/07/23/a-closer-look-at-the-hits-generates-2-new-hypotheses/">hypothesis that huntingtin binds poly ADP ribose,</a> our <a href="https://raytruantlab.ca/2018/09/07/is-huntingtin-a-par-binding-protein/">preliminary evidence that it does so</a>, <a href="https://raytruantlab.ca/2019/04/01/why-the-heck-does-huntingtin-bind-par/">frustrating attempts to understand why it does so</a>, and <a href="https://raytruantlab.ca/2019/07/15/huntingtin-chromatin-retention-in-parp-knockout-cells/">other confounding results</a>.</p>
<p>I wish this post was about a major breakthrough clarifying everything… but it’s just another normal, incremental piece of the puzzle (aka: science).</p>
<p>The full story and links to raw data can be found <a href="https://raytruantlab.ca/2019/09/24/update-on-the-huntingtin-par-binding-story/">HERE</a>.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/update-on-the-huntingtin-par-binding-story/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Deposition of structure of ALK2 with M4K2117</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/deposition-of-structure-of-alk2-with-m4k2117/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/deposition-of-structure-of-alk2-with-m4k2117/#respond</comments>
		<pubDate>Wed, 25 Sep 2019 14:19:55 +0000</pubDate>
		<dc:creator><![CDATA[Ellie Williams]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Ellie Williams]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>
		<category><![CDATA[SGC Univ. of Oxford]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=3575</guid>
		<description><![CDATA[One of the big aims of the M4K project was to look at the structure of potentially interesting molecules bound to the protein ALK2 and use that to try and design better molecules that might be useful in the treatment of the rare childhood brain cancer, DIPG. Ros did a lot of work on this <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/deposition-of-structure-of-alk2-with-m4k2117/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>One of the big aims of the M4K project was to look at the structure of potentially interesting molecules bound to the protein ALK2 and use that to try and design better molecules that might be useful in the treatment of the rare childhood brain cancer, DIPG. Ros did a lot of work on this project and documented a lot of it on her blog. Although currently this project has moved beyond the structure stage, there are still quite a lot of structures that need the final step of tidying up and then depositing into the PDB.</p>
<p>In the last couple of weeks among the other things I’ve needed to do, I’ve also managed to deposit the first of these structure – ALK2 bound to M4K2117.</p>
<p>This compound showed good potency as has been tested by Jong Fu and is one of the more promising compounds (of which there are around 5) with regards to both potency and selectivity for taking forward to the next stage of the process for more in-depth and extensive testing in both cell and animal models.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/09/M4K2117-1024x677.jpg" alt="" width="780" height="516" class="alignnone wp-image-3577 size-large" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/09/M4K2117-1024x677.jpg 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/09/M4K2117-300x198.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/09/M4K2117-768x508.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/09/M4K2117.jpg 1368w" sizes="(max-width: 780px) 100vw, 780px" /></p>
<p><em>A: The overall structure of ALK2 bound to M4K2117. B: The structure of the ligand M4K2117. C: A close up of the binding pocket of ALK2 bound to M4K2117 showing a standard H bond between the Nitrogen on the central ring and the backbone hinge region as well as a water mediated H bond between a methoxy group on the compound and a side chain at the back of the pocket of ALK2.</em></p>
<p>The final tweaks to the structures were made using Phenix Refine and Coot while I used the standard wwPDB One Dep system and this structure has been assigned the code: <a href="https://www.rcsb.org/structure/6SRH">6SRH</a></p>
<p>The files for the deposition can be found <a href="https://doi.org/10.5281/zenodo.3460724">over on Zenodo</a> and of course you can download the <a href="https://www.rcsb.org/structure/6SRH">PDB from the PDB website</a>.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/deposition-of-structure-of-alk2-with-m4k2117/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>More GS loop mutations – purification of ALK2 proteins.</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/more-gs-loop-mutations-purification-of-alk2-proteins/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/more-gs-loop-mutations-purification-of-alk2-proteins/#respond</comments>
		<pubDate>Fri, 20 Sep 2019 13:18:22 +0000</pubDate>
		<dc:creator><![CDATA[Ellie Williams]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Ellie Williams]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>
		<category><![CDATA[SGC Univ. of Oxford]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=3527</guid>
		<description><![CDATA[I’ve been getting my stocks in order recently and catching up on some other work which has been both busy but not the most thrilling in terms of experimental results. As such I’ve not had much time to write up a blog post so I’ll try and get back on top of that although it’s <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/more-gs-loop-mutations-purification-of-alk2-proteins/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>I’ve been getting my stocks in order recently and catching up on some other work which has been both busy but not the most thrilling in terms of experimental results. As such I’ve not had much time to write up a blog post so I’ll try and get back on top of that although it’s not likely to be that world shattering.</p>
<p>I’m still trying to purify versions of ALK2 where various phosphorylation sites at the top of the molecule have been mutated out so that we can see what influence each of the phospho-sites has on the overall activity of ALK2 and hopefully from that work out which sites are essential for ALK2’s function and which are merely ‘nice to haves’ or even not necessary at all. This will hopefully then combine with our study of the mutant version of ALK2 to try and see whether the mutation that causes FOP is interfering with this phosphorylation activation. Anyway – that’s the ultimate goal, in the mean time I’ve been slowly purifying more constructs so that I can do a big experiment comparing all the different constructs at the same time. The main reason I want to do them all at once is to try and minimise the variability that can occur between experimental runs despite best efforts to make everything identical. If I’m actually using the same stock of partner proteins and ATP and metal ions and incubating them for exactly the same time it makes the comparisons between the mutants easier and less likely to contain artefact errors.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/gsloop_mutations-1024x425.png" alt="" width="780" height="324" class="alignnone size-large wp-image-3300" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/gsloop_mutations-1024x425.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/gsloop_mutations-300x124.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/gsloop_mutations-768x318.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/gsloop_mutations.png 1165w" sizes="(max-width: 780px) 100vw, 780px" /></p>
<p><em>Figure 1: The different ACVR1 (ALK2) GS loop phosphomutants generated and their associated construct id&#8217;s. The top row shows the sequence of the wild type protein. Mutations are shown in red.</em></p>
<p>So – with that I’ve been stockpiling the various mutants as they come through the expression pipeline at the SGC. This time around I’ve been purifying three more constructs – ACVR1-105, 106 and 109 (constructs 107 and 108 and beyond are still to be sorted). <em>(Just a quick reminder that ALK2 and ACVR1 are different ways of referring to the same thing, technically ALK2 is the protein and ACVR1 is the gene)</em></p>
<p>I had a bit of a problem with some of them – I’m not sure why but they seem to require more Nickle resin than usual to purify compared to the wild type construct – I’m not sure if that’s because the mutation makes it harder for the tag to contact the resin or if it is because there is more background contamination than usual in this particular line of cells or whether it’s something else. Luckily they can be rescued through re-incubating the sample with more nickle to pull out the protein and continuing on as usual. This is, while annoying, not a complete loss so while I’m keen to find out why it’s behaving differently it’s not ruined my purification prep.</p>
<p>There are still a couple of constructs left to purify before I run the big experiment and there are a few other small optimisations I need to do as well but I’m getting closer to really getting to the bottom of this question.</p>
<p>If you want to read more about the exact purification process then please have a look over on <a href="https://doi.org/10.5281/zenodo.3453086">Zenodo</a>.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/more-gs-loop-mutations-purification-of-alk2-proteins/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>2D or not 2D: 3D is the answer</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/2d-or-not-2d-3d-is-the-answer/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/2d-or-not-2d-3d-is-the-answer/#comments</comments>
		<pubDate>Tue, 17 Sep 2019 12:27:36 +0000</pubDate>
		<dc:creator><![CDATA[Thomas Durcan]]></dc:creator>
				<category><![CDATA[ALS-Reproducible Antibody Platform (ALS-RAP)]]></category>
		<category><![CDATA[Amyotrophic Lateral Sclerosis (ALS)]]></category>
		<category><![CDATA[iPSC Disease modelling]]></category>
		<category><![CDATA[iPSCs]]></category>
		<category><![CDATA[Montreal Neurological Institute]]></category>
		<category><![CDATA[Parkinson's disease and neurodevelopmental disorders]]></category>
		<category><![CDATA[Thomas M. Durcan]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=3483</guid>
		<description><![CDATA[Well the summer has come and gone and after 9 blogs, I am ready for the big finale- Blog number 10. I did tell you when I started I was going to give you ten blogs over 2 months with multiple protocols to be published  online and now we are at the end. So, what shall we talk <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/2d-or-not-2d-3d-is-the-answer/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Well the summer has come and gone and after 9 blogs, I am ready for the big finale- <strong>Blog number 10</strong>. I did tell you when I started I was going to give you <a href="https://thesgc.github.io/static-openlabnotebooks/the-summer-notebook-of-a-principal-investigator/">ten blogs</a> <span style="font-size: 1.5rem;">over 2 months </span><span style="font-size: 1.5rem;">with multiple <a href="https://theconversation.com/to-cure-brain-diseases-neuroscientists-must-collaborate-thats-why-im-giving-my-data-away-118672">protocols</a> to be published  online and now we are at the end. So, what shall we talk about today, you want to know about my love of teenage mutant turtles, maybe my enjoyment of <a href="https://en.wikipedia.org/wiki/Dino-Riders">dinoriders</a> (look it up, it’s a classic dinosaur cartoon series from the 80s). Nope, today I will go out on a high, and tell you about one of my true passions. The Pamplemousse, otherwise affectionately known as the grapefruit. And not just any grapefruit, but the ruby red Pamplemousse. </span></p>
<p><span style="font-size: 1.5rem;"><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/09/grapefruit-300x200.jpg" alt="" width="300" height="200" class="size-medium wp-image-3484 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/09/grapefruit-300x200.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/09/grapefruit-768x512.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/09/grapefruit.jpg 910w" sizes="(max-width: 300px) 100vw, 300px" />For 17 years, I eat my lunch like clockwork, every day after noon and the Pamplemousse is my midday fruit (I am a man of routine). To me, i</span>t&#8217;s<span style="font-size: 1.5rem;"> soooo much better than an orange, and it has everything you want in a fruit, people have debated the merits of an orange with me, but I just don’t get the same joy. Fruitiness, sweetness, loveliness, all in one fruit. And it can be eaten all year long, not just summer. So, there you have it, you want to get on Tom’s good side, eat a Pamplemousse with him, and also bring some chocolate for me to consume.</span></p>
<p>Now, what does it have to do with the science topic of the day, well nothing in fact, except its 3D in shape and is great, just like 3D neuronal organoids, or what some refer to as “Minibrains”. For years we worked in flat two dimensions  with iPSC-derived neurons (and we still do). But, it was really only after <a href="https://www.mcgill.ca/neuro/category/article-categories/neuroxxceptional-stories/nguyen-vi-mohamed">Nguyen-Vi Mohamed</a> joined that we got some technicolor and added an extra dimension to our lives. She says, Tom lets put a shaker in an incubator, and see if we can make brain organoids. I say sure, why not, lets try it, and two years, 10,000 organoids later we have truly become immersed in all things 3D neuronal organoid. Yes, we are still pretty new, but we have some cool things going on, and we are happy to share how we make our organoids, to share histology slides if needed for testing antibodies, and have published our <a href="https://mniopenresearch.s3.amazonaws.com/manuscripts/13879/e6921813-a81d-4d1e-ade9-907ee5bbf470_12816_-_nguyen-vi_mohamed.pdf?doi=10.12688/mniopenres.12816.1&amp;numberOfBrowsableCollections=0&amp;numberOfBrowsableInstitutionalCollections=0&amp;numberOfBrowsableGateways=0">SOP</a> for everyone to see the methods we adapted in our group. In particular, as Parkinson’s disease is one of our big focus areas, the first organoids we made were midbrain organoids, organoids that are enriched in dopaminergic neurons.</p>
<p>So, you sit there, and think, man its must be hard to make these organoids, well you know something, it is, it really is&#8230;..</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p>I kid I kid. Anyone can make them, all you need is an incubator and a shaker inside the incubator, and a little sprinkle of magic knowhow. <img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/09/shaker-213x300.jpg" alt="" width="213" height="300" class="size-medium wp-image-3486 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/09/shaker-213x300.jpg 213w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/09/shaker.jpg 357w" sizes="(max-width: 213px) 100vw, 213px" /></p>
<p>You want low speed constant shaking, to ensure good oxygenation and nutrient distribution, keep the little organoids happy. These minibrains grow to 4mm in size, no more, sometimes less depending on how they form at the start.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/09/organoid-image-300x212.png" alt="" width="300" height="212" class="size-medium wp-image-3487 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/09/organoid-image-300x212.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/09/organoid-image.png 317w" sizes="(max-width: 300px) 100vw, 300px" /></p>
<p>And the first step is the most important, which is the iPSCs you use. Go back to my earlier blog, and you will see my points on <a href="https://thesgc.github.io/static-openlabnotebooks/the-never-ending-roundabout-of-ipsc-quality-control/">iPSC quality control</a>, this is crucial here, you need the best of the best cell-wise, otherwise your organoids will have problems and may not grow normally. This can manifest in the very earliest stages when you create a single cell mix of iPSCs to put in a low attachment U-bottom dish where it forms an embryoid body (EB). This is the seed for the minibrains, and you want it uniform and spherical (left image). But sometimes, things go awry and you get extra buds, extra pieces (right image- like mickey mouse) and this can all lead to organoids that vary in size and shape.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/09/embyoid-body-300x225.jpg" alt="" width="300" height="225" class="size-medium wp-image-3488 alignleft" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/09/embyoid-body-300x225.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/09/embyoid-body-350x263.jpg 350w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/09/embyoid-body-250x188.jpg 250w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/09/embyoid-body.jpg 352w" sizes="(max-width: 300px) 100vw, 300px" /><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/09/mickey-mouse-minibrain-300x225.jpg" alt="" width="300" height="225" class="size-medium wp-image-3489 alignright" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/09/mickey-mouse-minibrain-300x225.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/09/mickey-mouse-minibrain-350x263.jpg 350w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/09/mickey-mouse-minibrain-250x188.jpg 250w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/09/mickey-mouse-minibrain.jpg 352w" sizes="(max-width: 300px) 100vw, 300px" /></p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p>Another crucial step for making the midbrain and cerebral organoids for us is <span style="font-size: 1.5rem;">embedding in </span><span style="font-size: 1.5rem;">matrigel. We encase these organoids in this gelatinous structure, for 1 week once EBs are formed, and then transfer into the shaker. And from this point on, its all about looking, and nurturing. During a week, we do three media changes to ensure the nutrients are replenished, and these organoids can be maintained for as long you want. Some batches we grow for 15 days, others 30 days, 3 months, 6 months and our oldest was 18 months. Other groups have gone as high as 4 years, so imagine being a graduate student and waiting on your organoids to be ready in 48 months.</span></p>
<p>Organoids can be used for all manner of things, from being dissociated and analyzed by flow cytometry, to being lysed for proteomic analysis, to being fixed, embedded and sectioned for imaging studies. We have embedded in either parafilm or OCT and sectioned these 3D structures on a microtome or cryostat respectively. This is tricky to do, but it simply requires some practice, as organoid tissue is different to mouse brain tissue, as its much softer and less rigid. But when you get it right, you get some great images of the neurons in the neuronal organoids themselves (Midbrain organoid-Left image; Cerebral organoid- Right image)</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/09/minibrain-section-300x244.jpg" alt="" width="300" height="244" class="size-medium wp-image-3490 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/09/minibrain-section-300x244.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/09/minibrain-section-768x624.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/09/minibrain-section-1024x832.jpg 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/09/minibrain-section.jpg 1281w" sizes="(max-width: 300px) 100vw, 300px" /></p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/09/Cortical-MN-section-300x285.png" alt="" width="300" height="285" class="size-medium wp-image-3491 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/09/Cortical-MN-section-300x285.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/09/Cortical-MN-section.png 306w" sizes="(max-width: 300px) 100vw, 300px" /></p>
<p>Neuronal organoids are new, but are here to stay, and to date us and others have barely scratched the surface of their potential, which makes it even more exciting to see what we can learn from them about how neurons develop, how the brain might function and how a disease might occur.</p>
<p>Well I hope you enjoyed all the blogs, it was my way of discussing science and doing it with humor. And I hope I shed a light on what we are doing in my group. What started as a small rag tag group of three people in the iPSC/CRISPR platform, has now transformed into what we now call <a href="https://www.mcgill.ca/neuro/open-science/open-science-platforms/eddu">The Neuro’s Early Drug Discovery Unit (EDDU)</a>. We are still the same group as we were yesterday, but the new name reflects the diversity of activities we have ongoing and the exciting work we are undertaking as we look to do the early discovery work with both academics and industry in the quest for new drugs and therapies. Developing new therapies for CNS disorders is challenging and we hope to contribute something to finding new drugs that might one day soon enter the clinic.</p>
<p>I have one of the most committed and best bunch of people anyone could work with, and while this blog helped introduce you to me, I hope you will check out our new <a href="https://www.mcgill.ca/neuro/open-science/open-science-platforms/eddu">website</a>, where you will be introduced to each and every one of them. They are the reason I get to write my blogs, and who do all the cool science I tell you all about. So, come check out the new <a href="https://www.mcgill.ca/neuro/open-science/open-science-platforms/eddu">website</a>, learn more about us:</p>
<ul>
<li>Meet the <a href="https://www.mcgill.ca/neuro/open-science/open-science-platforms/eddu/meet-eddu-team">team</a> and learn what makes them tick</li>
<li>Find all our <a href="https://www.mcgill.ca/neuro/open-science/open-science-platforms/eddu/resources-researchers#OPEN%20SCIENCE%20TOOLS">SOPs and publications</a> we have published, and keep an eye out for new SOPs, publications, and software that will be added over the next year</li>
<li>Learn about the facilities we have to help you with your projects, including our automation platform and our FACs facility</li>
<li>Hear about our <a href="https://www.mcgill.ca/neuro/open-science/open-science-platforms/eddu/about-eddu">history.</a></li>
<li>Get involved with our group, if you think we can help you, please <a href="neuroeddu.mni@mcgill.ca">contact us</a>, the blogs and SOPs give you a flavor of what we have to offer, but we are happy to discuss all kinds of projects and ways to help researchers and others like yourself.</li>
</ul>
<p>I also want to say a special thanks to a few people for helping to build this website: Iain, Vanessa, Trisha, Anthony, Lenore, Kristiana and her team. They built it from the ground up and dealt with me on a daily basis, which is not an easy task. I am quite curmudgeonly. So thank you for all the hard work and effort you put into this.</p>
<p>Well, I hope you enjoyed reading my <span style="font-size: 1.5rem;">summer blog</span><span style="font-size: 1.5rem;"> as much as I enjoyed writing it. But fear not, this is not my last post, there will be more, just not every week, likely every 2-4 weeks. Further posts are planned in October when I will be putting our two posters online that we are presenting at the Society for Neuroscience (SFN) meeting. So stay tuned for more.</span></p>
<p>Don’t be a stranger, grab a Pamplemousse and come say hi, I will even teach you my secret technique for peeling it within 60 seconds, this is not something I put in a blog or SOP, top secret stuff.</p>
<p>&nbsp;</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/2d-or-not-2d-3d-is-the-answer/feed/</wfw:commentRss>
		<slash:comments>2</slash:comments>
		</item>
		<item>
		<title>All great things come in threes, even quality control of synuclein fibrils</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/all-great-things-come-in-threes-even-quality-control-of-synuclein-fibrils/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/all-great-things-come-in-threes-even-quality-control-of-synuclein-fibrils/#respond</comments>
		<pubDate>Wed, 04 Sep 2019 17:45:07 +0000</pubDate>
		<dc:creator><![CDATA[Thomas Durcan]]></dc:creator>
				<category><![CDATA[ALS-Reproducible Antibody Platform (ALS-RAP)]]></category>
		<category><![CDATA[Amyotrophic Lateral Sclerosis (ALS)]]></category>
		<category><![CDATA[iPSC Disease modelling]]></category>
		<category><![CDATA[iPSCs]]></category>
		<category><![CDATA[Montreal Neurological Institute]]></category>
		<category><![CDATA[Parkinson's disease and neurodevelopmental disorders]]></category>
		<category><![CDATA[Thomas M. Durcan]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=3421</guid>
		<description><![CDATA[Someone pointed out a funny quirk of mine the other day. I love things that come in threes. I never realized it before, but then I started thinking about it, and before you could say thirty-three, I was contemplating all the threes I like. Maybe its because the house I lived in growing up was <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/all-great-things-come-in-threes-even-quality-control-of-synuclein-fibrils/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Someone pointed out a funny quirk of mine the other day. I love things that come in threes. I never realized it before, but then I started thinking about it, and before you could say thirty-three, I was contemplating all the threes I like. Maybe its because the house I lived in growing up was 33 (a number to this day I have issues pronouncing, don&#8217;t ask), but I love things that come in threes, two always feels like too little, and four or more seems too much. When I go to McDonalds, I love the burger-fries-drinks combo; when I go to the movies I love the popcorn-candy-Drink combo, at restaurants its steak-fries-red wine. For breakfast I love tea-cornflakes-milk (I know reading this you think, man he is one exciting guy). My favorite dinosaur is the Velociraptor, with three claws per hand. Even when I write papers, I love titles with threes, if you want proof, check out my <a href="http://genesdev.cshlp.org/content/29/10/989">3P paper</a>. My favorite team has a world class top three. But its not just me, threes are everywhere, traffic lights come with three lights, movie series come in trilogies, hockey games have three periods, poutine has three ingredients (<em><strong>Can you name them?</strong></em>), the list goes on and on. So, three really is a magic number for me, which is why today’s protocol is based on three simple quality control steps for testing synuclein aggregates.</p>
<p>In my <a href="https://thesgc.github.io/static-openlabnotebooks/the-making-of-alpha-synuclein-a-villain-of-parkinsons-disease/">last entry</a>, I spoke about making the synuclein protein, but today I will take it one step further and explain how once we aggregate the synuclein into fibrils, we do <a href="https://zenodo.org/record/3385345#.XW87dShKg2w">“<em><strong>3</strong></em>” simple checks</a> to make sure its good to use in our experiments.</p>
<p>To make these aggregated fibril like structures, all we do is add synuclein into a tube and shake for 5-7 days. Once you are done, you have these long straw-like structures. And these are your fibrils. But they are way too long making them cumbersome to work with. So for our experiments, we like nice small, uniform fibrils, which is why we sonicate them. In a waterbath, we introduce a pulse-like vibration, that over 20 minutes breaks them into small pieces all of the same size. To come up with an analogy, if you ever had candy cane, smashing it into a hundred pieces to break it up, is like sonicating a synuclein fibril. This is pretty much sonication in a nutshell.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/09/synuclein-fibrils-300x152.png" alt="" width="444" height="225" class=" wp-image-3422 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/09/synuclein-fibrils-300x152.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/09/synuclein-fibrils-768x388.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/09/synuclein-fibrils.png 892w" sizes="(max-width: 444px) 100vw, 444px" /></p>
<p>Once this is done, you now have <a href="https://zenodo.org/record/3385345#.XW87dShKg2w">three simple steps</a> to make sure your fibrils are good.</p>
<p><span style="text-decoration: underline;"><strong>Step 1: Electron microscopy</strong></span>. Take some fibrils, both broken (sonicated) and not broken (unsonicated), and look at them under the electron microscope. With these images, you can measure the size, and determine whether they look good or not (Thanks to Dr. Esther Del Cild for the great EM images and quantification below). If something looks off, toss it out. This can mean inconsistent sizes, or fibrils are too long, or too small, or maybe you can’t even see them. Either way you have a problem with the batch you made.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/09/EM_example-300x215.jpg" alt="" width="430" height="308" class=" wp-image-3423 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/09/EM_example-300x215.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/09/EM_example-768x549.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/09/EM_example-1024x732.jpg 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/09/EM_example.jpg 1071w" sizes="(max-width: 430px) 100vw, 430px" /></p>
<p><span style="text-decoration: underline;"><strong>Step 2: Dynamic Light scattering</strong></span>: This is a cuvette based system we have (explained in our <a href="https://zenodo.org/record/3385345#.XW_2BShKiUl">protocol</a>), that shines a light on our suspension of fibrils. Based on the way the light hits, we can measure the size and distribution of fibrils in our batches. If we see large variabilities, then we toss the batch, as we are looking for a batch as uniform in size as possible.</p>
<p><span style="text-decoration: underline;"><strong>Step 3: Cellular uptake</strong></span>: if we are good in the first two steps, the last step is to check if the fibrils can get into cells. To do this, we add a fluorescent label (Alexa fluorescent dye) onto the fibrils, which provides us a quick and easy way to visualize the fibrils within the cells. Once we add the fibrils into the media the cells are grown in, they are rapidly taken up and after 24h we can image them in the cells. Below is an image of some fluorescent fibrils in Hela cells after only a few hours in the presence of cells. It was in the cell, as when cells are acid washed to remove any protein binding on its surface, the signal still remains intact, meaning the fibrils got into the cells.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/09/IF_HeLa_PFF-633.jpg" alt="" width="231" height="231" class=" wp-image-3425 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/09/IF_HeLa_PFF-633.jpg 200w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/09/IF_HeLa_PFF-633-150x150.jpg 150w" sizes="(max-width: 231px) 100vw, 231px" /></p>
<p>Once we quantify the rate of uptake in Hela or other cells being used, and are happy with the size, shape and uptake of our fibrils, then we are ready to use them in our studies. And you can do, we make synuclein and synuclein PFFs for our projects, but also provide to other users, if your interested in some for your experiments, let me <a href="https://www.linkedin.com/in/thomas-durcan-18593b23/">know</a> and we can discuss how to help.</p>
<p>Well I hope you enjoying learning a little about my quirks and fascination with things that come in threes, next time we will learn about my favorite letter in my final blog of the summer. Next week is blog #10 and the end of my summer series, when I will cover one last protocol and fill you in on what’s coming next for me and my group.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/all-great-things-come-in-threes-even-quality-control-of-synuclein-fibrils/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Evaluating the effect of 8 M4K compounds on the viability of DIPG patient-derived cells (SU-DIPG-VI, HSJD-DIPG-007 and SU-DIPG-IV)</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/evaluating-the-effect-of-8-m4k-compounds-on-the-viability-of-dipg-patient-derived-cells-su-dipg-vi-hsjd-dipg-007-and-su-dipg-iv/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/evaluating-the-effect-of-8-m4k-compounds-on-the-viability-of-dipg-patient-derived-cells-su-dipg-vi-hsjd-dipg-007-and-su-dipg-iv/#respond</comments>
		<pubDate>Wed, 28 Aug 2019 16:48:39 +0000</pubDate>
		<dc:creator><![CDATA[Jong Fu]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Brain Tumours]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Jong Fu Wong]]></category>
		<category><![CDATA[SGC Univ. of Oxford]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=3399</guid>
		<description><![CDATA[Background: I have been characterising the cellular potency of many M4K compounds against ALK2 and ALK5. Compounds that targets ALK2 and not ALK5 should be further evaluated for their ability to reduce the viability of cells derived from DIPG patients. Although these cultured cells might not reflect what actually happens in a DIPG patient&#8217;s brain, <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/evaluating-the-effect-of-8-m4k-compounds-on-the-viability-of-dipg-patient-derived-cells-su-dipg-vi-hsjd-dipg-007-and-su-dipg-iv/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p><strong>Background:</strong></p>
<p>I have been characterising the cellular potency of many M4K compounds against ALK2 and ALK5. Compounds that targets ALK2 and not ALK5 should be further evaluated for their ability to reduce the viability of cells derived from DIPG patients. Although these cultured cells might not reflect what actually happens in a DIPG patient&#8217;s brain, it is the the best tool we have on hand.</p>
<p>Viability assay using Celltiter Glo kit has been chosen as the main method for evaluating the efficacy of ALK2 inhibitors on DIPG patient-derived cells due to its robustness (refer to my previous post for how I determined this). As a pilot experiment, 8 ALK2 inhibitors with differing potencies were chosen and evaluated in 3 DIPG patient-derived cell lines (one with wild-type ALK2 and two with mutant ALK2).</p>
<p><strong>Experimental design:</strong></p>
<p>DIPG cells will be seeded into wells coated with laminin at densities optimised in previous experiments. After allowing the cells to adhere overnight, they will be treated with serial dilutions of various M4K compounds. After 7 days of compound treatment, the area occupied by the cells, the intensity of propidium iodide signal and Celltiter Glo luminescence signal will be determined.</p>
<div id="attachment_3400" style="width: 605px" class="wp-caption alignnone"><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/Figure-1.png" alt="" width="595" height="221" class=" wp-image-3400" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/Figure-1.png 1161w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/Figure-1-300x111.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/Figure-1-768x285.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/Figure-1-1024x380.png 1024w" sizes="(max-width: 595px) 100vw, 595px" /><p class="wp-caption-text">The timeline of this experiment</p></div>
<p><strong>Results:</strong></p>
<div id="attachment_3401" style="width: 949px" class="wp-caption alignnone"><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/Figure-2.png" alt="" width="939" height="508" class="size-full wp-image-3401" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/Figure-2.png 939w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/Figure-2-300x162.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/Figure-2-768x415.png 768w" sizes="(max-width: 939px) 100vw, 939px" /><p class="wp-caption-text">Chemical structures of the compounds tested in this experiment as well as the reasons they were selected (top table). The histone H3 and ACVR1/ALK2 mutation status of the different DIPG patient-derived cell lines is shown in bottom table.</p></div>
<div id="attachment_3402" style="width: 902px" class="wp-caption alignnone"><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/Figure-3.png" alt="" width="892" height="720" class="size-full wp-image-3402" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/Figure-3.png 892w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/Figure-3-300x242.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/Figure-3-768x620.png 768w" sizes="(max-width: 892px) 100vw, 892px" /><p class="wp-caption-text">Efficacy of the compounds assayed compounds. EC50, 95% confidence interval of the EC50 values and maximal observed reduction in viability are shown for each compounds.</p></div>
<div id="attachment_3403" style="width: 936px" class="wp-caption alignnone"><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/Figure-4.png" alt="" width="926" height="638" class="size-full wp-image-3403" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/Figure-4.png 926w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/Figure-4-300x207.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/Figure-4-768x529.png 768w" sizes="(max-width: 926px) 100vw, 926px" /><p class="wp-caption-text">EC50 curves for SU-DIPG-VI cells treated with dilution series of each compound for 7 days.</p></div>
<div id="attachment_3404" style="width: 950px" class="wp-caption alignnone"><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/Figure-5.png" alt="" width="940" height="638" class="size-full wp-image-3404" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/Figure-5.png 940w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/Figure-5-300x204.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/Figure-5-768x521.png 768w" sizes="(max-width: 940px) 100vw, 940px" /><p class="wp-caption-text">EC50 curves for HSJD-DIPG-007 cells treated with dilution series of each compound for 7 days.</p></div>
<div id="attachment_3405" style="width: 951px" class="wp-caption alignnone"><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/Figure-6.png" alt="" width="941" height="633" class="size-full wp-image-3405" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/Figure-6.png 941w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/Figure-6-300x202.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/Figure-6-768x517.png 768w" sizes="(max-width: 941px) 100vw, 941px" /><p class="wp-caption-text">EC50 curves for SU-DIPG-IV cells treated with dilution series of each compound for 7 days.</p></div>
<p><strong>Findings:</strong></p>
<p>1) HSJD-DIPG-007 was sensitive toward most of the M4K compounds.</p>
<p>2) SU-DIPG-IV was not sensitive to most of the compounds.</p>
<p>3) M4K3003 and M4K3007 which were based on patent literature were not efficacious despite having decent potency towards ALK2.</p>
<p>4) M4K2009 was reduced the viability of DIPG lines with ALK2 mutation while M4K2163 showed efficacy in all of the DIPG cell lines.</p>
<p><strong>In a nut shell:</strong></p>
<p>More compounds still needs to be evaluated in more DIPG patient-derived cell lines in order to better establish relationship between ALK2 IC50, compound chemical structure and efficacy in reducing the viability of DIPG cells.</p>
<p>For detailed description of my experimental protocol, please refer to my <a href="https://zenodo.org/record/3380446#.XWauDeNKiUk">Zenodo page</a>.</p>
<p>&nbsp;</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/evaluating-the-effect-of-8-m4k-compounds-on-the-viability-of-dipg-patient-derived-cells-su-dipg-vi-hsjd-dipg-007-and-su-dipg-iv/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>More Purifications</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/final-purifications/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/final-purifications/#respond</comments>
		<pubDate>Wed, 28 Aug 2019 15:56:46 +0000</pubDate>
		<dc:creator><![CDATA[Jacob McAuley]]></dc:creator>
				<category><![CDATA[Huntingtin]]></category>
		<category><![CDATA[Huntington’s disease]]></category>
		<category><![CDATA[Jacob McAuley]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=3393</guid>
		<description><![CDATA[Back again. Two posts in one month! I know its crazy, but I’m just excited to tell you about the progress that is being made on this project. First things first, purifications. This week I have four more to tell you about. The first is GFP, and unlike all of my other posts, this is <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/final-purifications/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Back again. Two posts in one month! I know its crazy, but I’m just excited to tell you about the progress that is being made on this project. First things first, purifications. This week I have four more to tell you about.</p>
<p>The first is <a href="https://zenodo.org/record/3380341#.XWakIehKjcs">GFP</a>, and unlike all of my other posts, this is not because it interacts with HTT. At the moment we are trying to over express FLAG-Tagged HTT in a neuronal cell line. Growing tagged HTT is usually done in HEK293 cells although this isn&#8217;t a great cell line for physiological studies of HTT due to its improper localization. Neurons are quite a bit more sensitive, so extra care has to be taken to properly over-express this protein. Using conventional methods of over expression don’t really work due to the physical characteristics of tagged HTT, mainly its size. To get the protein into the cell we are considering following the example of Lambert, H.<em> et. al.(</em> <a href="https://doi.org/10.1139/o90-105"><em>https://doi.org/10.1139/o90-105</em></a><em>). </em>Described in this paper, is a method of electroporating cells to insert protein into the cell instead of DNA to express said protein. The GFP will be used as an optimization tool to ensure our electroporation method can successfully insert a protein into a neuronal cell.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/GFPPP-300x260.png" alt="" width="300" height="260" class="size-medium wp-image-3394 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/GFPPP-300x260.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/GFPPP.png 323w" sizes="(max-width: 300px) 100vw, 300px" /></p>
<p>&nbsp;</p>
<p>The other proteins that were purified pertained to my main project. They were P97, PIAS1, GIT1 and SP1. <a href="https://zenodo.org/record/3367290#.XWaj9OhKjcs">P97</a> was purified successfully resulting in a final yield of approximately 3 mg. <a href="https://zenodo.org/record/3380355#.XWajz-hKjcs">PIAS1</a> was unable to be successfully purified. Previously a shorter construct of PIAS1 was expressed in E. coli BL21 codon plus RIL but now we attempted an SF9 construct of full length PIAS1, although it seemed to have very poor expression in this system. GIT1 and SP1 were also unsuccessfully purified. As always, more detailed methods of purification can be found on Zenodo, any of the hyperlinks of the proteins will bring you to the respective Zenodo post.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/final-purifications/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Development of mouse line with Grin1 variant Y647S</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/development-of-mouse-line-with-grin1-variant-y647s/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/development-of-mouse-line-with-grin1-variant-y647s/#respond</comments>
		<pubDate>Tue, 27 Aug 2019 17:29:02 +0000</pubDate>
		<dc:creator><![CDATA[Amy Ramsey]]></dc:creator>
				<category><![CDATA[Amy Ramsey]]></category>
		<category><![CDATA[Amy Ramsey (Univ. of Toronto)]]></category>
		<category><![CDATA[GRIN disorder]]></category>
		<category><![CDATA[NMDA]]></category>
		<category><![CDATA[Rare disease]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=3388</guid>
		<description><![CDATA[This is a test to figure out how we add links to zenodo lab notebooks, which are time stamped and have a DOI. I uploaded an image of our Sanger sequence that shows the introduction of base pair changes to cause an amino acid substitution Y647S. https://zenodo.org/record/3379009#.XWVmfpNKhp8]]></description>
				<content:encoded><![CDATA[<p>This is a test to figure out how we add links to zenodo lab notebooks, which are time stamped and have a DOI.</p>
<p>I uploaded an image of our Sanger sequence that shows the introduction of base pair changes to cause an amino acid substitution Y647S.</p>
<p><a href="https://zenodo.org/record/3379009#.XWVmfpNKhp8">https://zenodo.org/record/3379009#.XWVmfpNKhp8</a></p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/development-of-mouse-line-with-grin1-variant-y647s/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Open Science for GRIN</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/open-science-for-grin/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/open-science-for-grin/#respond</comments>
		<pubDate>Fri, 23 Aug 2019 02:51:04 +0000</pubDate>
		<dc:creator><![CDATA[Amy Ramsey]]></dc:creator>
				<category><![CDATA[Amy Ramsey]]></category>
		<category><![CDATA[Amy Ramsey (Univ. of Toronto)]]></category>
		<category><![CDATA[GRIN disorder]]></category>
		<category><![CDATA[NMDA]]></category>
		<category><![CDATA[Rare disease]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=3381</guid>
		<description><![CDATA[For the Ramsey Lab, this project began in 2018, when we were invited to a parent-organized meeting for caregivers of GRIN1 patients. Our group uses mice to study the NMDA receptor (which is encoded by the GRIN genes) in the context of schizophrenia and autism. It was only in 2018 that we learned of a <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/open-science-for-grin/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>For the Ramsey Lab, this project began in 2018, when we were invited to a parent-organized meeting for caregivers of GRIN1 patients. Our group uses mice to study the NMDA receptor (which is encoded by the GRIN genes) in the context of schizophrenia and autism. It was only in 2018 that we learned of a rare disease caused by de novo mutations in the GRIN1 gene. Then we learned that single base pair mutations have been discovered in all of the seven GRIN genes that encode NMDA receptors. We have since moved our research focus towards helping GRIN patients, and have joined the growing community of scientists and clinicians with this same goal.</p>
<p>What are the features of GRIN disorders? All patients have some level of intellectual disability, but other features of the syndrome are variable, depending on the individual patient and the nature of their specific variant. Many children have low muscle tone that leads to developmental delays in motor skills and difficulties feeding. Many children have seizures that vary in severity from brief and infrequent to life-threatening. Other common symptoms include reduced or absent speech, cortical visual impairment, hyperactive and repetitive movements, and sleep disturbances. In talking with caregivers, it seems clear that these symptoms are the ones where therapeutic interventions could greatly improve the quality of life.</p>
<p>It is an enormous task to care for and raise a child with such special needs. However, the GRIN parents have found each other through online support groups. They are providing helpful information about medicines and about the occupational therapy that enables GRIN patients to gain motor skills. They have also started foundations to help other parents and caregivers, or to provide financial support for research.</p>
<p>The support network that GRIN parents have developed should set the example for GRIN researchers. It is our hope that the cooperation of parents to share information and advance medical care will inspire scientists to build an open science community. If you would like to join the GRIN disorder open science platform to share your research, please email me at a.ramsey@utoronto.ca.</p>
<p>&nbsp;</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/open-science-for-grin/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
	</channel>
</rss>
