<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>Disease &#8211; openlabnotebooks.org</title>
	<atom:link href="https://thesgc.github.io/static-openlabnotebooks/category/disease-foundation/feed/" rel="self" type="application/rss+xml" />
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<description>A growing team of groundbreaking scientists around the world are now sharing their lab notebooks online</description>
	<lastBuildDate>Mon, 01 Jul 2019 16:49:26 +0000</lastBuildDate>
	<language>en-GB</language>
	<sy:updatePeriod>hourly</sy:updatePeriod>
	<sy:updateFrequency>1</sy:updateFrequency>
	<generator>https://wordpress.org/?v=4.9.10</generator>

<image>
	<url>https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/cropped-lab-jar02-32x32.png</url>
	<title>Disease &#8211; openlabnotebooks.org</title>
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<width>32</width>
	<height>32</height>
</image> 
	<item>
		<title>Strippers from outer space: recombinant PNGase F.</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/strippers-from-outer-space-recombinant-pngase-f/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/strippers-from-outer-space-recombinant-pngase-f/#respond</comments>
		<pubDate>Sun, 30 Jun 2019 18:41:29 +0000</pubDate>
		<dc:creator><![CDATA[Pietro Roversi]]></dc:creator>
				<category><![CDATA[Open Secretion Rescue]]></category>
		<category><![CDATA[Pietro Roversi]]></category>
		<category><![CDATA[Rare disease]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=3041</guid>
		<description><![CDATA[Nature has evolved molecules called glycoproteins that are proteins carrying sugar chains (&#8220;glycans&#8221;). Glycans can make proteins more soluble, they can shield them from the environment, they can be used as calling cards to enable protein-protein recognition; in Eukaryotes, a glucose-on/glucose-off quality control system uses glycans to monitor glycoproteins folding. In general, glycans contribute to <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/strippers-from-outer-space-recombinant-pngase-f/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<div id="attachment_3042" style="width: 239px" class="wp-caption alignnone"><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/06/mir.jpg" alt="" class="size-full wp-image-3042" width="229" height="220" /><p class="wp-caption-text">The enzyme PNGase F comes from a bacterium first isolated on the MIR space station.</p></div>
<p>Nature has evolved molecules called glycoproteins that are proteins carrying sugar chains (&#8220;glycans&#8221;). Glycans can make proteins more soluble, they can shield them from the environment, they can be used as calling cards to enable protein-protein recognition; in Eukaryotes, a glucose-on/glucose-off quality control system uses glycans to monitor glycoproteins folding.</p>
<p>In general, glycans contribute to expanding the repertoire of protein shapes and sizes that genomes code for, and sugars on proteins are a great asset to nature. Protein chemists on the other hand are not always happy with the glycans on the proteins they produce. This is especially true of molecular biologists forcing cells to synthesise custom glycoproteins: these &#8216;recombinant glycoproteins&#8217; may end up with glycans that differ from naturally occurring ones.</p>
<p>As it turns out, the ability of stripping sugars from glycoproteins has helped a number of species to survive, and enzymes that can cleave glycans from glycoproteins (&#8220;deglycosylases&#8221;) are encoded in many a genome. By offering such enzymes, nature comes therefore to the succour of molecular biologists and protein chemists who are unhappy with the glycans on the proteins they handle: by treating a glycoprotein with a deglycosylase, it is possible to get rid of the sugars and obtain a sugar-free protein.</p>
<p>In our case, crystals of a glycoprotein we are studying grew a flimsy lattice formed by glycan-glycan contacts, making the resulting glycoprotein structure all fuzzy. We are hoping that one way of improving on these crystals and obtaining a better structure would be to strip the glycoprotein of the glycans altogether, perhaps enabling the growth of different crystals, kept together by more stable, less mobile protein-protein contacts.</p>
<p>For this reason, we decided to treat our glycoprotein with a special deglycosylase called PNGase F. The enzyme belongs to a rare bacterium called <a href="https://en.wikipedia.org/wiki/Elizabethkingia_miricola"><em>Elizabethkingia miricola</em>, </a>first isolated from condensation water in the <a href="https://en.wikipedia.org/wiki/Mir">Space Station Mir</a>. But a small vial of the enzyme can cost up to £1000 when purchased from commercial sources! We therefore endeavoured to produce our own PNGase F, using <em>E.coli</em> bacteria transformed with the <em>Elizabethkingia miricola</em> PNGase F gene.</p>
<p>After ten days of work, <a href="https://zenodo.org/record/3263902#.XRj3jXt7l2Y">we obtained our stripper from outer space</a>, and were able to confirm its activity. Watch this less-than-outer space for the next experiments in which we shall try and grow our crystals again, this time with the protein without glycans!</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/strippers-from-outer-space-recombinant-pngase-f/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>The Year in Review</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/the-year-in-review/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/the-year-in-review/#respond</comments>
		<pubDate>Thu, 27 Jun 2019 16:24:03 +0000</pubDate>
		<dc:creator><![CDATA[Tamara Maiuri]]></dc:creator>
				<category><![CDATA[Huntingtin]]></category>
		<category><![CDATA[Huntington’s disease]]></category>
		<category><![CDATA[Ray Truant]]></category>
		<category><![CDATA[Tamara Maiuri]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=3034</guid>
		<description><![CDATA[A summary of the Berman Topper second year-end progress report, with links to the full report, are now posted on the Truant lab website.]]></description>
				<content:encoded><![CDATA[<p>A summary of the Berman Topper second year-end progress report, with links to the full report, are now posted on the <a href="https://raytruantlab.ca/2019/06/27/the-year-in-review/">Truant lab website</a>.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/06/Screen-Shot-2019-06-27-at-11.38.21-AM-300x110.png" alt="" width="300" height="110" class="alignnone size-medium wp-image-3035" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/06/Screen-Shot-2019-06-27-at-11.38.21-AM-300x110.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/06/Screen-Shot-2019-06-27-at-11.38.21-AM-768x280.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/06/Screen-Shot-2019-06-27-at-11.38.21-AM-1024x374.png 1024w" sizes="(max-width: 300px) 100vw, 300px" /></p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/the-year-in-review/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Selectivity against, and action of, TNIK kinase in relation to ALK2.</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/selectivity-against-and-action-of-tnik-kinase-in-relation-to-alk2/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/selectivity-against-and-action-of-tnik-kinase-in-relation-to-alk2/#respond</comments>
		<pubDate>Wed, 26 Jun 2019 15:58:08 +0000</pubDate>
		<dc:creator><![CDATA[Ellie Williams]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Ellie Williams]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>
		<category><![CDATA[SGC Univ. of Oxford]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=3015</guid>
		<description><![CDATA[Sorry I’ve not written for a while. I had some annual leave to use up and then I’ve been helping out with a residential EMBO course that was being co-organised by one of the group leaders in my department. The course was an introduction to a wide variety of biochemical and biophysical techniques. I was <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/selectivity-against-and-action-of-tnik-kinase-in-relation-to-alk2/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Sorry I’ve not written for a while. I had some annual leave to use up and then I’ve been helping out with a residential EMBO course that was being co-organised by one of the group leaders in my department. The course was an introduction to a wide variety of biochemical and biophysical techniques. I was helping out with a purification comparison between nickel resin, talon (cobalt) resin and streptactin resin and talking a bit about thermal shift assays.</p>
<p>Anyway, on with the science!</p>
<p><strong>TNIK Selectivity:</strong></p>
<p>TNIK is a protein kinase that Ros has previously talked about as being a potential off target hit for many of the M4K compounds along with DDR1. It is highly expressed in the brain which makes it of relevance to drug discovery for DIPG. Although its function is not entirely clear we know over-expression leads to cell rounding and it has been linked to some cell signalling effects in certain cancers. What makes it of most interest to us is that it seems to phosphorylate SMAD1 in an inhibitory position away from the tail where ALK2 phosphorylates SMAD1. Theoretically phosphorylation of SMAD1 by TNIK should inhibit phosphorylation elsewhere on SMAD1 by ALK2. This might make TNIK an indirect inhibitor of ALK2 function and thus we don’t want any drug we design against ALK2 to also hit TNIK.</p>
<p>I was working with a student, Mira Rothweiler, who purified TNIK that we then used in further experiments. Details for the purification can be found on <a href="https://doi.org/10.5281/zenodo.3257646">Zenodo</a> along with other details of the experiments done.</p>
<p>I have used Tm shift analysis (which is where you look at the change in melting point of a protein caused by the binding of a ligand, the tighter it binds the more it stabilises the protein and thus the larger the shift) to look at the binding of TNIK to all the M4K compounds and compared it to the shifts seen in ALK2. I plotted this data against each other with ALK2 shifts along the x-axis and the TNIK shifts along the y-axis. When compared against a 1:1 line, anything below the line is likely to be more selective for ALK2, anything above the line is likely to be selective for TNIK and anything close to the line shows minimal selectivity.</p>
<p>The data shows that while there are a small handful of M4K compounds that do hit TNIK, the vast majority seem to have a relatively low effect on TNIK and a high effect on ALK2 suggesting they are selective. One of the most promising M4K targets (M4K2009) gives a 14.1 °C shift for ALK2 but only 1.8 °C for TNIK. This is really good news.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/06/Tm_shifts_TNIK-816x1024.png" alt="" width="780" height="979" class="alignnone size-large wp-image-3017" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/06/Tm_shifts_TNIK-816x1024.png 816w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/06/Tm_shifts_TNIK-239x300.png 239w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/06/Tm_shifts_TNIK-768x964.png 768w" sizes="(max-width: 780px) 100vw, 780px" /></p>
<p><em>Tm shift results for ALK2 vs. TNIK for all M4K compounds. Top panel: plates K115 and K116. Bottom panel: plates K117 and K118.<br />
</em></p>
<p><strong>TNIK activity:</strong></p>
<p>While it has been reported that TNIK phosphorylates SMAD1 in a non-standard location in vivo, I wanted to see if I could get the same effect in vitro in my own experiments. I mixed SMAD1 with TNIK in different concentrations and for different lengths of time and took samples to run on mass spec to see whether SMAD1 was getting phosphorylated. My results showed that with concentrations of TNIK of 10mM and over after just 10 minutes I was starting to see phosphorylation on SMAD1. However this doesn’t tell me where that phosphorylation is happening.</p>
<p>I used another mass spec technique called ‘phosphomapping’ to identify where the phosphate was being added. Dr. Rod Chalk helped me with this for which I’m very thankful. This technique involved chopping the protein up into known chunks using different proteases that cut the protein at specific predictable places. We used five different proteases on different samples from the same incubation that all cut in different locations and then with each of the samples we ran half of it on the mass spec as was, and with the other half passed it over a Ti/Zr column to enrich the phosphopeptides within it and then ran it on the mass spec. The peptides detected are then run against a database of expected peptides created from knowing both the sequence of the protein and where the protease would have cut and then allowing for the possibility of modifications such as phosphorylation.</p>
<p>From this we detected a number of different phosphorylation sites on both SMAD1 and TNIK which are summarised in the diagram below.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/06/phosphomapping-1024x703.png" alt="" width="780" height="535" class="alignnone size-large wp-image-3016" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/06/phosphomapping-1024x703.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/06/phosphomapping-300x206.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/06/phosphomapping-768x527.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/06/phosphomapping.png 1314w" sizes="(max-width: 780px) 100vw, 780px" /></p>
<p><em>Summary of identified peptides showing overall coverage (red), definitive phosphorylation (blue) and ambiguous phosphorylation (orange). Consensus site for TNIK phosphorylation of SMAD1 from Wang et. al. 2016 shown boxed in blue.</em></p>
<p>&nbsp;</p>
<p>A number of sites were identified on TNIK which must be due to auto-phosphorylation. Three sites were identified on SMAD1 definitively and a further three ambiguously.</p>
<ul>
<li>Three of these sites (one firm and two ambiguous &#8211; PI<strong>SS</strong>V<strong>S</strong>) are known sites of phosphorylation by ALK2 however we know that ALK2 cannot phosphorylate them alone and requires a type II receptor for this to happen – it seems that under in-vitro conditions TNIK is not under the same restriction.</li>
<li>One of these sites (FE<strong>T</strong>VY) is not known to be involved in SMAD1 inhibition but it cannot be ruled out yet and may bear further investigation</li>
<li>Two of these sites (one firm and one ambiguous – RN<strong>ST</strong>IE) are in the location suggested by Wang et. al. (2016) as being the consensus sequence for TNIK phosphorylation of SMAD1.</li>
</ul>
<p>&nbsp;</p>
<p>The next thing to check will be if SMAD phosphorylated in the ‘inhibitory’ position stops ALK2 from phosphorylating SMAD in the ‘normal’ position.</p>
<p>You can find more detail on experimental work <a href="https://doi.org/10.5281/zenodo.3257646">over on Zenodo</a>.</p>
<p>&nbsp;</p>
<p>Reference:</p>
<p>Identification of Phosphorylation Consensus Sequences and Endogenous Neuronal Substrates of the Psychiatric Risk Kinase TNIK.  Wang Q, Amato SP, Rubitski DM, Hayward MM, Kormos BL, Verhoest PR, Xu L, Brandon NJ, Ehlers MD. Pharmacol Exp Ther. 2016 Feb;356(2):410-23. doi: 10.1124/jpet.115.229880. Epub 2015 Dec 8.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/selectivity-against-and-action-of-tnik-kinase-in-relation-to-alk2/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Production and Purification of Huntingtin interaction partners continued</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/production-and-purification-of-huntingtin-interaction-partners-continued/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/production-and-purification-of-huntingtin-interaction-partners-continued/#respond</comments>
		<pubDate>Tue, 25 Jun 2019 18:23:10 +0000</pubDate>
		<dc:creator><![CDATA[Jacob McAuley]]></dc:creator>
				<category><![CDATA[Huntingtin]]></category>
		<category><![CDATA[Huntington’s disease]]></category>
		<category><![CDATA[Jacob McAuley]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=3010</guid>
		<description><![CDATA[As the summer continues to progress, so does my project. After the last round of purification, detailed in my previous blog post, I am ready for another round. During this purification, I will be working with SH3G3 and PACN1. PACN1 is responsible for cytoskeletal remodeling through regulation of several other proteins, as well as neuronal <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/production-and-purification-of-huntingtin-interaction-partners-continued/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>As the summer continues to progress, so does my project. After the last round of purification, detailed in my <a href="https://thesgc.github.io/static-openlabnotebooks/first-round-of-huntingtin-interaction-partner-protein-production/">previous blog post</a>, I am ready for another round. During this purification, I will be working with <a href="https://www.uniprot.org/uniprot/Q9BY11">SH3G3</a> and <a href="https://www.uniprot.org/uniprot/Q99963">PACN1</a>. PACN1 is responsible for cytoskeletal remodeling through regulation of several other proteins, as well as neuronal vesicle endocytosis, among other things. It is thought that SH3G3 is more specialized in aiding endocytosis. Luckily, the purification of both proteins was successful, although that’s not to say everything went to plan.</p>
<p>The tags on these proteins allowed for multiple possible binding possibilities, meaning we had different avenues we could try for the purification. In the first attempt, we used SBP resin. In theory, the SBP would bind our protein of interest, allowing us to purify it, but in practice, the binding affinity was disappointingly low. In an attempt to salvage the purification, we then ran through a nickel purification, reverse nickel purification and size exclusion chromatography. Because of the design of the tag, our protein binds Nickel, but some proteins that occur naturally will also bind Nickel, so after collecting all of the nickel binding proteins, we can cut off the tag on our protein, using TEV, leaving it unable to bind Nickel. This means that is we mix nickel resin with the proteins; everything should bind the Nickel except our protein. After this, we ran it through a column with gel beads that contain tiny pores. Small proteins get trapped in the pores while larger proteins glide through, this gives us an additional way to purify our protein. For complete methods and data for <a href="https://zenodo.org/record/3256579#.XRJllIhKjcs">SH3G3</a> and <a href="https://zenodo.org/record/3256589#.XRJlr4hKjcs">PACN1</a> purifications, look to my posts on Zenodo.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/production-and-purification-of-huntingtin-interaction-partners-continued/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>First Round of Huntingtin Interaction Partner Protein Production</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/first-round-of-huntingtin-interaction-partner-protein-production/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/first-round-of-huntingtin-interaction-partner-protein-production/#respond</comments>
		<pubDate>Fri, 21 Jun 2019 19:42:06 +0000</pubDate>
		<dc:creator><![CDATA[Jacob McAuley]]></dc:creator>
				<category><![CDATA[Huntingtin]]></category>
		<category><![CDATA[Huntington’s disease]]></category>
		<category><![CDATA[Jacob McAuley]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2973</guid>
		<description><![CDATA[To start off my project at the SGC, my supervisor, Dr. Rachel Harding, and I produced and purified three proteins. These proteins were RAB1A, RAB1B, and MECP2. Both RAB1A and RAB1B have functions in the initiation of autophagy, which is a cellular system that helps clean out old debris and malfunctioning parts. MECP2 is a <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/first-round-of-huntingtin-interaction-partner-protein-production/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>To start off my project at the SGC, my supervisor, <a href="https://labscribbles.com/">Dr. Rachel Harding</a>, and I produced and purified three proteins. These proteins were <a href="https://www.uniprot.org/uniprot/Q9H0U4">RAB1A</a>, <a href="https://www.uniprot.org/uniprot/P62820">RAB1B</a>, and <a href="https://www.uniprot.org/uniprot/P51608">MECP2</a>. Both RAB1A and RAB1B have functions in the initiation of autophagy, which is a cellular system that helps clean out old debris and malfunctioning parts. MECP2 is a protein that can interact with DNA, changing the levels of expression of several genes. The purifications of RAB1A and RAB1B were successful, although MECP2 could not be adequately purified. These three proteins were hits with the <a href="https://zenodo.org/record/3234159#.XP7Go4hKjcs">BioID experiment</a> I mentioned in my<a href="https://thesgc.github.io/static-openlabnotebooks/project-overview-huntingtin-and-its-interaction-partners/"> previous post</a>.</p>
<p>To begin, we took <em>E. coli</em> cells and transformed them with plasmid DNA. The transformation process introduces the bacteria to the genetic instructions on how to make the protein, the cells were left to grow for about 24 hours. By the end, we had 6 L of liquid culture for both RAB1A and RAB1B as well as 4 L of MECP2. From this liquid culture, we collected the cells and proceeded to break them open using sound energy via a process called sonication. After opening the cells, we were able to purify our protein using two methods. The first was affinity chromatography, which allows the binding of a specific tag we put on our protein. To further refine the protein sample, we used size-exclusion chromatography, which is a process that separates proteins based on their size. For a more in-depth explanation of the methods used to purify <a href="https://zenodo.org/record/3252165#.XQ00qIhKjcs">RAB1A</a>, <a href="https://zenodo.org/record/3252174#.XQ00k4hKjcs">RAB1B</a>, and <a href="https://zenodo.org/record/3252176#.XQ0z74hKjcs">MECP2</a>, follow the links to the Zenodo posts about each protein.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/first-round-of-huntingtin-interaction-partner-protein-production/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Optimisation of viability assay for DIPG patient-derived cell lines</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/optimisation-of-viability-assay-for-dipg-patient-derived-cell-lines/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/optimisation-of-viability-assay-for-dipg-patient-derived-cell-lines/#respond</comments>
		<pubDate>Thu, 13 Jun 2019 22:41:42 +0000</pubDate>
		<dc:creator><![CDATA[Jong Fu]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Brain Tumours]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Jong Fu Wong]]></category>
		<category><![CDATA[SGC Univ. of Oxford]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2935</guid>
		<description><![CDATA[Background: Evaluation of the efficacy of M4K compounds in DIPG patient-derived cell lines is essential before any promising compounds can be further tested in mouse xenograft models. This approach can aid in narrowing down clinical compound candidates and reduce the time, resources and animal sacrifice needed downstream. A robust and efficient readout for the changes <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/optimisation-of-viability-assay-for-dipg-patient-derived-cell-lines/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p><strong>Background:</strong></p>
<p>Evaluation of the efficacy of M4K compounds in DIPG patient-derived cell lines is essential before any promising compounds can be further tested in mouse xenograft models. This approach can aid in narrowing down clinical compound candidates and reduce the time, resources and animal sacrifice needed downstream.</p>
<p>A robust and efficient readout for the changes in the viability of the DIPG cells needs to be established before it can be used to evaluate the M4K compounds. In addition, the amount of cells to be seeded at the beginning of the experiment has to be optimised to avoid overcrowding and starvation of the cells after extended culture times. Overcrowding and starvation will lead to increased cell death and prevent accurate estimation of the potency of M4K compounds (<a href="https://en.wikipedia.org/wiki/EC50">EC50</a>).</p>
<p><strong>Experimental design:</strong></p>
<p>The wells of 96-well plate with transparent bottom will be coated with laminin to synchronise the mode of growth (adherent or non-adherent) among the different patient-derived lines. Additionally, this will promote even distribution of cells and allow most cells to be in focus when images are taken.</p>
<p>An increasing cell number (125 to 20000) will be seeded into each well. The amount of cells after 7 days of growth in 100µl of medium will be measured based on the area occupied and the total amount of ATP present in each well. In addition, propidium iodide which stains dead cells will be added at the beginning the experiment. At seeding number that are not saturating, the amount of cells after 7 days should remain proportional to the initial seeding number.</p>
<p><strong>Results:</strong></p>
<p><a href="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/06/Figure-1.-Experimental-design.tif"><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/06/Figure-1.-Experimental-design.tif" alt="" class="alignnone size-full wp-image-2936" /></a></p>
<div id="attachment_2938" style="width: 436px" class="wp-caption alignnone"><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/06/Figure-1.-Experimental-design.gif" alt="" width="426" height="159" class=" wp-image-2938" /><p class="wp-caption-text">Timeline of experiment</p></div>
<p>&nbsp;</p>
<div id="attachment_2939" style="width: 973px" class="wp-caption alignnone"><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/06/Figure-2.-Quantification-of-confluency-by-Celigo-algorithms.gif" alt="" width="963" height="792" class="wp-image-2939" /><p class="wp-caption-text">Demonstration of how Celigo algorithm quantifies the area occupied by total cells and area occupied by propidium iodide signal. Images of HSJD-DIPG-007 cells after 7 days in culture were used.</p></div>
<div id="attachment_2940" style="width: 966px" class="wp-caption alignnone"><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/06/Figure-3.-Percentage-confluency.gif" alt="" width="956" height="471" class="wp-image-2940" /><p class="wp-caption-text">Percentage of area occupied by cells after 7 days in culture.</p></div>
<div id="attachment_2941" style="width: 971px" class="wp-caption alignnone"><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/06/Figure-4.-CellTiter-Glo.gif" alt="" width="961" height="453" class="wp-image-2941" /><p class="wp-caption-text">Luminescence signal from CellTiter-Glo at endpoint. ATP content is proportional to the amount of luminescence signal and is an indicator of cell viability.</p></div>
<div id="attachment_2942" style="width: 973px" class="wp-caption alignnone"><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/06/Figure-5.-Near-perfect-linearity-up-to-5000-cells-per-well.gif" alt="" width="963" height="465" class="wp-image-2942" /><p class="wp-caption-text">Near perfect linearity. The viability of the cells (CellTiter Glo signal) correlates almost perfectly with the area occupied by the cells (confluency) (below 5000 cells seeded per well). R squared values of the two measurements are indicated next to the fitted curves. A perfect correlation will have an R squared value of 1.</p></div>
<div id="attachment_2943" style="width: 841px" class="wp-caption alignnone"><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/06/Figure-6.-DIPG-cell-seeding-number-chosen.gif" alt="" width="831" height="458" class="wp-image-2943 size-full" /><p class="wp-caption-text">DIPG cell seeding numbers selected for future experiments</p></div>
<p><strong>Findings:</strong></p>
<p>1) Celigo and its algorithm is able to quantify the area occupied by the cells accurately.</p>
<p>2) SU-DIPG-IV and HSJD-DIPG-007 proliferate quickly. Seeding too many of these cells at the beginning of the experiment will saturate both confluency and CellTiter Glo readouts.</p>
<p>3) CellTiter Glo signal and confluency are good indicators of the actual quantity of cells in the well. Below 5000 cells seeded per well, both sets of measurements correlated almost perfectly with each other.</p>
<p>4) Cell death (as indicated by propidium iodide signal) in relation to total cell at endpoint increases when too many cells were seeded at the beginning of the experiment.</p>
<p>5) Initial cell seeding numbers for each patient-derived cell lines that do not lead to overcrowding at the end of the experiment can be chosen based on the linearity of CellTiter Glo signal and the intensity of propidium iodide.</p>
<p><strong>In a nut shell:</strong></p>
<p>We now have the parameters to go ahead and evaluate how well M4K compounds can stop the growth or kill DIPG cells.</p>
<p>For detailed experimental descriptions, please visit my <strong><a href="https://zenodo.org/record/3245503#.XQLMWRZKiUk">Zenodo</a> </strong>page.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/optimisation-of-viability-assay-for-dipg-patient-derived-cell-lines/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Synthesis of promiscuous ALK2 inhibitors-continuation-2</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/synthesis-of-promiscuous-alk2-inhibitors-continuation-2/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/synthesis-of-promiscuous-alk2-inhibitors-continuation-2/#respond</comments>
		<pubDate>Thu, 13 Jun 2019 20:34:07 +0000</pubDate>
		<dc:creator><![CDATA[Alfredo Picado]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Alfredo Picado]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>
		<category><![CDATA[SGC UNC Chapel Hill]]></category>
		<category><![CDATA[Understudied Kinases]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2924</guid>
		<description><![CDATA[Continuing with our goal to synthesize three compounds as ALK2 inhibitors, let’s refer back to Scheme 2 in the previous post on this topic, compounds 6, 8 and 9 were synthesized in good yields (65-90%), however compound 10 could not be obtained. &#160; Different conditions were tested to induce the cyclization of the bromo-pyrazine-2-carboxamide 9 <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/synthesis-of-promiscuous-alk2-inhibitors-continuation-2/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p style="text-align: justify;">Continuing with our goal to synthesize three compounds as ALK2 inhibitors, let’s refer back to Scheme 2 in the previous post on this topic, compounds <strong>6</strong>, <strong>8</strong> and <strong>9</strong> were synthesized in good yields (65-90%), however compound <strong>10</strong> could not be obtained.</p>
<div id="attachment_2597" style="width: 748px" class="wp-caption alignnone"><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Screen-Shot-2019-03-19-at-4.22.42-PM.png" alt="" width="738" height="522" class=" wp-image-2597" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Screen-Shot-2019-03-19-at-4.22.42-PM.png 918w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Screen-Shot-2019-03-19-at-4.22.42-PM-300x212.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Screen-Shot-2019-03-19-at-4.22.42-PM-768x544.png 768w" sizes="(max-width: 738px) 100vw, 738px" /><p class="wp-caption-text">Scheme 2. Synthesis route 1</p></div>
<p>&nbsp;</p>
<p style="text-align: justify;">Different conditions were tested to induce the cyclization of the bromo-pyrazine-2-carboxamide <strong>9</strong> to the target bromo-pyrazin-2-amine <strong>10</strong>. This reaction mostly yielded starting material <strong>9 </strong>and by-product <strong>11</strong> in low yields (&lt;15%). After several attempts, this approach (acid/heat) was abandoned.</p>
<p>&nbsp;</p>
<div id="attachment_2926" style="width: 746px" class="wp-caption alignnone"><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/06/Screen-Shot-2019-06-13-at-4.17.14-PM.png" alt="" width="736" height="592" class=" wp-image-2926" /><p class="wp-caption-text">Scheme 3. Synthesis route 2</p></div>
<p style="text-align: justify;">An alternative synthetic route is shown in Scheme 3.  Here, aminopirazine <strong>12</strong> will be reduced with diisobutylaluminium hydride (DIBAL-H) to yield carbaldehyde <strong>13</strong>, which then will be reacted with diamine <strong>6</strong> (previously synthesized) at 200 <sup>o</sup>C using dimethylacetamide (DMA) as a solvent. This last step can be carried out in a microwave reactor and should yield the cyclized compound pyridinpyrazin-2-amine <strong>14</strong>, which can be brominated with <em>N</em>-Bromosuccinimide (NBS) to yield compound <strong>10</strong>.</p>
<p style="text-align: justify;">As it was mentioned before, <strong>10 </strong>could not be synthesized by the previously proposed route. The new route is adapted from international patent WO 2008/071937 and we hope in this way we can access to this key intermediate in 150-200 mg scale to move forward with more reactions.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/synthesis-of-promiscuous-alk2-inhibitors-continuation-2/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Project Overview: Huntingtin and its interaction partners</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/project-overview-huntingtin-and-its-interaction-partners/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/project-overview-huntingtin-and-its-interaction-partners/#respond</comments>
		<pubDate>Mon, 10 Jun 2019 14:15:20 +0000</pubDate>
		<dc:creator><![CDATA[Jacob McAuley]]></dc:creator>
				<category><![CDATA[Huntingtin]]></category>
		<category><![CDATA[Huntington’s disease]]></category>
		<category><![CDATA[Jacob McAuley]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2905</guid>
		<description><![CDATA[&#160; Huntington’s disease (HD) is a progressive neurological disorder caused by a mutation in the HTT gene which encodes for the huntingtin protein. The current understanding of the function of HTT is limited, and there are only a few sources that indicate that it likely plays a critical role in several intracellular processes such as <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/project-overview-huntingtin-and-its-interaction-partners/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>&nbsp;</p>
<p>Huntington’s disease (HD) is a progressive neurological disorder caused by a mutation in the HTT gene which encodes for the huntingtin protein. The current understanding of the function of HTT is limited, and there are only a few sources that indicate that it likely plays a critical role in several intracellular processes such as proper neural development. This gap in the scientific literature continues onto the mutant version of the protein. In patients with HD, we know that the mutant protein misfolds and forms aggregates. However, it is unclear if this is the direct cause for the symptoms of the disease or if it is a downstream symptom of some other problem. To obtain a more conclusive understanding of huntingtins role in the cell, the SGC is trying to obtain a structure for huntingtin protein.<br />
So now the question is, how do we image such a molecule? At the SGC we use a technique called X-ray Crystallography. This is a process by which a very pure protein sample is crystallized and expose to X-rays. Using the diffraction of light through the crystal, we can piece together an atom by atom computer model of the sample protein. One complication with the huntingtin protein is that it contains several disordered regions which would likely prevent the formation of a consistent structure in the crystal. To get a full image of this protein, we need to find a way to force a consistent structure onto the disordered regions. Binding a protein to the disordered region causes consistent reordering of this region. The use of a <a href="https://zenodo.org/record/3234159#.XP5doIhKjcs">BioID experiment</a> has identified several proteins that bind huntingtin. In effect, this experiment consists of arming the huntingtin protein with a spray can. When another protein comes within its vicinity, it is tagged by the huntingtin molecule. By looking at tagged proteins, several interaction partners of huntingtin were identified.<br />
During my time at the SGC, I plan to go through this list of tentative interaction partners, producing and purifying them for use in future experiments. Furthermore, this could allow us to obtain crystal structures of huntingtin in complex with an interaction partner. Obtaining a list of all the interaction partners of any given protein can give us a good idea of the function of the protein. This is because we can then attribute the interaction partners function to that of the protein of interest. In the coming weeks, I will continue to post updates on the progress of the purification of the hits from the BioID experiment.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/project-overview-huntingtin-and-its-interaction-partners/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Identifying new members of the huntingtin interactome with BioID and more updates</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/identifying-new-members-of-the-huntingtin-interactome-with-bioid-and-more-updates/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/identifying-new-members-of-the-huntingtin-interactome-with-bioid-and-more-updates/#respond</comments>
		<pubDate>Thu, 30 May 2019 20:38:45 +0000</pubDate>
		<dc:creator><![CDATA[Rachel Harding]]></dc:creator>
				<category><![CDATA[Huntington’s disease]]></category>
		<category><![CDATA[Rachel Harding]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2890</guid>
		<description><![CDATA[I have been busy in the lab working at the bench and training our new recruits to the Huntington’s disease research team! Welcome Jacob and Claudia! Using a technique called BioID, we have identified proteins which are proximal or nearby to the HTT protein in cells. To date, no published literature details the use of <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/identifying-new-members-of-the-huntingtin-interactome-with-bioid-and-more-updates/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>I have been busy in the lab working at the bench and training our new recruits to the Huntington’s disease research team! Welcome Jacob and Claudia!</p>
<p>Using a technique called <a href="https://zenodo.org/record/3234159">BioID</a>, we have identified proteins which are proximal or nearby to the HTT protein in cells. To date, no published literature details the use of this methodology to identify proteins which might interact with HTT so this represents a novel approach. Working in collaboration with Prof. Anne-Claude Gingras (Lunenfeld Tanenbaum Research Institute, University of Toronto) and her postdoctoral fellow Dr. Geoff Hesketh, we have used this technique to finalise a short-list of supposed HTT interacting proteins, many of which are newly described in their connection to Huntington’s disease. This is an exciting step forward to this project and I am keen to follow up all of these new leads.</p>
<p>Working in collaboration with Prof. Hilal Lashuel and Driss Boudeffa as well as our cryoEM collaborators Prof. Susan Lea and Dr. Justin Deme, we have analysed how altering the <a href="https://zenodo.org/record/3234163">phosphorylation state of HTT might change the global structure</a>. We analysed our dephosphorylated samples by cryoEM and we can see the global structure of the protein from <a href="https://zenodo.org/record/3234167">analysis of the images</a> collected by Susan and Justin. I am looking forward to their next update on the remaining samples.</p>
<p>The stringent protein purification protocol for HTT is now <a href="https://zenodo.org/record/3234174">optimised</a> and we used this to purify both <a href="https://zenodo.org/record/3234182">HTT and HTT-HAP40 Q23</a> . These samples are ready for analysis and use in different interaction studies.</p>
<p>I am away from the lab for the next few weeks as I will be travelling to Europe to attend the <a href="https://www.grc.org/cag-triplet-repeat-disorders-conference/2019/">Gordon Research Conference on Triplet Repeat Disorders in Italy</a>. I am excited to give my talk and present my poster as well as catch up with many of my HD colleagues and collaborators.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/identifying-new-members-of-the-huntingtin-interactome-with-bioid-and-more-updates/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>ACVR1 Crystals taken to the Diamond Light Source (Beamline I04.) &#8211; no data sets collected.</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/acvr1-crystals-taken-to-the-diamond-light-source-beamline-i04-no-data-sets-collected/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/acvr1-crystals-taken-to-the-diamond-light-source-beamline-i04-no-data-sets-collected/#respond</comments>
		<pubDate>Mon, 13 May 2019 13:19:13 +0000</pubDate>
		<dc:creator><![CDATA[Ellie Williams]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Ellie Williams]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>
		<category><![CDATA[SGC Univ. of Oxford]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2861</guid>
		<description><![CDATA[Following up on the crystals from my last post I sent them along on the last Diamond trip we had to beamline I04 but regrettably didn’t get any usable datasets. Perhaps fortunately there is a possible explanation for this that isn’t just ‘the crystals were a bit too rubbish’. We send our crystals to the Diamond <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/acvr1-crystals-taken-to-the-diamond-light-source-beamline-i04-no-data-sets-collected/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Following up on the crystals from my last post I sent them along on the last Diamond trip we had to <a href="https://www.diamond.ac.uk/Instruments/Mx/I04">beamline I04</a> but regrettably didn’t get any usable datasets. Perhaps fortunately there is a possible explanation for this that isn’t just ‘the crystals were a bit too rubbish’. We send our crystals to the Diamond Light Source and because of the high number of crystals sent on each trip by the department (124 on the last trip) not everyone goes along on every trip and we quite often use something called ‘auto-centering’ that uses a computer algorithm to find the edge of the loop and to align the loop and your crystal in the beam. Centering is necessary but also quite tricky, because to collect a data set you need ‘slices’ of data taken with the crystal at different angles so the loop the crystals are mounted in, is rotated by a robotic arm so that the crystal also rotates around inside the x-ray beam. Thus if your crystal isn’t aligned properly, as the loop rotates it moves you crystal out of the beam and thus you get no data at that point.</p>
<p>Usually the automatic alignment is really good, however this time around it didn’t work so well and quite often either my crystal wasn’t in the beam at all (it mistook the loop stem for the crystal) or it started in the beam and then rotated out of it (which is what I think happened in the case shown below where you can see two ‘periods’ of data with dips in between. This is, of course, suboptimal and leads to a data set that you can’t really use.</p>
<p>I’m resending the crystals to the beamline this Friday and hopefully going with them myself. The silver lining to this is that I know that some of them definitely diffract which is encouraging. Hopefully they’ll behave well when fully in the beam.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/05/xtal_centering.png" alt="" width="940" height="430" class="alignnone size-full wp-image-2862" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/05/xtal_centering.png 940w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/05/xtal_centering-300x137.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/05/xtal_centering-768x351.png 768w" sizes="(max-width: 940px) 100vw, 940px" /></p>
<p><em>Fig. 1 – Beam aligned at very edge of loop showing cyclic diffraction suggesting the crystal was moving in and out of the beam rather than degrading in the beam.</em></p>
<p>&nbsp;</p>
<p>You can read more about this <a href="https://doi.org/10.5281/zenodo.2788112">over on Zenodo.</a></p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/acvr1-crystals-taken-to-the-diamond-light-source-beamline-i04-no-data-sets-collected/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
	</channel>
</rss>
