<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>Disease &#8211; openlabnotebooks.org</title>
	<atom:link href="https://thesgc.github.io/static-openlabnotebooks/category/disease-foundation/feed/" rel="self" type="application/rss+xml" />
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<description>A growing team of groundbreaking scientists around the world are now sharing their lab notebooks online</description>
	<lastBuildDate>Tue, 03 Mar 2020 04:46:39 +0000</lastBuildDate>
	<language>en-GB</language>
	<sy:updatePeriod>
	hourly	</sy:updatePeriod>
	<sy:updateFrequency>
	1	</sy:updateFrequency>
	<generator>https://wordpress.org/?v=5.3.2</generator>

<image>
	<url>https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/cropped-lab-jar02-32x32.png</url>
	<title>Disease &#8211; openlabnotebooks.org</title>
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<width>32</width>
	<height>32</height>
</image> 
	<item>
		<title>Its a small world we live in</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/its-a-small-world-we-live-in/</link>
				<comments>https://thesgc.github.io/static-openlabnotebooks/its-a-small-world-we-live-in/#respond</comments>
				<pubDate>Tue, 03 Mar 2020 04:46:39 +0000</pubDate>
		<dc:creator><![CDATA[Thomas Durcan]]></dc:creator>
				<category><![CDATA[ALS-Reproducible Antibody Platform (ALS-RAP)]]></category>
		<category><![CDATA[Amyotrophic Lateral Sclerosis (ALS)]]></category>
		<category><![CDATA[iPSC Disease modelling]]></category>
		<category><![CDATA[iPSCs]]></category>
		<category><![CDATA[Montreal Neurological Institute]]></category>
		<category><![CDATA[Parkinson's disease and neurodevelopmental disorders]]></category>
		<category><![CDATA[Thomas M. Durcan]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=4334</guid>
				<description><![CDATA[When I was a kid, I used to have a spinning globe lamp in my bedroom. What can I say? I was a big nerd child (I also had a passion for stamp collecting and making model airplanes, but thats a tale for another time). I loved plugging it in and when it lit up <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/its-a-small-world-we-live-in/">Read More ...</a>]]></description>
								<content:encoded><![CDATA[<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/03/spinning-lamp-286x300.jpg" alt="" width="286" height="300" class="size-medium wp-image-4335 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/03/spinning-lamp-286x300.jpg 286w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/03/spinning-lamp.jpg 425w" sizes="(max-width: 286px) 100vw, 286px" /></p>
<p>When I was a kid, I used to have a spinning globe lamp in my bedroom. What can I say? I was a big nerd child (I also had a passion for stamp collecting and making model airplanes, but thats a tale for another time). I loved plugging it in and when it lit up I would spend hours turning it around and memorizing counties and cities. Shame it didn’t help me with my high school geography as I never quite understand the difference between igneous and sedimentary rock. Looking at the globe, the world always seemed so massive. But Saturday I was reminded how small this world is that we live in.</p>
<p>For months (well since January 2019), I could sit and watch Liverpool games in peace. They win, no texts, no messages, I guess no one wanted to talk to me and ask if I saw the game and enjoyed the Liverpool performance. Then on Saturday, no sooner do Watford start scoring and Liverpool start losing the messages come thick and fast; from all corners of the globe. I never realized how eager people were to let me know how things were going, was I watching the liverpool match, did I have any thoughts on the manner of the loss. Imagine 50 years ago, or even 100 years ago, this wouldn’t have been possible. They would have written me a letter, and I would have received it 2-4 weeks later with their regards. And I could have just filed it in the pile saying return to sender.</p>
<p>Anyways, I appreciate all the best wishes and messages. While the world looks so big, its often surprising how small it really is and how easy it is to keep in touch with people from all corners of the world. Last week some good news came from one of these international partnerships I had struck up with a fellow scientist in a different part of the world, <a href="https://www.researchgate.net/profile/Nicolas_Unsain2">Dr Nico Unsain</a>, from Cordoba, Argentina.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/03/Nico-image-300x300.jpg" alt="" width="300" height="300" class="size-medium wp-image-4336 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/03/Nico-image-300x300.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/03/Nico-image-150x150.jpg 150w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/03/Nico-image.jpg 450w" sizes="(max-width: 300px) 100vw, 300px" /></p>
<p>I met Nico over ten years ago, when we worked beside each other in our respective labs. Nico taught me how to present properly, and how to give all my talks in a &#8220;BBC world voice&#8221;. He also taught me how to enjoy fine South American wines, and I have yet to find a Malbec I didn&#8217;t like.</p>
<p>A few years ago, Nico headed back to Argentina to set up his own group, but we always stayed in touch, discussing the merits of Lionel Messi (is he better or worse than Ronaldo), why do Argentina keep losing in major tournaments, and so on. Two years ago, I got to welcome Nico back into the MNI and my lab, as he came to do a 4 month exchange in my group.</p>
<p>Now, two years later, the fruits of our effort along with our other collaborators are now published in our new <a href="https://www.nature.com/articles/s41598-020-59856-w">scientific reports paper</a> (its also open access in case you ask). To sum up the work, the project involved blowing axons up in size (like filling plastic tubing with water and watching it expand) so we could analyze the periodicity of proteins within the neurons, a process termed expansion microscopy. This is some pretty cool work and am happy to have had chance to finally work with Nico on a project. If anyone is interested, I encourage you to check out the <a href="https://www.nature.com/articles/s41598-020-59856-w">paper</a>, and feel free to reach out to Nico, he&#8217;s one of the nicest guys you will. Hopefully we can continue this partnership and I am looking forward to the day I finally get an invite to come over to his lab in Cordoba, to sample fine Argentinian Malbec and to do some science in his neck of the woods. I also imagine the weather is a little nicer than what we are currently experiencing during Canadian winter.</p>
<p>As for Liverpool, they are back in action Tuesday in the FA cup, so I guess if all goes badly, I will be hearing from my good friends (and not so good friends) across the globe, checking in on me, and asking me how I feel about the Liverpool performance. If Liverpool play like I hope, then radio silence until their next defeat. Ah what a small world it is indeed.</p>
]]></content:encoded>
							<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/its-a-small-world-we-live-in/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
							</item>
		<item>
		<title>PILOT RESULTS: Grin1 Y647S Project</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/pilot-results-grin1p-y647s-project/</link>
				<comments>https://thesgc.github.io/static-openlabnotebooks/pilot-results-grin1p-y647s-project/#comments</comments>
				<pubDate>Tue, 18 Feb 2020 17:43:42 +0000</pubDate>
		<dc:creator><![CDATA[Katheron Intson]]></dc:creator>
				<category><![CDATA[Amy Ramsey (Univ. of Toronto)]]></category>
		<category><![CDATA[GRIN disorder]]></category>
		<category><![CDATA[Katheron Intson]]></category>
		<category><![CDATA[NMDA]]></category>
		<category><![CDATA[Rare disease]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=4274</guid>
				<description><![CDATA[Grin1 p.Tyr647Ser Project Background. Grin genes across species encode for proteins that make up the different NMDA receptor subunits. GRIN1 mutations (usually heterozygous) are associated with intellectual disability with epilepsy in humans, including the rare p.Tyr647Ser (Y647S) variant. This missense mutation occurs in the M3 helix of the transmembrane domain of GRIN1 which, importantly, is conserved between human GRIN1 and mouse <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/pilot-results-grin1p-y647s-project/">Read More ...</a>]]></description>
								<content:encoded><![CDATA[<p style="text-align: center;"><strong><em>Grin1 </em></strong><strong>p.Tyr647Ser Project</strong></p>
<p><strong>Background. </strong><em>Grin </em>genes across species encode for proteins that make up the different NMDA receptor subunits. <em>GRIN1 </em>mutations (usually heterozygous) are associated with intellectual disability with epilepsy in humans, including the rare p.Tyr647Ser (Y647S) variant. This missense mutation occurs in the M3 helix of the transmembrane domain of <em>GRIN1 </em>which, importantly, is conserved between human <em>GRIN1 </em>and mouse <em>Grin1</em>. Currently, no effective pharmacological intervention or gene therapy exists to treat human patients with <em>GRIN1 </em>disorders. Patients with different variants to the <em>GRIN1</em>gene vary widely in symptom presentation. Thus, it is imperative to independently study each of the variants preclinically when developing treatments.</p>
<p>To do this, we generated <em>Grin1<sup>Y647S </sup></em>mice using CRISPR and bred founder mice to develop F1  <em>Grin1<sup>+/Y647S </sup></em>progeny. As proof-of-concept for our study, our first aim is to evaluate the behaviour of <em>Grin1<sup>+/Y647S </sup></em>mice to ascertain whether there is phenotypic similarity to <em>GRIN1 </em>Y647S patients.</p>
<p>This entry describes <strong>pilot results </strong>from the Ramsey Laboratory regarding behavioural consequences of heterozygous Y647S mutation in mice.</p>
<p><strong>Behavioural Assays, Results, and Discussion:</strong></p>
<p>Each graph contains data from <em>n</em>= 4 WT and <em>n =</em>4 <em>Grin1<sup>+/Y647S </sup></em>mice of &gt;12 weeks of age mixed sexes.</p>
<ol>
<li>Open Field Test</li>
</ol>
<p><em>To measure locomotor activity.</em>Mice were placed in plexiglass arenas of 20x20x45 cm and their locomotor activity was measured using digital activity monitoring of infrared light beam breaks for 2h in 5min bins:</p>
<p><strong>This data has been removed in anticipation of a research grant outcome. Please email k.intson@utoronto.ca if you are interested in hearing about progress on this project. </strong></p>
<ol start="2">
<li>Social Affiliative Assay</li>
</ol>
<p><em>To measure social behaviour. </em>Mice were placed in an opaque open-top arena of 62x42x22 cm containing two wire cups. One wire cup contained a novel, same-age, same-sex mouse as a social stimulus, while the other cup was empty. Mice were allowed to freely explore the arena for 8min and their activity was measured using video recording motion software. The time spent in the 3cm zone surrounding each cup was recorded, as well as the track length each mouse accumulated around each cup, and the number of visits they made to each cup.</p>
<p><strong>This data has been removed in anticipation of a research grant outcome. Please email k.intson@utoronto.ca if you are interested in hearing about progress on this project. </strong></p>
<ol start="3">
<li>Elevated Plus Maze</li>
</ol>
<p><em>To measure anxiety-like behaviour.</em>Mice were placed in a elevated maze (39 cm high) composed of two opposing open arms (30.5&#215;5 cm) and two opposing closed arms (30.5x5x15.2 cm). Mice were placed in an open arm of the maze and allowed to freely explore for 8 min in dim light while being recorded. Motion tracking software measured the amount of time spent in open and closed arms.</p>
<p><strong>This data has been removed in anticipation of a research grant outcome. Please email k.intson@utoronto.ca if you are interested in hearing about progress on this project. </strong></p>
<p><strong>Other observations.</strong> I report that 1 male and 1 female <em>Grin1<sup>+/Y647S </sup></em>mouse each displayed a seizure when placed on the elevated plus maze. I would describe this seizure as atonic-like in nature: while mice stood on their hind legs, I observed rapid chewing movements, light convusions, and they fell to the floor of the EPM. Seizures lasted ~10s and mice rapidly regained consciousness.</p>
<p><strong>Planned experiments. </strong>Future experiments will involve boosting behaviour experimental group numbers, and examining biochemical consequences of Y647S on overall NMDA receptor function in order to determine treatment strategies for <em>GRIN1</em>patients.</p>
]]></content:encoded>
							<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/pilot-results-grin1p-y647s-project/feed/</wfw:commentRss>
		<slash:comments>2</slash:comments>
							</item>
		<item>
		<title>Safety first when working in the lab</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/safety-first-when-working-in-the-lab/</link>
				<comments>https://thesgc.github.io/static-openlabnotebooks/safety-first-when-working-in-the-lab/#comments</comments>
				<pubDate>Wed, 12 Feb 2020 13:28:35 +0000</pubDate>
		<dc:creator><![CDATA[Thomas Durcan]]></dc:creator>
				<category><![CDATA[ALS-Reproducible Antibody Platform (ALS-RAP)]]></category>
		<category><![CDATA[Amyotrophic Lateral Sclerosis (ALS)]]></category>
		<category><![CDATA[iPSC Disease modelling]]></category>
		<category><![CDATA[iPSCs]]></category>
		<category><![CDATA[Montreal Neurological Institute]]></category>
		<category><![CDATA[Parkinson's disease and neurodevelopmental disorders]]></category>
		<category><![CDATA[Thomas M. Durcan]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=4256</guid>
				<description><![CDATA[Do you know what the image above represents? I will give you a clue, it relates to lab safety. Any takers? Still nothing? Well, this my friends represents corrosive materials. This group includes many strong acids and bases and was a question I got wrong in my safety exam today (gasp, shock, you failed Tom). <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/safety-first-when-working-in-the-lab/">Read More ...</a>]]></description>
								<content:encoded><![CDATA[<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/02/Corrosion-GHS-vivid-learning-systems.jpg" alt="" width="200" height="200" class="size-full wp-image-4257 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/02/Corrosion-GHS-vivid-learning-systems.jpg 200w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/02/Corrosion-GHS-vivid-learning-systems-150x150.jpg 150w" sizes="(max-width: 200px) 100vw, 200px" /></p>
<p>Do you know what the image above represents? I will give you a clue, it relates to lab safety. Any takers? Still nothing? Well, this my friends represents <span style="text-decoration: underline;"><strong>corrosive materials</strong></span>. This group includes many strong acids and bases and was a question I got wrong in my safety exam today (<em><strong>gasp, shock, you failed Tom</strong></em>). I know, I know, after 18 years of working in a lab, I still have issues with safety tests every once in a while. So why mention this. Well, I said a few months ago in a presentation, if I was going to celebrate my successes, I need to also discuss the times I failed. So this is my reason for telling the story, to say its OK to fail and to learn from it.  But why am I taking a safety test in the first place.</p>
<p>Well every 3-4 years, when you work at McGill, they make us take a Workplace Hazardous Materials Information System (WHMIS) test to ensure we know safety regulations for working with hazardous materials, which in the lab is everything, even coffee. I passed WHMIS for the first time many years in 2007 and since then I could do the online refresher test every three years to remain certified. But, not this year. Oh no. Now its WHMIS 2015. New and improved. Which meant I had to sit through a 2h lecture all over again. So that was me Monday afternoon, sitting in a room Monday afternoon somewhere on McGill campus learning about the ten pictograms of hazardous materials, how to label a container, and in general how to act responsibly when working in a lab. Which after 17 years, I do pretty well at this, I might add (<em><strong>For any EHS inspectors reading</strong></em>).</p>
<p>Anyways, once you sign the attendance sheet, they send you the exam link which is also open book. And what did I do, I opened it this morning, and started answering away thinking I knew it all, hurrying through it and answering without thinking. And then <em><strong>BAM</strong></em>, results come in. Not good. It was almost as bad as the Iowa caucuses, although at least these results came within seconds and not days. 15 from 20 correct, and I needed 16 correct answers to pass. Stupid corrosive materials sent me astray. And to top things off, my student who started in lab did the test, and got 19, on her first attempt. Obviously she is great, but what a way to rub salts into the wound. Beaten by my own new graduate student.</p>
<p>I was stewing. Years and years of being in a lab, and I can’t pass a safety test. People now look at you different. Like &#8220;<em><strong>Hi Tom, so corrosive materials in your hands there, yeah you should put that down. You could hurt yourself</strong></em>&#8220;. The indignity of it all. So on it went, ridiculed by my colleagues. But, then light at the end of the tunnel, I got my retest link. Thank the lord, I got another shot at redemption, and this time boy do I do it right.</p>
<p>Multiple WHMIS manuals open, a large manual of SDS sheets for every chemical you could imagine and a banana to sustain me for the hour needed to complete the test. I will admit, it was this exam was a rocky road, corrosive vs flammable, is there a difference, well of course, but in the moment you question everything, fatigue sets in. 60 minutes pass and finally I finish. Then, the results come in. Booyah, I passed. 18 correct from 20, 90%, a passing grade, the clouds pass, and the sun shines again (obviously not in Montreal, we are peak winter and buried in snow, but I am sure somewhere else had sunshine, maybe <a href="https://thesgc.github.io/static-openlabnotebooks/rise-of-the-machines-and-lab-automation/">San Diego</a>). Corrosive materials are still my downfall, but now I know to never light a match around oxidizing materials, or the whole place goes up in smoke. Anyways, what a day in the lab life of Tom. The downs and ups. From being unsafe to  being safety-approved. And now I can back to my hazardous job, sitting on a computer writing emails, living the scientist dream.</p>
<p>In some science news, I want to highlight great work from the ALS field from Dr. Christine Vande Velde, an exceptional ALS researcher in Montreal at the CHUM, hope you all please check it out. <a style="font-size: 1.5rem;" href="https://www.jbc.org/content/early/2020/02/06/jbc.RA119.011215.long">https://www.jbc.org/content/early/2020/02/06/jbc.RA119.011215.long</a></p>
<p><span style="font-size: 1.5rem;">Next time, CRISPR editing will be discussed for all those editing </span>aficionados<span style="font-size: 1.5rem;">. </span></p>
]]></content:encoded>
							<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/safety-first-when-working-in-the-lab/feed/</wfw:commentRss>
		<slash:comments>2</slash:comments>
							</item>
		<item>
		<title>Purification of HTT N-HEAT_81-1643 construct with different buffer conditions</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/purification-of-htt-n-heat_81-1643-construct-with-different-buffer-conditions/</link>
				<comments>https://thesgc.github.io/static-openlabnotebooks/purification-of-htt-n-heat_81-1643-construct-with-different-buffer-conditions/#respond</comments>
				<pubDate>Fri, 07 Feb 2020 17:16:22 +0000</pubDate>
		<dc:creator><![CDATA[Claudia Alvarez]]></dc:creator>
				<category><![CDATA[Claudia Alvarez]]></category>
		<category><![CDATA[Huntingtin]]></category>
		<category><![CDATA[Huntington’s disease]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=4243</guid>
				<description><![CDATA[In my previous purification of the HTT N-HEAT_81-1643 construct, I showed that an additional heparin step did not remove any traces of nucleic acid material, changed purity or oligomeric state of the sample. Additionally, my DLS results showed that this construct forms large particles in solution. Thus, the DLS results in combination with the gel filtration <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/purification-of-htt-n-heat_81-1643-construct-with-different-buffer-conditions/">Read More ...</a>]]></description>
								<content:encoded><![CDATA[<p>In my <a href="https://zenodo.org/record/3658639#.Xj2VjyNOk2w">previous purification of the HTT N-HEAT_81-1643 construct</a>, I showed that an additional heparin step did not remove any traces of nucleic acid material, changed purity or oligomeric state of the sample. Additionally, my <a href="https://zenodo.org/record/3561096#.Xj2LZSNOk2w">DLS</a> results showed that this construct forms large particles in solution. Thus, the DLS results in combination with the gel filtration data gathered from the <a href="https://zenodo.org/record/3462496#.Xj2LgSNOk2w">first</a> and <a href="https://zenodo.org/record/3658639#.Xj2VjyNOk2w">second </a>rounds of purification imply that the HTT N-HEAT_81-1643 construct is in a high oligomeric state in solution. However, it is still unclear if these large particles in solution are the result of oligomerization, a monomeric state in an extended conformation (which would also register as large particles in both DLS and gel filtration chromatography), or binding to nucleic acid material. In order to learn more about the oligomeric state of this construct, I aimed at finding ideal buffer conditions to change the oligomeric state of the sample and favour the monomeric state.</p>
<p>First, I performed a preliminary buffer screen using <a href="https://zenodo.org/record/3561087#.Xj2L3CNOk2w">DSF</a> and <a href="https://zenodo.org/record/3519364#.Xj2LtCNOk2w">DSLS</a>. However, no differences in the temperature of aggregation (T<sub>agg</sub>) or melting temperature (T<sub>m</sub>) were observed in the initial conditions tested. Furthermore, the DSF data were difficult to fit due to the high initial fluorescence intensity. Like DSF, performing the buffer screen with DSLS would have been challenging to do. This is because our data showed that the T<sub>agg</sub> for this construct lies right at the detection limit of the technique (approx. 90 <sup>o</sup>C) meaning that changes in T<sub>agg­ </sub>would have been difficult to assess.</p>
<p>Second, I considered doing the buffer screen by changing the purification buffers and directly assessing the oligomeric state of the sample by gel filtration chromatography. This is what I did in <a href="https://zenodo.org/record/3658439#.Xj2VRSNOk2w">this round of purification</a>. The buffer screen tested the conditions listed in Table 1. I tested the addition of detergent, arginine, varying the pH, salt and glycerol. These conditions were selected based on purification strategies found in literature that were successfully applied to full length HTT or proteins in general. For instance, addition of small amounts of detergent (CHAPS) has been successfully used to purify full HTT.<sup>3, 4</sup> Further, including arginine in the lysis buffer has been shown to stabilize the monomeric state of proteins.<sup>5, 6</sup></p>
<p>Table 1. Buffer conditions tested for purification of the HTT N-HEAT_81-1643 construct</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/02/table_-post-300x63.png" alt="" class="alignnone wp-image-4244" width="508" height="107" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/02/table_-post-300x63.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/02/table_-post-768x162.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/02/table_-post-1024x216.png 1024w" sizes="(max-width: 508px) 100vw, 508px" /></p>
<p>My results show that the oligomeric state of the HTT N-HEAT_81-1643 remains unchanged as evidenced by the elution volume from the gel filtration column (Figure 2). Thus, none of the conditions tested were successful at stabilizing the monomeric state of the construct. One additional strategy that I would like to test next is to co-express this construct with HAP40, as this strategy proved successful for full length HTT.<sup>2, 4</sup> Further, in the future I will also send this sample for analysis by negative stain which can help me learn more about the oligomeric state of this construct.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/02/gel-filtration_post-300x75.png" alt="" class="alignnone wp-image-4245" width="508" height="127" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/02/gel-filtration_post-300x75.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/02/gel-filtration_post-768x192.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/02/gel-filtration_post.png 774w" sizes="(max-width: 508px) 100vw, 508px" /></p>
<p>Figure 1 Gel filtration chromatography profiles for the HTT N-HEAT_81-1643 construct. The concentrated elution samples of the HTT N-HEAT_81-1643 construct produced by using different purification buffers (Table 1 – FLAG buffer 1-3) were loaded on a Superose 6 10/300 GL column using the Superose 6 buffer (A-C). The absorbance of the eluted sample is monitored at 280 nm. The total volume of the column is Vc = 23.56 mL.</p>
<p>&nbsp;</p>
<p>References</p>
<ol>
<li>Huang, B.; Lucas, T.;  Kueppers, C.;  Dong, X. M.;  Krause, M.;  Bepperling, A.;  Buchner, J.;  Voshol, H.;  Weiss, A.;  Gerrits, B.; Kochanek, S., Scalable Production in Human Cells and Biochemical Characterization of Full-Length Normal and Mutant Huntingtin. <em>Plos One </em><strong>2015,</strong> <em>10</em> (3).</li>
<li>Guo, Q.; Huang, B.;  Cheng, J. D.;  Seefelder, M.;  Engler, T.;  Pfeifer, G.;  Oeckl, P.;  Otto, M.;  Moser, F.;  Maurer, M.;  Pautsch, A.;  Baumeister, W.;  Fernandez-Busnadiego, R.; Kochanek, S., The cryo-electron microscopy structure of huntingtin. <em>Nature </em><strong>2018,</strong> <em>555</em> (7694), 117-+.</li>
<li>Arakawa, T.; Ejima, D.;  Tsumoto, K.;  Obeyama, N.;  Tanaka, Y.;  Kita, Y.; Timasheff, S. N., Suppression of protein interactions by arginine: A proposed mechanism of the arginine effects. <em>Biophysical Chemistry </em><strong>2007,</strong> <em>127</em> (1), 1-8.</li>
<li>Tischer, A.; Lilie, H.;  Rudolph, R.; Lange, C., L-arginine hydrochloride increases the solubility of folded and unfolded recombinant plasminogen activator rPA. <em>Protein science : a publication of the Protein Society </em><strong>2010,</strong> <em>19</em> (9), 1783-1795.</li>
<li>Harding, R. J.; Loppnau, P.;  Ackloo, S.;  Lemak, A.;  Hutchinson, A.;  Hunt, B.;  Holehouse, A. S.;  Ho, J. C.;  Fan, L. X.;  Toledo-Sherman, L.;  Seitova, A.; Arrowsmith, C. H., Design and characterization of mutant and wildtype huntingtin proteins produced from a toolkit of scalable eukaryotic expression systems. <em>Journal of Biological Chemistry </em><strong>2019,</strong> <em>294</em> (17), 6986-7001.</li>
</ol>
]]></content:encoded>
							<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/purification-of-htt-n-heat_81-1643-construct-with-different-buffer-conditions/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
							</item>
		<item>
		<title>Rise of the machines&#8230;.and Lab automation</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/rise-of-the-machines-and-lab-automation/</link>
				<comments>https://thesgc.github.io/static-openlabnotebooks/rise-of-the-machines-and-lab-automation/#respond</comments>
				<pubDate>Thu, 06 Feb 2020 13:16:09 +0000</pubDate>
		<dc:creator><![CDATA[Thomas Durcan]]></dc:creator>
				<category><![CDATA[ALS-Reproducible Antibody Platform (ALS-RAP)]]></category>
		<category><![CDATA[Amyotrophic Lateral Sclerosis (ALS)]]></category>
		<category><![CDATA[iPSC Disease modelling]]></category>
		<category><![CDATA[iPSCs]]></category>
		<category><![CDATA[Montreal Neurological Institute]]></category>
		<category><![CDATA[Parkinson's disease and neurodevelopmental disorders]]></category>
		<category><![CDATA[Thomas M. Durcan]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=4234</guid>
				<description><![CDATA[Last week this was my view. This week nothing but wintry snow and cold everywhere. Oh, how I miss the sunnier climes of San Diego. I think I will add one more resolution to my first blog, spend some time every winter somewhere sunny, as this cold really gets into the bones. Anyways, I wasn’t <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/rise-of-the-machines-and-lab-automation/">Read More ...</a>]]></description>
								<content:encoded><![CDATA[<p>Last week this was my view.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/02/San-diego-palm-trees-225x300.jpg" alt="" width="225" height="300" class="size-medium wp-image-4237 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/02/San-diego-palm-trees-225x300.jpg 225w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/02/San-diego-palm-trees-768x1024.jpg 768w" sizes="(max-width: 225px) 100vw, 225px" /></p>
<p>This week nothing but wintry snow and cold everywhere.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/02/Mtl_snow-258x300.jpg" alt="" width="258" height="300" class="size-medium wp-image-4238 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/02/Mtl_snow-258x300.jpg 258w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/02/Mtl_snow-768x894.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/02/Mtl_snow-879x1024.jpg 879w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/02/Mtl_snow.jpg 900w" sizes="(max-width: 258px) 100vw, 258px" /></p>
<p>Oh, how I miss the sunnier climes of San Diego. I think I will add one more resolution to my first blog, spend some time every winter somewhere sunny, as this cold really gets into the bones. Anyways, I wasn’t simply in San Diego to look on in wonderment at palm trees. No I was there to learn about robots. More specifically, automation equipment for the lab, how we can make things faster, how robot arms in a lab can work with 500 cell-lines, do a thousand reactions a minute, all while making a pot of tea for me, and figuring out what we think about our latest data. Is this science fiction, no its reality. In fact, this was a glimpse into all the latest lab automation technology on display at this years <a href="https://www.slas2020.org/">annual SLAS meeting</a>, otherwise known as the “<a href="https://slas.org/">Society for Lab Automation and Screening</a>”. And can I say, my mind was blown. The technologies coming down the pipeline to analyze one single cell, all the way to growing a million organoids with new AI algorithms for analyzing all the data from these cells is not five years away. It’s already here. And ready to make our daily lab lives easier.</p>
<p>So are the machines taking over. Is it like Terminator 2? Are the machines ready to take over. Well I will be blunt, no. I arrive back in Montreal, and one of our fridges in the lab stops working. Just you know, no longer keeps things cold. Its now fixed, so yay. Then a few day later, a -80 freezer decides to crash on us. Oh, how I wish I was back in sunshine, where the machines worked wonders, here are freezers have issues staying operational, thank god for backup freezers ready to take over when one crashes. Anyways, my take on it all, machines and technology are good, but they won’t take over, they need us to repair them, and to make sure they are used properly. And I should never leave the lab again, when I leave, things break.</p>
<p>So I was delayed in blogging/social mediaing with all the travel over the past few weeks but am now back in Montreal and over the coming weeks will be writing more content as we release new work from the lab. For this week however, I wanted to bring peoples attention to two great pieces of work from elsewhere:</p>
<ul>
<li><a href="https://www.sciencedirect.com/science/article/pii/S0092867419313236?dgcid=author"><span style="text-decoration: underline;"><strong>Snake venom organoids:</strong></span></a> This is literally the coolest thing I have seen, its phenomenal work. Now we have a way to study snake venom in a dish. I think this will be a very powerful tool and I can’t wait to see what comes next from this group</li>
<li><a href="https://www.nature.com/articles/s41586-020-1962-0"><span style="text-decoration: underline;"><strong>Minibrains aren’t brains:</strong></span></a> A well designed study in Nature came out last week showing that 3D neuronal organoids or minibrains were not in fact miniature brains, and were also stressed. This confirms what we all suspected for a long time, but this opportunity opens the door for new work moving forward with 3D neuronal organoids.</li>
</ul>
<p>To finish, I also have to say thank you to <a href="https://cihr-irsc.gc.ca/e/193.html">CIHR</a>, or the Canadian Institute for Health Research, two weeks ago I got the great news that my <a href="https://cihr-irsc.gc.ca/e/51837.html">project grant</a> was funded, this was my view from Vancouver airport as I waited to board my flight when I got the news, top 10 happiest moments of my life that is for sure.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/02/Vancouver-airport-298x300.jpg" alt="" width="298" height="300" class="size-medium wp-image-4239 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/02/Vancouver-airport-298x300.jpg 298w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/02/Vancouver-airport-150x150.jpg 150w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/02/Vancouver-airport-768x773.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/02/Vancouver-airport-1018x1024.jpg 1018w" sizes="(max-width: 298px) 100vw, 298px" /></p>
<p>I took a picture of where I was so I would always remember where I was when I got my first CIHR research grant. I never got a CIHR grant before, or NIH funding or federal funding from any other country for that matter. I always found alternative ways to fund my work, but I was never able to convince these federal granting agencies that my work was fundable. With the CIHR decision, it feels like finally myself and my group are on the right way, vindicating my decision to become a scientist nearly two decades ago. You never forgot your first big grant, and I sure won’t, so thank you reviewers, colleagues, work team and everyone else for all their support, and also a special thank you to Vancouver airport, my new favourite airport.</p>
<p>This also means I can now cross one <a href="https://thesgc.github.io/static-openlabnotebooks/blog-2020_1-scientific-resolutions-for-2020/">resolution</a> from my list, <em><strong>I obtained a research grant as a research scientist</strong></em>. So, one down, nine resolutions to go. In the coming days, I will next be discussing the work ongoing on CRISPR genome editing in my group, and the tools we are making available. So stay tuned.</p>
]]></content:encoded>
							<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/rise-of-the-machines-and-lab-automation/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
							</item>
		<item>
		<title>UGGT:  a fly-paper glycoprotein catching troublesome flies?</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/uggt-is-the-misfolding-glycoprotein-checkpoint-like-misfolded-glycoprotein-fly-paper/</link>
				<comments>https://thesgc.github.io/static-openlabnotebooks/uggt-is-the-misfolding-glycoprotein-checkpoint-like-misfolded-glycoprotein-fly-paper/#respond</comments>
				<pubDate>Sun, 19 Jan 2020 15:04:40 +0000</pubDate>
		<dc:creator><![CDATA[Pietro Roversi]]></dc:creator>
				<category><![CDATA[Open Secretion Rescue]]></category>
		<category><![CDATA[Pietro Roversi]]></category>
		<category><![CDATA[Pietro Roversi (Univ. of Leicester)]]></category>
		<category><![CDATA[Rare disease]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=4013</guid>
				<description><![CDATA[&#160; About one quarter of the Eukaryotic genome encodes glycoproteins that are made in the Endoplasmic Reticulum (ER), and travel to special destinations such as the extracellular environment, the cell membrane or other organelles like the lysosomes. We can think of the ER as a nursery where baby glycoproteins quickly grow to attain their adult <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/uggt-is-the-misfolding-glycoprotein-checkpoint-like-misfolded-glycoprotein-fly-paper/">Read More ...</a>]]></description>
								<content:encoded><![CDATA[<p>&nbsp;</p>
<div id="attachment_4017" style="width: 255px" class="wp-caption alignleft"><img aria-describedby="caption-attachment-4017" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/01/FlyPaper-245x300.png" alt="" class="size-medium wp-image-4017" width="245" height="300" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/01/FlyPaper-245x300.png 245w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/01/FlyPaper-768x940.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/01/FlyPaper-837x1024.png 837w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/01/FlyPaper.png 1500w" sizes="(max-width: 245px) 100vw, 245px" /><p id="caption-attachment-4017" class="wp-caption-text">Mis-folded glycoproteins are caught by UGGT and cannot leave the ER.</p></div>
<p>About one quarter of the Eukaryotic genome encodes <a href="https://en.wikipedia.org/wiki/Glycoprotein">glycoprotein</a>s that are made in the <a href="https://en.wikipedia.org/wiki/Endoplasmic_reticulum">Endoplasmic Reticulum</a> (ER), and travel to special destinations such as the extracellular environment, the cell membrane or other organelles like the lysosomes. We can think of the ER as a nursery where baby glycoproteins quickly grow to attain their adult shape and size, in conditions that closely mimic the ones they evolved to find themselves in.</p>
<p>But how is it decided that a glycoprotein is fully and perfectly formed and can leave the ER? As it turns out, the Eukaryotic cell has evolved a formidable glycoprotein folding quality control (ERQC) system, which ensures that glycoproteins that are mis-folded &#8211; i.e. are still forming &#8211; cannot prematurely leave the ER. The checkpoint of this quality control system is the enzyme that detects a defects in a folding glycoprotein and flags it for ER retention: UDP-glucose Glycoprotein Glucosyl-Transferase (<a href="https://en.wikipedia.org/wiki/UGGT">UGGT</a>).</p>
<p>Thinking of the variety of sizes and shapes of the thousands of <a href="https://en.wikipedia.org/wiki/Secretory_protein">secretory proteins</a> that travel from the ER down the secretory pathway, a big question comes to mind: how can UGGT &#8211; a single enzyme &#8211; survey folding of so many and such different glycoprotein clients? It has been hypothesized that UGGT recognizes exposed &#8220;sticky&#8221; <a href="https://en.wikipedia.org/wiki/Amino_acid">amino acid</a> residues on the surface of a defective glycoprotein, but nobody has yet discovered how the recognition of that &#8220;stickiness&#8221; happens.</p>
<p>In the process of carrying out a control experiment, we have now obtained serendipitous data suggesting that UGGT itself may possess a &#8220;sticky&#8221; mis-folded patch, and that it may therefore catch glycoproteins that carry a folding defect via a &#8220;mis-fold:mis-fold&#8221; molecular recognition event. We prepared a special version of recombinant UGGT, treating mammalian cells with the mannosidase inhibitor <a href="https://en.wikipedia.org/wiki/Kifunensine">kifunensine</a>: this was meant to produce a UGGT molecule (which we call UGGT<sub>Kif</sub> ) bearing glycans identical to the ones carried by all glycoproteins in the ER (including UGGT&#8217;s substrates). Surprisingly, <a href="https://zenodo.org/record/3608191#.XiR76RfgrNA">the UGGT<sub>Kif </sub>crystal structure</a> revealed that one of the domains of the protein is mis-folded!</p>
<p>This observation &#8211; together with <a href="http://10.1016/j.bbrc.2015.04.105">published evidence</a> that UGGT bears de-mannosylated glycans (a hallmark of defective glycoproteins flagged for <a href="https://en.wikipedia.org/wiki/Endoplasmic-reticulum-associated_protein_degradation">ER associated degradation</a>) &#8211; now begs the question whether UGGT has evolved to maintain a mis-folded domain in order to catch mis-folded glycoproteins. Clearly, a physiologically mis-folded UGGT would be steadily degraded in the ER, so if nature has come up with this solution to the problem of mis-fold recognition &#8211; Eukaryotic cells would need to afford quick catabolism of UGGT and the costs associated with constant replenishment of ER UGGT levels.</p>
<p>In summary, when it comes to glycoproteins, <a href="https://doi.org/10.1101/2019.12.25.888438">does it take one to know one</a>? We are now pursuing experiments to test the hypothesis that UGGT be like mis-folded glycoprotein fly-paper, catching mis-folded glycoprotein flies. Watch this space!</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
]]></content:encoded>
							<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/uggt-is-the-misfolding-glycoprotein-checkpoint-like-misfolded-glycoprotein-fly-paper/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
							</item>
		<item>
		<title>Blog 2020_1: Scientific resolutions for 2020</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/blog-2020_1-scientific-resolutions-for-2020/</link>
				<comments>https://thesgc.github.io/static-openlabnotebooks/blog-2020_1-scientific-resolutions-for-2020/#comments</comments>
				<pubDate>Tue, 14 Jan 2020 12:46:50 +0000</pubDate>
		<dc:creator><![CDATA[Thomas Durcan]]></dc:creator>
				<category><![CDATA[ALS-Reproducible Antibody Platform (ALS-RAP)]]></category>
		<category><![CDATA[Amyotrophic Lateral Sclerosis (ALS)]]></category>
		<category><![CDATA[iPSC Disease modelling]]></category>
		<category><![CDATA[iPSCs]]></category>
		<category><![CDATA[Montreal Neurological Institute]]></category>
		<category><![CDATA[Parkinson's disease and neurodevelopmental disorders]]></category>
		<category><![CDATA[Thomas M. Durcan]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=3972</guid>
				<description><![CDATA[A happy new year all. Yes I know, it’s already mid-January and we are nearly all done with our resolutions. But what better way for me to start back to the open notebooking/blogging than to kick off with a top 10 list of my resolutions/hopes for 2020 (some science/some aspirational). Yes, I am back blogging <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/blog-2020_1-scientific-resolutions-for-2020/">Read More ...</a>]]></description>
								<content:encoded><![CDATA[<p>A happy new year all. Yes I know, it’s already mid-January and we are nearly all done with our resolutions. But what better way for me to start back to the open notebooking/blogging than to kick off with a top 10 list of my resolutions/hopes for 2020 (some science/some aspirational). Yes, I am back blogging and will be doing it on a regular basis for the rest of the year. Since finishing the last series of posts, the past months of 2019 were insanely busy with grants, meetings, admin duties, teaching, travel and life, so thankfully over Christmas and the new year I got to return to the motherland. Once in Ireland, I got to relax, recharge and reflect and did my best thinking and pontificating. So I am back, and ready to go.</p>
<p>For this year, you can expect <strong><u>40 blogs</u></strong> over the next 12 months, I know, you are like <em><strong>WHAT</strong></em>. 40 blogs. Are you crazy Tom. You&#8217;re so boring, please stop. Well yes I am dull (I enjoy naps and 9h of sleep over a nightclub), but no I will not stop, I have things to say, and by Jove I will say it. Unlike the summer blogs, this will be a little different. These will be more frequent but shorter in length and be more a diary of my comings and going, where I am, what I am doing, what work is coming out of the <a href="https://www.mcgill.ca/neuro/open-science/open-science-platforms/eddu">group</a>, and so forth.</p>
<p>All coinciding with a big year for Tom, as I turn a majestic 40 years old, which is also why 40 blogs, one for each year. So, lets not take any longer, with a new year, I have set a list of ten things I am hoping for/planning for over the next 12 months I will use the blog to track my progress on these items, some are a teeny tiny little bit bigger than others, but lets see how we get on, aim big as they say:</p>
<ol>
<li>Publish a minimum of ten scientific manuscripts. The past two years have been slow getting papers out (Large part my fault in not writing faster), so this year I have resolved to be an author on at least ten papers, last year I had four, so I want to double if not treble this number to show you all the cool science we are doing. As each new one comes out, you will be the first to know about it as we track my progress.</li>
<li>Witness an event 30 years in the making. If you know me, you know what this is, no more to be said. If this doesn&#8217;t happen, I would avoid me for like a month.</li>
<li>Turn 40 gracefully. We shall see how this goes. If I end up having an existential crisis in May we shall know it took a U turn.</li>
<li>Obtain a research grant as a lead scientist. We all need funds to do the science we are doing, so aim is to be the lead scientist on at least one grant. Lets see how I do. Fingers crossed.</li>
<li>Build three dinosaurs. I will be showing you my progress over the coming months. Which one shall I start with? (A kind Xmas gift from the lab, hours of endless fun). <img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/01/lego-dinos-282x300.jpg" alt="" width="282" height="300" class="size-medium wp-image-3973 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/01/lego-dinos-282x300.jpg 282w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/01/lego-dinos-768x816.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/01/lego-dinos-964x1024.jpg 964w" sizes="(max-width: 282px) 100vw, 282px" /></li>
<li>Launch a scientific company. I have a lab, but never a company. I have some ideas, so lets see if I can bring them to fruition and launch a company. If I do, you will all be the first to know.</li>
<li>Write 40 blogs (explained above)</li>
<li>Send cells into space. To trul<span style="font-size: 1.5rem;">y start exploring the universe, our cells need to go into space.</span></li>
<li>Identify new ways to make the work of my group more open and accessible. I have some ideas, so lets see how things go in 2020.</li>
<li><span style="font-size: 1.5rem;">Expand my butcher business of Tom Durcan meats. Some of the best deli meat you will ever taste.</span></li>
</ol>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/01/Tom-Durcan-meats-300x240.jpg" alt="" width="300" height="240" class="size-medium wp-image-3974 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/01/Tom-Durcan-meats-300x240.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/01/Tom-Durcan-meats-768x614.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/01/Tom-Durcan-meats-1024x819.jpg 1024w" sizes="(max-width: 300px) 100vw, 300px" /></p>
<p>As I succeed or fail with any of these items, then I will let you know. The next blog will be coming next week from rainy (snowy) Vancouver where I am travelling for the upcoming <a href="https://www.keystonesymposia.org/ks/Online/Events/2020J1/Details.aspx?EventKey=2020J1">Keystone Organoid</a> meeting. Am pumped for it, very excited to hear the latest work on organoids.</p>
<p>Until next week, here is some <a href="https://www.biorxiv.org/content/10.1101/2019.12.28.890012v1">new work</a> from the group obtained in a study led by Dr. Stefano Stefani, and our first manuscript of the year, so one down, nine more to go. Hope you enjoy and I will be back next week.</p>
<p><span style="text-decoration: underline;"><strong>Disclaimer:</strong></span> These are not my meats, I wish they were. It is my famous Irish namesake <a href="http://www.tomdurcanmeats.ie/">Tom Durcan</a>. He is a butcher from Cork whose deli meats are simply sensational. You should all visit Ireland to sample his produce, and then stay for the guinness and whiskey, just saying.</p>
]]></content:encoded>
							<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/blog-2020_1-scientific-resolutions-for-2020/feed/</wfw:commentRss>
		<slash:comments>1</slash:comments>
							</item>
		<item>
		<title>Mass spec analysis of SMAD1 phosphorylation by ALK2 and ALK2 GS mutants in the presence of ACVR2.</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/mass-spec-analysis-of-smad1-phosphorylation-by-alk2-and-alk2-gs-mutants-in-the-presence-of-acvr2/</link>
				<comments>https://thesgc.github.io/static-openlabnotebooks/mass-spec-analysis-of-smad1-phosphorylation-by-alk2-and-alk2-gs-mutants-in-the-presence-of-acvr2/#respond</comments>
				<pubDate>Wed, 18 Dec 2019 12:10:55 +0000</pubDate>
		<dc:creator><![CDATA[Ellie Williams]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Ellie Williams]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>
		<category><![CDATA[SGC Univ. of Oxford]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=3945</guid>
				<description><![CDATA[I decided to test a set of my ALK2 GS loop phosphomutants for activity. As I’ve written about previously, I needed to optimise the concentrations of receptors and decided that 60uM was the right concentration to use as showing activity with the mutants and also being workable in the assay. I set up three different <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/mass-spec-analysis-of-smad1-phosphorylation-by-alk2-and-alk2-gs-mutants-in-the-presence-of-acvr2/">Read More ...</a>]]></description>
								<content:encoded><![CDATA[<p>I decided to test a set of my ALK2 GS loop phosphomutants for activity. As I’ve written about previously, I needed to optimise the concentrations of receptors and decided that 60uM was the right concentration to use as showing activity with the mutants and also being workable in the assay.</p>
<p>I set up three different ALK2 mutants alongside the wild type ALK2 to compare them and used ACVR2 as my necessary type II receptor for SMAD1 activation.</p>
<p>I took time points at 0, 1, 5, 15, 30, 45 min, 1h, 1h15, 1h30, 1h45, 2h. Taking 1ul of sample and mixing it with 60ul of mass spec buffer (water + 0.1% formic acid)</p>
<p>I ran these on the mass spec to check for and monitor activity on all components but regrettably ended up seeing zero activity on SMAD1 even after 2h in the wildtype positive control. I did see phosphorylation on the type one receptors but nothing on the SMAD1 which suggested to me that something had gone horribly wrong with the type II receptor and led to it being unable to facilitate SMAD1 activation (as has been previously established to be necessary).</p>
<p>This was, to put it mildly, very frustrating.</p>
<p>So I went and purified a fresh stock of ACVR2 and did a small scale test using just the wild type to see if the positive control worked. This time it did so I set up a larger scale experiment using all the constructs we’ve managed to purify so far (which is all of those created so far except c107, c111 and c112).</p>
<p>This time we saw phosphorylation on SMAD and after 2h we could see strong signal and still make out the differences between the activity of the constructs and so further analysis was done on the 2h time point.</p>
<p>Differences between the different mutants stood out instantly – none of them increased the activity compared to the wild type but there were distinct effects on both the SMAD1 and ALK2 auto-phosphorylation patterns.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/12/MS_Traces.png" alt="" width="1478" height="1149" class="alignnone size-full wp-image-3946" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/12/MS_Traces.png 1478w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/12/MS_Traces-300x233.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/12/MS_Traces-768x597.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/12/MS_Traces-1024x796.png 1024w" sizes="(max-width: 1478px) 100vw, 1478px" /></p>
<p><em>Summary figures showing the phosphorylation of SMAD1 and ALK2 using various ALK2 GS loop mutants. The top row on both columns shows the wild type positive control for comparison. The constructs are then ranked from least to most effected moving down the first column (with matching SMAD1 and ALK2 traces) and then down the second column.</em></p>
<p>The last four were the most extreme in terms of knocking out SMAD phosphorylation with the fourth leading to very low levels of phosphorylation and last three basically completely stopping it. When we look at the mutations these four constructs contain we can see a theme in common.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/12/top_4_hits.png" alt="" width="436" height="60" class="alignnone size-full wp-image-3947" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/12/top_4_hits.png 436w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/12/top_4_hits-300x41.png 300w" sizes="(max-width: 436px) 100vw, 436px" /></p>
<p><em>Figure showing the mutations contained within the four most detrimental mutations.</em></p>
<p>The three most detrimental constructs (c109, c106 and c113) all contained a S190A mutation with the fourth (c105) containing a T189A mutation just prior to the S190 residue. This suggests that S190 has a key role in facilitating SMAD1 phosphorylation.</p>
<p>Next thing to test is the constructs I’m missing (they need expressing and purifying) and to repeat this experiment before also running the results on a western blot for confirmation.</p>
<p>If you&#8217;d like more details then please do visit <a href="https://doi.org/10.5281/zenodo.3582748">over on Zenodo</a>.</p>
]]></content:encoded>
							<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/mass-spec-analysis-of-smad1-phosphorylation-by-alk2-and-alk2-gs-mutants-in-the-presence-of-acvr2/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
							</item>
		<item>
		<title>Purification of HTT N-HEAT_81-1643 and initial biophysical characterization</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/purification-of-htt-n-heat_81-1643-and-initial-biophysical-characterization/</link>
				<comments>https://thesgc.github.io/static-openlabnotebooks/purification-of-htt-n-heat_81-1643-and-initial-biophysical-characterization/#respond</comments>
				<pubDate>Thu, 05 Dec 2019 16:24:38 +0000</pubDate>
		<dc:creator><![CDATA[Claudia Alvarez]]></dc:creator>
				<category><![CDATA[Claudia Alvarez]]></category>
		<category><![CDATA[Huntingtin]]></category>
		<category><![CDATA[Huntington’s disease]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=3827</guid>
				<description><![CDATA[In the second purification round of N-HEAT_81-1643, I added an additional purification step with heparin resin. My aim with doing this additional step was to improve the purity of the HTT N-HEAT_81-1643 sample from nucleic acid material/other proteins. Similar experiments proved successful for full HTT+HAP40 sample (by Dr. Rachel Harding). Our results showed that the <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/purification-of-htt-n-heat_81-1643-and-initial-biophysical-characterization/">Read More ...</a>]]></description>
								<content:encoded><![CDATA[<p>In the second purification round of <a href="https://zenodo.org/record/3562523#.XeflbdVOnb0">N-HEAT_81-1643</a>, I added an additional purification step with heparin resin. My aim with doing this additional step was to improve the purity of the HTT N-HEAT_81-1643 sample from nucleic acid material/other proteins. Similar experiments proved successful for full HTT+HAP40 sample (<a href="https://zenodo.org/record/3234174#.Xefm19VOnb0">by Dr. Rachel Harding</a>). Our results showed that the additional step with heparin does not change the purity, oligomeric state, or the content of nucleic acid in the sample of N-HEAT_81-1643  (A<sub>260/280 </sub>=  0.7).</p>
<p>I also performed DSF, DSLS and DLS experiments with HTT N-HEAT_81-1643. The DSLS experiments show the construct aggregates at high temperatures around 80 °C. The DSF experiments showed the construct has a melting temperature around 46 °C. No significant difference in the melting temperature (T<sub>m</sub>) or temperature of aggregation (T<sub>agg</sub>) was observed in the two buffer systems or protein concentrations tested.</p>
<p>The DLS experiments show that the HTT N-HEAT_81-1643 particles in solution have a radius of around 15 nm, implying the presence of large particles. These results are consistent with gel filtration chromatography results, which show N-HEAT_81-1643 elutes as a very large particle in solution (early elution volume). However, there is still the possibility that the construct HTT N-HEAT_81-1643 is in an extended conformation or in complex with nucleic acid material, which would also result in large particle size readings by DLS and gel filtration chromatography. So, in future experiments we will run negative stain experiments, which will tell us more about the conformation/oligomeric state of the sample.</p>
<div id="attachment_3831" style="width: 484px" class="wp-caption alignnone"><img aria-describedby="caption-attachment-3831" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/12/B08-DSLSDSF-DLS-300x62.png" alt="" class="wp-image-3831" width="474" height="98" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/12/B08-DSLSDSF-DLS-300x62.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/12/B08-DSLSDSF-DLS-768x159.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/12/B08-DSLSDSF-DLS-1024x213.png 1024w" sizes="(max-width: 474px) 100vw, 474px" /><p id="caption-attachment-3831" class="wp-caption-text"><a href="https://zenodo.org/record/3562523#.XeflzNVOnb0">Biophysical characterization of HTT N-HEAT_81-1643 by DSLS, DSF and DLS</a>. Representative curve of the DSLS experiments with HTT N-HEAT_81-1643 (0.2 mg/mL) in Buffer 1 (A) and the DSF experiments with HTT N-HEAT_81-1643 (0.1 mg/mL) in Buffer 2 (B). C. DLS results for HTT N-HEAT_81-1643 tested at different concentrations (n = 2).</p></div>
<p>&nbsp;</p>
<p>&nbsp;</p>
]]></content:encoded>
							<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/purification-of-htt-n-heat_81-1643-and-initial-biophysical-characterization/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
							</item>
		<item>
		<title>Updates on all the huntingtin samples I have made and what has been keeping me busy in the lab</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/updates-on-all-the-huntingtin-samples-i-have-made-and-what-has-been-keeping-me-busy-in-the-lab/</link>
				<comments>https://thesgc.github.io/static-openlabnotebooks/updates-on-all-the-huntingtin-samples-i-have-made-and-what-has-been-keeping-me-busy-in-the-lab/#respond</comments>
				<pubDate>Tue, 03 Dec 2019 10:37:25 +0000</pubDate>
		<dc:creator><![CDATA[Rachel Harding]]></dc:creator>
				<category><![CDATA[Huntingtin]]></category>
		<category><![CDATA[Huntington’s disease]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=3865</guid>
				<description><![CDATA[Updates on all the huntingtin samples I have made and what has been keeping me busy in the lab]]></description>
								<content:encoded><![CDATA[<blockquote class="wp-embedded-content" data-secret="bdZ2zpB8DG"><p><a href="https://labscribbles.com/2019/12/03/updates-on-all-the-huntingtin-samples-i-have-made-and-what-has-been-keeping-me-busy-in-the-lab/">Updates on all the huntingtin samples I have made and what has been keeping me busy in the lab</a></p></blockquote>
<p><iframe class="wp-embedded-content" sandbox="allow-scripts" security="restricted" src="https://labscribbles.com/2019/12/03/updates-on-all-the-huntingtin-samples-i-have-made-and-what-has-been-keeping-me-busy-in-the-lab/embed/#?secret=bdZ2zpB8DG" data-secret="bdZ2zpB8DG" width="600" height="338" title="&#8220;Updates on all the huntingtin samples I have made and what has been keeping me busy in the lab&#8221; &#8212; LabScribbles" frameborder="0" marginwidth="0" marginheight="0" scrolling="no"></iframe></p>
]]></content:encoded>
							<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/updates-on-all-the-huntingtin-samples-i-have-made-and-what-has-been-keeping-me-busy-in-the-lab/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
							</item>
	</channel>
</rss>
