<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>Disease &#8211; openlabnotebooks.org</title>
	<atom:link href="https://thesgc.github.io/static-openlabnotebooks/category/disease-foundation/feed/" rel="self" type="application/rss+xml" />
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<description>A growing team of groundbreaking scientists around the world are now sharing their lab notebooks online</description>
	<lastBuildDate>Thu, 12 Nov 2020 22:16:10 +0000</lastBuildDate>
	<language>en-GB</language>
	<sy:updatePeriod>
	hourly	</sy:updatePeriod>
	<sy:updateFrequency>
	1	</sy:updateFrequency>
	<generator>https://wordpress.org/?v=5.5.3</generator>

<image>
	<url>https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/cropped-lab-jar02-32x32.png</url>
	<title>Disease &#8211; openlabnotebooks.org</title>
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<width>32</width>
	<height>32</height>
</image> 
	<item>
		<title>Druggability and Genetic Variability of the ATPase Site and Central Channel of SARS-CoV-2 nsp13 Helicase Across Coronaviruses and SARS-CoV-2 Samples &#8211; Post 23</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/druggability-and-genetic-variability-of-the-atpase-site-and-central-channel-of-sars-cov-2-nsp13-helicase-across-coronaviruses-and-sars-cov-2-samples-post-23/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/druggability-and-genetic-variability-of-the-atpase-site-and-central-channel-of-sars-cov-2-nsp13-helicase-across-coronaviruses-and-sars-cov-2-samples-post-23/#comments</comments>
		
		<dc:creator><![CDATA[Setayesh Yazdani]]></dc:creator>
		<pubDate>Thu, 12 Nov 2020 07:56:23 +0000</pubDate>
				<category><![CDATA[COVID-19]]></category>
		<category><![CDATA[SARS-CoV-2 computational genomics]]></category>
		<category><![CDATA[Setayesh Yazdani]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=5388</guid>

					<description><![CDATA[One of the important enzymes in the replication cycle of the SARS-CoV-2 virus is the helicase, which is also known as non-structural protein 13 (nsp13). During the viral life cycle, the holo-RNA-dependent RNA polymerase, also known as nsp12, is thought to coordinate with several additional factors, including the nsp13 helicase (Snijder et al., 2016; Sola <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/druggability-and-genetic-variability-of-the-atpase-site-and-central-channel-of-sars-cov-2-nsp13-helicase-across-coronaviruses-and-sars-cov-2-samples-post-23/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p>One of the important enzymes in the replication cycle of the SARS-CoV-2 virus is the helicase, which is also known as non-structural protein 13 (nsp13). During the viral life cycle, the holo-RNA-dependent RNA polymerase, also known as nsp12, is thought to coordinate with several additional factors, including the nsp13 helicase (Snijder et al., 2016; Sola et al., 2015) as shown in figure 1.</p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/possible_targets_SARS-CoV-2.png" alt="" width="938" height="507" class="alignnone wp-image-5389 size-full" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/possible_targets_SARS-CoV-2.png 938w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/possible_targets_SARS-CoV-2-300x162.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/possible_targets_SARS-CoV-2-768x415.png 768w" sizes="(max-width: 938px) 100vw, 938px" /></p>
<p><strong>Figure 1. The possible targets for drug development against SARS-CoV-2.</strong> Adapted from Habtemariam et al., 2020.</p>
<p>The ATPase domain of nsp13 catalyzes an NTP-dependent unwinding of duplex RNAs into single strands in the 5’ to 3’ direction. (Habtemariam et al., 2020, Snijder et al., 2016) Previous reports on potent inhibitors targeting the ATPase domain of herpes simplex virus helicase suggest that helicase enzyme is chemically tractable. (Kleymann et al., 2002)</p>
<p>This post provides an overview of the druggability and amino acid variability of both the helicase ATP binding site and its central channel where the RNA molecule binds (figure 2) using the SARS-CoV-2 nsp13 helicase structures available from the protein data bank (PDB).</p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/atpase_site_central_channel_6zsl.png" alt="" width="975" height="645" class="alignnone wp-image-5390 size-full" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/atpase_site_central_channel_6zsl.png 975w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/atpase_site_central_channel_6zsl-300x198.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/atpase_site_central_channel_6zsl-768x508.png 768w" sizes="(max-width: 975px) 100vw, 975px" /></p>
<p><strong>Figure 2. Nsp13 helicase ATPase site and central channel (PDB: 6zsl).</strong> The RNA molecule is shown in ribbon diagram from the superimposed Upf1 complex (PDB code: 2xzl) and the ADP molecule shown in purple stick representation superimposed from SARS-CoV-2 nsp13 helicase structure bound to ADP (PDB: 6xez).</p>
<p>Using SiteMap (Schrodinger, NY) we assessed the druggability of the binding sites using the druggability score (Dscore). The ATP binding site of nsp13 and central channel have Dscores of 1.054 and 1.091 respectively. Both Dscores are indicative of druggable sites.</p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/druggability_nsp13.png" alt="" width="983" height="930" class="alignnone wp-image-5393 size-full" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/druggability_nsp13.png 983w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/druggability_nsp13-300x284.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/druggability_nsp13-768x727.png 768w" sizes="(max-width: 983px) 100vw, 983px" /></p>
<p><strong>Figure 3. Druggability analysis of SARS-CoV-2 nsp13 helicase ATPase site (top panel, PDB: 6xez chain F) and central channel (bottom pane, PDB: 6zsl). </strong></p>
<p>We then determined the residues that line the nsp13 helicase ATPase site and central channel using ICM PocketFinder <span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9">(</span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SpellingErrorV2 SCXW7658465 BCX9">Molsoft</span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9">, S</span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9">an Diego</span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9">) </span></span>and the nsp13 helicase structures. We identified 38 amino acid sidechains lining the nsp13 ATPase site (figure 5, <a href="https://doi.org/10.5281/zenodo.4269234">Zenodo report</a>) and 25 amino acid sidechains lining the central channel (figure 6, <a href="https://doi.org/10.5281/zenodo.4269234">Zenodo report</a>).</p>
<p>We conducted a broad survey of viral proteins from Alpha- and Betacoronaviruses to identify the most conserved druggable sites. By integrating variability and druggability data, we can then infer the most promising strategies for developing broad-spectrum viral inhibitors. This analysis can be valuable in the context of the emergence of future coronaviruses from circulating viral strains in bat reservoir species. We looked at twenty-seven reviewed nsp13 sequences from Uniprot, where six belong to Alphacoronavirus genus entries, and twenty-one belong to the Betacoronavirus genus. We then assessed the variability of amino acid lining the nsp13 helicase ATP binding site and its central channel by performing multiple sequence alignment.</p>
<p>We made diversity dendrograms focusing on the 38 sidechains of the ATPase site (figure 7, <a href="https://doi.org/10.5281/zenodo.4269234">zenodo report</a>) and the 25 sidechains of the central channel (figure 8, <a href="https://doi.org/10.5281/zenodo.4269234">zenodo report</a>) from step2. At the ATPase site, 26 out of 38 residues were found to be identical across these 27 entries which translates to 68% identity, indicating a relatively conserved site. At the nsp13 central channel, only 3 out of 25 residues were non-identical; 88% of residues were identical. This shows that the nsp13 helicase central channel where the RNA molecule binds is highly conserved.</p>
<p>We assessed the genetic variability of the ATPase site and the central channel of nsp13 among SARS-CoV-2 samples. Nicola De Maio, our collaborator from Nick Goldman’s lab at the European Bioinformatics Institute, looked at more than 39000 SARS-CoV-2 genome samples from COVID-19 patients and identified all mutations at the two sites of interest of nsp13 helicase. He identified 12 non-synonymous variants at the ATPase site (table 1, <a href="https://doi.org/10.5281/zenodo.4269234">zenodo report</a>) and no non-synonymous variant at the central channel of nsp13 helicase.</p>
<p>We conclude that the ATPase site of SARS-CoV-2 nsp13 helicase is less conserved than its RNA-binding central channel across Alpha- and Betacoronaviruses. We observe a similar pattern by looking at non-synonymous variants across SARS-CoV-2 samples from COVID-19 patients.  While the ATPase site where ADP binds is druggable and relatively conserved (%68), the central channel of nsp13 helicase where RNA binds is druggable and highly conserved (88%). Our finding supports the design of pan-coronavirus inhibitors targeting the central channel of nsp13 helicase.</p>
<p>You can view the full report on <a href="https://doi.org/10.5281/zenodo.4269234">Zenodo</a>.</p>
<p>Please contact me via the “Leave a comment” link at the top of this post. Stay Tuned for more updates on this project.</p>
<p><strong>References</strong></p>
<p>Habtemariam, S., Nabavi, S. F., Banach, M., Berindan-Neagoe, I., Sarkar, K., Sil, P. C., &amp; Nabavi, S. M. (2020). Should we try SARS-Cov-2 Helicase inhibitors for COVID-19 therapy? <em>Archives of Medical Research</em>, <em>51</em>(7), 733-735. <a href="https://doi.org/10.1016/j.arcmed.2020.05.024">https://doi.org/10.1016/j.arcmed.2020.05.024</a></p>
<p>Kleymann, G., Fischer, R., Betz, U. A., Hendrix, M., Bender, W., Schneider, U., Handke, G., Eckenberg, P., Hewlett, G., Pevzner, V., Baumeister, J., Weber, O., Henninger, K., Keldenich, J., Jensen, A., Kolb, J., Bach, U., Popp, A., Mäben, J., … Rübsamen-Waigmann, H. (2002). New helicase-primase inhibitors as drug candidates for the treatment of herpes simplex disease. <em>Nature Medicine</em>, <em>8</em>(4), 392-398. <a href="https://doi.org/10.1038/nm0402-392">https://doi.org/10.1038/nm0402-392</a></p>
<p>Snijder, E., Decroly, E., &amp; Ziebuhr, J. (2016). The Nonstructural proteins directing coronavirus RNA synthesis and processing. <em>Coronaviruses</em>, 59-126. <a href="https://doi.org/10.1016/bs.aivir.2016.08.008">https://doi.org/10.1016/bs.aivir.2016.08.008</a></p>
<p>Sola, I., Almazán, F., Zúñiga, S., &amp; Enjuanes, L. (2015). Continuous and discontinuous RNA synthesis in coronaviruses. <em>Annual Review of Virology</em>, <em>2</em>(1), 265-288. <a href="https://doi.org/10.1146/annurev-virology-100114-055218">https://doi.org/10.1146/annurev-virology-100114-055218</a></p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/druggability-and-genetic-variability-of-the-atpase-site-and-central-channel-of-sars-cov-2-nsp13-helicase-across-coronaviruses-and-sars-cov-2-samples-post-23/feed/</wfw:commentRss>
			<slash:comments>2</slash:comments>
		
		
			</item>
		<item>
		<title>Druggability and Genetic Variability of the ADP-bound Pocket of SARS-CoV-2 RNA-dependent RNA polymerase NiRAN domain Across Coronaviruses and SARS-CoV-2 Samples &#8211; Post 22</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/druggability-and-genetic-variability-of-the-adp-bound-pocket-of-sars-cov-2-rna-dependent-rna-polymerase-niran-domain-across-coronaviruses-and-sars-cov-2-samples-post-22/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/druggability-and-genetic-variability-of-the-adp-bound-pocket-of-sars-cov-2-rna-dependent-rna-polymerase-niran-domain-across-coronaviruses-and-sars-cov-2-samples-post-22/#respond</comments>
		
		<dc:creator><![CDATA[Setayesh Yazdani]]></dc:creator>
		<pubDate>Thu, 12 Nov 2020 03:39:52 +0000</pubDate>
				<category><![CDATA[COVID-19]]></category>
		<category><![CDATA[Ligandable Genome]]></category>
		<category><![CDATA[SARS-CoV-2 computational genomics]]></category>
		<category><![CDATA[Setayesh Yazdani]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=5384</guid>

					<description><![CDATA[The RNA-dependent RNA-polymerase (RdRp) also known as non-structural protein 12 (nsp12) is the target of antiviral agent remdesivir. Nsp12 has an important role in viral genome replication and transcription. (Chen et al., 2020)  Chen et al. identified a new pocket on the N-terminal extension of nsp12 occupied by ADP-Mg2+ after solving the structure of the helicase-polymerase complex. This ADP-bound pocket is on the N-terminal nidovirus RdRp-associated nucleotidyltransferase (NiRAN) domain which <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/druggability-and-genetic-variability-of-the-adp-bound-pocket-of-sars-cov-2-rna-dependent-rna-polymerase-niran-domain-across-coronaviruses-and-sars-cov-2-samples-post-22/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p><span data-contrast="auto">The RNA-dependent RNA-polymerase (RdRp) also known as non-structural protein 12 (nsp12) is the target of antiviral agent remdesivir. Nsp12 has an important role in viral genome replication and transcription. </span><span data-contrast="auto">(Chen et al., 2020)</span><span data-ccp-props="{&quot;201341983&quot;:0,&quot;335559739&quot;:160,&quot;335559740&quot;:259}"> </span></p>
<p><span data-contrast="auto">Chen et al. identified a new pocket on </span><span data-contrast="auto">the N-terminal extension of </span><span data-contrast="auto">nsp12 </span><span data-contrast="auto">occupied</span><span data-contrast="auto"> </span><span data-contrast="auto">by ADP-Mg2+ after solving the </span><span data-contrast="auto">structure of </span><span data-contrast="auto">the </span><span data-contrast="auto">helicase-polymerase complex. This </span><span data-contrast="auto">AD</span><span data-contrast="auto">P</span><span data-contrast="auto">-bound </span><span data-contrast="auto">pocket is on the N-terminal </span><span data-contrast="auto">nidovirus</span><span data-contrast="auto"> </span><span data-contrast="auto">RdRp</span><span data-contrast="auto">-associated </span><span data-contrast="auto">nucleotidyltransferase</span><span data-contrast="auto"> </span><span data-contrast="auto">(</span><span data-contrast="auto">NiRAN</span><span data-contrast="auto">) domain</span><span data-contrast="auto"> which is also well-resolved</span><span data-contrast="auto">.</span><span data-contrast="auto"> </span><span data-contrast="auto">(Chen et al., 2020)</span><span data-contrast="auto">.</span><span data-ccp-props="{&quot;201341983&quot;:0,&quot;335559739&quot;:160,&quot;335559740&quot;:259}"> </span></p>
<p><span data-contrast="auto">Based on </span><span data-contrast="auto">conservation profile</span><span data-contrast="auto"> across </span><span data-contrast="auto">nido</span><span data-contrast="auto">virales</span><span data-contrast="auto"> (an order of </span><span data-contrast="auto">positive-stranded RNA viruses</span><span data-contrast="auto"> including SARS-CoV-2)</span><span data-contrast="auto">, </span><span data-contrast="auto">the </span><span data-contrast="auto">NiRAN</span><span data-contrast="auto"> domain is beli</span><span data-contrast="auto">eved to have</span><span data-contrast="auto"> </span><span data-contrast="auto">nucleotidylation</span><span data-contrast="auto"> activity</span><span data-contrast="auto">, though the protein </span><span data-contrast="auto">substrates to which nucleotides are added </span><span data-contrast="auto">by the </span><span data-contrast="auto">NiRAN</span><span data-contrast="auto"> domain </span><span data-contrast="auto">are unknown</span><span data-contrast="auto">. </span><span data-contrast="auto"> </span><span data-contrast="auto">Reverse genetics was used to</span><span data-contrast="auto"> establish the importance of </span><span data-contrast="auto">NiRAN</span><span data-contrast="auto"> domain for </span><span data-contrast="auto">nidoviral</span><span data-contrast="auto"> </span><span data-contrast="auto">replication:</span><span data-contrast="auto"> m</span><span data-contrast="auto">utati</span><span data-contrast="auto">ng</span><span data-contrast="auto"> residues conserved in </span><span data-contrast="auto">NirAN</span><span data-contrast="auto"> domains i</span><span data-contrast="auto">nduces</span> <span data-contrast="auto">significant reduction of </span><span data-contrast="auto">SARS-CoV</span><span data-contrast="auto">-1</span><span data-contrast="auto"> and Equine viral arteritis (EAV) </span><span data-contrast="auto">viral titers in cell culture</span><span data-contrast="auto"> </span><span data-contrast="auto">(Lehmann et al., 2015)</span><span data-contrast="auto">.</span><span data-contrast="auto"> </span><span data-contrast="auto">Previous structural homology search of the SARS-CoV</span><span data-contrast="auto">-1</span><span data-contrast="auto"> nsp12 </span><span data-contrast="auto">NiRAN</span><span data-contrast="auto"> domain showed significant homology with protein kinases, including some key catalytic residues such as Asn209 and </span><span data-contrast="auto">NiRAN</span><span data-contrast="auto"> </span><span data-contrast="auto">conserved residues As218 and Phe219. (</span><span data-contrast="auto">Kirchdoerfer</span><span data-contrast="auto"> et al., 2019)</span><span data-ccp-props="{&quot;201341983&quot;:0,&quot;335559739&quot;:160,&quot;335559740&quot;:259}"> </span></p>
<p><span data-contrast="auto">For our project, w</span><span data-contrast="auto">e were interested in assessing the </span><span data-contrast="auto">druggability</span><span data-contrast="auto"> of this pocket and finding out how conserved is this </span><span data-contrast="auto">site both </span><span data-contrast="auto">across </span><span data-contrast="auto">past and </span><span data-contrast="auto">novel </span><span data-contrast="auto">coronaviruses and across SARS-CoV-2 samples</span><span data-contrast="auto"> </span><span data-contrast="auto">from COVID-19 patients. </span><span data-ccp-props="{&quot;201341983&quot;:0,&quot;335559739&quot;:160,&quot;335559740&quot;:259}"> </span></p>
<p><span class="TextRun SCXW204407608 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW204407608 BCX9">To achieve our first aim, we used a tool called sitemap (Schrodinger, NY) to assess the </span></span><span class="TextRun SCXW204407608 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SpellingErrorV2 SCXW204407608 BCX9">druggability</span></span><span class="TextRun SCXW204407608 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW204407608 BCX9"> of </span></span><span class="TextRun SCXW204407608 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW204407608 BCX9">the </span></span><span class="TextRun SCXW204407608 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW204407608 BCX9">ADP-bound pocket of </span></span><span class="TextRun SCXW204407608 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW204407608 BCX9">NiRAN</span></span><span class="TextRun SCXW204407608 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW204407608 BCX9"> domain of </span></span><span class="TextRun SCXW204407608 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SpellingErrorV2 SCXW204407608 BCX9">RdRp</span></span><span class="TextRun SCXW204407608 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW204407608 BCX9"> using the available nsp12 structure in </span></span><span class="TextRun SCXW204407608 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW204407608 BCX9">the </span></span><span class="TextRun SCXW204407608 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW204407608 BCX9">protein data bank (PDB code: </span></span><span class="TextRun SCXW204407608 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW204407608 BCX9">6xez</span></span><span class="TextRun SCXW204407608 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW204407608 BCX9">). </span></span>T<span class="TextRun SCXW2849510 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun CommentStart SCXW2849510 BCX0">he result indicate</span></span><span class="TextRun SCXW2849510 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW2849510 BCX0">s</span></span><span class="TextRun SCXW2849510 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW2849510 BCX0"> that this </span></span><span class="TextRun SCXW2849510 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW2849510 BCX0">highly polar </span></span><span class="TextRun SCXW2849510 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW2849510 BCX0">pocket</span></span><span class="TextRun SCXW2849510 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW2849510 BCX0"> is </span></span><span class="TextRun SCXW2849510 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW2849510 BCX0">a difficult </span></span><span class="TextRun SCXW2849510 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW2849510 BCX0">druggable </span></span><span class="TextRun SCXW2849510 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW2849510 BCX0">site </span></span><span class="TextRun SCXW2849510 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW2849510 BCX0">(</span></span><span class="TextRun SCXW2849510 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SpellingErrorV2 SCXW2849510 BCX0">Dscore</span></span><span class="TextRun SCXW2849510 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW2849510 BCX0"> =</span></span><span class="TextRun SCXW2849510 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW2849510 BCX0"> </span></span><span class="TextRun SCXW2849510 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW2849510 BCX0">0.85</span></span><span class="TextRun SCXW2849510 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW2849510 BCX0">)</span></span><span class="TextRun SCXW2849510 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW2849510 BCX0"> as shown in figure 1. (Halgren, 2009) </span></span><span class="EOP SCXW2849510 BCX0" data-ccp-props="{&quot;201341983&quot;:0,&quot;335559739&quot;:160,&quot;335559740&quot;:259}"> </span></p>
<p>&nbsp;</p>
<div id="attachment_5405" style="width: 985px" class="wp-caption alignnone"><img aria-describedby="caption-attachment-5405" loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/druggability_assessment_ADP-bound_pocket_NiRAN_domain_RdRp-1.png" alt="" width="975" height="471" class="wp-image-5405 size-full" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/druggability_assessment_ADP-bound_pocket_NiRAN_domain_RdRp-1.png 975w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/druggability_assessment_ADP-bound_pocket_NiRAN_domain_RdRp-1-300x145.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/druggability_assessment_ADP-bound_pocket_NiRAN_domain_RdRp-1-768x371.png 768w" sizes="(max-width: 975px) 100vw, 975px" /><p id="caption-attachment-5405" class="wp-caption-text">Figure 1. Druggability assessment of the ADP-bound pocket of NiRAN domain of RdRp.</p></div>
<p><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9">We picked 30 amino acid </span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9">residues </span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9">that li</span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9">n</span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9">e this</span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9"> site </span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9">of</span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9"> interest as identified by their proximity (~2.8</span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9">Å</span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9">) </span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9">to both</span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9"> the</span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9"> bound</span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9"> ADP</span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9"> molecule</span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9"> </span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9">and </span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9">the </span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9">V</span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9">a</span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9">n der Waals boundaries of the corresponding pocket</span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9"> </span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9">mapped by</span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9"> ICM</span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9">.</span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9"> (</span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SpellingErrorV2 SCXW7658465 BCX9">Molsoft</span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9">, S</span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9">an Diego</span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9">)</span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9">.</span></span><span class="TextRun SCXW7658465 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW7658465 BCX9"> (Figure 2, <a href="https://doi.org/10.5281/zenodo.4270496">Zenodo report</a>)</span></span></p>
<p><span class="TextRun SCXW159757282 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW159757282 BCX9">As part of our second aim, w</span></span><span class="TextRun SCXW159757282 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW159757282 BCX9">e conduct</span></span><span class="TextRun SCXW159757282 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW159757282 BCX9">ed</span></span><span class="TextRun SCXW159757282 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW159757282 BCX9"> a broad survey of viral proteins from Alpha- and </span></span><span class="TextRun SCXW159757282 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SpellingErrorV2 SCXW159757282 BCX9">Betacoronaviruses</span></span><span class="TextRun SCXW159757282 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW159757282 BCX9"> to identify the most conserved druggable sites. </span></span><span class="TextRun SCXW159757282 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW159757282 BCX9">By</span></span><span class="TextRun SCXW159757282 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW159757282 BCX9"> integrating variability and </span></span><span class="TextRun SCXW159757282 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SpellingErrorV2 SCXW159757282 BCX9">druggability</span></span><span class="TextRun SCXW159757282 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW159757282 BCX9"> data </span></span><span class="TextRun SCXW159757282 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW159757282 BCX9">we can</span></span><span class="TextRun SCXW159757282 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW159757282 BCX9"> infer the most promising strategies for developing broad-spectrum viral inhibitors</span></span><span class="TextRun SCXW159757282 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW159757282 BCX9"> against coronaviruses</span></span><span class="TextRun SCXW159757282 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW159757282 BCX9">. This analysis can be valuable in the context of the emergence of future coronaviruses circulating in bat reservoir species. </span></span><span class="EOP SCXW159757282 BCX9" data-ccp-props="{&quot;201341983&quot;:0,&quot;335559739&quot;:160,&quot;335559740&quot;:259}"> </span></p>
<p><span class="TextRun SCXW113687124 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW113687124 BCX9">To assess this site&#8217;s genetic variability, we made a diversity dendrogram by performing a multiple sequence alignment using reviewed sequences from </span></span><span class="TextRun SCXW113687124 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SpellingErrorV2 SCXW113687124 BCX9">Uniprot</span></span><span class="TextRun SCXW113687124 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW113687124 BCX9">, of which 6 are from the Alphacoronavirus genus and 21 from the </span><span class="NormalTextRun SpellingErrorV2 SCXW113687124 BCX9">Betacoronavirus</span><span class="NormalTextRun SCXW113687124 BCX9"> genus. (Figure </span></span><span class="TextRun SCXW113687124 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW113687124 BCX9">3</span></span><span class="TextRun SCXW113687124 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW113687124 BCX9">,<a href="https://doi.org/10.5281/zenodo.4270496"> Zenodo report</a>) <span class="TextRun SCXW93444954 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW93444954 BCX9">We observe that 21 out of </span></span><span class="TextRun SCXW93444954 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW93444954 BCX9">3</span></span><span class="TextRun SCXW93444954 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW93444954 BCX9">0</span></span><span class="TextRun SCXW93444954 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW93444954 BCX9"> residues are identical at this site of RdRp which translates into </span></span><span class="TextRun SCXW93444954 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW93444954 BCX9">70</span></span><span class="TextRun SCXW93444954 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW93444954 BCX9">% identity</span></span><span class="TextRun SCXW93444954 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW93444954 BCX9">. </span></span></span></span></p>
<p><span class="TextRun SCXW31895145 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW31895145 BCX9">As a follow-up to assessing the genetic variability among Alphacoronavirus and </span></span><span class="TextRun SCXW31895145 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SpellingErrorV2 SCXW31895145 BCX9">Betacoronavirus</span></span><span class="TextRun SCXW31895145 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW31895145 BCX9"> entries, </span></span><span class="TextRun SCXW31895145 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW31895145 BCX9">we also look</span></span><span class="TextRun SCXW31895145 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW31895145 BCX9">ed</span></span><span class="TextRun SCXW31895145 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW31895145 BCX9"> at the conservation of this site across SARS-CoV-2 samples. O</span></span><span class="TextRun SCXW31895145 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW31895145 BCX9">ur collaborator, Nicola De Maio, from Nick Goldman’s lab at the European Bioinformatics Institute has identified the non-synonymous mutations at </span></span><span class="TextRun SCXW31895145 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW31895145 BCX9">ADP-bound pocket of </span></span><span class="TextRun SCXW31895145 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW31895145 BCX9">NiRAN</span></span><span class="TextRun SCXW31895145 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW31895145 BCX9"> domain of </span></span><span class="TextRun SCXW31895145 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SpellingErrorV2 SCXW31895145 BCX9">RdRp</span></span><span class="TextRun SCXW31895145 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW31895145 BCX9"> </span></span><span class="TextRun SCXW31895145 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW31895145 BCX9">across more than </span></span><span class="TextRun SCXW31895145 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW31895145 BCX9">39000</span></span><span class="TextRun SCXW31895145 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW31895145 BCX9"> SARS-CoV-2 samples. Altogether he identified non-synonymous mutations at </span></span><span class="TextRun SCXW31895145 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW31895145 BCX9">6</span></span><span class="TextRun SCXW31895145 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW31895145 BCX9"> unique amino acid residues consisting of </span></span><span class="TextRun SCXW31895145 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW31895145 BCX9">6</span></span><span class="TextRun SCXW31895145 BCX9" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW31895145 BCX9"> unique non-synonymous variants. In table 1 of the <a href="https://doi.org/10.5281/zenodo.4270496">Zenodo report</a>, there are details about the mutation events underlying these variants. </span></span><span class="EOP SCXW31895145 BCX9" data-ccp-props="{&quot;201341983&quot;:0,&quot;335559739&quot;:160,&quot;335559740&quot;:259}"> </span></p>
<p><b><span data-contrast="auto">Comparison with previously analyzed </span></b><b><span data-contrast="auto">RdRp </span></b><b><span data-contrast="auto">pockets, <a href="https://doi.org/10.5281/zenodo.4041302">RdRp active site,</a> and <a href="https://doi.org/10.5281/zenodo.4247424">allosteric site (M666 pocket</a>):</span></b><span data-ccp-props="{&quot;201341983&quot;:0,&quot;335559739&quot;:160,&quot;335559740&quot;:259}"> </span></p>
<p>We previously analyzed the druggability and conservation of the catalytic site and an ectopic site (that we called M666 pocket) of RdRp. We find that the and ectopic pocket (Dscore = 1.03) is more druggable than the catalytic pocket (Dscore 0.86) occupied by RNA-conjugated Rmedesivir and NiRAN pocket (Dscore=0.85).</p>
<p><span data-contrast="auto">On the other end, the catalytic site o</span><span data-contrast="auto">f</span><span data-contrast="auto"> </span><span data-contrast="auto">RdRp</span><span data-contrast="auto"> is among the most conserved </span><span data-contrast="auto">pockets </span><span data-contrast="auto">that </span><span data-contrast="auto">we have seen across SARS-CoV-2 protein structures</span><span data-contrast="auto"> (83% conserved </span><span data-contrast="auto">in coronaviruses)</span><span data-contrast="auto"> follow</span><span data-contrast="auto">ed by </span><span data-contrast="auto">the </span><span data-contrast="auto">NiRAN</span><span data-contrast="auto"> pocket (</span><span data-contrast="auto">68% conserved) and the M666 pocket (</span><span data-contrast="auto">33% conserved).</span><span data-ccp-props="{&quot;201341983&quot;:0,&quot;335559739&quot;:160,&quot;335559740&quot;:259}"> </span></p>
<p><span data-contrast="auto">We observe a related</span><span data-contrast="auto"> trend when looking at mutation rates a</span><span data-contrast="auto">cross COVID-19 samples:</span><span data-contrast="auto"> </span><span data-contrast="auto">t</span><span data-contrast="auto">he </span><span data-contrast="auto">catalytic site and </span><span data-contrast="auto">NiRAN</span><span data-contrast="auto"> pocket were genetically more stable, with</span><span data-contrast="auto"> 6 unique </span><span data-contrast="auto">mutations</span><span data-contrast="auto"> </span><span data-contrast="auto">found in</span><span data-contrast="auto"> 6</span><span data-contrast="auto"> </span><span data-contrast="auto">samples</span><span data-contrast="auto"> each</span><span data-contrast="auto">, while the M666 pocket w</span><span data-contrast="auto">as more </span><span data-contrast="auto">prone to mutations (10 mutations found in </span><span data-contrast="auto">13 </span><span data-contrast="auto">variants</span><span data-contrast="auto">)</span><span data-contrast="auto">.</span><span data-ccp-props="{&quot;201341983&quot;:0,&quot;335559739&quot;:160,&quot;335559740&quot;:259}"> </span></p>
<p><span class="TextRun SCXW154553115 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun CommentStart SCXW154553115 BCX0">A</span></span><span class="TextRun SCXW154553115 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW154553115 BCX0"> strategy may be to develop compounds </span></span><span class="TextRun SCXW154553115 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW154553115 BCX0">that include a chelating moiety to engage the Mg2+ </span></span><span class="TextRun SCXW154553115 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW154553115 BCX0">observed</span></span><span class="TextRun SCXW154553115 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW154553115 BCX0"> in</span></span><span class="TextRun SCXW154553115 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW154553115 BCX0"> the ADP-bound</span></span><span class="TextRun SCXW154553115 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW154553115 BCX0"> </span></span><span class="TextRun SCXW154553115 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW154553115 BCX0">NiRAN pocket</span></span><span class="TextRun SCXW154553115 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW154553115 BCX0">. </span></span><span class="TextRun SCXW154553115 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW154553115 BCX0">This site</span></span><span class="TextRun SCXW154553115 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW154553115 BCX0"> </span></span><span class="TextRun SCXW154553115 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW154553115 BCX0">has</span></span><span class="TextRun SCXW154553115 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW154553115 BCX0"> </span></span><span class="TextRun SCXW154553115 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW154553115 BCX0">reasonable genetic stability and is </span></span><span class="TextRun SCXW154553115 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW154553115 BCX0">potentially </span></span><span class="TextRun SCXW154553115 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW154553115 BCX0">an interesting </span></span><span class="TextRun SCXW154553115 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW154553115 BCX0">target for the development of </span></span><span class="TextRun SCXW154553115 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW154553115 BCX0">broad-spectrum</span></span><span class="TextRun SCXW154553115 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW154553115 BCX0"> inhibitors</span></span><span class="TextRun SCXW154553115 BCX0" lang="EN-US" xml:lang="EN-US" data-contrast="auto"><span class="NormalTextRun SCXW154553115 BCX0">. </span></span><span class="EOP SCXW154553115 BCX0" data-ccp-props="{&quot;201341983&quot;:0,&quot;335559739&quot;:160,&quot;335559740&quot;:259}"> </span></p>
<p>You can view this report on <a href="https://doi.org/10.5281/zenodo.4270496">Zenodo</a> as well.</p>
<p>Please contact me via the “Leave a comment” link at the top of this post. Stay Tuned for more updates on this project.</p>
<p><b><span data-contrast="auto">References</span></b><span data-ccp-props="{&quot;201341983&quot;:0,&quot;335559739&quot;:160,&quot;335559740&quot;:259}"> </span></p>
<p><span data-contrast="auto">Chen, J., Malone, B., Llewellyn, E., Grasso, M., Shelton, P. M., </span><span data-contrast="auto">Olinares</span><span data-contrast="auto">, P. D., Maruthi, K., </span><span data-contrast="auto">Eng</span><span data-contrast="auto">, E., </span><span data-contrast="auto">Vatandaslar</span><span data-contrast="auto">, H., Chait, B. T., Kapoor, T., Darst, S. A., &amp; Campbell, E. A. (2020). Structural basis for helicase-polymerase coupling in the SARS-Cov-2 replication-transcription complex. </span><a href="https://doi.org/10.1101/2020.07.08.194084"><span data-contrast="none">https://doi.org/10.1101/2020.07.08.194084</span></a><span data-ccp-props="{&quot;201341983&quot;:0,&quot;335559685&quot;:720,&quot;335559739&quot;:160,&quot;335559740&quot;:259,&quot;335559991&quot;:720}"> </span></p>
<p><span data-contrast="auto">Halgren, T. A. (2009). Identifying and characterizing binding sites and assessing </span><span data-contrast="auto">Druggability</span><span data-contrast="auto">. </span><i><span data-contrast="auto">Journal of Chemical Information and Modeling</span></i><span data-contrast="auto">, </span><i><span data-contrast="auto">49</span></i><span data-contrast="auto">(2), 377-389. </span><a href="https://doi.org/10.1021/ci800324m"><span data-contrast="none">https://doi.org/10.1021/ci800324m</span></a><span data-ccp-props="{&quot;201341983&quot;:0,&quot;335559685&quot;:720,&quot;335559739&quot;:160,&quot;335559740&quot;:259,&quot;335559991&quot;:720}"> </span></p>
<p><span data-contrast="auto">Kirchdoerfer</span><span data-contrast="auto">, R. N., &amp; Ward, A. B. (2019). Structure of the SARS-</span><span data-contrast="auto">Cov</span><span data-contrast="auto"> NSP12 polymerase bound to NSP7 and NSP8 Co-factors. </span><a href="https://doi.org/10.1101/551986"><span data-contrast="none">https://doi.org/10.1101/551986</span></a><span data-ccp-props="{&quot;201341983&quot;:0,&quot;335559685&quot;:720,&quot;335559739&quot;:160,&quot;335559740&quot;:259,&quot;335559991&quot;:720}"> </span></p>
<p><span data-contrast="auto">Lehmann, K. C., Gulyaeva, A., </span><span data-contrast="auto">Zevenhoven</span><span data-contrast="auto">-Dobbe, J. C., Janssen, G., Ruben, M., </span><span data-contrast="auto">Overkleeft</span><span data-contrast="auto">, H. S., Van </span><span data-contrast="auto">Veelen</span><span data-contrast="auto">, P. A., </span><span data-contrast="auto">Samborskiy</span><span data-contrast="auto">, D., Kravchenko, A. A., </span><span data-contrast="auto">Leontovich</span><span data-contrast="auto">, A. M., Sidorov, I. A., Snijder, E. J., Posthuma, C. C., &amp; </span><span data-contrast="auto">Gorbalenya</span><span data-contrast="auto">, A. E. (2015). undefined. </span><i><span data-contrast="auto">Nucleic Acids Research</span></i><span data-contrast="auto">, </span><i><span data-contrast="auto">43</span></i><span data-contrast="auto">(17), 8416-8434. </span><a href="https://doi.org/10.1093/nar/gkv838"><span data-contrast="none">https://doi.org/10.1093/nar/gkv838</span></a><span data-contrast="none"> </span><span data-ccp-props="{&quot;201341983&quot;:0,&quot;335559685&quot;:720,&quot;335559739&quot;:160,&quot;335559740&quot;:259,&quot;335559991&quot;:720}"> </span></p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/druggability-and-genetic-variability-of-the-adp-bound-pocket-of-sars-cov-2-rna-dependent-rna-polymerase-niran-domain-across-coronaviruses-and-sars-cov-2-samples-post-22/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>Production and purification of Ectodomain of SARS-CoV-2 Spike (S) protein</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/production-and-purification-of-ectodomain-of-sars-cov-2-spike-s-protein/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/production-and-purification-of-ectodomain-of-sars-cov-2-spike-s-protein/#respond</comments>
		
		<dc:creator><![CDATA[Alma Seitova]]></dc:creator>
		<pubDate>Sat, 07 Nov 2020 23:35:17 +0000</pubDate>
				<category><![CDATA[Alma Seitova]]></category>
		<category><![CDATA[COVID-19]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=5353</guid>

					<description><![CDATA[Production and purification of Ectodomain of SARS-CoV-2 Spike (S) protein Two different expression plasmids for prefusion S ectodomain residues 1−1208 of 2019-nCoV S (GenBank: MN908947) with proline substitutions (2P) at residues 986 and 987(1) and with six proline substitutions (HexaPro (at residues 817, 892, 899, 942, 986 and 987 (2), a “GSAS” substitution at the <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/production-and-purification-of-ectodomain-of-sars-cov-2-spike-s-protein/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p></a><strong>Production and purification of Ectodomain of SARS-CoV-2 Spike (S) protein</strong><br />
      Two different expression plasmids for prefusion S ectodomain residues 1−1208 of 2019-nCoV S (GenBank: MN908947) with proline substitutions (2P) at residues 986 and 987(1) and with six proline substitutions (HexaPro (at residues 817, 892, 899, 942, 986 and 987 (2), a “GSAS” substitution at the furin cleavage site (residues 682–685), a C-terminal T4 fibritin trimerization motif, an HRV3C protease cleavage site, a TwinStrepTag and an 8XHisTag in the mammalian expression vector pαH were generously gifted by Dr.McLellan’s lab (1,2). </p>
<p>I. Production and purification of the 2P version of S-Ectodomain.<br />
During the first purifications from a small-scale production we have faced some challenges with the 2P version of this protein. (Figure 1.) </p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/1as.png" alt="" width="1032" height="575" class="alignnone size-full wp-image-5354" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/1as.png 1032w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/1as-300x167.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/1as-1024x571.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/1as-768x428.png 768w" sizes="(max-width: 1032px) 100vw, 1032px" /></p>
<p>Purification from a big scale-up production using another buffer (1x PBS) led to the same problems, and we evaluated protein concentrations by Pierce BCA Protein Assay (Please see Figure 2&#038;3) and comparative gel analysis. </p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Figure-2.png" alt="" width="1088" height="612" class="alignnone size-full wp-image-5362" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Figure-2.png 1088w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Figure-2-300x169.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Figure-2-1024x576.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Figure-2-768x432.png 768w" sizes="(max-width: 1088px) 100vw, 1088px" /></p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Figure-3.-as.png" alt="" width="1105" height="622" class="alignnone size-full wp-image-5368" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Figure-3.-as.png 1105w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Figure-3.-as-300x169.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Figure-3.-as-1024x576.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Figure-3.-as-768x432.png 768w" sizes="(max-width: 1105px) 100vw, 1105px" /></p>
<p>We have obtained total 12mg of Histag purified and dialyzed protein and run 9mg of protein  on the size-exclusion column. SEC profiles showed that a big portion of the protein were in aggregates (Figure 4).</p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Figure-4.-AS.png" alt="" width="1116" height="628" class="alignnone size-full wp-image-5364" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Figure-4.-AS.png 1116w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Figure-4.-AS-300x169.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Figure-4.-AS-1024x576.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Figure-4.-AS-768x432.png 768w" sizes="(max-width: 1116px) 100vw, 1116px" /></p>
<p>On the size-exclusion profile we observed 2 peaks of the aggregates and collected only a 3rd peak corresponding to the trimer of S- Ectodomain running close to 670kD.<br />
Based on the results of the purifications by affinity, size-exclusion protocols and exchanging notes with a Biotechnology group of SGC Oxford, we’ve come to conclusions that S- Ectodomain is prone to the aggregation almost at the every stage of purification, one nucleation of the aggregates “ curls up” a most part of the protein.<br />
We have optimised a few steps in the production and purification of this target to prevent or minimize aggregation:<br />
• Cultured cells were harvested in a different hours of the post-transfection times<br />
• Harvested culture supernatant clarified by two steps of centrifugation at RT was passed through a<br />
  0.22 μm filter before IMAC batch absorption<br />
• After IMAC purification, combined peak fractions were filtered through a 0.22 μm filter, before<br />
  concentrating and loading to the SEC<br />
• Concentrated combined fractions of the eluted protein at 16 degree celsius and  run on SEC<br />
Size-exclusion profiles of S-Ectodomain from these 4 purifications shown no major aggregates and a main fraction of the protein was corresponding to the expected MV as 670kD (Figure 5.), but a final yield of the purified by two-steps protein were very low as 0.2-0.3mg/L.  </p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Figure-5.-AS.png" alt="" width="1035" height="583" class="alignnone size-full wp-image-5365" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Figure-5.-AS.png 1035w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Figure-5.-AS-300x169.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Figure-5.-AS-1024x577.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Figure-5.-AS-768x433.png 768w" sizes="(max-width: 1035px) 100vw, 1035px" /></p>
<p>We have run protein standards on the SEC column used for this protein to confirm accuracy of the MV of the eluted protein (Figure 6.) </p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Figure-6.-as.png" alt="" width="1116" height="628" class="alignnone size-full wp-image-5366" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Figure-6.-as.png 1116w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Figure-6.-as-300x169.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Figure-6.-as-1024x576.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Figure-6.-as-768x432.png 768w" sizes="(max-width: 1116px) 100vw, 1116px" /></p>
<p>II. Production and purification of the HexaPro version of S-Ectodomain<br />
We have used the same optimized protocol for the HexaPro version of the S-Ectodomain (Figure 7).</p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Figure-7.-.png" alt="" width="1101" height="620" class="alignnone size-full wp-image-5370" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Figure-7.-.png 1101w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Figure-7.--300x169.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Figure-7.--1024x577.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/Figure-7.--768x432.png 768w" sizes="(max-width: 1101px) 100vw, 1101px" /></p>
<p>Final yield for this constructs was higher than for 2P version of S-Ectodomain, we’ve obtained 1mg/L from the first 800ml of production by transient transfection of EXPI293 cells. Further optimization might improve a final yield to the higher expression for this construct, but so far we&#8217;ve done only one production and purification trial with HexaPro. </p>
<p>References:<br />
1.Cryo-EM structure of the 2019-nCoV spike in the prefusion conformation<br />
Daniel Wrapp, Nianshuang Wang, Kizzmekia S. Corbett, Jory A. Goldsmith, Ching-Lin Hsieh, Olubukola Abiona, Barney S. Graham, Jason S. McLellan<br />
Science 367, 1260–1263 (2020) 13 March 2020</p>
<p>2. Structure-based design of prefusion-stabilized SARS-CoV-2 spikes<br />
Ching-Lin Hsieh, Jory A. Goldsmith, Jeffrey M. Schaub, Andrea M. DiVenere, Hung-Che Kuo,<br />
Kamyab Javanmardi, Kevin C. Le, Daniel Wrapp, Alison G. Lee, Yutong Liu, Chia-Wei Chou,<br />
Patrick O. Byrne, Christy K. Hjorth, Nicole V. Johnson, John Ludes-Meyers, Annalee W. Nguyen,<br />
Juyeon Park1, Nianshuang Wang1, Dzifa Amengor1, Jason J. Lavinder1,2, Gregory C. Ippolito,<br />
Jennifer A. Maynard, Ilya J. Finkelstein, Jason S. McLellan<br />
Science 369, 1501–1505 (2020) 18 September 2020              </p>
<p><a href="https://zenodo.org/record/4256998#.X6cqUS0ZPBI">Please see Zenodo page for the production and purification protocols</a></p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/production-and-purification-of-ectodomain-of-sars-cov-2-spike-s-protein/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>Genetic Variability at An Allosteric site of SARS-CoV-2 RNA-dependent RNA polymerase Across Coronaviruses and SARS-CoV-2 Samples &#8211; Post 21</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/genetic-variability-at-an-allosteric-site-of-sars-cov-2-rna-dependent-rna-polymerase-across-coronaviruses-and-sars-cov-2-samples/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/genetic-variability-at-an-allosteric-site-of-sars-cov-2-rna-dependent-rna-polymerase-across-coronaviruses-and-sars-cov-2-samples/#respond</comments>
		
		<dc:creator><![CDATA[Setayesh Yazdani]]></dc:creator>
		<pubDate>Thu, 05 Nov 2020 00:08:24 +0000</pubDate>
				<category><![CDATA[COVID-19]]></category>
		<category><![CDATA[SARS-CoV-2 computational genomics]]></category>
		<category><![CDATA[Setayesh Yazdani]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=5319</guid>

					<description><![CDATA[SARS-CoV-2 is a positive-strand RNA virus, depending on its multi-subunit machinery to replicate its RNA. This machinery is known as RNA-dependent RNA polymerase (RdRp). The catalytic subunit of RdRp, which is the core component of this machinery, is called nsp12. Nsp12 has little catalytic activity on its own and relies on accessory subunits to have <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/genetic-variability-at-an-allosteric-site-of-sars-cov-2-rna-dependent-rna-polymerase-across-coronaviruses-and-sars-cov-2-samples/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p>SARS-CoV-2 is a positive-strand RNA virus, depending on its multi-subunit machinery to replicate its RNA. This machinery is known as RNA-dependent RNA polymerase (RdRp). The catalytic subunit of RdRp, which is the core component of this machinery, is called nsp12. Nsp12 has little catalytic activity on its own and relies on accessory subunits to have full activity. These accessory factors, nsp7 and nsp8, increase RdRp template binding and processivity. (Yin et al., 2020)</p>
<p>RdRp is a well-established target for a class of antiviral drugs known as nucleotide analogs. Remdesivir is an example of such a group of drugs that inhibits RdRp. Remdesivir has been authorized to be used for special COVID-19 cases with severe symptoms in Canada. (https://healthycanadians.gc.ca/recall-alert-rappel-avis/hc-sc/2020/73621a-eng.php)</p>
<p>We previously analyzed the remdesivir-binding site of RdRp. We then shifted our focus on another druggable pocket of the RdRp complex, an allosteric site on the RdRp complex, which is this post&#8217;s topic.</p>
<p>In figure 1, we show the RdRp complex and highlight the site of interest, which is lined by amino acid residue M666 and which we refer to here as the M666 pocket. This site has a druggability score of 1.008 based on our results from Sitemap (Schrodinger, NY), indicating a druggable site (figure 2, <a href="https://doi.org/10.5281/zenodo.4247424">Zenodo report</a>).</p>
<p>M666 site is at the nsp8 binding site of nsp12. A compound binding this site with high potency may antagonize the formation of an active RdRp complex. It may inhibit the RdRp function, which is necessary for viral replication.</p>
<p>&nbsp;</p>
<div id="attachment_5321" style="width: 636px" class="wp-caption alignnone"><img aria-describedby="caption-attachment-5321" loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/nsp12-nsp82-nsp7.png" alt="" width="626" height="775" class="wp-image-5321 size-full" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/nsp12-nsp82-nsp7.png 626w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/nsp12-nsp82-nsp7-242x300.png 242w" sizes="(max-width: 626px) 100vw, 626px" /><p id="caption-attachment-5321" class="wp-caption-text">Figure 1. The apo nsp12-nsp7-nsp8 RdRp complex and the positioning of the M666 pocket at the interface of nsp12-nsp8.</p></div>
<p>&nbsp;</p>
<p>We then selected the residues surrounding this pocket in proximity to bound nsp8. The RdRp M666 pocket is lined by 24 amino acid sidechains (figure 3, <a href="https://doi.org/10.5281/zenodo.4247424">Zenodo report</a>).</p>
<p>We then conduct a broad survey of viral proteins from Alpha- and Betacoronaviruses to identify the most conserved druggable sites. We are integrating variability and druggability data to infer the most promising strategies for developing broad-spectrum viral inhibitors. This analysis can be valuable in the context of the emergence of future coronaviruses circulating in bat reservoir species.</p>
<p>To assess this site&#8217;s genetic variability, we first looked at 27 reviewed sequences from Uniprot, of which 6 are from the Alphacoronavirus genus and 21 from the Betacoronavirus genus. Altogether, eight out of twenty-four residues at M666 were identical across the 27 entries, equivalent to 33% identity at this site (figure 4, <a href="https://doi.org/10.5281/zenodo.4247424">Zenodo report</a>).</p>
<p>As a follow-up to assessing the genetic variability among Alphacoronavirus and Betacoronavirus entries, our collaborator, Nicola De Maio, from Nick Goldman’s lab at the European Bioinformatics Institute has identified the non-synonymous mutations at M666 sites across more than 15000 SARS-CoV-2 samples. Altogether he identified non-synonymous mutations at 10 unique amino acid residues consisting of 13 unique non-synonymous variants. In table 1 on our <a href="https://doi.org/10.5281/zenodo.4247424">Zenodo report</a>, there are details about the mutation events underlying these variants.</p>
<p>In terms of druggability, the M666 pocket (Dscore=1.028) is superior to the RdRp active site (where remdesivir binds) (Dscore=0.859). While the active site is highly conserved (83% identity), the M666 pocket is poorly conserved (33% identity). Similarly, across SARS-CoV-2 samples at the M666 site, there is high genetic variability compared to the remdesivir site, which is more genetically stable; only 6 out of 48 residues were mutated.</p>
<p>Remdesivir use for COVID-19 is only recommended for hospitalized patients who need supplemental oxygen therapy (Clinical Trials.gov number, NCT04280705), and only in those severe cases, remdesivir was superior to placebo in shortening the time to recovery. (Goldman et al., 2020) This phase3 study showed that despite remdesivir, there was still high mortality because of COVID-19.  Therefore, clinically we still need a better drug to treat COVID-19 through different approaches.  (https://www.nejm.org/doi/full/10.1056/NEJMoa2007764)</p>
<p>Targeting the M666 provides an alternate strategy to target RdRp function by disturbing its interaction with a key accessory factor, nsp8. It is important to note the high genetic variability of this site among coronaviruses and SARS-CoV-2 samples, which deprioritizes it as a target for pan-coronavirus anti-SARS-CoV-2 drugs.</p>
<p>You can view this report on <a href="https://doi.org/10.5281/zenodo.4247424">Zenodo</a> as well.</p>
<p>Please contact me via the “Leave a comment” link at the top of this post. Stay Tuned for more updates on this project.</p>
<p><strong>References:</strong></p>
<p>Yin, W., Mao, C., Luan, X., Shen, D., Shen, Q., Su, H., Wang, X., Zhou, F., Zhao, W., Gao, M., Chang, S., Xie, Y., Tian, G., Jiang, H., Tao, S., Shen, J., Jiang, Y., Jiang, H., Xu, Y., &amp; Xu, H. (2020). Structural basis for inhibition of the RNA-dependent RNA polymerase from SARS-CoV-2 by remdesivir. <em>Science</em>, <em>368</em>(6498), 1499-1504. <a href="https://doi.org/10.1126/science.abc1560">https://doi.org/10.1126/science.abc1560</a></p>
<p>Goldman, J. D., Lye, D. C.B., Hui, D. S., Marks, K. M., Bruno, R., Montejano, R., Spinner, C. D., Galli, M., Ahn, M., Nahass, R. G., Chen, Y., SenGupta, D., Hyland, R. H., Osinusi, A. O., Cao, H., Blair, C., Wei, X., Gaggar, A., Brainard, D. M., &amp; Subramanian, A. (2020). Remdesivir for 5 or 10 Days in Patients with Severe Covid-19. <em>The New England Journal of Medicine</em>. <a href="https://doi.org/10.1056/NEJMoa2015301">https://doi.org/10.1056/NEJMoa2015301</a></p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/genetic-variability-at-an-allosteric-site-of-sars-cov-2-rna-dependent-rna-polymerase-across-coronaviruses-and-sars-cov-2-samples/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>Production and purification of recombinant SARS COV-2 RBDSD1-Fc protein</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/production-and-purification-of-recombinant-sars-cov-2-rbdsd1-fc-protein/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/production-and-purification-of-recombinant-sars-cov-2-rbdsd1-fc-protein/#respond</comments>
		
		<dc:creator><![CDATA[Alma Seitova]]></dc:creator>
		<pubDate>Sun, 18 Oct 2020 12:32:26 +0000</pubDate>
				<category><![CDATA[Alma Seitova]]></category>
		<category><![CDATA[COVID-19]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=5250</guid>

					<description><![CDATA[Production and purification of recombinant SARS COV-2 RBDSD1-Fc protein from a transiently transfected EXPI293 mammalian cells. Expression plasmid for CoV-9-RBDSD1, residues of RBD R319−S591 with a C-terminal HRV3C protease cleavage site, a monomeric Fc tag and an 8XHisTag was generously gifted by Dr. McLellan’s lab (1). Secreted proteins were purified from a suspension culture cell <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/production-and-purification-of-recombinant-sars-cov-2-rbdsd1-fc-protein/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p><strong>Production and purification of recombinant SARS COV-2 RBDSD1-Fc protein from a transiently transfected EXPI293 mammalian cells.<br />
Expression plasmid for CoV-9-RBDSD1, residues of RBD R319−S591 with a C-terminal HRV3C protease cleavage site, a monomeric Fc tag and an 8XHisTag was generously gifted by Dr. McLellan’s lab (1). Secreted proteins were purified from a suspension culture cell supernatant with an expression level 15-20mg/L. For different application RBD-SD1 was purified by affinity and gel-filtration protocols, with and without purification tag (Figure 1.) </p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/RBDSD1.png" alt="" width="900" height="507" class="alignnone size-full wp-image-5242" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/RBDSD1.png 900w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/RBDSD1-300x169.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/RBDSD1-768x433.png 768w" sizes="(max-width: 900px) 100vw, 900px" /></p>
<p>Reference: 1.Cryo-EM structure of the 2019-nCoV spike in the prefusion conformation. Daniel Wrapp, Nianshuang Wang, Kizzmekia S. Corbett,<a href="https://zenodo.org/record/4085730#.X4crSdBKhaQ"></a>A. Goldsmith, Ching-Lin Hsieh,Olubukola Abiona, Barney S. Graham, Jason S. McLellan .Science 367, 1260–1263 (2020) 13 March 2020</p>
<p><a href="https://zenodo.org/record/4091746#.X4zVwC0ZNAY">Please visit corresponding Zenodo page for &#8220;Materials and Methods&#8221;</a></p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/production-and-purification-of-recombinant-sars-cov-2-rbdsd1-fc-protein/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>Production and purifications of the SARS-CoV-2 related proteins from the transiently transfected mammalian cells</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/production-and-purifications-of-the-sars-cov-2-related-proteins-from-the-transiently-transfected-mammalian-cells/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/production-and-purifications-of-the-sars-cov-2-related-proteins-from-the-transiently-transfected-mammalian-cells/#respond</comments>
		
		<dc:creator><![CDATA[Alma Seitova]]></dc:creator>
		<pubDate>Wed, 14 Oct 2020 16:48:32 +0000</pubDate>
				<category><![CDATA[Alma Seitova]]></category>
		<category><![CDATA[COVID-19]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=5215</guid>

					<description><![CDATA[Over the last few months, we have all become familiar with the COVID-19 global pandemic. The coronavirus is “decorated” with the Spike (S) protein, one of the 25 proteins of SARS-CoV-2 that are encoded in the viral genome. The infection starts with the attachment of the virus to the cell membrane to enter host cells. <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/production-and-purifications-of-the-sars-cov-2-related-proteins-from-the-transiently-transfected-mammalian-cells/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p>Over the last few months, we have all become familiar with the COVID-19 global pandemic. The coronavirus is “decorated” with the Spike (S) protein, one of the 25 proteins of SARS-CoV-2 that are encoded in the viral genome.<br />
   The infection starts with the attachment of the virus to the cell membrane to enter host cells.  S- protein mediates infection by interacting with the human angiotensin-converting enzyme 2 (ACE2) receptor (Figure 1).  This interaction occurs when the receptor-binding domain (RBD) of the S- protein which recognizes and binds ACE2. This interaction mediates membrane fusion and viral entry into human cells.</p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/Covid-image.png" alt="" width="576" height="324" class="alignnone size-full wp-image-5216" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/Covid-image.png 576w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/Covid-image-300x169.png 300w" sizes="(max-width: 576px) 100vw, 576px" /></p>
<p>So, it is no wonder why the S-protein, RBD of S-protein and human ACE2 became key targets in Covid-19 related scientific research and in the worldwide collaborative efforts to develop diagnostics such as therapeutic antibodies or a vaccine.</p>
<p>To support the fight against COVID-19 and fulfill the high demand for these proteins, I volunteered, together with my colleague, Ashley Hutchinson to produce and purify high-quality S-protein, RBD and human ACE2 in mammalian cells. It is not a trivial contribution. ACE2 protein, for example, would usually cost hundreds of dollars per ~10 µg. We started this challenging journey back in April, during the lockdown of the city and of the University of Toronto.  Since then, 64 aliquots of our products have been distributed among different labs in Toronto, across Canada and abroad. We are very happy to have feedback from researches indicating that our products are performing well in the projects and in some cases even better than proteins obtained from commercial sources. </p>
<p>Please visit corresponding <a href="https://zenodo.org/record/4085730#.X4crSdBKhaQ">Zenodo</a> page for the “Materials and Methods” and details of how we produced and purified these targets to share with the global research community. </p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/production-and-purifications-of-the-sars-cov-2-related-proteins-from-the-transiently-transfected-mammalian-cells/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>Genetic Variability at the Remdesivir-binding Pocket of SARS-CoV-2 RNA-dependent RNA polymerase Across Coronaviruses and SARS-CoV-2 Samples &#8211; Post 20</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/genetic-variability-at-the-remdesivir-binding-pocket-of-sars-cov-2-rna-dependent-rna-polymerase-across-coronaviruses-and-sars-cov-2-samples-post-20/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/genetic-variability-at-the-remdesivir-binding-pocket-of-sars-cov-2-rna-dependent-rna-polymerase-across-coronaviruses-and-sars-cov-2-samples-post-20/#respond</comments>
		
		<dc:creator><![CDATA[Setayesh Yazdani]]></dc:creator>
		<pubDate>Mon, 21 Sep 2020 13:09:33 +0000</pubDate>
				<category><![CDATA[COVID-19]]></category>
		<category><![CDATA[SARS-CoV-2 computational genomics]]></category>
		<category><![CDATA[Setayesh Yazdani]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=5176</guid>

					<description><![CDATA[In my previous post 19, I showed how we assessed the druggability of the remdesivir-binding site of SARS-CoV-2 RNA-dependent RNA polymerase (RdRp) and determined the residues lining that site. In this post, we analyze the genetic diversity of RdRp catalytic site across Alpha- and Betacoronavirus entries from UniProt and among SARS-CoV-2 samples. In the context <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/genetic-variability-at-the-remdesivir-binding-pocket-of-sars-cov-2-rna-dependent-rna-polymerase-across-coronaviruses-and-sars-cov-2-samples-post-20/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p>In my previous post <a href="https://thesgc.github.io/static-openlabnotebooks/the-next-target-sars-cov-2-rna-dependent-rna-polymerase-and-its-druggability-post-19/">19</a>, I showed how we assessed the druggability of the remdesivir-binding site of SARS-CoV-2 RNA-dependent RNA polymerase (RdRp) and determined the residues lining that site. In this post, we analyze the genetic diversity of RdRp catalytic site across Alpha- and Betacoronavirus entries from UniProt and among SARS-CoV-2 samples.</p>
<p>In the context of the emergence of future MERS-like or SARS-like coronaviruses from the bat strains circulating in bat reservoir species, it is imperative to do a broad survey of viral proteins to identify the best strategies for the development of broad-spectrum viral inhibitors. (Sheahan et al., 2020) For this exact reason, we are looking at twenty-seven reviewed sequences from Uniprot, where six belong to entries of the Alphacoronavirus genus and twenty-one belong to the Betacoronavirus genus. We then assess the variability of amino acid lining the binding pockets found at the RdRP active site (PDB: 7bv2).</p>
<p>I made a diversity dendrogram focusing on the <a href="https://thesgc.github.io/static-openlabnotebooks/the-next-target-sars-cov-2-rna-dependent-rna-polymerase-and-its-druggability-post-19/">48 amino acid sidechains lining the pockets</a> juxtaposed to Remdesivir. Shown below in figure 1, the consensus profile at each of the 48 sidechains is shown. Strikingly, 83% of residues, 40 out of 48, are conserved across these 27 entries. This is one of the most conserved SARS-CoV-2 binding sites that we have seen so far (I will post a systematic comparison once I have reviewed all sites on all SARS-CoV-2 proteins in the PDB). Here are the 8 non-conserved residues: F442, A443, Y453, Y455, M543, N552, A558, S681. Across the 21 entries of the Betacoronavirus genus (including SARS-CoV-2), 46 out of 48 residues are conserved at the RdRp active site. Here are the non-conserved residues: F442 (almost perfectly conserved), and A443.</p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/09/RdRp-alpha-beta-dendrogram.png" alt="" width="1616" height="760" class="alignnone wp-image-5184 size-full" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/09/RdRp-alpha-beta-dendrogram.png 1616w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/09/RdRp-alpha-beta-dendrogram-300x141.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/09/RdRp-alpha-beta-dendrogram-1024x482.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/09/RdRp-alpha-beta-dendrogram-768x361.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/09/RdRp-alpha-beta-dendrogram-1536x722.png 1536w" sizes="(max-width: 1616px) 100vw, 1616px" /></p>
<p><strong>Figure 1. </strong><strong>The diversity dendrogram of the active site of RdRp for the entries of Alpha- and Betacoronavirus genera</strong><strong>.</strong></p>
<p>We mapped the variations shown among the entries of Alpha- and Betacoronavirus genera onto the SARS-CoV-2 RdRp crystal structure (PDB: 7bv2) such that sidechains that are variable among the entries are highlighted in orange and the conserved sidechains are shown in green.</p>
<p>&nbsp;</p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/09/color-coded-RdRp-alpha-beta.png" alt="" width="975" height="684" class="alignnone wp-image-5191 size-full" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/09/color-coded-RdRp-alpha-beta.png 975w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/09/color-coded-RdRp-alpha-beta-300x210.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/09/color-coded-RdRp-alpha-beta-768x539.png 768w" sizes="(max-width: 975px) 100vw, 975px" /></p>
<p><strong>Figure 2. Mapping the genetic variation of UniProt entries belonging to Alpha- and Betacoronavirus genera onto SARS-CoV-2 RdRp Crystal structure (PDB: 7bv2). </strong>The eight non-conserved residues among the entries of both Alpha- and Betacoronavirus include F442, A443, Y453, Y455, M543, N552, A558, S681 while the two non-conserved residues among the Betacoronavirus entries include F442, A443.</p>
<p>Our next step was to assess the genetic variability of the catalytic site of RdRp among SARS-CoV-2 samples. Nicola De Maio, our collaborator from Nick Goldman’s lab at European Bioinformatics Institute, looked at more than 15000 SARS-CoV-2 samples and identified all mutations at the RdRp catalytic site pockets. He identified 6 non-synonymous variants as shown in table 1. For example, at residue number 443, the wild-type sidechain in SARS-CoV-2 RdRp is alanine. In the sample batch Nicola has looked at, he saw that 15870 samples had alanine at that position while there were 2 samples with valine at this exact position.</p>
<table width="629" style="width: 75.6566%;" height="365">
<tbody>
<tr>
<td width="47" style="width: 8.60861%;">Index</td>
<td width="123" style="width: 13.013%;">Non-synonymous Variants</td>
<td width="129" style="width: 12.1121%;">variants</td>
<td width="117" style="width: 11.9119%;">codon one</td>
<td width="108" style="width: 12.3123%;">codon two</td>
<td width="105" style="width: 14.3143%;">codon three</td>
</tr>
<tr>
<td width="47" style="width: 8.60861%;">1</td>
<td width="123" style="width: 13.013%;">A443</td>
<td width="129" style="width: 12.1121%;">A(15870), V(2)</td>
<td width="117" style="width: 11.9119%;">g(15872)</td>
<td width="108" style="width: 12.3123%;">c(15870), t(2)</td>
<td width="105" style="width: 14.3143%;">t(15872)</td>
</tr>
<tr>
<td width="47" style="width: 8.60861%;">2</td>
<td width="123" style="width: 13.013%;">Q444</td>
<td width="129" style="width: 12.1121%;">Q(15870), H(2)</td>
<td width="117" style="width: 11.9119%;">c(15872)</td>
<td width="108" style="width: 12.3123%;">a(15872)</td>
<td width="105" style="width: 14.3143%;">t(2), g(15870)</td>
</tr>
<tr>
<td width="47" style="width: 8.60861%;">3</td>
<td width="123" style="width: 13.013%;">M542</td>
<td width="129" style="width: 12.1121%;">M(15873), L(1)</td>
<td width="117" style="width: 11.9119%;">a(15874), c(1)</td>
<td width="108" style="width: 12.3123%;">t(15875)</td>
<td width="105" style="width: 14.3143%;">g(15874)</td>
</tr>
<tr>
<td width="47" style="width: 8.60861%;">4</td>
<td width="123" style="width: 13.013%;">A547</td>
<td width="129" style="width: 12.1121%;">A(15874), V(1)</td>
<td width="117" style="width: 11.9119%;">g(15875)</td>
<td width="108" style="width: 12.3123%;">c(15874), t(1)</td>
<td width="105" style="width: 14.3143%;">c(15875)</td>
</tr>
<tr>
<td width="47" style="width: 8.60861%;">5</td>
<td width="123" style="width: 13.013%;">A554</td>
<td width="129" style="width: 12.1121%;">A(15874), V(1)</td>
<td width="117" style="width: 11.9119%;">g(15875)</td>
<td width="108" style="width: 12.3123%;">c(15874), t(1)</td>
<td width="105" style="width: 14.3143%;">c(1), t(15874)</td>
</tr>
<tr>
<td width="47" style="width: 8.60861%;">6</td>
<td width="123" style="width: 13.013%;">V667</td>
<td width="129" style="width: 12.1121%;">V(15870), I(5)</td>
<td width="117" style="width: 11.9119%;">a(5), g(15870)</td>
<td width="108" style="width: 12.3123%;">t(15875)</td>
<td width="105" style="width: 14.3143%;">c(15875)</td>
</tr>
</tbody>
</table>
<p><strong>Table 1. </strong><strong>The non-synonymous variants at </strong><strong>RdRp active site </strong><strong>across more than 15000 SARS-CoV-2 samples.</strong></p>
<p>We could not predict the effect of these mutations on remdesivir binding due to the lack of RdRp structure where remdesivir has not been incorporated into the RNA primer sequence while it is sitting in a pocket in RdRp. Therefore, we can only comment on how the nature of the variant sidechains differ from the wild-type sidechains according to their physiochemical properties: the size and polarity of mutated residues are largely conserved in all cases (A-&gt;V, Q-&gt;H, V-&gt;I), suggesting high chances of a light effect on ligand binding.</p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/09/color-coded-RdRp-sars-cov-2.png" alt="" width="975" height="334" class="alignnone wp-image-5182 size-full" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/09/color-coded-RdRp-sars-cov-2.png 975w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/09/color-coded-RdRp-sars-cov-2-300x103.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/09/color-coded-RdRp-sars-cov-2-768x263.png 768w" sizes="(max-width: 975px) 100vw, 975px" /></p>
<p><strong>Figure 2. Left Panel: The surface representation of SARS-CoV-2 RdRp crystal structure with the non-conserved amino acid positions highlighted in orange. Right Panel: The sticks representation of the non-conserved residues among SARS-CoV-2 samples.</strong></p>
<p>In conclusion, from a structural standpoint, the catalytic site of RdRp is a good target for broad-spectrum inhibitors against coronaviruses considering that only 17% of the residues lining catalytic site binding pockets were not conserved across 27 Alpha- and Betacoronavirus entries. It is worth mentioning that clinically, the inhibition of RdRp which is achieved by remdesivir has not been fully confirmed to be an effective therapy for COVID-19.(Goldman et al, 2020)</p>
<p>According to the SARS-CoV-2 variants reported in table 1, the properties of most of the non-synonymous variants at the catalytic site were conserved, suggesting high chances of a light effect on ligand binding.</p>
<p>It would be crucial to investigate the structure of RdRp combined with remdesivir in the state right before it gets incorporated into the RNA primer. This would reveal the main interactions that remdesivir makes with RdRp sidechains and allow to better predict the effect of mutations both across coronaviruses broadly and more specifically across samples from COVID-19 patients on remdesivir binding.</p>
<p>To view the full report on SARS-CoV-2 RdRp, please visit this <a href="https://doi.org/10.5281/zenodo.4041302">Zenodo report. </a></p>
<p>As always, I would be happy to hear your comments. Please contact me via the <strong>“Leave a comment”</strong> link at the top of this post. Stay Tuned for my next post!</p>
<p><strong>References:</strong></p>
<p>Sheahan, T. P., Sims, A. C., Zhou, S., Graham, R. L., Hill, C. S., Leist, S. R., Schäfer, A., Dinnon, K. H., Montgomery, S. A., Agostini, M. L., Pruijssers, A. J., Chapell, J. D., Brown, A. J., Bluemling, G. R., Natchus, M. G., Saindane, M., Kolykhalov, A. A., Painter, G., Harcourt, J., … Baric, R. S. (2020). An orally bioavailable broad-spectrum antiviral inhibits SARS-Cov-2 and multiple endemic, epidemic and bat coronavirus. <a href="https://doi.org/10.1101/2020.03.19.997890">https://doi.org/10.1101/2020.03.19.997890</a></p>
<p>Goldman, J. D., Lye, D. C.B., Hui, D. S., Marks, K. M., Bruno, R., Montejano, R., Spinner, C. D., Galli, M., Ahn, M., Nahass, R. G., Chen, Y., SenGupta, D., Hyland, R. H., Osinusi, A. O., Cao, H., Blair, C., Wei, X., Gaggar, A., Brainard, D. M., &amp; Subramanian, A. (2020). Remdesivir for 5 or 10 Days in Patients with Severe Covid-19. <em>The New England Journal of Medicine</em>. <a href="https://doi.org/10.1056/NEJMoa2015301">https://doi.org/10.1056/NEJMoa2015301</a></p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/genetic-variability-at-the-remdesivir-binding-pocket-of-sars-cov-2-rna-dependent-rna-polymerase-across-coronaviruses-and-sars-cov-2-samples-post-20/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>The next target: SARS-CoV-2 RNA-dependent RNA polymerase and Its Druggability &#8211; Post 19</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/the-next-target-sars-cov-2-rna-dependent-rna-polymerase-and-its-druggability-post-19/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/the-next-target-sars-cov-2-rna-dependent-rna-polymerase-and-its-druggability-post-19/#respond</comments>
		
		<dc:creator><![CDATA[Setayesh Yazdani]]></dc:creator>
		<pubDate>Tue, 15 Sep 2020 16:35:45 +0000</pubDate>
				<category><![CDATA[COVID-19]]></category>
		<category><![CDATA[Setayesh Yazdani]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=5147</guid>

					<description><![CDATA[Previously we reported our analysis on the structural diversity of binding pockets found on the SARS-CoV-2 main protease, methyltransferase, and papain-like protease, macrodomain. We then shifted our focus on the SARS-CoV-2 RNA-dependent RNA polymerase (RdRp). SARS-CoV-2 is a positive-strand RNA virus, depending on its multi-subunit machinery to replicate its RNA. The catalytic subunit of RdRp <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/the-next-target-sars-cov-2-rna-dependent-rna-polymerase-and-its-druggability-post-19/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p>Previously we reported our analysis on the structural diversity of binding pockets found on the SARS-CoV-2 main protease, methyltransferase, and papain-like protease, macrodomain. We then shifted our focus on the SARS-CoV-2 RNA-dependent RNA polymerase (RdRp).</p>
<p>SARS-CoV-2 is a positive-strand RNA virus, depending on its multi-subunit machinery to replicate its RNA. The catalytic subunit of RdRp is the core component of this machinery, which is also known as nsp12. Nsp12 relies on accessory subunits to have full activity. The accessory factors, nsp7 and nsp8, increase RdRp template binding and processivity. (Yin et al., 2020)</p>
<p>RdRp is a well-established target for a class of antiviral drugs known as nucleotide analogs. One of the known and only FDA and Health Canada approved medications for COVID-19, remdesivir, targets RdRp.</p>
<p>Remdesivir is converted to its active form, remdesivir triphosphate (RTP), that gets incorporated into RNA and causes chain termination.</p>
<p>It is important to note that remdesivir has been tested in phase III clinical trial. The results were shared in The New England Journal of Medicine on May 27, 2020, by Goldman and his colleagues. (Goldman et al, 2020) The use of remdesivir for 5 to 10 days in patients with severe COVID-19 showed that there was no significant difference between the 5-day treatment and 10-day treatment groups. The overall effect on the survival rate was rather low. It is important to note this study did not have a placebo control arm and that means the magnitude of the benefit cannot be determined.</p>
<p>Yin et al solved the crystal structure of RdRp in complex with a remdesivir-incorporated RNA strand (PDB code 7bv2). The authors published two structures of RdRp; one in the apo form containing one nsp12, nsp7, and two nsp8 proteins (PDB: 7bv1) shown in figure 1A and another bound to the remdesivir-incorporated RNA, containing nsp12, one nsp7, and one nsp8 (PDB: 7bv2) shown in figure 1B.</p>
<p>Upon incorporation of the remdesivir into the primer strand, there is immediate termination of chain elongation. This inhibitory mechanism is called nonobligate RNA chain termination by remdesivir where the prodrug needs conversion to the active triphosphate form, RTP. (Yin et al., 2020)The interactions between remdesivir monophosphate (RMP) and RdRp are shown in figure 2.</p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/09/RdRp-1AB.png" alt="" width="506" height="339" class="alignnone wp-image-5150 " srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/09/RdRp-1AB.png 903w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/09/RdRp-1AB-300x201.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/09/RdRp-1AB-768x515.png 768w" sizes="(max-width: 506px) 100vw, 506px" /></p>
<p><strong>Figure 1. (A) Views of Cryo-electron microscopy structure of nsp12-nsp7-nsp8 RdRp complex</strong> <strong>(B) and nsp12-nsp7-nsp8 in complex with the primer RNA template and remdesivir</strong> (Yin et al., 2020)</p>
<p>Once RdRp is in the template-RTP form, it goes into a closed conformation where the 11 base-pairs template primer is held by the finger-palm-thumb subdomains. There are also extensive RNA-protein interactions that involve the RNA phosphate-ribose backbone. In the absence of specific interactions between any specific base-pairs and RdRp, the binding of RNA by RdRp becomes independent of the RNA sequence which explains why no specific sequence is needed for RdRp’s enzymatic activity. (Yin et al., 2020)</p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/09/RdRp-2.png.jpg" alt="" width="303" height="264" class="alignnone wp-image-5162 " srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/09/RdRp-2.png.jpg 476w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/09/RdRp-2.png-300x262.jpg 300w" sizes="(max-width: 303px) 100vw, 303px" /></p>
<p><strong>Figure 2. The chemical interactions between the RdRp catalytic site residues and RMP (purple). </strong>(Yin et al., 2020)</p>
<p>We aim to analyze the genetic diversity of RdRp catalytic site across coronaviruses and SARS-CoV-2 samples. To achieve this goal, we first assessed the druggability of the remdesivir-binding site of RdRp and determined the residues lining that site which I explain in this post.</p>
<p>Using SiteMap (Schrodinger, NY) we assessed the druggability of the binding sites using the druggability score (Dscore). We found that the active site of RdRp in the vicinity of bound remdesivir has a Dscore of 0.859 indicative of a difficult druggable site. We show a schematic of this site in figure 3.</p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/09/RdRp-druggability-rem.png" alt="" width="1004" height="570" class="alignnone wp-image-5153 size-full" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/09/RdRp-druggability-rem.png 1004w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/09/RdRp-druggability-rem-300x170.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/09/RdRp-druggability-rem-768x436.png 768w" sizes="(max-width: 1004px) 100vw, 1004px" /></p>
<p><strong>Figure 3. </strong><strong>Druggability analysis of SARS-CoV-2 RdRp active site. </strong>Cavities at the catalytic site (orange and magenta blobs) are found next to the position occupied by remdesivir monophosphate incorporated into the RNA template<strong> (purple stick). </strong></p>
<p>Using ICM PocketFinder, we find that RMP is not occupying a clearly defined binding pocket but is juxtaposed to two cavities. This probably reflects the fact that we are looking at a structure where remdesivir was already incorporated into RNA. The site is structurally conserved in the apo form (PDB: 7bv1). We hypothesize that compounds occupying these cavities would block the catalytic activity of RdRp. We identified 48 amino acid sidechains lining the pockets juxtaposed to Remdesivir shown in figures 3 and 4 (within 2.8 Å of the pocket): V410, F440, F441, F442, A443, Q444, D452, Y453, Y455, Y456, Y458, T540, M542, N543, L544, K545, Y546, A547, I548, S549, K551, N552, R553, A554, R555, T556, V557, A558, G559, Y619, P620, K621, C622, D623, R624, V667, K676, G679, T680, S681, S682, G683, D684, T687, A688, N691, S759, D760.</p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/09/RdRp-4-neigh_res_rem.png" alt="" width="671" height="467" class="alignnone wp-image-5154 " srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/09/RdRp-4-neigh_res_rem.png 1266w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/09/RdRp-4-neigh_res_rem-300x209.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/09/RdRp-4-neigh_res_rem-1024x713.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/09/RdRp-4-neigh_res_rem-768x534.png 768w" sizes="(max-width: 671px) 100vw, 671px" /></p>
<p><strong>Figure 4. </strong><strong>The neighboring residues lining the active site of RdRp.</strong></p>
<p>In my next post, I will show the sequence diversity across UniProt entries from the Alpha- and Betacoronavirus genera and map that to the RdRp remdesivir-binding site using its crystal structure.</p>
<p>As always, I would be happy to hear your comments. Please contact me via the “Leave a comment” link at the top of this post. Stay Tuned for my next post!</p>
<p><strong>References:</strong></p>
<p>Yin, W., Mao, C., Luan, X., Shen, D., Shen, Q., Su, H., Wang, X., Zhou, F., Zhao, W., Gao, M., Chang, S., Xie, Y., Tian, G., Jiang, H., Tao, S., Shen, J., Jiang, Y., Jiang, H., Xu, Y., &amp; Xu, H. (2020). Structural basis for inhibition of the RNA-dependent RNA polymerase from SARS-CoV-2 by remdesivir. <em>Science</em>, <em>368</em>(6498), 1499-1504. <a href="https://doi.org/10.1126/science.abc1560">https://doi.org/10.1126/science.abc1560</a></p>
<p>Goldman, J. D., Lye, D. C.B., Hui, D. S., Marks, K. M., Bruno, R., Montejano, R., Spinner, C. D., Galli, M., Ahn, M., Nahass, R. G., Chen, Y., SenGupta, D., Hyland, R. H., Osinusi, A. O., Cao, H., Blair, C., Wei, X., Gaggar, A., Brainard, D. M., &amp; Subramanian, A. (2020). Remdesivir for 5 or 10 Days in Patients with Severe Covid-19. <em>The New England Journal of Medicine</em>. <a href="https://doi.org/10.1056/NEJMoa2015301">https://doi.org/10.1056/NEJMoa2015301</a></p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/the-next-target-sars-cov-2-rna-dependent-rna-polymerase-and-its-druggability-post-19/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>Calculating the change in Gibbs free energy (ddGbind) of an improved alpha-ketoamide inhibitor binding associated with genetic variations of SARS-CoV-2 main protease &#8211; post18</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/calculating-the-change-in-gibbs-free-energy-ddgbind-of-an-improved-alpha-ketoamide-inhibitor-binding-associated-with-genetic-variations-of-sars-cov-2-main-protease-post18/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/calculating-the-change-in-gibbs-free-energy-ddgbind-of-an-improved-alpha-ketoamide-inhibitor-binding-associated-with-genetic-variations-of-sars-cov-2-main-protease-post18/#respond</comments>
		
		<dc:creator><![CDATA[Setayesh Yazdani]]></dc:creator>
		<pubDate>Mon, 24 Aug 2020 21:20:22 +0000</pubDate>
				<category><![CDATA[COVID-19]]></category>
		<category><![CDATA[SARS-CoV-2 computational genomics]]></category>
		<category><![CDATA[Setayesh Yazdani]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=5099</guid>

					<description><![CDATA[Since we posted our analysis of genetic variation of SARS-CoV-2 main protease (MPro) with inhibitor N3 (PDB: 7bqy), there have been other inhibitors co-crystallized with the MPro (PDB: 6lze, PDB: 6m0k, and PDB: 6y2f). In my last post, I talked about the two inhibitors, 11a and 11b (PDB: 6lze, PDB: 6m0k) and in today&#8217;s post, <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/calculating-the-change-in-gibbs-free-energy-ddgbind-of-an-improved-alpha-ketoamide-inhibitor-binding-associated-with-genetic-variations-of-sars-cov-2-main-protease-post18/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p>Since we posted <a href="https://thesgc.github.io/static-openlabnotebooks/calculating-the-change-in-gibbs-free-energy-ddgbind-of-inhibitor-binding-associated-with-genetic-variations-of-sars-cov-2-main-protease/">our analysis of genetic variation of SARS-CoV-2 main protease (MPro) with inhibitor N3</a> (PDB: 7bqy), there have been other inhibitors co-crystallized with the MPro (PDB: 6lze, PDB: 6m0k, and PDB: 6y2f).</p>
<p>In my <a href="https://thesgc.github.io/static-openlabnotebooks/back-to-sars-cov-2-main-protease-calculating-the-change-in-gibbs-free-energy-ddgbind-of-two-peptidomimetic-aldehyde-inhibitors-binding-associated-with-genetic-variations-of-sars-cov-2-main-protease/">last post</a>, I talked about the two inhibitors, 11a and 11b (PDB: 6lze, PDB: 6m0k) and in today&#8217;s post, we focus on another MPro structure co-crystallized with an inhibitor called 13b (PDB: 6y2f).</p>
<p>The premise for the MPro inhibitor discussed in this post is based on the authors&#8217; previous work where they designed a peptidomimetic α-ketoamide, 11r (Figure 1), as a broad-spectrum inhibitor of coronavirus and enterovirus replication. (1) In a subsequent study (2), the authors attempt to improve the half-life of the previous inhibitor in plasma by incorporating the p3-p2 amide bond in 11r into a pyridine ring in 13a. (Figure 1) They speculated that this change might hide the amide bond from cellular proteases and hence prevent its cleavage. Furthermore, the authors justified that to enhance the antiviral activity against betacoronaviruses of clade b (SARS-CoV-2 and SARS-CoV) they replaced the cyclohexyl moiety of 13a by cyclopropyl in 13b which is a smaller moiety. By changing 13a to 13b, they sacrificed the goal of broad-spectrum inhibitor activity of their compound, as the cyclohexyl in 13a was intended to fill in the enterovirus MPro which is replaced by a smaller cyclopropyl moiety in 13b.</p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/11r-13b.png" alt="" width="550" height="337" class="alignnone wp-image-5116 " srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/11r-13b.png 1294w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/11r-13b-300x184.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/11r-13b-1024x628.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/11r-13b-768x471.png 768w" sizes="(max-width: 550px) 100vw, 550px" /></p>
<p><strong>Figure 1. The design rationale for an improved alpha-ketoamide inhibitor, the path from 11r to 13b.</strong></p>
<p>The IC50 value of 13b was at 0.67 ± 0.18 μM for SARS-CoV-2 MPro and 0.90 ± 0.29 μM and 0.58 ± 0.22 μM for SARS-CoV Mpro and the MERS-CoV Mpro respectively. Additionally, the researchers in this study showed that 13b has lung tropism which means that 13b was detectable in mice’s lung tissue 4 hours following its subcutaneous administration. Lung tropism is desirable considering the tissue that COVID-19 mostly affects.</p>
<p>The catalytic residue of the MPro catalytic site, Cys145, performs a nucleophilic attack onto the α-keto group of 13b to form a thiohemiketal bond in a reversible reaction and thus MPro is inhibited.</p>
<p>Their study provided a framework for the development of pyridone-containing inhibitors to be used as antiviral drugs.</p>
<p>In this post, I will explain how the non-synonymous variants found at SARS-CoV-2 MPro catalytic site across thousands of SARS-CoV-2 samples from COVID-19 patients are predicted to affect 13b binding. (3)</p>
<p>To predict the effect of those identified mutations on inhibitor (13b) binding, we use ICM software (Molsoft, SD). In ICM, by mutating the residues, we can see the change in Gibbs free energy of inhibitor binding (ddGbind) which would indicate if the mutation penalizes the binding of the inhibitor significantly or not.</p>
<p>To see the detailed methods on how the change in Gibbs free energy of ligand binding is calculated, please view this Zenodo post <a href="https://doi.org/10.5281/zenodo.3928729"><u>here</u></a>. Below in table 1, we show the four non-synonymous mutations and the corresponding ddGbind values.</p>
<table width="463" style="width: 76.0487%; height: 214px;" height="313">
<tbody>
<tr style="height: 58px;">
<td width="75" style="height: 58px;">Residue number</td>
<td width="66" style="height: 58px;">Wild Type</td>
<td width="59" style="height: 58px;">Mutant</td>
<td width="73" style="height: 58px;"><strong>ddGbind</strong></td>
<td width="90" style="height: 58px;">dGbind_wild</td>
<td width="99" style="height: 58px;">dGbind_mut</td>
</tr>
<tr style="height: 39px;">
<td width="75" style="height: 39px;">49</td>
<td width="66" style="height: 39px;">Met</td>
<td width="59" style="height: 39px;">Ile</td>
<td width="73" style="height: 39px;">-0.6</td>
<td width="90" style="height: 39px;">-27.1</td>
<td width="99" style="height: 39px;">-27.7</td>
</tr>
<tr style="height: 39px;">
<td width="75" style="height: 39px;">52</td>
<td width="66" style="height: 39px;">Pro</td>
<td width="59" style="height: 39px;">Ser</td>
<td width="73" style="height: 39px;">-0.1</td>
<td width="90" style="height: 39px;">-27.1</td>
<td width="99" style="height: 39px;">-27.2</td>
</tr>
<tr style="height: 39px;">
<td width="75" style="height: 39px;">142</td>
<td width="66" style="height: 39px;">Asn</td>
<td width="59" style="height: 39px;">Ser</td>
<td width="73" style="height: 39px;">1.7</td>
<td width="90" style="height: 39px;">-27.1</td>
<td width="99" style="height: 39px;">-25.4</td>
</tr>
<tr style="height: 39px;">
<td width="75" style="height: 39px;">168</td>
<td width="66" style="height: 39px;">Pro</td>
<td width="59" style="height: 39px;">Ser</td>
<td width="73" style="height: 39px;">0.4</td>
<td width="90" style="height: 39px;">-27.1</td>
<td width="99" style="height: 39px;">-26.7</td>
</tr>
</tbody>
</table>
<p><strong>Table 1.</strong> <strong>The predicted change in Gibbs free energy associated with the non-synonymous variants found at MPro catalytic site across more than 15000 SARS-CoV-2 samples.</strong> ddGbind values are in Kcal/mol.</p>
<p>Based on the precision of our method, none of the non-synonymous mutations are predicted to affect 13b binding significantly, since all ddGbind values are less than 2.0 Kcal/mol. (4) Prediction of no significant changes in ddGbind values for the four non-synonymous mutations (M49I, P52S, N142S, and P168S) is possibly due to the conservation of key ligand-protein interactions after these residues are mutated. In figure 2, we show the 2D ligand-protein interaction diagram which outlines the chemical interactions between 13b and MPro catalytic site residues.</p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/13b-2D-ligand-protein-interactions.png" alt="" width="407" height="349" class="alignnone wp-image-5117 " srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/13b-2D-ligand-protein-interactions.png 683w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/13b-2D-ligand-protein-interactions-300x257.png 300w" sizes="(max-width: 407px) 100vw, 407px" /></p>
<p><strong>Figure 2. The 2D ligand-protein interaction diagram between ligand 13b and the residues of the MPro active site. (PDB: 6y2f).</strong> This diagram was created using Pose.View in Proteins.Plus.</p>
<p>We created an energy matrix table where we consider the effect of every possible mutation on the binding of the ligand (inhibitor 13b) with SARS-CoV-2-MPro. Table 2 includes a list of the 20 residues at MPro catalytic site expect Cys145. Cys145 is not included as it makes a covalent bond with 13b: any mutation will severely affect 13b binding.</p>
<p>It is likely that the residues of the rows with green color are not making optimal interactions with 13b. This is reflected in the very slight and non-significant lowering in ddGbind values of these residues (mutations are stabilizing) and hence the green color. On the other hand, the sidechains whose mutations are shown in orange are likely the positions where MPro would better interact with 13b which include H41, S144, M165, E166, P168, D187. As a result, the mutations of those sidechains are predicted to penalize inhibitor-binding significantly and hence the orange color. According to Zhang and her colleagues (3), His41 provides a hydrogen bond to stabilize the oxyanion group of thiohemiketal link between 13b which is also reflected in the orange row in table 3. The amide oxygen of 13b partly accepts a hydrogen bond from the main-chain amide of Ser144, which forms the canonical “oxyanion hole” of the cysteine protease, making S144 an important residue for 13b inhibitory mechanism.</p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/13b-energy-matrix.png" alt="" width="1908" height="728" class="alignnone wp-image-5118 size-full" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/13b-energy-matrix.png 1908w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/13b-energy-matrix-300x114.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/13b-energy-matrix-1024x391.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/13b-energy-matrix-768x293.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/13b-energy-matrix-1536x586.png 1536w" sizes="(max-width: 1908px) 100vw, 1908px" /></p>
<p><strong>Table 2. </strong><strong>The energy matrix showing the effect of all possible mutations at the SARS-COV-2 MPro catalytic site and the predicted effect on inhibitor 13b binding.</strong> No mutations at Cys145 were considered since it makes the covalent thiohemiketal linkage with 13b which is essential for the inhibitory mechanism in this scenario. The energy unit is kcal/mol.</p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/energy-matrix-color-coding.png" alt="" width="133" height="118" class="alignnone wp-image-5041" /></p>
<p>Altogether, for the improved α-ketoamide inhibitor, 13b, we can conclude that the rare variants found in SARS-CoV-2 samples are only expected to minimally affect 13b binding to the MPro.</p>
<p><strong>References:</strong></p>
<ol>
<li>Zhang, Linlin., α-Ketoamides as broad-spectrum inhibitors of coronavirus and enterovirus replication: Structure-based design, synthesis, and activity assessment. <em>Journal of Medicinal Chemistry</em>. 63 (9), 4562–4578 (2020). <a href="https://doi.org/10.1021/acs.jmedchem.9b01828">https://doi.org/10.1021/acs.jmedchem.9b01828</a></li>
<li>Zhang, Linlin., Crystal structure of SARS-CoV-2 main protease provides a basis for design of improved α-ketoamide inhibitors. <em>Science</em> 368(6489), 409-412 (2020). <a href="https://doi.org/10.1126/science.abb3405">https://doi.org/10.1126/science.abb3405</a></li>
<li>Nicola De Maio. <em>Genetic diversity at the catalytic site of the main SARS-Cov-2 protease.</em> (2020) openlabnotebooks.org –https://thesgc.github.io/static-openlabnotebooks/genetic-diversity-at-the-catalytic-site-of-the-main-sars-cov-2-protease/</li>
<li>Schapira, M., Totrov, M., Abagyan, R. Prediction of the Binding Energy for Small Molecules, Peptides and Proteins. <em>Journal of Molecular Recognition </em>12(3), 177-90 (1999). <a href="https://doi.org/10.1002/(SICI)1099-1352(199905/06)12:3%3c177::AID-JMR451%3e3.0.CO;2-Z">https://doi.org/10.1002/(SICI)1099-1352(199905/06)12:3&lt;177::AID-JMR451&gt;3.0.CO;2-Z</a></li>
</ol>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/calculating-the-change-in-gibbs-free-energy-ddgbind-of-an-improved-alpha-ketoamide-inhibitor-binding-associated-with-genetic-variations-of-sars-cov-2-main-protease-post18/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>Brachyury (TBXT) ligands for chordoma: 1. Fragment optimization strategy 2. fragment F9000505, B-site, PDB 5QRU</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/brachyury-tbxt-ligands-for-chordoma-1-fragment-optimization-strategy-2-fragment-f9000505-b-site-pdb-5qru/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/brachyury-tbxt-ligands-for-chordoma-1-fragment-optimization-strategy-2-fragment-f9000505-b-site-pdb-5qru/#respond</comments>
		
		<dc:creator><![CDATA[David Drewry]]></dc:creator>
		<pubDate>Mon, 24 Aug 2020 18:33:44 +0000</pubDate>
				<category><![CDATA[Chordoma]]></category>
		<category><![CDATA[David Drewry]]></category>
		<category><![CDATA[SGC UNC Chapel Hill]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=5102</guid>

					<description><![CDATA[Introduction In this blog post I will share details on the new molecules we have designed and synthesized or purchased based on one of our original hits from the x-ray crystallography fragment screen. The compound is called F9000505 and it is bound to pocket B. The crystal structure was deposited in the Protein Data Bank <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/brachyury-tbxt-ligands-for-chordoma-1-fragment-optimization-strategy-2-fragment-f9000505-b-site-pdb-5qru/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p><strong><u>Introduction</u></strong></p>
<p>In this blog post I will share details on the new molecules we have designed and synthesized or purchased based on one of our original hits from the x-ray crystallography fragment screen. The compound is called F9000505 and it is bound to pocket B. The crystal structure was deposited in the Protein Data Bank (pdb) with PDB ID 5QRU. This post will give you a feel for some of the ways we are approaching fragment optimization. Posts in the near future will describe strategies at other fragment sites and also provide data on the binding of these compounds to brachyury as it emerges.</p>
<p><strong><u>Background / Project Goal</u></strong></p>
<p>The expression of brachyury (TBXT) is a hallmark of chordoma. A growing body of data suggests that inhibition of TBXT protein activity, expression, or interactions with protein partners are predicted to be effective and specific means for treating chordoma. A key aim of this project is to discover compounds that bind directly to brachyury with sufficient affinity so that they ca be optimized into clinical candidates for the treatment of chordoma.</p>
<p><strong><u>Initial Fragment Optimization Strategy </u></strong></p>
<p>Fragment-based screening by x-ray crystallography is very powerful in that the location of the fragment hits on the target protein (brachyury) are known after the experiment, and the structure provides details of the interactions that the fragments make with brachyury. Crystallography can identify fragments that bind very weakly, so there is often a long way to go to grow the hit molecules into compound that bind potently enough to elicit the desired biological effects. We are taking an aggressive approach to exploring each of our hits so that we can soon focus efforts on the most promising molecules and binding sites.</p>
<p>The two sources of new molecules are purchase of commercially available compounds, and design and synthesis of new compounds. Purchase of compounds is cost effective and rapid in the early stages of a project but becomes limited as molecules become more complicated to capture the intricacies of the binding sites.</p>
<p>We have purchased compounds primarily through a company called ChemSpace. We performed substructure and similarity searches in the ChemSpace available compound database for each x-ray crystallography hit (“query” molecules for the searching). A substructure search means specific details of the hit need to be present in the molecules identified by the search. A similarity search relaxes the criteria and allows for selection of molecules that the computer tells us have certain similarities to our query molecules. These similarities could be in categories like size, shape, and electronic properties. Similarity searches can identify compounds that are more different from the hits which may be important if they identify new interactions, scaffolds that are easier to make new analogs of, or compound that explore different vectors (directions) around the hits.</p>
<p>We chose initial analogues for purchase to provide important information around each fragment in several domains:</p>
<ol>
<li>Compounds with functional groups that will serve as efficient handles for further elaboration.</li>
<li>Compounds that explore growth of the hit fragment in logical directions based on the crystal structure.</li>
<li>Compounds that add additional functionality that can make new interactions in order to enhance potency.</li>
<li>Compounds that teach us where linkers may be added in order to develop brachyury degraders.</li>
</ol>
<p>By making the fragments larger we can understand where growth is beneficial to potency. This will provide focus to future rounds of purchase or chemistry. Incorporation of chemical handles that can be rapidly elaborated with a variety of building blocks will allow us to make additional analogs quickly. Adding functionality (for example amides, ureas, phenols, heterocycles, substituted phenyls) that can make energetically favorable interactions (hydrogen bonds, pi-pi stacking, lipophilic interactions, charge-charge interactions) improves our chances of increasing potency. Utilizing commercially available analogs of the initial fragments found bound to the TBXT crystal structure provides the means to quickly generate structure activity relationships (SAR – this means how changing the structure changes how well the compound binds to brachyury) around each fragment. Initial SAR is critical as it provides confidence around each hit (can it be optimized, do several similar compounds also bind) and aids in site selection to narrow our focus.</p>
<p><strong><u>Fragment F9000505, binding site B, PDB ID: 5QRU </u></strong></p>
<p><em> </em><em>Compounds for purchase</em></p>
<p><strong>Figure 1</strong> depicts all the compounds that we could buy identified from substructure and similarity searching around fragment hit F9000505. This compound binds to site B, and more details on the way it binds will be found below in the new compound design section. Each dot represents a compound and they are organized in the plot according to two easy to calculate and important properties: lipophilicity (clogP) and molecular weight (MW). The green circles are the compounds that we have purchased and are screening. If any of these are active, we can purchase other compounds (blue dots) that can further supplement the SAR. This iterative cycle of design, synthesis (purchase if possible), and test is the life blood of medicinal chemistry compound optimization.</p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig1-1024x725.png" alt="" width="780" height="552" class="aligncenter wp-image-5104 size-large" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig1-1024x725.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig1-300x213.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig1-768x544.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig1.png 1200w" sizes="(max-width: 780px) 100vw, 780px" /></p>
<p><strong>Figure 2</strong> depicts examples of molecules we purchased from this set that demonstrate the availability of useful molecules that can answer some of the specific medicinal chemistry questions we need to ask. The blue boxes highlight the areas with structural modifications to the hit compound.</p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-2.png" alt="" width="643" height="604" class="aligncenter wp-image-5105 " srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-2.png 978w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-2-300x282.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-2-768x721.png 768w" sizes="(max-width: 643px) 100vw, 643px" /></p>
<p><em>Design and Synthesis of New Compounds</em></p>
<p>We have chemists on our team at UNC and also a talented team of chemists at a company called Piramal. We have been able to make excellent progress on new compound synthesis even with the initial lab shutdowns and then the reduced capacity necessitated by Covid-19. The team at Piramal has focused on preparation of analogs of fragments for which there are fewer commercial analogs, and on more elaborate chemistry to explore promising fragment sites in greater detail using the same principles outlined above.</p>
<p><strong>Figure 3</strong> depicts the crystal structure of fragment F9000505 bound to pocket B of brachyury. This simple compound results from the acylation of 2-methoxy-5-methylaniline with the amino acid glycine. The phenyl ring makes lipophilic contacts with the protein and the 5-methyl group tucks into a lipophilic pocket. The oxygen of the 2-methoxy group makes an intramolecular hydrogen bond with the adjacent NH, helping to orient the trajectory of the carbonyl of the amide and also the terminal amine of the glycine. In this locked in orientation the carbonyl of the fragment forms a hydrogen bond with Ala119 of brachyury and the primary amine forms a hydrogen bond to the carbonyl of Pro117.</p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog2-fig3-286x300.png" alt="" width="315" height="330" class="aligncenter wp-image-5106 " srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog2-fig3-286x300.png 286w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog2-fig3-768x806.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog2-fig3.png 806w" sizes="(max-width: 315px) 100vw, 315px" /></p>
<p>Based on this structure there are a number of avenues that need to be explored for optimization of binding. The methyl substituent in the lipophilic pocket may not be ideal. Groups with different size and electronic characteristics need to be explored (for example: F, Cl, ethyl, cyclopropyl). There is plenty of room to extend off of the methoxy group. Doing this may allow us to pick up additional interactions with the surface of the protein. In addition, this may be an attractive position for a linker that will be suitable for synthesis of a bifunctional degrader. Another strategy is to extend the primary amine in order to reach Glu116 which could increase affinity. The alpha carbon of the glycine moiety presents another opportunity. Simple substituents like methyl will add conformational constraint which can enhance potency. Because this building block comes from amino acids, there is opportunity to quickly explore stereochemistry at this position and a range of functional groups that have potential to form new interactions. Finally, we can form a 5- or 6-membered ring between the NH and the O of the fragment. This will serve to further lockdown the orientation of the carbonyl and NH2 that make key hydrogen bonds and offer different vectors for exploring interactions with other brachyury residues. Our initial compound design targeted 68 molecules. These molecules have all been synthesized and submitted from screening in the surface plasmon resonance (SPR) binding assay. A sampling of the 68 compounds are depicted in <strong>Figure 4</strong>. These show example modifications of the methyl group (<strong>4A</strong>), primary amine (<strong>4B</strong>), the alpha carbon of the glycine moiety (<strong>4C</strong>), and the methyl ether (<strong>4D</strong>).</p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-4.png" alt="" width="861" height="573" class="wp-image-5107 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-4.png 1538w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-4-300x200.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-4-1024x682.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-4-768x511.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-4-1536x1023.png 1536w" sizes="(max-width: 861px) 100vw, 861px" /></p>
<p><em>Synthetic chemistry schemes</em></p>
<p><strong>Figure 5</strong> show the chemistry scheme used to make many of the 68 new analogues of F9000505 synthesized for this project.</p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-5.png" alt="" width="698" height="172" class="alignnone wp-image-5108 " srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-5.png 1434w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-5-300x74.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-5-1024x253.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-5-768x190.png 768w" sizes="(max-width: 698px) 100vw, 698px" /></p>
<p><strong>Figure 6</strong> demonstrates utilization of an easily synthesized key intermediate to explore the addition of new functionality at three different distances from the core, linked with an amide. If any offer enhanced potency, follow up will focus on modifications of the pendant aryl ring.</p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-6.png" alt="" width="584" height="275" class="wp-image-5109 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-6.png 1332w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-6-300x141.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-6-1024x483.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-6-768x362.png 768w" sizes="(max-width: 584px) 100vw, 584px" /></p>
<p><strong>Figure 7</strong> depicts three different ways to functionalize the same key intermediate with robust chemistry that provides different sets of products. A key element of the initial fragment analogue strategy is to generate and use key intermediates that can be diversified in the last synthetic step.</p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-7.png" alt="" width="396" height="220" class="wp-image-5110 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-7.png 910w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-7-300x167.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-7-768x427.png 768w" sizes="(max-width: 396px) 100vw, 396px" /></p>
<p>The crystal structure suggests that new interactions may be possible if we grow the molecule in the ether position. In addition to making several different ethers, we have opted to incorporate both an alkyne linker and a triazole linker in this position. These choices offer different spacing, different angles, different functionalities, and can be expanded with robust chemistry. An overview of this linker modification chemistry is shown in <strong>figure 8</strong>.</p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-8.png" alt="" width="797" height="458" class="aligncenter wp-image-5111 " srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-8.png 1494w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-8-300x172.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-8-1024x588.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-8-768x441.png 768w" sizes="(max-width: 797px) 100vw, 797px" /></p>
<p>Finally, a key element of the strategy is to identify positions on the molecules where linkers can be attached to generate bifunctional degradation reagents. Crystallography will of course provide critical information to guide linker location. To improve our chances of quickly identifying potential linker locations, we are routinely incorporating some building blocks with functionality that lends itself to linker friendly chemistry. Currently these are mostly simple amides and Boc-protected amines, but azides and alkynes would also be valuable additions. Some newly synthesized compounds with “linker-friendly” functionality are shown in <strong>figure 9</strong>.</p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog2-fig-9.png" alt="" width="605" height="294" class="wp-image-5112 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog2-fig-9.png 1290w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog2-fig-9-300x146.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog2-fig-9-1024x499.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog2-fig-9-768x374.png 768w" sizes="(max-width: 605px) 100vw, 605px" /></p>
<p>&nbsp;</p>
<p>Stay tuned for details on compounds we have made and purchased for other sites and, importantly, the SPR screening data that should be available soon that lets us know how we are doing in terms of binding affinity!</p>
<p>Don&#8217;t hesitate to reach out with questions and comments.</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/brachyury-tbxt-ligands-for-chordoma-1-fragment-optimization-strategy-2-fragment-f9000505-b-site-pdb-5qru/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
	</channel>
</rss>
