<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>Link &#8211; openlabnotebooks.org</title>
	<atom:link href="https://thesgc.github.io/static-openlabnotebooks/type/link/feed/" rel="self" type="application/rss+xml" />
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<description>A growing team of groundbreaking scientists around the world are now sharing their lab notebooks online</description>
	<lastBuildDate>Mon, 13 May 2019 13:19:13 +0000</lastBuildDate>
	<language>en-GB</language>
	<sy:updatePeriod>hourly</sy:updatePeriod>
	<sy:updateFrequency>1</sy:updateFrequency>
	<generator>https://wordpress.org/?v=4.9.10</generator>

<image>
	<url>https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/cropped-lab-jar02-32x32.png</url>
	<title>Link &#8211; openlabnotebooks.org</title>
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<width>32</width>
	<height>32</height>
</image> 
	<item>
		<title>Design of split-intein mediated SCN1A protein trans-splicing</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/2047-2/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/2047-2/#respond</comments>
		<pubDate>Mon, 29 Oct 2018 00:28:39 +0000</pubDate>
		<dc:creator><![CDATA[Wen Yih Aw]]></dc:creator>
				<category><![CDATA[Dravet syndrome]]></category>
		<category><![CDATA[Gene therapy for Dravet Syndrome]]></category>
		<category><![CDATA[Wen Yih Aw]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2047</guid>
		<description><![CDATA[Previously, I provided a brief overview on SCN1A related Dravet Syndrome (DS) and the need for larger gene therapy vectors or alternative methods to deliver functional copies of SCN1A to affected cells in DS. The goal of my project is to explore one of these alternative methods, specifically a split-intein mediated protein trans-splicing method, that <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/2047-2/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Previously, I provided a brief overview on SCN1A related Dravet Syndrome (DS) and the need for larger gene therapy vectors or alternative methods to deliver functional copies of SCN1A to affected cells in DS. The goal of my project is to explore one of these alternative methods, specifically a split-intein mediated protein trans-splicing method, that could reduce the size of the passenger (SCN1A expression cassettes) for gene therapy vectors. In this post, I will talk about the overall design for this method.</p>
<p>To give you a little more background about inteins: Inteins (also known as intervening proteins) are basically mobile protein elements that are translated and embedded within another protein sequences. You can think of them as single-use, single-turnover enzymes that catalyze their own excision from a flanking precursor “exteins” protein sequences while concurrently joining the remaining exteins protein fragments together through the formation of a new peptide bond (Fig. 1).</p>
<p><img class="size-medium wp-image-2048 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/10/intein-inbase-300x185.png" alt="" width="300" height="185" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/10/intein-inbase-300x185.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/10/intein-inbase.png 736w" sizes="(max-width: 300px) 100vw, 300px" /></p>
<p style="text-align: center;">Figure 1. Intein-mediated protein splicing vs. RNA splicing.  Image source: Perler, F. B. (2002). InBase, the Intein Database. Nucleic Acids Res. 30, 383-384.</p>
<p>There are two types of inteins – cis splicing inteins that are embedded within protein sequences and catalyze protein splicing <em>in cis</em> and split- or trans-splicing inteins where the enzyme is naturally split and expressed into two parts. These two fragments of split-intein can associate with each other to form a catalytic active enzyme that can catalyze protein splicing <em>in trans. </em>Briefly, my project aims to split the SCN1A expression cassette into two parts and make use of the unique split-intein mediated trans-splicing capability to rejoin the expressed SCN1A fragments together (Fig. 2)</p>
<p><img class="size-medium wp-image-2050 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/10/dravet-pic4-216x300.png" alt="" width="216" height="300" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/10/dravet-pic4-216x300.png 216w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/10/dravet-pic4.png 279w" sizes="(max-width: 216px) 100vw, 216px" /></p>
<p style="text-align: center;">Figure 2. A basic design of split-intein mediated SCN1A protein trans-splicing</p>
<p>So, here goes the basic components and design of this cloning experiment. We are going to express SCN1A as two protein fragments (split-N terminal and split-C terminal SCN1A), each fused with split-N-term and C-term intein sequence (Fig. 2), respectively.</p>
<p>But&#8230; where do we split the protein?</p>
<p>One of the factors taken into consideration in designing the split sites is that for efficient protein trans-splicing, the split sites or the sites where split-intein sequences are inserted need to be surface exposed. SCN1A is a multipass membrane proteins with four homologous (unexposed) transmembrane domains (DI to DIV), each domain is connected to the next domain by large cytoplasmic loops (Fig. 3). In fulfilling this condition, we are left with the N and C-terminal cytoplasmic domain and the cytoplasmic loops connecting DI to DII, DII to DIII, and DIII to DIV to work with. Among these cytoplasmic regions, splitting cytoplasmic loops within and between DII to DIII can result in a significant reduction in the size of SCN1A expression cassettes.</p>
<p>Additionally, to initiate efficient splicing reaction, split-inteins require a certain combination of amino acids flanking the intein sequences. For instance, split DnaB intein from <em>Synechocystis</em> needs to be flanked by a glycine at the N-terminus of the intein and a serine at C-terminus of the intein to catalyze protein trans-splicing whereas Vma intein from <em>S. cerevisiae </em>requires a glycine and cysteine amino acids combination.  In addition to previous considerations, these specific requirements of amino acids combinations for efficient and seamless protein trans-splicing helped us further narrow down a short list of potential SCN1A split sites.</p>
<p><img class="size-medium wp-image-2051 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/10/scn1a-300x134.png" alt="" width="300" height="134" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/10/scn1a-300x134.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/10/scn1a-768x342.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/10/scn1a-1024x456.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/10/scn1a.png 1488w" sizes="(max-width: 300px) 100vw, 300px" /></p>
<p style="text-align: center;">Figure 3. A general topology of eukaryotic voltage-gated sodium channels.</p>
<p style="text-align: center;">Image source: Shen, Huaizong, et al. “Structure of a Eukaryotic Voltage-Gated Sodium Channel at near-Atomic Resolution.” <em>Science</em>, American Association for the Advancement of Science, 3 Mar. 2017, science.sciencemag.org/content/355/6328/eaal4326.long.</p>
<p>&nbsp;</p>
<p>How can we tell whether two split-SCN1A fragments trans-spliced?</p>
<p>To screen and identify constructs that have successfully reconstituted, I will attach split-GFP sequences to the N and C termini of the split-N-terminal-SCN1A intein and split-C-terminal-SCN1A-intein, respectively. Split-GFP is composed of two complementing split fluorescent protein fragments that irreversibly associate with each other to generate a fluorescent signal. When expressing both of these constructs in cells, we predicted that the split-GFP sequence would help bring the two split-proteins together (Fig. 4.) The split-intein would interact with each other to form an active enzyme and catalyze the excision of both itself and GFP protein to the cytoplasm. In addition to using the cytoplasmic GFP signal as a visual positive marker for trans-splicing, the split-GFP system could also be utilized as an alternative way to link two SCN1A fragments together.</p>
<p>So far, I have generated a pair of split-GFP construct to examine one of the split-site. You can find more details of these experiments on <a href="https://zenodo.org/record/1473859#.W9er4pNKh3g">zenodo</a>. Going forward, I will be transfecting these constructs into HEK293 cells to examine the following questions: Do these split-protein fragments reconstitute into full-length SCN1A. If so, do they localize and function similar to wild-type SCN1A? Does overproduction of split-protein fragments cause a dominant negative effect on wild-type SCN1A?</p>
<p>&nbsp;</p>
<p><img class="aligncenter size-large wp-image-2058" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/10/dravet-pic3-1024x223.png" alt="" width="780" height="170" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/10/dravet-pic3-1024x223.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/10/dravet-pic3-300x65.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/10/dravet-pic3-768x168.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/10/dravet-pic3.png 1132w" sizes="(max-width: 780px) 100vw, 780px" /></p>
<p style="text-align: center;">Figure 4. A fluorescence reporter system for identifying SCN1A trans-splicing.</p>
<p>&nbsp;</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/2047-2/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>ACVR1 – the link between FOP and DIPG</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/acvr1-the-link-between-fop-and-dipg/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/acvr1-the-link-between-fop-and-dipg/#comments</comments>
		<pubDate>Fri, 26 Jan 2018 15:20:16 +0000</pubDate>
		<dc:creator><![CDATA[Roslin Adamson]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>
		<category><![CDATA[Roslin Adamson]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=464</guid>
		<description><![CDATA[Background What is the link between a rare genetic disease that causes soft tissues to turn to bone, and a lethal childhood brain cancer? At first glance, with such different clinical phenotypes, it seems unlikely there could be any link, but it has been shown through whole genome sequencing of diffuse intrinsic pontine glioma (DIPG) <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/acvr1-the-link-between-fop-and-dipg/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p><strong>Background</strong></p>
<p>What is the link between a rare genetic disease that causes soft tissues to turn to bone, and a lethal childhood brain cancer? At first glance, with such different clinical phenotypes, it seems unlikely there could be any link, but it has been shown through whole genome sequencing of diffuse intrinsic pontine glioma (DIPG) patients in the past few years that in actual fact, the same mutated protein that causes fibrodysplasia ossificans progressiva (FOP), is present in 27 % of DIPG cases.</p>
<p>FOP is caused in 97 % of cases by a single mutation in the gene encoding for Activin A receptor Type I (ACVR1, also known as ALK2). The disease is usually diagnosed in childhood, where minor trauma to soft tissues causes inflammation followed by heterotopic ossification (HO) of those soft tissues, and gradually over time the patient grows a “second skeleton” from hardening of soft tissues into bone. There is no cure for FOP, and current therapies are symptomatic.</p>
<p>DIPG is the leading cause of death from paediatric brain tumours. The tumours occur in the ventral pons of the brain (the brainstem), making removal by surgery impossible. Radiation therapy can marginally increase lifespan, but for most children with DIPG, survival time is approximately 9 months from diagnosis. No chemotherapeutic agents have been found to slow or halt disease progression. In addition to mutations in ACVR1 in about a quarter of cases, DIPG patients have a K27M mutation in histone protein H3, resulting in changes in gene expression that ultimately lead to tumour formation.</p>
<p>ACVR1 is a membrane-bound receptor that sits in the plasma membrane of the cell and forms a heterotetrameric complex with Type II receptors when bone morphogenetic proteins (BMPs) bind. Both Type I and II receptors have serine/threonine kinase domains intracellularly, and when BMPs bind, the Type II receptors phosphorylate the Type I receptors, thus activating them (see Fig 1).</p>
<p>&nbsp;</p>
<div id="attachment_469" style="width: 790px" class="wp-caption aligncenter"><img class="wp-image-469 size-large" src="http://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/ACVR1-signalling-pathway-1024x724.jpg" alt="" width="780" height="551" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/ACVR1-signalling-pathway-1024x724.jpg 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/ACVR1-signalling-pathway-300x212.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/ACVR1-signalling-pathway-768x543.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/ACVR1-signalling-pathway.jpg 1123w" sizes="(max-width: 780px) 100vw, 780px" /><p class="wp-caption-text">Figure 1. ACVR1 signalling pathways.<br /> Two Type I and two Type II receptors tetramerise upon binding BMP ligand. The Type II receptors phosphorylate the Type I receptors in their GS domain, whereupon the Type I receptors are able to phosphorylate Smad proteins. The Smad proteins oligomerise and translocate to the nucleus, where they activate BMP target gene transcription. BMP signalling results in ectopic bone formation, but is also responsible for a number of other functions, such as maintenance of adult tissue homeostasis. The MAPK pathway is also involved in osteoclast differentiation.</p></div>
<p>The FOP R206H mutation, also present in DIPG, is in the so-called GS (Glycine-Serine-rich) loop of the kinase domain of ACVR1, where the amino acid Arginine is replaced by Histidine, an amino acid with very different properties (Fig 2). This causes the protein to bind less tightly to its inhibitor protein FKBP12, and to exhibit altered binding activity of activin, which acts as a competitor to BMPs, and, in FOP, ultimately results in HO. Other mutations in FOP, in DIPG and in both diseases, are shown in Figure 2, but the mutations in DIPG occur only in somatic cells, not germline as in FOP. It isn’t known how these mutations interact with the histone mutations to cause oncogenesis, but ACVR1 mutations on their own very rarely cause cancer.</p>
<p>&nbsp;</p>
<div id="attachment_471" style="width: 1290px" class="wp-caption aligncenter"><img class="wp-image-471 size-full" src="http://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/ACVR1-FOP-DIPG-mutations.jpg" alt="" width="1280" height="720" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/ACVR1-FOP-DIPG-mutations.jpg 1280w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/ACVR1-FOP-DIPG-mutations-300x169.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/ACVR1-FOP-DIPG-mutations-768x432.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/ACVR1-FOP-DIPG-mutations-1024x576.jpg 1024w" sizes="(max-width: 1280px) 100vw, 1280px" /><p class="wp-caption-text">Figure 2. Structure of ACVR1 in complex with dorsomorphin with FOP and DIPG mutation locations shown.<br /> Crystal structure of the ACVR1 kinase domain, with known FOP and DIPG mutation sites shown in space-filling representation (apart from Glycines), and with the dorsomorphin inhibitor shown as sticks in pink. FOP only mutations are shown in red. Mutations found in FOP and DIPG are shown in green. G328V is found in DIPG only. PDB 3H9R</p></div>
<p>&nbsp;</p>
<p><strong>Project overview</strong></p>
<p>The aim of the project is to find some way to inhibit the pathological signalling of the mutant ACVR1 in DIPG. FOP has been studied for longer than DIPG, and the structure of the kinase domain of the protein is known (although we do not have a structure of the active protein).  With some of the lessons learnt from FOP, it may be possible to use the known scaffolds of some of the inhibitors of ACVR1 to develop compounds to treat DIPG. The problem with numerous current inhibitors is that many kinases exist in the cell, all with different functions. Many of them have a very similar mode of binding, and thus, finding effective inhibitory drugs that are very selective for a specific kinase is a challenging task. In addition, in DIPG it is necessary to find a drug that can cross the blood-brain barrier.</p>
<p>We will try two crystallographic approaches to search for potential binding compounds. The first will be to test new and existing ATP competitive molecules to improve selectivity for ACVR1. This will be through co-crystallisation of compounds with the enzyme.</p>
<p>The second approach will use a process called fragment screening to find compounds that bind allosteric sites on the protein, and may be able to be built into effective, selective drugs. Fragment screening is a method that x-ray crystallographers use to screen large numbers of compound fragments against a protein to check where and how they bind. In order to see this, the crystallographer has to solve the structure of the protein. This is done by forming crystals from concentrated solutions of protein (not a trivial task), and then soaking them in different compounds that we hope will bind to the protein. The crystals are then placed in a x-ray beam, which has a very small wavelength, such that it is able to resolve atomic distances, and an idiosyncratic diffraction pattern is obtained. This is used to computationally solve the structure of the protein, with any bound molecules. This way we can gradually build up a picture of the shapes of the molecules that can bind to active sites on the protein, and adjust them to bind tighter, or to be more specific for a particular kinase. As an adjunct to this, biochemical and biophysical assays can be done to check the affinity and activity of the compounds.</p>
<p>I have already done an initial small set of fragment screens, and am currently in the process of solving the structures to find bound molecules. Watch this space!</p>
<p>&nbsp;</p>
<p><strong>References</strong></p>
<p><a href="http://www.jbc.org/content/287/44/36990.long">Structure of the Bone Morphogenetic Protein Receptor ALK2 and Implications for Fibrodysplasia Ossificans Progressiva</a></p>
<p><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4018681/">Recurrent activating ACVR1 mutations in diffuse intrinsic pontine glioma</a></p>
<p><a href="https://www.sciencedirect.com/science/article/pii/S875632821730265X?via%3Dihub">Shared ACVR1 mutations in FOP and DIPG: Opportunities and challenges in extending biological and clinical implications across rare diseases</a></p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/acvr1-the-link-between-fop-and-dipg/feed/</wfw:commentRss>
		<slash:comments>2</slash:comments>
		</item>
		<item>
		<title>Cloning an NSD3-Short-3xFLAG Construct into a Lentiviral Expression System</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/cloning-an-nsd3-short-3xflag-construct-into-a-lentiviral-expression-system/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/cloning-an-nsd3-short-3xflag-construct-into-a-lentiviral-expression-system/#respond</comments>
		<pubDate>Thu, 18 Jan 2018 16:30:22 +0000</pubDate>
		<dc:creator><![CDATA[David Dilworth]]></dc:creator>
				<category><![CDATA[David Dilworth]]></category>
		<category><![CDATA[Leukemia]]></category>
		<category><![CDATA[NSD3]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=493</guid>
		<description><![CDATA[Antibodies are fundamental tools in the exploration of cellular biology. Unfortunately, there is a lack of high-quality NSD3 antibodies available. To overcome this, I have generated a bicistronic NSD3-Short-3xFLAG &#8211; IRES &#8211; PuroR lentiviral expression plasmid for the purpose of creating cell lines that stably express 3xFLAG-tagged NSD3-short. This will allow me to use a <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/cloning-an-nsd3-short-3xflag-construct-into-a-lentiviral-expression-system/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Antibodies are fundamental tools in the exploration of cellular biology. Unfortunately, there is a lack of high-quality NSD3 antibodies available. To overcome this, I have generated a bicistronic NSD3-Short-3xFLAG &#8211; IRES &#8211; PuroR lentiviral expression plasmid for the purpose of creating cell lines that stably express 3xFLAG-tagged NSD3-short. This will allow me to use a well-characterized anti-FLAG antibody to study NSD3 in cancer cell lines.</p>
<p>Full experimental details have been posted to <a href="https://zenodo.org/record/1154862#.WmDItq6nHIU">zenodo</a>.</p>
<div id="attachment_501" style="width: 245px" class="wp-caption aligncenter"><img class="wp-image-501 size-medium" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/NSD3_BamHI-SalI_Digest-235x300.png" alt="" width="235" height="300" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/NSD3_BamHI-SalI_Digest-235x300.png 235w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/NSD3_BamHI-SalI_Digest.png 360w" sizes="(max-width: 235px) 100vw, 235px" /><p class="wp-caption-text">Figure 1. BamHI and SalI double restriction digest of RRL-Puro lentiviral backbone and an NSD3-Short-3xFLAG PCR product in preperation for ligation.</p></div>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/cloning-an-nsd3-short-3xflag-construct-into-a-lentiviral-expression-system/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
	</channel>
</rss>
