<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>Liz Brown &#8211; openlabnotebooks.org</title>
	<atom:link href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/liz-brown/feed/" rel="self" type="application/rss+xml" />
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<description>A growing team of groundbreaking scientists around the world are now sharing their lab notebooks online</description>
	<lastBuildDate>Fri, 01 Mar 2019 19:40:25 +0000</lastBuildDate>
	<language>en-GB</language>
	<sy:updatePeriod>hourly</sy:updatePeriod>
	<sy:updateFrequency>1</sy:updateFrequency>
	<generator>https://wordpress.org/?v=4.9.9</generator>

<image>
	<url>https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/cropped-lab-jar02-32x32.png</url>
	<title>Liz Brown &#8211; openlabnotebooks.org</title>
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<width>32</width>
	<height>32</height>
</image> 
	<item>
		<title>Checking I haven&#8217;t got my cells mixed up&#8230;</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/checking-i-havent-got-my-cells-mixed-up/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/checking-i-havent-got-my-cells-mixed-up/#respond</comments>
		<pubDate>Thu, 28 Feb 2019 18:56:05 +0000</pubDate>
		<dc:creator><![CDATA[Elizabeth Brown]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Brain Tumours]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>
		<category><![CDATA[Liz Brown]]></category>
		<category><![CDATA[Understudied Kinases]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2542</guid>
		<description><![CDATA[Did you know as many as 30% of the cells in labs could be misidentified? And what a nightmare it would be to find out years of your work are just completely wrong because you got your skin cells mixed up with your brain cells&#8230; So to avoid that catastrophe, I took some time to <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/checking-i-havent-got-my-cells-mixed-up/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Did you know as many as 30% of the cells in labs could be misidentified? And what a nightmare it would be to find out years of your work are just completely wrong because you got your skin cells mixed up with your brain cells&#8230;</p>
<p>So to avoid that catastrophe, I took some time to check the DNA sequence of the ACVR1 gene, because for this gene we know what should be mutated (or not) for each of my cell lines. This way I can make sure that I haven&#8217;t mixed them up in the past year (<em>and</em> that they haven&#8217;t picked up new mutations in this important area all by themselves).</p>
<p>All I had to do was collect the genomes of the cells, amplify specifically the ACVR1 gene, send it for overnight sequencing, and then I get a trace like this back from the facility:</p>
<div id="attachment_2544" style="width: 790px" class="wp-caption aligncenter"><a href="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/R206H-sequence-1.png"><img class="wp-image-2544 size-large" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/R206H-sequence-1-1024x396.png" alt="Sequencing trace covering the R206H mutation in ACVR1 found in FOP and DIPG" width="780" height="302" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/R206H-sequence-1-1024x396.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/R206H-sequence-1-300x116.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/R206H-sequence-1-768x297.png 768w" sizes="(max-width: 780px) 100vw, 780px" /></a><p class="wp-caption-text">Sequencing trace covering the R206H mutation in ACVR1 found in FOP and DIPG</p></div>
<p>Each DNA base has one peak, except one that has both a guanine (G) and an adenine (A) DNA base. The adenine, present on one chromosome, causes the change from an arginine (R) to a histidine (H) at the 206th position of the ACVR1 protein i.e. the R206H mutation.</p>
<p>This trace is from the HSJD-DIPG-007 cell line and is the mutation I expected. According to all my sequencing, none of my cell lines have gotten mixed up, yay!</p>
<p>You can read the full details of how I did this, <a href="https://zenodo.org/record/2579809#.XHgumMD7SUk">here on Zenodo</a>.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/checking-i-havent-got-my-cells-mixed-up/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Optimising seeding of normal and FOP cells for a later assay &#8211; prettier than expected</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/optimising-seeding-of-normal-and-fop-cells-for-a-later-assay-prettier-than-expected/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/optimising-seeding-of-normal-and-fop-cells-for-a-later-assay-prettier-than-expected/#respond</comments>
		<pubDate>Tue, 22 Jan 2019 19:22:08 +0000</pubDate>
		<dc:creator><![CDATA[Elizabeth Brown]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Brain Tumours]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>
		<category><![CDATA[Liz Brown]]></category>
		<category><![CDATA[Understudied Kinases]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2388</guid>
		<description><![CDATA[Further down the line I want to be able to test the activation of the BMP signalling pathway downstream of the ALK2 receptor. For this I plan to use the alkaline phosphatase gene as a readout &#8211; it&#8217;s a well known gene activated downstream of BMP signalling so a lot of easy to use assays <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/optimising-seeding-of-normal-and-fop-cells-for-a-later-assay-prettier-than-expected/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Further down the line I want to be able to test the activation of the BMP signalling pathway downstream of the ALK2 receptor. For this I plan to use the alkaline phosphatase gene as a readout &#8211; it&#8217;s a well known gene activated downstream of BMP signalling so a lot of easy to use assays already exist for it.</p>
<p>Unfortunately <a href="https://thesgc.github.io/static-openlabnotebooks/optimising-an-assay-to-detect-alkaline-phosphatase-activity-in-c2c12-cells/">when I tested this assay with the standard C2C12 cell line it didn&#8217;t work particularly well</a>Â so I&#8217;ve decided to try again with a new set of cells. This time I&#8217;m choosing connective tissue cells (fibroblasts) from FOP patients as a reporter cell line because they respond more strongly to the signals that activate the BMP pathway (called BMP ligands).</p>
<p>I&#8217;ve never worked with these cells before so, again, I needed to work out how many I&#8217;ll need to get a good result from the assay. These cells however grow as a very thin sheet which is so see-through that I initially couldn&#8217;t tell if the wells were full. To solve this I used stains called Calcein AM and Hoechst. Calcein AM can stain any living cell because they metabolise it from a see-through molecule into a fluorescent green product, whereas Hoechst can stick to DNA and turn all the cell nuclei blue, and so suddenly, all of my cells glow:</p>
<div id="attachment_2389" style="width: 1382px" class="wp-caption aligncenter"><a href="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/Calcein-AM-example.png"><img class="wp-image-2389 size-full" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/Calcein-AM-example.png" alt="Fibroblast cells stained with Calcein AM (green) and Hoechst (blue)" width="1372" height="784" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/Calcein-AM-example.png 1372w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/Calcein-AM-example-300x171.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/Calcein-AM-example-768x439.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/Calcein-AM-example-1024x585.png 1024w" sizes="(max-width: 1372px) 100vw, 1372px" /></a><p class="wp-caption-text">Fibroblast cells stained with Calcein AM (green) and Hoechst (blue)</p></div>
<p>Aren&#8217;t they pretty?</p>
<p>Once they&#8217;re this easy to see, it&#8217;s even easier to see how many cells are needed to fill a well, but not get overcrowded. You can read more about how I did that in my <a href="https://zenodo.org/record/2546911">Zenodo post</a>.</p>
<p>The only question I have left is why some cells didn&#8217;t stain with Calcein AM, even though their nucleus looks fine? Anyone know the answer?</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/optimising-seeding-of-normal-and-fop-cells-for-a-later-assay-prettier-than-expected/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Screening ACVR1 inhibitors on mutant and non-mutant ACVR1 DIPG cells &#8211; effectiveness may vary</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/screening-acvr1-inhibitors-on-mutant-and-wild-type-acvr1-dipg-cells-effectiveness-may-vary/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/screening-acvr1-inhibitors-on-mutant-and-wild-type-acvr1-dipg-cells-effectiveness-may-vary/#respond</comments>
		<pubDate>Fri, 07 Dec 2018 17:08:35 +0000</pubDate>
		<dc:creator><![CDATA[Elizabeth Brown]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Brain Tumours]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Liz Brown]]></category>
		<category><![CDATA[Understudied Kinases]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2226</guid>
		<description><![CDATA[Hi there! The last month of my life was taken over by making sure my PhD first year report was beautifully polished, but I have returned with results of a small compound screen: These are all compounds that Jong Fu has already tested with his assays so we know they effectively inhibit ACVR1, but I <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/screening-acvr1-inhibitors-on-mutant-and-wild-type-acvr1-dipg-cells-effectiveness-may-vary/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Hi there! The last month of my life was taken over by making sure my PhD first year report was beautifully polished, but I have returned with results of a small compound screen:</p>
<p>These are all compounds that <a href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/jong-fu-wong/">Jong Fu</a> has already tested with his assays so we know they effectively inhibit ACVR1, but I thought it would be a good idea to try them out on my DIPG cells and see how well they can kill them. I included a few different variables in this screen to test what could make a difference to the outcome:</p>
<ul>
<li>I tested cells grown in 2D and 3D because this has been noted to change the outcome of compound screens in other systems</li>
<li>I tested the compounds alone, or in addition to radiation in order to test whether the compounds were effective in addition to current DIPG treatments.</li>
</ul>
<p>I tested how effective each compound is by growing cells for 7 days in the presence of the compound. At the end I measured how alive or healthy the cells were (i.e. their viability) by using a pre-developed kit that measures the amount of ATP inside the cell by using an enzyme that can produce light when ATP is present. ATP is measured because this is sometimes called the &#8216;energy currency of the cell&#8217; so living cells have to have ATP inside them, whereas dead cells won&#8217;t and won&#8217;t be counted.</p>
<p>This screen produced a lot of data at once which is best visualised as heatmaps like these:</p>
<div id="attachment_2227" style="width: 790px" class="wp-caption aligncenter"><a href="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/ALK2i-figure-heatmap.png"><img class="size-large wp-image-2227" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/ALK2i-figure-heatmap-860x1024.png" alt="" width="780" height="929" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/ALK2i-figure-heatmap-860x1024.png 860w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/ALK2i-figure-heatmap-252x300.png 252w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/12/ALK2i-figure-heatmap-768x914.png 768w" sizes="(max-width: 780px) 100vw, 780px" /></a><p class="wp-caption-text"><strong>ACVR1 inhibitor M4K2009 is best at reducing mutant ACVR1 DIPG cell viability, but is not effective in ACVR1 wild-type cells</strong><br />Heatmaps showing the viability (relative to an untreated control) of a 2D culture of mutant ACVR1 DIPG cells (A) or wild-type ACVR1 DIPG cells (B) following 7 days in the presence of 1Î¼M of the indicated ALK2 inhibitors. The colour of each box indicates how viable the cells were at the end (where blue indicates an improvement in viability and red indicates a drop in viability). Each column indicates an individual (technical repeats, n1-3) grouped into those done at the same time (biological repeats 1-3).</p></div>
<p>From that heatmap you can see that M4K2009 is the best at killing ACVR1 mutant cells (the top heatmap), but it isn&#8217;t as effective in cells where the ACVR1 gene is not mutated (&#8220;wild-type&#8221; cells). In those cells instead M4K2096 is the most effective compound (but still not to the same extent as M4K2009).</p>
<p>But this is only the data for cells grown in 2 dimensions without a radiation treatment. If you want to see the full dataset, you can read my <a href="https://zenodo.org/record/2000673#.XAqo_FX7SUk">Zenodo post</a>!</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/screening-acvr1-inhibitors-on-mutant-and-wild-type-acvr1-dipg-cells-effectiveness-may-vary/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Presenting a poster at the annual Medical Sciences Division DPhil Day</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/presenting-a-poster-at-the-annual-medical-sciences-division-dphil-day/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/presenting-a-poster-at-the-annual-medical-sciences-division-dphil-day/#comments</comments>
		<pubDate>Wed, 29 Aug 2018 20:44:35 +0000</pubDate>
		<dc:creator><![CDATA[Elizabeth Brown]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Brain Tumours]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Liz Brown]]></category>
		<category><![CDATA[Understudied Kinases]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=1776</guid>
		<description><![CDATA[Every summer my department holds a special mini-conference for the DPhil (aka PhD) students so that we can get to know each other and practice presenting our research. This year I took along a poster outlining my own project â Understanding the pathogenic mechanism of ACVR1 mutations in Diffuse Intrinsic Pontine Glioma â and presented <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/presenting-a-poster-at-the-annual-medical-sciences-division-dphil-day/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Every summer my department holds a special mini-conference for the DPhil (aka PhD) students so that we can get to know each other and practice presenting our research.</p>
<p>This year I took along a poster outlining my own project â Understanding the pathogenic mechanism of ACVR1 mutations in Diffuse Intrinsic Pontine Glioma â and presented it to my peers (<a href="https://zenodo.org/record/1403108#.W4AH2s5KiUk">you can view it here</a>). I still find presenting my work a nerve-wracking experience, it doesnât come naturally to me at all, but I really enjoyed the chance to practice, and to take notes on how everyone else is presenting their work.</p>
<div id="attachment_1778" style="width: 790px" class="wp-caption aligncenter"><a href="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/08/IMG_20180720_165712-2.jpg"><img class="wp-image-1778 size-large" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/08/IMG_20180720_165712-2-1024x683.jpg" alt="Image of Elizabeth Brown standing beside her research poster." width="780" height="520" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/08/IMG_20180720_165712-2-1024x683.jpg 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/08/IMG_20180720_165712-2-300x200.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/08/IMG_20180720_165712-2-768x512.jpg 768w" sizes="(max-width: 780px) 100vw, 780px" /></a><p class="wp-caption-text">Me with my poster!</p></div>
<p>We also had an enlightening panel discussion about the publishing industry, and a brilliant key-note lecture from <a href="http://www.ox.ac.uk/news/2016-10-17-professor-kevin-marsh-recognised-contributions-health-africa">Professor Kevin Marsh</a> about the growth of scientific research in Africa and the <a href="http://www.afox.ox.ac.uk/">Africa Oxford Initiative</a> , which Iâve been chewing over ever since. Sometimes itâs daunting to take time away from the lab, but I find that the things I learn at conferences are always super useful!</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/presenting-a-poster-at-the-annual-medical-sciences-division-dphil-day/feed/</wfw:commentRss>
		<slash:comments>1</slash:comments>
		</item>
		<item>
		<title>Optimising an assay to detect alkaline phosphatase activity in C2C12 cells</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/optimising-an-assay-to-detect-alkaline-phosphatase-activity-in-c2c12-cells/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/optimising-an-assay-to-detect-alkaline-phosphatase-activity-in-c2c12-cells/#respond</comments>
		<pubDate>Mon, 18 Jun 2018 15:05:53 +0000</pubDate>
		<dc:creator><![CDATA[Elizabeth Brown]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Brain Tumours]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Liz Brown]]></category>
		<category><![CDATA[Understudied Kinases]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=1360</guid>
		<description><![CDATA[I donât just want to check whether the compounds Iâm testing kill DIPG cells, I want to check that theyâre doing this by altering BMP signalling too. However, I canât do this with the DIPG cells, because if a compound does work, and kill them, thereâll be no cell left to actually measure the amount <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/optimising-an-assay-to-detect-alkaline-phosphatase-activity-in-c2c12-cells/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>I donât just want to check whether the compounds Iâm testing kill DIPG cells, I want to check that theyâre doing this by altering BMP signalling too. However, I canât do this with the DIPG cells, because if a compound <em>does</em> work, and kill them, thereâll be no cell left to actually measure the amount of BMP signalling. For this purpose, I need to use my trusty C2C12 myoblast cell line instead, because they donât rely on BMP signalling to survive (like DIPG cells hopefully do), and because we know it has active BMP signalling that can be measured.</p>
<p>I chose to use the alkaline phosphatase (ALP) protein as an indication of how much BMP signalling has happened in the cell, because ALPâs production within the cell is activated by BMP signalling, and because this is a tried and tested method. In fact, I got instructions for how to carry out the test for another PhD student in the building (thank-you Chao!) At the end of the experiment all Iâll need to do is break open the cells (lyse them) and add 4-methylumbelliferyl (4-MUP) which ALP converts into a fluorescent product that I can easily measure.</p>
<p>Of course, as you may have noticed, setting up a new assay takes a little prior work to check exactly how it should be done. So, in this experiment I have tested how much ligand (BMP-7, which activates the signalling cascade) Iâll need to add in order to see a measurable response. I seeded enough cells to grow happily for the length of the experiment (only 110!), and then 3 days before the end I added different concentrations of BMP-7 to each well. (<a href="https://zenodo.org/record/1292062#.WyfKWFVKiUk">You can read the complete method here.</a>) This is the graph I got out at the end:</p>
<div id="attachment_1362" style="width: 790px" class="wp-caption aligncenter"><a href="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/06/ALP-activity-fold-changes-graph-ON.png"><img class="wp-image-1362 size-large" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/06/ALP-activity-fold-changes-graph-ON-1024x866.png" alt="" width="780" height="660" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/06/ALP-activity-fold-changes-graph-ON-1024x866.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/06/ALP-activity-fold-changes-graph-ON-300x254.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/06/ALP-activity-fold-changes-graph-ON-768x649.png 768w" sizes="(max-width: 780px) 100vw, 780px" /></a><p class="wp-caption-text">The fold change in alkaline phosphatase (ALP) activity (measured as fluorescence following 4-MUP addition) after 3 days of growth in the presence of BMP-7.</p></div>
<p>As you can see, adding more ligand increases the amount of ALP produced (no surprise there), but it doesnât really improve above 400ng/mL so I may not go higher than that. The amount of ALP produced when I added the ligand only 1 day before the end was far less, so I donât think I can use that time point (I wanted to test it out because I think that the longer the cells are treated with a compound before having the ligand added the bigger difference on BMP signalling it would have). Actually, as a 3 fold increase is quite small and 400ng/mL is quite a lot of BMP-7, I may go on to test adding BMP-7 to the cells even earlier.</p>
<p>What do you think?</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/optimising-an-assay-to-detect-alkaline-phosphatase-activity-in-c2c12-cells/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Attending the first ever CRUK London Brain Tumour Conference</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/attending-the-first-ever-cruk-london-brain-tumour-conference/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/attending-the-first-ever-cruk-london-brain-tumour-conference/#comments</comments>
		<pubDate>Fri, 11 May 2018 13:02:47 +0000</pubDate>
		<dc:creator><![CDATA[Elizabeth Brown]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Brain Tumours]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Jong Fu Wong]]></category>
		<category><![CDATA[Liz Brown]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=1197</guid>
		<description><![CDATA[Hello there! This week I wonât be filling you in on my latest experiment, because instead I spent three days at the first ever Cancer Research UK Brain Tumour Conference held in the Royal Society of Medicine in London! (Also the biggest conference Iâve been to so far). The conference gathered over 30 great speakers <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/attending-the-first-ever-cruk-london-brain-tumour-conference/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Hello there! This week I wonât be filling you in on my latest experiment, because instead I spent three days at the first ever Cancer Research UK Brain Tumour Conference held in the Royal Society of Medicine in London! (Also the biggest conference Iâve been to so far).</p>
<p>The conference gathered over 30 great speakers from around the world to give talks on both adult and paediatric brain tumours, and that covered topics from innovating clinical trial design, to best benefit patients, toÂ single cell -omics (i.e. studyingÂ allÂ of the DNA, RNA, protein and more present in a cell at one time).</p>
<p>Actually, there were three fantastic speakers currently studying DIPG:</p>
<ul>
<li><a href="https://med.stanford.edu/monje-lab.html">Michelle Monje</a> who did fascinating work into the fact that glioma cells can generate electrical impulses and form synapses with normal brain cells</li>
<li><a href="http://www.feinberg.northwestern.edu/faculty-profiles/az/profile.html?xid=37051">Oren Becher</a> who has made a whole suite of pre-clinical DIPG models</li>
<li>and Diana Carvalho (from <a href="https://www.icr.ac.uk/our-research/researchers-and-teams/professor-chris-jones">Chris Jonesâ lab</a>) who has been screening possible therapeutic compounds</li>
</ul>
<div id="attachment_1198" style="width: 4042px" class="wp-caption aligncenter"><a href="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/05/Oren-Becher-talk.jpg"><img class="wp-image-1198 size-full" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/05/Oren-Becher-talk.jpg" alt="" width="4032" height="1797" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/05/Oren-Becher-talk.jpg 4032w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/05/Oren-Becher-talk-300x134.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/05/Oren-Becher-talk-768x342.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/05/Oren-Becher-talk-1024x456.jpg 1024w" sizes="(max-width: 4032px) 100vw, 4032px" /></a><p class="wp-caption-text">Oren Becher giving a great talk on his new pre-clinical DIPG models.</p></div>
<p>In addition to set talks the speakers took part in panel discussions that created the space for some quite heated debates ! (How can clinicians, pathologists and lab scientists best work together to provide tissue for laboratory research? Weâre still sorting it outâ¦)</p>
<p>Finally, attendees presented over 80 different posters, including one from our very own lab that provided an overview of our groupâs research on DIPG! (<a href="https://zenodo.org/record/1246855#.WvtF5ogvzIU">You can read it here on Zenodo</a>).</p>
<p>You can see what everyone else has been saying about it at <a href="https://twitter.com/hashtag/crukbrainconf?src=hash">#CRUKBrainConf on Twitter</a></p>
<p>But now, Iâll get back to my experimentsâ¦</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/attending-the-first-ever-cruk-london-brain-tumour-conference/feed/</wfw:commentRss>
		<slash:comments>1</slash:comments>
		</item>
		<item>
		<title>Making single DIPG spheres</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/making-single-dipg-spheres/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/making-single-dipg-spheres/#respond</comments>
		<pubDate>Wed, 25 Apr 2018 20:40:55 +0000</pubDate>
		<dc:creator><![CDATA[Elizabeth Brown]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Brain Tumours]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Liz Brown]]></category>
		<category><![CDATA[Understudied Kinases]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=1045</guid>
		<description><![CDATA[Itâs usually a good idea to have more than one trick up your sleeve&#8230; So with that in mind, Iâve started developing another viability assay for my DIPG cell lines. The first assay relied on convincing a cell line that naturally grows as floating spheres to grow stuck down on the bottom of plastic wells, <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/making-single-dipg-spheres/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Itâs usually a good idea to have more than one trick up your sleeve&#8230; So with that in mind, Iâve started developing <em>another</em> viability assay for my DIPG cell lines. The first assay relied on convincing a cell line that naturally grows as floating spheres to grow stuck down on the bottom of plastic wells, but with this one I want to let them grow as a sphere.</p>
<p>However, the tricky part is to make sure the assay starts with one sphere of a reliable size, because this makes it easier to repeat the experiment with the cells behaving the same way each time. So I had to test how many cells to seed (as few as 10, right up to 2000 cells), and try a couple of different treatments. For example, I tried spinning them at 300 times the force of gravity to see if forcing them together at the bottom of the plate would help them form a sphere.</p>
<p>In the end this cell line is very happy to form tight regular spheres:</p>
<p><a href="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/04/Grown-spheres-image.png"><img class="aligncenter wp-image-1046 size-large" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/04/Grown-spheres-image-1024x561.png" alt="Brightfield images with red fluorescence overlaid (propidium iodide indicating cell death) of single DIPG spheres." width="780" height="427" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/04/Grown-spheres-image-1024x561.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/04/Grown-spheres-image-300x164.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/04/Grown-spheres-image-768x421.png 768w" sizes="(max-width: 780px) 100vw, 780px" /></a></p>
<p>(The red areas are where a compound I added on day 3, called propidium iodide, was able to soak into dead cells and show me when the spheres start to die in the middle because theyâre too big for oxygen or nutrients to diffuse into).</p>
<p>In fact the spheres looked so good that I went on to try out something for another interesting assay: On a subset of the spheres I added matrigel, an extract from a mouse tumour that mimics the scaffold (a.k.a. the extracellular matrix) in which cells grow.Â When cancer cells are grown like this they tend to invade into the matrix, as a tumour would in the body:</p>
<div id="attachment_1047" style="width: 790px" class="wp-caption aligncenter"><a href="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/04/Migrating-spheres-image.png"><img class="size-large wp-image-1047" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/04/Migrating-spheres-image-1024x552.png" alt="" width="780" height="420" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/04/Migrating-spheres-image-1024x552.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/04/Migrating-spheres-image-300x162.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/04/Migrating-spheres-image-768x414.png 768w" sizes="(max-width: 780px) 100vw, 780px" /></a><p class="wp-caption-text">Brightfield images of DIPG cells invading from a single sphere into a 3D matrix of matrigel.</p></div>
<p>I can then check if particular conditions or therapeutic compounds encourage or prevent this invasion, and thatâs a useful thing to know, because invasion into normal tissue is one of the behaviours that make cancer so dangerous to our bodies.</p>
<p>If you want to read about the method in detail, <a href="https://doi.org/10.5281/zenodo.1218669">I&#8217;ve uploaded a full report here</a>.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/making-single-dipg-spheres/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Developing an assay of viability in DIPG cell lines</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/developing-an-assay-of-viability-in-dipg-cell-lines/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/developing-an-assay-of-viability-in-dipg-cell-lines/#comments</comments>
		<pubDate>Wed, 28 Mar 2018 22:11:25 +0000</pubDate>
		<dc:creator><![CDATA[Elizabeth Brown]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Brain Tumours]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Liz Brown]]></category>
		<category><![CDATA[Understudied Kinases]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=984</guid>
		<description><![CDATA[Over the past few weeks Iâve been perfecting a method for screening a small set of therapeutic compounds on DIPG cell lines. Before starting I made sure to discuss the techniques used in other groups to make sure that I design an assay that is comparable to theirs, although I may make my own changes <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/developing-an-assay-of-viability-in-dipg-cell-lines/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Over the past few weeks Iâve been perfecting a method for screening a small set of therapeutic compounds on DIPG cell lines. Before starting I made sure to discuss the techniques used in other groups to make sure that I design an assay that is comparable to theirs, although I may make my own changes later, and see if this makes a difference to how effective drugs are.</p>
<p>My first attempt wasnât useful as a drug assay, but did allow me to measure the growth of the cells over time:</p>
<div id="attachment_986" style="width: 790px" class="wp-caption aligncenter"><a href="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/03/2nd-analysis-good-graph.png"><img class="wp-image-986 size-large" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/03/2nd-analysis-good-graph-1024x645.png" alt="" width="780" height="491" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/03/2nd-analysis-good-graph-1024x645.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/03/2nd-analysis-good-graph-300x189.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/03/2nd-analysis-good-graph-768x484.png 768w" sizes="(max-width: 780px) 100vw, 780px" /></a><p class="wp-caption-text">A graph of the growth of DIPG cells over time when growing adherently.</p></div>
<p>This was a great help for making choices about how many cells to seed and what time periods to use in the assay.</p>
<p>Eight iterations later, I now have an initial proof-of-concept dataset, and a (mostly) final protocol. The general scheme of the assay is as follows:</p>
<p>Coat the plates with laminin, a protein that allows the cells to stick to the plate, instead of growing as floating spheres (which is the current growth mode of choice in the field). Seed 2,000 cells into each well, in their usual growth medium, and let them grow undisturbed for 3 days. I then add a range of concentrations of each drug and a marker for cell death to the wells (with three repeats each). After four days, I add a solution (called CellTiter-Glo) that contains a luciferase enzyme and therefore glows in the presence of ATP. As ATP is only found in live cells, this light provides an indication of the number of live cells present.</p>
<p>Once I detect the luciferase glow with a plate reader, I can plot a curve of how many cells are alive at the end of each treatment (or, how many are dead if I look at the cell death marker), which looks like this:</p>
<div id="attachment_987" style="width: 510px" class="wp-caption aligncenter"><a href="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/03/Viability-assay-diagram.jpg"><img class="wp-image-987" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/03/Viability-assay-diagram-1024x922.jpg" alt="" width="500" height="450" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/03/Viability-assay-diagram-1024x922.jpg 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/03/Viability-assay-diagram-300x270.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/03/Viability-assay-diagram-768x691.jpg 768w" sizes="(max-width: 500px) 100vw, 500px" /></a><p class="wp-caption-text">Top: a visual representation of the luminescence readings recorded<br /> Bottom: a graph produced from such readings which illustrated the reduction of cell viability caused by drug treatment</p></div>
<p>For this compound, the assay worked well, however with another I noticed that I only seemed to be able to see inhibition for three days, and not four. This could be because the drug degrades or is digested over time and therefore that I may need to add the drug more than once in future .</p>
<p>You can read the complete method, and the full dataset <a href="https://zenodo.org/record/1208124#.WrwS5ojwbIU">on this page</a>.</p>
<p>P.S. If anyone has tips on how to present each biological repeat, my ears are open !</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/developing-an-assay-of-viability-in-dipg-cell-lines/feed/</wfw:commentRss>
		<slash:comments>3</slash:comments>
		</item>
		<item>
		<title>Recording the growth of a DIPG cell line</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/recording-the-growth-of-a-dipg-cell-line/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/recording-the-growth-of-a-dipg-cell-line/#respond</comments>
		<pubDate>Fri, 23 Feb 2018 17:37:46 +0000</pubDate>
		<dc:creator><![CDATA[Elizabeth Brown]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Brain Tumours]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Liz Brown]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=660</guid>
		<description><![CDATA[Repeatability is a cornerstone of science, but cells behaviour can change quite significantly if you aren&#8217;t careful when looking after them. To ensure that I have the right number of DIPG cells to complete an important experiment, and that theyâre all growing healthily, I want to record how fast they grow, and at what density <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/recording-the-growth-of-a-dipg-cell-line/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Repeatability is a cornerstone of science, but cells behaviour can change quite significantly if you aren&#8217;t careful when looking after them. To ensure that I have the right number of DIPG cells to complete an important experiment, and that theyâre all growing healthily, I want to record how fast they grow, and at what density they die or change in behaviour. (This will be <em>especially</em> important when I start comparing DIPG cell lines with different mutations, that all grow at different rates.)</p>
<p>Doing this is quite simple: I seeded patient derived HSJD-DIPG-07 cells in flasks from high to low density, with enough repeats that I can harvest the cells every two or three days, dye dead cells with trypan blue, and <a href="https://bitesizebio.com/13687/cell-counting-with-a-hemocytometer-easy-as-1-2-3/">count them in a haemocytometer</a>. You can read exactly how I did it <a href="https://zenodo.org/record/1172048#.WpBRSIPFKUk">here</a>.</p>
<p><span style="font-size: 1.5rem;">After a week of growth I could make a few observations:</span></p>
<p><span style="font-size: 1.5rem;">More than a million cells in a flask dirty their medium (turning it yellow) in less than three days (the rate at which I want to feed them). More than </span><em style="font-size: 1.5rem;">two </em><span style="font-size: 1.5rem;">million cells are so overcrowded that cells start dying in greater numbers, and theyâre so overcrowded and unhappy that the spheres they grow in start sticking to the bottom of the flask instead of floating in the medium. Thatâs a very noticeable change in behaviour, and something I definitely want to avoid during experiments or drug screening.</span></p>
<div id="attachment_663" style="width: 310px" class="wp-caption aligncenter"><a href="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/02/IMG_20180201_145119_2.jpg"><img class="wp-image-663 size-medium" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/02/IMG_20180201_145119_2-300x252.jpg" alt="A stack of flasks containing DIPG spheroids in suspension - the number of cells increases from top to bottom and is reflected in the colour change of the growth medium from pink to yellow" width="300" height="252" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/02/IMG_20180201_145119_2-300x252.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/02/IMG_20180201_145119_2-768x644.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/02/IMG_20180201_145119_2-1024x859.jpg 1024w" sizes="(max-width: 300px) 100vw, 300px" /></a><p class="wp-caption-text">A stack of flasks containing DIPG spheroids in suspension &#8211; the number of cells increases from top to bottom and is reflected in the colour change of the growth medium from pink to yellow</p></div>
<p>But even in such an unhealthy state, their growth didnât slow down:</p>
<div id="attachment_664" style="width: 790px" class="wp-caption aligncenter"><a href="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/02/Data-1.png"><img class="wp-image-664 size-large" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/02/Data-1-1024x828.png" alt="A graph showing the increasing cell number over 168h of the HSJD-DIPG-07 cell line." width="780" height="631" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/02/Data-1-1024x828.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/02/Data-1-300x242.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/02/Data-1-768x621.png 768w" sizes="(max-width: 780px) 100vw, 780px" /></a><p class="wp-caption-text">Either 300,000 cells (blue), 150,000 cells (red), 75,000 cells (green), or 37,500 cells (purple) were seeded and their final cell number recorded at 48, 96, or 186 hours after the initial split.</p></div>
<p>&nbsp;</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/recording-the-growth-of-a-dipg-cell-line/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Optimising transfection of the C2C12 myoblast cell line</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/optimising-transfection-of-the-c2c12-myoblast-cell-line/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/optimising-transfection-of-the-c2c12-myoblast-cell-line/#comments</comments>
		<pubDate>Wed, 31 Jan 2018 18:22:31 +0000</pubDate>
		<dc:creator><![CDATA[Elizabeth Brown]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Brain Tumours]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>
		<category><![CDATA[Liz Brown]]></category>
		<category><![CDATA[Understudied Kinases]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=586</guid>
		<description><![CDATA[Within a month (hopefully) Iâll be convincing C2C12 myoblast cells to express all kinds of mutant ALK2 to test its activity and interactions with other proteins within the cell. Iâll be doing this by treating them so that theyâll take up pieces of circular DNA (plasmids) that express those mutant proteins (i.e. transfecting them). Seeing <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/optimising-transfection-of-the-c2c12-myoblast-cell-line/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Within a month (hopefully) Iâll be convincing <a href="https://en.wikipedia.org/wiki/C2C12">C2C12 myoblast cells</a> to express all kinds of mutant ALK2 to test its activity and interactions with other proteins within the cell. Iâll be doing this by treating them so that theyâll take up pieces of circular DNA (plasmids) that express those mutant proteins (i.e. transfecting them).</p>
<p>Seeing as Iâm going to be doing that countless times, it makes sense to find the most efficient way to do that, whilst I wait for extra plasmids I need to arrive. Iâm looking for the way that will use the smallest amount of DNA and reagents, but will still transfect a high proportion of cells. So far, Iâve been using <a href="https://www.thermofisher.com/uk/en/home/brands/product-brand/lipofectamine/lipofectamine-2000.html">Lipofectamine-2000</a>, which traps the DNA in a lipid capsule and helps the cell absorb it, but I heard that the chemical polyethylenimine (a positive compound that helps DNA to stick to the negative surface of the cell) could work just as well with less DNA.</p>
<p>To compare the two methods, I performed them both on the cells with a plasmid that will express <a href="https://en.wikipedia.org/wiki/Green_fluorescent_protein">enhanced green fluorescent protein</a> (EGFP), because it visibly marks the cells which have been transfected:</p>
<div id="attachment_587" style="width: 790px" class="wp-caption aligncenter"><a href="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/180129-Transfection-image.png"><img class="wp-image-587 size-large" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/180129-Transfection-image-1024x355.png" alt="An image of C2C12 cells in culture, some of which express EGFP due to transfection" width="780" height="270" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/180129-Transfection-image-1024x355.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/180129-Transfection-image-300x104.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/180129-Transfection-image-768x267.png 768w" sizes="(max-width: 780px) 100vw, 780px" /></a><p class="wp-caption-text">Left: A brightfield image of C2C12 cells overlaid with the fluorescent image from expressed EGFP.<br /> Right: The fluorescent image alone.</p></div>
<p>Now I know that treating cells with PEI at a ratio of 4x the volume of PEI to the mass of DNA transfects more cells than the previous method that relied on Lipofectamine-2000. <a href="https://zenodo.org/record/1162362#.WnI1F6hl_IU">Full experimental details can be read here</a>.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/optimising-transfection-of-the-c2c12-myoblast-cell-line/feed/</wfw:commentRss>
		<slash:comments>2</slash:comments>
		</item>
	</channel>
</rss>
