<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>Nicola De Maio &#8211; openlabnotebooks.org</title>
	<atom:link href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/nicola-de-maio/feed/" rel="self" type="application/rss+xml" />
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<description>A growing team of groundbreaking scientists around the world are now sharing their lab notebooks online</description>
	<lastBuildDate>Mon, 01 Jun 2020 14:03:29 +0000</lastBuildDate>
	<language>en-GB</language>
	<sy:updatePeriod>
	hourly	</sy:updatePeriod>
	<sy:updateFrequency>
	1	</sy:updateFrequency>
	<generator>https://wordpress.org/?v=5.5.3</generator>

<image>
	<url>https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/cropped-lab-jar02-32x32.png</url>
	<title>Nicola De Maio &#8211; openlabnotebooks.org</title>
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<width>32</width>
	<height>32</height>
</image> 
	<item>
		<title>Genetic diversity at the catalytic site of the main SARS-CoV-2 protease</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/genetic-diversity-at-the-catalytic-site-of-the-main-sars-cov-2-protease/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/genetic-diversity-at-the-catalytic-site-of-the-main-sars-cov-2-protease/#respond</comments>
		
		<dc:creator><![CDATA[Nicola De Maio]]></dc:creator>
		<pubDate>Mon, 01 Jun 2020 09:57:10 +0000</pubDate>
				<category><![CDATA[COVID-19]]></category>
		<category><![CDATA[Nick Goldman (EBI)]]></category>
		<category><![CDATA[Nicola De Maio]]></category>
		<category><![CDATA[SARS-CoV-2 computational genomics]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=4818</guid>

					<description><![CDATA[We investigated genetic variation of SARS-CoV-2 at the catalytic site of the main SARS-CoV-2 protease, following the work of Setayesh Yazdani (see https://zenodo.org/record/3834875#.Xs1IHsZ7nyk and https://thesgc.github.io/static-openlabnotebooks/mapping-the-genetic-variations-of-sars-cov-2-onto-its-proteins-crystal-structures-post-1/ ). We collected information from more than 15,000 genomic sequences available from GISAID (https://www.epicov.org/) available on the 17th of May 2020, givig us considerable power to detect viral genetic variation <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/genetic-diversity-at-the-catalytic-site-of-the-main-sars-cov-2-protease/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p>We investigated genetic variation of SARS-CoV-2 at the catalytic site of the main SARS-CoV-2 protease, following the work of <span class="byline"><span class="author vcard">Setayesh Yazdani</span></span> (see <a href="https://zenodo.org/record/3834875#.Xs1IHsZ7nyk">https://zenodo.org/record/3834875#.Xs1IHsZ7nyk</a> and <a href="https://thesgc.github.io/static-openlabnotebooks/mapping-the-genetic-variations-of-sars-cov-2-onto-its-proteins-crystal-structures-post-1/">https://thesgc.github.io/static-openlabnotebooks/mapping-the-genetic-variations-of-sars-cov-2-onto-its-proteins-crystal-structures-post-1/ </a>).<br />
We collected information from more than 15,000 genomic sequences available from GISAID (<a href="https://www.epicov.org/">https://www.epicov.org/</a>) available on the 17th of May 2020, givig us considerable power to detect viral genetic variation within the current outbreak.</p>
<p>Due to the possible presence of errors within the sequences considered, We used a new approach to detect variants that are likely the result of sequencing issues, and that are therefore not reliable. Our analysis is based on phylogenetic inference of homoplasy (how many independent times the genetic mutation seems to have appeared along the phylogenetic tree), on the clustering of variants along the genome, on frequent ambiguous characters (positions of the genome where the sequence seems uncertain), and on the sequence metadata (in particular the laboratory that generated the sequence). The idea behind our approach is that sequencing artefacts are likely to result in homoplasic patterns, in ambiguous characters, in clustered rare variants, and in variants only detected in specific sequencing laboratories (or more generally specific sequencing and sample preparation methods).<br />
For details of our methods, see <a href="https://zenodo.org/record/3865719#.XtJ9UcZ7nyl">https://zenodo.org/record/3865719#.XtJ9UcZ7nyl</a> and <a href="http://virological.org/t/issues-with-sars-cov-2-sequencing-data/473">http://virological.org/t/issues-with-sars-cov-2-sequencing-data/473</a> .</p>
<p>We find that the positions considered are mostly conserved.<br />
Only one position considered (Thr25) seems affected by a likely sequencing artefact, which however results only in synonymous variants.<br />
The amino acid variants detected are only four: M49I, P52S, N142S, and P168S, and all of them appear only at extremely low frequencies (maximum of two samples each). More details for each position considered are given in Table 1 below.</p>
<p><a href="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/Table1.pdf">Table1</a></p>
<p>&nbsp;</p>
<p>&nbsp;</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/genetic-diversity-at-the-catalytic-site-of-the-main-sars-cov-2-protease/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
	</channel>
</rss>
