<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>Huntingtin &#8211; openlabnotebooks.org</title>
	<atom:link href="https://thesgc.github.io/static-openlabnotebooks/category/project/huntingtin/feed/" rel="self" type="application/rss+xml" />
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<description>A growing team of groundbreaking scientists around the world are now sharing their lab notebooks online</description>
	<lastBuildDate>Mon, 01 Jul 2019 16:49:26 +0000</lastBuildDate>
	<language>en-GB</language>
	<sy:updatePeriod>hourly</sy:updatePeriod>
	<sy:updateFrequency>1</sy:updateFrequency>
	<generator>https://wordpress.org/?v=4.9.10</generator>

<image>
	<url>https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/cropped-lab-jar02-32x32.png</url>
	<title>Huntingtin &#8211; openlabnotebooks.org</title>
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<width>32</width>
	<height>32</height>
</image> 
	<item>
		<title>The Year in Review</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/the-year-in-review/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/the-year-in-review/#respond</comments>
		<pubDate>Thu, 27 Jun 2019 16:24:03 +0000</pubDate>
		<dc:creator><![CDATA[Tamara Maiuri]]></dc:creator>
				<category><![CDATA[Huntingtin]]></category>
		<category><![CDATA[Huntington’s disease]]></category>
		<category><![CDATA[Ray Truant]]></category>
		<category><![CDATA[Tamara Maiuri]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=3034</guid>
		<description><![CDATA[A summary of the Berman Topper second year-end progress report, with links to the full report, are now posted on the Truant lab website.]]></description>
				<content:encoded><![CDATA[<p>A summary of the Berman Topper second year-end progress report, with links to the full report, are now posted on the <a href="https://raytruantlab.ca/2019/06/27/the-year-in-review/">Truant lab website</a>.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/06/Screen-Shot-2019-06-27-at-11.38.21-AM-300x110.png" alt="" width="300" height="110" class="alignnone size-medium wp-image-3035" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/06/Screen-Shot-2019-06-27-at-11.38.21-AM-300x110.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/06/Screen-Shot-2019-06-27-at-11.38.21-AM-768x280.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/06/Screen-Shot-2019-06-27-at-11.38.21-AM-1024x374.png 1024w" sizes="(max-width: 300px) 100vw, 300px" /></p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/the-year-in-review/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Production and Purification of Huntingtin interaction partners continued</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/production-and-purification-of-huntingtin-interaction-partners-continued/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/production-and-purification-of-huntingtin-interaction-partners-continued/#respond</comments>
		<pubDate>Tue, 25 Jun 2019 18:23:10 +0000</pubDate>
		<dc:creator><![CDATA[Jacob McAuley]]></dc:creator>
				<category><![CDATA[Huntingtin]]></category>
		<category><![CDATA[Huntington’s disease]]></category>
		<category><![CDATA[Jacob McAuley]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=3010</guid>
		<description><![CDATA[As the summer continues to progress, so does my project. After the last round of purification, detailed in my previous blog post, I am ready for another round. During this purification, I will be working with SH3G3 and PACN1. PACN1 is responsible for cytoskeletal remodeling through regulation of several other proteins, as well as neuronal <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/production-and-purification-of-huntingtin-interaction-partners-continued/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>As the summer continues to progress, so does my project. After the last round of purification, detailed in my <a href="https://thesgc.github.io/static-openlabnotebooks/first-round-of-huntingtin-interaction-partner-protein-production/">previous blog post</a>, I am ready for another round. During this purification, I will be working with <a href="https://www.uniprot.org/uniprot/Q9BY11">SH3G3</a> and <a href="https://www.uniprot.org/uniprot/Q99963">PACN1</a>. PACN1 is responsible for cytoskeletal remodeling through regulation of several other proteins, as well as neuronal vesicle endocytosis, among other things. It is thought that SH3G3 is more specialized in aiding endocytosis. Luckily, the purification of both proteins was successful, although that’s not to say everything went to plan.</p>
<p>The tags on these proteins allowed for multiple possible binding possibilities, meaning we had different avenues we could try for the purification. In the first attempt, we used SBP resin. In theory, the SBP would bind our protein of interest, allowing us to purify it, but in practice, the binding affinity was disappointingly low. In an attempt to salvage the purification, we then ran through a nickel purification, reverse nickel purification and size exclusion chromatography. Because of the design of the tag, our protein binds Nickel, but some proteins that occur naturally will also bind Nickel, so after collecting all of the nickel binding proteins, we can cut off the tag on our protein, using TEV, leaving it unable to bind Nickel. This means that is we mix nickel resin with the proteins; everything should bind the Nickel except our protein. After this, we ran it through a column with gel beads that contain tiny pores. Small proteins get trapped in the pores while larger proteins glide through, this gives us an additional way to purify our protein. For complete methods and data for <a href="https://zenodo.org/record/3256579#.XRJllIhKjcs">SH3G3</a> and <a href="https://zenodo.org/record/3256589#.XRJlr4hKjcs">PACN1</a> purifications, look to my posts on Zenodo.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/production-and-purification-of-huntingtin-interaction-partners-continued/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>First Round of Huntingtin Interaction Partner Protein Production</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/first-round-of-huntingtin-interaction-partner-protein-production/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/first-round-of-huntingtin-interaction-partner-protein-production/#respond</comments>
		<pubDate>Fri, 21 Jun 2019 19:42:06 +0000</pubDate>
		<dc:creator><![CDATA[Jacob McAuley]]></dc:creator>
				<category><![CDATA[Huntingtin]]></category>
		<category><![CDATA[Huntington’s disease]]></category>
		<category><![CDATA[Jacob McAuley]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2973</guid>
		<description><![CDATA[To start off my project at the SGC, my supervisor, Dr. Rachel Harding, and I produced and purified three proteins. These proteins were RAB1A, RAB1B, and MECP2. Both RAB1A and RAB1B have functions in the initiation of autophagy, which is a cellular system that helps clean out old debris and malfunctioning parts. MECP2 is a <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/first-round-of-huntingtin-interaction-partner-protein-production/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>To start off my project at the SGC, my supervisor, <a href="https://labscribbles.com/">Dr. Rachel Harding</a>, and I produced and purified three proteins. These proteins were <a href="https://www.uniprot.org/uniprot/Q9H0U4">RAB1A</a>, <a href="https://www.uniprot.org/uniprot/P62820">RAB1B</a>, and <a href="https://www.uniprot.org/uniprot/P51608">MECP2</a>. Both RAB1A and RAB1B have functions in the initiation of autophagy, which is a cellular system that helps clean out old debris and malfunctioning parts. MECP2 is a protein that can interact with DNA, changing the levels of expression of several genes. The purifications of RAB1A and RAB1B were successful, although MECP2 could not be adequately purified. These three proteins were hits with the <a href="https://zenodo.org/record/3234159#.XP7Go4hKjcs">BioID experiment</a> I mentioned in my<a href="https://thesgc.github.io/static-openlabnotebooks/project-overview-huntingtin-and-its-interaction-partners/"> previous post</a>.</p>
<p>To begin, we took <em>E. coli</em> cells and transformed them with plasmid DNA. The transformation process introduces the bacteria to the genetic instructions on how to make the protein, the cells were left to grow for about 24 hours. By the end, we had 6 L of liquid culture for both RAB1A and RAB1B as well as 4 L of MECP2. From this liquid culture, we collected the cells and proceeded to break them open using sound energy via a process called sonication. After opening the cells, we were able to purify our protein using two methods. The first was affinity chromatography, which allows the binding of a specific tag we put on our protein. To further refine the protein sample, we used size-exclusion chromatography, which is a process that separates proteins based on their size. For a more in-depth explanation of the methods used to purify <a href="https://zenodo.org/record/3252165#.XQ00qIhKjcs">RAB1A</a>, <a href="https://zenodo.org/record/3252174#.XQ00k4hKjcs">RAB1B</a>, and <a href="https://zenodo.org/record/3252176#.XQ0z74hKjcs">MECP2</a>, follow the links to the Zenodo posts about each protein.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/first-round-of-huntingtin-interaction-partner-protein-production/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Project Overview: Huntingtin and its interaction partners</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/project-overview-huntingtin-and-its-interaction-partners/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/project-overview-huntingtin-and-its-interaction-partners/#respond</comments>
		<pubDate>Mon, 10 Jun 2019 14:15:20 +0000</pubDate>
		<dc:creator><![CDATA[Jacob McAuley]]></dc:creator>
				<category><![CDATA[Huntingtin]]></category>
		<category><![CDATA[Huntington’s disease]]></category>
		<category><![CDATA[Jacob McAuley]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2905</guid>
		<description><![CDATA[&#160; Huntington’s disease (HD) is a progressive neurological disorder caused by a mutation in the HTT gene which encodes for the huntingtin protein. The current understanding of the function of HTT is limited, and there are only a few sources that indicate that it likely plays a critical role in several intracellular processes such as <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/project-overview-huntingtin-and-its-interaction-partners/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>&nbsp;</p>
<p>Huntington’s disease (HD) is a progressive neurological disorder caused by a mutation in the HTT gene which encodes for the huntingtin protein. The current understanding of the function of HTT is limited, and there are only a few sources that indicate that it likely plays a critical role in several intracellular processes such as proper neural development. This gap in the scientific literature continues onto the mutant version of the protein. In patients with HD, we know that the mutant protein misfolds and forms aggregates. However, it is unclear if this is the direct cause for the symptoms of the disease or if it is a downstream symptom of some other problem. To obtain a more conclusive understanding of huntingtins role in the cell, the SGC is trying to obtain a structure for huntingtin protein.<br />
So now the question is, how do we image such a molecule? At the SGC we use a technique called X-ray Crystallography. This is a process by which a very pure protein sample is crystallized and expose to X-rays. Using the diffraction of light through the crystal, we can piece together an atom by atom computer model of the sample protein. One complication with the huntingtin protein is that it contains several disordered regions which would likely prevent the formation of a consistent structure in the crystal. To get a full image of this protein, we need to find a way to force a consistent structure onto the disordered regions. Binding a protein to the disordered region causes consistent reordering of this region. The use of a <a href="https://zenodo.org/record/3234159#.XP5doIhKjcs">BioID experiment</a> has identified several proteins that bind huntingtin. In effect, this experiment consists of arming the huntingtin protein with a spray can. When another protein comes within its vicinity, it is tagged by the huntingtin molecule. By looking at tagged proteins, several interaction partners of huntingtin were identified.<br />
During my time at the SGC, I plan to go through this list of tentative interaction partners, producing and purifying them for use in future experiments. Furthermore, this could allow us to obtain crystal structures of huntingtin in complex with an interaction partner. Obtaining a list of all the interaction partners of any given protein can give us a good idea of the function of the protein. This is because we can then attribute the interaction partners function to that of the protein of interest. In the coming weeks, I will continue to post updates on the progress of the purification of the hits from the BioID experiment.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/project-overview-huntingtin-and-its-interaction-partners/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Why the heck does huntingtin bind PAR?</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/why-the-heck-does-huntingtin-bind-par/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/why-the-heck-does-huntingtin-bind-par/#respond</comments>
		<pubDate>Tue, 23 Apr 2019 19:45:27 +0000</pubDate>
		<dc:creator><![CDATA[Tamara Maiuri]]></dc:creator>
				<category><![CDATA[Huntingtin]]></category>
		<category><![CDATA[Huntington’s disease]]></category>
		<category><![CDATA[Ray Truant]]></category>
		<category><![CDATA[Tamara Maiuri]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2742</guid>
		<description><![CDATA[Updates on the huntingtin PAR binding story can be found on the Truant lab website.]]></description>
				<content:encoded><![CDATA[<p>Updates on the huntingtin PAR binding story can be found <a href="https://raytruantlab.ca/2019/04/01/why-the-heck-does-huntingtin-bind-par/">on the Truant lab website</a>.</p>
<div id="attachment_2746" style="width: 310px" class="wp-caption alignnone"><img class="size-medium wp-image-2746" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/2019-04-01-FRAP-crop-300x151.jpg" alt="" width="300" height="151" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/2019-04-01-FRAP-crop-300x151.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/2019-04-01-FRAP-crop-768x388.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/2019-04-01-FRAP-crop-1024x517.jpg 1024w" sizes="(max-width: 300px) 100vw, 300px" /><p class="wp-caption-text">FRAP experiment testing nuclear huntingtin dynamics in response to oxidative stress.</p></div>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/why-the-heck-does-huntingtin-bind-par/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Update on meetings, manuscripts and mighty cool experiments…</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/update-on-meetings-manuscripts-and-mighty-cool-experiments/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/update-on-meetings-manuscripts-and-mighty-cool-experiments/#respond</comments>
		<pubDate>Thu, 21 Mar 2019 16:00:43 +0000</pubDate>
		<dc:creator><![CDATA[Rachel Harding]]></dc:creator>
				<category><![CDATA[Huntingtin]]></category>
		<category><![CDATA[Huntington’s disease]]></category>
		<category><![CDATA[Rachel Harding]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2674</guid>
		<description><![CDATA[Update on meetings, manuscripts and mighty cool experiments&#8230;]]></description>
				<content:encoded><![CDATA[<blockquote class="wp-embedded-content" data-secret="jM2a5gTbw1"><p><a href="http://labscribbles.com/2019/03/21/update-on-meetings-manuscripts-and-mighty-cool-experiments/">Update on meetings, manuscripts and mighty cool experiments&#8230;</a></p></blockquote>
<p><iframe class="wp-embedded-content" sandbox="allow-scripts" security="restricted" src="http://labscribbles.com/2019/03/21/update-on-meetings-manuscripts-and-mighty-cool-experiments/embed/#?secret=jM2a5gTbw1" data-secret="jM2a5gTbw1" width="600" height="338" title="&#8220;Update on meetings, manuscripts and mighty cool experiments&#8230;&#8221; &#8212; LabScribbles" frameborder="0" marginwidth="0" marginheight="0" scrolling="no"></iframe></p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/update-on-meetings-manuscripts-and-mighty-cool-experiments/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Optimisation of HTT-HAP40 purification using heparin affinity chromatography</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/optimisation-of-htt-hap40-purification-using-heparin-affinity-chromatography/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/optimisation-of-htt-hap40-purification-using-heparin-affinity-chromatography/#respond</comments>
		<pubDate>Wed, 30 Jan 2019 16:00:28 +0000</pubDate>
		<dc:creator><![CDATA[Rachel Harding]]></dc:creator>
				<category><![CDATA[Huntingtin]]></category>
		<category><![CDATA[Huntington’s disease]]></category>
		<category><![CDATA[Rachel Harding]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2672</guid>
		<description><![CDATA[Optimisation of HTT-HAP40 purification using heparin affinity chromatography]]></description>
				<content:encoded><![CDATA[<blockquote class="wp-embedded-content" data-secret="PMPBXb2A6l"><p><a href="http://labscribbles.com/2019/01/30/optimisation-of-htt-hap40-purification-using-heparin-affinity-chromatography/">Optimisation of HTT-HAP40 purification using heparin affinity chromatography</a></p></blockquote>
<p><iframe class="wp-embedded-content" sandbox="allow-scripts" security="restricted" src="http://labscribbles.com/2019/01/30/optimisation-of-htt-hap40-purification-using-heparin-affinity-chromatography/embed/#?secret=PMPBXb2A6l" data-secret="PMPBXb2A6l" width="600" height="338" title="&#8220;Optimisation of HTT-HAP40 purification using heparin affinity chromatography&#8221; &#8212; LabScribbles" frameborder="0" marginwidth="0" marginheight="0" scrolling="no"></iframe></p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/optimisation-of-htt-hap40-purification-using-heparin-affinity-chromatography/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Further Investigation of the Reduction of HTT Protein through Chemical Inhibition</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/further-investigation-of-the-reduction-of-htt-protein-through-chemical-inhibition/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/further-investigation-of-the-reduction-of-htt-protein-through-chemical-inhibition/#respond</comments>
		<pubDate>Tue, 15 Jan 2019 18:56:28 +0000</pubDate>
		<dc:creator><![CDATA[Jolene Ho]]></dc:creator>
				<category><![CDATA[Huntingtin]]></category>
		<category><![CDATA[Huntington’s disease]]></category>
		<category><![CDATA[Jolene Ho]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2378</guid>
		<description><![CDATA[The experimental set-up and results covered in this post can be found here: 10.5281/zenodo.2541053. A lot has happened in the past few months and it is with mixed feelings that I am finishing up my postdoc here at SGC! I feel very privileged to have been part of the great science going on at SGC <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/further-investigation-of-the-reduction-of-htt-protein-through-chemical-inhibition/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p style="text-align: justify;">The experimental set-up and results covered in this post can be found here: <a href="http://doi.org/10.5281/zenodo.2541053">10.5281/zenodo.2541053</a>.</p>
<p style="text-align: justify;">A lot has happened in the past few months and it is with mixed feelings that I am finishing up my postdoc here at SGC! I feel very privileged to have been part of the great science going on at SGC and a pioneer of the Open Notebook initiative here, contributing to open science. Nonetheless, the science has to go on! Hopefully, my posts in this Open Notebook have provided new perspectives to people working on similar projects and trying to learn more about Huntington’s Disease and huntingtin (HTT), inspired people to strive on for science despite the negative and/or inconclusive data, or at least given some insight to how life on the bench is like in general.</p>
<p style="text-align: justify;">Previously, I showed that the use of PRMT5 inhibitors from the SGC probe library, namely GSK591 and LLY283, were able to decrease levels of HTT in HeLa cells. SGC2096, the inactive control to GSK591, was not able to decrease HTT protein levels. This, together with the different mechanisms of action of GSK591 and LLY283 in inhibiting PRMT5, demonstrates that it is indeed through the specific inhibition of PRMT5 that HTT protein levels were decreased. So far, I have not come across any drug that is able to regulate the levels of HTT protein. As such, the inhibition of PRMT5 to reduce the expression of HTT at the protein level presents a potential and exciting therapeutic avenue for the treatment of Huntington’s disease.</p>
<p style="text-align: justify;">The findings that PRMT5 inhibition itself can affect the expression of HTT at the protein level suggests that there might be some interaction and possible feedback loop in play between PRMT5 and HTT. We thus further investigated how PRMT5 inhibition may contribute to the decrease in HTT protein levels to elucidate the mechanisms involved in this process. As PRMT5 is the main enzyme known to mediate symmetric demethylation of arginine (SDMA), we thus checked how the levels of SDMA are affected by PRMT5 inhibition and HTT knockdown.</p>
<p style="text-align: justify;"><img class="alignnone wp-image-2379" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/HeLa-G3BP1-PRMT5i-in-HTT-KD-300x225.jpg" alt="" width="771" height="578" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/HeLa-G3BP1-PRMT5i-in-HTT-KD-300x225.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/HeLa-G3BP1-PRMT5i-in-HTT-KD-768x576.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/HeLa-G3BP1-PRMT5i-in-HTT-KD-350x263.jpg 350w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/HeLa-G3BP1-PRMT5i-in-HTT-KD-780x586.jpg 780w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/HeLa-G3BP1-PRMT5i-in-HTT-KD-250x188.jpg 250w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/HeLa-G3BP1-PRMT5i-in-HTT-KD.jpg 960w" sizes="(max-width: 771px) 100vw, 771px" /></p>
<p style="text-align: justify;">Unsurprisingly, PRMT5 inhibition with GSK591 or LLY283 led to the decrease in SDMA. However, we also noticed a slight decrease in SDMA across all 3 HTT knockdown samples! Although this decrease is marginal, its occurrence across all the HTT knockdown samples we tested gives confidence to this observation and corroborates the hypothesis that PRMT5 and HTT are part of a feedback loop rather than that HTT is simply downstream of PRMT5, since the inhibition of PRMT5 leads to a decrease in HTT, and a decrease in HTT by way of HTT knockdown leads to a decrease in SDMA, which is mainly mediated by PRMT5.</p>
<p style="text-align: justify;">The two-way crosstalk demonstrated by PRMT5 activity levels affecting the expression of HTT and the presence (or absence, since we used a knockdown model) of HTT likewise decreasing SDMA levels suggests that the relationship between PRMT5 and HTT is complicated and reciprocal, with one regulating the other possibly through post-translational modifications such as SDMA and other epigenetic mechanisms, and warrants further investigation as to how the activity and levels of PRMT5 and HTT are related. Some important future experiments would be to test if PRMT inhibition is also able to decrease the protein levels of mutant HTT similarly to how it affects wild-type HTT, and if so, this will be an exciting opportunity to follow up on in the search of a potential therapy for Huntington’s disease.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/further-investigation-of-the-reduction-of-htt-protein-through-chemical-inhibition/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Manuscript finished! Design and characterisation of mutant and wild-type huntingtin proteins produced from a toolkit of scalable eukaryotic expression systems</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/manuscript-finished-design-and-characterisation-of-mutant-and-wild-type-huntingtin-proteins-produced-from-a-toolkit-of-scalable-eukaryotic-expression-systems/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/manuscript-finished-design-and-characterisation-of-mutant-and-wild-type-huntingtin-proteins-produced-from-a-toolkit-of-scalable-eukaryotic-expression-systems/#respond</comments>
		<pubDate>Tue, 11 Dec 2018 16:00:26 +0000</pubDate>
		<dc:creator><![CDATA[Rachel Harding]]></dc:creator>
				<category><![CDATA[Huntingtin]]></category>
		<category><![CDATA[Huntington’s disease]]></category>
		<category><![CDATA[Rachel Harding]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2670</guid>
		<description><![CDATA[Manuscript finished! Design and characterisation of mutant and wild-type huntingtin proteins produced from a toolkit of scalable eukaryotic expression systems]]></description>
				<content:encoded><![CDATA[<blockquote class="wp-embedded-content" data-secret="bgco7KH9zr"><p><a href="http://labscribbles.com/2018/12/11/manuscript-finished-design-and-characterisation-of-mutant-and-wild-type-huntingtin-proteins-produced-from-a-toolkit-of-scalable-eukaryotic-expression-systems/">Manuscript finished! Design and characterisation of mutant and wild-type huntingtin proteins produced from a toolkit of scalable eukaryotic expression systems</a></p></blockquote>
<p><iframe class="wp-embedded-content" sandbox="allow-scripts" security="restricted" src="http://labscribbles.com/2018/12/11/manuscript-finished-design-and-characterisation-of-mutant-and-wild-type-huntingtin-proteins-produced-from-a-toolkit-of-scalable-eukaryotic-expression-systems/embed/#?secret=bgco7KH9zr" data-secret="bgco7KH9zr" width="600" height="338" title="&#8220;Manuscript finished! Design and characterisation of mutant and wild-type huntingtin proteins produced from a toolkit of scalable eukaryotic expression systems&#8221; &#8212; LabScribbles" frameborder="0" marginwidth="0" marginheight="0" scrolling="no"></iframe></p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/manuscript-finished-design-and-characterisation-of-mutant-and-wild-type-huntingtin-proteins-produced-from-a-toolkit-of-scalable-eukaryotic-expression-systems/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Reduction of HTT Protein through Chemical Inhibition</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/reduction-of-htt-protein-through-chemical-inhibition/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/reduction-of-htt-protein-through-chemical-inhibition/#comments</comments>
		<pubDate>Thu, 22 Nov 2018 20:34:38 +0000</pubDate>
		<dc:creator><![CDATA[Jolene Ho]]></dc:creator>
				<category><![CDATA[Huntingtin]]></category>
		<category><![CDATA[Huntington’s disease]]></category>
		<category><![CDATA[Jolene Ho]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2169</guid>
		<description><![CDATA[The experimental set-up and results covered in this post can be found here: 10.5281/zenodo.1495129. Apologies for the long delay between this post and my last post, it really has been quite some time! As postdocs, we generally have several projects running simultaneously and we have to split our time between these projects. I am no <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/reduction-of-htt-protein-through-chemical-inhibition/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p style="text-align: justify;">The experimental set-up and results covered in this post can be found here: <a href="http://doi.org/10.5281/zenodo.1495129">10.5281/zenodo.1495129</a>.</p>
<p style="text-align: justify;">Apologies for the long delay between this post and my last post, it really has been quite some time! As postdocs, we generally have several projects running simultaneously and we have to split our time between these projects. I am no exception and lately I have been caught up in trying to wrap up several projects. Besides my work in HTT, I am involved in screening various cell lines against our SGC probe library and identifying probes that have a differential effect in these cell lines. At the SGC, chemical compounds only qualify as probes if they have an in vitro potency of &lt;100 nM, &gt;30-fold selectivity vs other subfamilies, and have significant on-target cell activity at 1 µM. These probes are essential for the early stages of drug development as positive hits in our screens allow us to identify potential drug targets, protein families, and pathways for which we can further investigate in terms of disease mechanism and therapeutic strategies. These probes identified as positive hits in our screens may then be further developed and refined as drugs which may be used in the clinics eventually. You can read more about the SGC’s probes here: <a href="https://www.thesgc.org/chemical-probes">https://www.thesgc.org/chemical-probes</a>.</p>
<p style="text-align: justify;">In one of the other projects I’m working on, we identified PRMT5 as a potential therapeutic target in glioblastoma through the action of GSK591 and LLY283, 2 probes that inhibit PRMT5 through different and independent mechanisms. The inhibition of PRMT5 as a therapeutic strategy is not novel in itself, with several other groups having already published on this.</p>
<p>&nbsp;</p>
<p style="text-align: justify;">PRMT5 as a druggable target for glioblastoma therapy.</p>
<p style="text-align: justify;">Banasavadi-Siddegowda YK, Welker AM, An M, Yang X, Zhou W, Shi G, Imitola J, Li C, Hsu S, Wang J, Phelps M, Zhang J, Beattie CE, Baiocchi R, Kaur B.</p>
<p style="text-align: justify;">Neuro Oncol. 2018 May 18;20(6):753-763. doi: <a href="http://doi.org/10.1093/neuonc/nox206">10.1093/neuonc/nox206</a>.</p>
<p>&nbsp;</p>
<p style="text-align: justify;">Coordinated Splicing of Regulatory Detained Introns within Oncogenic Transcripts Creates an Exploitable Vulnerability in Malignant Glioma.</p>
<p style="text-align: justify;">Braun CJ, Stanciu M, Boutz PL, Patterson JC, Calligaris D, Higuchi F, Neupane R, Fenoglio S, Cahill DP, Wakimoto H, Agar NYR, Yaffe MB, Sharp PA, Hemann MT, Lees JA.</p>
<p style="text-align: justify;">Cancer Cell. 2017 Oct 9;32(4):411-426.e11. doi: <a href="http://doi.org/10.1016/j.ccell.2017.08.018">10.1016/j.ccell.2017.08.018</a>. Epub 2017 Sep 28.</p>
<p>&nbsp;</p>
<p style="text-align: justify;">However, we had some interesting findings through the RNA-sequencing done on cells treated with these PRMT5 inhibitors. Serendipitously, HTT emerged as one of the genes whose expression was consistently downregulated with PRMT5 inhibition! I tested these PRMT5 inhibitors in the HeLa cells I used to generate the 3 stable HTT knockdown lines previously (read more about that here: https://thesgc.github.io/static-openlabnotebooks/generating-htt-knockdown-models/) and looked at how the HTT protein levels were affected in these lines.</p>
<p style="text-align: justify;"><img class="alignnone wp-image-2170" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/20181116-HeLa-G3BP1-PRMT5i-and-HTT-KD-300x225.jpg" alt="" width="841" height="631" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/20181116-HeLa-G3BP1-PRMT5i-and-HTT-KD-300x225.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/20181116-HeLa-G3BP1-PRMT5i-and-HTT-KD-768x576.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/20181116-HeLa-G3BP1-PRMT5i-and-HTT-KD-350x263.jpg 350w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/20181116-HeLa-G3BP1-PRMT5i-and-HTT-KD-780x586.jpg 780w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/20181116-HeLa-G3BP1-PRMT5i-and-HTT-KD-250x188.jpg 250w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/20181116-HeLa-G3BP1-PRMT5i-and-HTT-KD.jpg 960w" sizes="(max-width: 841px) 100vw, 841px" /></p>
<p style="text-align: justify;">HTT protein levels were clearly decreased with PRMT5 inhibition! I ran these samples alongside the stable HTT knockdown lines and although the HTT levels did not decrease as drastically as with the knockdowns, the decrease with the PRMT5 inhibitors is still substantial. I also included SGC2096, the inactive control to GSK591, and HTT protein levels were not affected by it, showing that it is indeed through the activity of PRMT5 that HTT levels were decreased.</p>
<p style="text-align: justify;">The mechanism by which HTT is downregulated by PRMT5 inhibitors is still unclear and it will be very interesting to further investigate the link between HTT and PRMT5. As of now, the only publication I can find linking HTT to PRMT5 is the following:</p>
<p>&nbsp;</p>
<p style="text-align: justify;">PRMT5- mediated symmetric arginine dimethylation is attenuated by mutant huntingtin and is impaired in Huntington&#8217;s disease (HD).</p>
<p style="text-align: justify;">Ratovitski T, Arbez N, Stewart JC, Chighladze E, Ross CA.</p>
<p style="text-align: justify;">Cell Cycle. 2015;14(11):1716-29. doi: <a href="http://doi.org/10.1080/15384101.2015.1033595">10.1080/15384101.2015.1033595</a>.</p>
<p>&nbsp;</p>
<p style="text-align: justify;">This paper showed the colocalisation and interaction of HTT and PRMT5 and shows that this interaction results in the stimulation of PRMT5 activity by HTT in vitro. Mutant HTT has a greater affinity for PRMT5; however, it has a reduced ability to stimulate PRMT5 activity, as demonstrated by the reduction of symmetric dimethylation of arginine (SDMA), which is mainly mediated by PRMT5, of histones H2 and H4. Decreased levels of SDMA on known PRMT5 targets were detected in HD brain samples, suggesting that the reduction in PRMT5 activity may contribute to the pathogenesis of HD. The knockdown of JMJD6, which removes arginine demethylation, was able to rescue the phenotype of increased cell death seen with mutant HTT, further suggesting that the restoration of SDMA to normal levels may be a potential therapeutic strategy.</p>
<p style="text-align: justify;">However, our findings that PRMT5 inhibition itself can affect the expression of HTT at the protein level suggests that there might be a feedback loop in play between PRMT5 and HTT. It will be interesting to further investigate how the levels of SDMA and the other proteins involved are affected with PRMT5 inhibition and elucidate the mechanisms involved in this process. It should also be noted that our results only reflect the expression of wild-type HTT and not mutant (expanded) HTT, and it will be highly interesting and of great clinical importance to find out if PRMT5 inhibition is able to decrease the protein levels of mutant HTT similarly to how it affects wild-type HTT.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/reduction-of-htt-protein-through-chemical-inhibition/feed/</wfw:commentRss>
		<slash:comments>4</slash:comments>
		</item>
	</channel>
</rss>
