<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>Amyotrophic Lateral Sclerosis (ALS) &#8211; openlabnotebooks.org</title>
	<atom:link href="https://thesgc.github.io/static-openlabnotebooks/category/disease-foundation/amyotrophic-lateral-sclerosis-als/feed/" rel="self" type="application/rss+xml" />
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<description>A growing team of groundbreaking scientists around the world are now sharing their lab notebooks online</description>
	<lastBuildDate>Thu, 20 Jun 2019 11:26:10 +0000</lastBuildDate>
	<language>en-GB</language>
	<sy:updatePeriod>hourly</sy:updatePeriod>
	<sy:updateFrequency>1</sy:updateFrequency>
	<generator>https://wordpress.org/?v=4.9.10</generator>

<image>
	<url>https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/cropped-lab-jar02-32x32.png</url>
	<title>Amyotrophic Lateral Sclerosis (ALS) &#8211; openlabnotebooks.org</title>
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<width>32</width>
	<height>32</height>
</image> 
	<item>
		<title>Cell types we work with</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/cell-types-we-work-with/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/cell-types-we-work-with/#respond</comments>
		<pubDate>Sat, 23 Mar 2019 18:06:56 +0000</pubDate>
		<dc:creator><![CDATA[Thomas Durcan]]></dc:creator>
				<category><![CDATA[ALS-Reproducible Antibody Platform (ALS-RAP)]]></category>
		<category><![CDATA[Amyotrophic Lateral Sclerosis (ALS)]]></category>
		<category><![CDATA[iPSC Disease modelling]]></category>
		<category><![CDATA[iPSCs]]></category>
		<category><![CDATA[Parkinson's disease and neurodevelopmental disorders]]></category>
		<category><![CDATA[Thomas M. Durcan]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2607</guid>
		<description><![CDATA[Hello again, Three weeks have now passed since my first blog, and the response has been overwhelming, who knew this kid with a love of dinosaurs could become an internet blogger. Your encouragement to make my t-Rex has motivated me to do just that, so let’s see how it all goes. Also fun fact, the <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/cell-types-we-work-with/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Hello again,</p>
<p>Three weeks have now passed since my first blog, and the response has been overwhelming, who knew this kid with a love of dinosaurs could become an internet blogger. Your encouragement to make my t-Rex has motivated me to do just that, so let’s see how it all goes. Also fun fact, the biggest and oldest T-Rex was found in <a href="https://www.ctvnews.ca/sci-tech/paleontologists-discover-world-s-biggest-and-oldest-tyrannosaurus-rex-in-canada-1.4348206">Canada</a>, so looks like I am in right place for dinosaurs.</p>
<p>But, in the meantime, I do have a day, night and weekend job running a lab and academic drug discovery group, and so its time to discuss science. More specifically, what cells we work with in our work. Our bread and butter are the iPSC, short for induced pluripotent stem cells. Made from the blood of patients and healthy volunteers, when you look at them, they look like regular old cells, tightly packed into colonies, as they love being close to their fellow cells, problems arise when they want to move away and get some distance from their neighbors. Below are some images of what they look like, courtesy of Carol Chen and Narges Abdian, who help manage and oversee our catalog of stem cell lines in my group, along with Zhipeng You. As you can see, the cells are tightly packed, with each cell having the potential to grow into any cell type you want, all it needs is the right ingredients to direct it into the cell type of your choosing.</p>
<p><a href="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/iPSC-cell-image.tif"><img class="alignnone size-medium wp-image-2610" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/iPSC-cell-image.tif" alt="" width="1" height="1" /><img class="size-medium wp-image-2611 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Image-of-IPSCs_forblog-300x249.jpg" alt="" width="300" height="249" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Image-of-IPSCs_forblog-300x249.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Image-of-IPSCs_forblog-768x638.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Image-of-IPSCs_forblog-1024x851.jpg 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Image-of-IPSCs_forblog.jpg 1175w" sizes="(max-width: 300px) 100vw, 300px" /></a></p>
<p>And what might these cells be, well we are interested in diseases of the brain, and in particular the cells lost during the progression of these degenerative disorders. This means our focus is on getting these stem cells to form neurons, of which everyone of us has over 100 billion. We can make many different types and over the coming months, I will be putting online our different SOPs from our group outlining how we make many of these different neurons. One of the main interests for our group is Parkinson’s disease, and the cells lost in this disease are dopaminergic neurons, making it our mission to understand why these neurons are being lost in the disease. Other cell types we make are motor neurons, sensory neurons and cortical neurons, all different types of neurons that are involved in different diseases, playing different roles. For those who wonder what they look like, here are some images below, staining for proteins that are specific for these neurons (Dopaminergic-Top; Cortical- Bottom). Images are courtesy of Carol Chen and Cecilia DeRocha Desouza.</p>
<p><img class="wp-image-2612 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/DA-neurons-300x232.jpg" alt="" width="299" height="231" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/DA-neurons-300x232.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/DA-neurons-768x594.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/DA-neurons-1024x792.jpg 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/DA-neurons.jpg 1323w" sizes="(max-width: 299px) 100vw, 299px" /><img class="wp-image-2613 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/cortical-neurons.jpg" alt="" width="296" height="234" /></p>
<p>In addition to neurons, we also make the support cells for these neurons, and one of these cells is astrocytes. This cell can be the good guy-bad guy and has been demonstrated to support neuronal survival, and on the flip side, to promote the loss of neurons. Courtesy of Vincent Soubannier, here are some images of what these cells look like in the dish.<img class="size-medium wp-image-2615 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/astrocytes-1-300x241.jpg" alt="" width="300" height="241" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/astrocytes-1-300x241.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/astrocytes-1-768x617.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/astrocytes-1.jpg 908w" sizes="(max-width: 300px) 100vw, 300px" /></p>
<p>Now, I know, you are saying Tom, these neurons and astrocytes are beautiful, but they are flat, having been grown in 2D dishes. This isn’t like a brain; the brain is three dimensional and complex. And you know what, you would be right. Growing cells in 2D is artificial, but it lets us grow and image neurons and astrocytes for doing research on human cells that otherwise wouldn’t be accessible. But it wasn’t until the pioneering work by Dr <a href="https://www2.mrc-lmb.cam.ac.uk/groups/lancaster/">Madeline Lancaster</a> and others that the idea for making complex brain structures emerged. From pioneering work in her lab, she showed that neuronal organoids or “minibrains” could be formed in a dish. More complex then anything grown in 2D, these minibrains could be expanded, grown for several months, if not years, and show many characteristics of different regions within the human brain. Over 18 months ago, we turned our attention to minibrains and now after all this time, we can make three different types in our group: midbrain, cerebral and forebrain neuronal organoids or “minibrains”. You can learn more about this work through the enclosed link, with <a href="https://www.mcgill.ca/neuro/category/article-categories/neuroxxceptional-stories/nguyen-vi-mohamed">Dr Nguyen-Vi Mohamed</a> showing how these are made in the lab.</p>
<p>These can grow up to 4mm in size depending on the minibrain grown, and are comprised of neurons, astrocytes and many other cell types. They can be maintained for months, and our oldest minibrains were over 14 months old. Just as a preview of what they look like, below is a section through one of these minibrains at 30 days, courtesy of Dr Nguyen-Vi Mohamed.</p>
<p><img class=" wp-image-2616 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/minibrain-section-300x244.jpg" alt="" width="304" height="247" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/minibrain-section-300x244.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/minibrain-section-768x624.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/minibrain-section-1024x832.jpg 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/minibrain-section.jpg 1281w" sizes="(max-width: 304px) 100vw, 304px" /></p>
<p>With incubators packed with these little guys, we now have over 5000 growing with the capacity to maintain close to 10,000 at any one time. These are probably the coolest things I have ever seen and am amazed at the things that can be done with them. Its such a new model, we are barely scratching the surface at what can be done, but I am looking forward to trying new things and seeing what others do. To end, I wanted to bring peoples attention to some remarkable work from from the Lancaster group published this <a href="https://www.theguardian.com/science/2019/mar/18/scientists-grow-mini-brain-on-the-move-that-can-contract-muscle">week</a> (I am like a fanboy of her work). They could connect a minibrain and muscle, with the minibrain taking control over the muscle and causing it to contract. This is just the tip of the iceberg and now I can’t wait to see what other structures we can connect to a minibrain. It won’t be long before we have little mini bodies on a dish, with minibrains connected to miniature versions of all the major organs in the body. But for now, we wait and see how the field progresses. For those wondering how we make minibrains, our <a href="https://mniopenresearch.org/articles/3-1/v1">methods paper</a> is coming out in next week or so, and once its out, I will use this blog to cover the main points on how we make these organoid structures amongst other hot topics in the stem cell field. Next blog, I will be discuss how to thaw out a vial of iPSCs for all those new users wondering what I should do when I get a tube of these cells. Well don’t panic, I am here to help you. Feel free to connect with me on <a href="https://www.linkedin.com/in/thomas-durcan-18593b23/">LinkedIn</a>, if I don’t know you please at least introduce yourself and who you are. I also now on the twitter (@thomas.durcan), so you can follow me there too, I am a relative novice at this whole social media malarkey, so we shall see how that goes.</p>
<p>&nbsp;</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/cell-types-we-work-with/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Modelling diseases of the brain through stem cells</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/modelling-diseases-of-the-brain-through-stem-cells/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/modelling-diseases-of-the-brain-through-stem-cells/#respond</comments>
		<pubDate>Thu, 28 Feb 2019 00:41:34 +0000</pubDate>
		<dc:creator><![CDATA[Thomas Durcan]]></dc:creator>
				<category><![CDATA[ALS-Reproducible Antibody Platform (ALS-RAP)]]></category>
		<category><![CDATA[Amyotrophic Lateral Sclerosis (ALS)]]></category>
		<category><![CDATA[iPSC Disease modelling]]></category>
		<category><![CDATA[iPSCs]]></category>
		<category><![CDATA[Parkinson's disease and neurodevelopmental disorders]]></category>
		<category><![CDATA[Thomas M. Durcan]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2536</guid>
		<description><![CDATA[Hello SGC members and to the millions of scientific readers across the globe, My name is Thomas Durcan, and I am an assistant professor at the Montreal Neurological Institute (MNI) and McGill University, where I also lead the SGC tissue platform in Montreal. You can call me the stem cell guy, as everything I and my <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/modelling-diseases-of-the-brain-through-stem-cells/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Hello SGC members and to the millions of scientific readers across the globe,</p>
<p>My name is Thomas Durcan, and I am an assistant professor at the <a href="https://www.mcgill.ca/neuro/">Montreal Neurological Institute (MNI) </a>and <a href="https://www.mcgill.ca/">McGill University</a>, where I also lead the <a href="https://ultra-dd.org/tissue-platforms">SGC tissue platform</a> in Montreal. You can call me the stem cell guy, as everything I and my team does is centered on human stem cells, developing disease models for understanding and treating diseases of the brain. So who am I, well let me introduce myself. I hail from Dublin, Ireland where I was born and raised. I wanted to clone dinosaurs from the moment I saw Jurassic park, so I did everything in my power to become a scientist. During my time at <a href="https://www.ucd.ie/">University College Dublin</a>, I looked at the DNA of plants, although no breakthroughs emerged in my quest for dinosaurs. With a degree in hand, I headed across to North America, first to the USA and the <a href="https://www.nd.edu/">University of Notre Dame</a>, and then onto colder climes in Canada, settling in Montreal, Quebec. It is here in Montreal, where I have learned to survive winter, to sample the finest bagels, to develop a passion for wine and where I got the opportunity to understand the cellular and molecular causes of Parkinson&#8217;s disease working in the lab of <a href="https://www.mcgill.ca/neuro/research/researchers/fon">Dr. Edward Fon</a>.</p>
<p>However, all my studies were with regular old Hela or HEK293T cells, cells that are clearly not neuronal. When studying a disease of the brain, it would make sense to actually study a human neuron, but these aren&#8217;t readily accessible. I can&#8217;t go up to someone and ask them for their neurons as what you are born with is kind of what you need. So for years, our field was stuck, we had some advances, but it was always in a Hela-type cell, or animal model, which often lacked many of the features of the human disease itself. Then the breakthrough came, in 2006 with the Yamanaka factors. A set of four transcription factors that when introduced into skin cells from a mouse or human, brought them back to the beginning. It reprogrammed the cell, and made it pluripotent, coining the term induced pluripotent stem cell or iPSC. By reprogramming the skin cell, it removed all epigenetics and created a stem cell, that was capable of generating any tissue or cell type within the human body. All it needed, was the correct set of factors to do so.</p>
<p>SInce this seminal publication, the field has grown exponentially and I started working on stem cells in 2014, leading to the creation of the <a href="https://www.mcgill.ca/ipsquebec-mni/">iPSC platform</a> at the MNI, under the direction of Dr. Edward Fon, and myself as group leader, and in partnership with the iPSC platform at the Universite of Laval, under the leadership of <a href="http://www.jpuymiratlab.crchul.ulaval.ca/">Jack Puymerat</a>.</p>
<p>When we began, we were three people, and fast forward to 2019, our group is now over 30 full time staff and students, and growing every month, with the iPSC platform starting its transition this year into the Early Drug Discovery Unit (Neuro-EDDU- further details to follow). With iPSCs at the core of our mission we are now engaged in many early drug discovery projects with researchers, biotech and large pharma, generating neurons for studying Parkinson&#8217;s disease, Amyotrophic Lateral Sclerosis (ALS) and neurodevelopmental biology. Not only are we generating neurons on 2D dishes, but we have been growing minibrains by the thousands, with these 3D neuronal organoids mimicking the human brain more closely then anything seen with 2D cultures. At the same time, we have been fortunate that CRISPR genome editing has emerged, providing us the tools to edit a gene at the base pair level. This technology is at the core of our involvement with the <a href="https://als-rap.org/">ALS-RAP</a> project, in which we are making a panel of over 30 KO cells against ALS-associated genes, using these cells to validate antibodies. With all these tools now set up within the group, new big data approaches are being explored by members of my team, both for image based and MultiOmics based analysis of neurons and other disease-relevant cells generated from patient iPSCs.</p>
<p>This is just a snap-shot into the type of things we are doing, and we are now a step closer to making dinosaur stem cells, fulfilling a life long dream of mine, while also advancing new therapies into the clinic that we hope will one day treat these devastating disorders. Yet, we can&#8217;t do this alone, which is why for my blog I will be starting to make available all our SOPs as part of the open science mission of the MNI and TOSI. All our tips and tricks will be posted on a regular basis, from how to grow an iPSC all the way to to how to make a minibrain will be added to the blog over the coming months, to show everyone how to work with iPSCs. In parallel, as one of the three leads on the <a href="https://als-rap.org/">ALS-RAP</a> project, I will be posting updates on our antibody validation efforts and progress on this project, keeping people up to date on antibodies we are testing, and pointing to ones you should consider working with.</p>
<p>I look forward to opening the doors to our group, and hopefully over the coming months you will enjoy hearing about the work being done in Montreal, and at the MNI on stem cells and antibody validation. Next time, I will start by introducing our SOPs for working with iPSCs, starting with how to thaw a frozen vial of iPSCs. To talk or to lean more  feel free to reach me through <a href="https://www.linkedin.com/in/thomas-durcan-18593b23/">linkedin</a>.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/modelling-diseases-of-the-brain-through-stem-cells/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Welcome to the SGC-Karolinska side of ALS-RAP: the scientists, the screening, and the antibody selections</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/welcome-to-the-sgc-karolinska-side-of-als-rap-the-scientists-the-screening-and-the-antibody-selections/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/welcome-to-the-sgc-karolinska-side-of-als-rap-the-scientists-the-screening-and-the-antibody-selections/#respond</comments>
		<pubDate>Thu, 31 Jan 2019 23:10:29 +0000</pubDate>
		<dc:creator><![CDATA[Carolyn Marks]]></dc:creator>
				<category><![CDATA[ALS-Reproducible Antibody Platform (ALS-RAP)]]></category>
		<category><![CDATA[Amyotrophic Lateral Sclerosis (ALS)]]></category>
		<category><![CDATA[Carolyn Marks]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2419</guid>
		<description><![CDATA[Hello fellow scientists and open lab notebook readers! I’m Carolyn Marks and I’m working in the ALS-RAP partnership with Susanne Gräslund, who leads the Recombinant Antibodies Group at SGC-Karolinska in Stockholm, Sweden. SGC-Karolinska has had a long-standing collaboration with Helena Persson Lotsholm, Director of the Human Antibody Therapeutics Facility within the Drug Discovery and Development <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/welcome-to-the-sgc-karolinska-side-of-als-rap-the-scientists-the-screening-and-the-antibody-selections/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p style="text-align: justify;">Hello fellow scientists and open lab notebook readers!</p>
<p style="text-align: justify;">I’m Carolyn Marks and I’m working in the ALS-RAP partnership with <a href="https://thesgc.org/profile/karolinska/sgraslund">Susanne Gräslund</a>, who leads the Recombinant Antibodies Group at <a href="https://www.thesgc.org/karolinska">SGC-Karolinska</a> in Stockholm, Sweden. SGC-Karolinska has had a long-standing collaboration with <a href="https://www.scilifelab.se/facilities/human-antibody-therapeutics/">Helena Persson Lotsholm</a>, Director of the Human Antibody Therapeutics Facility within the <a href="https://www.scilifelab.se/platforms/ddd">Drug Discovery and Development Platform at Science for Life Laboratory in Stockholm</a>. I am fortunate to work with and learn from several experts to generate the highest quality antibodies for the ALS-Reproducible Antibody Platform. I joined the <a href="https://www.als-rap.org/team">ALS-RAP team</a> in September of last year and from here on out I’ll be sharing live updates of our work with this community.</p>
<p style="text-align: justify;">Perhaps I should tell you a little bit about me, what we do at SGC-Karolinska, and how I got here. I’m a wet lab kind of gal and a neuroscientist at heart with a lot of years of experience under my belt. The methods that I use here at SGC-Karolinska for the ALS-RAP are broad and range from phage display technology to cloning to high-throughput screening to patient-derived cell-based assays. Actually, many of the methods that I&#8217;m using now to generate and validate recombinant antibodies for the ALS-RAP, I used for many years to study neurodegenerative diseases, including ALS.</p>
<p style="text-align: justify;">I´m originally from Texas where my love of neurophysiology began, and where I was initially trained as an electrophysiologist. From there, I deferred medical school and took an internship at NINDS at the NIH to have a break before med school began. My 1-year internship turned into two, and I quickly understood why everyone at NIH kept telling me “Carolyn, you reek of a PhD!” I soon found the NIH-Karolinska Graduate Partnerships Program and hopped on a plane. That&#8217;s how the girl from Texas ended up in Sweden. I received my PhD in 2012 from the NIH-Karolinska Collaborative Doctoral Program in Neuroscience; and I recently completed my Post-Doc, also at Karolinska, characterizing the ubiquitin proteasome system as therapeutic targets in neurodegenerative diseases.</p>
<p style="text-align: justify;">How did I find my way to the SGC, you might wonder? In my research over the years, I have experienced first-hand the need for reliable high-quality antibodies – and the lack thereof, as well. I hate that we as scientists are limited by the tools that companies provide. These limitations are crippling our research. Sadly, the ones who suffer the most are the patients. This was my motivation for joining SGC-Karolinska and particularly this project. I wanted to make better tools for the research community. Consequently, I believe that better research tools will yield better science and hopefully better therapeutics for the patients who need it. That&#8217;s why I&#8217;m here.</p>
<p style="text-align: justify;">How does the <a href="https://www.als-rap.org/overview">ALS-RAP</a> work? Our team at Karolinska is only 1 of many. We work together primarily with scientists at <a href="https://thesgc.github.io/static-openlabnotebooks/introducing-the-team-the-gileadi-group-in-oxford/">Oxford</a> and in <a href="https://www.mcgill.ca/neuro/research/researchers/thomas-m-durcan-phd">Montreal</a>. The ALS-RAP was created by three major ALS charities from Canada, the UK, and the US. The long-term goal of this partnership is to provide the ALS research community with reproducible, thoroughly validated antibodies for selected ALS-related proteins. This includes testing existing commercial antibodies as well as generating new ones. The work is done in three different labs, as recently outlined by my fellow ALS-RAP partner and open-lab-notebooker, <a href="https://thesgc.github.io/static-openlabnotebooks/validated-antibodies-for-als-research-the-als-rap-project/">Opher Gileadi</a>.</p>
<p style="text-align: justify;">In the Stockholm team, we use phage display technology to create recombinant antibodies. Don´t know what phage display is? The inventors of this technique were actually awarded the <a href="https://www.nobelprize.org/prizes/chemistry/2018/summary/">Nobel Prize in Chemistry in 2018</a>. While this technology has many uses, we use phage display to generate recombinant antibodies. How does it work you might wonder? When we receive antigens of our ALS proteins from our friends in <a href="https://www.ndm.ox.ac.uk/principal-investigators/researcher/opher-gileadi">Oxford</a>, we then use the phage display technique to select out antibodies that binds these antigens. We use extensive libraries of human antibody candidates where high-quality binders can be obtained with the phage display method. We then perform several rounds of screening and selection to ensure that we get the best possible antibodies. We then evaluate their specificity and affinity and finally we validate the antibodies using immunoprecipitation followed by mass spectrometry. It sounds simple, but trust me it’s a thorough process! Last but not least, we will release the sequence of the best antibody for each target on our website and make it available to anyone who wants to use it. Our collaborators in Montreal are creating CRISPR-knockouts of these targets in both standard cell lines and iPSCs for further validation and application specificity.</p>
<p style="text-align: justify;">We have a really exciting <a href="https://als-rap.org/target-list">target list</a>, built by the help of our ALS-RAP advisory board. Check it out and see if your favorite protein or that much needed antibody is on our list. If not, click <a href="https://als-rap.org/your-input">here</a> and request it. So far, we have generated and selected our first set of antibodies for 7 of these targets. Now onto primary and secondary validation.</p>
<p style="text-align: justify;">Stay tuned to find out who the lucky 7 are…</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/welcome-to-the-sgc-karolinska-side-of-als-rap-the-scientists-the-screening-and-the-antibody-selections/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Validated antibodies for ALS research: the ALS-RAP project</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/validated-antibodies-for-als-research-the-als-rap-project/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/validated-antibodies-for-als-research-the-als-rap-project/#respond</comments>
		<pubDate>Mon, 03 Dec 2018 10:06:28 +0000</pubDate>
		<dc:creator><![CDATA[Opher Gileadi]]></dc:creator>
				<category><![CDATA[ALS-Reproducible Antibody Platform (ALS-RAP)]]></category>
		<category><![CDATA[Amyotrophic Lateral Sclerosis (ALS)]]></category>
		<category><![CDATA[Opher Gileadi]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2212</guid>
		<description><![CDATA[We are part of ALS-RAP, the ALS (Amyotrophic Lateral Sclerosis) Reproducible Antibody Platform. We’re addressing one of the big conundrums to biomedical research: the lack, or the poor quality, of many antibodies used for research ranging from cell biology to histopathology. ALS, also called MND (motor neurone disease), is a group of diseases that cause <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/validated-antibodies-for-als-research-the-als-rap-project/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>We are part of <a href="https://www.als-rap.org/">ALS-RAP</a>, the ALS (Amyotrophic Lateral Sclerosis) Reproducible Antibody Platform. We’re addressing one of the big conundrums to biomedical research: the lack, or the poor quality, of many antibodies used for research ranging from cell biology to histopathology.</p>
<p>ALS, also called MND (motor neurone disease), is a group of diseases that cause degeneration of motor neurones. The symptoms vary, but the disease is debilitating, life-threatening and currently incurable. Around 5,000 people in the UK, and 12,000 people in the US, are currently affected. There are charities in several countries devoted to helping patients and supporting research, including the <a href="http://www.alsa.org/">ALS Association</a> (US), <a href="https://www.mndassociation.org/">MNDA association</a> (UK) and the <a href="https://www.als.ca/">ALS Society of Canada</a>; they all have very informative web sites. These three charities have joined forces to fund the ALS-RAP project</p>
<p>Up to 10% of cases are hereditary, and the genetic changes have been mapped for many of these cases. Mutations of C9ORF72 and SOD1, TDP-43 and FUS are most common, but there are dozens of other genes that are mutated less frequently. Studying the ALS-related genes and their roles in healthy nerve function and in disease are keys to understanding the biology and pathology and, eventually, finding cures. To do that, we need high-quality, validated reagents; in many cases, these do not exist (or we have no way of knowing, because we haven’t tested them properly!)</p>
<p>So here’s our plan. We have set up a collaboration with Susanne Gräslund at the Karolinska Institute in Stockholm and with Tom Durcan and Peter McPherson at the Montreal Neurological Institute (McGill University).</p>
<p>First, we consulted our advisory board to build an initial <a href="https://www.als-rap.org/target-list">target list</a>; this is mostly based on genetic data, nicely summarized <a href="https://doi.org/10.2147/DNND.S84956">here</a>.</p>
<p>The Canadian groups are establishing a robust validation process: basically, using cells that express “native” levels of the target protein, and matched CRISPR knockout cells, to probe the sensitivity and specificity of antibodies. They have started to probe commercial antibodies</p>
<p>The Stockholm team has libraries of single-chain antibody variable domains (scFv) in bacteriophage; they will screen the library to isolate and characterize antibodies to each target protein. We also expect contributions of new antibodies from commercial collaborators, who will (of course) put these on their catalogues.</p>
<p>The Oxford team (that’s Tracy Keates and me) are expressing and purifying the target proteins as biotinylated, well-folded proteins or protein domains, to be used as antigens for antibody screening.</p>
<p><strong>You can see the details in our web page: <a href="http://www.als-rap.org">www.als-rap.org</a> , including the target list, target status, plasmids and protocols, validation data (soon) etc. </strong></p>
<p>In the next blog, I’ll tell more about our cloning campaign to generate the antigens.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/validated-antibodies-for-als-research-the-als-rap-project/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Introducing the team: The Gileadi Group in Oxford.</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/introducing-the-team-the-gileadi-group-in-oxford/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/introducing-the-team-the-gileadi-group-in-oxford/#respond</comments>
		<pubDate>Fri, 30 Nov 2018 16:31:10 +0000</pubDate>
		<dc:creator><![CDATA[Opher Gileadi]]></dc:creator>
				<category><![CDATA[Alzheimer’s disease]]></category>
		<category><![CDATA[Amyotrophic Lateral Sclerosis (ALS)]]></category>
		<category><![CDATA[Chordoma]]></category>
		<category><![CDATA[Opher Gileadi]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2198</guid>
		<description><![CDATA[Hi! I’m Opher Gileadi, I head a group at the Structural Genomics Consortium (SGC) in Oxford. For many years, we’ve been looking at proteins involved in human. We’ve looked at a wide variety of proteins: from DNA repair enzymes and transcription factors to enzymes involved in signalling. In most cases, we focus on early work: <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/introducing-the-team-the-gileadi-group-in-oxford/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Hi! I’m Opher Gileadi, I head a group at the Structural Genomics Consortium (SGC) in Oxford. For many years, we’ve been looking at proteins involved in human. We’ve looked at a wide variety of proteins: from DNA repair enzymes and transcription factors to enzymes involved in signalling. In most cases, we focus on early work: structure determination, activity assays, antibody generation, and finding initial small molecule “hits”. Having the right reagents are the key to enabling new drug targets; we expect that our data will allow – or encourage – further work by medicinal chemists and biologists to seriously test therapeutic hypotheses (we call our data and reagents sets <a href="https://www.thesgc.org/tep">TEPs</a> – Target Enabling Packages).</p>
<p>My team and I will be blogging and sharing our experiments on three areas of interest:</p>
<p><strong>Alzheimer’s disease</strong>: We are funded by the NIH through the <a href="https://www.nia.nih.gov/research/amp-ad">AMP-AD</a> project (Accelerating Medicines Partnership &#8211; Alzheimer&#8217;s Disease). The project has integrated data from brain autopsies and model systems to identify new possible protein targets for treatment or prevention of Alzheimer’s (annotated in the <a href="https://agora.ampadportal.org/genes/(genes-router:genes-list)">Agora</a> web site, run by <a href="http://sagebionetworks.org/research-projects/agora/">SAGE Bionetworks</a>. Our group is generating TEPs for some of these targets – we’ll share our progress in future posts.</p>
<p><strong>ALS (Amyotrophic Lateral Sclerosis)</strong>: We are part of <a href="https://www.als-rap.org/">ALS-RAP</a>, the ALS reproducible antibody platform. We’re addressing one of the big conundrums to biomedical research: the lack, or the poor quality, of many antibodies used for research ranging from cell biology to histopathology.</p>
<p>Together with colleagues in Stockholm and Montreal we aim to validate a set of antibodies to 30 proteins involved in the disease. We will test commercial antibodies as well as generate new ones through phage display technologies.</p>
<p><strong>Chordoma</strong>: <a href="https://www.chordomafoundation.org/understanding-chordoma/">Chordoma</a> is a rare type of cancer that occurs in the bones of the skull base and spine. It is resistant to all forms of chemotherapy; patients are treated by surgery and radiation, but recurrence is common and, eventually, untreatable. A unifying molecular characteristic of chordoma is overexpression of a variant of the transcription factor T-Brachyury (TBXT), which is essential for survival of the cancer cells. Joseph Newman in my group has solved structures of the human Brachyury protein (PDB codes <a href="http://www.rcsb.org/structure/6F58">6F58</a>, <a href="http://www.rcsb.org/structure/6F59">6F59</a>); we are working together with groups in the University of North Carolina and in Baylor College (Texas) to explore TBXT as a drug target.</p>
<p>That’s all for today – we’ll follow-up with stories in these three areas.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/introducing-the-team-the-gileadi-group-in-oxford/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
	</channel>
</rss>
