<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>David Dilworth &#8211; openlabnotebooks.org</title>
	<atom:link href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/david-dilworth/feed/" rel="self" type="application/rss+xml" />
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<description>A growing team of groundbreaking scientists around the world are now sharing their lab notebooks online</description>
	<lastBuildDate>Mon, 01 Jul 2019 15:02:43 +0000</lastBuildDate>
	<language>en-GB</language>
	<sy:updatePeriod>
	hourly	</sy:updatePeriod>
	<sy:updateFrequency>
	1	</sy:updateFrequency>
	<generator>https://wordpress.org/?v=5.3.2</generator>

<image>
	<url>https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/cropped-lab-jar02-32x32.png</url>
	<title>David Dilworth &#8211; openlabnotebooks.org</title>
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<width>32</width>
	<height>32</height>
</image> 
	<item>
		<title>Analysis of NSD3 Isoform Expression from TCGA-LUSC Data</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/analysis-of-nsd3-isoform-expression-from-tcga-lusc-data/</link>
				<comments>https://thesgc.github.io/static-openlabnotebooks/analysis-of-nsd3-isoform-expression-from-tcga-lusc-data/#respond</comments>
				<pubDate>Wed, 15 Aug 2018 14:16:29 +0000</pubDate>
		<dc:creator><![CDATA[David Dilworth]]></dc:creator>
				<category><![CDATA[David Dilworth]]></category>
		<category><![CDATA[NSD3]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=1707</guid>
				<description><![CDATA[There are two major NSD3 isoforms expressed, long (aa 1-1437) and short (aa 1-645, differing in sequence from 620-645). The short isoform lacks a methyltransferase domain and, perhaps surprisingly, is the form attributed to NSD3&#8217;s role in acute myeloid leukemia (AML) [1]. I&#8217;ve previously used the TCGA-LUSC data-set [2] to test for differential expression based on the <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/analysis-of-nsd3-isoform-expression-from-tcga-lusc-data/">Read More ...</a>]]></description>
								<content:encoded><![CDATA[<p>There are two major NSD3 isoforms expressed, long (aa 1-1437) and short (aa 1-645, differing in sequence from 620-645). The short isoform lacks a methyltransferase domain and, perhaps surprisingly, is the form attributed to NSD3&#8217;s role in acute myeloid leukemia (AML) [1]. I&#8217;ve <a href="https://thesgc.github.io/static-openlabnotebooks/gene-expression-profiling-of-lung-squamous-cancer-stratified-by-nsd3-expression/">previously</a> used the TCGA-LUSC data-set [2] to test for differential expression based on the abundance of NSD3 transcripts at the gene-level. Next, I was interested in using the isoform-level expression data from <a href="http://firebrowse.org/">Firebrowse</a> to look at how different isoforms may be regulated. Is expression correlated or are there more complex regulatory mechanisms that control splicing decisions?</p>
<div id="attachment_1717" style="width: 790px" class="wp-caption aligncenter"><img aria-describedby="caption-attachment-1717" class="wp-image-1717 size-large" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/08/exp28-1-1024x705.png" alt="" width="780" height="537" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/08/exp28-1-1024x705.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/08/exp28-1-300x207.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/08/exp28-1-768x529.png 768w" sizes="(max-width: 780px) 100vw, 780px" /><p id="caption-attachment-1717" class="wp-caption-text">Figure 1. Isoform Expression Analysis (A) Schematic representation of NSD3 isoforms. Introns lengths have been shortened in several places for visualization. (B) Normalized expression levels of NSD3 isoforms (RSEM). (C) Relative expression of short vs long isoforms.</p></div>
<p>Firstly, I found that the short isoform is expressed roughly 10-fold over long isoforms, making it the dominant isoform in the LUSC data (Figure 1 &#8211; B).  Next, when comparing transcript levels of long vs short, I observed a significant moderately positive correlation, suggesting that the isoforms are likely co-regulated. I hope the analysis will be helpful in framing the shared and separate functions of the two isoforms uncovered in future experiments.</p>
<p>As always, the complete analysis &amp; code can be found on <a href="https://zenodo.org/record/1345216#.W3MoOOhKjb0">zenodo</a>.</p>
<p><strong>Acknowledgement:</strong> The results shown here are in whole or part based upon data generated by the TCGA Research Network: http://cancergenome.nih.gov/.</p>
<p><strong>References</strong></p>
<p>1. Shen C, Ipsaro JJ, Shi J, et al. NSD3-short is an adaptor protein that couples BRD4 to the CHD8 chromatin remodeler. Molecular cell. 2015;60(6):847-859. doi:10.1016/j.molcel.2015.10.033.<br />
2. Weinstein JN, Collisson EA, Mills GB, et al. The Cancer Genome Atlas Pan-Cancer Analysis Project. Nature genetics. 2013;45(10):1113-1120. doi:10.1038/ng.2764.</p>
]]></content:encoded>
							<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/analysis-of-nsd3-isoform-expression-from-tcga-lusc-data/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
							</item>
		<item>
		<title>Purification of Recombinant Cas9</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/purification-of-recombinant-cas9/</link>
				<comments>https://thesgc.github.io/static-openlabnotebooks/purification-of-recombinant-cas9/#comments</comments>
				<pubDate>Wed, 01 Aug 2018 22:39:20 +0000</pubDate>
		<dc:creator><![CDATA[David Dilworth]]></dc:creator>
				<category><![CDATA[David Dilworth]]></category>
		<category><![CDATA[NSD3]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=1628</guid>
				<description><![CDATA[These days CRISPR gene editing needs little introduction. From humble origins as a bacterial defence system evolved to thwart viral invaders, CRISPR-based technologies have recently made headlines for their promise to revolutionize medicine. While there is a lot of excitement surrounding the use of gene editing in the clinic, it is also an incredibly valuable tool for studying the <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/purification-of-recombinant-cas9/">Read More ...</a>]]></description>
								<content:encoded><![CDATA[<p>These days CRISPR gene editing needs little introduction. From humble origins as a bacterial defence system evolved to thwart viral invaders, CRISPR-based technologies have recently made headlines for their promise to revolutionize medicine. While there is a lot of excitement surrounding the use of gene editing in the clinic, it is also an incredibly valuable tool for studying the function of proteins in cells. For my project, I will be using CRISPR to make genetic changes to the NSD3 gene and study the outcome, beginning first by using the technology to create NSD3 knock out cell lines.</p>
<p>There is an ever-expanding catalogue of CRISPR reagents available to scientists. I&#8217;ve decided to use a method that involves the in vitro (in a tube) assembly of ribonucleoproteins (in this case Cas9 + annealed tracrRNA:crRNA &#8211; RNPs), which can then be introduced into a cell via lipofection or electroporation. Using RNPs as opposed to viral delivery or plasmid-based transfections is advantageous as the expression of Cas9 is transient limiting off-target effects and it is compatible with many different cell types.</p>
<p>Using this approach, the first step would be to either buy or purify recombinant Cas9. Since I work at the SGC, where expertise in protein production is abundant, I opted to purify recombinant Cas9 using Jennifer Doudna&#8217;s expression plasmid from AddGene (<a href="https://www.addgene.org/69090/"># 69090</a>). With help from Rachel Harding and Mani Ravichandran, I was able to make highly pure, low-endotoxin, active Cas9 protein for my future editing experiments (<a href="https://zenodo.org/record/1325937#.W2I2FtJKjb0">see extended details of the protein prep &amp; QC on </a>zenodo). Next steps &#8211; getting the RNPs into cells, testing for INDELs at the cut site, and screening clones for NSD3 disruption.</p>
<div id="attachment_1629" style="width: 310px" class="wp-caption aligncenter"><img aria-describedby="caption-attachment-1629" class="wp-image-1629 size-medium" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/08/g948-300x237.png" alt="" width="300" height="237" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/08/g948-300x237.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/08/g948-768x607.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/08/g948-1024x809.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/08/g948.png 1355w" sizes="(max-width: 300px) 100vw, 300px" /><p id="caption-attachment-1629" class="wp-caption-text">Figure 1. Purified Cas9 (A). 1 µg of purified Cas9 run on an SDS-PAGE gel and stained with Coomassie Blue (B) In vitro activity assay showing specific cleavage of NSD3-short PCR-amplified DNA when the Cas9-RNP is assembled with NSD3-targeting crRNA, but not the non-targeting control RNA.</p></div>
<p>&nbsp;</p>
<p><strong>Acknowledgements: </strong>A big thank you to Rachel Harding and Mani Ravichandran at the SGC Toronto for help optimizing the protocol as well as running instrumentation for size exclusion and cation exchange. I would also like to thank Dr. Jennifer Doudna for sharing her Cas9 expression construct on addgene  (Addgene plasmid <a href="https://www.addgene.org/69090/"># 69090</a>) [1].</p>
<p><strong>References:</strong></p>
<p>1. Enhanced homology-directed human genome engineering by controlled timing of CRISPR/Cas9 delivery. Lin S, Staahl BT, Alla RK, Doudna JA. Elife. 2014 Dec 15;3:e04766. doi: 10.7554/eLife.04766. 10.7554/eLife.04766 PubMed 25497837</p>
]]></content:encoded>
							<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/purification-of-recombinant-cas9/feed/</wfw:commentRss>
		<slash:comments>1</slash:comments>
							</item>
		<item>
		<title>Gene Expression Profiling of Lung Squamous Cancer Stratified by NSD3 Expression</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/gene-expression-profiling-of-lung-squamous-cancer-stratified-by-nsd3-expression/</link>
				<comments>https://thesgc.github.io/static-openlabnotebooks/gene-expression-profiling-of-lung-squamous-cancer-stratified-by-nsd3-expression/#comments</comments>
				<pubDate>Fri, 20 Jul 2018 16:44:10 +0000</pubDate>
		<dc:creator><![CDATA[David Dilworth]]></dc:creator>
				<category><![CDATA[David Dilworth]]></category>
		<category><![CDATA[NSD3]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=1526</guid>
				<description><![CDATA[NSD3 is located on a region of chromosome 8 (8p11-12) that is frequently amplified in several forms of cancer. It has been suggested that NSD3 (WHSC1L1) may be a driver of tumorigenesis in this context (Mahmood et al. 2013). However, it is still unclear how amplification of NSD3 contributes to the disease and whether or not it <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/gene-expression-profiling-of-lung-squamous-cancer-stratified-by-nsd3-expression/">Read More ...</a>]]></description>
								<content:encoded><![CDATA[<p>NSD3 is located on a region of chromosome 8 (8p11-12) that is frequently amplified in several forms of cancer. It has been suggested that NSD3 (WHSC1L1) may be a driver of tumorigenesis in this context (Mahmood et al. 2013). However, it is still unclear how amplification of NSD3 contributes to the disease and whether or not it is a good candidate for targeted therapy.</p>
<p>Given the phenotypes I have observed in lung cancer models, I was interested in using publically available lung cancer patient data from the Cancer Genome Atlas (TCGA) to try exploring this question.</p>
<p>In an earlier analysis (full details + code <a href="https://zenodo.org/record/1219208#.W1IP_tJKjb0">here</a>), I found that amplification of NSD3 was more commonly associated with squamous cell cancer than adenocarcinoma (Figure 1a). Next, looking at gene expression data, I found that amplification of NSD3 leads to increased expression levels, supporting the possibility of NSD3-driven downstream events in the context of 8p11-12 amplifications. I also looked at patient survival as well as mutational profiles of these tumours, finding no significant difference compared to patients that do not have an 8p11-12 amplification (Figure 1d-e).</p>
<div id="attachment_1561" style="width: 766px" class="wp-caption aligncenter"><img aria-describedby="caption-attachment-1561" class="wp-image-1561 size-full" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/07/figures_blog-3.png" alt="" width="756" height="870" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/07/figures_blog-3.png 756w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/07/figures_blog-3-261x300.png 261w" sizes="(max-width: 756px) 100vw, 756px" /><p id="caption-attachment-1561" class="wp-caption-text">Figure 1. Analysis of NSD3 Amplification in Lung Adenocarcinoma (LUAD) and Lung Squamous Cell Carcinoma (LUSC) TCGA Datasets (a) Expression level (RSEM) of NSD3 in normal versus tumour samples (b) Histogram of NSD3 copy-number status. A value greater than 2 indicates high-level confidence that the gene is amplified. (c) NSD3 expression levels (RSEM) in amplified versus non-amplified samples. (d) Survival curve of patients with an NSD3 amplification compared to non-amplified. (e) Mutational profiles of TCGA-LUSC patients stratified by NSD3 amplification status.</p></div>
<p>Next, I was interested in how NSD3 amplification may influence gene expression patterns and whether this may reveal a potential susceptibility of 8p11-12 amplified cancers ( See full detail &amp; code on <a href="https://zenodo.org/record/1318236#.W1IQbtJKjb0">Zenodo</a>). After performing differential expression analysis of amplified vs non-amplified TCGA-LUSC RNA-Seq data, I found that many of the upregulated genes were associated with the 8p11-12 amplicon (Figure 2). There are a number of genes encoded in this region, including chromatin modifiers and assumed oncogenes. Therefore, it is difficult to untangle NSD3-dependent changes in gene expression versus those that may be driven by other gene products originating from this region.</p>
<div id="attachment_1565" style="width: 790px" class="wp-caption aligncenter"><img aria-describedby="caption-attachment-1565" class="wp-image-1565 size-large" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/07/rect26-1024x384.png" alt="" width="780" height="293" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/07/rect26-1024x384.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/07/rect26-300x112.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/07/rect26-768x288.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/07/rect26.png 1695w" sizes="(max-width: 780px) 100vw, 780px" /><p id="caption-attachment-1565" class="wp-caption-text">Figure 2. (a) Gene set enrichment analysis of amplified vs non-amplified LUSC gene expression data. We can see an enrichment in genes from the 8p11-12 amplicon (b) GSEA plot of the 8p11-12 gene set.</p></div>
<p>To focus in on NSD3-associated gene expression profiles, I next took the samples with the highest NSD3 expression (top 20%) and performed differential expression analysis against those with the lowest NSD3 expression (bottom 20%) (Figure 3). Here, I identified significantly more differentially expressed genes than when comparing amplified to non-amplified samples (Figure 4b). I&#8217;ve included an interactive plot and table with the results below. Feel free to explore the data, and please let me know if you see something interesting/informative.</p>
<p>I haven&#8217;t had time yet to explore this analysis in significant depth. What I have found is that genes that were upregulated in the high NSD3 expressing samples showed enrichment in hedgehog signalling genes and targets of E2F transcription factor. Surprisingly, downregulated genes were actually enriched in EMT factors. This observation is in disagreement with my hypothesis that NSD3 promotes EMT. At this point is difficult to interpret the significance of these transcriptional changes and how they may relate to what I&#8217;ve seen in my experiments, lots more research and experiments are needed before drawing any solid conclusions from this analysis. It will be helpful to keep this data in mind and revisit it as I move the project forward.</p>
<div id="attachment_1566" style="width: 790px" class="wp-caption alignnone"><img aria-describedby="caption-attachment-1566" class="wp-image-1566 size-large" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/07/rect2665-1024x672.png" alt="" width="780" height="512" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/07/rect2665-1024x672.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/07/rect2665-300x197.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/07/rect2665-768x504.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/07/rect2665.png 1994w" sizes="(max-width: 780px) 100vw, 780px" /><p id="caption-attachment-1566" class="wp-caption-text">Figure 3. (a) Stratification of Samples based on NSD3 expression. Plot shows h High (top 20%) versus Low (bottom 20%) NSD3 TCGA-LUSC Samples. (b) Volcano plot of differentially expressed genes comparing high-low NSD3 samples. (c) Hallmark MSigDB gene sets enriched in upregulated genes (c) and downregulated (d).</p></div>
<p>&nbsp;</p>
<p><strong>Acknowledgement.</strong> This analysis relies on several amazing &amp; <strong>open</strong> R packages, which are cited below in no particular order. Additionally, the results shown here are based upon data generated by the TCGA Research Network.</p>
<p>&nbsp;</p>
<p><iframe src="https://s3.amazonaws.com/tcgaluscnsd3/MD-Plot.html" name="frame1" width="1000px" height="1000px" frameborder="yes" scrolling="auto" align="center"><span data-mce-type="bookmark" style="display: inline-block; width: 0px; overflow: hidden; line-height: 0;" class="mce_SELRES_start"><span data-mce-type="bookmark" style="display: inline-block; width: 0px; overflow: hidden; line-height: 0;" class="mce_SELRES_start">﻿</span>﻿</span><br />
</iframe></p>
<p><strong>References:</strong></p>
<ol>
<li>Mahmood SF et al. (2013). PPAPDC1B and WHSC1L1 are common drivers of the 8p11-12 amplicon, not only in breast tumors<span style="font-size: 1.5rem;"> but also in pancreatic adenocarcinomas and lung </span>tumors<span style="font-size: 1.5rem;">. <span role="menubar"><a title="The American journal of pathology." role="menuitem" href="https://www.ncbi.nlm.nih.gov/pubmed#" aria-expanded="false" aria-haspopup="true">Am J Pathol.</a></span> 2013 Nov;183(5):1634-1644. doi: 10.1016/j.ajpath.2013.07.028</span></li>
<li>Weinstein JN, Collisson EA, Mills GB, et al. The Cancer Genome Atlas Pan-Cancer Analysis Project. Nature genetics. 2013;45(10):1113-1120. doi<span style="font-size: 1.5rem;">:10.1038/ng.2764.</span></li>
<li>Martin Morgan, Valerie Obenchain, Jim Hester and Hervé Pagès (2017). SummarizedExperiment: SummarizedExperiment container. R package version 1.8.1.</li>
<li>BioMart and Bioconductor: a powerful link between biological databases and microarray data analysis. Steffen Durinck, Yves Moreau, Arek Kasprzyk, Sean Davis, Bart De Moor, Alvis Brazma and Wolfgang Huber, Bioinformatics 21, 3439-3440 (2005).</li>
<li>Antonio Colaprico, Tiago Chedraoui Silva, Catharina Olsen, Luciano Garofano, Claudia Cava, Davide Garolini, Thais Sabedot, Tathiane Malta, Stefano M. Pagnotta, Isabella Castiglioni,Michele Ceccarelli, Gianluca Bontempi Houtan Noushmehr. TCGAbiolinks: An R/Bioconductor package for integrative analysis of TCGA data Nucleic Acids Research (05 May 2016) 44 (8): e71. (doi:10.1093/nar/gkv1507) Mario Deng (2016). FirebrowseR<span style="font-size: 1.5rem;">: An &#8216;API&#8217; Client for Broads &#8216;Firehose&#8217; Pipeline. R package version </span>1.1.35.</li>
<li>Hadley Wickham (2017). tidyverse<span style="font-size: 1.5rem;">: Easily Install and Load the &#8216;Tidyverse&#8217;. R package version 1.2.1. </span>https://CRAN.R-project.org/package=tidyverse</li>
<li>Guangchuang Yu, Li-Gen Wang, Yanyan Han and Qing-Yu He. clusterProfiler: an R package for comparing biological themes among gene clusters. OMICS: A Journal of Integrative Biology 2012, 16(5):284-287</li>
<li>Ritchie, M.E., Phipson, B., Wu, D., Hu, Y., Law, C.W., Shi, W., and Smyth, G.K. (2015). limma powers differential expression analyses for RNA-sequencing and microarray studies. Nucleic Acids Research 43(7), e47</li>
<li> Shian Su, Charity W. Law, Casey Ah-Cann, Marie-Liesse Asselin-Labat, Marnie E. Blewitt, Matthew E. Ritchie; Glimma: interactive graphics for gene expression analysis. Bioinformatics 2017 btx094. doi<span style="font-size: 1.5rem;">: </span>10.1093/bioinformatics/btx094</li>
<li>Stefan Milton Bache and Hadley Wickham (2014). magrittr<span style="font-size: 1.5rem;">: A Forward-Pipe Operator for R. R package </span>version 1.5. https://CRAN.R-project.org/package=magrittr</li>
<li>Robinson MD, McCarthy DJ and Smyth GK (2010). edgeR: a Bioconductor package for differential expression analysis of digital gene expression data. Bioinformatics 26, 139-140</li>
<li>Mayakonda, A. and H.P. Koeffler, Maftools: Efficient analysis, visualization and summarization of MAF files from large-scale<br />
cohort based cancer studies. bioRxiv, 2016. doi: http://dx.doi.org/10.1101/052662</li>
</ol>
<p>&nbsp;</p>
]]></content:encoded>
							<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/gene-expression-profiling-of-lung-squamous-cancer-stratified-by-nsd3-expression/feed/</wfw:commentRss>
		<slash:comments>3</slash:comments>
							</item>
		<item>
		<title>NSD3&#8217;s Role in Promoting Epithelial to Mesenchymal Transition is Isoform Dependent</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/nsd3s-role-in-promoting-epithelial-to-mesenchymal-transition-is-isoform-dependent/</link>
				<comments>https://thesgc.github.io/static-openlabnotebooks/nsd3s-role-in-promoting-epithelial-to-mesenchymal-transition-is-isoform-dependent/#respond</comments>
				<pubDate>Tue, 05 Jun 2018 22:55:41 +0000</pubDate>
		<dc:creator><![CDATA[David Dilworth]]></dc:creator>
				<category><![CDATA[David Dilworth]]></category>
		<category><![CDATA[NSD3]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=1305</guid>
				<description><![CDATA[To understand what proteins do, scientists often test the consequence their absence has on a cell. A common method uses a cells regulatory machinery to disrupt gene expression with complementary RNA molecules, known as RNA interference. Here I used small interfering RNAs (siRNAs) to deplete NSD3 and test the effect this has on epithelial cell identity in <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/nsd3s-role-in-promoting-epithelial-to-mesenchymal-transition-is-isoform-dependent/">Read More ...</a>]]></description>
								<content:encoded><![CDATA[<p>To understand what proteins do, scientists often test the consequence their absence has on a cell. A common method uses a cells regulatory machinery to disrupt gene expression with complementary RNA molecules, known as <a href="https://en.wikipedia.org/wiki/RNA_interference">RNA interference</a>. Here I used small interfering RNAs (siRNAs) to deplete NSD3 and test the effect this has on epithelial cell identity in a lung cancer model. Importantly, siRNA molecules can be designed to target specific isoforms of NSD3, allowing us to test their individual functions (Figure 1 &#8211; Isoforms of NSD3).</p>
<div id="attachment_1311" style="width: 593px" class="wp-caption aligncenter"><img aria-describedby="caption-attachment-1311" class="wp-image-1311" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/06/FireShot-Capture-30-laml_isoform_figure-768x547.png-768×5_-https___opennotebook.thesgc.org_wp-1024x311.png" alt="" width="583" height="177" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/06/FireShot-Capture-30-laml_isoform_figure-768x547.png-768×5_-https___opennotebook.thesgc.org_wp-1024x311.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/06/FireShot-Capture-30-laml_isoform_figure-768x547.png-768×5_-https___opennotebook.thesgc.org_wp-300x91.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/06/FireShot-Capture-30-laml_isoform_figure-768x547.png-768×5_-https___opennotebook.thesgc.org_wp-768x233.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/06/FireShot-Capture-30-laml_isoform_figure-768x547.png-768×5_-https___opennotebook.thesgc.org_wp.png 1181w" sizes="(max-width: 583px) 100vw, 583px" /><p id="caption-attachment-1311" class="wp-caption-text">Figure 1. Gene-level depiction of known NSD3 isoforms.</p></div>
<p>I first tested several different concentrations of siRNA on H1299 cells and determined the degree of NSD3 loss by western blotting (Figure 2). Surprisingly, I observed depletion of NSD3 with only one of two antibodies tested (Figure 2, <a href="https://doi.org/10.5281/zenodo.1219201">full experimental details</a>). This suggests abcam&#8217;s NSD3 antibody (<a href="http://www.abcam.com/nsd3-antibody-epr13813-n-terminal-ab180500.html">ab180500</a>), which was advertised as knock-out validated, likely recognizes a completely different protein. Additionally, ab180500 did not recognize NSD3-Short, even though the epitope is present in the short isoform. This experiment shows how important it is to validate antibodies and that this reagent should not be used to study NSD3.</p>
<div class="mceTemp"></div>
<div id="attachment_1308" style="width: 579px" class="wp-caption aligncenter"><a href="https://zenodo.org/record/1219201#.WxUrD0gvzb0"><img aria-describedby="caption-attachment-1308" class="wp-image-1308" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/06/Fig1_WB-1024x477.png" alt="" width="569" height="265" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/06/Fig1_WB-1024x477.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/06/Fig1_WB-300x140.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/06/Fig1_WB-768x358.png 768w" sizes="(max-width: 569px) 100vw, 569px" /></a><p id="caption-attachment-1308" class="wp-caption-text">Figure 2. siRNA-mediated Knockdown of NSD3 in H1299 Lung Cancer Cells. Western blots show transfection with different nanomolar amounts of an NSD3-targeting siRNA probing with either A. anti-NSD3 (proteintech: 11345-1-AP) or B. anti-NSD3 (abcam: ab180500), with actin as a loading control.</p></div>
<p>Next, I looked at E-cadherin expression as a marker of an epithelial cell state (Figure 3, <a href="https://zenodo.org/record/1263398">full experimental details</a>).  With siRNA that target both long and short isoforms, I observed an increase in E-cadherin when NSD3 was depleted. <a href="https://thesgc.github.io/static-openlabnotebooks/following-up-on-nsd3s-involvement-in-epithelial-to-mesenchymal-reprogramming/">Previously</a> I showed overexpression of NSD3 decreased E-cadherin, together these results suggest the amount of NSD3 present in a cell influences the transcriptional programs that control cell identity. To test if this function is isoform specific, I used siRNA molecules that target either the long or short isoforms of NSD3 (Figure 4, full experimental details &#8211; <a href="https://zenodo.org/record/1258215">A</a>, <a href="https://zenodo.org/record/1258223">B</a>). I found that knockdown of the short isoform and not the long increased E-cadherin expression levels and reduced cell migration. As the catalytic SET domain is absent in NSD3-short, NSD3&#8217;s H3K36me methyltransferase activity does not appear to be required for this function. Importantly, this supports the idea that NSD3-Short is an important mediator of cell identity and, in the context of cancer promotes, metastatic properties.</p>
<div id="attachment_1314" style="width: 672px" class="wp-caption aligncenter"><img aria-describedby="caption-attachment-1314" class="wp-image-1314" style="font-size: 1.5rem;" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/06/g916-1024x381.png" alt="" width="662" height="247" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/06/g916-1024x381.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/06/g916-300x112.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/06/g916-768x286.png 768w" sizes="(max-width: 662px) 100vw, 662px" /><p id="caption-attachment-1314" class="wp-caption-text">Figure 3. NSD3 Represses E-cadherin Expression in H1299 Lung Cancer Cells. A. Western blot of H1299 cells treated with NSD3-targeting siRNA in triplicate. B. Quantification of western blot signal relative to either actin or histone h3 loading controls.</p></div>
<div id="attachment_1309" style="width: 684px" class="wp-caption aligncenter"><img aria-describedby="caption-attachment-1309" class="wp-image-1309" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/06/g1031-1024x322.png" alt="" width="674" height="212" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/06/g1031-1024x322.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/06/g1031-300x94.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/06/g1031-768x242.png 768w" sizes="(max-width: 674px) 100vw, 674px" /><p id="caption-attachment-1309" class="wp-caption-text">Figure 4. The short Isoform of NSD3 mediates its function in EMT. A. Western blot of E-cadherin protein levels in response to isoform-specific siRNAs in A549 lung cancer cells. B. Wound healing assay in A549 cells at 24 and 48 hours, indicates the short isoform of NSD3 is responsible for promoting migration in lung epithelial cells.</p></div>
]]></content:encoded>
							<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/nsd3s-role-in-promoting-epithelial-to-mesenchymal-transition-is-isoform-dependent/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
							</item>
		<item>
		<title>Following Up on NSD3&#8217;s Involvement in Epithelial to Mesenchymal Reprogramming</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/following-up-on-nsd3s-involvement-in-epithelial-to-mesenchymal-reprogramming/</link>
				<comments>https://thesgc.github.io/static-openlabnotebooks/following-up-on-nsd3s-involvement-in-epithelial-to-mesenchymal-reprogramming/#comments</comments>
				<pubDate>Sun, 22 Apr 2018 23:24:29 +0000</pubDate>
		<dc:creator><![CDATA[David Dilworth]]></dc:creator>
				<category><![CDATA[David Dilworth]]></category>
		<category><![CDATA[NSD3]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=1073</guid>
				<description><![CDATA[In my last blog post, I observed morphological changes that were indicative of an epithelial to mesenchymal transition (EMT) when NSD3short was overexpressed from a stably integrated transgene in the cancer cell line H1299. Importantly, NSD3 has been identified as amplified in a number of cancer types, including lung (Figure 1) (cBioPortal: Cerami et al., Cancer Discov. 2012 <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/following-up-on-nsd3s-involvement-in-epithelial-to-mesenchymal-reprogramming/">Read More ...</a>]]></description>
								<content:encoded><![CDATA[<p>In my last blog <a href="https://thesgc.github.io/static-openlabnotebooks/over-expression-of-the-nsd3-short-isoform-in-h1299-lung-cancer-cells-alters-cellular-morphology/">post</a>, I observed morphological changes that were indicative of an epithelial to mesenchymal transition (EMT) when NSD3short was overexpressed from a stably integrated transgene in the cancer cell line <a href="https://www.atcc.org/Products/All/CRL-5803.aspx">H1299</a>. Importantly, NSD3 has been identified as amplified in a number of cancer types, including lung (Figure 1) (<b>cBioPortal</b><b>:</b> <a href="https://www.ncbi.nlm.nih.gov/pubmed/22588877">Cerami et al., Cancer Discov. 2012</a> and <a href="https://www.ncbi.nlm.nih.gov/pubmed/23550210">Gao et al. Sci. Signal. 2013</a>). Therefore, it is important to study how increased NSD3 levels may influence a cancer cell.</p>
<div id="attachment_1075" style="width: 1139px" class="wp-caption alignnone"><img aria-describedby="caption-attachment-1075" class="wp-image-1075 size-full" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/04/NSD3_Status_By_Cancer.png" alt="" width="1129" height="579" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/04/NSD3_Status_By_Cancer.png 1129w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/04/NSD3_Status_By_Cancer-300x154.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/04/NSD3_Status_By_Cancer-768x394.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/04/NSD3_Status_By_Cancer-1024x525.png 1024w" sizes="(max-width: 1129px) 100vw, 1129px" /><p id="caption-attachment-1075" class="wp-caption-text">Figure 1. NSD3 Status Across Multiple Cancer Types (Data from cBioPortal)</p></div>
<p>To follow up on this putative phenotype, I have tested the same cell lines for <a href="https://zenodo.org/record/1226772#.Wt0MI4jwbb0">E-cadherin expression</a> and in <a href="https://zenodo.org/record/1226774#.Wt0OFYjwbb0">wound healing assays</a> (Figure 2 &#8211; full experimental details and results in the links). During epithelial to mesenchymal transitioning (EMT), transcription networks become activated and cytoskeletal rearrangements take place, promoting migratory capacity and invasiveness. Cadherins are a family of cell adhesion molecules that are important regulators of these events. In particular, E-cadherin, a marker for polarized epithelial cells, is downregulated during EMT. When we assay E-cadherin levels in the H1299 cell lines, we observe decreased expression with either wild-type or PWWP1 mutant (W284A) NSD3 transgene overexpression (Figure 2A). Next, to test if these cells display increased migratory capacity, I turned to a field standard, the wound healing assay. This involves making a scratch in a cell monolayer and measuring the extent to which cells migrate and divide into the cleared space over time. We observe that these cell lines appear to close wounds faster than the control (Empty Vector). Collectively, these results support the hypothesis that NSD3short is a potentiator of EMT, although PWWP1 function does not appear to be required. These results will be validated by repeating with independently transduced and puro-selected stable lines and in NSD3-knockdown conditions.</p>
<div id="attachment_1081" style="width: 790px" class="wp-caption alignnone"><img aria-describedby="caption-attachment-1081" class="size-large wp-image-1081" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/04/NSD3_Ecad_WH_04220218-1-1024x623.png" alt="" width="780" height="475" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/04/NSD3_Ecad_WH_04220218-1-1024x623.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/04/NSD3_Ecad_WH_04220218-1-300x182.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/04/NSD3_Ecad_WH_04220218-1-768x467.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/04/NSD3_Ecad_WH_04220218-1.png 1311w" sizes="(max-width: 780px) 100vw, 780px" /><p id="caption-attachment-1081" class="wp-caption-text">Figure 2. (A) Western blot for E-cadherin Expression (B) Representative Wells from a Wound Healing Assay at 0 and 72 hours</p></div>
]]></content:encoded>
							<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/following-up-on-nsd3s-involvement-in-epithelial-to-mesenchymal-reprogramming/feed/</wfw:commentRss>
		<slash:comments>1</slash:comments>
							</item>
		<item>
		<title>Over-expression of the NSD3-Short Isoform in H1299 Lung Cancer Cells Alters Cellular Morphology</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/over-expression-of-the-nsd3-short-isoform-in-h1299-lung-cancer-cells-alters-cellular-morphology/</link>
				<comments>https://thesgc.github.io/static-openlabnotebooks/over-expression-of-the-nsd3-short-isoform-in-h1299-lung-cancer-cells-alters-cellular-morphology/#comments</comments>
				<pubDate>Fri, 16 Mar 2018 16:37:20 +0000</pubDate>
		<dc:creator><![CDATA[David Dilworth]]></dc:creator>
				<category><![CDATA[David Dilworth]]></category>
		<category><![CDATA[NSD3]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=899</guid>
				<description><![CDATA[Recently, a study of cancer-focused protein-protein interactions identified a direct interaction between NSD3 and the oncoprotein Myc in the non-small cell lung cancer cell line H1299 (Li, Z et al. 2017).  Importantly, the study found that increased NSD3 expression levels can activate Myc in H1299 cells as measured by an E-box luciferase reporter. However, the functional significance of <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/over-expression-of-the-nsd3-short-isoform-in-h1299-lung-cancer-cells-alters-cellular-morphology/">Read More ...</a>]]></description>
								<content:encoded><![CDATA[<p>Recently, a study of cancer-focused protein-protein interactions identified a direct interaction between NSD3 and the oncoprotein Myc in the non-small cell lung cancer cell line H1299 <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5316855/">(Li, Z et al. 2017)</a>.  Importantly, the study found that increased NSD3 expression levels can activate Myc in H1299 cells as measured by an E-box luciferase reporter. However, the functional significance of this activity in the context of lung cancer remains unclear.</p>
<p>To study NSD3 in this context, as well as other cell types, I have been optimizing lentiviral vectors to express a tagged NSD3short transgene. Initially, I had <a href="https://thesgc.github.io/static-openlabnotebooks/western-blots-lentiviruses-building-tools-to-study-nsd3/">limited success</a> producing viral particles capable of integrating a bicistronic expression cassette containing NSD3short and a puromycin resistance marker separated by an internal ribosome entry site (IRES). I hypothesized that this may be due to packaging constraints of the lentiviral system and opted to decrease the size of my construct by replacing the IRES element with a sequence encoding a T2A self-cleaving peptide (details of mutagenesis can be found here &#8211; <a href="https://zenodo.org/record/1170478#.WqkjKejwbSE">Zenodo &#8211; Exp016</a>). The 2A self-cleaving system has the added benefit of equimolar expression, while the IRES sequence tends to favour expression of the first coding sequence at approximately 3:1 <a href="https://www.ncbi.nlm.nih.gov/pubmed/22114669">(Goedhart et al. 2011)</a>. For a helpful overview of this topic, I suggest checking out an excellent<a href="http://blog.addgene.org/plasmids-101-multicistronic-vectors"> blog post</a> on the subject over at addgene. This modification to the initial construct improved production of functional virus, as I was able to select for stable integration and expression of NSD3short in multiple cells lines (full experimental details can be found here &#8211; <a href="https://zenodo.org/record/1197974#.WqkjW-jwbSE">Zenodo &#8211; Exp018</a>). Interestingly, overexpression of either NSD3short or a PWWP1 mutant (W284A) deficient in H3K36me2 binding resulted in altered morphological features in H1299 cells (Figure1). In cells overexpressing NSD3short, I observed greater eccentricities in their appearance with some cells displaying long projections. This may be indicative of an<a href="https://en.wikipedia.org/wiki/Epithelial%E2%80%93mesenchymal_transition"> </a>epithelial-mesenchymal transition (<a href="https://en.wikipedia.org/wiki/Epithelial%E2%80%93mesenchymal_transition">EMT</a>), which would be in line with elevated Myc activity (<a href="https://www.ncbi.nlm.nih.gov/pubmed/20144848">Cho, KB et al. 2010</a>). EMT is a developmental switch that can promote invasion and migration of cancer cells, phenotypes associated with metastasis.  However, before we can make any claim with respect to NSD3&#8217;s involvement in this process, I will have to follow up these observations by assaying hallmarks of EMT, like downregulation of E-cadherin, with multiple biological replicates.</p>
<p>&nbsp;</p>
<div id="attachment_900" style="width: 478px" class="wp-caption alignnone"><img aria-describedby="caption-attachment-900" class="wp-image-900" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/03/Figure1_Exp018-300x158.png" alt="" width="468" height="246" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/03/Figure1_Exp018-300x158.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/03/Figure1_Exp018-768x405.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/03/Figure1_Exp018-1024x540.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/03/Figure1_Exp018.png 1052w" sizes="(max-width: 468px) 100vw, 468px" /><p id="caption-attachment-900" class="wp-caption-text">Figure 1. Expression of NSD3short-3xFLAG in H1299 Lung Cancer Cells (A) Western blot of NSD3 and FLAG in stable cell lines (B) Representative phase contrast images (C) Boxplot of cellular<br />eccentricity (mean of 4 images across 4 wells per sample).</p></div>
]]></content:encoded>
							<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/over-expression-of-the-nsd3-short-isoform-in-h1299-lung-cancer-cells-alters-cellular-morphology/feed/</wfw:commentRss>
		<slash:comments>3</slash:comments>
							</item>
		<item>
		<title>Western blots &#038; Lentiviruses: Building Tools to Study NSD3</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/western-blots-lentiviruses-building-tools-to-study-nsd3/</link>
				<comments>https://thesgc.github.io/static-openlabnotebooks/western-blots-lentiviruses-building-tools-to-study-nsd3/#comments</comments>
				<pubDate>Mon, 12 Feb 2018 17:57:11 +0000</pubDate>
		<dc:creator><![CDATA[David Dilworth]]></dc:creator>
				<category><![CDATA[David Dilworth]]></category>
		<category><![CDATA[NSD3]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=693</guid>
				<description><![CDATA[The early stages of a project often involve spending considerable time building tools to effectively tackle the problem. In cell biology, this often includes testing antibodies and expression constructs in your cell lines of interest. Putting in the work to properly validate these reagents early on in a study is critical. Poorly validated antibodies are, at least <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/western-blots-lentiviruses-building-tools-to-study-nsd3/">Read More ...</a>]]></description>
								<content:encoded><![CDATA[<p>The early stages of a project often involve spending considerable time building tools to effectively tackle the problem. In cell biology, this often includes testing antibodies and expression constructs in your cell lines of interest. Putting in the work to properly validate these reagents early on in a study is critical. Poorly validated antibodies are, at least partially, to blame for <a href="https://www.nature.com/news/reproducibility-crisis-blame-it-on-the-antibodies-1.17586">poor reproducibility</a> in life sciences. Recently, I tested two additional commercial antibodies to NSD3, which displayed quite a bit of cross-reactivity in western blotting (full experimental details are up on zenodo<a href="https://zenodo.org/record/1170474#.WoHSbyXwbIU"> &#8211; exp012</a>). To date, the AbCam (a180500) antibody seems to perform best in my hands.</p>
<p>To express 3xFLAG tagged NSD3 short in different cell backgrounds, I have been working with a lentivirus expression system. Recently, I have made a W284A mutant NSD3-short expression plasmid to test PWWP1-dependent phenotypes (see zenodo &#8211; <a href="https://zenodo.org/record/1170478#.WoHS_SXwbIU">exp 013</a>). NSD3&#8217;s PWWP1 domain recognizes and binds H3K36me2 methylated histones, and it is still unclear how this module contributes to NSD3&#8217;s role in oncogenesis. Also, I have tested lentivirus generated from the RRL-NSD3short-IRES-Puro plasmid, in H1299 cells, a lung cancer model. Unfortunately, this construct did not generate infectious particles relative to empty-vector or GFP positive control (see zenodo &#8211; <a href="https://zenodo.org/record/1170482#.WoHTwiXwbIU">exp014</a>). I believe this may be due to the inherent packaging limits of the virus I am using ~10 000kb. My current expression vector exceeds this by about 500 bp, which may be inhibiting viral production. I should be able to reduce the size under this threshold by swapping out the IRES bicistronic element and replacing it with a P2A self-cleaving peptide.</p>
<p>&nbsp;</p>
<p><img class="alignnone size-medium wp-image-698" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/02/exp012_12-02-2018-1-248x300.png" alt="" width="248" height="300" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/02/exp012_12-02-2018-1-248x300.png 248w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/02/exp012_12-02-2018-1.png 367w" sizes="(max-width: 248px) 100vw, 248px" /></p>
<p>&nbsp;</p>
]]></content:encoded>
							<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/western-blots-lentiviruses-building-tools-to-study-nsd3/feed/</wfw:commentRss>
		<slash:comments>1</slash:comments>
							</item>
		<item>
		<title>Western Blot of NSD3 in AML Cell Lines</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/western-blot-of-nsd3-in-aml-cell-lines/</link>
				<comments>https://thesgc.github.io/static-openlabnotebooks/western-blot-of-nsd3-in-aml-cell-lines/#respond</comments>
				<pubDate>Sun, 28 Jan 2018 22:41:20 +0000</pubDate>
		<dc:creator><![CDATA[David Dilworth]]></dc:creator>
				<category><![CDATA[David Dilworth]]></category>
		<category><![CDATA[Leukemia]]></category>
		<category><![CDATA[NSD3]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=581</guid>
				<description><![CDATA[In order to study NSD3 in the context of acute myeloid leukemia, I first need to determine the best reagents/tools. This includes characterizing model cell lines as well as antibodies that can be used for future experiments. Here I looked at the expression of NSD3 by western blotting in three AML cell lines using a <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/western-blot-of-nsd3-in-aml-cell-lines/">Read More ...</a>]]></description>
								<content:encoded><![CDATA[<p>In order to study NSD3 in the context of acute myeloid leukemia, I first need to determine the best reagents/tools. This includes characterizing model cell lines as well as antibodies that can be used for future experiments. Here I looked at the expression of NSD3 by western blotting in three AML cell lines using a rabbit monoclonal antibody from AbCam (<a href="http://www.abcam.com/nsd3-antibody-epr13813-n-terminal-ab180500.html">ab180500</a>). It appears there may be some protein degradation in two of three samples (MOLM-13 and HL-60). I will be testing another antibody this week and additional cell lines.</p>
<p>Full experimental details can be found at <a href="https://zenodo.org/record/1161786#.Wm5Ow3WnHCI">zenodo.</a></p>
<p><img class="alignnone size-medium wp-image-582" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/westernblot-193x300.png" alt="" width="193" height="300" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/westernblot-193x300.png 193w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/westernblot.png 642w" sizes="(max-width: 193px) 100vw, 193px" /></p>
]]></content:encoded>
							<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/western-blot-of-nsd3-in-aml-cell-lines/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
							</item>
		<item>
		<title>Cloning an NSD3-Short-3xFLAG Construct into a Lentiviral Expression System</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/cloning-an-nsd3-short-3xflag-construct-into-a-lentiviral-expression-system/</link>
				<comments>https://thesgc.github.io/static-openlabnotebooks/cloning-an-nsd3-short-3xflag-construct-into-a-lentiviral-expression-system/#respond</comments>
				<pubDate>Thu, 18 Jan 2018 16:30:22 +0000</pubDate>
		<dc:creator><![CDATA[David Dilworth]]></dc:creator>
				<category><![CDATA[David Dilworth]]></category>
		<category><![CDATA[Leukemia]]></category>
		<category><![CDATA[NSD3]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=493</guid>
				<description><![CDATA[Antibodies are fundamental tools in the exploration of cellular biology. Unfortunately, there is a lack of high-quality NSD3 antibodies available. To overcome this, I have generated a bicistronic NSD3-Short-3xFLAG &#8211; IRES &#8211; PuroR lentiviral expression plasmid for the purpose of creating cell lines that stably express 3xFLAG-tagged NSD3-short. This will allow me to use a <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/cloning-an-nsd3-short-3xflag-construct-into-a-lentiviral-expression-system/">Read More ...</a>]]></description>
								<content:encoded><![CDATA[<p>Antibodies are fundamental tools in the exploration of cellular biology. Unfortunately, there is a lack of high-quality NSD3 antibodies available. To overcome this, I have generated a bicistronic NSD3-Short-3xFLAG &#8211; IRES &#8211; PuroR lentiviral expression plasmid for the purpose of creating cell lines that stably express 3xFLAG-tagged NSD3-short. This will allow me to use a well-characterized anti-FLAG antibody to study NSD3 in cancer cell lines.</p>
<p>Full experimental details have been posted to <a href="https://zenodo.org/record/1154862#.WmDItq6nHIU">zenodo</a>.</p>
<div id="attachment_501" style="width: 245px" class="wp-caption aligncenter"><img aria-describedby="caption-attachment-501" class="wp-image-501 size-medium" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/NSD3_BamHI-SalI_Digest-235x300.png" alt="" width="235" height="300" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/NSD3_BamHI-SalI_Digest-235x300.png 235w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/NSD3_BamHI-SalI_Digest.png 360w" sizes="(max-width: 235px) 100vw, 235px" /><p id="caption-attachment-501" class="wp-caption-text">Figure 1. BamHI and SalI double restriction digest of RRL-Puro lentiviral backbone and an NSD3-Short-3xFLAG PCR product in preperation for ligation.</p></div>
]]></content:encoded>
							<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/cloning-an-nsd3-short-3xflag-construct-into-a-lentiviral-expression-system/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
							</item>
		<item>
		<title>Project Overview: Understanding Enhancer Biology in Cancer</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/project-overview-understanding-enhancer-biology-in-cancer/</link>
				<comments>https://thesgc.github.io/static-openlabnotebooks/project-overview-understanding-enhancer-biology-in-cancer/#respond</comments>
				<pubDate>Wed, 03 Jan 2018 21:52:12 +0000</pubDate>
		<dc:creator><![CDATA[David Dilworth]]></dc:creator>
				<category><![CDATA[David Dilworth]]></category>
		<category><![CDATA[Leukemia]]></category>
		<category><![CDATA[NSD3]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=329</guid>
				<description><![CDATA[Every cell in your body contains the exact same set of genetic instructions encoded in its genome. However, for complex multicellular organisms such as ourselves, we depend on many specialized cell types that exhibit strikingly different features. This is largely accomplished by precisely controlling how and when genetic information is read out in each cell, <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/project-overview-understanding-enhancer-biology-in-cancer/">Read More ...</a>]]></description>
								<content:encoded><![CDATA[<p>Every cell in your body contains the exact same set of genetic instructions encoded in its genome. However, for complex multicellular organisms such as ourselves, we depend on many specialized cell types that exhibit strikingly different features. This is largely accomplished by precisely controlling how and when genetic information is read out in each cell, known as a cell’s gene expression profile or transcriptional program. Cancer results from mutations, which drive abnormal transcriptional programs that ultimately manifest as the <a href="http://www.cell.com/abstract/S0092-8674(11)00127-9">hallmarks of cancer</a>, including evasion of cell death and unlimited replicative potential. If we can learn more about how these transcriptional programs are executed both in health and disease, we may be able to find ways to rewire dangerous gene expression patterns to either destroy cancer cells or force them into a benign state.</p>
<p>One way in which transcriptional programs are enforced is through distal tissue-specific enhancer regions that activate gene expression in specific cell types (Figure 1). Enhancer regions also become co-opted in some cancers, resulting in increased production of oncogenes (genes that promote cancer) [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4254433/">1</a>]. Cancer cells often become dependent on the expression of these genes to maintain tumour growth, a feature that is frequently referred to as oncogene addiction. If we can find ways to disrupt the enhancer elements that are driving expression of these genes, we may be able to turn off oncogenic transcription programs. My research focuses on the components that mediate enhancer biology and I hope this work will expand our understanding of the regulation and misregulation of transcriptional programs.</p>
<div id="attachment_330" style="width: 790px" class="wp-caption aligncenter"><img aria-describedby="caption-attachment-330" class="wp-image-330 size-large" src="http://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/enhancer_figure-1024x454.png" alt="" width="780" height="346" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/enhancer_figure-1024x454.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/enhancer_figure-300x133.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/enhancer_figure-768x341.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/enhancer_figure.png 1665w" sizes="(max-width: 780px) 100vw, 780px" /><p id="caption-attachment-330" class="wp-caption-text">Figure 1 &#8211; Enhancers are distal genetic elements that become bound by transcriptional activators to promote the transcription of a gene with the potential to influence cell identity.</p></div>
<p>For this project, which I will be sharing my data prepublication through this blog, I will focus primarily on a novel regulator of enhancer function called NSD3. In a recent publication, Chen et al found that NSD3 is part of a protein complex that promotes enhancer activity and is required for the maintenance of acute myeloid leukemia (<a href="https://en.wikipedia.org/wiki/Acute_myeloid_leukemia">AML</a>), a cancer of the blood [<a href="https://www.ncbi.nlm.nih.gov/pubmed/26626481">2</a>]. The authours of this study found that a shortened version (isoform) of NSD3, which lacks several functional regions of NSD3, including its histone methyltransferase domain, is responsible for mediating an interaction between the proteins <a href="http://www.uniprot.org/uniprot/O60885">BRD4</a> and<a href="http://www.uniprot.org/uniprot/Q9HCK8"> CHD8</a>, two components that are critical for enhancer function. The short isoform also turns out to be the most dominant form present in AML patient samples described in the public <a href="https://cancergenome.nih.gov/">TCGA</a> database (Figure 2). Importantly, further research is needed to determine NSD3&#8217;s exact role at enhancers and whether it is a suitable drug target for the treatment of AML and other cancers. To address this problem, I will use bioinformatic methods to mine publically available datasets as well as perform cell biology experiments to further explore how NSD3 contributes to enhancer activity in cancer. All data for this project will be hosted on <a href="https://zenodo.org/">Zenodo</a> with links, updates, and brief literature reviews relevant to the work posted to this blog.</p>
<div id="attachment_331" style="width: 610px" class="wp-caption aligncenter"><img aria-describedby="caption-attachment-331" class="wp-image-331" src="http://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/laml_isoform_figure-1024x730.png" alt="" width="600" height="428" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/laml_isoform_figure-1024x730.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/laml_isoform_figure-300x214.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/laml_isoform_figure-768x547.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/laml_isoform_figure.png 1242w" sizes="(max-width: 600px) 100vw, 600px" /><p id="caption-attachment-331" class="wp-caption-text">Figure 2 &#8211; (A) NSD3 isoforms annotations from<a href="https://genome.ucsc.edu/"> UCSC genome browser</a>. The short isoform is highlighted. Intron lengths have been shortened where indicated (//) for clarity.  (B) Median RNA-Seq expression levels in Transcripts per Million Reads (TPM) for all NSD3 isoforms across 173 TCGA-LAML patient samples obtained from <a href="http://wiki.tgilab.org/ISOexpresso/">ISOexpresso</a> platform [<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4983006/">3</a>]. (C) Box plot of NSD3 isoform expression (TPM) from TCGA-LAML dataset. The short NSD3 isoform is the most dominant.</p></div>
<h6></h6>
<h6>[1]<a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4254433/"> Kron, KJ et al. (2014) Genome Med. 6:77 </a></h6>
<h6>[2] <a href="https://www.ncbi.nlm.nih.gov/pubmed/26626481">Shen, C et al. (2015). Mol Cell. 60 (6):847-59 </a></h6>
<h6>[3] <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4983006/">Seok Yang, I et al. (2016) BMC Genomics. 17:631</a></h6>
<div class="fm-citation half_rhythm no_top_margin clearfix">
<div class="small">
<div class="inline_block eight_col va_top">
<div></div>
</div>
</div>
</div>
<div class="half_rhythm">
<div class="contrib-group fm-author"></div>
</div>
]]></content:encoded>
							<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/project-overview-understanding-enhancer-biology-in-cancer/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
							</item>
	</channel>
</rss>
