<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>Cell assay reports &#8211; openlabnotebooks.org</title>
	<atom:link href="https://thesgc.github.io/static-openlabnotebooks/category/project/cell-assay-reports/feed/" rel="self" type="application/rss+xml" />
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<description>A growing team of groundbreaking scientists around the world are now sharing their lab notebooks online</description>
	<lastBuildDate>Mon, 26 Oct 2020 19:00:59 +0000</lastBuildDate>
	<language>en-GB</language>
	<sy:updatePeriod>
	hourly	</sy:updatePeriod>
	<sy:updateFrequency>
	1	</sy:updateFrequency>
	<generator>https://wordpress.org/?v=5.5.3</generator>

<image>
	<url>https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/cropped-lab-jar02-32x32.png</url>
	<title>Cell assay reports &#8211; openlabnotebooks.org</title>
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<width>32</width>
	<height>32</height>
</image> 
	<item>
		<title>FBXW11 interaction with β-catenin &#8211; cellular assay</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/fbxw11-interaction-with-%ce%b2-catenin-cellular-assay/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/fbxw11-interaction-with-%ce%b2-catenin-cellular-assay/#respond</comments>
		
		<dc:creator><![CDATA[Magdalena Szewczyk]]></dc:creator>
		<pubDate>Mon, 26 Oct 2020 17:05:14 +0000</pubDate>
				<category><![CDATA[Cell assay reports]]></category>
		<category><![CDATA[E3 ligases]]></category>
		<category><![CDATA[Magdalena Szewczyk]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=5309</guid>

					<description><![CDATA[Background The SCF (SKP1-CUL1-F-box protein) E3 ubiquitin ligase is a complex of proteins that helps facilitate the ubiquitination of substrates for targeted proteasomal degradation. FBXW11 is F-box containing protein that has 11 WD-repeats and is a substrate recognition component of the SCF complex. FBXW11 recognizes and binds its substrates, including beta-catenin, at phosphorylated sites or <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/fbxw11-interaction-with-%ce%b2-catenin-cellular-assay/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p><strong>Background</strong></p>
<p>The SCF (SKP1-CUL1-F-box protein) E3 ubiquitin ligase is a complex of proteins that helps facilitate the ubiquitination of substrates for targeted proteasomal degradation. FBXW11 is F-box containing protein that has 11 WD-repeats and is a substrate recognition component of the SCF complex. FBXW11 recognizes and binds its substrates, including beta-catenin, at phosphorylated sites or phosphodegrons, and recruits it to the SCF complex for subsequent ubiquitination and degradation (PMID: 10321728)</p>
<p><strong>Assay validation</strong></p>
<p>Assay measuring the interaction of FBXW11 with B-catenin was developed using NanoBRET technology &#8211; a proximity-based assay that can detect protein interactions by measuring energy transfer from a bioluminescent protein donor (NanoLuc luciferase, NL) to a fluorescent protein acceptor (fluorescent ligand bound to HaloTag, HT). In the beginning, we determined the optimal tag orientation of donor constructs: N or C-terminally NL-tagged β-catenin and acceptor constructs: N or C-terminally HT-tagged FBXW11 in HEK293T cells. NL and HT proteins, expressed alone, were used as controls to determine background signal. The best results were obtained with C-terminally NL-tagged B-catenin and C-terminally HT-tagged FBXW11 (Fig.1). The assay was further validated with binding deficient FBXW11 mutant (R447E) designed by Matthieu Schapira. R447E mutation resulted in a significant decrease in the interaction of FBXW11 with B-catenin (Fig.2).</p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/f11b-300x134.png" alt="" width="582" height="260" class=" wp-image-5311 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/f11b-300x134.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/f11b-768x343.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/f11b.png 993w" sizes="(max-width: 582px) 100vw, 582px" /></p>
<p><strong>Fig.1. FBXW11 interaction with B-catenin &#8211; NanoBRET assay. </strong>HEK293T cells were co-transfected with C- or N-terminally NanoLuc (NL)-tagged B-catenin or NL alone and C- or N-terminally Halo-tagged (HT) FBXW11 or HT alone for 24 h. The interaction was measured using NanoBRET assay. The results are MEAN of 2 technical replicates.</p>
<p><strong> <img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/fbxw11-300x168.png" alt="" width="491" height="275" class="wp-image-5310 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/fbxw11-300x168.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/fbxw11-1024x572.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/fbxw11-768x429.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/fbxw11.png 1063w" sizes="(max-width: 491px) 100vw, 491px" /></strong></p>
<p><strong>Fig.2. </strong><strong>R447E mutation decreases the interaction of FBXW11 with B-catenin.</strong> HEK293T cells were co-transfected with C-terminally Halo-tagged (HT) FBXW11 or HT and C-terminally NanoLuc (NL)-tagged B-catenin for 24 h. The interaction was measured using NanoBRET assay. The results are MEAN of 2 technical replicates.</p>
<p>Please go to <a href="https://zenodo.org/record/4134923#.X5ccxohKg2w">Zenodo</a> for experimental details.</p>
<p><strong>Acknowledgments:</strong></p>
<p>Matthieu Schapira &#8211; mutant design, Aidan Levinsky &#8211; help with mutagenesis</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/fbxw11-interaction-with-%ce%b2-catenin-cellular-assay/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>FBXW7 interaction with Cyclin E1 &#8211; cellular assay</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/fbxw7-interaction-with-cyclin-e1-cellular-assay/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/fbxw7-interaction-with-cyclin-e1-cellular-assay/#respond</comments>
		
		<dc:creator><![CDATA[Magdalena Szewczyk]]></dc:creator>
		<pubDate>Mon, 26 Oct 2020 16:42:54 +0000</pubDate>
				<category><![CDATA[Cell assay reports]]></category>
		<category><![CDATA[E3 ligases]]></category>
		<category><![CDATA[Magdalena Szewczyk]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=5302</guid>

					<description><![CDATA[Background The SCF (SKP1-CUL1-F-box protein) E3 ubiquitin ligase is a complex of proteins that helps facilitate the ubiquitination of substrates for targeted proteasomal degradation. FBXW7 (F-box/WD repeat-containing protein 7) is an F-box protein component of the SCF complex and is involved in substrate recognition. The F-box domain tethers FBXW7 to the SCF complex by interacting <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/fbxw7-interaction-with-cyclin-e1-cellular-assay/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p><strong>Background</strong></p>
<p>The SCF (SKP1-CUL1-F-box protein) E3 ubiquitin ligase is a complex of proteins that helps facilitate the ubiquitination of substrates for targeted proteasomal degradation. FBXW7 (F-box/WD repeat-containing protein 7) is an F-box protein component of the SCF complex and is involved in substrate recognition. The F-box domain tethers FBXW7 to the SCF complex by interacting with the SKP1 component while the WD40 repeats are involved in substrate binding. FBXW7 recognizes and binds its substrates at phosphorylated sites or phosphodegrons and recruits them to the SCF complex for subsequent ubiquitination and degradation (PMID:17434132). A variety of substrates exist for FBXW7, many of which are potentially oncogenic, including cyclin E1, MYC, JUN, and NOTCH (PMID: 11565034, 15103331, 14739463, 29149593). The targeted degradation of growth-promoting factors by FBXW7 suggests it likely plays an important role as a tumor suppressor. FBXW7 Arg→Cys mutation at position 465 hinders its ability to interact with SKP1 and thus the SCF complex; this mutation is associated with acute lymphoblastic leukemia (PMID: 11565033, 28007894).</p>
<p><strong>Assay validation</strong></p>
<p>Assay measuring the interaction of FBXW7 with cyclin E1 was developed using NanoBRET technology &#8211; a proximity-based assay that can detect protein interactions by measuring energy transfer from a bioluminescent protein donor (NanoLuc, NL) to a fluorescent protein acceptor (fluorescent ligand bound to HaloTag, HT). In the beginning, we determined the optimal tag orientation of donor constructs: N or C-terminally NL-tagged cyclin E1 and acceptor constructs: N or C-terminally HT- tagged FBXW7 in HEK293T cells. NanoLuc and halo tag proteins, expressed alone, were used as controls to determine background signal. The best results were obtained with C-terminally NL-tagged cyclin E1 and C-terminally HT-tagged FBXW7 (Fig.1). The assay was further validated with a binding deficient FBXW7 mutant (R465E, R505E) designed by Matthieu Schapira. R465E, R505E double mutation resulted in a significant decrease in the interaction of FBXW7 with cyclin E1 (Fig.2).</p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/fbx711-300x132.jpg" alt="" width="598" height="263" class=" wp-image-5307 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/fbx711-300x132.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/fbx711-1024x451.jpg 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/fbx711-768x338.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/fbx711.jpg 1280w" sizes="(max-width: 598px) 100vw, 598px" /></p>
<p><strong>Fig.1. FBXW7 interaction with cyclin E1 &#8211; NanoBRET assay. </strong>HEK293T cells were co-transfected with C- or N-terminally NanoLuc (NL)-tagged cyclin E1 or NL alone and C- or N-terminally Halo-tagged (HT) FBXW7 or HT alone for 24 h. The interaction was measured using NanoBRET assay. The results are MEAN of 2 technical replicates.</p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/fbx7a-300x119.jpg" alt="" width="635" height="252" class=" wp-image-5306 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/fbx7a-300x119.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/fbx7a-1024x405.jpg 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/fbx7a-768x304.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/10/fbx7a.jpg 1145w" sizes="(max-width: 635px) 100vw, 635px" /></p>
<p><strong> </strong><strong>Fig.2. R465E, R505E double</strong><strong> mutation decreases the interaction of FBXW7 with cyclin E1.</strong> HEK293T cells were co-transfected with C-terminally Halo-tagged (HT) FBXW7 (WT or R465E/R505E or HT and C-terminally NanoLuc (NL)-tagged cyclin E1 for 24 h. The interaction was measured using NanoBRET assay. The results are MEAN of 2 technical replicates.</p>
<p>Please go to<a href="https://zenodo.org/record/4134887#.X5b8RohKg2w"> Zenodo</a> for experimental details.</p>
<p><strong>Acknowledgments:</strong></p>
<p>Matthieu Schapira &#8211; mutant design, Aidan Levinsky &#8211; help with mutagenesis</p>
<p>&nbsp;</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/fbxw7-interaction-with-cyclin-e1-cellular-assay/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>WDR5 and histone H3 interaction cell assay report</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/wdr5-and-histone-h3-interaction-cell-assay-report/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/wdr5-and-histone-h3-interaction-cell-assay-report/#respond</comments>
		
		<dc:creator><![CDATA[Magdalena Szewczyk]]></dc:creator>
		<pubDate>Tue, 21 Jul 2020 12:14:51 +0000</pubDate>
				<category><![CDATA[Cell assay reports]]></category>
		<category><![CDATA[Chromatin factors]]></category>
		<category><![CDATA[Magdalena Szewczyk]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<category><![CDATA[USPs]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=5016</guid>

					<description><![CDATA[  Background WDR5 belongs to the WD-repeat protein family and is an important component of various complexes, including SET1/MLL, NuRD, and NSL complex. It interacts with several binding partners via the &#8220;WDR5-interacting&#8221; (Win) site (e.g. symmetrically dimethylated histone H3R2, MLL1-4, SET1-2) or &#8220;WDR5-binding motif&#8221; (WBM) site (e.g. c,l,n-MYC, RBBP5). WDR5 emerged as a prime target <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/wdr5-and-histone-h3-interaction-cell-assay-report/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p><strong> </strong></p>
<p><strong>Background</strong></p>
<p>WDR5 belongs to the WD-repeat protein family and is an important component of various complexes, including SET1/MLL, NuRD, and NSL complex. It interacts with several binding partners via the &#8220;WDR5-interacting&#8221; (Win) site (e.g. symmetrically dimethylated histone H3R2, MLL1-4, SET1-2) or &#8220;WDR5-binding motif&#8221; (WBM) site (e.g. c,l,n-MYC, RBBP5). WDR5 emerged as a prime target for anti-cancer therapy due to its important roles in gene transcription, proliferation, and self-renewal (PIMID 29385767).</p>
<p><strong> </strong><strong>Assay validation</strong></p>
<p>Assay measuring the interaction of WDR5 with histone H3 was developed using NanoBRET technology &#8211; a proximity-based assay that can detect protein interactions by measuring energy transfer from a bioluminescent protein donor to a fluorescent protein acceptor. We tested different donor (N- or C-terminally NanoLuc® tagged WDR5) to acceptor (C-terminally HaloTag®Fusion-tagged histone H3.3) ratios in two different cell lines (U2Os and HEK293T). The assay was validated with WDR5 Win site antagonist &#8211; OICR9429 (PIMID:2695870). The best NanoBRET ratio and the greatest compound effect was obtained in U2Os cells transfected with 0.01 ug of C-terminally HaloTag®Fusion-tagged histone H3.3 and 0.001 µg of C- terminally NanoLuc® -tagged-WDR5 per 96-well (Fig.1).</p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/wdr5-228x300.jpg" alt="" width="368" height="484" class="wp-image-5018 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/wdr5-228x300.jpg 228w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/wdr5.jpg 490w" sizes="(max-width: 368px) 100vw, 368px" /></p>
<p><strong>Fig.1. OICR9429 decreases WDR5 interaction with histone H3.3 &#8211; NanoBRET assay.  </strong>U2Os or HEK293T cells were co-transfected with 0.001 µg of N- or C-terminally NL-tagged WDR5 and different amounts of C-terminally HT-tagged histone H3.3, as indicated (per 96/well) and treated with OICR9429 for 4 h. * &#8211; best assay condition,  NL-NanoLuc® tag, HT- HaloTag®Fusion tag</p>
<p>Experimental details can be found on <a href="https://zenodo.org/record/3953928#.Xxbb955Kg2w">Zenodo</a>.</p>
<p>&nbsp;</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/wdr5-and-histone-h3-interaction-cell-assay-report/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>L3MBTL1 and histone H3 interaction cellular assay</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/l3mbtl1-and-histone-h3-interaction-cellular-assay/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/l3mbtl1-and-histone-h3-interaction-cellular-assay/#respond</comments>
		
		<dc:creator><![CDATA[Magdalena Szewczyk]]></dc:creator>
		<pubDate>Tue, 21 Jul 2020 11:59:04 +0000</pubDate>
				<category><![CDATA[Cell assay reports]]></category>
		<category><![CDATA[Magdalena Szewczyk]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=5011</guid>

					<description><![CDATA[Background Malignant-Brain-Tumor (MBT) protein L3MBTL1 contains three tandem MBT repeats (3xMBT), which interact with mono- and dimethylated lysines of histone H4, H3 and H1.4 resulting in transcriptional repression. The interaction requires conserved aspartic acid (D355) in the second MBT repeat (PMID:17540172, 18408754, 20870725, 18408754). Assay validation Assay measuring the interaction of L3MBTL1 to histone H3 was <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/l3mbtl1-and-histone-h3-interaction-cellular-assay/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p><strong>Background</strong></p>
<p>Malignant-Brain-Tumor (MBT) protein L3MBTL1 contains three tandem MBT repeats (3xMBT), which interact with mono- and dimethylated lysines of histone H4, H3 and H1.4 resulting in transcriptional repression. The interaction requires conserved aspartic acid (D355) in the second MBT repeat (PMID:17540172, 18408754, 20870725, 18408754).</p>
<p><strong>Assay validation</strong></p>
<p>Assay measuring the interaction of L3MBTL1 to histone H3 was developed using NanoBRET technology &#8211; a proximity-based assay that can detect protein interactions by measuring energy transfer from a bioluminescent protein donor to a fluorescent protein acceptor. At the beginning we determined the optimal donor to acceptor ratio of four donor constructs and their binding deficient mutants (D355N): C- or N-terminally NanoLuc® tagged 3xMBT domain of L3MBTL1 and full L3MBTL1 and two acceptor constructs: C-terminally HaloTag®Fusion tagged histone H3 and H4. The best results were obtained with C-terminally NanoLuc® tagged 3xMBT domain of L3MBTL1 and histone H3 at  1:100 ratio, respectively (Fig.1). To validate the assay, we treated the cells with available weak antagonists UNC1215 and UNC669 (PIMID:<strong> 23292653</strong>, <strong>21417280).</strong> Both antagonists at high concentrations decreased the NanoBRET signal in a concentration-dependent manner (Fig 2.).</p>
<p><strong> <img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/L3MBTL1-300x225.png" alt="" width="447" height="336" class=" wp-image-5013 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/L3MBTL1-300x225.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/L3MBTL1-768x576.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/L3MBTL1-350x263.png 350w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/L3MBTL1-780x586.png 780w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/L3MBTL1-250x188.png 250w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/L3MBTL1.png 960w" sizes="(max-width: 447px) 100vw, 447px" /></strong></p>
<p><strong>Fig.1. 3xMBT(L3MBTL1) D355N mutant shows a lower NanoBRET ratio than the wild type protein</strong>. HEK293T cells were co-transfected with C-terminally NanoLuc® tagged wild type 3xMBT(L3MBTL1) or D355N mutant and histone HaloTag®Fusion tagged histone H3- at ratio 1:100, respectively. The results are mean+/- SEM of 3 replicates.</p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/L3MBTL1-1-300x154.png" alt="" width="551" height="283" class=" wp-image-5012 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/L3MBTL1-1-300x154.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/L3MBTL1-1-768x394.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/L3MBTL1-1.png 924w" sizes="(max-width: 551px) 100vw, 551px" /></p>
<p><strong>Fig.2. UNC1215 and UNC669 decrease the interaction of 3xMBT(L3MBTL1) with histone H3 in a dose-dependent manner. </strong>HEK293T cells were co-transfected with C-terminally NanoLuc® tagged wild type 3xMBT(L3MBTL1) or D355N mutant and C-terminally HaloTag®Fusion tagged histone H3. Cells were treated with indicated concentrations of antagonists for 20 h. The results are mean +/-SEM of 3 replicates.</p>
<p>Please go to <a href="https://zenodo.org/record/3953887#.XxbYPZ5Kg2w">Zenodo</a> for experimental details.</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/l3mbtl1-and-histone-h3-interaction-cellular-assay/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>STIP1(HOP) and STUB1(CHIP) interaction with HSPA8 &#8211; cellular assay</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/stip1hop-and-stub1chip-interaction-with-hspa8-cellular-assay/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/stip1hop-and-stub1chip-interaction-with-hspa8-cellular-assay/#respond</comments>
		
		<dc:creator><![CDATA[Magdalena Szewczyk]]></dc:creator>
		<pubDate>Fri, 05 Jun 2020 19:02:55 +0000</pubDate>
				<category><![CDATA[Cell assay reports]]></category>
		<category><![CDATA[E3 ligases]]></category>
		<category><![CDATA[Magdalena Szewczyk]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=4866</guid>

					<description><![CDATA[Background HSP70 heat-shock proteins family, including constitutively expressed HSPA8 variant, facilitate correct folding of client proteins and degradation of unfolded clients. To achieve folding or degradation of client proteins, HSP70 cooperates with co-chaperones, which mostly bind to the highly conserved sequence EEVD in the C-terminal domain of HSP70 via their tetratricopeptide repeat (TPR) domain (PMID: <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/stip1hop-and-stub1chip-interaction-with-hspa8-cellular-assay/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p><strong>Background</strong></p>
<p>HSP70 heat-shock proteins family, including constitutively expressed HSPA8 variant, facilitate correct folding of client proteins and degradation of unfolded clients. To achieve folding or degradation of client proteins, HSP70 cooperates with co-chaperones, which mostly bind to the highly conserved sequence EEVD in the C-terminal domain of HSP70 via their tetratricopeptide repeat (TPR) domain (PMID: 26565746). STIP1(HOP) and ubiquitin ligase STUB1(CHIP) and are TPR-containing co-chaperones that assist with protein folding or degradation, respectively (PIMID:22824801). It was shown previously that mutations within TPR domains of STIP1 (K8A) and STUB1 (K30A) abolished their interaction with HSPA8 (PIMID: 23865999, 26565746).</p>
<p><strong>Assay validation</strong></p>
<p>Assay measuring the interaction of HSPA8 to STIP1 and STUB1 co-chaperons was developed using NanoBRET technology &#8211; a proximity-based assay that can detect protein interactions by measuring energy transfer from a bioluminescent protein donor to a fluorescent protein acceptor. At the beginning we determined the optimal tag orientation and donor to acceptor ratio of donor constructs: N or C-terminally NanoLuc® tagged STIP1/STIP1 K8A mutant or STUB1/STUB1 K30A and acceptor construct: C-terminally HaloTag®Fusion tagged HSPA8. The best results were obtained with C-terminally tagged donor constructs and 1:9 donor to acceptor ratio. Consistent with previous findings the interaction (NanoBRET ratio) of HSPA8 and mutated co-chaperons was significantly lower compared to wild type proteins (Fig.1).</p>
<p><strong> <img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/stipstub-300x170.jpg" alt="" width="671" height="380" class="wp-image-4867 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/stipstub-300x170.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/stipstub-768x434.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/stipstub.jpg 945w" sizes="(max-width: 671px) 100vw, 671px" /></strong></p>
<p><strong>Fig.1. HSPA8 interaction with STIP1 (HOP) and STUB1 (CHIP) co-chaperones. </strong>HEK293T cells were co-transfected with Nanoluc (NL)-tagged STIP1 (WT or K8A mutant) or NL-tagged STUB1 (WT or K30A mutant) and Halo-tagged (HT) HSPA8 for 24 h. The interaction was measured using NanoBRET assay. The results are MEAN+/-SD of 3 technical replicates.</p>
<p>Experimental details are posted on <a href="https://zenodo.org/record/3879514#.XtqWgzpKg2w">Zenodo</a>.</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/stip1hop-and-stub1chip-interaction-with-hspa8-cellular-assay/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>PRMT7 cellular assay</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/prmt7-cellular-assay/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/prmt7-cellular-assay/#respond</comments>
		
		<dc:creator><![CDATA[Magdalena Szewczyk]]></dc:creator>
		<pubDate>Fri, 05 Jun 2020 18:44:13 +0000</pubDate>
				<category><![CDATA[Cell assay reports]]></category>
		<category><![CDATA[Magdalena Szewczyk]]></category>
		<category><![CDATA[PRMTs]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=4859</guid>

					<description><![CDATA[Background PRMT7 is a type III protein arginine methyltransferase that monomethylates arginine residues mainly within the RXR motif (Fig.1) (PIMID: 24247247). PRMT7 was shown to play an important role in muscle physiology, stem cell biology, and cellular stress response (PIMID: 32409666, 27225728, 26854227).  Fig.1. Mammalian PRMTs. There are 9 members and 3 types of the <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/prmt7-cellular-assay/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p><strong>Background</strong></p>
<p>PRMT7 is a type III protein arginine methyltransferase that monomethylates arginine residues mainly within the RXR motif (Fig.1) (PIMID: 24247247). PRMT7 was shown to play an important role in muscle physiology, stem cell biology, and cellular stress response (PIMID: 32409666, 27225728, 26854227).</p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1-300x119.png" alt="" width="643" height="255" class="wp-image-4626 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1-300x119.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1-1024x405.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1-768x304.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1.png 1214w" sizes="(max-width: 643px) 100vw, 643px" /></p>
<p><strong> </strong><strong>Fig.1. Mammalian PRMTs. </strong>There are 9 members and 3 types of the PRMT family. Type I, II, and III PRMTs catalyze the formation of monomethyl arginines (Rme1). Type I PRMTs catalyze asymmetric arginine methylation (Rme2a), while type II PRMTs form symmetric arginine methylation (Rme2s). Type III PRMT can only monomethylate arginine residues. Modified from (PMID:29378138).</p>
<p><strong> </strong><strong>Assay validation</strong></p>
<p><strong> </strong>It has been reported previously that PRMT7 is the main contributor to HSP70 R469 monomethylation in cells (PIMID:32409666). We found that<em> PRMT7</em> knockout or knockdown reduces HSP70 monomethylation in various cell lines (Fig. 2). The assay was further validated with the PRMT7 selective chemical probe SGC3027 (PIMID: 32409666). SGC3027 but not its inactive control SGC3027N decreased PRMT7 dependent HSP70 monomethylation in C2C12 cells in a dose-dependent manner (Fig.3). The Z factor for the assay equals 0.67.</p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/prmt7-1-300x94.png" alt="" width="744" height="233" class="wp-image-4861 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/prmt7-1-300x94.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/prmt7-1-768x240.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/prmt7-1.png 936w" sizes="(max-width: 744px) 100vw, 744px" /></p>
<p><strong>Fig. 2<em>. PRMT7</em> (P7) knockout (KO) or knockdown (KD) reduces HSP70 methylation in various cell lines.</strong> Western blot analysis of HSP70 monomethylation. 94A, 21B—HCT116 CRISPR <em>PRMT7</em> KO clones; P parental C2C12; C-C2C12 expressing control guide RNA; 32 C2C12 CRISPR clone expressing PRMT7 catalytic mutant (delY35,A35S); 4,57,74—C2C12 CRISPR <em>Prmt7</em> KO clones. <em>PRMT7</em> was knocked down in HEK293T and MCF7 cells using siRNA. Con-control, P7KD-<em>PRMT7</em> knockdown. Modified from PIMID:32409666. (Shawna Organ, Magdalena Szewczyk)</p>
<p>&nbsp;</p>
<p><strong> <img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/prmt7-300x137.png" alt="" width="740" height="338" class="wp-image-4860 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/prmt7-300x137.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/prmt7-768x351.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/prmt7.png 988w" sizes="(max-width: 740px) 100vw, 740px" /></strong></p>
<p><strong>Fig.3.</strong> <strong>SGC3027 but not negative control compound SGC3027N inhibits PRMT7-dependent HSP70 monomethylation in C2C12 cells in a dose-dependent manner.</strong> Cells were treated with the compound for 2 days. <strong>A.</strong> A representative blot for SGC3027 and SGC3027N effects on HSP70 methylation. Rme1—arginine monomethylation.<strong>B</strong> Quantification of SGC3027 and SGC3027N effects on HSP70 monomethylation in C2C12 cells. The graphs represent non-linear fits of Rme1 signal intensities normalized to intensities of HSP70. SGC3027: <em>n</em> = 11, four separate experiments, IC<sub>50</sub> = 2.4 ± 0.1 µM; SGC3027N: <em>n</em> = 4 technical replicates, IC<sub>50</sub> &gt; 40 µM (mean ± SEM). Modified from PIMID:32409666. (Magdalena Szewczyk)</p>
<p>For experimental details please go to <a href="https://zenodo.org/record/3879397#.XtqPzzpKg2w">Zenodo</a>.</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/prmt7-cellular-assay/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>NSD3(PWWP1) and histone H3 interaction cellular assay</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/nsd3pwwp1-and-histone-h3-interaction-cellular-assay/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/nsd3pwwp1-and-histone-h3-interaction-cellular-assay/#respond</comments>
		
		<dc:creator><![CDATA[Magdalena Szewczyk]]></dc:creator>
		<pubDate>Fri, 29 May 2020 17:13:56 +0000</pubDate>
				<category><![CDATA[Cell assay reports]]></category>
		<category><![CDATA[Magdalena Szewczyk]]></category>
		<category><![CDATA[NSD3]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=4807</guid>

					<description><![CDATA[Background NSD3 is a member of the NSD (nuclear receptor SET domain- containing) family of H3K36 methyltransferases. It has two isoforms: long -1437 amino acid and short &#8211; 645 amino acids, which are derived from alternative splicing of exon 10.  In contrast to long isoform, the short isoform lacks the methyltransferase domain and it contains <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/nsd3pwwp1-and-histone-h3-interaction-cellular-assay/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p><strong>Background</strong></p>
<p>NSD3 is a member of the NSD (nuclear receptor SET domain- containing) family of H3K36 methyltransferases. It has two isoforms: long -1437 amino acid and short &#8211; 645 amino acids, which are derived from alternative splicing of exon 10.  In contrast to long isoform, the short isoform lacks the methyltransferase domain and it contains only one histone methyl-lysine reader domain (PWWP1).  The overexpression of both NSD3 isoforms is capable of oncogenic transformation indicating that the oncogenic function is not related to methyltransferase activity. Short NSD3 isoform was shown to sustain AML leukemia by linking BRD4 to the CHD8 chromatin remodeler, through the PWWP chromatin reader domain.  (PMID: 20940404, 23269674, 25942451, 26626481)</p>
<p><strong>Assay validation</strong></p>
<p>Assay measuring the interaction of NSD3-PWWP1 to histone H3 was developed using NanoBRET technology &#8211; a proximity-based assay that can detect protein interactions by measuring energy transfer from a bioluminescent protein donor to a fluorescent protein acceptor. The assay was optimized and validated using NSD3 and H3 antagonist BI-9466 and W284/E318K binding deficient mutant. The best results were obtained with U2Os cells transfected with C-terminally NanoLuc tagged NSD3-PWWP1 and H3 at 1:10 ratio. As shown in Fig 1. chemical probe antagonist (BI-9466) but not its negative control (BI-9321) selectively decreased the BRET signal only in WT NSD3-PWWP1. No effect was observed on binding deficient W284A/E318K mutant and closely related NSD2-PWWP1 domain. BI-9466 decreased NSD3-PWWP1 and H3 interaction in a dose-dependent manner (Fig2). (PIMID: 31285596)</p>
<p><u><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/nsd3-1-300x237.png" alt="" width="537" height="424" class="wp-image-4810 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/nsd3-1-300x237.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/nsd3-1-768x607.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/nsd3-1.png 801w" sizes="(max-width: 537px) 100vw, 537px" /></u></p>
<p><strong> </strong><strong>Fig.1. BI-9466 antagonizes interaction between histone H3 and WT NSD3-PWWP1 but not NSD3-PWWP1(W284A/E318K) mutant or NSD2-PWWP1 in NanoBRET assay. </strong>U2Os cells were co-transfected with C-terminally HaloTag Fusion-tagged histone H3 and C-terminally NanoLuc-tagged NSD3-PWWP1, NSD3-PWWP1(W284A/E318K), or NSD2-PWWP1 and treated with 10 µM BI-9321 (antagonist) or BI-9466 (negative control) for 24 h. Shown is the mean ± SD of two biological replicates, run in quadruplicate. (David Dilworth, Magdalena Szewczyk)</p>
<p>&nbsp;</p>
<p><strong> <img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/nsd3-2-300x215.png" alt="" width="546" height="391" class="wp-image-4809 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/nsd3-2-300x215.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/nsd3-2-768x551.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/nsd3-2.png 870w" sizes="(max-width: 546px) 100vw, 546px" /></strong></p>
<p><strong>Fig.2. BI-9466 but not negative control compound BI-9321 decreases the interaction of NSD3-PWWP1 and histone H3 in a dose-dependent manner. </strong>NanoBRET based on the interaction of C-terminally NanoLuc-tagged NSD3-PWWP1 with C-terminally HaloTag Fusion tagged histone H3 with increasing concentrations of either BI-9321 (blue) or control compound BI-9466 (green). Shown is a representative dose-response curve. IC50 was calculated from the mean ± SD of three independent replicates. (David Dilworth, Magdalena Szewczyk)</p>
<p>For experimental details go to <a href="https://zenodo.org/record/3865777#.XtFCUzpKg2w">Zenodo</a>.</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/nsd3pwwp1-and-histone-h3-interaction-cellular-assay/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>NSD2(PWWP1) and histone H3 interaction cellular assay</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/nsd2pwwp1-and-histone-h3-interaction-cellular-assay/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/nsd2pwwp1-and-histone-h3-interaction-cellular-assay/#respond</comments>
		
		<dc:creator><![CDATA[Magdalena Szewczyk]]></dc:creator>
		<pubDate>Fri, 29 May 2020 16:26:47 +0000</pubDate>
				<category><![CDATA[Cell assay reports]]></category>
		<category><![CDATA[Magdalena Szewczyk]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=4800</guid>

					<description><![CDATA[Background NSD2 is a member of the NSD (nuclear receptor SET domain- containing) family of H3K36 methyltransferases which contain a conserved catalytic SET domain and several chromatin-associated domains comprising four PHD fingers, two PWWP domains, and an HMG box. NSD2 is implicated in several diseases including Wolf Hirschhorn syndrome and its overexpression or hyperactivity may <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/nsd2pwwp1-and-histone-h3-interaction-cellular-assay/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p><strong>Background</strong></p>
<p>NSD2 is a member of the NSD (nuclear receptor SET domain- containing) family of H3K36 methyltransferases which contain a conserved catalytic SET domain and several chromatin-associated domains comprising four PHD fingers, two PWWP domains, and an HMG box. NSD2 is implicated in several diseases including Wolf Hirschhorn syndrome and its overexpression or hyperactivity may cause cancer. The first PWWP domain of NSD2 preferentially binds dimethylated H3K36, which stabilizes NDS2 at chromatin and regulates NSD2 occupancy at its gene targets. PWWP1 does not directly impact NSD2 catalytic activity, however, the increase of cell proliferation caused by NSD2 overexpression is abolished by mutations in the PWWP1 domain affecting its binding to H3K36me2 (PMID:23823660, 26912663, 28054944).</p>
<p><strong>Assay validation</strong></p>
<p>Assay measuring the interaction of NSD2 to histone H3 was developed using NanoBRET technology &#8211; a proximity-based assay that can detect protein interactions by measuring energy transfer from a bioluminescent protein donor to a fluorescent protein acceptor. The assay was optimized and validated using NSD2 and H3 antagonist UNC6034 and NSD2-F266A binding deficient mutant. The best results were obtained with U2Os cells transfected with C-terminally NanoLuc-tagged NSD2-PWWP1 and H3 at 1:10 ratio. As shown in Fig 1. chemical probe antagonist (UNC6934) but not its negative control (UNC7145) selectively decreased the BRET signal only in WT NSD2-PWWP1. No effect was observed on F266A binding deficient mutant and closely related NSD3-PWWP1 domain. UNC6934 decreased NSD3-PWWP1 and H3 interaction in a dose-dependent manner (Fig2).</p>
<p><strong> <img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/nsd2-2-300x166.png" alt="" width="450" height="249" class="wp-image-4801 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/nsd2-2-300x166.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/nsd2-2-1024x567.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/nsd2-2-768x425.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/nsd2-2.png 1218w" sizes="(max-width: 450px) 100vw, 450px" /></strong></p>
<p><strong> </strong></p>
<p><strong>Fig.1. UNC6934 antagonizes interaction between histone H3 and WT NSD2-PWWP1 but not NSD2-PWWP1(F266A) mutant or NSD3-PWWP1 in NanoBRET assay. </strong>U2Os cells were co-transfected with C-terminally HaloTag Fusion-tagged histone H3 and C-terminally NanoLuc-tagged NSD2-PWWP1, NSD2-PWWP1(F266A), or NSD3-PWWP1 and treated with 10 µM compounds for 24 h. Shown is the mean ± SD of three biological replicates, run in triplicates. (David Dilworth, Magdalena Szewczyk)</p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/nsd2-1-300x174.png" alt="" width="429" height="249" class="wp-image-4802 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/nsd2-1-300x174.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/nsd2-1-768x445.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/nsd2-1.png 920w" sizes="(max-width: 429px) 100vw, 429px" /></p>
<p><strong>Fig.2. UNC6934 but not negative control compound UNC7145 decrease NSD2-PWWP1 interaction with H3 in a dose-dependent manner. </strong>U2Os cells were co-transfected with C-terminally NanoLuc-tagged NSD2-PWWP1 or NSD2-PWWP1(F266A) and C-terminally HaloTag Fusion-tagged histone H3 and treated with compounds for 24 h. Shown is a representative dose-response curve. IC50 was calculated from the mean ± SD of three biological replicates, run in triplicates. (David Dilworth)</p>
<p>The experimental details are posted on <a href="https://zenodo.org/record/3865717#.XtE3STpKg2w">Zenodo</a>.</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/nsd2pwwp1-and-histone-h3-interaction-cellular-assay/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>p300/CBP cellular assay</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/p300-cbp-cellular-assay/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/p300-cbp-cellular-assay/#respond</comments>
		
		<dc:creator><![CDATA[Magdalena Szewczyk]]></dc:creator>
		<pubDate>Thu, 28 May 2020 16:23:26 +0000</pubDate>
				<category><![CDATA[Cell assay reports]]></category>
		<category><![CDATA[Magdalena Szewczyk]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<category><![CDATA[USPs]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=4793</guid>

					<description><![CDATA[Background Transcriptional coactivators p300 and CBP catalyze the acetylation of lysine residues of a large variety of histone and non-histone proteins (PMID:28426192). p300 and CBP are considered functional homologous proteins that share around 86% amino acid sequence identity in the HAT domain (PIMID:7870179).  The upregulation of p300/CBP has been associated with various cancers, indicating an important role in <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/p300-cbp-cellular-assay/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p><strong>Background</strong></p>
<p>Transcriptional coactivators p300 and CBP catalyze the acetylation of lysine residues of a large variety of histone and non-histone proteins (PMID:28426192). p300 and CBP are considered functional homologous proteins that share around 86% amino acid sequence identity in the HAT domain (PIMID:7870179).  The upregulation of p300/CBP has been associated with various cancers, indicating an important role in tumorigenesis (PIMID:27881443).</p>
<p><strong>Assay validation</strong></p>
<p>P300/CBP has been reported previously to acetylate p53 at lysine 320 and histone H3 at lysine 27 (PIMID: 25545885, 29804834). Based on the findings, we developed p300 cellular assays measuring p53 acetylation. As shown in Fig.1 overexpression of p300 but not its catalytic mutant (Y1467F) increased p53 acetylation levels. We also developed p300/CBP cellular assay measuring H3K27 acetylation.  Knocking down of p300/CBP resulted in a global decrease of H3K27 acetylation levels (Fig 2A). The H3K27 assay was further validated with p300/CBP selective inhibitor A-485 (PIMID: 29769713). A-485 decreased H3K27 acetylation levels in a dose-dependent manner in HFF cells (Fig.2B). Similar results were obtained in other cell lines (data not shown). The z factor for the assay equals 0.7.</p>
<p><strong> </strong></p>
<p><strong> <img loading="lazy" class="wp-image-4795 alignleft" height="363" width="290" alt="" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p300-1.png" /></strong></p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p><strong>Fig.1.</strong><strong> The wild type but not catalytic mutant of p300 acetylated exogenous p53.</strong>  HEK293T cells were co-transfected with FLAG-tagged p53 and empty vector(control),  p300 wild type, or p300 Y1467F catalytic mutant for 24 h<strong>.</strong> The acetylation levels were analyzed in Western blot. (Taylor Mitchell)</p>
<p><strong> </strong></p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p><strong> <img loading="lazy" class="alignnone wp-image-4794" height="248" width="845" alt="" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p300-2-300x88.png" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p300-2-300x88.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p300-2-1024x301.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p300-2-768x226.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p300-2.png 1280w" sizes="(max-width: 845px) 100vw, 845px" /></strong></p>
<p><strong> </strong><strong>Fig.2. The loss of p300/CBP activity results in a decrease of H3K27 acetylation levels. A. </strong>p300/CBP knock-down. HEK293T and MCF7 cells were treated with siRNA targeted for p300/CBP  and control siRNA for 3 days. <strong>B.</strong> p300/CBP inhibition. HFF cells were treated with p300/CBP selective inhibitor A-481 at indicated concentrations for 3 days. H3K27 acetylation levels were determined by Western blot. (Dalia Barsyte, Magdalena Szewczyk)</p>
<p>Experimental details are posted on <a href="https://zenodo.org/record/3862780#.Xs_k6DpKg2w">Zenodo</a>.</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/p300-cbp-cellular-assay/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>RPA2 and RFWD3 interaction cellular assay</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/rpa2-and-rfwd3-interaction-cellular-assay/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/rpa2-and-rfwd3-interaction-cellular-assay/#respond</comments>
		
		<dc:creator><![CDATA[Magdalena Szewczyk]]></dc:creator>
		<pubDate>Thu, 21 May 2020 16:00:40 +0000</pubDate>
				<category><![CDATA[Cell assay reports]]></category>
		<category><![CDATA[E3 ligases]]></category>
		<category><![CDATA[Magdalena Szewczyk]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=4771</guid>

					<description><![CDATA[Background RFWD3 (E3 ligase RING finger and WD repeat domain-containing protein 3) and RPA2 (Replication protein A2) were shown to functionally interact and participate in replication checkpoint control. At stalled replication forks, the RPA complex binds single-stranded DNA to prevent nucleolytic digestions and also to initiate replication checkpoint signaling by recruitment checkpoint proteins. RFWD3 is <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/rpa2-and-rfwd3-interaction-cellular-assay/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p><strong>Background</strong></p>
<p>RFWD3 (E3 ligase RING finger and WD repeat domain-containing protein 3) and RPA2 (Replication protein A2) were shown to functionally interact and participate in replication checkpoint control. At stalled replication forks, the RPA complex binds single-stranded DNA to prevent nucleolytic digestions and also to initiate replication checkpoint signaling by recruitment checkpoint proteins. RFWD3 is recruited to stalled replication forks and interacts directly with RPA2 in response to replication stress. The deletion of the RPA2 binding region on RFWD3 impairs its localization to stalled replication forks (PIMID: 21504906).</p>
<p><strong>Summary</strong></p>
<p>Assay measuring the interaction of RPA2 to RFWD3 was developed using NanoBRET technology &#8211; a proximity-based assay that can detect protein interactions by measuring energy transfer from a bioluminescent protein donor to a fluorescent protein acceptor. We determined the optimal donor to acceptor ratio and levels of expression of 2 donor constructs: C- or N-terminally NanoLuc®(NL)- tagged RFWD3 and acceptor construct: N-terminally HaloTag®Fusion (HT)-tagged RPA2. We compared the results to background NanoBRET signal from HT-RPA2 and NL protein alone. The best results were obtained with N-terminally NL-tagged RFWD3 (0.001 µg/96-well) and N-terminally HT- tagged RPA2 (0.03 µg/96-well) (Fig.1)</p>
<p><strong> </strong></p>
<p><strong> <img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/rpa2-300x152.png" alt="" width="513" height="260" class="wp-image-4772 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/rpa2-300x152.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/rpa2-768x390.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/rpa2.png 962w" sizes="(max-width: 513px) 100vw, 513px" /></strong></p>
<p><strong>Fig 1. RPA2 – RFWD3</strong><strong> nanoBRET assay. </strong>HEK293T cells were plated in 96-well plates and transfected with indicated amounts of C- or N- terminally NL-tagged RFWD3 and N-terminally HT-tagged RPA2. Next day cells were treated +/- fluorescent ligand and 4 h later the luciferase substrate was added, and the signal was read. The results are mean +/- SD of 3 replicates.</p>
<p>Go to <a href="https://zenodo.org/record/3837867#.Xsak7WhKg2w">Zenodo</a> for experimental details.</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/rpa2-and-rfwd3-interaction-cellular-assay/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
	</channel>
</rss>
