<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>Pietro Roversi (Univ. of Leicester) &#8211; openlabnotebooks.org</title>
	<atom:link href="https://thesgc.github.io/static-openlabnotebooks/category/laboratory/pietro-roversi-leic/feed/" rel="self" type="application/rss+xml" />
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<description>A growing team of groundbreaking scientists around the world are now sharing their lab notebooks online</description>
	<lastBuildDate>Wed, 13 May 2020 15:38:15 +0000</lastBuildDate>
	<language>en-GB</language>
	<sy:updatePeriod>
	hourly	</sy:updatePeriod>
	<sy:updateFrequency>
	1	</sy:updateFrequency>
	<generator>https://wordpress.org/?v=5.4.1</generator>

<image>
	<url>https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/cropped-lab-jar02-32x32.png</url>
	<title>Pietro Roversi (Univ. of Leicester) &#8211; openlabnotebooks.org</title>
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<width>32</width>
	<height>32</height>
</image> 
	<item>
		<title>A blind date with an old friend: crystal structures of Cys-S-sulfonic acid GAPDH</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/a-blind-date-with-an-old-friend-crystal-structures-of-cys-s-sulfonic-acid-gapdh/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/a-blind-date-with-an-old-friend-crystal-structures-of-cys-s-sulfonic-acid-gapdh/#respond</comments>
		
		<dc:creator><![CDATA[Pietro Roversi]]></dc:creator>
		<pubDate>Tue, 21 Apr 2020 15:56:58 +0000</pubDate>
				<category><![CDATA[Open Secretion Rescue]]></category>
		<category><![CDATA[Pietro Roversi]]></category>
		<category><![CDATA[Pietro Roversi (Univ. of Leicester)]]></category>
		<category><![CDATA[Rare disease]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=4525</guid>

					<description><![CDATA[Biologists interested in the function or the 3D shape of a certain protein face the task of isolating billions of copies of that protein from cells (a process called &#8220;protein purification&#8221; which does not involve washing or holy water). The aim of protein purification is to start with cells that contain the protein of interest <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/a-blind-date-with-an-old-friend-crystal-structures-of-cys-s-sulfonic-acid-gapdh/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<div id="attachment_4526" style="width: 310px" class="wp-caption alignleft"><img aria-describedby="caption-attachment-4526" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/Wheel-300x180.jpg" alt="The familiar, improved" class="size-medium wp-image-4526" width="300" height="180" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/Wheel-300x180.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/Wheel.jpg 700w" sizes="(max-width: 300px) 100vw, 300px" /><p id="caption-attachment-4526" class="wp-caption-text">The familiar, rediscovered.</p></div>
<p>Biologists interested in the function or the 3D shape of a certain protein face the task of isolating billions of copies of that protein from cells (a process called &#8220;protein purification&#8221; which does not involve washing or holy water). The aim of protein purification is to start with cells that contain the protein of interest in large amounts and arrive at a sample made of billions of copies of that protein &#8211; and not much else. Typically &#8211; the pure protein ends up in a plastic tube, in water and perhaps some salt or some pH buffer to make sure that the proteins are happy and stable in the water solution. Protein purification has <a href="https://www.biologicscorp.com/blog/overview-of-protein-purification-and-characterization/">an illustrious history,</a> and early pioneers such as <a class="" title="James B. Sumner" href="https://www.nobelprize.org/prizes/chemistry/1946/sumner/facts/">James B. Sumner, </a><a class="" title="John H. Northrop" href="https://www.nobelprize.org/prizes/chemistry/1946/northrop/facts/">John H. Northrop </a>and<a class="" title="Wendell M. Stanley" href="https://www.nobelprize.org/prizes/chemistry/1946/stanley/facts/"> Wendell M. Stanley</a> won the <a href="https://www.nobelprize.org/prizes/chemistry/1946/speedread/">Nobel Prize for preparing pure proteins</a>. Once a sample of protein is pure, depending on the questions surrounding it, it can either be used for biochemical assays, or looked at with electromagnetic radiation of some kind, and the results add to the knowledge about the protein, answering questions such as what does the protein do? how does it do it? what 3D shape does it have? how does it move in time? and so on.</p>
<p>Of course &#8211; however simple or complex a human endeavour may be &#8211; what can go wrong will go wrong &#8211; and protein purification is no exception. Proteins are not visible to the naked eye, nor with an optical microscope, and the starting cellular sample contains tens of thousands of different proteins &#8211; so every so often biologists set out to isolate protein X &#8211; and end up with protein Y (&#8220;a <a href="https://en.wikipedia.org/wiki/Contamination">contaminant</a>&#8220;) in their tube (see for example <a href="https://doi.org/10.1016/j.jmb.2006.11.023">here</a>). This is an unfortunate outcome: protein purification (like most complex human endeavours) is not for free. At what point is the mistake apparent? Wise biologists set out to check their samples early during the process, and in fact the most important skill for successful protein purification is to be able to track the protein one wants along the potentially long and laborious steps with which it is progressively separated from the initial mix. This way, as soon as it transpires that the protein of interest is lost (this indeed can happen &#8211; or perhaps it was not there in the first place?) one goes back to square one without further costly efforts. What is not there (anymore) cannot be isolated.</p>
<div id="attachment_4545" style="width: 235px" class="wp-caption alignright"><img aria-describedby="caption-attachment-4545" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/Serendipity-225x300.gif" alt="Serendipity" class="size-medium wp-image-4545" width="225" height="300" /><p id="caption-attachment-4545" class="wp-caption-text">Serendipity</p></div>
<p>However, there&#8217;s always <a href="https://www.jbc.org/content/284/16/10285.full">serendipity</a>. Perhaps protein Y is a novel, interesting one &#8211; and its accidental purification will constitute a good result &#8211; it may answer a different set of questions. Or perhaps one isolates protein X with another protein Y &#8211; and the fact that they stick together reveals that they interact. The proteins that nobody wants to accidentally purify are the ones that we know very well already: ending up with one of those in the pure protein tube is tantamount to discovering that one&#8217;s blind date is an old friend. And of course the more abundant and common a protein is &#8211; the higher the likelihood that it will be the protein Y that the biologist purifies without wanting to. One such very abundant protein is one of the very first enzymes studied, Glyceraldehyde 3-phosphate dehydrogenase aka <a href="https://en.wikipedia.org/wiki/Glyceraldehyde_3-phosphate_dehydrogenase">GAPDH</a>. It is so necessary to cells across all animal kingdoms that it is found in large amounts in bacteria, fungi and animals; its stability makes it a very successful contaminant, and one that has been serendipitously isolated in many a study (see for example <a href="https://doi.org/10.1107/S0907444903041441">[1]</a>, <a href="doi: 10.1074/jbc.M801401200">[2]</a>, <a href="doi: 10.1074/jbc.M113.466870">[3]</a>). The repeated unexpected isolation of GAPDH from protein samples in which biologists did not expect it to be present in the first place has actually revealed that the protein moonlights in multiple roles and very diverse cellular and extracellular environments. Or looking at it from a different angle, a promiscuous protein with a track record for surprise, and one that may well end up turning up on one&#8217;s blind date.</p>
<div id="attachment_4547" style="width: 310px" class="wp-caption alignleft"><img aria-describedby="caption-attachment-4547" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/CSUs-300x220.png" alt="GAPDH Cys-S-sulfonated" class="size-medium wp-image-4547" width="300" height="220" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/CSUs-300x220.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/CSUs.png 661w" sizes="(max-width: 300px) 100vw, 300px" /><p id="caption-attachment-4547" class="wp-caption-text">Structures of human GAPDH carrying S-sulfonic acid on the catalytic Cys152. Eight different conformations of the Cys152 S-sulfonic acid (from as many monomers in the asymmetric unit of the crystal) are superposed and represented in sticks (cyan: C atoms; yellow: S atoms; red: O atoms).</p></div>
<p>We have now joined the group of biologists stumbling on the disappointing discovery of GAPDH in their pure protein tube. In the course of a research effort aimed at the purification of an unrelated enzyme from a fungus, <a href="https://doi.org/10.5281/zenodo.3760776">we purified, crystallised and determined crystal structures of human GAPDH</a>. The protein made it into our sample from the human cells we were using to purify the protein wanted. At the time of this writing, the <a href="https://www.rcsb.org/">Protein Data Bank</a> (PDB) contains more than 142 GAPDH entries, eleven of which are of human GAPDH isoforms &#8211; so these structures are not exactly a big deal. On the other hand, our crystals turned out to contain GAPDH protein that is chemically modified and bears a Cys-S-sulfonic acid on its catalytic Cystein residue. Although GAPDH Cys-S-sulfonation is known to occur and has already been detected and described, one of these novel human GAPDH crystal structures is the highest resolution human GAPDH structure to date &#8211; and the first Cys-S-sulfonated human GAPDH  structure to be determined. Sometimes the old friend one ends up in a blind date with is not as familiar as one thought &#8211; and may still make for an interesting night out after all.</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/a-blind-date-with-an-old-friend-crystal-structures-of-cys-s-sulfonic-acid-gapdh/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>UGGT:  a fly-paper glycoprotein catching troublesome flies?</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/uggt-is-the-misfolding-glycoprotein-checkpoint-like-misfolded-glycoprotein-fly-paper/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/uggt-is-the-misfolding-glycoprotein-checkpoint-like-misfolded-glycoprotein-fly-paper/#respond</comments>
		
		<dc:creator><![CDATA[Pietro Roversi]]></dc:creator>
		<pubDate>Sun, 19 Jan 2020 15:04:40 +0000</pubDate>
				<category><![CDATA[Open Secretion Rescue]]></category>
		<category><![CDATA[Pietro Roversi]]></category>
		<category><![CDATA[Pietro Roversi (Univ. of Leicester)]]></category>
		<category><![CDATA[Rare disease]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=4013</guid>

					<description><![CDATA[&#160; About one quarter of the Eukaryotic genome encodes glycoproteins that are made in the Endoplasmic Reticulum (ER), and travel to special destinations such as the extracellular environment, the cell membrane or other organelles like the lysosomes. We can think of the ER as a nursery where baby glycoproteins quickly grow to attain their adult <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/uggt-is-the-misfolding-glycoprotein-checkpoint-like-misfolded-glycoprotein-fly-paper/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p>&nbsp;</p>
<div id="attachment_4017" style="width: 255px" class="wp-caption alignleft"><img aria-describedby="caption-attachment-4017" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/01/FlyPaper-245x300.png" alt="" class="size-medium wp-image-4017" width="245" height="300" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/01/FlyPaper-245x300.png 245w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/01/FlyPaper-768x940.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/01/FlyPaper-837x1024.png 837w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/01/FlyPaper.png 1500w" sizes="(max-width: 245px) 100vw, 245px" /><p id="caption-attachment-4017" class="wp-caption-text">Mis-folded glycoproteins are caught by UGGT and cannot leave the ER.</p></div>
<p>About one quarter of the Eukaryotic genome encodes <a href="https://en.wikipedia.org/wiki/Glycoprotein">glycoprotein</a>s that are made in the <a href="https://en.wikipedia.org/wiki/Endoplasmic_reticulum">Endoplasmic Reticulum</a> (ER), and travel to special destinations such as the extracellular environment, the cell membrane or other organelles like the lysosomes. We can think of the ER as a nursery where baby glycoproteins quickly grow to attain their adult shape and size, in conditions that closely mimic the ones they evolved to find themselves in.</p>
<p>But how is it decided that a glycoprotein is fully and perfectly formed and can leave the ER? As it turns out, the Eukaryotic cell has evolved a formidable glycoprotein folding quality control (ERQC) system, which ensures that glycoproteins that are mis-folded &#8211; i.e. are still forming &#8211; cannot prematurely leave the ER. The checkpoint of this quality control system is the enzyme that detects a defects in a folding glycoprotein and flags it for ER retention: UDP-glucose Glycoprotein Glucosyl-Transferase (<a href="https://en.wikipedia.org/wiki/UGGT">UGGT</a>).</p>
<p>Thinking of the variety of sizes and shapes of the thousands of <a href="https://en.wikipedia.org/wiki/Secretory_protein">secretory proteins</a> that travel from the ER down the secretory pathway, a big question comes to mind: how can UGGT &#8211; a single enzyme &#8211; survey folding of so many and such different glycoprotein clients? It has been hypothesized that UGGT recognizes exposed &#8220;sticky&#8221; <a href="https://en.wikipedia.org/wiki/Amino_acid">amino acid</a> residues on the surface of a defective glycoprotein, but nobody has yet discovered how the recognition of that &#8220;stickiness&#8221; happens.</p>
<p>In the process of carrying out a control experiment, we have now obtained serendipitous data suggesting that UGGT itself may possess a &#8220;sticky&#8221; mis-folded patch, and that it may therefore catch glycoproteins that carry a folding defect via a &#8220;mis-fold:mis-fold&#8221; molecular recognition event. We prepared a special version of recombinant UGGT, treating mammalian cells with the mannosidase inhibitor <a href="https://en.wikipedia.org/wiki/Kifunensine">kifunensine</a>: this was meant to produce a UGGT molecule (which we call UGGT<sub>Kif</sub> ) bearing glycans identical to the ones carried by all glycoproteins in the ER (including UGGT&#8217;s substrates). Surprisingly, <a href="https://zenodo.org/record/3608191#.XiR76RfgrNA">the UGGT<sub>Kif </sub>crystal structure</a> revealed that one of the domains of the protein is mis-folded!</p>
<p>This observation &#8211; together with <a href="http://10.1016/j.bbrc.2015.04.105">published evidence</a> that UGGT bears de-mannosylated glycans (a hallmark of defective glycoproteins flagged for <a href="https://en.wikipedia.org/wiki/Endoplasmic-reticulum-associated_protein_degradation">ER associated degradation</a>) &#8211; now begs the question whether UGGT has evolved to maintain a mis-folded domain in order to catch mis-folded glycoproteins. Clearly, a physiologically mis-folded UGGT would be steadily degraded in the ER, so if nature has come up with this solution to the problem of mis-fold recognition &#8211; Eukaryotic cells would need to afford quick catabolism of UGGT and the costs associated with constant replenishment of ER UGGT levels.</p>
<p>In summary, when it comes to glycoproteins, <a href="https://doi.org/10.1101/2019.12.25.888438">does it take one to know one</a>? We are now pursuing experiments to test the hypothesis that UGGT be like mis-folded glycoprotein fly-paper, catching mis-folded glycoprotein flies. Watch this space!</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/uggt-is-the-misfolding-glycoprotein-checkpoint-like-misfolded-glycoprotein-fly-paper/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>Strippers from outer space: recombinant PNGase F.</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/strippers-from-outer-space-recombinant-pngase-f/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/strippers-from-outer-space-recombinant-pngase-f/#comments</comments>
		
		<dc:creator><![CDATA[Pietro Roversi]]></dc:creator>
		<pubDate>Sun, 30 Jun 2019 18:41:29 +0000</pubDate>
				<category><![CDATA[Open Secretion Rescue]]></category>
		<category><![CDATA[Pietro Roversi]]></category>
		<category><![CDATA[Pietro Roversi (Univ. of Leicester)]]></category>
		<category><![CDATA[Rare disease]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=3041</guid>

					<description><![CDATA[Nature has evolved molecules called glycoproteins that are proteins carrying sugar chains (&#8220;glycans&#8221;). Glycans can make proteins more soluble, they can shield them from the environment, they can be used as calling cards to enable protein-protein recognition; in Eukaryotes, a glucose-on/glucose-off quality control system uses glycans to monitor glycoproteins folding. In general, glycans contribute to <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/strippers-from-outer-space-recombinant-pngase-f/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<div id="attachment_3042" style="width: 239px" class="wp-caption alignnone"><img aria-describedby="caption-attachment-3042" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/06/mir.jpg" alt="" class="size-full wp-image-3042" width="229" height="220" /><p id="caption-attachment-3042" class="wp-caption-text">The enzyme PNGase F comes from a bacterium first isolated on the MIR space station.</p></div>
<p>Nature has evolved molecules called glycoproteins that are proteins carrying sugar chains (&#8220;glycans&#8221;). Glycans can make proteins more soluble, they can shield them from the environment, they can be used as calling cards to enable protein-protein recognition; in Eukaryotes, a glucose-on/glucose-off quality control system uses glycans to monitor glycoproteins folding.</p>
<p>In general, glycans contribute to expanding the repertoire of protein shapes and sizes that genomes code for, and sugars on proteins are a great asset to nature. Protein chemists on the other hand are not always happy with the glycans on the proteins they produce. This is especially true of molecular biologists forcing cells to synthesise custom glycoproteins: these &#8216;recombinant glycoproteins&#8217; may end up with glycans that differ from naturally occurring ones.</p>
<p>As it turns out, the ability of stripping sugars from glycoproteins has helped a number of species to survive, and enzymes that can cleave glycans from glycoproteins (&#8220;deglycosylases&#8221;) are encoded in many a genome. By offering such enzymes, nature comes therefore to the succour of molecular biologists and protein chemists who are unhappy with the glycans on the proteins they handle: by treating a glycoprotein with a deglycosylase, it is possible to get rid of the sugars and obtain a sugar-free protein.</p>
<p>In our case, crystals of a glycoprotein we are studying grew a flimsy lattice formed by glycan-glycan contacts, making the resulting glycoprotein structure all fuzzy. We are hoping that one way of improving on these crystals and obtaining a better structure would be to strip the glycoprotein of the glycans altogether, perhaps enabling the growth of different crystals, kept together by more stable, less mobile protein-protein contacts.</p>
<p>For this reason, we decided to treat our glycoprotein with a special deglycosylase called PNGase F. The enzyme belongs to a rare bacterium called <a href="https://en.wikipedia.org/wiki/Elizabethkingia_miricola"><em>Elizabethkingia miricola</em>, </a>first isolated from condensation water in the <a href="https://en.wikipedia.org/wiki/Mir">Space Station Mir</a>. But a small vial of the enzyme can cost up to £1000 when purchased from commercial sources! We therefore endeavoured to produce our own PNGase F, using <em>E.coli</em> bacteria transformed with the <em>Elizabethkingia miricola</em> PNGase F gene.</p>
<p>After ten days of work, <a href="https://zenodo.org/record/3263902#.XRj3jXt7l2Y">we obtained our stripper from outer space</a>, and were able to confirm its activity. Watch this less-than-outer space for the next experiments in which we shall try and grow our crystals again, this time with the protein without glycans!</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/strippers-from-outer-space-recombinant-pngase-f/feed/</wfw:commentRss>
			<slash:comments>1</slash:comments>
		
		
			</item>
		<item>
		<title>Crystal structures of the CtUGGT double mutant S180C:T742C</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/crystal-structures-of-the-ctuggt-double-mutant-s180ct742c/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/crystal-structures-of-the-ctuggt-double-mutant-s180ct742c/#comments</comments>
		
		<dc:creator><![CDATA[Pietro Roversi]]></dc:creator>
		<pubDate>Wed, 15 Aug 2018 14:59:01 +0000</pubDate>
				<category><![CDATA[Open Secretion Rescue]]></category>
		<category><![CDATA[Pietro Roversi]]></category>
		<category><![CDATA[Pietro Roversi (Univ. of Leicester)]]></category>
		<category><![CDATA[Rare disease]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=1705</guid>

					<description><![CDATA[The inter-domain conformational mobility of UGGT, the eukaryotic secretion glycoprotein folding checkpoint, is likely key to its ability of recognising and re-glucosylating a vast number of different misfolded glycoproteins, bearing N-linked glycan(s) at various different distances from the site(s) of misfold. Ongoing Molecular Dynamics simulations by Juan Blanco, Carlos Modenutti, and Marcelo Martí in Buenos <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/crystal-structures-of-the-ctuggt-double-mutant-s180ct742c/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p>The inter-domain conformational mobility of UGGT, the eukaryotic secretion glycoprotein folding checkpoint, is likely key to its ability of recognising and re-glucosylating a vast number of different misfolded glycoproteins, bearing N-linked glycan(s) at various different distances from the site(s) of <a href="https://www.ncbi.nlm.nih.gov/pubmed/28739903">misfold.<br />
</a></p>
<p>Ongoing Molecular Dynamics simulations by <a href="http://www.iquibicen.fcen.uba.ar/investigador/marcelo-marti/">Juan Blanco, Carlos Modenutti, and Marcelo Martí</a> in Buenos Aires have highlighted a motion called clamping, which brings the TRXL1 and the TRXL3 domains of UGGT close together. To test the importance of this clamping motion we have engineered a double Cys mutant of <em>Chaetomium thermophilum</em> UGGT (aka <em>Ct</em>UGGT) which locks the clamping motions shut: S180C:T742C. This mutant is called <em>Ct</em>UGGT-N for short.</p>
<p>The cloning, protein expression and purification were successful. Mass spectrometry confirmed the existence of the desired disulphide bond. We have managed to crystallise the <em>Ct</em>UGGT-N protein in two distinct crystal forms and we have determined <a href="https://zenodo.org/record/1345671#.W3Q61X4nYfw">the structures</a>.</p>
<p>In the trigonal form, the orientation of TRXL2 with respect to the rest of the protein is the most opened ever observed so far (a rotation of 60 degrees away from the TRXL2 in <a href="https://www.rcsb.org/structure/5NV4">PDB ID 5NV4</a>). The orthorhombic form proves that the clamped shut structure (<em>Ct</em>UGGT-N P2<sub>1</sub>2<sub>1</sub>2<sub>1</sub>) is compatible with the bent shut structure (<em>Ct</em>UGGT<img class="size-medium wp-image-1728 alignleft" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/08/Slide4-300x225.jpg" alt="" width="300" height="225" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/08/Slide4-300x225.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/08/Slide4-768x576.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/08/Slide4-350x263.jpg 350w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/08/Slide4-780x586.jpg 780w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/08/Slide4-250x188.jpg 250w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/08/Slide4.jpg 841w" sizes="(max-width: 300px) 100vw, 300px" />-H, P2<sub>1</sub>2<sub>1</sub>2<sub>1</sub>, PDB ID 5NV4) <em>i.e.</em> both clamping and bending movements can close at the same time. A quadruple <em>Ct</em>UGGT mutant, containing both D611C:G1050C and S180C:T742C, should be feasible and its human UGGT1 equivalent could provide the best rigidification of the human structure conceivable so far &#8211; in order to aid crystallisation and obtain the so far elusive human protein crystal structure.</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/crystal-structures-of-the-ctuggt-double-mutant-s180ct742c/feed/</wfw:commentRss>
			<slash:comments>1</slash:comments>
		
		
			</item>
	</channel>
</rss>
