<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>Magdalena Szewczyk &#8211; openlabnotebooks.org</title>
	<atom:link href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/magdalena-szewczyk/feed/" rel="self" type="application/rss+xml" />
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<description>A growing team of groundbreaking scientists around the world are now sharing their lab notebooks online</description>
	<lastBuildDate>Fri, 05 Jun 2020 19:03:24 +0000</lastBuildDate>
	<language>en-GB</language>
	<sy:updatePeriod>
	hourly	</sy:updatePeriod>
	<sy:updateFrequency>
	1	</sy:updateFrequency>
	<generator>https://wordpress.org/?v=5.4.2</generator>

<image>
	<url>https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/cropped-lab-jar02-32x32.png</url>
	<title>Magdalena Szewczyk &#8211; openlabnotebooks.org</title>
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<width>32</width>
	<height>32</height>
</image> 
	<item>
		<title>STIP1(HOP) and STUB1(CHIP) interaction with HSPA8 &#8211; cellular assay</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/stip1hop-and-stub1chip-interaction-with-hspa8-cellular-assay/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/stip1hop-and-stub1chip-interaction-with-hspa8-cellular-assay/#respond</comments>
		
		<dc:creator><![CDATA[Magdalena Szewczyk]]></dc:creator>
		<pubDate>Fri, 05 Jun 2020 19:02:55 +0000</pubDate>
				<category><![CDATA[Cell assay reports]]></category>
		<category><![CDATA[E3 ligases]]></category>
		<category><![CDATA[Magdalena Szewczyk]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=4866</guid>

					<description><![CDATA[Background HSP70 heat-shock proteins family, including constitutively expressed HSPA8 variant, facilitate correct folding of client proteins and degradation of unfolded clients. To achieve folding or degradation of client proteins, HSP70 cooperates with co-chaperones, which mostly bind to the highly conserved sequence EEVD in the C-terminal domain of HSP70 via their tetratricopeptide repeat (TPR) domain (PMID: <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/stip1hop-and-stub1chip-interaction-with-hspa8-cellular-assay/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p><strong>Background</strong></p>
<p>HSP70 heat-shock proteins family, including constitutively expressed HSPA8 variant, facilitate correct folding of client proteins and degradation of unfolded clients. To achieve folding or degradation of client proteins, HSP70 cooperates with co-chaperones, which mostly bind to the highly conserved sequence EEVD in the C-terminal domain of HSP70 via their tetratricopeptide repeat (TPR) domain (PMID: 26565746). STIP1(HOP) and ubiquitin ligase STUB1(CHIP) and are TPR-containing co-chaperones that assist with protein folding or degradation, respectively (PIMID:22824801). It was shown previously that mutations within TPR domains of STIP1 (K8A) and STUB1 (K30A) abolished their interaction with HSPA8 (PIMID: 23865999, 26565746).</p>
<p><strong>Assay validation</strong></p>
<p>Assay measuring the interaction of HSPA8 to STIP1 and STUB1 co-chaperons was developed using NanoBRET technology &#8211; a proximity-based assay that can detect protein interactions by measuring energy transfer from a bioluminescent protein donor to a fluorescent protein acceptor. At the beginning we determined the optimal tag orientation and donor to acceptor ratio of donor constructs: N or C-terminally NanoLuc® tagged STIP1/STIP1 K8A mutant or STUB1/STUB1 K30A and acceptor construct: C-terminally HaloTag®Fusion tagged HSPA8. The best results were obtained with C-terminally tagged donor constructs and 1:9 donor to acceptor ratio. Consistent with previous findings the interaction (NanoBRET ratio) of HSPA8 and mutated co-chaperons was significantly lower compared to wild type proteins (Fig.1).</p>
<p><strong> <img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/stipstub-300x170.jpg" alt="" width="671" height="380" class="wp-image-4867 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/stipstub-300x170.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/stipstub-768x434.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/stipstub.jpg 945w" sizes="(max-width: 671px) 100vw, 671px" /></strong></p>
<p><strong>Fig.1. HSPA8 interaction with STIP1 (HOP) and STUB1 (CHIP) co-chaperones. </strong>HEK293T cells were co-transfected with Nanoluc (NL)-tagged STIP1 (WT or K8A mutant) or NL-tagged STUB1 (WT or K30A mutant) and Halo-tagged (HT) HSPA8 for 24 h. The interaction was measured using NanoBRET assay. The results are MEAN+/-SD of 3 technical replicates.</p>
<p>Experimental details are posted on <a href="https://zenodo.org/record/3879514#.XtqWgzpKg2w">Zenodo</a>.</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/stip1hop-and-stub1chip-interaction-with-hspa8-cellular-assay/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>PRMT7 cellular assay</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/prmt7-cellular-assay/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/prmt7-cellular-assay/#respond</comments>
		
		<dc:creator><![CDATA[Magdalena Szewczyk]]></dc:creator>
		<pubDate>Fri, 05 Jun 2020 18:44:13 +0000</pubDate>
				<category><![CDATA[Cell assay reports]]></category>
		<category><![CDATA[Magdalena Szewczyk]]></category>
		<category><![CDATA[PRMTs]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=4859</guid>

					<description><![CDATA[Background PRMT7 is a type III protein arginine methyltransferase that monomethylates arginine residues mainly within the RXR motif (Fig.1) (PIMID: 24247247). PRMT7 was shown to play an important role in muscle physiology, stem cell biology, and cellular stress response (PIMID: 32409666, 27225728, 26854227).  Fig.1. Mammalian PRMTs. There are 9 members and 3 types of the <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/prmt7-cellular-assay/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p><strong>Background</strong></p>
<p>PRMT7 is a type III protein arginine methyltransferase that monomethylates arginine residues mainly within the RXR motif (Fig.1) (PIMID: 24247247). PRMT7 was shown to play an important role in muscle physiology, stem cell biology, and cellular stress response (PIMID: 32409666, 27225728, 26854227).</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1-300x119.png" alt="" width="643" height="255" class="wp-image-4626 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1-300x119.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1-1024x405.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1-768x304.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1.png 1214w" sizes="(max-width: 643px) 100vw, 643px" /></p>
<p><strong> </strong><strong>Fig.1. Mammalian PRMTs. </strong>There are 9 members and 3 types of the PRMT family. Type I, II, and III PRMTs catalyze the formation of monomethyl arginines (Rme1). Type I PRMTs catalyze asymmetric arginine methylation (Rme2a), while type II PRMTs form symmetric arginine methylation (Rme2s). Type III PRMT can only monomethylate arginine residues. Modified from (PMID:29378138).</p>
<p><strong> </strong><strong>Assay validation</strong></p>
<p><strong> </strong>It has been reported previously that PRMT7 is the main contributor to HSP70 R469 monomethylation in cells (PIMID:32409666). We found that<em> PRMT7</em> knockout or knockdown reduces HSP70 monomethylation in various cell lines (Fig. 2). The assay was further validated with the PRMT7 selective chemical probe SGC3027 (PIMID: 32409666). SGC3027 but not its inactive control SGC3027N decreased PRMT7 dependent HSP70 monomethylation in C2C12 cells in a dose-dependent manner (Fig.3). The Z factor for the assay equals 0.67.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/prmt7-1-300x94.png" alt="" width="744" height="233" class="wp-image-4861 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/prmt7-1-300x94.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/prmt7-1-768x240.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/prmt7-1.png 936w" sizes="(max-width: 744px) 100vw, 744px" /></p>
<p><strong>Fig. 2<em>. PRMT7</em> (P7) knockout (KO) or knockdown (KD) reduces HSP70 methylation in various cell lines.</strong> Western blot analysis of HSP70 monomethylation. 94A, 21B—HCT116 CRISPR <em>PRMT7</em> KO clones; P parental C2C12; C-C2C12 expressing control guide RNA; 32 C2C12 CRISPR clone expressing PRMT7 catalytic mutant (delY35,A35S); 4,57,74—C2C12 CRISPR <em>Prmt7</em> KO clones. <em>PRMT7</em> was knocked down in HEK293T and MCF7 cells using siRNA. Con-control, P7KD-<em>PRMT7</em> knockdown. Modified from PIMID:32409666. (Shawna Organ, Magdalena Szewczyk)</p>
<p>&nbsp;</p>
<p><strong> <img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/prmt7-300x137.png" alt="" width="740" height="338" class="wp-image-4860 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/prmt7-300x137.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/prmt7-768x351.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/prmt7.png 988w" sizes="(max-width: 740px) 100vw, 740px" /></strong></p>
<p><strong>Fig.3.</strong> <strong>SGC3027 but not negative control compound SGC3027N inhibits PRMT7-dependent HSP70 monomethylation in C2C12 cells in a dose-dependent manner.</strong> Cells were treated with the compound for 2 days. <strong>A.</strong> A representative blot for SGC3027 and SGC3027N effects on HSP70 methylation. Rme1—arginine monomethylation.<strong>B</strong> Quantification of SGC3027 and SGC3027N effects on HSP70 monomethylation in C2C12 cells. The graphs represent non-linear fits of Rme1 signal intensities normalized to intensities of HSP70. SGC3027: <em>n</em> = 11, four separate experiments, IC<sub>50</sub> = 2.4 ± 0.1 µM; SGC3027N: <em>n</em> = 4 technical replicates, IC<sub>50</sub> &gt; 40 µM (mean ± SEM). Modified from PIMID:32409666. (Magdalena Szewczyk)</p>
<p>For experimental details please go to <a href="https://zenodo.org/record/3879397#.XtqPzzpKg2w">Zenodo</a>.</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/prmt7-cellular-assay/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>NSD3(PWWP1) and histone H3 interaction cellular assay</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/nsd3pwwp1-and-histone-h3-interaction-cellular-assay/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/nsd3pwwp1-and-histone-h3-interaction-cellular-assay/#respond</comments>
		
		<dc:creator><![CDATA[Magdalena Szewczyk]]></dc:creator>
		<pubDate>Fri, 29 May 2020 17:13:56 +0000</pubDate>
				<category><![CDATA[Cell assay reports]]></category>
		<category><![CDATA[Magdalena Szewczyk]]></category>
		<category><![CDATA[NSD3]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=4807</guid>

					<description><![CDATA[Background NSD3 is a member of the NSD (nuclear receptor SET domain- containing) family of H3K36 methyltransferases. It has two isoforms: long -1437 amino acid and short &#8211; 645 amino acids, which are derived from alternative splicing of exon 10.  In contrast to long isoform, the short isoform lacks the methyltransferase domain and it contains <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/nsd3pwwp1-and-histone-h3-interaction-cellular-assay/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p><strong>Background</strong></p>
<p>NSD3 is a member of the NSD (nuclear receptor SET domain- containing) family of H3K36 methyltransferases. It has two isoforms: long -1437 amino acid and short &#8211; 645 amino acids, which are derived from alternative splicing of exon 10.  In contrast to long isoform, the short isoform lacks the methyltransferase domain and it contains only one histone methyl-lysine reader domain (PWWP1).  The overexpression of both NSD3 isoforms is capable of oncogenic transformation indicating that the oncogenic function is not related to methyltransferase activity. Short NSD3 isoform was shown to sustain AML leukemia by linking BRD4 to the CHD8 chromatin remodeler, through the PWWP chromatin reader domain.  (PMID: 20940404, 23269674, 25942451, 26626481)</p>
<p><strong>Assay validation</strong></p>
<p>Assay measuring the interaction of NSD3-PWWP1 to histone H3 was developed using NanoBRET technology &#8211; a proximity-based assay that can detect protein interactions by measuring energy transfer from a bioluminescent protein donor to a fluorescent protein acceptor. The assay was optimized and validated using NSD3 and H3 antagonist BI-9466 and W284/E318K binding deficient mutant. The best results were obtained with U2Os cells transfected with C-terminally NanoLuc tagged NSD3-PWWP1 and H3 at 1:10 ratio. As shown in Fig 1. chemical probe antagonist (BI-9466) but not its negative control (BI-9321) selectively decreased the BRET signal only in WT NSD3-PWWP1. No effect was observed on binding deficient W284A/E318K mutant and closely related NSD2-PWWP1 domain. BI-9466 decreased NSD3-PWWP1 and H3 interaction in a dose-dependent manner (Fig2). (PIMID: 31285596)</p>
<p><u><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/nsd3-1-300x237.png" alt="" width="537" height="424" class="wp-image-4810 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/nsd3-1-300x237.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/nsd3-1-768x607.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/nsd3-1.png 801w" sizes="(max-width: 537px) 100vw, 537px" /></u></p>
<p><strong> </strong><strong>Fig.1. BI-9466 antagonizes interaction between histone H3 and WT NSD3-PWWP1 but not NSD3-PWWP1(W284A/E318K) mutant or NSD2-PWWP1 in NanoBRET assay. </strong>U2Os cells were co-transfected with C-terminally HaloTag Fusion-tagged histone H3 and C-terminally NanoLuc-tagged NSD3-PWWP1, NSD3-PWWP1(W284A/E318K), or NSD2-PWWP1 and treated with 10 µM BI-9321 (antagonist) or BI-9466 (negative control) for 24 h. Shown is the mean ± SD of two biological replicates, run in quadruplicate. (David Dilworth, Magdalena Szewczyk)</p>
<p>&nbsp;</p>
<p><strong> <img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/nsd3-2-300x215.png" alt="" width="546" height="391" class="wp-image-4809 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/nsd3-2-300x215.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/nsd3-2-768x551.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/nsd3-2.png 870w" sizes="(max-width: 546px) 100vw, 546px" /></strong></p>
<p><strong>Fig.2. BI-9466 but not negative control compound BI-9321 decreases the interaction of NSD3-PWWP1 and histone H3 in a dose-dependent manner. </strong>NanoBRET based on the interaction of C-terminally NanoLuc-tagged NSD3-PWWP1 with C-terminally HaloTag Fusion tagged histone H3 with increasing concentrations of either BI-9321 (blue) or control compound BI-9466 (green). Shown is a representative dose-response curve. IC50 was calculated from the mean ± SD of three independent replicates. (David Dilworth, Magdalena Szewczyk)</p>
<p>For experimental details go to <a href="https://zenodo.org/record/3865777#.XtFCUzpKg2w">Zenodo</a>.</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/nsd3pwwp1-and-histone-h3-interaction-cellular-assay/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>NSD2(PWWP1) and histone H3 interaction cellular assay</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/nsd2pwwp1-and-histone-h3-interaction-cellular-assay/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/nsd2pwwp1-and-histone-h3-interaction-cellular-assay/#respond</comments>
		
		<dc:creator><![CDATA[Magdalena Szewczyk]]></dc:creator>
		<pubDate>Fri, 29 May 2020 16:26:47 +0000</pubDate>
				<category><![CDATA[Cell assay reports]]></category>
		<category><![CDATA[Magdalena Szewczyk]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=4800</guid>

					<description><![CDATA[Background NSD2 is a member of the NSD (nuclear receptor SET domain- containing) family of H3K36 methyltransferases which contain a conserved catalytic SET domain and several chromatin-associated domains comprising four PHD fingers, two PWWP domains, and an HMG box. NSD2 is implicated in several diseases including Wolf Hirschhorn syndrome and its overexpression or hyperactivity may <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/nsd2pwwp1-and-histone-h3-interaction-cellular-assay/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p><strong>Background</strong></p>
<p>NSD2 is a member of the NSD (nuclear receptor SET domain- containing) family of H3K36 methyltransferases which contain a conserved catalytic SET domain and several chromatin-associated domains comprising four PHD fingers, two PWWP domains, and an HMG box. NSD2 is implicated in several diseases including Wolf Hirschhorn syndrome and its overexpression or hyperactivity may cause cancer. The first PWWP domain of NSD2 preferentially binds dimethylated H3K36, which stabilizes NDS2 at chromatin and regulates NSD2 occupancy at its gene targets. PWWP1 does not directly impact NSD2 catalytic activity, however, the increase of cell proliferation caused by NSD2 overexpression is abolished by mutations in the PWWP1 domain affecting its binding to H3K36me2 (PMID:23823660, 26912663, 28054944).</p>
<p><strong>Assay validation</strong></p>
<p>Assay measuring the interaction of NSD2 to histone H3 was developed using NanoBRET technology &#8211; a proximity-based assay that can detect protein interactions by measuring energy transfer from a bioluminescent protein donor to a fluorescent protein acceptor. The assay was optimized and validated using NSD2 and H3 antagonist UNC6034 and NSD2-F266A binding deficient mutant. The best results were obtained with U2Os cells transfected with C-terminally NanoLuc-tagged NSD2-PWWP1 and H3 at 1:10 ratio. As shown in Fig 1. chemical probe antagonist (UNC6934) but not its negative control (UNC7145) selectively decreased the BRET signal only in WT NSD2-PWWP1. No effect was observed on F266A binding deficient mutant and closely related NSD3-PWWP1 domain. UNC6934 decreased NSD3-PWWP1 and H3 interaction in a dose-dependent manner (Fig2).</p>
<p><strong> <img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/nsd2-2-300x166.png" alt="" width="450" height="249" class="wp-image-4801 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/nsd2-2-300x166.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/nsd2-2-1024x567.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/nsd2-2-768x425.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/nsd2-2.png 1218w" sizes="(max-width: 450px) 100vw, 450px" /></strong></p>
<p><strong> </strong></p>
<p><strong>Fig.1. UNC6934 antagonizes interaction between histone H3 and WT NSD2-PWWP1 but not NSD2-PWWP1(F266A) mutant or NSD3-PWWP1 in NanoBRET assay. </strong>U2Os cells were co-transfected with C-terminally HaloTag Fusion-tagged histone H3 and C-terminally NanoLuc-tagged NSD2-PWWP1, NSD2-PWWP1(F266A), or NSD3-PWWP1 and treated with 10 µM compounds for 24 h. Shown is the mean ± SD of three biological replicates, run in triplicates. (David Dilworth, Magdalena Szewczyk)</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/nsd2-1-300x174.png" alt="" width="429" height="249" class="wp-image-4802 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/nsd2-1-300x174.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/nsd2-1-768x445.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/nsd2-1.png 920w" sizes="(max-width: 429px) 100vw, 429px" /></p>
<p><strong>Fig.2. UNC6934 but not negative control compound UNC7145 decrease NSD2-PWWP1 interaction with H3 in a dose-dependent manner. </strong>U2Os cells were co-transfected with C-terminally NanoLuc-tagged NSD2-PWWP1 or NSD2-PWWP1(F266A) and C-terminally HaloTag Fusion-tagged histone H3 and treated with compounds for 24 h. Shown is a representative dose-response curve. IC50 was calculated from the mean ± SD of three biological replicates, run in triplicates. (David Dilworth)</p>
<p>The experimental details are posted on <a href="https://zenodo.org/record/3865717#.XtE3STpKg2w">Zenodo</a>.</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/nsd2pwwp1-and-histone-h3-interaction-cellular-assay/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>p300/CBP cellular assay</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/p300-cbp-cellular-assay/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/p300-cbp-cellular-assay/#respond</comments>
		
		<dc:creator><![CDATA[Magdalena Szewczyk]]></dc:creator>
		<pubDate>Thu, 28 May 2020 16:23:26 +0000</pubDate>
				<category><![CDATA[Cell assay reports]]></category>
		<category><![CDATA[Magdalena Szewczyk]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<category><![CDATA[USPs]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=4793</guid>

					<description><![CDATA[Background Transcriptional coactivators p300 and CBP catalyze the acetylation of lysine residues of a large variety of histone and non-histone proteins (PMID:28426192). p300 and CBP are considered functional homologous proteins that share around 86% amino acid sequence identity in the HAT domain (PIMID:7870179).  The upregulation of p300/CBP has been associated with various cancers, indicating an important role in <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/p300-cbp-cellular-assay/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p><strong>Background</strong></p>
<p>Transcriptional coactivators p300 and CBP catalyze the acetylation of lysine residues of a large variety of histone and non-histone proteins (PMID:28426192). p300 and CBP are considered functional homologous proteins that share around 86% amino acid sequence identity in the HAT domain (PIMID:7870179).  The upregulation of p300/CBP has been associated with various cancers, indicating an important role in tumorigenesis (PIMID:27881443).</p>
<p><strong>Assay validation</strong></p>
<p>P300/CBP has been reported previously to acetylate p53 at lysine 320 and histone H3 at lysine 27 (PIMID: 25545885, 29804834). Based on the findings, we developed p300 cellular assays measuring p53 acetylation. As shown in Fig.1 overexpression of p300 but not its catalytic mutant (Y1467F) increased p53 acetylation levels. We also developed p300/CBP cellular assay measuring H3K27 acetylation.  Knocking down of p300/CBP resulted in a global decrease of H3K27 acetylation levels (Fig 2A). The H3K27 assay was further validated with p300/CBP selective inhibitor A-485 (PIMID: 29769713). A-485 decreased H3K27 acetylation levels in a dose-dependent manner in HFF cells (Fig.2B). Similar results were obtained in other cell lines (data not shown). The z factor for the assay equals 0.7.</p>
<p><strong> </strong></p>
<p><strong> <img class="wp-image-4795 alignleft" height="363" width="290" alt="" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p300-1.png" /></strong></p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p><strong>Fig.1.</strong><strong> The wild type but not catalytic mutant of p300 acetylated exogenous p53.</strong>  HEK293T cells were co-transfected with FLAG-tagged p53 and empty vector(control),  p300 wild type, or p300 Y1467F catalytic mutant for 24 h<strong>.</strong> The acetylation levels were analyzed in Western blot. (Taylor Mitchell)</p>
<p><strong> </strong></p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p><strong> <img class="alignnone wp-image-4794" height="248" width="845" alt="" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p300-2-300x88.png" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p300-2-300x88.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p300-2-1024x301.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p300-2-768x226.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p300-2.png 1280w" sizes="(max-width: 845px) 100vw, 845px" /></strong></p>
<p><strong> </strong><strong>Fig.2. The loss of p300/CBP activity results in a decrease of H3K27 acetylation levels. A. </strong>p300/CBP knock-down. HEK293T and MCF7 cells were treated with siRNA targeted for p300/CBP  and control siRNA for 3 days. <strong>B.</strong> p300/CBP inhibition. HFF cells were treated with p300/CBP selective inhibitor A-481 at indicated concentrations for 3 days. H3K27 acetylation levels were determined by Western blot. (Dalia Barsyte, Magdalena Szewczyk)</p>
<p>Experimental details are posted on <a href="https://zenodo.org/record/3862780#.Xs_k6DpKg2w">Zenodo</a>.</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/p300-cbp-cellular-assay/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>RPA2 and RFWD3 interaction cellular assay</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/rpa2-and-rfwd3-interaction-cellular-assay/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/rpa2-and-rfwd3-interaction-cellular-assay/#respond</comments>
		
		<dc:creator><![CDATA[Magdalena Szewczyk]]></dc:creator>
		<pubDate>Thu, 21 May 2020 16:00:40 +0000</pubDate>
				<category><![CDATA[Cell assay reports]]></category>
		<category><![CDATA[E3 ligases]]></category>
		<category><![CDATA[Magdalena Szewczyk]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=4771</guid>

					<description><![CDATA[Background RFWD3 (E3 ligase RING finger and WD repeat domain-containing protein 3) and RPA2 (Replication protein A2) were shown to functionally interact and participate in replication checkpoint control. At stalled replication forks, the RPA complex binds single-stranded DNA to prevent nucleolytic digestions and also to initiate replication checkpoint signaling by recruitment checkpoint proteins. RFWD3 is <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/rpa2-and-rfwd3-interaction-cellular-assay/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p><strong>Background</strong></p>
<p>RFWD3 (E3 ligase RING finger and WD repeat domain-containing protein 3) and RPA2 (Replication protein A2) were shown to functionally interact and participate in replication checkpoint control. At stalled replication forks, the RPA complex binds single-stranded DNA to prevent nucleolytic digestions and also to initiate replication checkpoint signaling by recruitment checkpoint proteins. RFWD3 is recruited to stalled replication forks and interacts directly with RPA2 in response to replication stress. The deletion of the RPA2 binding region on RFWD3 impairs its localization to stalled replication forks (PIMID: 21504906).</p>
<p><strong>Summary</strong></p>
<p>Assay measuring the interaction of RPA2 to RFWD3 was developed using NanoBRET technology &#8211; a proximity-based assay that can detect protein interactions by measuring energy transfer from a bioluminescent protein donor to a fluorescent protein acceptor. We determined the optimal donor to acceptor ratio and levels of expression of 2 donor constructs: C- or N-terminally NanoLuc®(NL)- tagged RFWD3 and acceptor construct: N-terminally HaloTag®Fusion (HT)-tagged RPA2. We compared the results to background NanoBRET signal from HT-RPA2 and NL protein alone. The best results were obtained with N-terminally NL-tagged RFWD3 (0.001 µg/96-well) and N-terminally HT- tagged RPA2 (0.03 µg/96-well) (Fig.1)</p>
<p><strong> </strong></p>
<p><strong> <img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/rpa2-300x152.png" alt="" width="513" height="260" class="wp-image-4772 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/rpa2-300x152.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/rpa2-768x390.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/rpa2.png 962w" sizes="(max-width: 513px) 100vw, 513px" /></strong></p>
<p><strong>Fig 1. RPA2 – RFWD3</strong><strong> nanoBRET assay. </strong>HEK293T cells were plated in 96-well plates and transfected with indicated amounts of C- or N- terminally NL-tagged RFWD3 and N-terminally HT-tagged RPA2. Next day cells were treated +/- fluorescent ligand and 4 h later the luciferase substrate was added, and the signal was read. The results are mean +/- SD of 3 replicates.</p>
<p>Go to <a href="https://zenodo.org/record/3837867#.Xsak7WhKg2w">Zenodo</a> for experimental details.</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/rpa2-and-rfwd3-interaction-cellular-assay/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>SMYD3 cellular assay</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/smyd3-cellular-assay/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/smyd3-cellular-assay/#respond</comments>
		
		<dc:creator><![CDATA[Magdalena Szewczyk]]></dc:creator>
		<pubDate>Thu, 21 May 2020 15:26:30 +0000</pubDate>
				<category><![CDATA[Cell assay reports]]></category>
		<category><![CDATA[Magdalena Szewczyk]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=4766</guid>

					<description><![CDATA[Background SMYD3 (SET and MYND domain containing protein 3) is a member of the SMYD lysine methylase family and plays an important role in lysine methylation of various histone and non-histone proteins such as H2A.Z.1-K101, VEDFR1-K831, MAP3K2-K260, HER2-K175 or AKT1-K14. SMYD3 is overexpressed in numerous human tumors and is a potential therapeutic target and prognostic <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/smyd3-cellular-assay/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p><strong>Background</strong></p>
<p>SMYD3 (SET and MYND domain containing protein 3) is a member of the SMYD lysine methylase family and plays an important role in lysine methylation of various histone and non-histone proteins such as H2A.Z.1-K101, VEDFR1-K831, MAP3K2-K260, HER2-K175 or AKT1-K14. SMYD3 is overexpressed in numerous human tumors and is a potential therapeutic target and prognostic factor in various solid cancers. (PIMID: 31935919).</p>
<p><strong> </strong><strong>Assay validation</strong></p>
<p>SMYD3 has been reported to trimethylate MAP3K2 at lysine 260, which for the first time linked lysine methylation to Ras-driven signaling pathways in cancer (PIMID: 24847881). Based on the findings, we developed SMYD3 cellular assay in which exogenous HA-tagged MAP3K2 is methylated by exogenous SMYD3 in the HeLa cervical cancer cell line (Fig.1). Overexpression of SMYD3 but not its catalytic mutant (ΔNHSCDPN, Fig.1) increased MAP3K2-K260me3 levels (Fig.2). The assay was further validated with SMYD3 selective inhibitors EPZ031686 and BAY-6035. Both inhibitors decreased SMYD3 dependent exogenous MAP3K2-K260 trimethylation in a dose-dependent manner (Fig.3). The Z factor for the assay equals 0.86.</p>
<p><strong> <img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/smyd3-1-300x153.png" alt="" width="682" height="348" class="wp-image-4767 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/smyd3-1-300x153.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/smyd3-1-1024x521.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/smyd3-1-768x391.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/smyd3-1.png 1080w" sizes="(max-width: 682px) 100vw, 682px" /></strong></p>
<p><strong> </strong><strong>Fig.1. Assay schematic. </strong>Modified from (PIMID:20943667).</p>
<p><strong> <img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/smyd3-2-300x190.png" alt="" width="456" height="289" class=" wp-image-4768 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/smyd3-2-300x190.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/smyd3-2.png 622w" sizes="(max-width: 456px) 100vw, 456px" /></strong></p>
<p><strong> </strong><strong>Fig.2.</strong><strong> The wild type but not catalytic mutant of SMYD3 (ΔNHSCDPN) trimethylated MAP3K2-K260.</strong>  HEK293T cells were co-transfected with empty vector (control), SMYD3 wild type or catalytic mutant (ΔSMYD3), and HA-tagged MAP3K2 for 24 h. The methylation levels were analyzed in Western Blot. (Shawna Organ)</p>
<p><strong><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/smyd3-3-300x169.png" alt="" width="619" height="349" class="wp-image-4769 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/smyd3-3-300x169.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/smyd3-3-1024x576.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/smyd3-3-768x432.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/smyd3-3.png 1280w" sizes="(max-width: 619px) 100vw, 619px" /> </strong><strong>Fig.3.</strong><strong> SMYD3 selective inhibitors decreased SMYD3 dependent MAP3K2-K260 trimethylation in a dose-dependent manner in cells.</strong>  HeLa cells were co-transfected with SMYD3 and HA-tagged MAP3K2 and treated with EPZ031686 (<strong>A</strong>) and BAY-6035 (<strong>B</strong>) for 20 h. MAP3K2-K260me3 levels were determined by Western blot. The graphs represent the nonlinear fits of MAP3K2-K260me3 signal intensities normalized to HA(MAP3K2). The results are mean +/- SEM of 2 (A) and 3 (B) technical replicates. (Shawna Organ, Magdalena Szewczyk)</p>
<p>Experimental details are posted on <a href="https://zenodo.org/record/3837707#.XsadE2hKg2w">Zenodo</a>.</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/smyd3-cellular-assay/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>PRMT8 cellular assay</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/prmt8-cellular-assay/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/prmt8-cellular-assay/#respond</comments>
		
		<dc:creator><![CDATA[Magdalena Szewczyk]]></dc:creator>
		<pubDate>Fri, 15 May 2020 19:28:09 +0000</pubDate>
				<category><![CDATA[Cell assay reports]]></category>
		<category><![CDATA[Magdalena Szewczyk]]></category>
		<category><![CDATA[PRMTs]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=4718</guid>

					<description><![CDATA[Background PRMT8 is type I protein arginine methyltransferase expressed mainly in the brain, which mono- and asymmetrically dimethylates histone and nonhistone proteins (Fig.1). It shares over 80% of amino acid sequence homology with PRMT1. At the N-terminus, it harbors a glycine that can be myristoylated which results in its plasma membrane localization (PIMID: 16051612). Fig.1. <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/prmt8-cellular-assay/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p><strong>Background</strong></p>
<p>PRMT8 is type I protein arginine methyltransferase expressed mainly in the brain, which mono- and asymmetrically dimethylates histone and nonhistone proteins (Fig.1). It shares over 80% of amino acid sequence homology with PRMT1. At the N-terminus, it harbors a glycine that can be myristoylated which results in its plasma membrane localization (PIMID: 16051612).</p>
<p><strong><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1-300x119.png" alt="" width="597" height="237" class="wp-image-4626 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1-300x119.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1-1024x405.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1-768x304.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/04/prmtall-1.png 1214w" sizes="(max-width: 597px) 100vw, 597px" /></strong></p>
<p><strong>Fig.1. </strong><strong>Mammalian PRMTs. </strong>There are 9 members and 3 types of the PRMT family. Type I, II, and III PRMTs catalyze the formation of monomethyl arginines (Rme1). Type I PRMTs catalyze asymmetric arginine methylation (Rme2a), while type II PRMTs form symmetric arginine methylation (Rme2s). Type III PRMT can only monomethylate arginine residues. Modified from (PMID:29378138).</p>
<p><strong>Assay validation</strong></p>
<p>It has been reported that PRMT8 can methylate pro-oncoprotein encoded by the Ewing sarcoma gene – EWS (PIMID:18698489).  However, in several cell lines, the overexpressed EWS was already methylated even in the absence of exogenous PRMT8 possibly due to high activity of PRMT1. Thus we knocked down PRMT1 and overexpressed PRMT8 wt and catalytically inactive mutant (E185E) together with EWS to see if in that setting EWS would be methylated by PRMT8. The wild type PRMT8 but not catalytic mutant methylated ectopic EWS (Fig.2). We tested several pan asymmetric dimethylarginine antibodies and the methylation was only detected with Asym25 (Millipore,#09-814) but not with Asym24 (Millipore, #07-414) or Adme-R (CST,#13522). The assay was further validated with PRMT type I selective and PRMT4/6 selective chemical probes, MS023 and MS049, respectively (PIMID: 26598975, 27584694). As expected, only MS023 but not MS049 (negative control) decreased PRMT8 dependent asymmetric dimethylation of exogenous EWS (Fig.3). Although MS023 is very potent in inhibiting PRMT8 in <em>in vitro</em> assays, in cells high concentrations of MS023 are required to see the effect. The Z factor for the assay equals 0.58 (n=3).</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p8-1-300x105.png" alt="" width="628" height="220" class="wp-image-4719 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p8-1-300x105.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p8-1.png 654w" sizes="(max-width: 628px) 100vw, 628px" /></p>
<p><strong> </strong><strong>Fig.2. The wild type but not catalytic mutant of PRMT8 asymmetrically dimethylated ectopic EWS.</strong>  HEK293T cells were transduced with an inducible PRMT1 knockdown vector. After 3 days of doxycycline treatment, PRMT1 levels were drastically reduced (<strong>A)</strong> and cells were co-transfected with FLAG-tagged PRMT8 wild type or catalytic mutant (E185Q) and GFP-tagged EWS.  The protein expression and levels of methyl mark were analyzed in Western blot. <strong>B.</strong> Exogenous EWS is asymmetrically dimethylated by overexpressed wild type PRMT8 when PRMT1 is knocked-down.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p8-2-300x165.png" alt="" width="672" height="370" class="wp-image-4720 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p8-2-300x165.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p8-2-768x424.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/p8-2.png 1010w" sizes="(max-width: 672px) 100vw, 672px" /></p>
<p><strong>Fig.3. MS023 decreased PRMT8 dependent EWS asymmetric dimethylation at high concentration when cellular PRMT1 levels are reduced.</strong> PRMT1 knockdown HEK293T cells were co-transfected with FLAG-tagged PRMT8 (wild type or catalytic E185Q mutant) and GFP-tagged EWS-GFP and treated with MS023 (PRMT type I inhibitor) and MS049 (PRMT4 and 6 inhibitor) for 20h.</p>
<p>Experimental details are posted on <a href="https://zenodo.org/record/3829641#.Xr7tDWhKg2w">Zenodo</a>.</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/prmt8-cellular-assay/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>PRDM9 cellular assay</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/prdm9-cellular-assay/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/prdm9-cellular-assay/#respond</comments>
		
		<dc:creator><![CDATA[Magdalena Szewczyk]]></dc:creator>
		<pubDate>Thu, 14 May 2020 17:24:00 +0000</pubDate>
				<category><![CDATA[Cell assay reports]]></category>
		<category><![CDATA[Magdalena Szewczyk]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=4697</guid>

					<description><![CDATA[Background PRDM9 (PR domain-containing protein 9) contains a subclass of the SET (PR/SET) domain that trimethylates histone 3 on lysine 4 and 36 (PIMID:24095733). It is expressed mainly in germ cells playing an important role in meiotic recombination (PIMID:27362481). Its aberrant expression has also been associated with genomic instability in cancer (PIMID:30341163). Assay Validation Consistent <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/prdm9-cellular-assay/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p><strong>Background</strong></p>
<p>PRDM9 (PR domain-containing protein 9) contains a subclass of the SET (PR/SET) domain that trimethylates histone 3 on lysine 4 and 36 (PIMID:24095733). It is expressed mainly in germ cells playing an important role in meiotic recombination (PIMID:27362481). Its aberrant expression has also been associated with genomic instability in cancer (PIMID:30341163).</p>
<p><strong>Assay Validation</strong></p>
<p>Consistent with previous findings, the overexpression of FLAG-tagged wild type PRDM9 but not its catalytic mutant (Y357A) led to an increase in histone 3 lysine 4 trimethylation levels (H3K4me3) of exogenous (GFP-tagged) as well as endogenous histone H3 in cells (Fig.1). Due to high basal H3K4me3 levels, only a small increase in methylation was observed upon PRDM9 overexpression in endogenous H3, making the system with exogenous histone H3 preferential assay for compounds screening. The assay was additionally validated with PRDM9 selective inhibitor MRK-740 (PIMID:31848333). MRK-740 decreased PRDM9 dependent H3K4 trimethylation of exogenous histone H3 in a dose-dependent manner (PIMID:31848333, Fig.2). The assay z factor equals 0.83.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/PRDM9-2-268x300.png" alt="" width="338" height="378" class="wp-image-4699 alignleft" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/PRDM9-2-268x300.png 268w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/PRDM9-2.png 552w" sizes="(max-width: 338px) 100vw, 338px" /></p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p><strong>Fig.1.</strong><strong> The wild type but not catalytic mutant of PRDM9 methylated exogenous and endogenous histone H3K4.</strong>  HEK293T cells were co-transfected with GFP-tagged histone H3 and empty vector(control), FLAG-tagged wild type, or Y357A catalytic mutant PRDM9 24 h<strong>.</strong> The methylation levels were analyzed in Western Blot. (Shawna Organ)</p>
<p>&nbsp;</p>
<p><strong> </strong></p>
<p><strong> </strong><strong> </strong></p>
<p><strong> </strong><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/PRDM9-1-300x94.png" alt="" width="714" height="224" class="alignnone wp-image-4698" style="font-size: 1.5rem;" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/PRDM9-1-300x94.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/PRDM9-1-768x241.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/PRDM9-1.png 959w" sizes="(max-width: 714px) 100vw, 714px" /></p>
<p><strong>Fig.2.</strong><strong> MRK-740 decreases PRDM9 dependent H3K4 trimethylation in cells</strong>. HEK293T cells were co-transfected with FLAG-tagged PRMT9 (wild type or Y357A mutant) and GFP-tagged histone H3 and treated with inhibitor for 20 h. H3K4me3 levels were determined by Western blot. The graph represents the nonlinear fit of H3K4me3 signal intensities normalized to GFP. The results are mean +/- SEM of 3 replicates. (Magdalena Szewczyk)</p>
<p>Experimental details are posted on <a href="https://zenodo.org/record/3827543#.Xr1-LGhKg2w">Zenodo</a>.</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/prdm9-cellular-assay/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>SMYD2 cellular assay</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/smyd2-cellular-assay/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/smyd2-cellular-assay/#respond</comments>
		
		<dc:creator><![CDATA[Magdalena Szewczyk]]></dc:creator>
		<pubDate>Wed, 13 May 2020 17:42:48 +0000</pubDate>
				<category><![CDATA[Cell assay reports]]></category>
		<category><![CDATA[Magdalena Szewczyk]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=4674</guid>

					<description><![CDATA[Background SMYD2 is protein-lysine N-methyltransferase that methylates both histones and non-histone proteins, including p53. It monomethylates K370 of p53 leading to decreased DNA-binding and consequently the lower transcriptional regulation activity of p53 (PIMID:17108971). Assay validation Based on the findings (PIMID:17108971), we developed SMYD2 cellular assay in which exogenous FLAG-tagged p53 is methylated by exogenous V5-tagged <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/smyd2-cellular-assay/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p><strong>Background</strong></p>
<p>SMYD2 is protein-lysine N-methyltransferase that methylates both histones and non-histone proteins, including p53. It monomethylates K370 of p53 leading to decreased DNA-binding and consequently the lower transcriptional regulation activity of p53 (PIMID:17108971).</p>
<p><strong>Assay validation </strong></p>
<p>Based on the findings (PIMID:17108971), we developed SMYD2 cellular assay in which exogenous FLAG-tagged p53 is methylated by exogenous V5-tagged SMYD2. The assay was validated with two SMYD2 selective chemical probes: BAY-598 and PFI-5 (PIMID: 27075367, 31415173). Both compounds decreased monomethylation of exogenous p53-K370 in cells overexpressed with V5-SMYD2 (Fig.1,2). The z factor for the assay equals 0.87 (n=3).</p>
<p>&nbsp;</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/smyd2-1-300x187.png" alt="" width="529" height="330" class="wp-image-4678 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/smyd2-1-300x187.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/smyd2-1.png 654w" sizes="(max-width: 529px) 100vw, 529px" /></p>
<p><strong>Fig.1. BAY-598 decreases SMYD2 dependent p53K370 monomethylation in cells.</strong> HEK293T cells were co-transfected with V5-tagged SMYD2 and FLAG-tagged p53 and treated with 1 µM BAY-598 for 20 h. The monomethylation levels of p53K370 were analyzed in Western blot.</p>
<p><strong> <img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/smyd2-2-300x74.png" alt="" width="693" height="171" class="alignnone wp-image-4677" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/smyd2-2-300x74.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/05/smyd2-2.png 635w" sizes="(max-width: 693px) 100vw, 693px" /></strong></p>
<p><strong>Fig.2. PFI-5 decreases SMYD2 dependent p53K370 monomethylation in cells in a dose-dependent manner.</strong> MCF7 cells were co-transfected with V5-tagged SMYD2 and FLAG-tagged p53 and treated with PFI-5 for 20 h. The monomethylation levels of p53K370 were analyzed in Western Blot. The graph represents the nonlinear fit of p53K370me signal intensities normalized to p53(FLAG). The results are mean +/- SEM of 3 replicates.</p>
<p>Go to <a href="https://zenodo.org/record/3825216#.XrwxOmhKg2w">Zenodo</a> for experimental details.</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/smyd2-cellular-assay/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
	</channel>
</rss>
