<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>Antoine Claessens &#8211; openlabnotebooks.org</title>
	<atom:link href="https://thesgc.github.io/static-openlabnotebooks/category/laboratory/antoine-claessens-laboratories/feed/" rel="self" type="application/rss+xml" />
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<description>A growing team of groundbreaking scientists around the world are now sharing their lab notebooks online</description>
	<lastBuildDate>Fri, 27 Sep 2019 13:24:29 +0000</lastBuildDate>
	<language>en-GB</language>
	<sy:updatePeriod>hourly</sy:updatePeriod>
	<sy:updateFrequency>1</sy:updateFrequency>
	<generator>https://wordpress.org/?v=4.9.11</generator>

<image>
	<url>https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/cropped-lab-jar02-32x32.png</url>
	<title>Antoine Claessens &#8211; openlabnotebooks.org</title>
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<width>32</width>
	<height>32</height>
</image> 
	<item>
		<title>PhD position available in malaria parasite genomics and transcriptomics</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/phd-position-available-in-malaria-parasite-genomics-and-transcriptomics/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/phd-position-available-in-malaria-parasite-genomics-and-transcriptomics/#respond</comments>
		<pubDate>Tue, 23 Apr 2019 13:49:53 +0000</pubDate>
		<dc:creator><![CDATA[Antoine Claessens]]></dc:creator>
				<category><![CDATA[Antoine Claessens]]></category>
		<category><![CDATA[Malaria]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2737</guid>
		<description><![CDATA[Summary: –       3-year PhD funded by CNRS for genomics and transcriptomics data analysis of the Plasmodium falciparum malaria parasite. –       In a young dynamic team based in Montpellier, South of France, with many worldwide collaborators –       Keywords: population genetics &#38; genomics, single-cell RNA-seq, molecular epidemiology, antigenic variation.   Our research Most malaria deaths are caused <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/phd-position-available-in-malaria-parasite-genomics-and-transcriptomics/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p><strong>Summary:</strong></p>
<p><strong>–       3-year PhD funded by CNRS for genomics and transcriptomics data analysis of the <em>Plasmodium falciparum</em> malaria parasite.</strong></p>
<p><strong>–       In a young dynamic team based in Montpellier, South of France, with many worldwide collaborators</strong></p>
<p><strong>–       Keywords: population genetics &amp; genomics, single-cell RNA-seq, molecular epidemiology, antigenic variation.</strong></p>
<p><strong> </strong></p>
<p><strong>Our research</strong></p>
<p>Most malaria deaths are caused by <em>Plasmodium falciparum</em>, a unicellular eukaryote parasite transmitted by Anopheles mosquitoes. Despite the heavy burden of the disease, most human infections are asymptomatic (afebrile) and can last for months/years. In regions where malaria is seasonal, such as in The Gambia, West Africa, these asymptomatic infections probably act as a reservoir during the dry season when there is no or little transmission. <strong>Long-lasting infections could be turned to our advantage if we study them as a new host-pathogen interaction model</strong>, by regular sampling of an asymptomatic volunteer, investigating the parasite biology in its natural environment. More details on our research can be found on this <a href="https://thesgc.github.io/static-openlabnotebooks/plasmodium-falciparum-asymptomatic-infections-in-search-of-the-elephant-in-the-room/">on this post</a>.</p>
<p>From 2014 to 2017, we collected fingerprick blood samples every 3 months in a population of ~1000 inhabitants in a Gambian village. At the start of the 2017 dry season, we also recruited a cohort of 50 volunteers who were all asymptomatic carriers for <em>P. falciparum</em>. Monthly blood samples were taken for 6 months over the entire dry season. A total of 435 blood samples were collected, including this time series of monthly samples from the same volunteers over a 6-month period. <strong>This dataset represents a unique opportunity to address novel questions about the parasite biology</strong>. More specifically, we are interested in discovering how the parasite establishes a chronic infection, through changes at the transcriptomic levels.</p>
<p><strong>Your projects</strong></p>
<p>Projects here under are only a suggestion, the exact project will be defined based on the student’s abilities and expertise. Your work will be entirely computer-based, but you will be part of a  team mixing ‘dry’ and ‘wet’ lab research.</p>
<p><strong>Project 1: <em>P. falciparum</em> population genetics and genomics</strong></p>
<p>As outlined above, we collected blood samples all year round from a village over two and a half year. <em>P. falciparum</em> DNA was extracted and genotyped and/or whole genome sequenced in collaboration with MalariaGEN at the Sanger Institute. A dataset of about 400 parasite DNA barcodes and 100 parasite genomes is now available. Here we want to describe the first parasite genomes sequenced from asymptomatic infections, and investigate the impact of the dry season selective pressure on the parasite population. This longitudinal approach also allows us to tackle the question of the duration of a <em>P. falciparum</em> chronic infection. Preliminary analysis revealed, for example, that a child was infected asymptomatically with the same <em>P. falciparum </em>strain for over 2.5 years. This overturns the dogma on duration of a <em>P. falciparum</em> infection.</p>
<p>Your task is to fully characterise the parasite population genetic diversity, using genotyping, genomic and epidemiologic data. The analysis will include PCA, Fst, chromosome painting, EHH, iHS, etc.</p>
<p><strong>Project 2: Can <em><u>P. falciparum</u></em> sense its environment and adapt to it via transcriptional regulation?</strong></p>
<p>In areas where rains are seasonal, two distinct patterns of malaria fluctuate, from high prevalence with a wide range of clinical outcome in the wet season, to low prevalence mainly asymptomatic in the dry season. The mechanisms allowing the parasite to survive in chronic infections during the dry season, and restart transmission as vector population increases with the ensuing wet season, remain to be investigated. We hypothesize that <em>P. falciparum</em> is able to sense its environment and dampens its virulence with seasonality.</p>
<p>In collaboration with colleagues at the London School of Hygiene &amp; Trop Med, we are currently optimizing a single-cell RNA-seq approach to <em>P. falciparum</em>.</p>
<p>Your task, using ‘bulk’ and single-cell parasite transcriptomes, is to identify differentially regulated genes in the dry vs wet season, in symptomatic vs asymptomatic infections, and identify putative candidates involved in cell growth. The analysis will include normalization, PCA for clustering and tSNE for representation, correlating bulk and single cell data with Pearson correlation, drawing gene co-expression networks within each identified cluster (subpopulation), etc.</p>
<p>The transcriptome dataset, which will be linked with <em>in vitro </em>growth assays, will be crucial in understanding how the parasite successfully establishes a chronic infection over the dry season, yet triggers a new epidemic over the following transmission season. It also has the potential of identifying dormant parasites, a discovery that would guide malaria elimination strategies.</p>
<p><strong>Other projects:</strong></p>
<p>– Population genomics with over 100 <em>P. falciparum</em> genomes collected from severe malaria or uncomplicated cases in Benin. In collaboration with Gwladys Bertin (Paris), Joe Zhu and Jacob Garcia (Oxford, UK).</p>
<p>– Identification of the micro-indels and structural variant mutation rates in <em>P. falciparum</em> in vitro using data from “clone trees” (Hamilton 2016 NAR and unpublished data).</p>
<p>– And many, many more…</p>
<p>&nbsp;</p>
<p><strong>Your skills (essentials)</strong></p>
<ul>
<li>A Master in biostatistics / bioinformatics / population genetics, or in molecular biology with a strong background in biostatistics and programming</li>
<li>Demonstrated experience in “Omics” data analysis</li>
<li>Able to work independently</li>
<li>Great communication skills in English</li>
<li>Stay focused on answering a specific biological question, from the initial analysis to publication</li>
<li>Passionate about scientific discoveries</li>
</ul>
<p><strong>Your skills (desirable)</strong></p>
<ul>
<li>Knowledge about malaria biology and population genetics</li>
<li>Wet-lab experience is a plus, but not essential</li>
<li>Willing to supervise a Master student in bioinformatics / epidemiology</li>
<li>Participate actively in the lab and institute life</li>
</ul>
<p><strong>Who we are</strong></p>
<p>Antoine Claessens (PI) is a malariologist who trained at Edinburgh University, the Sanger Institute, LSHTM and the MRC-Gambia. He recently joined Montpellier University as a “Chargé de Recherche INSERM”. He was awarded an ATIP and an ANR-JC.</p>
<p>Currently the team consists of one post-doc and one Master student. Two PhD students will be recruited around Sep 2019, one for lab-based research and one for data analysis (this advert).  Each year, 2 Master students in bioinformatics / epidemiology will join the team for up to 6 months. We are likely to recruit another post-doc towards late 2019.</p>
<p><strong>What you can expect / what we can offer</strong></p>
<p>The 3-year PhD will be funded by CNRS (substantially higher salary than other sources of funding). You will be part of <a href="http://www.dimnp.univ-montp2.fr/">DIMNP</a> and you will closely interact with experienced bioinformaticians from the bioinformatics platform and with population geneticists from <a href="https://www.mivegec.ird.fr/fr/">MIVEGEC</a>. Both ‘UMR’ are very international, with highly competent and friendly scientists. More generally, Montpellier is a large hub for research in Life Sciences, particularly in the field of evolutionary biology.</p>
<p>I will try my best to transmit my passion about <em>P. falciparum</em> biology; you might find yourself dreaming about <em>var</em> genes too. As we will be a small team, you can expect more of my time dedicated to you, but I do not want to micromanage. I expect you to work hard, yet you will have freedom on working hours etc. All our current research projects are in collaboration with international laboratories, there is an opportunity to work in the USA/UK for a short period of time. If you are planning to stay in academic research, I will help you secure a post-doc position.</p>
<p>If you are from abroad: Being able to speak French is not a requirement; your colleagues will speak English fluently. We will help you with the administrative task of moving to France. Many aspects of the cost of life, such as childcare, are relatively cheap. Most importantly, the quality of life in South of France is hard to beat!</p>
<p><strong>Key collaborators and recent publications:<br />
</strong>Umberto D’Alessandro (MRC-Gambia)<br />
Teun Bousema (Radboud UMC, Netherlands)<br />
David Conway (LSHTM, UK)<br />
Dominic Kwiatkowski, MalariaGEN, Lia Chappell (Sanger Institute)<br />
Franck Prugnolle and Virginie Rougeron (MIVEGEC, Montpellier)<br />
Arthur Talman (Sanger / MIVEGEC)<br />
Silvia Portugal (Heidelberg, Germany)<br />
Ian Cheeseman (Texas Bio Med, USA)</p>
<table width="510">
<tbody>
<tr>
<td width="326"><strong>Selected publications:</strong></td>
<td width="184"><strong>Key discovery</strong></td>
</tr>
<tr>
<td width="326"><u>Claessens A</u>, Adams Y, …, Bozdech Z and Rowe JA. Group A-like PfEMP1s mediate cytoadherence to human brain endothelial cells<em>.</em> <strong>Proc Natl Acad Sci USA</strong> 2012.</td>
<td width="184">How <em>P. falciparum</em> bind to blood microvessels in vitro (microarray transcriptomic approach)</td>
</tr>
<tr>
<td width="326"><u>Claessens A</u>*, Hamilton W*, …, Rayner J, Kwiatkowski D. Generation of antigenic diversity in Plasmodium falciparum by structured rearrangement of var genes during mitosis. <strong>PLOS Genetics</strong> 2014. *joint first author.</td>
<td width="184">New chimeric <em>var</em> gene sequences are generated during parasite growth <em>in vitro</em>.</td>
</tr>
<tr>
<td width="326">Hamilton W*, <u>Claessens A</u>*, Otto T, …, Rayner J, Kwiatkowski D. Extreme mutation bias and high AT content in Plasmodium falciparum<em>.</em> <strong>Nucleic Acid Research</strong> 2016<em>. </em></td>
<td width="184">An excess of G:C to A:T transitions explains the AT-richness of <em>P.f.</em> genome</td>
</tr>
<tr>
<td width="326"><u>Claessens A</u>, Affara M, Assefa S, Kwiatkowski D &amp; Conway D. Culture adaptation of malaria parasites selects for convergent loss-of-function mutants<em>. </em><strong>Scientific Reports</strong> 2017<em>. </em></td>
<td width="184"><em>In vitro</em> culture selects for loss-of-function mutations (genomic approach)</td>
</tr>
<tr>
<td width="326"><u>Claessens A*</u>, Harris LM*, Stanojcic S*, Merrick C. RecQ helicases in the malaria parasite Plasmodium falciparum affect genome stability, gene expression patterns and DNA replication dynamics. <strong>PLOS Genetics</strong> 2018.</td>
<td width="184">Helicase PfWRN maintains genome integrity</td>
</tr>
</tbody>
</table>
<p>&nbsp;</p>
<p><strong>Interested in applying? Please send an email to <a href="mailto:antoineclaessens@gmail.com">antoineclaessens@gmail.com</a> with subject “PhD #0002 – keep me updated”. Starting date: September 2019 (to be discussed).</strong></p>
<p>&nbsp;</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/phd-position-available-in-malaria-parasite-genomics-and-transcriptomics/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Post-doc position available for a bioinformatician</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/post-doc-position-available-for-a-bioinformatician/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/post-doc-position-available-for-a-bioinformatician/#respond</comments>
		<pubDate>Fri, 17 Aug 2018 18:31:49 +0000</pubDate>
		<dc:creator><![CDATA[Antoine Claessens]]></dc:creator>
				<category><![CDATA[Antoine Claessens]]></category>
		<category><![CDATA[Malaria]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=1746</guid>
		<description><![CDATA[Post-doctoral position for an ‘Omic-oriented’ parasitologist/bioinformatician in Montpellier University, France &#160; Update April 2019: If you are skilled and motivated, this position is still open! Drop me an email (see address at the bottom) Summary: &#8211;       Up to 3 years post-doc position in genomics and transcriptomics data analysis of the Plasmodium falciparum malaria parasite. &#8211;       <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/post-doc-position-available-for-a-bioinformatician/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<h2>Post-doctoral position for an ‘Omic-oriented’ parasitologist/bioinformatician in Montpellier University, France</h2>
<p>&nbsp;</p>
<p><span style="text-decoration: underline;"><em>Update April 2019: If you are skilled and motivated, this position is still open! Drop me an email (see address at the bottom)</em></span></p>
<table width="424">
<tbody>
<tr>
<td width="424"><strong>Summary:</strong></p>
<p><strong>&#8211;       Up to 3 years post-doc position in genomics and transcriptomics data analysis of the <em>Plasmodium falciparum</em> malaria parasite.</strong></p>
<p><strong>&#8211;       In a young dynamic team based in Montpellier, South of France, with many worldwide collaborators</strong></p>
<p><strong>&#8211;       Keywords: population genetics &amp; genomics, single-cell RNA-seq, molecular epidemiology, antigenic variation.</strong></td>
</tr>
</tbody>
</table>
<p><strong> </strong></p>
<h3>Our research</h3>
<p>Most malaria deaths are caused by <em>Plasmodium falciparum</em>, a unicellular eukaryote parasite transmitted by Anopheles mosquitoes. Despite the heavy burden of the disease, most human infections are asymptomatic (afebrile) and can last for months/years. In regions where malaria is seasonal, such as in The Gambia, West Africa, it is thought that these asymptomatic infections act as a reservoir during the dry season when there is no or little transmission. <strong>Long-lasting infections could be turned to our advantage if we study them as a new host-pathogen interaction model</strong>, by regular sampling of an asymptomatic volunteer, investigating the parasite biology in its natural environment. More details on our research can be found on this <a href="https://thesgc.github.io/static-openlabnotebooks/plasmodium-falciparum-asymptomatic-infections-in-search-of-the-elephant-in-the-room/">on this post</a>.</p>
<p>From 2014 to 2017, we collected fingerprick blood samples every 3 months in a population of ~1000 inhabitants in a Gambian village. At the start of the 2017 dry season, we also recruited a cohort of 50 volunteers who were all asymptomatic carriers for <em>P. falciparum</em>. Monthly blood samples were taken for 6 months over the entire dry season. A total of 435 blood samples were collected, including this time series of monthly samples from the same volunteers over a 6-month period. <strong>This dataset represents a unique opportunity to address novel questions about the parasite biology</strong>. More specifically, we are interested in discovering how the parasite establishes a chronic infection, through changes at the transcriptomic levels.</p>
<h3>Your projects</h3>
<p><strong><u>Project 1: </u></strong><strong><u>Can <em>P. falciparum</em> sense its environment and adapt to it via transcriptional regulation?</u></strong></p>
<p>In areas where rains are seasonal, two distinct patterns of malaria fluctuate, from high prevalence with a wide range of clinical outcome in the wet season, to low prevalence mainly asymptomatic in the dry season. The mechanisms allowing the parasite to survive in chronic infections during the dry season, and restart transmission as vector population increases with the ensuing wet season, remain to be investigated. We hypothesize that <em>P. falciparum</em> is able to sense its environment and dampens its virulence with seasonality.</p>
<p>In collaboration with colleagues at the London School of Hygiene &amp; Tropical Medecine, we are sequencing about 100 transcriptomes of parasite isolates collected from the wet &amp; dry season, from symptomatic and asymptomatic infections, and from the monthly time series. In parallel, using the same dataset of Gambian isolates, we are measuring the parasite multiplication rate in vitro, with the hypothesis that parasite growth is reduced in isolates from the dry season.</p>
<p>Also, in collaboration with colleagues at the Sanger Institute, we are currently optimizing a single-cell RNA-seq approach to <em>P. falciparum</em>. The second part of project 1 is to identify putative subpopulations within each isolate.</p>
<p>Your task, using these ‘bulk’ and single-cell parasite transcriptomes, is to identify differentially regulated genes in the dry vs wet season, in symptomatic vs asymptomatic infections, and identify putative candidates involved in cell growth. The analysis will include normalization, PCA for clustering and tSNE for representation, correlating bulk and single cell data with Pearson correlation, drawing gene co-expression networks within each identified cluster (subpopulation), etc.</p>
<p>The transcriptome dataset, which will be linked to the growth assays, will be crucial in understanding how the parasite successfully establishes a chronic infection over the dry season, yet triggers a new epidemic over the following transmission season. It also has the potential of identifying dormant parasites, a discovery that would guide malaria elimination strategies.</p>
<p><strong><u>Project 2: <em>P. falciparum</em> population genetics and genomics</u></strong></p>
<p>As outlined above, we collected blood samples all year round from a village over two and a half year. <em>P. falciparum</em> DNA was extracted and genotyped and/or whole genome sequenced in collaboration with MalariaGEN at the Sanger Institute. A dataset of about 350 parasite DNA barcodes and 100 parasite genomes is now available. Here we want to describe the first parasite genomes sequenced from asymptomatic infections, and investigate the impact of the dry season selective pressure on the parasite population. This longitudinal approach also allows us to tackle the question of the duration of a <em>P. falciparum</em> chronic infection. Preliminary analysis revealed, for example, that a child was infected with the same <em>P. falciparum </em>strain for over 2.5 years.</p>
<p>Your task is to fully characterise the parasite population genetic diversity, using genotyping, genomic and epidemiologic data. The analysis will include PCA, Fst, chromosome painting, EHH, iHS, etc.</p>
<p><strong><u>Other projects:</u></strong></p>
<p>&#8211; Population genomics with over 100 <em>P. falciparum</em> genomes collected from severe malaria or uncomplicated cases in Benin. In collaboration with Gwladys Bertin (Paris), Joe Zhu and Jacob Garcia (Oxford, UK).</p>
<p>&#8211; Identification of the micro-indels and structural variant mutation rates in <em>P. falciparum</em> in vitro using data from “clone trees” (Hamilton 2016 NAR and unpublished data).</p>
<p>&#8211; And many, many more…</p>
<p>&nbsp;</p>
<h3>Your skills (essentials)</h3>
<ul>
<li>PhD in biostatistics/bioinformatics, or in molecular biology with a strong background in biostatistics and programming</li>
<li>Demonstrated experience in “Omics” data analysis</li>
<li>Able to work independently</li>
<li>Great communication skills in English</li>
<li>Stay focused on answering a specific biological question, from the initial analysis to publication</li>
<li>Passionate about scientific discoveries</li>
</ul>
<h3>Your skills (desirable)</h3>
<ul>
<li>Knowledge about malaria biology and population genetics</li>
<li>Wet-lab experience would be a plus, particularly FACS and cell-culture</li>
<li>Willing to supervise a Master student in bioinformatics / epidemiology</li>
<li>Participate actively in the lab and institute life</li>
<li>Appreciate XKCD humour</li>
</ul>
<h3>Who we are</h3>
<p>Antoine Claessens (PI) is a malariologist who trained at Edinburgh University, the Sanger Institute, LSHTM and the MRC-Gambia. He recently joined Montpellier University as a “Chargé de Recherche INSERM”. He was awarded an ATIP and an ANR-JC, with which this position will be funded.</p>
<p>A permanent Research Assistant will join the team full-time from October 2018. Each year, 2 or 3 Master students in bioinformatics / epidemiology will join the team for up to 6 months. We plan to recruit a PhD student in molecular biology in mid-2019.</p>
<h3>What you can expect / what we can offer</h3>
<p>The initial contract will be for two years, renewable for at least one year. You will be part of <a href="http://www.dimnp.univ-montp2.fr/">DIMNP</a> and you will closely interact with experienced bioinformaticians from the bioinformatics platform and with population geneticists from <a href="https://www.mivegec.ird.fr/fr/">MIVEGEC</a>. Both ‘UMR’ are very international, with highly competent and friendly scientists. More generally, Montpellier is a large hub for research in Life Sciences, particularly in the field of evolutionary biology. .</p>
<p>I will try my best to transmit my passion about <em>P. falciparum</em> biology; you might find yourself dreaming about <em>var</em> genes too. As we will be a small team, you can expect more of my time dedicated to you, but I do not want to micromanage. I expect you to work hard, yet you will have freedom on working hours etc. You are welcome to bring your own side projects if you wish. If you are planning to stay in academic research, I will help you secure your own funding (e.g. writing a Marie Curie Fellowship application).</p>
<p>If you are from abroad: Being able to speak French is not a requirement; your colleagues will speak English fluently. We will help you with the administrative task of moving to France. Many aspects of the cost of life, such as childcare, are relatively cheap. Most importantly, the quality of life in South of France is hard to beat!</p>
<p><strong>Key collaborators and recent publications:<br />
</strong>Umberto D’Alessandro (MRC-Gambia)<br />
Teun Bousema (Radboud UMC, Netherlands)<br />
David Conway (LSHTM, UK)<br />
Dominic Kwiatkowski, MalariaGEN, Lia Chappell (Sanger Institute)<br />
Franck Prugnolle and Virginie Rougeron (MIVEGEC, Montpellier)<br />
Arthur Talman (Sanger / MIVEGEC)<br />
Silvia Portugal (Heidelberg, Germany)</p>
<table width="510">
<tbody>
<tr>
<td width="326"><strong>Selected publications:</strong></td>
<td width="184"><strong>Key discovery</strong></td>
</tr>
<tr>
<td width="326"><u>Claessens A</u>, Adams Y, …, Bozdech Z and Rowe JA. Group A-like PfEMP1s mediate cytoadherence to human brain endothelial cells<em>.</em> <strong>Proc Natl Acad Sci USA</strong> 2012.</td>
<td width="184">How <em>P. falciparum</em> bind to blood microvessels in vitro (microarray transcriptomic approach)</td>
</tr>
<tr>
<td width="326"><u>Claessens A</u>*, Hamilton W*, …, Rayner J, Kwiatkowski D. Generation of antigenic diversity in Plasmodium falciparum by structured rearrangement of var genes during mitosis. <strong>PLOS Genetics</strong> 2014. *joint first author.</td>
<td width="184">New chimeric <em>var</em> gene sequences are generated during parasite growth <em>in vitro</em>.</td>
</tr>
<tr>
<td width="326">Hamilton W*, <u>Claessens A</u>*, Otto T, …, Rayner J, Kwiatkowski D. Extreme mutation bias and high AT content in Plasmodium falciparum<em>.</em> <strong>Nucleic Acid Research</strong> 2016<em>. </em></td>
<td width="184">An excess of G:C to A:T transitions explains the AT-richness of <em>P.f.</em> genome</td>
</tr>
<tr>
<td width="326"><u>Claessens A</u>, Affara M, Assefa S, Kwiatkowski D &amp; Conway D. Culture adaptation of malaria parasites selects for convergent loss-of-function mutants<em>. </em><strong>Scientific Reports</strong> 2017<em>. </em></td>
<td width="184"><em>In vitro</em> culture selects for loss-of-function mutations (genomic approach)</td>
</tr>
<tr>
<td width="326"><u>Claessens A*</u>, Harris LM*, Stanojcic S*, Merrick C. RecQ helicases in the malaria parasite Plasmodium falciparum affect genome stability, gene expression patterns and DNA replication dynamics. <strong>PLOS Genetics</strong> 2018.</td>
<td width="184">Helicase PfWRN maintains genome integrity</td>
</tr>
</tbody>
</table>
<p>&nbsp;</p>
<p><strong>Interested in applying? </strong><strong>Please send you application on the <a href="http://bit.ly/2NCJMtC">CNRS page</a>, and send an email to <a href="mailto:antoineclaessens@gmail.com">antoineclaessens@gmail.com</a> with subject “post-doc #0001 &#8211; keep me updated”. Starting date: January 2019 (to be discussed).</strong></p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/post-doc-position-available-for-a-bioinformatician/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Plasmodium falciparum asymptomatic infections: in search of the elephant in the room</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/plasmodium-falciparum-asymptomatic-infections-in-search-of-the-elephant-in-the-room/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/plasmodium-falciparum-asymptomatic-infections-in-search-of-the-elephant-in-the-room/#comments</comments>
		<pubDate>Mon, 18 Jun 2018 13:43:11 +0000</pubDate>
		<dc:creator><![CDATA[Antoine Claessens]]></dc:creator>
				<category><![CDATA[Antoine Claessens]]></category>
		<category><![CDATA[Malaria]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=1351</guid>
		<description><![CDATA[Hello Open World! In this very first post I will give an overview of my current and future research on malaria. I would like to sincerely thank everyone here (particularly Matthieu and Mandeep) for welcoming me so warmly. ABSTRACT: A major challenge for the elimination of malaria is Plasmodium falciparum asymptomatic infections, the hidden infectious <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/plasmodium-falciparum-asymptomatic-infections-in-search-of-the-elephant-in-the-room/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p><em>Hello Open World!<br />
</em><em>In this very first post I will give an overview of my current and future research on malaria. I would like to sincerely thank everyone here (particularly Matthieu and Mandeep) for welcoming me so warmly. </em></p>
<p><span style="text-decoration: underline;">ABSTRACT:</span><br />
A major challenge for the elimination of malaria is <em>Plasmodium falciparum </em>asymptomatic infections, the hidden infectious reservoir. Yet I believe these infections can provide useful information on the host-pathogen interactions over an extended period of time. My research aims to understand how <em>P. falciparum </em>parasites can survive in chronic infections over the 6-month long dry season, using an already available collection of blood samples from a cohort of asymptomatic volunteers in The Gambia. With whole genome sequencing, we will identify parasite alleles under selection during the dry season population bottleneck. With single-cell RNA-seq, we will explore how the parasite senses its environment and adapt to it. With qPCR and flow cytometry, we will test the central dogma that antigenic variation is at the basis of a chronic infection. This work will greatly advance our understanding of the parasite, host and vector interactions, and could reveal key molecular drivers of asymptomatic infections, a critical step to enable the eradication of malaria.</p>
<p><span style="text-decoration: underline;">INTRODUCTION:</span></p>
<p><strong>Basic facts about malaria:</strong></p>
<ul>
<li>Caused by <em>Plasmodium </em>parasites (unicellular eukaryote)</li>
<li>Transmitted by Anopheles mosquitoes</li>
<li><em>Plasmodium </em>parasites spend most of their asexual life inside red blood cells, with a reinvasion cycle every 48 hours.</li>
<li>Over half a million deaths per year, mainly children in Sub-Saharan Africa infected with <em>Plasmodium falciparum</em></li>
</ul>
<p><strong>Malaria in The Gambia, West Africa</strong></p>
<p>Where malaria transmission is seasonal such as The Gambia in West Africa, this occurs during and immediately after the rains (June-December). During the dry season (January-May) there is no transmission, hardly any clinical case is diagnosed during these dry months, and there are fewer mosquitoes[1,2]. Most of these infections will clear naturally, inducing a parasite population bottleneck (Fig. 1). However some <em>P. falciparum </em>parasites survive by establishing chronic, asymptomatic infections across the whole dry season. These infections are the reservoir from which the seasonal peak will restart at the next transmission season.</p>
<table width="454">
<tbody>
<tr>
<td style="width: 540px;" width="262"><img class="alignnone wp-image-1352 size-full" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/06/Ofig1.png" alt="" width="532" height="192" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/06/Ofig1.png 532w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/06/Ofig1-300x108.png 300w" sizes="(max-width: 532px) 100vw, 532px" /></td>
<td style="width: 880px; text-align: left;" width="191"><strong>Figure</strong> <strong>1.</strong> <strong><em>Plasmodium falciparum</em> prevalence in a village of ~1000 inhabitants in the eastern Gambia</strong>, detected by nested PCR or quantitative PCR. The blue and orange shadings indicate the rain and dry seasons, respectively. (Claessens, unpublished data).</td>
</tr>
</tbody>
</table>
<p><strong> </strong><strong> </strong><strong>My current research focuses on <em>P. falciparum</em> asymptomatic infections for three main reasons:</strong><strong> </strong></p>
<ol>
<li><em> Asymptomatic infections: out of sight, out of mind</em></li>
</ol>
<p>On any given day, the vast majority of all <em>P. falciparum </em>infections are asymptomatic, yet they have been poorly investigated (Fig 2). Currently, our knowledge of the parasite biology is based on isolates derived from clinical cases, and usually from clonal culture-adapted parasites. How asymptomatic infections differ from clinical cases has hardly ever been investigated, with no genome, epigenome, transcriptome, proteome or phenotypic description of such parasites published to date, due to the technical challenges associated with very low amount of biological material. Today’s technology is at a turning point for addressing these issues.</p>
<table>
<tbody>
<tr>
<td style="width: 319px;" width="120"><img class="alignnone wp-image-1353" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/06/Ofig2.png" alt="" width="291" height="207" /></td>
<td style="width: 1102px;" width="113"><strong><br />
Figure 2.</strong> Depending on malaria endemicity, the proportion of asymptomatic infections ranges from 73 to 98% [3].</td>
</tr>
</tbody>
</table>
<ol start="2">
<li><em> Asymptomatic infections, a challenge for malaria elimination</em></li>
</ol>
<p>This reservoir, shown to produce gametocytes and transmit them[4], represents the biggest challenge for malaria eradication. However, clearing all infections would include treating carriers without clinical symptoms who are unlikely to seek treatment. A campaign that would only target clinical cases, the most virulent forms of the parasite, is at risk of inadvertently selecting for a population with an “asymptomatic profile” (i.e. low-parasitaemia, chronic infections), complicating the elimination effort.</p>
<ol start="3">
<li><em> Asymptomatic infections as a new model for host-pathogen interactions</em></li>
</ol>
<p>Currently, pathogenesis studies typically collect a single blood sample from the patient on its arrival at hospital, giving us only a snapshot of the host-pathogen interaction. I believe we could “record the video” of the host-pathogen interaction with regular blood sampling of the same chronic asymptomatic carrier, to study the evolution of the same infection in real time (Fig 3).</p>
<p><img class="alignnone wp-image-1354" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/06/Ofig3-300x96.png" alt="" width="238" height="76" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/06/Ofig3-300x96.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/06/Ofig3.png 451w" sizes="(max-width: 238px) 100vw, 238px" />    <strong>Figure 3</strong></p>
<p>This is why in The Gambia we recruited a cohort of ~50 volunteers who were asymptomatic carriers at the start of the dry season (December 2016). These individuals were sampled monthly for up to 6 months. Epidemiology results will be described in a separate post.<br />
Understanding how the parasite can survive in a human host for months will help us develop novel genetic surveillance &amp; therapeutic tools.</p>
<p>&nbsp;</p>
<p><span style="text-decoration: underline;">AIMS</span>:</p>
<p>To understand how the parasite is able to survive over the dry season, and start a new epidemic at the following wet season, we will characterize <em>P. falciparum </em>from chronic infections, at the genomic, transcriptomic and phenotypic levels, as well as the associated host immune response. More specifically:</p>
<p><strong>Project 1: Identifying P. falciparum alleles under selection during the dry season</strong></p>
<p>Sequencing of <em>P. falciparum </em>genomes collected during the wet and dry season will identify parasite alleles that are selected for by the parasite population bottleneck due to the lack of transmission in the dry season.</p>
<p><strong>Project 2: Can <em>P. falciparum </em>sense its environment and adapt to it via transcriptional regulation?</strong></p>
<p><strong>2a</strong>: We will measure the in vitro parasite multiplication rate to determine whether isolates from asymptomatic infections grow slower than virulent infections.</p>
<p><strong>2b</strong>: Using single-cell (sc) RNA-seq, we will compare the transcriptome of parasites collected in dry and wet seasons. Our hypothesis is that at the start of the dry season some subpopulations of parasites switch to a dormant-like state.</p>
<p><strong>Project 3: Can antigenic variation explain chronic infections?</strong></p>
<p>We will record <em>var</em> gene expression switching and its associated immune response monthly over the course of the same infections. This will reveal a long postulated but never formally tested hypothesis that variant surface antigen switching is sufficient for the parasite to evade the host immune system.</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p>References:</p>
<p>1        Ceesay, S.J. <em>et al.</em> (2008) Changes in malaria indices between 1999 and 2007 in The Gambia: a retrospective analysis. <em>Lancet</em> 372, 1545–54</p>
<p>2        Ceesay, S.J. <em>et al.</em> (2010) Continued decline of malaria in The Gambia with implications for elimination. <em>PLoS One</em> 5, 4–13</p>
<p>3        Lindblade, K. a <em>et al.</em> (2013) The silent threat: asymptomatic parasitemia and malaria transmission. <em>Expert Rev. Anti. Infect. Ther.</em> 11, 623–39</p>
<p>4        Stone, W. <em>et al.</em> (2015) Assessing the infectious reservoir of falciparum malaria: Past and future. <em>Trends Parasitol.</em> 31, 287–296</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/plasmodium-falciparum-asymptomatic-infections-in-search-of-the-elephant-in-the-room/feed/</wfw:commentRss>
		<slash:comments>3</slash:comments>
		</item>
	</channel>
</rss>
