<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	
	>
<channel>
	<title>Comments on: Overexpression of Expanded HTT</title>
	<atom:link href="https://thesgc.github.io/static-openlabnotebooks/overexpression-of-expanded-htt/feed/" rel="self" type="application/rss+xml" />
	<link>https://thesgc.github.io/static-openlabnotebooks/overexpression-of-expanded-htt/</link>
	<description>A growing team of groundbreaking scientists around the world are now sharing their lab notebooks online</description>
	<lastBuildDate>Wed, 11 Dec 2019 16:43:59 +0000</lastBuildDate>
	<sy:updatePeriod>hourly</sy:updatePeriod>
	<sy:updateFrequency>1</sy:updateFrequency>
	<generator>https://wordpress.org/?v=4.9.13</generator>
	<item>
		<title>By: Overexpression of Expanded HTT in Brain Cancer Cell Lines &#8211; Extreme Open Science Initiative</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/overexpression-of-expanded-htt/#comment-75</link>
		<dc:creator><![CDATA[Overexpression of Expanded HTT in Brain Cancer Cell Lines &#8211; Extreme Open Science Initiative]]></dc:creator>
		<pubDate>Tue, 19 Jun 2018 17:06:52 +0000</pubDate>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=1037#comment-75</guid>
		<description><![CDATA[[&#8230;] was not very good, especially for HTT with longer Q repeats (have a look at these results here). Anyway, forging ahead, I decided to try the baculovirus overexpression in brain cells, which [&#8230;]]]></description>
		<content:encoded><![CDATA[<p>[&#8230;] was not very good, especially for HTT with longer Q repeats (have a look at these results here). Anyway, forging ahead, I decided to try the baculovirus overexpression in brain cells, which [&#8230;]</p>
]]></content:encoded>
	</item>
	<item>
		<title>By: Jolene Ho</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/overexpression-of-expanded-htt/#comment-49</link>
		<dc:creator><![CDATA[Jolene Ho]]></dc:creator>
		<pubDate>Tue, 24 Apr 2018 01:42:44 +0000</pubDate>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=1037#comment-49</guid>
		<description><![CDATA[Hey J.B. Shaw, thanks for the video, that&#039;s really interesting! I suppose the paper the video is referencing is this: https://www.nature.com/articles/s41557-018-0046-3? I personally didn&#039;t know anything about i-motifs until I watched your link and did some research. It seems like the i-motif was discovered quite some time ago, in 1993, although it has not be shown in living cells until now. I believe this is the article in which i-motifs were first described, but please correct me if I am wrong: https://www.nature.com/articles/363561a0.

Anyway, the video you linked mentioned that the i-motifs were observed to appear and disappear, and that the i-motifs might be involved in turning genes on and off. This would strongly suggest that the i-motifs are acting as epigenetic switches.

As for the links between HTT and the i-motifs, I don&#039;t think anyone has looked at that yet (again, please correct me if I am wrong). So far, people have attributed the etiology of HD to the misfolded mutant HTT protein. However, the mutant HTT is misfolded due to the expanded CAG region, and the i-motifs occur in cytosine-rich regions. I&#039;m not sure if the expanded CAG region of mutant HTT is cytosine-rich enough for i-motifs to form, but it would be very interesting if such motifs were found in the mutant HTT gene!]]></description>
		<content:encoded><![CDATA[<p>Hey J.B. Shaw, thanks for the video, that&#8217;s really interesting! I suppose the paper the video is referencing is this: <a href="https://www.nature.com/articles/s41557-018-0046-3" rel="nofollow">https://www.nature.com/articles/s41557-018-0046-3</a>? I personally didn&#8217;t know anything about i-motifs until I watched your link and did some research. It seems like the i-motif was discovered quite some time ago, in 1993, although it has not be shown in living cells until now. I believe this is the article in which i-motifs were first described, but please correct me if I am wrong: <a href="https://www.nature.com/articles/363561a0" rel="nofollow">https://www.nature.com/articles/363561a0</a>.</p>
<p>Anyway, the video you linked mentioned that the i-motifs were observed to appear and disappear, and that the i-motifs might be involved in turning genes on and off. This would strongly suggest that the i-motifs are acting as epigenetic switches.</p>
<p>As for the links between HTT and the i-motifs, I don&#8217;t think anyone has looked at that yet (again, please correct me if I am wrong). So far, people have attributed the etiology of HD to the misfolded mutant HTT protein. However, the mutant HTT is misfolded due to the expanded CAG region, and the i-motifs occur in cytosine-rich regions. I&#8217;m not sure if the expanded CAG region of mutant HTT is cytosine-rich enough for i-motifs to form, but it would be very interesting if such motifs were found in the mutant HTT gene!</p>
]]></content:encoded>
	</item>
	<item>
		<title>By: J.B. Shaw</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/overexpression-of-expanded-htt/#comment-48</link>
		<dc:creator><![CDATA[J.B. Shaw]]></dc:creator>
		<pubDate>Mon, 23 Apr 2018 23:05:38 +0000</pubDate>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=1037#comment-48</guid>
		<description><![CDATA[Jolene Ho, Are eye-motifs epigenetic switches? (NATURE Chemistry) MSN Short video clip seen here) https://www.msn.com/en-us/video/wonder/new-dna-structure-discovered-inside-living-human-cells/vi-AAwfiyx?ocid=spartanntp  If so, where&#039;s the eye-motif for HTT? Enjoy :-)]]></description>
		<content:encoded><![CDATA[<p>Jolene Ho, Are eye-motifs epigenetic switches? (NATURE Chemistry) MSN Short video clip seen here) <a href="https://www.msn.com/en-us/video/wonder/new-dna-structure-discovered-inside-living-human-cells/vi-AAwfiyx?ocid=spartanntp" rel="nofollow">https://www.msn.com/en-us/video/wonder/new-dna-structure-discovered-inside-living-human-cells/vi-AAwfiyx?ocid=spartanntp</a>  If so, where&#8217;s the eye-motif for HTT? Enjoy ðŸ™‚</p>
]]></content:encoded>
	</item>
	<item>
		<title>By: Jolene Ho</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/overexpression-of-expanded-htt/#comment-46</link>
		<dc:creator><![CDATA[Jolene Ho]]></dc:creator>
		<pubDate>Wed, 18 Apr 2018 16:30:05 +0000</pubDate>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=1037#comment-46</guid>
		<description><![CDATA[Great question! Actually, that might be one of the possible reasons we get such low expression levels with the larger constructs. It might be possible to try detecting aggregates in the cells.]]></description>
		<content:encoded><![CDATA[<p>Great question! Actually, that might be one of the possible reasons we get such low expression levels with the larger constructs. It might be possible to try detecting aggregates in the cells.</p>
]]></content:encoded>
	</item>
	<item>
		<title>By: Matthieu Schapira</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/overexpression-of-expanded-htt/#comment-43</link>
		<dc:creator><![CDATA[Matthieu Schapira]]></dc:creator>
		<pubDate>Mon, 16 Apr 2018 19:27:00 +0000</pubDate>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=1037#comment-43</guid>
		<description><![CDATA[Jolene, if pathogenic variants of Huntingtin are misfolded, should their baculo-expressed constructs be soluble in mammalian cell lines?]]></description>
		<content:encoded><![CDATA[<p>Jolene, if pathogenic variants of Huntingtin are misfolded, should their baculo-expressed constructs be soluble in mammalian cell lines?</p>
]]></content:encoded>
	</item>
</channel>
</rss>
