<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>Past Scientist &#8211; openlabnotebooks.org</title>
	<atom:link href="https://thesgc.github.io/static-openlabnotebooks/category/past-scientist/feed/" rel="self" type="application/rss+xml" />
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<description>A growing team of groundbreaking scientists around the world are now sharing their lab notebooks online</description>
	<lastBuildDate>Fri, 01 Mar 2019 09:51:22 +0000</lastBuildDate>
	<language>en-GB</language>
	<sy:updatePeriod>hourly</sy:updatePeriod>
	<sy:updateFrequency>1</sy:updateFrequency>
	<generator>https://wordpress.org/?v=4.9.9</generator>

<image>
	<url>https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/cropped-lab-jar02-32x32.png</url>
	<title>Past Scientist &#8211; openlabnotebooks.org</title>
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<width>32</width>
	<height>32</height>
</image> 
	<item>
		<title>Further Investigation of the Reduction of HTT Protein through Chemical Inhibition</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/further-investigation-of-the-reduction-of-htt-protein-through-chemical-inhibition/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/further-investigation-of-the-reduction-of-htt-protein-through-chemical-inhibition/#respond</comments>
		<pubDate>Tue, 15 Jan 2019 18:56:28 +0000</pubDate>
		<dc:creator><![CDATA[Jolene Ho]]></dc:creator>
				<category><![CDATA[Huntingtin]]></category>
		<category><![CDATA[Huntington’s disease]]></category>
		<category><![CDATA[Jolene Ho]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2378</guid>
		<description><![CDATA[The experimental set-up and results covered in this post can be found here: 10.5281/zenodo.2541053. A lot has happened in the past few months and it is with mixed feelings that I am finishing up my postdoc here at SGC! I feel very privileged to have been part of the great science going on at SGC <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/further-investigation-of-the-reduction-of-htt-protein-through-chemical-inhibition/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p style="text-align: justify;">The experimental set-up and results covered in this post can be found here: <a href="http://doi.org/10.5281/zenodo.2541053">10.5281/zenodo.2541053</a>.</p>
<p style="text-align: justify;">A lot has happened in the past few months and it is with mixed feelings that I am finishing up my postdoc here at SGC! I feel very privileged to have been part of the great science going on at SGC and a pioneer of the Open Notebook initiative here, contributing to open science. Nonetheless, the science has to go on! Hopefully, my posts in this Open Notebook have provided new perspectives to people working on similar projects and trying to learn more about Huntington’s Disease and huntingtin (HTT), inspired people to strive on for science despite the negative and/or inconclusive data, or at least given some insight to how life on the bench is like in general.</p>
<p style="text-align: justify;">Previously, I showed that the use of PRMT5 inhibitors from the SGC probe library, namely GSK591 and LLY283, were able to decrease levels of HTT in HeLa cells. SGC2096, the inactive control to GSK591, was not able to decrease HTT protein levels. This, together with the different mechanisms of action of GSK591 and LLY283 in inhibiting PRMT5, demonstrates that it is indeed through the specific inhibition of PRMT5 that HTT protein levels were decreased. So far, I have not come across any drug that is able to regulate the levels of HTT protein. As such, the inhibition of PRMT5 to reduce the expression of HTT at the protein level presents a potential and exciting therapeutic avenue for the treatment of Huntington’s disease.</p>
<p style="text-align: justify;">The findings that PRMT5 inhibition itself can affect the expression of HTT at the protein level suggests that there might be some interaction and possible feedback loop in play between PRMT5 and HTT. We thus further investigated how PRMT5 inhibition may contribute to the decrease in HTT protein levels to elucidate the mechanisms involved in this process. As PRMT5 is the main enzyme known to mediate symmetric demethylation of arginine (SDMA), we thus checked how the levels of SDMA are affected by PRMT5 inhibition and HTT knockdown.</p>
<p style="text-align: justify;"><img class="alignnone wp-image-2379" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/HeLa-G3BP1-PRMT5i-in-HTT-KD-300x225.jpg" alt="" width="771" height="578" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/HeLa-G3BP1-PRMT5i-in-HTT-KD-300x225.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/HeLa-G3BP1-PRMT5i-in-HTT-KD-768x576.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/HeLa-G3BP1-PRMT5i-in-HTT-KD-350x263.jpg 350w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/HeLa-G3BP1-PRMT5i-in-HTT-KD-780x586.jpg 780w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/HeLa-G3BP1-PRMT5i-in-HTT-KD-250x188.jpg 250w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/HeLa-G3BP1-PRMT5i-in-HTT-KD.jpg 960w" sizes="(max-width: 771px) 100vw, 771px" /></p>
<p style="text-align: justify;">Unsurprisingly, PRMT5 inhibition with GSK591 or LLY283 led to the decrease in SDMA. However, we also noticed a slight decrease in SDMA across all 3 HTT knockdown samples! Although this decrease is marginal, its occurrence across all the HTT knockdown samples we tested gives confidence to this observation and corroborates the hypothesis that PRMT5 and HTT are part of a feedback loop rather than that HTT is simply downstream of PRMT5, since the inhibition of PRMT5 leads to a decrease in HTT, and a decrease in HTT by way of HTT knockdown leads to a decrease in SDMA, which is mainly mediated by PRMT5.</p>
<p style="text-align: justify;">The two-way crosstalk demonstrated by PRMT5 activity levels affecting the expression of HTT and the presence (or absence, since we used a knockdown model) of HTT likewise decreasing SDMA levels suggests that the relationship between PRMT5 and HTT is complicated and reciprocal, with one regulating the other possibly through post-translational modifications such as SDMA and other epigenetic mechanisms, and warrants further investigation as to how the activity and levels of PRMT5 and HTT are related. Some important future experiments would be to test if PRMT inhibition is also able to decrease the protein levels of mutant HTT similarly to how it affects wild-type HTT, and if so, this will be an exciting opportunity to follow up on in the search of a potential therapy for Huntington’s disease.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/further-investigation-of-the-reduction-of-htt-protein-through-chemical-inhibition/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Reduction of HTT Protein through Chemical Inhibition</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/reduction-of-htt-protein-through-chemical-inhibition/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/reduction-of-htt-protein-through-chemical-inhibition/#comments</comments>
		<pubDate>Thu, 22 Nov 2018 20:34:38 +0000</pubDate>
		<dc:creator><![CDATA[Jolene Ho]]></dc:creator>
				<category><![CDATA[Huntingtin]]></category>
		<category><![CDATA[Huntington’s disease]]></category>
		<category><![CDATA[Jolene Ho]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2169</guid>
		<description><![CDATA[The experimental set-up and results covered in this post can be found here: 10.5281/zenodo.1495129. Apologies for the long delay between this post and my last post, it really has been quite some time! As postdocs, we generally have several projects running simultaneously and we have to split our time between these projects. I am no <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/reduction-of-htt-protein-through-chemical-inhibition/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p style="text-align: justify;">The experimental set-up and results covered in this post can be found here: <a href="http://doi.org/10.5281/zenodo.1495129">10.5281/zenodo.1495129</a>.</p>
<p style="text-align: justify;">Apologies for the long delay between this post and my last post, it really has been quite some time! As postdocs, we generally have several projects running simultaneously and we have to split our time between these projects. I am no exception and lately I have been caught up in trying to wrap up several projects. Besides my work in HTT, I am involved in screening various cell lines against our SGC probe library and identifying probes that have a differential effect in these cell lines. At the SGC, chemical compounds only qualify as probes if they have an in vitro potency of &lt;100 nM, &gt;30-fold selectivity vs other subfamilies, and have significant on-target cell activity at 1 µM. These probes are essential for the early stages of drug development as positive hits in our screens allow us to identify potential drug targets, protein families, and pathways for which we can further investigate in terms of disease mechanism and therapeutic strategies. These probes identified as positive hits in our screens may then be further developed and refined as drugs which may be used in the clinics eventually. You can read more about the SGC’s probes here: <a href="https://www.thesgc.org/chemical-probes">https://www.thesgc.org/chemical-probes</a>.</p>
<p style="text-align: justify;">In one of the other projects I’m working on, we identified PRMT5 as a potential therapeutic target in glioblastoma through the action of GSK591 and LLY283, 2 probes that inhibit PRMT5 through different and independent mechanisms. The inhibition of PRMT5 as a therapeutic strategy is not novel in itself, with several other groups having already published on this.</p>
<p>&nbsp;</p>
<p style="text-align: justify;">PRMT5 as a druggable target for glioblastoma therapy.</p>
<p style="text-align: justify;">Banasavadi-Siddegowda YK, Welker AM, An M, Yang X, Zhou W, Shi G, Imitola J, Li C, Hsu S, Wang J, Phelps M, Zhang J, Beattie CE, Baiocchi R, Kaur B.</p>
<p style="text-align: justify;">Neuro Oncol. 2018 May 18;20(6):753-763. doi: <a href="http://doi.org/10.1093/neuonc/nox206">10.1093/neuonc/nox206</a>.</p>
<p>&nbsp;</p>
<p style="text-align: justify;">Coordinated Splicing of Regulatory Detained Introns within Oncogenic Transcripts Creates an Exploitable Vulnerability in Malignant Glioma.</p>
<p style="text-align: justify;">Braun CJ, Stanciu M, Boutz PL, Patterson JC, Calligaris D, Higuchi F, Neupane R, Fenoglio S, Cahill DP, Wakimoto H, Agar NYR, Yaffe MB, Sharp PA, Hemann MT, Lees JA.</p>
<p style="text-align: justify;">Cancer Cell. 2017 Oct 9;32(4):411-426.e11. doi: <a href="http://doi.org/10.1016/j.ccell.2017.08.018">10.1016/j.ccell.2017.08.018</a>. Epub 2017 Sep 28.</p>
<p>&nbsp;</p>
<p style="text-align: justify;">However, we had some interesting findings through the RNA-sequencing done on cells treated with these PRMT5 inhibitors. Serendipitously, HTT emerged as one of the genes whose expression was consistently downregulated with PRMT5 inhibition! I tested these PRMT5 inhibitors in the HeLa cells I used to generate the 3 stable HTT knockdown lines previously (read more about that here: https://thesgc.github.io/static-openlabnotebooks/generating-htt-knockdown-models/) and looked at how the HTT protein levels were affected in these lines.</p>
<p style="text-align: justify;"><img class="alignnone wp-image-2170" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/20181116-HeLa-G3BP1-PRMT5i-and-HTT-KD-300x225.jpg" alt="" width="841" height="631" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/20181116-HeLa-G3BP1-PRMT5i-and-HTT-KD-300x225.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/20181116-HeLa-G3BP1-PRMT5i-and-HTT-KD-768x576.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/20181116-HeLa-G3BP1-PRMT5i-and-HTT-KD-350x263.jpg 350w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/20181116-HeLa-G3BP1-PRMT5i-and-HTT-KD-780x586.jpg 780w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/20181116-HeLa-G3BP1-PRMT5i-and-HTT-KD-250x188.jpg 250w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/20181116-HeLa-G3BP1-PRMT5i-and-HTT-KD.jpg 960w" sizes="(max-width: 841px) 100vw, 841px" /></p>
<p style="text-align: justify;">HTT protein levels were clearly decreased with PRMT5 inhibition! I ran these samples alongside the stable HTT knockdown lines and although the HTT levels did not decrease as drastically as with the knockdowns, the decrease with the PRMT5 inhibitors is still substantial. I also included SGC2096, the inactive control to GSK591, and HTT protein levels were not affected by it, showing that it is indeed through the activity of PRMT5 that HTT levels were decreased.</p>
<p style="text-align: justify;">The mechanism by which HTT is downregulated by PRMT5 inhibitors is still unclear and it will be very interesting to further investigate the link between HTT and PRMT5. As of now, the only publication I can find linking HTT to PRMT5 is the following:</p>
<p>&nbsp;</p>
<p style="text-align: justify;">PRMT5- mediated symmetric arginine dimethylation is attenuated by mutant huntingtin and is impaired in Huntington&#8217;s disease (HD).</p>
<p style="text-align: justify;">Ratovitski T, Arbez N, Stewart JC, Chighladze E, Ross CA.</p>
<p style="text-align: justify;">Cell Cycle. 2015;14(11):1716-29. doi: <a href="http://doi.org/10.1080/15384101.2015.1033595">10.1080/15384101.2015.1033595</a>.</p>
<p>&nbsp;</p>
<p style="text-align: justify;">This paper showed the colocalisation and interaction of HTT and PRMT5 and shows that this interaction results in the stimulation of PRMT5 activity by HTT in vitro. Mutant HTT has a greater affinity for PRMT5; however, it has a reduced ability to stimulate PRMT5 activity, as demonstrated by the reduction of symmetric dimethylation of arginine (SDMA), which is mainly mediated by PRMT5, of histones H2 and H4. Decreased levels of SDMA on known PRMT5 targets were detected in HD brain samples, suggesting that the reduction in PRMT5 activity may contribute to the pathogenesis of HD. The knockdown of JMJD6, which removes arginine demethylation, was able to rescue the phenotype of increased cell death seen with mutant HTT, further suggesting that the restoration of SDMA to normal levels may be a potential therapeutic strategy.</p>
<p style="text-align: justify;">However, our findings that PRMT5 inhibition itself can affect the expression of HTT at the protein level suggests that there might be a feedback loop in play between PRMT5 and HTT. It will be interesting to further investigate how the levels of SDMA and the other proteins involved are affected with PRMT5 inhibition and elucidate the mechanisms involved in this process. It should also be noted that our results only reflect the expression of wild-type HTT and not mutant (expanded) HTT, and it will be highly interesting and of great clinical importance to find out if PRMT5 inhibition is able to decrease the protein levels of mutant HTT similarly to how it affects wild-type HTT.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/reduction-of-htt-protein-through-chemical-inhibition/feed/</wfw:commentRss>
		<slash:comments>3</slash:comments>
		</item>
		<item>
		<title>Generating HTT Knockdown Models</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/generating-htt-knockdown-models/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/generating-htt-knockdown-models/#comments</comments>
		<pubDate>Mon, 17 Sep 2018 20:45:58 +0000</pubDate>
		<dc:creator><![CDATA[Jolene Ho]]></dc:creator>
				<category><![CDATA[Huntingtin]]></category>
		<category><![CDATA[Huntington’s disease]]></category>
		<category><![CDATA[Jolene Ho]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=1860</guid>
		<description><![CDATA[The experimental set-up and results covered in this post can be found here: doi.org/10.5281/zenodo.1420537 I was previously using an HTT overexpression model to look at any changes in proteins related to HTT or its function. Unfortunately, none of the related proteins I looked at showed any significant change in protein levels with HTT overexpression. There <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/generating-htt-knockdown-models/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>The experimental set-up and results covered in this post can be found here: <a href="http://doi.org/10.5281/zenodo.1420537">doi.org/10.5281/zenodo.1420537</a></p>
<p>I was previously using an HTT overexpression model to look at any changes in proteins related to HTT or its function. Unfortunately, none of the related proteins I looked at showed any significant change in protein levels with HTT overexpression. There are several reasons for this: the HTT overexpression might not be strong enough (especially for the larger constructs of D4 and D6, which are harder to express) or the level of HTT protein in the cells is already saturated, hence an overexpression does not further affect these proteins of interest. I therefore decided to use a knockdown model instead, which might provide more insight into the functions and effects of HTT.</p>
<p>A slightly different system was used to generate the HTT knockdown models, compared to the HTT overexpression model. For the overexpression model, a baculovirus system was used. In the knockdown model, a lentivirus system was used.</p>
<p>Lentiviruses can be easily produced through the transfection of HEK293T cells with several plasmids which code for different components of the lentivirus. The HEK293T cells, which are very easy to transfect, can then produce the various components of the lentivirus and assemble the lentiviruses, releasing them into the cell media. As HEK293T cells are grown adherently, these lentiviruses can be harvested by simply collecting the supernatant. When the plasmids encoding the different components of the lentivirus are transfected together with a plasmid encoding an shRNA for the gene of interest (in this case, HTT), the lentiviruses produced have the ability to disrupt the gene of interest through RNA interference (RNAi) when they transduce the target cells.</p>
<p><img class="alignnone  wp-image-1862" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/09/Lentivirus-Production-300x225.jpg" alt="" width="771" height="578" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/09/Lentivirus-Production-300x225.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/09/Lentivirus-Production-768x576.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/09/Lentivirus-Production-350x263.jpg 350w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/09/Lentivirus-Production-780x586.jpg 780w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/09/Lentivirus-Production-250x188.jpg 250w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/09/Lentivirus-Production.jpg 960w" sizes="(max-width: 771px) 100vw, 771px" /></p>
<p>Figure adapted from Figure 4 of Nucleic Acid Delivery: Lentiviral and Retroviral Vectors by Erin P O’Keefe; Princeton University, United States (MATER METHODS 2013;3:174; <a href="http://dx.doi.org/10.13070/mm.en.3.174">dx.doi.org/10.13070/mm.en.3.174</a>).</p>
<p>&nbsp;</p>
<p>Upon transduction of the target cells, the lentivirus delivers the plasmid encoding the shRNA to the nucleus. The plasmid produces the shRNA, which is processed by the target cell, leading to the production of a sequence complementary to that of the mRNA of the gene of interest, HTT. The binding of this complementary sequence to the mRNA of the gene of interest causes the formation of the RNA-induced silencing complex (RISC). The RISC, consisting of both the complementary sequence from the shRNA and the mRNA of the gene of interest, is then degraded, preventing the expression of the target protein, which is HTT in this case, leading to the knockdown of HTT. As the plasmid in the target cell continuously produces the shRNA, the mRNA is continuously degraded and the cells are stably knocked down for HTT.</p>
<p>&nbsp;</p>
<p><img class="alignnone  wp-image-1863" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/09/DdRNAi_diagram-211x300.jpg" alt="" width="325" height="462" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/09/DdRNAi_diagram-211x300.jpg 211w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/09/DdRNAi_diagram.jpg 571w" sizes="(max-width: 325px) 100vw, 325px" /></p>
<p>Figure from Figure 1 of DNA-directed RNA interference (<a href="http://www.wikiwand.com/en/DNA-directed_RNA_interference">http://www.wikiwand.com/en/DNA-directed_RNA_interference</a>).</p>
<p>&nbsp;</p>
<p>Using this method, I managed to establish 3 stable HTT knockdowns using 3 independent shRNA constructs in HeLa cells.</p>
<p><img class="alignnone  wp-image-1861" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/09/20180620-HeLa-G3BP1-HTT-KD245-Stable-300x225.jpg" alt="" width="773" height="580" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/09/20180620-HeLa-G3BP1-HTT-KD245-Stable-300x225.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/09/20180620-HeLa-G3BP1-HTT-KD245-Stable-768x576.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/09/20180620-HeLa-G3BP1-HTT-KD245-Stable-350x263.jpg 350w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/09/20180620-HeLa-G3BP1-HTT-KD245-Stable-780x586.jpg 780w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/09/20180620-HeLa-G3BP1-HTT-KD245-Stable-250x188.jpg 250w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/09/20180620-HeLa-G3BP1-HTT-KD245-Stable.jpg 960w" sizes="(max-width: 773px) 100vw, 773px" /></p>
<p>As you can see, the levels of HTT are much lower in the HTT knockdown cells compared to the no treatment and H4 (luciferase) controls. I will next use these cell lines to identify any changes in expression of HTT-related proteins.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/generating-htt-knockdown-models/feed/</wfw:commentRss>
		<slash:comments>1</slash:comments>
		</item>
		<item>
		<title>Checking Disruption of Proteins of Interest with Overexpression of HTT</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/checking-disruption-of-proteins-of-interest-with-overexpression-of-htt/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/checking-disruption-of-proteins-of-interest-with-overexpression-of-htt/#comments</comments>
		<pubDate>Mon, 30 Jul 2018 22:45:47 +0000</pubDate>
		<dc:creator><![CDATA[Jolene Ho]]></dc:creator>
				<category><![CDATA[Huntingtin]]></category>
		<category><![CDATA[Huntington’s disease]]></category>
		<category><![CDATA[Jolene Ho]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=1582</guid>
		<description><![CDATA[The experimental set-up and results covered in this post can be found here: 10.5281/zenodo.1323902 As I’ve been having problems overexpression HTT properly in my cancer cell lines, I decided to continue looking at other proteins of interest using our best overexpression model so far, which is in HEK293T cells. Although it may not be a cancer <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/checking-disruption-of-proteins-of-interest-with-overexpression-of-htt/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p style="text-align: justify;">The experimental set-up and results covered in this post can be found here: <a href="http://doi.org/10.5281/zenodo.1323902">10.5281/zenodo.1323902</a></p>
<p style="text-align: justify;">
<p style="text-align: justify;">As I’ve been having problems overexpression HTT properly in my cancer cell lines, I decided to continue looking at other proteins of interest using our best overexpression model so far, which is in HEK293T cells. Although it may not be a cancer cell line, it is very widely used due to the ease of genetic manipulation, as evident in my success with overexpressing HTT in this line, and would be useful for the elucidation of possible mechanisms of action, which can then be validated in other more relevant but difficult cells.</p>
<p style="text-align: justify;">
<p style="text-align: justify;">On my hunt for possible target proteins to look at in my HTT overexpression model, I found several interesting papers on posttranslational modifications of proteins. Posttranslational modifications occur on proteins after they have been translated from RNA to protein and are important in the normal function of proteins, including ensuring that proteins are folded properly and do not aggregate. As the pathological feature of Huntington’s disease is the formation of protein aggregates due to the expanded polyQ region, it is highly relevant to look at changes in posttranslational modifications in the context of huntingtin overexpression.</p>
<p style="text-align: justify;">
<p>&nbsp;</p>
<p style="text-align: justify;"><strong>PRMT5- mediated symmetric arginine dimethylation is attenuated by mutant huntingtin and is impaired in Huntington&#8217;s disease (HD)</strong></p>
<p style="text-align: justify;">Ratovitski T, Arbez N, Stewart JC, Chighladze E, Ross CA.</p>
<p style="text-align: justify;">Cell Cycle. 2015;14(11):1716-29.</p>
<p style="text-align: justify;"><a href="https://doi.org/10.1080/15384101.2015.1033595">https://doi.org/10.1080/15384101.2015.1033595</a></p>
<p style="text-align: justify;">
<p>&nbsp;</p>
<p style="text-align: justify;">This paper showed that arginine dimethylation was decreased by mutant huntingtin in Huntington’s disease. Interestingly, normal HTT, but not mutant HTT, stimulated arginine methylation mediated by PRMT5 activity. The main epigenetic mark mediated by PRMT5 is symmetric dimethylation of arginine (RMe2s). I thus decided to look at the levels of RMe2s in cells with HTT OE to see if there were any changes in PRMT5 activity. As a control, I also looked at asymmetric demethylation of arginine (RMe2a), which should not be affected by changes in PRMT5 activity.</p>
<p style="text-align: justify;">
<p style="text-align: justify;">I also found several other pathways that have been reported to be disrupted by HTT. In particular, HTT is known to be important for nuclear integrity and nuclear export, and several papers looked at how the nuclear lamina is regulated by Huntingtin. I decided to examine the nuclear integrity of HTT OE cells by looking at Lamin B1, which is a protein involved in the nuclear lamina and which was also used in some of these papers to study nuclear integrity.</p>
<p style="text-align: justify;">
<p>&nbsp;</p>
<p style="text-align: justify;">
<p style="text-align: justify;"><strong>Polyglutamine-Expanded Huntingtin Exacerbates Age-Related Disruption of Nuclear Integrity and Nucleocytoplasmic Transport</strong></p>
<p style="text-align: justify;">Gasset-Rosa F, Chillon-Marinas C, Goginashvili A, Atwal RS, Artates JW, Tabet R, Wheeler VC, Bang AG, Cleveland DW, Lagier-Tourenne C.</p>
<p style="text-align: justify;">Neuron. 2017 Apr 5;94(1):48-57.e4.</p>
<p style="text-align: justify;"><a href="https://doi.org/10.1016/j.neuron.2017.03.027">https://doi.org/10.1016/j.neuron.2017.03.027</a></p>
<p style="text-align: justify;">
<p>&nbsp;</p>
<p style="text-align: justify;"><strong>Focal distortion of the nuclear envelope by huntingtin aggregates revealed by lamin immunostaining</strong></p>
<p style="text-align: justify;">Chapple JP, Bros-Facer V, Butler R, Gallo JM.</p>
<p style="text-align: justify;">Neurosci Lett. 2008 Dec 12;447(2-3):172-4. Epub 2008 Oct 1.</p>
<p style="text-align: justify;"><a href="https://doi.org/10.1016/j.neulet.2008.09.075">https://doi.org/10.1016/j.neulet.2008.09.075</a></p>
<p style="text-align: justify;">
<p>&nbsp;</p>
<p style="text-align: justify;"><strong>B10 Nuclear Lamina Is Differentially Altered In Huntington’s Disease Brain Regions</strong></p>
<p style="text-align: justify;">J Neurol Neurosurg Psychiatry 2014;85:A12.</p>
<p style="text-align: justify;">Alcalá R, Creus-Muncunill J, Azkona G, Alberch J, Pérez-Navarro E.</p>
<p style="text-align: justify;"><a href="http://doi.org/10.1136/jnnp-2014-309032.38">http://doi.org/10.1136/jnnp-2014-309032.38</a></p>
<p style="text-align: justify;">
<p>&nbsp;</p>
<p style="text-align: justify;">Huntingtin was also previously reported to be involved in perinuclear localisation and centrosomal organisation. PCNT is a protein that localises to the centrosome and is involved in the formation of the pericentriolar matrix (PCM), which is key for proper centrosome formation and cell cycle progression. Its role in both perinuclear localisation and centrosomal organisation makes it an important candidate to look at in the context of Huntington’s disease. Moreover, it is also involved in ciliogenesis, which has been reported to be disrupted in patients with Huntington’s disease.</p>
<p style="text-align: justify;">
<p>&nbsp;</p>
<p style="text-align: justify;"><strong>Perinuclear localization of huntingtin as a consequence of its binding to microtubules through an interaction with beta-tubulin: relevance to Huntington&#8217;s disease</strong></p>
<p style="text-align: justify;">Hoffner G, Kahlem P, Djian P.</p>
<p style="text-align: justify;">J Cell Sci. 2002 Mar 1;115(Pt 5):941-8.</p>
<p style="text-align: justify;"><a href="http://jcs.biologists.org/content/115/5/941.long">http://jcs.biologists.org/content/115/5/941.long</a></p>
<p style="text-align: justify;">
<p>&nbsp;</p>
<p style="text-align: justify;"><strong>Centrosome disorganization in fibroblast cultures derived from R6/2 Huntington&#8217;s disease (HD) transgenic mice and HD patients</strong></p>
<p style="text-align: justify;">Sathasivam K, Woodman B, Mahal A, Bertaux F, Wanker EE, Shima DT, Bates GP.</p>
<p style="text-align: justify;">Hum Mol Genet. 2001 Oct 1;10(21):2425-35.</p>
<p style="text-align: justify;"><a href="http://doi.org/10.1093/hmg/10.21.2425">http://doi.org/10.1093/hmg/10.21.2425</a></p>
<p style="text-align: justify;">
<p>&nbsp;</p>
<p style="text-align: justify;"><strong>Ciliogenesis is regulated by a huntingtin-HAP1-PCM1 pathway and is altered in Huntington disease</strong></p>
<p style="text-align: justify;">Keryer G, Pineda JR, Liot G, Kim J, Dietrich P, Benstaali C, Smith K, Cordelières FP, Spassky N, Ferrante RJ, Dragatsis I, Saudou F.</p>
<p style="text-align: justify;">J Clin Invest. 2011 Nov;121(11):4372-82. Epub 2011 Oct 10.</p>
<p style="text-align: justify;"><a href="http://doi.org/10.1172/JCI57552">http://doi.org/10.1172/JCI57552</a></p>
<p style="text-align: justify;">
<p>&nbsp;</p>
<p style="text-align: justify;">Finally, I also looked at coilin, a major component of cajal bodies. There have been reports that cajal bodies may play a role in the pathogenesis of CAG repeat diseases such as Huntington’s disease. Furthermore, it is a known substrate of PRMT5 and may be affected indirectly by Huntingtin through methylation by PRMT5.</p>
<p style="text-align: justify;">
<p>&nbsp;</p>
<p style="text-align: justify;"><strong>Interaction between neuronal intranuclear inclusions and promyelocytic leukemia protein nuclear and coiled bodies in CAG repeat diseases</strong></p>
<p style="text-align: justify;">Yamada M, Sato T, Shimohata T, Hayashi S, Igarashi S, Tsuji S, Takahashi H.</p>
<p style="text-align: justify;">Am J Pathol. 2001 Nov;159(5):1785-95.</p>
<p style="text-align: justify;"><a href="http://doi.org/10.1016/S0002-9440(10)63025-8">http://doi.org/10.1016/S0002-9440(10)63025-8</a></p>
<p style="text-align: justify;">
<p>&nbsp;</p>
<p style="text-align: justify;">My final list of proteins to look thus consisted of: RMe2s, RMe2a, Lamin B1, PCNT and coilin, with GAPDH as the loading control.</p>
<p style="text-align: justify;"><img class="alignnone  wp-image-1583" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/07/20180524-HeLa-G3BP1-HEK293T-HTT-OE-300x225.jpg" alt="" width="827" height="620" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/07/20180524-HeLa-G3BP1-HEK293T-HTT-OE-300x225.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/07/20180524-HeLa-G3BP1-HEK293T-HTT-OE-768x576.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/07/20180524-HeLa-G3BP1-HEK293T-HTT-OE-350x263.jpg 350w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/07/20180524-HeLa-G3BP1-HEK293T-HTT-OE-780x586.jpg 780w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/07/20180524-HeLa-G3BP1-HEK293T-HTT-OE-250x188.jpg 250w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/07/20180524-HeLa-G3BP1-HEK293T-HTT-OE.jpg 960w" sizes="(max-width: 827px) 100vw, 827px" /></p>
<p style="text-align: justify;">Unfortunately, none of these proteins showed any significant change in protein levels! It is possible that the HTT overexpression was not good enough, especially for the larger constructs of D4 and D6. However, even comparing between the HTT WT overexpression (D1: Q23) and the PBS control, there is no obvious difference in protein levels. Another possibility is that HTT levels is not limiting in the pathways looked at and hence an overexpression does not affect these proteins of interest. It might thus be more prudent to look at knockdowns of HTT instead.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/checking-disruption-of-proteins-of-interest-with-overexpression-of-htt/feed/</wfw:commentRss>
		<slash:comments>2</slash:comments>
		</item>
		<item>
		<title>CLK2 inhibitor candidates</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/clk2-inhibitor-candidates/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/clk2-inhibitor-candidates/#comments</comments>
		<pubDate>Fri, 06 Jul 2018 18:26:46 +0000</pubDate>
		<dc:creator><![CDATA[Carla Alamillo-Ferrer]]></dc:creator>
				<category><![CDATA[Carla Alamillo]]></category>
		<category><![CDATA[Understudied Kinases]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=1466</guid>
		<description><![CDATA[In my previous post, I presented different CLK2 inhibitors (https://thesgc.github.io/static-openlabnotebooks/clk-chemical-probe/), including structures, and IC50 curves that were generated using a cell target engagement assay called NanoBRET. My colleagues Carrow Wells and Julie Pickett generated these data. The BRET of NanoBRET stands for bioluminescence resonance energy transfer, and it is an in-vitro quantitative technique to identify <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/clk2-inhibitor-candidates/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p style="text-align: justify;">In my previous post, I presented different CLK2 inhibitors (<a href="https://thesgc.github.io/static-openlabnotebooks/clk-chemical-probe/">https://thesgc.github.io/static-openlabnotebooks/clk-chemical-probe/</a>), including structures, and IC<sub>50</sub> curves that were generated using a cell target engagement assay called NanoBRET. My colleagues Carrow Wells and Julie Pickett generated these data. The BRET of NanoBRET stands for bioluminescence resonance energy transfer, and it is an <em>in-vitro</em> quantitative technique to identify target engagement of the drug with the protein of interest. As shown in the picture below, the kinase has to be modified somehow to express the NanoLuc, used as a detector. Next, a promiscuous and fluorescent tracer has to be identified and be introduced to engage with the kinase, which results in a BRET signal. Finally, our test compound, hopefully a selective inhibitor (called the unmodified drug in the picture) is introduced, resulting in a decrease of the BRET signal, due to displacement of the fluorescent tracer from the active site of the tagged kinase.</p>
<p style="text-align: justify;"><img class="aligncenter  wp-image-1467" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/07/nanobretPic-300x118.png" alt="" width="488" height="192" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/07/nanobretPic-300x118.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/07/nanobretPic.png 658w" sizes="(max-width: 488px) 100vw, 488px" /></p>
<p style="text-align: justify;">Compounds CAF022 and CAF061, previously identified as CLK2 inhibitor candidates, were tested using NanoBRET cell engagement. IC<sub>50 </sub>curves were generated with 10 different inhibitor concentrations, providing an IC<sub>50</sub>(CAF022) = 32 nM and IC<sub>50</sub>(CAF061) = 24 nM, demonstrating high affinity CLK2 target engagement in cells.</p>
<p><img class="aligncenter  wp-image-1469" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/07/IC50pic-300x132.png" alt="" width="475" height="209" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/07/IC50pic-300x132.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/07/IC50pic.png 711w" sizes="(max-width: 475px) 100vw, 475px" /></p>
<p style="text-align: justify;">In order to meet the SGC criteria and compounds become a chemical probe, potency is not enough and other chemical/biological factors have to be addressed. Selectivity is a key factor, and the requirement is a &gt;30-fold difference within the family. We are currently assessing kinome wide selectivity of our top candidates. This involves a kinome scan at Discoverx (&gt;400 kinases) testing our inhibitors at a concentration of 1 micromolar.</p>
<p style="text-align: justify;">The picture below shows the kinome scan results of both <strong><span style="text-decoration: underline;">CAF022</span></strong> and <strong><span style="text-decoration: underline;">CAF061</span></strong>.</p>
<p style="text-align: justify;">Disclosure of <strong>CAF022</strong>, this compound is already pretty selective, hitting 8 kinases with &gt;90% of remaining activity, containing 3 false positives (checked by Kd’s follow-up). These 5 kinases include: PIP4K2C, MAPK15, CLK1, STK16 and CLK2 (yellow dots). Furthermore, between 89-75% of remaining activity, only 6 kinases (2 of them false positive) were hit, including NEK6, NEK7, HIPK1 and TGFBR2 (blue dots).</p>
<p style="text-align: justify;">Regarding <strong>CAF061</strong>, this compound shows a cleaner kinome scan, hitting only 5 kinases with &gt;90% of remaining activity (Kd value determination is in progress to check on false positives). These kinases included: HIPK1, HIPK2, CLK1, CLK2 and CLK4 (red pentagons). An extra 5 kinases were found to have a remaining activity between 89-75%, including: HIPK3, STK16, CDKL3, ERK8 and JAK1 (JH2 pseudo-kinase) (yellow triangles). Once again Kd determination to determine true off-targets is in progress.</p>
<p><img class="wp-image-1468 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/07/kinomeScanPic-300x255.png" alt="" width="559" height="475" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/07/kinomeScanPic-300x255.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/07/kinomeScanPic.png 599w" sizes="(max-width: 559px) 100vw, 559px" /></p>
<p style="text-align: justify;">Having found common kinases off-targets, Kd values will be generated for those and other potential off targets and/or in a similar nanoBRET cellular assay to get IC<sub>50</sub>. Having found a selective compound, then we are a step closer to a potential chemical probe. Feel free to suggest further ideas and if you have any questions or comments or want more detail on anything, please let me know and feel free to contact me or any of my SGC colleagues.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/clk2-inhibitor-candidates/feed/</wfw:commentRss>
		<slash:comments>4</slash:comments>
		</item>
		<item>
		<title>The process of HDAC11 Assay Development: Z’-factor</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/the-process-of-hdac11-assay-development-z-factor/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/the-process-of-hdac11-assay-development-z-factor/#respond</comments>
		<pubDate>Wed, 04 Jul 2018 19:40:23 +0000</pubDate>
		<dc:creator><![CDATA[Megha Abbey]]></dc:creator>
				<category><![CDATA[HDACs]]></category>
		<category><![CDATA[Megha Abbey]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=1454</guid>
		<description><![CDATA[As a last step, to check if the assay qualifies for the screening purposes, Z’-factor was calculated. The dataset can be viewed on Zenodo.]]></description>
				<content:encoded><![CDATA[<p>As a last step, to check if the assay qualifies for the screening purposes, Z’-factor was calculated. The dataset can be viewed on <a href="https://zenodo.org/record/1305375#.Wz0iMNVKjIU">Zenodo.</a></p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/the-process-of-hdac11-assay-development-z-factor/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>The process of HDAC11 Assay Development: Effect of DMSO</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/the-process-of-hdac11-assay-development-effect-of-dmso/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/the-process-of-hdac11-assay-development-effect-of-dmso/#respond</comments>
		<pubDate>Tue, 03 Jul 2018 20:23:46 +0000</pubDate>
		<dc:creator><![CDATA[Megha Abbey]]></dc:creator>
				<category><![CDATA[HDACs]]></category>
		<category><![CDATA[Megha Abbey]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=1452</guid>
		<description><![CDATA[For the purpose of compound library screenings, the compounds would be made available in DMSO and thus, when they would be incubated with the protein, the protein solution will now contain DMSO as well. It is important to determine the concentration of DMSO that could be tolerated by the protein such that the protein stays <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/the-process-of-hdac11-assay-development-effect-of-dmso/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>For the purpose of compound library screenings, the compounds would be made available in DMSO and thus, when they would be incubated with the protein, the protein solution will now contain DMSO as well. It is important to determine the concentration of DMSO that could be tolerated by the protein such that the protein stays active. Hence, the effect of varying concentration of DMSO was analyzed on HDAC11 activity. The dataset can be viewed on <a href="https://zenodo.org/record/1304433#.WzvbAdVKjIU">Zenodo</a>.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/the-process-of-hdac11-assay-development-effect-of-dmso/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>The process of HDAC11 Assay Development: Effect of Detergent</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/the-process-of-hdac11-assay-development-effect-of-detergent/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/the-process-of-hdac11-assay-development-effect-of-detergent/#respond</comments>
		<pubDate>Tue, 03 Jul 2018 20:08:56 +0000</pubDate>
		<dc:creator><![CDATA[Megha Abbey]]></dc:creator>
				<category><![CDATA[HDACs]]></category>
		<category><![CDATA[Megha Abbey]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=1449</guid>
		<description><![CDATA[The presence of detergent can prevent the sticking of the fluorophore to the sides of the wells of the plate and thus improve the signals. Hence, the effect of detergent was observed on the overall signals and the activity of HDAC11. The dataset can be viewed on Zenodo.]]></description>
				<content:encoded><![CDATA[<p>The presence of detergent can prevent the sticking of the fluorophore to the sides of the wells of the plate and thus improve the signals. Hence, the effect of detergent was observed on the overall signals and the activity of HDAC11. The dataset can be viewed on <a href="https://zenodo.org/record/1304424#.WzvXn9VKjIU">Zenodo</a>.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/the-process-of-hdac11-assay-development-effect-of-detergent/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>The process of HDAC11 Assay Development: Check for linearity and Km-repeat</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/the-process-of-hdac11-assay-development-check-for-linearity-and-km-repeat/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/the-process-of-hdac11-assay-development-check-for-linearity-and-km-repeat/#respond</comments>
		<pubDate>Tue, 03 Jul 2018 19:38:21 +0000</pubDate>
		<dc:creator><![CDATA[Megha Abbey]]></dc:creator>
				<category><![CDATA[HDACs]]></category>
		<category><![CDATA[Megha Abbey]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=1447</guid>
		<description><![CDATA[The linearity for HDAC11 activity for Boc-Lys-(TFA)-AMC substrate was monitored using the newly optimized buffer conditions. This was followed upon by Km calculation. The dataset can be viewed on zenodo.]]></description>
				<content:encoded><![CDATA[<p>The linearity for HDAC11 activity for Boc-Lys-(TFA)-AMC substrate was monitored using the newly optimized buffer conditions. This was followed upon by Km calculation. The dataset can be viewed on <a href="https://zenodo.org/record/1304365#.WzvQd9VKjIU">zenodo</a>.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/the-process-of-hdac11-assay-development-check-for-linearity-and-km-repeat/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>The process of HDAC11 Assay Development: Effect of reducing agent</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/the-process-of-hdac11-assay-development-effect-of-reducing-agent/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/the-process-of-hdac11-assay-development-effect-of-reducing-agent/#respond</comments>
		<pubDate>Tue, 03 Jul 2018 19:37:07 +0000</pubDate>
		<dc:creator><![CDATA[Megha Abbey]]></dc:creator>
				<category><![CDATA[HDACs]]></category>
		<category><![CDATA[Megha Abbey]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=1445</guid>
		<description><![CDATA[An effect of adding reducing agent on the HDAC11 activity was monitored. Dataset can be found on Zenodo.]]></description>
				<content:encoded><![CDATA[<p>An effect of adding reducing agent on the HDAC11 activity was monitored. Dataset can be found on <a href="https://zenodo.org/record/1304316#.WzvQIdVKjIU">Zenodo</a>.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/the-process-of-hdac11-assay-development-effect-of-reducing-agent/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
	</channel>
</rss>
