<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>Past Scientist &#8211; openlabnotebooks.org</title>
	<atom:link href="https://thesgc.github.io/static-openlabnotebooks/category/past-scientist/feed/" rel="self" type="application/rss+xml" />
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<description>A growing team of groundbreaking scientists around the world are now sharing their lab notebooks online</description>
	<lastBuildDate>Mon, 01 Jul 2019 16:49:26 +0000</lastBuildDate>
	<language>en-GB</language>
	<sy:updatePeriod>hourly</sy:updatePeriod>
	<sy:updateFrequency>1</sy:updateFrequency>
	<generator>https://wordpress.org/?v=4.9.10</generator>

<image>
	<url>https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/cropped-lab-jar02-32x32.png</url>
	<title>Past Scientist &#8211; openlabnotebooks.org</title>
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<width>32</width>
	<height>32</height>
</image> 
	<item>
		<title>CRISPR knockout of EZH1 in AML cell line</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/crispr-knockout-of-ezh1-in-aml-cell-line/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/crispr-knockout-of-ezh1-in-aml-cell-line/#respond</comments>
		<pubDate>Fri, 10 May 2019 21:37:11 +0000</pubDate>
		<dc:creator><![CDATA[Genna Luciani]]></dc:creator>
				<category><![CDATA[Genna Luciani]]></category>
		<category><![CDATA[Leukemia]]></category>
		<category><![CDATA[PRC2]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2855</guid>
		<description><![CDATA[Hello again. My apologies for the delay in posting &#8211; I&#8217;ve been busy with other projects lately. As a follow up to my previous post, we wanted to further confirm our EZH1 antibody and test it in a knockout line. To do this, we generated a CRISPR knockout of EZH1 in an AML cell line <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/crispr-knockout-of-ezh1-in-aml-cell-line/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Hello again. My apologies for the delay in posting &#8211; I&#8217;ve been busy with other projects lately. As a follow up to my <a href="https://thesgc.github.io/static-openlabnotebooks/finally-finding-an-antibody-to-detect-ezh1-protein-expression/">previous</a> post, we wanted to further confirm our EZH1 antibody and test it in a knockout line. To do this, we generated a CRISPR knockout of EZH1 in an AML cell line that will be a great tool for future experiments. To generate the knockout line, we were helped by Fraser Soares, a postdoctoral fellow in Hansen He&#8217;s lab. He provided us with an AML cell line (OCI-AML-3) that expresses Cas9, along with guide RNAs that would allow us to knockout EZH1. Once the knockouts were generated, we tested the expression of EZH1 and EZH2. We determined that the guides knocked out EZH1, but had no effect on related EZH2 expression levels (Figure 1). We can now conclude we have a specific EZH1 antibody and EZH1 knockout lines that we can use in the future. For full experimental details, please see <a href="https://zenodo.org/record/2718713#.XNXV6TBKjIU">Zenodo</a> post.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/05/EZH1-crispr-2-300x100.png" alt="" width="342" height="114" class="wp-image-2857 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/05/EZH1-crispr-2-300x100.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/05/EZH1-crispr-2-768x255.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/05/EZH1-crispr-2-1024x340.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/05/EZH1-crispr-2.png 1462w" sizes="(max-width: 342px) 100vw, 342px" /></p>
<p>Figure 1: Westerns of Crispr knockouts in OCI-AML-3 Cas9 cells with either control guide RNA (sgLacZ) or guide RNA to EZH1. Levels of EZH1 were undetectable with knockout (A) and did not affect EZH2 levels (B).</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/crispr-knockout-of-ezh1-in-aml-cell-line/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>PhD position available in malaria parasite genomics and transcriptomics</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/phd-position-available-in-malaria-parasite-genomics-and-transcriptomics/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/phd-position-available-in-malaria-parasite-genomics-and-transcriptomics/#respond</comments>
		<pubDate>Tue, 23 Apr 2019 13:49:53 +0000</pubDate>
		<dc:creator><![CDATA[Antoine Claessens]]></dc:creator>
				<category><![CDATA[Antoine Claessens]]></category>
		<category><![CDATA[Malaria]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2737</guid>
		<description><![CDATA[Summary: –       3-year PhD funded by CNRS for genomics and transcriptomics data analysis of the Plasmodium falciparum malaria parasite. –       In a young dynamic team based in Montpellier, South of France, with many worldwide collaborators –       Keywords: population genetics &#38; genomics, single-cell RNA-seq, molecular epidemiology, antigenic variation.   Our research Most malaria deaths are caused <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/phd-position-available-in-malaria-parasite-genomics-and-transcriptomics/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p><strong>Summary:</strong></p>
<p><strong>–       3-year PhD funded by CNRS for genomics and transcriptomics data analysis of the <em>Plasmodium falciparum</em> malaria parasite.</strong></p>
<p><strong>–       In a young dynamic team based in Montpellier, South of France, with many worldwide collaborators</strong></p>
<p><strong>–       Keywords: population genetics &amp; genomics, single-cell RNA-seq, molecular epidemiology, antigenic variation.</strong></p>
<p><strong> </strong></p>
<p><strong>Our research</strong></p>
<p>Most malaria deaths are caused by <em>Plasmodium falciparum</em>, a unicellular eukaryote parasite transmitted by Anopheles mosquitoes. Despite the heavy burden of the disease, most human infections are asymptomatic (afebrile) and can last for months/years. In regions where malaria is seasonal, such as in The Gambia, West Africa, these asymptomatic infections probably act as a reservoir during the dry season when there is no or little transmission. <strong>Long-lasting infections could be turned to our advantage if we study them as a new host-pathogen interaction model</strong>, by regular sampling of an asymptomatic volunteer, investigating the parasite biology in its natural environment. More details on our research can be found on this <a href="https://thesgc.github.io/static-openlabnotebooks/plasmodium-falciparum-asymptomatic-infections-in-search-of-the-elephant-in-the-room/">on this post</a>.</p>
<p>From 2014 to 2017, we collected fingerprick blood samples every 3 months in a population of ~1000 inhabitants in a Gambian village. At the start of the 2017 dry season, we also recruited a cohort of 50 volunteers who were all asymptomatic carriers for <em>P. falciparum</em>. Monthly blood samples were taken for 6 months over the entire dry season. A total of 435 blood samples were collected, including this time series of monthly samples from the same volunteers over a 6-month period. <strong>This dataset represents a unique opportunity to address novel questions about the parasite biology</strong>. More specifically, we are interested in discovering how the parasite establishes a chronic infection, through changes at the transcriptomic levels.</p>
<p><strong>Your projects</strong></p>
<p>Projects here under are only a suggestion, the exact project will be defined based on the student’s abilities and expertise. Your work will be entirely computer-based, but you will be part of a  team mixing ‘dry’ and ‘wet’ lab research.</p>
<p><strong>Project 1: <em>P. falciparum</em> population genetics and genomics</strong></p>
<p>As outlined above, we collected blood samples all year round from a village over two and a half year. <em>P. falciparum</em> DNA was extracted and genotyped and/or whole genome sequenced in collaboration with MalariaGEN at the Sanger Institute. A dataset of about 400 parasite DNA barcodes and 100 parasite genomes is now available. Here we want to describe the first parasite genomes sequenced from asymptomatic infections, and investigate the impact of the dry season selective pressure on the parasite population. This longitudinal approach also allows us to tackle the question of the duration of a <em>P. falciparum</em> chronic infection. Preliminary analysis revealed, for example, that a child was infected asymptomatically with the same <em>P. falciparum </em>strain for over 2.5 years. This overturns the dogma on duration of a <em>P. falciparum</em> infection.</p>
<p>Your task is to fully characterise the parasite population genetic diversity, using genotyping, genomic and epidemiologic data. The analysis will include PCA, Fst, chromosome painting, EHH, iHS, etc.</p>
<p><strong>Project 2: Can <em><u>P. falciparum</u></em> sense its environment and adapt to it via transcriptional regulation?</strong></p>
<p>In areas where rains are seasonal, two distinct patterns of malaria fluctuate, from high prevalence with a wide range of clinical outcome in the wet season, to low prevalence mainly asymptomatic in the dry season. The mechanisms allowing the parasite to survive in chronic infections during the dry season, and restart transmission as vector population increases with the ensuing wet season, remain to be investigated. We hypothesize that <em>P. falciparum</em> is able to sense its environment and dampens its virulence with seasonality.</p>
<p>In collaboration with colleagues at the London School of Hygiene &amp; Trop Med, we are currently optimizing a single-cell RNA-seq approach to <em>P. falciparum</em>.</p>
<p>Your task, using ‘bulk’ and single-cell parasite transcriptomes, is to identify differentially regulated genes in the dry vs wet season, in symptomatic vs asymptomatic infections, and identify putative candidates involved in cell growth. The analysis will include normalization, PCA for clustering and tSNE for representation, correlating bulk and single cell data with Pearson correlation, drawing gene co-expression networks within each identified cluster (subpopulation), etc.</p>
<p>The transcriptome dataset, which will be linked with <em>in vitro </em>growth assays, will be crucial in understanding how the parasite successfully establishes a chronic infection over the dry season, yet triggers a new epidemic over the following transmission season. It also has the potential of identifying dormant parasites, a discovery that would guide malaria elimination strategies.</p>
<p><strong>Other projects:</strong></p>
<p>– Population genomics with over 100 <em>P. falciparum</em> genomes collected from severe malaria or uncomplicated cases in Benin. In collaboration with Gwladys Bertin (Paris), Joe Zhu and Jacob Garcia (Oxford, UK).</p>
<p>– Identification of the micro-indels and structural variant mutation rates in <em>P. falciparum</em> in vitro using data from “clone trees” (Hamilton 2016 NAR and unpublished data).</p>
<p>– And many, many more…</p>
<p>&nbsp;</p>
<p><strong>Your skills (essentials)</strong></p>
<ul>
<li>A Master in biostatistics / bioinformatics / population genetics, or in molecular biology with a strong background in biostatistics and programming</li>
<li>Demonstrated experience in “Omics” data analysis</li>
<li>Able to work independently</li>
<li>Great communication skills in English</li>
<li>Stay focused on answering a specific biological question, from the initial analysis to publication</li>
<li>Passionate about scientific discoveries</li>
</ul>
<p><strong>Your skills (desirable)</strong></p>
<ul>
<li>Knowledge about malaria biology and population genetics</li>
<li>Wet-lab experience is a plus, but not essential</li>
<li>Willing to supervise a Master student in bioinformatics / epidemiology</li>
<li>Participate actively in the lab and institute life</li>
</ul>
<p><strong>Who we are</strong></p>
<p>Antoine Claessens (PI) is a malariologist who trained at Edinburgh University, the Sanger Institute, LSHTM and the MRC-Gambia. He recently joined Montpellier University as a “Chargé de Recherche INSERM”. He was awarded an ATIP and an ANR-JC.</p>
<p>Currently the team consists of one post-doc and one Master student. Two PhD students will be recruited around Sep 2019, one for lab-based research and one for data analysis (this advert).  Each year, 2 Master students in bioinformatics / epidemiology will join the team for up to 6 months. We are likely to recruit another post-doc towards late 2019.</p>
<p><strong>What you can expect / what we can offer</strong></p>
<p>The 3-year PhD will be funded by CNRS (substantially higher salary than other sources of funding). You will be part of <a href="http://www.dimnp.univ-montp2.fr/">DIMNP</a> and you will closely interact with experienced bioinformaticians from the bioinformatics platform and with population geneticists from <a href="https://www.mivegec.ird.fr/fr/">MIVEGEC</a>. Both ‘UMR’ are very international, with highly competent and friendly scientists. More generally, Montpellier is a large hub for research in Life Sciences, particularly in the field of evolutionary biology.</p>
<p>I will try my best to transmit my passion about <em>P. falciparum</em> biology; you might find yourself dreaming about <em>var</em> genes too. As we will be a small team, you can expect more of my time dedicated to you, but I do not want to micromanage. I expect you to work hard, yet you will have freedom on working hours etc. All our current research projects are in collaboration with international laboratories, there is an opportunity to work in the USA/UK for a short period of time. If you are planning to stay in academic research, I will help you secure a post-doc position.</p>
<p>If you are from abroad: Being able to speak French is not a requirement; your colleagues will speak English fluently. We will help you with the administrative task of moving to France. Many aspects of the cost of life, such as childcare, are relatively cheap. Most importantly, the quality of life in South of France is hard to beat!</p>
<p><strong>Key collaborators and recent publications:<br />
</strong>Umberto D’Alessandro (MRC-Gambia)<br />
Teun Bousema (Radboud UMC, Netherlands)<br />
David Conway (LSHTM, UK)<br />
Dominic Kwiatkowski, MalariaGEN, Lia Chappell (Sanger Institute)<br />
Franck Prugnolle and Virginie Rougeron (MIVEGEC, Montpellier)<br />
Arthur Talman (Sanger / MIVEGEC)<br />
Silvia Portugal (Heidelberg, Germany)<br />
Ian Cheeseman (Texas Bio Med, USA)</p>
<table width="510">
<tbody>
<tr>
<td width="326"><strong>Selected publications:</strong></td>
<td width="184"><strong>Key discovery</strong></td>
</tr>
<tr>
<td width="326"><u>Claessens A</u>, Adams Y, …, Bozdech Z and Rowe JA. Group A-like PfEMP1s mediate cytoadherence to human brain endothelial cells<em>.</em> <strong>Proc Natl Acad Sci USA</strong> 2012.</td>
<td width="184">How <em>P. falciparum</em> bind to blood microvessels in vitro (microarray transcriptomic approach)</td>
</tr>
<tr>
<td width="326"><u>Claessens A</u>*, Hamilton W*, …, Rayner J, Kwiatkowski D. Generation of antigenic diversity in Plasmodium falciparum by structured rearrangement of var genes during mitosis. <strong>PLOS Genetics</strong> 2014. *joint first author.</td>
<td width="184">New chimeric <em>var</em> gene sequences are generated during parasite growth <em>in vitro</em>.</td>
</tr>
<tr>
<td width="326">Hamilton W*, <u>Claessens A</u>*, Otto T, …, Rayner J, Kwiatkowski D. Extreme mutation bias and high AT content in Plasmodium falciparum<em>.</em> <strong>Nucleic Acid Research</strong> 2016<em>. </em></td>
<td width="184">An excess of G:C to A:T transitions explains the AT-richness of <em>P.f.</em> genome</td>
</tr>
<tr>
<td width="326"><u>Claessens A</u>, Affara M, Assefa S, Kwiatkowski D &amp; Conway D. Culture adaptation of malaria parasites selects for convergent loss-of-function mutants<em>. </em><strong>Scientific Reports</strong> 2017<em>. </em></td>
<td width="184"><em>In vitro</em> culture selects for loss-of-function mutations (genomic approach)</td>
</tr>
<tr>
<td width="326"><u>Claessens A*</u>, Harris LM*, Stanojcic S*, Merrick C. RecQ helicases in the malaria parasite Plasmodium falciparum affect genome stability, gene expression patterns and DNA replication dynamics. <strong>PLOS Genetics</strong> 2018.</td>
<td width="184">Helicase PfWRN maintains genome integrity</td>
</tr>
</tbody>
</table>
<p>&nbsp;</p>
<p><strong>Interested in applying? Please send an email to <a href="mailto:antoineclaessens@gmail.com">antoineclaessens@gmail.com</a> with subject “PhD #0002 – keep me updated”. Starting date: September 2019 (to be discussed).</strong></p>
<p>&nbsp;</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/phd-position-available-in-malaria-parasite-genomics-and-transcriptomics/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Three new ALK2 structures, one of them not the one I thought!</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/three-new-alk2-structures-one-of-them-not-the-one-i-thought/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/three-new-alk2-structures-one-of-them-not-the-one-i-thought/#respond</comments>
		<pubDate>Tue, 19 Feb 2019 17:46:47 +0000</pubDate>
		<dc:creator><![CDATA[Roslin Adamson]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Brain Tumours]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>
		<category><![CDATA[Roslin Adamson]]></category>
		<category><![CDATA[SGC Univ. of Oxford]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2507</guid>
		<description><![CDATA[I’ve not posted in a while, mainly because things have been really busy, but good busy, as I now have 3 new datasets/structures, of ALK2 with M4K2118, M4K2149 and, amazingly, M4K2009. As you know, this is our lead candidate, which until now has not behaved very well in crystal trials, and it hasn’t been possible <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/three-new-alk2-structures-one-of-them-not-the-one-i-thought/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>I’ve not posted in a while, mainly because things have been really busy, but good busy, as I now have 3 new datasets/structures, of ALK2 with M4K2118, M4K2149 and, amazingly, M4K2009. As you know, this is our lead candidate, which until now has not behaved very well in crystal trials, and it hasn’t been possible to get decent crystals. In my last post I told you how I had somehow managed to mix up the TNIK and ACVR1/ALK2 tubes when I was setting up the plates, and accidentally put ACVR1 into a coarse screen and TNIK into a fine screen for ACVR1. Well, after I got lovely big crystals for what I thought was ACVR1 with M4K3003, I sent them to the synchrotron, got beautiful data to 1.4 Å, phased it, and lo and behold! It was M4K2009 snugly fitted into the binding pocket (see the ligand density after molecular replacement below). Of course, I didn’t notice this was a trimethoxy containing compound (how??) before I excitedly emailed everyone to tell them I had M4K3003, and then had to retract … However, the story ends well as M4K2009 is really important.</p>
<p><img class="aligncenter  wp-image-2511" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/M4k2009-1024x576.jpg" alt="" width="904" height="509" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/M4k2009-1024x576.jpg 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/M4k2009-300x169.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/M4k2009-768x432.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/M4k2009.jpg 1280w" sizes="(max-width: 904px) 100vw, 904px" /></p>
<p>Additionally, I got data for M4K2149 (~2.6 Å), one of the amide head group containing compounds we’re interested to see, and M4K2118 (~1.4 Å), which is doing well in some of the other assays too. I’ve put the .mtz files <a href="https://zenodo.org/record/2573142#.XGw9g-j7SUk">here</a> on Zenodo if you want to play with high resolution data, and some images below showing the ligand density and the ligands binding in the pocket. The amide head group of M4K2149 appears to preserve the water network in the pocket that we’ve observed in other structures, but as the resolution is too low for waters, we can’t tell til we get higher resolution data. Which hopefully will be this week, as I’ve grown some really pretty crystals for M4K2149 that I hope will diffract better. These appeared quite by accident again, as I’d designed a fine screen around the D5 well condition of my mega composite screen, but neglected to notice when I validated the screen that I’d put 10 x less ammonium sulphate in one row than intended – should have been 0.9 M, but ended up being 0.09 M. I should probably start calling myself ‘the accidental crystallographer’ at this rate.</p>
<p><img class="aligncenter  wp-image-2509" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/M4K2118-2149-1024x576.png" alt="" width="883" height="497" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/M4K2118-2149-1024x576.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/M4K2118-2149-300x169.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/M4K2118-2149-768x432.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/M4K2118-2149.png 1280w" sizes="(max-width: 883px) 100vw, 883px" /></p>
<p>In the meantime, I’ve got some small crystals for M4K3007, another structure we really want, and some pretty ones for M4K2184, another amide head group compound. I’ve followed up the M4K3007, just waiting to see if anything appears, and will send the tiny ones to the synchrotron while waiting.</p>
<p><img class="aligncenter  wp-image-2508" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/crystal-pics-1024x576.png" alt="" width="849" height="478" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/crystal-pics-1024x576.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/crystal-pics-300x169.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/crystal-pics-768x432.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/crystal-pics.png 1280w" sizes="(max-width: 849px) 100vw, 849px" /></p>
<p>I’m also very happy to say that our new M4K2117 crystal system for XChem fragment screening has worked very nicely so far, in the sense that the crystals are large and relatively robust and can handle soaking in low salt conditions as well as compounds with DMSO for a few hours. They then still diffract to around 1.2-1.8 Å, consistently in the same space group, C 1 2 1. This is a vast improvement over the previous system with LDN-193189. We did a solvent characterisation and pre-screen a couple of weeks ago with 100 crystals, and of the 20 or so hits we got, none was in the allosteric pocket we are looking for. This could be because of the 1.4 M ammonium sulphate and 0.2 M Na/K tartrate I have in there, which might prevent access of fragments to the pocket. Thus, using the remarkable shifter that Nathan Wright at SGC designed, we’ll be transferring loads of crystals from grow plates to low salt soak plates before doing the compound soaks this week, with the same 100 compounds, in order to check if the hit rate/location changes.</p>
<p>I will mention though, for anyone who may never have noticed this, the DMSO concentration in the protein solution for crystallisation is very important! I had previously been using Batch 1 of M4K2117 for my co-crystallisation, but it ran out so I used Batch 2, which was a 50 mM stock, as opposed to the previous 25 mM stock. The plates I set up with this, the same in every other way, gave showers of rather ugly, crunchy, small crystals. So I tried a few things – diluted the stock to 25 mM in DMSO and set up plates with that, with both lower and higher concentrations of protein (8 and 14 mg/ml along with the usual 10-11 mg/ml), as well as using the stock as is, with lower and higher protein concentrations. The lower stock concentrations brought back the big, perfect, beautiful crystals from before, the higher stock, no matter the protein concentration, gave ugly chunks and showers. The final concentration of compound was the same in all, so clearly the DMSO percentage is a very important factor.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/three-new-alk2-structures-one-of-them-not-the-one-i-thought/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Last post on Split-SCN1A</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/last-post-on-split-scn1a/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/last-post-on-split-scn1a/#respond</comments>
		<pubDate>Sun, 10 Feb 2019 22:47:17 +0000</pubDate>
		<dc:creator><![CDATA[Wen Yih Aw]]></dc:creator>
				<category><![CDATA[Catalyst UNC Chapel Hill]]></category>
		<category><![CDATA[Dravet syndrome]]></category>
		<category><![CDATA[Gene therapy for Dravet Syndrome]]></category>
		<category><![CDATA[Wen Yih Aw]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2452</guid>
		<description><![CDATA[The idea for split-intein based SCN1A reconstitution started when I first learned of the split-Cas9 system. Long story short, packaging Cas9 with a guideRNA sequence into an AAV vector is pushing the packaging capacity of AAV vector. To circumvent this issue, the authors design a split-Cas9 system where the coding sequence for Cas9 is split <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/last-post-on-split-scn1a/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>The idea for split-intein based SCN1A reconstitution started when I first learned of the split-Cas9 system. Long story short, packaging Cas9 with a guideRNA sequence into an AAV vector is pushing the packaging capacity of AAV vector. To circumvent this issue, the authors design a split-Cas9 system where the coding sequence for Cas9 is split into two fragments, each fused to a split-inteins sequence. When co-expressed in cells, the split-inteins can function as glue joining two protein fragments together.</p>
<p>In an attempt to “shrink” SCN1A to be packageable in AAV vectors, I ran into several design and cloning issues, ultimately leading to the decision to terminate this project. Cas9 is a well-studied cytoplasmic protein with structural information while SCN1A is a large multi-pass transmembrane protein that weaves in and out of the plasma membrane. While success packaging and application of the split-Cas9 system for gene editing is encouraging, my attempt to translate similar design to SCN1A, without structural information, proves to be an oversight. Design aside, the coding sequence for human SCN1A is difficult to maintain. The SCN1A cDNA sequence tends to accumulate deletions and rearranges when transformed and propagated in E. coli. One way to improve plasmid stability and reduce unwanted rearrangement is by propagating plasmid in E. coli strains with recA1 genotype at room temperature. recA1 confers a loss of function in RecA1, a protein that is essential for homologous recombinational DNA repair in E. coli. While this strategy helps to alleviate the problem, it is time-consuming and eventually, I decided to make use of a codon-optimized human SCN1A cDNA as the cloning template for split-SCN1A constructs. While this solves the plasmid stability issue, the codon-optimized cDNA has not been well characterized and while the proteins expressed reconstituted, they failed to traffic to the correct cellular compartment. These raise an issue that needs to be addressed – how to propagate mutation-free gene therapy vectors with SCN1A sequence in large scale, have any of us managed to move the project to treatment pipeline.</p>
<p>Thank you all for reading. While this project didn&#8217;t work out as planned, I truly believe that this is an exciting time for Dravet Syndrome. With recent positive Fenfluramine clinical trial results, exciting new drugs in the pipeline, and cross-continental efforts in pushing various gene therapy approaches for treating Dravet, I am hopeful that we can see these exciting results moving into medical treatment in the foreseeable future.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/last-post-on-split-scn1a/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Welcome to the SGC-Karolinska side of ALS-RAP: the scientists, the screening, and the antibody selections</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/welcome-to-the-sgc-karolinska-side-of-als-rap-the-scientists-the-screening-and-the-antibody-selections/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/welcome-to-the-sgc-karolinska-side-of-als-rap-the-scientists-the-screening-and-the-antibody-selections/#respond</comments>
		<pubDate>Thu, 31 Jan 2019 23:10:29 +0000</pubDate>
		<dc:creator><![CDATA[Carolyn Marks]]></dc:creator>
				<category><![CDATA[ALS-Reproducible Antibody Platform (ALS-RAP)]]></category>
		<category><![CDATA[Amyotrophic Lateral Sclerosis (ALS)]]></category>
		<category><![CDATA[Carolyn Marks]]></category>
		<category><![CDATA[SGC Karolinska]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2419</guid>
		<description><![CDATA[Hello fellow scientists and open lab notebook readers! I’m Carolyn Marks and I’m working in the ALS-RAP partnership with Susanne Gräslund, who leads the Recombinant Antibodies Group at SGC-Karolinska in Stockholm, Sweden. SGC-Karolinska has had a long-standing collaboration with Helena Persson Lotsholm, Director of the Human Antibody Therapeutics Facility within the Drug Discovery and Development <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/welcome-to-the-sgc-karolinska-side-of-als-rap-the-scientists-the-screening-and-the-antibody-selections/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p style="text-align: justify;">Hello fellow scientists and open lab notebook readers!</p>
<p style="text-align: justify;">I’m Carolyn Marks and I’m working in the ALS-RAP partnership with <a href="https://thesgc.org/profile/karolinska/sgraslund">Susanne Gräslund</a>, who leads the Recombinant Antibodies Group at <a href="https://www.thesgc.org/karolinska">SGC-Karolinska</a> in Stockholm, Sweden. SGC-Karolinska has had a long-standing collaboration with <a href="https://www.scilifelab.se/facilities/human-antibody-therapeutics/">Helena Persson Lotsholm</a>, Director of the Human Antibody Therapeutics Facility within the <a href="https://www.scilifelab.se/platforms/ddd">Drug Discovery and Development Platform at Science for Life Laboratory in Stockholm</a>. I am fortunate to work with and learn from several experts to generate the highest quality antibodies for the ALS-Reproducible Antibody Platform. I joined the <a href="https://www.als-rap.org/team">ALS-RAP team</a> in September of last year and from here on out I’ll be sharing live updates of our work with this community.</p>
<p style="text-align: justify;">Perhaps I should tell you a little bit about me, what we do at SGC-Karolinska, and how I got here. I’m a wet lab kind of gal and a neuroscientist at heart with a lot of years of experience under my belt. The methods that I use here at SGC-Karolinska for the ALS-RAP are broad and range from phage display technology to cloning to high-throughput screening to patient-derived cell-based assays. Actually, many of the methods that I&#8217;m using now to generate and validate recombinant antibodies for the ALS-RAP, I used for many years to study neurodegenerative diseases, including ALS.</p>
<p style="text-align: justify;">I´m originally from Texas where my love of neurophysiology began, and where I was initially trained as an electrophysiologist. From there, I deferred medical school and took an internship at NINDS at the NIH to have a break before med school began. My 1-year internship turned into two, and I quickly understood why everyone at NIH kept telling me “Carolyn, you reek of a PhD!” I soon found the NIH-Karolinska Graduate Partnerships Program and hopped on a plane. That&#8217;s how the girl from Texas ended up in Sweden. I received my PhD in 2012 from the NIH-Karolinska Collaborative Doctoral Program in Neuroscience; and I recently completed my Post-Doc, also at Karolinska, characterizing the ubiquitin proteasome system as therapeutic targets in neurodegenerative diseases.</p>
<p style="text-align: justify;">How did I find my way to the SGC, you might wonder? In my research over the years, I have experienced first-hand the need for reliable high-quality antibodies – and the lack thereof, as well. I hate that we as scientists are limited by the tools that companies provide. These limitations are crippling our research. Sadly, the ones who suffer the most are the patients. This was my motivation for joining SGC-Karolinska and particularly this project. I wanted to make better tools for the research community. Consequently, I believe that better research tools will yield better science and hopefully better therapeutics for the patients who need it. That&#8217;s why I&#8217;m here.</p>
<p style="text-align: justify;">How does the <a href="https://www.als-rap.org/overview">ALS-RAP</a> work? Our team at Karolinska is only 1 of many. We work together primarily with scientists at <a href="https://thesgc.github.io/static-openlabnotebooks/introducing-the-team-the-gileadi-group-in-oxford/">Oxford</a> and in <a href="https://www.mcgill.ca/neuro/research/researchers/thomas-m-durcan-phd">Montreal</a>. The ALS-RAP was created by three major ALS charities from Canada, the UK, and the US. The long-term goal of this partnership is to provide the ALS research community with reproducible, thoroughly validated antibodies for selected ALS-related proteins. This includes testing existing commercial antibodies as well as generating new ones. The work is done in three different labs, as recently outlined by my fellow ALS-RAP partner and open-lab-notebooker, <a href="https://thesgc.github.io/static-openlabnotebooks/validated-antibodies-for-als-research-the-als-rap-project/">Opher Gileadi</a>.</p>
<p style="text-align: justify;">In the Stockholm team, we use phage display technology to create recombinant antibodies. Don´t know what phage display is? The inventors of this technique were actually awarded the <a href="https://www.nobelprize.org/prizes/chemistry/2018/summary/">Nobel Prize in Chemistry in 2018</a>. While this technology has many uses, we use phage display to generate recombinant antibodies. How does it work you might wonder? When we receive antigens of our ALS proteins from our friends in <a href="https://www.ndm.ox.ac.uk/principal-investigators/researcher/opher-gileadi">Oxford</a>, we then use the phage display technique to select out antibodies that binds these antigens. We use extensive libraries of human antibody candidates where high-quality binders can be obtained with the phage display method. We then perform several rounds of screening and selection to ensure that we get the best possible antibodies. We then evaluate their specificity and affinity and finally we validate the antibodies using immunoprecipitation followed by mass spectrometry. It sounds simple, but trust me it’s a thorough process! Last but not least, we will release the sequence of the best antibody for each target on our website and make it available to anyone who wants to use it. Our collaborators in Montreal are creating CRISPR-knockouts of these targets in both standard cell lines and iPSCs for further validation and application specificity.</p>
<p style="text-align: justify;">We have a really exciting <a href="https://als-rap.org/target-list">target list</a>, built by the help of our ALS-RAP advisory board. Check it out and see if your favorite protein or that much needed antibody is on our list. If not, click <a href="https://als-rap.org/your-input">here</a> and request it. So far, we have generated and selected our first set of antibodies for 7 of these targets. Now onto primary and secondary validation.</p>
<p style="text-align: justify;">Stay tuned to find out who the lucky 7 are…</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/welcome-to-the-sgc-karolinska-side-of-als-rap-the-scientists-the-screening-and-the-antibody-selections/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Further Investigation of the Reduction of HTT Protein through Chemical Inhibition</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/further-investigation-of-the-reduction-of-htt-protein-through-chemical-inhibition/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/further-investigation-of-the-reduction-of-htt-protein-through-chemical-inhibition/#respond</comments>
		<pubDate>Tue, 15 Jan 2019 18:56:28 +0000</pubDate>
		<dc:creator><![CDATA[Jolene Ho]]></dc:creator>
				<category><![CDATA[Huntingtin]]></category>
		<category><![CDATA[Huntington’s disease]]></category>
		<category><![CDATA[Jolene Ho]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2378</guid>
		<description><![CDATA[The experimental set-up and results covered in this post can be found here: 10.5281/zenodo.2541053. A lot has happened in the past few months and it is with mixed feelings that I am finishing up my postdoc here at SGC! I feel very privileged to have been part of the great science going on at SGC <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/further-investigation-of-the-reduction-of-htt-protein-through-chemical-inhibition/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p style="text-align: justify;">The experimental set-up and results covered in this post can be found here: <a href="http://doi.org/10.5281/zenodo.2541053">10.5281/zenodo.2541053</a>.</p>
<p style="text-align: justify;">A lot has happened in the past few months and it is with mixed feelings that I am finishing up my postdoc here at SGC! I feel very privileged to have been part of the great science going on at SGC and a pioneer of the Open Notebook initiative here, contributing to open science. Nonetheless, the science has to go on! Hopefully, my posts in this Open Notebook have provided new perspectives to people working on similar projects and trying to learn more about Huntington’s Disease and huntingtin (HTT), inspired people to strive on for science despite the negative and/or inconclusive data, or at least given some insight to how life on the bench is like in general.</p>
<p style="text-align: justify;">Previously, I showed that the use of PRMT5 inhibitors from the SGC probe library, namely GSK591 and LLY283, were able to decrease levels of HTT in HeLa cells. SGC2096, the inactive control to GSK591, was not able to decrease HTT protein levels. This, together with the different mechanisms of action of GSK591 and LLY283 in inhibiting PRMT5, demonstrates that it is indeed through the specific inhibition of PRMT5 that HTT protein levels were decreased. So far, I have not come across any drug that is able to regulate the levels of HTT protein. As such, the inhibition of PRMT5 to reduce the expression of HTT at the protein level presents a potential and exciting therapeutic avenue for the treatment of Huntington’s disease.</p>
<p style="text-align: justify;">The findings that PRMT5 inhibition itself can affect the expression of HTT at the protein level suggests that there might be some interaction and possible feedback loop in play between PRMT5 and HTT. We thus further investigated how PRMT5 inhibition may contribute to the decrease in HTT protein levels to elucidate the mechanisms involved in this process. As PRMT5 is the main enzyme known to mediate symmetric demethylation of arginine (SDMA), we thus checked how the levels of SDMA are affected by PRMT5 inhibition and HTT knockdown.</p>
<p style="text-align: justify;"><img class="alignnone wp-image-2379" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/HeLa-G3BP1-PRMT5i-in-HTT-KD-300x225.jpg" alt="" width="771" height="578" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/HeLa-G3BP1-PRMT5i-in-HTT-KD-300x225.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/HeLa-G3BP1-PRMT5i-in-HTT-KD-768x576.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/HeLa-G3BP1-PRMT5i-in-HTT-KD-350x263.jpg 350w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/HeLa-G3BP1-PRMT5i-in-HTT-KD-780x586.jpg 780w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/HeLa-G3BP1-PRMT5i-in-HTT-KD-250x188.jpg 250w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/HeLa-G3BP1-PRMT5i-in-HTT-KD.jpg 960w" sizes="(max-width: 771px) 100vw, 771px" /></p>
<p style="text-align: justify;">Unsurprisingly, PRMT5 inhibition with GSK591 or LLY283 led to the decrease in SDMA. However, we also noticed a slight decrease in SDMA across all 3 HTT knockdown samples! Although this decrease is marginal, its occurrence across all the HTT knockdown samples we tested gives confidence to this observation and corroborates the hypothesis that PRMT5 and HTT are part of a feedback loop rather than that HTT is simply downstream of PRMT5, since the inhibition of PRMT5 leads to a decrease in HTT, and a decrease in HTT by way of HTT knockdown leads to a decrease in SDMA, which is mainly mediated by PRMT5.</p>
<p style="text-align: justify;">The two-way crosstalk demonstrated by PRMT5 activity levels affecting the expression of HTT and the presence (or absence, since we used a knockdown model) of HTT likewise decreasing SDMA levels suggests that the relationship between PRMT5 and HTT is complicated and reciprocal, with one regulating the other possibly through post-translational modifications such as SDMA and other epigenetic mechanisms, and warrants further investigation as to how the activity and levels of PRMT5 and HTT are related. Some important future experiments would be to test if PRMT inhibition is also able to decrease the protein levels of mutant HTT similarly to how it affects wild-type HTT, and if so, this will be an exciting opportunity to follow up on in the search of a potential therapy for Huntington’s disease.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/further-investigation-of-the-reduction-of-htt-protein-through-chemical-inhibition/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>A new ALK2 structure, and proof that mistakes in the lab aren’t always a bad thing</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/a-new-alk2-structure-and-proof-that-mistakes-in-the-lab-arent-always-a-bad-thing/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/a-new-alk2-structure-and-proof-that-mistakes-in-the-lab-arent-always-a-bad-thing/#respond</comments>
		<pubDate>Fri, 04 Jan 2019 15:28:47 +0000</pubDate>
		<dc:creator><![CDATA[Roslin Adamson]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Brain Tumours]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>
		<category><![CDATA[Roslin Adamson]]></category>
		<category><![CDATA[SGC Univ. of Oxford]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2364</guid>
		<description><![CDATA[Hello everyone and happy New Year! I really, really wanted to end the year on a high note in December before I went off on holiday, but as it turned out, I didn’t have enough time to set up 18 plates and write a blog post on the last day. So I’ll just have to <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/a-new-alk2-structure-and-proof-that-mistakes-in-the-lab-arent-always-a-bad-thing/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Hello everyone and happy New Year! I really, really wanted to end the year on a high note in December before I went off on holiday, but as it turned out, I didn’t have enough time to set up 18 plates and write a blog post on the last day. So I’ll just have to start the year on a high note.</p>
<p>I’m very happy to say I got another ACVR1/ALK2 structure in December, at 2 Å resolution, co-crystallised with M4K2158 (see images below, and the .mtz autoprocessed file is available <a href="https://zenodo.org/record/2531714#.XC94jlz7SUk">here</a> on Zenodo). The structure is not fully refined yet, but what’s interesting is the solvent side of the molecule is being stabilised by either a water contact (Chain A) or Asp293 (Chain B). This gives it really good density.</p>
<p>&nbsp;</p>
<div id="attachment_2367" style="width: 865px" class="wp-caption aligncenter"><img class="wp-image-2367 " src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/M4K2158_structure-1024x576.png" alt="" width="855" height="481" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/M4K2158_structure-1024x576.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/M4K2158_structure-300x169.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/M4K2158_structure-768x432.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/M4K2158_structure.png 1280w" sizes="(max-width: 855px) 100vw, 855px" /><p class="wp-caption-text">ACVR1 bound to M4K2158 at ~2 Å, showing interactions with His286 and a water at the back of the pocket</p></div>
<p>&nbsp;</p>
<p>I also seem to have some new <span style="font-size: 1.5rem;">pretty </span><span style="font-size: 1.5rem;">hits with M4K2118, M4K2149 and M4K3003 (crystal pics below). What’s interesting about the M4K3003 hits is that I’ve really struggled to get crystals with this compound (as well as M4K3007 and 3010), and it is a pretty desirable structure. These hits came about as a result of a mistake, where I somehow managed to mix up my tubes and put ACVR1 with M4K3003 into my mega composite screen without seeds, and TNIK into a fine screen I had intended for ACVR1 with seeds. I only discovered this because I mass specced the TNIK just before setting up the plates, then rushed off to do them as it was the SGC Christmas party that night and I needed to finish (best not to rush experiments). When I checked the mass spec result later, I got the exact mass of ACVR1! So I went back to the crystallography room and recovered what protein I could from the source plate for the TNIK plates, and mass specced it again. Same result – ACVR1. But when I looked at the drop images for those plates (now containing ACVR1 + M4K3003), I found a bunch of crystals! Happy serendipity!! These will go on the trip to Diamond on the 21</span><sup>st</sup><span style="font-size: 1.5rem;"> of January and then we’ll know whether the mistake really ended well or not. Fingers crossed!</span></p>
<p>&nbsp;</p>
<div id="attachment_2366" style="width: 893px" class="wp-caption aligncenter"><img class="wp-image-2366" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/Crystal-hits-Dec_Jan-2018_9-1024x576.png" alt="" width="883" height="497" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/Crystal-hits-Dec_Jan-2018_9-1024x576.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/Crystal-hits-Dec_Jan-2018_9-300x169.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/Crystal-hits-Dec_Jan-2018_9-768x432.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/Crystal-hits-Dec_Jan-2018_9.png 1280w" sizes="(max-width: 883px) 100vw, 883px" /><p class="wp-caption-text">ACVR1 co-crystallised with M4K2149, 2118 and 3003 &#8211; hits to send to the synchrotron</p></div>
<p>&nbsp;</p>
<p>You may well ask, why was I setting up plates with TNIK? As it happens, some of our collaborators’ assays show M4K2009, our main compound of interest, hits both TNIK and DDR1 with pretty high affinity. We’ve been struggling to get crystals of ACVR1 with M4K2009 – for whatever reason, it doesn’t want to make crystals. So it seemed like a good idea to see if TNIK or DDR1 will co-crystallise with it. TNIK (TRAF2 and NCK Interacting Kinase) is a serine/threonine kinase that functions as an activator of the Wnt signalling pathway. DDR1 (Discoidin Domain Receptor Tyrosine Kinase 1) is a tyrosine kinase that functions as cell surface receptor for fibrillar collagen and regulates cell attachment to the extracellular matrix, remodelling of the extracellular matrix, cell migration, differentiation, survival and cell proliferation (according to its GeneCard).</p>
<p>While I was about it, I thought it would be useful to set up plates with BMPR1A (ALK3), for the same reason I set up TGFBR1 (ALK5) before – to see what off-target structures look like with some of these compounds. The purification protocols for TNIK and BMPR1A were as standard, but they’re available <a href="https://zenodo.org/record/2531714#.XC94jlz7SUk">here</a> on Zenodo.</p>
<p>Some other good news is that I’ve figured out how to reproduce my XChem crystals fairly reliably. On the last day before the holidays I set up 6 plates of ACVR1 with M4K2117, which gives really beautiful, well-diffracting crystals. I used 10, 12 and 15 mg/ml protein, and made plates with and without seeds. Well, the ones without the seeds grew fabulously, within 2 weeks, and the ones with seeds didn’t! The next step with these then is to test what happens if I soak them in different formulations of lower salt with and without some PEG3350. That&#8217;ll be for next week. Have a great weekend!</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/a-new-alk2-structure-and-proof-that-mistakes-in-the-lab-arent-always-a-bad-thing/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Update on crystal hits, XChem and more purification</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/update-on-crystal-hits-xchem-and-more-purification/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/update-on-crystal-hits-xchem-and-more-purification/#comments</comments>
		<pubDate>Wed, 28 Nov 2018 16:22:44 +0000</pubDate>
		<dc:creator><![CDATA[Roslin Adamson]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Brain Tumours]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>
		<category><![CDATA[Roslin Adamson]]></category>
		<category><![CDATA[SGC Univ. of Oxford]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2191</guid>
		<description><![CDATA[Things have been really busy in the lab the last few weeks. I’ve done a couple of purifications of ACVR1 (ALK2), which I’ll post in Zenodo (getting loads of protein now), but probably won’t write up the details of every purification every single time anymore, as it’s the same process every time for this construct. <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/update-on-crystal-hits-xchem-and-more-purification/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Things have been really busy in the lab the last few weeks. I’ve done a couple of purifications of ACVR1 (ALK2), which I’ll post in <a href="https://zenodo.org/record/1639900#.W_6-6Wj7SUk">Zenodo</a> (getting loads of protein now), but probably won’t write up the details of every purification every single time anymore, as it’s the same process every time for this construct. Please feel free to access earlier Zenodo posts for the protocol, or contact me and I’ll send you the links. I’ve also set up at least 50 crystal plates over the past month, most using the composite screen I made a while back, as well as follow up screens for hits in those conditions.</p>
<p>&nbsp;</p>
<p>I haven’t been lucky with other compounds again like I was with the big, beautiful M4K2117 crystals, but, after initial apparent abject failure to reproduce these, I now appear to have grown a plate of simply gorgeous beauties (about 40 wells with lovely crystals, which in theory should diffract around 1.5 Å or better) (see pics below, verified as protein with UV imaging). I’ve done this for the fragment screening side of the project, where we’re using x-ray crystallography to screen compound fragments in search of an allosteric binding site on ACVR1, as described in earlier posts. The crystal system I’ve been using is with LDN-193189 co-crystallised with ACVR1. The crystals are big and beautiful, but suffer from some issues: they give variable space groups, tricky for the pandda software to deal with, they have thick skins, which makes soaking somewhat variable and necessarily increases the soak time, as well as making mounting tricky. In many cases while mounting, I noticed the skin was formed almost as an envelope around the top and bottom of the drop, thus when you try to pull back the skin, it pulls the crystal with it and then wraps it up tightly in a big lump of sticky skin. I’m not that great at the micro-surgery needed to separate the mess out. Additionally, many of these crystals melt or become friable when soaked too long (more than ~2 hours).</p>
<p>&nbsp;</p>
<p>The M4K2117 ACVR1 co-crystals however are in nice clear drops, although the conditions are very salty (0.1 M citrate pH 5.0 – 6.0, 1.4M ammonium sulphate, 0.2M sodium/potassium tartrate). This could be an issue for fragment binding – acidic conditions with lots of charged species around. Thus, I intend to test what happens if I soak them in different formulations of lower salt with and without some PEG3350. Also, I need to figure out how to grow them reproducibly, as only those 40 wells from one plate out of 6 set up actually grew. It’s unclear why 2 other plates the same conditions and 3 other plates with essentially the same condition didn’t grow. Or perhaps I just need to wait a little longer (they take about a month to grow).  Many of the wells without crystals have a lot of phase separation, as shown in the pic below as well.</p>
<p>&nbsp;</p>
<div id="attachment_2194" style="width: 521px" class="wp-caption aligncenter"><img class="wp-image-2194 size-full" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/Allosteric-pocket.png" alt="" width="511" height="687" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/Allosteric-pocket.png 511w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/Allosteric-pocket-223x300.png 223w" sizes="(max-width: 511px) 100vw, 511px" /><p class="wp-caption-text">The allosteric site in question, and the current LDN-193189 co-crystals we are using</p></div>
<div id="attachment_2195" style="width: 902px" class="wp-caption aligncenter"><img class="wp-image-2195 size-full" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/M4K2117-1.png" alt="" width="892" height="657" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/M4K2117-1.png 892w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/M4K2117-1-300x221.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/M4K2117-1-768x566.png 768w" sizes="(max-width: 892px) 100vw, 892px" /><p class="wp-caption-text">ACVR1-M4K2117 co-crystals, with corresponding UV images showing these are indeed protein. The lower right pics show the phase separation that is in the majority of wells.</p></div>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/update-on-crystal-hits-xchem-and-more-purification/feed/</wfw:commentRss>
		<slash:comments>2</slash:comments>
		</item>
		<item>
		<title>Reduction of HTT Protein through Chemical Inhibition</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/reduction-of-htt-protein-through-chemical-inhibition/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/reduction-of-htt-protein-through-chemical-inhibition/#comments</comments>
		<pubDate>Thu, 22 Nov 2018 20:34:38 +0000</pubDate>
		<dc:creator><![CDATA[Jolene Ho]]></dc:creator>
				<category><![CDATA[Huntingtin]]></category>
		<category><![CDATA[Huntington’s disease]]></category>
		<category><![CDATA[Jolene Ho]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2169</guid>
		<description><![CDATA[The experimental set-up and results covered in this post can be found here: 10.5281/zenodo.1495129. Apologies for the long delay between this post and my last post, it really has been quite some time! As postdocs, we generally have several projects running simultaneously and we have to split our time between these projects. I am no <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/reduction-of-htt-protein-through-chemical-inhibition/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p style="text-align: justify;">The experimental set-up and results covered in this post can be found here: <a href="http://doi.org/10.5281/zenodo.1495129">10.5281/zenodo.1495129</a>.</p>
<p style="text-align: justify;">Apologies for the long delay between this post and my last post, it really has been quite some time! As postdocs, we generally have several projects running simultaneously and we have to split our time between these projects. I am no exception and lately I have been caught up in trying to wrap up several projects. Besides my work in HTT, I am involved in screening various cell lines against our SGC probe library and identifying probes that have a differential effect in these cell lines. At the SGC, chemical compounds only qualify as probes if they have an in vitro potency of &lt;100 nM, &gt;30-fold selectivity vs other subfamilies, and have significant on-target cell activity at 1 µM. These probes are essential for the early stages of drug development as positive hits in our screens allow us to identify potential drug targets, protein families, and pathways for which we can further investigate in terms of disease mechanism and therapeutic strategies. These probes identified as positive hits in our screens may then be further developed and refined as drugs which may be used in the clinics eventually. You can read more about the SGC’s probes here: <a href="https://www.thesgc.org/chemical-probes">https://www.thesgc.org/chemical-probes</a>.</p>
<p style="text-align: justify;">In one of the other projects I’m working on, we identified PRMT5 as a potential therapeutic target in glioblastoma through the action of GSK591 and LLY283, 2 probes that inhibit PRMT5 through different and independent mechanisms. The inhibition of PRMT5 as a therapeutic strategy is not novel in itself, with several other groups having already published on this.</p>
<p>&nbsp;</p>
<p style="text-align: justify;">PRMT5 as a druggable target for glioblastoma therapy.</p>
<p style="text-align: justify;">Banasavadi-Siddegowda YK, Welker AM, An M, Yang X, Zhou W, Shi G, Imitola J, Li C, Hsu S, Wang J, Phelps M, Zhang J, Beattie CE, Baiocchi R, Kaur B.</p>
<p style="text-align: justify;">Neuro Oncol. 2018 May 18;20(6):753-763. doi: <a href="http://doi.org/10.1093/neuonc/nox206">10.1093/neuonc/nox206</a>.</p>
<p>&nbsp;</p>
<p style="text-align: justify;">Coordinated Splicing of Regulatory Detained Introns within Oncogenic Transcripts Creates an Exploitable Vulnerability in Malignant Glioma.</p>
<p style="text-align: justify;">Braun CJ, Stanciu M, Boutz PL, Patterson JC, Calligaris D, Higuchi F, Neupane R, Fenoglio S, Cahill DP, Wakimoto H, Agar NYR, Yaffe MB, Sharp PA, Hemann MT, Lees JA.</p>
<p style="text-align: justify;">Cancer Cell. 2017 Oct 9;32(4):411-426.e11. doi: <a href="http://doi.org/10.1016/j.ccell.2017.08.018">10.1016/j.ccell.2017.08.018</a>. Epub 2017 Sep 28.</p>
<p>&nbsp;</p>
<p style="text-align: justify;">However, we had some interesting findings through the RNA-sequencing done on cells treated with these PRMT5 inhibitors. Serendipitously, HTT emerged as one of the genes whose expression was consistently downregulated with PRMT5 inhibition! I tested these PRMT5 inhibitors in the HeLa cells I used to generate the 3 stable HTT knockdown lines previously (read more about that here: https://thesgc.github.io/static-openlabnotebooks/generating-htt-knockdown-models/) and looked at how the HTT protein levels were affected in these lines.</p>
<p style="text-align: justify;"><img class="alignnone wp-image-2170" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/20181116-HeLa-G3BP1-PRMT5i-and-HTT-KD-300x225.jpg" alt="" width="841" height="631" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/20181116-HeLa-G3BP1-PRMT5i-and-HTT-KD-300x225.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/20181116-HeLa-G3BP1-PRMT5i-and-HTT-KD-768x576.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/20181116-HeLa-G3BP1-PRMT5i-and-HTT-KD-350x263.jpg 350w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/20181116-HeLa-G3BP1-PRMT5i-and-HTT-KD-780x586.jpg 780w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/20181116-HeLa-G3BP1-PRMT5i-and-HTT-KD-250x188.jpg 250w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/20181116-HeLa-G3BP1-PRMT5i-and-HTT-KD.jpg 960w" sizes="(max-width: 841px) 100vw, 841px" /></p>
<p style="text-align: justify;">HTT protein levels were clearly decreased with PRMT5 inhibition! I ran these samples alongside the stable HTT knockdown lines and although the HTT levels did not decrease as drastically as with the knockdowns, the decrease with the PRMT5 inhibitors is still substantial. I also included SGC2096, the inactive control to GSK591, and HTT protein levels were not affected by it, showing that it is indeed through the activity of PRMT5 that HTT levels were decreased.</p>
<p style="text-align: justify;">The mechanism by which HTT is downregulated by PRMT5 inhibitors is still unclear and it will be very interesting to further investigate the link between HTT and PRMT5. As of now, the only publication I can find linking HTT to PRMT5 is the following:</p>
<p>&nbsp;</p>
<p style="text-align: justify;">PRMT5- mediated symmetric arginine dimethylation is attenuated by mutant huntingtin and is impaired in Huntington&#8217;s disease (HD).</p>
<p style="text-align: justify;">Ratovitski T, Arbez N, Stewart JC, Chighladze E, Ross CA.</p>
<p style="text-align: justify;">Cell Cycle. 2015;14(11):1716-29. doi: <a href="http://doi.org/10.1080/15384101.2015.1033595">10.1080/15384101.2015.1033595</a>.</p>
<p>&nbsp;</p>
<p style="text-align: justify;">This paper showed the colocalisation and interaction of HTT and PRMT5 and shows that this interaction results in the stimulation of PRMT5 activity by HTT in vitro. Mutant HTT has a greater affinity for PRMT5; however, it has a reduced ability to stimulate PRMT5 activity, as demonstrated by the reduction of symmetric dimethylation of arginine (SDMA), which is mainly mediated by PRMT5, of histones H2 and H4. Decreased levels of SDMA on known PRMT5 targets were detected in HD brain samples, suggesting that the reduction in PRMT5 activity may contribute to the pathogenesis of HD. The knockdown of JMJD6, which removes arginine demethylation, was able to rescue the phenotype of increased cell death seen with mutant HTT, further suggesting that the restoration of SDMA to normal levels may be a potential therapeutic strategy.</p>
<p style="text-align: justify;">However, our findings that PRMT5 inhibition itself can affect the expression of HTT at the protein level suggests that there might be a feedback loop in play between PRMT5 and HTT. It will be interesting to further investigate how the levels of SDMA and the other proteins involved are affected with PRMT5 inhibition and elucidate the mechanisms involved in this process. It should also be noted that our results only reflect the expression of wild-type HTT and not mutant (expanded) HTT, and it will be highly interesting and of great clinical importance to find out if PRMT5 inhibition is able to decrease the protein levels of mutant HTT similarly to how it affects wild-type HTT.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/reduction-of-htt-protein-through-chemical-inhibition/feed/</wfw:commentRss>
		<slash:comments>4</slash:comments>
		</item>
		<item>
		<title>Checking expression and localization of split-SCN1A constructs in HEK293 cells</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/checking-expression-and-localization-of-split-scn1a-constructs-in-hek293-cells/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/checking-expression-and-localization-of-split-scn1a-constructs-in-hek293-cells/#respond</comments>
		<pubDate>Sun, 18 Nov 2018 19:41:36 +0000</pubDate>
		<dc:creator><![CDATA[Wen Yih Aw]]></dc:creator>
				<category><![CDATA[Catalyst UNC Chapel Hill]]></category>
		<category><![CDATA[Dravet syndrome]]></category>
		<category><![CDATA[Gene therapy for Dravet Syndrome]]></category>
		<category><![CDATA[Wen Yih Aw]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2163</guid>
		<description><![CDATA[In my last posts, I have constructed a pair of split-SCN1A constructs (split- SCN1A-GFP(1-10) and SCN1A-GFP(11)). Here, I tested the functionality of these constructs by looking at whether these split- constructs reconstitute into a full-length SCN1A and generate a fluorescent signal when co-transfected in HEK293 cells, and if so, do the reconstituted SCN1A (with an <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/checking-expression-and-localization-of-split-scn1a-constructs-in-hek293-cells/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>In my last posts, I have constructed a pair of split-SCN1A constructs (split- SCN1A-GFP<sub>(1-10)</sub> and SCN1A-GFP<sub>(11)</sub>). Here, I tested the functionality of these constructs by looking at whether these split- constructs reconstitute into a full-length SCN1A and generate a fluorescent signal when co-transfected in HEK293 cells, and if so, do the reconstituted SCN1A (with an internal GFP tag) localize correctly at the plasma membrane.</p>
<p>Below are my imaging results. The detailed experimental procedure can be found at <a href="https://zenodo.org/record/1490851#.W_G_bpNKh3g">https://zenodo.org/record/1490851#.W_G_bpNKh3g</a></p>
<p><img class="aligncenter size-large wp-image-2165" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/dravet-pic5-1024x752.jpg" alt="" width="780" height="573" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/dravet-pic5-1024x752.jpg 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/dravet-pic5-300x220.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/dravet-pic5-768x564.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/dravet-pic5.jpg 1424w" sizes="(max-width: 780px) 100vw, 780px" /></p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p><img class="aligncenter size-large wp-image-2164" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/dravet-pic6-1024x404.png" alt="" width="780" height="308" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/dravet-pic6-1024x404.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/dravet-pic6-300x118.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/dravet-pic6-768x303.png 768w" sizes="(max-width: 780px) 100vw, 780px" /></p>
<p>&nbsp;</p>
<p>In short, these split-constructs managed to find each other and reconstitute a GFP signal in HEK293 cells. However, the GFP signals do not overlap with cell cortex or plasma membrane markers (cell cortex marked by cortical actin staining and plasma membrane marked by CD8, a transmembrane glycoprotein.) The distribution of reconstituted SCN1A-GFP signals resembles misfolded proteins that are trapped in the ER.  Co-expression of SCN1B, a beta subunit which is thought to promote sodium channel trafficking, does not seem to improve the localization of reconstituted SCN1A-GFP.</p>
<p>There are several caveats with the design of these constructs which might have affected folding and trafficking of reconstituted SCN1A: 1) instead of using the canonical SCN1A cDNA sequence (which is  unstable and prone to mutagenize when propagated in E. coli), a codon-optimized SCN1A cDNA was used as a template in cloning these constructs and 2) the reconstituted protein has a foreign GFP sequence which might have interfered with SCN1A trafficking. Moving forward, I will revisit and examine the localization of codon optimized SCN1A in HEK293 cells while continue constructing the rest of the split-intein constructs.</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/checking-expression-and-localization-of-split-scn1a-constructs-in-hek293-cells/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
	</channel>
</rss>
