<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>Past Scientist &#8211; openlabnotebooks.org</title>
	<atom:link href="https://thesgc.github.io/static-openlabnotebooks/category/past-scientist/feed/" rel="self" type="application/rss+xml" />
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<description>A growing team of groundbreaking scientists around the world are now sharing their lab notebooks online</description>
	<lastBuildDate>Thu, 20 Jun 2019 11:26:10 +0000</lastBuildDate>
	<language>en-GB</language>
	<sy:updatePeriod>hourly</sy:updatePeriod>
	<sy:updateFrequency>1</sy:updateFrequency>
	<generator>https://wordpress.org/?v=4.9.10</generator>

<image>
	<url>https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/cropped-lab-jar02-32x32.png</url>
	<title>Past Scientist &#8211; openlabnotebooks.org</title>
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<width>32</width>
	<height>32</height>
</image> 
	<item>
		<title>Three new ALK2 structures, one of them not the one I thought!</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/three-new-alk2-structures-one-of-them-not-the-one-i-thought/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/three-new-alk2-structures-one-of-them-not-the-one-i-thought/#respond</comments>
		<pubDate>Tue, 19 Feb 2019 17:46:47 +0000</pubDate>
		<dc:creator><![CDATA[Roslin Adamson]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Brain Tumours]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>
		<category><![CDATA[Roslin Adamson]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2507</guid>
		<description><![CDATA[I’ve not posted in a while, mainly because things have been really busy, but good busy, as I now have 3 new datasets/structures, of ALK2 with M4K2118, M4K2149 and, amazingly, M4K2009. As you know, this is our lead candidate, which until now has not behaved very well in crystal trials, and it hasn’t been possible <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/three-new-alk2-structures-one-of-them-not-the-one-i-thought/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>I’ve not posted in a while, mainly because things have been really busy, but good busy, as I now have 3 new datasets/structures, of ALK2 with M4K2118, M4K2149 and, amazingly, M4K2009. As you know, this is our lead candidate, which until now has not behaved very well in crystal trials, and it hasn’t been possible to get decent crystals. In my last post I told you how I had somehow managed to mix up the TNIK and ACVR1/ALK2 tubes when I was setting up the plates, and accidentally put ACVR1 into a coarse screen and TNIK into a fine screen for ACVR1. Well, after I got lovely big crystals for what I thought was ACVR1 with M4K3003, I sent them to the synchrotron, got beautiful data to 1.4 Å, phased it, and lo and behold! It was M4K2009 snugly fitted into the binding pocket (see the ligand density after molecular replacement below). Of course, I didn’t notice this was a trimethoxy containing compound (how??) before I excitedly emailed everyone to tell them I had M4K3003, and then had to retract … However, the story ends well as M4K2009 is really important.</p>
<p><img class="aligncenter  wp-image-2511" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/M4k2009-1024x576.jpg" alt="" width="904" height="509" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/M4k2009-1024x576.jpg 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/M4k2009-300x169.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/M4k2009-768x432.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/M4k2009.jpg 1280w" sizes="(max-width: 904px) 100vw, 904px" /></p>
<p>Additionally, I got data for M4K2149 (~2.6 Å), one of the amide head group containing compounds we’re interested to see, and M4K2118 (~1.4 Å), which is doing well in some of the other assays too. I’ve put the .mtz files <a href="https://zenodo.org/record/2573142#.XGw9g-j7SUk">here</a> on Zenodo if you want to play with high resolution data, and some images below showing the ligand density and the ligands binding in the pocket. The amide head group of M4K2149 appears to preserve the water network in the pocket that we’ve observed in other structures, but as the resolution is too low for waters, we can’t tell til we get higher resolution data. Which hopefully will be this week, as I’ve grown some really pretty crystals for M4K2149 that I hope will diffract better. These appeared quite by accident again, as I’d designed a fine screen around the D5 well condition of my mega composite screen, but neglected to notice when I validated the screen that I’d put 10 x less ammonium sulphate in one row than intended – should have been 0.9 M, but ended up being 0.09 M. I should probably start calling myself ‘the accidental crystallographer’ at this rate.</p>
<p><img class="aligncenter  wp-image-2509" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/M4K2118-2149-1024x576.png" alt="" width="883" height="497" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/M4K2118-2149-1024x576.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/M4K2118-2149-300x169.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/M4K2118-2149-768x432.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/M4K2118-2149.png 1280w" sizes="(max-width: 883px) 100vw, 883px" /></p>
<p>In the meantime, I’ve got some small crystals for M4K3007, another structure we really want, and some pretty ones for M4K2184, another amide head group compound. I’ve followed up the M4K3007, just waiting to see if anything appears, and will send the tiny ones to the synchrotron while waiting.</p>
<p><img class="aligncenter  wp-image-2508" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/crystal-pics-1024x576.png" alt="" width="849" height="478" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/crystal-pics-1024x576.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/crystal-pics-300x169.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/crystal-pics-768x432.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/crystal-pics.png 1280w" sizes="(max-width: 849px) 100vw, 849px" /></p>
<p>I’m also very happy to say that our new M4K2117 crystal system for XChem fragment screening has worked very nicely so far, in the sense that the crystals are large and relatively robust and can handle soaking in low salt conditions as well as compounds with DMSO for a few hours. They then still diffract to around 1.2-1.8 Å, consistently in the same space group, C 1 2 1. This is a vast improvement over the previous system with LDN-193189. We did a solvent characterisation and pre-screen a couple of weeks ago with 100 crystals, and of the 20 or so hits we got, none was in the allosteric pocket we are looking for. This could be because of the 1.4 M ammonium sulphate and 0.2 M Na/K tartrate I have in there, which might prevent access of fragments to the pocket. Thus, using the remarkable shifter that Nathan Wright at SGC designed, we’ll be transferring loads of crystals from grow plates to low salt soak plates before doing the compound soaks this week, with the same 100 compounds, in order to check if the hit rate/location changes.</p>
<p>I will mention though, for anyone who may never have noticed this, the DMSO concentration in the protein solution for crystallisation is very important! I had previously been using Batch 1 of M4K2117 for my co-crystallisation, but it ran out so I used Batch 2, which was a 50 mM stock, as opposed to the previous 25 mM stock. The plates I set up with this, the same in every other way, gave showers of rather ugly, crunchy, small crystals. So I tried a few things – diluted the stock to 25 mM in DMSO and set up plates with that, with both lower and higher concentrations of protein (8 and 14 mg/ml along with the usual 10-11 mg/ml), as well as using the stock as is, with lower and higher protein concentrations. The lower stock concentrations brought back the big, perfect, beautiful crystals from before, the higher stock, no matter the protein concentration, gave ugly chunks and showers. The final concentration of compound was the same in all, so clearly the DMSO percentage is a very important factor.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/three-new-alk2-structures-one-of-them-not-the-one-i-thought/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Last post on Split-SCN1A</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/last-post-on-split-scn1a/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/last-post-on-split-scn1a/#respond</comments>
		<pubDate>Sun, 10 Feb 2019 22:47:17 +0000</pubDate>
		<dc:creator><![CDATA[Wen Yih Aw]]></dc:creator>
				<category><![CDATA[Dravet syndrome]]></category>
		<category><![CDATA[Gene therapy for Dravet Syndrome]]></category>
		<category><![CDATA[Wen Yih Aw]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2452</guid>
		<description><![CDATA[The idea for split-intein based SCN1A reconstitution started when I first learned of the split-Cas9 system. Long story short, packaging Cas9 with a guideRNA sequence into an AAV vector is pushing the packaging capacity of AAV vector. To circumvent this issue, the authors design a split-Cas9 system where the coding sequence for Cas9 is split <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/last-post-on-split-scn1a/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>The idea for split-intein based SCN1A reconstitution started when I first learned of the split-Cas9 system. Long story short, packaging Cas9 with a guideRNA sequence into an AAV vector is pushing the packaging capacity of AAV vector. To circumvent this issue, the authors design a split-Cas9 system where the coding sequence for Cas9 is split into two fragments, each fused to a split-inteins sequence. When co-expressed in cells, the split-inteins can function as glue joining two protein fragments together.</p>
<p>In an attempt to “shrink” SCN1A to be packageable in AAV vectors, I ran into several design and cloning issues, ultimately leading to the decision to terminate this project. Cas9 is a well-studied cytoplasmic protein with structural information while SCN1A is a large multi-pass transmembrane protein that weaves in and out of the plasma membrane. While success packaging and application of the split-Cas9 system for gene editing is encouraging, my attempt to translate similar design to SCN1A, without structural information, proves to be an oversight. Design aside, the coding sequence for human SCN1A is difficult to maintain. The SCN1A cDNA sequence tends to accumulate deletions and rearranges when transformed and propagated in E. coli. One way to improve plasmid stability and reduce unwanted rearrangement is by propagating plasmid in E. coli strains with recA1 genotype at room temperature. recA1 confers a loss of function in RecA1, a protein that is essential for homologous recombinational DNA repair in E. coli. While this strategy helps to alleviate the problem, it is time-consuming and eventually, I decided to make use of a codon-optimized human SCN1A cDNA as the cloning template for split-SCN1A constructs. While this solves the plasmid stability issue, the codon-optimized cDNA has not been well characterized and while the proteins expressed reconstituted, they failed to traffic to the correct cellular compartment. These raise an issue that needs to be addressed – how to propagate mutation-free gene therapy vectors with SCN1A sequence in large scale, have any of us managed to move the project to treatment pipeline.</p>
<p>Thank you all for reading. While this project didn&#8217;t work out as planned, I truly believe that this is an exciting time for Dravet Syndrome. With recent positive Fenfluramine clinical trial results, exciting new drugs in the pipeline, and cross-continental efforts in pushing various gene therapy approaches for treating Dravet, I am hopeful that we can see these exciting results moving into medical treatment in the foreseeable future.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/last-post-on-split-scn1a/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Further Investigation of the Reduction of HTT Protein through Chemical Inhibition</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/further-investigation-of-the-reduction-of-htt-protein-through-chemical-inhibition/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/further-investigation-of-the-reduction-of-htt-protein-through-chemical-inhibition/#respond</comments>
		<pubDate>Tue, 15 Jan 2019 18:56:28 +0000</pubDate>
		<dc:creator><![CDATA[Jolene Ho]]></dc:creator>
				<category><![CDATA[Huntingtin]]></category>
		<category><![CDATA[Huntington’s disease]]></category>
		<category><![CDATA[Jolene Ho]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2378</guid>
		<description><![CDATA[The experimental set-up and results covered in this post can be found here: 10.5281/zenodo.2541053. A lot has happened in the past few months and it is with mixed feelings that I am finishing up my postdoc here at SGC! I feel very privileged to have been part of the great science going on at SGC <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/further-investigation-of-the-reduction-of-htt-protein-through-chemical-inhibition/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p style="text-align: justify;">The experimental set-up and results covered in this post can be found here: <a href="http://doi.org/10.5281/zenodo.2541053">10.5281/zenodo.2541053</a>.</p>
<p style="text-align: justify;">A lot has happened in the past few months and it is with mixed feelings that I am finishing up my postdoc here at SGC! I feel very privileged to have been part of the great science going on at SGC and a pioneer of the Open Notebook initiative here, contributing to open science. Nonetheless, the science has to go on! Hopefully, my posts in this Open Notebook have provided new perspectives to people working on similar projects and trying to learn more about Huntington’s Disease and huntingtin (HTT), inspired people to strive on for science despite the negative and/or inconclusive data, or at least given some insight to how life on the bench is like in general.</p>
<p style="text-align: justify;">Previously, I showed that the use of PRMT5 inhibitors from the SGC probe library, namely GSK591 and LLY283, were able to decrease levels of HTT in HeLa cells. SGC2096, the inactive control to GSK591, was not able to decrease HTT protein levels. This, together with the different mechanisms of action of GSK591 and LLY283 in inhibiting PRMT5, demonstrates that it is indeed through the specific inhibition of PRMT5 that HTT protein levels were decreased. So far, I have not come across any drug that is able to regulate the levels of HTT protein. As such, the inhibition of PRMT5 to reduce the expression of HTT at the protein level presents a potential and exciting therapeutic avenue for the treatment of Huntington’s disease.</p>
<p style="text-align: justify;">The findings that PRMT5 inhibition itself can affect the expression of HTT at the protein level suggests that there might be some interaction and possible feedback loop in play between PRMT5 and HTT. We thus further investigated how PRMT5 inhibition may contribute to the decrease in HTT protein levels to elucidate the mechanisms involved in this process. As PRMT5 is the main enzyme known to mediate symmetric demethylation of arginine (SDMA), we thus checked how the levels of SDMA are affected by PRMT5 inhibition and HTT knockdown.</p>
<p style="text-align: justify;"><img class="alignnone wp-image-2379" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/HeLa-G3BP1-PRMT5i-in-HTT-KD-300x225.jpg" alt="" width="771" height="578" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/HeLa-G3BP1-PRMT5i-in-HTT-KD-300x225.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/HeLa-G3BP1-PRMT5i-in-HTT-KD-768x576.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/HeLa-G3BP1-PRMT5i-in-HTT-KD-350x263.jpg 350w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/HeLa-G3BP1-PRMT5i-in-HTT-KD-780x586.jpg 780w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/HeLa-G3BP1-PRMT5i-in-HTT-KD-250x188.jpg 250w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/HeLa-G3BP1-PRMT5i-in-HTT-KD.jpg 960w" sizes="(max-width: 771px) 100vw, 771px" /></p>
<p style="text-align: justify;">Unsurprisingly, PRMT5 inhibition with GSK591 or LLY283 led to the decrease in SDMA. However, we also noticed a slight decrease in SDMA across all 3 HTT knockdown samples! Although this decrease is marginal, its occurrence across all the HTT knockdown samples we tested gives confidence to this observation and corroborates the hypothesis that PRMT5 and HTT are part of a feedback loop rather than that HTT is simply downstream of PRMT5, since the inhibition of PRMT5 leads to a decrease in HTT, and a decrease in HTT by way of HTT knockdown leads to a decrease in SDMA, which is mainly mediated by PRMT5.</p>
<p style="text-align: justify;">The two-way crosstalk demonstrated by PRMT5 activity levels affecting the expression of HTT and the presence (or absence, since we used a knockdown model) of HTT likewise decreasing SDMA levels suggests that the relationship between PRMT5 and HTT is complicated and reciprocal, with one regulating the other possibly through post-translational modifications such as SDMA and other epigenetic mechanisms, and warrants further investigation as to how the activity and levels of PRMT5 and HTT are related. Some important future experiments would be to test if PRMT inhibition is also able to decrease the protein levels of mutant HTT similarly to how it affects wild-type HTT, and if so, this will be an exciting opportunity to follow up on in the search of a potential therapy for Huntington’s disease.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/further-investigation-of-the-reduction-of-htt-protein-through-chemical-inhibition/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>A new ALK2 structure, and proof that mistakes in the lab aren’t always a bad thing</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/a-new-alk2-structure-and-proof-that-mistakes-in-the-lab-arent-always-a-bad-thing/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/a-new-alk2-structure-and-proof-that-mistakes-in-the-lab-arent-always-a-bad-thing/#respond</comments>
		<pubDate>Fri, 04 Jan 2019 15:28:47 +0000</pubDate>
		<dc:creator><![CDATA[Roslin Adamson]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Brain Tumours]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>
		<category><![CDATA[Roslin Adamson]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2364</guid>
		<description><![CDATA[Hello everyone and happy New Year! I really, really wanted to end the year on a high note in December before I went off on holiday, but as it turned out, I didn’t have enough time to set up 18 plates and write a blog post on the last day. So I’ll just have to <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/a-new-alk2-structure-and-proof-that-mistakes-in-the-lab-arent-always-a-bad-thing/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Hello everyone and happy New Year! I really, really wanted to end the year on a high note in December before I went off on holiday, but as it turned out, I didn’t have enough time to set up 18 plates and write a blog post on the last day. So I’ll just have to start the year on a high note.</p>
<p>I’m very happy to say I got another ACVR1/ALK2 structure in December, at 2 Å resolution, co-crystallised with M4K2158 (see images below, and the .mtz autoprocessed file is available <a href="https://zenodo.org/record/2531714#.XC94jlz7SUk">here</a> on Zenodo). The structure is not fully refined yet, but what’s interesting is the solvent side of the molecule is being stabilised by either a water contact (Chain A) or Asp293 (Chain B). This gives it really good density.</p>
<p>&nbsp;</p>
<div id="attachment_2367" style="width: 865px" class="wp-caption aligncenter"><img class="wp-image-2367 " src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/M4K2158_structure-1024x576.png" alt="" width="855" height="481" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/M4K2158_structure-1024x576.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/M4K2158_structure-300x169.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/M4K2158_structure-768x432.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/M4K2158_structure.png 1280w" sizes="(max-width: 855px) 100vw, 855px" /><p class="wp-caption-text">ACVR1 bound to M4K2158 at ~2 Å, showing interactions with His286 and a water at the back of the pocket</p></div>
<p>&nbsp;</p>
<p>I also seem to have some new <span style="font-size: 1.5rem;">pretty </span><span style="font-size: 1.5rem;">hits with M4K2118, M4K2149 and M4K3003 (crystal pics below). What’s interesting about the M4K3003 hits is that I’ve really struggled to get crystals with this compound (as well as M4K3007 and 3010), and it is a pretty desirable structure. These hits came about as a result of a mistake, where I somehow managed to mix up my tubes and put ACVR1 with M4K3003 into my mega composite screen without seeds, and TNIK into a fine screen I had intended for ACVR1 with seeds. I only discovered this because I mass specced the TNIK just before setting up the plates, then rushed off to do them as it was the SGC Christmas party that night and I needed to finish (best not to rush experiments). When I checked the mass spec result later, I got the exact mass of ACVR1! So I went back to the crystallography room and recovered what protein I could from the source plate for the TNIK plates, and mass specced it again. Same result – ACVR1. But when I looked at the drop images for those plates (now containing ACVR1 + M4K3003), I found a bunch of crystals! Happy serendipity!! These will go on the trip to Diamond on the 21</span><sup>st</sup><span style="font-size: 1.5rem;"> of January and then we’ll know whether the mistake really ended well or not. Fingers crossed!</span></p>
<p>&nbsp;</p>
<div id="attachment_2366" style="width: 893px" class="wp-caption aligncenter"><img class="wp-image-2366" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/Crystal-hits-Dec_Jan-2018_9-1024x576.png" alt="" width="883" height="497" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/Crystal-hits-Dec_Jan-2018_9-1024x576.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/Crystal-hits-Dec_Jan-2018_9-300x169.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/Crystal-hits-Dec_Jan-2018_9-768x432.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/01/Crystal-hits-Dec_Jan-2018_9.png 1280w" sizes="(max-width: 883px) 100vw, 883px" /><p class="wp-caption-text">ACVR1 co-crystallised with M4K2149, 2118 and 3003 &#8211; hits to send to the synchrotron</p></div>
<p>&nbsp;</p>
<p>You may well ask, why was I setting up plates with TNIK? As it happens, some of our collaborators’ assays show M4K2009, our main compound of interest, hits both TNIK and DDR1 with pretty high affinity. We’ve been struggling to get crystals of ACVR1 with M4K2009 – for whatever reason, it doesn’t want to make crystals. So it seemed like a good idea to see if TNIK or DDR1 will co-crystallise with it. TNIK (TRAF2 and NCK Interacting Kinase) is a serine/threonine kinase that functions as an activator of the Wnt signalling pathway. DDR1 (Discoidin Domain Receptor Tyrosine Kinase 1) is a tyrosine kinase that functions as cell surface receptor for fibrillar collagen and regulates cell attachment to the extracellular matrix, remodelling of the extracellular matrix, cell migration, differentiation, survival and cell proliferation (according to its GeneCard).</p>
<p>While I was about it, I thought it would be useful to set up plates with BMPR1A (ALK3), for the same reason I set up TGFBR1 (ALK5) before – to see what off-target structures look like with some of these compounds. The purification protocols for TNIK and BMPR1A were as standard, but they’re available <a href="https://zenodo.org/record/2531714#.XC94jlz7SUk">here</a> on Zenodo.</p>
<p>Some other good news is that I’ve figured out how to reproduce my XChem crystals fairly reliably. On the last day before the holidays I set up 6 plates of ACVR1 with M4K2117, which gives really beautiful, well-diffracting crystals. I used 10, 12 and 15 mg/ml protein, and made plates with and without seeds. Well, the ones without the seeds grew fabulously, within 2 weeks, and the ones with seeds didn’t! The next step with these then is to test what happens if I soak them in different formulations of lower salt with and without some PEG3350. That&#8217;ll be for next week. Have a great weekend!</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/a-new-alk2-structure-and-proof-that-mistakes-in-the-lab-arent-always-a-bad-thing/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Update on crystal hits, XChem and more purification</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/update-on-crystal-hits-xchem-and-more-purification/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/update-on-crystal-hits-xchem-and-more-purification/#comments</comments>
		<pubDate>Wed, 28 Nov 2018 16:22:44 +0000</pubDate>
		<dc:creator><![CDATA[Roslin Adamson]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Brain Tumours]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>
		<category><![CDATA[Roslin Adamson]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2191</guid>
		<description><![CDATA[Things have been really busy in the lab the last few weeks. I’ve done a couple of purifications of ACVR1 (ALK2), which I’ll post in Zenodo (getting loads of protein now), but probably won’t write up the details of every purification every single time anymore, as it’s the same process every time for this construct. <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/update-on-crystal-hits-xchem-and-more-purification/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Things have been really busy in the lab the last few weeks. I’ve done a couple of purifications of ACVR1 (ALK2), which I’ll post in <a href="https://zenodo.org/record/1639900#.W_6-6Wj7SUk">Zenodo</a> (getting loads of protein now), but probably won’t write up the details of every purification every single time anymore, as it’s the same process every time for this construct. Please feel free to access earlier Zenodo posts for the protocol, or contact me and I’ll send you the links. I’ve also set up at least 50 crystal plates over the past month, most using the composite screen I made a while back, as well as follow up screens for hits in those conditions.</p>
<p>&nbsp;</p>
<p>I haven’t been lucky with other compounds again like I was with the big, beautiful M4K2117 crystals, but, after initial apparent abject failure to reproduce these, I now appear to have grown a plate of simply gorgeous beauties (about 40 wells with lovely crystals, which in theory should diffract around 1.5 Å or better) (see pics below, verified as protein with UV imaging). I’ve done this for the fragment screening side of the project, where we’re using x-ray crystallography to screen compound fragments in search of an allosteric binding site on ACVR1, as described in earlier posts. The crystal system I’ve been using is with LDN-193189 co-crystallised with ACVR1. The crystals are big and beautiful, but suffer from some issues: they give variable space groups, tricky for the pandda software to deal with, they have thick skins, which makes soaking somewhat variable and necessarily increases the soak time, as well as making mounting tricky. In many cases while mounting, I noticed the skin was formed almost as an envelope around the top and bottom of the drop, thus when you try to pull back the skin, it pulls the crystal with it and then wraps it up tightly in a big lump of sticky skin. I’m not that great at the micro-surgery needed to separate the mess out. Additionally, many of these crystals melt or become friable when soaked too long (more than ~2 hours).</p>
<p>&nbsp;</p>
<p>The M4K2117 ACVR1 co-crystals however are in nice clear drops, although the conditions are very salty (0.1 M citrate pH 5.0 – 6.0, 1.4M ammonium sulphate, 0.2M sodium/potassium tartrate). This could be an issue for fragment binding – acidic conditions with lots of charged species around. Thus, I intend to test what happens if I soak them in different formulations of lower salt with and without some PEG3350. Also, I need to figure out how to grow them reproducibly, as only those 40 wells from one plate out of 6 set up actually grew. It’s unclear why 2 other plates the same conditions and 3 other plates with essentially the same condition didn’t grow. Or perhaps I just need to wait a little longer (they take about a month to grow).  Many of the wells without crystals have a lot of phase separation, as shown in the pic below as well.</p>
<p>&nbsp;</p>
<div id="attachment_2194" style="width: 521px" class="wp-caption aligncenter"><img class="wp-image-2194 size-full" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/Allosteric-pocket.png" alt="" width="511" height="687" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/Allosteric-pocket.png 511w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/Allosteric-pocket-223x300.png 223w" sizes="(max-width: 511px) 100vw, 511px" /><p class="wp-caption-text">The allosteric site in question, and the current LDN-193189 co-crystals we are using</p></div>
<div id="attachment_2195" style="width: 902px" class="wp-caption aligncenter"><img class="wp-image-2195 size-full" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/M4K2117-1.png" alt="" width="892" height="657" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/M4K2117-1.png 892w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/M4K2117-1-300x221.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/M4K2117-1-768x566.png 768w" sizes="(max-width: 892px) 100vw, 892px" /><p class="wp-caption-text">ACVR1-M4K2117 co-crystals, with corresponding UV images showing these are indeed protein. The lower right pics show the phase separation that is in the majority of wells.</p></div>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/update-on-crystal-hits-xchem-and-more-purification/feed/</wfw:commentRss>
		<slash:comments>2</slash:comments>
		</item>
		<item>
		<title>Reduction of HTT Protein through Chemical Inhibition</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/reduction-of-htt-protein-through-chemical-inhibition/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/reduction-of-htt-protein-through-chemical-inhibition/#comments</comments>
		<pubDate>Thu, 22 Nov 2018 20:34:38 +0000</pubDate>
		<dc:creator><![CDATA[Jolene Ho]]></dc:creator>
				<category><![CDATA[Huntingtin]]></category>
		<category><![CDATA[Huntington’s disease]]></category>
		<category><![CDATA[Jolene Ho]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2169</guid>
		<description><![CDATA[The experimental set-up and results covered in this post can be found here: 10.5281/zenodo.1495129. Apologies for the long delay between this post and my last post, it really has been quite some time! As postdocs, we generally have several projects running simultaneously and we have to split our time between these projects. I am no <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/reduction-of-htt-protein-through-chemical-inhibition/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p style="text-align: justify;">The experimental set-up and results covered in this post can be found here: <a href="http://doi.org/10.5281/zenodo.1495129">10.5281/zenodo.1495129</a>.</p>
<p style="text-align: justify;">Apologies for the long delay between this post and my last post, it really has been quite some time! As postdocs, we generally have several projects running simultaneously and we have to split our time between these projects. I am no exception and lately I have been caught up in trying to wrap up several projects. Besides my work in HTT, I am involved in screening various cell lines against our SGC probe library and identifying probes that have a differential effect in these cell lines. At the SGC, chemical compounds only qualify as probes if they have an in vitro potency of &lt;100 nM, &gt;30-fold selectivity vs other subfamilies, and have significant on-target cell activity at 1 µM. These probes are essential for the early stages of drug development as positive hits in our screens allow us to identify potential drug targets, protein families, and pathways for which we can further investigate in terms of disease mechanism and therapeutic strategies. These probes identified as positive hits in our screens may then be further developed and refined as drugs which may be used in the clinics eventually. You can read more about the SGC’s probes here: <a href="https://www.thesgc.org/chemical-probes">https://www.thesgc.org/chemical-probes</a>.</p>
<p style="text-align: justify;">In one of the other projects I’m working on, we identified PRMT5 as a potential therapeutic target in glioblastoma through the action of GSK591 and LLY283, 2 probes that inhibit PRMT5 through different and independent mechanisms. The inhibition of PRMT5 as a therapeutic strategy is not novel in itself, with several other groups having already published on this.</p>
<p>&nbsp;</p>
<p style="text-align: justify;">PRMT5 as a druggable target for glioblastoma therapy.</p>
<p style="text-align: justify;">Banasavadi-Siddegowda YK, Welker AM, An M, Yang X, Zhou W, Shi G, Imitola J, Li C, Hsu S, Wang J, Phelps M, Zhang J, Beattie CE, Baiocchi R, Kaur B.</p>
<p style="text-align: justify;">Neuro Oncol. 2018 May 18;20(6):753-763. doi: <a href="http://doi.org/10.1093/neuonc/nox206">10.1093/neuonc/nox206</a>.</p>
<p>&nbsp;</p>
<p style="text-align: justify;">Coordinated Splicing of Regulatory Detained Introns within Oncogenic Transcripts Creates an Exploitable Vulnerability in Malignant Glioma.</p>
<p style="text-align: justify;">Braun CJ, Stanciu M, Boutz PL, Patterson JC, Calligaris D, Higuchi F, Neupane R, Fenoglio S, Cahill DP, Wakimoto H, Agar NYR, Yaffe MB, Sharp PA, Hemann MT, Lees JA.</p>
<p style="text-align: justify;">Cancer Cell. 2017 Oct 9;32(4):411-426.e11. doi: <a href="http://doi.org/10.1016/j.ccell.2017.08.018">10.1016/j.ccell.2017.08.018</a>. Epub 2017 Sep 28.</p>
<p>&nbsp;</p>
<p style="text-align: justify;">However, we had some interesting findings through the RNA-sequencing done on cells treated with these PRMT5 inhibitors. Serendipitously, HTT emerged as one of the genes whose expression was consistently downregulated with PRMT5 inhibition! I tested these PRMT5 inhibitors in the HeLa cells I used to generate the 3 stable HTT knockdown lines previously (read more about that here: https://thesgc.github.io/static-openlabnotebooks/generating-htt-knockdown-models/) and looked at how the HTT protein levels were affected in these lines.</p>
<p style="text-align: justify;"><img class="alignnone wp-image-2170" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/20181116-HeLa-G3BP1-PRMT5i-and-HTT-KD-300x225.jpg" alt="" width="841" height="631" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/20181116-HeLa-G3BP1-PRMT5i-and-HTT-KD-300x225.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/20181116-HeLa-G3BP1-PRMT5i-and-HTT-KD-768x576.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/20181116-HeLa-G3BP1-PRMT5i-and-HTT-KD-350x263.jpg 350w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/20181116-HeLa-G3BP1-PRMT5i-and-HTT-KD-780x586.jpg 780w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/20181116-HeLa-G3BP1-PRMT5i-and-HTT-KD-250x188.jpg 250w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/20181116-HeLa-G3BP1-PRMT5i-and-HTT-KD.jpg 960w" sizes="(max-width: 841px) 100vw, 841px" /></p>
<p style="text-align: justify;">HTT protein levels were clearly decreased with PRMT5 inhibition! I ran these samples alongside the stable HTT knockdown lines and although the HTT levels did not decrease as drastically as with the knockdowns, the decrease with the PRMT5 inhibitors is still substantial. I also included SGC2096, the inactive control to GSK591, and HTT protein levels were not affected by it, showing that it is indeed through the activity of PRMT5 that HTT levels were decreased.</p>
<p style="text-align: justify;">The mechanism by which HTT is downregulated by PRMT5 inhibitors is still unclear and it will be very interesting to further investigate the link between HTT and PRMT5. As of now, the only publication I can find linking HTT to PRMT5 is the following:</p>
<p>&nbsp;</p>
<p style="text-align: justify;">PRMT5- mediated symmetric arginine dimethylation is attenuated by mutant huntingtin and is impaired in Huntington&#8217;s disease (HD).</p>
<p style="text-align: justify;">Ratovitski T, Arbez N, Stewart JC, Chighladze E, Ross CA.</p>
<p style="text-align: justify;">Cell Cycle. 2015;14(11):1716-29. doi: <a href="http://doi.org/10.1080/15384101.2015.1033595">10.1080/15384101.2015.1033595</a>.</p>
<p>&nbsp;</p>
<p style="text-align: justify;">This paper showed the colocalisation and interaction of HTT and PRMT5 and shows that this interaction results in the stimulation of PRMT5 activity by HTT in vitro. Mutant HTT has a greater affinity for PRMT5; however, it has a reduced ability to stimulate PRMT5 activity, as demonstrated by the reduction of symmetric dimethylation of arginine (SDMA), which is mainly mediated by PRMT5, of histones H2 and H4. Decreased levels of SDMA on known PRMT5 targets were detected in HD brain samples, suggesting that the reduction in PRMT5 activity may contribute to the pathogenesis of HD. The knockdown of JMJD6, which removes arginine demethylation, was able to rescue the phenotype of increased cell death seen with mutant HTT, further suggesting that the restoration of SDMA to normal levels may be a potential therapeutic strategy.</p>
<p style="text-align: justify;">However, our findings that PRMT5 inhibition itself can affect the expression of HTT at the protein level suggests that there might be a feedback loop in play between PRMT5 and HTT. It will be interesting to further investigate how the levels of SDMA and the other proteins involved are affected with PRMT5 inhibition and elucidate the mechanisms involved in this process. It should also be noted that our results only reflect the expression of wild-type HTT and not mutant (expanded) HTT, and it will be highly interesting and of great clinical importance to find out if PRMT5 inhibition is able to decrease the protein levels of mutant HTT similarly to how it affects wild-type HTT.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/reduction-of-htt-protein-through-chemical-inhibition/feed/</wfw:commentRss>
		<slash:comments>4</slash:comments>
		</item>
		<item>
		<title>Checking expression and localization of split-SCN1A constructs in HEK293 cells</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/checking-expression-and-localization-of-split-scn1a-constructs-in-hek293-cells/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/checking-expression-and-localization-of-split-scn1a-constructs-in-hek293-cells/#respond</comments>
		<pubDate>Sun, 18 Nov 2018 19:41:36 +0000</pubDate>
		<dc:creator><![CDATA[Wen Yih Aw]]></dc:creator>
				<category><![CDATA[Dravet syndrome]]></category>
		<category><![CDATA[Gene therapy for Dravet Syndrome]]></category>
		<category><![CDATA[Wen Yih Aw]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2163</guid>
		<description><![CDATA[In my last posts, I have constructed a pair of split-SCN1A constructs (split- SCN1A-GFP(1-10) and SCN1A-GFP(11)). Here, I tested the functionality of these constructs by looking at whether these split- constructs reconstitute into a full-length SCN1A and generate a fluorescent signal when co-transfected in HEK293 cells, and if so, do the reconstituted SCN1A (with an <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/checking-expression-and-localization-of-split-scn1a-constructs-in-hek293-cells/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>In my last posts, I have constructed a pair of split-SCN1A constructs (split- SCN1A-GFP<sub>(1-10)</sub> and SCN1A-GFP<sub>(11)</sub>). Here, I tested the functionality of these constructs by looking at whether these split- constructs reconstitute into a full-length SCN1A and generate a fluorescent signal when co-transfected in HEK293 cells, and if so, do the reconstituted SCN1A (with an internal GFP tag) localize correctly at the plasma membrane.</p>
<p>Below are my imaging results. The detailed experimental procedure can be found at <a href="https://zenodo.org/record/1490851#.W_G_bpNKh3g">https://zenodo.org/record/1490851#.W_G_bpNKh3g</a></p>
<p><img class="aligncenter size-large wp-image-2165" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/dravet-pic5-1024x752.jpg" alt="" width="780" height="573" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/dravet-pic5-1024x752.jpg 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/dravet-pic5-300x220.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/dravet-pic5-768x564.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/dravet-pic5.jpg 1424w" sizes="(max-width: 780px) 100vw, 780px" /></p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p><img class="aligncenter size-large wp-image-2164" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/dravet-pic6-1024x404.png" alt="" width="780" height="308" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/dravet-pic6-1024x404.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/dravet-pic6-300x118.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/dravet-pic6-768x303.png 768w" sizes="(max-width: 780px) 100vw, 780px" /></p>
<p>&nbsp;</p>
<p>In short, these split-constructs managed to find each other and reconstitute a GFP signal in HEK293 cells. However, the GFP signals do not overlap with cell cortex or plasma membrane markers (cell cortex marked by cortical actin staining and plasma membrane marked by CD8, a transmembrane glycoprotein.) The distribution of reconstituted SCN1A-GFP signals resembles misfolded proteins that are trapped in the ER.  Co-expression of SCN1B, a beta subunit which is thought to promote sodium channel trafficking, does not seem to improve the localization of reconstituted SCN1A-GFP.</p>
<p>There are several caveats with the design of these constructs which might have affected folding and trafficking of reconstituted SCN1A: 1) instead of using the canonical SCN1A cDNA sequence (which is  unstable and prone to mutagenize when propagated in E. coli), a codon-optimized SCN1A cDNA was used as a template in cloning these constructs and 2) the reconstituted protein has a foreign GFP sequence which might have interfered with SCN1A trafficking. Moving forward, I will revisit and examine the localization of codon optimized SCN1A in HEK293 cells while continue constructing the rest of the split-intein constructs.</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/checking-expression-and-localization-of-split-scn1a-constructs-in-hek293-cells/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Some structure pictures for your viewing pleasure</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/some-structure-pictures-for-your-viewing-pleasure/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/some-structure-pictures-for-your-viewing-pleasure/#respond</comments>
		<pubDate>Thu, 08 Nov 2018 16:04:46 +0000</pubDate>
		<dc:creator><![CDATA[Roslin Adamson]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Brain Tumours]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>
		<category><![CDATA[Roslin Adamson]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2127</guid>
		<description><![CDATA[So I reported here a couple of weeks ago that I&#8217;d obtained datasets for ACVR1 bound to M4K2117 and M4K2121, at 1.25 and 2 Å resolution, respectively, and I put up a link to the data. I&#8217;m now refining the data and got some pretty good structures so far. These are what the current stats look <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/some-structure-pictures-for-your-viewing-pleasure/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>So I reported here a couple of weeks ago that I&#8217;d obtained datasets for ACVR1 bound to M4K2117 and M4K2121, at 1.25 and 2 Å resolution, respectively, and I put up a link to the data. I&#8217;m now refining the data and got some pretty good structures so far. These are what the current stats look like, below. There&#8217;s still some way to go with the refinement, but they look pretty good so far!</p>
<p><img class="aligncenter  wp-image-2129" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/Structure-stats-1024x576.png" alt="" width="873" height="491" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/Structure-stats-1024x576.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/Structure-stats-300x169.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/Structure-stats-768x432.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/Structure-stats.png 1280w" sizes="(max-width: 873px) 100vw, 873px" /></p>
<p>I&#8217;ve also made some pretty pictures of the bound ligands, some or which were used for the M4K progress meeting yesterday. As can be seen below, the water molecule co-ordinating two of the Os on the trimethoxyphenyl is there, as in previous structures, and this is coordinated by another water and Lys235. The interaction of the hinge region of both ligands with His286 is also there. The solvent side is uncoordinated in the M4K2117 structure, but there&#8217;s a water there in M4K2121, which is less mobile in that region anyway, so that is not too surprising. It&#8217;s truly fabulous how we can easily see in these structures the chemical space that can still be explored to fill up the pocket, but I&#8217;m acutely aware of how many other important criteria a drug candidate has to fulfil in order to actually do its job properly. Hopefully, with the excellent expertise we have working on this project in all the different areas needed to make an effective drug, the structural information will be of use. Now, I have some more compounds to screen &#8230;</p>
<p><img class="aligncenter  wp-image-2130" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/M4K2117-1024x576.png" alt="" width="842" height="474" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/M4K2117-1024x576.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/M4K2117-300x169.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/M4K2117-768x432.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/M4K2117.png 1280w" sizes="(max-width: 842px) 100vw, 842px" /> <img class="aligncenter  wp-image-2131" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/M4K2121-1024x576.png" alt="" width="830" height="466" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/M4K2121-1024x576.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/M4K2121-300x169.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/M4K2121-768x432.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/M4K2121.png 1152w" sizes="(max-width: 830px) 100vw, 830px" /></p>
<p>&nbsp;</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/some-structure-pictures-for-your-viewing-pleasure/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Design of split-intein mediated SCN1A protein trans-splicing</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/2047-2/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/2047-2/#respond</comments>
		<pubDate>Mon, 29 Oct 2018 00:28:39 +0000</pubDate>
		<dc:creator><![CDATA[Wen Yih Aw]]></dc:creator>
				<category><![CDATA[Dravet syndrome]]></category>
		<category><![CDATA[Gene therapy for Dravet Syndrome]]></category>
		<category><![CDATA[Wen Yih Aw]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2047</guid>
		<description><![CDATA[Previously, I provided a brief overview on SCN1A related Dravet Syndrome (DS) and the need for larger gene therapy vectors or alternative methods to deliver functional copies of SCN1A to affected cells in DS. The goal of my project is to explore one of these alternative methods, specifically a split-intein mediated protein trans-splicing method, that <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/2047-2/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Previously, I provided a brief overview on SCN1A related Dravet Syndrome (DS) and the need for larger gene therapy vectors or alternative methods to deliver functional copies of SCN1A to affected cells in DS. The goal of my project is to explore one of these alternative methods, specifically a split-intein mediated protein trans-splicing method, that could reduce the size of the passenger (SCN1A expression cassettes) for gene therapy vectors. In this post, I will talk about the overall design for this method.</p>
<p>To give you a little more background about inteins: Inteins (also known as intervening proteins) are basically mobile protein elements that are translated and embedded within another protein sequences. You can think of them as single-use, single-turnover enzymes that catalyze their own excision from a flanking precursor “exteins” protein sequences while concurrently joining the remaining exteins protein fragments together through the formation of a new peptide bond (Fig. 1).</p>
<p><img class="size-medium wp-image-2048 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/10/intein-inbase-300x185.png" alt="" width="300" height="185" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/10/intein-inbase-300x185.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/10/intein-inbase.png 736w" sizes="(max-width: 300px) 100vw, 300px" /></p>
<p style="text-align: center;">Figure 1. Intein-mediated protein splicing vs. RNA splicing.  Image source: Perler, F. B. (2002). InBase, the Intein Database. Nucleic Acids Res. 30, 383-384.</p>
<p>There are two types of inteins – cis splicing inteins that are embedded within protein sequences and catalyze protein splicing <em>in cis</em> and split- or trans-splicing inteins where the enzyme is naturally split and expressed into two parts. These two fragments of split-intein can associate with each other to form a catalytic active enzyme that can catalyze protein splicing <em>in trans. </em>Briefly, my project aims to split the SCN1A expression cassette into two parts and make use of the unique split-intein mediated trans-splicing capability to rejoin the expressed SCN1A fragments together (Fig. 2)</p>
<p><img class="size-medium wp-image-2050 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/10/dravet-pic4-216x300.png" alt="" width="216" height="300" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/10/dravet-pic4-216x300.png 216w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/10/dravet-pic4.png 279w" sizes="(max-width: 216px) 100vw, 216px" /></p>
<p style="text-align: center;">Figure 2. A basic design of split-intein mediated SCN1A protein trans-splicing</p>
<p>So, here goes the basic components and design of this cloning experiment. We are going to express SCN1A as two protein fragments (split-N terminal and split-C terminal SCN1A), each fused with split-N-term and C-term intein sequence (Fig. 2), respectively.</p>
<p>But&#8230; where do we split the protein?</p>
<p>One of the factors taken into consideration in designing the split sites is that for efficient protein trans-splicing, the split sites or the sites where split-intein sequences are inserted need to be surface exposed. SCN1A is a multipass membrane proteins with four homologous (unexposed) transmembrane domains (DI to DIV), each domain is connected to the next domain by large cytoplasmic loops (Fig. 3). In fulfilling this condition, we are left with the N and C-terminal cytoplasmic domain and the cytoplasmic loops connecting DI to DII, DII to DIII, and DIII to DIV to work with. Among these cytoplasmic regions, splitting cytoplasmic loops within and between DII to DIII can result in a significant reduction in the size of SCN1A expression cassettes.</p>
<p>Additionally, to initiate efficient splicing reaction, split-inteins require a certain combination of amino acids flanking the intein sequences. For instance, split DnaB intein from <em>Synechocystis</em> needs to be flanked by a glycine at the N-terminus of the intein and a serine at C-terminus of the intein to catalyze protein trans-splicing whereas Vma intein from <em>S. cerevisiae </em>requires a glycine and cysteine amino acids combination.  In addition to previous considerations, these specific requirements of amino acids combinations for efficient and seamless protein trans-splicing helped us further narrow down a short list of potential SCN1A split sites.</p>
<p><img class="size-medium wp-image-2051 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/10/scn1a-300x134.png" alt="" width="300" height="134" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/10/scn1a-300x134.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/10/scn1a-768x342.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/10/scn1a-1024x456.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/10/scn1a.png 1488w" sizes="(max-width: 300px) 100vw, 300px" /></p>
<p style="text-align: center;">Figure 3. A general topology of eukaryotic voltage-gated sodium channels.</p>
<p style="text-align: center;">Image source: Shen, Huaizong, et al. “Structure of a Eukaryotic Voltage-Gated Sodium Channel at near-Atomic Resolution.” <em>Science</em>, American Association for the Advancement of Science, 3 Mar. 2017, science.sciencemag.org/content/355/6328/eaal4326.long.</p>
<p>&nbsp;</p>
<p>How can we tell whether two split-SCN1A fragments trans-spliced?</p>
<p>To screen and identify constructs that have successfully reconstituted, I will attach split-GFP sequences to the N and C termini of the split-N-terminal-SCN1A intein and split-C-terminal-SCN1A-intein, respectively. Split-GFP is composed of two complementing split fluorescent protein fragments that irreversibly associate with each other to generate a fluorescent signal. When expressing both of these constructs in cells, we predicted that the split-GFP sequence would help bring the two split-proteins together (Fig. 4.) The split-intein would interact with each other to form an active enzyme and catalyze the excision of both itself and GFP protein to the cytoplasm. In addition to using the cytoplasmic GFP signal as a visual positive marker for trans-splicing, the split-GFP system could also be utilized as an alternative way to link two SCN1A fragments together.</p>
<p>So far, I have generated a pair of split-GFP construct to examine one of the split-site. You can find more details of these experiments on <a href="https://zenodo.org/record/1473859#.W9er4pNKh3g">zenodo</a>. Going forward, I will be transfecting these constructs into HEK293 cells to examine the following questions: Do these split-protein fragments reconstitute into full-length SCN1A. If so, do they localize and function similar to wild-type SCN1A? Does overproduction of split-protein fragments cause a dominant negative effect on wild-type SCN1A?</p>
<p>&nbsp;</p>
<p><img class="aligncenter size-large wp-image-2058" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/10/dravet-pic3-1024x223.png" alt="" width="780" height="170" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/10/dravet-pic3-1024x223.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/10/dravet-pic3-300x65.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/10/dravet-pic3-768x168.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/10/dravet-pic3.png 1132w" sizes="(max-width: 780px) 100vw, 780px" /></p>
<p style="text-align: center;">Figure 4. A fluorescence reporter system for identifying SCN1A trans-splicing.</p>
<p>&nbsp;</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/2047-2/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Some good news! I have data! Not just any data &#8211; 1.25 Å data!</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/some-good-news-i-have-data-not-just-any-data-1-25-a-data/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/some-good-news-i-have-data-not-just-any-data-1-25-a-data/#comments</comments>
		<pubDate>Fri, 19 Oct 2018 15:58:24 +0000</pubDate>
		<dc:creator><![CDATA[Roslin Adamson]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Brain Tumours]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>
		<category><![CDATA[Roslin Adamson]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2019</guid>
		<description><![CDATA[So, the week has been super busy, but all for good reasons this time!! I&#8217;m absolutely thrilled to say that amongst all the salt crystals I found in my crystal plates, I also found the crystals below, one of which diffracted to 1.25 Å. This is definitely the highest resolution I&#8217;ve ever worked with. It&#8217;s amazing! <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/some-good-news-i-have-data-not-just-any-data-1-25-a-data/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>So, the week has been super busy, but all for good reasons this time!! I&#8217;m absolutely thrilled to say that amongst all the salt crystals I found in my crystal plates, I also found the crystals below, one of which diffracted to 1.25 Å. This is definitely the highest resolution I&#8217;ve ever worked with. It&#8217;s amazing! You can see everything &#8211; as shown in the pic below, phenix found the ligand quite easily. This was with M4K2117. The other two datasets I got were at about 2 Å, also really good, and they&#8217;re for M4K2121. Clearly the use of the composite screen I set up has increased the hit rate, and allowed me to screen a lot more compounds more quickly and at both room temperature and 4 degrees.</p>
<p>The link to the autoprocessed .mtz files is <a href="https://zenodo.org/record/1475852#.W9mR4JP7SUk">here.</a></p>
<p>I&#8217;ve already set up 20 more plates with 10 more compounds &#8211; fingers crossed we get more crystals and more data! A brilliant end to a tough few weeks!</p>
<p>&nbsp;</p>
<p><img class="aligncenter size-large wp-image-2020" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/10/ACVR1A-c096-datasets-1024x576.png" alt="" width="780" height="439" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/10/ACVR1A-c096-datasets-1024x576.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/10/ACVR1A-c096-datasets-300x169.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/10/ACVR1A-c096-datasets-768x432.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/10/ACVR1A-c096-datasets.png 1280w" sizes="(max-width: 780px) 100vw, 780px" /></p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/some-good-news-i-have-data-not-just-any-data-1-25-a-data/feed/</wfw:commentRss>
		<slash:comments>5</slash:comments>
		</item>
	</channel>
</rss>
