<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>David Drewry &#8211; openlabnotebooks.org</title>
	<atom:link href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/david-drewry/feed/" rel="self" type="application/rss+xml" />
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<description>A growing team of groundbreaking scientists around the world are now sharing their lab notebooks online</description>
	<lastBuildDate>Mon, 24 Aug 2020 18:33:44 +0000</lastBuildDate>
	<language>en-GB</language>
	<sy:updatePeriod>
	hourly	</sy:updatePeriod>
	<sy:updateFrequency>
	1	</sy:updateFrequency>
	<generator>https://wordpress.org/?v=5.4.2</generator>

<image>
	<url>https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/cropped-lab-jar02-32x32.png</url>
	<title>David Drewry &#8211; openlabnotebooks.org</title>
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<width>32</width>
	<height>32</height>
</image> 
	<item>
		<title>Brachyury (TBXT) ligands for chordoma: 1. Fragment optimization strategy 2. fragment F9000505, B-site, PDB 5QRU</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/brachyury-tbxt-ligands-for-chordoma-1-fragment-optimization-strategy-2-fragment-f9000505-b-site-pdb-5qru/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/brachyury-tbxt-ligands-for-chordoma-1-fragment-optimization-strategy-2-fragment-f9000505-b-site-pdb-5qru/#respond</comments>
		
		<dc:creator><![CDATA[David Drewry]]></dc:creator>
		<pubDate>Mon, 24 Aug 2020 18:33:44 +0000</pubDate>
				<category><![CDATA[Chordoma]]></category>
		<category><![CDATA[David Drewry]]></category>
		<category><![CDATA[SGC UNC Chapel Hill]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=5102</guid>

					<description><![CDATA[Introduction In this blog post I will share details on the new molecules we have designed and synthesized or purchased based on one of our original hits from the x-ray crystallography fragment screen. The compound is called F9000505 and it is bound to pocket B. The crystal structure was deposited in the Protein Data Bank <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/brachyury-tbxt-ligands-for-chordoma-1-fragment-optimization-strategy-2-fragment-f9000505-b-site-pdb-5qru/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p><strong><u>Introduction</u></strong></p>
<p>In this blog post I will share details on the new molecules we have designed and synthesized or purchased based on one of our original hits from the x-ray crystallography fragment screen. The compound is called F9000505 and it is bound to pocket B. The crystal structure was deposited in the Protein Data Bank (pdb) with PDB ID 5QRU. This post will give you a feel for some of the ways we are approaching fragment optimization. Posts in the near future will describe strategies at other fragment sites and also provide data on the binding of these compounds to brachyury as it emerges.</p>
<p><strong><u>Background / Project Goal</u></strong></p>
<p>The expression of brachyury (TBXT) is a hallmark of chordoma. A growing body of data suggests that inhibition of TBXT protein activity, expression, or interactions with protein partners are predicted to be effective and specific means for treating chordoma. A key aim of this project is to discover compounds that bind directly to brachyury with sufficient affinity so that they ca be optimized into clinical candidates for the treatment of chordoma.</p>
<p><strong><u>Initial Fragment Optimization Strategy </u></strong></p>
<p>Fragment-based screening by x-ray crystallography is very powerful in that the location of the fragment hits on the target protein (brachyury) are known after the experiment, and the structure provides details of the interactions that the fragments make with brachyury. Crystallography can identify fragments that bind very weakly, so there is often a long way to go to grow the hit molecules into compound that bind potently enough to elicit the desired biological effects. We are taking an aggressive approach to exploring each of our hits so that we can soon focus efforts on the most promising molecules and binding sites.</p>
<p>The two sources of new molecules are purchase of commercially available compounds, and design and synthesis of new compounds. Purchase of compounds is cost effective and rapid in the early stages of a project but becomes limited as molecules become more complicated to capture the intricacies of the binding sites.</p>
<p>We have purchased compounds primarily through a company called ChemSpace. We performed substructure and similarity searches in the ChemSpace available compound database for each x-ray crystallography hit (“query” molecules for the searching). A substructure search means specific details of the hit need to be present in the molecules identified by the search. A similarity search relaxes the criteria and allows for selection of molecules that the computer tells us have certain similarities to our query molecules. These similarities could be in categories like size, shape, and electronic properties. Similarity searches can identify compounds that are more different from the hits which may be important if they identify new interactions, scaffolds that are easier to make new analogs of, or compound that explore different vectors (directions) around the hits.</p>
<p>We chose initial analogues for purchase to provide important information around each fragment in several domains:</p>
<ol>
<li>Compounds with functional groups that will serve as efficient handles for further elaboration.</li>
<li>Compounds that explore growth of the hit fragment in logical directions based on the crystal structure.</li>
<li>Compounds that add additional functionality that can make new interactions in order to enhance potency.</li>
<li>Compounds that teach us where linkers may be added in order to develop brachyury degraders.</li>
</ol>
<p>By making the fragments larger we can understand where growth is beneficial to potency. This will provide focus to future rounds of purchase or chemistry. Incorporation of chemical handles that can be rapidly elaborated with a variety of building blocks will allow us to make additional analogs quickly. Adding functionality (for example amides, ureas, phenols, heterocycles, substituted phenyls) that can make energetically favorable interactions (hydrogen bonds, pi-pi stacking, lipophilic interactions, charge-charge interactions) improves our chances of increasing potency. Utilizing commercially available analogs of the initial fragments found bound to the TBXT crystal structure provides the means to quickly generate structure activity relationships (SAR – this means how changing the structure changes how well the compound binds to brachyury) around each fragment. Initial SAR is critical as it provides confidence around each hit (can it be optimized, do several similar compounds also bind) and aids in site selection to narrow our focus.</p>
<p><strong><u>Fragment F9000505, binding site B, PDB ID: 5QRU </u></strong></p>
<p><em> </em><em>Compounds for purchase</em></p>
<p><strong>Figure 1</strong> depicts all the compounds that we could buy identified from substructure and similarity searching around fragment hit F9000505. This compound binds to site B, and more details on the way it binds will be found below in the new compound design section. Each dot represents a compound and they are organized in the plot according to two easy to calculate and important properties: lipophilicity (clogP) and molecular weight (MW). The green circles are the compounds that we have purchased and are screening. If any of these are active, we can purchase other compounds (blue dots) that can further supplement the SAR. This iterative cycle of design, synthesis (purchase if possible), and test is the life blood of medicinal chemistry compound optimization.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig1-1024x725.png" alt="" width="780" height="552" class="aligncenter wp-image-5104 size-large" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig1-1024x725.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig1-300x213.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig1-768x544.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig1.png 1200w" sizes="(max-width: 780px) 100vw, 780px" /></p>
<p><strong>Figure 2</strong> depicts examples of molecules we purchased from this set that demonstrate the availability of useful molecules that can answer some of the specific medicinal chemistry questions we need to ask. The blue boxes highlight the areas with structural modifications to the hit compound.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-2.png" alt="" width="643" height="604" class="aligncenter wp-image-5105 " srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-2.png 978w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-2-300x282.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-2-768x721.png 768w" sizes="(max-width: 643px) 100vw, 643px" /></p>
<p><em>Design and Synthesis of New Compounds</em></p>
<p>We have chemists on our team at UNC and also a talented team of chemists at a company called Piramal. We have been able to make excellent progress on new compound synthesis even with the initial lab shutdowns and then the reduced capacity necessitated by Covid-19. The team at Piramal has focused on preparation of analogs of fragments for which there are fewer commercial analogs, and on more elaborate chemistry to explore promising fragment sites in greater detail using the same principles outlined above.</p>
<p><strong>Figure 3</strong> depicts the crystal structure of fragment F9000505 bound to pocket B of brachyury. This simple compound results from the acylation of 2-methoxy-5-methylaniline with the amino acid glycine. The phenyl ring makes lipophilic contacts with the protein and the 5-methyl group tucks into a lipophilic pocket. The oxygen of the 2-methoxy group makes an intramolecular hydrogen bond with the adjacent NH, helping to orient the trajectory of the carbonyl of the amide and also the terminal amine of the glycine. In this locked in orientation the carbonyl of the fragment forms a hydrogen bond with Ala119 of brachyury and the primary amine forms a hydrogen bond to the carbonyl of Pro117.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog2-fig3-286x300.png" alt="" width="315" height="330" class="aligncenter wp-image-5106 " srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog2-fig3-286x300.png 286w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog2-fig3-768x806.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog2-fig3.png 806w" sizes="(max-width: 315px) 100vw, 315px" /></p>
<p>Based on this structure there are a number of avenues that need to be explored for optimization of binding. The methyl substituent in the lipophilic pocket may not be ideal. Groups with different size and electronic characteristics need to be explored (for example: F, Cl, ethyl, cyclopropyl). There is plenty of room to extend off of the methoxy group. Doing this may allow us to pick up additional interactions with the surface of the protein. In addition, this may be an attractive position for a linker that will be suitable for synthesis of a bifunctional degrader. Another strategy is to extend the primary amine in order to reach Glu116 which could increase affinity. The alpha carbon of the glycine moiety presents another opportunity. Simple substituents like methyl will add conformational constraint which can enhance potency. Because this building block comes from amino acids, there is opportunity to quickly explore stereochemistry at this position and a range of functional groups that have potential to form new interactions. Finally, we can form a 5- or 6-membered ring between the NH and the O of the fragment. This will serve to further lockdown the orientation of the carbonyl and NH2 that make key hydrogen bonds and offer different vectors for exploring interactions with other brachyury residues. Our initial compound design targeted 68 molecules. These molecules have all been synthesized and submitted from screening in the surface plasmon resonance (SPR) binding assay. A sampling of the 68 compounds are depicted in <strong>Figure 4</strong>. These show example modifications of the methyl group (<strong>4A</strong>), primary amine (<strong>4B</strong>), the alpha carbon of the glycine moiety (<strong>4C</strong>), and the methyl ether (<strong>4D</strong>).</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-4.png" alt="" width="861" height="573" class="wp-image-5107 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-4.png 1538w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-4-300x200.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-4-1024x682.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-4-768x511.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-4-1536x1023.png 1536w" sizes="(max-width: 861px) 100vw, 861px" /></p>
<p><em>Synthetic chemistry schemes</em></p>
<p><strong>Figure 5</strong> show the chemistry scheme used to make many of the 68 new analogues of F9000505 synthesized for this project.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-5.png" alt="" width="698" height="172" class="alignnone wp-image-5108 " srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-5.png 1434w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-5-300x74.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-5-1024x253.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-5-768x190.png 768w" sizes="(max-width: 698px) 100vw, 698px" /></p>
<p><strong>Figure 6</strong> demonstrates utilization of an easily synthesized key intermediate to explore the addition of new functionality at three different distances from the core, linked with an amide. If any offer enhanced potency, follow up will focus on modifications of the pendant aryl ring.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-6.png" alt="" width="584" height="275" class="wp-image-5109 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-6.png 1332w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-6-300x141.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-6-1024x483.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-6-768x362.png 768w" sizes="(max-width: 584px) 100vw, 584px" /></p>
<p><strong>Figure 7</strong> depicts three different ways to functionalize the same key intermediate with robust chemistry that provides different sets of products. A key element of the initial fragment analogue strategy is to generate and use key intermediates that can be diversified in the last synthetic step.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-7.png" alt="" width="396" height="220" class="wp-image-5110 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-7.png 910w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-7-300x167.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-7-768x427.png 768w" sizes="(max-width: 396px) 100vw, 396px" /></p>
<p>The crystal structure suggests that new interactions may be possible if we grow the molecule in the ether position. In addition to making several different ethers, we have opted to incorporate both an alkyne linker and a triazole linker in this position. These choices offer different spacing, different angles, different functionalities, and can be expanded with robust chemistry. An overview of this linker modification chemistry is shown in <strong>figure 8</strong>.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-8.png" alt="" width="797" height="458" class="aligncenter wp-image-5111 " srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-8.png 1494w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-8-300x172.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-8-1024x588.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-8-768x441.png 768w" sizes="(max-width: 797px) 100vw, 797px" /></p>
<p>Finally, a key element of the strategy is to identify positions on the molecules where linkers can be attached to generate bifunctional degradation reagents. Crystallography will of course provide critical information to guide linker location. To improve our chances of quickly identifying potential linker locations, we are routinely incorporating some building blocks with functionality that lends itself to linker friendly chemistry. Currently these are mostly simple amides and Boc-protected amines, but azides and alkynes would also be valuable additions. Some newly synthesized compounds with “linker-friendly” functionality are shown in <strong>figure 9</strong>.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog2-fig-9.png" alt="" width="605" height="294" class="wp-image-5112 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog2-fig-9.png 1290w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog2-fig-9-300x146.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog2-fig-9-1024x499.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog2-fig-9-768x374.png 768w" sizes="(max-width: 605px) 100vw, 605px" /></p>
<p>&nbsp;</p>
<p>Stay tuned for details on compounds we have made and purchased for other sites and, importantly, the SPR screening data that should be available soon that lets us know how we are doing in terms of binding affinity!</p>
<p>Don&#8217;t hesitate to reach out with questions and comments.</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/brachyury-tbxt-ligands-for-chordoma-1-fragment-optimization-strategy-2-fragment-f9000505-b-site-pdb-5qru/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>TBXT (brachyury) and Chordoma: Project Introduction</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/tbxt-brachyury-and-chordoma-project-introduction/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/tbxt-brachyury-and-chordoma-project-introduction/#comments</comments>
		
		<dc:creator><![CDATA[David Drewry]]></dc:creator>
		<pubDate>Tue, 14 Apr 2020 19:49:19 +0000</pubDate>
				<category><![CDATA[Chordoma]]></category>
		<category><![CDATA[David Drewry]]></category>
		<category><![CDATA[SGC UNC Chapel Hill]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=4516</guid>

					<description><![CDATA[In my Chordoma Open Lab Notebook entries, I will tell you about a project that is very important to me, targeting the transcription factor brachyury (also known as TBXT) for the rare cancer chordoma. In the coming weeks and months, the amount of science strategy, science details, and science progress shared here will rise. Today’s <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/tbxt-brachyury-and-chordoma-project-introduction/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p>In my Chordoma Open Lab Notebook entries, I will tell you about a project that is very important to me, targeting the transcription factor brachyury (also known as TBXT) for the rare cancer chordoma. In the coming weeks and months, the amount of science strategy, science details, and science progress shared here will rise. Today’s entry will simply introduce the project to you, tell you how we got here, and share my motivation. By the way, it will help me and the team if you let me (us!) know what you think, what questions you have, and what you’d like to know. Please feel free to share these posts with others via email links or social media. In addition, a long career has taught me that I don’t have all the answers! If you have ideas, please share them with us!</p>
<p>Working on Chordoma is personal for me. Eighteen years ago, my daughter Carolyn was in first grade and a bright and joyful little boy in her class named Justin (pictured below, graphic from Chordoma Foundation) was diagnosed with chordoma. Of course, I had never heard about this rare cancer which strikes only one in a million people. In the US your chances of getting hit by lightning are higher. Justin and his wonderful parents shared his difficult journey with his classmates and their parents. Justin wanted to talk about this. Justin drew his friends in and inspired them to find joy and strength and motivation in his journey.  Their enthusiasm in turn inspired me. As a scientist I wanted to help for Justin, Justin’s parents, my daughter Carolyn, and all of Justin’s classmates who gave what they could of their time, talents, and treasure.</p>
<p><img src="https://chordomafoundation.org/wp-content/uploads/2011/09/Justin%20Perseverance.jpg" width="427" height="285" class="alignleft" /></p>
<p>A number of years after Justin’s diagnosis I met Josh Sommer, a Chordoma patient himself, and Co-Founder and Executive Director of the Chordoma Foundation. Josh attended a number of fund raisers that my daughter and her classmates were involved in, and one thing led to another and I found ways to get involved. Josh inspired me with his positive attitude, drive, and ability to band people together. He is a connector, and science thrives on connections. We are all better when we work together. Josh introduced me to Dr. Adrienne Flanagan, a leader in chordoma research, a key thought leader on the links between brachyury and chordoma, and a real blessing to the chordoma community for her hard work, insights, and passion. I was able to get involved by leveraging an open science project I was working on at the pharmaceutical company GlaxoSmithKline (GSK) sharing inhibitors of an important class of proteins called kinases. Adrienne and her team tested them, and we were able to make some interesting connections between kinase inhibition and chordoma cell line growth. I believe there is more to be done with kinase inhibition and chordoma, but that’s a story for another day.</p>
<p>I have since moved from GSK to the Eshelman School of Pharmacy at the University of North Carolina at Chapel Hill. Our SGC-UNC team (<a href="https://www.sgc-unc.org/">https://www.sgc-unc.org/</a>) is the first US site of the Structural Genomics Consortium (SGC, <a href="https://www.thesgc.org/">https://www.thesgc.org/</a>) a worldwide public private partnership committed to supporting the discovery of new medicines through open science. We generate tools and reagents and share them with the scientific community openly and without restrictions. We want to move quickly, we want to facilitate research programs in other groups, and we want to help patients.</p>
<p><strong><u>TBXT (brachyury): a key target for chordoma</u></strong></p>
<p>Emerging data suggests that the transcription factor TBXT is a key vulnerability of chordoma and a prime candidate to target in order to create medicines to treat chordoma. For example, TBXT knockdown stops the proliferation of chordoma cell lines and reduces their viability. TBXT is the top selectively essential gene in chordoma. A great place to start if you want to know more about TBXT and chordoma is this paper by Tanaz Sharifnia and colleagues (link here: <a href="https://pubmed.ncbi.nlm.nih.gov/30664779/">https://pubmed.ncbi.nlm.nih.gov/30664779/</a>). As a transcription factor, it falls in the “difficult target” category. There is no active site tailor made for small molecules. Details of downstream signaling via protein-protein interactions are poorly understood. On the plus side, TBXT is expressed in development but almost nonexistent (or very low expression) in most adult human tissues, suggesting that if we are successful, compounds may have a useful therapeutic index. The high unmet need and weight of evidence convince us that TBXT is an Achilles heel of chordoma that needs to be addressed through a targeted drug discovery program.</p>
<p>Our initial thought was to see if we could identify small molecule binders for TBXT using x-ray crystallography. To get there we needed a number of things to fall into place. I am grateful to the Mark Foundation and Chordoma Foundation for funding our initial proposal that allowed us to generate enough data to convince ourselves that this strategy holds great promise.</p>
<p>As part of the SGC, I was well aware of the SGC Target Enabling Package (TEP) program and realized it was an ideal mechanism to facilitate study of brachyury. You can read more about the TEP program here: <a href="https://www.thesgc.org/tep">https://www.thesgc.org/tep</a>. I was excited to nominate TBXT as a TEP target, and absolutely thrilled when SGC scientist Opher Gileadi (Oxford University, SGC) agreed to work on the project. Opher and his talented team started the project by expressing and purifying the TBXT protein and then in short order they identified conditions to reproducibly grow crystals of the protein. They then collaborated with the team at Diamond to do an x-ray crystallography based high throughput fragment screen using the XChem platform (more information on XChem here:  <a href="https://www.diamond.ac.uk/Instruments/Mx/Fragment-Screening.html">https://www.diamond.ac.uk/Instruments/Mx/Fragment-Screening.html#</a>). This resulted in more than 40 fragments binding to a number of different sites on the transcription factor. As a medicinal chemist, it is an exciting day when you see a range of small molecules bound to your target of interest! With x-ray evidence that there are hot spots for binding small molecules to TBXT, we have now set our sights on a more challenging goal and bigger prize, optimization of these fragments into molecules that modulate TBXT biology and kill chordoma cells. In coming posts, we will provide more detail on the initial fragment screen and start walking through the steps we are taking.</p>
<p><strong><u>Some of our project goals</u></strong></p>
<ul>
<li>Develop and provide reagents the community can use to explore TBXT biology (eg: compounds, protein constructs, biophysical assays, cell assays, antibodies)</li>
<li>Develop assays to measure affinity (thermal shift, SPR)</li>
<li>Synthesize analogues of fragments at each site in order to optimize binding potency</li>
<li>Generate crystal structures of new and improved compounds</li>
<li>Generate Structure Activity Relationships that allow us or others to generate reagents that can lead to the degradation of TBXT</li>
<li>Understand impact potent fragments (and degraders) have on TBXT biology</li>
</ul>
<p><strong><u>Project Summary – things you will hear more about as the project progresses</u></strong></p>
<p>To date we have exemplified reproducible crystallography of TBXT and demonstrated the utility of fragment screening as a means of identifying TBXT hotspots for binding small molecules. The primary aim of the current project is to develop small molecule inhibitors of TBXT or of its interaction partners and <strong><em>identify at least one tractable lead molecule targeting TBXT that has potential for final optimization into a candidate compound</em></strong>. As we progress towards this end, we also aim to understand the structural basis of the activities of TBXT and contribute to the understanding of the mechanism of the disease. <strong><em>At the end of this two-year project, we will have the infrastructure, assays, reagents, and compounds in place that form the basis for a drug discovery project to move TBXT modulators into the clinic.</em></strong></p>
<p>In the coming weeks different members of the project team will provide updates. Here is a very brief team introduction. Currently our team is located at three sites.</p>
<ul>
<li>SGC-UNC at the UNC Eshelman School of Pharmacy: David Drewry, Carrow Wells</li>
<li>SGC at Oxford University: Opher Gileadi, Joseph Newman, Angeline Gavard</li>
<li>Institute for Cancer Research: Paul Workman, Paul Clarke, Hadley Sheppard</li>
</ul>
<p>We also have guidance and support from the teams at our generous funders, the Mark Foundation for Cancer Research (Becky Bish, Ryan Schoenfeld, Michele Cleary) and the Chordoma Foundation (Josh Sommer, Joan Levy). Thank you!</p>
<p>You can find out more about these organizations here:</p>
<p>Mark Foundation for Cancer Research: <a href="https://themarkfoundation.org/">https://themarkfoundation.org/</a></p>
<p>The Chordoma Foundation: <a href="https://www.chordomafoundation.org/">https://www.chordomafoundation.org/</a></p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/tbxt-brachyury-and-chordoma-project-introduction/feed/</wfw:commentRss>
			<slash:comments>18</slash:comments>
		
		
			</item>
	</channel>
</rss>
