<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>Danton Ivanochko &#8211; openlabnotebooks.org</title>
	<atom:link href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/danton-ivanochko/feed/" rel="self" type="application/rss+xml" />
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<description>A growing team of groundbreaking scientists around the world are now sharing their lab notebooks online</description>
	<lastBuildDate>Mon, 09 Nov 2020 14:37:06 +0000</lastBuildDate>
	<language>en-GB</language>
	<sy:updatePeriod>
	hourly	</sy:updatePeriod>
	<sy:updateFrequency>
	1	</sy:updateFrequency>
	<generator>https://wordpress.org/?v=5.5.3</generator>

<image>
	<url>https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/cropped-lab-jar02-32x32.png</url>
	<title>Danton Ivanochko &#8211; openlabnotebooks.org</title>
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<width>32</width>
	<height>32</height>
</image> 
	<item>
		<title>Interrogating PR/SET domains for the ability to bind SAH</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/interrogating-pr-set-domains-for-the-ability-to-bind-sah/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/interrogating-pr-set-domains-for-the-ability-to-bind-sah/#respond</comments>
		
		<dc:creator><![CDATA[Danton Ivanochko]]></dc:creator>
		<pubDate>Mon, 09 Nov 2020 14:37:06 +0000</pubDate>
				<category><![CDATA[Cheryl Arrowsmith (Univ. of Toronto)]]></category>
		<category><![CDATA[Danton Ivanochko]]></category>
		<category><![CDATA[PRDM methyltransferases]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=5380</guid>

					<description><![CDATA[This is an excerpt from my PhD thesis where I investigated 13 human PR/SET domains from the PRDM protein family and assessed their ability to bind to a fluorinated SAH analog. The research helps to identify which of the PRDMs could be methyltransferases. This work was carried out in the laboratory of Dr. Cheryl Arrowsmith <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/interrogating-pr-set-domains-for-the-ability-to-bind-sah/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<blockquote><p>This is an excerpt from my PhD thesis where I investigated 13 human PR/SET domains from the PRDM protein family and assessed their ability to bind to a fluorinated SAH analog. The research helps to identify which of the PRDMs could be methyltransferases. This work was carried out in the laboratory of Dr. Cheryl Arrowsmith at the University of Toronto and the University Health Network.</p></blockquote>
<h1><strong>Interrogating PR/SET domains for the ability to bind SAH</strong></h1>
<p><strong><br />
</strong><br />
<strong>Authorship attribution:<br />
</strong>This chapter contains unpublished work and I am responsible for the overall generation and interpretation of the data included herein with the following mentions: The F-SAH compound was synthesized and kindly provided by Carlos Zepeda, Ontario Institute for Cancer Research (<em>OICR</em>). Shili Duan provided guidance and technical assistance in protein production. Additional proteins were kindly provided by Taraneh Hajian, Elisa Gibson and Dr. Masoud Vedadi from the SGC Toronto. Dr. Scott Houliston advised and supported all NMR experiments, and Drs. Houliston and Cheryl Arrowsmith aided in data analysis and provided guidance throughout the study. This project was prematurely halted due to a catastrophic freezer malfunction during the COVID-19 shutdown.</p>
<p>&nbsp;</p>
<p><strong><u>Introduction </u></strong><strong><br />
</strong><br />
Lysine methyltransferases are a group epigenetic regulators currently being investigated in clinical and pharmaceutical studies for the treatment of certain cancers (Rugo, et al., 2020) (Ilango, et al., 2020) (Hoy, 2020) (Dilworth &amp; Barsyte-Lovejoy, 2019). The majority of all KMT proteins possess a domain with a SET-fold (Schapira, 2011). The PRDM protein family is characterized by the presence of a PRDI-BF1 and RIZ1 (PR/SET) domain, which is a subgroup of the SET domain (<strong>FIGURE A1</strong>). Currently KMT activity has been reported for some PRDMs (<strong>TABLE A1</strong>), while some PRDMs are thought to be inactive pseudoenzymes. Since PRDMs are less represented in the academic literature compared to canonical SET proteins (<strong>FIGURE A4</strong>), it is unclear if these PRDMs are truly pseudoenzyme or if their KMT activity is only yet to be characterized. Distinguishing which PRDMs possess KMT activity would further our understanding of lysine epigenetics and pseudoenzyme-driven biology (Arrowsmith, et al., 2012) (Ribeiro, et al., 2019).</p>
<p>All domains with a SET-fold interact with SAM through a conserved hydrogen bonding network split across two distinct domain regions (Campagna-Slater, et al., 2011). These SAM binding residues in the canonical SET-domains are well conserved, however the corresponding residues in PR/SET domains are less well conserved (<strong>FIGURE A2</strong>) (H-binding clusters at Regions 1 and 2)]. The only holo-enzyme structure of a PR/SET domains is of mouse PRDM9 bound to a histone H3 peptide and SAH (Wu, et al., 2013). The human and mouse PRDM9 PR/SET domains possess many of the highly conserved SAM-binding residues found in the canonical SET-domains. Lacking any structural evidence, it is unclear if and how many of the PR/SET domains may bind to SAM. Therefore, detecting cofactor binding in PR/SET domains would aid in distinguishing whether certain PRDMs are enzymes or pseudoenzymes.</p>
<p>Here I investigate cofactor binding to understand which PR/SET domains may have the ability to catalyze the KMT reaction. Using <sup>19</sup>F-nuclear magnetic resonance (NMR) spectroscopy, we measured the binding response for the majority of the human PR/SET domains with a fluorinated analog of the methyltransferase by-product S-adenosylhomocysteine (SAH), 2-fluoro-SAH (F-SAH). <sup>19</sup>F-NMR is a sensitive and versatile method to study protein-ligand interactions (Dalvit &amp; Vulpetti, 2019). Using the F-SAH binding response with the PRDM9 PR/SET domain as a positive control for the screening assays, we identified several PR/SET domains that bind F-SAH and therefore likely bind to SAM. Additionally, we detected several PR/SET domains that do not show any evidence of binding to F-SAH, which suggests a lesser affinity for SAM or potentially no affinity. This dataset can be used as a starting point for future discovery of KMT activity in PRDM proteins.</p>
<p><strong><u>RESULTS</u></strong></p>
<p><strong>Establishment of the F-SAH binding assay</strong></p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/FIGURE_K1-Copy-295x300.png" alt="" class="alignnone wp-image-5381" width="955" height="971" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/FIGURE_K1-Copy-295x300.png 295w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/FIGURE_K1-Copy-1008x1024.png 1008w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/FIGURE_K1-Copy-768x780.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/FIGURE_K1-Copy-1512x1536.png 1512w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/FIGURE_K1-Copy.png 1887w" sizes="(max-width: 955px) 100vw, 955px" /><br />
<strong>Figure K1. Characterization of F-SAH binding to PRDM9. </strong>(<strong>a</strong>) Chemical structure of 2-fluoro-SAH (F-SAH) and (<strong>b</strong>) model of F-SAH bound to PRDM9 based on a SAM-bound structure (PDBID: 6nm4). Colour coding corresponds to (<strong>a</strong>). The effects of F-SAH binding to PRDM9 were measured by <sup>19</sup>F-NMR. (<strong>c-d</strong>) Spectra of 16 µM F-SAH alone and with 1X and 5X molar equivalents of PRDM9 display a concentration dependent decrease in the unbound F-SAH peak and the emergence of a bound F-SAH peak.</p>
<p>&nbsp;</p>
<p>We utilized <sup>19</sup>F nuclear magnetic resonance spectroscopy (NMR) spectroscopy to assess whether PR/SET domains may have the ability to bind to the methyltransferase enzyme cofactor. Fluorinated SAH was chosen, rather than fluorinated SAM because SAH and its chemical precursors are more stable across a wide pH range and were therefore more suited to chemical synthesis. The chemical structure of 2-fluoro-SAH (F-SAH) shows that SAH is fluorinated at the 2-position of the adenine ring<strong> (FIGURE K1a)</strong>. Using our structure of the human PRDM9 PR/SET domain bound by SAM, we generated a theoretical model of F-SAH bound to PRDM9 <strong>(FIGURE K1b)</strong>. We observed that the fluorine atom projects toward the solvent and therefore is unlikely to alter the interaction with the protein. Nevertheless, because fluorine’s chemical shift is highly sensitive to any change in its microenvironment, we expected to see perturbation and/or broadening of the <sup>19</sup>F resonance when F-SAH binds to a PR/SET domain.</p>
<p>First, we investigated F-SAH alone by measuring a 1D <sup>19</sup>F spectrum of F-SAH, which gave rise to a single peak at ~-52.4 ppm. Next, we monitored for F-SAH binding to PRDM9, which is the most extensively characterized methyltransferase among all the PR/SET domain containing proteins (Eram, et al., 2014) (Blazer, et al., 2016) (Allali-Hassani, et al., 2019) (Powers, et al., 2016). We assessed the F-SAH NMR signal alone and in the presence of equal and 5-fold molar equivalents of PRDM9 PR/SET domain, we detected a concentration-dependent decrease in the intensity of the unbound F-SAH <sup>19</sup>F resonance <strong>(FIGURE K1c-d)</strong>.  Moreover, we observed the emergence of a second <sup>19</sup>F signal (~ 1045 Hz up-field at -54.3 ppm) that is from protein-bound F-SAH. As these results were reproducible using two separate PRDM9 protein preparations, we selected 5-fold molar equivalents of PR/SET domain to F-SAH for further experiments.</p>
<p><strong>Screening human PR/SET domains with the F-SAH binding assay</strong></p>
<p><img loading="lazy" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/FIGURE_K2-Copy-204x300.png" alt="" class="alignnone wp-image-5382" width="869" height="1278" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/FIGURE_K2-Copy-204x300.png 204w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/11/FIGURE_K2-Copy-1391x2048.png 1391w" sizes="(max-width: 869px) 100vw, 869px" /><br />
<strong>Figure K2. Screening for F-SAH binders among PR domains.</strong> (<strong>a</strong>) <sup>19</sup>F-NMR spectra showing 16 µM F-SAH in the presence of 80 µM of the indicated PR/SET domains. (<strong>b</strong>) Plots of <sup>19</sup>F peak integrals for F-SAH in the presence of individual PR/SET domains. All <sup>19</sup>F peaks were normalized to the peak integral of F-SAH alone in buffer and the normalized integrals of the Bound and 1-Unbound peaks relative to F-SAH alone were plotted (i.e. F-SAH alone would be located at position [0,0]).</p>
<p>&nbsp;</p>
<p>We performed F-SAH binding assays with PR/SET domains from 13 of the 19 human PRDM family members. The PR/SET domains of PRDM 1, 2, 4, 5, 6, 11, 12, 14 and 17 were provided by the SGC Toronto and I purified PRDM 3, 7, 9 and 16. I was able to purify PR/SET protein constructs for PRDM 8, 10, 13 and 15, however these constructs suffered stability issues in our <sup>19</sup>F-NMR buffer during dialysis (see Methods) and were therefore not suitable for further study. We collected <sup>19</sup>F-NMR spectra for F-SAH with 5-fold molar equivalents of each of the 13 human PR/SET domains (<strong>Figure K2a</strong>). The relative intensities of the unbound and bound F-SAH peaks varied considerably among the different PR/SET proteins indicating variable binding affinities across the panel of PRDM proteins. To quantify the binding for each PRDM PR/SET domain, we normalized the peak integrals of the unbound and bound peaks for each protein to the unbound peak of F-SAH alone. Specifically, we divided the integral of each peak by the integral of the peak for F-SAH alone. We plotted the normalized integrals of the 1 minus the unbound peak against the bound peak for each PR/SET domain to differentiate each protein’s affinity for F-SAH (<strong>Figure K2b</strong>). PR/SET datapoints located in the upper, right region of the plot displayed strong binding to F-SAH, while datapoints at the origin indicate no evidence of binding. We used the F-SAH signal in the presence of PRDM9 as an internal control to distinguish a strong binding response. PRDM 4 and 5 were plotted near to PRDM9, indicating a strong evidence response. Additionally, PRDM 14, 12, 7, 11 and 17 exhibited a very strong binding response. We were unable to detect any binding response for F-SAH in the presence of PRDM 3, 1 or 16, using our assay conditions, though we did detected a weak binding response for PRDM 2 and 6. Taken together, this data provides evidence to help prioritize further assays to identify methyltransferase activity among the PRDM family members.<em><br />
</em></p>
<p><strong><u>Discussion</u></strong><strong><br />
</strong><br />
<strong>Using F-SAH to probe KMT domains</strong></p>
<p>Here we used <sup>19</sup>F-NMR to investigate the potential for 13 of the 19 human PR/SET domains to bind a fluorinated analog of the methylation reaction by-product SAH. Using our F-SAH binding assays with PR/SET domains, we identified 7 domains (PRDM 4, 5, 7, 11, 12, 14 and 17) that showed a binding response similar to, or stronger than PRDM9. Of these ‘binding-competent’ domains only PRDM 9 and 7 have reported KMT activity, with accompanying enzyme kinetics data (Eram, et al., 2014) (Blazer, et al., 2016). Interestingly, PRDM7 showed stronger binding to F-SAH compared to PRDM9 (<strong>Figure K2</strong>). PRDM7 possesses much weaker KMT activity for H3K4me2 substrate peptide compared to PRDM9, with a 100-fold lower rate of catalysis (<em>k</em><sub>cat</sub>=1.9&#215;10<sup>2</sup>/<em>h </em>vs 1.9&#215;10<sup>4</sup>/h, for PRDM 7 and 9, respectively) (Blazer, et al., 2016) (Eram, et al., 2014). These catalytic rate differences are mostly attributed to the distinct capacities to facilitate the methyl cation transferase by the catalytic residues (Y357 and S357, in PRDM 9 and 7, respectively). Based on our data, another contributing factor could be that PRDM7 has a stronger affinity for the reaction by-product SAH, which inhibits the catalytic rate. In our F-SAH binding assays, PRDM 2 and 6 showed weak, but detectable, binding compared to PRDM9 (<strong>Figure K2</strong>). KMT activities towards H3K9 and H4K20 have been reported for PRDM 2 and 6, respectively (Kim, et al., 2003) (Congdon, et al., 2014) (Wu, et al., 2008). This indicates that a weak binding response for F-SAH is associated with known active KMTs. Finally, in all cases where we observed F-SAH binding, we measured a consistent up-field shift (~1040 Hz) in the F-SAH <sup>19</sup>F resonance (<strong>Figure K2a</strong>). This indicates there is a common binding modality with a highly similar microenvironment formed in the bound state across the family.</p>
<p>We also demonstrated the utility of F-SAH as a synthetic chemical probe for methyltransferase proteins. Our approach is comparable to a previous study that reported the binding of the MLL1 SET-domain to a SAM-analog conjugated with a fluorescent tag (Luan, et al., 2016). Comparatively, F-SAH benefits from the smaller size of fluorine, which is positioned on the adenine ring in an orientation that is unlikely to interfere with binding (<strong>Figure K2a-b</strong>). Additionally, <sup>19</sup>F-NMR is capable of detecting low affinity protein-ligand interactions, such as is required for fragment-based screening (Dalvit &amp; Vulpetti, 2019). We surveyed two publicly available databases that quantify protein-ligand interaction data (<strong>TABLE K1</strong>). We found that SET-fold containing proteins have a range of affinities for SAH with dissociation constants (K<sub>D</sub>) from low µM to high nM, as well as a similar range for reported half-maximal inhibition (IC<sub>50</sub>) by SAH. Only RBCMT had affinity data for both SAH and SAM, showing that SAM binds RBCMT with an ~70-fold higher affinity when compared with SAH  (Couture, et al., 2006). The authors reported conserved binding modes in crystal structures of RBCMT-SAM and RBCMT-SAH implying that the variations in affinity could not be attributed to the binding conformations, but may rather be attributed to a higher entropic penalty for bound SAH, which possesses additional bond rotational freedom compared to SAM (Couture, et al., 2006). Taken together, we can predict that the upper-limit for the affinity of PRDM9 for F-SAH is less than the known affinity of SAM, which is 18.6 µM (Eram, et al., 2014).</p>
<p><strong>Table K1. Known dissociation constant (K<sub>D</sub>) and half maximal inhibition values (IC<sub>50</sub>) of SAM and SAH for SET domain proteins. </strong>Binding affinity data available from the BindingDB (www.bindingdb.org) and BindingMOAD (www.bindingmoad.org/) was accessed from the RCSB PDB (www.rcsb.org). Note that BindingDB reports a large range of IC<sub>50</sub> values for EZH2.</p>
<table width="95%">
<tbody>
<tr>
<td style="width: 95.5333px;"><strong>Protein</strong></td>
<td style="width: 207.883px;"><strong>Source</strong></td>
<td style="width: 115.967px;"><strong>Ligand</strong></td>
<td style="width: 115.967px;"><strong>K<sub>D</sub> (µM)</strong></td>
<td style="width: 115.967px;"><strong>IC<sub>50</sub> (µM)</strong></td>
<td style="width: 228.283px;"><strong>Source</strong></td>
</tr>
<tr>
<td style="width: 95.5333px;">EHMT1</td>
<td style="width: 207.883px;"><em>Homo sapiens</em></td>
<td style="width: 115.967px;">SAH</td>
<td style="width: 115.967px;"></td>
<td style="width: 115.967px;">2.3</td>
<td style="width: 228.283px;">BindingDB</td>
</tr>
<tr>
<td style="width: 95.5333px;">EHMT2</td>
<td style="width: 207.883px;"><em>Homo sapiens</em></td>
<td style="width: 115.967px;">SAH</td>
<td style="width: 115.967px;">0.57</td>
<td style="width: 115.967px;">2.0</td>
<td style="width: 228.283px;">BindingDB</td>
</tr>
<tr>
<td style="width: 95.5333px;">EZH2</td>
<td style="width: 207.883px;"><em>Homo sapiens</em></td>
<td style="width: 115.967px;">SAH</td>
<td style="width: 115.967px;"></td>
<td style="width: 115.967px;">0.263-16.6</td>
<td style="width: 228.283px;">BindingDB</td>
</tr>
<tr>
<td style="width: 95.5333px;">KMT2A</td>
<td style="width: 207.883px;"><em>Homo sapiens</em></td>
<td style="width: 115.967px;">SAH</td>
<td style="width: 115.967px;"></td>
<td style="width: 115.967px;">2.3</td>
<td style="width: 228.283px;">BindingDB</td>
</tr>
<tr>
<td style="width: 95.5333px;">Kmt5b</td>
<td style="width: 207.883px;"><em>Mus musculus</em></td>
<td style="width: 115.967px;">SAM</td>
<td style="width: 115.967px;">11.2</td>
<td style="width: 115.967px;"></td>
<td style="width: 228.283px;">BindingMOAD</td>
</tr>
<tr>
<td style="width: 95.5333px;">Kmt5c</td>
<td style="width: 207.883px;"><em>Mus musculus</em></td>
<td style="width: 115.967px;">SAH</td>
<td style="width: 115.967px;">17.2</td>
<td style="width: 115.967px;">10</td>
<td style="width: 228.283px;">BindingDB</td>
</tr>
<tr>
<td style="width: 95.5333px;">PRDM9</td>
<td style="width: 207.883px;"><em>Homo sapiens</em></td>
<td style="width: 115.967px;">SAM</td>
<td style="width: 115.967px;">18.6</td>
<td style="width: 115.967px;"></td>
<td style="width: 228.283px;">(Eram, et al., 2014)</td>
</tr>
<tr>
<td style="width: 95.5333px;">RBCMT</td>
<td style="width: 207.883px;"><em>Pisum sativum</em></td>
<td style="width: 115.967px;">SAM</td>
<td style="width: 115.967px;">0.29</td>
<td style="width: 115.967px;"></td>
<td style="width: 228.283px;">BindingMOAD</td>
</tr>
<tr>
<td style="width: 95.5333px;">RBCMT</td>
<td style="width: 207.883px;"><em>Pisum sativum</em></td>
<td style="width: 115.967px;">SAH</td>
<td style="width: 115.967px;">21.2</td>
<td style="width: 115.967px;"></td>
<td style="width: 228.283px;">BindingMOAD</td>
</tr>
<tr>
<td style="width: 95.5333px;">SETD7</td>
<td style="width: 207.883px;"><em>Homo sapiens</em></td>
<td style="width: 115.967px;">SAH</td>
<td style="width: 115.967px;"></td>
<td style="width: 115.967px;">30</td>
<td style="width: 228.283px;">BindingDB</td>
</tr>
<tr>
<td style="width: 95.5333px;">mSMYD2</td>
<td style="width: 207.883px;"><em>Mus musculus</em></td>
<td style="width: 115.967px;">SAH</td>
<td style="width: 115.967px;"></td>
<td style="width: 115.967px;">0.18</td>
<td style="width: 228.283px;">BindingDB</td>
</tr>
<tr>
<td style="width: 95.5333px;">SMYD2</td>
<td style="width: 207.883px;"><em>Homo sapiens</em></td>
<td style="width: 115.967px;">SAH</td>
<td style="width: 115.967px;"></td>
<td style="width: 115.967px;">0.18</td>
<td style="width: 228.283px;">BindingDB</td>
</tr>
</tbody>
</table>
<p><strong>Interpreting the absence of binding evidence</strong></p>
<p>We identified that PRDM 1, 3, and 16 do not bind to F-SAH in our assay conditions, which is suggestive of the inability to bind to SAM. PRDM1 (initially identified as Blimp-1) is widely understood to lack KMT activity and instead functions as a gene repressor by recruiting specific co-repressors proteins to PRDM1-binding sites throughout the genome (Minnich, et al., 2016). PRDM1 does affect histone methylation, but it does so through interactions with the KMT protein G9a/EHMT2 (Gyory, et al., 2004), the arginine methyltransferase PRMT5 (Ancelin, et al., 2006) and the lysine demethylase LSD1 (Su, et al., 2009). Whether the PR/SET domain of PRDM1 functions as a “reader domain” for specific histone tail modifications is undetermined, however our data suggests that it is unlikely that SAH or SAM would play a role in this ability.</p>
<p>We found that the PR/SET domains of PRDM 3 and 16 do not bind to F-SAH under our assay conditions. However, this does not exclude the possibility that larger or differently generated constructs may possess the ability to bind SAM or SAH. For instance, longer or full-length PRDM 3 and 16 protein constructs were reported to possess KMT activity when purified from either HeLa cells and mouse fibroblasts or SF9 insect cells (Pinheiro, et al., 2012) (Zhou, et al., 2016). Our PRDM 3 and 16 constructs were purified from E. coli and each contained the PR/SET domain along with the first proximal C-terminal zinc-finger motif (ZnF1), which is similar to previous structural and enzyme characterization studies of the PR/SET and ZnF1 of PRDM9 (Eram, et al., 2014) (Wu, et al., 2013). It is unclear if the additional regions outside the PRDM 3 and 16 PR/SET domains and ZnF1 could function to stabilize the cofactor binding site in the PR/SET domain. Interestingly, a previously solved NMR solution structure of PRDM16 demonstrated the absence of internal β-strands that are present in crystal and solution structures of all other PR/SET domains, which could possibly destabilize the cofactor and substrate binding sites (<strong>FIGURE A3d</strong>). Furthermore, it is unclear if additional factors present in eukaryotic expression systems enable cofactor binding or enzymatic activity.</p>
<p><strong>Future analysis of PRDMs<br />
</strong><br />
Future investigation to complete the F-SAH binding screen with the untested members of the PRDM family (PRDM 8, 10, 13, 15, FOG1, and FOG2) would provide valuable data.  Additionally, assaying F-SAH binding in competition with (unlabeled) SAM could qualitatively delineate the affinities of the methyltransferase reaction cofactor and by-product, providing further evidence of potential enzymatic function. We detected F-SAH binding in PRDM 4, 5, 11, 12, 14, and 17, and we could not find any studies reporting KMT activity with these proteins. Intriguingly, PRDM 4, 5, 11, 12, and 14 all possess the highly conserved “catalytic tyrosine” (<strong>FIGURE A2</strong>), which is further evidence supporting KMT activity for these proteins. Importantly, several studies have attributed oncogenic roles for PRDM 4, 12 and 14 in specific cancers (Casamassimi, et al., 2020). Future discovery of enzymatic function in any of these PR/SET domains could provide a novel, actionable target for oncological drug discovery.</p>
<p>&nbsp;</p>
<p><strong><u>Methods</u></strong><strong><br />
</strong><strong>Protein production </strong><br />
PRDM9 (195-385) and PRDM7 (195-392) with an N-terminal 6xHis-tag and TEV protease site were expressed in <em>Escherichia coli</em> BL21 (DE3) codon plus cells from a pET28-MHL vector. PRDM3 (69-235) and PRDM16 (73-256) with an N-terminal 6xHis-tag and TEV protease site were expressed in <em>Escherichia coli</em> BL21 (DE3) codon plus cells from a pET15b/MHL vector. Cell were grown at 37°C in M9 minimal medium in the presence of 50 µg/ml of kanamycin (PRDM 7 and 9) or 100 µg/ml of ampicillin (PRDM 3 and 16) to an OD600 of 0.8 and induced by isopropyl-1-thio-D-galactopyranoside (IPTG), final concentration 0.5 mM and incubated overnight at 15°C. Cells were harvested by centrifugation at 7,000 rpm and cell pellets were stored at -80°C. Cell were lyzed on ice in lysis buffer (20 mM Tris pH 7.5, 300 mM NaCl, 10 mM imidazole, 50 µM ZnSO<sub>4</sub>, 5 mM β-mercaptoethanol, 0.5 mM TCEP, 0.1% Triton-X 100, 2.5% glycerol, 1mM PMSF, 2 mM benzamidine and Roche complete EDTA-free protease inhibitor cocktail tablet) using a probe sonicator. The crude extract was cleared by ultracentrifugation for 50 min at 50000xg and the supernatant was incubated with TALON Metal Affinity Resin (Takara) at 4<sup>o</sup>C for 120 minutes with agitation. Resin was washed in wash buffer (lysis buffer with 0.01% Triton-X 100 and lacking protease inhibitors) and bound proteins were eluted using elution buffer (wash buffer with 400 mM imidazole), monitored by Bradford analysis. Protein was dialyzed overnight at 4°C in TEV dialysis buffer (20 mM Tris pH 7.5, 300 mM NaCl, 20µM ZnCL<sub>2</sub>, 2.5% glycerol, 5 mM β-mercaptoethanol and 0.5 mM TCEP) and incubated with His-tagged TEV protease produced in house at a 1/20 dilution by mass. 0.5 mM CHAPS was added to the TEV dialysis buffer for PRDM 9 and 7, but not PRDM 3 and 16. TEV and uncleaved proteins were removed using TALON resin and soluble  protein was loaded onto a Superdex 75 Increase 10/300 GL colum (GE Healthcare), equilibrated with 20 mM Tris-HCl buffer, pH 7.5, and 150 mM NaCl, 2.5% glycerol, 5 mM β-mercaptoethanol and 0.5 mM TCEP at flow rate 0.7 ml/min. Fractions containing PRDM9 protein were pooled and dialyzed into low salt buffer (20 mM Tris-HCl pH 8.5, 100 mM NaCl, 5% glycerol, 5 mM β-mercaptoethanol and 0.5 mM TCEP). PRDM9 protein was further purified by anion-exchange chromatography on 2 tandemly joined 5 ml HiTrap<img src="https://s.w.org/images/core/emoji/13.0.0/72x72/2122.png" alt="™" class="wp-smiley" style="height: 1em; max-height: 1em;" /> DEAE FF (GE Healthcare) columns along a linear gradient up to 1.0 M NaCl. Purified PRDM9 protein was dialyzed into a final buffer containing 20 mM Tris pH 7.5, 150 mM NaCl and 2 mM TCEP. The proteins for PRDM1 (38-223), PRDM2 (2-160), PRDM4 (390-540), PRDM5 (2-226), PRDM6 (194-405), PRDM11 (79-314), PRDM12 (60-229), PRDM14 (212-422), and PRDM17 (1-205) were purified from E. coli expression and kindly provided by Taraneh Hajian, Elisa Gibson and Dr. Masoud Vedadi from the SGC Toronto.</p>
<p><strong>F-SAH binding by <sup>19</sup>F NMR<br />
</strong>F-SAH was synthesized and kindly provided by Carlos Zepeda, Ontario Institute for Cancer Research (<em>OICR</em>). All PR/SET domain containing proteins were dialyzed overnight at 4<sup>o</sup>C into <sup>19</sup>F-NMR buffer (20 mM Tris pH 7.3, 150 mM NaCl, 2 mM TCEP and 20 µM ZnCl<sub>2</sub>) also containing 16 µM sodium trifluoroacetate. The concentration of each protein was measured using an absorbance reading at 280 nm and evaluated using their respective theoretical extinction coefficients calculated using ProtParam (Gasteiger, et al., 2005). 100 mM F-SAH in d6-DMSO was diluted to 16 µM F-SAH by adding the dialyzed protein solution to a final protein concentration of 80 µM and then topping off with <sup>19</sup>F-NMR buffer to a final volume of 500 µl. For the F-SAH alone sample, only <sup>19</sup>F-NMR buffer was added. The solutions were mixed in 1.5 ml polypropylene tubes and then transferred to 5 mm glass NMR tubes.</p>
<p>1D <sup>19</sup>F spectra were acquired at 298K on a Bruker Avance III spectrometer operating at 600 MHz and equipped with a QCI cryoprobe with an independent <sup>19</sup>F detection coil. Each spectrum was acquired with 3072 scans, an acquisition time of 577 ms, and processed by applying an exponential window function (LB = 20). There was no difference observed in the F-SAH <sup>19</sup>F line-shape when signal acquisition was acquired with or without proton decoupling; therefore, all spectra were acquired without proton decoupling.  The spectral window was measured from -37.89 ppm to -88.09 ppm and <sup>19</sup>F peaks were identified at -75.53 ppm, -52.40 ppm, and -54.21 ppm, for trifluoroacetate, F-SAH unbound, and F-SAH bound, respectively. Processing and analysis were carried out with Topspin 3.5 (Bruker BioSpin). To control for pipetting errors that could result in differences in F-SAH signal intensities, the integral of the trifluoroacetate <sup>19</sup>F peak of each PR/SET containing sample was set to equal the integral of the trifluoroacetate <sup>19</sup>F peak of the sample with F-SAH alone. To normalize F-SAH peaks, the integral for each F-SAH peak was divided by the integral of the F-SAH peak from the sample with F-SAH alone (i.e. integral of F-SAH alone = 1).</p>
<h1>References</h1>
<p>Allali-Hassani, A. et al., 2019. Discovery of a chemical probe for PRDM9.. <em>Nat Commun., </em>10(1), p. 5759.</p>
<p>Ancelin, K. et al., 2006. Blimp1 associates with Prmt5 and directs histone arginine methylation in mouse germ cells.. <em>Nat Cell Biol, </em>8(6), pp. 623-30.</p>
<p>Arrowsmith, C. et al., 2012. Epigenetic protein families: a new frontier for drug discovery.. <em>Nat Rev Drug Discov, </em>11(5), pp. 384-400.</p>
<p>Blazer, L. et al., 2016. PR Domain-containing Protein 7 (PRDM7) Is a Histone 3 Lysine 4 Trimethyltransferase.. <em>J Biol Chem, </em>291(26), pp. 13509-19.</p>
<p>Campagna-Slater, V. et al., 2011. Structural chemistry of the histone methyltransferases cofactor binding site.. <em>J Chem Inf Model, </em>51(3), pp. 612-23.</p>
<p>Casamassimi, A. et al., 2020. Multifaceted Role of PRDM Proteins in Human Cancer.. <em>Int J Mol Sci., </em>21(7), p. E2648.</p>
<p>Congdon, L., Sims, J., Tuzon, C. &amp; Rice, J., 2014. The PR-Set7 binding domain of Riz1 is required for the H4K20me1-H3K9me1 trans-tail &#8216;histone code&#8217; and Riz1 tumor suppressor function.. <em>Nucleic Acids Res, </em>42(6), pp. 3580-9.</p>
<p>Couture, J. et al., 2006. Catalytic roles for carbon-oxygen hydrogen bonding in SET domain lysine methyltransferases.. <em>J Biol Chem, </em>281(28), pp. 19280-7.</p>
<p>Dalvit, C. &amp; Vulpetti, A., 2019. Ligand-Based Fluorine NMR Screening: Principles and Applications in Drug Discovery Projects.. <em>J Med Chem, </em>62(5), pp. 2218-44.</p>
<p>Dilworth, D. &amp; Barsyte-Lovejoy, D., 2019. Targeting protein methylation: from chemical tools to precision medicines.. <em>Cell Mol Life Sci, </em>76(15), pp. 2967-85.</p>
<p>Eram, M. et al., 2014. Trimethylation of histone H3 lysine 36 by human methyltransferase PRDM9 protein.. <em>J Biol Chem, </em>289(17), pp. 12177-88.</p>
<p>Gasteiger, E. et al., 2005. Protein Identification and Analysis Tools on the ExPASy Server. In: J. M. Walker, ed. <em>The Proteomics Protocols Handbook. </em>s.l.:Humana Press, pp. 571-607.</p>
<p>Gyory, I. et al., 2004. PRDI-BF1 recruits the histone H3 methyltransferase G9a in transcriptional silencing.. <em>Nat Immunol, </em>5(3), pp. 299-08.</p>
<p>Hoy, S., 2020. Tazemetostat: First Approval.. <em>Drugs, </em>80(5), pp. 513-21.</p>
<p>Ilango, S. et al., 2020. Epigenetic alterations in cancer.. <em>Front Biosci (Landmark Ed), </em>25(1), pp. 1058-109.</p>
<p>Kim, K., Geng, L. &amp; Huang, S., 2003. Inactivation of a histone methyltransferase by mutations in human cancers.. <em>Cancer Res, </em>63(22), pp. 7619-23.</p>
<p>Luan, Y. et al., 2016. Design of a fluorescent ligand targeting the S-adenosylmethionine binding site of the histone methyltransferase MLL1. <em>Org Biomol Chem, </em>14(2), pp. 631-8.</p>
<p>Minnich, M. et al., 2016. Multifunctional role of the transcription factor Blimp-1 in coordinating plasma cell differentiation.. <em>Nat Immunol, </em>17(3), pp. 331-43.</p>
<p>Pinheiro, I. et al., 2012. Prdm3 and Prdm16 are H3K9me1 methyltransferases required for mammalian heterochromatin integrity.. <em>Cell, </em>150(5), pp. 948-60.</p>
<p>Powers, N. et al., 2016. The Meiotic Recombination Activator PRDM9 Trimethylates Both H3K36 and H3K4 at Recombination Hotspots In Vivo.. <em>PLoS Genet, </em>12(6), p. e1006146.</p>
<p>Ribeiro, A. et al., 2019. Emerging concepts in pseudoenzyme classification, evolution, and signaling.. <em>Sci Signal, </em>12(594), p. eaat9797.</p>
<p>Rugo, H. et al., 2020. The Promise for Histone Methyltransferase Inhibitors for Epigenetic Therapy in Clinical Oncology: A Narrative Review.. <em>Adv Ther. , </em>37(7), pp. 3059-82.</p>
<p>Schapira, M., 2011. Structural Chemistry of Human SET Domain Protein Methyltransferases.. <em>Curr Chem Genomics, </em>5(S1), pp. 85-94.</p>
<p>Su, S. et al., 2009. Involvement of histone demethylase LSD1 in Blimp-1-mediated gene repression during plasma cell differentiation.. <em>Mol Cell Biol, </em>29(6), pp. 1421-31.</p>
<p>Wu, H. et al., 2013. Molecular basis for the regulation of the H3K4 methyltransferase activity of PRDM9.. <em>Cell Rep, </em>5(1), pp. 13-20.</p>
<p>Wu, Y. et al., 2008. PRDM6 is enriched in vascular precursors during development and inhibits endothelial cell proliferation, survival, and differentiation.. <em>J Mol Cell Cardiol, </em>44(1), pp. 47-58.</p>
<p>Zhou, B. et al., 2016. PRDM16 Suppresses MLL1r Leukemia via Intrinsic Histone Methyltransferase Activity.. <em>Mol Cell, </em>62(2), pp. 222-36.</p>
<p>&nbsp;</p>
<p><a href="https://zenodo.org/record/4253145#.X6lMCVB7lEY">Download this as a PDF from Zenodo.</a></p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/interrogating-pr-set-domains-for-the-ability-to-bind-sah/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
	</channel>
</rss>
