<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>Chordoma &#8211; openlabnotebooks.org</title>
	<atom:link href="https://thesgc.github.io/static-openlabnotebooks/category/disease-foundation/chordoma/feed/" rel="self" type="application/rss+xml" />
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<description>A growing team of groundbreaking scientists around the world are now sharing their lab notebooks online</description>
	<lastBuildDate>Tue, 12 Mar 2019 11:57:59 +0000</lastBuildDate>
	<language>en-GB</language>
	<sy:updatePeriod>hourly</sy:updatePeriod>
	<sy:updateFrequency>1</sy:updateFrequency>
	<generator>https://wordpress.org/?v=4.9.10</generator>

<image>
	<url>https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/cropped-lab-jar02-32x32.png</url>
	<title>Chordoma &#8211; openlabnotebooks.org</title>
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<width>32</width>
	<height>32</height>
</image> 
	<item>
		<title>Introducing the team: The Gileadi Group in Oxford.</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/introducing-the-team-the-gileadi-group-in-oxford/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/introducing-the-team-the-gileadi-group-in-oxford/#respond</comments>
		<pubDate>Fri, 30 Nov 2018 16:31:10 +0000</pubDate>
		<dc:creator><![CDATA[Opher Gileadi]]></dc:creator>
				<category><![CDATA[Alzheimer’s disease]]></category>
		<category><![CDATA[Amyotrophic Lateral Sclerosis (ALS)]]></category>
		<category><![CDATA[Chordoma]]></category>
		<category><![CDATA[Opher Gileadi]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2198</guid>
		<description><![CDATA[Hi! I’m Opher Gileadi, I head a group at the Structural Genomics Consortium (SGC) in Oxford. For many years, we’ve been looking at proteins involved in human. We’ve looked at a wide variety of proteins: from DNA repair enzymes and transcription factors to enzymes involved in signalling. In most cases, we focus on early work: <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/introducing-the-team-the-gileadi-group-in-oxford/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Hi! I’m Opher Gileadi, I head a group at the Structural Genomics Consortium (SGC) in Oxford. For many years, we’ve been looking at proteins involved in human. We’ve looked at a wide variety of proteins: from DNA repair enzymes and transcription factors to enzymes involved in signalling. In most cases, we focus on early work: structure determination, activity assays, antibody generation, and finding initial small molecule “hits”. Having the right reagents are the key to enabling new drug targets; we expect that our data will allow – or encourage – further work by medicinal chemists and biologists to seriously test therapeutic hypotheses (we call our data and reagents sets <a href="https://www.thesgc.org/tep">TEPs</a> – Target Enabling Packages).</p>
<p>My team and I will be blogging and sharing our experiments on three areas of interest:</p>
<p><strong>Alzheimer’s disease</strong>: We are funded by the NIH through the <a href="https://www.nia.nih.gov/research/amp-ad">AMP-AD</a> project (Accelerating Medicines Partnership &#8211; Alzheimer&#8217;s Disease). The project has integrated data from brain autopsies and model systems to identify new possible protein targets for treatment or prevention of Alzheimer’s (annotated in the <a href="https://agora.ampadportal.org/genes/(genes-router:genes-list)">Agora</a> web site, run by <a href="http://sagebionetworks.org/research-projects/agora/">SAGE Bionetworks</a>. Our group is generating TEPs for some of these targets – we’ll share our progress in future posts.</p>
<p><strong>ALS (Amyotrophic Lateral Sclerosis)</strong>: We are part of <a href="https://www.als-rap.org/">ALS-RAP</a>, the ALS reproducible antibody platform. We’re addressing one of the big conundrums to biomedical research: the lack, or the poor quality, of many antibodies used for research ranging from cell biology to histopathology.</p>
<p>Together with colleagues in Stockholm and Montreal we aim to validate a set of antibodies to 30 proteins involved in the disease. We will test commercial antibodies as well as generate new ones through phage display technologies.</p>
<p><strong>Chordoma</strong>: <a href="https://www.chordomafoundation.org/understanding-chordoma/">Chordoma</a> is a rare type of cancer that occurs in the bones of the skull base and spine. It is resistant to all forms of chemotherapy; patients are treated by surgery and radiation, but recurrence is common and, eventually, untreatable. A unifying molecular characteristic of chordoma is overexpression of a variant of the transcription factor T-Brachyury (TBXT), which is essential for survival of the cancer cells. Joseph Newman in my group has solved structures of the human Brachyury protein (PDB codes <a href="http://www.rcsb.org/structure/6F58">6F58</a>, <a href="http://www.rcsb.org/structure/6F59">6F59</a>); we are working together with groups in the University of North Carolina and in Baylor College (Texas) to explore TBXT as a drug target.</p>
<p>That’s all for today – we’ll follow-up with stories in these three areas.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/introducing-the-team-the-gileadi-group-in-oxford/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Screening EGFR and GAK Inhibitors in Chordoma</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/screening-egfr-and-gak-inhibitors-in-chordoma/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/screening-egfr-and-gak-inhibitors-in-chordoma/#respond</comments>
		<pubDate>Thu, 24 May 2018 16:40:59 +0000</pubDate>
		<dc:creator><![CDATA[Kaleb Naegeli]]></dc:creator>
				<category><![CDATA[Chordoma]]></category>
		<category><![CDATA[EGFR and GAK]]></category>
		<category><![CDATA[Kaleb Naegeli]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=1259</guid>
		<description><![CDATA[A major therapeutic avenue for chordoma is targeting the receptor tyrosine kinase, EGFR. Its ligand, EGF, is overexpressed as a transcriptional target of Brachyury and polysomy of EGFR’s region of chromosome 7 is frequent in chordoma. To this extent I, in collaboration with SGC-UNC’s post-doc Chris Asquith, am working on the profiling of a number <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/screening-egfr-and-gak-inhibitors-in-chordoma/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>A major therapeutic avenue for chordoma is targeting the receptor tyrosine kinase, EGFR. Its ligand, EGF, is overexpressed as a transcriptional target of Brachyury and polysomy of EGFR’s region of chromosome 7 is frequent in chordoma. To this extent I, in collaboration with SGC-UNC’s post-doc Chris Asquith, am working on the profiling of a number of EGFR inhibitors. Of particular interest, though, is this: many commercial EGFR inhibitors also hit another kinase – GAK (cyclin-G associated kinase). GAK plays a major role in prostate cancer, but its role in chordoma is unknown. Based on gene expression data from the Chordoma Foundation, we do know GAK is expressed in most chordoma cell lines.</p>
<p>First, we screened a set of published EGFR inhibitors, some of which also have GAK affinity, on the chordoma cell lines U-CH1 and U-CH2 (Table 1). Most of these compounds have strong potency (&lt;1 μM) on U-CH1 but not as strong of potency on U-CH2; this is expected, as previous work indicates U-CH2 is more resistant to EGFR inhibitors<sup>1</sup>. Following this initial screen, Chris has developed and I am characterizing a new set of compounds. This is a useful set because some of them have high affinity for EGFR, some have high affinity for GAK, and others show relatively equal affinity for both kinases. Using these compounds will allow us to interrogate if GAK plays a role in chordoma.</p>
<p>I performed a series of experiments to assess how much of each compound was required to kill (or at least stop the growth of) chordoma cells (U-CH1 and U-CH2) as well as normal skin fibroblasts (WS1). An ideal therapeutic would have a high potency for U-CH1 and U-CH2 without being toxic for WS1. A number of compounds have low values in U-CH1 but, like the published compounds in referenced above, do not show activity in U-CH2. A few compounds show activity in both U-CH1 and U-CH2, but they tend to also show similar potency in WS1. The mechanisms of these compounds warrant further investigation, as they will continue to help us understand potential interplay between EGFR and GAK, but their toxicity in normal skin cells may indicate they aren’t great therapeutic candidates.</p>
<p><img class="wp-image-1261 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/05/20180522-Table2-300x241.png" alt="" width="348" height="280" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/05/20180522-Table2-300x241.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/05/20180522-Table2-768x617.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/05/20180522-Table2-1024x823.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/05/20180522-Table2.png 1501w" sizes="(max-width: 348px) 100vw, 348px" /></p>
<p style="text-align: center;"><strong>Table 1</strong>: Compound potency in chordoma cells and normal fibroblasts in vitro. Values shown are EC<sub>50</sub> in μM.</p>
<p>Going forward, I am now trying to work out the mechanism of action of a subset of these compounds: do they kill the cells or just stop their growth? Does specific genetic knockdown of GAK produce similar phenotypes? These data will help us understand if GAK has a role in chordoma and how we can therapeutically target this kinase.</p>
<p>You can read greater detail about these experiments, their methodology, and see more data on <a href="https://zenodo.org/record/1251296#.Wwbq-CAh1EY">Zenodo</a>.</p>
<p><sup>1</sup>Scheipl S, Barnard M, Cottone L, <em>et al.</em> EGFR inhibitors identified as a potential treatment for chordoma in a focused compound screen. <em>J Pathol</em> 2016; <strong>239</strong>: 320–334.</p>
<p>&nbsp;</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/screening-egfr-and-gak-inhibitors-in-chordoma/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Chordoma: A Brief Background</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/choromda-a-brief-background/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/choromda-a-brief-background/#comments</comments>
		<pubDate>Tue, 15 May 2018 17:48:27 +0000</pubDate>
		<dc:creator><![CDATA[Kaleb Naegeli]]></dc:creator>
				<category><![CDATA[Chordoma]]></category>
		<category><![CDATA[Kaleb Naegeli]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=1237</guid>
		<description><![CDATA[Greetings! My first post here will be a brief introduction. More in-depth experiment and data posts are coming soon! I am a post-doctoral research associate in the UNC Catalyst for Rare Diseases1. Throughout my education and training, I have been interested in cancer – I worked on epigenetics of ovarian cancer as an undergraduate and <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/choromda-a-brief-background/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Greetings! My first post here will be a brief introduction. More in-depth experiment and data posts are coming soon!</p>
<p>I am a post-doctoral research associate in the <a href="https://pharmacy.unc.edu/research/unc-catalyst-rare-diseases/" target="_blank" rel="noopener">UNC Catalyst for Rare Diseases</a><sup>1</sup>. Throughout my education and training, I have been interested in cancer – I worked on epigenetics of ovarian cancer as an undergraduate and then mechanisms of cell invasion (in a developmental biology system) as a Ph.D. student. Now as a post-doc with the UNC Catalyst for Rare Diseases, my interest in cancer continues in the form of a project on chordoma, a rare bone cancer.</p>
<p>At the recent International Chordoma Research Workshop, the idea of sharing data, reagents, and research progress was raised. Given the affiliation of the UNC Catalyst for Rare Diseases with the open-science SGC, our group seemed like optimal candidates to initiate this undertaking. <strong>Other chordoma researchers: I welcome you to join us! Open science will accelerate the pace of research!<br />
</strong></p>
<p>Chordoma is a rare cancer that occurs along the bones of the central nervous system – from the base of the skull to the tip of the spine. Chordomas arise from remnants of the notochord – a developmental precursor to the vertebral column – and thus are almost always driven by expression of the notochord transcription factor Brachyury (encoded by the gene <em>T</em>). Therapeutic avenues for chordoma are tricky because they are rare (roughly 1 in 10<sup>6</sup> people), slow-growing, and often difficult to resect because of proximity to vital and sensitive structures. Current standard of care is surgical resection followed by radiation, but recurrence, metastasis, and mortality still happens in many cases despite these best practices.</p>
<p>Thus, improved knowledge of the biology of chordoma and the development of new therapeutics are vital for bettering the lives of patients with this rare bone cancer. I have two primary interests in Chordoma which will appear frequently in this blog:</p>
<ul>
<li><strong>Brachyury</strong> – this embryonic transcription factor of the notochord is encoded by the gene <em>T</em>. But T is going to give you way too many Google search results, so most always we’ll stick with Brachyury. This transcription factor is also overexpressed in some other cancers, though its role as a driver in these instances is unclear. While some of the biology of this transcription factor is known – including some of the genes it may regulate – much remains to be learned, including potential chaperones and binding partners. Brachyury presents an intriguing therapeutic avenue for this cancer because of its ubiquitous expression.</li>
<li><strong>EGFR</strong> – this receptor tyrosine kinase is overexpressed (but not mutated) in many chordomas and its ligand, EGF, is a direct transcriptional target of Brachyury. EGFR inhibitors are a potential therapeutic for chordoma, and in collaboration with SGC-UNC, we are working on new inhibitors of EGFR and other kinases as potential chemotherapeutics for chordoma (more information: https://www.chordomafoundation.org/targets/egfr/)</li>
</ul>
<p>With this background out of the way, a backlog of data will be forthcoming.</p>
<p>My work owes many thanks to the Chordoma Foundation for their inspiring array of reagents and data. They are a superb example of how a patient advocacy/disease awareness foundation should be run and how the sharing of reagents and insights can accelerate the progress of research and therapeutic development.</p>
<p>If you are unfamiliar with chordoma, suggested references include the following:</p>
<p><a href="https://www.chordomafoundation.org/understanding-chordoma/" target="_blank" rel="noopener">Chordoma Foundation &#8211; Understanding Chordoma</a><br />
<a href="http://www.thelancet.com/journals/lanonc/article/PIIS1470-2045%2814%2971190-8/abstract">Chordoma Overview (Lancet, 2015)</a><br />
<a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/path.2816" target="_blank" rel="noopener">Role of Brachyury in Chordoma; Presnet et al (2010) J Pathology</a><br />
<a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/path.4082" target="_blank" rel="noopener">The Brachyury Functional Network; Nelson et al (2012) J Pathology</a><br />
<a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/path.4729" target="_blank" rel="noopener">EGFR and EGFR inhibitors in chordoma; Scheipl et al (2016) J Pathology</a></p>
<p><sup>1</sup><strong><a href="https://pharmacy.unc.edu/research/unc-catalyst-rare-diseases/" target="_blank" rel="noopener">The UNC Catalyst for Rare Diseases</a>:</strong> We are a biology group started by the SGC (Structural Genomics Consortium) at the University of North Carolina through funding from the Eshelman Institute for Innovation (in the UNC Eshelman School of Pharmacy). Our commitment and objective is to be a research group that can be a resource for patients and other researchers, a hub with wet lab expertise to do be a source of knowledge in rare diseases. As an “SGC spin-off”, as I like to call our lab, we are committed to a practice of open science. <a href="https://twitter.com/UNC_Catalyst" target="_blank" rel="noopener">You can follow us on Twitter here!</a></p>
<p>&nbsp;</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/choromda-a-brief-background/feed/</wfw:commentRss>
		<slash:comments>1</slash:comments>
		</item>
	</channel>
</rss>
