<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>Chordoma &#8211; openlabnotebooks.org</title>
	<atom:link href="https://thesgc.github.io/static-openlabnotebooks/category/disease-foundation/chordoma/feed/" rel="self" type="application/rss+xml" />
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<description>A growing team of groundbreaking scientists around the world are now sharing their lab notebooks online</description>
	<lastBuildDate>Mon, 24 Aug 2020 18:33:44 +0000</lastBuildDate>
	<language>en-GB</language>
	<sy:updatePeriod>
	hourly	</sy:updatePeriod>
	<sy:updateFrequency>
	1	</sy:updateFrequency>
	<generator>https://wordpress.org/?v=5.4.2</generator>

<image>
	<url>https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/cropped-lab-jar02-32x32.png</url>
	<title>Chordoma &#8211; openlabnotebooks.org</title>
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<width>32</width>
	<height>32</height>
</image> 
	<item>
		<title>Brachyury (TBXT) ligands for chordoma: 1. Fragment optimization strategy 2. fragment F9000505, B-site, PDB 5QRU</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/brachyury-tbxt-ligands-for-chordoma-1-fragment-optimization-strategy-2-fragment-f9000505-b-site-pdb-5qru/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/brachyury-tbxt-ligands-for-chordoma-1-fragment-optimization-strategy-2-fragment-f9000505-b-site-pdb-5qru/#respond</comments>
		
		<dc:creator><![CDATA[David Drewry]]></dc:creator>
		<pubDate>Mon, 24 Aug 2020 18:33:44 +0000</pubDate>
				<category><![CDATA[Chordoma]]></category>
		<category><![CDATA[David Drewry]]></category>
		<category><![CDATA[SGC UNC Chapel Hill]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=5102</guid>

					<description><![CDATA[Introduction In this blog post I will share details on the new molecules we have designed and synthesized or purchased based on one of our original hits from the x-ray crystallography fragment screen. The compound is called F9000505 and it is bound to pocket B. The crystal structure was deposited in the Protein Data Bank <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/brachyury-tbxt-ligands-for-chordoma-1-fragment-optimization-strategy-2-fragment-f9000505-b-site-pdb-5qru/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p><strong><u>Introduction</u></strong></p>
<p>In this blog post I will share details on the new molecules we have designed and synthesized or purchased based on one of our original hits from the x-ray crystallography fragment screen. The compound is called F9000505 and it is bound to pocket B. The crystal structure was deposited in the Protein Data Bank (pdb) with PDB ID 5QRU. This post will give you a feel for some of the ways we are approaching fragment optimization. Posts in the near future will describe strategies at other fragment sites and also provide data on the binding of these compounds to brachyury as it emerges.</p>
<p><strong><u>Background / Project Goal</u></strong></p>
<p>The expression of brachyury (TBXT) is a hallmark of chordoma. A growing body of data suggests that inhibition of TBXT protein activity, expression, or interactions with protein partners are predicted to be effective and specific means for treating chordoma. A key aim of this project is to discover compounds that bind directly to brachyury with sufficient affinity so that they ca be optimized into clinical candidates for the treatment of chordoma.</p>
<p><strong><u>Initial Fragment Optimization Strategy </u></strong></p>
<p>Fragment-based screening by x-ray crystallography is very powerful in that the location of the fragment hits on the target protein (brachyury) are known after the experiment, and the structure provides details of the interactions that the fragments make with brachyury. Crystallography can identify fragments that bind very weakly, so there is often a long way to go to grow the hit molecules into compound that bind potently enough to elicit the desired biological effects. We are taking an aggressive approach to exploring each of our hits so that we can soon focus efforts on the most promising molecules and binding sites.</p>
<p>The two sources of new molecules are purchase of commercially available compounds, and design and synthesis of new compounds. Purchase of compounds is cost effective and rapid in the early stages of a project but becomes limited as molecules become more complicated to capture the intricacies of the binding sites.</p>
<p>We have purchased compounds primarily through a company called ChemSpace. We performed substructure and similarity searches in the ChemSpace available compound database for each x-ray crystallography hit (“query” molecules for the searching). A substructure search means specific details of the hit need to be present in the molecules identified by the search. A similarity search relaxes the criteria and allows for selection of molecules that the computer tells us have certain similarities to our query molecules. These similarities could be in categories like size, shape, and electronic properties. Similarity searches can identify compounds that are more different from the hits which may be important if they identify new interactions, scaffolds that are easier to make new analogs of, or compound that explore different vectors (directions) around the hits.</p>
<p>We chose initial analogues for purchase to provide important information around each fragment in several domains:</p>
<ol>
<li>Compounds with functional groups that will serve as efficient handles for further elaboration.</li>
<li>Compounds that explore growth of the hit fragment in logical directions based on the crystal structure.</li>
<li>Compounds that add additional functionality that can make new interactions in order to enhance potency.</li>
<li>Compounds that teach us where linkers may be added in order to develop brachyury degraders.</li>
</ol>
<p>By making the fragments larger we can understand where growth is beneficial to potency. This will provide focus to future rounds of purchase or chemistry. Incorporation of chemical handles that can be rapidly elaborated with a variety of building blocks will allow us to make additional analogs quickly. Adding functionality (for example amides, ureas, phenols, heterocycles, substituted phenyls) that can make energetically favorable interactions (hydrogen bonds, pi-pi stacking, lipophilic interactions, charge-charge interactions) improves our chances of increasing potency. Utilizing commercially available analogs of the initial fragments found bound to the TBXT crystal structure provides the means to quickly generate structure activity relationships (SAR – this means how changing the structure changes how well the compound binds to brachyury) around each fragment. Initial SAR is critical as it provides confidence around each hit (can it be optimized, do several similar compounds also bind) and aids in site selection to narrow our focus.</p>
<p><strong><u>Fragment F9000505, binding site B, PDB ID: 5QRU </u></strong></p>
<p><em> </em><em>Compounds for purchase</em></p>
<p><strong>Figure 1</strong> depicts all the compounds that we could buy identified from substructure and similarity searching around fragment hit F9000505. This compound binds to site B, and more details on the way it binds will be found below in the new compound design section. Each dot represents a compound and they are organized in the plot according to two easy to calculate and important properties: lipophilicity (clogP) and molecular weight (MW). The green circles are the compounds that we have purchased and are screening. If any of these are active, we can purchase other compounds (blue dots) that can further supplement the SAR. This iterative cycle of design, synthesis (purchase if possible), and test is the life blood of medicinal chemistry compound optimization.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig1-1024x725.png" alt="" width="780" height="552" class="aligncenter wp-image-5104 size-large" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig1-1024x725.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig1-300x213.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig1-768x544.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig1.png 1200w" sizes="(max-width: 780px) 100vw, 780px" /></p>
<p><strong>Figure 2</strong> depicts examples of molecules we purchased from this set that demonstrate the availability of useful molecules that can answer some of the specific medicinal chemistry questions we need to ask. The blue boxes highlight the areas with structural modifications to the hit compound.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-2.png" alt="" width="643" height="604" class="aligncenter wp-image-5105 " srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-2.png 978w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-2-300x282.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-2-768x721.png 768w" sizes="(max-width: 643px) 100vw, 643px" /></p>
<p><em>Design and Synthesis of New Compounds</em></p>
<p>We have chemists on our team at UNC and also a talented team of chemists at a company called Piramal. We have been able to make excellent progress on new compound synthesis even with the initial lab shutdowns and then the reduced capacity necessitated by Covid-19. The team at Piramal has focused on preparation of analogs of fragments for which there are fewer commercial analogs, and on more elaborate chemistry to explore promising fragment sites in greater detail using the same principles outlined above.</p>
<p><strong>Figure 3</strong> depicts the crystal structure of fragment F9000505 bound to pocket B of brachyury. This simple compound results from the acylation of 2-methoxy-5-methylaniline with the amino acid glycine. The phenyl ring makes lipophilic contacts with the protein and the 5-methyl group tucks into a lipophilic pocket. The oxygen of the 2-methoxy group makes an intramolecular hydrogen bond with the adjacent NH, helping to orient the trajectory of the carbonyl of the amide and also the terminal amine of the glycine. In this locked in orientation the carbonyl of the fragment forms a hydrogen bond with Ala119 of brachyury and the primary amine forms a hydrogen bond to the carbonyl of Pro117.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog2-fig3-286x300.png" alt="" width="315" height="330" class="aligncenter wp-image-5106 " srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog2-fig3-286x300.png 286w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog2-fig3-768x806.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog2-fig3.png 806w" sizes="(max-width: 315px) 100vw, 315px" /></p>
<p>Based on this structure there are a number of avenues that need to be explored for optimization of binding. The methyl substituent in the lipophilic pocket may not be ideal. Groups with different size and electronic characteristics need to be explored (for example: F, Cl, ethyl, cyclopropyl). There is plenty of room to extend off of the methoxy group. Doing this may allow us to pick up additional interactions with the surface of the protein. In addition, this may be an attractive position for a linker that will be suitable for synthesis of a bifunctional degrader. Another strategy is to extend the primary amine in order to reach Glu116 which could increase affinity. The alpha carbon of the glycine moiety presents another opportunity. Simple substituents like methyl will add conformational constraint which can enhance potency. Because this building block comes from amino acids, there is opportunity to quickly explore stereochemistry at this position and a range of functional groups that have potential to form new interactions. Finally, we can form a 5- or 6-membered ring between the NH and the O of the fragment. This will serve to further lockdown the orientation of the carbonyl and NH2 that make key hydrogen bonds and offer different vectors for exploring interactions with other brachyury residues. Our initial compound design targeted 68 molecules. These molecules have all been synthesized and submitted from screening in the surface plasmon resonance (SPR) binding assay. A sampling of the 68 compounds are depicted in <strong>Figure 4</strong>. These show example modifications of the methyl group (<strong>4A</strong>), primary amine (<strong>4B</strong>), the alpha carbon of the glycine moiety (<strong>4C</strong>), and the methyl ether (<strong>4D</strong>).</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-4.png" alt="" width="861" height="573" class="wp-image-5107 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-4.png 1538w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-4-300x200.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-4-1024x682.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-4-768x511.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-4-1536x1023.png 1536w" sizes="(max-width: 861px) 100vw, 861px" /></p>
<p><em>Synthetic chemistry schemes</em></p>
<p><strong>Figure 5</strong> show the chemistry scheme used to make many of the 68 new analogues of F9000505 synthesized for this project.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-5.png" alt="" width="698" height="172" class="alignnone wp-image-5108 " srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-5.png 1434w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-5-300x74.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-5-1024x253.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-5-768x190.png 768w" sizes="(max-width: 698px) 100vw, 698px" /></p>
<p><strong>Figure 6</strong> demonstrates utilization of an easily synthesized key intermediate to explore the addition of new functionality at three different distances from the core, linked with an amide. If any offer enhanced potency, follow up will focus on modifications of the pendant aryl ring.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-6.png" alt="" width="584" height="275" class="wp-image-5109 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-6.png 1332w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-6-300x141.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-6-1024x483.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-6-768x362.png 768w" sizes="(max-width: 584px) 100vw, 584px" /></p>
<p><strong>Figure 7</strong> depicts three different ways to functionalize the same key intermediate with robust chemistry that provides different sets of products. A key element of the initial fragment analogue strategy is to generate and use key intermediates that can be diversified in the last synthetic step.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-7.png" alt="" width="396" height="220" class="wp-image-5110 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-7.png 910w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-7-300x167.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-7-768x427.png 768w" sizes="(max-width: 396px) 100vw, 396px" /></p>
<p>The crystal structure suggests that new interactions may be possible if we grow the molecule in the ether position. In addition to making several different ethers, we have opted to incorporate both an alkyne linker and a triazole linker in this position. These choices offer different spacing, different angles, different functionalities, and can be expanded with robust chemistry. An overview of this linker modification chemistry is shown in <strong>figure 8</strong>.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-8.png" alt="" width="797" height="458" class="aligncenter wp-image-5111 " srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-8.png 1494w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-8-300x172.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-8-1024x588.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog-2-fig-8-768x441.png 768w" sizes="(max-width: 797px) 100vw, 797px" /></p>
<p>Finally, a key element of the strategy is to identify positions on the molecules where linkers can be attached to generate bifunctional degradation reagents. Crystallography will of course provide critical information to guide linker location. To improve our chances of quickly identifying potential linker locations, we are routinely incorporating some building blocks with functionality that lends itself to linker friendly chemistry. Currently these are mostly simple amides and Boc-protected amines, but azides and alkynes would also be valuable additions. Some newly synthesized compounds with “linker-friendly” functionality are shown in <strong>figure 9</strong>.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog2-fig-9.png" alt="" width="605" height="294" class="wp-image-5112 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog2-fig-9.png 1290w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog2-fig-9-300x146.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog2-fig-9-1024x499.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/blog2-fig-9-768x374.png 768w" sizes="(max-width: 605px) 100vw, 605px" /></p>
<p>&nbsp;</p>
<p>Stay tuned for details on compounds we have made and purchased for other sites and, importantly, the SPR screening data that should be available soon that lets us know how we are doing in terms of binding affinity!</p>
<p>Don&#8217;t hesitate to reach out with questions and comments.</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/brachyury-tbxt-ligands-for-chordoma-1-fragment-optimization-strategy-2-fragment-f9000505-b-site-pdb-5qru/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>Introducing the team: The Workman Group at the Institute of Cancer Research (ICR), London</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/introducing-the-team-the-workman-group-at-the-institute-of-cancer-research-icr-london/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/introducing-the-team-the-workman-group-at-the-institute-of-cancer-research-icr-london/#comments</comments>
		
		<dc:creator><![CDATA[Hadley Sheppard]]></dc:creator>
		<pubDate>Thu, 30 Apr 2020 20:06:27 +0000</pubDate>
				<category><![CDATA[Chordoma]]></category>
		<category><![CDATA[Hadley Sheppard]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=4618</guid>

					<description><![CDATA[Hello everyone! My name is Hadley Sheppard, and I am an incoming Postdoctoral Fellow joining Professor Paul Workman’s Signal Transduction and Molecular Pharmacology (STMP) Team at the Institute of Cancer Research (ICR), London. Dr. Workman’s group is based in the Cancer Research UK Cancer Therapeutics Unit at ICR and has extensive experience in many areas <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/introducing-the-team-the-workman-group-at-the-institute-of-cancer-research-icr-london/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[
<p style="text-align: left;">Hello everyone! <br /><br />My name is Hadley Sheppard, and I am an incoming Postdoctoral Fellow joining Professor Paul Workman’s Signal Transduction and Molecular Pharmacology (STMP) Team at the Institute of Cancer Research (ICR), London. Dr. Workman’s group is based in the Cancer Research UK Cancer Therapeutics Unit at ICR and has extensive experience in many areas of cancer drug discovery – from identification of drug targets, to discovery of novel cancer drugs, and their progression to clinical trials in cancer patients.<br /><br />I recently finished my post-graduate PhD in the lab of Dr. Charles Lin at Baylor College of Medicine. My PhD thesis was actually focused on chordoma research and I have been collaborating with the Chordoma Foundation since the beginning of my graduate work in 2017. I am very passionate about my work in chordoma; not only do I find our work very impactful as it can potentially better the lives of affected patients, but chordoma also serves as an excellent system to study transcriptional dependencies in cancer. Chordoma is similar to many other cancers in being dependent on an oncogenic transcription factor (TF) that drives gene expression required for the cancer to develop and progress. My post-graduate work centered on understanding the role of brachyury, the oncogenic transcription factor found in all chordomas, and its potential as a drug target. I’m very excited to join the STMP team and work closely with David Drewry and Opher Gileadi’s groups, under the support of both the Chordoma Foundation and the Mark Foundation for Cancer Research, as we now move towards directly drugging brachyury. <br /><br />The Drewry, Gileadi, and Workman teams have come together to form a multidisciplinary group with the overall objective of drugging brachyury. Dr. Gileadi and his team bring expertise in structural biology, Dr. Drewry and his team expertise in medicinal chemistry, and finally we in the Workman group bring expertise in molecular pharmacology and chemical biology.<br /><br />Given this, my primary role in this collaborative project — under the mentorship of Dr. Workman and Dr. Paul Clarke, group leader in the Workman lab — will be to validate and characterize compounds targeting brachyury in intact chordoma cells. As you may have read in the previous chordoma open lab notebook posts, Dr. Gileadi and his team have been able to crystalize brachyury’s DNA binding domain in complex with small molecule fragments. These small fragments tell us about regions of brachyury that could potentially be targeted with a drug. <br /><br />Importantly, these fragments also provide a starting point for developing a brachyury-targeting drug. Dr. Drewry and his team are now working on chemically modifying these small molecules to improve their characteristics and, at this stage especially, their potency in inhibiting brachyury function. While it has been possible to measure certain characteristics of these small molecules in biochemical assays, the next critical step will be to test the best small molecules in intact chordoma cancer cells. This will include validation of brachyury binding and also inhibition of brachyury function through a suite of cellular assays whose development was initiated during my PhD. With the knowledge that we gain from these assays in the Workman lab, we will be able to better inform Dr. Drewry and Dr. Gileadi’s teams so that we can continue to optimize our brachyury-targeting small-molecules. <br /><br />For those interested in further reading on fragment-based drug discovery and its clinical successes, I highly recommend the following review:<br /><br />Structure-based drug design: aiming for a perfect fit.<br />van Montfort RLM, Workman P. Structure-based drug design: aiming for a perfect fit. Essays Biochem. 2017;61(5):431‐437. Published 2017 Nov 8. doi:10.1042/EBC20170052<br /><br />That is all I have for today – I will be following up on our progress in future blog posts!</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/introducing-the-team-the-workman-group-at-the-institute-of-cancer-research-icr-london/feed/</wfw:commentRss>
			<slash:comments>2</slash:comments>
		
		
			</item>
		<item>
		<title>TBXT (brachyury) and Chordoma: Project Introduction</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/tbxt-brachyury-and-chordoma-project-introduction/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/tbxt-brachyury-and-chordoma-project-introduction/#comments</comments>
		
		<dc:creator><![CDATA[David Drewry]]></dc:creator>
		<pubDate>Tue, 14 Apr 2020 19:49:19 +0000</pubDate>
				<category><![CDATA[Chordoma]]></category>
		<category><![CDATA[David Drewry]]></category>
		<category><![CDATA[SGC UNC Chapel Hill]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=4516</guid>

					<description><![CDATA[In my Chordoma Open Lab Notebook entries, I will tell you about a project that is very important to me, targeting the transcription factor brachyury (also known as TBXT) for the rare cancer chordoma. In the coming weeks and months, the amount of science strategy, science details, and science progress shared here will rise. Today’s <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/tbxt-brachyury-and-chordoma-project-introduction/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p>In my Chordoma Open Lab Notebook entries, I will tell you about a project that is very important to me, targeting the transcription factor brachyury (also known as TBXT) for the rare cancer chordoma. In the coming weeks and months, the amount of science strategy, science details, and science progress shared here will rise. Today’s entry will simply introduce the project to you, tell you how we got here, and share my motivation. By the way, it will help me and the team if you let me (us!) know what you think, what questions you have, and what you’d like to know. Please feel free to share these posts with others via email links or social media. In addition, a long career has taught me that I don’t have all the answers! If you have ideas, please share them with us!</p>
<p>Working on Chordoma is personal for me. Eighteen years ago, my daughter Carolyn was in first grade and a bright and joyful little boy in her class named Justin (pictured below, graphic from Chordoma Foundation) was diagnosed with chordoma. Of course, I had never heard about this rare cancer which strikes only one in a million people. In the US your chances of getting hit by lightning are higher. Justin and his wonderful parents shared his difficult journey with his classmates and their parents. Justin wanted to talk about this. Justin drew his friends in and inspired them to find joy and strength and motivation in his journey.  Their enthusiasm in turn inspired me. As a scientist I wanted to help for Justin, Justin’s parents, my daughter Carolyn, and all of Justin’s classmates who gave what they could of their time, talents, and treasure.</p>
<p><img src="https://chordomafoundation.org/wp-content/uploads/2011/09/Justin%20Perseverance.jpg" width="427" height="285" class="alignleft" /></p>
<p>A number of years after Justin’s diagnosis I met Josh Sommer, a Chordoma patient himself, and Co-Founder and Executive Director of the Chordoma Foundation. Josh attended a number of fund raisers that my daughter and her classmates were involved in, and one thing led to another and I found ways to get involved. Josh inspired me with his positive attitude, drive, and ability to band people together. He is a connector, and science thrives on connections. We are all better when we work together. Josh introduced me to Dr. Adrienne Flanagan, a leader in chordoma research, a key thought leader on the links between brachyury and chordoma, and a real blessing to the chordoma community for her hard work, insights, and passion. I was able to get involved by leveraging an open science project I was working on at the pharmaceutical company GlaxoSmithKline (GSK) sharing inhibitors of an important class of proteins called kinases. Adrienne and her team tested them, and we were able to make some interesting connections between kinase inhibition and chordoma cell line growth. I believe there is more to be done with kinase inhibition and chordoma, but that’s a story for another day.</p>
<p>I have since moved from GSK to the Eshelman School of Pharmacy at the University of North Carolina at Chapel Hill. Our SGC-UNC team (<a href="https://www.sgc-unc.org/">https://www.sgc-unc.org/</a>) is the first US site of the Structural Genomics Consortium (SGC, <a href="https://www.thesgc.org/">https://www.thesgc.org/</a>) a worldwide public private partnership committed to supporting the discovery of new medicines through open science. We generate tools and reagents and share them with the scientific community openly and without restrictions. We want to move quickly, we want to facilitate research programs in other groups, and we want to help patients.</p>
<p><strong><u>TBXT (brachyury): a key target for chordoma</u></strong></p>
<p>Emerging data suggests that the transcription factor TBXT is a key vulnerability of chordoma and a prime candidate to target in order to create medicines to treat chordoma. For example, TBXT knockdown stops the proliferation of chordoma cell lines and reduces their viability. TBXT is the top selectively essential gene in chordoma. A great place to start if you want to know more about TBXT and chordoma is this paper by Tanaz Sharifnia and colleagues (link here: <a href="https://pubmed.ncbi.nlm.nih.gov/30664779/">https://pubmed.ncbi.nlm.nih.gov/30664779/</a>). As a transcription factor, it falls in the “difficult target” category. There is no active site tailor made for small molecules. Details of downstream signaling via protein-protein interactions are poorly understood. On the plus side, TBXT is expressed in development but almost nonexistent (or very low expression) in most adult human tissues, suggesting that if we are successful, compounds may have a useful therapeutic index. The high unmet need and weight of evidence convince us that TBXT is an Achilles heel of chordoma that needs to be addressed through a targeted drug discovery program.</p>
<p>Our initial thought was to see if we could identify small molecule binders for TBXT using x-ray crystallography. To get there we needed a number of things to fall into place. I am grateful to the Mark Foundation and Chordoma Foundation for funding our initial proposal that allowed us to generate enough data to convince ourselves that this strategy holds great promise.</p>
<p>As part of the SGC, I was well aware of the SGC Target Enabling Package (TEP) program and realized it was an ideal mechanism to facilitate study of brachyury. You can read more about the TEP program here: <a href="https://www.thesgc.org/tep">https://www.thesgc.org/tep</a>. I was excited to nominate TBXT as a TEP target, and absolutely thrilled when SGC scientist Opher Gileadi (Oxford University, SGC) agreed to work on the project. Opher and his talented team started the project by expressing and purifying the TBXT protein and then in short order they identified conditions to reproducibly grow crystals of the protein. They then collaborated with the team at Diamond to do an x-ray crystallography based high throughput fragment screen using the XChem platform (more information on XChem here:  <a href="https://www.diamond.ac.uk/Instruments/Mx/Fragment-Screening.html">https://www.diamond.ac.uk/Instruments/Mx/Fragment-Screening.html#</a>). This resulted in more than 40 fragments binding to a number of different sites on the transcription factor. As a medicinal chemist, it is an exciting day when you see a range of small molecules bound to your target of interest! With x-ray evidence that there are hot spots for binding small molecules to TBXT, we have now set our sights on a more challenging goal and bigger prize, optimization of these fragments into molecules that modulate TBXT biology and kill chordoma cells. In coming posts, we will provide more detail on the initial fragment screen and start walking through the steps we are taking.</p>
<p><strong><u>Some of our project goals</u></strong></p>
<ul>
<li>Develop and provide reagents the community can use to explore TBXT biology (eg: compounds, protein constructs, biophysical assays, cell assays, antibodies)</li>
<li>Develop assays to measure affinity (thermal shift, SPR)</li>
<li>Synthesize analogues of fragments at each site in order to optimize binding potency</li>
<li>Generate crystal structures of new and improved compounds</li>
<li>Generate Structure Activity Relationships that allow us or others to generate reagents that can lead to the degradation of TBXT</li>
<li>Understand impact potent fragments (and degraders) have on TBXT biology</li>
</ul>
<p><strong><u>Project Summary – things you will hear more about as the project progresses</u></strong></p>
<p>To date we have exemplified reproducible crystallography of TBXT and demonstrated the utility of fragment screening as a means of identifying TBXT hotspots for binding small molecules. The primary aim of the current project is to develop small molecule inhibitors of TBXT or of its interaction partners and <strong><em>identify at least one tractable lead molecule targeting TBXT that has potential for final optimization into a candidate compound</em></strong>. As we progress towards this end, we also aim to understand the structural basis of the activities of TBXT and contribute to the understanding of the mechanism of the disease. <strong><em>At the end of this two-year project, we will have the infrastructure, assays, reagents, and compounds in place that form the basis for a drug discovery project to move TBXT modulators into the clinic.</em></strong></p>
<p>In the coming weeks different members of the project team will provide updates. Here is a very brief team introduction. Currently our team is located at three sites.</p>
<ul>
<li>SGC-UNC at the UNC Eshelman School of Pharmacy: David Drewry, Carrow Wells</li>
<li>SGC at Oxford University: Opher Gileadi, Joseph Newman, Angeline Gavard</li>
<li>Institute for Cancer Research: Paul Workman, Paul Clarke, Hadley Sheppard</li>
</ul>
<p>We also have guidance and support from the teams at our generous funders, the Mark Foundation for Cancer Research (Becky Bish, Ryan Schoenfeld, Michele Cleary) and the Chordoma Foundation (Josh Sommer, Joan Levy). Thank you!</p>
<p>You can find out more about these organizations here:</p>
<p>Mark Foundation for Cancer Research: <a href="https://themarkfoundation.org/">https://themarkfoundation.org/</a></p>
<p>The Chordoma Foundation: <a href="https://www.chordomafoundation.org/">https://www.chordomafoundation.org/</a></p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/tbxt-brachyury-and-chordoma-project-introduction/feed/</wfw:commentRss>
			<slash:comments>18</slash:comments>
		
		
			</item>
		<item>
		<title>Growing co-crystallized fragments to ligands of the transcription factor brachyury (T)</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/growing-co-crystallized-fragments-to-ligands-of-the-transcription-factor-brachyury-t/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/growing-co-crystallized-fragments-to-ligands-of-the-transcription-factor-brachyury-t/#respond</comments>
		
		<dc:creator><![CDATA[Louisa Temme]]></dc:creator>
		<pubDate>Wed, 24 Apr 2019 21:15:14 +0000</pubDate>
				<category><![CDATA[Chordoma]]></category>
		<category><![CDATA[Louisa Temme]]></category>
		<category><![CDATA[SGC UNC Chapel Hill]]></category>
		<category><![CDATA[Understudied Kinases]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2758</guid>

					<description><![CDATA[In order to develop ligands addressing different possible binding sites of the transcription factor brachyury, fragments co-crystallized with this protein should be developed into ligands by using the program SparkTM (Cresset®). Brachyury (T, TBXT) is an essential regulator of the notochord development. Brachyury is primarily expressed in the embryo but not in the majority of normal <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/growing-co-crystallized-fragments-to-ligands-of-the-transcription-factor-brachyury-t/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p>In order to develop ligands addressing different possible binding sites of the transcription factor brachyury, fragments co-crystallized with this protein should be developed into ligands by using the program Spark<sup>TM </sup>(Cresset<sup><span style="font-size: 11.25px;">®</span></sup>).</p>
<p>Brachyury (T, TBXT) is an essential regulator of the notochord development. Brachyury is primarily expressed in the embryo but not in the majority of normal adult tissue. It is known that brachyury is overexpressed in chordoma, a primary bone cancer, and plays a role in the disease pathogenesis. Chordoma affects the skull-base, the mobile spine and the sacrum. It is a slow-growing, malign cancer that is locally invasive and has a tendency to recur although a surgical and/or radiation therapy was conducted.<sup>1</sup></p>
<p>It is hypothesized that chordoma is originated from embryonic notochordal remnants. Brachyury is proposed to act as an aberrantly activated transcription factor and thereby be oncogenic. No approved targeted therapies, chemotherapies or immunotherapies are available for chordoma and no small-molecule inhibitors for brachyury have been developed.<sup>1 </sup>Therefore, a fragment-based approach should be explored to develop ligands for brachyury to study this protein.</p>
<p>Our colleagues Joseph Newman and Opher Gileadi at SGC Oxford co-crystallized wild-type human brachyury protein with various fragments which bind to the sites A-E (Figure 1, left). Thankfully, they provided the fragment and protein structures to us so that we can explore the usage of the Spark<sup>TM </sup>growing approach to develop ligands for these binding sites.</p>
<p><img class="wp-image-2759 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Picture1-300x121.png" alt="" width="592" height="239" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Picture1-300x121.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Picture1-768x310.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Picture1.png 974w" sizes="(max-width: 592px) 100vw, 592px" />Figure 1. Left: Aligned X-ray structures of the human wild-type brachyury protein in complex with fragments at different binding sites (A-E). These first crystallization structures contain Cd<sup>2+</sup>ions (green balls). Right: Binding site A close to loop with D177 and in proximity to residue Y88.<sup>2</sup></p>
<p>First, we focused on binding site A since most fragments bind to this binding site and site A is close to a loop that is mutated in chordoma-linked variant G177D<sup>3 </sup>and to the amino acid residue Y88 that, if mutated, affects binding to effector proteins<sup>4 </sup>(Figure 1, left).</p>
<p><img class="wp-image-2761 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Picture2-300x86.png" alt="" width="607" height="174" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Picture2-300x86.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Picture2-768x220.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Picture2.png 974w" sizes="(max-width: 607px) 100vw, 607px" /></p>
<p>Figure 2. Fragment <strong>1 </strong>and <strong>2 </strong>co-crystallized in the binding site A of wild-type TBXT.</p>
<p>21 fragments were co-crystallized to binding site A. In Figure 2, the binding modes of two co-crystallized fragments are displayed. Fragments <strong>1 </strong>and <strong>2 </strong>show interactions or close proximity to two of the three amino acids Ser129, Ser89 and Arg180. Therefore, the growing approach of the program Spark<sup>TM </sup>should be used to identify new ligands that are able to form all three interactions. One growing approach started from the amino group of fragment <strong>1 </strong>and was guided by fragment <strong>2 </strong>towards the region of its carboxylate group (Figure 3).</p>
<p><img class="wp-image-2763 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Picture3-300x171.png" alt="" width="484" height="276" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Picture3-300x171.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Picture3-768x438.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Picture3-780x445.png 780w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Picture3.png 781w" sizes="(max-width: 484px) 100vw, 484px" /></p>
<p>Figure 3: Spark<sup>TM </sup>growing approach starting from fragment <strong>1 </strong>towards Arg180 guided by fragment <strong>2</strong>. The functional groups of fragments <strong>1 </strong>and <strong>2 </strong>revealing H-bond interactions with amino acid residues are marked with the following colors: Ser129 (red), Ser89 (blue), Arg180 (green).</p>
<p>To achieve new ligands with similar properties in certain areas so-called field points were used. They are ‘condensed representations of electrostatic, hydrophobic and shape properties’ (Cresset<sup><span style="font-size: 11.25px;">®</span></sup>) of those areas where they are placed by the program Spark<sup>TM</sup>. Regarding the growing approach of fragment <strong>1 </strong>and <strong>2 </strong>the hydrophobic field point of the phenyl ring and the negative field point of the carboxylate group of fragment <strong>2 </strong>were used. In total three of the 21 fragments bound to site A were used as starting points and each of them were grown in the area of two different fragments. The results were clustered by Spark<sup>TM </sup>depending on their structural similarity. Those compounds with the highest score of the first 20 clusters ranked according their scores were chosen to analyze their interactions by docking into binding site A using the docking program Flare<sup>TM </sup>(Cresset<sup><span style="font-size: 11.25px;">®</span></sup>). 4,000 docking results were inspected visually and compounds with a high docking score, high electrostatic complementarity and high number of interactions to the binding site were picked.</p>
<p>Figure 4 shows the docking pose of the example compound <strong>3 </strong>that was obtained by docking <strong>3 </strong>in binding site A of brachyury using Flare<sup>TM</sup>. Besides other criteria like docking score and electrostatic complementarity following interactions of <strong>3 </strong>are promising for the co-crystallization of this compound with brachyury: two H-bond interactions of the positively charged amino moiety to Ser129 and His126, the proximity of the amide carbonyl-group to Ser89 and the H-bond interactions of the oxazole O-atom and N-atom towards Arg180.</p>
<p><img class="wp-image-2765 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Picture4-300x232.png" alt="" width="454" height="351" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Picture4-300x232.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Picture4.png 633w" sizes="(max-width: 454px) 100vw, 454px" />Figure 4. Binding mode of new compound <strong>3 </strong>obtained by docking into binding site A of wild-type brachyury with Flare<sup>TM</sup>. H-bond interactions are marked with green dotted lines.</p>
<p>Additionally, new compounds were obtained by searching in the Enamine REAL database for similar compounds to those compounds that were found by the Spark<sup>TM </sup>approach and that showed promising docking results in Flare<sup>TM</sup>.</p>
<p>A selection of compounds that are identical or slightly different than those found in the Spark<sup>TM </sup>approach as well as compounds that represent similar ones to the Spark<sup>TM </sup>hits were purchased and will be tested regarding their ability to co-crystallize with brachyury. If the compounds bind to this transcription factor, they will represent promising starting points for future ligands of brachyury.</p>
<p>&nbsp;</p>
<p>(1) Sharifnia, T., <em> Med. (N. Y., NY, U. S.)</em>, <strong>2019</strong>, <em>25</em>, 292-300.</p>
<p>(2) Joseph Newman and Opher Gileadi, SGC Oxford, University of Oxford, UK.</p>
<p>(3) Pillay, N., <em>Nat. Genet.</em>, <strong>2012</strong>, <em>44</em>, 1185-1187.</p>
<p>(4) Beisaw, A., <em>EMBO reports</em>, <strong>2018</strong>, <em>19</em>, 118-134.</p>
<p>&nbsp;</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/growing-co-crystallized-fragments-to-ligands-of-the-transcription-factor-brachyury-t/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>Introducing the team: The Gileadi Group in Oxford.</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/introducing-the-team-the-gileadi-group-in-oxford/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/introducing-the-team-the-gileadi-group-in-oxford/#respond</comments>
		
		<dc:creator><![CDATA[Opher Gileadi]]></dc:creator>
		<pubDate>Fri, 30 Nov 2018 16:31:10 +0000</pubDate>
				<category><![CDATA[Alzheimer’s disease]]></category>
		<category><![CDATA[Amyotrophic Lateral Sclerosis (ALS)]]></category>
		<category><![CDATA[Chordoma]]></category>
		<category><![CDATA[Opher Gileadi]]></category>
		<category><![CDATA[SGC Univ. of Oxford]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2198</guid>

					<description><![CDATA[Hi! I’m Opher Gileadi, I head a group at the Structural Genomics Consortium (SGC) in Oxford. For many years, we’ve been looking at proteins involved in human. We’ve looked at a wide variety of proteins: from DNA repair enzymes and transcription factors to enzymes involved in signalling. In most cases, we focus on early work: <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/introducing-the-team-the-gileadi-group-in-oxford/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p>Hi! I’m Opher Gileadi, I head a group at the Structural Genomics Consortium (SGC) in Oxford. For many years, we’ve been looking at proteins involved in human. We’ve looked at a wide variety of proteins: from DNA repair enzymes and transcription factors to enzymes involved in signalling. In most cases, we focus on early work: structure determination, activity assays, antibody generation, and finding initial small molecule “hits”. Having the right reagents are the key to enabling new drug targets; we expect that our data will allow – or encourage – further work by medicinal chemists and biologists to seriously test therapeutic hypotheses (we call our data and reagents sets <a href="https://www.thesgc.org/tep">TEPs</a> – Target Enabling Packages).</p>
<p>My team and I will be blogging and sharing our experiments on three areas of interest:</p>
<p><strong>Alzheimer’s disease</strong>: We are funded by the NIH through the <a href="https://www.nia.nih.gov/research/amp-ad">AMP-AD</a> project (Accelerating Medicines Partnership &#8211; Alzheimer&#8217;s Disease). The project has integrated data from brain autopsies and model systems to identify new possible protein targets for treatment or prevention of Alzheimer’s (annotated in the <a href="https://agora.ampadportal.org/genes/(genes-router:genes-list)">Agora</a> web site, run by <a href="http://sagebionetworks.org/research-projects/agora/">SAGE Bionetworks</a>. Our group is generating TEPs for some of these targets – we’ll share our progress in future posts.</p>
<p><strong>ALS (Amyotrophic Lateral Sclerosis)</strong>: We are part of <a href="https://www.als-rap.org/">ALS-RAP</a>, the ALS reproducible antibody platform. We’re addressing one of the big conundrums to biomedical research: the lack, or the poor quality, of many antibodies used for research ranging from cell biology to histopathology.</p>
<p>Together with colleagues in Stockholm and Montreal we aim to validate a set of antibodies to 30 proteins involved in the disease. We will test commercial antibodies as well as generate new ones through phage display technologies.</p>
<p><strong>Chordoma</strong>: <a href="https://www.chordomafoundation.org/understanding-chordoma/">Chordoma</a> is a rare type of cancer that occurs in the bones of the skull base and spine. It is resistant to all forms of chemotherapy; patients are treated by surgery and radiation, but recurrence is common and, eventually, untreatable. A unifying molecular characteristic of chordoma is overexpression of a variant of the transcription factor T-Brachyury (TBXT), which is essential for survival of the cancer cells. Joseph Newman in my group has solved structures of the human Brachyury protein (PDB codes <a href="http://www.rcsb.org/structure/6F58">6F58</a>, <a href="http://www.rcsb.org/structure/6F59">6F59</a>); we are working together with groups in the University of North Carolina and in Baylor College (Texas) to explore TBXT as a drug target.</p>
<p>That’s all for today – we’ll follow-up with stories in these three areas.</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/introducing-the-team-the-gileadi-group-in-oxford/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>Screening EGFR and GAK Inhibitors in Chordoma</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/screening-egfr-and-gak-inhibitors-in-chordoma/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/screening-egfr-and-gak-inhibitors-in-chordoma/#respond</comments>
		
		<dc:creator><![CDATA[Kaleb Naegeli]]></dc:creator>
		<pubDate>Thu, 24 May 2018 16:40:59 +0000</pubDate>
				<category><![CDATA[Catalyst UNC Chapel Hill]]></category>
		<category><![CDATA[Chordoma]]></category>
		<category><![CDATA[EGFR and GAK]]></category>
		<category><![CDATA[Kaleb Naegeli]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=1259</guid>

					<description><![CDATA[A major therapeutic avenue for chordoma is targeting the receptor tyrosine kinase, EGFR. Its ligand, EGF, is overexpressed as a transcriptional target of Brachyury and polysomy of EGFR’s region of chromosome 7 is frequent in chordoma. To this extent I, in collaboration with SGC-UNC’s post-doc Chris Asquith, am working on the profiling of a number <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/screening-egfr-and-gak-inhibitors-in-chordoma/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p>A major therapeutic avenue for chordoma is targeting the receptor tyrosine kinase, EGFR. Its ligand, EGF, is overexpressed as a transcriptional target of Brachyury and polysomy of EGFR’s region of chromosome 7 is frequent in chordoma. To this extent I, in collaboration with SGC-UNC’s post-doc Chris Asquith, am working on the profiling of a number of EGFR inhibitors. Of particular interest, though, is this: many commercial EGFR inhibitors also hit another kinase – GAK (cyclin-G associated kinase). GAK plays a major role in prostate cancer, but its role in chordoma is unknown. Based on gene expression data from the Chordoma Foundation, we do know GAK is expressed in most chordoma cell lines.</p>
<p>First, we screened a set of published EGFR inhibitors, some of which also have GAK affinity, on the chordoma cell lines U-CH1 and U-CH2 (Table 1). Most of these compounds have strong potency (&lt;1 μM) on U-CH1 but not as strong of potency on U-CH2; this is expected, as previous work indicates U-CH2 is more resistant to EGFR inhibitors<sup>1</sup>. Following this initial screen, Chris has developed and I am characterizing a new set of compounds. This is a useful set because some of them have high affinity for EGFR, some have high affinity for GAK, and others show relatively equal affinity for both kinases. Using these compounds will allow us to interrogate if GAK plays a role in chordoma.</p>
<p>I performed a series of experiments to assess how much of each compound was required to kill (or at least stop the growth of) chordoma cells (U-CH1 and U-CH2) as well as normal skin fibroblasts (WS1). An ideal therapeutic would have a high potency for U-CH1 and U-CH2 without being toxic for WS1. A number of compounds have low values in U-CH1 but, like the published compounds in referenced above, do not show activity in U-CH2. A few compounds show activity in both U-CH1 and U-CH2, but they tend to also show similar potency in WS1. The mechanisms of these compounds warrant further investigation, as they will continue to help us understand potential interplay between EGFR and GAK, but their toxicity in normal skin cells may indicate they aren’t great therapeutic candidates.</p>
<p><img class="wp-image-1261 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/05/20180522-Table2-300x241.png" alt="" width="348" height="280" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/05/20180522-Table2-300x241.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/05/20180522-Table2-768x617.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/05/20180522-Table2-1024x823.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/05/20180522-Table2.png 1501w" sizes="(max-width: 348px) 100vw, 348px" /></p>
<p style="text-align: center;"><strong>Table 1</strong>: Compound potency in chordoma cells and normal fibroblasts in vitro. Values shown are EC<sub>50</sub> in μM.</p>
<p>Going forward, I am now trying to work out the mechanism of action of a subset of these compounds: do they kill the cells or just stop their growth? Does specific genetic knockdown of GAK produce similar phenotypes? These data will help us understand if GAK has a role in chordoma and how we can therapeutically target this kinase.</p>
<p>You can read greater detail about these experiments, their methodology, and see more data on <a href="https://zenodo.org/record/1251296#.Wwbq-CAh1EY">Zenodo</a>.</p>
<p><sup>1</sup>Scheipl S, Barnard M, Cottone L, <em>et al.</em> EGFR inhibitors identified as a potential treatment for chordoma in a focused compound screen. <em>J Pathol</em> 2016; <strong>239</strong>: 320–334.</p>
<p>&nbsp;</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/screening-egfr-and-gak-inhibitors-in-chordoma/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>Chordoma: A Brief Background</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/choromda-a-brief-background/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/choromda-a-brief-background/#comments</comments>
		
		<dc:creator><![CDATA[Kaleb Naegeli]]></dc:creator>
		<pubDate>Tue, 15 May 2018 17:48:27 +0000</pubDate>
				<category><![CDATA[Catalyst UNC Chapel Hill]]></category>
		<category><![CDATA[Chordoma]]></category>
		<category><![CDATA[Kaleb Naegeli]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=1237</guid>

					<description><![CDATA[Greetings! My first post here will be a brief introduction. More in-depth experiment and data posts are coming soon! I am a post-doctoral research associate in the UNC Catalyst for Rare Diseases1. Throughout my education and training, I have been interested in cancer – I worked on epigenetics of ovarian cancer as an undergraduate and <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/choromda-a-brief-background/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p>Greetings! My first post here will be a brief introduction. More in-depth experiment and data posts are coming soon!</p>
<p>I am a post-doctoral research associate in the <a href="https://pharmacy.unc.edu/research/unc-catalyst-rare-diseases/" target="_blank" rel="noopener">UNC Catalyst for Rare Diseases</a><sup>1</sup>. Throughout my education and training, I have been interested in cancer – I worked on epigenetics of ovarian cancer as an undergraduate and then mechanisms of cell invasion (in a developmental biology system) as a Ph.D. student. Now as a post-doc with the UNC Catalyst for Rare Diseases, my interest in cancer continues in the form of a project on chordoma, a rare bone cancer.</p>
<p>At the recent International Chordoma Research Workshop, the idea of sharing data, reagents, and research progress was raised. Given the affiliation of the UNC Catalyst for Rare Diseases with the open-science SGC, our group seemed like optimal candidates to initiate this undertaking. <strong>Other chordoma researchers: I welcome you to join us! Open science will accelerate the pace of research!<br />
</strong></p>
<p>Chordoma is a rare cancer that occurs along the bones of the central nervous system – from the base of the skull to the tip of the spine. Chordomas arise from remnants of the notochord – a developmental precursor to the vertebral column – and thus are almost always driven by expression of the notochord transcription factor Brachyury (encoded by the gene <em>T</em>). Therapeutic avenues for chordoma are tricky because they are rare (roughly 1 in 10<sup>6</sup> people), slow-growing, and often difficult to resect because of proximity to vital and sensitive structures. Current standard of care is surgical resection followed by radiation, but recurrence, metastasis, and mortality still happens in many cases despite these best practices.</p>
<p>Thus, improved knowledge of the biology of chordoma and the development of new therapeutics are vital for bettering the lives of patients with this rare bone cancer. I have two primary interests in Chordoma which will appear frequently in this blog:</p>
<ul>
<li><strong>Brachyury</strong> – this embryonic transcription factor of the notochord is encoded by the gene <em>T</em>. But T is going to give you way too many Google search results, so most always we’ll stick with Brachyury. This transcription factor is also overexpressed in some other cancers, though its role as a driver in these instances is unclear. While some of the biology of this transcription factor is known – including some of the genes it may regulate – much remains to be learned, including potential chaperones and binding partners. Brachyury presents an intriguing therapeutic avenue for this cancer because of its ubiquitous expression.</li>
<li><strong>EGFR</strong> – this receptor tyrosine kinase is overexpressed (but not mutated) in many chordomas and its ligand, EGF, is a direct transcriptional target of Brachyury. EGFR inhibitors are a potential therapeutic for chordoma, and in collaboration with SGC-UNC, we are working on new inhibitors of EGFR and other kinases as potential chemotherapeutics for chordoma (more information: https://www.chordomafoundation.org/targets/egfr/)</li>
</ul>
<p>With this background out of the way, a backlog of data will be forthcoming.</p>
<p>My work owes many thanks to the Chordoma Foundation for their inspiring array of reagents and data. They are a superb example of how a patient advocacy/disease awareness foundation should be run and how the sharing of reagents and insights can accelerate the progress of research and therapeutic development.</p>
<p>If you are unfamiliar with chordoma, suggested references include the following:</p>
<p><a href="https://www.chordomafoundation.org/understanding-chordoma/" target="_blank" rel="noopener">Chordoma Foundation &#8211; Understanding Chordoma</a><br />
<a href="http://www.thelancet.com/journals/lanonc/article/PIIS1470-2045%2814%2971190-8/abstract">Chordoma Overview (Lancet, 2015)</a><br />
<a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/path.2816" target="_blank" rel="noopener">Role of Brachyury in Chordoma; Presnet et al (2010) J Pathology</a><br />
<a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/path.4082" target="_blank" rel="noopener">The Brachyury Functional Network; Nelson et al (2012) J Pathology</a><br />
<a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/path.4729" target="_blank" rel="noopener">EGFR and EGFR inhibitors in chordoma; Scheipl et al (2016) J Pathology</a></p>
<p><sup>1</sup><strong><a href="https://pharmacy.unc.edu/research/unc-catalyst-rare-diseases/" target="_blank" rel="noopener">The UNC Catalyst for Rare Diseases</a>:</strong> We are a biology group started by the SGC (Structural Genomics Consortium) at the University of North Carolina through funding from the Eshelman Institute for Innovation (in the UNC Eshelman School of Pharmacy). Our commitment and objective is to be a research group that can be a resource for patients and other researchers, a hub with wet lab expertise to do be a source of knowledge in rare diseases. As an “SGC spin-off”, as I like to call our lab, we are committed to a practice of open science. <a href="https://twitter.com/UNC_Catalyst" target="_blank" rel="noopener">You can follow us on Twitter here!</a></p>
<p>&nbsp;</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/choromda-a-brief-background/feed/</wfw:commentRss>
			<slash:comments>1</slash:comments>
		
		
			</item>
	</channel>
</rss>
