<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>SGC Univ. of Oxford &#8211; openlabnotebooks.org</title>
	<atom:link href="https://thesgc.github.io/static-openlabnotebooks/category/laboratory/sgc-oxford/feed/" rel="self" type="application/rss+xml" />
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<description>A growing team of groundbreaking scientists around the world are now sharing their lab notebooks online</description>
	<lastBuildDate>Wed, 19 Aug 2020 17:07:37 +0000</lastBuildDate>
	<language>en-GB</language>
	<sy:updatePeriod>
	hourly	</sy:updatePeriod>
	<sy:updateFrequency>
	1	</sy:updateFrequency>
	<generator>https://wordpress.org/?v=5.4.2</generator>

<image>
	<url>https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/cropped-lab-jar02-32x32.png</url>
	<title>SGC Univ. of Oxford &#8211; openlabnotebooks.org</title>
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<width>32</width>
	<height>32</height>
</image> 
	<item>
		<title>Electroporation: a second look</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/electroporation-a-second-look/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/electroporation-a-second-look/#respond</comments>
		
		<dc:creator><![CDATA[Elizabeth Brown]]></dc:creator>
		<pubDate>Wed, 19 Aug 2020 17:07:37 +0000</pubDate>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Brain Tumours]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>
		<category><![CDATA[Liz Brown]]></category>
		<category><![CDATA[SGC Univ. of Oxford]]></category>
		<category><![CDATA[Understudied Kinases]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=5092</guid>

					<description><![CDATA[It’s never too late to take another look at your data. The incredibly knowledgeable Sally Cowley suggested that I shouldn’t just think about how many cells contain any AlexFluor488 antibody after electroporation, but how bright each cell is (i.e. the amplitude of fluorescence). Looking at this instead will give me an idea of how much <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/electroporation-a-second-look/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p>It’s never too late to take another look at your data.</p>
<p>The incredibly knowledgeable <a href="https://www.stemcells.ox.ac.uk/sally-cowley">Sally Cowley</a> suggested that I shouldn’t just think about how many cells contain <em>any </em>AlexFluor488 antibody after electroporation, but <em>how bright each cell is </em>(i.e. the amplitude of fluorescence). Looking at this instead will give me an idea of <em>how much</em> protein enters each cell. So. I went back to the old data, saw that the electroporation pre-sets <em>do</em> have an effect on fluorescence amplitude, and added a new gate to quantify the difference.</p>
<p>With this I could see that my previous best pre-set produced only 27% “hyper-positive” cells, compared with 93% hyper-positive cells in the new Best Electroporation Pre-set!</p>
<p>&nbsp;</p>
<p><a href="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/Image-1.png"><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/Image-1-1024x478.png" alt="Two side by side graphs entitled 'old &quot;best&quot; settings' and 'new best settings!' that display the number of cells versus the amplitude of Alexa-Fluor 488 flourescence. The right hand graph show that these settings produce more highly fluorescent cells." class="aligncenter wp-image-5093 size-large" width="780" height="364" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/Image-1-1024x478.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/Image-1-300x140.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/Image-1-768x359.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/Image-1-1536x717.png 1536w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/Image-1-2048x956.png 2048w" sizes="(max-width: 780px) 100vw, 780px" /></a></p>
<p>At the same time I can see that the three best pre-sets also produce the highest levels of cell death, but I think this is actually a good sign. If the electroporation has more of an impact on the cell membrane it makes sense that more protein can enter the cell, but also that more of them will die. And the level of cell death is still low enough that in the end there are more hyper-positive cells.</p>
<p>Hopefully this rethink will make my future editing attempts even more effective!</p>
<p>You can read the full details of what I did in <a href="https://zenodo.org/record/3990047#.Xz1cM0HTVaQ">my Zenodo report</a>.</p>
<p>&nbsp;</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/electroporation-a-second-look/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>Structure and X-ray Fragment screening of SARS-Cov-2 helicase (Nsp13)</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/structure-and-x-ray-fragment-screening-of-sars-cov-2-helicase-nsp13/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/structure-and-x-ray-fragment-screening-of-sars-cov-2-helicase-nsp13/#respond</comments>
		
		<dc:creator><![CDATA[Joseph Newman]]></dc:creator>
		<pubDate>Mon, 17 Aug 2020 18:53:46 +0000</pubDate>
				<category><![CDATA[COVID-19]]></category>
		<category><![CDATA[Joseph Newman]]></category>
		<category><![CDATA[SGC Univ. of Oxford]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=5084</guid>

					<description><![CDATA[My name is Joseph Newman and I have been working in Lab of Opher Gileadi at SGC Oxford since 2013 mainly on targeting DNA repair factors as synthetic lethal targets for early stage development of new cancer therapeutics. Like many scientists my normal laboratory based research activities were disrupted by the lockdown, and when Opher <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/structure-and-x-ray-fragment-screening-of-sars-cov-2-helicase-nsp13/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p>My name is Joseph Newman and I have been working in Lab of Opher Gileadi at SGC Oxford since 2013 mainly on targeting DNA repair factors as synthetic lethal targets for early stage development of new cancer therapeutics. Like many scientists my normal laboratory based research activities were disrupted by the lockdown, and when Opher suggested to work on targeting the SARS-Cov-2 helicase (a family that I have significant experience and interest in) I was keen to give it a try. The SARA-Cov-2 helicase is well conserved across and appears to be indispensable across all coronavirus species, and for this reason has been highlighted as one of the attractive targets for the development of anti-viral drugs(1, 2). The helicase is a Upf1 like SF1b helicase that unwinds both DNA and RNA in a 5’ to 3’ direction with a relaxed nucleotide specificity <em>in vitro</em>. Whilst its precise role in viral replication in not currently well-defined recent studies show it acts in concert with the replication-transcription complex (Nsp7/Nsp8<sub>2</sub>/Nsp12), possibly being involved in either disrupting downstream RNA secondary structures or template switching(3).</p>
<p>Previously crystal structures of Nsp13 have been solved for MERS-CoV(4) and SARS-Cov-1(5) to 3.0 Å and 2.8 Å respectively which indicated that the protein may be crystallizable, although the resolution of these structures is probably too low to reliably detect fragment binders in a X-ray Fragment screen. We began lab work on this project in mid-April and made quite rapid progress obtaining a new crystal form that is reproducible and diffracts routinely to around 2.0 Å. We have deposited the Nnsp13 structure in the pdb (6ZSL) and recently ran a full fragment screen with the help of XChem team at Diamond Light Source. The screen went about as well as could be expected producing around 50-60 hits from a screen of around 600 compounds with several compounds binding in both the nucleotide and RNA binding sites. I am still working on finalising the hits and the results should be available soon on the Diamond fragalysis user interface, but I thought I would show you an overview of what we found in the image below<a href="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/nsp13-DNA-atp.tif"><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/nsp13-DNA-atp.tif" alt="" class="alignnone size-medium wp-image-5085" width="1" height="1" /></a></p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/nsp13-DNA-atp.tif" alt="" class="alignnone size-medium wp-image-5085" width="1" height="1" /><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/nsp13-DNA-atp-300x120.jpg" alt="" class="alignnone wp-image-5088" width="1225" height="490" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/nsp13-DNA-atp-300x120.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/nsp13-DNA-atp-1024x411.jpg 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/08/nsp13-DNA-atp-768x308.jpg 768w" sizes="(max-width: 1225px) 100vw, 1225px" /></p>
<p><em>The structure of Nsp13 is shown on the left panel along with the expected positions of the Nucleotide and RNA which have been positioned into the enzyme based on structures of homologues. The right hand panel shows a subset of the fragment hits which can be seen to cluster around areas of potential interest for inhibition.</em></p>
<p>This project was a team effort and I want to thank Yuliana Yosaatmadja for help with the protein work and the whole XChem team at Diamond in particular Alice Douangamath for help in setting crystallization experiments and mounting all the crystals. If you are reading this and are interested in collaborating towards the development of Nsp13 inhibitors, then please get in touch.</p>
<p>For a detailed protocol of Nsp13 purification and crystallization please refer to my<a href="https://zenodo.org/record/3988536"> Zenodo post</a>.</p>
<p><strong>References</strong></p>
<ol>
<li>Habtemariam S, Nabavi SF, Banach M, Berindan-Neagoe I, Sarkar K, Sil PC, et al. Should We Try SARS-CoV-2 Helicase Inhibitors for COVID-19 Therapy? Arch Med Res. 2020.</li>
<li>Keum YS, Jeong YJ. Development of chemical inhibitors of the SARS coronavirus: viral helicase as a potential target. Biochem Pharmacol. 2012;84(10):1351-8.</li>
<li>Chen J, Malone B, Llewellyn E, Grasso M, Shelton PMM, Olinares PDB, et al. Structural basis for helicase-polymerase coupling in the SARS-CoV-2 replication-transcription complex. bioRxiv. 2020.</li>
<li>Hao W, Wojdyla JA, Zhao R, Han R, Das R, Zlatev I, et al. Crystal structure of Middle East respiratory syndrome coronavirus helicase. PLoS Pathog. 2017;13(6):e1006474.</li>
<li>Jia Z, Yan L, Ren Z, Wu L, Wang J, Guo J, et al. Delicate structural coordination of the Severe Acute Respiratory Syndrome coronavirus Nsp13 upon ATP hydrolysis. Nucleic Acids Res. 2019;47(12):6538-50.</li>
</ol>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/structure-and-x-ray-fragment-screening-of-sars-cov-2-helicase-nsp13/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>Generation of stable mammalian cell lines for the expression of proteins related to COVID-19 through random integration</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/generation-of-stable-mammalian-cell-lines-for-the-expression-of-proteins-related-to-covid-19-through-random-integration/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/generation-of-stable-mammalian-cell-lines-for-the-expression-of-proteins-related-to-covid-19-through-random-integration/#respond</comments>
		
		<dc:creator><![CDATA[Jong Fu]]></dc:creator>
		<pubDate>Fri, 24 Jul 2020 09:27:17 +0000</pubDate>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[COVID-19]]></category>
		<category><![CDATA[Jong Fu Wong]]></category>
		<category><![CDATA[SGC Univ. of Oxford]]></category>
		<category><![CDATA[Spike]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=5048</guid>

					<description><![CDATA[Background: Scientists centred around University of Oxford came together in the current COVID-19 pandemic to contribute to relevant studies collaboratively. Researchers with no prior experience in virology studies, myself included, had volunteered our time to contribute whichever way we can based on our field of expertise. The Biotech team of SGC Oxford, headed by Nicola <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/generation-of-stable-mammalian-cell-lines-for-the-expression-of-proteins-related-to-covid-19-through-random-integration/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p><strong>Background:</strong></p>
<p>Scientists centred around University of Oxford came together in the current COVID-19 pandemic to contribute to relevant studies collaboratively. Researchers with no prior experience in virology studies, myself included, had volunteered our time to contribute whichever way we can based on our field of expertise. The Biotech team of SGC Oxford, headed by Nicola Burgess-Brown had taken up the role of producing proteins used in serological assays, basic research to understand COVID-19 pathophysiology and hunt for treatment compounds. The initial production of these proteins was based on transient transfection of expression construct DNA into mammalian cells. This approach was laborious and consumed large amounts of plasmid DNA. Therefore, I volunteered to generate stable expression cell lines that will negate the constant need for transfection.</p>
<p><strong>Experimental design:</strong></p>
<p>I aimed to stably integrate the expression constructs into the genome of Expi293F human cells commonly used in large scale protein production. The expression constructs carried neomycin marker for selection in mammalian cells. Toxicity to the cells is not a concern in this case because the expressed proteins are secreted into the medium and do not build up in the cells. The expression constructs are randomly integrated into the genome mainly via non-homologous end-joining (NHEJ). An advantage of this approach is that molecular sub-cloning is not necessary.</p>
<p>In order to maximise the rate of correct integration of the expression constructs, restriction digestion was performed to cut once between the expression cassette and antibiotic resistance cassette (refer to cartoon in Figure 1). Without this step, the construct DNA will be broken at a random position during the integration step, potentially disrupting the expression cassette. I have also tested the effect of actively creating double strand breaks (DSBs) in the genome of the cells on the integration efficiency. More DSBs in the genome might increase the incidence of NHEJ and integration of our constructs. This was achieved by co-transfection of AAVS1 CRISPR/eSpCas9(1.1) construct (refer to cartoon in Figure 1). The eSpCas9(1.1)_No_FLAG_AAVS1_T2 construct was a gift from Yannick Doyon (Addgene plasmid # 79888 ; http://n2t.net/addgene:79888 ; RRID:Addgene_79888)</p>
<p><strong>Results:</strong></p>
<div id="attachment_5049" style="width: 1209px" class="wp-caption alignnone"><img aria-describedby="caption-attachment-5049" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/Figure-1.jpg" alt="" width="1199" height="661" class="size-full wp-image-5049" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/Figure-1.jpg 1199w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/Figure-1-300x165.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/Figure-1-1024x565.jpg 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/Figure-1-768x423.jpg 768w" sizes="(max-width: 1199px) 100vw, 1199px" /><p id="caption-attachment-5049" class="wp-caption-text">Cartoons of approaches to enhance desired stable integration of protein expression constructs.</p></div>
<div id="attachment_5050" style="width: 1108px" class="wp-caption alignnone"><img aria-describedby="caption-attachment-5050" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/Figure-2.jpg" alt="" width="1098" height="623" class="size-full wp-image-5050" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/Figure-2.jpg 1098w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/Figure-2-300x170.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/Figure-2-1024x581.jpg 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/Figure-2-768x436.jpg 768w" sizes="(max-width: 1098px) 100vw, 1098px" /><p id="caption-attachment-5050" class="wp-caption-text">Schematic of the workflow to generate stable cell lines</p></div>
<div id="attachment_5051" style="width: 557px" class="wp-caption alignnone"><img aria-describedby="caption-attachment-5051" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/Figure-3.jpg" alt="" width="547" height="579" class="size-full wp-image-5051" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/Figure-3.jpg 547w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/Figure-3-283x300.jpg 283w" sizes="(max-width: 547px) 100vw, 547px" /><p id="caption-attachment-5051" class="wp-caption-text">Comparison of the efficiency of correct stable integration under different conditions. Numbers of surviving cells were counted after G418 antibiotic selection and the expression of desired proteins was detected using Western Blot.</p></div>
<p><strong>Findings:</strong></p>
<ol>
<li>Linearisation of expression construct increased stable integration.</li>
<li>Induction of AAVS1 double strand break enhanced stable integration further.</li>
<li>The efficiency of stable integration is highly dependent on construct size. E.g. 12.3 kb construct is about 30 times less efficiently integrated than 8.2 kb construct.</li>
<li>More than 5-fold increase in purified proteins compared to same volume of transient transfected cells.</li>
</ol>
<p><strong> </strong><strong>In a nut shell:</strong></p>
<p>Stable cells that continuously produce and secrete high level of COVID-19-related proteins have been successfully generated. These cells are suitable for large scale culture and protein purification. These stable cells will help to streamline protein production for serological assays and other COVID-19-related studies.</p>
<p>For detailed protocol and information, please refer to my <a href="https://zenodo.org/record/3958921#.XxqpYFVKipo">Zenodo post</a>.</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/generation-of-stable-mammalian-cell-lines-for-the-expression-of-proteins-related-to-covid-19-through-random-integration/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>Evaluating the inhibition of ALK2 phosphorylation of SMAD1/5 by M4K lead compounds in DIPG patient-derived cells (SU-DIPG-IV, HSJD-DIPG-007, HSJD-DIPG-018 and SU-DIPG-XXI)</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/evaluating-the-inhibition-of-alk2-phosphorylation-of-smad1-5-by-m4k-lead-compounds-in-dipg-patient-derived-cells-su-dipg-iv-hsjd-dipg-007-hsjd-dipg-018-and-su-dipg-xxi/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/evaluating-the-inhibition-of-alk2-phosphorylation-of-smad1-5-by-m4k-lead-compounds-in-dipg-patient-derived-cells-su-dipg-iv-hsjd-dipg-007-hsjd-dipg-018-and-su-dipg-xxi/#respond</comments>
		
		<dc:creator><![CDATA[Jong Fu]]></dc:creator>
		<pubDate>Mon, 20 Jul 2020 13:57:12 +0000</pubDate>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Brain Tumours]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Jong Fu Wong]]></category>
		<category><![CDATA[SGC Univ. of Oxford]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=5001</guid>

					<description><![CDATA[Background: The binding potency of M4K compounds to ALK2 has been assayed in biochemical assay and cellular assays in HEK293 or C2C12 myoblast cell lines. However, the potency of ALK2 inhibition by M4K compounds has not determined directly in DIPG patient-derived cell lines. While no major deviation from existing assay data is expected, direct experimental <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/evaluating-the-inhibition-of-alk2-phosphorylation-of-smad1-5-by-m4k-lead-compounds-in-dipg-patient-derived-cells-su-dipg-iv-hsjd-dipg-007-hsjd-dipg-018-and-su-dipg-xxi/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p><strong>Background:</strong></p>
<p>The binding potency of M4K compounds to ALK2 has been assayed in biochemical assay and cellular assays in HEK293 or C2C12 myoblast cell lines. However, the potency of ALK2 inhibition by M4K compounds has not determined directly in DIPG patient-derived cell lines. While no major deviation from existing assay data is expected, direct experimental evidence is essential.</p>
<p><strong>Experimental design:</strong></p>
<p>DIPG cells will be trypsinized and resuspended in TSM-base medium without any growth factor for one-hour starvation. Subsequently, equal volume of TSM-base with 2X Activin A (200ng/mL) and M4K compounds or DMSO vehicle control will be added. After one-hour treatment, the cells will be pelleted and lysed in buffer with protease and phosphatase inhibitors for Western Blot analysis.</p>
<p><strong>Results:</strong></p>
<div id="attachment_5007" style="width: 558px" class="wp-caption alignnone"><img aria-describedby="caption-attachment-5007" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/Compound-structures.jpg" alt="" width="548" height="540" class="size-full wp-image-5007" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/Compound-structures.jpg 548w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/Compound-structures-300x296.jpg 300w" sizes="(max-width: 548px) 100vw, 548px" /><p id="caption-attachment-5007" class="wp-caption-text">Chemical structures of the compounds tested</p></div>
<div id="attachment_5002" style="width: 1290px" class="wp-caption alignnone"><img aria-describedby="caption-attachment-5002" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/SU-DIPG-IV.jpg" alt="" width="1280" height="720" class="size-full wp-image-5002" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/SU-DIPG-IV.jpg 1280w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/SU-DIPG-IV-300x169.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/SU-DIPG-IV-1024x576.jpg 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/SU-DIPG-IV-768x432.jpg 768w" sizes="(max-width: 1280px) 100vw, 1280px" /><p id="caption-attachment-5002" class="wp-caption-text">Figure 1. Inhibition of ALK2 phosphorylation of SMAD1/5 by M4K lead compounds in SU-DIPG-IV DIPG patient-derived cell line. Phosphorylated proteins were detected in Western Blot using specific antibodies.</p></div>
<div id="attachment_5003" style="width: 1290px" class="wp-caption alignnone"><img aria-describedby="caption-attachment-5003" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/SU-DIPG-007.jpg" alt="" width="1280" height="720" class="size-full wp-image-5003" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/SU-DIPG-007.jpg 1280w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/SU-DIPG-007-300x169.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/SU-DIPG-007-1024x576.jpg 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/SU-DIPG-007-768x432.jpg 768w" sizes="(max-width: 1280px) 100vw, 1280px" /><p id="caption-attachment-5003" class="wp-caption-text">Figure 2. Inhibition of ALK2 phosphorylation of SMAD1/5 by M4K lead compounds in HSJD-DIPG-007 DIPG patient-derived cell line. Phosphorylated proteins were detected in Western Blot using specific antibodies.</p></div>
<div id="attachment_5004" style="width: 1290px" class="wp-caption alignnone"><img aria-describedby="caption-attachment-5004" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/SU-DIPG-018.jpg" alt="" width="1280" height="720" class="size-full wp-image-5004" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/SU-DIPG-018.jpg 1280w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/SU-DIPG-018-300x169.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/SU-DIPG-018-1024x576.jpg 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/SU-DIPG-018-768x432.jpg 768w" sizes="(max-width: 1280px) 100vw, 1280px" /><p id="caption-attachment-5004" class="wp-caption-text">Figure 3. Inhibition of ALK2 phosphorylation of SMAD1/5 by M4K lead compounds in HSJD-DIPG-018 DIPG patient-derived cell line. Phosphorylated proteins were detected in Western Blot using specific antibodies.</p></div>
<div id="attachment_5005" style="width: 1290px" class="wp-caption alignnone"><img aria-describedby="caption-attachment-5005" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/SU-DIPG-XXI.jpg" alt="" width="1280" height="720" class="size-full wp-image-5005" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/SU-DIPG-XXI.jpg 1280w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/SU-DIPG-XXI-300x169.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/SU-DIPG-XXI-1024x576.jpg 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/07/SU-DIPG-XXI-768x432.jpg 768w" sizes="(max-width: 1280px) 100vw, 1280px" /><p id="caption-attachment-5005" class="wp-caption-text">Figure 4. Inhibition of ALK2 phosphorylation of SMAD1/5 by M4K lead compounds in SU-DIPG-XXI DIPG patient-derived cell line. Phosphorylated proteins were detected in Western Blot using specific antibodies.</p></div>
<p>&nbsp;</p>
<p><strong>Findings:</strong></p>
<p>1) All M4K lead compounds inhibited ALK2 kinase activity in DIPG cells.</p>
<p>2) M4K3078 appeared to be less potent even though its potency was highest when measured in biochemical assay and cellular target engagement assay in HEK293. Additional experiments are necessary to verify this result before hypothesis of the underlying reason can be developed.</p>
<p><strong>In a nut shell:</strong></p>
<p>On target ALK2 inhibition in DIPG patient-derived cell lines by M4K lead compounds had been successfully verified using Western Blot.</p>
<p>For detailed experimental information, please refer to my <a href="https://zenodo.org/record/3952506#.XxWiWVVKipo">Zenodo</a> post.</p>
<p>&nbsp;</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/evaluating-the-inhibition-of-alk2-phosphorylation-of-smad1-5-by-m4k-lead-compounds-in-dipg-patient-derived-cells-su-dipg-iv-hsjd-dipg-007-hsjd-dipg-018-and-su-dipg-xxi/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>Mass spec analysis of SMAD1 phosphorylation by ALK2 and ALK2 GS mutants in the presence of ACVR2.</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/mass-spec-analysis-of-smad1-phosphorylation-by-alk2-and-alk2-gs-mutants-in-the-presence-of-acvr2/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/mass-spec-analysis-of-smad1-phosphorylation-by-alk2-and-alk2-gs-mutants-in-the-presence-of-acvr2/#respond</comments>
		
		<dc:creator><![CDATA[Ellie Williams]]></dc:creator>
		<pubDate>Wed, 18 Dec 2019 12:10:55 +0000</pubDate>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Ellie Williams]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>
		<category><![CDATA[SGC Univ. of Oxford]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=3945</guid>

					<description><![CDATA[I decided to test a set of my ALK2 GS loop phosphomutants for activity. As I’ve written about previously, I needed to optimise the concentrations of receptors and decided that 60uM was the right concentration to use as showing activity with the mutants and also being workable in the assay. I set up three different <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/mass-spec-analysis-of-smad1-phosphorylation-by-alk2-and-alk2-gs-mutants-in-the-presence-of-acvr2/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p>I decided to test a set of my ALK2 GS loop phosphomutants for activity. As I’ve written about previously, I needed to optimise the concentrations of receptors and decided that 60uM was the right concentration to use as showing activity with the mutants and also being workable in the assay.</p>
<p>I set up three different ALK2 mutants alongside the wild type ALK2 to compare them and used ACVR2 as my necessary type II receptor for SMAD1 activation.</p>
<p>I took time points at 0, 1, 5, 15, 30, 45 min, 1h, 1h15, 1h30, 1h45, 2h. Taking 1ul of sample and mixing it with 60ul of mass spec buffer (water + 0.1% formic acid)</p>
<p>I ran these on the mass spec to check for and monitor activity on all components but regrettably ended up seeing zero activity on SMAD1 even after 2h in the wildtype positive control. I did see phosphorylation on the type one receptors but nothing on the SMAD1 which suggested to me that something had gone horribly wrong with the type II receptor and led to it being unable to facilitate SMAD1 activation (as has been previously established to be necessary).</p>
<p>This was, to put it mildly, very frustrating.</p>
<p>So I went and purified a fresh stock of ACVR2 and did a small scale test using just the wild type to see if the positive control worked. This time it did so I set up a larger scale experiment using all the constructs we’ve managed to purify so far (which is all of those created so far except c107, c111 and c112).</p>
<p>This time we saw phosphorylation on SMAD and after 2h we could see strong signal and still make out the differences between the activity of the constructs and so further analysis was done on the 2h time point.</p>
<p>Differences between the different mutants stood out instantly – none of them increased the activity compared to the wild type but there were distinct effects on both the SMAD1 and ALK2 auto-phosphorylation patterns.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/12/MS_Traces.png" alt="" width="1478" height="1149" class="alignnone size-full wp-image-3946" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/12/MS_Traces.png 1478w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/12/MS_Traces-300x233.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/12/MS_Traces-768x597.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/12/MS_Traces-1024x796.png 1024w" sizes="(max-width: 1478px) 100vw, 1478px" /></p>
<p><em>Summary figures showing the phosphorylation of SMAD1 and ALK2 using various ALK2 GS loop mutants. The top row on both columns shows the wild type positive control for comparison. The constructs are then ranked from least to most effected moving down the first column (with matching SMAD1 and ALK2 traces) and then down the second column.</em></p>
<p>The last four were the most extreme in terms of knocking out SMAD phosphorylation with the fourth leading to very low levels of phosphorylation and last three basically completely stopping it. When we look at the mutations these four constructs contain we can see a theme in common.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/12/top_4_hits.png" alt="" width="436" height="60" class="alignnone size-full wp-image-3947" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/12/top_4_hits.png 436w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/12/top_4_hits-300x41.png 300w" sizes="(max-width: 436px) 100vw, 436px" /></p>
<p><em>Figure showing the mutations contained within the four most detrimental mutations.</em></p>
<p>The three most detrimental constructs (c109, c106 and c113) all contained a S190A mutation with the fourth (c105) containing a T189A mutation just prior to the S190 residue. This suggests that S190 has a key role in facilitating SMAD1 phosphorylation.</p>
<p>Next thing to test is the constructs I’m missing (they need expressing and purifying) and to repeat this experiment before also running the results on a western blot for confirmation.</p>
<p>If you&#8217;d like more details then please do visit <a href="https://doi.org/10.5281/zenodo.3582748">over on Zenodo</a>.</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/mass-spec-analysis-of-smad1-phosphorylation-by-alk2-and-alk2-gs-mutants-in-the-presence-of-acvr2/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>Purification of more ALK2 GS loop mutants and fresh ACVR2.</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/purification-of-more-alk2-gs-loop-mutants-and-fresh-acvr2/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/purification-of-more-alk2-gs-loop-mutants-and-fresh-acvr2/#respond</comments>
		
		<dc:creator><![CDATA[Ellie Williams]]></dc:creator>
		<pubDate>Thu, 14 Nov 2019 13:38:31 +0000</pubDate>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Ellie Williams]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>
		<category><![CDATA[SGC Univ. of Oxford]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=3773</guid>

					<description><![CDATA[So I’m still ‘trying to catch them all’ as the young folks say (!) and building up my collection of purified GS loop mutations so that I can test them all together and start building up a picture of what phosphorylation sites are necessary for SMAD1 activation and which are not. This time around I <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/purification-of-more-alk2-gs-loop-mutants-and-fresh-acvr2/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p>So I’m still ‘trying to catch them all’ as the young folks say (!) and building up my collection of purified GS loop mutations so that I can test them all together and start building up a picture of what phosphorylation sites are necessary for SMAD1 activation and which are not.</p>
<p>This time around I was aided ably by Jong Fu who was trying his hand at protein purification for the first time. I think (at least I hope!) you’ll agree he did a pretty good job!</p>
<p>The two mutants we were purifying this time were c108 and c133 which contain one and three mutations respectively. We were also purifying a fresh set of ACVR2A as the last test I did showed no SMAD1 activity (I’m going to write about this in the next blog post along with the follow-up experiment I did with this purification of ACVR2) but did show ACVR1 autophosphorylation suggesting the type I was active but something had happened to the type II protein to inactivate it.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/gsloop_mutations-1024x425.png" alt="" width="780" height="324" class="alignnone wp-image-3300 size-large" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/gsloop_mutations-1024x425.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/gsloop_mutations-300x124.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/gsloop_mutations-768x318.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/gsloop_mutations.png 1165w" sizes="(max-width: 780px) 100vw, 780px" /></p>
<p><em>Fig. 1 – Showing the phospho-mutations introduced into the GS domain of ALK2</em>.</p>
<p>This time around we also tried a slightly different method of purification using Talon resin bound to cobalt ions rather than using nickle – partly this was because our nickle resin was in a sorry state after many regenerations and partly this was because we’d got access to some cobalt talon resin from the membrane protein group who, because they require super clean resin for each purification, only use it once and then pass it onto other groups for use with higher expressing proteins.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/11/Talon_purification-1024x584.png" alt="" width="780" height="445" class="alignnone wp-image-3775 size-large" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/11/Talon_purification-1024x584.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/11/Talon_purification-300x171.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/11/Talon_purification-768x438.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/11/Talon_purification.png 1188w" sizes="(max-width: 780px) 100vw, 780px" /></p>
<p><em>Fig. 2. SDS-PAGE gel of Talon column purification for all constructs showing flowthrough, washes and elutions. Boxed fractions have Tev added and taken on to gel filtration purification.</em></p>
<p>As you can see the talon resin worked out really well and we ended up with a really decent purification from it. The boxed fractions we cleaved overnight with Tev protease to remove the His tag and then concentrated them down to &lt;5ml to purify on a GF75 size exclusion column.</p>
<p>This is shown in the gel below – in this case the boxed fractions are the ones chosen to be taken on for use in experiments – the other fractions, even though they contain the protein of interest, weren’t part of the main gel filtration peak and thus probably represent aggregates of the protein that we don’t want.</p>
<p>We then concentrated down the samples and froze them ready for future assays.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/11/GF_purification.png" alt="" width="908" height="796" class="alignnone wp-image-3774 size-full" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/11/GF_purification.png 908w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/11/GF_purification-300x263.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/11/GF_purification-768x673.png 768w" sizes="(max-width: 908px) 100vw, 908px" /></p>
<p><em>Fig. 3. SDS-PAGE gels (left) and akta traces (right) of the purification of ACVR1Z-c108, ACVR1Z-c113 and ACVR2A-c46 by size exclusion chromatography.</em></p>
<p>&nbsp;</p>
<p>For full details on the purification you can <a href="https://doi.org/10.5281/zenodo.3541824">head over to Zenodo and download the files</a>.</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/purification-of-more-alk2-gs-loop-mutants-and-fresh-acvr2/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>Purification of ALK2 and SUFU</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/purification-of-alk2-and-sufu/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/purification-of-alk2-and-sufu/#respond</comments>
		
		<dc:creator><![CDATA[Ellie Williams]]></dc:creator>
		<pubDate>Fri, 18 Oct 2019 15:28:02 +0000</pubDate>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Ellie Williams]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>
		<category><![CDATA[SGC Univ. of Oxford]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=3709</guid>

					<description><![CDATA[I’ve been working recently on purifying some proteins for use in future experiments and wanted to explain a little bit about what I need each construct for as they’re all for different things. Purification, as always, is a time consuming business but vital for any biophysical experiment that I do given all my work is <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/purification-of-alk2-and-sufu/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p>I’ve been working recently on purifying some proteins for use in future experiments and wanted to explain a little bit about what I need each construct for as they’re all for different things. Purification, as always, is a time consuming business but vital for any biophysical experiment that I do given all my work is in vitro rather than in a cellular context. It’s important to know what you’re putting in any given experiment so that you can interpret the results accurately!</p>
<p>The three proteins I was purifying were:</p>
<p><strong>ALK2</strong> – a short version of the construct containing the Kinase domain only, no GS loop, and a Q207D mutation that makes it consituativly active. We use this construct regularly for co-crystallisation studies and in this case I need some of it for growing some crystals for some mini-fragment screening later on.</p>
<p><strong>ALK2 GS loop mutant</strong> – as described previously, we’ve mutated out various different phosphorylation sites in different combinations within the GS loop of ALK2 to see how this effects the activity. Overall we have 12 different version of this protein, this is one of them. I want to purify several so that I can compare them directly in an experiment all at once.</p>
<p><strong>SUFU</strong> – so this is a slightly different project harking back to some stuff I did a few years ago so bear with me. In the BMP signalling pathway there is a protein called Endofin (ZFYVE16) which is involved in endosomal signalling/trafficking and may have some role in membrane localisation of the receptors and/or their substrates. In the TGFβ pathway the equivalent protein is called SARA and I managed to crystallise a previously unknown domain (a DUF domain or ‘domain of unknown function’) which is in the <a href="https://www.rcsb.org/structure/4BKW">pdb as 4BKW</a>. This domain had three sub-folds, two of which were known and one of which was novel. The two familiar folds looked a lot like SUFU which is part of the Hedgehog signalling pathway although there is practically no sequence similarity. It is known that Endofin and SARA bind to the protein TOM1 which is also thought to be involved in trafficking. To confirm whether there is a structural or sequence basis to the binding, one test would be to see if TOM1 also associates with SUFU – given the sequence differences between SUFU and SARA it is highly unlikely that TOM1 will bind but it’s worth testing just in case. I obtained a plasmid for an E.coli expressible version of SUFU from <a href="https://www.ndm.ox.ac.uk/principal-investigators/researcher/christian-siebold">Christian Seibold</a> and a large scale culture was grown up by Charlotte Manning, our new lab assistant.</p>
<p>I’m happy to say that all constructs expressed well and are now safely frozen at -80°C (GS loop AKL2 mutant and SUFU) or used to set up in crystal plates for mini-fragment screening which I’ll do a separate post about soon.</p>
<p>I’ve also manged to grow up a large scale culture of TOM1 (6L of cells) which I’ll set about purifying probably next week so I can then test whether it binds to SUFU.</p>
<p>&nbsp;</p>
<p>You can find all the experimental details<a href="https://doi.org/10.5281/zenodo.3507756"> over on Zenodo.</a></p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/purification-of-alk2-and-sufu/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>Deposition of ALK2 with M4K2009</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/deposition-of-alk2-with-m4k2009/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/deposition-of-alk2-with-m4k2009/#respond</comments>
		
		<dc:creator><![CDATA[Ellie Williams]]></dc:creator>
		<pubDate>Thu, 03 Oct 2019 16:14:59 +0000</pubDate>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Ellie Williams]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>
		<category><![CDATA[SGC Univ. of Oxford]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=3643</guid>

					<description><![CDATA[Hello folks, another day another deposition. That makes it sound horribly commonplace but actually Ros put in immense effort into crystallising and solving these proteins and it’s my job just to add the final polish (fix those last stubborn geometry errors), give it a once over to make sure there’s nothing obviously wrong, and whip <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/deposition-of-alk2-with-m4k2009/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p>Hello folks, another day another deposition. That makes it sound horribly commonplace but actually Ros put in immense effort into crystallising and solving these proteins and it’s my job just to add the final polish (fix those last stubborn geometry errors), give it a once over to make sure there’s nothing obviously wrong, and whip it into shape to put into the PDB.</p>
<p>This one is another of the M4K structures, this time with M4K2009 which is one of our most promising candidates as shown by Jong Fu and colleagues and is one of the ones being taken forward into more tests. There were two molecules of ALK2 in the asymmetric unit</p>
<p>&nbsp;</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/10/Picture1-1024x587.jpg" alt="" width="780" height="447" class="alignnone size-large wp-image-3645" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/10/Picture1-1024x587.jpg 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/10/Picture1-300x172.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/10/Picture1-768x440.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/10/Picture1.jpg 1183w" sizes="(max-width: 780px) 100vw, 780px" /></p>
<p>A: Overall structure of ALK2 bound to M4K2009. B: Details of the molecule M4K2009. C: Closeup of the binding pocket showing water mediated H bonds between molecule and side-chains and directly between M4K2009 and the protein hinge region.</p>
<p>The structure is what we expect in a sense, the overall structure of the kinase domain is very similar to usual. Of most interest is the position of water molecules in the binding pocket that help mediate contacts between the protein and the ligand – these help improve the stability and the fit of the compound in the binding site</p>
<p>One other thing of note was the presence of two additional ligand molecules bound to the top of one of the molecules in the asymmetric unit. It is unlikely that these are physiologically relevant and are most likely there as a crystallographic artefact as they are making good contact with chains from the neighbouring unit cell in the crystal.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/10/odd_ligands-292x300.png" alt="" width="292" height="300" class="alignnone size-medium wp-image-3644" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/10/odd_ligands-292x300.png 292w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/10/odd_ligands.png 736w" sizes="(max-width: 292px) 100vw, 292px" /></p>
<p>Structure of one copy of ALK2 in the asymmetric unit showing two additional M4K2009 compounds bound non-physiologically at the crystallographic interface.</p>
<p>I used the standard wwPDB One Dep system and this structure has been assigned the code: 6SZM and should be released into the wild (that is, once it has been curated by the people at the pdb) asap!</p>
<p>The files for the deposition can be found<a href="https://doi.org/10.5281/zenodo.3471671"> over on Zenodo.</a></p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/deposition-of-alk2-with-m4k2009/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>Deposition of structure of ALK2 with M4K2117</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/deposition-of-structure-of-alk2-with-m4k2117/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/deposition-of-structure-of-alk2-with-m4k2117/#respond</comments>
		
		<dc:creator><![CDATA[Ellie Williams]]></dc:creator>
		<pubDate>Wed, 25 Sep 2019 14:19:55 +0000</pubDate>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Ellie Williams]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>
		<category><![CDATA[SGC Univ. of Oxford]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=3575</guid>

					<description><![CDATA[One of the big aims of the M4K project was to look at the structure of potentially interesting molecules bound to the protein ALK2 and use that to try and design better molecules that might be useful in the treatment of the rare childhood brain cancer, DIPG. Ros did a lot of work on this <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/deposition-of-structure-of-alk2-with-m4k2117/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p>One of the big aims of the M4K project was to look at the structure of potentially interesting molecules bound to the protein ALK2 and use that to try and design better molecules that might be useful in the treatment of the rare childhood brain cancer, DIPG. Ros did a lot of work on this project and documented a lot of it on her blog. Although currently this project has moved beyond the structure stage, there are still quite a lot of structures that need the final step of tidying up and then depositing into the PDB.</p>
<p>In the last couple of weeks among the other things I’ve needed to do, I’ve also managed to deposit the first of these structure – ALK2 bound to M4K2117.</p>
<p>This compound showed good potency as has been tested by Jong Fu and is one of the more promising compounds (of which there are around 5) with regards to both potency and selectivity for taking forward to the next stage of the process for more in-depth and extensive testing in both cell and animal models.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/09/M4K2117-1024x677.jpg" alt="" width="780" height="516" class="alignnone wp-image-3577 size-large" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/09/M4K2117-1024x677.jpg 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/09/M4K2117-300x198.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/09/M4K2117-768x508.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/09/M4K2117.jpg 1368w" sizes="(max-width: 780px) 100vw, 780px" /></p>
<p><em>A: The overall structure of ALK2 bound to M4K2117. B: The structure of the ligand M4K2117. C: A close up of the binding pocket of ALK2 bound to M4K2117 showing a standard H bond between the Nitrogen on the central ring and the backbone hinge region as well as a water mediated H bond between a methoxy group on the compound and a side chain at the back of the pocket of ALK2.</em></p>
<p>The final tweaks to the structures were made using Phenix Refine and Coot while I used the standard wwPDB One Dep system and this structure has been assigned the code: <a href="https://www.rcsb.org/structure/6SRH">6SRH</a></p>
<p>The files for the deposition can be found <a href="https://doi.org/10.5281/zenodo.3460724">over on Zenodo</a> and of course you can download the <a href="https://www.rcsb.org/structure/6SRH">PDB from the PDB website</a>.</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/deposition-of-structure-of-alk2-with-m4k2117/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>More GS loop mutations – purification of ALK2 proteins.</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/more-gs-loop-mutations-purification-of-alk2-proteins/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/more-gs-loop-mutations-purification-of-alk2-proteins/#respond</comments>
		
		<dc:creator><![CDATA[Ellie Williams]]></dc:creator>
		<pubDate>Fri, 20 Sep 2019 13:18:22 +0000</pubDate>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Ellie Williams]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>
		<category><![CDATA[SGC Univ. of Oxford]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=3527</guid>

					<description><![CDATA[I’ve been getting my stocks in order recently and catching up on some other work which has been both busy but not the most thrilling in terms of experimental results. As such I’ve not had much time to write up a blog post so I’ll try and get back on top of that although it’s <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/more-gs-loop-mutations-purification-of-alk2-proteins/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p>I’ve been getting my stocks in order recently and catching up on some other work which has been both busy but not the most thrilling in terms of experimental results. As such I’ve not had much time to write up a blog post so I’ll try and get back on top of that although it’s not likely to be that world shattering.</p>
<p>I’m still trying to purify versions of ALK2 where various phosphorylation sites at the top of the molecule have been mutated out so that we can see what influence each of the phospho-sites has on the overall activity of ALK2 and hopefully from that work out which sites are essential for ALK2’s function and which are merely ‘nice to haves’ or even not necessary at all. This will hopefully then combine with our study of the mutant version of ALK2 to try and see whether the mutation that causes FOP is interfering with this phosphorylation activation. Anyway – that’s the ultimate goal, in the mean time I’ve been slowly purifying more constructs so that I can do a big experiment comparing all the different constructs at the same time. The main reason I want to do them all at once is to try and minimise the variability that can occur between experimental runs despite best efforts to make everything identical. If I’m actually using the same stock of partner proteins and ATP and metal ions and incubating them for exactly the same time it makes the comparisons between the mutants easier and less likely to contain artefact errors.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/gsloop_mutations-1024x425.png" alt="" width="780" height="324" class="alignnone size-large wp-image-3300" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/gsloop_mutations-1024x425.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/gsloop_mutations-300x124.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/gsloop_mutations-768x318.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/gsloop_mutations.png 1165w" sizes="(max-width: 780px) 100vw, 780px" /></p>
<p><em>Figure 1: The different ACVR1 (ALK2) GS loop phosphomutants generated and their associated construct id&#8217;s. The top row shows the sequence of the wild type protein. Mutations are shown in red.</em></p>
<p>So – with that I’ve been stockpiling the various mutants as they come through the expression pipeline at the SGC. This time around I’ve been purifying three more constructs – ACVR1-105, 106 and 109 (constructs 107 and 108 and beyond are still to be sorted). <em>(Just a quick reminder that ALK2 and ACVR1 are different ways of referring to the same thing, technically ALK2 is the protein and ACVR1 is the gene)</em></p>
<p>I had a bit of a problem with some of them – I’m not sure why but they seem to require more Nickle resin than usual to purify compared to the wild type construct – I’m not sure if that’s because the mutation makes it harder for the tag to contact the resin or if it is because there is more background contamination than usual in this particular line of cells or whether it’s something else. Luckily they can be rescued through re-incubating the sample with more nickle to pull out the protein and continuing on as usual. This is, while annoying, not a complete loss so while I’m keen to find out why it’s behaving differently it’s not ruined my purification prep.</p>
<p>There are still a couple of constructs left to purify before I run the big experiment and there are a few other small optimisations I need to do as well but I’m getting closer to really getting to the bottom of this question.</p>
<p>If you want to read more about the exact purification process then please have a look over on <a href="https://doi.org/10.5281/zenodo.3453086">Zenodo</a>.</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/more-gs-loop-mutations-purification-of-alk2-proteins/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
	</channel>
</rss>
