<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>Fibrodysplasia ossificans progressiva &#8211; openlabnotebooks.org</title>
	<atom:link href="https://thesgc.github.io/static-openlabnotebooks/category/disease-foundation/fop/feed/" rel="self" type="application/rss+xml" />
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<description>A growing team of groundbreaking scientists around the world are now sharing their lab notebooks online</description>
	<lastBuildDate>Mon, 19 Aug 2019 08:46:42 +0000</lastBuildDate>
	<language>en-GB</language>
	<sy:updatePeriod>hourly</sy:updatePeriod>
	<sy:updateFrequency>1</sy:updateFrequency>
	<generator>https://wordpress.org/?v=4.9.10</generator>

<image>
	<url>https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/cropped-lab-jar02-32x32.png</url>
	<title>Fibrodysplasia ossificans progressiva &#8211; openlabnotebooks.org</title>
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<width>32</width>
	<height>32</height>
</image> 
	<item>
		<title>Purification of GS loop mutants of ALK2</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/purification-of-gs-loop-mutants-of-alk2/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/purification-of-gs-loop-mutants-of-alk2/#respond</comments>
		<pubDate>Fri, 09 Aug 2019 10:48:22 +0000</pubDate>
		<dc:creator><![CDATA[Ellie Williams]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Ellie Williams]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>
		<category><![CDATA[SGC Univ. of Oxford]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=3298</guid>
		<description><![CDATA[&#8230;and analysing their activity! Following on from the work I talked about here I’ve been able to start looking at the influence the different phosphorylation sites have over the ability of ALK2 to activate SMAD1. Just coming through the SGC expression pipeline are several constructs containing mutations in various combinations of the sites known to <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/purification-of-gs-loop-mutants-of-alk2/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>&#8230;and analysing their activity!</p>
<p>Following on from the work I talked about <a href="https://thesgc.github.io/static-openlabnotebooks/phosphorylation-over-time-and-phosphomapping-experiments-to-investigate-the-influence-of-alk2-mutations-on-activity/">here</a> I’ve been able to start looking at the influence the different phosphorylation sites have over the ability of ALK2 to activate SMAD1.</p>
<p>Just coming through the SGC expression pipeline are several constructs containing mutations in various combinations of the sites known to be phosphorylated in and around the ‘GS loop’ of ALK2. This will prevent phosphorylation on that site and the change that makes to the activity of ALK2 will tell us something about how that site influences ALK2 function. These are shown in the figure below. Here they’re all labelled as ACVR1 due to our internal system using the gene name rather than the protein name.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/gsloop_mutations-1024x425.png" alt="" width="780" height="324" class="alignnone size-large wp-image-3300" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/gsloop_mutations-1024x425.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/gsloop_mutations-300x124.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/gsloop_mutations-768x318.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/gsloop_mutations.png 1165w" sizes="(max-width: 780px) 100vw, 780px" /></p>
<p><em>A list of ALK2 constructs containing mutations in the GS loop region of the protein. The mutations are from either Ser or Thr to Ala to knock out phosphorylation on that site.</em></p>
<p>One of the key findings from my previous experiments was that there seemed to be some sites that needed to be phosphorylated for maximal activation of SMAD – for example when a kinase dead type II binding partner was used some sites were not phosphorylated on ALK2 and this also corresponded to a reduction in SMAD1 phosphorylation. These mutants should help me explore which sites are vital for ALK2 activity and ultimately help me understand why the FOP mutation causes increased activity.</p>
<p>Anyway… now that these have been cloned into a baculo expression vector I can start actually purifying the protein and doing some experiments with them.</p>
<p>I had 1l of c103 cells and c104 cells which I purified with fairly high yields. I also had 1l of c105 cells but something went really wrong with the purification and I ended up not having enough to do any actual experiments with. I arranged to get some more c105 cells and 1l c106 cells but while those were being grown I went ahead and did a small scale experiment looking at how c103 and c104 behaved in comparison to the wild type ALK2.</p>
<p>I used my standard assay for monitoring SMAD1 phosphorylation. I incubated SMAD1 with ALK2 and with a type II receptor (in this case ACVR2 and BMPR2) at high enough concentrations to allow receptor association under normal circumstances. I then added activating quantities of ATP along with DTT, Mg2+ and Mn2+ ions necessary for activity and took 1ul samples at regular time intervals up to 30 minutes. The samples were quenched in MS buffer (which contains enough acid to stop any phosphorylation by a kinase) and at the end the results were run on the mass spec machine. Full details of the experimental protocol and the purification can be found over on Zenodo.</p>
<p>The results were fairly interesting but without other experiments remain somewhat inconclusive.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/SMAD1_P-1024x274.png" alt="" width="780" height="209" class="alignnone size-large wp-image-3301" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/SMAD1_P-1024x274.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/SMAD1_P-300x80.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/SMAD1_P-768x205.png 768w" sizes="(max-width: 780px) 100vw, 780px" /></p>
<p><em>Graphs showing the mass peaks detected by mass spec for SMAD1 in its unphosphorylated state and in various states of phosphorylation at +80 Da intervals at both 0 min and 25min for ALK2 wild type, ALK2 c103 and ALK2 c104 mutants.</em></p>
<p>What I can see is that compared to wildtype ALK2, it looks like there is much reduced phosphorylation on SMAD1 using the phosphomutants over 25 minutes.</p>
<p>I also looked at the phosphorylation of ALK2 in this experiment which can be seen in the figure below. This shows that there is a reduction of one occupied phosphorylation site compared to WT which is to be expected given one site has been removed by mutation however given we know from other experiments that there are up to 8 possible phosphorylation sites it’s not clear if the mutation is influencing which of these sites then is occupied.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/ALK2_P-1024x265.png" alt="" width="780" height="202" class="alignnone size-large wp-image-3299" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/ALK2_P-1024x265.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/ALK2_P-300x78.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/ALK2_P-768x199.png 768w" sizes="(max-width: 780px) 100vw, 780px" /></p>
<p><em>Graphs showing the mass peaks detected by mass spec for ALK2 in it’s unphosphorylated state and in various states of phosphorylation at +80 Da intervals at both 0min and 25min for ALK2 wild type, ALK2 c103 and ALK2 c104 mutants.</em></p>
<p><em> </em></p>
<p>There are questions unanswered however, such as do the mutations reduce the affinity between the type I and type II proteins such that a higher concentration might be needed to induce association (given that we already need to artificially induce their association via high concentrations to make up for the lack of membrane co-localisation that would occur in the cell) and if left for longer, would they show some activity rather than practically nothing? This suggests my next line of experimental inquiry which will be to alter the concentration of proteins in my experimental setup to see if increasing the concentrations of ALK2 and ACVR2 make any difference to the results in comparison to a wild type control. I’ll also let the experiment run for longer to see if this gives any indication as to whether the activity has been stopped or merely reduced. Increasing the time will also help pick up any differences in ALK2 phosphorylation as it may allow more sites to become occupied and thus pick up any limitations that the mutation is putting on that aside from the removal of one site by the mutation its self.</p>
<p>I’ll also think about looking at SMAD1 phosphorylation by western blot in addition to mass spec as we have a good antibody for looking at specifically phosphorylated SMAD1 which will show up very small amounts of phosphorylated SMAD1 that the mass spec might miss.</p>
<p>&nbsp;</p>
<p>For details on the experiments you can find it over on <a href="https://doi.org/10.5281/zenodo.3364406">Zenodo here</a>:</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/purification-of-gs-loop-mutants-of-alk2/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Selectivity against, and action of, TNIK kinase in relation to ALK2.</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/selectivity-against-and-action-of-tnik-kinase-in-relation-to-alk2/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/selectivity-against-and-action-of-tnik-kinase-in-relation-to-alk2/#respond</comments>
		<pubDate>Wed, 26 Jun 2019 15:58:08 +0000</pubDate>
		<dc:creator><![CDATA[Ellie Williams]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Ellie Williams]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>
		<category><![CDATA[SGC Univ. of Oxford]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=3015</guid>
		<description><![CDATA[Sorry I’ve not written for a while. I had some annual leave to use up and then I’ve been helping out with a residential EMBO course that was being co-organised by one of the group leaders in my department. The course was an introduction to a wide variety of biochemical and biophysical techniques. I was <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/selectivity-against-and-action-of-tnik-kinase-in-relation-to-alk2/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Sorry I’ve not written for a while. I had some annual leave to use up and then I’ve been helping out with a residential EMBO course that was being co-organised by one of the group leaders in my department. The course was an introduction to a wide variety of biochemical and biophysical techniques. I was helping out with a purification comparison between nickel resin, talon (cobalt) resin and streptactin resin and talking a bit about thermal shift assays.</p>
<p>Anyway, on with the science!</p>
<p><strong>TNIK Selectivity:</strong></p>
<p>TNIK is a protein kinase that Ros has previously talked about as being a potential off target hit for many of the M4K compounds along with DDR1. It is highly expressed in the brain which makes it of relevance to drug discovery for DIPG. Although its function is not entirely clear we know over-expression leads to cell rounding and it has been linked to some cell signalling effects in certain cancers. What makes it of most interest to us is that it seems to phosphorylate SMAD1 in an inhibitory position away from the tail where ALK2 phosphorylates SMAD1. Theoretically phosphorylation of SMAD1 by TNIK should inhibit phosphorylation elsewhere on SMAD1 by ALK2. This might make TNIK an indirect inhibitor of ALK2 function and thus we don’t want any drug we design against ALK2 to also hit TNIK.</p>
<p>I was working with a student, Mira Rothweiler, who purified TNIK that we then used in further experiments. Details for the purification can be found on <a href="https://doi.org/10.5281/zenodo.3257646">Zenodo</a> along with other details of the experiments done.</p>
<p>I have used Tm shift analysis (which is where you look at the change in melting point of a protein caused by the binding of a ligand, the tighter it binds the more it stabilises the protein and thus the larger the shift) to look at the binding of TNIK to all the M4K compounds and compared it to the shifts seen in ALK2. I plotted this data against each other with ALK2 shifts along the x-axis and the TNIK shifts along the y-axis. When compared against a 1:1 line, anything below the line is likely to be more selective for ALK2, anything above the line is likely to be selective for TNIK and anything close to the line shows minimal selectivity.</p>
<p>The data shows that while there are a small handful of M4K compounds that do hit TNIK, the vast majority seem to have a relatively low effect on TNIK and a high effect on ALK2 suggesting they are selective. One of the most promising M4K targets (M4K2009) gives a 14.1 °C shift for ALK2 but only 1.8 °C for TNIK. This is really good news.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/06/Tm_shifts_TNIK-816x1024.png" alt="" width="780" height="979" class="alignnone size-large wp-image-3017" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/06/Tm_shifts_TNIK-816x1024.png 816w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/06/Tm_shifts_TNIK-239x300.png 239w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/06/Tm_shifts_TNIK-768x964.png 768w" sizes="(max-width: 780px) 100vw, 780px" /></p>
<p><em>Tm shift results for ALK2 vs. TNIK for all M4K compounds. Top panel: plates K115 and K116. Bottom panel: plates K117 and K118.<br />
</em></p>
<p><strong>TNIK activity:</strong></p>
<p>While it has been reported that TNIK phosphorylates SMAD1 in a non-standard location in vivo, I wanted to see if I could get the same effect in vitro in my own experiments. I mixed SMAD1 with TNIK in different concentrations and for different lengths of time and took samples to run on mass spec to see whether SMAD1 was getting phosphorylated. My results showed that with concentrations of TNIK of 10mM and over after just 10 minutes I was starting to see phosphorylation on SMAD1. However this doesn’t tell me where that phosphorylation is happening.</p>
<p>I used another mass spec technique called ‘phosphomapping’ to identify where the phosphate was being added. Dr. Rod Chalk helped me with this for which I’m very thankful. This technique involved chopping the protein up into known chunks using different proteases that cut the protein at specific predictable places. We used five different proteases on different samples from the same incubation that all cut in different locations and then with each of the samples we ran half of it on the mass spec as was, and with the other half passed it over a Ti/Zr column to enrich the phosphopeptides within it and then ran it on the mass spec. The peptides detected are then run against a database of expected peptides created from knowing both the sequence of the protein and where the protease would have cut and then allowing for the possibility of modifications such as phosphorylation.</p>
<p>From this we detected a number of different phosphorylation sites on both SMAD1 and TNIK which are summarised in the diagram below.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/06/phosphomapping-1024x703.png" alt="" width="780" height="535" class="alignnone size-large wp-image-3016" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/06/phosphomapping-1024x703.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/06/phosphomapping-300x206.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/06/phosphomapping-768x527.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/06/phosphomapping.png 1314w" sizes="(max-width: 780px) 100vw, 780px" /></p>
<p><em>Summary of identified peptides showing overall coverage (red), definitive phosphorylation (blue) and ambiguous phosphorylation (orange). Consensus site for TNIK phosphorylation of SMAD1 from Wang et. al. 2016 shown boxed in blue.</em></p>
<p>&nbsp;</p>
<p>A number of sites were identified on TNIK which must be due to auto-phosphorylation. Three sites were identified on SMAD1 definitively and a further three ambiguously.</p>
<ul>
<li>Three of these sites (one firm and two ambiguous &#8211; PI<strong>SS</strong>V<strong>S</strong>) are known sites of phosphorylation by ALK2 however we know that ALK2 cannot phosphorylate them alone and requires a type II receptor for this to happen – it seems that under in-vitro conditions TNIK is not under the same restriction.</li>
<li>One of these sites (FE<strong>T</strong>VY) is not known to be involved in SMAD1 inhibition but it cannot be ruled out yet and may bear further investigation</li>
<li>Two of these sites (one firm and one ambiguous – RN<strong>ST</strong>IE) are in the location suggested by Wang et. al. (2016) as being the consensus sequence for TNIK phosphorylation of SMAD1.</li>
</ul>
<p>&nbsp;</p>
<p>The next thing to check will be if SMAD phosphorylated in the ‘inhibitory’ position stops ALK2 from phosphorylating SMAD in the ‘normal’ position.</p>
<p>You can find more detail on experimental work <a href="https://doi.org/10.5281/zenodo.3257646">over on Zenodo</a>.</p>
<p>&nbsp;</p>
<p>Reference:</p>
<p>Identification of Phosphorylation Consensus Sequences and Endogenous Neuronal Substrates of the Psychiatric Risk Kinase TNIK.  Wang Q, Amato SP, Rubitski DM, Hayward MM, Kormos BL, Verhoest PR, Xu L, Brandon NJ, Ehlers MD. Pharmacol Exp Ther. 2016 Feb;356(2):410-23. doi: 10.1124/jpet.115.229880. Epub 2015 Dec 8.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/selectivity-against-and-action-of-tnik-kinase-in-relation-to-alk2/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Synthesis of promiscuous ALK2 inhibitors-continuation-2</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/synthesis-of-promiscuous-alk2-inhibitors-continuation-2/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/synthesis-of-promiscuous-alk2-inhibitors-continuation-2/#respond</comments>
		<pubDate>Thu, 13 Jun 2019 20:34:07 +0000</pubDate>
		<dc:creator><![CDATA[Alfredo Picado]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Alfredo Picado]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>
		<category><![CDATA[SGC UNC Chapel Hill]]></category>
		<category><![CDATA[Understudied Kinases]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2924</guid>
		<description><![CDATA[Continuing with our goal to synthesize three compounds as ALK2 inhibitors, let’s refer back to Scheme 2 in the previous post on this topic, compounds 6, 8 and 9 were synthesized in good yields (65-90%), however compound 10 could not be obtained. &#160; Different conditions were tested to induce the cyclization of the bromo-pyrazine-2-carboxamide 9 <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/synthesis-of-promiscuous-alk2-inhibitors-continuation-2/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p style="text-align: justify;">Continuing with our goal to synthesize three compounds as ALK2 inhibitors, let’s refer back to Scheme 2 in the previous post on this topic, compounds <strong>6</strong>, <strong>8</strong> and <strong>9</strong> were synthesized in good yields (65-90%), however compound <strong>10</strong> could not be obtained.</p>
<div id="attachment_2597" style="width: 748px" class="wp-caption alignnone"><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Screen-Shot-2019-03-19-at-4.22.42-PM.png" alt="" width="738" height="522" class=" wp-image-2597" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Screen-Shot-2019-03-19-at-4.22.42-PM.png 918w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Screen-Shot-2019-03-19-at-4.22.42-PM-300x212.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Screen-Shot-2019-03-19-at-4.22.42-PM-768x544.png 768w" sizes="(max-width: 738px) 100vw, 738px" /><p class="wp-caption-text">Scheme 2. Synthesis route 1</p></div>
<p>&nbsp;</p>
<p style="text-align: justify;">Different conditions were tested to induce the cyclization of the bromo-pyrazine-2-carboxamide <strong>9</strong> to the target bromo-pyrazin-2-amine <strong>10</strong>. This reaction mostly yielded starting material <strong>9 </strong>and by-product <strong>11</strong> in low yields (&lt;15%). After several attempts, this approach (acid/heat) was abandoned.</p>
<p>&nbsp;</p>
<div id="attachment_2926" style="width: 746px" class="wp-caption alignnone"><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/06/Screen-Shot-2019-06-13-at-4.17.14-PM.png" alt="" width="736" height="592" class=" wp-image-2926" /><p class="wp-caption-text">Scheme 3. Synthesis route 2</p></div>
<p style="text-align: justify;">An alternative synthetic route is shown in Scheme 3.  Here, aminopirazine <strong>12</strong> will be reduced with diisobutylaluminium hydride (DIBAL-H) to yield carbaldehyde <strong>13</strong>, which then will be reacted with diamine <strong>6</strong> (previously synthesized) at 200 <sup>o</sup>C using dimethylacetamide (DMA) as a solvent. This last step can be carried out in a microwave reactor and should yield the cyclized compound pyridinpyrazin-2-amine <strong>14</strong>, which can be brominated with <em>N</em>-Bromosuccinimide (NBS) to yield compound <strong>10</strong>.</p>
<p style="text-align: justify;">As it was mentioned before, <strong>10 </strong>could not be synthesized by the previously proposed route. The new route is adapted from international patent WO 2008/071937 and we hope in this way we can access to this key intermediate in 150-200 mg scale to move forward with more reactions.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/synthesis-of-promiscuous-alk2-inhibitors-continuation-2/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>ACVR1 Crystals taken to the Diamond Light Source (Beamline I04.) &#8211; no data sets collected.</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/acvr1-crystals-taken-to-the-diamond-light-source-beamline-i04-no-data-sets-collected/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/acvr1-crystals-taken-to-the-diamond-light-source-beamline-i04-no-data-sets-collected/#respond</comments>
		<pubDate>Mon, 13 May 2019 13:19:13 +0000</pubDate>
		<dc:creator><![CDATA[Ellie Williams]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Ellie Williams]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>
		<category><![CDATA[SGC Univ. of Oxford]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2861</guid>
		<description><![CDATA[Following up on the crystals from my last post I sent them along on the last Diamond trip we had to beamline I04 but regrettably didn’t get any usable datasets. Perhaps fortunately there is a possible explanation for this that isn’t just ‘the crystals were a bit too rubbish’. We send our crystals to the Diamond <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/acvr1-crystals-taken-to-the-diamond-light-source-beamline-i04-no-data-sets-collected/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Following up on the crystals from my last post I sent them along on the last Diamond trip we had to <a href="https://www.diamond.ac.uk/Instruments/Mx/I04">beamline I04</a> but regrettably didn’t get any usable datasets. Perhaps fortunately there is a possible explanation for this that isn’t just ‘the crystals were a bit too rubbish’. We send our crystals to the Diamond Light Source and because of the high number of crystals sent on each trip by the department (124 on the last trip) not everyone goes along on every trip and we quite often use something called ‘auto-centering’ that uses a computer algorithm to find the edge of the loop and to align the loop and your crystal in the beam. Centering is necessary but also quite tricky, because to collect a data set you need ‘slices’ of data taken with the crystal at different angles so the loop the crystals are mounted in, is rotated by a robotic arm so that the crystal also rotates around inside the x-ray beam. Thus if your crystal isn’t aligned properly, as the loop rotates it moves you crystal out of the beam and thus you get no data at that point.</p>
<p>Usually the automatic alignment is really good, however this time around it didn’t work so well and quite often either my crystal wasn’t in the beam at all (it mistook the loop stem for the crystal) or it started in the beam and then rotated out of it (which is what I think happened in the case shown below where you can see two ‘periods’ of data with dips in between. This is, of course, suboptimal and leads to a data set that you can’t really use.</p>
<p>I’m resending the crystals to the beamline this Friday and hopefully going with them myself. The silver lining to this is that I know that some of them definitely diffract which is encouraging. Hopefully they’ll behave well when fully in the beam.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/05/xtal_centering.png" alt="" width="940" height="430" class="alignnone size-full wp-image-2862" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/05/xtal_centering.png 940w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/05/xtal_centering-300x137.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/05/xtal_centering-768x351.png 768w" sizes="(max-width: 940px) 100vw, 940px" /></p>
<p><em>Fig. 1 – Beam aligned at very edge of loop showing cyclic diffraction suggesting the crystal was moving in and out of the beam rather than degrading in the beam.</em></p>
<p>&nbsp;</p>
<p>You can read more about this <a href="https://doi.org/10.5281/zenodo.2788112">over on Zenodo.</a></p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/acvr1-crystals-taken-to-the-diamond-light-source-beamline-i04-no-data-sets-collected/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Re-Purification of ALK2 for crystallisation with different compounds from M4K.</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/re-purification-of-alk2-for-crystallisation-with-different-compounds-from-m4k/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/re-purification-of-alk2-for-crystallisation-with-different-compounds-from-m4k/#respond</comments>
		<pubDate>Wed, 24 Apr 2019 14:50:59 +0000</pubDate>
		<dc:creator><![CDATA[Ellie Williams]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Ellie Williams]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>
		<category><![CDATA[SGC Univ. of Oxford]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2753</guid>
		<description><![CDATA[I’ve been lucky enough to have a student (Mira) in the lab working with me for the past few weeks and so I’d like to write about the work that we’ve been doing together. We spent some time cleaning up an old sample of ALK2 that Ros had left over in the freezer to set <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/re-purification-of-alk2-for-crystallisation-with-different-compounds-from-m4k/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>I’ve been lucky enough to have a student (Mira) in the lab working with me for the past few weeks and so I’d like to write about the work that we’ve been doing together.</p>
<p>We spent some time cleaning up an old sample of ALK2 that Ros had left over in the freezer to set up in crystal plates with some compounds from M4K Pharma that they were relatively keen to see the structures of.</p>
<p>We used a specialised coarse screen that Ros had designed containing only conditions where we had previously seen crystals in the wider coarse screens. This means that we can narrow down the field where we’re expecting to find crystals and test fewer conditions to find the one we need – this means setting up fewer plates and thus needing less protein per compound tested, so is overall more efficient.</p>
<p>We set up the plates (at 4C and 20C) at around 9mg/ml with compounds M4K2163, M4K2143, M4K2192, M4K2207 and M4K2194 at 1mM concentration from 25mM stocks.</p>
<p>We found some interesting crystals, one of which was mountable as is (from M4K2192) and several across all the compounds, which showed small crystals that would be worth following up in more specific conditions.</p>
<p><img class="alignnone size-full wp-image-2754" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/M4K2192.png" alt="" width="403" height="294" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/M4K2192.png 403w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/M4K2192-300x219.png 300w" sizes="(max-width: 403px) 100vw, 403px" /></p>
<p><em>Crystals of ALK2/ACVR1A with compound M4K2192</em></p>
<p>&nbsp;</p>
<p>So Mira designed some follow up screens based on the conditions that showed small crystals and we set up some more plates with those conditions in the hope of turning those small crystals into larger, better crystals. We had enough protein to set up followup plates for compounds M4K2163 and M4K2143 at 9.5mg/ml. From these plates we mounted 5 crystals across the board and combined with those above ended up with crystals of unknown quality across three compounds. These have been confirmed as containing protein by imaging the drops by UV light (UV light interacts with protein to make crystals only visible under UV light if they contain protein rather than just being salt) and are now waiting to be sent to the synchrotron to see if they actually diffract or not.  You can read more about the technical details <a href="https://zenodo.org/record/2650203#.XMB29jBKiUk">here on Zenodo.</a></p>
<p>&nbsp;</p>
<p><span style="font-size: 1.5rem;">Yesterday (23</span><span style="font-size: 11.25px; height: 0px; line-height: 0; position: relative; vertical-align: baseline; bottom: 1ex;">rd</span><span style="font-size: 1.5rem;">) also marked FOP awareness day and so, shamefully a day late, I’d like to add my picture to the many others out there celebrating</span><span style="font-size: 1.5rem;"> </span><strong style="font-size: 1.5rem;">#FunFeet4FOP</strong><span style="font-size: 1.5rem;">! Here are my toes.</span></p>
<p><img class="alignnone size-medium wp-image-2755" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Feet4FOP_crop-300x223.jpg" alt="" width="300" height="223" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Feet4FOP_crop-300x223.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Feet4FOP_crop-768x570.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Feet4FOP_crop-1024x761.jpg 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/04/Feet4FOP_crop.jpg 1143w" sizes="(max-width: 300px) 100vw, 300px" /></p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/re-purification-of-alk2-for-crystallisation-with-different-compounds-from-m4k/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>The Benefits of Conference attendance:  BMP Signalling in Cancer II.</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/the-benefits-of-conference-attendance-bmp-signalling-in-cancer-ii/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/the-benefits-of-conference-attendance-bmp-signalling-in-cancer-ii/#respond</comments>
		<pubDate>Mon, 15 Apr 2019 10:44:07 +0000</pubDate>
		<dc:creator><![CDATA[Ellie Williams]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Ellie Williams]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>
		<category><![CDATA[SGC Univ. of Oxford]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2728</guid>
		<description><![CDATA[I’ve got a few things to catch up with on the blog – one of them is to talk about a conference I attended in Oxford recently. I attended the Biochemical Society BMP Signalling in Cancer II meeting which was held in St. Annes College, Oxford. At this meeting I presented a poster on the <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/the-benefits-of-conference-attendance-bmp-signalling-in-cancer-ii/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>I’ve got a few things to catch up with on the blog – one of them is to talk about a conference I attended in Oxford recently. I attended the <a href="https://www.biochemistry.org/Events/tabid/379/MeetingNo/SA217/view/Conference/Default.aspx">Biochemical Society BMP Signalling in Cancer II</a> meeting which was held in St. Annes College, Oxford. At this meeting I presented a poster on the work I’ve been doing (and which has been written about here as it’s happened) on looking at how the mutations in ALK2 effect the underlying activity of the kinase as well as effecting how well FKBP12 can inhibit ALK2.</p>
<p>The conference brought together 60 researchers in the field from around the world talking about a variety of different cancers but also other BMP related conditions such as FOP. Over three days there were some fantastic talks by the leaders in the field including from Dr. Caroline Hill from the Crick Institute in London who spoke about receptor cross talk in BMP signalling and how that might affect signalling through ALK2 and Prof. Eileen Shore from the university of Pennsylvania who spoke about mechanosensing and tissue stiffness in FOP.</p>
<p>I had a chance to get into some really interesting conversations with some of the attendees about the way their work and my work intersected and how we might be able to collaborate in the future – particularly interesting was talking to some Postdocs from Caroline Hill’s group who are looking at similar problems to me but from an in-cell perspective rather than using purified proteins. From that some interesting questions have been thrown up that I now want to test in my in-vitro systems and that will be detailed here as and when I’ve had a chance to try them. Hopefully this can lead to validation of both my work and theirs by complementing in-vivo and in-vitro approaches.</p>
<p>I thought it would be good to share the poster I created here – so you can find it <a href="https://zenodo.org/record/2640488#.XLRgljBKiUk">over on Zenodo</a>. It is designed for a scientific audience but not necessarily an FOP specialist audience so hopefully it should be fairly easy to follow. It leads the reader through the story of explaining that Gain of Function (GOF) mutations in ALK2 are involved in diseases like FOP and DIPG. I then go on to present data that shows that these mutations disrupt the binding of the inhibitor protein FKBP12 but that this isn’t the whole story (as regular readers of this blog will know). Then I talk about what I found out about what is needed for SMAD activation by ALK2 before looking at the increased phosphorylation of SMAD1 by the mutant ALK2 proteins compared to the wild type. This covers both the western blot data that shows more SMAD1 activation but also the mass spec data I’ve shared previously that looks at more SMAD1 and more ALK2 phosphorylation. Finally I showed the phosphomapping data on ALK2 that is starting to pick apart the exact phosphorylation locations on ALK2 that are necessary for SMAD1 activation (Also data I’ve talked about on this blog before). In this case it was nice to see that all the effort Rod and I had put into phosphomapping had paid off! Other groups had said they’d tried and been unable to map ALK2 using trypsin alone but our method of using 5 different enzymes both enriched and unenriched seems to have done the trick (despite it being 10x more work!).</p>
<p>There is great value from attending a conference like this and in this case I feel that I really did make some very important connections and had some very informative discussions that were directly relevant to my research and that will hopefully bear fruit in time and guide the direction of my experiments.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/the-benefits-of-conference-attendance-bmp-signalling-in-cancer-ii/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Synthesis of promiscuous ALK2 inhibitors</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/synthesis-of-promiscuous-alk2-inhibitors/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/synthesis-of-promiscuous-alk2-inhibitors/#respond</comments>
		<pubDate>Tue, 19 Mar 2019 20:34:09 +0000</pubDate>
		<dc:creator><![CDATA[Alfredo Picado]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Alfredo Picado]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>
		<category><![CDATA[SGC UNC Chapel Hill]]></category>
		<category><![CDATA[Understudied Kinases]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2598</guid>
		<description><![CDATA[In a collaboration between SGC-University of North Carolina-Chapel Hill and M4K Pharma, we plan to synthesize 3 different imidazopyridin pyrazines as promiscuous kinase ALK2 inhibitors, compounds 1-3, Scheme 1. The first step will be the synthesis of key intermediate 10, Scheme 2.  The proposed synthesis begins with a nucleophilic aromatic substitution on compound 4 to <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/synthesis-of-promiscuous-alk2-inhibitors/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>In a collaboration between SGC-University of North Carolina-Chapel Hill and M4K Pharma, we plan to synthesize 3 different imidazopyridin pyrazines as promiscuous kinase ALK2 inhibitors, compounds <strong>1-3</strong>, Scheme 1.</p>
<div id="attachment_2596" style="width: 994px" class="wp-caption alignnone"><img class="size-full wp-image-2596" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Screen-Shot-2019-03-19-at-4.22.27-PM.png" alt="" width="984" height="346" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Screen-Shot-2019-03-19-at-4.22.27-PM.png 984w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Screen-Shot-2019-03-19-at-4.22.27-PM-300x105.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Screen-Shot-2019-03-19-at-4.22.27-PM-768x270.png 768w" sizes="(max-width: 984px) 100vw, 984px" /><p class="wp-caption-text">Scheme 1. ALK2 inhibitors</p></div>
<p>The first step will be the synthesis of key intermediate <strong>10</strong>, Scheme 2.  The proposed synthesis begins with a nucleophilic aromatic substitution on compound <strong>4</strong> to get nitropyridine <strong>5</strong>, then reduction with hydrogen on palladium/carbon to obtain the pyridine-3,4-diamine <strong>6</strong>.</p>
<p>On the other hand, aminopyrazine <strong>7</strong> will be saponified to the aminopyrazine sodium salt <strong>8</strong>, which will be coupled with  compound <strong>6</strong> using peptide coupling reagent HATU to form bromopyrazine carboxamide <strong>9</strong>, followed by heating in presence of acetic acid to yield the cyclized intermediate 5-bromo-3-(1-ethyl-1<em>H</em>-imidazo[4,5-<em>c</em>]pyridin-2-yl)pyrazin-2-amine <strong>10</strong>.</p>
<div id="attachment_2597" style="width: 928px" class="wp-caption alignnone"><img class="size-full wp-image-2597" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Screen-Shot-2019-03-19-at-4.22.42-PM.png" alt="" width="918" height="650" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Screen-Shot-2019-03-19-at-4.22.42-PM.png 918w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Screen-Shot-2019-03-19-at-4.22.42-PM-300x212.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/Screen-Shot-2019-03-19-at-4.22.42-PM-768x544.png 768w" sizes="(max-width: 918px) 100vw, 918px" /><p class="wp-caption-text">Scheme 2. Synthesis route 1</p></div>
<p>If we succeed with the proposed synthesis of imidazopyridine pyrazine <strong>10</strong>, then soon we will present next steps to get to target compounds <strong>1-3</strong></p>
<p>This collaboration is the under supervision of Dr. David Drewry.</p>
<p>Synthesis routes were adapted from international patents WO 2006/063167 A1, WO 2008/071937 and <em>ACS Med. Chem. Lett</em>. 2016, 7, 5, 514-519.</p>
<p>&nbsp;</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/synthesis-of-promiscuous-alk2-inhibitors/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Phosphorylation over time and phosphomapping experiments to investigate the influence of ALK2 mutations on activity.</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/phosphorylation-over-time-and-phosphomapping-experiments-to-investigate-the-influence-of-alk2-mutations-on-activity/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/phosphorylation-over-time-and-phosphomapping-experiments-to-investigate-the-influence-of-alk2-mutations-on-activity/#respond</comments>
		<pubDate>Tue, 12 Mar 2019 11:56:29 +0000</pubDate>
		<dc:creator><![CDATA[Ellie Williams]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Ellie Williams]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>
		<category><![CDATA[SGC Univ. of Oxford]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2579</guid>
		<description><![CDATA[In this post I’d like to return to looking at my work on phosphorylation. Phosphorylation is a key signalling method for activating the BMP pathway and inducing activity. Specifically activation of ALK2 by phosphorylation leads to phosphorylation of SMAD1 which leads to gene transcription that ultimately is responsible for bone formation. In this post I’m <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/phosphorylation-over-time-and-phosphomapping-experiments-to-investigate-the-influence-of-alk2-mutations-on-activity/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>In this post I’d like to return to looking at my work on phosphorylation. Phosphorylation is a key signalling method for activating the BMP pathway and inducing activity. Specifically activation of ALK2 by phosphorylation leads to phosphorylation of SMAD1 which leads to gene transcription that ultimately is responsible for bone formation.</p>
<p>In this post I’m going to talk about two experiments – the first is looking at the rate of phosphorylation (or how active) of two ALK2 mutants, L196P and R258S. The second is to look at mapping the exact sites of phosphorylation on two of the more common ALK2 mutations in FOP, R206H and Q027E.</p>
<p>&nbsp;</p>
<p><strong>Phosphorylation over time in ALK2 L196P and R258S.</strong></p>
<p>I’ve talked <a href="https://thesgc.github.io/static-openlabnotebooks/phosphorylation-of-smad1-rates-as-a-function-of-alk2-mutations-and-type-ii-association/">previously</a> about how phosphorylation is effected by the Q207E and R206H mutations in ALK2. This time I wanted to look at some additional mutants that are found in a small proportion of cases of FOP, specifically L196P, R258S, G328E and R375P. I purified the long forms of these ALK2 mutations in a standard fashion. Regrettably only L196P and R258S gave a good yield of expressed protein while the other two constructs gave me no expression of protein. I set up my standard activation assay with these ALK2 mutants (I mixed them with SMAD1 and with either ACVR2 or BMPR2, before adding ATP, DTT and Mg2+/Mn2+ in order to start the reaction), taking samples over time periodically up to 30 minutes. The samples were then analysed by mass spec to look for intact mass shifts corresponding to phosphorylation.</p>
<p>I plotted the peaks corresponding to phosphorylated states of both ALK2 and SMAD1 at the different time points and used this 3D graph to compare different mutant’s behaviour with respect to the activity of ALK2. I also compared this data to previously obtained data from the previous blog post.</p>
<p><img class="alignnone size-large wp-image-2580" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/MS_timecourse_blogpost-806x1024.jpg" alt="" width="780" height="991" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/MS_timecourse_blogpost-806x1024.jpg 806w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/MS_timecourse_blogpost-236x300.jpg 236w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/03/MS_timecourse_blogpost-768x975.jpg 768w" sizes="(max-width: 780px) 100vw, 780px" /></p>
<p><em>Time course plots for phosphorylation of SMAD1 (top) and ALK2 L196P or R258S (bottom) in the presence of ACVR2 (left) or BMPR2 (right). Legend shows time point in seconds.</em></p>
<p>&nbsp;</p>
<p>Overall we can see that R258S appears to be more activating than L196P with both more SMAD activation over the first five minutes (including the appearance of a potentially interesting third SMAD binding site). This is the case when using either ACVR2 or BMPR2. Also in general ACVR2 appears to lead to more SMAD activation than BMPR2 regardless of mutation.</p>
<p>When looking at ALK2 phosphorylation, we can see that L196P is quickly phosphorylated up to three times compared to R258S but that over the course of five minutes, R258S accumulates up to 6 phosphorylation’s much more rapidly than L196P. This effect is most pronounced when paired with ACVR2.</p>
<p>What this means in very broad terms is that the different ALK2 FOP mutants are differently activating and that their response to different type II receptors change depending on which mutation is present. This is similar to the results seen with other ALK2 mutants studied previously. This builds towards understanding why the ALK2 FOP mutations leads to increased signalling and thus increased bone formation and exactly what is going on.</p>
<p>&nbsp;</p>
<p><strong>Phosphomapping of ALK2 R206H and Q207E:</strong></p>
<p>Another aspect of looking at the phosphorylation of ALK2 and SMAD1 is not just how much it is phosphorylated but also where it is phosphorylated. For this I worked with Rod Chalk in our mass spec department to map the phosphorylation sites present in samples of protein incubated as previously mentioned. In this case I was looking at Q207E and R206H mutants of ALK2 paired with either ACVR2 or BMPR2, all in the presence of SMAD. This suggested that many of the phosphorylations could be traced to the following positions in the GS region of the kinase regardless of ALK2 mutation:</p>
<p><strong><u>ST</u></strong>LADLLDH<strong><u>S</u></strong>C<strong><u>TS</u></strong>G<strong><u>S</u></strong>G<strong><u>S</u></strong>GLPFLVQRTV</p>
<p>However not all these sites are seen at the same time and it is hard to determine which are essential for transmission of signal and ALK2 activation.</p>
<p>Similarly phosphorylation on SMAD1 could be traced to the following common locations at the C-terminal tail of the protein:</p>
<p>TQMGSPHNPI<strong><u>SS</u></strong>V<strong><u>S</u></strong></p>
<p>In this case it is the last three serines in the tail that are able to be phosphorylated and again, are seen with both mutations. Combinations of the three possible sites and all three at once were seen as possibilities within the phosphomapping data.</p>
<p>What this means however is a bit less clear. When considering the time course data from the previous post linked as well as the new mutant time course data and the phosphomapping it seems to suggest that you require multiple (up to 6) ALK2 phosphorylations to get maximal (3 site) SMAD1 phosphorylation and that there are at least this many sites in the GS loop and the region just prior to it.</p>
<p>In order to properly probe the necessity of each of these sites, it is going to be necessary to repeat these mass spec experiments on knock out mutants that remove the phosphorylation sites in various combinations, to really see which are essential for ALK2 activation. These are currently being cloned so hopefully I’ll be able to test them soon.</p>
<p>&nbsp;</p>
<p>You can find the technical details for all these experiments over on <a href="https://doi.org/10.5281/zenodo.2591062">Zenodo.</a></p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/phosphorylation-over-time-and-phosphomapping-experiments-to-investigate-the-influence-of-alk2-mutations-on-activity/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Checking I haven&#8217;t got my cells mixed up&#8230;</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/checking-i-havent-got-my-cells-mixed-up/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/checking-i-havent-got-my-cells-mixed-up/#respond</comments>
		<pubDate>Thu, 28 Feb 2019 18:56:05 +0000</pubDate>
		<dc:creator><![CDATA[Elizabeth Brown]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Brain Tumours]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>
		<category><![CDATA[Liz Brown]]></category>
		<category><![CDATA[SGC Univ. of Oxford]]></category>
		<category><![CDATA[Understudied Kinases]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2542</guid>
		<description><![CDATA[Did you know as many as 30% of the cells in labs could be misidentified? And what a nightmare it would be to find out years of your work are just completely wrong because you got your skin cells mixed up with your brain cells&#8230; So to avoid that catastrophe, I took some time to <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/checking-i-havent-got-my-cells-mixed-up/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Did you know as many as 30% of the cells in labs could be misidentified? And what a nightmare it would be to find out years of your work are just completely wrong because you got your skin cells mixed up with your brain cells&#8230;</p>
<p>So to avoid that catastrophe, I took some time to check the DNA sequence of the ACVR1 gene, because for this gene we know what should be mutated (or not) for each of my cell lines. This way I can make sure that I haven&#8217;t mixed them up in the past year (<em>and</em> that they haven&#8217;t picked up new mutations in this important area all by themselves).</p>
<p>All I had to do was collect the genomes of the cells, amplify specifically the ACVR1 gene, send it for overnight sequencing, and then I get a trace like this back from the facility:</p>
<div id="attachment_2544" style="width: 790px" class="wp-caption aligncenter"><a href="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/R206H-sequence-1.png"><img class="wp-image-2544 size-large" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/R206H-sequence-1-1024x396.png" alt="Sequencing trace covering the R206H mutation in ACVR1 found in FOP and DIPG" width="780" height="302" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/R206H-sequence-1-1024x396.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/R206H-sequence-1-300x116.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/R206H-sequence-1-768x297.png 768w" sizes="(max-width: 780px) 100vw, 780px" /></a><p class="wp-caption-text">Sequencing trace covering the R206H mutation in ACVR1 found in FOP and DIPG</p></div>
<p>Each DNA base has one peak, except one that has both a guanine (G) and an adenine (A) DNA base. The adenine, present on one chromosome, causes the change from an arginine (R) to a histidine (H) at the 206th position of the ACVR1 protein i.e. the R206H mutation.</p>
<p>This trace is from the HSJD-DIPG-007 cell line and is the mutation I expected. According to all my sequencing, none of my cell lines have gotten mixed up, yay!</p>
<p>You can read the full details of how I did this, <a href="https://zenodo.org/record/2579809#.XHgumMD7SUk">here on Zenodo</a>.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/checking-i-havent-got-my-cells-mixed-up/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>ZFYVE9C purification and association test with TGFBR1 and TGFBR2</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/zfyve9c-purification-and-association-test-with-tgfbr1-and-tgfbr2/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/zfyve9c-purification-and-association-test-with-tgfbr1-and-tgfbr2/#comments</comments>
		<pubDate>Fri, 22 Feb 2019 14:05:47 +0000</pubDate>
		<dc:creator><![CDATA[Ellie Williams]]></dc:creator>
				<category><![CDATA[ACVR1/ALK2]]></category>
		<category><![CDATA[Diffuse intrinsic pontine glioma]]></category>
		<category><![CDATA[Ellie Williams]]></category>
		<category><![CDATA[Fibrodysplasia ossificans progressiva]]></category>
		<category><![CDATA[SGC Univ. of Oxford]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2515</guid>
		<description><![CDATA[ZFYVE9C (Otherwise known as SARA) is a scaffold protein involved in the association of various protein components as part of the TGFB signalling pathway. The equivalent in the BMP signalling pathway is a protein known as Endofin so studying SARA (which is slightly easier) might give us useful insights for both proteins. For more details <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/zfyve9c-purification-and-association-test-with-tgfbr1-and-tgfbr2/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>ZFYVE9C (Otherwise known as SARA) is a scaffold protein involved in the association of various protein components as part of the TGFB signalling pathway. The equivalent in the BMP signalling pathway is a protein known as Endofin so studying SARA (which is slightly easier) might give us useful insights for both proteins. For more details on this project please see <a href="https://thesgc.github.io/static-openlabnotebooks/a-summary-of-my-alk2-project-goals/">this blog post.</a></p>
<p>One of the unanswered questions about SARA is what does one of the major domains that it has actually do? (The domain is known as a DUF domain, or a Domain of Unknown Function!) We know the structure but that hasn’t given us any clues as to its function. One thought was that it might be involved in recruiting type I or type II receptors close to their binding partners such as one of the SMADs to allow signal propagation.</p>
<p>I purified the DUF domain of SARA and mixed it with either excess TGFBR1 or excess TGFBR2 (the two relevant receptors in the TGFB pathway). I tested the receptors in both the unphosphorylated (inactive) and phosphorylated (active) states. After incubating the samples for 30 minutes while concentrating the volume back down to &lt;5ml, I ran them down a gel filtration column to see if the two proteins bound together sufficiently to shift the peaks coming off the column which would indicate they were acting as a complex of larger mass.</p>
<p><img class="alignnone size-large wp-image-2516" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/gf_trace_zfyve9c-1024x491.png" alt="" width="780" height="374" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/gf_trace_zfyve9c-1024x491.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/gf_trace_zfyve9c-300x144.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/gf_trace_zfyve9c-768x368.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/02/gf_trace_zfyve9c.png 1578w" sizes="(max-width: 780px) 100vw, 780px" /></p>
<p><em>Overlay of the UV traces from all the size exclusion gel filtration runs to show the relative elution volumes of the peaks confirming no shift in the ZFYVE9C peak consistent with no complex being formed.</em></p>
<p>&nbsp;</p>
<p>Unfortunately I didn’t see any shift in the peak for SARA suggesting that either the two proteins don’t bind or that the affinity is weak enough that they fall apart under the conditions of the gel filtration column.</p>
<p>The next thing to try will be to purify the proteins again but not to cleave the tag on one of the binding partners (either the receptors or the SARA protein) and try to do a pull down experiment to test whether this shows any association or not.</p>
<p>&nbsp;</p>
<p>You can find more details over at<a href="https://doi.org/10.5281/zenodo.2575581"> Zenodo.</a></p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/zfyve9c-purification-and-association-test-with-tgfbr1-and-tgfbr2/feed/</wfw:commentRss>
		<slash:comments>2</slash:comments>
		</item>
	</channel>
</rss>
