<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>Victoria Vu &#8211; openlabnotebooks.org</title>
	<atom:link href="https://thesgc.github.io/static-openlabnotebooks/category/scientist/victoria-vu/feed/" rel="self" type="application/rss+xml" />
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<description>A growing team of groundbreaking scientists around the world are now sharing their lab notebooks online</description>
	<lastBuildDate>Mon, 01 Jun 2020 21:14:49 +0000</lastBuildDate>
	<language>en-GB</language>
	<sy:updatePeriod>
	hourly	</sy:updatePeriod>
	<sy:updateFrequency>
	1	</sy:updateFrequency>
	<generator>https://wordpress.org/?v=5.4.2</generator>

<image>
	<url>https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/cropped-lab-jar02-32x32.png</url>
	<title>Victoria Vu &#8211; openlabnotebooks.org</title>
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<width>32</width>
	<height>32</height>
</image> 
	<item>
		<title>UBE3A Wnt signaling Cell Assay</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/ube3a-wnt-signaling-cell-assay/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/ube3a-wnt-signaling-cell-assay/#respond</comments>
		
		<dc:creator><![CDATA[Victoria Vu]]></dc:creator>
		<pubDate>Mon, 01 Jun 2020 21:14:49 +0000</pubDate>
				<category><![CDATA[E3 ligases]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<category><![CDATA[Victoria Vu]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=4826</guid>

					<description><![CDATA[Background Apart from my HTT protein project, I also work on UBE3A, a HECT domain E3 ligase. E3 ligases work as a part of the ubiquitination system interacting with E2 ubiquitin-conjugating enzymes to transfer ubiquitin onto target substrate proteins. Ubiquitination of substrate proteins regulate a diversity of cell processes and is also important for ubiquitin-mediated <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/ube3a-wnt-signaling-cell-assay/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p><strong><em><u>Background</u></em></strong></p>
<p>Apart from my HTT protein project, I also work on UBE3A, a HECT domain E3 ligase. E3 ligases work as a part of the ubiquitination system interacting with E2 ubiquitin-conjugating enzymes to transfer ubiquitin onto target substrate proteins. Ubiquitination of substrate proteins regulate a diversity of cell processes and is also important for ubiquitin-mediated proteosomal degradation of proteins.</p>
<p>UBE3A reported substrates include the tumor suppressor p53 (<a href="https://pubmed.ncbi.nlm.nih.gov/8221889/">PMID8221889</a>), the PDZ domain-containing protein scribble (<a href="https://pubmed.ncbi.nlm.nih.gov/11027293/">PMID11027293</a>), and NFX1-91, a transcriptional repressor of the gene encoding hTERT (<a href="https://pubmed.ncbi.nlm.nih.gov/15371341/">PMID15371341</a>). UBE3A was initially identified as the cellular factor that stimulates ubiquitin-dependent degradation of the tumor suppressor p53 driven by the E6 protein of cervical cancer-associated human papillomavirus (HPV) types 16 and 18 (<a href="https://pubmed.ncbi.nlm.nih.gov/8380895/">PMID8380895</a>). UBE3A also functions as a transcriptional coactivator for steroid hormone receptors (<a href="https://pubmed.ncbi.nlm.nih.gov/18432313/">PMID18432313</a>).</p>
<p>I am interested in understanding the role of UBE3A as an E3 ligase in the context of neurological diseases. UBE3A shows brain-specific imprinting and the loss of function of maternally inherited UBE3A has been linked to Angelman syndrome (<a href="https://pubmed.ncbi.nlm.nih.gov/8988171/">PMID8988171</a>). In addition, the autism-linked UBE3A-T485A mutation disrupts phosphorylation control of UBE3A and enhances UBE3A ubiquitin ligase activity (<a href="https://pubmed.ncbi.nlm.nih.gov/28559284/">PMID28559284</a>). This suggest that dysregulation of UBE3A, both downregulation and upregulation, can affect neurological function.</p>
<p>To better understand the cellular function of UBE3A, we can use chemical probes to inhibit it. To identify compounds that affect UBE3A, we can develop reporter assays that detection activation of pathways involving our protein of interest. UBE3A can stimulate Wnt pathway activation and protect β-catenin from proteasomal degradation with these activities requiring the ubiquitin ligase activity of UBE3A (<a href="https://pubmed.ncbi.nlm.nih.gov/27902311/">PMID27902311</a>) (Fig 1). Here, I describe a UBE3A Wnt-signalling reporter cell assay developed in collaboration with Dalia Barsyte-Lovejoy. The methods described in this post can be found on <a href="http://doi.org/10.5281/zenodo.3872219">Zenodo</a>.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/zbc0311770290007-936x1024.jpg" alt="" width="266" height="291" class="wp-image-4831 aligncenter" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/zbc0311770290007-936x1024.jpg 936w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/zbc0311770290007-274x300.jpg 274w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/zbc0311770290007-768x840.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/zbc0311770290007.jpg 989w" sizes="(max-width: 266px) 100vw, 266px" /></p>
<p><strong>Figure 1. UBE3A regulation of Wnt pathway (from <a href="https://pubmed.ncbi.nlm.nih.gov/28559284/">PMID28559284</a>).</strong></p>
<p><strong><em><u>Results</u></em></strong></p>
<p>To assess the transcriptional output of Wnt pathway we utilized the TCF response element 8xTOP-Flash-FLuc (Firefly luciferase) reporter that is co-transfected into HEK293 cells together with increasing amounts of UBE3A and Renilla luciferase (Rluc) expressing plasmid that is used for transfection efficiency normalization purposes (Fig 2).</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/Fig2-1024x183.png" alt="" width="780" height="139" class="aligncenter size-large wp-image-4827" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/Fig2-1024x183.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/Fig2-300x54.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/Fig2-768x138.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/Fig2.png 1189w" sizes="(max-width: 780px) 100vw, 780px" /></p>
<p><strong>Figure 2. Wnt activity reporter assay system schematic. Reporter constructs were previously described in <a href="https://pubmed.ncbi.nlm.nih.gov/12699626/">PMID12699626</a>.</strong></p>
<p>To determine if UBE3A specifically affects the Wnt reporter and is not affecting the control Rluc or the mutant TCF response element reporter Fluc, we assessed the Rluc and Fluc separately and as a normalized ratio (Fig 3). UBE3A drastically upregulated the 8xTOP reporter signal, slightly upregulated Rluc driven by TK promoter and had no effect on mutant FOP reporter. We also use a GSK3 inhibitor to ensure that the 8XTOP-Flash-FLuc reporter responds to Wnt activation. Normally, GSK3 phosphorylates β-catenin triggering its degradation.  Inhibition of GSK3 will stabilize β-catenin, translating to increased Wnt signalling. The 8xTOP Flash reporter is activated by GSK3 inhibition and resulting Wnt activation, but the mutant FOP Flash reporter is not.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/Fig3-1024x373.png" alt="" width="780" height="284" class="aligncenter size-large wp-image-4828" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/Fig3-1024x373.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/Fig3-300x109.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/Fig3-768x280.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/Fig3.png 1445w" sizes="(max-width: 780px) 100vw, 780px" /></p>
<p><strong>Figure 3. UBE3A reporter assay specificity testing. Left panel represents data on baseline reporter activity, while right graph is data obtained upon the activation of wnt pathway by inhibiting GSK3. </strong>(Dalia Barsyte-Lovejoy)</p>
<p>Apart from testing the specificity of the assay, we also want to show that the system is it is responsive to varying levels of Wnt activation. We find that increasing amounts of three different isoforms of UBE3A resulted in Wnt reporter activation in a dose-response manner (Fig 4). UBE3A isoform III gave the highest relative activity Fluc/Rluc (i.e. lowest signal:background ratio) which makes it the most suitable for chemical probe screening. We will use this isoform moving forward.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/Fig4-1024x436.png" alt="" width="780" height="332" class="aligncenter size-large wp-image-4829" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/Fig4-1024x436.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/Fig4-300x128.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/Fig4-768x327.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/Fig4.png 1396w" sizes="(max-width: 780px) 100vw, 780px" /></p>
<p><strong>Figure 4. Wnt reporter activity in response to increasing UBE3A. Left panel represents data on baseline reporter activity, while the right graph is data obtained upon the activation of Wnt pathway by inhibiting GSK3. </strong>(Dalia Barsyte-Lovejoy)</p>
<p>To further validate the assay, we test whether the Wnt activation is dependent on the catalytic activity of UBE3A. Using site-directed mutagenesis, I made the UBE3A-C833A catalytically inactive mutant (<a href="https://pubmed.ncbi.nlm.nih.gov/9497376/">PMID9497376</a>) where the alanine substitution of the active site cysteine obliterates UBE3A’s ability of forming a thiolester bond with ubiquitin. The wildtype UBE3A protein showed Wnt reporter activation, while the catalytic cysteine C833 mutant UBE3A did not (Fig 5).</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/Fig5.png" alt="" width="362" height="332" class="aligncenter size-full wp-image-4830" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/Fig5.png 362w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2020/06/Fig5-300x275.png 300w" sizes="(max-width: 362px) 100vw, 362px" /></p>
<p><strong>Figure 5. UBE3A driven Wnt activation is dependent on its catalytic activity. </strong>(Victoria Vu)</p>
<p>Now that I have a Wnt activation cell assay that is specific, dose-responsive and depends on the catalytic activity of UBE3A, I can screen for compounds that may inhibit UBE3A catalytic activity and use it to better understand the function of UBE3A in neurological disorders.</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/ube3a-wnt-signaling-cell-assay/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>Overexpressing HTT in neuronal cells: Update</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/overexpressing-htt-in-neuronal-cells-update/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/overexpressing-htt-in-neuronal-cells-update/#respond</comments>
		
		<dc:creator><![CDATA[Victoria Vu]]></dc:creator>
		<pubDate>Mon, 18 Nov 2019 20:02:42 +0000</pubDate>
				<category><![CDATA[Huntington’s disease]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<category><![CDATA[Victoria Vu]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=3796</guid>

					<description><![CDATA[In my last blog post, I described how I am trying to overexpress HTT in neuronal cells. While the BacMAM system was easy to implement in cells such as HEK293, it proved far more challenging to express in SH-SH5Y neuronal cells. Since then, I have attempted to use the BacMAM system in two other neuronal <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/overexpressing-htt-in-neuronal-cells-update/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p>In my <a href="https://thesgc.github.io/static-openlabnotebooks/project-overview-overexpressing-htt-in-neuronal-cells/">last blog post</a>, I described how I am trying to overexpress HTT in neuronal cells. While the BacMAM system was easy to implement in cells such as HEK293, it proved far more challenging to express in SH-SH5Y neuronal cells. Since then, I have attempted to use the BacMAM system in two other neuronal cell lines, Neuro-2a and N38. However, I found that even the positive control GFP-BacMAM was low expressing and thus the lines were not very promising for HTT expression (data not shown). These results were disheartening so I began exploring alternative means of expressing HTT in cells, including protein transfection and protein electroporation. The methods described in this post can be found on <a href="https://doi.org/10.5281/zenodo.3546328">Zenodo</a>.</p>
<p>I tested protein transfection using HEPES as a DIY protein transfection reagent following protocols from <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6357789/">PMC6357789</a>. Jacob McAuley <a href="https://thesgc.github.io/static-openlabnotebooks/final-purifications/">purified some GFP protein</a> for me to use as a test protein. HTT is a far larger protein so I rationalized that if the method did not even work for GFP, it was unlikely to work with HTT which was both larger and cannot be concentrated to the same high level as GFP. The HEPES transfection did not work in HEK293, Neuro-2a, N38 and SH-SH5Y cells, with no discernible amount of GFP detected on a fluorescent microscope in all four cell lines (data not shown).</p>
<p>Electroporation has also been reported to successfully deliver proteins into cells. Using the methods from <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6656429/">PMC6656429</a>, and conditions from the <a href="https://www.thermofisher.com/content/dam/LifeTech/migration/en/filelibrary/cell-culture/neon-protocols.par.61129.file.dat/hek%20293-kidney.pdf">manufacturer</a>, I began with electroporating some HEK293 cells with GFP protein to identify conditions where I got the highest level of protein electroporation and sufficiently high viability post-electroporation. I was able to successful electroporate in the GFP protein (Figure 1). I noted that the protein electroporation is most successful with higher levels of protein concentration and with post-electroporation treatment of cells with trypsin. Alas, when I tried this method with HTT, I was not able to get measurable HTT signal by Western blot (Figure 2). This is likely because HTT is a protein that is difficult to concentrated to levels higher than 4mg/ml. While the method did not work for this specific application, it does offer a useful approach to introduce difficult to express proteins into cells and examine how they behave within cells. Antibodies can also be introduced into cells in this manner, as well as sgRNAs/cas9 protein for cell engineering.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/11/293TRex_PtnElectroporation_OLN-01-1024x623.png" alt="" width="780" height="475" class="aligncenter size-large wp-image-3798" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/11/293TRex_PtnElectroporation_OLN-01-1024x623.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/11/293TRex_PtnElectroporation_OLN-01-300x183.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/11/293TRex_PtnElectroporation_OLN-01-768x467.png 768w" sizes="(max-width: 780px) 100vw, 780px" /></p>
<p><strong>Figure 1. GFP protein electroporation in HEK293 cells.</strong></p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/11/HTT_electroporated_OLN-01-1-1024x512.jpg" alt="" width="780" height="390" class="aligncenter size-large wp-image-3803" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/11/HTT_electroporated_OLN-01-1-1024x512.jpg 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/11/HTT_electroporated_OLN-01-1-300x150.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/11/HTT_electroporated_OLN-01-1-768x384.jpg 768w" sizes="(max-width: 780px) 100vw, 780px" /></p>
<p><strong>Figure 2. HTT protein electroporation in HEK293 cells.</strong></p>
<p>In the end, I decided to go back to the BacMAM system in SH-SY5Y cells and spend some time tweaking the cell culture media and timing of the transduction to improve BacMAM HTT expression. In my last post, I observed that SH-SHY5 cells began dying 2 days post-transduction and that more cells died in higher v/v BacMAM virus. Some possible causes included the SH-SY5Y cells may be dying from either too much virus, the side effects of overexpressing HTT, or incompatibility with the Sf9 media the BacMAM virus is in. I noted that with greater v/v BacMAM virus, the SH-SY5Y cell media was acidifying more quickly, and I hypothesized that this may be because the sf9 media is incompatible with the cells. I scoured the internet looking for advice on what SH-SY5Y media modifications I can make to improve cell growth. Most publications refer to standard growth media conditions identical to my own, but after investigation into the exact formulation of some medias used, I noticed that some contained higher levels of sodium bicarbonate, which is used as part of the CO<sub>2 </sub>buffering system.</p>
<p>Using modified SH-SY5Y media with a higher percentage of FBS (15%) and supplemented with B27, non-essential amino acids and additional sodium bicarbonate, I was able to successfully express HTT in SH-SY5Y cells (Figure 3). Overall, I observe that there is a trade-off between increased HTT expression and cell death with higher v/v BacMAM virus levels or longer transduction times. Moving forward, I will need to optimize these parameters to get the highest HTT expression with lowest SH-SY5Y cell death. I also want to attempt to express other HTT mutant forms with varying number of CAG repeats.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/11/HTT_18.09.2019_OLN-01-1024x509.png" alt="" width="780" height="388" class="aligncenter size-large wp-image-3797" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/11/HTT_18.09.2019_OLN-01-1024x509.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/11/HTT_18.09.2019_OLN-01-300x149.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/11/HTT_18.09.2019_OLN-01-768x382.png 768w" sizes="(max-width: 780px) 100vw, 780px" /><strong>Figure 3. BacMAM HTT protein expression in SH-SY5Y cells.</strong></p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/overexpressing-htt-in-neuronal-cells-update/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
		<item>
		<title>Project Overview: Overexpressing HTT in neuronal cells</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/project-overview-overexpressing-htt-in-neuronal-cells/</link>
					<comments>https://thesgc.github.io/static-openlabnotebooks/project-overview-overexpressing-htt-in-neuronal-cells/#respond</comments>
		
		<dc:creator><![CDATA[Victoria Vu]]></dc:creator>
		<pubDate>Tue, 06 Aug 2019 04:19:23 +0000</pubDate>
				<category><![CDATA[Huntingtin]]></category>
		<category><![CDATA[Huntington’s disease]]></category>
		<category><![CDATA[SGC Univ. of Toronto]]></category>
		<category><![CDATA[Victoria Vu]]></category>
		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=3260</guid>

					<description><![CDATA[My main project revolves around the E3 ligase UBE3A (I will be writing a project overview for this later), but I am also continuing some work in the same vein as Jolene Ho&#8217;s work looking at role of HTT in neurological disorders. HTT encodes a protein called huntingtin whose function is not known but appears <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/project-overview-overexpressing-htt-in-neuronal-cells/">Read More ...</a>]]></description>
										<content:encoded><![CDATA[<p>My main project revolves around the E3 ligase UBE3A (I will be writing a project overview for this later), but I am also continuing some work in the same vein as <a href="https://thesgc.github.io/static-openlabnotebooks/category/past-scientist/jolene-ho/">Jolene Ho&#8217;s work</a> looking at role of HTT in neurological disorders.</p>
<p>HTT encodes a protein called huntingtin whose function is not known but appears to be crucial to neuron function.  Mutant HTT is frequently associated with Huntington&#8217;s Disease (HD), where the 5&#8242; end of the <em>HTT</em> gene is observed to have an expanded CAG repeat region &#8211; the trinucleotide is repeated more times in individuals with HD than in normal individuals. In HD, mutant HTT is observed to affect neuronal function and structure leading to progressive neurodegeneration, however the exact disease mechanism is not well understood.</p>
<p>To study the function of HTT and the effects of mutant HTT protein <em>in vivo</em>, I will try to introduce trackable copies of <em>HTT </em>into mammalian cells, specifically neuronal cells. By overexpressing the HTT protein in varying amounts and with varying number of CAG repeats, we can try to elucidate the normal function of HTT in a non-disease context and examine how mutant HTT protein affects cell function and leads to neurodegeneration.</p>
<p>As seen in <a href="https://thesgc.github.io/static-openlabnotebooks/category/past-scientist/jolene-ho/">Jolene&#8217;s previous posts</a>, it is can be difficult to express such a large protein in cells. Jolene attempted to express HTT in <a href="https://thesgc.github.io/static-openlabnotebooks/overexpression-of-expanded-htt-in-brain-cancer-cell-lines/">two brain cancer cell lines</a>, but found limited success. I am using the same <a href="https://thesgc.github.io/static-openlabnotebooks/overexpressing-htt-in-mammalian-cells/">BacMAM virus system</a> previously established at the SGC and attempting to apply it in the widely used neuroblast cell line SH-SY5Y. The protocols for the experiments described below and additional data can be found on <a href="https://zenodo.org/record/3361015#.XUj8POhKiMo">Zenodo</a>.</p>
<p>With limited examples of the use of BacMAM in SH-SY5Y in the literature, I decided to test a range of virus amounts and included a BacMAM GFP control which I could visualize under the microscope to see how effective it was in SH-SY5Y. In line with <a href="https://thesgc.github.io/static-openlabnotebooks/overexpressing-htt-in-mammalian-cells/">Jolene&#8217;s optimization experiments</a> and a paper I found using BacMAM to overexpress wildtype and mutant LRRK2 implicated in Parkinson&#8217;s Disease  (PMID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3429506/">22952710</a>), I used up to 25% v/v of p3 (passage 3) BacMAM virus to transduce SH-SY5Y and included HEK293 for comparison.</p>
<p>The LRRK2 article (PMID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3429506/">22952710</a>) assayed cells for protein expression 48 hours post-transduction, so I thought I would get decent HTT expression within the same time-frame. I tracked GFP expression for 48 hours to get an overall idea of how effective the virus is in SH-SHY5. To detect HTT expression, I followed-up with FLAG-tag immunofluorescence (IF) in the SH-SY5Y (note: HEK293 don&#8217;t adhere very well and wash off easily so IF can&#8217;t be done with them). I find that GFP is very well expressed in both the HEK293 and SH-SY5Y cell lines after 48hrs, however, I did not detect any FLAG-tagged HTT in my IF (see RFP signal in figure below). This suggest that expression levels of the FLAG-tagged HTT from my BacMAM virus is likely too low to be detected using IF.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/BacMAM_HTT_48hrs-01-300x137.png" alt="" width="654" height="299" class="aligncenter wp-image-3285" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/BacMAM_HTT_48hrs-01-300x137.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/BacMAM_HTT_48hrs-01-768x350.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/BacMAM_HTT_48hrs-01-1024x467.png 1024w" sizes="(max-width: 654px) 100vw, 654px" /></p>
<p>I decided to repeat the transduction in a larger 24-well format and harvest cells for a western to see if I could detect any HTT expression. I also changed the harvest point to 3 days post-transduction as Jolene successfully in detected overexpressed HTT in HEK293 with a longer 3-day post-transduction expression period. I observed some SH-SHY5 cells began dying 2 days post-transduction, with more cells dying and lifting off in higher v/v of BacMAM virus. After 3 days, I was only able to harvest the cells with 5% BacMAM virus for my western as most of the cells at higher v/v BacMAM virus died.</p>
<p><img src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/HTT_combined-01-300x148.png" alt="" width="675" height="333" class="aligncenter wp-image-3284" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/HTT_combined-01-300x148.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/HTT_combined-01-768x379.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/HTT_combined-01-1024x506.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2019/08/HTT_combined-01.png 1338w" sizes="(max-width: 675px) 100vw, 675px" /></p>
<p>In my western, I detected FLAG-tagged HTT in the positive controls (D1 &#8211; 23 CAG repeats, D2 &#8211; 54 CAG repeats), but I did not detect any in both my SH-SY5Y grown for 3 days and 6 days post-transduction. It is possible that 5% v/v BacMAM virus is too low, but before I can move forward with higher v/v, I need to optimize conditions for the transduction in SH-SY5Y so that fewer cells die. The SH-SY5Y cells may be dying from either too much virus, the side effects of overexpressing HTT, or incompatibility with the Sf9 media the BacMAM virus is in. Moving forward, I will have to test these possibilities and optimize transduction conditions to maximize overexpression HTT while reducing cell death.</p>
]]></content:encoded>
					
					<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/project-overview-overexpressing-htt-in-neuronal-cells/feed/</wfw:commentRss>
			<slash:comments>0</slash:comments>
		
		
			</item>
	</channel>
</rss>
