<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>PRC2 &#8211; openlabnotebooks.org</title>
	<atom:link href="https://thesgc.github.io/static-openlabnotebooks/category/project/prc2/feed/" rel="self" type="application/rss+xml" />
	<link>http://thesgc.github.io/static-openlabnotebooks/</link>
	<description>A growing team of groundbreaking scientists around the world are now sharing their lab notebooks online</description>
	<lastBuildDate>Thu, 31 Jan 2019 23:10:29 +0000</lastBuildDate>
	<language>en-GB</language>
	<sy:updatePeriod>hourly</sy:updatePeriod>
	<sy:updateFrequency>1</sy:updateFrequency>
	<generator>https://wordpress.org/?v=4.9.9</generator>

<image>
	<url>https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/cropped-lab-jar02-32x32.png</url>
	<title>PRC2 &#8211; openlabnotebooks.org</title>
	<link>http://thesgc.github.io/static-openlabnotebooks/</link>
	<width>32</width>
	<height>32</height>
</image> 
	<item>
		<title>Light the Night Walk and detecting EZH1 in patient cells</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/light-the-night-walk-and-detecting-ezh1-in-patient-cells/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/light-the-night-walk-and-detecting-ezh1-in-patient-cells/#comments</comments>
		<pubDate>Thu, 15 Nov 2018 00:15:52 +0000</pubDate>
		<dc:creator><![CDATA[Genna Luciani]]></dc:creator>
				<category><![CDATA[Genna Luciani]]></category>
		<category><![CDATA[Leukemia]]></category>
		<category><![CDATA[PRC2]]></category>

		<guid isPermaLink="false">https://openlabnotebooks.org/?p=2145</guid>
		<description><![CDATA[Hello again! My previous post discussed how I finally had a specific antibody to EZH1 in a cell line (HEK293). As a follow up, I wanted to determine if we could detect EZH1 expression in patient cells. I&#8217;m happy to report I was able to detect the EZH1 protein in patient cells (Figure 1). Experimental <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/light-the-night-walk-and-detecting-ezh1-in-patient-cells/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Hello again! My <a href="https://openlabnotebooks.org/finally-finding-an-antibody-to-detect-ezh1-protein-expression/">previous post</a> discussed how I finally had a specific antibody to EZH1 in a cell line (HEK293). As a follow up, I wanted to determine if we could detect EZH1 expression in patient cells. I&#8217;m happy to report I was able to detect the EZH1 protein in patient cells (Figure 1). Experimental details can be found on <a href="https://zenodo.org/record/1488312#.W-yj2jhKjIU">Zenodo</a>. We can now use this antibody to determine EZH1 expression in other patient cells.</p>
<p><img class="size-medium wp-image-2151 aligncenter" src="https://openlabnotebooks.org/wp-content/uploads/2018/11/EZH1-in-patient-cells-300x185.png" alt="" width="300" height="185" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/EZH1-in-patient-cells-300x185.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/EZH1-in-patient-cells.png 756w" sizes="(max-width: 300px) 100vw, 300px" /></p>
<p style="text-align: center;">Figure 1: Western blot for EZH1 (green) and actin (red) with ~50ug protein loaded in HEK293 cells (293) and patient cells. A band of expected size (~85kDa) is seen in all samples except 150860 that we did not expect EZH1 expression.</p>
<p>Something else that I wanted to share was members of SGC Toronto (including other open notebookers <a href="http://labscribbles.com/">Rachel</a>, <a href="https://openlabnotebooks.org/category/scientist/mandeep-mann/">Mandeep</a> and <a href="https://openlabnotebooks.org/category/scientist/david-dilworth/">Dave</a>) recently participated in the Leukemia and Lymphoma Society of Canada&#8217;s (LLSC) <a href="https://www.lightthenight.ca/about-us/">Light the Night Walk</a>. The walk is the LLSC&#8217;s annual fundraising event to raise money to support those impacted by blood cancers, as well as funding for leukemia and lymphoma research. LLSC is a great organization that funds my research at the <a href="http://www.llscanada.org/someday-is-today/stories/dr-cheryl-arrowsmith">SGC</a>, including the work posted on my open notebook. Through the generosity of our supporters, team SGC was able to raise over $2000 for LLSC! It was a truly memorable experience seeing all the lanterns lighting up the night in support of fighting blood cancers and I  definitely won&#8217;t forget the touching stories we heard from survivors, patients and their loved ones battling leukemia and lymphoma.</p>
<p><img class="size-medium wp-image-2150 aligncenter" src="https://openlabnotebooks.org/wp-content/uploads/2018/11/TeamSGC_LLSC-300x225.jpg" alt="" width="300" height="225" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/TeamSGC_LLSC-300x225.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/TeamSGC_LLSC-768x576.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/TeamSGC_LLSC.jpg 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/TeamSGC_LLSC-350x263.jpg 350w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/TeamSGC_LLSC-780x586.jpg 780w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/TeamSGC_LLSC-250x188.jpg 250w" sizes="(max-width: 300px) 100vw, 300px" /></p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/light-the-night-walk-and-detecting-ezh1-in-patient-cells/feed/</wfw:commentRss>
		<slash:comments>2</slash:comments>
		</item>
		<item>
		<title>Finally finding an antibody to detect EZH1 protein expression</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/finally-finding-an-antibody-to-detect-ezh1-protein-expression/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/finally-finding-an-antibody-to-detect-ezh1-protein-expression/#respond</comments>
		<pubDate>Wed, 26 Sep 2018 16:25:19 +0000</pubDate>
		<dc:creator><![CDATA[Genna Luciani]]></dc:creator>
				<category><![CDATA[Genna Luciani]]></category>
		<category><![CDATA[Leukemia]]></category>
		<category><![CDATA[PRC2]]></category>

		<guid isPermaLink="false">https://openlabnotebooks.org/?p=1923</guid>
		<description><![CDATA[Hello again! As a follow up to my previous post,I&#8217;ve continued the search for an antibody to detect EZH1. I&#8217;m happy to report that I finally found one. I was able to detect a band in HEK293 cells using the Cell Signaling Antibody. Using knockdowns targeting EZH1, I was able to determine that the band <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/finally-finding-an-antibody-to-detect-ezh1-protein-expression/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Hello again! As a follow up to my previous <a href="https://openlabnotebooks.org/follow-up-to-finding-an-antibody-to-detect-ezh1-protein-expression-the-search-continues/">post</a>,I&#8217;ve continued the search for an antibody to detect EZH1. I&#8217;m happy to report that I finally found one. I was able to detect a band in HEK293 cells using the Cell Signaling Antibody. Using knockdowns targeting EZH1, I was able to determine that the band detected is specific to EZH1 (Figure 1). Compared with control knockdown, I see a decreased signal in the band that corresponds to EZH1 in size using both A7 and A9 knockdowns (Figure 1A) but not the related EZH2 protein (Figure 1B) with the A9 knockdown. A7 knockdown shows some decrease in EZH2 expression and is likely due to off target affect of the knockdown.  Full details are available on <a href="https://zenodo.org/record/1435780#.W6uYcntKjIU">Zenodo. </a></p>
<p><img class="wp-image-1924 aligncenter" src="https://openlabnotebooks.org/wp-content/uploads/2018/09/EZH1-knockdown-300x121.png" alt="" width="342" height="138" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/09/EZH1-knockdown-300x121.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/09/EZH1-knockdown-768x309.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/09/EZH1-knockdown-1024x411.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/09/EZH1-knockdown.png 1431w" sizes="(max-width: 342px) 100vw, 342px" /></p>
<p style="text-align: center;">Figure 1: Western blots for (A) EZH1 and (B) EZH2 in HEK293 cells with control (shLuc2) or EZH1 (shEZH1 A9 or A7) knockdown.</p>
<p>Now that we have an antibody for EZH1, I will test it in our AML patient cells.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/finally-finding-an-antibody-to-detect-ezh1-protein-expression/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Follow up to finding an antibody to detect EZH1 protein expression&#8230;the search continues</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/follow-up-to-finding-an-antibody-to-detect-ezh1-protein-expression-the-search-continues/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/follow-up-to-finding-an-antibody-to-detect-ezh1-protein-expression-the-search-continues/#comments</comments>
		<pubDate>Thu, 02 Aug 2018 21:16:23 +0000</pubDate>
		<dc:creator><![CDATA[Genna Luciani]]></dc:creator>
				<category><![CDATA[Genna Luciani]]></category>
		<category><![CDATA[Leukemia]]></category>
		<category><![CDATA[PRC2]]></category>

		<guid isPermaLink="false">https://openlabnotebooks.org/?p=1669</guid>
		<description><![CDATA[Following up on my previous post, I tested two antibodies I was planning on trying from Abcam and Novus that were used in previous publications. This time I tested directly in patient sample lysates and either saw no band, or many nonspecific bands (Figure 1). I have since reached out to the Orlando lab who <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/follow-up-to-finding-an-antibody-to-detect-ezh1-protein-expression-the-search-continues/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Following up on my previous <a href="https://openlabnotebooks.org/not-quite-finding-an-antibody-to-detect-ezh1-protein-expression/">post</a>, I tested two antibodies I was planning on trying from Abcam and Novus that were used in previous publications. This time I tested directly in patient sample lysates and either saw no band, or many nonspecific bands (Figure 1). I have since reached out to the Orlando lab who recently published a paper (<a href="https://www.ncbi.nlm.nih.gov/pubmed/28346433">PMID: 28346433</a>) where they looked at EZH1 expression levels by Western. A post doc in their lab (Peng Liu) informed me that they were also using the EZH1 antibody from Abcam that is no longer available, but recommended that I try a Cell Signaling Technologies antibody. That is what I will be doing next!</p>
<p><img class="size-medium wp-image-1670 aligncenter" src="https://openlabnotebooks.org/wp-content/uploads/2018/08/EZH1-part-2-fig-1-300x165.png" alt="" width="300" height="165" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/08/EZH1-part-2-fig-1-300x165.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/08/EZH1-part-2-fig-1-768x422.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/08/EZH1-part-2-fig-1-1024x562.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/08/EZH1-part-2-fig-1.png 1249w" sizes="(max-width: 300px) 100vw, 300px" /></p>
<p style="text-align: center;">Figure 1: Western blots using actin (red) and EZH1 antibodies from (A) Novus and (B) Abcam (ab176115). An EZH1 band was not detected using Novus (A) and many nonspecific bands were seen with Abcam (B).</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/follow-up-to-finding-an-antibody-to-detect-ezh1-protein-expression-the-search-continues/feed/</wfw:commentRss>
		<slash:comments>2</slash:comments>
		</item>
		<item>
		<title>(Not quite) finding an antibody to detect EZH1 protein expression</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/not-quite-finding-an-antibody-to-detect-ezh1-protein-expression/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/not-quite-finding-an-antibody-to-detect-ezh1-protein-expression/#comments</comments>
		<pubDate>Thu, 26 Jul 2018 20:58:13 +0000</pubDate>
		<dc:creator><![CDATA[Genna Luciani]]></dc:creator>
				<category><![CDATA[Genna Luciani]]></category>
		<category><![CDATA[Leukemia]]></category>
		<category><![CDATA[PRC2]]></category>

		<guid isPermaLink="false">https://openlabnotebooks.org/?p=1573</guid>
		<description><![CDATA[Hello again! Here is my first attempt to find an EZH1 antibody. I have mainly focused on EZH2 in the PRC2 complex as the methyltransferase responsible for the H3K27me3 mark. However, EZH1 can functionally substitute for EZH2 in PRC2. We have a good antibody to detect EZH2 levels in patient AML samples. However, for some <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/not-quite-finding-an-antibody-to-detect-ezh1-protein-expression/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Hello again! Here is my first attempt to find an EZH1 antibody. I have mainly focused on EZH2 in the <a href="https://openlabnotebooks.org/project-overview-testing-polycomb-repressive-complex-2-prc2-inhibitors-in-acute-myeloid-leukemia/">PRC2</a> complex as the methyltransferase responsible for the H3K27me3 mark. However, EZH1 can functionally substitute for EZH2 in PRC2. We have a good antibody to detect EZH2 levels in patient AML samples. However, for some samples that we see detectable levels of H3K27me3 mark but no EZH2 expression, we suspect that EZH1 might be substituting for EZH2. We would like to get a good EZH1 antibody to determine expression levels of this protein in patient samples. I tested three antibodies for EZH1 using western blot, and despite some initial results of detecting a potential band for EZH1 in brain tissue with an Active Motif antibody (Figure 1A), we were unable to see it in patient cells (Figure 1B). Full experimental detail can be found on <a href="https://zenodo.org/record/1322109#.W1odstVKjIU">Zenodo</a>. Also listed in my experimental details is our plan moving forward to obtain an EZH1 antibody!</p>
<p><img class="alignnone  wp-image-1579 aligncenter" src="https://openlabnotebooks.org/wp-content/uploads/2018/07/opennotebook-post-jul-2018-2-300x191.png" alt="" width="349" height="222" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/07/opennotebook-post-jul-2018-2-300x191.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/07/opennotebook-post-jul-2018-2-768x489.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/07/opennotebook-post-jul-2018-2-1024x652.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/07/opennotebook-post-jul-2018-2.png 1314w" sizes="(max-width: 349px) 100vw, 349px" /></p>
<p style="text-align: center;">Figure 1: Western blots using Active Motif EZH1 antibody (green) and actin (red) loading control in (A) brain lysate, (B) AML cell line (UCSD-AML1) and patient sample (151077)we expect would express EZH1. We detect a faint band of the expected size (~85kDa) in (A) but do not detect any band in (B).</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/not-quite-finding-an-antibody-to-detect-ezh1-protein-expression/feed/</wfw:commentRss>
		<slash:comments>3</slash:comments>
		</item>
		<item>
		<title>Optimizing methodology for the detection of H3K27me3 levels using flow cytometry</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/optimizing-methodology-for-the-detection-of-h3k27me3-levels-using-flow-cytometry/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/optimizing-methodology-for-the-detection-of-h3k27me3-levels-using-flow-cytometry/#comments</comments>
		<pubDate>Tue, 29 May 2018 21:47:45 +0000</pubDate>
		<dc:creator><![CDATA[Genna Luciani]]></dc:creator>
				<category><![CDATA[Genna Luciani]]></category>
		<category><![CDATA[Leukemia]]></category>
		<category><![CDATA[PRC2]]></category>

		<guid isPermaLink="false">https://openlabnotebooks.org/?p=1275</guid>
		<description><![CDATA[Hello again! In previous posts I talked about how we were using a co-culture system to grow our leukemic cells. Although this is a great way to maintain cells, it can complicate some analyses where we cannot distinguish mouse (stroma) from human (AML) cells. We wanted to determine H3K27me3 levels (the mark that EZH2 is <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/optimizing-methodology-for-the-detection-of-h3k27me3-levels-using-flow-cytometry/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Hello again! In <a href="https://openlabnotebooks.org/generation-of-fluorescently-labelled-op9-stromal-cells-for-growing-patient-leukemic-cells/">previous</a> <a href="https://openlabnotebooks.org/determining-cd34-expression-in-the-presence-and-absence-of-stroma/">posts</a> I talked about how we were using a co-culture system to grow our leukemic cells. Although this is a great way to maintain cells, it can complicate some analyses where we cannot distinguish mouse (stroma) from human (AML) cells. We wanted to determine H3K27me3 levels (the mark that EZH2 is responsible for) in the AML patient cells we are working with. Usually this can be performed using the western blots. However, the antibody for the mark recognizes both human and mouse H3K27me3. To overcome this, we wanted to see if we could determine H3K27me3 levels using flow cytometry where we can distinguish the human cells from the mouse cells using the human CD45 marker. Experimental details can be found at <a href="https://zenodo.org/record/1253386#.WwiN16QvzIU">Zenodo</a>.</p>
<p><img class="size-medium wp-image-1277 aligncenter" src="https://openlabnotebooks.org/wp-content/uploads/2018/05/7d-h3k27-300x218.png" alt="" width="300" height="218" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/05/7d-h3k27-300x218.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/05/7d-h3k27-768x557.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/05/7d-h3k27-1024x743.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/05/7d-h3k27.png 1048w" sizes="(max-width: 300px) 100vw, 300px" /></p>
<p>Figure 1: Histogram of H3K27me3 levels in CD45 positive OCI-AML-20 with (EPZ6438; red and orange) and without (DMSO; blue and light blue) 7 days of EZH2 inhibitor treatment. Number of events (cells) is displayed, as well as median fluorescent intensity (MFI) for cell populations. Unstained show almost no fluorescence. DMSO treated cells show a substantial fluorescence signal that is greatly reduced in the presence of EZH2 inhibitor.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/optimizing-methodology-for-the-detection-of-h3k27me3-levels-using-flow-cytometry/feed/</wfw:commentRss>
		<slash:comments>1</slash:comments>
		</item>
		<item>
		<title>Determining the plasticity of CD34 expression</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/determining-the-plasticity-of-cd34-expression/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/determining-the-plasticity-of-cd34-expression/#respond</comments>
		<pubDate>Wed, 04 Apr 2018 18:44:28 +0000</pubDate>
		<dc:creator><![CDATA[Genna Luciani]]></dc:creator>
				<category><![CDATA[Genna Luciani]]></category>
		<category><![CDATA[Leukemia]]></category>
		<category><![CDATA[PRC2]]></category>

		<guid isPermaLink="false">https://openlabnotebooks.org/?p=1002</guid>
		<description><![CDATA[Previously, we established that OCI-AML-20 requires stroma to maintain CD34 expression (https://zenodo.org/record/1187083#.Wr5-1C7wbIV). In this experiment, we wanted to see whether those cells grown in suspension are able to regain CD34 expression when cells are reintroduced to stroma. We find that OCI-AML-20 regains CD34 expression when put back on to stroma (either MS5 or OP9). Full <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/determining-the-plasticity-of-cd34-expression/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Previously, we established that OCI-AML-20 requires stroma to maintain CD34 expression (<a href="https://zenodo.org/record/1187083#.Wr5-1C7wbIV">https://zenodo.org/record/1187083#.Wr5-1C7wbIV</a>). In this experiment, we wanted to see whether those cells grown in suspension are able to regain CD34 expression when cells are reintroduced to stroma. We find that OCI-AML-20 regains CD34 expression when put back on to stroma (either MS5 or OP9). Full experimental details can be found at <a href="https://zenodo.org/record/1212577#.WsUXhy7wbIU">Zenodo</a>.</p>
<p><img class=" wp-image-1005 aligncenter" src="https://openlabnotebooks.org/wp-content/uploads/2018/04/plasticity-300x240.png" alt="" width="363" height="290" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/04/plasticity-300x240.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/04/plasticity.png 553w" sizes="(max-width: 363px) 100vw, 363px" /></p>
<p style="text-align: center;">Figure 1: Summary of flow cytometry analysis of cells at 4 weeks. Cells that were transferred from suspension to stroma regain CD34 expression, and for CD34+CD38- we see expression levels seen with cells grown continuously on stroma.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/determining-the-plasticity-of-cd34-expression/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Determining CD34 expression in the presence and absence of stroma</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/determining-cd34-expression-in-the-presence-and-absence-of-stroma/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/determining-cd34-expression-in-the-presence-and-absence-of-stroma/#respond</comments>
		<pubDate>Fri, 02 Mar 2018 17:27:23 +0000</pubDate>
		<dc:creator><![CDATA[Genna Luciani]]></dc:creator>
				<category><![CDATA[Genna Luciani]]></category>
		<category><![CDATA[Leukemia]]></category>
		<category><![CDATA[PRC2]]></category>

		<guid isPermaLink="false">https://openlabnotebooks.org/?p=822</guid>
		<description><![CDATA[We are currently working on a manuscript to characterize one of the cell lines that we established from growing patient samples. I previously discussed how we established OCI-AML-20 and I&#8217;m currently working on some experiments to complete the manuscript. One of those experiments is determining whether the CD34 expression seen in OCI-AML-20 relies on the <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/determining-cd34-expression-in-the-presence-and-absence-of-stroma/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>We are currently working on a manuscript to characterize one of the cell lines that we established from growing patient samples. I previously discussed how we <a href="https://openlabnotebooks.org/report-culturing-patient-leukemic-cells/">established OCI-AML-20</a> and I&#8217;m currently working on some experiments to complete the manuscript. One of those experiments is determining whether the CD34 expression seen in OCI-AML-20 relies on the presence of stroma. We found that OCI-AML-20 relies on stroma (either OP9 or MS5) for CD34 expression (Figure 1). For experimental details, visit <a href="https://zenodo.org/record/1187083#.WpltoYPwbIU">Zenodo</a>.</p>
<p><img class="size-medium wp-image-826 aligncenter" src="https://openlabnotebooks.org/wp-content/uploads/2018/03/stroma-1-300x163.png" alt="" width="300" height="163" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/03/stroma-1-300x163.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/03/stroma-1.png 764w" sizes="(max-width: 300px) 100vw, 300px" /></p>
<p style="text-align: center;">Figure 1: Summary of CD34 and CD38 expression in OCI-AML-20 cells grown with and without stroma (OP9 and MS5) over two weeks. OCI-AML-20 require stroma for CD34 expression maintenance</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/determining-cd34-expression-in-the-presence-and-absence-of-stroma/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Report: Culturing patient leukemic cells</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/report-culturing-patient-leukemic-cells/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/report-culturing-patient-leukemic-cells/#comments</comments>
		<pubDate>Thu, 01 Mar 2018 18:31:37 +0000</pubDate>
		<dc:creator><![CDATA[Genna Luciani]]></dc:creator>
				<category><![CDATA[Genna Luciani]]></category>
		<category><![CDATA[Leukemia]]></category>
		<category><![CDATA[PRC2]]></category>

		<guid isPermaLink="false">https://openlabnotebooks.org/?p=816</guid>
		<description><![CDATA[Hello again! I have written a summary report of how we are culturing patient leukemic cells using two patients as an example, 150333 and OCI-AML-20. The report can be found at Zenodo. We are monitoring the growth of cultured cells by looking at the expression of CD34 and CD38 markers over a number of weeks. These <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/report-culturing-patient-leukemic-cells/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Hello again! I have written a summary report of how we are culturing patient leukemic cells using two patients as an example, 150333 and OCI-AML-20. The report can be found at <a href="https://zenodo.org/record/1186848#.WphD-oPwbIU">Zenodo</a>.</p>
<p>We are monitoring the growth of cultured cells by looking at the expression of CD34 and CD38 markers over a number of weeks. These are important markers for leukemic stem cells (LSCs). LSCs are cells with the capacity to self-renew and give rise to progenitor cells. The LSCs are thought to be the cells that are responsible for relapse in AML. To be able to screen chemical probes, we would like to be able to maintain CD34 expression, a marker of LSCs (CD34+CD38- are often markers of LSCs). The epigenetic probes that we are using often require long term exposure, so we wanted to determine if we could maintain CD34 expression long term to be able to treat these cells with probes.</p>
<p><img class="wp-image-817 aligncenter" src="https://openlabnotebooks.org/wp-content/uploads/2018/03/OCI-AML-20-image.png" alt="" width="214" height="215" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/03/OCI-AML-20-image.png 292w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/03/OCI-AML-20-image-150x150.png 150w" sizes="(max-width: 214px) 100vw, 214px" /></p>
<p style="text-align: center;">Figure 1: Image of OCI-AML-20 cells in culture. Image generated using IncuCyte live cell imager.</p>
<p>&nbsp;</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/report-culturing-patient-leukemic-cells/feed/</wfw:commentRss>
		<slash:comments>1</slash:comments>
		</item>
		<item>
		<title>Generation of fluorescently labelled OP9 stromal cells for growing patient leukemic cells</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/generation-of-fluorescently-labelled-op9-stromal-cells-for-growing-patient-leukemic-cells/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/generation-of-fluorescently-labelled-op9-stromal-cells-for-growing-patient-leukemic-cells/#comments</comments>
		<pubDate>Wed, 17 Jan 2018 22:37:27 +0000</pubDate>
		<dc:creator><![CDATA[Genna Luciani]]></dc:creator>
				<category><![CDATA[Genna Luciani]]></category>
		<category><![CDATA[Leukemia]]></category>
		<category><![CDATA[PRC2]]></category>

		<guid isPermaLink="false">https://openlabnotebooks.org/?p=477</guid>
		<description><![CDATA[Our experiments use cells from patients with leukemia. These primary cells are notoriously difficult to culture, especially long term. To overcome this, we are using a co-culture system where we grow the leukemia cells with mouse stromal cells (OP9) to try to mimic some aspects of the bone marrow environment present in leukemia. Although essential <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/generation-of-fluorescently-labelled-op9-stromal-cells-for-growing-patient-leukemic-cells/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Our experiments use cells from patients with leukemia. These primary cells are notoriously difficult to culture, especially long term. To overcome this, we are using a co-culture system where we grow the leukemia cells with mouse stromal cells (OP9) to try to mimic some aspects of the bone marrow environment present in leukemia. Although essential for maintenance of leukemia cells, the stromal cells can complicate many of the analyses that are required to establish the effective targeting of the leukemia cells. To get around this, we introduced fluorescent markers in the OP9 cells to easily distinguish the stromal cells from the patient cells in various applications such as microscopy and flow cytometry. For full experimental details, please visit <a href="https://zenodo.org/record/1154257#.Wl_N4FWnHIU">Zenodo</a>.</p>
<p><img class="alignnone size-medium wp-image-479" src="http://openlabnotebooks.org/wp-content/uploads/2018/01/GFP-stromal-cells-300x225.png" alt="" width="300" height="225" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/GFP-stromal-cells-300x225.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/GFP-stromal-cells-768x575.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/GFP-stromal-cells-350x263.png 350w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/GFP-stromal-cells-780x586.png 780w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/GFP-stromal-cells-250x188.png 250w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/GFP-stromal-cells.png 865w" sizes="(max-width: 300px) 100vw, 300px" /> <img class="alignnone size-medium wp-image-480" src="http://openlabnotebooks.org/wp-content/uploads/2018/01/mcherry-stromal-cells-300x224.png" alt="" width="300" height="224" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/mcherry-stromal-cells-300x224.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/mcherry-stromal-cells-768x574.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/mcherry-stromal-cells-350x263.png 350w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/mcherry-stromal-cells-780x586.png 780w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/mcherry-stromal-cells-250x188.png 250w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/mcherry-stromal-cells.png 865w" sizes="(max-width: 300px) 100vw, 300px" /></p>
<p>Figure 1: mCherry and GFP labelled OP9 cells. Image generated using IncuCyte live cell imager.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/generation-of-fluorescently-labelled-op9-stromal-cells-for-growing-patient-leukemic-cells/feed/</wfw:commentRss>
		<slash:comments>1</slash:comments>
		</item>
		<item>
		<title>Project Overview: Testing polycomb repressive complex 2 (PRC2) inhibitors in acute myeloid leukemia</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/project-overview-testing-polycomb-repressive-complex-2-prc2-inhibitors-in-acute-myeloid-leukemia/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/project-overview-testing-polycomb-repressive-complex-2-prc2-inhibitors-in-acute-myeloid-leukemia/#comments</comments>
		<pubDate>Mon, 15 Jan 2018 18:13:36 +0000</pubDate>
		<dc:creator><![CDATA[Genna Luciani]]></dc:creator>
				<category><![CDATA[Genna Luciani]]></category>
		<category><![CDATA[Leukemia]]></category>
		<category><![CDATA[PRC2]]></category>

		<guid isPermaLink="false">https://openlabnotebooks.org/?p=414</guid>
		<description><![CDATA[Acute myeloid leukemia or AML is a type of cancer that is the result of abnormal division and development of blood cells. AML has very poor outcomes, with 5-year survival of less than 50 percent and even less in patients over the age of 60 [1]. Within AML, there are many subgroups that can dictate <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/project-overview-testing-polycomb-repressive-complex-2-prc2-inhibitors-in-acute-myeloid-leukemia/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Acute myeloid leukemia or AML is a type of cancer that is the result of abnormal division and development of blood cells. AML has very poor outcomes, with 5-year survival of less than 50 percent and even less in patients over the age of 60 [<a href="#_ENREF_1">1</a>]. Within AML, there are many subgroups that can dictate prognosis and potential treatment options. A group of particularly poor prognosis patients are those with an inversion in chromosome 3 (denoted as inv(3)(q21q26.2) or t(3;3)(q21;q26.2)) [<a href="#_ENREF_2">2</a>]. The inversion causes activation of a cancer promoting gene, EVI1 (Figure 1) [<a href="#_ENREF_3">3</a>]. We have focused our study on understanding how EVI1 causes AML and identifying potential novel therapies for the inv(3) group because these poor prognosis patients do not respond to the current treatment options.</p>
<p><img class="alignnone wp-image-422 " src="http://openlabnotebooks.org/wp-content/uploads/2018/01/inv3_1-1-1024x259.png" alt="" width="634" height="160" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/inv3_1-1-1024x259.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/inv3_1-1-300x76.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/inv3_1-1-768x194.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/inv3_1-1.png 1218w" sizes="(max-width: 634px) 100vw, 634px" /></p>
<p>Figure 1: Inversion 3 causes an enhancer that normally allows the expression of GATA2 to rearrange, leading to inappropriate expression of EVI1.</p>
<p>Inv(3) can co-occur with a number of other mutations that contribute to the poor prognosis such as loss of chromosome 7. Chromosome 7 carries a <a href="http://atlasgeneticsoncology.org/Indexbychrom/idxa_7.html">number of genes</a> and one of those genes is enhancer of zeste 2 (EZH2) [<a href="#_ENREF_4">4</a>] that functions as a methyltransferase in the polycomb repressive complex 2 (PRC2) (Figure 2). The PRC2 complex comprises of a number proteins that coordinate the regulation of gene expression by adding methyl groups at lysine 27 of histone 3 (H3K27) (Figure 2) [<a href="#_ENREF_5">5</a>].  It has been reported that EVI1 recruits PRC2 to repress an important tumor suppressor gene, PTEN [<a href="#_ENREF_6">6</a>].</p>
<p><img class="alignnone wp-image-416" src="http://openlabnotebooks.org/wp-content/uploads/2018/01/ezh2-286x300.png" alt="" width="311" height="326" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/ezh2-286x300.png 286w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/ezh2-768x806.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/ezh2.png 778w" sizes="(max-width: 311px) 100vw, 311px" /></p>
<p>Figure 2: PRC2 complex is responsible for the methylation of lysine 27 on histone 3. Chemical probes/inhibitors to PRC2 (A395, UNC1999 and EPZ6438) will be used in this study.</p>
<p>We are interested in how the oncogenic protein EVI1 interacts with the PRC2 complex components and how modulating the PRC2 by using chemical probes affects leukemia. This blog will follow our experiments to further examine the PRC2 and EVI1 interactions in inv(3) AML with or without deletion of chromosome 7, elucidate the importance of these molecules in leukemia cell survival and finally investigate the therapeutic potential of targeting these complexes to cure inv(3) AML.</p>
<ol>
<li>Gregory, T.K., et al., <em>Molecular prognostic markers for adult acute myeloid leukemia with normal cytogenetics.</em> J Hematol Oncol, 2009. <strong>2</strong>: p. 23.</li>
<li>Lugthart, S., et al., <em>Clinical, molecular, and prognostic significance of WHO type inv(3)(q21q26.2)/t(3;3)(q21;q26.2) and various other 3q abnormalities in acute myeloid leukemia.</em> J Clin Oncol, 2010. <strong>28</strong>(24): p. 3890-8.</li>
<li>Glass, C., et al., <em>The role of EVI1 in myeloid malignancies.</em> Blood Cells Mol Dis, 2014. <strong>53</strong>(1-2): p. 67-76.</li>
<li>Lavallee, V.P., et al., <em>EVI1-rearranged acute myeloid leukemias are characterized by distinct molecular alterations.</em> Blood, 2015. <strong>125</strong>(1): p. 140-3.</li>
<li>Greenblatt, S.M. and S.D. Nimer, <em>Chromatin modifiers and the promise of epigenetic therapy in acute leukemia.</em> Leukemia, 2014. <strong>28</strong>(7): p. 1396-406.</li>
<li>Yoshimi, A., et al., <em>Evi1 represses PTEN expression and activates PI3K/AKT/mTOR via interactions with polycomb proteins.</em> Blood, 2011. <strong>117</strong>(13): p. 3617-28.</li>
</ol>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/project-overview-testing-polycomb-repressive-complex-2-prc2-inhibitors-in-acute-myeloid-leukemia/feed/</wfw:commentRss>
		<slash:comments>1</slash:comments>
		</item>
	</channel>
</rss>
