<?xml version="1.0" encoding="UTF-8"?><rss version="2.0"
	xmlns:content="http://purl.org/rss/1.0/modules/content/"
	xmlns:wfw="http://wellformedweb.org/CommentAPI/"
	xmlns:dc="http://purl.org/dc/elements/1.1/"
	xmlns:atom="http://www.w3.org/2005/Atom"
	xmlns:sy="http://purl.org/rss/1.0/modules/syndication/"
	xmlns:slash="http://purl.org/rss/1.0/modules/slash/"
	>

<channel>
	<title>Catalyst UNC Chapel Hill &#8211; openlabnotebooks.org</title>
	<atom:link href="https://thesgc.github.io/static-openlabnotebooks/category/laboratory/unc-catalyst/feed/" rel="self" type="application/rss+xml" />
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<description>A growing team of groundbreaking scientists around the world are now sharing their lab notebooks online</description>
	<lastBuildDate>Fri, 07 Feb 2020 17:16:22 +0000</lastBuildDate>
	<language>en-GB</language>
	<sy:updatePeriod>hourly</sy:updatePeriod>
	<sy:updateFrequency>1</sy:updateFrequency>
	<generator>https://wordpress.org/?v=4.9.13</generator>

<image>
	<url>https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/01/cropped-lab-jar02-32x32.png</url>
	<title>Catalyst UNC Chapel Hill &#8211; openlabnotebooks.org</title>
	<link>https://thesgc.github.io/static-openlabnotebooks/</link>
	<width>32</width>
	<height>32</height>
</image> 
	<item>
		<title>Last post on Split-SCN1A</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/last-post-on-split-scn1a/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/last-post-on-split-scn1a/#respond</comments>
		<pubDate>Sun, 10 Feb 2019 22:47:17 +0000</pubDate>
		<dc:creator><![CDATA[Wen Yih Aw]]></dc:creator>
				<category><![CDATA[Catalyst UNC Chapel Hill]]></category>
		<category><![CDATA[Dravet syndrome]]></category>
		<category><![CDATA[Gene therapy for Dravet Syndrome]]></category>
		<category><![CDATA[Wen Yih Aw]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2452</guid>
		<description><![CDATA[The idea for split-intein based SCN1A reconstitution started when I first learned of the split-Cas9 system. Long story short, packaging Cas9 with a guideRNA sequence into an AAV vector is pushing the packaging capacity of AAV vector. To circumvent this issue, the authors design a split-Cas9 system where the coding sequence for Cas9 is split <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/last-post-on-split-scn1a/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>The idea for split-intein based SCN1A reconstitution started when I first learned of the split-Cas9 system. Long story short, packaging Cas9 with a guideRNA sequence into an AAV vector is pushing the packaging capacity of AAV vector. To circumvent this issue, the authors design a split-Cas9 system where the coding sequence for Cas9 is split into two fragments, each fused to a split-inteins sequence. When co-expressed in cells, the split-inteins can function as glue joining two protein fragments together.</p>
<p>In an attempt to “shrink” SCN1A to be packageable in AAV vectors, I ran into several design and cloning issues, ultimately leading to the decision to terminate this project. Cas9 is a well-studied cytoplasmic protein with structural information while SCN1A is a large multi-pass transmembrane protein that weaves in and out of the plasma membrane. While success packaging and application of the split-Cas9 system for gene editing is encouraging, my attempt to translate similar design to SCN1A, without structural information, proves to be an oversight. Design aside, the coding sequence for human SCN1A is difficult to maintain. The SCN1A cDNA sequence tends to accumulate deletions and rearranges when transformed and propagated in E. coli. One way to improve plasmid stability and reduce unwanted rearrangement is by propagating plasmid in E. coli strains with recA1 genotype at room temperature. recA1 confers a loss of function in RecA1, a protein that is essential for homologous recombinational DNA repair in E. coli. While this strategy helps to alleviate the problem, it is time-consuming and eventually, I decided to make use of a codon-optimized human SCN1A cDNA as the cloning template for split-SCN1A constructs. While this solves the plasmid stability issue, the codon-optimized cDNA has not been well characterized and while the proteins expressed reconstituted, they failed to traffic to the correct cellular compartment. These raise an issue that needs to be addressed – how to propagate mutation-free gene therapy vectors with SCN1A sequence in large scale, have any of us managed to move the project to treatment pipeline.</p>
<p>Thank you all for reading. While this project didn&#8217;t work out as planned, I truly believe that this is an exciting time for Dravet Syndrome. With recent positive Fenfluramine clinical trial results, exciting new drugs in the pipeline, and cross-continental efforts in pushing various gene therapy approaches for treating Dravet, I am hopeful that we can see these exciting results moving into medical treatment in the foreseeable future.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/last-post-on-split-scn1a/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Checking expression and localization of split-SCN1A constructs in HEK293 cells</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/checking-expression-and-localization-of-split-scn1a-constructs-in-hek293-cells/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/checking-expression-and-localization-of-split-scn1a-constructs-in-hek293-cells/#respond</comments>
		<pubDate>Sun, 18 Nov 2018 19:41:36 +0000</pubDate>
		<dc:creator><![CDATA[Wen Yih Aw]]></dc:creator>
				<category><![CDATA[Catalyst UNC Chapel Hill]]></category>
		<category><![CDATA[Dravet syndrome]]></category>
		<category><![CDATA[Gene therapy for Dravet Syndrome]]></category>
		<category><![CDATA[Wen Yih Aw]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2163</guid>
		<description><![CDATA[In my last posts, I have constructed a pair of split-SCN1A constructs (split- SCN1A-GFP(1-10) and SCN1A-GFP(11)). Here, I tested the functionality of these constructs by looking at whether these split- constructs reconstitute into a full-length SCN1A and generate a fluorescent signal when co-transfected in HEK293 cells, and if so, do the reconstituted SCN1A (with an <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/checking-expression-and-localization-of-split-scn1a-constructs-in-hek293-cells/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>In my last posts, I have constructed a pair of split-SCN1A constructs (split- SCN1A-GFP<sub>(1-10)</sub> and SCN1A-GFP<sub>(11)</sub>). Here, I tested the functionality of these constructs by looking at whether these split- constructs reconstitute into a full-length SCN1A and generate a fluorescent signal when co-transfected in HEK293 cells, and if so, do the reconstituted SCN1A (with an internal GFP tag) localize correctly at the plasma membrane.</p>
<p>Below are my imaging results. The detailed experimental procedure can be found at <a href="https://zenodo.org/record/1490851#.W_G_bpNKh3g">https://zenodo.org/record/1490851#.W_G_bpNKh3g</a></p>
<p><img class="aligncenter size-large wp-image-2165" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/dravet-pic5-1024x752.jpg" alt="" width="780" height="573" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/dravet-pic5-1024x752.jpg 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/dravet-pic5-300x220.jpg 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/dravet-pic5-768x564.jpg 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/dravet-pic5.jpg 1424w" sizes="(max-width: 780px) 100vw, 780px" /></p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p><img class="aligncenter size-large wp-image-2164" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/dravet-pic6-1024x404.png" alt="" width="780" height="308" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/dravet-pic6-1024x404.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/dravet-pic6-300x118.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/11/dravet-pic6-768x303.png 768w" sizes="(max-width: 780px) 100vw, 780px" /></p>
<p>&nbsp;</p>
<p>In short, these split-constructs managed to find each other and reconstitute a GFP signal in HEK293 cells. However, the GFP signals do not overlap with cell cortex or plasma membrane markers (cell cortex marked by cortical actin staining and plasma membrane marked by CD8, a transmembrane glycoprotein.) The distribution of reconstituted SCN1A-GFP signals resembles misfolded proteins that are trapped in the ER.  Co-expression of SCN1B, a beta subunit which is thought to promote sodium channel trafficking, does not seem to improve the localization of reconstituted SCN1A-GFP.</p>
<p>There are several caveats with the design of these constructs which might have affected folding and trafficking of reconstituted SCN1A: 1) instead of using the canonical SCN1A cDNA sequence (which is  unstable and prone to mutagenize when propagated in E. coli), a codon-optimized SCN1A cDNA was used as a template in cloning these constructs and 2) the reconstituted protein has a foreign GFP sequence which might have interfered with SCN1A trafficking. Moving forward, I will revisit and examine the localization of codon optimized SCN1A in HEK293 cells while continue constructing the rest of the split-intein constructs.</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/checking-expression-and-localization-of-split-scn1a-constructs-in-hek293-cells/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Design of split-intein mediated SCN1A protein trans-splicing</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/2047-2/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/2047-2/#respond</comments>
		<pubDate>Mon, 29 Oct 2018 00:28:39 +0000</pubDate>
		<dc:creator><![CDATA[Wen Yih Aw]]></dc:creator>
				<category><![CDATA[Catalyst UNC Chapel Hill]]></category>
		<category><![CDATA[Dravet syndrome]]></category>
		<category><![CDATA[Gene therapy for Dravet Syndrome]]></category>
		<category><![CDATA[Wen Yih Aw]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=2047</guid>
		<description><![CDATA[Previously, I provided a brief overview on SCN1A related Dravet Syndrome (DS) and the need for larger gene therapy vectors or alternative methods to deliver functional copies of SCN1A to affected cells in DS. The goal of my project is to explore one of these alternative methods, specifically a split-intein mediated protein trans-splicing method, that <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/2047-2/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Previously, I provided a brief overview on SCN1A related Dravet Syndrome (DS) and the need for larger gene therapy vectors or alternative methods to deliver functional copies of SCN1A to affected cells in DS. The goal of my project is to explore one of these alternative methods, specifically a split-intein mediated protein trans-splicing method, that could reduce the size of the passenger (SCN1A expression cassettes) for gene therapy vectors. In this post, I will talk about the overall design for this method.</p>
<p>To give you a little more background about inteins: Inteins (also known as intervening proteins) are basically mobile protein elements that are translated and embedded within another protein sequences. You can think of them as single-use, single-turnover enzymes that catalyze their own excision from a flanking precursor “exteins” protein sequences while concurrently joining the remaining exteins protein fragments together through the formation of a new peptide bond (Fig. 1).</p>
<p><img class="size-medium wp-image-2048 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/10/intein-inbase-300x185.png" alt="" width="300" height="185" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/10/intein-inbase-300x185.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/10/intein-inbase.png 736w" sizes="(max-width: 300px) 100vw, 300px" /></p>
<p style="text-align: center;">Figure 1. Intein-mediated protein splicing vs. RNA splicing.  Image source: Perler, F. B. (2002). InBase, the Intein Database. Nucleic Acids Res. 30, 383-384.</p>
<p>There are two types of inteins – cis splicing inteins that are embedded within protein sequences and catalyze protein splicing <em>in cis</em> and split- or trans-splicing inteins where the enzyme is naturally split and expressed into two parts. These two fragments of split-intein can associate with each other to form a catalytic active enzyme that can catalyze protein splicing <em>in trans. </em>Briefly, my project aims to split the SCN1A expression cassette into two parts and make use of the unique split-intein mediated trans-splicing capability to rejoin the expressed SCN1A fragments together (Fig. 2)</p>
<p><img class="size-medium wp-image-2050 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/10/dravet-pic4-216x300.png" alt="" width="216" height="300" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/10/dravet-pic4-216x300.png 216w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/10/dravet-pic4.png 279w" sizes="(max-width: 216px) 100vw, 216px" /></p>
<p style="text-align: center;">Figure 2. A basic design of split-intein mediated SCN1A protein trans-splicing</p>
<p>So, here goes the basic components and design of this cloning experiment. We are going to express SCN1A as two protein fragments (split-N terminal and split-C terminal SCN1A), each fused with split-N-term and C-term intein sequence (Fig. 2), respectively.</p>
<p>But&#8230; where do we split the protein?</p>
<p>One of the factors taken into consideration in designing the split sites is that for efficient protein trans-splicing, the split sites or the sites where split-intein sequences are inserted need to be surface exposed. SCN1A is a multipass membrane proteins with four homologous (unexposed) transmembrane domains (DI to DIV), each domain is connected to the next domain by large cytoplasmic loops (Fig. 3). In fulfilling this condition, we are left with the N and C-terminal cytoplasmic domain and the cytoplasmic loops connecting DI to DII, DII to DIII, and DIII to DIV to work with. Among these cytoplasmic regions, splitting cytoplasmic loops within and between DII to DIII can result in a significant reduction in the size of SCN1A expression cassettes.</p>
<p>Additionally, to initiate efficient splicing reaction, split-inteins require a certain combination of amino acids flanking the intein sequences. For instance, split DnaB intein from <em>Synechocystis</em> needs to be flanked by a glycine at the N-terminus of the intein and a serine at C-terminus of the intein to catalyze protein trans-splicing whereas Vma intein from <em>S. cerevisiae </em>requires a glycine and cysteine amino acids combination.  In addition to previous considerations, these specific requirements of amino acids combinations for efficient and seamless protein trans-splicing helped us further narrow down a short list of potential SCN1A split sites.</p>
<p><img class="size-medium wp-image-2051 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/10/scn1a-300x134.png" alt="" width="300" height="134" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/10/scn1a-300x134.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/10/scn1a-768x342.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/10/scn1a-1024x456.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/10/scn1a.png 1488w" sizes="(max-width: 300px) 100vw, 300px" /></p>
<p style="text-align: center;">Figure 3. A general topology of eukaryotic voltage-gated sodium channels.</p>
<p style="text-align: center;">Image source: Shen, Huaizong, et al. “Structure of a Eukaryotic Voltage-Gated Sodium Channel at near-Atomic Resolution.” <em>Science</em>, American Association for the Advancement of Science, 3 Mar. 2017, science.sciencemag.org/content/355/6328/eaal4326.long.</p>
<p>&nbsp;</p>
<p>How can we tell whether two split-SCN1A fragments trans-spliced?</p>
<p>To screen and identify constructs that have successfully reconstituted, I will attach split-GFP sequences to the N and C termini of the split-N-terminal-SCN1A intein and split-C-terminal-SCN1A-intein, respectively. Split-GFP is composed of two complementing split fluorescent protein fragments that irreversibly associate with each other to generate a fluorescent signal. When expressing both of these constructs in cells, we predicted that the split-GFP sequence would help bring the two split-proteins together (Fig. 4.) The split-intein would interact with each other to form an active enzyme and catalyze the excision of both itself and GFP protein to the cytoplasm. In addition to using the cytoplasmic GFP signal as a visual positive marker for trans-splicing, the split-GFP system could also be utilized as an alternative way to link two SCN1A fragments together.</p>
<p>So far, I have generated a pair of split-GFP construct to examine one of the split-site. You can find more details of these experiments on <a href="https://zenodo.org/record/1473859#.W9er4pNKh3g">zenodo</a>. Going forward, I will be transfecting these constructs into HEK293 cells to examine the following questions: Do these split-protein fragments reconstitute into full-length SCN1A. If so, do they localize and function similar to wild-type SCN1A? Does overproduction of split-protein fragments cause a dominant negative effect on wild-type SCN1A?</p>
<p>&nbsp;</p>
<p><img class="aligncenter size-large wp-image-2058" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/10/dravet-pic3-1024x223.png" alt="" width="780" height="170" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/10/dravet-pic3-1024x223.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/10/dravet-pic3-300x65.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/10/dravet-pic3-768x168.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/10/dravet-pic3.png 1132w" sizes="(max-width: 780px) 100vw, 780px" /></p>
<p style="text-align: center;">Figure 4. A fluorescence reporter system for identifying SCN1A trans-splicing.</p>
<p>&nbsp;</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/2047-2/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Overview &#8211; Dravet Syndrome</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/overview-dravet-syndrome/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/overview-dravet-syndrome/#comments</comments>
		<pubDate>Sun, 30 Sep 2018 00:16:04 +0000</pubDate>
		<dc:creator><![CDATA[Wen Yih Aw]]></dc:creator>
				<category><![CDATA[Catalyst UNC Chapel Hill]]></category>
		<category><![CDATA[Dravet syndrome]]></category>
		<category><![CDATA[Gene therapy for Dravet Syndrome]]></category>
		<category><![CDATA[Wen Yih Aw]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=1930</guid>
		<description><![CDATA[Hello Open Sciences World! In my very first post for the SGC’s Extreme Open Sciences project, I will start by providing an overview of my project – Gene therapy for Dravet Syndrome. Dravet Syndrome is a catastrophic, rare form of infantile epilepsy. The disease onset occurs in the first year of life where children with <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/overview-dravet-syndrome/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Hello Open Sciences World! In my very first post for the SGC’s Extreme Open Sciences project, I will start by providing an overview of my project – Gene therapy for Dravet Syndrome.</p>
<p>Dravet Syndrome is a catastrophic, rare form of infantile epilepsy. The disease onset occurs in the first year of life where children with Dravet Syndrome suffer from frequent and prolonged seizures, cognitive and motor dysfunction and a substantially elevated risk of coma and death. There is currently no cure for Dravet Syndrome. The current standard of care involves using anti-epileptic drugs to relieve its symptoms, but, unfortunately, most anti-epileptic drugs are not very effective.</p>
<p>The majority (&gt;90%) of Dravet Syndrome is caused by mutations in SCN1A gene. This gene encodes a complex pore-forming protein that forms voltage-gated sodium channels on the surface of our neurons. The primary function of these channels is to initiate and transmit electrochemical communicative signals in neurons. They open in response to changes in electrical membrane potential across cell membranes and enable the influx of sodium ions into our neurons. Similar to the electrical currents that power our batteries, you can think of the inflow of ions as electrochemical signals that powers our neural circuit.</p>
<p>Most Dravet Syndrome related mutations result in haploinsufficiency of SCN1A where one copy of SCN1A does not function and having one healthy functioning copy of SCN1A gene is not enough for normal cellular function. One way to treating loss of function or haploinsufficiency disorders is via gene therapy where we make use of a vector, an engineered virus, to deliver back-up functional copies of genes into affected cells to increase the production and to restore functions of the missing proteins. While gene therapy methods have been approved to treat retinal dystrophy and several other diseases, the development of gene therapy for Dravet Syndrome remains a challenge. The engineered virus that we use as a vehicle to deliver gene can only fit ~4.7kb of DNA and SCN1A (~6kb) is too big for this vehicle.</p>
<p>My research focuses on developing a split-intein protein “gluing” system where the SCN1A gene can be split into two smaller fragments that are sufficiently small to be packaged into common gene therapy vehicles. Upon delivery in cells, two fragments of SCN1A protein, each fused to a split-intein, will be expressed. Inteins are protein sequences that can catalyze self-excision and concurrently glue its flanking protein fragments back together. When two fragments of SCN1A proteins fused to split-intein are next to each other, the split-intein can glue two parts of the proteins together, thereby reconstituting functional SCN1A proteins in targeted cells.</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/overview-dravet-syndrome/feed/</wfw:commentRss>
		<slash:comments>4</slash:comments>
		</item>
		<item>
		<title>Screening EGFR and GAK Inhibitors in Chordoma</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/screening-egfr-and-gak-inhibitors-in-chordoma/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/screening-egfr-and-gak-inhibitors-in-chordoma/#respond</comments>
		<pubDate>Thu, 24 May 2018 16:40:59 +0000</pubDate>
		<dc:creator><![CDATA[Kaleb Naegeli]]></dc:creator>
				<category><![CDATA[Catalyst UNC Chapel Hill]]></category>
		<category><![CDATA[Chordoma]]></category>
		<category><![CDATA[EGFR and GAK]]></category>
		<category><![CDATA[Kaleb Naegeli]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=1259</guid>
		<description><![CDATA[A major therapeutic avenue for chordoma is targeting the receptor tyrosine kinase, EGFR. Its ligand, EGF, is overexpressed as a transcriptional target of Brachyury and polysomy of EGFR’s region of chromosome 7 is frequent in chordoma. To this extent I, in collaboration with SGC-UNC’s post-doc Chris Asquith, am working on the profiling of a number <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/screening-egfr-and-gak-inhibitors-in-chordoma/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>A major therapeutic avenue for chordoma is targeting the receptor tyrosine kinase, EGFR. Its ligand, EGF, is overexpressed as a transcriptional target of Brachyury and polysomy of EGFR’s region of chromosome 7 is frequent in chordoma. To this extent I, in collaboration with SGC-UNC’s post-doc Chris Asquith, am working on the profiling of a number of EGFR inhibitors. Of particular interest, though, is this: many commercial EGFR inhibitors also hit another kinase – GAK (cyclin-G associated kinase). GAK plays a major role in prostate cancer, but its role in chordoma is unknown. Based on gene expression data from the Chordoma Foundation, we do know GAK is expressed in most chordoma cell lines.</p>
<p>First, we screened a set of published EGFR inhibitors, some of which also have GAK affinity, on the chordoma cell lines U-CH1 and U-CH2 (Table 1). Most of these compounds have strong potency (&lt;1 μM) on U-CH1 but not as strong of potency on U-CH2; this is expected, as previous work indicates U-CH2 is more resistant to EGFR inhibitors<sup>1</sup>. Following this initial screen, Chris has developed and I am characterizing a new set of compounds. This is a useful set because some of them have high affinity for EGFR, some have high affinity for GAK, and others show relatively equal affinity for both kinases. Using these compounds will allow us to interrogate if GAK plays a role in chordoma.</p>
<p>I performed a series of experiments to assess how much of each compound was required to kill (or at least stop the growth of) chordoma cells (U-CH1 and U-CH2) as well as normal skin fibroblasts (WS1). An ideal therapeutic would have a high potency for U-CH1 and U-CH2 without being toxic for WS1. A number of compounds have low values in U-CH1 but, like the published compounds in referenced above, do not show activity in U-CH2. A few compounds show activity in both U-CH1 and U-CH2, but they tend to also show similar potency in WS1. The mechanisms of these compounds warrant further investigation, as they will continue to help us understand potential interplay between EGFR and GAK, but their toxicity in normal skin cells may indicate they aren’t great therapeutic candidates.</p>
<p><img class="wp-image-1261 aligncenter" src="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/05/20180522-Table2-300x241.png" alt="" width="348" height="280" srcset="https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/05/20180522-Table2-300x241.png 300w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/05/20180522-Table2-768x617.png 768w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/05/20180522-Table2-1024x823.png 1024w, https://thesgc.github.io/static-openlabnotebooks/wp-content/uploads/2018/05/20180522-Table2.png 1501w" sizes="(max-width: 348px) 100vw, 348px" /></p>
<p style="text-align: center;"><strong>Table 1</strong>: Compound potency in chordoma cells and normal fibroblasts in vitro. Values shown are EC<sub>50</sub> in μM.</p>
<p>Going forward, I am now trying to work out the mechanism of action of a subset of these compounds: do they kill the cells or just stop their growth? Does specific genetic knockdown of GAK produce similar phenotypes? These data will help us understand if GAK has a role in chordoma and how we can therapeutically target this kinase.</p>
<p>You can read greater detail about these experiments, their methodology, and see more data on <a href="https://zenodo.org/record/1251296#.Wwbq-CAh1EY">Zenodo</a>.</p>
<p><sup>1</sup>Scheipl S, Barnard M, Cottone L, <em>et al.</em> EGFR inhibitors identified as a potential treatment for chordoma in a focused compound screen. <em>J Pathol</em> 2016; <strong>239</strong>: 320–334.</p>
<p>&nbsp;</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/screening-egfr-and-gak-inhibitors-in-chordoma/feed/</wfw:commentRss>
		<slash:comments>0</slash:comments>
		</item>
		<item>
		<title>Chordoma: A Brief Background</title>
		<link>https://thesgc.github.io/static-openlabnotebooks/choromda-a-brief-background/</link>
		<comments>https://thesgc.github.io/static-openlabnotebooks/choromda-a-brief-background/#comments</comments>
		<pubDate>Tue, 15 May 2018 17:48:27 +0000</pubDate>
		<dc:creator><![CDATA[Kaleb Naegeli]]></dc:creator>
				<category><![CDATA[Catalyst UNC Chapel Hill]]></category>
		<category><![CDATA[Chordoma]]></category>
		<category><![CDATA[Kaleb Naegeli]]></category>

		<guid isPermaLink="false">https://thesgc.github.io/static-openlabnotebooks/?p=1237</guid>
		<description><![CDATA[Greetings! My first post here will be a brief introduction. More in-depth experiment and data posts are coming soon! I am a post-doctoral research associate in the UNC Catalyst for Rare Diseases1. Throughout my education and training, I have been interested in cancer – I worked on epigenetics of ovarian cancer as an undergraduate and <a class="more-link" href="https://thesgc.github.io/static-openlabnotebooks/choromda-a-brief-background/">Read More ...</a>]]></description>
				<content:encoded><![CDATA[<p>Greetings! My first post here will be a brief introduction. More in-depth experiment and data posts are coming soon!</p>
<p>I am a post-doctoral research associate in the <a href="https://pharmacy.unc.edu/research/unc-catalyst-rare-diseases/" target="_blank" rel="noopener">UNC Catalyst for Rare Diseases</a><sup>1</sup>. Throughout my education and training, I have been interested in cancer – I worked on epigenetics of ovarian cancer as an undergraduate and then mechanisms of cell invasion (in a developmental biology system) as a Ph.D. student. Now as a post-doc with the UNC Catalyst for Rare Diseases, my interest in cancer continues in the form of a project on chordoma, a rare bone cancer.</p>
<p>At the recent International Chordoma Research Workshop, the idea of sharing data, reagents, and research progress was raised. Given the affiliation of the UNC Catalyst for Rare Diseases with the open-science SGC, our group seemed like optimal candidates to initiate this undertaking. <strong>Other chordoma researchers: I welcome you to join us! Open science will accelerate the pace of research!<br />
</strong></p>
<p>Chordoma is a rare cancer that occurs along the bones of the central nervous system – from the base of the skull to the tip of the spine. Chordomas arise from remnants of the notochord – a developmental precursor to the vertebral column – and thus are almost always driven by expression of the notochord transcription factor Brachyury (encoded by the gene <em>T</em>). Therapeutic avenues for chordoma are tricky because they are rare (roughly 1 in 10<sup>6</sup> people), slow-growing, and often difficult to resect because of proximity to vital and sensitive structures. Current standard of care is surgical resection followed by radiation, but recurrence, metastasis, and mortality still happens in many cases despite these best practices.</p>
<p>Thus, improved knowledge of the biology of chordoma and the development of new therapeutics are vital for bettering the lives of patients with this rare bone cancer. I have two primary interests in Chordoma which will appear frequently in this blog:</p>
<ul>
<li><strong>Brachyury</strong> – this embryonic transcription factor of the notochord is encoded by the gene <em>T</em>. But T is going to give you way too many Google search results, so most always we’ll stick with Brachyury. This transcription factor is also overexpressed in some other cancers, though its role as a driver in these instances is unclear. While some of the biology of this transcription factor is known – including some of the genes it may regulate – much remains to be learned, including potential chaperones and binding partners. Brachyury presents an intriguing therapeutic avenue for this cancer because of its ubiquitous expression.</li>
<li><strong>EGFR</strong> – this receptor tyrosine kinase is overexpressed (but not mutated) in many chordomas and its ligand, EGF, is a direct transcriptional target of Brachyury. EGFR inhibitors are a potential therapeutic for chordoma, and in collaboration with SGC-UNC, we are working on new inhibitors of EGFR and other kinases as potential chemotherapeutics for chordoma (more information: https://www.chordomafoundation.org/targets/egfr/)</li>
</ul>
<p>With this background out of the way, a backlog of data will be forthcoming.</p>
<p>My work owes many thanks to the Chordoma Foundation for their inspiring array of reagents and data. They are a superb example of how a patient advocacy/disease awareness foundation should be run and how the sharing of reagents and insights can accelerate the progress of research and therapeutic development.</p>
<p>If you are unfamiliar with chordoma, suggested references include the following:</p>
<p><a href="https://www.chordomafoundation.org/understanding-chordoma/" target="_blank" rel="noopener">Chordoma Foundation &#8211; Understanding Chordoma</a><br />
<a href="http://www.thelancet.com/journals/lanonc/article/PIIS1470-2045%2814%2971190-8/abstract">Chordoma Overview (Lancet, 2015)</a><br />
<a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/path.2816" target="_blank" rel="noopener">Role of Brachyury in Chordoma; Presnet et al (2010) J Pathology</a><br />
<a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/path.4082" target="_blank" rel="noopener">The Brachyury Functional Network; Nelson et al (2012) J Pathology</a><br />
<a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/path.4729" target="_blank" rel="noopener">EGFR and EGFR inhibitors in chordoma; Scheipl et al (2016) J Pathology</a></p>
<p><sup>1</sup><strong><a href="https://pharmacy.unc.edu/research/unc-catalyst-rare-diseases/" target="_blank" rel="noopener">The UNC Catalyst for Rare Diseases</a>:</strong> We are a biology group started by the SGC (Structural Genomics Consortium) at the University of North Carolina through funding from the Eshelman Institute for Innovation (in the UNC Eshelman School of Pharmacy). Our commitment and objective is to be a research group that can be a resource for patients and other researchers, a hub with wet lab expertise to do be a source of knowledge in rare diseases. As an “SGC spin-off”, as I like to call our lab, we are committed to a practice of open science. <a href="https://twitter.com/UNC_Catalyst" target="_blank" rel="noopener">You can follow us on Twitter here!</a></p>
<p>&nbsp;</p>
]]></content:encoded>
			<wfw:commentRss>https://thesgc.github.io/static-openlabnotebooks/choromda-a-brief-background/feed/</wfw:commentRss>
		<slash:comments>1</slash:comments>
		</item>
	</channel>
</rss>
